<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-02 06:30:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>22</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>52</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>276</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>237</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Sequencing-based mapping of ensemble pairwise interactions among regulatory elements support the existence of topological assemblies known as promoter-enhancer hubs or cliques in cancer. Yet, prevalence, regulators, and functions of promoter-enhancer hubs in individual cancer cells remain unclear. Here, we systematically integrated functional genomics, transcription factor screening, and optical mapping of promoter-enhancer interactions to identify key promoter-enhancer hubs, examine heterogeneity of their assembly, determine their regulators, and elucidate their role in gene expression control in individual triple negative breast cancer (TNBC) cells. Optical mapping of individual SOX9 and MYC alleles revealed the existence of frequent multiway interactions among promoters and enhancers within spatial hubs. Our single-allele studies further demonstrated that lineage-determining SOX9 and signaling-dependent NOTCH1 transcription factors compact MYC and SOX9 hubs. Together, our findings suggest that promoter-enhancer hubs are dynamic and heterogeneous topological assemblies, which are controlled by oncogenic transcription factors and facilitate subtype-restricted gene expression in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8df926269367eb0273d95bc28b249a4b8bb66d" target='_blank'>
              Oncogenic transcription factors instruct promoter-enhancer hubs in individual triple negative breast cancer cells
              </a>
            </td>
          <td>
            Jingru Zhao, Yeqiao Zhou, Ilias Tzelepis, Noah G Burget, Junwei Shi, R. Faryabi
          </td>
          <td>2024-08-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Significance Metastatic tumor cells must cope with the compression forces generated by interstitial migration and with the visco-elastic differences of various tissue microenvironments. Our study reveals that mechanical stress generated by confined migration overrides the ATR, ATM, and spindle assembly checkpoint (SAC) responses that protect nuclear and chromosome integrity. Thus, tumor cells migrating across constrictions exhibit aberrant chromosome segregation, chromosome instability, and accumulation of extra- chromosomal DNA circles. Confined migration also leads to amplification of the cMYC locus. Hence, mechanical stress has a causative role in generating cancer aneuploidy, genome instability, and oncogenic events, possibly explaining why cancer aneuploidy often does not correlate with mutations in the SAC genes and why c-MYC amplifications are strongly linked to metastatic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca38a8ab95f9ad79f4a0c6ed3aa7e9ffc62776bb" target='_blank'>
              Mechanical stress during confined migration causes aberrant mitoses and c-MYC amplification
              </a>
            </td>
          <td>
            G. Bastianello, G. Kidiyoor, Conor Lowndes, Qingsen Li, R. Bonnal, Jeffrey Godwin, Fabio Iannelli, Lorenzo Drufuca, Ramona Bason, Fabrizio Orsenigo, D. Parazzoli, Mattia Pavani, V. Cancila, Stefano Piccolo, G. Scita, Andrea Ciliberto, C. Tripodo, Massimiliano Pagani, Marco Foiani
          </td>
          <td>2024-07-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dec3a33e2b0139c6a02f10afa4c9e80abbaad7a" target='_blank'>
              Extrachromosomal circular DNA promotes prostate cancer progression through the FAM84B/CDKN1B/MYC/WWP1 axis
              </a>
            </td>
          <td>
            Wei Jin, Zhenqun Xu, Yan Song, Fangjie Chen
          </td>
          <td>2024-07-12</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V. Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="Interactions between multiple genes or cis-regulatory elements (CREs) underlie a wide range of biological processes in both health and disease. High-throughput screens using dCas9 fused to epigenome editing domains have allowed researchers to assess the impact of activation or repression of both coding and non-coding genomic regions on a phenotype of interest, but assessment of genetic interactions between those elements has been limited to pairs. Here, we combine a hyper-efficient version of Lachnospiraceae bacterium dCas12a (dHyperLbCas12a) with RNA Polymerase II expression of long CRISPR RNA (crRNA) arrays to enable efficient highly-multiplexed epigenome editing. We demonstrate that this system is compatible with several activation and repression domains, including the P300 histone acetyltransferase domain and SIN3A interacting domain (SID). We also show that the dCas12a platform can perform simultaneous activation and repression using a single crRNA array via co-expression of multiple dCas12a orthologues. Lastly, demonstrate that the dCas12a system is highly effective for high-throughput screens. We use dHyperLbCas12a-KRAB and a ∼19,000-member barcoded library of crRNA arrays containing six crRNAs each to dissect the independent and combinatorial contributions of CREs to the dose-dependent control of gene expression at a glucocorticoid-responsive locus. The tools and methods introduced here create new possibilities for highly multiplexed control of gene expression in a wide variety of biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c036dd7ee414b69b8ef2f177d76c80471f4540e3" target='_blank'>
              HyperCas12a enables highly-multiplexed epigenome editing screens
              </a>
            </td>
          <td>
            Schuyler M. Melore, Marisa C. Hamilton, Timothy E. Reddy
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b93e43a1982115416b67edb36296e1d7dd7f71fe" target='_blank'>
              The DNA methylome of pediatric brain tumors appears shaped by structural variation and predicts survival
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2024-08-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W. Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Background Approximately half of all high-grade serous ovarian carcinomas (HGSCs) have a therapeutically targetable defect in the homologous recombination (HR) DNA repair mechanism. While there are genomic and transcriptomic methods, developed for other cancer types, to identify HR deficient (HRD) samples, there are no gene expression-based tools to predict HR repair status in HGSC specifically. We have built the first HGSC-specific model to predict HR repair status using gene expression. Methods We separated The Cancer Genome Atlas (TCGA) cohort of HGSCs (n = 361) into training (n = 288) and testing (n = 73) sets and labelled each case as HRD or HR proficient (HRP) based on the clinical standard for classification, being a score of HRD genomic damage. Using the training set, we performed differential gene expression analysis between HRD and HRP cases. The 2604 significantly differentially expressed genes were then used to tune and train a penalised logistic regression model. Results IdentifiHR is an elastic net penalised logistic regression model that uses the expression of 209 genes to predict HR status in HGSC. These genes capture known regions of HR-specific copy number alteration, which impact gene expression levels, and preserve the genomic damage signal. IdentifiHR has an accuracy of 85% in the TCGA test set and of 91% in an independent cohort of 99 samples, collected from primary tumours before (n = 74/99) and after autopsy (n = 6/99), in addition to ascites (n = 12/99) and normal fallopian tube samples (n = 7/99). Further, IdentifiHR is 84% accurate in pseudobulked single-cell HGSC sequencing from 37 patients and outperforms existing gene expression-based methods to predict HR status, being BRCAness, MutliscaleHRD and expHRD. Conclusions IdentifiHR is an accurate model to predict HR status in HGSC using gene expression alone, that is available as an R package from https://github.com/DavidsonGroup/IdentifiHR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bb043799881c3a8ed9a4b69ffa433b94653148" target='_blank'>
              IdentifiHR: predicting homologous recombination deficiency in high-grade serous ovarian carcinoma using gene expression
              </a>
            </td>
          <td>
            Ashley L. Weir, Samuel C. Lee, Mengbo Li, Chin Wee Tan, S. Ramus, Nadia M. Davidson
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Mitochondria are organelles present in most eukaryotic cells, have their own genome and replicate independently of the nuclear genome. Alterations of mitochondrial DNA (mtDNA) have been shown to be associated with cancer. In particular, each cell has many mtDNA copies, the copy number depends on the cell type and the pathological state of the cell. To understand the role of mtDNA copy number in colorectal cancer (CRC), we quantified mtDNA copy number using real-time PCR method on 115 pairs of cancer tissue samples and 40 blood samples of CRC patients which were provided by hospital K, Military Hospital 103 and 67 blood samples of healthy people were used as controls. The results showed that, on tissue samples, most patients had mtDNA copy numbers in the range of 500-1500. The mtDNA copy number in the tissue sample of colon cancer patients (median: 1487.90) was higher than that of rectal cancer patients (median: 694.84) (p<0.05). The mtDNA copy number tends to increase with age, women are higher than men, and decreases with lymph node metastasis and cancer stage. The mtDNA copy number of blood samples from cancer patients is 3.4 times higher than that of blood samples from healthy people (p<0.05). The correlation of mtDNA copy number between tissue and blood samples from the same cancer patient is weak.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b8c83c3a98ecb964dca6742fc3062e9287e505" target='_blank'>
              Mitochondrial DNA Copy Number in Colorectal Cancer in Vietnamese Patients
              </a>
            </td>
          <td>
            P. Bich, T. Thái
          </td>
          <td>2024-08-30</td>
          <td>VNU Journal of Science: Natural Sciences and Technology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin within the nucleus is crucial for gene regulation. However, the 3D architectural features that coordinate the activation of an entire chromosome remain largely unknown. We introduce an omics method, RNA-associated chromatin DNA-DNA interactions, that integrates RNA polymerase II (RNAPII)–mediated regulome with stochastic optical reconstruction microscopy to investigate the landscape of noncoding RNA roX2-associated chromatin topology for gene equalization to achieve dosage compensation. Our findings reveal that roX2 anchors to the target gene transcription end sites (TESs) and spreads in a distinctive boot-shaped configuration, promoting a more open chromatin state for hyperactivation. Furthermore, roX2 arches TES to transcription start sites to enhance transcriptional loops, potentially facilitating RNAPII convoying and connecting proximal promoter-promoter transcriptional hubs for synergistic gene regulation. These TESs cluster as roX2 compartments, surrounded by inactive domains for coactivation of multiple genes within the roX2 territory. In addition, roX2 structures gradually form and scaffold for stepwise coactivation in dosage compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc0d29c3125cfd0f353817880f4b6081993ce9" target='_blank'>
              3D chromatin structures associated with ncRNA roX2 for hyperactivation and coactivation across the entire X chromosome
              </a>
            </td>
          <td>
            S. Tian, Yang Yang, Duo Ning, Ke Fang, Kai Jing, Guangyu Huang, Yewen Xu, Pengfei Yin, Haibo Huang, Gengzhan Chen, Yuqing Deng, Shaohong Zhang, Zhimin Zhang, Zhen-Xia Chen, Tong Gao, Wei Chen, Guoliang Li, Ruilin Tian, Yijun Ruan, Yiming Li, Meizhen Zheng
          </td>
          <td>2024-07-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There are two key signatures of pediatric cancers: (a) higher prevalence of germline alterations and (b) heterogeneity in alteration types. Recent population-based assessments have demonstrated that children with birth defects (BDs) are more likely to develop cancer even without chromosomal anomalies; therefore, explorations of genetic alterations in children with BDs and cancers could provide new insights into the underlying mechanisms for pediatric tumor development. We performed whole-genome sequencing (WGS) on blood-derived DNA for 1556 individuals without chromosomal anomalies, including 454 BD probands with at least one type of malignant tumor, 757 cancer-free children with BDs, and 345 healthy individuals, focusing on copy number variation (CNV) analysis. Roughly half of the children with BD-cancer have CNVs that are not identified in BD-only/healthy individuals, and CNVs are not evenly distributed among these patients. Strong heterogeneity was observed, with a limited number of cancer predisposition genes containing CNVs in more than three patients. Moreover, functional enrichments of genes with CNVs showed that dozens of patients have variations related to the same biological pathways, such as deletions of genes with neurological functions and duplications of immune response genes. Phenotype clustering uncovered recurrences of patients with sarcoma: A notable enrichment was observed involving non-coding RNA regulators, showing strong signals related to growth and cancer regulations in functional analysis. In conclusion, we conducted one of the first genomic studies exploring the impact of CNVs on cancer development in children with BDs, unveiling new insights into the underlying biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0421f07c60f2733c310940e72c92010ab69f33fb" target='_blank'>
              Copy number variations contribute to malignant tumor development in children with serious birth defects.
              </a>
            </td>
          <td>
            Yichuan Liu, J. Glessner, Hui-Qi Qu, X. Chang, H. Qiu, Tiancheng Wang, F. Mentch, H. Hakonarson
          </td>
          <td>2024-08-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The recent paradoxical dual citizenships of Runt-related transcription factor 3 (Runx3) in tumorigenesis remains poorly characterized. Here, we report the oncogenic capacity of Runx3 as chromatin modulator in metastatic gastric cancer model. Runx3 exists as homodimer and binds cooperatively to modified nucleosomes. Additionally, we detected a synergistic functional enhancement of octamer transfer, nucleosome sliding and stochiometric integrity of SWI/SNF by Runx3. We found that Runx3 depletion increased nucleosomes occupancy and promoted chromatin silencing by heterochromatin condensation and HP1 oligomerization. ATAC-seq analysis revealed differential accessibility per chromosome due to Runx3 null expression with dysregulation of multiple inflammatory response and DNA repair pathways. Mechanistically, these modulations resulted in aberrant DNA damage repair response, which is rescued by RUNX3 overexpression. These findings reveal a new paradigm in Runx3 biology as dynamic chromatin regulatory element vital for the maintenances of cancerous phenotype. Significance To the best of our knowledge, the present study is the first to explore the role of Runx3 as homodimeic chromatin binding factor and establishes its oncogenic-function as modulator of heterochromatin de-condensation and SWI/SNF chromatin remodeling activities. These emerging features of Runx3 at the epigenetic level imply a promising direction to screen for anti-Runx3 epigenetic drugs “Epi-drugs” in search of novel gastric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65693fd8ec877e2185ca7654a9454ed2dc3bd82f" target='_blank'>
              Runx3 Acts as Homodimeric Chromatin Binding Factor Regulating Heterochromatin-Mediated Cancerous Phenotype
              </a>
            </td>
          <td>
            Salma Awad Merghani, Isabelle Bonne, Aik Yong Sim, Gehan El-Enain
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="An obligate step in the life cycle of HIV-1 and other retroviruses is the establishment of the provirus in target cell chromosomes. Transcriptional regulation of proviruses is complex, and understanding the mechanisms underlying this regulation has ramifications for fundamental biology, human health, and gene therapy implementation. The three core components of the Human Silencing Hub (HUSH) complex, TASOR, MPHOSPH8 (MPP8), and PPHLN1 (Periphilin 1), were identified in forward genetic screens for host genes that repress provirus expression. Subsequent loss-of-function screens revealed accessory proteins that collaborate with the HUSH complex to silence proviruses in particular contexts. To identify proteins associated with a HUSH complex-repressed provirus in human cells, we developed a technique, Provirus Proximal Proteomics, based on proximity labeling with C-BERST (dCas9-APEX2 biotinylation at genomic elements by restricted spatial tagging). Our screen exploited a lentiviral reporter that is silenced by the HUSH complex in a manner that is independent of the integration site in chromatin. Our data reveal that proviruses silenced by the HUSH complex are associated with DNA repair, mRNA processing, and transcriptional silencing proteins, including L3MBTL2, a member of the non-canonical polycomb repressive complex 1.6 (PRC1.6). A forward genetic screen confirmed that PRC1.6 components L3MBTL2 and MGA contribute to HUSH complex-mediated silencing. PRC1.6 was then shown to silence HUSH-sensitive proviruses in a promoter-specific manner. Genome wide profiling showed striking colocalization of the PRC1.6 and HUSH complexes on chromatin, primarily at sites of active promoters. Finally, PRC1.6 binding at a subset of genes that are silenced by the HUSH complex was dependent on the core HUSH complex component MPP8. These studies offer new tools with great potential for studying the transcriptional regulation of proviruses and reveal crosstalk between the HUSH complex and PRC1.6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6151d660aa506b4ad3a4c66d40fa26e99798e8" target='_blank'>
              PRC1.6 localizes on chromatin with the human silencing hub (HUSH) complex for promoter-specific silencing
              </a>
            </td>
          <td>
            Tom S Rodriguez, Leonid A Yurkovetskiy, Karthika Nagalekshmi, Chin Hung Oscar Lam, Eva Jazbec, Stacy A. Maitland, S. Wolfe, E. Sontheimer, Jeremy Luban
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer molecularly characterized by arrested myogenesis. The defining genetic driver, PAX3::FOXO1, functions as a chimeric gain-of-function transcription factor. An incomplete understanding of PAX3::FOXO1’s in vivo epigenetic mechanisms has hindered therapeutic development. Here, we establish a PAX3::FOXO1 zebrafish injection model and semi-automated ChIP-seq normalization strategy to evaluate how PAX3::FOXO1 initially interfaces with chromatin in a developmental context. We investigated PAX3::FOXO1’s recognition of chromatin and subsequent transcriptional consequences. We find that PAX3::FOXO1 interacts with inaccessible chromatin through partial/homeobox motif recognition consistent with pioneering activity. However, PAX3::FOXO1-genome binding through a composite paired-box/homeobox motif alters chromatin accessibility and redistributes H3K27ac to activate neural transcriptional programs. We uncover neural signatures that are highly representative of clinical rhabdomyosarcoma gene expression programs that are enriched following chemotherapy. Overall, we identify partial/homeobox motif recognition as a new mode for PAX3::FOXO1 pioneer function and identify neural signatures as a potentially critical PAX3::FOXO1 tumor initiation event.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34ac92eb278acb6b0ddaf9def7a0af29156e864" target='_blank'>
              Rhabdomyosarcoma fusion oncoprotein initially pioneers a neural signature in vivo
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, C. Taslim, Meng Wang, Matthew V. Cannon, Katherine M. Silvius, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c780b28cafd38289d7c874a5207cf0360a5c59d4" target='_blank'>
              Co-inheritance of recombined chromatids maintains heterozygosity in a parthenogenetic ant
              </a>
            </td>
          <td>
            Kip D. Lacy, Taylor Hart, D. Kronauer
          </td>
          <td>2024-07-16</td>
          <td>Nature Ecology & Evolution</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract DNA Double-strand breaks (DSBs) are harmful lesions and major sources of genomic instability. Studies have suggested that DSBs induce local transcriptional silencing that consequently promotes genomic stability. Several factors have been proposed to actively participate in this process, including ATM and Polycomb repressive complex 1 (PRC1). Here we found that disrupting PRC1 clustering disrupts DSB-induced gene silencing. Interactome analysis of PHC2, a PRC1 subunit that promotes the formation of the Polycomb body, found several nucleoporins that constitute the Nuclear Pore Complex (NPC). Similar to PHC2, depleting the nucleoporins also disrupted the DSB-induced gene silencing. We found that some of these nucleoporins, such as NUP107 and NUP43, which are members of the Y-complex of NPC, localize to DSB sites. These nucleoporin-enriched DSBs were distant from the nuclear periphery. The presence of nucleoporins and PHC2 at DSB regions were inter-dependent, suggesting that they act cooperatively in the DSB-induced gene silencing. We further found two structural components within NUP107 to be necessary for the transcriptional repression at DSBs: ATM/ATR-mediated phosphorylation at Serine37 residue within the N-terminal disordered tail, and the NUP133-binding surface at the C-terminus. These results provide a new functional interplay among nucleoporins, ATM and the Polycomb proteins in the DSB metabolism, and underscore their emerging roles in genome stability maintenance. *Hongseon Song, Yubin Bae, Sangin Kim, and Dante Deascanis contributed equally to this work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b0e95cd1e0dcce638bc5888df3682c199f2d24" target='_blank'>
              Nucleoporins cooperate with Polycomb silencers to promote transcriptional repression and repair at DNA double strand breaks
              </a>
            </td>
          <td>
            Hongseon Song*, Yubin Bae*, Sangin Kim, Dante Deascanis*, Yujin Lee, G. Róna, Ethan Lane, Seoyeong Lee, Sujung Kim, Michele Pagano, Kyungjae Myung, Y. Kee
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b28bde62e7c6ec79bde107b1d678f6c8aab03e" target='_blank'>
              MethNet: a robust approach to identify regulatory hubs and their distal targets from cancer data
              </a>
            </td>
          <td>
            Theodore Sakellaropoulos, Catherine Do, Guimei Jiang, Giulia Cova, Peter Meyn, D. Dimartino, Sitharam Ramaswami, Adriana Heguy, A. Tsirigos, Jane A Skok
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome integrity is safeguarded by the DNA damage response (DDR). Controlled transcriptional dampening of genes surrounding DNA double-strand breaks (DSBs) has been shown to facilitate DNA repair. This phenomenon, defined as DSB-induced silencing in cis (DISC), involves the DDR apical kinase ATM and the Polycomb Repressive Complex 1 (PRC1). Conversely, DSBs have also been reported to induce de novo transcription of damaged-induced long non-coding RNAs (dilncRNAs) in a MRE11-RAD50-NBS1 (MNR) complex-dependent manner. MRN also controls the recruitment to DSB of the ribonuclease DROSHA, which together with DICER, stimulates DDR signaling and DNA repair. Here, we reconcile these apparently contrasting observations by showing that dilncRNA, together with DROSHA and DICER, but not GW182-like proteins required for miRNA-mediated gene silencing, controls DISC. Indeed, similarly to ATM, MRN inhibition abolishes DISC while pharmacological enhancement of DICER ribonuclease activity by Enoxacin improves DISC. Importantly, Enoxacin administration restores DISC upon ATM inhibition, demonstrating that DICER promotes DISC independently from ATM. Differently, Enoxacin does not restore DISC upon MRN inhibition, suggesting that DICER acts downstream to dilncRNA biogenesis and DROSHA recruitment. Mechanistically, we show that DROSHA and DICER control the recruitment of the PRC1 component BMI1 at DSBs and the consequent H2A-K119 ubiquitination. Upon DSBs formation, BMI1 and DROSHA interact in an RNA-dependent manner. Indeed, BMI1 associates to dilncRNA and do so in a DROSHA- and DICER-dependent manner. Importantly, inhibition of dilncRNA function by antisense oligonucleotides or Cas13-mediated targeting is sufficient to reduce BMI1 recruitment and DISC at individual loci. We propose that dilncRNAs together with DROSHA and DICER control DISC at genomic DSB by supporting PRC1 recruitment and chromatin ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbe46331f4c7eaaba94ede95e2ee9569f12ce8df" target='_blank'>
              DROSHA, DICER and Damage-Induced long ncRNA control BMI1-dependent transcriptional repression at DNA double-strand break
              </a>
            </td>
          <td>
            Francesca Esposito, Ilaria Capozzo, Stefania Modafferi, Ubaldo Gioia, Letizia Manfredi, F. Iannelli, Alessio Colantoni, Adelaide Riccardi, Alessia di Lillo, Sara Tavella, Matteo Cabrini, F. d’Adda di Fagagna, S. Francia
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="After egg fertilization, an initially silent embryonic genome is transcriptionally activated during the maternal-to-zygotic transition. In zebrafish, maternal vertebrate pluripotency factors Nanog, Pou5f3 (OCT4 homolog), and Sox19b (SOX2 homolog) (NPS) play essential roles in orchestrating embryonic genome activation, acting as “pioneers” that open condensed chromatin and mediate acquisition of activating histone modifications. However, some embryonic gene transcription still occurs in the absence of these factors, suggesting the existence of other mechanisms regulating genome activation. To identify chromatin signatures of these unknown pathways, we profiled the histone modification landscape of zebrafish embryos using CUT&RUN. Our regulatory map revealed two subclasses of enhancers distinguished by presence or absence of H3K4me2. Enhancers lacking H3K4me2 tend to require NPS factors for de novo activation, while enhancers bearing H3K4me2 are epigenetically bookmarked by DNA hypomethylation to recapitulate gamete activity in the embryo, independent of NPS pioneering. Thus, parallel enhancer activation pathways combine to induce transcriptional reprogramming to pluripotency in the early embryo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/915e897ed23cf5b7d63f2ee24fa9d133d64c79b4" target='_blank'>
              H3K4me2 distinguishes a distinct class of enhancers during the maternal-to-zygotic transition
              </a>
            </td>
          <td>
            Matthew D. Hurton, Jennifer M. Miller, Miler T. Lee
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c429a219f0b035a7a915d9f1d9d1a5ce65fcffe" target='_blank'>
              DKK1-SE recruits AP1 to activate the target gene DKK1 thereby promoting pancreatic cancer progression
              </a>
            </td>
          <td>
            Lan Shao, Haoran Yu, Mengyun Wang, Lu Chen, Boshu Ji, Tong Wu, Xiangqi Teng, Mu Su, Xiao Han, Weikai Shi, Xin Hu, Ziwen Wang, Hongjuan He, Guiping Han, Yan Zhang, Qiong Wu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a139b2492c263398f29fd9ba2ccbb0286a9c476b" target='_blank'>
              The Ty1 retrotransposon harbors a DNA region that performs dual functions as both a gene silencing and chromatin insulator
              </a>
            </td>
          <td>
            Hiroshi Masumoto, Hideki Muto, Koichi Yano, Yohei Kurosaki, Hironori Niki
          </td>
          <td>2024-07-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc75014daa55260f9205d28f8165ad0ad389192e" target='_blank'>
              Genetic hallmarks and clinical implications of chromothripsis in childhood T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            A. Pastorczak, Z. Urbańska, Borys Styka, Karolina Miarka-Walczyk, L. Sędek, K. Wypyszczak, A. Wakulinska, Zuzanna Nowicka, Tomasz Szczepański, Marcin Stańczak, Wojciech Fendler, Jerzy Kowalczyk, W. Młynarski, Monika Lejman
          </td>
          <td>2024-08-27</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="A key step of Canonical Nonhomologous End Joining (C-NHEJ) is synapsis of DNA double strand break (DSB) ends for ligation. The DNA-PKcs dimer mediates synapsis in a long-range complex with DSB ends remaining apart, whereas the XLF homodimer can mediate synapsis in both long-range and short-range complexes. Recent structural studies found the PAXX homodimer may also facilitate synapsis in long-range complexes with DNA-PKcs via its interactions with Ku70. Thus, we examined the influence of PAXX in C-NHEJ of chromosomal DSBs, which we compared to another Ku-binding factor, MRI. Using EJ of blunt DSBs with Cas9 reporters as a readout for C-NHEJ, we found that PAXX and/or MRI are dispensable. However, when combined with disruption of DNA-PKcs, particularly with DNA-PKcs kinase inhibition, PAXX becomes important for blunt DSB EJ. In contrast, while DNA-PKcs is also important to suppress short deletion mutations with microhomology, this effect is not magnified with PAXX loss. MRI loss had no effect combined with DNA-PKcs disruption, but becomes important for blunt DSB EJ when combined with disruption of XLF, as is PAXX. Finally, XLF loss causes an increase in larger deletions compared to DNA-PKcs inhibition, which is magnified with combined loss of MRI. Altogether, we suggest that PAXX promotes DSB end synapsis during C-NHEJ in a manner that is partially redundant with DNA-PKcs and XLF, whereas MRI appears to be mainly important in the context of XLF disruption. SIGNIFICANCE STATEMENT Canonical Nonhomologous End Joining (C-NHEJ) is a DNA double strand break repair pathway that can repair diverse types of DNA ends, but is prone to causing genetic mutations. In particular, C-NHEJ can cause insertion or deletion mutations that can cause human disease. Therefore, it is critical to understand the mechanisms of C-NHEJ, including and how various factors within the pathway affect repair outcomes. In this study, we elucidate the role of factors PAXX and MRI during chromosomal EJ and their interplay with the C-NHEJ factors DNA-PKcs and XLF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6710460a50993f582d4fe63d4fd0ff8229a07e" target='_blank'>
              Distinct functions of PAXX and MRI during chromosomal end joining
              </a>
            </td>
          <td>
            Metztli Cisneros-Aguirre, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Ruby Chen, Jeremy M. Stark
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary High levels of the transcription factor Grainyhead-like protein 2 (GRHL2) contribute to worse outcomes for patients with breast cancer tumors that express estrogen receptor (ER). Using multiple methods to increase GRHL2 expression in breast cancer cells, we found that high GRHL2 promotes both epithelial and mesenchymal phenotypes. This is indicative of an intermediate state between epithelial and mesenchymal cells, which is associated with metastasis. We also observed that elevated GRHL2 stimulates properties characteristic of cellular plasticity, including a decrease in cellular proliferation with a reciprocal increase in dormancy and stem cell markers. Elevated levels of GRHL2 broadly changed its control of gene expression at the level of DNA and shifted nearby transcription factor binding sites from those associated with development to those associated with disease progression. These results provide a possible explanation for how elevated GRHL2 levels could contribute to poorer prognosis in ER-positive breast cancer. Abstract Estrogen receptor (ER)-positive breast cancer is characterized by late recurrences following initial treatment. The epithelial cell fate transcription factor Grainyhead-like protein 2 (GRHL2) is overexpressed in ER-positive breast cancers and is linked to poorer prognosis as compared to ER-negative breast cancers. To understand how GRHL2 contributes to progression, GRHL2 was overexpressed in ER-positive cells. We demonstrated that elevated GRHL2 imparts plasticity with stem cell- and dormancy-associated traits. RNA sequencing and immunocytochemistry revealed that high GRHL2 not only strengthens the epithelial identity but supports a hybrid epithelial to mesenchymal transition (EMT). Proliferation and tumor studies exhibited a decrease in growth and an upregulation of dormancy markers, such as NR2F1 and CDKN1B. Mammosphere assays and flow cytometry revealed enrichment of stem cell markers CD44 and ALDH1, and increased self-renewal capacity. Cistrome analyses revealed a change in transcription factor motifs near GRHL2 sites from developmental factors to those associated with disease progression. Together, these data support the idea that the plasticity and properties induced by elevated GRHL2 may provide a selective advantage to explain the association between GRHL2 and breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c72323ad8abaa131c1ae72b9d382e27082cefdf" target='_blank'>
              Elevated GRHL2 Imparts Plasticity in ER-Positive Breast Cancer Cells
              </a>
            </td>
          <td>
            Christy Zheng, Kaelyn O. Allen, Tianrui Liu, Natalia M. Solodin, Mark B Meyer, Kelley Salem, Phillipos K. Tsourkas, Sean J. McIlwain, Jessica M. Vera, Erika R. Cromwell, M. S. Ozers, Amy M. Fowler, E. Alarid
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The AurkA serine/threonine kinase is a key regulator of cell division controlling mitotic entry, centrosome maturation, and chromosome segregation. The microtubule-associated protein TPX2 controls spindle assembly and is the main AurkA regulator, contributing to AurkA activation, localisation, and stabilisation. Since their identification, AurkA and TPX2 have been described as being overexpressed in cancer, with a significant correlation with highly proliferative and aneuploid tumours. Despite the frequent occurrence of AurkA/TPX2 co-overexpression in cancer, the investigation of their involvement in tumorigenesis and cancer therapy resistance mostly arises from studies focusing only on one at the time. Here, we review the existing literature and discuss the mitotic phenotypes described under conditions of AurkA, TPX2, or AurkA/TPX2 overexpression, to build a picture that may help clarify their oncogenic potential through the induction of chromosome instability. We highlight the relevance of the AurkA/TPX2 complex as an oncogenic unit, based on which we discuss recent strategies under development that aim at disrupting the complex as a promising therapeutic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1887c3163d0fe8ceb6fc5dcedbb98394ccd0c097" target='_blank'>
              Contribution of AurkA/TPX2 Overexpression to Chromosomal Imbalances and Cancer
              </a>
            </td>
          <td>
            Federica Polverino, Anna Mastrangelo, Giulia Guarguaglini
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Three-stranded DNA-RNA structures known as R-loops that form during papillomavirus transcription can cause transcription-replication conflicts and lead to DNA damage. We found that R-loops accumulated at the viral early promoter in human papillomavirus (HPV) episomal cells but were greatly reduced in cells with integrated HPV genomes. RNA-DNA helicases unwind R-loops and allow for transcription and replication to proceed. Depletion of the RNA-DNA helicase senataxin (SETX) using siRNAs increased the presence of R-loops at the viral early promoter in HPV-31 (CIN612) and HPV-16 (W12) episomal HPV cell lines. Depletion of SETX reduced viral transcripts in episomal HPV cell lines. The viral E2 protein, which binds with high affinity to specific palindromes near the promoter and origin, complexes with SETX, and both SETX and E2 are present at the viral p97 promoter in CIN612 and W12 cells. SETX overexpression increased E2 transcription activity on the p97 promoter. SETX depletion also significantly increased integration of viral genomes in CIN612 cells. Our results demonstrate that SETX resolves viral R-loops to proceed with HPV transcription and prevent genome integration.IMPORTANCEPapillomaviruses contain small circular genomes of approximately 8 kilobase pairs and undergo unidirectional transcription from the sense strand of the viral genome. Co-transcriptional R-loops were recently reported to be present at high levels in cells that maintain episomal HPV and were also detected at the early viral promoter. R-loops can inhibit transcription and DNA replication. The process that removes R-loops from the PV genome and the requisite enzymes are unknown. We propose a model in which the host RNA-DNA helicase senataxin assembles on the HPV genome to resolve R-loops in order to maintain the episomal status of the viral genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3257970da042eeca9c08a9a174f0a9b3a46f14" target='_blank'>
              Senataxin mediates R-loop resolution on HPV episomes.
              </a>
            </td>
          <td>
            L. Jose, Keely Smith, Anaiya Crowner, EJ Androphy, M. DeSmet
          </td>
          <td>2024-07-24</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Senescence —the endpoint of replicative lifespan for normal cells— is established via a complex sequence of molecular events. One such event is the dramatic reorganization of CTCF into senescence-induced clusters (SICCs). However, the molecular determinants, genomic consequences, and functional purpose of SICCs remained unknown. Here, we combine functional assays, super-resolution imaging, and 3D genomics with computational modelling to dissect SICC emergence. We establish that the competition between CTCF-bound and non-bound loci dictates clustering propensity. Upon senescence entry, cells repurpose SRRM2 —a key component of nuclear speckles— and BANF1 —a ‘molecular glue’ for chromosomes— to cluster CTCF and rewire genome architecture. This CTCF-centric reorganization in reference to nuclear speckles functionally sustains the senescence splicing program, as SICC disruption fully reverts alternative splicing patterns. We therefore uncover a new paradigm, whereby cells translate changes in nuclear biochemistry into architectural changes directing splicing choices so as to commit to the fate of senescence. GRAPHICAL ABSTRACT HIGHLIGHTS HMGB2-bound loci compete with CTCF-bound ones for nuclear speckle association Senescent cells repurpose SRRM2 and BANF1 to cluster CTCF on speckles BANF1 is essential, but not sufficient for CTCF clustering The SRRM2 RNA-binding domain directs CTCF clustering SICCs rewire chromatin positioning to sustain the senescence splicing program">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d2e2f24883e3f53705549d5245cfd1c159e708" target='_blank'>
              Senescent cells cluster CTCF on nuclear speckles to sustain their splicing program
              </a>
            </td>
          <td>
            Spiros Palikyras, Vassiliki Varamogiani-Mamatsi, Yajie Zhu, Shyam Ramasamy, Athanasia Mizi, Isabel Liebermann, Athanasia Stavropoulou, Ioanna Papadionysiou, Deniz Bartsch, Yulia Kargapolova, K. Sofiadis, Christoforos Nikolaou, Leo Kurian, A. M. Oudelaar, Mariano Barbieri, A. Papantonis
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Structural variants (SVs) are important contributors to human disease. Their characterization remains however difficult due to their size and association with repetitive regions. Long-read sequencing (LRS) and optical genome mapping (OGM) can aid as their molecules span multiple kilobases and capture SVs in full. In this study, we selected six individuals who presented with unresolved SVs. We applied LRS onto all individuals and OGM to a subset of three complex cases. LRS detected and fully resolved the interrogated SV in all samples. This enabled a precise molecular diagnosis in two individuals. Overall, LRS identified 100% of the junctions at single-basepair level, providing valuable insights into their formation mechanisms without need for additional data sources. Application of OGM added straightforward variant phasing, aiding in the unravelment of complex rearrangements. These results highlight the potential of LRS and OGM as follow-up molecular tests for complete SV characterization. We show that they can assess clinically relevant structural variation at unprecedented resolution. Additionally, they detect (complex) cryptic rearrangements missed by conventional methods. This ultimately leads to an increased diagnostic yield, emphasizing their added benefit in a diagnostic setting. To aid their rapid adoption, we provide detailed laboratory and bioinformatics workflows in this manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d491356850d1dca650c1719115928a885707ff0" target='_blank'>
              Full characterization of unresolved structural variation through long-read sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Griet De Clercq, Lies Vantomme, Barbara Dewaele, Bert Callewaert, Olivier Vanakker, Sandra Janssens, Bart Loeys, Mojca Strazisar, W. D. Coster, J. Vermeesch, A. Dheedene, B. Menten
          </td>
          <td>2024-07-19</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Sex-chromosome dosage represents a challenge for heterogametic species to maintain correct proportion of gene products across chromosomes in each sex. While therian mammals (XX/XY system) achieve near-perfect balance of X-chromosome mRNAs through X-upregulation and X-inactivation, birds (ZW/ZZ system) have been found to lack efficient compensation at RNA level, challenging the necessity of resolving major gene-dosage discrepancies in avian cells. Through allele-resolved multiome analyses, we comprehensively examined dosage compensation in female (ZW), male (ZZ), and rare intersex (ZZW) chicken. Remarkably, this revealed that females exhibit upregulation of their single Z through increased transcriptional burst frequency similar to mammalian X-upregulation, and that Z-protein levels are further balanced via enhanced translation efficiency in females. Global analyses of transcriptional kinetics elements in birds demonstrate remarkable conservation of the genomic encoding of burst kinetics between mammals and birds. Our study uncovers new mechanisms for achieving sex-chromosome dosage compensation and highlights the importance of gene-dosage balance across diverse species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f4b40e70c2988331617fc51e819b31adbb3451" target='_blank'>
              Multi-layered dosage compensation of the avian Z chromosome
              </a>
            </td>
          <td>
            Natali Papanicolaou, Antonio Lentini, Sebastian Wettersten, Michael Hagemann-Jensen, Annika Krüger, Jilin Zhang, Christos Coucoravas, Ioannis Petrosian, Xian Xin, Ilhan Ceyhan, Joanna Rorbach, D. Wright, B. Reinius
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Higher levels of aneuploidy, characterized by imbalanced chromosome numbers, are associated with lethal progression in prostate cancer. However, how aneuploidy contributes to prostate cancer aggressiveness remains poorly understood. In this study, we assessed in patients which genes on chromosome 8q, one of the most frequently gained chromosome arms in prostate tumors, were most strongly associated with long-term risk of cancer progression to metastases and death from prostate cancer (lethal disease) in 403 patients and found the strongest candidate was cohesin subunit gene, RAD21, with an odds ratio of 3.7 (95% CI 1.8, 7.6) comparing the highest vs. lowest tertiles of mRNA expression and adjusting for overall aneuploidy burden and Gleason score, both strong prognostic factors in primary prostate cancer. Studying prostate cancer driven by the TMPRSS2-ERG oncogenic fusion, found in about half of all prostate tumors, we found that increased RAD21 alleviated toxic oncogenic stress and DNA damage caused by oncogene expression. Data from both organoids and patients indicate that increased RAD21 thereby enables aggressive tumors to sustain tumor proliferation, and more broadly suggests one path through which tumors benefit from aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f061db09f8cb87400e631c0dc03d0078eb8a3c" target='_blank'>
              RAD21 promotes oncogenesis and lethal progression of prostate cancer.
              </a>
            </td>
          <td>
            Xiaofeng A Su, Konrad H. Stopsack, Daniel R. Schmidt, Duanduan Ma, Zhe Li, P. Scheet, K. Penney, T. Lotan, W. Abida, Elise G. DeArment, Kate Lu, Thomas Janas, Sofia Hu, M. V. Vander Heiden, Massimo Loda, Monica G. Boselli, Angelika Amon, Lorelei Mucci
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="A genome-wide chromatin profiling technology, named as Nicking Enzyme Epitope targeted DNA sequencing (NEED-seq) in which antibody-targeted controlled nicking by Nt.CviPII-pGL is used to study specific protein-DNA complexes. NEED-seq is performed in situ in formaldehyde fixed cells, allowing for both visual and genomic resolution of epitope bound chromatin. When applied to nuclei, NEED-seq yielded genome-wide chromatin associated proteins and histone post-translational modifications (PTMs). NEED-seq of lamin B1/B2 demonstrated their association with heterochromatin. Lamin B1 and B2 associated domains (LAD) segregated to three different states, and states with stronger LAD correlated with heterochromatic marks. Hi-C analysis displayed A and B compartment with equal lamin B1/B2 distribution, although methylated DNA remained high in B compartment. LAD clustering with Hi-C resulted in subcompartments, with lamin B1-B2 partitioning to facultative and constitutive heterochromatin respectively and were associated with neuronal development. Thus, lamin B1 and B2 have structural and functional partitioning in mammalian nucleus. Figure 1: Graphical abstract: Model depicting association of lamin B1 and B2 in A (facultative heterochromatin) and B (constitutive heterochromatin) compartmentalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3c50fd768031283d19ccee2e6223fbeee2130c" target='_blank'>
              Distinct structural and functional heterochromatin partitioning of lamin B1 and B2 revealed using genome-wide Nicking Enzyme Epitope targeted DNA sequencing
              </a>
            </td>
          <td>
            Sagnik Sen, P. Estève, Karthikeyan Raman, Julie Beaulieu, H. Chin, G. R. Feehery, U. Vishnu, Shuang-yong Xu, James C. Samuelson, S. Pradhan
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="The Y-linked gene DDX3Y and its X-linked homolog DDX3X survived the evolution of the human sex chromosomes from ordinary autosomes. DDX3X encodes a multi-functional RNA helicase, with mutations causing developmental disorders and cancers. We find that, among X-linked genes with surviving Y homologs, DDX3X is extraordinarily dosage-sensitive. Studying cells of individuals with sex chromosome aneuploidy, we observe that when the number of Y chromosomes increases, DDX3X transcript levels fall; conversely, when the number of X chromosomes increases, DDX3Y transcript levels fall. In 46,XY cells, CRISPRi knockdown of either DDX3X or DDX3Y causes transcript levels of the homologous gene to rise. In 46,XX cells, chemical inhibition of DDX3X protein activity elicits an increase in DDX3X transcript levels. Thus, perturbation of either DDX3X or DDX3Y expression is buffered – by negative cross-regulation of DDX3X and DDX3Y in 46,XY cells, and by negative auto-regulation of DDX3X in 46,XX cells. DDX3X-DDX3Y cross-regulation is mediated through mRNA destabilization – as shown by metabolic labeling of newly transcribed RNA – and buffers total levels of DDX3X and DDX3Y protein in human cells. We infer that post-transcriptional auto-regulation of the ancestral (autosomal) DDX3 gene transmuted into auto- and cross-regulation of DDX3X and DDX3Y as these sex-linked genes evolved from ordinary alleles of their autosomal precursor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21706998708cb42f8ef2259c9b9706f33b1d0c74" target='_blank'>
              Post-transcriptional cross- and auto-regulation buffer expression of the human RNA helicases DDX3X and DDX3Y
              </a>
            </td>
          <td>
            Shruthi Rengarajan, Jason Derks, D. W. Bellott, N. Slavov, David C. Page
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7234a63fd75fc71e91d39632a18b128298ba3fb" target='_blank'>
              Genome-wide identification of copy number variations in wrinkled skin cases of Xiang pigs
              </a>
            </td>
          <td>
            Xiaoli Liu, Fenbin Hu, Wei Wang, Xia Chen, Xi Niu, Shihui Huang, Zhou Wang, Jiafu Wang, Xueqin Ran
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb0c25603cf78f65def294ef7f4472ac1c4808b" target='_blank'>
              Simultaneous de novo calling and phasing of genetic variants at chromosome-scale using NanoStrand-seq
              </a>
            </td>
          <td>
            Xiuzhen Bai, Zonggui Chen, Kexuan Chen, Zixin Wu, Rui Wang, Jun'e Liu, Liang Chang, Lu Wen, Fuchou Tang
          </td>
          <td>2024-07-09</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="During the sexual cycle, programmed genome rearrangement (PGR) in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal Internal Eliminated Sequences (IESs) from the newly developing somatic nucleus. In addition to the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5’-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolX (a,b,c,d), four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, 1-nucleotide deletions and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during PGR and have acquired tight nuclear anchoring properties through their N-terminal region, which contains a predicted BRCT domain. Finally, we show that PolXa accumulates in nuclear foci together with other NHEJ proteins and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these “DNA repair foci” correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of iesRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8a893b216f167d132f42d3e505df3cd9a940a5" target='_blank'>
              A development-specific POLX essential for programmed genome rearrangement in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Heterochromatin is a gene-poor and repeat-rich genomic compartment universally found in eukaryotes. Despite its low transcriptional activity, heterochromatin plays important roles in maintaining genome stability, organizing chromosomes, and suppressing transposable elements (TEs). Given the importance of these functions, it is expected that the genes involved in heterochromatin regulation would be highly conserved. Yet, a handful of these genes were found to evolve rapidly. To investigate whether these previous findings are anecdotal or general to genes modulating heterochromatin, we compile an exhaustive list of 106 candidate genes involved in heterochromatin functions and investigate their evolution over short and long evolutionary time scales in Drosophila. Our analyses find that these genes exhibit significantly more frequent evolutionary changes, both in the forms of amino acid substitutions and gene copy number change, when compared to genes involved in Polycomb-based repressive chromatin. While positive selection drives amino acid changes within both structured domains with diverse functions and intrinsically disordered regions (IDRs), purifying selection may have maintained the proportions of IDRs of these proteins. Together with the observed negative associations between evolutionary rates of these genes and genomic TE abundance, we propose an evolutionary model where the fast evolution of genes involved in heterochromatin functions is an inevitable outcome of the unique functional roles of heterochromatin, while the rapid evolution of TEs may be an effect rather than cause. Our study provides an important global view of the evolution of genes involved in this critical cellular domain and provides insights into the factors driving the distinctive evolution of heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6e4293419d3640277d1a81925a746d1d427525a" target='_blank'>
              Prevalent fast evolution of genes involved in heterochromatin functions.
              </a>
            </td>
          <td>
            Leila Lin, Yuheng Huang, Jennifer McIntyre, Ching-Ho Chang, Serafin U. Colmenares, Y. C. G. Lee
          </td>
          <td>2024-08-27</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Development of the malaria parasite, Plasmodium falciparum, is regulated by a limited number of sequence-specific transcription factors (TFs). However, the mechanisms by which these TFs recognize genome-wide binding sites is largely unknown. To address TF specificity, we investigated the binding of two TF subsets that either bind CACACA or GTGCAC DNA sequence motifs and further characterized two additional ApiAP2 TFs, PfAP2-G and PfAP2-EXP, which bind unique DNA motifs (GTAC and TGCATGCA). We also interrogated the impact of DNA sequence and chromatin context on P. falciparum TF binding by integrating high-throughput in vitro and in vivo binding assays, DNA shape predictions, epigenetic post-translational modifications, and chromatin accessibility. We found that DNA sequence context minimally impacts binding site selection for paralogous CACACA-binding TFs, while chromatin accessibility, epigenetic patterns, co-factor recruitment, and dimerization correlate with differential binding. In contrast, GTGCAC-binding TFs prefer different DNA sequence context in addition to chromatin dynamics. Finally, we determined that TFs that preferentially bind divergent DNA motifs may bind overlapping genomic regions due to low-affinity binding to other sequence motifs. Our results demonstrate that TF binding site selection relies on a combination of DNA sequence and chromatin features, thereby contributing to the complexity of P. falciparum gene regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de1da5fe0ac5a53ba6dea6fb346647650ca1dccf" target='_blank'>
              DNA sequence and chromatin differentiate sequence-specific transcription factor binding in the human malaria parasite Plasmodium falciparum.
              </a>
            </td>
          <td>
            Victoria A. Bonnell, Yuning Zhang, Alan S. Brown, John Horton, G. Josling, Tsu-Pei Chiu, R. Rohs, Shaun Mahony, Raluca Gordân, M. Llinás
          </td>
          <td>2024-07-05</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Neuro-Oncological Ventral Antigen 1 (NOVA1) is best known for its role in mediating an alternative splicing (AS) program in neurons, yet was first discovered as an antigen expressed in breast tumors, causing rare autoimmune reactions and paraneoplastic neurological disorders (PNDs). The PND model suggests a plausible role of the tumor antigen expression in tumor suppression, whereas it has emerged that NOVA may function as an oncogene in a variety of cancers. In addition, whether NOVA mediates AS in breast cancer remains unanswered. Here we examine the AS profiles of breast invasive carcinoma (BRCA) tumor samples and demonstrate that ectopic NOVA1 expression led to the activation of neuron-like splicing patterns in many genes, including exons targeted by NOVA in the brain. The splicing dysregulation is especially prevalent in cell periphery and cytoskeleton genes related to cell-cell communication, actin-based movement, and neuronal functions. We find that NOVA1-mediated AS is most prominent in Luminal A tumors and high NOVA1 expression in this subtype is associated with poorer prognosis. Our results suggest that ectopic NOVA1 in tumors has regulatory activity affecting pathways with high relevance to tumor progression and that this might be a more general mechanism for PND antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e50ff50fd94e0098cbe1b6b46456df93e0c5100" target='_blank'>
              Oncogenic NOVA1 expression dysregulates alternative splicing in breast cancer
              </a>
            </td>
          <td>
            Daniel F. Moakley, Chaolin Zhang
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) leads as a primary cause of cancer-related premature mortality in Western populations. This study leverages cutting-edge gene-expression-profiling technologies to perform an in-depth molecular characterization of NSCLC specimens, with the objective of uncovering tumor-specific genomic alterations. By employing DNA microarray analysis, our research aims to refine the classification of NSCLC for early detection, guide molecular-targeted treatment approaches, enhance prognostication, and broaden the scientific understanding of the disease’s biology. We identified widespread genomic abnormalities in our samples, including the recurrent loss of chromosomal regions 3p, 5q, 13q, and 21q and the gain of 12p. Furthermore, utilizing Metascape for bioinformatic analysis revealed critical biological pathways disrupted in NSCLC, offering promising leads for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489eff9e89a558831da14bfa1d262e700e732bd4" target='_blank'>
              Specific Deletions of Chromosomes 3p, 5q, 13q, and 21q among Patients with G2 Grade of Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Agata Kolecka-Bednarczyk, M. Frydrychowicz, B. Budny, M. Ruciński, Claudia Dompe, Piotr Gabryel, B. Płachno, M. Ruchała, K. Ziemnicka, Paweł Zieliński, Joanna Budna-Tukan
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Cancer stemness plays an important role in cancer initiation and progression, and is the major cause of tumor invasion, metastasis, recurrence, and poor prognosis. Non-coding RNAs (ncRNAs) are a class of RNA transcripts that generally cannot encode proteins and have been demonstrated to play a critical role in regulating cancer stemness. Here, we developed the ncStem database to record manually curated and predicted ncRNAs associated with cancer stemness. In total, ncStem contains 645 experimentally verified entries, including 159 long non-coding RNAs (lncRNAs), 254 microRNAs (miRNAs), 39 circular RNAs (circRNAs), and 5 other ncRNAs. The detailed information of each entry includes the ncRNA name, ncRNA identifier, disease, reference, expression direction, tissue, species, and so on. In addition, ncStem also provides computationally predicted cancer stemness-associated ncRNAs for 33 TCGA cancers, which were prioritized using the random walk with restart (RWR) algorithm based on regulatory and co-expression networks. The total predicted cancer stemness-associated ncRNAs included 11 132 lncRNAs and 972 miRNAs. Moreover, ncStem provides tools for functional enrichment analysis, survival analysis, and cell location interrogation for cancer stemness-associated ncRNAs. In summary, ncStem provides a platform to retrieve cancer stemness-associated ncRNAs, which may facilitate research on cancer stemness and offer potential targets for cancer treatment. Database URL: http://www.nidmarker-db.cn/ncStem/index.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb97313e3402ad277fe895b46bce91bdb6c21f55" target='_blank'>
              ncStem: a comprehensive resource of curated and predicted ncRNAs in cancer stemness
              </a>
            </td>
          <td>
            Hui Liu, Nan Zhang, Yijie Jia, Jun Wang, Aokun Ye, Siru Yang, Honghan Zhou, Yingli Lv, Chaohan Xu, Shuyuan Wang
          </td>
          <td>2024-08-13</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks (SSB) arising from base excision repair (BER) during misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination-deficient (HRD) cells. At templated AP-sites, HMCES DNA-protein crosslinks (DPC) regulate replication fork speed while avoiding APE1-mediated cytotoxic double-strand breaks (DSB). Whereas the role of HMCES at template DNA strand is well studied, its consequences on nascent DNA are less understood. Here, we provide evidence that HMCES play detrimental roles during removal of 5- hydroxymethyl-2’-deoxycytidine (5hmdC)-derived 5-hydroxymethyl-2’-deoxyuridine (5hmdU) by BER at replication forks. HRD cells display heightened HMCES chromatin levels, which increase upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP-sites. HMCES depletion largely suppresses 5hmdC- mediated Fancd2-/- replication fork defects, chromosomal aberrations and cell lethality, suggesting that HMCES is responsible for the replication fork impairment and lethality observed in HRD cells. Therefore, HMCES-DPCs are a novel source of BER-initiated PRIMPOL-mediated ssDNA gaps, implying endogenous DPCs as a source of DNA damage in HRD tumours. Teaser Covalent binding of HMCES to nascent DNA blocks replication progression and kills homologous recombination deficient cancer cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301e05a5b2dbeda5a785b9d3a1c934b12bac03" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martin, Marta del Rio Oliva, Jean-Yves Masson, JC Reyes, Iván V. Rosado
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The Chromodomain helicase DNA-binding protein 1-like (CHD1L) is a nucleosome remodeling enzyme, which plays a key role in chromatin relaxation during the DNA damage response. Genome editing has shown that deletion of CHD1L sensitizes cells to PARPi, but the effect of its pharmacological inhibition has not been defined. Triple-negative breast cancer SUM149PT, HCC1937, and MDA-MB-231 cells were used to assess the mechanism of action of the CHD1Li OTI-611. Cytotoxicity as a single agent or in combination with standard-of-care treatments was assessed in tumor organoids. Immunofluorescence was used to assess the translocation of PAR and AIF to the cytoplasm or the nucleus and to study markers of DNA damage or apoptosis. Trapping of PARP1/2 or CHD1L onto chromatin was also assessed by in situ subcellular fractionation and immunofluorescence and validated by Western blot. We show that the inhibition of CHD1L’s ATPase activity by OTI-611 is cytotoxic to triple-negative breast cancer tumor organoids and synergizes with PARPi and chemotherapy independently of the BRCA mutation status. The inhibition of the remodeling function blocks the phosphorylation of H2AX, traps CHD1L on chromatin, and leaves PAR chains on PARP1/2 open for hydrolysis. PAR hydrolysis traps PARP1/2 at DNA damage sites and mediates PAR translocation to the cytoplasm, release of AIF from the mitochondria, and induction of PARthanatos. The targeted inhibition of CHD1L’s oncogenic function by OTI-611 signifies an innovative therapeutic strategy for breast cancer and other cancers. This approach capitalizes on CHD1L-mediated DNA repair and cell survival vulnerabilities, thereby creating synergy with standard-of-care therapies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b6daf1d2a20088ba1aa72a3182ab37cb9c8e4" target='_blank'>
              CHD1L Regulates Cell Survival in Breast Cancer and Its Inhibition by OTI-611 Impedes the DNA Damage Response and Induces PARthanatos
              </a>
            </td>
          <td>
            Rita Sala, Hector Esquer, Timothy Kellett, Jeffrey T. Kearns, Paul Awolade, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Maintaining genome integrity is an essential and challenging process. RAD51 recombinase, the central player of several crucial processes in repairing and protecting genome integrity, forms filaments on DNA. RAD51 filaments are tightly regulated. One of these regulators is FIGNL1, that prevents persistent RAD51 foci post-damage and genotoxic chromatin association in cells. The cryogenic electron microscopy structure of FIGNL1 in complex with RAD51 reveals that the FIGNL1 forms a non-planar hexamer and RAD51 N-terminus is enclosed in the FIGNL1 hexamer pore. Mutations in pore loop or catalytic residues of FIGNL1 render it defective in filament disassembly and are lethal in mouse embryonic stem cells. Our study reveals a unique mechanism for removing RAD51 from DNA and provides the molecular basis for FIGNL1 in maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b11fcc1a323386baea335e7bd61f5bc4543b03e" target='_blank'>
              Molecular basis of FIGNL1 in dissociating RAD51 from DNA and chromatin
              </a>
            </td>
          <td>
            Alexander Carver, Tai-Yuan Yu, Luke A Yates, Travis White, Raymond Wang, Katie Lister, M. Jasin, Xiaodong Zhang
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a51aa320dccf7a4322f2753df2fe86ac6b9b3a8" target='_blank'>
              Unraveling the three-dimensional (3D) genome architecture in Neurodevelopmental Disorders (NDDs).
              </a>
            </td>
          <td>
            P. Carballo-Pacoret, A. Carracedo, C. Rodriguez-Fontenla
          </td>
          <td>2024-08-27</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, Tengjiang Chen, Jie Lei, Changwen Chen, Shunqin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current Cancer Drug Targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature Plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Gene regulators physically associate to the genome, in a combinatorial fashion, to drive tissue-specific gene expression programs. Uncovering the genome-wide activity of all gene regulators across tissues is therefore needed to understand how the genome is regulated during development. Here, we take a first step forward towards achieving this goal. Using CUT&RUN, we systematically measured the genome-wide binding profiles of key transcription factors and cofactors that mediate the activity of ontogenetically relevant signaling pathways in select mouse tissues at two developmental stages. Computation of the numerous genome-wide binding datasets unveiled a large degree of tissue and time-specific activity for each gene regulator, and several factor-specific idiosyncrasies. Moreover, we identified “popular” regulatory regions that are bound by a multitude of pathway regulators. Popular regions tend to be more evolutionarily conserved, implying their essentiality. Consistently, they lie in the proximity of genes whose dysregulation causes early embryonic lethality in the mouse. Moreover, the human homologs of these regions are also bound by many gene regulators and are highly conserved in human populations, indicating that they retain functional relevance for human development. This work constitutes a decisive step towards understanding how the genome is simultaneously read and used by gene regulators in a holistic fashion and unveils multiple genomic mechanisms that drive embryonic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea44cb9e15a021f4221a0294719b3ece65eeee5" target='_blank'>
              The Transcription Factor Binding Landscape of Mouse Development
              </a>
            </td>
          <td>
            A. Nordin, Gianluca Zambanini, Mattias Jonasson, Tamina Weiss, Yorick Bernardus Cornelis van de Grift, P. Pagella, Claudio Cantù
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non-SMC condensin I complex subunit D2 (NCAPD2) is a newly identified oncogene; however, the specific biological function and molecular mechanism of NCAPD2 in liver cancer progression remain unknown. In the present study, the aberrant expression of NCAPD2 in liver cancer was investigated using public tumor databases, including TNMplot, The Cancer Genome Atlas and the International Cancer Genome Consortium based on bioinformatics analyses, and it was validated using a clinical cohort. It was revealed that NCAPD2 was significantly upregulated in liver cancer tissues compared with in control liver tissues, and NCAPD2 served as an independent prognostic factor and predicted poor prognosis in liver cancer. In addition, the expression of NCAPD2 was positively correlated with the percentage of Ki67+ cells. Finally, single-cell sequencing data, gene-set enrichment analyses and in vitro investigations, including cell proliferation assay, Transwell assay, wound healing assay, cell cycle experiments, cell apoptosis assay and western blotting, were carried out in human liver cancer cell lines to assess the biological mechanisms of NCAPD2 in patients with liver cancer. The results revealed that the upregulation of NCAPD2 enhanced tumor cell proliferation, invasion and cell cycle progression at the G2/M-phase transition, and inhibited apoptosis in liver cancer cells. Furthermore, NCAPD2 overexpression was closely associated with the phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR)/c-Myc signaling pathway and epithelial-mesenchymal transition (EMT) progression in HepG2 and Huh7 cells. In addition, upregulated NCAPD2 was shown to have adverse effects on overall survival and disease-specific survival in liver cancer. In conclusion, the overexpression of NCAPD2 was shown to lead to cell cycle progression at the G2/M-phase transition, activation of the PI3K-Akt-mTOR/c-Myc signaling pathway and EMT progression in human liver cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6873a90e35d5a8f62be3a51b8f0dd6c3ab4c961" target='_blank'>
              NCAPD2 augments the tumorigenesis and progression of human liver cancer via the PI3K-Akt-mTOR signaling pathway
              </a>
            </td>
          <td>
            Jiangxue Gu, Ke Huang, Wei-Lin Zhao, Xiao-Ming Zheng, Yu-Qin Wu, Shirong Yan, Yu-Gang Huang, Pei Hu
          </td>
          <td>2024-07-31</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M. Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The genomic architecture of 22q11.2 Deletion Syndrome (22q11.2DS) has focused on analysis of white genomes. However, Black individuals appear to have a lower prevalence of 22q11.2DS compared to whites. To improve the understanding of different populations in relation to 22q11.2DS, optical mapping data from 106 genomes across various Black and white genomes were used to determine the organization of 22q11.2 genomic structures. This revealed extensive variability between the groups regarding copy number and orientation changes of the elements comprising the 22q11.2 low copy repeats (LCR22s). Several novel CNVs and whole haplotype configurations, private and of different prevalence to each group were detected. The diversity of CNVs within Black genomes compared to white genomes was especially striking. To determine the impact of this variability, Black families with de novo 22q11.2DS probands were compared to white families. The highly variable configurations of Black and white haplotypes led to several unique non-allelic homologous recombination (NAHR) scenarios with recombinations at different loci. In particular, Black families had unique recombinations yet to be observed. Thus, the unique and highly variable haplotype configurations of LCR22s in Black individuals may play a role in their decreased incidence of 22q11.2DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654ffdc07a99700bb41c69f30252e8dc1b2d3481" target='_blank'>
              Optically Mapped Black Genomes: Distinct Structures and 22q11.2 Deletion Syndrome Mechanisms
              </a>
            </td>
          <td>
            Steven Pastor, Oanh Tran, Ryan Lapointe, Arnold Z. Olali, Douglas C. Wallace, Bernice Morrow, E. Zackai, Donna M. McDonald-McGinn, B. Emanuel
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Polyploidization plays a crucial role in plant evolution and is becoming increasingly important in breeding. Structural variations and epigenomic repatterning have been observed in synthetic polyploidizations. However, the mechanisms underlying the occurrence and their effects on gene expression and phenotype remain unknown. Here, we investigated genome-wide large deletion/duplication regions (DelDups) and genomic methylation dynamics in leaf organs of progeny from the first eight generations of synthetic tetraploids derived from Chinese cabbage (Brassica rapa L. ssp. pekinensis) and cabbage (Brassica oleracea L. var. capitata). One- or two-copy DelDups, with a mean size of 5.70 Mb (400 kb - 65.85 Mb), occurred from the first generation of selfing and thereafter. The duplication of a fragment in one subgenome consistently coincided with the deletion of its syntenic fragment in the other subgenome, and vice versa, indicating that these DelDups were generated by homoeologous exchanges (HEs). Interestingly, the larger the genomic syntenic region, the higher the frequency of DelDups, further suggesting that the pairing of large homoeologous fragments is crucial for HEs. Moreover, we found that the active transcription of continuously distributed genes in local regions is positively associated with the occurrence of HE breakpoints. In addition, the expression of genes within DelDups exhibited a dosage effect, and plants with extra parental genomic fragments generally displayed phenotypes biased towards the corresponding parent. Genome-wide methylation fluctuated remarkably, which did not clearly affect gene expression on a large scale. Our findings provide insights into the early evolution of polyploid genomes, offering valuable knowledge for polyploidization-based breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc22507c27b064911f132b6bdaa4330162e63d6f" target='_blank'>
              Regional Active Transcription Associates with Homoeologous Exchange Breakpoints in Synthetic Brassica Tetraploids.
              </a>
            </td>
          <td>
            Chengcheng Cai, Yinqing Yang, Lei Zhang, Yinan Cui, Jian Wu, Jianli Liang, Xing Li, Lingkui Zhang, Xin Zhang, Yiyue Zhang, Zhongwei Guo, Shumin Chen, Kang Zhang, M. Freeling, Xiaowu Wang, Feng Cheng
          </td>
          <td>2024-08-20</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor in adults and is notorious for resistance to treatments. Chemotherapy is largely based on alkylating agents that induce lethal secondary DNA double-strand breaks (DSB). Therefore, modulation of DNA DSB-repair pathways that are involved in the innate or acquired tumor resistance, is frequently under consideration for improved chemotherapy. In the present study, we used a RAD51 inhibitor to modulate the DSB repair pathway homologous recombination (HR) and a PARPi to abolish the PARP-dependent non-homologous end-joining (NHEJ) as a backup DSB-repair mechanism in human glioblastoma cells with functional p53 and PTEN. In cell culture, synthetic lethality-like effects of combined treatment with PARPi and RAD51i were observed as well as increased sensitivity to temozolomide when combined with RAD51i or PARPi. In vivo in the orthotopic model, combination of PARPi and RAD51i significantly increased mouse median survival and reduced the tumorigenic potential of cancer xenoplants. Improved survival correlated with increased DNA damage and apoptotic cell numbers, as well as reduced numbers of proliferating cells in tumor brain sections of mice treated with PARPi and RAD51i. Our results suggest that combining RAD51i with PARPi might be beneficial for glioblastoma treatment and suggests RAD51 inhibition as novel strategy in glioma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064f750891c9080de686323c9c30b6ded299a104" target='_blank'>
              Targeting DNA Repair Pathways in Orthotopic Glioblastoma Xenoplants
              </a>
            </td>
          <td>
            AA Müller, TG Hofmann
          </td>
          <td>2024-07-15</td>
          <td>Journal of Oncology Research and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>154</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs.
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-07-01</td>
          <td>Genome research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis to generate new combinations of variants. Recombination also causes new mutations, affects the GC content of the genome and reduces selective interference. Here, we use HiFi long-read sequencing to directly detect crossover and gene conversion events from switches between the two haplotypes along single HiFi-reads from testis tissue of humans, chimpanzees and gorillas as well as human sperm samples. Furthermore, based on DNA methylation calls, we classify the cellular origin of reads to either somatic or germline cells in the testis tissue. We identify 1692 crossovers and 1032 gene conversions in nine samples and investigate their chromosomal distribution. Crossovers are more telomeric and correlate better with recombination maps than gene conversions. We show a strong concordance between a human double-strand break map and the human samples, but not for the other species, supporting different PRDM9-programmed double-strand break loci. We estimate the average gene conversion tract lengths to be similar and very short in all three species (means 40-100 bp, fitted well by a geometric distribution) and that 95-98% of non-crossover events do not involve tracts intersecting with polymorphism and are therefore not detectable. Finally, we detect a GC bias in the gene conversion of both single and multiple SNVs and show that the GC-biased gene conversion affects SNVs flanking crossover events. This implies that gene conversion events associated with crossover events are much longer (estimated above 500 bp) than those associated with non-crossover events. Highly accurate long-read sequencing combined with the classification of reads to specific cell types provides a new, powerful way to make individual, detailed maps of gene conversion and crossovers for any species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e343940af051e78f3a88674b4f23d5e3391f5fb" target='_blank'>
              Insights into gene conversion and crossing-over processes from long-read sequencing of human, chimpanzee and gorilla testes and sperm
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. B. Winge, C. Hvilsom, Marta Pelizzola, S. Laurentino, N. Neuhaus, A. Hobolth, Thomas Bataillon, Kristian Almstrup, Søren Besenbacher, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4935f35210bf40b045d154a339912e6ddb86073" target='_blank'>
              Decoding high mobility group A2 protein expression regulation and implications in human cancers
              </a>
            </td>
          <td>
            Farah Khazem, A. B. Zetoune
          </td>
          <td>2024-07-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Heterochromatic loci marked by histone H3 lysine 9 dimethylation (H3K9me2) are enriched at the nuclear periphery in metazoans, but the effect of spatial position on heterochromatin function has not been defined. Here, we remove three nuclear lamins and lamin B receptor (LBR) in mouse embryonic stem cells (mESCs) and show that heterochromatin detaches from the nuclear periphery. Mutant mESCs sustain naïve pluripotency and maintain H3K9me2 across the genome but cannot repress H3K9me2-marked genes or transposons. Further, mutant cells fail to differentiate into epiblast-like cells (EpiLCs), a transition that requires the expansion of H3K9me2 across the genome. Mutant EpiLCs can silence naïve pluripotency genes and activate epiblast-stage genes. However, H3K9me2 cannot repress markers of alternative fates, including primitive endoderm. We conclude that the nuclear periphery controls the spatial position, dynamic remodeling, and repressive capacity of H3K9me2-marked heterochromatin to shape cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61fbe8552a0a7b09256167e0e62254d13819ef24" target='_blank'>
              The nuclear periphery confers repression on H3K9me2-marked genes and transposons to shape cell fate
              </a>
            </td>
          <td>
            Harold Marin, Eric Simental, Charlie Allen, Eric Martin, Barbara Panning, Bassem Al-Sady, A. Buchwalter
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The centrosome is the microtubule-organizing center and a crucial part of cell division. Centrosomal RNAs (cnRNAs) have been reported to enable precise spatiotemporal control of gene expression during cell division in many species. Whether and how cnRNAs exist in C. elegans are unclear. Here, using the nuclear RNAi Argonaute protein NRDE-3 as a reporter, we observed potential peri-centrosome localized small interfering (si)RNAs in C. elegans. NRDE-3 was previously shown to associate with pre-mRNAs and pre-rRNAs via a process involving the presence of complementary siRNAs. We generated a GFP-NRDE-3 knock-in transgene through CRISPR/Cas9 technology and observed that NRDE-3 formed peri-centrosomal foci neighboring the tubulin protein TBB-2, other centriole proteins and pericentriolar material (PCM) components in C. elegans embryos. The peri-centrosomal accumulation of NRDE-3 depends on RNA-dependent RNA polymerase (RdRP)-synthesized 22G siRNAs and the PAZ domain of NRDE-3, which is essential for siRNA binding. Mutation of eri-1, ergo-1, or drh-3 significantly increased the percentage of pericentrosome-enriched NRDE-3. At the metaphase of the cell cycle, NRDE-3 was enriched in both the peri-centrosomal region and the spindle. Moreover, the integrity of centriole proteins and pericentriolar material (PCM) components is also required for the peri-centrosomal accumulation of NRDE-3. Therefore, we concluded that siRNAs could accumulate in the peri-centrosomal region in C. elegans and suggested that the peri-centrosomal region may also be a platform for RNAi-mediated gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65bde153af05e00aebd6944ec9c41c84ee6f36d" target='_blank'>
              Peri-centrosomal localization of small interfering RNAs in C. elegans
              </a>
            </td>
          <td>
            Qile Jin, Xuezhu Feng, Minjie Hong, Ke Wang, Xiangyang Chen, Jiewei Cheng, Yan Kuang, Xiaoyue Si, Mingjing Xu, Xinya Huang, Shouhong Guang, Chengming Zhu
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d53c60ef5486ebcbf381bfd90970c04135b42" target='_blank'>
              Low-input PacBio sequencing generates high-quality individual fly genomes and characterizes mutational processes
              </a>
            </td>
          <td>
            Hangxing Jia, Shengjun Tan, Yingao Cai, Ya-Dong Guo, Jieyu Shen, Yaqiong Zhang, Huijing Ma, Qingzhu Zhang, Jinfeng Chen, Gexia Qiao, Jue Ruan, Yong E Zhang
          </td>
          <td>2024-07-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4a5e737d6cb299cb80200022679595c834f0d4" target='_blank'>
              The novel circFKBP8/miR-432-5p/E2F7 cascade functions as a regulatory network in breast cancer
              </a>
            </td>
          <td>
            Zhongkui Jin, Wang Xu, Kunlin Yu, Cailu Luo, Xiaodan Luo, Tao Lian, Changshan Liu
          </td>
          <td>2024-08-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="All cancers are associated with massive reorganisation of cellular epigenetic patterns, including extensive changes in the genomic patterns of DNA methylation. However, the huge scale of these changes has made it very challenging to identify key DNA methylation changes responsible for driving cancer development. Here, we present a novel approach to address this problem called methylation mapping. Through comparison of multiple types of B-lymphocyte derived malignancies and normal cell populations, this approach can define the origins of methylation changes as proliferation-driven, differentiation-driven and disease-driven (including both cancer-specific changes and cancer absent changes). Each of these categories of methylation change were found to occur at genomic regions that vary in sequence context, chromatin structure and associated transcription factors, implying underlying mechanistic differences behind the acquisition of methylation at each category. This analysis determined that only a very small fraction (about 3%) of DNA methylation changes in B-cell cancers are disease related, with the overwhelming majority (97%) being driven by normal biological processes, predominantly cell proliferation. Furthermore, the low level of true disease-specific changes can potentially simplify identification of functionally relevant DNA methylation changes, allowing identification of previously unappreciated candidate drivers of cancer development, as illustrated here by the identification and functional confirmation of SLC22A15 as a novel tumour suppressor candidate in acute lymphoblastic leukaemia. Overall, this approach should lead to a clearer understanding of the role of altered DNA methylation in cancer development, facilitate the identification of DNA methylation targeted genes with genuine functional roles in cancer development and thus identify novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f876685ea691f693bc76361ff33252fea93a62c" target='_blank'>
              Deconvolution of cancer methylation patterns determines that altered methylation in cancer is dominated by a non-disease associated proliferation signal
              </a>
            </td>
          <td>
            H. Lalchungnunga, H. Atasoy, EC Schwalbe, CM Bacon, G. Strathdee
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            W. Zhang, X. Zhang, C. Qiu, Z. Zhang, K.-J. Su, Z. Luo, Q. Tian, M. Liu, B. Zhao, L. Wu, H. Shen, C. Wu, H.-W. Deng
          </td>
          <td>2024-08-13</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cfb136406d6e0103812e5102a4ed1670edf2da" target='_blank'>
              METTL3-mediated m6A modification of circGLIS3 promotes prostate cancer progression and represents a potential target for ARSI therapy
              </a>
            </td>
          <td>
            Xiaofeng Cheng, Heng Yang, Yujun Chen, Zhenhao Zeng, Yifu Liu, Xiaochen Zhou, Cheng Zhang, An Xie, Gongxian Wang
          </td>
          <td>2024-08-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Alu retrotransposons, which form the largest family of mobile DNA elements in the human genome, have recently come to attention as a potential source of regulatory novelties, most notably by participating in enhancer function. Even though Alu transcription by RNA polymerase III is subjected to tight epigenetic silencing, their expression has long been known to increase in response to various types of stress, including viral infection. Here we show that, in primary human fibroblasts, adenovirus small e1a triggered derepression of hundreds of individual Alus by promoting TFIIIB recruitment by Alu-bound TFIIIC. Epigenome profiling revealed an e1a-induced decrease of H3K27 acetylation and increase of H3K4 monomethylation at derepressed Alus, making them resemble poised enhancers. The enhancer nature of e1a-targeted Alus was confirmed by the enrichment, in their upstream regions, of the EP300/CBP acetyltransferase, EP400 chromatin remodeler and YAP1 and FOS transcription factors. The physical interaction of e1a with EP400 was critical for Alu derepression, which was abrogated upon EP400 ablation. Our data suggest that e1a targets a subset of enhancer Alus whose transcriptional activation, which requires EP400 and is mediated by the e1a-EP400 interaction, may participate in the manipulation of enhancer activity by adenoviruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36f9b26ed47425d2917decac2468f0de41c5512" target='_blank'>
              Adenovirus small E1A directs activation of Alu transcription at YAP/TEAD- and AP-1-bound enhancers through interactions with the EP400 chromatin remodeler.
              </a>
            </td>
          <td>
            Simona Cantarella, M. Vezzoli, Davide Carnevali, M. Morselli, Nathan R. Zemke, B. Montanini, Coralie F. Daussy, Harald Wodrich, Martin Teichmann, M. Pellegrini, Arnold J. Berk, G. Dieci, Roberto Ferrari
          </td>
          <td>2024-07-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are involved in the pathogenesis of esophageal squamous cell carcinoma (ESCC). This study aimed to explore the mechanisms of aberrant expression and functions of circ_0006168 in ESCC. In this study, real-time qPCR and fluorescence in situ hybridization (FISH) are adopted to estimate the expression and localization of circ_0006168 in cancer tissues and cells. Methylated RNA immunoprecipitation (MeRIP) was performed to detect the N6-methyladenosine (m6A) modification of circ_0006168. Gain- and loss-of-functions of circ_0006168 were performed to identify its role in ESCC progression. RNA-binding protein immunoprecipitation (RIP) was used to detect the interaction of circ_0006168 with IGF2BP2. Luciferase reporter assay and RIP are used to confirm the circ_0006168/miR-384/STAT3 ceRNA network. Our results showed that the expression of circ_0006168 was upregulated in ESCC tissues and cells. METTL3-mediated m6A modification increased the expression of circ_0006168 via IGF2BP2-dependent way in TE-1 cells. Circ_0006168 promoted cell proliferation, migration, invasion, cell cycle progression and inhibited cell apoptosis, while knockdown of circ_0006168 had the reverse effects. Mechanistically, circ_0006168 acted its functions via miR-384/STAT3/Snail axis in TE-1 cells. In conclusion, circ_0006168 is upregulated in ESCC and m6A methylation increased its expression via IGF2BP2. CircRNA_0006168 promotes cell migration, invasion by regulating EMT via miR-384/STAT3/Snail axis in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5633a4be443d2222f80bed6a025bc3273be4d5d6" target='_blank'>
              N6-methyladenosine-modified circ_0006168 promotes epithelial mesenchymal transition via miR-384/STAT3/Snail axis in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Guandi Wu, Qin Hou, Zhe Liu, Zejin Pu, Lingfei Wu
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Microphthalmia associated transcription factor (Mitf) is a critical regulator of the melanocyte lineage and also plays an important role in eye development. Mitf activity in different cell types is controlled in part by ten alternative promoters and their resulting isoforms. A useful tool for melanocyte-based research, the Mitf-cre transgene was designed to express Cre recombinase from the Mitf-M promoter, which is melanocyte specific. However, Mitf-cre mice are also microphthalmic, perhaps because of insertional mutagenesis or disrupted gene expression. Here, we investigated these possibilities. We determined that the eye phenotype arises early, with Mitf-cre embryos at E13.5 exhibiting variable ocular sizes and abnormalities, but all with coloboma. Targeted locus amplification and next generation sequencing indicated that multiple copies of the transgene integrated into an intergenic region on chromosome 2, in between Spred1 and Meis2. The BAC transgene used to make Mitf-cre was larger than expected, carrying three upstream alternative promoters, Mitf-H, Mitf-D, and Mitf-B, which could express their isoforms intact off the transgene. RT-qPCR using eye tissue demonstrated a 5-fold increase in Mitf transcripts containing exon 1B1b, which is shared by Mitf-H, Mitf-D, and Mitf-B, while Spred1 and Meis2 did not differ in their expression. These findings clarify and support the usage of Mitf-cre in conditional mutagenesis in melanocytes. The specific over-expression of the Mitf-H and Mitf-D isoforms, which are preferentially expressed in the RPE, presents a unique resource for those interested in eye development and coloboma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f416be172f324512ebeadaba72cc51d72c665bd9" target='_blank'>
              Mitf over-expression leads to microphthalmia and coloboma in Mitf-cre mice
              </a>
            </td>
          <td>
            A. Longakit, Hannah Bourget, C. D. Van Raamsdonk
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite the established role of DNA methylation in transcriptional silencing, most of the genes remain unchanged upon its global erasing from the genome, while expression of other genes is actually raised. Since its discovery a few decades ago, this unsolved enigma has hampered true understanding of DNA methylation role. By applying an updated mathematical model to the results of the original studies, we showed that this paradoxical effect is due to between-gene differences in the balance between methylation-guided enhancers and silencers, which protect particular disease-causing genes from the otherwise harsh effects of global methylation perturbation. Nevertheless, local methylation alternation may still affect these genes. The results a settle a long-standing wondering and suggest that DNA-methylation regulatory role is significantly greater than previously thought.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f326a9727e723fecbefe9526db8fc37ada53d66" target='_blank'>
              On the enigmatic null effect of global methylation perturbation
              </a>
            </td>
          <td>
            Y. Edrei, A. Hellman
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Radiation cytogenetics has a rich history seldom appreciated by those outside the field. Early radiobiology was dominated by physics and biophysical concepts that borrowed heavily from the study of radiation-induced chromosome aberrations. From such studies, quantitative relationships between biological effect and changes in absorbed dose, dose rate and ionization density were codified into key concepts of radiobiological theory that have persisted for nearly a century. This review aims to provide a historical perspective of some of these concepts, including evidence supporting the contention that chromosome aberrations underlie development of many, if not most, of the biological effects of concern for humans exposed to ionizing radiations including cancer induction, on the one hand, and tumor eradication on the other. The significance of discoveries originating from these studies has widened and extended far beyond their original scope. Chromosome structural rearrangements viewed in mitotic cells were first attributed to the production of breaks by the radiations during interphase, followed by the rejoining or mis-rejoining among ends of other nearby breaks. These relatively modest beginnings eventually led to the discovery and characterization of DNA repair of double-strand breaks by non-homologous end joining, whose importance to various biological processes is now widely appreciated. Two examples, among many, are V(D)J recombination and speciation. Rapid technological advancements in cytogenetics, the burgeoning fields of molecular radiobiology and third-generation sequencing served as a point of confluence between the old and new. As a result, the emergent field of "cytogenomics" now becomes uniquely positioned for the purpose of more fully understanding mechanisms underlying the biological effects of ionizing radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c57db9d0b65c8dcb0db634d392394a245d1aa2a" target='_blank'>
              The Central Role of Cytogenetics in Radiation Biology.
              </a>
            </td>
          <td>
            Susan M. Bailey, Stephen R. Kunkel, Joel S. Bedford, Michael N. Cornforth
          </td>
          <td>2024-07-10</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Chromocenters in mouse cells are membrane-less nuclear compartments that represent typical heterochromatin stably maintained during the cell cycle. Here, we explore how histone H3 variants, replicative H3.1/H3.2 or replacement H3.3, mark these domains during the cell cycle. In mouse embryonic stem cells (ESCs), neuronal precursor cells (NPCs) as well as immortalized 3T3 cells, we find a strong and distinct H3.1 enrichment at chromocenters, with some variation in ESCs. Mechanistically, this H3.1 selective enrichment depends on the DNA Synthesis Coupled (DSC) deposition pathway operating in S phase. Yet, this selective enrichment is challenged when we target H3.3 deposition through the DNA Synthesis Independent (DSI) deposition pathway mediated by HIRA. Altering the H3.1/H3.3 equilibrium at chromocenters in ESCs affects its heterochromatin properties leading to mitotic defects. We thus reveal opposing mechanisms for H3.1 and H3.3 deposition with different enforcement according to cell cycle and potency which determine their ratio at chromocenters and are critical for genome stability and cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d8dd976e66cd05d04e95c15ce7087e63b41055" target='_blank'>
              H3.3 deposition counteracts the replication-dependent enrichment of H3.1 at chromocenters in embryonic stem cells
              </a>
            </td>
          <td>
            S. Arfè, T. Karagyozova, A. Forest, H. Hmidan, E. Meshorer, J. Quivy, G. Almouzni
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Trinucleotide repeat (TNR) expansion is the cause of over 40 neurodegenerative diseases, including Huntington’s disease and Friedreich’s ataxia (FRDA). There are no effective treatments for these diseases due to the poor understanding of molecular mechanisms underlying somatic TNR expansion and contraction in neural systems. We and others have found that DNA base excision repair (BER) actively modulates TNR instability, shedding light on the development of effective treatments for the diseases by contracting expanded repeats through DNA repair. In this study, temozolomide (TMZ) was employed as a model DNA base damaging agent to reveal the mechanisms of the BER pathway in modulating GAA repeat instability at the frataxin (FXN) gene in FRDA neural cells and transgenic mouse mice. We found that TMZ induced large GAA repeat contraction in FRDA mouse brain tissue, neurons, and FRDA iPSC-differentiated neural cells, increasing frataxin protein levels in FRDA mouse brain and neural cells. Surprisingly, we found that TMZ could also inhibit H3K9 methyltransferases, leading to open chromatin and increasing ssDNA breaks and recruitment of the key BER enzyme, pol β, on the repeats in FRDA neural cells. We further demonstrated that the H3K9 methyltransferase inhibitor BIX01294 also induced the contraction of the expanded repeats and increased frataxin protein in FRDA neural cells by opening the chromatin and increasing the endogenous ssDNA breaks and recruitment of pol β on the repeats. Our study provides new mechanistic insight illustrating that inhibition of H3K9 methylation can crosstalk with BER to induce GAA repeat contraction in FRDA. Our results will open a new avenue for developing novel gene therapy by targeting histone methylation and the BER pathway for repeat expansion diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88103a73ada31d035f661593eaffbaace8d21383" target='_blank'>
              DNA Base Damage Repair Crosstalks with Chromatin Structures to Contract Expanded GAA Repeats in Friedreich’s Ataxia
              </a>
            </td>
          <td>
            Yanhao Lai, Nicole Diaz, Rhyisa Armbrister, Irina Agoulnik, Yuan Liu
          </td>
          <td>2024-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. ATAC-seq and ChIP-seq revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLC. Analyses of human SCLC tumors and patient-derived xenografts (PDXs) also showed increase in TREX1 expression in post-chemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 induces innate immune response in drug-resistant Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J. Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ca92a03051caca152f854fcb5c2f6b12d56f76" target='_blank'>
              Integrated genomic/epigenomic analysis stratifies subtypes of clear cell ovarian carcinoma, highlighting their cellular origin
              </a>
            </td>
          <td>
            Akira Nishijima, K. Oda, Kosei Hasegawa, Takahiro Koso, K. Asada, Yuji Ikeda, Ayumi Taguchi, Daichi Maeda, G. Nagae, S. Tsuji, K. Tatsuno, Y. Uehara, A. Kurosaki, S. Sato, M. Tanikawa, K. Sone, M. Mori, M. Ikemura, Keiichi Fujiwara, Tetsuo Ushiku, Yutaka Osuga, Hiroyuki Aburatani
          </td>
          <td>2024-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Mutations in the tumor-suppressor genes BRCA1 and BRCA2 resulting in BRCA1/2 deficiency are frequently identified in breast, ovarian, prostate, pancreatic, and other cancers. Poly(ADP-ribose) polymerase (PARP) inhibitors (PARPis) selectively kill BRCA1/2-deficient cancer cells by inducing synthetic lethality, providing an effective biomarker-guided strategy for targeted cancer therapy. However, a substantial fraction of cancer patients carrying BRCA1/2 mutations do not respond to PARPis, and most patients develop resistance to PARPis over time, highlighting a major obstacle to PARPi therapy in the clinic. Recent studies have revealed that changes of specific functional defects of BRCA1/2-deficient cells, particularly their defects in suppressing and protecting single-stranded DNA gaps, contribute to the gain or loss of PARPi-induced synthetic lethality. These findings not only shed light on the mechanism of action of PARPis, but also lead to revised models that explain how PARPis selectively kill BRCA-deficient cancer cells. Furthermore, new mechanistic principles of PARPi sensitivity and resistance have emerged from these studies, generating potentially useful guidelines for predicting the PARPi response and design therapies for overcoming PARPi resistance. In this Review, we will discuss these recent studies and put them in context with the classic views of PARPi-induced synthetic lethality, aiming to stimulate the development of new therapeutic strategies to overcome PARPi resistance and improve PARPi therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2cad9670fd9d04e8ac0fa76598900aad338e25c" target='_blank'>
              BRCAness, DNA gaps, and gain and loss of PARP inhibitor–induced synthetic lethality
              </a>
            </td>
          <td>
            Xin Li, Lee Zou
          </td>
          <td>2024-07-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii Der, Ida Chen, C. Charles, Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>None</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891d9fb97d4fae14734f86d5adf86855004117f2" target='_blank'>
              Clonal inactivation of TERT impairs stem cell competition
              </a>
            </td>
          <td>
            Kazuteru Hasegawa, Yang Zhao, A. Garbuzov, M. Corces, P. Neuhöfer, Victoria M Gillespie, Peggie Cheung, J. Belk, Yung-Hsin Huang, Yuning Wei, Lu Chen, Howard Y. Chang, S. Artandi
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, , Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) in eukaryotic RNA is an epigenetic modification that is critical for RNA metabolism, gene expression regulation, and the development of organisms. Aberrant expression of m6A components appears in a variety of human diseases. RNA m6A modification in Drosophila has proven to be involved in sex determination regulated by Sxl and may affect X chromosome expression through the MSL complex. The dosage-related effects under the condition of genomic imbalance (i.e., aneuploidy) are related to various epigenetic regulatory mechanisms. Here, we investigated the roles of RNA m6A modification in unbalanced genomes using aneuploid Drosophila. The results showed that the expression of m6A components changed significantly under genomic imbalance, and affected the abundance and genome-wide distribution of m6A, which may be related to the developmental abnormalities of aneuploids. The relationships between methylation status and classical dosage effect, dosage compensation, and inverse dosage effect were also studied. In addition, we demonstrated that RNA m6A methylation may affect dosage-dependent gene regulation through dosage-sensitive modifiers, alternative splicing, the MSL complex, and other processes. More interestingly, there seems to be a closely relationship between MSL complex and RNA m6A modification. It is found that ectopically overexpressed MSL complex, especially the levels of H4K16Ac through MOF could influence the expression levels of m6A modification and genomic imbalance may be involved in this interaction. We found that m6A could affect the levels of H4K16Ac through MOF, a component of the MSL complex, and that genomic imbalance may be involved in this interaction. Altogether, our work reveals the dynamic and regulatory role of RNA m6A modification in unbalanced genomes, and may shed new light on the mechanisms of aneuploidy-related developmental abnormalities and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3774028a6a169a3f6cb2d8f929963755d335f38e" target='_blank'>
              Dynamics and Regulatory Roles of RNA m6A Methylation in Unbalanced Genomes
              </a>
            </td>
          <td>
            Shuai Zhang, Xinyu Liu, Ruixue Wang, Junhan Wang, Ludan Zhang, Lin Sun
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Objectives The complex system of cancer has led to an emphasis on understanding the more general causal relationship within the disease. In this context, concepts of symmetry and symmetry-breaking in distinct biological cell features or components have been examined as an approach to cancer investigation. However, there can be possible limitations in directly interpreting the symmetry-based approach from a physical viewpoint due to the lack of understanding of physical laws governing symmetry in complex systems like cancer. Methods Fractal geometry and DNA walk representation were employed to investigate the geometric features i.e., self-similarity and heterogeneity in DNA nucleotide coding sequences of wild-type and mutated oncogenes, tumour-suppressor, and other unclassified genes. The mutation-facilitated self-similar and heterogenous features were quantified by the fractal dimension and lacunarity coefficient measures, respectively. Additionally, the geometrical orderedness and disorderedness in the analyzed sequences were interpreted from the combination of the fractal measures. Results The findings showed distinct fractal geometric features in the case of fusion mutations. It also highlights the possible interpretation of the observed fractal features as geometric analogues concerning explicit observations corresponding to specific cancer types. In addition, the two-dimensional multi-fractal analysis highlighted the presence of a single exponent in the scaling of mutation-mediated gene sequence self-similarity/complexity and heterogeneity. Conclusions The approach identified mutation-induced geometric features in gene sequences, demonstrating the potential of DNA walks and fractal analysis in translational research regarding cancer. The findings suggest that investigating fractal parameters can capture unique geometric features in nucleotide sequences, contributing to the understanding of cancer’s molecular complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a73efc783fc74b512f9c22058780b5d74879423" target='_blank'>
              DNA walk of specific fused oncogenes exhibit distinct fractal geometric characteristics in nucleotide patterns
              </a>
            </td>
          <td>
            Abhijeet Das, Manas Sehgal, A. Singh, Rishabh Goyal, Mallika Prabhakar, Jeremy Fricke, Isa Mambetsariev, Prakash Kulkarni, M. Jolly, Ravi Salgia
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Abstract Recent advances in DNA sequencing technology have enabled the precise decoding of genomes in non-model organisms, providing a basis for unraveling the patterns and mechanisms of sex chromosome evolution. Studies of different species have yielded conflicting results regarding the traditional theory that sex chromosomes evolve from autosomes via the accumulation of deleterious mutations and degeneration of the Y (or W) chromosome. The concept of the ‘sex chromosome cycle,’ emerging from this context, posits that at any stage of the cycle (i.e., differentiation, degeneration, or loss), sex chromosome turnover can occur while maintaining stable sex determination. Thus, understanding the mechanisms that drive both the persistence and turnover of sex chromosomes at each stage of the cycle is crucial. In this review, we integrate recent findings on the mechanisms underlying maintenance and turnover, with a special focus on several organisms having unique sex chromosomes. Our review suggests that the diversity of sex chromosomes in the maintenance of stable sex determination is underappreciated and emphasizes the need for more research on the sex chromosome cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ad3585db829ba2d4ab84372e9e70791e552b39" target='_blank'>
              Sex chromosome cycle as a mechanism of stable sex determination
              </a>
            </td>
          <td>
            Shun Hayashi, Takuya Abe, Takeshi Igawa, Yukako Katsura, Yusuke Kazama, Masafumi Nozawa
          </td>
          <td>2024-07-10</td>
          <td>Journal of Biochemistry</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Hepatocyte polyploidy and maturity are critical to acquiring specialized liver functions. Multiple intra- and extracellular factors influence ploidy, but how they cooperate temporally to steer liver polyploidization and maturation or how post-transcriptional mechanisms integrate into these paradigms is unknown. Here, we identified an important regulatory hierarchy in which postnatal activation of Epithelial-Splicing-Regulatory-Protein-2 (ESRP2) stimulates biogenesis of liver-specific microRNA (miR-122), thereby facilitating polyploidization, maturation, and functional competence of hepatocytes. By determining transcriptome-wide protein-RNA interactions in vivo and integrating them with single-cell and bulk hepatocyte RNA-seq datasets, we delineate an ESRP2-driven RNA processing program that drives sequential replacement of fetal-to-adult transcript isoforms. Specifically, ESRP2 binds the primary miR-122 host gene transcript to promote its processing/biogenesis. Combining constitutive and inducible ESRP2 gain- and loss-of-function mice models with miR-122 rescue experiments, we demonstrate that timed activation of ESRP2 augments miR-122-driven program of cytokinesis failure, ensuring proper onset and extent of hepatocyte polyploidization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b843bf46c0b24b11154afa20c28344a1ac3278d5" target='_blank'>
              ESRP2-microRNA-122 axis directs the postnatal onset of liver polyploidization and maturation
              </a>
            </td>
          <td>
            Sushant Bangru, Jackie Chen, Nicholas Baker, Diptatanu Das, U. V. Chembazhi, J. Derham, Sandip Chorghade, W. Arif, Frances Alencastro, Andrew W. Duncan, Russ P. Carstens, A. Kalsotra
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Dysregulation of the tumour suppressor gene Adenomatous Polyposis Coli (APC) is a canonical step in colorectal cancer development. Curiously, most colorectal tumours carry biallelic mutations that result in only partial loss of APC function, suggesting that a ‘just-right’ level of APC inactivation, and hence Wnt signalling, provides the optimal conditions for tumorigenesis. Mutational processes act variably across the APC gene, which could contribute to the bias against complete APC inactivation. Thus the selective consequences of partial APC loss are unclear. Here we propose a mathematical model to quantify the tumorigenic effect of biallelic APC genotypes, controlling for somatic mutational processes. Analysing sequence data from >2500 colorectal cancers, we find that APC genotypes resulting in partial protein function confer about 50 times higher probability of progressing to cancer compared to complete APC inactivation. The optimal inactivation level varies with anatomical location and additional mutations of Wnt pathway regulators. We use this context dependency to assess the regulatory effect of secondary Wnt drivers in combination with APC in vivo, and provide evidence that mutant AMER1 combines with APC genotypes that lead to relatively low Wnt. The fitness landscape of APC inactivation is consistent across microsatellite unstable and POLE-deficient colorectal cancers and tumours in patients with Familial Adenomatous Polyposis suggesting a general ‘just-right’ optimum, and pointing to Wnt hyperactivation as a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5f474ac3dba3c99ab4d1fd2c61a492c95822b9" target='_blank'>
              Quantifying ‘just-right’ APC inactivation for colorectal cancer initiation
              </a>
            </td>
          <td>
            Meritxell Brunet Guasch, Nathalie Feeley, Ignacio Soriano, Steve Thorn, Ian Tomlinson, Michael D. Nicholson, Tibor Antal
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA methylation-based indices predict mortality and provide insights into aging mechanisms. Even though sex chromosomes are thought to play an important role in sex-specific aging, they have been largely ignored in epigenetic aging studies. We identified age-related CpG (AR-CpG) sites in the full methylomes of two avian species (zebra finch and jackdaw), including for the first time the avian sex chromosomes (Z and the female-specific, haploid W). In both species, we find a high concentration of AR-CpG sites on the W chromosome, with DNA methylation levels predominantly decreasing with age. Our results highlight distinct age-related changes in sex chromosome DNA methylation compared to the rest of the genome, suggesting that this feature of sex chromosomes may contribute to sex-dependent aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31690344c13aafc58b5419447fd6edd4d52dcc4" target='_blank'>
              Sex-chromosome-dependent aging in female heterogametic methylomes
              </a>
            </td>
          <td>
            Marianthi Tangili, Joanna Sudyka, Fabrício Furni, P. Palsbøll, Simon Verhulst
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. We specify that the DnaA/DNA ratio may be a more rigorous feature in determining the variability of initiation time in E. coli than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work is a continued examination of our hypothesis of the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the variable DnaA/DNA ratio may serve as an essential source of variability of the replication initiation event and the ensuing chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi A. Gheber, Mario Feingold, I. Fishov
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma.
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69e1db6874aa879e7cfcdc2ee3255c993cf70dae" target='_blank'>
              USP9X regulates the proliferation, survival, migration and invasion of gastric cancer cells by stabilizing MTH1
              </a>
            </td>
          <td>
            Wenji Xu, Yaping Zhang, Yingrui Su, Libin Li, Xinxia Yang, Lixing Wang, Hongzhi Gao
          </td>
          <td>2024-07-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Detection of minor DNA allele alterations is becoming increasingly important for early detection and monitoring of cancer. We describe a new method that uses ultraviolet light to eliminate wild-type DNA alleles and enables improved detection of minor genetic or epigenetic changes.


METHODS
Pyrimidine-dependent UV-based minor-allele enrichment (PD-UVME) employed oligonucleotide probes that incorporated a UVA-sensitive 3-cyanovinylcarbazole (CNVK), placed directly opposite interrogated pyrimidines, such as thymine (T) or cytosine (C) in wild-type (WT) DNA. Upon UVA-illumination, CNVK cross-linked with T/C, preventing subsequent amplification. Mutations that removed the T/C escaped cross-linking and were amplified and detected. Similarly, CNVK discriminated between methylated and unmethylated cytosine in CpG dinucleotides, enabling direct enrichment of unmethylated DNA targets. PD-UVME was combined with digital droplet PCR (ddPCR) to detect serine/threonine-protein kinase B-Raf (BRAF) V600E mutations in model systems, thyroid patient cancer tissue samples, and circulating DNA of tumor origin (ctDNA) from melanoma patients.


RESULTS
One thyroid cancer sample out of 9, and 6 circulating-DNA samples out of 7 were found to be BRAF V600E-positive via PD-UVME while classified as negative by conventional ddPCR. Positive samples via conventional ddPCR were also found positive via PD-UVME. All 10 circulating cell-free DNA (cfDNA) samples obtained from normal volunteers were negative via both approaches. Furthermore, preferential enrichment of unmethylated alleles in MAGEA1 promoters using PD-UVME was demonstrated.


CONCLUSIONS
PD-UVME mutation/methylation enrichment performed prior to ddPCR magnifies low-level mutations or epigenetic changes and increases sensitivity and confidence in the results. It can assist with clinical decisions that hinge on the presence of trace alterations like BRAF V600E.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362fd25c2fb0f063f30c56509492e4fe9cd161b" target='_blank'>
              Pyrimidine-Dependent UV-Mediated Cross-Linking Magnifies Minor Genetic or Epigenetic Changes in Clinical Samples.
              </a>
            </td>
          <td>
            F. Yu, Farzana Ahmed, S. Smilkou, S. Yasmin-Karim, Farzaneh Darbeheshti, A. Markou, Martyn Bullock, Ioannis Boukovinas, V. Adalsteinsson, E. Lianidou, G. M. Makrigiorgos
          </td>
          <td>2024-07-23</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4d231cf341ecb2bcc080757179bed2d4c422c6" target='_blank'>
              Hypoxia-enhanced YAP1-EIF4A3 interaction drives circ_0007386 circularization by competing with CRIM1 pre-mRNA linear splicing and promotes non-small cell lung cancer progression
              </a>
            </td>
          <td>
            Lixia Li, Dewei Liu, Tingting Chen, Chunhui Wei, Youping Qiao, Weiliang Liu, Yanmei Liang, Zhu Liang, Chunyuan Chen, Dongming Li, Bin Wu, Xuanna Zhao, Dan Huang, Dong Wu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcf3e13b55bc25cd806bf0acc955517dc375f3ab" target='_blank'>
              CircRNA as an Achilles heel of cancer: characterization, biomarker and therapeutic modalities
              </a>
            </td>
          <td>
            Jun Zhang, Zai Luo, Yang Zheng, Mingyu Duan, Zheng-Jun Qiu, Chen Huang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3338350362bc2189e1725a808911e7a879595b1f" target='_blank'>
              TESC associated with poor prognosis enhances cancer stemness and migratory properties in liver cancer
              </a>
            </td>
          <td>
            Peng Ye, Shahang Luo, Junyu Huang, Xihua Fu, Xiaoxia Chi, Jong-Ho Cha, Yumei Chen, Yanjun Mai, Kai-Wen Hsu, Xiuwen Yan, Wen-Hao Yang
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1238d9815100a9d52edecd0a776b444d4a9b4a2f" target='_blank'>
              N6-methyladenosine modification of circMARK2 enhances cytoplasmic export and stabilizes LIN28B, contributing to the progression of Wilms tumor
              </a>
            </td>
          <td>
            Guannan Shu, Zhang Zhao, Tianxin Zhao, Changmi Deng, Jiangquan Zhu, Yufeng Han, Minyu Chen, Jiajia Jing, Gaochen Bai, Dian Li, Feng Li, Jing He, W. Fu, Guochang Liu
          </td>
          <td>2024-07-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5395af718a191f5fa3586c7dc9b0e07227b5b78c" target='_blank'>
              Subunit-specific analysis of cohesin-mutant myeloid malignancies reveals distinct ontogeny and outcomes
              </a>
            </td>
          <td>
            Joahnn-Christoph Jann, Christopher B. Hergott, Marisa Winkler, Yiwen Liu, Benjamin Braun, A. Charles, K. Copson, Shougat Barua, M. Meggendorfer, N. Nadarajah, S. Shimony, Eric S. Winer, M. Wadleigh, Richard M Stone, D. DeAngelo, Jacqueline S Garcia, Torsten Haferlach, R. C. Lindsley, M. Luskin, Maximilian Stahl, Zuzana Tothova
          </td>
          <td>2024-07-20</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Understanding the regulatory mechanisms of gene expression is a crucial objective in genomics. Although the DNA sequence near the transcription start site (TSS) offers valuable insights, recent methods suggest that analyzing only the surrounding DNA may not suffice to accurately predict gene expression levels. We developed GENet (Gene Expression Network from Histone and Transcription Factor Integration), a novel approach that integrates essential regulatory signals from transcription factors and histone modifications into a graph-based model. GENet extends beyond simple DNA sequence analysis by incorporating additional layers of genetic control, which are vital for determining gene expression. Our method markedly enhances the prediction of mRNA levels compared to previous models that depend solely on DNA sequence data. The results underscore the significance of including comprehensive regulatory information in gene expression studies. GENet emerges as a promising tool for researchers, with potential applications extending from fundamental biological research to the development of medical therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f603b69c3085904fb437d7844f2d8696ae07e11" target='_blank'>
              GENet: A Graph-Based Model Leveraging Histone Marks and Transcription Factors for Enhanced Gene Expression Prediction
              </a>
            </td>
          <td>
            Mahdieh Labani, Amin Beheshti, Tracey A O'Brien
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Here, we introduce the highly versatile circular polymerase chain reaction (CiPCR) technique, propose a mechanism of action, and describe a number of examples demonstrating the versatility of this technique. CiPCR takes place between two fragments of dsDNA with two homologous regions, as long as one of the fragments carries said regions at its 3′- and 5′-ends. Upon hybridization, elongation by a polymerase occurs from all 3′-ends continuously until a 5′-end is reached, leading to stable circular dsDNA with staggered nicks. When both dsDNA fragments carry the homology at their 3′- and 5′-ends (Type I CiPCR), all four 3′-ends effectively prime amplification of the intervening region and CiPCR products can function as template during the reaction. In contrast, when only one of the two dsDNA fragments carries the homologous regions at its 3′- and 5′-ends and the other carries such regions internally (Type II CiPCR), only two 3′-ends can be amplified and CiPCR products possess no template activity. We demonstrate the applicability of both CiPCR types via well-illustrated experimental examples. CiPCR is well adapted to the quick resolution of most of the molecular cloning challenges faced by the biology/biomedicine laboratory, including the generation of insertions, deletions, and mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3cb2ea36d681e36d6cc2ddc27dd7f88df313b0b" target='_blank'>
              Circular PCR as an efficient and precise umbrella of methods for the generation of circular dsDNA with staggered nicks: Mechanism and types
              </a>
            </td>
          <td>
            Pedro Ferro-Gallego, Antón Vila-Sanjurjo, Andrea Katherine Valderrama Pereira, Gonzalo Porres Pérez, Lourdes Domínguez-Gerpe
          </td>
          <td>2024-07-19</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Turner syndrome (TS) is caused by a complete or partial absence of an X or Y chromosome, including chromosomal mosaicism, affecting 1 in 2500 female live births. Sister chromatid exchange (SCE) is used as a sensitive indicator of spontaneous chromosome instability. Cells from mosaic patients constitute useful material for SCE evaluations as they grow under the influence of the same genetic background and endogenous and exogenous factors. We evaluated the proliferation dynamics and SCE frequencies of 45,X and 46,XN cells of 17 mosaic TS patients. In two participants, the 45,X cells exhibited a proliferative disadvantage in relation to 46,XN cells after 72 h of cultivation. The analysis of the mean proliferation index (PI) showed a trend for a significant difference between the 45,X and 46,X+der(X)/der(Y) cell lineages; however, there were no intra-individual differences. On the other hand, mean SCE frequencies showed that 46,X+der(X) had the highest mean value and 46,XX the lowest, with 45,X occupying an intermediate position among the lineages found in at least three participants; moreover, there were intra-individual differences in five patients. Although 46,X+der(X)/der(Y) cell lineages, found in more than 70% of participants, were the most unstable, they had a slightly higher mean PI than the 45,X cell lineages in younger (≤17 years) mosaic TS participants. This suggests that cells with a karyotype distinct from 45,X may increase with time in mosaic TS children and adolescents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a35420c99ba97fe4f5acd7a759bcc6a732cb8d1" target='_blank'>
              Cell Cycle Kinetics and Sister Chromatid Exchange in Mosaic Turner Syndrome
              </a>
            </td>
          <td>
            M. B. Goulart, Eduardo Vieira Neto, Daniela R. Ney Garcia, Marília Martins Guimarães, Isaías Soares de Paiva, Karina de Ferran, Nathalia Correia Krause dos Santos, Luciana Santos Barbosa, Amanda F. de Figueiredo, M. Ribeiro, Márcia Gonçalves Ribeiro
          </td>
          <td>2024-07-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb263a8ed53aaba0d5cf504ca6351b84a2417a1a" target='_blank'>
              MksB is a novel mycobacterial condensin that orchestrates spatiotemporal positioning of replication machinery
              </a>
            </td>
          <td>
            Hanna Bułacz, Joanna Hołówka, Wiktoria Wójcik, Jolanta Zakrzewska-Czerwińska
          </td>
          <td>2024-08-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec9eb84ca8c610e2b08f8d4ce038f9c0401c702" target='_blank'>
              De novo variants in the RNU4-2 snRNA cause a frequent neurodevelopmental syndrome
              </a>
            </td>
          <td>
            Yuyang Chen, R. Dawes, Hyung Chul Kim, A. Ljungdahl, S. Stenton, Susan Walker, Jenny Lord, Gabrielle Lemire, Alexandra C Martin-Geary, Vijay S Ganesh, Jialan Ma, J. Ellingford, E. Delage, Elston N. D’Souza, Shan Dong, David R Adams, Kirsten Allan, Madhura Bakshi, Erin E Baldwin, Seth I Berger, Jonathan A Bernstein, Ishita Bhatnagar, Ed Blair, Natasha J Brown, Lindsay C Burrage, Kimberly Chapman, David J Coman, A. Compton, Chloe A Cunningham, Precilla D’Souza, P. Danecek, E. Délot, Kerith-Rae Dias, Ellen R Elias, F. Elmslie, Carey-Anne Evans, L. Ewans, Kimberly M Ezell, Jamie L Fraser, L. Gallacher, C. Genetti, A. Goriely, Christina L Grant, Tobias B Haack, Jenny E Higgs, A. Hinch, M. Hurles, Alma Kuechler, Katherine L Lachlan, S. Lalani, F. Lecoquierre, E. Leitão, A. L. Fevre, R. Leventer, J. Liebelt, S. Lindsay, Paul J Lockhart, Alan S Ma, Ellen F. Macnamara, Sahar Mansour, Taylor M Maurer, Hector R Mendez, Kay Metcalfe, Stephen B. Montgomery, Mariya Moosajee, M. Nassogne, Serena Neumann, Michael O'Donoghue, Melanie O'Leary, E. E. Palmer, N. Pattani, John Phillips, Georgia Pitsava, Ryan Pysar, Heidi L. Rehm, Chloe M. Reuter, N. Revencu, A. Riess, Rocio Rius, Lance Rodan, T. Roscioli, Jill A. Rosenfeld, R. Sachdev, Charles J Shaw-Smith, Cas Simons, Sanjay M. Sisodiya, Penny Snell, Laura St Clair, Zornitza Stark, Helen Stewart, T. Tan, Natalie B Tan, Suzanna E L Temple, D. Thorburn, C. Tifft, E. Uebergang, G. VanNoy, Pradeep Vasudevan, Eric Vilain, David Viskochil, Laura Wedd, M. Wheeler, Susan M. White, Monica Wojcik, Lynne A Wolfe, Zoe Wolfenson, Caroline F Wright, Changrui Xiao, David Zocche, J. L. Rubenstein, Eirene Markenscoff-Papadimitriou, S. Fica, Diana Baralle, C. Depienne, D. MacArthur, Joanna M M Howson, Stephan J. Sanders, A. O’Donnell-Luria, Nicola Whiffin
          </td>
          <td>2024-07-11</td>
          <td>Nature</td>
          <td>2</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec67b6cd2c5b516c2d3f922caad0a44789a5bdd" target='_blank'>
              TRF1 and TRF2: pioneering targets in telomere-based cancer therapy
              </a>
            </td>
          <td>
            Anoop Kallingal, Radosław Krzemieniecki, Natalia Maciejewska, Wioletta Brankiewicz-Kopcińska, Maciej Baginski
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A significant number of the genetic alterations observed in cancer patients lie within nonprotein-coding segments of the genome, including regions coding for long noncoding RNAs (lncRNAs). LncRNAs display aberrant expression in breast cancer (BrCa), but the functional implications of this altered expression remain to be elucidated. By performing transcriptome screen in a triple negative BrCa (TNBC) isogenic 2D and 3D spheroid model, we observed aberrant expression of >1000 lncRNAs during BrCa progression. The chromatin-associated lncRNA MANCR shows elevated expression in metastatic TNBC. MANCR is upregulated in response to cellular stress and modulates DNA repair and cell proliferation. MANCR promotes metastasis as MANCR-depleted cells show reduced cell migration, invasion, and wound healing in vitro, and reduced metastatic lung colonization in xenograft experiments in vivo. Transcriptome analyses reveal that MANCR modulates expression and pre-mRNA splicing of genes, controlling DNA repair and checkpoint response. MANCR promotes the transcription of NET1A, a Rho-GEF that regulates DNA damage checkpoint and metastatic processes in cis, by differential promoter usage. Experiments suggest that MANCR regulates the expression of cancer-associated genes by modulating the association of various transcription factors and RNA-binding proteins. Our results identified the metastasis-promoting activities of MANCR in TNBC by cis-regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917a434f9bc31bbdd4efde82c1a4158058876264" target='_blank'>
              MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF.
              </a>
            </td>
          <td>
            D. K. Singh, Zhengmin Cong, Y. Song, Minxue Liu, Ritu Chaudhary, Dazhen Liu, Yu Wang, Rishabh Prasanth, Rajendra K C, Simon Lizarazo, Miriam Akhnoukh, Omid Gholamalamdari, Anurupa Moitra, Lisa M. Jenkins, Rohit Bhargava, Erik R Nelson, Kevin Van Bortle, S. Prasanth, K. Prasanth
          </td>
          <td>2024-08-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="MOTIVATION
Circulating-cell free DNA (cfDNA) is widely explored as a non-invasive biomarker for cancer screening and diagnosis. The ability to decode the cells of origin in cfDNA would provide biological insights into pathophysiological mechanisms, aiding in cancer characterization and directing clinical management and follow-up.


RESULTS
We developed a DNA methylation signature-based deconvolution algorithm, MetDecode, for cancer tissue origin identification. We built a reference atlas exploiting de novo and published whole-genome methylation sequencing data for colorectal, breast, ovarian and cervical cancer, and blood-cell-derived entities. MetDecode models the contributors absent in the atlas with methylation patterns learnt on-the-fly from the input cfDNA methylation profiles. Additionally, our model accounts for the coverage of each marker region to alleviate potential sources of noise. In-silico experiments showed a limit of detection down to 2.88% of tumour tissue contribution in cfDNA. MetDecode produced Pearson correlation coefficients above 0.95 and outperformed other methods in simulations (p < 0.001; T-test; one-sided). In plasma cfDNA profiles from cancer patients, MetDecode assigned the correct tissue-of-origin in 84.2% of cases. In conclusion, MetDecode can unravel alterations in the cfDNA pool components by accurately estimating the contribution of multiple tissues, while supplied with an imperfect reference atlas.


AVAILABILITY
MetDecode is available at https://github.com/JorisVermeeschLab/MetDecode.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6103b5c6abb6f1e1d9da1b8923c2a277930a1a19" target='_blank'>
              MetDecode: Methylation-based deconvolution of cell-free DNA for non-invasive multi-cancer typing.
              </a>
            </td>
          <td>
            Antoine Passemiers, Stefania Tuveri, Dhanya Sudhakaran, Tatjana Jatsenko, T. Laga, K. Punie, Sigrid Hatse, Sabine Tejpar, A. Coosemans, E. Van Nieuwenhuysen, Dirk Timmerman, G. Floris, A. Van Rompuy, Xavier Sagaert, Antonia Testa, Daniela Ficherova, Daniele Raimondi, Frédéric Amant, L. Lenaerts, Yves Moreau, J. R. Vermeesch
          </td>
          <td>2024-08-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Senataxin is an RNA:DNA helicase that plays an important role in the resolution of RNA:DNA hybrids (R-loops) formed during transcription. R-loops are involved in the regulation of biological processes such as immunoglobulin class switching, gene expression and DNA repair. Excessive accumulation of R-loops results in DNA damage and loss of genomic integrity. Senataxin is critical for maintaining optimal levels of R-loops to prevent DNA damage and acts as a genome guardian. Within the nucleus, senataxin interacts with various RNA processing factors and DNA damage response and repair proteins. Senataxin interactors include survival motor neuron and zinc finger protein 1, with whom it co-localizes in sub-nuclear bodies. Despite its ubiquitous expression, mutations in senataxin specifically affect neurons and result in distinct neurodegenerative diseases such as amyotrophic lateral sclerosis type 4 and ataxia with oculomotor apraxia type 2, which are attributed to the gain-of-function and the loss-of-function mutations in senataxin, respectively. In addition, low levels of senataxin (loss-of-function) in spinal muscular atrophy result in the accumulation of R-loops causing DNA damage and motor neuron degeneration. Senataxin may play multiple functions in diverse cellular processes; however, its emerging role in R-loop resolution and maintenance of genomic integrity is gaining attention in the field of neurodegenerative diseases. In this review, we highlight the role of senataxin in R-loop resolution and its potential as a therapeutic target to treat neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0918b5cad58e2e8aa658850e44f481a80ca9f6d7" target='_blank'>
              Role of senataxin in R-loop-mediated neurodegeneration
              </a>
            </td>
          <td>
            Annapoorna Kannan, Shyni Gangadharan Leela, Dana Branzei, L. Gangwani
          </td>
          <td>2024-07-15</td>
          <td>Brain Communications</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Oxidative damage triggers micronuclear membrane rupture and defective repair Errors in chromosome segregation during cell division (mitosis) can lead to chromosomal instability, a hallmark of human cancer. These mitotic errors give rise to the formation of micronuclei—small cytoplasmic structures that are spatially separated from the primary nucleus and contain lagging chromosomes or chromosome fragments. Unlike the primary nucleus of a cell, micronuclei frequently undergo irreparable rupture and collapse. This breakdown releases micronuclear DNA into the cytosol, causing DNA damage, chromosomal rearrangements, and genomic instability, thereby contributing to cancer progression. The rupture of micronuclei also activates a component of the innate immune system called the cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING) pathway, leading to tumorrelated inflammation and metastasis (1). On pages 951 and 952 of this issue, Di Bona et al. (2) and Martin et al. (3), respectively, report the molecular underpinnings of micronuclear collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3bfa9de0e1bf705d54ddc01f0ac3869ca5a791" target='_blank'>
              Micronuclear collapse mechanisms in cancer
              </a>
            </td>
          <td>
            Marianna Maddaluno, C. Settembre
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Maintenance of genome stability is of paramount importance for the survival of an organism. However, genomic integrity is constantly being challenged by various endogenous and exogenous processes that damage DNA. Therefore, cells are heavily reliant on DNA repair pathways that have evolved to deal with every type of genotoxic insult that threatens to compromise genome stability. Notably, inherited mutations in genes encoding proteins involved in these protective pathways trigger the onset of disease that is driven by chromosome instability e.g. neurodevelopmental abnormalities, neurodegeneration, premature ageing, immunodeficiency and cancer development. The ability of cells to regulate the recruitment of specific DNA repair proteins to sites of DNA damage is extremely complex but is primarily mediated by protein post-translational modifications (PTMs). Ubiquitylation is one such PTM, which controls genome stability by regulating protein localisation, protein turnover, protein-protein interactions and intra-cellular signalling. Over the past two decades, numerous ubiquitin (Ub) E3 ligases have been identified to play a crucial role not only in the initiation of DNA replication and DNA damage repair but also in the efficient termination of these processes. In this review, we discuss our current understanding of how different Ub E3 ligases (RNF168, TRAIP, HUWE1, TRIP12, FANCL, BRCA1, RFWD3) function to regulate DNA repair and replication and the pathological consequences arising from inheriting deleterious mutations that compromise the Ub-dependent DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d67379a570ccc6355d856b2e241bb15b1ce2e1c" target='_blank'>
              E3 ligases: a ubiquitous link between DNA repair, DNA replication and human disease
              </a>
            </td>
          <td>
            Anoop S. Chauhan, Satpal S. Jhujh, Grant S Stewart
          </td>
          <td>2024-07-10</td>
          <td>Biochemical Journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Long-read sequencing technology enables highly accurate detection of allele-specific RNA expression, providing insights into the effects of genetic variation on splicing and RNA abundance. Furthermore, the ability to directly sequence RNA promises the detection of RNA modifications in tandem with ascertaining the allelic origin of each molecule. Here, we leverage these advantages to determine allele-biased patterns of N6-methyladenosine (m6A) modifications in native mRNA. We utilized human and mouse cells with known genetic variants to assign allelic origin of each mRNA molecule combined with a supervised machine learning model to detect read-level m6A modification ratios. Our analyses revealed the importance of sequences adjacent to the DRACH-motif in determining m6A deposition, in addition to allelic differences that directly alter the motif. Moreover, we discovered allele-specific m6A modification (ASM) events with no genetic variants in close proximity to the differentially modified nucleotide, demonstrating the unique advantage of using long reads and surpassing the capabilities of antibody-based short-read approaches. This technological advancement promises to advance our understanding of the role of genetics in determining mRNA modifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc670683629b81daed9b31ffe9b2b01d01648b9e" target='_blank'>
              Long-read RNA sequencing reveals allele-specific N6-methyladenosine modifications
              </a>
            </td>
          <td>
            Dayea Park, Can Cenik
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b11c9fdbfd43bb000396d5cb58b5ead9a774473" target='_blank'>
              Hypermethylated RASAL1’s promotive role in chemoresistance and tumorigenesis of choriocarcinoma was regulated by TET2 but not DNMTs
              </a>
            </td>
          <td>
            Xianling Zeng, Rui An, Ruixia Guo, Han Li
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNAs (eccDNAs), a type of double‐stranded DNAs (dsDNAs) that facilitate the activation of the DNA sensing machinery, have been implicated in the progression and prognosis of various diseases. While the roles of eccDNAs remain contentious, their significance in diffuse large B‐cell lymphoma (DLBCL) has not been reported. Methods Circular DNA sequencing (circle‐seq) was used to demonstrate the expression profile of eccDNAs in DLBCL, and atomic force microscopy to validate the presence of eccDNAs. CCK‐8 and scRNA‐seq techniques were employed to uncover the activation of eccDNA in the STING pathway, leading to enhanced cell proliferation. Chemotherapeutic drugs were used to test the hypothesis that DNA damage induces the production of eccDNA, thereby activating the STING pathway independent of cGAS. GEO databases were used for verification of the prognosis of the eccDNA‐related genes, and animal models were used to investigate the synergistic effects of DNA damage therapy in combination with STING inhibitors on anti‐tumour responses. Results EccDNAs were widely expressed in DLBCL and associated with the prognosis of patients. Elevated abundance of eccDNAs promoted the progression of DLBCL. Chemotherapeutic drugs‐induced DNA damage triggered the generation of eccDNAs, resulting in the activation of the STING signalling in a cGAS‐independent manner. Moreover, inhibition of STING exerted a synergistic anti‐tumour effect with cisplatin. Conclusions EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. This finding offers a rational therapeutic strategy combining chemotherapy with targeting STING. Highlights EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. The combined treatment of chemotherapeutic drugs with STING inhibitor significantly delayed the tumor progression, providing new insights into the therapeutic strategy for patients with DLBCL, particularly the relapsed and/or refractory (R/R) ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e573ea771bfdb5c81c97bedfb9abf4331d619e8" target='_blank'>
              Multi‐omics integration reveals the oncogenic role of eccDNAs in diffuse large B‐cell lymphoma through STING signalling
              </a>
            </td>
          <td>
            Zijuan Wu, Wei Zhang, Luqiao Wang, Jiayan Leng, Yongle Li, Zhou Fan, Mengtao Zhan, Lei Cao, Yongning Jiang, Yan Jiang, Bing Sun, Jianxin Fu, Jianyong Li, W. Shi, H. Jin
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is an aggressive primary intestinal malignancy with the third-highest incidence and second-highest mortality among all cancer types worldwide. Transcription factors (TFs) regulate cell development and differentiation owing to their ability to recognize specific DNA sequences upstream of genes. Numerous studies have demonstrated a strong correlation between TFs, the etiology of tumors, and therapeutic approaches. Here, we aimed to explore prognosis-related TFs and comprehend their carcinogenic mechanisms, thereby offering novel insights into the diagnosis and management of CRC. Materials and Methods Differentially expressed TFs between CRC and normal tissues were identified leveraging The Cancer Genome Atlas database, Weighted correlation network analysis and Cox regression analysis were performed to identify prognosis-related TFs. The cellular functions of hub TF zinc finger E-box binding homeobox 1 (ZEB1) were determined using by 5-ethynyl-2′-deoxyuridine and cell invasion assays in CRC cells. RNA-sequencing, Kyoto Encyclopedia of Genes and Genomes enrichment, and gene set enrichment analyses were used to identify the cellular processes in which ZEB1 participates. Immunoaffinity purification, silver staining mass spectrometry, and a chromatin immunoprecipitation assay were conducted to search for proteins that might interact with ZEB1 and the target genes they jointly regulate. Results Thirteen central TFs related to prognosis were identified through bioinformatics analysis techniques. Among these TFs, ZEB1 emerged as the TF most closely associated with CRC, as determined through a combination of regulatory network diagrams, survival curves, and phenotype analyses. ZEB1 promotes CRC cell growth by recruiting the NuRD(MTA1) complex, and the ZEB1/NuRD(MTA1) complex transcriptionally represses glycolysis-associated tumor suppressor genes. Conclusion Our study not only identified a hub biomarker related to CRC prognosis but also revealed the specific molecular mechanisms through which ZEB1 affects cancer progression. These insights provide crucial evidence for the diagnosis of CRC and potential treatment opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951b486f89f4fbb493f66d9c80785b0eee594498" target='_blank'>
              Transcription factor ZEB1 coordinating with NuRD complex to promote oncogenesis through glycolysis in colorectal cancer
              </a>
            </td>
          <td>
            Tianyang Gao, Xinhui Hao, Jing-yao Zhang, Miaomiao Huo, Ting Hu, Tianyu Ma, Hefen Yu, X. Teng, Yong Wang, Yunkai Yang, Wei Huang, Yan Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92a939542e292edaa1bc766f527a0552465d51" target='_blank'>
              PTIP epigenetically regulates DNA damage-induced cell cycle arrest by upregulating PRDM1
              </a>
            </td>
          <td>
            Y. Nakata, Shion Nagasawa, Y. Sera, N. Yamasaki, A. Kanai, K. Kobatake, Takeshi Ueda, M. Koizumi, Ichiro Manabe, Osamu Kaminuma, Hiroaki Honda
          </td>
          <td>2024-08-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The H3K4 methyltransferase SETD1A plays an essential role in both development and cancer. However, essential components involved in SETD1A chromatin binding remain unclear. Here, we discovered that BOD1L exhibits the highest correlated SETD1A co-dependency in human cancer cell lines. BOD1L knockout reduces leukemia cells in vitro and in vivo, and mimics the transcriptional profiles observed in SETD1A knockout cells. The loss of BOD1L immediately reduced SETD1A distribution at transcriptional start sites (TSS), induced transcriptional elongation defect, and increased the RNA polymerase II content at TSS; however, it did not reduce H3K4me3. The Shg1 domain of BOD1L has a DNA binding ability, and a tryptophan residue (W104) in the domain recruits SETD1A to chromatin through the association with SETD1A FLOS domain. In addition, the BOD1L-SETD1A complex associates with transcriptional regulators, including E2Fs. These results reveal that BOD1L mediates chromatin and SETD1A, and regulates the non-canonical function of SETD1A in transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea47ee3cc9995eedcb6c115549fadf580ad7e9c9" target='_blank'>
              BOD1L mediates chromatin binding and non-canonical function of H3K4 methyltransferase SETD1A.
              </a>
            </td>
          <td>
            Takayuki Hoshii, Sota Kikuchi, T. Kujirai, Takeshi Masuda, Tomoko Ito, Satoshi Yasuda, Makoto Matsumoto, Bahityar Rahmutulla, Masaki Fukuyo, Takeshi Murata, Hitoshi Kurumizaka, Atsushi Kaneda
          </td>
          <td>2024-07-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Precise double-strand break (DSB) repair is paramount for genome stability. Homologous recombination (HR) is preferred to repair DSBs when a nearby sister chromatid ensures an error-free template. In Saccharomyces cerevisiae, this preference extends into anaphase and telophase (late mitosis; late-M), despite sister chromatids having been pulled apart. Previously, we identified the nuclear envelope (NE) protein Msc1 as important for late-M DSB repair. Here, we report that Msc1 faces the NE lumen, and its depletion leads to DSB-independent over-compartmentalization of the nucleus, which hampers approximation of sister loci after DSB. Depletion of Msc1 also leads to nuclear pore complex mislocation, a phenotype shared by the highly conserved NE healing complex ESCRT-III. Critically, we show that both Msc1 and ESCRT-III also ensure DSB repair in G2/M, and that there is synergism between Msc1 and the ESCRT-III subunit Snf7. These findings highlight the essential role of NE health in DSB repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/411d62fef8c8eeb83c687f479b3594e76b289bd0" target='_blank'>
              Continuous nuclear envelope surveillance is required for DNA double-strand break repair
              </a>
            </td>
          <td>
            Sara Medina-Suárez, F. Machín
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaf9c2f335952f249d3df58f150d64ce452ea5c" target='_blank'>
              CircXPO6 promotes breast cancer progression through competitively inhibiting the ubiquitination degradation of c-Myc.
              </a>
            </td>
          <td>
            Aiqi Xu, Xi Li, Qiaoting Cai, Ciqiu Yang, Mei Yang, Hongfei Gao, M. Cheng, Xianzhe Chen, F. Ji, Hailin Tang, Kun Wang
          </td>
          <td>2024-08-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley
          </td>
          <td>2024-08-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Disruptive variants in the chromodomain helicase CHD8, which acts as a transcriptional regulator during neurodevelopment, are strongly associated with risk for autism spectrum disorder (ASD). Loss of CHD8 function is hypothesized to perturb gene regulatory networks in the developing brain, thereby contributing to ASD etiology. However, insight into the cell type-specific transcriptional effects of CHD8 loss of function remains limited. We used single-cell and single-nucleus RNA-sequencing to globally profile gene expression and identify dysregulated genes in the embryonic and juvenile wild type and Chd8+/− mouse cortex, respectively. Chd8 and other ASD risk-associated genes showed a convergent expression trajectory that was largely conserved between the mouse and human developing cortex, increasing from the progenitor zones to the cortical plate. Genes associated with risk for neurodevelopmental disorders and genes involved in neuron projection development, chromatin remodeling, signaling, and migration were dysregulated in Chd8+/− embryonic day (E) 12.5 radial glia. Genes implicated in synaptic organization and activity were dysregulated in Chd8+/− postnatal day (P) 25 deep- and upper-layer excitatory cortical neurons, suggesting a delay in synaptic maturation or impaired synaptogenesis due to CHD8 loss of function. Our findings reveal a complex pattern of transcriptional dysregulation in Chd8+/− developing cortex, potentially with distinct biological impacts on progenitors and maturing neurons in the excitatory neuronal lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6506acfbabe2f922b8852083b3df7cdf4715bd" target='_blank'>
              Cell type-specific dysregulation of gene expression due to Chd8 haploinsufficiency during mouse cortical development
              </a>
            </td>
          <td>
            Kristina M. Yim, Marybeth Baumgartner, M. Krenzer, María F. Rosales Larios, Guillermina Hill-Terán, Timothy Nottoli, Rebecca A. Muhle, James P. Noonan
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Recent research indicates a positive correlation between DEP structural domain‐containing 1B (DEPDC1B) and the cell cycle in various tumors. However, the role of DEPDC1B in the infiltration of the tumor immune microenvironment (TIME) remains unexplored. Methods We analyzed the differential expression and prognostic significance of DEPDC1B in colon adenocarcinoma (COAD) using the R package “limma” and the Gene Expression Profiling Interactive Analysis (GEPIA) website. Gene set enrichment analysis (GSEA) was employed to investigate the functions and interactions of DEPDC1B expression in COAD. Cell Counting Kit‐8 (CCK‐8) assays and colony formation assays were utilized to assess the proliferative function of DEPDC1B. Correlations between DEPDC1B expression and tumor‐infiltrating immune cells, immune checkpoints, tumor mutational burden (TMB), and microsatellite instability (MSI) status were examined using Spearman correlation analysis and CIBERSORT. Results DEPDC1B was highly expressed in COAD. Elevated DEPDC1B expression was associated with lower epithelial‐to‐mesenchymal transition (EMT) and TNM stages, leading to a favorable prognosis. DEPDC1B mRNA was prominently expressed in COAD cell lines. CCK‐8 and colony formation assays demonstrated that DEPDC1B inhibited the proliferation of COAD cells. Analysis using the CIBERSORT database and Spearman correlation revealed that DEPDC1B correlated with four types of tumor‐infiltrating immune cells. Furthermore, high DEPDC1B expression was linked to the expression of PD‐L1, CTLA4, SIGLEC15, PD‐L2, TMB, and MSI‐H. High DEPDC1B expression also indicated responsiveness to anti‐PD‐L1 immunotherapy. Conclusions DEPDC1B inhibits the proliferation of COAD cells and positively regulates the cell cycle, showing a positive correlation with CCNB1 and PBK expression. DEPDC1B expression in COAD is associated with tumor‐infiltrating immune cells, immune checkpoints, TMB, and MSI‐H in the tumor immune microenvironment. This suggests that DEPDC1B may serve as a novel prognostic marker and a potential target for immunotherapy in COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/293cb0f525ffa84f15dbabefe7162aab0692a337" target='_blank'>
              DEPDC1B: A novel tumor suppressor gene associated with immune infiltration in colon adenocarcinoma
              </a>
            </td>
          <td>
            Dandan Zhu, Huolun Feng, Zhixiong Zhang, Jiaqi Li, Yong Li, Tieying Hou
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Histones are hyper modified proteins that regulate chromatin accessibility and DNA readout. Co-existing post-translational modifications (PTMs) on histones affect interaction affinities of chromatin-associated proteins in ways that are still mostly unexplored. Here, we focus on the biological role of a specific histone code made of two PTMs with supposedly opposing biological functions, i.e. H3K9me3 marker of constitutive heterochromatin and H3K14ac benchmarking accessible chromatin. By applying multi-dimensional mass spectrometry, we demonstrated that EXOSC4 interacts with H3K9me3 + acetyls and affects post-transcriptional regulation. Specifically, EXOSC4 depletion leads to down-regulation of the RNA surveillance machinery and increased expression of non-coding transcripts, including anti-sense RNAs. Together, we elucidate the role of a co-modified histone tail and demonstrate its involvement in the RNA machinery and spurious transcription surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414a923cc5fe1402e6de4d18710208900c79f29c" target='_blank'>
              EXOSC4 is recruited by histone H3 co-modified with K9me3 and acetylations to surveil non-coding transcription
              </a>
            </td>
          <td>
            S. Stransky, Ronald Cutler, Jennifer T. Aguilan, Joseph D. DeAngelo, D. Shechter, Simone Sidoli
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="MOTIVATION
5-hydroxymethylcytosine (5hmC), a crucial epigenetic mark with a significant role in regulating tissue-specific gene expression, is essential for understanding the dynamic functions of the human genome. Despite its importance, predicting 5hmC modification across the genome remains a challenging task, especially when considering the complex interplay between DNA sequences and various epigenetic factors such as histone modifications and chromatin accessibility.


RESULTS
Using tissue-specific 5hmC sequencing data, we introduce Deep5hmC, a multimodal deep learning framework that integrates both the DNA sequence and epigenetic features such as histone modification and chromatin accessibility to predict genome-wide 5hmC modification. The multimodal design of Deep5hmC demonstrates remarkable improvement in predicting both qualitative and quantitative 5hmC modification compared to unimodal versions of Deep5hmC and state-of-the-art machine learning methods. This improvement is demonstrated through benchmarking on a comprehensive set of 5hmC sequencing data collected at four developmental stages during forebrain organoid development and across 17 human tissues. Compared to DeepSEA and random forest, Deep5hmC achieves close 4 % and 17% improvement of AUROC across four forebrain developmental stages, and 6% and 27% across 17 human tissues for predicting binary 5hmC modification sites; and 8% and 22% improvement of Spearman correlation coefficient across four forebrain developmental stages, and 17% and 30% across 17 human tissues for predicting continuous 5hmC modification. Notably, Deep5hmC showcases its practical utility by accurately predicting gene expression and identifying differentially hydroxymethylated regions in a case-control study of Alzheimer's disease. Deep5hmC significantly improves our understanding of tissue-specific gene regulation and facilitates the development of new biomarkers for complex diseases.


AVAILABILITY AND IMPLEMENTATION
Deep5hmC is available via https://github.com/lichen-lab/Deep5hmC.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14adc9b4233580127e9b1670a5d719223757a6d5" target='_blank'>
              Deep5hmC: Predicting genome-wide 5-Hydroxymethylcytosine landscape via a multimodal deep learning model.
              </a>
            </td>
          <td>
            Xin Ma, Sai Ritesh Thela, Fengdi Zhao, Bing Yao, Zhexing Wen, Peng Jin, Jinying Zhao, Li Chen
          </td>
          <td>2024-08-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed75306e651376970012e59c0ec2cae9bb7cf37a" target='_blank'>
              ESRP1-mediated biogenesis of circPTPN12 inhibits hepatocellular carcinoma progression by PDLIM2/ NF-κB pathway
              </a>
            </td>
          <td>
            Yang Ji, Chuangye Ni, Yanjun Shen, Zhenggang Xu, Lei Tang, Fei Yu, Lingbang Zhu, Hao Lu, Chuanyong Zhang, Shikun Yang, Xuehao Wang
          </td>
          <td>2024-07-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="TAD boundaries are genomic elements that separate biological processes in neighboring domains by blocking DNA loops that are formed through Cohesin-mediated loop extrusion. Most TAD boundaries consist of arrays of binding sites for the CTCF protein, whose interaction with the Cohesin complex blocks loop extrusion. TAD boundaries are not fully impermeable though and allow a limited amount of inter-TAD loop formation. Based on the reanalysis of Nano-C data, a multicontact Chromosome Conformation Capture assay, we propose a model whereby clustered CTCF binding sites promote the successive stalling of Cohesin and subsequent dissociation from the chromatin. A fraction of Cohesin nonetheless achieves boundary read-through. Due to a constant rate of Cohesin dissociation elsewhere in the genome, the maximum length of inter-TAD loops is restricted though. We speculate that the DNA-encoded organization of stalling sites regulates TAD boundary permeability and discuss implications for enhancer-promoter loop formation and other genomic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7707b0397caa81eae89d125f02c4b85e10532d65" target='_blank'>
              Permeable TAD boundaries and their impact on genome-associated functions.
              </a>
            </td>
          <td>
            Li-Hsin Chang, D. Noordermeer
          </td>
          <td>2024-08-02</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="This article theoretically speculates that after replication of a pair of DNA they must be allocated to two daughter cells in a transverse division or erect division manner. Transverse division represents differentiation, while erect division represents transformation. The molecular structure determines that DNA has greater hydrophobicity than RNA. The hydrophobic modification of chromatin regulates gene activity and centromere similarity between new and old DNA. A difference in division between cancer cells and embryonic stem cells is discussed">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c6b78e19de2f6d2d4dda38ace7242257dc237cb" target='_blank'>
              On the Cancer
              </a>
            </td>
          <td>
            Xiao En Wang
          </td>
          <td>2024-08-31</td>
          <td>Journal of Cancer Research Reviews &amp; Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ubiquitin modification and alternative polyadenylation play crucial roles in the onset and progression of cancer. Hence, this study aims to comprehensively and deeply understand gene regulation and associated biological processes in lung adenocarcinoma (LUAD) by integrating both mechanisms. Alternative polyadenylation (APA)-related E3 ubiquitin ligases in LUAD were identified through multiple databases, and the association between selected genetic loci influencing gene expression (apaQTL-SNPs) and LUAD risk were evaluated through the GWAS database of the Female Lung Cancer Consortium in Asia (FLCCA). Subsequently, the interaction between RNF213 and ZBTB20, as well as their functional mechanisms in LUAD, were investigated using bioinformatics analysis, Western blot, co-immunoprecipitation, and colony formation experiments. A total of five apaQTL-SNPs (rs41301932, rs4494603, rs9890400, rs56066320, and rs41301932), located on RNF213, were significantly associated with LUAD risk (p < 0.05), and they inhibit tumor growth through ubiquitin-mediated degradation of ZBTB20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c9426c96fac5af729404fd06cbe66dceed7c43" target='_blank'>
              Crosstalk among Alternative Polyadenylation, Genetic Variants and Ubiquitin Modification Contribute to Lung Adenocarcinoma Risk
              </a>
            </td>
          <td>
            Yutong Wu, Yanqiong Yuan, Huiwen Xu, Wendi Zhang, Anhui Ning, Siqi Li, Qiong Chen, Xiaobo Tao, Gongbu Pan, Tian Tian, Lei Zhang, Minjie Chu, Jiahua Cui
          </td>
          <td>2024-07-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Microsatellite instability (MSI) is a crucial feature in cancer biology, yet its prevalence and significance in canine cancers remain largely unexplored. This study conducted a comprehensive analysis of MSI across 10 distinct canine cancer histotypes using whole-exome sequencing data from 692 tumor-normal sample pairs. MSI was detected in 64% of tumors, with prevalence varying significantly among cancer types. B-cell lymphomas exhibited the highest MSI burden, contrasting with human studies. A novel “MSI-burden” score was developed, correlating significantly with tumor mutational burden. MSI-high (MSI-H) tumors showed elevated somatic mutation counts compared to MSI-low and microsatellite stable tumors. The study identified 3632 recurrent MSI-affected genomic regions across cancer types. Notably, seven of the ten cancer types exhibited MSI-H tumors, with prevalence ranging from 1.5% in melanomas to 37% in B-cell lymphomas. These findings highlight the potential importance of MSI in canine cancer biology and suggest opportunities for targeted therapies, particularly immunotherapies. The high prevalence of MSI in canine cancers, especially in B-cell lymphomas, warrants further investigation into its mechanistic role and potential as a biomarker for prognosis and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb71f690b2dee2b9e6e72217515901fac7161789" target='_blank'>
              Comprehensive Analysis of Microsatellite Instability in Canine Cancers: Implications for Comparative Oncology and Personalized Veterinary Medicine
              </a>
            </td>
          <td>
            Eugenio Mazzone, Luca Aresu
          </td>
          <td>2024-08-27</td>
          <td>Animals</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gene conversions are broadly defined as the transfer of genetic material from a ‘donor’ to an ‘acceptor’ sequence and can happen both in meiosis and mitosis. They are a subset of non-crossover events and like crossover events, gene conversion can generate new combinations of alleles, erode linkage disequilibrium, and even counteract the mutation load by reverting germline mutations through GC-biased gene conversion. Estimating the rate of gene conversion and the distribution of gene conversion tract lengths remains challenging. Here, we present a new method for estimating tract length, rate and detection probability of non-crossover events directly in HiFi PacBio long read data. The method can be applied with data from a single individual, is unbiased even under low single nucleotide variant densities and does not necessitate any demographic or evolutionary assumptions. We apply the method to gene conversion events observed directly in Pacbio HiFI read data from a human sperm sample and find that human gene conversion tracts are shorter (mean of 50 base pairs) than estimates from yeast or Drosophila. We also estimate that typical human male gametes undergo on average 280 non-crossover events where approximately 7 are expected to become visible as gene conversions moving variants from one donor haplotype to an acceptor haplotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f463a46e1b5590cae1deadecdd92d1f8b1f8321" target='_blank'>
              Estimating gene conversion tract length and rate from PacBio HiFi data
              </a>
            </td>
          <td>
            A. P. Charmouh, Peter Porsborg Sørud, T. Bataillon, A. Hobolth, Lasse Thorup Hansen, Søren Besenbacher, S. B. Winge, K. Almstrup, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf95f1dc0f1af0cb6e8bd77dab713b5fc609919" target='_blank'>
              CircRHBDD1 promotes immune escape via IGF2BP2/PD-L1 signaling and acts as a nanotherapeutic target in gastric cancer
              </a>
            </td>
          <td>
            Yanna Li, Zhixiong Wang, Peng Gao, Danping Cao, R. Dong, Menglin Zhu, Yao Fei, X. Zuo, Juan Cai
          </td>
          <td>2024-07-30</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="During mammalian embryogenesis, both the 5-cytosine DNA methylation (5meC) landscape and three dimensional (3D) chromatin architecture are profoundly remodeled during a process known as 'epigenetic reprogramming.' An understudied aspect of epigenetic reprogramming is how the 5meC flux, per se, affects the 3D genome. This is pertinent given the 5meC-sensitivity of DNA binding for a key regulator of chromosome folding: CTCF. We profiled the CTCF binding landscape using a mouse embryonic stem cell (ESC) differentiation protocol that models embryonic 5meC dynamics. Mouse ESCs lacking DNA methylation machinery are able to exit naive pluripotency, thus allowing for dissection of subtle effects of CTCF on gene expression. We performed CTCF HiChIP in both wild-type and mutant conditions to assess gained CTCF-CTCF contacts in the absence of 5meC. We performed H3K27ac HiChIP to determine the impact that ectopic CTCF binding has on cis-regulatory contacts. Using 5meC epigenome editing, we demonstrated that the methyl-mark is able to impair CTCF binding at select loci. Finally, a detailed dissection of the imprinted Zdbf2 locus showed how 5meC-antagonism of CTCF allows for proper gene regulation during differentiation. This work provides a comprehensive overview of how 5meC impacts the 3D genome in a relevant model for early embryonic events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4286154f07173854ef3d5bb731fafdeabe579e02" target='_blank'>
              The impact of the embryonic DNA methylation program on CTCF-mediated genome regulation.
              </a>
            </td>
          <td>
            Ana Monteagudo-Sánchez, Julien Richard Albert, Margherita Scarpa, D. Noordermeer, Maxim V. C. Greenberg
          </td>
          <td>2024-08-24</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1230bf0b0e23a37cb0ef069af27c70638d7b9578" target='_blank'>
              Cell-free DNA from germline TP53 mutation carriers reflect cancer-like fragmentation patterns
              </a>
            </td>
          <td>
            Derek Wong, Maha Tageldein, Ping Luo, Erik Ensminger, Jeffrey P. Bruce, L. Oldfield, Haifan Gong, N. Fischer, B. Laverty, Vallijah Subasri, Scott Davidson, Reem Khan, A. Villani, A. Shlien, R. Kim, David Malkin, Trevor J. Pugh
          </td>
          <td>2024-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Breast cancer (BC) is the most frequently diagnosed cancer and the primary cause of cancer-related mortality in women. Treatment of triple-negative breast cancer (TNBC) remains particularly challenging due to its resistance to chemotherapy and poor prognosis. Extensive research efforts in BC screening and therapy have improved clinical outcomes for BC patients. Therefore, identifying reliable biomarkers for TNBC is of great clinical importance. Here, we found that tyrosine aminotransferase (TAT) expression was significantly reduced in BC and strongly correlated with the poor prognosis of BC patients, which distinguished BC patients from normal individuals, indicating that TAT is a valuable biomarker for early BC diagnosis. Mechanistically, we uncovered that methylation of the TAT promoter was significantly increased by DNA methyltransferase 3 (DNMT3A/3B). In addition, reduced TAT contributes to DNA replication and cell cycle activation by regulating homologous recombination repair and mismatch repair to ensure genomic stability, which may be one of the reasons for TNBC resistance to chemotherapy. Furthermore, we demonstrated that Diazinon increases TAT expression as an inhibitor of DNMT3A/3B and inhibits the growth of BC by blocking downstream pathways. Taken together, we revealed that TAT is silenced by DNMT3A/3B in BC, especially in TNBC, which promotes the proliferation of tumor cells by supporting DNA replication, activating cell cycle, and enhancing DNA damage repair. These results provide fresh insights and a theoretical foundation for the clinical diagnosis and treatment of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a77f352d33a7a2b9ba7ce61fe82a2e23ebd12e" target='_blank'>
              Identification of TAT as a Biomarker Involved in Cell Cycle and DNA Repair in Breast Cancer
              </a>
            </td>
          <td>
            Fei Xie, Saiwei Hua, Yajuan Guo, Taoyuan Wang, Changliang Shan, Lianwen Zhang, 
          </td>
          <td>2024-08-30</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cbd49dc4f182de3dedf2bd9170f18d407483728" target='_blank'>
              VolcanoSV enables accurate and robust structural variant calling in diploid genomes from single-molecule long read sequencing
              </a>
            </td>
          <td>
            Can Luo, Y. Liu, X. Zhou
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25bc89ba9898a52d88ff5aa506e261a761286bfb" target='_blank'>
              piR-823 tale as emerging cancer-hallmark molecular marker in different cancer types: a step-toward ncRNA-precision.
              </a>
            </td>
          <td>
            Fatma H Shaker, E. Sanad, H. Elghazaly, Shih-Min Hsia, N. Hamdy
          </td>
          <td>2024-08-05</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Early embryos often have relatively unstructured chromatin that lacks active and inactive domains typical of differentiated cells. In many species, these regulatory domains are established during zygotic genome activation (ZGA). In Drosophila, ZGA occurs after 13 fast, reductive, syncytial nuclear divisions during which the nuclear to cytoplasmic (N/C) ratio grows exponentially. These divisions incorporate maternally-loaded, cytoplasmic pools of histones into chromatin. Previous work found that chromatin incorporation of replication-coupled histone H3 decreases while its variant H3.3 increases in the cell cycles leading up to ZGA. In other cell types, H3.3 is associated with sites of active transcription as well as heterochromatin, suggesting a link between H3.3 incorporation and ZGA. Here, we examine the factors that contribute to H3.3 incorporation at ZGA. We identify a more rapid decrease in the nuclear availability of H3 than H3.3 over the final pre-ZGA cycles. We also observe an N/C ratio-dependent increase in H3.3 incorporation in mutant embryos with non-uniform local N/C ratios. We find that chaperone binding, not gene expression, controls incorporation patterns using H3/H3.3 chimeric proteins at the endogenous H3.3A locus. We test the specificity of the H3.3 chaperone pathways for H3.3 incorporation using Hira (H3.3 chaperone) mutant embryos. Overall, we propose a model in which local N/C ratios and specific chaperone binding regulate differential incorporation of H3.3 during ZGA. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddcf3e70edf5f1afb44643d1130dfd3762c01db0" target='_blank'>
              Local nuclear to cytoplasmic ratio regulates chaperone-dependent H3 variant incorporation during zygotic genome activation
              </a>
            </td>
          <td>
            Anusha D. Bhatt, Madeleine G. Brown, Aurora B. Wackford, Yuki Shindo, Amanda A. Amodeo
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7098328b990f26128e9d5dd53699039600c56625" target='_blank'>
              Noncanonical formation of SNX5 gene-derived circular RNA regulates cancer growth
              </a>
            </td>
          <td>
            Yi-Tung Chen, Hui-Ju Tsai, C. Kan, Chung-Pei Ma, Hui-Wen Chen, I. Chang, Hsuan Liu, Chih-Ching Wu, Wei-Yun Chu, Ya-Chun Wu, Kai-Ping Chang, Jau-Song Yu, B. Tan
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e1918f5013376fd130c9f594868a7f384598c2" target='_blank'>
              HMOX1: A pivotal regulator of prognosis and immune dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Jinfa Huang, Ruiwan Tan
          </td>
          <td>2024-08-30</td>
          <td>BMC Women's Health</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the genetic regulation, for example, gene expressions (GEs) by copy number variations and methylations, is crucial to uncover the development and progression of complex diseases. Advancing from early studies that are mostly focused on homogeneous groups of patients, some recent studies have shifted their focus toward different patient groups, explored their commonalities and differences, and led to insightful findings. However, the analysis can be very challenging with one GE possibly regulated by multiple regulators and one regulator potentially regulating the expressions of multiple genes, leading to two distinct types of commonalities/differences in the patterns of genetic regulation. In addition, the high dimensionality of both sides of regulation poses challenges to computation. In this study, we develop a two-way fusion integrative analysis approach, which innovatively applies two fusion penalties to simultaneously identify commonalities/differences in the regulated pattern of GEs and regulating pattern of regulators, and adopt a Huber loss function to accommodate the possible data contamination. Moreover, a simple yet efficient iterative optimization algorithm is developed, which does not need to introduce any auxiliary variables and extra tuning parameters and is guaranteed to converge to a globally optimal solution. The advantages of the proposed approach are demonstrated in extensive simulations. The analysis of The Cancer Genome Atlas data on melanoma and lung cancer leads to interesting findings and satisfactory prediction performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c30f4bc0f5f2a42888c972380cc183c3c5a04c6" target='_blank'>
              Unveiling Commonalities and Differences in Genetic Regulations via Two-Way Fusion.
              </a>
            </td>
          <td>
            Biao Mei, Yu Jiang, Yifan Sun
          </td>
          <td>2024-08-12</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="“Telomere DNA and ribosomal DNA co-regulation model for cell senescence” suggests that cell senescence and Hayflick limits are co-regulated by telomere and rDNA. To this end, by partially knockout 45S rDNA, we found that common aging markers increased and cell passage times decreased, preliminarily proving that cell aging and Hayflick limits are co-regulated by 45S rDNA and telomeres. We also found that telomere length and 45S rDNA copy number of hESC and hiPSC are significantly increased, which may be the reason why hESC and hiPSC remain immortal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e1973b7270ad9e9b64dd522eff7ae60852ce9" target='_blank'>
              Experimental verification on “Telomere DNA and ribosomal DNA co-regulation model for cell senescence”
              </a>
            </td>
          <td>
            , 
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Approximately 40% ERα-positive breast cancer patients suffer from therapeutic resistance to tamoxifen. Although reduced ERα level is the major cause of tamoxifen resistance, the underlying mechanisms remain elusive. Here, we report that FRMD8 raises the level of ERα at both transcriptional and post-translational layers. FRMD8 deficiency in MMTV-Cre+; Frmd8fl/fl; PyMT mice accelerates mammary tumor growth and loss of luminal phenotype, and confers tamoxifen resistance. Single-cell RNA profiling reveals that Frmd8 loss decreases the proportion of hormone-sensing differentiated epithelial cells and downregulates the levels of ERα. Mechanically, on one hand, loss of FRMD8 inhibits ESR1 transcription via suppressing the expression of FOXO3A, a transcription factor of ESR1. On the other hand, FRMD8 interacts both with ERα and UBE3A, and disrupts the interaction of UBE3A with ERα, thereby blocking UBE3A-mediated ERα degradation. In breast cancer patients, FRMD8 gene promoter is found hypermethylated and low level of FRMD8 predicts poor prognosis. Therefore, FRMD8 is an important regulator of ERα and may control therapeutic sensitivity to tamoxifen in ERα-positive breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0f03ffffae72049184669f9bd59cfb3a196499" target='_blank'>
              Loss function of tumor suppressor FRMD8 confers resistance to tamoxifen therapy via a dual mechanism
              </a>
            </td>
          <td>
            Weijie Wu, Miao Yu, Qianchen Li, Yiqian Zhao, Lei Zhang, Yi Sun, Zhenbin Wang, Yuqing Gong, Wenjing Wang, Chenying Liu, Jing Zhang, Yan Tang, Xiaojie Xu, Xiaojing Guo, Jun Zhan, Hongquan Zhang
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recent exome-wide association studies have explored the role of coding variants in breast cancer risk, highlighting the role of rare variants in multiple genes including BRCA1, BRCA2, CHEK2, ATM and PALB2, as well as new susceptibility genes e.g., MAP3K1. These genes, however, explain a small proportion of the missing heritability of the disease. Much of the missing heritability likely lies in the non-coding genome. We evaluated the role of rare variants in the 5' and 3' untranslated regions (UTRs) of 18,676 genes, and 35,201 putative promoter regions, using whole-genome sequencing data from UK Biobank on 8,001 women with breast cancer and 92,534 women without breast cancer. Burden tests and SKAT-O tests were performed in UTR and promoter regions. For UTR regions of 35 putative breast cancer susceptibility genes, we additionally performed a meta-analysis with a large breast cancer case-control dataset. Associations for 8 regions at P<0.0001 were identified, including several with known roles in tumorigenesis. The strongest evidence of association was for variants in the 5' UTR of CDK5R1 (P=8.5x10-7). These results highlight the potential role of non-coding regulatory regions in breast cancer susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202db6ee770b7ecafdb01fbe1ede9ead1c66b51f" target='_blank'>
              Analysis of variants in untranslated and promoter regions and breast cancer risk using whole genome sequencing data.
              </a>
            </td>
          <td>
            Abubakar 2, U. Ahearn 2, L. Andrulis, Antonis 4, Antoniou 1, N. Bogdanova, Stig E. Bojesen 8-10 5-7, M. Bolla, Hiltrud Brauch 11-13, Nicola J. Jenny Chang-Claude, 16 Kamila Chang-Claude 15, Thilo Dörk Czene 17, D. G. 6, A. 19Peter, Jonine D Fasching 20, 21 22 Figueroa 2, 23 Flyger, E. Gardner, Anna González-Neira 25, Eric Guénel 26, 28 Per Hall 27, Mikael Hartman 29, Maartje J. Hooning 30-32, Anna 33, 35 Elza 34, 37 Vessela N 36, Jingmei Kristensen 38, A. Lindblom, A. Lophatananon, Arto Mannermaa 44-46, Manoukian, R. Milne, Rocio Nuñez-Torres 51, 53 Mihalis Obi 52, Panayiotidis, Sue K. Park 55-57, 58 R.B. Perry 24, U. Muhammad, 60 Saloustros 61 59, E. Sawyer, K. Marjanka, Melissa C. Southey 63-65, 66 Amanda B. Spurdle 50, Diana Torres 59, Jacques Qin Wang 1, Soo Hwang Teo 69, 72 AlisonM.Dunning, 73 PeterDevilee, F. Douglas, 72 . Easton 1, 1. Centre, Cancer Genetic, Epidemiology
          </td>
          <td>2024-07-05</td>
          <td>None</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="In recent years, the detection and analysis of circulating tumor DNA (ctDNA) have emerged as a new focus in the field of cancer research, particularly in the early diagnosis of hepatocellular carcinoma (HCC) and monitoring of therapeutic efficacy. ctDNA, which refers to cell-free DNA fragments released into the bloodstream from tumor cells upon cell death or shedding, carries tumor-specific genetic and epigenetic alterations, thereby providing a non-invasive approach for cancer diagnosis and prognosis. The concentration of ctDNA in the blood is higher compared to that in healthy individuals or other liquid biopsies from early-stage cancers, which is closely associated with the early diagnosis and comprehensive sequencing studies of HCC. Recent studies have indicated that sequential ctDNA analysis in patients receiving primary or adjuvant therapy for HCC can detect treatment resistance and recurrence before visible morphological changes in the tumor, making it a valuable basis for rapid adjustment of treatment strategies. However, this technology is continuously being optimized and improved. Challenges such as enhancing the accuracy of ctDNA sequencing tests, reducing the burden of high-throughput sequencing on a large number of samples, and controlling variables in the assessment of the relationship between ctDNA concentration and tumor burden, need to be addressed. Overall, despite the existing challenges, the examination and analysis of ctDNA have opened up new avenues for early diagnosis and therapeutic efficacy monitoring in hepatocellular carcinoma, expanding the horizons of this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe795a1ff20add2c9eb3d87e4f8cefe083305b7e" target='_blank'>
              Circulating tumor DNA mutation analysis: advances in its application for early diagnosis of hepatocellular carcinoma and therapeutic efficacy monitoring
              </a>
            </td>
          <td>
            Jing Yang, Na Lin, Miaomiao Niu, Boshu Yin
          </td>
          <td>2024-07-19</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In light of the post-genomic era, epigenetics brings about an opportunity to better understand how the molecular machinery works and is led by a complex dynamic set of mechanisms, often intricate and complementary in many aspects. In particular, epigenetics links developmental biology and genetics, as well as many other areas of knowledge. The present work highlights substantial scopes and relevant discoveries related to the development of the term from its first notions. To our understanding, the concept of epigenetics needs to be revisited, as it is one of the most relevant and multifaceted terms in human knowledge. To redirect future novel experimental or theoretical efforts, it is crucial to compile all significant issues that could impact human and ecological benefit in the most precise and accurate manner. In this paper, the reader can find one of the widest compilations of the landmarks and epistemic considerations of the knowledge of epigenetics across the history of biology from the earliest epigenetic formulation to genetic determinism until the present. In the present work, we link the current body of knowledge and earlier pre-genomic concepts in order to propose a new definition of epigenetics that is faithful to its regulatory nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d95143e930f9759529d2eb02d359dc5d641be6" target='_blank'>
              Revisiting Epigenetics Fundamentals and Its Biomedical Implications
              </a>
            </td>
          <td>
            T. Meza-Menchaca, Arnulfo Albores-Medina, Alma Heredia-Mendez, E. Ruíz-May, Jorge Ricaño-Rodríguez, Veronica Gallegos-García, Adriana Esquivel, Giancarlo Vettoretti-Maldonado, Alma Campos-Parra
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In developing B cells, V(D)J gene recombination is initiated by the RAG1/2 endonuclease complex, introducing double-stranded DNA breaks (DSBs) in V, D, and J genes and resulting in the formation of the hypervariable parts of immunoglobulins (Ig). Persistent or aberrant RAG1/2 targeting is a potential threat to genome integrity. While RAG1 and RAG2 have been shown to bind various regions genome-wide, the in vivo off-target DNA damage instigated by RAG1/2 endonuclease remains less well understood. In the current study, we identified regions containing RAG1/2-induced DNA breaks in mouse pre-B cells on a genome-wide scale using a global DNA DSB detection strategy. We detected 1489 putative RAG1/2-dependent DSBs, most of which were located outside the Ig loci. DNA sequence motif analysis showed a specific enrichment of RAG1/2-induced DNA DSBs at GA- and CA-repeats and GC-rich motifs. These findings provide further insights into RAG1/2 off-target activity. The ability of RAG1/2 to introduce DSBs on the non-Ig loci during the endogenous V(D)J recombination emphasizes its genotoxic potential in developing lymphocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c3ee9eea0d95b181f8a61de27812d8926453b6" target='_blank'>
              RAG1/2 induces double-stranded DNA breaks at non-Ig loci in the proximity of single sequence repeats in developing B cells.
              </a>
            </td>
          <td>
            Katarina Ochodnicka-Mackovicova, Michal Mokry, Martin Haagmans, Ted E. Bradley, C. V. van Noesel, J. Guikema
          </td>
          <td>2024-07-24</td>
          <td>European journal of immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Chromosomal fusions play an integral role in genome remodeling and karyotype evolution. Fusions that join a sex chromosome to an autosome are particularly abundant across the tree of life. However, previous models on the establishment of such fusions have not accounted for the physical structure of the chromosomes. We predict a fusion joining an autosome to the pseudoautosomal region (PAR) of a sex chromosome will not remain stable, and the fusion will switch from the X to the Y chromosome each generation due to recombination. We have produced a forward-time population genetic simulation to explore the outcomes of fusions to both the PAR and non-PAR of sex chromosomes. The model can simulate the fusion of an autosome containing a sexually antagonistic locus to either the PAR or non-PAR end of a sex chromosome. Our model is diploid, two-locus, and biallelic. Our results show a clear pattern where fusions to the non-PAR are favored in the presence of sexual antagonism, whereas fusions to the PAR are disfavored in the presence of sexual antagonism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417dce982562c863a7e38c28a474ba15591636ef" target='_blank'>
              Worse than nothing at all: the inequality of fusions joining autosomes to the PAR and non-PAR portions of sex chromosomes
              </a>
            </td>
          <td>
            Kayla T. Wilhoit, Emmarie P. Alexander, Heath Blackmon
          </td>
          <td>2024-07-23</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tight control of the mitotic phase of differentiation is crucial to prevent tumourigenesis while securing tissue homeostasis. In the Drosophila female germline, differentiation involves precisely four mitotic divisions, and accumulating evidence suggests that bag-of- marbles (bam), the initiator of differentiation, is also involved in controlling the number of divisions. To test this hypothesis, we depleted Bam from differentiating cells and found a reduced number of mitotic divisions. We examined the regulation of Bam using RNA imaging methods and found that the bam 3’ UTR conveys instability to the transcript in the 8-cell cyst and early 16-cell cyst. We show that the RNA binding protein, Rbp9, is responsible for timing bam mRNA decay. Rbp9 itself is part of a sequential cascade of RNA binding proteins activated downstream of Bam, and we show that it is regulated through a change in transcription start site, driven by Rbfox1. Altogether, we propose a model in which Bam expression at the dawn of differentiation initiates a series of events that eventually terminates the Bam expression domain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ac40f2eb322caeece920877298412a69c4a5a9" target='_blank'>
              Destabilisation of bam transcripts terminates the mitotic phase of Drosophila female germline differentiation
              </a>
            </td>
          <td>
            Tamsin J. Samuels, Elizabeth J Torley, Emily L Naden, Phoebe E Blair, Frankjel A Hernandez Frometa, Felipe Karam Teixeira
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/798b264fbd39508ed9588f5472e886b8b6303884" target='_blank'>
              Navigating the complexity of p53-DNA binding: implications for cancer therapy
              </a>
            </td>
          <td>
            Kelly M. Thayer, Sean Stetson, Fernando Caballero, Christopher Chiu, I. Han
          </td>
          <td>2024-07-11</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="The p53 genome is a gene that has truly taken the scientific world by storm. Once referred to as an oncogene, p53 has proved itself to be more complex and dubious time and time again. This paper will talk about the beginning doubts and queries revolving around p53 and the discoveries that led to it being referred to as a tumour suppressor gene, rather than being mislabeled as an oncogene. It will touch upon the importance that p53 brings to our daily lives such as repairing the DNA, maintaining the cell cycle and starting apoptosis, as well as highlight the differences between the gene’s two types: wild-type p53 and mutant p53. Furthermore, it will talk about how a slight mutation in p53, can cause it to form many protein isoforms, a leading contributing factor in cancers. It will also mention the effects post-translational modification will have on the two types of p53. In addition to this, it will delve into the negative and potentially harmful effects that p53 could have on our bodies. Finally, it will also talk about p53-mediated therapy and targeting mutant p53 with autophagy, which are 2 processes that have shown successful outcomes in reducing the complications caused by mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc70ab96424bdf601f1a4ba7f9b8f2ff8a5b550" target='_blank'>
              P53: Our Saviour and Our Destroyer
              </a>
            </td>
          <td>
            Narayani Haldar
          </td>
          <td>2024-07-23</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The emergence of cancers involves numerous coding and non-coding genes. Understanding the contribution of non-coding RNAs (ncRNAs) to the cancer neighborhood is crucial for interpreting the interaction between molecular markers of cancer. However, there is a lack of systematic studies on the involvement of ncRNAs in the cancer neighborhood. In this paper, we construct an interaction network which encompasses multiple genes. We focus on the fundamental topological indicator, namely connectivity, and evaluate its performance when applied to cancer-affected genes using statistical indices. Our findings reveal that ncRNAs significantly enhance the connectivity of affected genes and mediate the inclusion of more genes in the cancer module. To further explore the role of ncRNAs in the network, we propose a connectivity-based method which leverages the bridging function of ncRNAs across cancer-affected genes and reveals the non-coding RNAs extended omnigenic module (NeOModule). Topologically, this module promotes the formation of cancer patterns involving ncRNAs. Biologically, it is enriched with cancer pathways and treatment targets, providing valuable insights into disease relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c53c94851580faedc72696e1e649d6cf0d594b0" target='_blank'>
              Non-Coding RNAs Extended Omnigenic Module of Cancers
              </a>
            </td>
          <td>
            Jie Li, Bingbo Wang, Xiujuan Ma
          </td>
          <td>2024-07-27</td>
          <td>Entropy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Cellular angiofibroma, a rare benign mesenchymal neoplasm, is classified within the 13q/RB1 family of tumors due to morphological, immunohistochemical, and genetic similarities with spindle cell lipoma. Here, genetic data reveal pathogenetic heterogeneity in cellular angiofibroma.


METHODS
Three cellular angiofibromas were studied using G-banding/Karyotyping, array comparative genomic hybridization, RNA sequencing, and direct cycling sequencing.


RESULTS
The first tumor carried a del(13)(q12) together with heterozygous loss and minimal expression of the RB1 gene. Tumors two and three displayed chromosome 8 abnormalities associated with chimeras of the pleomorphic adenoma gene 1 (PLAG1). In tumor 2, the cathepsin B (CTSB) fused to PLAG1 (CTSB::PLAG1) while in tumor 3, the mir-99a-let-7c cluster host gene (MIR99AHG) fused to PLAG1 (MIR99AHG::PLAG1), both leading to elevated expression of PLAG1 and insulin growth factor 2.


CONCLUSION
This study uncovers two genetic pathways contributing to the pathogenetic heterogeneity within cellular angiofibromas. The first aligns with the 13q/RB1 family of tumors and the second involves PLAG1-chimeras. These findings highlight the diverse genetic landscape of cellular angiofibromas, providing insights into potential diagnostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72129daa715b46d571770497bea5325bf92afaeb" target='_blank'>
              Genetic Heterogeneity in Cellular Angiofibromas.
              </a>
            </td>
          <td>
            Ioannis Panagopoulos, K. Andersen, I. Lobmaier, M. Lund-Iversen
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fe3c701c57ad11bfba55ea29642c8877198c85" target='_blank'>
              Single-cell analysis identified PDIA3 as regulator of malignant characteristics and macrophage function in human cancers.
              </a>
            </td>
          <td>
            Wantao Wu, Gang Peng, Kaiyue Wang, Yijian Yang, Zhikun Liu, Gelei Xiao
          </td>
          <td>2024-08-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary It is known that both B cells and myeloblasts from acute myeloid leukemia (AML) express immunoglobulin (Ig). However, the difference between Ig from myeloblasts and B cells has not been explored; therefore, the function and significance of AML-Ig are not well known. Here, we performed 5′ RACE-related PCR coupled with PacBio sequencing to analyze the Ig repertoire of Chinese AML patients. Myeloblasts expressed all five classes of IgH, especially Igγ, with a high expression frequency. Compared with B-Ig in the same AML patient, AML-Ig showed different biased V(D)J usages and mutation patterns. More importantly, mutations of AML-Ig frequently occurred at the sites of post-translational modification. In summary, our results showed that AML-Ig had different sequence characteristics than those of B-Ig and that the unique V(D)J usages of AML-Ig may serve as a novel biomarker for personalized minimal/measurable residual disease monitoring and therapeutic targets in AML patients. Abstract It is common knowledge that immunoglobulin (Ig) is produced by B lymphocytes and mainly functions as an antibody. However, it has been shown recently that myeloblasts from acute myeloid leukemia (AML) could also express Ig and that AML-Ig played a role in leukemogenesis and AML progression. The difference between Ig from myeloblasts and B cells has not been explored. Studying the characteristics of the Ig repertoire in myeloblasts and B cells will be helpful to understand the function and significance of AML-Ig. We performed 5′ RACE-related PCR coupled with PacBio sequencing to analyze the Ig repertoire in myeloblasts and B cells from Chinese AML patients. Myeloblasts expressed all five classes of IgH, especially Igγ, with a high expression frequency. Compared with B-Ig in the same patient, AML-Ig showed different biased V(D)J usages and mutation patterns. In addition, the CDR3 length distribution of AML-Ig was significantly different from those of B-Ig. More importantly, mutations of AML-IgH, especially Igμ, Igα, and Igδ, were different from that of B-IgH in each AML patient, and the mutations frequently occurred at the sites of post-translational modification. AML-Ig has distinct characteristics of variable regions and mutations, which may have implications for disease monitoring and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b72edfacf57bf0df201db104f6a0b0c301157e" target='_blank'>
              A Comparative Analysis of the Immunoglobulin Repertoire in Leukemia Cells and B Cells in Chinese Acute Myeloid Leukemia by High-Throughput Sequencing
              </a>
            </td>
          <td>
            Huige Yan, Lina Wu, Pingzhang Wang, Miaoran Xia, Zhan Shi, Xinmei Huang, Sha Yin, Qian Jiang, C Cameron Yin, Xiangyu Zhao, Xiaoyan Qiu
          </td>
          <td>2024-08-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>