<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-09 06:32:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>22</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>52</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>276</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>237</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Sequencing-based mapping of ensemble pairwise interactions among regulatory elements support the existence of topological assemblies known as promoter-enhancer hubs or cliques in cancer. Yet, prevalence, regulators, and functions of promoter-enhancer hubs in individual cancer cells remain unclear. Here, we systematically integrated functional genomics, transcription factor screening, and optical mapping of promoter-enhancer interactions to identify key promoter-enhancer hubs, examine heterogeneity of their assembly, determine their regulators, and elucidate their role in gene expression control in individual triple negative breast cancer (TNBC) cells. Optical mapping of individual SOX9 and MYC alleles revealed the existence of frequent multiway interactions among promoters and enhancers within spatial hubs. Our single-allele studies further demonstrated that lineage-determining SOX9 and signaling-dependent NOTCH1 transcription factors compact MYC and SOX9 hubs. Together, our findings suggest that promoter-enhancer hubs are dynamic and heterogeneous topological assemblies, which are controlled by oncogenic transcription factors and facilitate subtype-restricted gene expression in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8df926269367eb0273d95bc28b249a4b8bb66d" target='_blank'>
              Oncogenic transcription factors instruct promoter-enhancer hubs in individual triple negative breast cancer cells
              </a>
            </td>
          <td>
            Jingru Zhao, Yeqiao Zhou, Ilias Tzelepis, Noah G Burget, Junwei Shi, R. Faryabi
          </td>
          <td>2024-08-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dec3a33e2b0139c6a02f10afa4c9e80abbaad7a" target='_blank'>
              Extrachromosomal circular DNA promotes prostate cancer progression through the FAM84B/CDKN1B/MYC/WWP1 axis
              </a>
            </td>
          <td>
            Wei Jin, Zhenqun Xu, Yan Song, Fangjie Chen
          </td>
          <td>2024-07-12</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b93e43a1982115416b67edb36296e1d7dd7f71fe" target='_blank'>
              The DNA methylome of pediatric brain tumors appears shaped by structural variation and predicts survival
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2024-08-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W. Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. A. Awuah, Vivek Sanker, H. Bharadwaj, Nicholas Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, Sama Hesham Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Repetitive DNA sequences can form non-canonical structures such as H-DNA which is an intramolecular triplex DNA structure. The new Telomere-to-Telomere (T2T) genome assembly for the human genome has eliminated gaps, enabling the examination of highly repetitive regions including centromeric and pericentromeric repeats and ribosomal DNA arrays. This gapless assembly allows for the examination of the distribution of H-DNA sequences in parts of the human genome that were not previously annotated. We find that H-DNA appears once every 30,000 bps in the human genome. Its distribution is highly inhomogeneous with H-DNA motif hotspots being detectable in acrocentric chromosomes. Ribosomal DNA arrays in acrocentric chromosomes are the genomic element with the highest H-DNA enrichment, with 13.22% of total H-DNA motifs being found in ribosomal DNA arrays, representing a 42.65-fold enrichment over what would be expected by chance. Across the acrocentric chromosomes we report that 55.87% of all H-DNA motifs found in these chromosomes are in rDNA array loci. The H-DNA motifs are primarily found in the intergenic spacer regions of the ribosomal DNA arrays, generating repeated clusters. We also discover that binding sites for PRDM9, a protein that regulates the formation of double-strand breaks and determines the meiotic recombination hotspots in humans and most mammals, are over 5-fold enriched for H-DNA motifs. Finally, we provide evidence that our findings are consistent in other non-human great ape genomes. We conclude that ribosomal DNA arrays are the most enriched genomic loci for H-DNA sequences in human and other great ape genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec5edf88573fda646ad1eb9d23c9425162a3d2f" target='_blank'>
              Ribosomal DNA arrays are the most H-DNA rich element in the human genome
              </a>
            </td>
          <td>
            Nikol Chantzi, Michail Patsakis, Akshatha Nayak, Austin Montgomery, I. Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e11890b80227e402b42551347179de59480a8fc" target='_blank'>
              HMGA1 orchestrates chromatin compartmentalization and sequesters genes into 3D networks coordinating senescence heterogeneity
              </a>
            </td>
          <td>
            Ioana Olan, Masami Ando-Kuri, Aled J. Parry, Testuya Handa, S. Schoenfelder, Peter Fraser, Yasuyuki Ohkawa, Hiroshi Kimura, M. Narita, Masashi Narita
          </td>
          <td>2024-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="H3K36 methylation is emerging as a key epigenetic modification with strong implications in genetic disease and cancer. However, the mechanisms through which H3K36me impacts the epigenome and asserts its functional consequences are far from understood. Here, we use mouse mesenchymal stem cell lines with successive knockouts of the H3K36 methyltransferases: NSD1, NSD2, SETD2, NSD3, and ASH1L, which result in progressive depletion of H3K36me and its complete absence in quintuple knockout cells, to finely dissect the role of H3K36me2 in shaping the epigenome and transcriptome. We show that H3K36me2, which targets active enhancers, is important for maintaining enhancer activity, and its depletion results in downregulation of enhancer-dependent genes. We demonstrate the roles of H3K36me2/3 in preventing the invasion of gene bodies by the repressive H3K27me modifications. Finally, we observe a previously undescribed relationship between H3K36me and H3K9me3: Following the depletion of H3K36me2, H3K9me3 is redistributed away from large heterochromatic domains and towards euchromatin. This results in a drastic decompartmentalization of the genome, weakening the boundaries between active and inactive compartments, and a catastrophic loss of long-range inter-compartment interactions. By studying cells totally devoid of H3K36 methyltransferase activity, we uncover a broad range of crucial functions of H3K36me in maintaining epigenome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c996ce7c85d697b9c8fb195bf8183e8099242ceb" target='_blank'>
              H3K36 Methylation - a Guardian of Epigenome Integrity
              </a>
            </td>
          <td>
            Reinnier Padilla, Gerry A. Shipman, Cynthia Horth, E. Bareke, Jacek Majewski
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin within the nucleus is crucial for gene regulation. However, the 3D architectural features that coordinate the activation of an entire chromosome remain largely unknown. We introduce an omics method, RNA-associated chromatin DNA-DNA interactions, that integrates RNA polymerase II (RNAPII)–mediated regulome with stochastic optical reconstruction microscopy to investigate the landscape of noncoding RNA roX2-associated chromatin topology for gene equalization to achieve dosage compensation. Our findings reveal that roX2 anchors to the target gene transcription end sites (TESs) and spreads in a distinctive boot-shaped configuration, promoting a more open chromatin state for hyperactivation. Furthermore, roX2 arches TES to transcription start sites to enhance transcriptional loops, potentially facilitating RNAPII convoying and connecting proximal promoter-promoter transcriptional hubs for synergistic gene regulation. These TESs cluster as roX2 compartments, surrounded by inactive domains for coactivation of multiple genes within the roX2 territory. In addition, roX2 structures gradually form and scaffold for stepwise coactivation in dosage compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc0d29c3125cfd0f353817880f4b6081993ce9" target='_blank'>
              3D chromatin structures associated with ncRNA roX2 for hyperactivation and coactivation across the entire X chromosome
              </a>
            </td>
          <td>
            S. Tian, Yang Yang, Duo Ning, Ke Fang, Kai Jing, Guangyu Huang, Yewen Xu, Pengfei Yin, Haibo Huang, Gengzhan Chen, Yuqing Deng, Shaohong Zhang, Zhimin Zhang, Zhen-Xia Chen, Tong Gao, Wei Chen, Guoliang Li, Ruilin Tian, Yijun Ruan, Yiming Li, Meizhen Zheng
          </td>
          <td>2024-07-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There are two key signatures of pediatric cancers: (a) higher prevalence of germline alterations and (b) heterogeneity in alteration types. Recent population-based assessments have demonstrated that children with birth defects (BDs) are more likely to develop cancer even without chromosomal anomalies; therefore, explorations of genetic alterations in children with BDs and cancers could provide new insights into the underlying mechanisms for pediatric tumor development. We performed whole-genome sequencing (WGS) on blood-derived DNA for 1556 individuals without chromosomal anomalies, including 454 BD probands with at least one type of malignant tumor, 757 cancer-free children with BDs, and 345 healthy individuals, focusing on copy number variation (CNV) analysis. Roughly half of the children with BD-cancer have CNVs that are not identified in BD-only/healthy individuals, and CNVs are not evenly distributed among these patients. Strong heterogeneity was observed, with a limited number of cancer predisposition genes containing CNVs in more than three patients. Moreover, functional enrichments of genes with CNVs showed that dozens of patients have variations related to the same biological pathways, such as deletions of genes with neurological functions and duplications of immune response genes. Phenotype clustering uncovered recurrences of patients with sarcoma: A notable enrichment was observed involving non-coding RNA regulators, showing strong signals related to growth and cancer regulations in functional analysis. In conclusion, we conducted one of the first genomic studies exploring the impact of CNVs on cancer development in children with BDs, unveiling new insights into the underlying biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0421f07c60f2733c310940e72c92010ab69f33fb" target='_blank'>
              Copy number variations contribute to malignant tumor development in children with serious birth defects.
              </a>
            </td>
          <td>
            Yichuan Liu, J. Glessner, Hui-Qi Qu, X. Chang, H. Qiu, Tiancheng Wang, F. Mentch, H. Hakonarson
          </td>
          <td>2024-08-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="An obligate step in the life cycle of HIV-1 and other retroviruses is the establishment of the provirus in target cell chromosomes. Transcriptional regulation of proviruses is complex, and understanding the mechanisms underlying this regulation has ramifications for fundamental biology, human health, and gene therapy implementation. The three core components of the Human Silencing Hub (HUSH) complex, TASOR, MPHOSPH8 (MPP8), and PPHLN1 (Periphilin 1), were identified in forward genetic screens for host genes that repress provirus expression. Subsequent loss-of-function screens revealed accessory proteins that collaborate with the HUSH complex to silence proviruses in particular contexts. To identify proteins associated with a HUSH complex-repressed provirus in human cells, we developed a technique, Provirus Proximal Proteomics, based on proximity labeling with C-BERST (dCas9-APEX2 biotinylation at genomic elements by restricted spatial tagging). Our screen exploited a lentiviral reporter that is silenced by the HUSH complex in a manner that is independent of the integration site in chromatin. Our data reveal that proviruses silenced by the HUSH complex are associated with DNA repair, mRNA processing, and transcriptional silencing proteins, including L3MBTL2, a member of the non-canonical polycomb repressive complex 1.6 (PRC1.6). A forward genetic screen confirmed that PRC1.6 components L3MBTL2 and MGA contribute to HUSH complex-mediated silencing. PRC1.6 was then shown to silence HUSH-sensitive proviruses in a promoter-specific manner. Genome wide profiling showed striking colocalization of the PRC1.6 and HUSH complexes on chromatin, primarily at sites of active promoters. Finally, PRC1.6 binding at a subset of genes that are silenced by the HUSH complex was dependent on the core HUSH complex component MPP8. These studies offer new tools with great potential for studying the transcriptional regulation of proviruses and reveal crosstalk between the HUSH complex and PRC1.6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6151d660aa506b4ad3a4c66d40fa26e99798e8" target='_blank'>
              PRC1.6 localizes on chromatin with the human silencing hub (HUSH) complex for promoter-specific silencing
              </a>
            </td>
          <td>
            Tom S Rodriguez, Leonid A Yurkovetskiy, Karthika Nagalekshmi, Chin Hung Oscar Lam, Eva Jazbec, Stacy A. Maitland, S. Wolfe, E. Sontheimer, Jeremy Luban
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer molecularly characterized by arrested myogenesis. The defining genetic driver, PAX3::FOXO1, functions as a chimeric gain-of-function transcription factor. An incomplete understanding of PAX3::FOXO1’s in vivo epigenetic mechanisms has hindered therapeutic development. Here, we establish a PAX3::FOXO1 zebrafish injection model and semi-automated ChIP-seq normalization strategy to evaluate how PAX3::FOXO1 initially interfaces with chromatin in a developmental context. We investigated PAX3::FOXO1’s recognition of chromatin and subsequent transcriptional consequences. We find that PAX3::FOXO1 interacts with inaccessible chromatin through partial/homeobox motif recognition consistent with pioneering activity. However, PAX3::FOXO1-genome binding through a composite paired-box/homeobox motif alters chromatin accessibility and redistributes H3K27ac to activate neural transcriptional programs. We uncover neural signatures that are highly representative of clinical rhabdomyosarcoma gene expression programs that are enriched following chemotherapy. Overall, we identify partial/homeobox motif recognition as a new mode for PAX3::FOXO1 pioneer function and identify neural signatures as a potentially critical PAX3::FOXO1 tumor initiation event.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34ac92eb278acb6b0ddaf9def7a0af29156e864" target='_blank'>
              Rhabdomyosarcoma fusion oncoprotein initially pioneers a neural signature in vivo
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, C. Taslim, Meng Wang, Matthew V. Cannon, Katherine M. Silvius, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, André Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transcription factor 3 (TCF3), a pivotal member of the TCF/LEF family, plays a critical role in tumorigenesis. Nonetheless, its impact on the tumor microenvironment (TME) and cancer phenotypes remains elusive. We perform an exhaustive analysis of TCF3 expression, DNA variation profiles, prognostic implications, and associations with the TME and immunological aspects. This study is based on a large-scale pan-cancer cohort, encompassing over 17,000 cancer patients from multiple independent datasets, validated by in vitro assays. Our results show that TCF3/4/7 exhibits differential expression patterns between normal and tumor tissues across pan-cancer analyses. Mutational analysis of TCF3 across diverse cancer types reveals the highest alteration rates in biliary tract cancer. Additionally, mutations and single nucleotide variants in TCF3/4/7 are found to exert varied effects on patient prognosis. Importantly, TCF3 emerges as a robust predictor of survival across all cancer cohorts and among patients receiving immune checkpoint inhibitors. Elevated TCF3 expression is correlated with more aggressive cancer subtypes, as validated by immunohistochemistry and diverse cohort data. Furthermore, TCF3 expression is positively correlated with intratumoral heterogeneity and angiogenesis. In vitro investigations demonstrate that TCF3 is involved in epithelial-mesenchymal transition, migration, invasion, and angiogenesis. These effects are likely mediated through the interaction of TCF3 with the NF-κB/MMP2 pathway, which is modulated by IL-17A in human uveal melanoma MUM2B cells. This study elucidates, for the first time, the significant associations of TCF3 with DNA variation profiles, prognostic outcomes, and the TME in multiple cancer contexts. TCF3 holds promise as a molecular marker for diagnosis and as a potential target for novel therapeutic strategies, particularly in uveal melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bec429a74c0a32bc4300e0227ccff7d9af711e" target='_blank'>
              TCF3 as a multidimensional biomarker: oncogenicity, genomic alterations, and immune landscape in pan-cancer analysis.
              </a>
            </td>
          <td>
            Huiling Nie, Yang Yu, Siqi Zhou, Yue Xu, Xi Chen, Xun Qin, Zhangyu Liu, Jiayu Huang, Hailiang Zhang, Jin Yao, Qin Jiang, Bingbing Wei, Xiaojian Qin
          </td>
          <td>2024-08-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract DNA Double-strand breaks (DSBs) are harmful lesions and major sources of genomic instability. Studies have suggested that DSBs induce local transcriptional silencing that consequently promotes genomic stability. Several factors have been proposed to actively participate in this process, including ATM and Polycomb repressive complex 1 (PRC1). Here we found that disrupting PRC1 clustering disrupts DSB-induced gene silencing. Interactome analysis of PHC2, a PRC1 subunit that promotes the formation of the Polycomb body, found several nucleoporins that constitute the Nuclear Pore Complex (NPC). Similar to PHC2, depleting the nucleoporins also disrupted the DSB-induced gene silencing. We found that some of these nucleoporins, such as NUP107 and NUP43, which are members of the Y-complex of NPC, localize to DSB sites. These nucleoporin-enriched DSBs were distant from the nuclear periphery. The presence of nucleoporins and PHC2 at DSB regions were inter-dependent, suggesting that they act cooperatively in the DSB-induced gene silencing. We further found two structural components within NUP107 to be necessary for the transcriptional repression at DSBs: ATM/ATR-mediated phosphorylation at Serine37 residue within the N-terminal disordered tail, and the NUP133-binding surface at the C-terminus. These results provide a new functional interplay among nucleoporins, ATM and the Polycomb proteins in the DSB metabolism, and underscore their emerging roles in genome stability maintenance. *Hongseon Song, Yubin Bae, Sangin Kim, and Dante Deascanis contributed equally to this work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b0e95cd1e0dcce638bc5888df3682c199f2d24" target='_blank'>
              Nucleoporins cooperate with Polycomb silencers to promote transcriptional repression and repair at DNA double strand breaks
              </a>
            </td>
          <td>
            Hongseon Song*, Yubin Bae*, Sangin Kim, Dante Deascanis*, Yujin Lee, G. Róna, Ethan Lane, Seoyeong Lee, Sujung Kim, Michele Pagano, Kyungjae Myung, Y. Kee
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b28bde62e7c6ec79bde107b1d678f6c8aab03e" target='_blank'>
              MethNet: a robust approach to identify regulatory hubs and their distal targets from cancer data
              </a>
            </td>
          <td>
            Theodore Sakellaropoulos, Catherine Do, Guimei Jiang, Giulia Cova, Peter Meyn, D. Dimartino, Sitharam Ramaswami, Adriana Heguy, A. Tsirigos, Jane A Skok
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="After egg fertilization, an initially silent embryonic genome is transcriptionally activated during the maternal-to-zygotic transition. In zebrafish, maternal vertebrate pluripotency factors Nanog, Pou5f3 (OCT4 homolog), and Sox19b (SOX2 homolog) (NPS) play essential roles in orchestrating embryonic genome activation, acting as “pioneers” that open condensed chromatin and mediate acquisition of activating histone modifications. However, some embryonic gene transcription still occurs in the absence of these factors, suggesting the existence of other mechanisms regulating genome activation. To identify chromatin signatures of these unknown pathways, we profiled the histone modification landscape of zebrafish embryos using CUT&RUN. Our regulatory map revealed two subclasses of enhancers distinguished by presence or absence of H3K4me2. Enhancers lacking H3K4me2 tend to require NPS factors for de novo activation, while enhancers bearing H3K4me2 are epigenetically bookmarked by DNA hypomethylation to recapitulate gamete activity in the embryo, independent of NPS pioneering. Thus, parallel enhancer activation pathways combine to induce transcriptional reprogramming to pluripotency in the early embryo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/915e897ed23cf5b7d63f2ee24fa9d133d64c79b4" target='_blank'>
              H3K4me2 distinguishes a distinct class of enhancers during the maternal-to-zygotic transition
              </a>
            </td>
          <td>
            Matthew D. Hurton, Jennifer M. Miller, Miler T. Lee
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a139b2492c263398f29fd9ba2ccbb0286a9c476b" target='_blank'>
              The Ty1 retrotransposon harbors a DNA region that performs dual functions as both a gene silencing and chromatin insulator
              </a>
            </td>
          <td>
            Hiroshi Masumoto, Hideki Muto, Koichi Yano, Yohei Kurosaki, Hironori Niki
          </td>
          <td>2024-07-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="A key step of Canonical Nonhomologous End Joining (C-NHEJ) is synapsis of DNA double strand break (DSB) ends for ligation. The DNA-PKcs dimer mediates synapsis in a long-range complex with DSB ends remaining apart, whereas the XLF homodimer can mediate synapsis in both long-range and short-range complexes. Recent structural studies found the PAXX homodimer may also facilitate synapsis in long-range complexes with DNA-PKcs via its interactions with Ku70. Thus, we examined the influence of PAXX in C-NHEJ of chromosomal DSBs, which we compared to another Ku-binding factor, MRI. Using EJ of blunt DSBs with Cas9 reporters as a readout for C-NHEJ, we found that PAXX and/or MRI are dispensable. However, when combined with disruption of DNA-PKcs, particularly with DNA-PKcs kinase inhibition, PAXX becomes important for blunt DSB EJ. In contrast, while DNA-PKcs is also important to suppress short deletion mutations with microhomology, this effect is not magnified with PAXX loss. MRI loss had no effect combined with DNA-PKcs disruption, but becomes important for blunt DSB EJ when combined with disruption of XLF, as is PAXX. Finally, XLF loss causes an increase in larger deletions compared to DNA-PKcs inhibition, which is magnified with combined loss of MRI. Altogether, we suggest that PAXX promotes DSB end synapsis during C-NHEJ in a manner that is partially redundant with DNA-PKcs and XLF, whereas MRI appears to be mainly important in the context of XLF disruption. SIGNIFICANCE STATEMENT Canonical Nonhomologous End Joining (C-NHEJ) is a DNA double strand break repair pathway that can repair diverse types of DNA ends, but is prone to causing genetic mutations. In particular, C-NHEJ can cause insertion or deletion mutations that can cause human disease. Therefore, it is critical to understand the mechanisms of C-NHEJ, including and how various factors within the pathway affect repair outcomes. In this study, we elucidate the role of factors PAXX and MRI during chromosomal EJ and their interplay with the C-NHEJ factors DNA-PKcs and XLF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6710460a50993f582d4fe63d4fd0ff8229a07e" target='_blank'>
              Distinct functions of PAXX and MRI during chromosomal end joining
              </a>
            </td>
          <td>
            Metztli Cisneros-Aguirre, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Ruby Chen, Jeremy M. Stark
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pediatric brain cancer is the leading cause of disease-related mortality in children, and many aggressive tumors still lack effective treatment strategies. Despite extensive studies characterizing these tumor genomes, alternative transcriptional splicing patterns remain underexplored. Here, we systematically characterized aberrant alternative splicing across pediatric brain tumors, identifying pediatric high-grade gliomas (HGGs) among the most heterogeneous. Through integration with UniProt Knowledgebase annotations, we identified 12,145 splice events in 5,424 genes, leading to functional changes in protein activation, folding, and localization. We discovered that the master splicing factor and cell-cycle modulator, CDC- like kinase 1 (CLK1), is aberrantly spliced in HGGs to include exon 4, resulting in a gain of two phosphorylation sites and subsequent activation of CLK1. Inhibition of CLK1 with Cirtuvivint in the pediatric HGG KNS-42 cell line significantly decreased both cell viability and proliferation in a dose-dependent manner. Morpholino-mediated depletion of CLK1 exon 4 splicing reduced RNA expression, protein abundance, and cell viability. Notably, KNS-42 cells treated with the CLK1 exon 4 morpholino demonstrated differential expression 78 genes and differential splicing with loss or gain of a functional site in 193 genes annotated as oncogene or tumor suppressor genes (TSGs). These genes were enriched for cancer-associated pathways, with 20 identified as significant gene dependencies in pediatric HGGs. Our findings highlight a dependency of pediatric HGGs on CLK1 and its roles contributing to tumor splicing heterogeneity through transcriptional dysregulation of splicing factors and transcriptional modulation of oncogenes. Overall, aberrant splicing in HGGs and other pediatric brain tumors represents a potentially targetable oncogenic pathway contributing to tumor growth and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624af22ec9a4e826208669859a67ff2ad429da0e" target='_blank'>
              Characterization of aberrant splicing in pediatric central nervous system tumors reveals CLK1 as a candidate oncogenic dependency
              </a>
            </td>
          <td>
            Ammar S. Naqvi, Ryan J. Corbett, Priyanka Seghal, Karina L. Conkrite, Komal S. Rathi, Brian M. Ennis, Katharina E. Hayer, Bo Zhang, Miguel A. Brown, Daniel P. Miller, Adam A. Kraya, Joseph M. Dybas, Zhuangzhuang Geng, Christopher Blackden, Shehbeel Arif, Antonia Chroni, Aditya Lahiri, Madison L. Hollawell, Phillip B. Storm, J. Foster, Mateusz Koptyra, Peter Madsen, S. Diskin, A. Thomas-Tikhonenko, Adam C. Resnick, J. L. Rokita
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The ParABS system plays a critical role in bacterial chromosome segregation. The key component of this system, ParB, loads and spreads along DNA to form a local protein-DNA condensate known as a partition complex. As bacterial chromosomes are heavily supercoiled due to the continuous action of RNA polymerases, gyrases, and nucleoid-associated proteins, it is important to study the impact of DNA supercoiling on the ParB-DNA partition complex formation. Here, we use an in vitro single- molecule assay to visualize ParB on supercoiled DNA. Unlike most DNA-binding proteins, individual ParB proteins are found to not pin plectonemes on supercoiled DNA, but freely diffuse along supercoiled DNA. We find that DNA supercoiling enhances ParB-DNA condensation which initiates at lower ParB concentrations than on DNA that is torsionally relaxed. ParB proteins induce a DNA- protein condensate that strikingly absorbs all supercoiling writhe. Our findings provide mechanistic insights that have important implications for our understanding of bacterial chromosome organization and segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44217dd76122fe3389e0cee61b40b2f74765918" target='_blank'>
              DNA supercoiling enhances DNA condensation by ParB proteins
              </a>
            </td>
          <td>
            Alejandro Martin Gonzalez, M. Tišma, Brian T. Analikwu, Anders Barth, R. Janissen, Hammam Antar, Gianluca Kemps, Stephan Gruber, C. Dekker
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc75014daa55260f9205d28f8165ad0ad389192e" target='_blank'>
              Genetic hallmarks and clinical implications of chromothripsis in childhood T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            A. Pastorczak, Z. Urbańska, Borys Styka, Karolina Miarka-Walczyk, L. Sędek, K. Wypyszczak, A. Wakulinska, Zuzanna Nowicka, Tomasz Szczepański, Marcin Stańczak, Wojciech Fendler, Jerzy Kowalczyk, W. Młynarski, Monika Lejman
          </td>
          <td>2024-08-27</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary High levels of the transcription factor Grainyhead-like protein 2 (GRHL2) contribute to worse outcomes for patients with breast cancer tumors that express estrogen receptor (ER). Using multiple methods to increase GRHL2 expression in breast cancer cells, we found that high GRHL2 promotes both epithelial and mesenchymal phenotypes. This is indicative of an intermediate state between epithelial and mesenchymal cells, which is associated with metastasis. We also observed that elevated GRHL2 stimulates properties characteristic of cellular plasticity, including a decrease in cellular proliferation with a reciprocal increase in dormancy and stem cell markers. Elevated levels of GRHL2 broadly changed its control of gene expression at the level of DNA and shifted nearby transcription factor binding sites from those associated with development to those associated with disease progression. These results provide a possible explanation for how elevated GRHL2 levels could contribute to poorer prognosis in ER-positive breast cancer. Abstract Estrogen receptor (ER)-positive breast cancer is characterized by late recurrences following initial treatment. The epithelial cell fate transcription factor Grainyhead-like protein 2 (GRHL2) is overexpressed in ER-positive breast cancers and is linked to poorer prognosis as compared to ER-negative breast cancers. To understand how GRHL2 contributes to progression, GRHL2 was overexpressed in ER-positive cells. We demonstrated that elevated GRHL2 imparts plasticity with stem cell- and dormancy-associated traits. RNA sequencing and immunocytochemistry revealed that high GRHL2 not only strengthens the epithelial identity but supports a hybrid epithelial to mesenchymal transition (EMT). Proliferation and tumor studies exhibited a decrease in growth and an upregulation of dormancy markers, such as NR2F1 and CDKN1B. Mammosphere assays and flow cytometry revealed enrichment of stem cell markers CD44 and ALDH1, and increased self-renewal capacity. Cistrome analyses revealed a change in transcription factor motifs near GRHL2 sites from developmental factors to those associated with disease progression. Together, these data support the idea that the plasticity and properties induced by elevated GRHL2 may provide a selective advantage to explain the association between GRHL2 and breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c72323ad8abaa131c1ae72b9d382e27082cefdf" target='_blank'>
              Elevated GRHL2 Imparts Plasticity in ER-Positive Breast Cancer Cells
              </a>
            </td>
          <td>
            Christy Zheng, Kaelyn O. Allen, Tianrui Liu, Natalia M. Solodin, Mark B Meyer, Kelley Salem, Phillipos K. Tsourkas, Sean J. McIlwain, Jessica M. Vera, Erika R. Cromwell, M. S. Ozers, Amy M. Fowler, E. Alarid
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The AurkA serine/threonine kinase is a key regulator of cell division controlling mitotic entry, centrosome maturation, and chromosome segregation. The microtubule-associated protein TPX2 controls spindle assembly and is the main AurkA regulator, contributing to AurkA activation, localisation, and stabilisation. Since their identification, AurkA and TPX2 have been described as being overexpressed in cancer, with a significant correlation with highly proliferative and aneuploid tumours. Despite the frequent occurrence of AurkA/TPX2 co-overexpression in cancer, the investigation of their involvement in tumorigenesis and cancer therapy resistance mostly arises from studies focusing only on one at the time. Here, we review the existing literature and discuss the mitotic phenotypes described under conditions of AurkA, TPX2, or AurkA/TPX2 overexpression, to build a picture that may help clarify their oncogenic potential through the induction of chromosome instability. We highlight the relevance of the AurkA/TPX2 complex as an oncogenic unit, based on which we discuss recent strategies under development that aim at disrupting the complex as a promising therapeutic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1887c3163d0fe8ceb6fc5dcedbb98394ccd0c097" target='_blank'>
              Contribution of AurkA/TPX2 Overexpression to Chromosomal Imbalances and Cancer
              </a>
            </td>
          <td>
            Federica Polverino, Anna Mastrangelo, Giulia Guarguaglini
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1082aba026dea72fee57aef4a45e7856e9efcf8" target='_blank'>
              Topoisomerase-modulated genome-wide DNA supercoiling domains colocalize with nuclear compartments and regulate human gene expression.
              </a>
            </td>
          <td>
            Qian Yao, Linying Zhu, Zhen Shi, Subhadra Banerjee, Chongyi Chen
          </td>
          <td>2024-08-16</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Three-stranded DNA-RNA structures known as R-loops that form during papillomavirus transcription can cause transcription-replication conflicts and lead to DNA damage. We found that R-loops accumulated at the viral early promoter in human papillomavirus (HPV) episomal cells but were greatly reduced in cells with integrated HPV genomes. RNA-DNA helicases unwind R-loops and allow for transcription and replication to proceed. Depletion of the RNA-DNA helicase senataxin (SETX) using siRNAs increased the presence of R-loops at the viral early promoter in HPV-31 (CIN612) and HPV-16 (W12) episomal HPV cell lines. Depletion of SETX reduced viral transcripts in episomal HPV cell lines. The viral E2 protein, which binds with high affinity to specific palindromes near the promoter and origin, complexes with SETX, and both SETX and E2 are present at the viral p97 promoter in CIN612 and W12 cells. SETX overexpression increased E2 transcription activity on the p97 promoter. SETX depletion also significantly increased integration of viral genomes in CIN612 cells. Our results demonstrate that SETX resolves viral R-loops to proceed with HPV transcription and prevent genome integration.IMPORTANCEPapillomaviruses contain small circular genomes of approximately 8 kilobase pairs and undergo unidirectional transcription from the sense strand of the viral genome. Co-transcriptional R-loops were recently reported to be present at high levels in cells that maintain episomal HPV and were also detected at the early viral promoter. R-loops can inhibit transcription and DNA replication. The process that removes R-loops from the PV genome and the requisite enzymes are unknown. We propose a model in which the host RNA-DNA helicase senataxin assembles on the HPV genome to resolve R-loops in order to maintain the episomal status of the viral genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3257970da042eeca9c08a9a174f0a9b3a46f14" target='_blank'>
              Senataxin mediates R-loop resolution on HPV episomes.
              </a>
            </td>
          <td>
            L. Jose, Keely Smith, Anaiya Crowner, EJ Androphy, M. DeSmet
          </td>
          <td>2024-07-24</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Senescence —the endpoint of replicative lifespan for normal cells— is established via a complex sequence of molecular events. One such event is the dramatic reorganization of CTCF into senescence-induced clusters (SICCs). However, the molecular determinants, genomic consequences, and functional purpose of SICCs remained unknown. Here, we combine functional assays, super-resolution imaging, and 3D genomics with computational modelling to dissect SICC emergence. We establish that the competition between CTCF-bound and non-bound loci dictates clustering propensity. Upon senescence entry, cells repurpose SRRM2 —a key component of nuclear speckles— and BANF1 —a ‘molecular glue’ for chromosomes— to cluster CTCF and rewire genome architecture. This CTCF-centric reorganization in reference to nuclear speckles functionally sustains the senescence splicing program, as SICC disruption fully reverts alternative splicing patterns. We therefore uncover a new paradigm, whereby cells translate changes in nuclear biochemistry into architectural changes directing splicing choices so as to commit to the fate of senescence. GRAPHICAL ABSTRACT HIGHLIGHTS HMGB2-bound loci compete with CTCF-bound ones for nuclear speckle association Senescent cells repurpose SRRM2 and BANF1 to cluster CTCF on speckles BANF1 is essential, but not sufficient for CTCF clustering The SRRM2 RNA-binding domain directs CTCF clustering SICCs rewire chromatin positioning to sustain the senescence splicing program">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d2e2f24883e3f53705549d5245cfd1c159e708" target='_blank'>
              Senescent cells cluster CTCF on nuclear speckles to sustain their splicing program
              </a>
            </td>
          <td>
            Spiros Palikyras, Vassiliki Varamogiani-Mamatsi, Yajie Zhu, Shyam Ramasamy, Athanasia Mizi, Isabel Liebermann, Athanasia Stavropoulou, Ioanna Papadionysiou, Deniz Bartsch, Yulia Kargapolova, K. Sofiadis, Christoforos Nikolaou, Leo Kurian, A. M. Oudelaar, Mariano Barbieri, A. Papantonis
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="R-loops are trimeric nucleic acid structures that form when an RNA molecule hybridizes with its complementary DNA strand, displacing the opposite strand. These structures regulate transcription as well as replication, but aberrant R-loops can form, leading to DNA breaks and genomic instability if unresolved. R-loop levels are elevated in many cancers as well as cells that maintain high-risk human papillomaviruses. We investigated how the distribution as well as function of R-loops changed between normal keratinocytes and HPV positive cells derived from a precancerous lesion of the cervix (CIN I). The levels of R-loops associated with cellular genes were found to be up to 10-fold higher in HPV positive cells than in normal keratinocytes while increases at ALU1 elements increased by up to 500-fold. The presence of enhanced R-loops resulted in altered levels of gene transcription, with equal numbers increased as decreased. While no uniform global effects on transcription due to the enhanced levels of R-loops were detected, genes in several pathways were coordinately increased or decreased in expression only in the HPV positive cells. This included the downregulation of genes in the innate immune pathway, such as DDX58, IL-6, STAT1, IFN-β, and NLRP3. All differentially expressed innate immune genes dependent on R-loops were also associated with H3K36me3 modified histones. Genes that were upregulated by the presence of R-loops in HPV positive cells included those in the DNA damage repair such as ATM, ATRX, and members of the Fanconi Anemia pathway. These genes exhibited a linkage between R-loops and H3K36me3 as well as γH2AX histone marks only in HPV positive cells. These studies identify a potential link in HPV positive cells between DNA damage repair as well as innate immune regulatory pathways with R-loops and γH2AX/H3K36me3 histone marks that may contribute to regulating important functions for HPV pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30cc12d40eed32c558ede6628c4c8dfd335ab28" target='_blank'>
              HPV induced R-loop formation represses innate immune gene expression while activating DNA damage repair pathways
              </a>
            </td>
          <td>
            Conor W Templeton, Lou Laimins
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Topologically associated domains (TADs) are believed to be involved in the regulation of gene expression. While the impact of TAD perturbations is usually studied in developmental genes with highly cell-type-specific expression patterns, this study examines genes with broad expression profiles divided by a strong insulatory boundary. We focused on the Slc29a3/Unc5b locus, which encompasses two distinct TADs. Our analysis demonstrates that deletions of CTCF binding sites within this locus lead to alterations in local chromatin architecture, disrupting existing loops and forming novel long-range interactions. We evaluated the transcription changes of Unc5b, Slc29a3, Psap, Vsir, Cdh23, and Sgpl1 genes across various organs, finding that TAD boundary disruption results in variable transcriptional responses, where not only magnitude, but also direction of gene expression changes are tissue-specific. Current models of genome architecture, including enhancer competition and hijacking, only partially account for these transcriptional changes, indicating the need for further investigation into the mechanisms underlying TAD function and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c912d81a6af96e3ee3f74faa2fc1bde4c441880" target='_blank'>
              Direction and modality of transcription changes caused by TAD boundary disruption in Slc29a3/Unc5b locus depends on tissue-specific epigenetic context
              </a>
            </td>
          <td>
            Paul Salnikov, P. Belokopytova, Alexandra Yan, E. Viesná, Alexey Korablev, Irina Serova, V. Lukyanchikova, Yana Stepanchuk, N. Torgunakov, Savelii Tikhomirov, V. Fishman
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast Cancer Type 1 Susceptibility Protein (BRCA1) is a tumor-suppressor protein that regulates various cellular pathways, including those that are essential for preserving genome stability. One essential mechanism involves a BRCA1-A complex that is recruited to double-strand breaks (DSBs) by RAP80 before initiating DNA damage repair (DDR). How RAP80 itself is recruited to DNA damage sites, however, is unclear. Here, we demonstrate an intrinsic correlation between a methyltransferase DOT1L-mediated RAP80 methylation and BRCA1-A complex chromatin recruitment that occurs during cancer cell radiotherapy resistance. Mechanistically, DOT1L is quickly recruited onto chromatin and methylates RAP80 at multiple lysines in response to DNA damage. Methylated RAP80 is then indispensable for binding to ubiquitinated H2A and subsequently triggering BRCA1-A complex recruitment onto DSBs. Importantly, DOT1L-catalyzed RAP80 methylation and recruitment of BRCA1 have clinical relevance, as inhibition of DOT1L or RAP80 methylation seems to enhance the radiosensitivity of cancer cells both in vivo and in vitro. These data reveal a crucial role for DOT1L in DDR through initiating recruitment of RAP80 and BRCA1 onto chromatin and underscore a therapeutic strategy based on targeting DOT1L to overcome tumor radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a53d6b1d8429c7df99fb0b7dc3e6119436184d" target='_blank'>
              DOT1L-mediated RAP80 methylation promotes BRCA1 recruitment to elicit DNA repair.
              </a>
            </td>
          <td>
            Huangqi Tang, Ya-Fei Lu, Rongsheng Zeng, Chaohua Liu, Yuxin Shu, Yupei Wu, Jiajie Su, Longjiang Di, Jinqin Qian, Jun Zhang, Yuan Tian, Xiaopeng Lu, Xin-Hai Pei, Qian Zhu, Wei-Guo Zhu
          </td>
          <td>2024-08-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Disruption of the epigenome is a hallmark of human disease including liver cirrhosis and hepatocellular carcinoma (HCC). While genetic heterogeneity is an established effector of pathologic phenotypes, epigenetic heterogeneity is less well understood. Environmental exposures alter the liver-specific DNA methylation landscape and influence the onset of liver cancer. Given that currently available treatments are unable to target frequently mutated genes in HCC, there is an unmet need for novel therapeutics to prevent or reverse liver damage leading to hepatic tumorigenesis, which the epigenome may provide. We performed genome-wide profiling of DNA methylation, copy number, and gene expression from multiple liver regions from 31 patients with liver disease to examine their crosstalk and define the individual and combinatorial contribution of these processes to liver disease progression. We identify epigenetic heterogeneity hotspots that are conserved across patients. Elevated epigenetic heterogeneity associates with increased gene expression heterogeneity. Cirrhotic regions comprise two distinct cohorts, one exclusively epigenetic, and a second where epigenetic and copy number variations collaborate. Epigenetic heterogeneity hotspots are enriched for genes central to liver function (e.g., HNF1A) and known tumor suppressors (e.g., RASSF1A). These hotspots encompass genes including ACSL1, ACSL5, MAT1A, and ELFN1, which have phenotypic effects in functional screens, supporting their relevance to hepatocarcinogenesis. Moreover, epigenetic heterogeneity hotspots are linked to clinical measures of outcome.


CONCLUSION
Substantial epigenetic heterogeneity arises early in liver disease development, targeting key pathways in the progression and initiation of both cirrhosis and HCC. Integration of epigenetic and transcriptional heterogeneity unveils putative epigenetic regulators of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46b475bb15d42990ae95e5c459fd1ea63a3f93cd" target='_blank'>
              Epigenetic heterogeneity hotspots in human liver disease progression.
              </a>
            </td>
          <td>
            R. Hlady, Xia Zhao, Louis El Khoury, Ryan T. Wagner, Aesis M. Luna, Kien Pham, Nikolaos Pyrosopoulos, Dhanpat Jain, Liguo Wang, Chen Liu, Keith D. Robertson
          </td>
          <td>2024-07-19</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Higher levels of aneuploidy, characterized by imbalanced chromosome numbers, are associated with lethal progression in prostate cancer. However, how aneuploidy contributes to prostate cancer aggressiveness remains poorly understood. In this study, we assessed in patients which genes on chromosome 8q, one of the most frequently gained chromosome arms in prostate tumors, were most strongly associated with long-term risk of cancer progression to metastases and death from prostate cancer (lethal disease) in 403 patients and found the strongest candidate was cohesin subunit gene, RAD21, with an odds ratio of 3.7 (95% CI 1.8, 7.6) comparing the highest vs. lowest tertiles of mRNA expression and adjusting for overall aneuploidy burden and Gleason score, both strong prognostic factors in primary prostate cancer. Studying prostate cancer driven by the TMPRSS2-ERG oncogenic fusion, found in about half of all prostate tumors, we found that increased RAD21 alleviated toxic oncogenic stress and DNA damage caused by oncogene expression. Data from both organoids and patients indicate that increased RAD21 thereby enables aggressive tumors to sustain tumor proliferation, and more broadly suggests one path through which tumors benefit from aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f061db09f8cb87400e631c0dc03d0078eb8a3c" target='_blank'>
              RAD21 promotes oncogenesis and lethal progression of prostate cancer.
              </a>
            </td>
          <td>
            Xiaofeng A Su, Konrad H. Stopsack, Daniel R. Schmidt, Duanduan Ma, Zhe Li, P. Scheet, K. Penney, T. Lotan, W. Abida, Elise G. DeArment, Kate Lu, Thomas Janas, Sofia Hu, M. V. Vander Heiden, Massimo Loda, Monica G. Boselli, Angelika Amon, Lorelei Mucci
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b09daa5b66a34ba0df34226e316e985b17964e4" target='_blank'>
              Satellite DNAs and the evolution of the multiple X1X2Y sex chromosomes in the wolf fish Hoplias malabaricus (Teleostei; Characiformes)
              </a>
            </td>
          <td>
            G. A. Toma, A. Sember, C. A. G. Goes, Rafael Kretschmer, F. Porto‐Foresti, L. Bertollo, Thomas Liehr, R. Utsunomia, Marcelo de Bello Cioffi
          </td>
          <td>2024-09-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Aggressive breast cancers harbor TP53 missense mutations. Tumor cells with TP53 missense mutations exhibit enhanced growth and survival through transcriptional rewiring. To delineate how TP53 mutations in breast cancer contribute to tumorigenesis and progression in vivo, we created a somatic mouse model driven by mammary epithelial cell-specific expression of Trp53 mutations. Mice developed primary mammary tumors reflecting the human molecular subtypes of luminal A, luminal B, HER2-enriched, and triple-negative breast cancer with metastases. Transcriptomic analyses comparing MaPR172H/− or MaPR245W/− mammary tumors to MaP−/− tumors revealed (1) differences in cancer-associated pathways activated in both p53 mutants and (2) Nr5a2 as a novel transcriptional mediator of distinct pathways in p53 mutants. Meta-analyses of human breast tumors corroborated these results. In vitro assays demonstrate mutant p53 upregulates specific target genes that are enriched for Nr5a2 response elements in their promoters. Co-immunoprecipitation studies revealed p53R172H and p53R245W interact with Nr5a2. These findings implicate NR5A2 as a novel mediator of mutant p53 transcriptional activity in breast cancer. Significance: Our findings implicate NR5A2 as a novel mediator of mutant p53 transcriptional activity in breast cancer. NR5A2 may be an important therapeutic target in hard-to-treat breast cancers such as endocrine-resistant tumors and metastatic triple-negative breast cancers harboring TP53 missense mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3a3a065368157acdf353ec042f3861378226ba" target='_blank'>
              p53R172H and p53R245W Hotspot Mutations Drive Distinct Transcriptomes in Mouse Mammary Tumors Through a Convergent Transcriptional Mediator
              </a>
            </td>
          <td>
            Joy M. McDaniel, R. Morrissey, D. Dibra, Lalit R Patel, S. Xiong, Yun Zhang, G. Chau, Xiaoping Su, Yuan Qi, Adel K El-Naggar, Guillermina Lozano
          </td>
          <td>2024-07-12</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b45077911b50bf1147820528b1db3588bf1b0d" target='_blank'>
              A sequence of SVA retrotransposon insertions in ASIP shaped human pigmentation
              </a>
            </td>
          <td>
            Nolan Kamitaki, Margaux L. A. Hujoel, Ronen E. Mukamel, Edward Gebara, S. Mccarroll, Po-Ru Loh
          </td>
          <td>2024-07-24</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>115</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378973c9450b39532ddda952e6b94210cb3bdd4f" target='_blank'>
              The genetic evolution of acral melanoma
              </a>
            </td>
          <td>
            Meng Wang, Satoshi Fukushima, Y. Sheen, E. Ramelyte, Noel Cruz-Pacheco, Chenxu Shi, Shanshan Liu, I. Banik, Jamie D. Aquino, Martin Sangueza Acosta, Mitchell P. Levesque, Reinhard Dummer, Jau-Yu Liau, Chia-Yu Chu, A. Shain, I. Yeh, Boris C. Bastian
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7234a63fd75fc71e91d39632a18b128298ba3fb" target='_blank'>
              Genome-wide identification of copy number variations in wrinkled skin cases of Xiang pigs
              </a>
            </td>
          <td>
            Xiaoli Liu, Fenbin Hu, Wei Wang, Xia Chen, Xi Niu, Shihui Huang, Zhou Wang, Jiafu Wang, Xueqin Ran
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="During the sexual cycle, programmed genome rearrangement (PGR) in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal Internal Eliminated Sequences (IESs) from the newly developing somatic nucleus. In addition to the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5’-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolX (a,b,c,d), four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, 1-nucleotide deletions and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during PGR and have acquired tight nuclear anchoring properties through their N-terminal region, which contains a predicted BRCT domain. Finally, we show that PolXa accumulates in nuclear foci together with other NHEJ proteins and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these “DNA repair foci” correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of iesRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8a893b216f167d132f42d3e505df3cd9a940a5" target='_blank'>
              A development-specific POLX essential for programmed genome rearrangement in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Heterochromatin is a gene-poor and repeat-rich genomic compartment universally found in eukaryotes. Despite its low transcriptional activity, heterochromatin plays important roles in maintaining genome stability, organizing chromosomes, and suppressing transposable elements (TEs). Given the importance of these functions, it is expected that the genes involved in heterochromatin regulation would be highly conserved. Yet, a handful of these genes were found to evolve rapidly. To investigate whether these previous findings are anecdotal or general to genes modulating heterochromatin, we compile an exhaustive list of 106 candidate genes involved in heterochromatin functions and investigate their evolution over short and long evolutionary time scales in Drosophila. Our analyses find that these genes exhibit significantly more frequent evolutionary changes, both in the forms of amino acid substitutions and gene copy number change, when compared to genes involved in Polycomb-based repressive chromatin. While positive selection drives amino acid changes within both structured domains with diverse functions and intrinsically disordered regions (IDRs), purifying selection may have maintained the proportions of IDRs of these proteins. Together with the observed negative associations between evolutionary rates of these genes and genomic TE abundance, we propose an evolutionary model where the fast evolution of genes involved in heterochromatin functions is an inevitable outcome of the unique functional roles of heterochromatin, while the rapid evolution of TEs may be an effect rather than cause. Our study provides an important global view of the evolution of genes involved in this critical cellular domain and provides insights into the factors driving the distinctive evolution of heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6e4293419d3640277d1a81925a746d1d427525a" target='_blank'>
              Prevalent fast evolution of genes involved in heterochromatin functions.
              </a>
            </td>
          <td>
            Leila Lin, Yuheng Huang, Jennifer McIntyre, Ching-Ho Chang, Serafin U. Colmenares, Y. C. G. Lee
          </td>
          <td>2024-08-27</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Cancer stemness plays an important role in cancer initiation and progression, and is the major cause of tumor invasion, metastasis, recurrence, and poor prognosis. Non-coding RNAs (ncRNAs) are a class of RNA transcripts that generally cannot encode proteins and have been demonstrated to play a critical role in regulating cancer stemness. Here, we developed the ncStem database to record manually curated and predicted ncRNAs associated with cancer stemness. In total, ncStem contains 645 experimentally verified entries, including 159 long non-coding RNAs (lncRNAs), 254 microRNAs (miRNAs), 39 circular RNAs (circRNAs), and 5 other ncRNAs. The detailed information of each entry includes the ncRNA name, ncRNA identifier, disease, reference, expression direction, tissue, species, and so on. In addition, ncStem also provides computationally predicted cancer stemness-associated ncRNAs for 33 TCGA cancers, which were prioritized using the random walk with restart (RWR) algorithm based on regulatory and co-expression networks. The total predicted cancer stemness-associated ncRNAs included 11 132 lncRNAs and 972 miRNAs. Moreover, ncStem provides tools for functional enrichment analysis, survival analysis, and cell location interrogation for cancer stemness-associated ncRNAs. In summary, ncStem provides a platform to retrieve cancer stemness-associated ncRNAs, which may facilitate research on cancer stemness and offer potential targets for cancer treatment. Database URL: http://www.nidmarker-db.cn/ncStem/index.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb97313e3402ad277fe895b46bce91bdb6c21f55" target='_blank'>
              ncStem: a comprehensive resource of curated and predicted ncRNAs in cancer stemness
              </a>
            </td>
          <td>
            Hui Liu, Nan Zhang, Yijie Jia, Jun Wang, Aokun Ye, Siru Yang, Honghan Zhou, Yingli Lv, Chaohan Xu, Shuyuan Wang
          </td>
          <td>2024-08-13</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple myeloma is the second most hematological cancer. RUVBL1 and RUVBL2 form a subcomplex of many chromatin remodeling complexes implicated in cancer progression. As an inhibitor specific to the RUVBL1/2 complex, CB-6644 exhibits remarkable anti-tumor activity in xenograft models of Burkitt’s lymphoma and multiple myeloma (MM). In this work, we defined transcriptional signatures corresponding to CB-6644 treatment in MM cells and determined underlying epigenetic changes in terms of chromatin accessibility. CB-6644 upregulated biological processes related to interferon response and downregulated those linked to cell proliferation in MM cells. Transcriptional regulator inference identified E2Fs as regulators for downregulated genes and MED1 and MYC as regulators for upregulated genes. CB-6644-induced changes in chromatin accessibility occurred mostly in non-promoter regions. Footprinting analysis identified transcription factors implied in modulating chromatin accessibility in response to CB-6644 treatment, including ATF4/CEBP and IRF4. Lastly, integrative analysis of transcription responses to various chemical compounds of the molecular signature genes from public gene expression data identified CB-5083, a p97 inhibitor, as a synergistic candidate with CB-6644 in MM cells, but experimental validation refuted this hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff9ee9c96de0ec030a3d9d628090554b2bbd4f2" target='_blank'>
              Molecular Signatures of CB-6644 Inhibition of the RUVBL1/2 Complex in Multiple Myeloma
              </a>
            </td>
          <td>
            Weijun Yi, Sebastian A. Dziadowicz, Rachel S. Mangano, Lei Wang, Joseph C. McBee, Steven M. Frisch, Lori A. Hazlehurst, Donald A. Adjeroh, Gangqing Hu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human cell line models, including the neuronal precursor line LUHMES, are important for investigating developmental transcriptional dynamics within imprinted regions, particularly the 15q11-q13 Angelman (AS) and Prader-Willi (PWS) syndrome locus. AS results from loss of maternal UBE3A in neurons, where the paternal allele is silenced by a convergent antisense transcript UBE3A-ATS, a lncRNA that terminates at PWAR1 in non-neurons. qRT-PCR analysis confirmed the exclusive and progressive increase in UBE3A-ATS in differentiating LUHMES neurons, validating their use for studying UBE3A silencing. Genome-wide transcriptome analyses revealed changes to 11 834 genes during neuronal differentiation, including the upregulation of most genes within the 15q11-q13 locus. To identify dynamic changes in chromatin loops linked to transcriptional activity, we performed a HiChIP validated by 4C, which identified two neuron-specific CTCF loops between MAGEL2-SNRPN and PWAR1-UBE3A. To determine if allele-specific differentially methylated regions (DMR) may be associated with CTCF loop anchors, whole genome long-read nanopore sequencing was performed. We identified a paternally hypomethylated DMR near the SNRPN upstream loop anchor exclusive to neurons and a paternally hypermethylated DMR near the PWAR1 CTCF anchor exclusive to undifferentiated cells, consistent with increases in neuronal transcription. Additionally, DMRs near CTCF loop anchors were observed in both cell types, indicative of allele-specific differences in chromatin loops regulating imprinted transcription. These results provide an integrated view of the 15q11-q13 epigenetic landscape during LUHMES neuronal differentiation, underscoring the complex interplay of transcription, chromatin looping, and DNA methylation. They also provide insights for future therapeutic approaches for AS and PWS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c616b0f402f95ed53dc1945c207ee4eb663c1600" target='_blank'>
              Integration of CTCF loops, methylome, and transcriptome in differentiating LUHMES as a model for imprinting dynamics of the 15q11-q13 locus in human neurons.
              </a>
            </td>
          <td>
            Orangel J. Gutierrez Fugón, Osman Sharifi, Nicholas Heath, Daniela C. Soto, J. A. Gomez, Dag Yasui, A. J. P. Mendiola, H. O’Geen, U. Beitnere, Marketa Tomkova, Viktoria Haghani, Greg Dillon, David J. Segal, J. LaSalle
          </td>
          <td>2024-07-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks (SSB) arising from base excision repair (BER) during misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination-deficient (HRD) cells. At templated AP-sites, HMCES DNA-protein crosslinks (DPC) regulate replication fork speed while avoiding APE1-mediated cytotoxic double-strand breaks (DSB). Whereas the role of HMCES at template DNA strand is well studied, its consequences on nascent DNA are less understood. Here, we provide evidence that HMCES play detrimental roles during removal of 5- hydroxymethyl-2’-deoxycytidine (5hmdC)-derived 5-hydroxymethyl-2’-deoxyuridine (5hmdU) by BER at replication forks. HRD cells display heightened HMCES chromatin levels, which increase upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP-sites. HMCES depletion largely suppresses 5hmdC- mediated Fancd2-/- replication fork defects, chromosomal aberrations and cell lethality, suggesting that HMCES is responsible for the replication fork impairment and lethality observed in HRD cells. Therefore, HMCES-DPCs are a novel source of BER-initiated PRIMPOL-mediated ssDNA gaps, implying endogenous DPCs as a source of DNA damage in HRD tumours. Teaser Covalent binding of HMCES to nascent DNA blocks replication progression and kills homologous recombination deficient cancer cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301e05a5b2dbeda5a785b9d3a1c934b12bac03" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martin, Marta del Rio Oliva, Jean-Yves Masson, JC Reyes, Iván V. Rosado
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The Chromodomain helicase DNA-binding protein 1-like (CHD1L) is a nucleosome remodeling enzyme, which plays a key role in chromatin relaxation during the DNA damage response. Genome editing has shown that deletion of CHD1L sensitizes cells to PARPi, but the effect of its pharmacological inhibition has not been defined. Triple-negative breast cancer SUM149PT, HCC1937, and MDA-MB-231 cells were used to assess the mechanism of action of the CHD1Li OTI-611. Cytotoxicity as a single agent or in combination with standard-of-care treatments was assessed in tumor organoids. Immunofluorescence was used to assess the translocation of PAR and AIF to the cytoplasm or the nucleus and to study markers of DNA damage or apoptosis. Trapping of PARP1/2 or CHD1L onto chromatin was also assessed by in situ subcellular fractionation and immunofluorescence and validated by Western blot. We show that the inhibition of CHD1L’s ATPase activity by OTI-611 is cytotoxic to triple-negative breast cancer tumor organoids and synergizes with PARPi and chemotherapy independently of the BRCA mutation status. The inhibition of the remodeling function blocks the phosphorylation of H2AX, traps CHD1L on chromatin, and leaves PAR chains on PARP1/2 open for hydrolysis. PAR hydrolysis traps PARP1/2 at DNA damage sites and mediates PAR translocation to the cytoplasm, release of AIF from the mitochondria, and induction of PARthanatos. The targeted inhibition of CHD1L’s oncogenic function by OTI-611 signifies an innovative therapeutic strategy for breast cancer and other cancers. This approach capitalizes on CHD1L-mediated DNA repair and cell survival vulnerabilities, thereby creating synergy with standard-of-care therapies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b6daf1d2a20088ba1aa72a3182ab37cb9c8e4" target='_blank'>
              CHD1L Regulates Cell Survival in Breast Cancer and Its Inhibition by OTI-611 Impedes the DNA Damage Response and Induces PARthanatos
              </a>
            </td>
          <td>
            Rita Sala, Hector Esquer, Timothy Kellett, Jeffrey T. Kearns, Paul Awolade, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Maintaining genome integrity is an essential and challenging process. RAD51 recombinase, the central player of several crucial processes in repairing and protecting genome integrity, forms filaments on DNA. RAD51 filaments are tightly regulated. One of these regulators is FIGNL1, that prevents persistent RAD51 foci post-damage and genotoxic chromatin association in cells. The cryogenic electron microscopy structure of FIGNL1 in complex with RAD51 reveals that the FIGNL1 forms a non-planar hexamer and RAD51 N-terminus is enclosed in the FIGNL1 hexamer pore. Mutations in pore loop or catalytic residues of FIGNL1 render it defective in filament disassembly and are lethal in mouse embryonic stem cells. Our study reveals a unique mechanism for removing RAD51 from DNA and provides the molecular basis for FIGNL1 in maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b11fcc1a323386baea335e7bd61f5bc4543b03e" target='_blank'>
              Molecular basis of FIGNL1 in dissociating RAD51 from DNA and chromatin
              </a>
            </td>
          <td>
            Alexander Carver, Tai-Yuan Yu, Luke A Yates, Travis White, Raymond Wang, Katie Lister, M. Jasin, Xiaodong Zhang
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00690bfe5789fdccfdd4c5cebda562df27157a03" target='_blank'>
              A new evaluation of the rearranged non-coding control region of JC virus in patients with colorectal cancer
              </a>
            </td>
          <td>
            Azadeh Haghi Navand, Shahram Jalilian, Kambiz Ahmadi Angali, Mohammad Karimi Babaahmadi, A. Talaiezadeh, M. Makvandi
          </td>
          <td>2024-08-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, Tengjiang Chen, Jie Lei, Changwen Chen, Shunqin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current Cancer Drug Targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature Plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8cd8dcee3abb97837fb9cb766c024fc2216aaa" target='_blank'>
              Canine hyper-sociability structural variants associated with altered three-dimensional chromatin state
              </a>
            </td>
          <td>
            Dhriti Tandon, E. Kubinyi, Sára Sándor, Hannah Faughnan, Ádám Miklósi, B. Vonholdt
          </td>
          <td>2024-08-07</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Gene regulators physically associate to the genome, in a combinatorial fashion, to drive tissue-specific gene expression programs. Uncovering the genome-wide activity of all gene regulators across tissues is therefore needed to understand how the genome is regulated during development. Here, we take a first step forward towards achieving this goal. Using CUT&RUN, we systematically measured the genome-wide binding profiles of key transcription factors and cofactors that mediate the activity of ontogenetically relevant signaling pathways in select mouse tissues at two developmental stages. Computation of the numerous genome-wide binding datasets unveiled a large degree of tissue and time-specific activity for each gene regulator, and several factor-specific idiosyncrasies. Moreover, we identified “popular” regulatory regions that are bound by a multitude of pathway regulators. Popular regions tend to be more evolutionarily conserved, implying their essentiality. Consistently, they lie in the proximity of genes whose dysregulation causes early embryonic lethality in the mouse. Moreover, the human homologs of these regions are also bound by many gene regulators and are highly conserved in human populations, indicating that they retain functional relevance for human development. This work constitutes a decisive step towards understanding how the genome is simultaneously read and used by gene regulators in a holistic fashion and unveils multiple genomic mechanisms that drive embryonic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea44cb9e15a021f4221a0294719b3ece65eeee5" target='_blank'>
              The Transcription Factor Binding Landscape of Mouse Development
              </a>
            </td>
          <td>
            A. Nordin, Gianluca Zambanini, Mattias Jonasson, Tamina Weiss, Yorick Bernardus Cornelis van de Grift, P. Pagella, Claudio Cantù
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, B. van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non-SMC condensin I complex subunit D2 (NCAPD2) is a newly identified oncogene; however, the specific biological function and molecular mechanism of NCAPD2 in liver cancer progression remain unknown. In the present study, the aberrant expression of NCAPD2 in liver cancer was investigated using public tumor databases, including TNMplot, The Cancer Genome Atlas and the International Cancer Genome Consortium based on bioinformatics analyses, and it was validated using a clinical cohort. It was revealed that NCAPD2 was significantly upregulated in liver cancer tissues compared with in control liver tissues, and NCAPD2 served as an independent prognostic factor and predicted poor prognosis in liver cancer. In addition, the expression of NCAPD2 was positively correlated with the percentage of Ki67+ cells. Finally, single-cell sequencing data, gene-set enrichment analyses and in vitro investigations, including cell proliferation assay, Transwell assay, wound healing assay, cell cycle experiments, cell apoptosis assay and western blotting, were carried out in human liver cancer cell lines to assess the biological mechanisms of NCAPD2 in patients with liver cancer. The results revealed that the upregulation of NCAPD2 enhanced tumor cell proliferation, invasion and cell cycle progression at the G2/M-phase transition, and inhibited apoptosis in liver cancer cells. Furthermore, NCAPD2 overexpression was closely associated with the phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR)/c-Myc signaling pathway and epithelial-mesenchymal transition (EMT) progression in HepG2 and Huh7 cells. In addition, upregulated NCAPD2 was shown to have adverse effects on overall survival and disease-specific survival in liver cancer. In conclusion, the overexpression of NCAPD2 was shown to lead to cell cycle progression at the G2/M-phase transition, activation of the PI3K-Akt-mTOR/c-Myc signaling pathway and EMT progression in human liver cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6873a90e35d5a8f62be3a51b8f0dd6c3ab4c961" target='_blank'>
              NCAPD2 augments the tumorigenesis and progression of human liver cancer via the PI3K-Akt-mTOR signaling pathway
              </a>
            </td>
          <td>
            Jiangxue Gu, Ke Huang, Wei-Lin Zhao, Xiao-Ming Zheng, Yu-Qin Wu, Shirong Yan, Yu-Gang Huang, Pei Hu
          </td>
          <td>2024-07-31</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5269207a9cd9eb0e78ac9fd2c09cf5480ed4d833" target='_blank'>
              PRC2-AgeIndex as a universal biomarker of aging and rejuvenation
              </a>
            </td>
          <td>
            M. Moqri, Andrea Cipriano, Daniel J. Simpson, S. Rasouli, Tara Murty, Tineke Anna de Jong, D. Nachun, Guilherme de Sena Brandine, Kejun Ying, A. Tarkhov, K. Aberg, E. van den Oord, Wanding Zhou, Andrew Smith, Crystal L Mackall, V. N. Gladyshev, S. Horvath, M. Snyder, Vittorio Sebastiano
          </td>
          <td>2024-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The use of antitumor drugs represents a reliable strategy for cancer therapy. Unfortunately, drug resistance has become increasingly common and contributes to tumor metastasis and local recurrence. The tumor immune microenvironment (TME) consists of immune cells, cytokines and immunomodulators, and collectively they influence the response to treatment. Epigenetic changes including DNA methylation and histone modification, as well as increased drug exportation have been reported to contribute to the development of drug resistance in cancers. In the past few years, the majority of studies on tumors have only focused on the development and progression of a tumor from a mechanistic standpoint; few studies have examined whether the changes in the TME can also affect tumor growth and drug resistance. Recently, emerging evidence have raised more concerns regarding the role of TME in the development of drug resistance. In the present review, it was discussed how the suppressive TME adapts to drug resistance characterized by the cooperation of immune cells, cytokines, immunomodulators, stromal cells and extracellular matrix. Furthermore, it was reviewed how these immunological or metabolic changes alter immuno‑surveillance and thus facilitate tumor drug resistance. In addition, potential targets present in the TME for developing novel therapeutic strategies to improve individualized therapy for cancer treatment were revealed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdfa9d18c0bfe84e176528ce335232eb883a650" target='_blank'>
              Drug resistance and tumor immune microenvironment: An overview of current understandings (Review).
              </a>
            </td>
          <td>
            Yan Liu, Jun Liang, Yanping Zhang, Q. Guo
          </td>
          <td>2024-08-30</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The mechanisms that work alongside BRAFV600E oncogene in melanoma development, in addition to ultraviolet (UV) radiation (UVR), are of great interest. Analysis of human melanoma tumors [data from The Cancer Genome Atlas (TCGA)] revealed that 50% or more of the samples expressed no or low amounts of serine/threonine protein kinase STK11 (also known as LKB1) protein. Here, we report that, in a mouse model, concomitant neonatal BrafV600E activation and Lkb1 tumor suppressor ablation in melanocytes led to full melanoma development. A single postnatal dose of UVB radiation had no effect on melanoma onset in Lkb1-depleted mice compared with BrafV600E-irradiated mice, but increased tumor multiplicity. In concordance with these findings and previous reports, Lkb1-null irradiated mice exhibited deficient DNA damage repair (DDR). Histologically, tumors lacking Lkb1 were enriched in neural-like tumor morphology. Genetic profiling and gene set enrichment analyses of tumor sample mutated genes indicated that loss of Lkb1 promoted the selection of altered genes associated with neural differentiation processes. Thus, these results suggest that the loss of Lkb1 cooperates with BrafV600E and UVR, impairing the DDR and increasing melanoma multiplicity and neural-like dedifferentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468890566d37157a91439461401c20d4b0665a15" target='_blank'>
              Loss of Lkb1 cooperates with BrafV600E and ultraviolet radiation, increasing melanoma multiplicity and neural-like dedifferentiation.
              </a>
            </td>
          <td>
            K. McGrail, Elena González-Sánchez, Paula Granado-Martínez, Roberto Orsenigo, Yuxin Ding, Berta Ferrer, Javier Hernández-Losa, Iván Ortega, Juan Martín-Caballero, Eva Muñoz-Couselo, Vicente García-Patos, Juan A. Recio
          </td>
          <td>2024-08-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeting CDC20 can enhance the radiosensitivity of tumor cells, but the function and mechanism of CDC20 on DNA damage repair response remains vague. To examine that issue, tumor cell lines, including KYSE200, KYSE450, and HCT116, were utilized to detect the expression, function, and underlying mechanism of CDC20 in radio-chemoresistance. Western blot and immunofluorescence staining were employed to confirm CDC20 expression and location, and radiation could upregulate the expression of CDC20 in the cell nucleus. The homologous recombination (HR) and non-homologous end joining (NHEJ) reporter gene systems were utilized to explore the impact of CDC20 on DNA damage repair, indicating that CDC20 could promote HR repair and radio/chemo-resistance. In the early stages of DNA damage, CDC20 stabilizes the RPA1 protein through protein-protein interactions, activating the ATR-mediated signaling cascade, thereby aiding in genomic repair. In the later stages, CDC20 assists in the subsequent steps of damage repair by the ubiquitin-mediated degradation of RPA1. CCK-8 and colony formation assay were used to detect the function of CDC20 in cell vitality and proliferation, and targeting CDC20 can exacerbate the increase in DNA damage levels caused by cisplatin or etoposide. A tumor xenograft model was conducted in BALB/c-nu/nu mice to confirm the function of CDC20 in vivo, confirming the in vitro results. In conclusion, this study provides further validation of the potential clinical significance of CDC20 as a strategy to overcome radio-chemoresistance via uncovering a novel role of CDC20 in regulating RPA1 during DNA damage repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ebddd7f8266727872b7593baec6eddbe1783f9" target='_blank'>
              CDC20 Holds Novel Regulation Mechanism in RPA1 during Different Stages of DNA Damage to Induce Radio-Chemoresistance
              </a>
            </td>
          <td>
            Yang Gao, Pengbo Wen, Chenran Shao, Cheng Ye, Yuji Chen, Junyu You, Zhongjing Su
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is decisive for genetic stability. DSB repair act according to two steps: 1-the choice between nonhomologous end-joining (NHEJ) versus resection of DNA ends, generating single-stranded DNA (ssDNA); 2-on ssDNA, homologous recombination (HR) and sub-processes. Here we show that 53BP1, which plays a prime role at the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of the cAMP dependant protein kinase A (PKAcs). PKA favors the recruitment of 53BP1 at DNA damages sites, and consistently, prevents resection, favoring NHEJ. Inhibition of PKA stimulated resection and coherently reduces NHEJ. Reversely, activation of PKA with 8-Bromo-cAMP stimulate NHEJ and reduces HR. These data open new avenues for potential anticancer strategies. More generally, they underline the high complexity of the regulation of DSB repair, identifying PKA as a novel actor of the DNA damage response, acting at the choice of the DSB repair process, which might be essential for genome integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb615918d35bbed71f2c3e0b64876e126043052" target='_blank'>
              PKA prevents resection of DNA double strand breaks and favor non-homologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, I. Magdalou, Vasily Ogryzko, Bernard S. Lopez
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Polyploidization plays a crucial role in plant evolution and is becoming increasingly important in breeding. Structural variations and epigenomic repatterning have been observed in synthetic polyploidizations. However, the mechanisms underlying the occurrence and their effects on gene expression and phenotype remain unknown. Here, we investigated genome-wide large deletion/duplication regions (DelDups) and genomic methylation dynamics in leaf organs of progeny from the first eight generations of synthetic tetraploids derived from Chinese cabbage (Brassica rapa L. ssp. pekinensis) and cabbage (Brassica oleracea L. var. capitata). One- or two-copy DelDups, with a mean size of 5.70 Mb (400 kb - 65.85 Mb), occurred from the first generation of selfing and thereafter. The duplication of a fragment in one subgenome consistently coincided with the deletion of its syntenic fragment in the other subgenome, and vice versa, indicating that these DelDups were generated by homoeologous exchanges (HEs). Interestingly, the larger the genomic syntenic region, the higher the frequency of DelDups, further suggesting that the pairing of large homoeologous fragments is crucial for HEs. Moreover, we found that the active transcription of continuously distributed genes in local regions is positively associated with the occurrence of HE breakpoints. In addition, the expression of genes within DelDups exhibited a dosage effect, and plants with extra parental genomic fragments generally displayed phenotypes biased towards the corresponding parent. Genome-wide methylation fluctuated remarkably, which did not clearly affect gene expression on a large scale. Our findings provide insights into the early evolution of polyploid genomes, offering valuable knowledge for polyploidization-based breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc22507c27b064911f132b6bdaa4330162e63d6f" target='_blank'>
              Regional Active Transcription Associates with Homoeologous Exchange Breakpoints in Synthetic Brassica Tetraploids.
              </a>
            </td>
          <td>
            Chengcheng Cai, Yinqing Yang, Lei Zhang, Yinan Cui, Jian Wu, Jianli Liang, Xing Li, Lingkui Zhang, Xin Zhang, Yiyue Zhang, Zhongwei Guo, Shumin Chen, Kang Zhang, M. Freeling, Xiaowu Wang, Feng Cheng
          </td>
          <td>2024-08-20</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor in adults and is notorious for resistance to treatments. Chemotherapy is largely based on alkylating agents that induce lethal secondary DNA double-strand breaks (DSB). Therefore, modulation of DNA DSB-repair pathways that are involved in the innate or acquired tumor resistance, is frequently under consideration for improved chemotherapy. In the present study, we used a RAD51 inhibitor to modulate the DSB repair pathway homologous recombination (HR) and a PARPi to abolish the PARP-dependent non-homologous end-joining (NHEJ) as a backup DSB-repair mechanism in human glioblastoma cells with functional p53 and PTEN. In cell culture, synthetic lethality-like effects of combined treatment with PARPi and RAD51i were observed as well as increased sensitivity to temozolomide when combined with RAD51i or PARPi. In vivo in the orthotopic model, combination of PARPi and RAD51i significantly increased mouse median survival and reduced the tumorigenic potential of cancer xenoplants. Improved survival correlated with increased DNA damage and apoptotic cell numbers, as well as reduced numbers of proliferating cells in tumor brain sections of mice treated with PARPi and RAD51i. Our results suggest that combining RAD51i with PARPi might be beneficial for glioblastoma treatment and suggests RAD51 inhibition as novel strategy in glioma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064f750891c9080de686323c9c30b6ded299a104" target='_blank'>
              Targeting DNA Repair Pathways in Orthotopic Glioblastoma Xenoplants
              </a>
            </td>
          <td>
            AA Müller, TG Hofmann
          </td>
          <td>2024-07-15</td>
          <td>Journal of Oncology Research and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BRAF and MEK inhibitors (BRAF/MEKi) have radically changed the treatment landscape of advanced BRAF mutation-positive tumours. However, limited efficacy and emergence of drug resistance are major handicaps for successful treatments. Here, by using relevant preclinical models, we found that Connexin43 (Cx43), a protein that plays a role in cell-to-cell communication, increases effectiveness of BRAF/MEKi by recruiting DNA repair complexes to lamin-associated domains and promoting persistent DNA damage and cellular senescence. The nuclear compartmentalization promoted by Cx43 contributes to genome instability and synthetic lethality caused by excessive DNA damage, which could lead to a novel therapeutic approach for these tumours to overcome drug resistance. Based on these findings, we designed an innovative drug combination using small extracellular vesicles (sEVs) to deliver the full-Cx43 in combination with the BRAF/MEKi. This study reveals Cx43 as a new player on DNA repair and BRAF/MEKi response, underlining the therapeutical potential that this approach could eventually have in the clinic to overcome the limitations of current therapies and improve treatment outcomes for patients with advanced BRAF mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2931f384d734ebe11eb6f5f59fbfc1a5a5d9c4e2" target='_blank'>
              Cx43 Enhances Response to BRAF/MEK Inhibitors by Reducing DNA Repair Capacity
              </a>
            </td>
          <td>
            A. Varela-Vázquez, A. Guitián-Caamaño, P. Carpintero-Fernández, Vanesa Álvarez, Alexander Carneiro-Figueira, M. Varela-Eirín, Teresa Calleja-Chuclá, S. B. Bravo-López, Anxo Vidal, Juan Sendón-Lago, M. Rodríguez-Candela Mateos, José R Caeiro, Miguel G. Blanco, Guadalupe Sabio, María Quindós, Carmen Rivas, David Santamaría, Carlos Fernández-Lozano, E. Fonseca, Pablo Huertas, Berta Sanchez-Laorden, Constance Alabert, M. Mayán
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromatin within the nucleus adopts complex three-dimensional structures that are crucial for gene regulation and cellular function. Recent studies have revealed the presence of distinct chromatin subcompartments beyond the traditional A/B compartments (eu- and hetero-chromatin), each exhibiting unique structural and functional properties. Here, we introduce TECSAS (Transformer of Epigenetics to Chromatin Structural AnnotationS), a deep learning model based on the Transformer architecture, designed to predict chromatin subcompartment annotations directly from epigenomic data. TECSAS leverages information from histone modifications, transcription factor binding profiles, and RNA-Seq data to decode the relationship between the biochemical composition of chromatin and its 3D structural behavior. TECSAS achieves high accuracy in predicting subcompartment annotations and reveals the influence of long-range epigenomic context on chromatin organization. Furthermore, we demonstrate the model’s capability to predict the association of loci with nuclear bodies, such as the lamina, nucleoli, and speckles, providing insights into the role of these structures in shaping the 3D genome organization. This study highlights the potential of deep learning models for deciphering the complex interplay between epigenomic features and 3D genome organization, allowing us to better understand genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46d3eeafc78bbe750f181271e73a57733705362e" target='_blank'>
              Epigenetics is all you need: A Transformer to decode chromatin structural compartments from the epigenome
              </a>
            </td>
          <td>
            E. Dodero-Rojas, V. Contessoto, Yao Fehlis, Nicolas Mayala, J. Onuchic
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>154</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-07-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Genomic imprinting is the primary epigenetic phenomenon that results in parent-of-origin effects on mammalian development and growth. Research on genomic imprinting in domesticated animals has lagged due to a primary focus on orthologous imprinted genes. This emphasis has limited the discovery of imprinted genes specific to livestock. To identify genomic imprinting in pigs, we generated parthenogenetic porcine embryos alongside biparental normal embryos, and then performed whole-genome bisulfite sequencing and RNA sequencing on these samples. Results In our analyses, we discovered a maternally methylated differentially methylated region within the orthologous ZNF791 locus in pigs. Additionally, we identified both a major imprinted isoform of the ZNF791-like gene and an unannotated antisense transcript that has not been previously annotated. Importantly, our comparative analyses of the orthologous ZNF791 gene in various eutherian mammals, including humans, non-human primates, rodents, artiodactyls, and dogs, revealed that this gene is subjected to genomic imprinting exclusively in domesticated animals, thereby highlighting lineage-specific imprinting. Furthermore, we explored the potential mechanisms behind the establishment of maternal DNA methylation imprints in porcine and bovine oocytes, supporting the notion that integration of transposable elements, active transcription, and histone modification may collectively contribute to the methylation of embedded intragenic CpG island promoters. Conclusions Our findings convey fundamental insights into molecular and evolutionary aspects of livestock species-specific genomic imprinting and provide critical agricultural implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae305979ce13f1ce99e877938bff267ad6753a3a" target='_blank'>
              Lineage-specific genomic imprinting in the ZNF791 locus
              </a>
            </td>
          <td>
            J. Ahn, I. Hwang, Mi-Ryung Park, Milca Rosa-Velázquez, In-Cheol Cho, A. Relling, Seongsoo Hwang, Kichoon Lee
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The epigenetic layer of regulation has become increasingly relevant in the research focused on tumor suppressors. KLF4 is a well-described zinc-finger transcription factor, mainly known for its role in the acquisition of cell pluripotency. Here we report and describe the most relevant epigenetic regulation mechanisms that affect KLF4 expression in tumors. CpG island methylation emerges as the most common mechanism in several tumors including lung adenocarcinoma, hepatocellular carcinoma, non-Hodgkin lymphomas, among others. Further layers of regulation represented by histone methylation and acetylation and by non-coding RNAs are described. Overall, KLF4 emerges as a crucial target in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e9c4ef46a7844e1c0cc57e2b10d6f8fefd7118" target='_blank'>
              KLF4 is an epigenetically modulated, context-dependent tumor suppressor
              </a>
            </td>
          <td>
            R. Frazzi
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4a5e737d6cb299cb80200022679595c834f0d4" target='_blank'>
              The novel circFKBP8/miR-432-5p/E2F7 cascade functions as a regulatory network in breast cancer
              </a>
            </td>
          <td>
            Zhongkui Jin, Wang Xu, Kunlin Yu, Cailu Luo, Xiaodan Luo, Tao Lian, Changshan Liu
          </td>
          <td>2024-08-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="All cancers are associated with massive reorganisation of cellular epigenetic patterns, including extensive changes in the genomic patterns of DNA methylation. However, the huge scale of these changes has made it very challenging to identify key DNA methylation changes responsible for driving cancer development. Here, we present a novel approach to address this problem called methylation mapping. Through comparison of multiple types of B-lymphocyte derived malignancies and normal cell populations, this approach can define the origins of methylation changes as proliferation-driven, differentiation-driven and disease-driven (including both cancer-specific changes and cancer absent changes). Each of these categories of methylation change were found to occur at genomic regions that vary in sequence context, chromatin structure and associated transcription factors, implying underlying mechanistic differences behind the acquisition of methylation at each category. This analysis determined that only a very small fraction (about 3%) of DNA methylation changes in B-cell cancers are disease related, with the overwhelming majority (97%) being driven by normal biological processes, predominantly cell proliferation. Furthermore, the low level of true disease-specific changes can potentially simplify identification of functionally relevant DNA methylation changes, allowing identification of previously unappreciated candidate drivers of cancer development, as illustrated here by the identification and functional confirmation of SLC22A15 as a novel tumour suppressor candidate in acute lymphoblastic leukaemia. Overall, this approach should lead to a clearer understanding of the role of altered DNA methylation in cancer development, facilitate the identification of DNA methylation targeted genes with genuine functional roles in cancer development and thus identify novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f876685ea691f693bc76361ff33252fea93a62c" target='_blank'>
              Deconvolution of cancer methylation patterns determines that altered methylation in cancer is dominated by a non-disease associated proliferation signal
              </a>
            </td>
          <td>
            H. Lalchungnunga, H. Atasoy, EC Schwalbe, CM Bacon, G. Strathdee
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cfb136406d6e0103812e5102a4ed1670edf2da" target='_blank'>
              METTL3-mediated m6A modification of circGLIS3 promotes prostate cancer progression and represents a potential target for ARSI therapy
              </a>
            </td>
          <td>
            Xiaofeng Cheng, Heng Yang, Yujun Chen, Zhenhao Zeng, Yifu Liu, Xiaochen Zhou, Cheng Zhang, An Xie, Gongxian Wang
          </td>
          <td>2024-08-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment.
              </a>
            </td>
          <td>
            M. Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-02</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1f187326818c3bc4b9634f53afc541e906548d2" target='_blank'>
              Targeting PRMT5 enhances the radiosensitivity of tumor cells grown in vitro and in vivo
              </a>
            </td>
          <td>
            Charlotte M. Degorre, Steven Lohard, Christina N. Bobrek, Komal N. Rawal, Skyler Kuhn, P. Tofilon
          </td>
          <td>2024-07-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are involved in the pathogenesis of esophageal squamous cell carcinoma (ESCC). This study aimed to explore the mechanisms of aberrant expression and functions of circ_0006168 in ESCC. In this study, real-time qPCR and fluorescence in situ hybridization (FISH) are adopted to estimate the expression and localization of circ_0006168 in cancer tissues and cells. Methylated RNA immunoprecipitation (MeRIP) was performed to detect the N6-methyladenosine (m6A) modification of circ_0006168. Gain- and loss-of-functions of circ_0006168 were performed to identify its role in ESCC progression. RNA-binding protein immunoprecipitation (RIP) was used to detect the interaction of circ_0006168 with IGF2BP2. Luciferase reporter assay and RIP are used to confirm the circ_0006168/miR-384/STAT3 ceRNA network. Our results showed that the expression of circ_0006168 was upregulated in ESCC tissues and cells. METTL3-mediated m6A modification increased the expression of circ_0006168 via IGF2BP2-dependent way in TE-1 cells. Circ_0006168 promoted cell proliferation, migration, invasion, cell cycle progression and inhibited cell apoptosis, while knockdown of circ_0006168 had the reverse effects. Mechanistically, circ_0006168 acted its functions via miR-384/STAT3/Snail axis in TE-1 cells. In conclusion, circ_0006168 is upregulated in ESCC and m6A methylation increased its expression via IGF2BP2. CircRNA_0006168 promotes cell migration, invasion by regulating EMT via miR-384/STAT3/Snail axis in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5633a4be443d2222f80bed6a025bc3273be4d5d6" target='_blank'>
              N6-methyladenosine-modified circ_0006168 promotes epithelial mesenchymal transition via miR-384/STAT3/Snail axis in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Guandi Wu, Qin Hou, Zhe Liu, Zejin Pu, Lingfei Wu
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Microphthalmia associated transcription factor (Mitf) is a critical regulator of the melanocyte lineage and also plays an important role in eye development. Mitf activity in different cell types is controlled in part by ten alternative promoters and their resulting isoforms. A useful tool for melanocyte-based research, the Mitf-cre transgene was designed to express Cre recombinase from the Mitf-M promoter, which is melanocyte specific. However, Mitf-cre mice are also microphthalmic, perhaps because of insertional mutagenesis or disrupted gene expression. Here, we investigated these possibilities. We determined that the eye phenotype arises early, with Mitf-cre embryos at E13.5 exhibiting variable ocular sizes and abnormalities, but all with coloboma. Targeted locus amplification and next generation sequencing indicated that multiple copies of the transgene integrated into an intergenic region on chromosome 2, in between Spred1 and Meis2. The BAC transgene used to make Mitf-cre was larger than expected, carrying three upstream alternative promoters, Mitf-H, Mitf-D, and Mitf-B, which could express their isoforms intact off the transgene. RT-qPCR using eye tissue demonstrated a 5-fold increase in Mitf transcripts containing exon 1B1b, which is shared by Mitf-H, Mitf-D, and Mitf-B, while Spred1 and Meis2 did not differ in their expression. These findings clarify and support the usage of Mitf-cre in conditional mutagenesis in melanocytes. The specific over-expression of the Mitf-H and Mitf-D isoforms, which are preferentially expressed in the RPE, presents a unique resource for those interested in eye development and coloboma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f416be172f324512ebeadaba72cc51d72c665bd9" target='_blank'>
              Mitf over-expression leads to microphthalmia and coloboma in Mitf-cre mice
              </a>
            </td>
          <td>
            A. Longakit, Hannah Bourget, C. D. Van Raamsdonk
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/609bb25145d5246600ce4feb000ce7f09c4474b7" target='_blank'>
              Building a translational cancer dependency map for The Cancer Genome Atlas
              </a>
            </td>
          <td>
            Xu Shi, Christos Gekas, Daniel Verduzco, Sakina M. Petiwala, Cynthia Jeffries, Charles Lu, Erin Murphy, Tifani Anton, Andy H Vo, Zhiguang Xiao, Padmini Narayanan, Bee-Chun Sun, Aloma L D'Souza, J. M. Barnes, Somdutta Roy, Cyril Ramathal, Michael J. Flister, Zoltan Dezso
          </td>
          <td>2024-07-15</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd29289a45e4a7c7bcab171c48d3c292a1ad2638" target='_blank'>
              Targeting DNA Damage Response Deficiency in Thoracic Cancers.
              </a>
            </td>
          <td>
            A. Bzura, Jake B Spicer, S. Dulloo, T.A. Yap, D. Fennell
          </td>
          <td>2024-07-13</td>
          <td>Drugs</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. ATAC-seq and ChIP-seq revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLC. Analyses of human SCLC tumors and patient-derived xenografts (PDXs) also showed increase in TREX1 expression in post-chemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 induces innate immune response in drug-resistant Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J. Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the tumor-suppressor genes BRCA1 and BRCA2 resulting in BRCA1/2 deficiency are frequently identified in breast, ovarian, prostate, pancreatic, and other cancers. Poly(ADP-ribose) polymerase (PARP) inhibitors (PARPis) selectively kill BRCA1/2-deficient cancer cells by inducing synthetic lethality, providing an effective biomarker-guided strategy for targeted cancer therapy. However, a substantial fraction of cancer patients carrying BRCA1/2 mutations do not respond to PARPis, and most patients develop resistance to PARPis over time, highlighting a major obstacle to PARPi therapy in the clinic. Recent studies have revealed that changes of specific functional defects of BRCA1/2-deficient cells, particularly their defects in suppressing and protecting single-stranded DNA gaps, contribute to the gain or loss of PARPi-induced synthetic lethality. These findings not only shed light on the mechanism of action of PARPis, but also lead to revised models that explain how PARPis selectively kill BRCA-deficient cancer cells. Furthermore, new mechanistic principles of PARPi sensitivity and resistance have emerged from these studies, generating potentially useful guidelines for predicting the PARPi response and design therapies for overcoming PARPi resistance. In this Review, we will discuss these recent studies and put them in context with the classic views of PARPi-induced synthetic lethality, aiming to stimulate the development of new therapeutic strategies to overcome PARPi resistance and improve PARPi therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2cad9670fd9d04e8ac0fa76598900aad338e25c" target='_blank'>
              BRCAness, DNA gaps, and gain and loss of PARP inhibitor–induced synthetic lethality
              </a>
            </td>
          <td>
            Xin Li, Lee Zou
          </td>
          <td>2024-07-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891d9fb97d4fae14734f86d5adf86855004117f2" target='_blank'>
              Clonal inactivation of TERT impairs stem cell competition
              </a>
            </td>
          <td>
            Kazuteru Hasegawa, Yang Zhao, A. Garbuzov, M. Corces, P. Neuhöfer, Victoria M Gillespie, Peggie Cheung, J. Belk, Yung-Hsin Huang, Yuning Wei, Lu Chen, Howard Y. Chang, S. Artandi
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, Yu-Hang Xing, Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/948860eb063cb703e9fda97c23b09a74cb48fa42" target='_blank'>
              Secrets of DNA-PKcs beyond DNA repair
              </a>
            </td>
          <td>
            Sydney Camfield, Sayan Chakraborty, Shailendra Kumar Dhar Dwivedi, Pijush Kanti Pramanik, Priyabrata Mukherjee, Resham Bhattacharya
          </td>
          <td>2024-07-23</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9824378cb72ff7a6ac9e3450ad8fb3f2d8abaa3a" target='_blank'>
              Human genomic DNA is widely interspersed with i-motif structures.
              </a>
            </td>
          <td>
            C. D. Peña Martinez, Mahdi Zeraati, R. Rouet, Ohan Mazigi, Jake Y Henry, Brian Gloss, J. Kretzmann, Cameron W Evans, E. Ruggiero, Irene Zanin, Maja Marušič, J. Plavec, Sara N. Richter, Tracy M. Bryan, Nicole M Smith, Marcel E. Dinger, Sarah Kummerfeld, Daniel Christ
          </td>
          <td>2024-08-29</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The holoparasitic plant Lophophytum mirabile exhibits remarkable levels of mitochondrial horizontal gene transfer (HGT). Gathering comparative data from other individuals and host plants can provide insights into the HGT process. We sequenced the mitochondrial genome (mtDNA) from individuals of two species of Lophophytum and from mimosoid hosts. We applied a stringent phylogenomic approach to elucidate the origin of the whole mtDNAs, estimate the timing of the transfers, and understand the molecular mechanisms involved. Ancestral and recent HGT events replaced and enlarged the multichromosomal mtDNA of Lophophytum spp., with the foreign DNA ascending to 74%. A total of 14 foreign mitochondrial chromosomes originated from continuous regions in the host mtDNA flanked by short direct repeats. These foreign tracts are circularized by microhomology-mediated repair pathways and replicate independently until they are lost or they eventually recombine with other chromosomes. The foreign noncoding chromosomes are variably present in the population and likely evolve by genetic drift. We present the 'circle-mediated HGT' model in which foreign mitochondrial DNA tracts become circular and are maintained as plasmid-like molecules. This model challenges the conventional belief that foreign DNA must be integrated into the recipient genome for successful HGT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3d29e903592234bfcff719a1e822abd49a6998" target='_blank'>
              Horizontally transferred mitochondrial DNA tracts become circular by microhomology-mediated repair pathways.
              </a>
            </td>
          <td>
            M. Roulet, Luis F. Ceriotti, L. Gatica-Soria, M. Sanchez-Puerta
          </td>
          <td>2024-07-23</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a global health concern with persistently high incidence and mortality rates. However, the specific pathogenesis of CRC remains poorly understood. This study aims to investigate the role and pathogenesis of serine and arginine rich splicing factor 10 (SRSF10) in colorectal cancer. Methods Bioinformatics analysis was employed to predict SRSF10 gene expression in CRC patients. Functional experiments involving SRSF10 knockdown and overexpression were conducted using CCK8, transwell, scratch assay, and flow cytometry. Additionally, the PRIdictor website was utilized to predict the SRSF10 interaction site with RFC5. The identification of different transcripts of SRSF10-acting RFC5 pre-mRNA was achieved through agarose gel electrophoresis. Result The knockdown of SRSF10 inhibited the proliferation and migration ability of CRC cells, while promoting apoptosis and altering the DNA replication of CRC cells. Conversely, when SRSF10 was highly expressed, it enhanced the proliferation and migration ability of CRC cells and caused changes in the cell cycle of colorectal cancer cells. This study revealed a change in the replicating factor C subunit 5 (RFC5) gene in colorectal cancer cells following SRSF10 knockdown. Furthermore, it was confirmed that SRSF10 increased RFC5 exon2-AS1(S) transcription variants, thereby promoting the development of colorectal cancer through AS1 exclusion to exon 2 of RFC5. Conclusion In summary, this study demonstrates that SRSF10 promotes the progression of colorectal cancer by generating an aberrantly spliced exclusion isoform of AS1 within RFC5 exon 2. These findings suggest that SRSF10 could serve as a crucial target for the clinical diagnosis and treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183e863439be0cd3db11d90e783d10b2b9aa76e8" target='_blank'>
              High Expression of SRSF10 Promotes Colorectal Cancer Progression by Aberrant Alternative Splicing of RFC5
              </a>
            </td>
          <td>
            Shuai Xu, Fangmin Zhong, Junyao Jiang, Fangyi Yao, Meiyong Li, Mengxin Tang, Ying Cheng, Yulin Yang, Wen Wen, Xueru Zhang, Bo Huang, Xiao-Zhong Wang
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. We specify that the DnaA/DNA ratio may be a more rigorous feature in determining the variability of initiation time in E. coli than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work is a continued examination of our hypothesis of the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the variable DnaA/DNA ratio may serve as an essential source of variability of the replication initiation event and the ensuing chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi A. Gheber, Mario Feingold, I. Fishov
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Homologous recombination (HR) is a universally conserved mechanism of DNA strand exchange between homologous sequences, driven in bacteria by the universal recombinase RecA. HR is key for the maintenance of bacterial genomes via replication fork restart and DNA repair, as well as for their plasticity via the widespread mechanism of natural transformation. Transformation involves the capture and internalisation of exogenous DNA in the form of single strands (ssDNA), followed by chromosomal integration via HR. In the human pathogen Streptococcus pneumoniae, transformation occurs during a transient, stress-induced physiological state called competence. RecA and its partner DNA branch migration translocase RadA both cooperate in HR during transformation and in some recombinational DNA repair pathways of genome maintenance. Both recA and radA genes are basally expressed and transcriptionally induced during competence. In this study, we explored the importance of competence induction of recA and radA expression in transformation and genome maintenance processes. We confirmed that competence induction of recA was important for optimal transformation, but found this was not the case for radA. In contrast, the competence induction of both genes was required for optimal tolerance faced with transient exposure to the lethal genotoxic agent methyl methanesulfonate (MMS). However, this was not the case for another DNA-damaging agent, norfloxacin. These results show that competence induction of HR effectors is important for the increased tolerance to genotoxic stress provided to competent pneumococcal cells. This reinforces the finding that pneumococcal competence is a stress-sensing mechanism, transiently increasing the expression of some genes not to optimise transformation but to improve survival faced with specific lethal stresses. Importance Homologous recombination (HR) is a mechanism of DNA strand exchange important for both the maintenance and plasticity of bacterial genomes. Bacterial HR is driven by the recombinase RecA along with many accessory partner proteins, which define multiple dedicated pathways crucial to genome biology. Thus, a main mechanism of genome plasticity in bacteria is natural genetic transformation, which involves uptake and chromosomal integration of exogenous DNA via HR. In the human pathogen Streptococcus pneumoniae, transformation occurs during a transient, stress-induced physiological state called competence. RecA and the helicase RadA are key for both genome maintenance and transformation, and both are over-produced during competence. Here, we explore the importance of this over-production for transformation and genome maintenance, quantified by tolerance to genotoxic stress. While over-production of RecA was important for both processes, over-production of RadA was required only for genotoxic stress tolerance. This highlights the importance of competence as a stress-responsive mechanism, with induction of HR genes important for genotoxic stress tolerance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e28357f136a9c3076953939ae89a7a4058fe918" target='_blank'>
              Competence induction of homologous recombination genes protects pneumococcal cells from genotoxic stress
              </a>
            </td>
          <td>
            David De Lemos, Anne-Lise Soulet, Violette Morales, Mathieu Berge, P. Polard, Calum H. G. Johnston
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3239b6a328047bdf1fed75ded2d7ba86fd49bc1" target='_blank'>
              RBM8A, a new target of TEAD4, promotes breast cancer progression by regulating IGF1R and IRS-2
              </a>
            </td>
          <td>
            Fang Li, Xiaofei Wang, Jing Zhang, Jin-yan Zhang, X. Jing, Q. Jiang, Jing Zhou, Li Cao, Hang Peng, D. Tong, Chen Huang
          </td>
          <td>2024-09-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Detection of minor DNA allele alterations is becoming increasingly important for early detection and monitoring of cancer. We describe a new method that uses ultraviolet light to eliminate wild-type DNA alleles and enables improved detection of minor genetic or epigenetic changes.


METHODS
Pyrimidine-dependent UV-based minor-allele enrichment (PD-UVME) employed oligonucleotide probes that incorporated a UVA-sensitive 3-cyanovinylcarbazole (CNVK), placed directly opposite interrogated pyrimidines, such as thymine (T) or cytosine (C) in wild-type (WT) DNA. Upon UVA-illumination, CNVK cross-linked with T/C, preventing subsequent amplification. Mutations that removed the T/C escaped cross-linking and were amplified and detected. Similarly, CNVK discriminated between methylated and unmethylated cytosine in CpG dinucleotides, enabling direct enrichment of unmethylated DNA targets. PD-UVME was combined with digital droplet PCR (ddPCR) to detect serine/threonine-protein kinase B-Raf (BRAF) V600E mutations in model systems, thyroid patient cancer tissue samples, and circulating DNA of tumor origin (ctDNA) from melanoma patients.


RESULTS
One thyroid cancer sample out of 9, and 6 circulating-DNA samples out of 7 were found to be BRAF V600E-positive via PD-UVME while classified as negative by conventional ddPCR. Positive samples via conventional ddPCR were also found positive via PD-UVME. All 10 circulating cell-free DNA (cfDNA) samples obtained from normal volunteers were negative via both approaches. Furthermore, preferential enrichment of unmethylated alleles in MAGEA1 promoters using PD-UVME was demonstrated.


CONCLUSIONS
PD-UVME mutation/methylation enrichment performed prior to ddPCR magnifies low-level mutations or epigenetic changes and increases sensitivity and confidence in the results. It can assist with clinical decisions that hinge on the presence of trace alterations like BRAF V600E.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362fd25c2fb0f063f30c56509492e4fe9cd161b" target='_blank'>
              Pyrimidine-Dependent UV-Mediated Cross-Linking Magnifies Minor Genetic or Epigenetic Changes in Clinical Samples.
              </a>
            </td>
          <td>
            F. Yu, Farzana Ahmed, S. Smilkou, S. Yasmin-Karim, Farzaneh Darbeheshti, A. Markou, Martyn Bullock, Ioannis Boukovinas, V. Adalsteinsson, E. Lianidou, G. M. Makrigiorgos
          </td>
          <td>2024-07-23</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a1e72cd7656def115fe3bbce4a02021bb17ead" target='_blank'>
              Involvement of the splicing factor SART1 in the BRCA1-dependent homologous recombination repair of DNA double-strand breaks
              </a>
            </td>
          <td>
            Kie Ozaki, Reona Kato, Takaaki Yasuhara, Yuki Uchihara, Miyako Hirakawa, Yu Abe, Hiroki Shibata, Reika Kawabata-Iwakawa, Aizhan Shakayeva, Palina Kot, Kiyoshi Miyagawa, Keiji Suzuki, Naoki Matsuda, Atsushi Shibata, M. Yamauchi
          </td>
          <td>2024-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4d231cf341ecb2bcc080757179bed2d4c422c6" target='_blank'>
              Hypoxia-enhanced YAP1-EIF4A3 interaction drives circ_0007386 circularization by competing with CRIM1 pre-mRNA linear splicing and promotes non-small cell lung cancer progression
              </a>
            </td>
          <td>
            Lixia Li, Dewei Liu, Tingting Chen, Chunhui Wei, Youping Qiao, Weiliang Liu, Yanmei Liang, Zhu Liang, Chunyuan Chen, Dongming Li, Bin Wu, Xuanna Zhao, Dan Huang, Dong Wu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3734fdaa3ed3777431c0f723757629a9b7e4aa2" target='_blank'>
              High-complexity of DNA double-strand breaks is key for alternative end-joining choice
              </a>
            </td>
          <td>
            Zhiyang Hou, Tianxiang Yu, Qiyi Yi, Yan Du, Libin Zhou, Ye Zhao, Yuejin Wu, Lijun Wu, Ting Wang, Po Bian
          </td>
          <td>2024-08-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5395af718a191f5fa3586c7dc9b0e07227b5b78c" target='_blank'>
              Subunit-specific analysis of cohesin-mutant myeloid malignancies reveals distinct ontogeny and outcomes
              </a>
            </td>
          <td>
            Joahnn-Christoph Jann, Christopher B. Hergott, Marisa Winkler, Yiwen Liu, Benjamin Braun, A. Charles, K. Copson, Shougat Barua, M. Meggendorfer, N. Nadarajah, S. Shimony, Eric S. Winer, M. Wadleigh, Richard M Stone, D. DeAngelo, Jacqueline S Garcia, Torsten Haferlach, R. C. Lindsley, M. Luskin, Maximilian Stahl, Zuzana Tothova
          </td>
          <td>2024-07-20</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Chromatin states, fundamental to gene regulation and cellular identity, are defined by a unique combination of histone post-translational modifications. Despite their importance, comprehensive patterns within chromatin state sequences, which could provide insights into key biological functions, remain largely unexplored. In this study, we introduce ChromBERT, a BERT-based model specifically designed to detect distinct patterns of chromatin state annotation data sequences. Notably, ChromBERT was pre-trained on promoter regions across a diverse range of epigenomes and subsequently fine-tuned using a dataset from multiple cell lines where RNA-seq data were available, highlighting the model’s ability to discern conserved chromatin state patterns within these regions. In addition to its predictive powers across tasks, evidenced by high AUC scores, ChromBERT provides further analysis through the incorporation of motif clustering using Dynamic Time Warping (DTW). This method enhances the model’s ability to dissect chromatin state sequence motifs, typically involving transcription and enhancer sites. The introduction of motif clustering with DTW into ChromBERT’s workflow is poised to facilitate the discovery of genomic regions linked to novel biological functions, deepening our understanding of chromatin state dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758a0c4bd29ca771e1573835ff87961f8849bdb7" target='_blank'>
              ChromBERT: Uncovering Chromatin State Motifs in the Human Genome Using a BERT-based Approach
              </a>
            </td>
          <td>
            Seohyun Lee, Che Lin, Chien-Yu Chen, Ryuichiro Nakato
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Telomeres are specialized structures at the ends of linear chromosomes that protect them from degradation and fusion. It’s replication is a complex process that involves both DNA polymerases and a specialized enzyme called telomerase which is a ribonucleoprotein complex that synthesizes telomeric DNA by using an internal RNA template. However, telomerase alone cannot fully replicate the telomeric DNA, and requires the cooperation of other factors, such as shelterin, CST, and DNA repair proteins. Telomere replication is tightly regulated by various mechanisms, such as cell cycle checkpoints, telomere length homeostasis, and telomere position effect and dysregulation of it can lead to genomic instability, cellular senescence, and cancer. Therefore, understanding the molecular details of telomere replication is crucial for elucidating the role of telomeres in aging and disease. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e590a2bd1420d5e3b8ed0e5debba5cfdd475b0f8" target='_blank'>
              Unravelling the Intricacies of Telomere Replication: A Molecular Conundrum
              </a>
            </td>
          <td>
            Neelabh Datta
          </td>
          <td>2024-07-16</td>
          <td>Canadian Journal for the Academic Mind</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary With advanced bioinformatic and sequencing technologies, many non-coding RNAs have been found to harbor non-canonical open reading frames (ncORFs). Some studies suggest that many ncORF-encoded microproteins or micropeptides possess important roles in cancer biology, yet the functional roles of these ncORF-encoded peptides are mostly unknown. The purpose of this review is to provide a comprehensive overview of existing knowledge on ncORF-encoded microproteins in cancer biology and highlight their roles as potential prognostic and therapeutic targets in cancer. Abstract Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The protein sequence and spatial structure of DNA helicase HELQ are highly conserved, spanning from archaea to humans. Aside from its helicase activity, which is based on DNA binding and translocation, it has also been recently reconfirmed that human HELQ possesses DNA–strand–annealing activity, similar to that of the archaeal HELQ homolog StoHjm. These biochemical functions play an important role in regulating various double–strand break (DSB) repair pathways, as well as multiple steps in different DSB repair processes. HELQ primarily facilitates repair in end–resection–dependent DSB repair pathways, such as homologous recombination (HR), single–strand annealing (SSA), microhomology–mediated end joining (MMEJ), as well as the sub-pathways’ synthesis–dependent strand annealing (SDSA) and break–induced replication (BIR) within HR. The biochemical functions of HELQ are significant in end resection and its downstream pathways, such as strand invasion, DNA synthesis, and gene conversion. Different biochemical activities are required to support DSB repair at various stages. This review focuses on the functional studies of the biochemical roles of HELQ during different stages of diverse DSB repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fd5a992183030db3a19f5e1b5b64289405976c" target='_blank'>
              Helicase HELQ: Molecular Characters Fit for DSB Repair Function
              </a>
            </td>
          <td>
            Yuqin Zhao, Kai-Yuan Hou, Yu Liu, Yinan Na, Chao Li, Haoyuan Luo, Hailong Wang
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer.
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec67b6cd2c5b516c2d3f922caad0a44789a5bdd" target='_blank'>
              TRF1 and TRF2: pioneering targets in telomere-based cancer therapy
              </a>
            </td>
          <td>
            Anoop Kallingal, Radosław Krzemieniecki, Natalia Maciejewska, Wioletta Brankiewicz-Kopcińska, Maciej Baginski
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="MOTIVATION
Circulating-cell free DNA (cfDNA) is widely explored as a non-invasive biomarker for cancer screening and diagnosis. The ability to decode the cells of origin in cfDNA would provide biological insights into pathophysiological mechanisms, aiding in cancer characterization and directing clinical management and follow-up.


RESULTS
We developed a DNA methylation signature-based deconvolution algorithm, MetDecode, for cancer tissue origin identification. We built a reference atlas exploiting de novo and published whole-genome methylation sequencing data for colorectal, breast, ovarian and cervical cancer, and blood-cell-derived entities. MetDecode models the contributors absent in the atlas with methylation patterns learnt on-the-fly from the input cfDNA methylation profiles. Additionally, our model accounts for the coverage of each marker region to alleviate potential sources of noise. In-silico experiments showed a limit of detection down to 2.88% of tumour tissue contribution in cfDNA. MetDecode produced Pearson correlation coefficients above 0.95 and outperformed other methods in simulations (p < 0.001; T-test; one-sided). In plasma cfDNA profiles from cancer patients, MetDecode assigned the correct tissue-of-origin in 84.2% of cases. In conclusion, MetDecode can unravel alterations in the cfDNA pool components by accurately estimating the contribution of multiple tissues, while supplied with an imperfect reference atlas.


AVAILABILITY
MetDecode is available at https://github.com/JorisVermeeschLab/MetDecode.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6103b5c6abb6f1e1d9da1b8923c2a277930a1a19" target='_blank'>
              MetDecode: Methylation-based deconvolution of cell-free DNA for non-invasive multi-cancer typing.
              </a>
            </td>
          <td>
            Antoine Passemiers, Stefania Tuveri, Dhanya Sudhakaran, Tatjana Jatsenko, T. Laga, K. Punie, Sigrid Hatse, Sabine Tejpar, A. Coosemans, E. Van Nieuwenhuysen, Dirk Timmerman, G. Floris, A. Van Rompuy, Xavier Sagaert, Antonia Testa, Daniela Ficherova, Daniele Raimondi, Frédéric Amant, L. Lenaerts, Yves Moreau, J. R. Vermeesch
          </td>
          <td>2024-08-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Horizontal gene transfer is a key driver of bacterial evolution, but it also presents severe risks to bacteria by introducing invasive mobile genetic elements. To counter these threats, bacteria have developed various defense systems, including prokaryotic Argonautes (pAgo) and the DNA Defense Module DdmDE system. Through biochemical analysis, structural determination, and in vivo plasmid clearance assays, we elucidate the assembly and activation mechanisms of DdmDE, which eliminates small, multicopy plasmids. We demonstrate that DdmE, a pAgo-like protein, acts as a catalytically inactive, DNA-guided, DNA-targeting defense module. In the presence of guide DNA, DdmE targets plasmids and recruits a dimeric DdmD, which contains nuclease and helicase domains. Upon binding to DNA substrates, DdmD transitions from an autoinhibited dimer to an active monomer, which then translocates along and cleaves the plasmids. Together, our findings reveal the intricate mechanisms underlying DdmDE-mediated plasmid clearance, offering fundamental insights into bacterial defense systems against plasmid invasions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac230afd0ac05cd0569717ab193d8b0059c7437" target='_blank'>
              DdmDE eliminates plasmid invasion by DNA-guided DNA targeting
              </a>
            </td>
          <td>
            Xiao-Yuan Yang, Zhangfei Shen, Chen Wang, Kotaro Nakanishi, Tian-Min Fu
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Senataxin is an RNA:DNA helicase that plays an important role in the resolution of RNA:DNA hybrids (R-loops) formed during transcription. R-loops are involved in the regulation of biological processes such as immunoglobulin class switching, gene expression and DNA repair. Excessive accumulation of R-loops results in DNA damage and loss of genomic integrity. Senataxin is critical for maintaining optimal levels of R-loops to prevent DNA damage and acts as a genome guardian. Within the nucleus, senataxin interacts with various RNA processing factors and DNA damage response and repair proteins. Senataxin interactors include survival motor neuron and zinc finger protein 1, with whom it co-localizes in sub-nuclear bodies. Despite its ubiquitous expression, mutations in senataxin specifically affect neurons and result in distinct neurodegenerative diseases such as amyotrophic lateral sclerosis type 4 and ataxia with oculomotor apraxia type 2, which are attributed to the gain-of-function and the loss-of-function mutations in senataxin, respectively. In addition, low levels of senataxin (loss-of-function) in spinal muscular atrophy result in the accumulation of R-loops causing DNA damage and motor neuron degeneration. Senataxin may play multiple functions in diverse cellular processes; however, its emerging role in R-loop resolution and maintenance of genomic integrity is gaining attention in the field of neurodegenerative diseases. In this review, we highlight the role of senataxin in R-loop resolution and its potential as a therapeutic target to treat neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0918b5cad58e2e8aa658850e44f481a80ca9f6d7" target='_blank'>
              Role of senataxin in R-loop-mediated neurodegeneration
              </a>
            </td>
          <td>
            Annapoorna Kannan, Shyni Gangadharan Leela, Dana Branzei, L. Gangwani
          </td>
          <td>2024-07-15</td>
          <td>Brain Communications</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Oxidative damage triggers micronuclear membrane rupture and defective repair Errors in chromosome segregation during cell division (mitosis) can lead to chromosomal instability, a hallmark of human cancer. These mitotic errors give rise to the formation of micronuclei—small cytoplasmic structures that are spatially separated from the primary nucleus and contain lagging chromosomes or chromosome fragments. Unlike the primary nucleus of a cell, micronuclei frequently undergo irreparable rupture and collapse. This breakdown releases micronuclear DNA into the cytosol, causing DNA damage, chromosomal rearrangements, and genomic instability, thereby contributing to cancer progression. The rupture of micronuclei also activates a component of the innate immune system called the cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING) pathway, leading to tumorrelated inflammation and metastasis (1). On pages 951 and 952 of this issue, Di Bona et al. (2) and Martin et al. (3), respectively, report the molecular underpinnings of micronuclear collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3bfa9de0e1bf705d54ddc01f0ac3869ca5a791" target='_blank'>
              Micronuclear collapse mechanisms in cancer
              </a>
            </td>
          <td>
            Marianna Maddaluno, C. Settembre
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b11c9fdbfd43bb000396d5cb58b5ead9a774473" target='_blank'>
              Hypermethylated RASAL1’s promotive role in chemoresistance and tumorigenesis of choriocarcinoma was regulated by TET2 but not DNMTs
              </a>
            </td>
          <td>
            Xianling Zeng, Rui An, Ruixia Guo, Han Li
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Function as a potential cancer biomarker, DNA methylation shows great significance in cancer diagnosis, prognosis, and treatment monitoring. While the lack of an ultrasensitive, specific, and accurate method at the single-molecule level hinders the analysis of the exceedingly low levels of DNA methylation. Herein, based on the outstanding recognition and digestion ability of methylation-sensitive restriction endonuclease (MSRE), we established a single MSRE-based cascade exponential amplification method, which requires only two ingeniously designed primers and only one recognition site of MSRE for the detection of DNA methylation. Differentiated by MSRE digestion, the cleaved unmethylated DNA is too short to induce any amplification reactions, while methylated DNA remains intact to trigger cascade exponential amplification and the subsequent CRISPR/Cas12a system. By integrating the two exponential amplification reactions, as low as 1 aM methylated DNA can be accurately detected, which corresponds to 6 molecules in a 10 μL system, indicating that our method is more sensitive than single amplification-based methods with the ability to detect DNA methylation at the single-molecule level. In addition, 0.1% methylated DNA can be effectively distinguished from large amounts of unmethylated DNA. Our method is further introduced to exploit the expression difference of DNA methylation among normal cells and cancer cells. Moreover, the visual detection of DNA methylation is also realized by the full hybridization between amplification products and the crRNA of CRISPR/Cas12a. Therefore, the proposed method has great potential to be a promising and robust bisulfite-free method for the detection of DNA methylation at the single-molecule level, which is of great importance for early diagnosis of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be58d60315986e0f65e35e915fb70f5754644223" target='_blank'>
              Single Methylation Sensitive Restriction Endonuclease-Based Cascade Exponential Amplification Assay for Visual Detection of DNA Methylation at Single-Molecule Level.
              </a>
            </td>
          <td>
            Parezhati Pataer, Pengbo Zhang, Zhengping Li
          </td>
          <td>2024-08-02</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNAs (eccDNAs), a type of double‐stranded DNAs (dsDNAs) that facilitate the activation of the DNA sensing machinery, have been implicated in the progression and prognosis of various diseases. While the roles of eccDNAs remain contentious, their significance in diffuse large B‐cell lymphoma (DLBCL) has not been reported. Methods Circular DNA sequencing (circle‐seq) was used to demonstrate the expression profile of eccDNAs in DLBCL, and atomic force microscopy to validate the presence of eccDNAs. CCK‐8 and scRNA‐seq techniques were employed to uncover the activation of eccDNA in the STING pathway, leading to enhanced cell proliferation. Chemotherapeutic drugs were used to test the hypothesis that DNA damage induces the production of eccDNA, thereby activating the STING pathway independent of cGAS. GEO databases were used for verification of the prognosis of the eccDNA‐related genes, and animal models were used to investigate the synergistic effects of DNA damage therapy in combination with STING inhibitors on anti‐tumour responses. Results EccDNAs were widely expressed in DLBCL and associated with the prognosis of patients. Elevated abundance of eccDNAs promoted the progression of DLBCL. Chemotherapeutic drugs‐induced DNA damage triggered the generation of eccDNAs, resulting in the activation of the STING signalling in a cGAS‐independent manner. Moreover, inhibition of STING exerted a synergistic anti‐tumour effect with cisplatin. Conclusions EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. This finding offers a rational therapeutic strategy combining chemotherapy with targeting STING. Highlights EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. The combined treatment of chemotherapeutic drugs with STING inhibitor significantly delayed the tumor progression, providing new insights into the therapeutic strategy for patients with DLBCL, particularly the relapsed and/or refractory (R/R) ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e573ea771bfdb5c81c97bedfb9abf4331d619e8" target='_blank'>
              Multi‐omics integration reveals the oncogenic role of eccDNAs in diffuse large B‐cell lymphoma through STING signalling
              </a>
            </td>
          <td>
            Zijuan Wu, Wei Zhang, Luqiao Wang, Jiayan Leng, Yongle Li, Zhou Fan, Mengtao Zhan, Lei Cao, Yongning Jiang, Yan Jiang, Bing Sun, Jianxin Fu, Jianyong Li, W. Shi, H. Jin
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="R-loops, structures that play a crucial role in various biological processes, are integral to gene expression, the maintenance of genome stability, and the formation of epigenomic signatures. When these R-loops are deregulated, they can contribute to the development of serious health conditions, including cancer and neurodegenerative diseases. The detection of R-loops is a complex process that involves several approaches. These include S9.6 antibody- or RNAse H-based immunoprecipitation, non-denaturing bisulfite footprinting, gel electrophoresis, and electron microscopy. Each of these methods offers unique insights into the nature and behavior of R-loops. In our study, we introduce a novel protocol that has been developed based on a single-molecule DNA combing assay. This innovative approach allows for the direct and simultaneous visualization of RNA:DNA hybrids and replication forks, providing a more comprehensive understanding of these structures. Our findings confirm the transcriptional origin of the hybrids, adding to the body of knowledge about their formation. Furthermore, we demonstrate that these hybrids have an inhibitory effect on the progression of replication forks, highlighting their potential impact on DNA replication and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d0ec1a89d3fa05ea164f315ebe78596d3a2da" target='_blank'>
              Simultaneous Visualization of R-Loops/RNA:DNA Hybrids and Replication Forks in a DNA Combing Assay
              </a>
            </td>
          <td>
            Miroslav Penchev Ivanov, Heather Zecchini, Petra Hamerlik
          </td>
          <td>2024-09-03</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92a939542e292edaa1bc766f527a0552465d51" target='_blank'>
              PTIP epigenetically regulates DNA damage-induced cell cycle arrest by upregulating PRDM1
              </a>
            </td>
          <td>
            Y. Nakata, Shion Nagasawa, Y. Sera, N. Yamasaki, A. Kanai, K. Kobatake, Takeshi Ueda, M. Koizumi, Ichiro Manabe, Osamu Kaminuma, Hiroaki Honda
          </td>
          <td>2024-08-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe961497d985de77bb27297cd1364379e55768f" target='_blank'>
              Circular RNA in cancer.
              </a>
            </td>
          <td>
            Vanessa M. Conn, A. Chinnaiyan, Simon J. Conn
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The fidelity of replication, especially in the presence of DNA damage, is essential for the proper function of cells. Mutations that inactivate genes involved in DNA damage repair or bypass are enriched in several types of cancer cells. Thus, it is important to further our understanding of the mechanisms governing replication fidelity. PCNA is a ring-shaped complex that encircles DNA at the front of the replication fork, at the double-stranded/single-stranded DNA junction. It serves as a processivity factor for the different DNA replication polymerases, allowing them to replicate longer stretches of DNA by physically tethering them to the DNA and preventing their detachment. In addition, PCNA also regulates and coordinates different DNA damage bypass pathways meant to allow DNA replication in the presence of DNA damage. Due to its essentiality and the numerous functions it has in the cell, much is still unclear about PCNA. Here, we utilize PCNA mutants that lower the stability of the PCNA complex on the chromatin, and thus tend to disassociate and fall from the DNA. Using these mutants, we show that PCNA’s physical presence on the DNA can prevent DNA misalignment at repetitive sequences, leading to increased mutation formation. We also show that PCNA-interacting proteins play an important role in strengthening the ring’s stability on the chromatin. Such repetitive sequence-induced mutations are common in several human diseases and it is important to study their formation and the mechanisms guarding against them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4a21eb48fc762d33559ed0c2e7196cd781bc5" target='_blank'>
              Effects of PCNA Stability on the Formation of Mutations
              </a>
            </td>
          <td>
            Matan Arbel-Groissman, Batia Liefshitz, M. Kupiec
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22a32ba49d7731185468f66f4a9a15ddb1efe55" target='_blank'>
              Is cross-species horizontal gene transfer responsible for gallbladder carcinogenesis
              </a>
            </td>
          <td>
            Monika Rajput, Manoj Pandey, R. Dixit, Vijay K. Shukla
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Precise double-strand break (DSB) repair is paramount for genome stability. Homologous recombination (HR) is preferred to repair DSBs when a nearby sister chromatid ensures an error-free template. In Saccharomyces cerevisiae, this preference extends into anaphase and telophase (late mitosis; late-M), despite sister chromatids having been pulled apart. Previously, we identified the nuclear envelope (NE) protein Msc1 as important for late-M DSB repair. Here, we report that Msc1 faces the NE lumen, and its depletion leads to DSB-independent over-compartmentalization of the nucleus, which hampers approximation of sister loci after DSB. Depletion of Msc1 also leads to nuclear pore complex mislocation, a phenotype shared by the highly conserved NE healing complex ESCRT-III. Critically, we show that both Msc1 and ESCRT-III also ensure DSB repair in G2/M, and that there is synergism between Msc1 and the ESCRT-III subunit Snf7. These findings highlight the essential role of NE health in DSB repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/411d62fef8c8eeb83c687f479b3594e76b289bd0" target='_blank'>
              Continuous nuclear envelope surveillance is required for DNA double-strand break repair
              </a>
            </td>
          <td>
            Sara Medina-Suárez, F. Machín
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaf9c2f335952f249d3df58f150d64ce452ea5c" target='_blank'>
              CircXPO6 promotes breast cancer progression through competitively inhibiting the ubiquitination degradation of c-Myc.
              </a>
            </td>
          <td>
            Aiqi Xu, Xi Li, Qiaoting Cai, Ciqiu Yang, Mei Yang, Hongfei Gao, M. Cheng, Xianzhe Chen, F. Ji, Hailin Tang, Kun Wang
          </td>
          <td>2024-08-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, Andreas Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction Pediatric acute myeloid leukemia (AML) with NUP98 rearrangements (NUP98r) is associated with dismal prognoses. NUP98 fusion partners are linked with specific AML subtypes, such as NUP98::NSD1 with myeloblastic AML and NUP98::KDM5A with acute megakaryocytic leukemia (AMKL). Although mostly present in AML, various NUP98r are also found in other malignancies such as T-cell acute lymphoblastic leukemia (T-ALL) and myelodysplastic syndrome (MDS). Despite genomic studies showing associations between FLT3 internal tandem duplications (FLT3-ITD) and WT1 mutations with NUP98::NSD1 AML, the genome-wide mutational status of NUP98r leukemias is to be elucidated. Also, experimental models showed that NUP98 fusion oncoproteins (FOs) regulate gene expression by binding to genomic loci such as HOX genes or MEIS1; however, how different partners contribute to specific diseases remains unstudied. Methods We established a pediatric NUP98r leukemia cohort consisting of 185 samples from 177 cases from our institute, public databases, or collaborators. AML is the dominant disease (n=154), whereas it also included T-ALL (n=18), therapy-related myeloid neoplasms (t-MN, n=4), and MDS (n=1). RNA-sequencing (RNA-seq) data was obtained for all samples, and whole-genome/exome sequencing (WGS/WES) data was obtained from 94 samples. Fusion genes were called from RNA-seq. Somatic alterations were called from both WGS/WES and RNA-seq. The epigenetic status and NUP98 FO-binding of 14 cases were studied using CUT&RUN with antibodies against the N-terminal NUP98, H3K27ac, H3K4me, and K3H27ac. Results Among 177 cases, NUP98::NSD1 was most prevalent (n=92) followed by NUP98::KDM5A (n=45). This cohort also included 11 cases with NUP98::RAP1GDS1 enriched in T-ALL and immature AML, rarely reported in previous NUP98r studies. Genomic profiling confirmed known associations with NUP98::NSD1 AML with FLT3-ITD or WT1, along with novel associations represented by NOTCH1, CCND3, and NRAS mutations with NUP98r T-ALL. Transcriptional analyses with other AML subtypes (n=821) showed NUP98r cases are distributed in five large clusters, where fusions and mutations were associated with specific clusters but not exclusively, suggesting that fusion partners, cooperating alterations, and differentiation states contribute to the disease phenotypes. Epigenetic profiling of 14 cases (NSD1: n=5, KDM5A: n=6, RAP1GDS1: n=3) showed that H3K27ac patterns reflect disease types while H3K27me3 and NUP98-FO binding patterns are intrinsic to fusion partners. NUP98 FOs bound to genes commonly expressed in NUP98r leukemia such as HOXA-B genes or ETV6, whereas NUP98-FOs bound to differentiation-related genes depending on the leukemia subtype (e.g., NUP98::KDM5A binding to the GFI1B loci in AMKL). Conclusion These data suggest that NUP98 FOs specify disease types by directly regulating differentiation-related genes. Further investigation by introducing somatic mutations (e.g., WT1 mutations, RB1 loss) in NUP98r models would reveal its contribution to gene regulation and disease types.
 Citation Format: Masayuki Umeda, Nicole Michmerhuizen, Ryan Hiltenbrand, Juan M. Barajas, Michael P. Walsh, Guangchun Song, Jing Ma, Tamara Westover, Cristina Mecucci, Danika Di Giacomo, Franco Locatelli, Riccardo Masetti, Salvatore Bertuccio, Martina Pigazzi, Ilaria Iacobucci, Charles G. Mullighan, Jeffery M. Klco. Genetic and epigenetic characterization of NUP98-rearranged leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d94d1c36d05baa7cc25171e29df8b2f95233b1" target='_blank'>
              Abstract A019 Genetic and epigenetic characterization of NUP98-rearranged leukemia
              </a>
            </td>
          <td>
            Masayuki Umeda, Nicole Michmerhuizen, R. Hiltenbrand, Juan M Barajas, Michael P. Walsh, Guangchun Song, Jing Ma, Tamara Westover, Cristina Mecucci, Danika Di Giacomo, Franco Locatelli, Riccardo Masetti, S. Bertuccio, M. Pigazzi, Ilaria Iacobucci, C. Mullighan, J. Klco
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2245c6c9be5dc1954f3bd2bc26c574b380581c40" target='_blank'>
              Comprehensive genomic analysis reveals clonal origin and subtype-specific evolution in a case of sporadic multiple meningiomas.
              </a>
            </td>
          <td>
            M. Sakaguchi, Masafumi Horie, Yukinobu Ito, Shingo Tanaka, Keishi Mizuguchi, Hiroko Ikeda, Etsuko Kiyokawa, Mitsutoshi Nakada, Daichi Maeda
          </td>
          <td>2024-07-27</td>
          <td>Brain tumor pathology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="TAD boundaries are genomic elements that separate biological processes in neighboring domains by blocking DNA loops that are formed through Cohesin-mediated loop extrusion. Most TAD boundaries consist of arrays of binding sites for the CTCF protein, whose interaction with the Cohesin complex blocks loop extrusion. TAD boundaries are not fully impermeable though and allow a limited amount of inter-TAD loop formation. Based on the reanalysis of Nano-C data, a multicontact Chromosome Conformation Capture assay, we propose a model whereby clustered CTCF binding sites promote the successive stalling of Cohesin and subsequent dissociation from the chromatin. A fraction of Cohesin nonetheless achieves boundary read-through. Due to a constant rate of Cohesin dissociation elsewhere in the genome, the maximum length of inter-TAD loops is restricted though. We speculate that the DNA-encoded organization of stalling sites regulates TAD boundary permeability and discuss implications for enhancer-promoter loop formation and other genomic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7707b0397caa81eae89d125f02c4b85e10532d65" target='_blank'>
              Permeable TAD boundaries and their impact on genome-associated functions.
              </a>
            </td>
          <td>
            Li-Hsin Chang, D. Noordermeer
          </td>
          <td>2024-08-02</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ubiquitin modification and alternative polyadenylation play crucial roles in the onset and progression of cancer. Hence, this study aims to comprehensively and deeply understand gene regulation and associated biological processes in lung adenocarcinoma (LUAD) by integrating both mechanisms. Alternative polyadenylation (APA)-related E3 ubiquitin ligases in LUAD were identified through multiple databases, and the association between selected genetic loci influencing gene expression (apaQTL-SNPs) and LUAD risk were evaluated through the GWAS database of the Female Lung Cancer Consortium in Asia (FLCCA). Subsequently, the interaction between RNF213 and ZBTB20, as well as their functional mechanisms in LUAD, were investigated using bioinformatics analysis, Western blot, co-immunoprecipitation, and colony formation experiments. A total of five apaQTL-SNPs (rs41301932, rs4494603, rs9890400, rs56066320, and rs41301932), located on RNF213, were significantly associated with LUAD risk (p < 0.05), and they inhibit tumor growth through ubiquitin-mediated degradation of ZBTB20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c9426c96fac5af729404fd06cbe66dceed7c43" target='_blank'>
              Crosstalk among Alternative Polyadenylation, Genetic Variants and Ubiquitin Modification Contribute to Lung Adenocarcinoma Risk
              </a>
            </td>
          <td>
            Yutong Wu, Yanqiong Yuan, Huiwen Xu, Wendi Zhang, Anhui Ning, Siqi Li, Qiong Chen, Xiaobo Tao, Gongbu Pan, Tian Tian, Lei Zhang, Minjie Chu, Jiahua Cui
          </td>
          <td>2024-07-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In mammals, tail length is controlled by several genetic determinants, amongst which Hox13 genes located at the posterior extremities of Hox clusters, whose main function are to terminate the extension of the body axis. In this view, the precise timing in the transcriptional activation of these genes may impact upon body length. Unlike other Hox clusters, HoxB lacks all posterior genes between Hoxb9 and Hoxb13, two genes separated by a ca. 70 kb large DNA segment containing an unusually high number of CTCF sites, suggesting it isolates Hoxb13 from the rest of the cluster, thereby delaying its negative impact on trunk extension. We deleted the spacer DNA to induce a potential heterochronic gain of function of Hoxb13 at physiological concentration and observed a shortening of the tail as well as other abnormal phenotypes, which were all rescued by inactivating Hoxb13 in-cis with the deletion. A comparable gain of function was observed in mutant ES cells grown as pseudo-embryos in vitro, which allowed us to examine in details the importance of both the number and the orientation of CTCF sites in the insulating activity of the DNA spacer. A short cassette containing all the CTCF sites was sufficient to insulate Hoxb13 from the rest of HoxB and additional modifications of this CTCF cassette showed that two CTCF sites in convergent orientations are already capable of importantly delaying Hoxb13 activation in these conditions. We discuss the relative importance of genomic distance versus number and orientation of CTCF sites in preventing Hoxb13 to be activated too early during trunk extension and hence to modulate tail length.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bc27ccdd6c53e8dcd3d37c1bc58024dfc056eb7" target='_blank'>
              CTCF-DEPENDENT INSULATION OF Hoxb13 AND THE HETEROCHRONIC CONTROL OF TAIL LENGTH
              </a>
            </td>
          <td>
            Lucille Lopez-Delisle, J. Zákány, Célia Bochaton, P. Osteil, Alexandre Mayran, Fabrice Darbellay, Bénédicte Mascrez, Hocine Rekaik, Denis Duboule
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Head and neck squamous cell carcinomas (HNSCCs), a heterogeneous group of cancers that arise from the mucosal epithelia cells in the head and neck areas, present great challenges in diagnosis, treatment and prognosis due to their complex aetiology and various clinical manifestations. Several factors, including smoking, alcohol consumption, oncogenic genes, growth factors, Epstein-Barr virus and human papillomavirus infections can contribute to HNSCC development. The unpredictable tumour microenvironment adds to the complexity of managing HNSCC. Despite significant advances in therapies, the prediction of outcome after treatment for patients with HNSCC remains poor, and the 5-year overall survival rate is low due to late diagnosis. Early detection greatly increases the chances of successful treatment. The present review aimed to bring together the latest findings related to the molecular mechanisms of HNSCC carcinogenesis and progression. Comprehensive genomic, transcriptomic, metabolomic, microbiome and proteomic analyses allow researchers to identify important biological markers such as genetic alterations, gene expression signatures and protein markers that drive HNSCC tumours. These biomarkers associated with the stages of initiation, progression and metastasis of cancer are useful in the management of patients with cancer in order to improve their life expectancy and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103d6f12f2b9f899f2264b3f78096ec88ebba8ce" target='_blank'>
              Epithelial‑derived head and neck squamous tumourigenesis (Review)
              </a>
            </td>
          <td>
            Charles Shirima, C. Bleotu, Demetrios A. Spandidos, Adel K El-Naggar, G. Pîrcălăbioru, Ioannis Michalopoulos
          </td>
          <td>2024-08-21</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028d055f673c0c4d806112c2a9d63a57da73eaef" target='_blank'>
              A brief review of noncoding RNA
              </a>
            </td>
          <td>
            Tina P. George, Suja Subramanian, M. H. Supriya
          </td>
          <td>2024-09-02</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf95f1dc0f1af0cb6e8bd77dab713b5fc609919" target='_blank'>
              CircRHBDD1 promotes immune escape via IGF2BP2/PD-L1 signaling and acts as a nanotherapeutic target in gastric cancer
              </a>
            </td>
          <td>
            Yanna Li, Zhixiong Wang, Peng Gao, Danping Cao, R. Dong, Menglin Zhu, Yao Fei, X. Zuo, Juan Cai
          </td>
          <td>2024-07-30</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="During mammalian embryogenesis, both the 5-cytosine DNA methylation (5meC) landscape and three dimensional (3D) chromatin architecture are profoundly remodeled during a process known as 'epigenetic reprogramming.' An understudied aspect of epigenetic reprogramming is how the 5meC flux, per se, affects the 3D genome. This is pertinent given the 5meC-sensitivity of DNA binding for a key regulator of chromosome folding: CTCF. We profiled the CTCF binding landscape using a mouse embryonic stem cell (ESC) differentiation protocol that models embryonic 5meC dynamics. Mouse ESCs lacking DNA methylation machinery are able to exit naive pluripotency, thus allowing for dissection of subtle effects of CTCF on gene expression. We performed CTCF HiChIP in both wild-type and mutant conditions to assess gained CTCF-CTCF contacts in the absence of 5meC. We performed H3K27ac HiChIP to determine the impact that ectopic CTCF binding has on cis-regulatory contacts. Using 5meC epigenome editing, we demonstrated that the methyl-mark is able to impair CTCF binding at select loci. Finally, a detailed dissection of the imprinted Zdbf2 locus showed how 5meC-antagonism of CTCF allows for proper gene regulation during differentiation. This work provides a comprehensive overview of how 5meC impacts the 3D genome in a relevant model for early embryonic events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4286154f07173854ef3d5bb731fafdeabe579e02" target='_blank'>
              The impact of the embryonic DNA methylation program on CTCF-mediated genome regulation.
              </a>
            </td>
          <td>
            Ana Monteagudo-Sánchez, Julien Richard Albert, Margherita Scarpa, D. Noordermeer, Maxim V. C. Greenberg
          </td>
          <td>2024-08-24</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Salih Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (AML) exhibits distinct characteristics between infants and children, manifested by variations in clinical features, cytogenetic abnormalities, and molecular aberrations. While most studies have focused on the associations between genomic abnormalities, age groups, and disease status, the transcriptional profiles and gene regulatory mechanisms underlying these differences remain underexplored. In this study, through differential gene expression analysis and Weighted Gene Co-expression Network Analysis, I have identified gene groups associated with AML and further categorized them by infant and childhood subgroups. The findings reveal three distinct gene groups with age- and tissue-specific expression patterns. Additionally, I propose a gene regulatory circuitry that elucidates the differences between infant and childhood AML. Several novel markers demonstrating significant expression changes in tumors were identified. Moreover, a comprehensive gene regulatory network for pediatric AML was constructed using differentially expressed transcription factors and protein-gene interaction data. This study highlights new gene regulation mechanisms in pediatric AML and offers potential avenues for developing novel biomarkers and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdcb7517a7ea15fd2b0b9215c835acfed1519a12" target='_blank'>
              Dissecting the transcriptional regulation of infant and childhood acute myeloid leukemia
              </a>
            </td>
          <td>
            Rasoul Godini
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1230bf0b0e23a37cb0ef069af27c70638d7b9578" target='_blank'>
              Cell-free DNA from germline TP53 mutation carriers reflect cancer-like fragmentation patterns
              </a>
            </td>
          <td>
            Derek Wong, Maha Tageldein, Ping Luo, Erik Ensminger, Jeffrey P. Bruce, L. Oldfield, Haifan Gong, N. Fischer, B. Laverty, Vallijah Subasri, Scott Davidson, Reem Khan, A. Villani, A. Shlien, R. Kim, David Malkin, Trevor J. Pugh
          </td>
          <td>2024-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is the most frequently diagnosed cancer and the primary cause of cancer-related mortality in women. Treatment of triple-negative breast cancer (TNBC) remains particularly challenging due to its resistance to chemotherapy and poor prognosis. Extensive research efforts in BC screening and therapy have improved clinical outcomes for BC patients. Therefore, identifying reliable biomarkers for TNBC is of great clinical importance. Here, we found that tyrosine aminotransferase (TAT) expression was significantly reduced in BC and strongly correlated with the poor prognosis of BC patients, which distinguished BC patients from normal individuals, indicating that TAT is a valuable biomarker for early BC diagnosis. Mechanistically, we uncovered that methylation of the TAT promoter was significantly increased by DNA methyltransferase 3 (DNMT3A/3B). In addition, reduced TAT contributes to DNA replication and cell cycle activation by regulating homologous recombination repair and mismatch repair to ensure genomic stability, which may be one of the reasons for TNBC resistance to chemotherapy. Furthermore, we demonstrated that Diazinon increases TAT expression as an inhibitor of DNMT3A/3B and inhibits the growth of BC by blocking downstream pathways. Taken together, we revealed that TAT is silenced by DNMT3A/3B in BC, especially in TNBC, which promotes the proliferation of tumor cells by supporting DNA replication, activating cell cycle, and enhancing DNA damage repair. These results provide fresh insights and a theoretical foundation for the clinical diagnosis and treatment of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a77f352d33a7a2b9ba7ce61fe82a2e23ebd12e" target='_blank'>
              Identification of TAT as a Biomarker Involved in Cell Cycle and DNA Repair in Breast Cancer
              </a>
            </td>
          <td>
            Fei Xie, Saiwei Hua, Yajuan Guo, Taoyuan Wang, Changliang Shan, Lianwen Zhang, Tao He
          </td>
          <td>2024-08-30</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25bc89ba9898a52d88ff5aa506e261a761286bfb" target='_blank'>
              piR-823 tale as emerging cancer-hallmark molecular marker in different cancer types: a step-toward ncRNA-precision.
              </a>
            </td>
          <td>
            Fatma H Shaker, E. Sanad, H. Elghazaly, Shih-Min Hsia, N. Hamdy
          </td>
          <td>2024-08-05</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Early embryos often have relatively unstructured chromatin that lacks active and inactive domains typical of differentiated cells. In many species, these regulatory domains are established during zygotic genome activation (ZGA). In Drosophila, ZGA occurs after 13 fast, reductive, syncytial nuclear divisions during which the nuclear to cytoplasmic (N/C) ratio grows exponentially. These divisions incorporate maternally-loaded, cytoplasmic pools of histones into chromatin. Previous work found that chromatin incorporation of replication-coupled histone H3 decreases while its variant H3.3 increases in the cell cycles leading up to ZGA. In other cell types, H3.3 is associated with sites of active transcription as well as heterochromatin, suggesting a link between H3.3 incorporation and ZGA. Here, we examine the factors that contribute to H3.3 incorporation at ZGA. We identify a more rapid decrease in the nuclear availability of H3 than H3.3 over the final pre-ZGA cycles. We also observe an N/C ratio-dependent increase in H3.3 incorporation in mutant embryos with non-uniform local N/C ratios. We find that chaperone binding, not gene expression, controls incorporation patterns using H3/H3.3 chimeric proteins at the endogenous H3.3A locus. We test the specificity of the H3.3 chaperone pathways for H3.3 incorporation using Hira (H3.3 chaperone) mutant embryos. Overall, we propose a model in which local N/C ratios and specific chaperone binding regulate differential incorporation of H3.3 during ZGA. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddcf3e70edf5f1afb44643d1130dfd3762c01db0" target='_blank'>
              Local nuclear to cytoplasmic ratio regulates chaperone-dependent H3 variant incorporation during zygotic genome activation
              </a>
            </td>
          <td>
            Anusha D. Bhatt, Madeleine G. Brown, Aurora B. Wackford, Yuki Shindo, Amanda A. Amodeo
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Neoadjuvant chemoradiotherapy (NCRT) followed by surgery is a standard treatment for locally advanced esophageal squamous cell carcinomas (ESCCs). However, the evolution of genome and immunogenome in ESCCs driven by NCRT remains incompletely elucidated. We performed whole-exome sequencing of 51 ESCC tumors collected before and after NCRT, 36 of which were subjected to transcriptome sequencing. Clonal analysis identified clonal extinction in 13 ESCC patients wherein all pre-NCRT clones disappeared after NCRT, and clonal persistence in 9 patients wherein clones endured following NCRT. The clone-persistent patients showed higher pre-NCRT genomic intratumoral heterogeneity and worse prognosis than the clone-extinct ones. In contrast to the clone-extinct patients, the clone-persistent patients demonstrated a high proportion of subclonal neoantigens within pre-treatment specimens. Transcriptome analysis revealed increased immune infiltrations and up-regulated immune-related pathways after NCRT, especially in the clone-extinct patients. The number of T cell receptor-neoantigen interactions was higher in the clone-extinct patients than in the clone-persistent ones. The decrease in T cell repertoire evenness positively correlated to the decreased number of clonal neoantigens after NCRT, especially in the clone-extinct patients. In conclusion, we identified two prognosis-related clonal dynamic modes driven by NCRT in ESCCs. This study extended our knowledge of the ESCC genome and immunogenome evolutions driven by NCRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c14b263bfbb1734f528f0fb9d5e5bd5af9edb6d" target='_blank'>
              Evolution of genome and immunogenome in esophageal squamous cell carcinomas driven by neoadjuvant chemoradiotherapy.
              </a>
            </td>
          <td>
            Zelin Weng, Zihang Mai, Jianye Yuan, Qianwen Liu, Fangqi Deng, Hong Yang, Y. Ling, Xiuying Xie, Xiaodan Lin, Ting Lin, Jiyang Chen, Xiaoli Wei, K. Luo, Jianhua Fu, Jing Wen
          </td>
          <td>2024-07-31</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Current diagnostic tools are unable to distinguish low‐grade indolent prostate cancer (PrCa) from that with a propensity to become metastatic and/or lethal. Recent evidence suggests that reprogramming of the transcriptome may drive the metastatic phenotype, and that this reprogramming is controlled, at least in part, by epigenetic changes to the DNA of cancer cells, including methylation. These changes, referred to as ‘epigenetic drivers,’ have previously been associated with cancer cell survival. Methods Here, using Illumina Methylation EPIC array data of paired primary PrCa and metastatic bone samples, we identified WNT5A as a putative epi‐driver of PrCa metastasis to the bone, which was further validated in vitro. Results Significantly higher WNT5A methylation was observed in primary PrCa samples and 22Rv1 cells compared to metastatic bone samples and PC‐3 cells. This higher methylation was associated with significantly lower WNT5A gene expression. Conclusion Given the limited effective therapies available for metastatic cancer sufferers, particularly those whose disease has metastasised to the bone, WNT5A presents as a potential putative target for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b055e884a7d4d2969ce7c73d7f93b2c051be4c83" target='_blank'>
              WNT5A is a putative epi‐driver of prostate cancer metastasis to the bone
              </a>
            </td>
          <td>
            Emma J Wilkinson, Kelsie Raspin, R. Malley, S. Donovan, Louise M Nott, Adele F Holloway, Joanne L Dickinson
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Double-strand break repair of eukaryotic DNA is mainly accomplished by non-homologous end joining and homologous recombination. Providing exogenous templates during homologous recombination repair can result in the editing of target genes, which is the central mechanism of the well-established CRISPR gene editing system. Currently, exogenous templates are mainly DNA molecules, which can provoke a cellular immune response within the cell. In order to verify the feasibility of RNA molecules as repair templates for homologous recombination in mammalian cell genome editing, we fused RNA template molecules to the 3´-end of sgRNA, so that the sgRNA and the homologous template RNA form a single RNA molecule. The results show this construct can be used as a repair template to achieve target gene editing in mammalian cells. In addition, the factors influencing homologous recombination mediated by RNA template molecules were investigated and it was found that increasing the length of homologous arms and inducing an R-loop near the DSB can effectively promote homologous recombination repair. Further, intracellular homologous chromosomes may compete with exogenous RNA templates. The findings in this paper provide a reference for the utilization of RNA template molecules to mediate target gene editing in eukaryotic cells, as well as a basis for the study of the mechanism by which RNA molecules mediate the repair of DSBs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8213616e8bc7e7049f87c65b52c718b1231c8c19" target='_blank'>
              Genome editing of mammalian cells through RNA transcript-mediated homologous recombination repair.
              </a>
            </td>
          <td>
            Yangmin Wang, Meilin Liu, Xinjian Lin, Haozheng Wang, Na Dong, Hengshen Liu, Hong-wei Shao, Wenfeng Zhang
          </td>
          <td>2024-07-24</td>
          <td>Human gene therapy</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdedb81bfc2b32fe60e68b41b7f434fc5731813" target='_blank'>
              Role of homeobox genes in cancer: immune system interactions, long non-coding RNAs, and tumor progression.
              </a>
            </td>
          <td>
            Saade Abdalkareem Jasim, Shireen Hamid Farhan, Irfan Ahmad, Ahmed Hjazi, Ashwani Kumar, Mohammed Abed Jawad, Atreyi Pramanik, Farag M. A. Altalbawy, Salim B. Alsaadi, M. Abosaoda
          </td>
          <td>2024-09-06</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e1918f5013376fd130c9f594868a7f384598c2" target='_blank'>
              HMOX1: A pivotal regulator of prognosis and immune dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Jinfa Huang, Ruiwan Tan
          </td>
          <td>2024-08-30</td>
          <td>BMC Women's Health</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization aﬀects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of diﬀerentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative eﬀects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. Smirnov, T. Shnaider, I. Pristyazhnuk, S. Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-02</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The SAGA complex is an evolutionarily conserved histone acetyltransferase complex and transcription coactivator essential for development and disease. Dysregulation of SAGA is implicated in various human diseases, including cancer. In this issue of Genes & Development, Chen et al. (doi/10.1101/gad.351789.124) uncover a critical role for SAGA in multiple myeloma wherein SAGA's ADA2B component is required for the expression of mTORC1 pathway genes and targets of the MYC, E2F, and MAF (musculoaponeurotic fibrosarcoma) transcription factors. SAGA cooperates with MYC and MAF to sustain oncogenic gene expression programs vital for multiple myeloma survival and thus may serve as a therapeutic target for future cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa0500e20ce37a63c1e23fe219d04a1b016c2f1c" target='_blank'>
              The MYC-MAF-SAGA axis drives oncogenic gene expression in multiple myeloma.
              </a>
            </td>
          <td>
            Hongkuan Wang, Hong Wen, Xiaobing Shi
          </td>
          <td>2024-08-21</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intratumoral hypoxia is associated with tumor progression, aggressiveness, and therapeutic resistance in several cancers. Hypoxia causes cancer cells to experience replication stress, thereby activating DNA damage and repair pathways. MutT homologue-1 (MTH1, also known as NUDT1), a member of the Nudix family, maintains the genomic integrity and viability of tumor cells in the hypoxic tumor microenvironment. Although hypoxia is associated with poor prognosis and can cause therapeutic resistance by regulating the microenvironment, it has not been considered a treatable target in cancer. This study aimed to investigate whether hypoxia-induced MTH1 is a useful target for immunotherapy and whether hypoxic conditions influence the antitumor activity of immune cells. Our results showed that MTH1 expression was elevated under hypoxic conditions in head and neck cancer cell lines. Furthermore, we identified a novel MTH1-targeting epitope peptide that can activate peptide-specific CD4+ helper T cells with cytotoxic activity. The proliferation and cytotoxic activity of T cells were maintained under hypoxic conditions, and PD-1 blockade further augmented the cytotoxicity. These results indicate that MTH1-targeted immunotherapy combined with checkpoint blockade can be an effective strategy for the treatment of hypoxic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fd79a48171b74f56ec3547f207f7b41e323150" target='_blank'>
              Hypoxia-Targeted Immunotherapy with PD-1 Blockade in Head and Neck Cancer
              </a>
            </td>
          <td>
            Risa Wakisaka, Hidekiyo Yamaki, Michihisa Kono, Takahiro Inoue, Ryosuke Sato, H. Komatsuda, K. Ohara, A. Kosaka, T. Ohkuri, T. Nagato, K. Kishibe, Koh Nakayama, Hiroya Kobayashi, T. Kumai, Miki Takahara
          </td>
          <td>2024-08-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>