<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-16 06:31:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>24</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>52</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>101</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>279</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>240</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, Jens Luebeck, Kaiyuan Zhu, Joshua Lange, Roman Sasik, Chad Phillips, Sayed Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Rationale: Extrachromosomal circular DNA is a hallmark of cancer, but its role in shaping the genome heterogeneity of urothelial bladder carcinoma (UBC) remains poorly understood. Here, we comprehensively analyzed the features of extrachromosomal circular DNA in 80 UBC patients. Methods: We performed whole-genome/exome sequencing (WGS/WES), Circle-Seq, single-molecule real-time (SMRT) long-read sequencing of circular DNA, and RNA sequencing (RNA-Seq) on 80 pairs of tumor and AT samples. We used our newly developed circular DNA analysis software, Circle-Map++ to detect small extrachromosomal circular DNA from Circle-Seq data. Results: We observed a high load and significant heterogeneity of extrachromosomal circular DNAs in UBC, including numerous single-locus and complex chimeric circular DNAs originating from different chromosomes. This includes highly chimeric circular DNAs carrying seven oncogenes and circles from nine chromosomes. We also found that large tumor-specific extrachromosomal circular DNAs could influence genome-wide gene expression, and are detectable in time-matched urinary sediments. Additionally, we found that the extrachromosomal circular DNA correlates with hypermutation, copy number variation, oncogene amplification, and clinical outcome. Conclusions: Overall, our study provides a comprehensive extrachromosomal circular DNA map of UBC, along with valuable data resources and bioinformatics tools for future cancer and extrachromosomal circular DNA research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0454d85a5ac1363a4fa012edcd2f47cff5be9b32" target='_blank'>
              Extrachromosomal circular DNA orchestrates genome heterogeneity in urothelial bladder carcinoma
              </a>
            </td>
          <td>
            Wei Lv, Xiaoguang Pan, Peng Han, Shuang Wu, Yuchen Zeng, Qingqing Wang, Lidong Guo, Mengyang Xu, Yanwei Qi, Li Deng, Zhe Xu, Conghui Li, Tianxi Yu, Xin Cui, Huajing Teng, Chongjun Xiang, Haotian Tan, Yue Li, Ning Liang, Huiying Tao, Qingqing Gao, Guohua Yu, Jia Mi, Fuyi Xu, Benjiao Gong, Lei Shi, Tao Wang, Huanming Yang, Wei Dong, Lars Bolund, Lin Lin, Wenting Wang, Hanbo Li, Jinrong Huang, Chunhua Lin, Yonglun Luo
          </td>
          <td>2024-08-19</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm ‘HYENA’ to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Sequencing-based mapping of ensemble pairwise interactions among regulatory elements support the existence of topological assemblies known as promoter-enhancer hubs or cliques in cancer. Yet, prevalence, regulators, and functions of promoter-enhancer hubs in individual cancer cells remain unclear. Here, we systematically integrated functional genomics, transcription factor screening, and optical mapping of promoter-enhancer interactions to identify key promoter-enhancer hubs, examine heterogeneity of their assembly, determine their regulators, and elucidate their role in gene expression control in individual triple negative breast cancer (TNBC) cells. Optical mapping of individual SOX9 and MYC alleles revealed the existence of frequent multiway interactions among promoters and enhancers within spatial hubs. Our single-allele studies further demonstrated that lineage-determining SOX9 and signaling-dependent NOTCH1 transcription factors compact MYC and SOX9 hubs. Together, our findings suggest that promoter-enhancer hubs are dynamic and heterogeneous topological assemblies, which are controlled by oncogenic transcription factors and facilitate subtype-restricted gene expression in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8df926269367eb0273d95bc28b249a4b8bb66d" target='_blank'>
              Oncogenic transcription factors instruct promoter-enhancer hubs in individual triple negative breast cancer cells
              </a>
            </td>
          <td>
            Jingru Zhao, Yeqiao Zhou, Ilias Tzelepis, Noah G Burget, Junwei Shi, R. Faryabi
          </td>
          <td>2024-08-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Regulating gene expression involves significant and frequent changes in the chromatin environment at the locus level, especially at regulatory sequences. However, their modulation in response to pharmacological treatments or pathological conditions remain mostly undetermined. Here, we report versatile locus-specific proteomics tools to address this knowledge gap, which combine the targeting ability of the CRISPR/Cas9 system and the protein-labelling capability of the highly reactive biotin ligases TurboID (in CasTurbo) and UltraID (in CasUltra). CasTurbo and CasUltra enabled rapid chromatin protein labelling under mild conditions at repetitive sequences like centromeres and telomeres, as well as non-amplified genes. We applied CasUltra to A375 melanoma cell lines to decipher the protein environment of the MYC promoter and characterize the molecular effects of the bromodomain inhibitor JQ1, which targets bromodomain and extra-terminal (BET) proteins that regulate MYC expression. We quantified the consequences of BET protein displacement from the MYC promoter and found that it was associated with a considerable reorganisation of the chromatin composition. In addition, BET protein retention at the MYC promoter was consistent with a model of increased JQ1 resistance. Thus, through the combination of proximity biotinylation and CRISPR-Cas9-dependent genomic targeting, CasTurbo and CasUltra have successfully demonstrated their utility in profiling the proteome associated with a genomic locus in living cells. In Brief Kougnassoukou Tchara et al. report the development and application of CasTurbo and CasUltra, two locus-specific proteomics tools that fuse catalytically dead Cas9 to the engineered biotin ligases TurboID and UltraID. These tools enabled the quantitative mapping of locus-specific chromatin remodelling due to pharmacological inhibition. Highlights CasTurbo and CasUltra were developed for locus-specific label-free proteomics CasTurbo mapped the proteins localized to the centromeres and telomeres Proteins bound to the MYC promoter were quantified in melanoma cells with CasUltra CasUltra is compatible with investigating pharmacological treatment effects Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce2d069fe2369eb465a0930b8b4ef5639c3f142c" target='_blank'>
              Coupling proximity biotinylation with genomic targeting to characterize locus-specific changes in chromatin environments
              </a>
            </td>
          <td>
            Pata-Eting Kougnassoukou Tchara, J. Loehr, Jean-Philippe Lambert
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Allele-specific expression (ASE) is the differential abundance in levels of mRNAs that originated from the paternal and maternal copies of a gene. Such allelic imbalances can contribute to phenotypic variation and influence disease traits, including cancer. There is common ASE in tumors that results from somatic copy-number alterations (CNAs) at the DNA level, but there also exist other causes of ASE: cis-acting genetic or epigenetic variation that can lead to differential expression between the two alleles. However, the latter, non-CNA mechanisms of ASE remain understudied in cancer, as well as their role in tumor evolution and impact on clinical outcomes. By integrating a wide variety of genomic and transcriptomic pan-cancer data from the TCGA project, we show that ASE favoring the preferential expression of the mutant allele in some driver genes is subject to positive selection, and that these events are associated with worse overall survival across all cancer types. We found that the impact of ASE triggered by non-CNA causes is substantial, and we propose that some instances of cis-ASE are explained by the epigenetic changes affecting alleles differently. Furthermore, as a second mechanism, we find that splicing-altering mutations are selected in various cancer genes and result in ASE. We anticipate that the study and understanding of the role of mutant allele imbalances at the mRNA level can help understand epigenetic changes during cancer evolution, as well as identify new prognostic markers and therapeutic approaches that target altered allelic expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a918fe3f2472f99c1403cdc4bbe583961852532" target='_blank'>
              Allele-specific expression is selected in tumorigenesis, results from epigenetic changes and has prognostic relevance
              </a>
            </td>
          <td>
            Guillermo Palou-Márquez, Pere Pericot-Masdevall, F. Supek
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e11890b80227e402b42551347179de59480a8fc" target='_blank'>
              HMGA1 orchestrates chromatin compartmentalization and sequesters genes into 3D networks coordinating senescence heterogeneity
              </a>
            </td>
          <td>
            Ioana Olan, Masami Ando-Kuri, Aled J. Parry, Testuya Handa, S. Schoenfelder, Peter Fraser, Yasuyuki Ohkawa, Hiroshi Kimura, M. Narita, Masashi Narita
          </td>
          <td>2024-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature Plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="H3K36 methylation is emerging as a key epigenetic modification with strong implications in genetic disease and cancer. However, the mechanisms through which H3K36me impacts the epigenome and asserts its functional consequences are far from understood. Here, we use mouse mesenchymal stem cell lines with successive knockouts of the H3K36 methyltransferases: NSD1, NSD2, SETD2, NSD3, and ASH1L, which result in progressive depletion of H3K36me and its complete absence in quintuple knockout cells, to finely dissect the role of H3K36me2 in shaping the epigenome and transcriptome. We show that H3K36me2, which targets active enhancers, is important for maintaining enhancer activity, and its depletion results in downregulation of enhancer-dependent genes. We demonstrate the roles of H3K36me2/3 in preventing the invasion of gene bodies by the repressive H3K27me modifications. Finally, we observe a previously undescribed relationship between H3K36me and H3K9me3: Following the depletion of H3K36me2, H3K9me3 is redistributed away from large heterochromatic domains and towards euchromatin. This results in a drastic decompartmentalization of the genome, weakening the boundaries between active and inactive compartments, and a catastrophic loss of long-range inter-compartment interactions. By studying cells totally devoid of H3K36 methyltransferase activity, we uncover a broad range of crucial functions of H3K36me in maintaining epigenome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c996ce7c85d697b9c8fb195bf8183e8099242ceb" target='_blank'>
              H3K36 Methylation - a Guardian of Epigenome Integrity
              </a>
            </td>
          <td>
            Reinnier Padilla, Gerry A. Shipman, Cynthia Horth, E. Bareke, Jacek Majewski
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin within the nucleus is crucial for gene regulation. However, the 3D architectural features that coordinate the activation of an entire chromosome remain largely unknown. We introduce an omics method, RNA-associated chromatin DNA-DNA interactions, that integrates RNA polymerase II (RNAPII)–mediated regulome with stochastic optical reconstruction microscopy to investigate the landscape of noncoding RNA roX2-associated chromatin topology for gene equalization to achieve dosage compensation. Our findings reveal that roX2 anchors to the target gene transcription end sites (TESs) and spreads in a distinctive boot-shaped configuration, promoting a more open chromatin state for hyperactivation. Furthermore, roX2 arches TES to transcription start sites to enhance transcriptional loops, potentially facilitating RNAPII convoying and connecting proximal promoter-promoter transcriptional hubs for synergistic gene regulation. These TESs cluster as roX2 compartments, surrounded by inactive domains for coactivation of multiple genes within the roX2 territory. In addition, roX2 structures gradually form and scaffold for stepwise coactivation in dosage compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc0d29c3125cfd0f353817880f4b6081993ce9" target='_blank'>
              3D chromatin structures associated with ncRNA roX2 for hyperactivation and coactivation across the entire X chromosome
              </a>
            </td>
          <td>
            S. Tian, Yang Yang, Duo Ning, Ke Fang, Kai Jing, Guangyu Huang, Yewen Xu, Pengfei Yin, Haibo Huang, Gengzhan Chen, Yuqing Deng, Shaohong Zhang, Zhimin Zhang, Zhen-Xia Chen, Tong Gao, Wei Chen, Guoliang Li, Ruilin Tian, Yijun Ruan, Yiming Li, Meizhen Zheng
          </td>
          <td>2024-07-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There are two key signatures of pediatric cancers: (a) higher prevalence of germline alterations and (b) heterogeneity in alteration types. Recent population-based assessments have demonstrated that children with birth defects (BDs) are more likely to develop cancer even without chromosomal anomalies; therefore, explorations of genetic alterations in children with BDs and cancers could provide new insights into the underlying mechanisms for pediatric tumor development. We performed whole-genome sequencing (WGS) on blood-derived DNA for 1556 individuals without chromosomal anomalies, including 454 BD probands with at least one type of malignant tumor, 757 cancer-free children with BDs, and 345 healthy individuals, focusing on copy number variation (CNV) analysis. Roughly half of the children with BD-cancer have CNVs that are not identified in BD-only/healthy individuals, and CNVs are not evenly distributed among these patients. Strong heterogeneity was observed, with a limited number of cancer predisposition genes containing CNVs in more than three patients. Moreover, functional enrichments of genes with CNVs showed that dozens of patients have variations related to the same biological pathways, such as deletions of genes with neurological functions and duplications of immune response genes. Phenotype clustering uncovered recurrences of patients with sarcoma: A notable enrichment was observed involving non-coding RNA regulators, showing strong signals related to growth and cancer regulations in functional analysis. In conclusion, we conducted one of the first genomic studies exploring the impact of CNVs on cancer development in children with BDs, unveiling new insights into the underlying biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0421f07c60f2733c310940e72c92010ab69f33fb" target='_blank'>
              Copy number variations contribute to malignant tumor development in children with serious birth defects.
              </a>
            </td>
          <td>
            Yichuan Liu, J. Glessner, Hui-Qi Qu, X. Chang, H. Qiu, Tiancheng Wang, F. Mentch, H. Hakonarson
          </td>
          <td>2024-08-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transcription factor 3 (TCF3), a pivotal member of the TCF/LEF family, plays a critical role in tumorigenesis. Nonetheless, its impact on the tumor microenvironment (TME) and cancer phenotypes remains elusive. We perform an exhaustive analysis of TCF3 expression, DNA variation profiles, prognostic implications, and associations with the TME and immunological aspects. This study is based on a large-scale pan-cancer cohort, encompassing over 17,000 cancer patients from multiple independent datasets, validated by in vitro assays. Our results show that TCF3/4/7 exhibits differential expression patterns between normal and tumor tissues across pan-cancer analyses. Mutational analysis of TCF3 across diverse cancer types reveals the highest alteration rates in biliary tract cancer. Additionally, mutations and single nucleotide variants in TCF3/4/7 are found to exert varied effects on patient prognosis. Importantly, TCF3 emerges as a robust predictor of survival across all cancer cohorts and among patients receiving immune checkpoint inhibitors. Elevated TCF3 expression is correlated with more aggressive cancer subtypes, as validated by immunohistochemistry and diverse cohort data. Furthermore, TCF3 expression is positively correlated with intratumoral heterogeneity and angiogenesis. In vitro investigations demonstrate that TCF3 is involved in epithelial-mesenchymal transition, migration, invasion, and angiogenesis. These effects are likely mediated through the interaction of TCF3 with the NF-κB/MMP2 pathway, which is modulated by IL-17A in human uveal melanoma MUM2B cells. This study elucidates, for the first time, the significant associations of TCF3 with DNA variation profiles, prognostic outcomes, and the TME in multiple cancer contexts. TCF3 holds promise as a molecular marker for diagnosis and as a potential target for novel therapeutic strategies, particularly in uveal melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bec429a74c0a32bc4300e0227ccff7d9af711e" target='_blank'>
              TCF3 as a multidimensional biomarker: oncogenicity, genomic alterations, and immune landscape in pan-cancer analysis.
              </a>
            </td>
          <td>
            Huiling Nie, Yang Yu, Siqi Zhou, Yue Xu, Xi Chen, Xun Qin, Zhangyu Liu, Jiayu Huang, Hailiang Zhang, Jin Yao, Qin Jiang, Bingbing Wei, Xiaojian Qin
          </td>
          <td>2024-08-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The primary constriction site of the M-phase chromosome is an established marker for the kinetochore position, often used to determine the karyotype of each species. Underlying this observation is the concept that the kinetochore is spatially linked with the pericentromere where sister-chromatids are most tightly cohered. Here, we found an unconventional pericentromere specification with sister chromatids mainly cohered at a chromosome end, spatially separated from the kinetochore in Peromyscus mouse oocytes. This distal locus enriched cohesin protectors, such as the Chromosomal Passenger Complex (CPC) and PP2A, at a higher level compared to its centromere/kinetochore region, acting as the primary site for sister-chromatid cohesion. Chromosomes with the distal cohesion site exhibited enhanced cohesin protection at anaphase I compared to those without it, implying that these distal cohesion sites may have evolved to ensure sister-chromatid cohesion during meiosis. In contrast, mitotic cells enriched CPC only near the kinetochore and the distal locus was not cohered between sister chromatids, suggesting a meiosis-specific mechanism to protect cohesin at this distal locus. We found that this distal locus corresponds to an additional centromeric satellite block, located far apart from the centromeric satellite block that builds the kinetochore. Several Peromyscus species carry chromosomes with two such centromeric satellite blocks. Analyses on three Peromyscus species revealed that the internal satellite consistently assembles the kinetochore in both mitosis and meiosis, whereas the distal satellite selectively enriches cohesin protectors in meiosis to promote sister-chromatid cohesion at that site. Thus, our study demonstrates that pericentromere specification is remarkably flexible and can control chromosome segregation in a cell-type and context dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15bd1988d69721d61e9bea006887737188fa047" target='_blank'>
              Meiosis-specific decoupling of the pericentromere from the kinetochore
              </a>
            </td>
          <td>
            Bo Pan, Melania Bruno, Todd S Macfarlan, Takashi Akera
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4149f430cd50897e383d0f368cb3d1c8239aed" target='_blank'>
              Distinct dynamics of parental 5-hydroxymethylcytosine during human preimplantation development regulate early lineage gene expression
              </a>
            </td>
          <td>
            Dan Liang, Rui Yan, Xin Long, Dongmei Ji, Bing Song, Mengyao Wang, Fan Zhang, Xin Cheng, Fengyuan Sun, Ran Zhu, Xinling Hou, Tianjuan Wang, Weiwei Zou, Ying Zhang, Zhixin Pu, Jing Zhang, Zhiguo Zhang, Ya-Jing Liu, Yuqiong Hu, Xiaojin He, Yunxia Cao, Fan Guo
          </td>
          <td>2024-07-30</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome integrity is safeguarded by the DNA damage response (DDR). Controlled transcriptional dampening of genes surrounding DNA double-strand breaks (DSBs) has been shown to facilitate DNA repair. This phenomenon, defined as DSB-induced silencing in cis (DISC), involves the DDR apical kinase ATM and the Polycomb Repressive Complex 1 (PRC1). Conversely, DSBs have also been reported to induce de novo transcription of damaged-induced long non-coding RNAs (dilncRNAs) in a MRE11-RAD50-NBS1 (MNR) complex-dependent manner. MRN also controls the recruitment to DSB of the ribonuclease DROSHA, which together with DICER, stimulates DDR signaling and DNA repair. Here, we reconcile these apparently contrasting observations by showing that dilncRNA, together with DROSHA and DICER, but not GW182-like proteins required for miRNA-mediated gene silencing, controls DISC. Indeed, similarly to ATM, MRN inhibition abolishes DISC while pharmacological enhancement of DICER ribonuclease activity by Enoxacin improves DISC. Importantly, Enoxacin administration restores DISC upon ATM inhibition, demonstrating that DICER promotes DISC independently from ATM. Differently, Enoxacin does not restore DISC upon MRN inhibition, suggesting that DICER acts downstream to dilncRNA biogenesis and DROSHA recruitment. Mechanistically, we show that DROSHA and DICER control the recruitment of the PRC1 component BMI1 at DSBs and the consequent H2A-K119 ubiquitination. Upon DSBs formation, BMI1 and DROSHA interact in an RNA-dependent manner. Indeed, BMI1 associates to dilncRNA and do so in a DROSHA- and DICER-dependent manner. Importantly, inhibition of dilncRNA function by antisense oligonucleotides or Cas13-mediated targeting is sufficient to reduce BMI1 recruitment and DISC at individual loci. We propose that dilncRNAs together with DROSHA and DICER control DISC at genomic DSB by supporting PRC1 recruitment and chromatin ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbe46331f4c7eaaba94ede95e2ee9569f12ce8df" target='_blank'>
              DROSHA, DICER and Damage-Induced long ncRNA control BMI1-dependent transcriptional repression at DNA double-strand break
              </a>
            </td>
          <td>
            Francesca Esposito, Ilaria Capozzo, Stefania Modafferi, Ubaldo Gioia, Letizia Manfredi, F. Iannelli, Alessio Colantoni, Adelaide Riccardi, Alessia di Lillo, Sara Tavella, Matteo Cabrini, F. d’Adda di Fagagna, S. Francia
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="After egg fertilization, an initially silent embryonic genome is transcriptionally activated during the maternal-to-zygotic transition. In zebrafish, maternal vertebrate pluripotency factors Nanog, Pou5f3 (OCT4 homolog), and Sox19b (SOX2 homolog) (NPS) play essential roles in orchestrating embryonic genome activation, acting as “pioneers” that open condensed chromatin and mediate acquisition of activating histone modifications. However, some embryonic gene transcription still occurs in the absence of these factors, suggesting the existence of other mechanisms regulating genome activation. To identify chromatin signatures of these unknown pathways, we profiled the histone modification landscape of zebrafish embryos using CUT&RUN. Our regulatory map revealed two subclasses of enhancers distinguished by presence or absence of H3K4me2. Enhancers lacking H3K4me2 tend to require NPS factors for de novo activation, while enhancers bearing H3K4me2 are epigenetically bookmarked by DNA hypomethylation to recapitulate gamete activity in the embryo, independent of NPS pioneering. Thus, parallel enhancer activation pathways combine to induce transcriptional reprogramming to pluripotency in the early embryo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/915e897ed23cf5b7d63f2ee24fa9d133d64c79b4" target='_blank'>
              H3K4me2 distinguishes a distinct class of enhancers during the maternal-to-zygotic transition
              </a>
            </td>
          <td>
            Matthew D. Hurton, Jennifer M. Miller, Miler T. Lee
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Animal chromosomes can persist with recognizable homology over hundreds of millions of years, in spite of homology-obfuscating processes such as chromosomal fusion and translocation. The frequency and pace of these major genome structural changes are unknown, and it remains unclear whether or how they impact long-term genome evolution. Here, we compare whole chromosomal sequences of 3,631 genomes from 2,291 species spanning all major animal clades and show that animal karyotypes evolve primarily via karyotype contraction, associated with increased rates of chromosomal fusion-with-mixing and dispersion that largely obey chromosomal algebra1, or karyotype expansion, which breaks up ancestral linkage groups and forms new chromosomal elements via non-algebraic changes. We show that chromosomal changes can be associated with major extinction events. Using a multi-scale encoding of pan-animal genome homology and a manifold representation of genomic changes, we find that genome evolution is not only driven by changes at the chromosomal level, but that subchromosomal mixing and irreversibility define clade-specific evolution. Using this ‘evolutionary genome topology’ approach, we calculate extrema of irreversible genomic configurations and identify species that occupy intermediate manifold positions, providing evidence for distinct macro-evolutionary trajectories. We propose that investigation of mixed state accumulation around important gene loci (such as Hox) will be crucial in capturing and further study of clade-specific regulatory innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bcfa759f5e30b92ec5cb430b3076d461c210e1e" target='_blank'>
              Topological mixing and irreversibility in animal chromosome evolution
              </a>
            </td>
          <td>
            D. Schultz, Arno Blümel, Dalila Destanović, Fatih Sarigol, Oleg Simakov
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Although the role of G-quadruplex (G4) DNA structures has been suggested in chromosomal looping this was not tested directly. Here, to test causal function, an array of G4s, or control sequence that does not form G4s, were inserted within chromatin in cells. In vivo G4 formation of the inserted G4 sequence array, and not the control sequence, was confirmed using G4-selective antibody. Compared to the control insert, we observed a remarkable increase in the number of 3D chromatin looping interactions from the inserted G4 array. This was evident within the immediate topologically associated domain (TAD) and throughout the genome. Locally, recruitment of enhancer histone marks and the transcriptional coactivator p300/Acetylated-p300 increased in the G4-array, but not in the control insertion. Resulting promoter-enhancer interactions and gene activation were clear up to 5 Mb away from the insertion site. Together, these show the causal role of G4s in enhancer function and long-range chromatin interactions. Mechanisms of 3D topology are primarily based on DNA-bound architectural proteins that induce/stabilize long-range interactions. Involvement of the underlying intrinsic DNA sequence/structure in 3D looping shown here therefore throws new light on how long-range chromosomal interactions might be induced or maintained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4a14fedf36592373d312c9d23b12df8fd19b77" target='_blank'>
              Artificially inserted strong promoter containing multiple G-quadruplexes induces long-range chromatin modification
              </a>
            </td>
          <td>
            S. Roy, Sulochana Bagri, Soujanya Vinayagamurthy, Avik Sengupta, Claudia Regina Then, Rahul Kumar, Sriram Sridharan, Shantanu Chowdhury
          </td>
          <td>2024-08-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Eukaryotic genomes are organized into chromatin domains through long-range chromatin interactions which are mediated by the binding of architectural proteins, such as CTCF and cohesin, and histone modifications. Based on the published Hi-C and ChIP-seq datasets in human monocyte-derived macrophages, we identified 206 and 127 differential chromatin interactions (DCIs) that were not located within transcription readthrough regions in influenza A virus- and interferon β-treated cells, respectively, and found that the binding positions of CTCF and RAD21 within more than half of the DCI sites did not change. However, five histone modifications, H3K4me3, H3K27ac, H3K36me3, H3K9me3, and H3K27me3, showed significantly more dramatic changes than CTCF and RAD21 within the DCI sites. For H3K4me3, H3K27ac, H3K36me3, and H3K27me3, significantly more dramatic changes were observed outside than within the DCI sites. We further applied a motif scanning approach to discover proteins that might correlate with changes in histone modifications and chromatin interactions and found that PRDM9, ZNF384, and STAT2 frequently bound to DNA sequences corresponding to 1 kb genomic intervals with gains or losses of a histone modification within the DCI sites. This study explores the dynamic regulation of chromatin interactions and extends the current knowledge of the relationship between histone modifications and chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962acc7e0c5158c0467415488e49df41c02448fb" target='_blank'>
              Dynamic Changes in Histone Modifications Are Associated with Differential Chromatin Interactions
              </a>
            </td>
          <td>
            Yumin Nie, Mengjie Wang
          </td>
          <td>2024-07-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multimeric SWI/SNF chromatin remodelers assemble into discrete conformations with unique complex functionalities difficult to dissect. Distinct cancers harbor mutations in specific subunits, altering the chromatin landscape, such as the PBAF-specific component ARID2 in melanoma. Here, we performed comprehensive epigenomic profiling of SWI/SNF complexes and their associated chromatin states in melanoma and melanocytes and uncovered a subset of PBAF-exclusive regions that coexist with PRC2 and repressive chromatin. Time-resolved approaches revealed that PBAF regions are generally less sensitive to ATPase-mediated remodeling than BAF sites. Moreover, PBAF/PRC2-bound loci are enriched for REST, a transcription factor that represses neuronal genes. In turn, absence of ARID2 and consequent PBAF complex disruption hinders the ability of REST to bind and inactivate its targets, leading to upregulation of synaptic transcripts. Remarkably, this gene signature is conserved in melanoma patients with ARID2 mutations. In sum, we demonstrate a unique role for PBAF in generating accessibility for a silencing transcription factor at repressed chromatin, with important implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523ec2477b747fcc0089e58355e10a3265c7a242" target='_blank'>
              The SWI/SNF PBAF complex facilitates REST occupancy at repressive chromatin
              </a>
            </td>
          <td>
            Elena Grossi, Christie B. Nguyen, Saul Carcamo, Shannon Moran, Valentina Kirigin Callaú, Dan Filipescu, Dan Hasson, Emily Bernstein
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="A key step of Canonical Nonhomologous End Joining (C-NHEJ) is synapsis of DNA double strand break (DSB) ends for ligation. The DNA-PKcs dimer mediates synapsis in a long-range complex with DSB ends remaining apart, whereas the XLF homodimer can mediate synapsis in both long-range and short-range complexes. Recent structural studies found the PAXX homodimer may also facilitate synapsis in long-range complexes with DNA-PKcs via its interactions with Ku70. Thus, we examined the influence of PAXX in C-NHEJ of chromosomal DSBs, which we compared to another Ku-binding factor, MRI. Using EJ of blunt DSBs with Cas9 reporters as a readout for C-NHEJ, we found that PAXX and/or MRI are dispensable. However, when combined with disruption of DNA-PKcs, particularly with DNA-PKcs kinase inhibition, PAXX becomes important for blunt DSB EJ. In contrast, while DNA-PKcs is also important to suppress short deletion mutations with microhomology, this effect is not magnified with PAXX loss. MRI loss had no effect combined with DNA-PKcs disruption, but becomes important for blunt DSB EJ when combined with disruption of XLF, as is PAXX. Finally, XLF loss causes an increase in larger deletions compared to DNA-PKcs inhibition, which is magnified with combined loss of MRI. Altogether, we suggest that PAXX promotes DSB end synapsis during C-NHEJ in a manner that is partially redundant with DNA-PKcs and XLF, whereas MRI appears to be mainly important in the context of XLF disruption. SIGNIFICANCE STATEMENT Canonical Nonhomologous End Joining (C-NHEJ) is a DNA double strand break repair pathway that can repair diverse types of DNA ends, but is prone to causing genetic mutations. In particular, C-NHEJ can cause insertion or deletion mutations that can cause human disease. Therefore, it is critical to understand the mechanisms of C-NHEJ, including and how various factors within the pathway affect repair outcomes. In this study, we elucidate the role of factors PAXX and MRI during chromosomal EJ and their interplay with the C-NHEJ factors DNA-PKcs and XLF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6710460a50993f582d4fe63d4fd0ff8229a07e" target='_blank'>
              Distinct functions of PAXX and MRI during chromosomal end joining
              </a>
            </td>
          <td>
            Metztli Cisneros-Aguirre, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Ruby Chen, Jeremy M. Stark
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pediatric brain cancer is the leading cause of disease-related mortality in children, and many aggressive tumors still lack effective treatment strategies. Despite extensive studies characterizing these tumor genomes, alternative transcriptional splicing patterns remain underexplored. Here, we systematically characterized aberrant alternative splicing across pediatric brain tumors, identifying pediatric high-grade gliomas (HGGs) among the most heterogeneous. Through integration with UniProt Knowledgebase annotations, we identified 12,145 splice events in 5,424 genes, leading to functional changes in protein activation, folding, and localization. We discovered that the master splicing factor and cell-cycle modulator, CDC- like kinase 1 (CLK1), is aberrantly spliced in HGGs to include exon 4, resulting in a gain of two phosphorylation sites and subsequent activation of CLK1. Inhibition of CLK1 with Cirtuvivint in the pediatric HGG KNS-42 cell line significantly decreased both cell viability and proliferation in a dose-dependent manner. Morpholino-mediated depletion of CLK1 exon 4 splicing reduced RNA expression, protein abundance, and cell viability. Notably, KNS-42 cells treated with the CLK1 exon 4 morpholino demonstrated differential expression 78 genes and differential splicing with loss or gain of a functional site in 193 genes annotated as oncogene or tumor suppressor genes (TSGs). These genes were enriched for cancer-associated pathways, with 20 identified as significant gene dependencies in pediatric HGGs. Our findings highlight a dependency of pediatric HGGs on CLK1 and its roles contributing to tumor splicing heterogeneity through transcriptional dysregulation of splicing factors and transcriptional modulation of oncogenes. Overall, aberrant splicing in HGGs and other pediatric brain tumors represents a potentially targetable oncogenic pathway contributing to tumor growth and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624af22ec9a4e826208669859a67ff2ad429da0e" target='_blank'>
              Characterization of aberrant splicing in pediatric central nervous system tumors reveals CLK1 as a candidate oncogenic dependency
              </a>
            </td>
          <td>
            Ammar S. Naqvi, Ryan J. Corbett, Priyanka Seghal, Karina L. Conkrite, Komal S. Rathi, Brian M. Ennis, Katharina E. Hayer, Bo Zhang, Miguel A. Brown, Daniel P. Miller, Adam A. Kraya, Joseph M. Dybas, Zhuangzhuang Geng, Christopher Blackden, Shehbeel Arif, Antonia Chroni, Aditya Lahiri, Madison L. Hollawell, Phillip B. Storm, J. Foster, Mateusz Koptyra, Peter Madsen, S. Diskin, A. Thomas-Tikhonenko, Adam C. Resnick, J. L. Rokita
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc75014daa55260f9205d28f8165ad0ad389192e" target='_blank'>
              Genetic hallmarks and clinical implications of chromothripsis in childhood T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            A. Pastorczak, Z. Urbańska, Borys Styka, Karolina Miarka-Walczyk, L. Sędek, K. Wypyszczak, A. Wakulinska, Zuzanna Nowicka, Tomasz Szczepański, Marcin Stańczak, Wojciech Fendler, Jerzy Kowalczyk, W. Młynarski, Monika Lejman
          </td>
          <td>2024-08-27</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary High levels of the transcription factor Grainyhead-like protein 2 (GRHL2) contribute to worse outcomes for patients with breast cancer tumors that express estrogen receptor (ER). Using multiple methods to increase GRHL2 expression in breast cancer cells, we found that high GRHL2 promotes both epithelial and mesenchymal phenotypes. This is indicative of an intermediate state between epithelial and mesenchymal cells, which is associated with metastasis. We also observed that elevated GRHL2 stimulates properties characteristic of cellular plasticity, including a decrease in cellular proliferation with a reciprocal increase in dormancy and stem cell markers. Elevated levels of GRHL2 broadly changed its control of gene expression at the level of DNA and shifted nearby transcription factor binding sites from those associated with development to those associated with disease progression. These results provide a possible explanation for how elevated GRHL2 levels could contribute to poorer prognosis in ER-positive breast cancer. Abstract Estrogen receptor (ER)-positive breast cancer is characterized by late recurrences following initial treatment. The epithelial cell fate transcription factor Grainyhead-like protein 2 (GRHL2) is overexpressed in ER-positive breast cancers and is linked to poorer prognosis as compared to ER-negative breast cancers. To understand how GRHL2 contributes to progression, GRHL2 was overexpressed in ER-positive cells. We demonstrated that elevated GRHL2 imparts plasticity with stem cell- and dormancy-associated traits. RNA sequencing and immunocytochemistry revealed that high GRHL2 not only strengthens the epithelial identity but supports a hybrid epithelial to mesenchymal transition (EMT). Proliferation and tumor studies exhibited a decrease in growth and an upregulation of dormancy markers, such as NR2F1 and CDKN1B. Mammosphere assays and flow cytometry revealed enrichment of stem cell markers CD44 and ALDH1, and increased self-renewal capacity. Cistrome analyses revealed a change in transcription factor motifs near GRHL2 sites from developmental factors to those associated with disease progression. Together, these data support the idea that the plasticity and properties induced by elevated GRHL2 may provide a selective advantage to explain the association between GRHL2 and breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c72323ad8abaa131c1ae72b9d382e27082cefdf" target='_blank'>
              Elevated GRHL2 Imparts Plasticity in ER-Positive Breast Cancer Cells
              </a>
            </td>
          <td>
            Christy Zheng, Kaelyn O. Allen, Tianrui Liu, Natalia M. Solodin, Mark B Meyer, Kelley Salem, Phillipos K. Tsourkas, Sean J. McIlwain, Jessica M. Vera, Erika R. Cromwell, M. S. Ozers, Amy M. Fowler, E. Alarid
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1082aba026dea72fee57aef4a45e7856e9efcf8" target='_blank'>
              Topoisomerase-modulated genome-wide DNA supercoiling domains colocalize with nuclear compartments and regulate human gene expression.
              </a>
            </td>
          <td>
            Qian Yao, Linying Zhu, Zhen Shi, Subhadra Banerjee, Chongyi Chen
          </td>
          <td>2024-08-16</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs), widespread genetic parasites, pose potential threats to the stability of their host genomes. Hence, the interactions observed today between TEs and their host genomes, as well as among the different TE species coexisting in the same host, likely reflect those that did not lead to the extinction of either the host or the TEs. It is not clear to what extent the expression and integration steps of the TE replication cycles are involved in this ‘peaceful’ coexistence. Here, we show that four Drosophila LTR RetroTransposable Elements (LTR-RTEs), although sharing the same overall integration mechanism, preferentially integrate into distinct open chromatin domains of the host germline. Notably, the differential expressions of the gtwin and ZAM LTR-RTEs in ovarian and embryonic somatic tissues, respectively, result in differential integration timings and targeting of accessible chromatin landscapes that differ between early and late embryonic nuclei, highlighting connections between temporal and spatial LTR-RTEs niche partitionings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ef122580a3d05b44b334371537616b10346674" target='_blank'>
              Temporal and spatial niche partitioning in a retrotransposon community of the Drosophila genome
              </a>
            </td>
          <td>
            Marion Varoqui, Mourdas Mohamed, B. Mugat, Daniel Gourion, Maëlys Lemoine, Alain Pélisson, Charlotte Grimaud, S. Chambeyron
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionary conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as scm3 cells are sensitive to DNA damage. We further observe that the scm3 mutant interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with the DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. Thus, the revelation that these chaperones confer genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b8f002bffe33b7b0eebb870a6636324b784a0d" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway
              </a>
            </td>
          <td>
            Prakhar Agarwal, S. Ghosh
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Three-stranded DNA-RNA structures known as R-loops that form during papillomavirus transcription can cause transcription-replication conflicts and lead to DNA damage. We found that R-loops accumulated at the viral early promoter in human papillomavirus (HPV) episomal cells but were greatly reduced in cells with integrated HPV genomes. RNA-DNA helicases unwind R-loops and allow for transcription and replication to proceed. Depletion of the RNA-DNA helicase senataxin (SETX) using siRNAs increased the presence of R-loops at the viral early promoter in HPV-31 (CIN612) and HPV-16 (W12) episomal HPV cell lines. Depletion of SETX reduced viral transcripts in episomal HPV cell lines. The viral E2 protein, which binds with high affinity to specific palindromes near the promoter and origin, complexes with SETX, and both SETX and E2 are present at the viral p97 promoter in CIN612 and W12 cells. SETX overexpression increased E2 transcription activity on the p97 promoter. SETX depletion also significantly increased integration of viral genomes in CIN612 cells. Our results demonstrate that SETX resolves viral R-loops to proceed with HPV transcription and prevent genome integration.IMPORTANCEPapillomaviruses contain small circular genomes of approximately 8 kilobase pairs and undergo unidirectional transcription from the sense strand of the viral genome. Co-transcriptional R-loops were recently reported to be present at high levels in cells that maintain episomal HPV and were also detected at the early viral promoter. R-loops can inhibit transcription and DNA replication. The process that removes R-loops from the PV genome and the requisite enzymes are unknown. We propose a model in which the host RNA-DNA helicase senataxin assembles on the HPV genome to resolve R-loops in order to maintain the episomal status of the viral genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3257970da042eeca9c08a9a174f0a9b3a46f14" target='_blank'>
              Senataxin mediates R-loop resolution on HPV episomes.
              </a>
            </td>
          <td>
            L. Jose, Keely Smith, Anaiya Crowner, EJ Androphy, M. DeSmet
          </td>
          <td>2024-07-24</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Structural variants (SVs) are important contributors to human disease. Their characterization remains however difficult due to their size and association with repetitive regions. Long-read sequencing (LRS) and optical genome mapping (OGM) can aid as their molecules span multiple kilobases and capture SVs in full. In this study, we selected six individuals who presented with unresolved SVs. We applied LRS onto all individuals and OGM to a subset of three complex cases. LRS detected and fully resolved the interrogated SV in all samples. This enabled a precise molecular diagnosis in two individuals. Overall, LRS identified 100% of the junctions at single-basepair level, providing valuable insights into their formation mechanisms without need for additional data sources. Application of OGM added straightforward variant phasing, aiding in the unravelment of complex rearrangements. These results highlight the potential of LRS and OGM as follow-up molecular tests for complete SV characterization. We show that they can assess clinically relevant structural variation at unprecedented resolution. Additionally, they detect (complex) cryptic rearrangements missed by conventional methods. This ultimately leads to an increased diagnostic yield, emphasizing their added benefit in a diagnostic setting. To aid their rapid adoption, we provide detailed laboratory and bioinformatics workflows in this manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d491356850d1dca650c1719115928a885707ff0" target='_blank'>
              Full characterization of unresolved structural variation through long-read sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Griet De Clercq, Lies Vantomme, Barbara Dewaele, B. Callewaert, Olivier Vanakker, Sandra Janssens, Bart Loeys, Mojca Strazisar, W. D. Coster, J. Vermeesch, A. Dheedene, B. Menten
          </td>
          <td>2024-07-19</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="R-loops are trimeric nucleic acid structures that form when an RNA molecule hybridizes with its complementary DNA strand, displacing the opposite strand. These structures regulate transcription as well as replication, but aberrant R-loops can form, leading to DNA breaks and genomic instability if unresolved. R-loop levels are elevated in many cancers as well as cells that maintain high-risk human papillomaviruses. We investigated how the distribution as well as function of R-loops changed between normal keratinocytes and HPV positive cells derived from a precancerous lesion of the cervix (CIN I). The levels of R-loops associated with cellular genes were found to be up to 10-fold higher in HPV positive cells than in normal keratinocytes while increases at ALU1 elements increased by up to 500-fold. The presence of enhanced R-loops resulted in altered levels of gene transcription, with equal numbers increased as decreased. While no uniform global effects on transcription due to the enhanced levels of R-loops were detected, genes in several pathways were coordinately increased or decreased in expression only in the HPV positive cells. This included the downregulation of genes in the innate immune pathway, such as DDX58, IL-6, STAT1, IFN-β, and NLRP3. All differentially expressed innate immune genes dependent on R-loops were also associated with H3K36me3 modified histones. Genes that were upregulated by the presence of R-loops in HPV positive cells included those in the DNA damage repair such as ATM, ATRX, and members of the Fanconi Anemia pathway. These genes exhibited a linkage between R-loops and H3K36me3 as well as γH2AX histone marks only in HPV positive cells. These studies identify a potential link in HPV positive cells between DNA damage repair as well as innate immune regulatory pathways with R-loops and γH2AX/H3K36me3 histone marks that may contribute to regulating important functions for HPV pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30cc12d40eed32c558ede6628c4c8dfd335ab28" target='_blank'>
              HPV induced R-loop formation represses innate immune gene expression while activating DNA damage repair pathways
              </a>
            </td>
          <td>
            Conor W Templeton, Lou Laimins
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Topologically associated domains (TADs) are believed to be involved in the regulation of gene expression. While the impact of TAD perturbations is usually studied in developmental genes with highly cell-type-specific expression patterns, this study examines genes with broad expression profiles divided by a strong insulatory boundary. We focused on the Slc29a3/Unc5b locus, which encompasses two distinct TADs. Our analysis demonstrates that deletions of CTCF binding sites within this locus lead to alterations in local chromatin architecture, disrupting existing loops and forming novel long-range interactions. We evaluated the transcription changes of Unc5b, Slc29a3, Psap, Vsir, Cdh23, and Sgpl1 genes across various organs, finding that TAD boundary disruption results in variable transcriptional responses, where not only magnitude, but also direction of gene expression changes are tissue-specific. Current models of genome architecture, including enhancer competition and hijacking, only partially account for these transcriptional changes, indicating the need for further investigation into the mechanisms underlying TAD function and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c912d81a6af96e3ee3f74faa2fc1bde4c441880" target='_blank'>
              Direction and modality of transcription changes caused by TAD boundary disruption in Slc29a3/Unc5b locus depends on tissue-specific epigenetic context
              </a>
            </td>
          <td>
            Paul Salnikov, P. Belokopytova, Alexandra Yan, E. Viesná, Alexey Korablev, Irina Serova, V. Lukyanchikova, Yana Stepanchuk, N. Torgunakov, Savelii Tikhomirov, V. Fishman
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Many integrative mobile genetic elements (MGEs) are sensitive to RecA-DNA filaments, suggesting their activation may enable them to evade deletion through homologous recombination (HR). This was studied using pneumococcal prophage. Most pneumococci contained prophage that inhibited transformation-mediated HR through chromosomal modification. These included a commonly-targeted attB that reduced transformation by modifying the small cytoplasmic RNA. The prophage were almost universally regulated by C1-type or ImmAR-type systems. Representatives with either type were efficiently deleted by HR. Mutations that impaired a C1-regulated prophage increased this deletion rate, reflecting this element being activated by RecA-DNA filaments imported during transformation. ImmAR-regulated prophage instead responded to transient stimuli by excising as deletion-resistant pseudolysogens, with varying RecA sensitivity, gradually committing to lysing cells following longer-duration signals. These processes were modulated by interactions with the DNA-binding protein DprA. Therefore bacterial diversification though HR is impeded by MGEs that selfishly prevent themselves from being eliminated from the chromosome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a246916c36cd9a48942af1a067709e422a31a41c" target='_blank'>
              Chromosomal curing drives prophage evolution and bacterial epidemiology
              </a>
            </td>
          <td>
            M. Kwun, Alexandru V. Ion, Katinka J. Apagyi, N. Croucher
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d667ed72276b4514cb71b1b131d78ceeae693d2" target='_blank'>
              miR-876-3p is a tumor suppressor on 9p21 that is inactivated in melanoma and targets ERK
              </a>
            </td>
          <td>
            V. Bezrookove, Imran Khan, Anukana Bhattacharjee, Juifang Fan, Robyn Jones, Anima Sharma, M. Nosrati, P. Desprez, Nathan Salomonis, Yihui Shi, A. Dar, M. Kashani-Sabet
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast Cancer Type 1 Susceptibility Protein (BRCA1) is a tumor-suppressor protein that regulates various cellular pathways, including those that are essential for preserving genome stability. One essential mechanism involves a BRCA1-A complex that is recruited to double-strand breaks (DSBs) by RAP80 before initiating DNA damage repair (DDR). How RAP80 itself is recruited to DNA damage sites, however, is unclear. Here, we demonstrate an intrinsic correlation between a methyltransferase DOT1L-mediated RAP80 methylation and BRCA1-A complex chromatin recruitment that occurs during cancer cell radiotherapy resistance. Mechanistically, DOT1L is quickly recruited onto chromatin and methylates RAP80 at multiple lysines in response to DNA damage. Methylated RAP80 is then indispensable for binding to ubiquitinated H2A and subsequently triggering BRCA1-A complex recruitment onto DSBs. Importantly, DOT1L-catalyzed RAP80 methylation and recruitment of BRCA1 have clinical relevance, as inhibition of DOT1L or RAP80 methylation seems to enhance the radiosensitivity of cancer cells both in vivo and in vitro. These data reveal a crucial role for DOT1L in DDR through initiating recruitment of RAP80 and BRCA1 onto chromatin and underscore a therapeutic strategy based on targeting DOT1L to overcome tumor radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a53d6b1d8429c7df99fb0b7dc3e6119436184d" target='_blank'>
              DOT1L-mediated RAP80 methylation promotes BRCA1 recruitment to elicit DNA repair.
              </a>
            </td>
          <td>
            Huangqi Tang, Ya-Fei Lu, Rongsheng Zeng, Chaohua Liu, Yuxin Shu, Yupei Wu, Jiajie Su, Longjiang Di, Jinqin Qian, Jun Zhang, Yuan Tian, Xiaopeng Lu, Xin-Hai Pei, Qian Zhu, Wei-Guo Zhu
          </td>
          <td>2024-08-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b09daa5b66a34ba0df34226e316e985b17964e4" target='_blank'>
              Satellite DNAs and the evolution of the multiple X1X2Y sex chromosomes in the wolf fish Hoplias malabaricus (Teleostei; Characiformes)
              </a>
            </td>
          <td>
            G. A. Toma, A. Sember, C. A. G. Goes, Rafael Kretschmer, F. Porto‐Foresti, L. Bertollo, Thomas Liehr, R. Utsunomia, Marcelo de Bello Cioffi
          </td>
          <td>2024-09-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Recently, new data have been added to the interaction between non-coding RNAs (ncRNAs) and epigenetic machinery. Epigenetics includes enzymes involved in DNA methylation, histone modifications, and RNA modifications, and mechanisms underlying chromatin structure, repressive states, and active states operating in transcription. The main focus is on long ncRNAs (lncRNAs) acting as scaffolds to assemble protein complexes. This review does not cover RNA’s role in sponging microRNAs, or decoy functions. Several lncRNAs were shown to regulate chromatin activation and repression by interacting with Polycomb repressive complexes and mixed-lineage leukemia (MLL) activating complexes. Various groups reported on enhancer of zeste homolog 2 (EZH2) interactions with regulatory RNAs. Knowledge of the function of these complexes opens the perspective to develop new therapeutics for cancer treatment. Lastly, the interplay between lncRNAs and epitranscriptomic modifications in cancers paves the way for new targets in cancer therapy. The approach to inhibit lncRNAs interaction with protein complexes and perspective to regulate epitrascriptomics-regulated RNAs may bring new compounds as therapeuticals in various types of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7d7f52a87e7225cdb1f45c21451b2412bd46ba" target='_blank'>
              Regulatory RNAs: role as scaffolds assembling protein complexes and their epigenetic deregulation
              </a>
            </td>
          <td>
            P. Poltronieri
          </td>
          <td>2024-07-22</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7234a63fd75fc71e91d39632a18b128298ba3fb" target='_blank'>
              Genome-wide identification of copy number variations in wrinkled skin cases of Xiang pigs
              </a>
            </td>
          <td>
            Xiaoli Liu, Fenbin Hu, Wei Wang, Xia Chen, Xi Niu, Shihui Huang, Zhou-cai Wang, Jiafu Wang, Xueqin Ran
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Excess centrosomes cause defects in mitosis, cell-signaling, and cell migration, and therefore their assembly is tightly regulated. Plk4 controls centriole duplication at the heart of centrosome assembly, and elevation of Plk4 promotes centrosome amplification (CA), a founding event of tumorigenesis. Here, we investigate the transcriptional consequences of elevated Plk4 and find Unkempt, a gene encoding an RNA binding protein with roles in translational regulation, to be one of only two upregulated mRNAs. Unk protein localizes to centrosomes and Cep131-positive centriolar satellites and is required for Plk4-induced centriole overduplication in an RNA-binding dependent manner. Translation is enriched at centrosomes and centriolar satellites with Unk and Cep131 promoting this localized translation. A transient centrosomal downregulation of translation occurs early in Plk4-induced CA. CNOT9, an Unk interactor and component of the translational inhibitory CCR4-NOT complex, localizes to centrosomes at this time. In summary, centriolar satellites and Unk promote local translation as part of a translational program that ensures centriole duplication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6c9c911209145f33e4cb8a10948a1237a931b7" target='_blank'>
              The Unkempt RNA binding protein reveals a local translation program in centriole overduplication
              </a>
            </td>
          <td>
            Abraham Martinez, Alexander J. Stemm-Wolf, Ryan M. Sheridan, Matthew J. Taliaferro, Chad G. Pearson
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Histone modifications, known as histone marks, are pivotal in regulating gene expression within cells. The vast array of potential combinations of histone marks presents a considerable challenge in decoding the regulatory mechanisms solely through biological experimental approaches. To overcome this challenge, we have developed a method called CatLearning. It utilizes a modified convolutional neural network architecture with a specialized adaptation Residual Network to quantitatively interpret histone marks and predict gene expression. This architecture integrates long-range histone information up to 500Kb and learns chromatin interaction features without 3D information. By using only one histone mark, CatLearning achieves a high level of accuracy. Furthermore, CatLearning predicts gene expression by simulating changes in histone modifications at enhancers and throughout the genome. These findings help comprehend the architecture of histone marks and develop diagnostic and therapeutic targets for diseases with epigenetic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5803e9ae50a15f5317e9960e85094d317adea14" target='_blank'>
              CatLearning: highly accurate gene expression prediction from histone mark
              </a>
            </td>
          <td>
            Weining Lu, Yin Tang, Yu Liu, Shiyi Lin, Qifan Shuai, Bin Liang, Rongqing Zhang, Yu Cheng, Dong Fang
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple myeloma is the second most hematological cancer. RUVBL1 and RUVBL2 form a subcomplex of many chromatin remodeling complexes implicated in cancer progression. As an inhibitor specific to the RUVBL1/2 complex, CB-6644 exhibits remarkable anti-tumor activity in xenograft models of Burkitt’s lymphoma and multiple myeloma (MM). In this work, we defined transcriptional signatures corresponding to CB-6644 treatment in MM cells and determined underlying epigenetic changes in terms of chromatin accessibility. CB-6644 upregulated biological processes related to interferon response and downregulated those linked to cell proliferation in MM cells. Transcriptional regulator inference identified E2Fs as regulators for downregulated genes and MED1 and MYC as regulators for upregulated genes. CB-6644-induced changes in chromatin accessibility occurred mostly in non-promoter regions. Footprinting analysis identified transcription factors implied in modulating chromatin accessibility in response to CB-6644 treatment, including ATF4/CEBP and IRF4. Lastly, integrative analysis of transcription responses to various chemical compounds of the molecular signature genes from public gene expression data identified CB-5083, a p97 inhibitor, as a synergistic candidate with CB-6644 in MM cells, but experimental validation refuted this hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff9ee9c96de0ec030a3d9d628090554b2bbd4f2" target='_blank'>
              Molecular Signatures of CB-6644 Inhibition of the RUVBL1/2 Complex in Multiple Myeloma
              </a>
            </td>
          <td>
            Weijun Yi, Sebastian A. Dziadowicz, Rachel S. Mangano, Lei Wang, Joseph C. McBee, Steven M. Frisch, Lori A. Hazlehurst, Donald A. Adjeroh, Gangqing Hu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human cell line models, including the neuronal precursor line LUHMES, are important for investigating developmental transcriptional dynamics within imprinted regions, particularly the 15q11-q13 Angelman (AS) and Prader-Willi (PWS) syndrome locus. AS results from loss of maternal UBE3A in neurons, where the paternal allele is silenced by a convergent antisense transcript UBE3A-ATS, a lncRNA that terminates at PWAR1 in non-neurons. qRT-PCR analysis confirmed the exclusive and progressive increase in UBE3A-ATS in differentiating LUHMES neurons, validating their use for studying UBE3A silencing. Genome-wide transcriptome analyses revealed changes to 11 834 genes during neuronal differentiation, including the upregulation of most genes within the 15q11-q13 locus. To identify dynamic changes in chromatin loops linked to transcriptional activity, we performed a HiChIP validated by 4C, which identified two neuron-specific CTCF loops between MAGEL2-SNRPN and PWAR1-UBE3A. To determine if allele-specific differentially methylated regions (DMR) may be associated with CTCF loop anchors, whole genome long-read nanopore sequencing was performed. We identified a paternally hypomethylated DMR near the SNRPN upstream loop anchor exclusive to neurons and a paternally hypermethylated DMR near the PWAR1 CTCF anchor exclusive to undifferentiated cells, consistent with increases in neuronal transcription. Additionally, DMRs near CTCF loop anchors were observed in both cell types, indicative of allele-specific differences in chromatin loops regulating imprinted transcription. These results provide an integrated view of the 15q11-q13 epigenetic landscape during LUHMES neuronal differentiation, underscoring the complex interplay of transcription, chromatin looping, and DNA methylation. They also provide insights for future therapeutic approaches for AS and PWS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c616b0f402f95ed53dc1945c207ee4eb663c1600" target='_blank'>
              Integration of CTCF loops, methylome, and transcriptome in differentiating LUHMES as a model for imprinting dynamics of the 15q11-q13 locus in human neurons.
              </a>
            </td>
          <td>
            Orangel J. Gutierrez Fugón, Osman Sharifi, Nicholas Heath, Daniela C. Soto, J. A. Gomez, Dag Yasui, A. J. P. Mendiola, H. O’Geen, U. Beitnere, Marketa Tomkova, Viktoria Haghani, Greg Dillon, David J. Segal, J. LaSalle
          </td>
          <td>2024-07-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Deoxyribonucleic acid (DNA) methylation plays a key role in gene regulation and is critical for development and human disease. Techniques such as whole-genome bisulfite sequencing (WGBS) and reduced representation bisulfite sequencing (RRBS) allow DNA methylation analysis at the genome scale, with Illumina NovaSeq 6000 and MGI Tech DNBSEQ-T7 being popular due to their efficiency and affordability. However, detailed comparative studies of their performance are not available. In this study, we constructed 60 WGBS and RRBS libraries for two platforms using different types of clinical samples and generated approximately 2.8 terabases of sequencing data. We systematically compared quality control metrics, genomic coverage, CpG methylation levels, intra- and interplatform correlations, and performance in detecting differentially methylated positions. Our results revealed that the DNBSEQ platform exhibited better raw read quality, although base quality recalibration indicated potential overestimation of base quality. The DNBSEQ platform also showed lower sequencing depth and less coverage uniformity in GC-rich regions than did the NovaSeq platform and tended to enrich methylated regions. Overall, both platforms demonstrated robust intra- and interplatform reproducibility for RRBS and WGBS, with NovaSeq performing better for WGBS, highlighting the importance of considering these factors when selecting a platform for bisulfite sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1781d9e0ece91d256378a0a534b6341d62451d73" target='_blank'>
              Beyond the base pairs: comparative genome-wide DNA methylation profiling across sequencing technologies
              </a>
            </td>
          <td>
            Xin Liu, Yu Pang, Junqi Shan, Yunfei Wang, Yanhua Zheng, Yuhang Xue, Xuerong Zhou, Wenjun Wang, Yanlai Sun, Xiaojing Yan, Jiantao Shi, Xiaoxue Wang, Hongcang Gu, Fan Zhang
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00690bfe5789fdccfdd4c5cebda562df27157a03" target='_blank'>
              A new evaluation of the rearranged non-coding control region of JC virus in patients with colorectal cancer
              </a>
            </td>
          <td>
            Azadeh Haghi Navand, Shahram Jalilian, Kambiz Ahmadi Angali, Mohammad Karimi Babaahmadi, A. Talaiezadeh, M. Makvandi
          </td>
          <td>2024-08-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy.
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, Tengjiang Chen, Jie Lei, Changwen Chen, Shunqin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene regulators physically associate to the genome, in a combinatorial fashion, to drive tissue-specific gene expression programs. Uncovering the genome-wide activity of all gene regulators across tissues is therefore needed to understand how the genome is regulated during development. Here, we take a first step forward towards achieving this goal. Using CUT&RUN, we systematically measured the genome-wide binding profiles of key transcription factors and cofactors that mediate the activity of ontogenetically relevant signaling pathways in select mouse tissues at two developmental stages. Computation of the numerous genome-wide binding datasets unveiled a large degree of tissue and time-specific activity for each gene regulator, and several factor-specific idiosyncrasies. Moreover, we identified “popular” regulatory regions that are bound by a multitude of pathway regulators. Popular regions tend to be more evolutionarily conserved, implying their essentiality. Consistently, they lie in the proximity of genes whose dysregulation causes early embryonic lethality in the mouse. Moreover, the human homologs of these regions are also bound by many gene regulators and are highly conserved in human populations, indicating that they retain functional relevance for human development. This work constitutes a decisive step towards understanding how the genome is simultaneously read and used by gene regulators in a holistic fashion and unveils multiple genomic mechanisms that drive embryonic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea44cb9e15a021f4221a0294719b3ece65eeee5" target='_blank'>
              The Transcription Factor Binding Landscape of Mouse Development
              </a>
            </td>
          <td>
            A. Nordin, Gianluca Zambanini, Mattias Jonasson, Tamina Weiss, Yorick Bernardus Cornelis van de Grift, P. Pagella, Claudio Cantù
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            , Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aflatoxin B1 (AFB1), a potent mycotoxin, is one of the environmental risk factors that cause liver cancer. In the liver, the bioactivated AFB1 intercalates into the DNA double helix to form a bulky DNA adduct which will lead to mutation if left unrepaired. Here, we adapted the tXR-seq method to measure the nucleotide excision repair of AFB1-induced DNA adducts at single-nucleotide resolution on a genome-wide scale, and compared it with repair data obtained from conventional UV-damage XR-seq. Our results showed that transcription-coupled repair plays a major role in the damage removal process. We further analyzed the distribution of nucleotide excision repair sites for AFB1-induced DNA adducts within the 3D human genome organization. Our analysis revealed a heterogeneous AFB1-dG repair across four different organization levels, including chromosome territories, A/B compartments, TADs, and chromatin loops. We found that chromosomes positioned closer to the nuclear center and regions within A compartments have higher levels of nucleotide excision repair. Notably, we observed high repair activity around both TAD boundaries and loop anchors. These findings provide insights into the complex interplay between AFB1-induced DNA damage repair, transcription, and 3D genome organization, shedding light on the mechanisms underlying AFB1-induced mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6fed0c3af4e712a403c0af2af9bb447bf31d09" target='_blank'>
              Nucleotide excision repair of aflatoxin-induced DNA damage within the 3D human genome organization.
              </a>
            </td>
          <td>
            Yiran Wu, Muhammad Muzammal Adeel, Dian Xia, Aziz Sancar, Wentao Li
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The mechanisms that work alongside BRAFV600E oncogene in melanoma development, in addition to ultraviolet (UV) radiation (UVR), are of great interest. Analysis of human melanoma tumors [data from The Cancer Genome Atlas (TCGA)] revealed that 50% or more of the samples expressed no or low amounts of serine/threonine protein kinase STK11 (also known as LKB1) protein. Here, we report that, in a mouse model, concomitant neonatal BrafV600E activation and Lkb1 tumor suppressor ablation in melanocytes led to full melanoma development. A single postnatal dose of UVB radiation had no effect on melanoma onset in Lkb1-depleted mice compared with BrafV600E-irradiated mice, but increased tumor multiplicity. In concordance with these findings and previous reports, Lkb1-null irradiated mice exhibited deficient DNA damage repair (DDR). Histologically, tumors lacking Lkb1 were enriched in neural-like tumor morphology. Genetic profiling and gene set enrichment analyses of tumor sample mutated genes indicated that loss of Lkb1 promoted the selection of altered genes associated with neural differentiation processes. Thus, these results suggest that the loss of Lkb1 cooperates with BrafV600E and UVR, impairing the DDR and increasing melanoma multiplicity and neural-like dedifferentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468890566d37157a91439461401c20d4b0665a15" target='_blank'>
              Loss of Lkb1 cooperates with BrafV600E and ultraviolet radiation, increasing melanoma multiplicity and neural-like dedifferentiation.
              </a>
            </td>
          <td>
            K. McGrail, Elena González-Sánchez, Paula Granado-Martínez, Roberto Orsenigo, Yuxin Ding, Berta Ferrer, Javier Hernández-Losa, Iván Ortega, Juan Martín-Caballero, Eva Muñoz-Couselo, Vicente García-Patos, Juan A. Recio
          </td>
          <td>2024-08-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8200f580864196977ce1aaadc3c51eb18d6c8f71" target='_blank'>
              Evolutionary patterns and functional effects of 3D chromatin structures in butterflies with extensive genome rearrangements
              </a>
            </td>
          <td>
            Botong Zhou, Ping Hu, Guichun Liu, Zhou Chang, Zhiwei Dong, Zihe Li, Yuan Yin, Zunzhe Tian, Ge Han, Wen Wang, Xueyan Li
          </td>
          <td>2024-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromatin within the nucleus adopts complex three-dimensional structures that are crucial for gene regulation and cellular function. Recent studies have revealed the presence of distinct chromatin subcompartments beyond the traditional A/B compartments (eu- and hetero-chromatin), each exhibiting unique structural and functional properties. Here, we introduce TECSAS (Transformer of Epigenetics to Chromatin Structural AnnotationS), a deep learning model based on the Transformer architecture, designed to predict chromatin subcompartment annotations directly from epigenomic data. TECSAS leverages information from histone modifications, transcription factor binding profiles, and RNA-Seq data to decode the relationship between the biochemical composition of chromatin and its 3D structural behavior. TECSAS achieves high accuracy in predicting subcompartment annotations and reveals the influence of long-range epigenomic context on chromatin organization. Furthermore, we demonstrate the model’s capability to predict the association of loci with nuclear bodies, such as the lamina, nucleoli, and speckles, providing insights into the role of these structures in shaping the 3D genome organization. This study highlights the potential of deep learning models for deciphering the complex interplay between epigenomic features and 3D genome organization, allowing us to better understand genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46d3eeafc78bbe750f181271e73a57733705362e" target='_blank'>
              Epigenetics is all you need: A Transformer to decode chromatin structural compartments from the epigenome
              </a>
            </td>
          <td>
            E. Dodero-Rojas, V. Contessoto, Yao Fehlis, Nicolas Mayala, J. Onuchic
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>155</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-07-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Uterine cancer is the most common gynecologic malignancy in the United States, with endometrioid endometrial adenocarcinoma (EC) being the most common histologic sub-type. Considering the molecular classifications of EC, efforts have been made to identify additional biomarkers that can assist in diagnosis, prognosis, and individualized therapy. We sought to explore the relationship of Repressor Element 1 (RE1) silencing transcription factor (REST), which downregulates neuronal genes in non-neuronal tissue, along with matrix metalloproteinase-24 (MMP24) and EC. We analyzed the expression of REST and MMP24 in 31 cases of endometrial cancer and 16 controls. We then explored the baseline expression of REST and MMP24 in two EC cell lines (Ishikawa and HEC-1-A) compared to a benign cell line (t-HESC) and subsequently evaluated proliferation, migration, and invasion in the setting of loss of REST gene expression. REST and MMP24 expression were significantly lower in human EC samples compared to control samples. REST was highly expressed in EC cell lines, but decreasing REST gene expression increased proliferation (FC: 1.13X, p < 0.0001), migration (1.72X, p < 0.0001), and invasion (FC: 7.77X, p < 0.05) in Ishikawa cells, which are hallmarks of cancer progression and metastasis. These findings elicit a potential role for REST as a putative tumor suppressor in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d640a56866f332d95647c4ba33b72698badf61e" target='_blank'>
              Functional Analysis of RE1 Silencing Transcription Factor as a Putative Tumor Suppressor in Human Endometrial Cancer
              </a>
            </td>
          <td>
            Yasmin Abedin, P. Minchella, Riley Peterson, F. Gonnella, Amanda Graham, Ian Cook, Melissa Javellana, Andrea Jewell, Lori Spoozak, Warren B. Nothnick
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The epigenetic layer of regulation has become increasingly relevant in the research focused on tumor suppressors. KLF4 is a well-described zinc-finger transcription factor, mainly known for its role in the acquisition of cell pluripotency. Here we report and describe the most relevant epigenetic regulation mechanisms that affect KLF4 expression in tumors. CpG island methylation emerges as the most common mechanism in several tumors including lung adenocarcinoma, hepatocellular carcinoma, non-Hodgkin lymphomas, among others. Further layers of regulation represented by histone methylation and acetylation and by non-coding RNAs are described. Overall, KLF4 emerges as a crucial target in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e9c4ef46a7844e1c0cc57e2b10d6f8fefd7118" target='_blank'>
              KLF4 is an epigenetically modulated, context-dependent tumor suppressor
              </a>
            </td>
          <td>
            R. Frazzi
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675901519e822bb662f6c8d88bdb2a0a5f8a3f55" target='_blank'>
              Mitochondrial DNA copy number and risk of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Materah Salem Alwehaidah, Rana Al-Awadhi, Moody Al Roomy, Tahani Al Baqer
          </td>
          <td>2024-08-02</td>
          <td>BMC Endocrine Disorders</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Thyroid cancer incidence increases worldwide annually, primarily due to factors such as ionizing radiation (IR), iodine intake, and genetics. Papillary carcinoma of the thyroid (PTC) accounts for about 80% of thyroid cancer cases. RET/PTC1 (coiled‐coil domain containing 6 [CCDC6]‐rearranged during transfection) rearrangement is a distinctive feature in over 70% of thyroid cancers who exposed to low doses of IR in Chernobyl and Hiroshima‒Nagasaki atomic bombings. This study aims to elucidate mechanism between RET/PTC1 rearrangement and IR in PTC. N‐thy‐ori‐3‐1 cells were subjected to varying doses of IR (2/1/0.5/0.2/0.1/0.05 Gy) of IR at different days, and result showed low‐dose IR‐induced RET/PTC1 rearrangement in a dose‐dependent manner. RET/PTC1 has been observed to promote PTC both in vivo and in vitro. To delineate the role of different DNA repair pathways, SCR7, RI‐1, and Olaparib were employed to inhibit non‐homologous end joining (NHEJ), homologous recombination (HR), and microhomology‐mediated end joining (MMEJ), respectively. Notably, inhibiting NHEJ enhanced HR repair efficiency and reduced IR‐induced RET/PTC1 rearrangement. Conversely, inhibiting HR increased NHEJ repair efficiency and subsequent RET/PTC1 rearrangement. The MMEJ did not show a markable role in this progress. Additionally, inhibiting DNA‐dependent protein kinase catalytic subunit (DNA‐PKcs) decreased the efficiency of NHEJ and thus reduced IR‐induced RET/PTC1 rearrangement. To conclude, the data suggest that NHEJ, rather than HR or MMEJ, is the critical cause of IR‐induced RET/PTC1 rearrangement. Targeting DNA‐PKcs to inhibit the NHEJ has emerged as a promising therapeutic strategy for addressing IR‐induced RET/PTC1 rearrangement in PTC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a7f3ce0a1b1698f736c70e4abe3cceb9234a15" target='_blank'>
              Low‐dose ionizing radiation‐induced RET/PTC1 rearrangement via the non‐homologous end joining pathway to drive thyroid cancer
              </a>
            </td>
          <td>
            Yuhao Liu, Jiaojiao Zhu, Shenghui Zhou, Yifan Hou, Ziyan Yan, Xingkun Ao, Ping Wang, Lin Zhou, Huixi Chen, Xinxin Liang, Hua Guan, Shan-shan Gao, Dafei Xie, Yongqing Gu, Ping-Kun Zhou
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment.
              </a>
            </td>
          <td>
            M. Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-02</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Tandem duplication (TD) is a common and important type of structural variation in the human genome. TDs have been shown to play an essential role in many diseases, including cancer. However, it is difficult to accurately detect TDs due to the uneven distribution of reads and the inherent complexity of next-generation sequencing (NGS) data. Methods This article proposes a method called DTDHM (detection of tandem duplications based on hybrid methods), which utilizes NGS data to detect TDs in a single sample. DTDHM builds a pipeline that integrates read depth (RD), split read (SR), and paired-end mapping (PEM) signals. To solve the problem of uneven distribution of normal and abnormal samples, DTDHM uses the K-nearest neighbor (KNN) algorithm for multi-feature classification prediction. Then, the qualified split reads and discordant reads are extracted and analyzed to achieve accurate localization of variation sites. This article compares DTDHM with three other methods on 450 simulated datasets and five real datasets. Results In 450 simulated data samples, DTDHM consistently maintained the highest F1-score. The average F1-score of DTDHM, SVIM, TARDIS, and TIDDIT were 80.0%, 56.2%, 43.4%, and 67.1%, respectively. The F1-score of DTDHM had a small variation range and its detection effect was the most stable and 1.2 times that of the suboptimal method. Most of the boundary biases of DTDHM fluctuated around 20 bp, and its boundary deviation detection ability was better than TARDIS and TIDDIT. In real data experiments, five real sequencing samples (NA19238, NA19239, NA19240, HG00266, and NA12891) were used to test DTDHM. The results showed that DTDHM had the highest overlap density score (ODS) and F1-score of the four methods. Conclusions Compared with the other three methods, DTDHM achieved excellent results in terms of sensitivity, precision, F1-score, and boundary bias. These results indicate that DTDHM can be used as a reliable tool for detecting TDs from NGS data, especially in the case of low coverage depth and tumor purity samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332aeecb63c38630dbdee5179707b4c99d4bb7e6" target='_blank'>
              DTDHM: detection of tandem duplications based on hybrid methods using next-generation sequencing data
              </a>
            </td>
          <td>
            Tianting Yuan, Jinxin Dong, Baoxian Jia, Hua Jiang, Zuyao Zhao, Mengjiao Zhou
          </td>
          <td>2024-07-26</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Characterizing DNA methylation patterns is important for addressing key questions in evolutionary biology, geroscience, and medical genomics. While costs are decreasing, whole-genome DNA methylation profiling remains prohibitively expensive for most population-scale studies, creating a need for cost-effective, reduced representation approaches (i.e., assays that rely on microarrays, enzyme digests, or sequence capture to target a subset of the genome). Most common whole genome and reduced representation techniques rely on bisulfite conversion, which can damage DNA resulting in DNA loss and sequencing biases. Enzymatic methyl sequencing (EM-seq) was recently proposed to overcome these issues, but thorough benchmarking of EM-seq combined with cost-effective, reduced representation strategies has not yet been performed. To do so, we optimized Targeted Methylation Sequencing protocol (TMS)—which profiles ∼4 million CpG sites—for miniaturization, flexibility, and multispecies use at a cost of ∼$80. First, we tested modifications to increase throughput and reduce cost, including increasing multiplexing, decreasing DNA input, and using enzymatic rather than mechanical fragmentation to prepare DNA. Second, we compared our optimized TMS protocol to commonly used techniques, specifically the Infinium MethylationEPIC BeadChip (n=55 paired samples) and whole genome bisulfite sequencing (n=6 paired samples). In both cases, we found strong agreement between technologies (R² = 0.97 and 0.99, respectively). Third, we tested the optimized TMS protocol in three non-human primate species (rhesus macaques, geladas, and capuchins). We captured a high percentage (mean=77.1%) of targeted CpG sites and produced methylation level estimates that agreed with those generated from reduced representation bisulfite sequencing (R² = 0.98). Finally, we applied our protocol to profile age-associated DNA methylation variation in two subsistence-level populations—the Tsimane of lowland Bolivia and the Orang Asli of Peninsular Malaysia—and found age-methylation patterns that were strikingly similar to those reported in high income cohorts, despite known differences in age-health relationships between lifestyle contexts. Altogether, our optimized TMS protocol will enable cost-effective, population-scale studies of genome-wide DNA methylation levels across human and non-human primate species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ea49a3598e0f07df9c655927729fa7dc601194" target='_blank'>
              Cost-effective solutions for high-throughput enzymatic DNA methylation sequencing
              </a>
            </td>
          <td>
            Amy Longtin, Marina M. Watowich, Baptiste Sadoughi, Rachel M. Petersen, Sarah F. Brosnan, Kenneth Buetow, Qiuyin Cai, M. Gurven, Heather M. Highland, Yi-Ting Huang, Hillard Kaplan, Thomas S Kraft, Yvonne A L Lim, J. Long, Amanda D. Melin, Jamie Roberson, Kee-Seong Ng, J. Stieglitz, B. Trumble, Vivek V Venkataraman, Ian J Wallace, Jie Wu, N. Snyder‐Mackler, Angela Jones, Alexander G. Bick, Amanda J. Lea
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bba6936c2ac2d76bcee0a653b32c7b9ef4038fa" target='_blank'>
              Yeast EndoG prevents genome instability by degrading extranuclear DNA species
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Ruofan Yu, P. Thakre, Brenna S. McCauley, Nic Nikoloutsos, Yang Yu, Qian Li, P. Hastings, Weiwei Dang, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-09-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Myc is a major driver of tumor initiation, progression, and maintenance. Up-regulation of Myc protein level rather than acquisition of neomorphic properties appears to underlie most Myc-driven cancers. Cellular mechanisms governing Myc expression remain incompletely defined. In this study, we show that ribosome-associated quality control (RQC) plays a critical role in maintaining Myc protein level. Ribosomes stall during the synthesis of the N-terminal portion of cMyc, generating aberrant cMyc species and necessitating deployment of the early RQC factor ZNF598 to handle translational stress and restore cMyc translation. ZNF598 expression is up-regulated in human glioblastoma (GBM), and its expression positively correlates with that of cMyc. ZNF598 knockdown inhibits human GBM neurosphere formation in cell culture and Myc-dependent tumor growth in vivo in Drosophila. Intriguingly, the SARS-COV-2-encoded translational regulator Nsp1 impinges on ZNF598 to restrain cMyc translation and consequently cMyc-dependent cancer growth. Remarkably, Nsp1 exhibits synthetic toxicity with the translation and RQC-related factor ATP-binding cassette subfamily E member 1, which, despite its normally positive correlation with cMyc in cancer cells, is co-opted by Nsp1 to down-regulate cMyc and inhibit tumor growth. Ribosome stalling during c-myc translation thus offers actionable cancer cell vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e20c4cb54679180301d5c681441f377ad983fd" target='_blank'>
              Ribosome stalling during c-myc translation presents actionable cancer cell vulnerability
              </a>
            </td>
          <td>
            T. P. Khaket, Suman Rimal, Xingjun Wang, Sunil Bhurtel, Yen-Chi Wu, Bingwei Lu
          </td>
          <td>2024-08-01</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Accurate reference genomes are fundamental to understanding biological evolution, biodiversity, hereditary phenomena and diseases. However, many assembled nuclear chromosomes are often contaminated by organelle genomes, which will mislead bioinformatic analysis, and genomic and transcriptomic data interpretation. Methods To address this issue, we developed a tool named Chlomito, aiming at precise identification and elimination of organelle genome contamination from nuclear genome assembly. Compared to conventional approaches, Chlomito utilized new metrics, alignment length coverage ratio (ALCR) and sequencing depth ratio (SDR), thereby effectively distinguishing true organelle genome sequences from those transferred into nuclear genomes via horizontal gene transfer (HGT). Results The accuracy of Chlomito was tested using sequencing data from Plum, Mango and Arabidopsis. The results confirmed that Chlomito can accurately detect contigs originating from the organelle genomes, and the identified contigs covered most regions of the organelle reference genomes, demonstrating efficiency and precision of Chlomito. Considering user convenience, we further packaged this method into a Docker image, simplified the data processing workflow. Discussion Overall, Chlomito provides an efficient, accurate and convenient method for identifying and removing contigs derived from organelle genomes in genomic assembly data, contributing to the improvement of genome assembly quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2949fcd41e909221ac38423a6ad3cf2f0aef961d" target='_blank'>
              Chlomito: a novel tool for precise elimination of organelle genome contamination from nuclear genome assembly
              </a>
            </td>
          <td>
            Wei Song, Chong Li, Yanming Lu, Dawei Shen, Yunxiao Jia, Yixin Huo, Weilan Piao, Hua Jin
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="RNA-guided endonucleases are involved in processes ranging from adaptive immunity to site-specific transposition and have revolutionized genome editing. CRISPR-Cas9, -Cas12 and related proteins use guide RNAs to recognize ∼20-nucleotide target sites within genomic DNA by mechanisms that are not yet fully understood. We used structural and biochemical methods to assess early steps in DNA recognition by Cas12a protein-guide RNA complexes. We show here that Cas12a initiates DNA target recognition by bending DNA to induce transient nucleotide flipping that exposes nucleobases for DNA-RNA hybridization. Cryo-EM structural analysis of a trapped Cas12a-RNA-DNA surveillance complex and fluorescence-based conformational probing show that Cas12a-induced DNA helix destabilization enables target discovery and engagement. This mechanism of initial DNA interrogation resembles that of CRISPR-Cas9 despite distinct evolutionary origins and different RNA-DNA hybridization directionality of these enzyme families. Our findings support a model in which RNA-mediated DNA engineering begins with local helix distortion by transient CRISPR-Cas protein binding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca8177ea3bf5a7611f5cd9ffa09c83dca74a49" target='_blank'>
              CRISPR-Cas12a bends DNA to destabilize base pairs during target interrogation
              </a>
            </td>
          <td>
            K. Soczek, Joshua C. Cofsky, Owen T. Tuck, Honglue Shi, Jennifer A. Doudna
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Noncoding single nucleotide variants are the predominant class of genetic variation in whole genome sequencing and are key drivers of phenotypic variation. However, their functional annotation remains challenging. To address this, we develop a hypothesis-driven functional annotation scheme for CTCF binding sites given CTCF’s critical roles in gene regulation and extensive profiling in regulatory datasets. We synthesize CTCF’s binding patterns at 1,063,879 genomic loci across 214 biological contexts into a summary metric, which we refer to as binding activity. We find that binding activity is significantly enriched for both conserved nucleotides (Pearson R = 0.31, p < 2.2 x 10-16) and sequences that contain high-quality CTCF binding motifs (Pearson R = 0.63, p = 2.9 x 10-12). We then integrate binding activity with high confidence change in precision weight matrix scores. By applying this framework to 1,253,330 SNVs in gnomAD, we explore signatures of selection acting against the disruption of CTCF binding. We find a strong, positive relationship between the mutability adjusted proportion of singletons (MAPS) metric and the loss of CTCF binding at loci with high in vitro activity (Pearson R = 0.67, p = 1.5 x 10-14). To contextualize these findings, we apply MAPS to other functional classes of variation and find that a subset of 198,149 loss of CTCF binding variants are observed as infrequently as missense variants. This work implicates these thousands of rare, noncoding variants that disrupt CTCF binding for further functional studies while providing a blueprint for the interpretable annotation of noncoding variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea605606df9f3fb36737c3abacedd277ab50869b" target='_blank'>
              Identifying deleterious noncoding variation through gain and loss of CTCF binding activity
              </a>
            </td>
          <td>
            Colby Tubbs, Mary Lauren Benton, Evonne McArthur, John A. Capra, Douglas M. Ruderfer
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, Yu-Hang Xing, Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/948860eb063cb703e9fda97c23b09a74cb48fa42" target='_blank'>
              Secrets of DNA-PKcs beyond DNA repair
              </a>
            </td>
          <td>
            Sydney Camfield, Sayan Chakraborty, Shailendra Kumar Dhar Dwivedi, Pijush Kanti Pramanik, Priyabrata Mukherjee, Resham Bhattacharya
          </td>
          <td>2024-07-23</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9824378cb72ff7a6ac9e3450ad8fb3f2d8abaa3a" target='_blank'>
              Human genomic DNA is widely interspersed with i-motif structures.
              </a>
            </td>
          <td>
            C. D. Peña Martinez, Mahdi Zeraati, R. Rouet, Ohan Mazigi, Jake Y Henry, Brian Gloss, J. Kretzmann, Cameron W Evans, E. Ruggiero, Irene Zanin, Maja Marušič, J. Plavec, Sara N. Richter, Tracy M. Bryan, Nicole M Smith, Marcel E. Dinger, Sarah Kummerfeld, Daniel Christ
          </td>
          <td>2024-08-29</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The holoparasitic plant Lophophytum mirabile exhibits remarkable levels of mitochondrial horizontal gene transfer (HGT). Gathering comparative data from other individuals and host plants can provide insights into the HGT process. We sequenced the mitochondrial genome (mtDNA) from individuals of two species of Lophophytum and from mimosoid hosts. We applied a stringent phylogenomic approach to elucidate the origin of the whole mtDNAs, estimate the timing of the transfers, and understand the molecular mechanisms involved. Ancestral and recent HGT events replaced and enlarged the multichromosomal mtDNA of Lophophytum spp., with the foreign DNA ascending to 74%. A total of 14 foreign mitochondrial chromosomes originated from continuous regions in the host mtDNA flanked by short direct repeats. These foreign tracts are circularized by microhomology-mediated repair pathways and replicate independently until they are lost or they eventually recombine with other chromosomes. The foreign noncoding chromosomes are variably present in the population and likely evolve by genetic drift. We present the 'circle-mediated HGT' model in which foreign mitochondrial DNA tracts become circular and are maintained as plasmid-like molecules. This model challenges the conventional belief that foreign DNA must be integrated into the recipient genome for successful HGT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3d29e903592234bfcff719a1e822abd49a6998" target='_blank'>
              Horizontally transferred mitochondrial DNA tracts become circular by microhomology-mediated repair pathways.
              </a>
            </td>
          <td>
            M. Roulet, Luis F. Ceriotti, L. Gatica-Soria, M. Sanchez-Puerta
          </td>
          <td>2024-07-23</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec6e64b128aecafe6f2ababd8df012b838d2b7a" target='_blank'>
              Cohesin prevents cross-domain gene coactivation
              </a>
            </td>
          <td>
            Peng Dong, Shu Zhang, Valentina Gandin, Liangqi Xie, Lihua Wang, Andrew L. Lemire, Wenhong Li, Hideo Otsuna, Takashi Kawase, A. Lander, Howard Y. Chang, Zhe J. Liu
          </td>
          <td>2024-07-24</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a global health concern with persistently high incidence and mortality rates. However, the specific pathogenesis of CRC remains poorly understood. This study aims to investigate the role and pathogenesis of serine and arginine rich splicing factor 10 (SRSF10) in colorectal cancer. Methods Bioinformatics analysis was employed to predict SRSF10 gene expression in CRC patients. Functional experiments involving SRSF10 knockdown and overexpression were conducted using CCK8, transwell, scratch assay, and flow cytometry. Additionally, the PRIdictor website was utilized to predict the SRSF10 interaction site with RFC5. The identification of different transcripts of SRSF10-acting RFC5 pre-mRNA was achieved through agarose gel electrophoresis. Result The knockdown of SRSF10 inhibited the proliferation and migration ability of CRC cells, while promoting apoptosis and altering the DNA replication of CRC cells. Conversely, when SRSF10 was highly expressed, it enhanced the proliferation and migration ability of CRC cells and caused changes in the cell cycle of colorectal cancer cells. This study revealed a change in the replicating factor C subunit 5 (RFC5) gene in colorectal cancer cells following SRSF10 knockdown. Furthermore, it was confirmed that SRSF10 increased RFC5 exon2-AS1(S) transcription variants, thereby promoting the development of colorectal cancer through AS1 exclusion to exon 2 of RFC5. Conclusion In summary, this study demonstrates that SRSF10 promotes the progression of colorectal cancer by generating an aberrantly spliced exclusion isoform of AS1 within RFC5 exon 2. These findings suggest that SRSF10 could serve as a crucial target for the clinical diagnosis and treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183e863439be0cd3db11d90e783d10b2b9aa76e8" target='_blank'>
              High Expression of SRSF10 Promotes Colorectal Cancer Progression by Aberrant Alternative Splicing of RFC5
              </a>
            </td>
          <td>
            Shuai Xu, Fangmin Zhong, Junyao Jiang, Fangyi Yao, Meiyong Li, Mengxin Tang, Ying Cheng, Yulin Yang, Wen Wen, Xueru Zhang, Bo Huang, Xiao-Zhong Wang
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Atypical Teratoid Rhabdoid Tumor (ATRT) is a highly aggressive pediatric brain tumor and characterized by SMARCB1 deletion. SMARCB1 is a member of the SWI/SNF chromatin remodeling complex that is responsible for determining cellular pluripotency and lineage commitment. We performed an RNAi-based functional genomic screen to identify key epigenetic regulators that co-operate with SMARCB1 loss and contribute to tumorigenesis in ATRT. 406 genes associated with epigenetic regulation in patient derived ATRT cell lines were targeted. This screen identified BMI1, a component of the Polycomb Repressive Complex 1 (PRC1), as one of the top targets essential for ATRT cell growth. Using RNA-seq and ChIP-Seq analysis we demonstrated that BMI1 co-operates with SMARCB1 deletion to suppress transcription of pro-differentiation pathways and promote self-renewal of tumor stem cells. In SMARCB1 deficient cells BMI1 forms a partial PRC1 complex devoid of DNA binding components. We found that genetic inactivation of BMI1 suppresses ATRT cell clonogenicity in vitro and decreases S-phase length. By Doxycycline-inducible SMARCB1 re-expression system we revealed activation of proapoptotic proteins p16 and p21. With immunoprecipitation for BMI1, followed by mass spectrometry analysis we demonstrated that re-expression of SMARCB1 activates two PRC1 chromatin localizing components CBX4 and CBX8. Moreover, inhibition of BMI1 significantly prolongs mice survival and decreases tumor size as evaluated by IVIS imaging system and T2-MRI. Thus, we demonstrated that SMARCB1 deletion results in reprograming of BMI1 chromatin occupancy away from lineage specification by altering the components of the PRC1 complex and that PRC1/BMI1 inhibition is a novel therapeutic approach in ATRT.
 Citation Format: Irina Irina Alimova, Gillian Murdock, Angela M. Pierce, Dong Wang, Nathan Dahl, Sujatha Venkataraman, Rajeev Vibhakar. PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d643026696b6226900916830deb821d92b3f76f" target='_blank'>
              Abstract A080 PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Irina Irina Alimova, Gillian Murdock, Angela M Pierce, Dong Wang, N. Dahl, S. Venkataraman, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. We specify that the DnaA/DNA ratio may be a more rigorous feature in determining the variability of initiation time in E. coli than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work is a continued examination of our hypothesis of the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the variable DnaA/DNA ratio may serve as an essential source of variability of the replication initiation event and the ensuing chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi A. Gheber, Mario Feingold, I. Fishov
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1daefafcb98a739594660ed94aebb04da4a2f" target='_blank'>
              DNA polymerase swapping in Caudoviricetes bacteriophages
              </a>
            </td>
          <td>
            N. Yutin, I. Tolstoy, Pascal Mutz, Y. Wolf, M. Krupovic, Eugene V. Koonin
          </td>
          <td>2024-08-26</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Detection of minor DNA allele alterations is becoming increasingly important for early detection and monitoring of cancer. We describe a new method that uses ultraviolet light to eliminate wild-type DNA alleles and enables improved detection of minor genetic or epigenetic changes.


METHODS
Pyrimidine-dependent UV-based minor-allele enrichment (PD-UVME) employed oligonucleotide probes that incorporated a UVA-sensitive 3-cyanovinylcarbazole (CNVK), placed directly opposite interrogated pyrimidines, such as thymine (T) or cytosine (C) in wild-type (WT) DNA. Upon UVA-illumination, CNVK cross-linked with T/C, preventing subsequent amplification. Mutations that removed the T/C escaped cross-linking and were amplified and detected. Similarly, CNVK discriminated between methylated and unmethylated cytosine in CpG dinucleotides, enabling direct enrichment of unmethylated DNA targets. PD-UVME was combined with digital droplet PCR (ddPCR) to detect serine/threonine-protein kinase B-Raf (BRAF) V600E mutations in model systems, thyroid patient cancer tissue samples, and circulating DNA of tumor origin (ctDNA) from melanoma patients.


RESULTS
One thyroid cancer sample out of 9, and 6 circulating-DNA samples out of 7 were found to be BRAF V600E-positive via PD-UVME while classified as negative by conventional ddPCR. Positive samples via conventional ddPCR were also found positive via PD-UVME. All 10 circulating cell-free DNA (cfDNA) samples obtained from normal volunteers were negative via both approaches. Furthermore, preferential enrichment of unmethylated alleles in MAGEA1 promoters using PD-UVME was demonstrated.


CONCLUSIONS
PD-UVME mutation/methylation enrichment performed prior to ddPCR magnifies low-level mutations or epigenetic changes and increases sensitivity and confidence in the results. It can assist with clinical decisions that hinge on the presence of trace alterations like BRAF V600E.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c362fd25c2fb0f063f30c56509492e4fe9cd161b" target='_blank'>
              Pyrimidine-Dependent UV-Mediated Cross-Linking Magnifies Minor Genetic or Epigenetic Changes in Clinical Samples.
              </a>
            </td>
          <td>
            F. Yu, Farzana Ahmed, S. Smilkou, S. Yasmin-Karim, Farzaneh Darbeheshti, A. Markou, Martyn Bullock, Ioannis Boukovinas, V. Adalsteinsson, E. Lianidou, G. M. Makrigiorgos
          </td>
          <td>2024-07-23</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a1e72cd7656def115fe3bbce4a02021bb17ead" target='_blank'>
              Involvement of the splicing factor SART1 in the BRCA1-dependent homologous recombination repair of DNA double-strand breaks
              </a>
            </td>
          <td>
            Kie Ozaki, Reona Kato, Takaaki Yasuhara, Yuki Uchihara, Miyako Hirakawa, Yu Abe, Hiroki Shibata, Reika Kawabata-Iwakawa, Aizhan Shakayeva, Palina Kot, Kiyoshi Miyagawa, Keiji Suzuki, Naoki Matsuda, Atsushi Shibata, M. Yamauchi
          </td>
          <td>2024-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract The bacterium Deinococcus radiodurans is known to survive high doses of DNA damaging agents. This resistance is the result of robust antioxidant systems which protect efficient DNA repair mechanisms that are unique to Deinococcus species. The protein DdrC has been identified as an important component of this repair machinery. DdrC is known to bind to DNA in vitro and has been shown to circularize and compact DNA fragments. The mechanism and biological relevance of this activity is poorly understood. Here, we show that the DdrC homodimer is a lesion-sensing protein that binds to two single-strand (ss) or double-strand (ds) breaks. The immobilization of DNA breaks in pairs consequently leads to the circularization of linear DNA and the compaction of nicked DNA. The degree of compaction is directly proportional with the number of available nicks. Previously, the structure of the DdrC homodimer was solved in an unusual asymmetric conformation. Here, we solve the structure of DdrC under different crystallographic environments and confirm that the asymmetry is an endogenous feature of DdrC. We propose a dynamic structural mechanism where the asymmetry is necessary to trap a pair of lesions. We support this model with mutant disruption and computational modeling experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28aafcf5c41af6c5cd747f9262cba3fcf3d744c7" target='_blank'>
              DdrC, a unique DNA repair factor from D. radiodurans, senses and stabilizes DNA breaks through a novel lesion-recognition mechanism
              </a>
            </td>
          <td>
            R. Szabla, Mingyi Li, Victoria Warner, Yifeng Song, Murray Junop
          </td>
          <td>2024-07-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3338350362bc2189e1725a808911e7a879595b1f" target='_blank'>
              TESC associated with poor prognosis enhances cancer stemness and migratory properties in liver cancer
              </a>
            </td>
          <td>
            Peng Ye, Shahang Luo, Junyu Huang, Xihua Fu, Xiaoxia Chi, Jong-Ho Cha, Yumei Chen, Yanjun Mai, Kai-Wen Hsu, Xiuwen Yan, Wen-Hao Yang
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5395af718a191f5fa3586c7dc9b0e07227b5b78c" target='_blank'>
              Subunit-specific analysis of cohesin-mutant myeloid malignancies reveals distinct ontogeny and outcomes
              </a>
            </td>
          <td>
            Joahnn-Christoph Jann, Christopher B. Hergott, Marisa Winkler, Yiwen Liu, Benjamin Braun, A. Charles, K. Copson, Shougat Barua, M. Meggendorfer, N. Nadarajah, S. Shimony, Eric S. Winer, M. Wadleigh, Richard M Stone, D. DeAngelo, Jacqueline S Garcia, Torsten Haferlach, R. C. Lindsley, M. Luskin, Maximilian Stahl, Zuzana Tothova
          </td>
          <td>2024-07-20</td>
          <td>Leukemia</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Whole exon sequencing (WES) data are frequently used for cancer diagnosis and genome-wide association studies (GWAS), hinging upon high-coverage read mapping, informative variant calling, and high-quality reference genomes. The center position of the currently used genome assembly, GRCh38, is now challenged by two newly publicized telomere-to-telomere or T2T genomes, T2T-CHM13 and T2T-YAO, and it becomes urgent to have a comparative study to test population specificity using the three reference genomes based on real case WES data. We here report our analysis along this line for 19 tumor samples collected from Chinese patients. The primary comparison of the exon regions among the three references reveals that the sequences in up to ∼1% target regions in YAO are widely diversified from GRCh38 and may lead to off-target in sequence capture. However, YAO still outperforms GRCh38 genomes by obtaining 7.41% more mapped reads. Due to more reliable read-mapping and closer phylogenetic relationship with the samples than GRCh38, YAO reduces half of variant calls of clinical significance which are mostly benign while keeping sensitivity in identifying pathogenic variants. YAO also outperforms CHM13 in reducing calls of Chinese-specific variants. Our findings highlight the critical need for employing population-specific reference genomes in genomic analysis to ensure accurate variant analysis and the significant benefits of tailoring these approaches to the unique genetic backgrounds of each ethnic group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125110dee90b36e3730eebfc5dcf7afd43e85aa6" target='_blank'>
              Enhancing Variant Calling in Whole Exome Sequencing (WES) Data Using Population-Matched Reference Genomes
              </a>
            </td>
          <td>
            Shuming Guo, Zhuo Huang, Yanming Zhang, Yukun He, Xiangju Chen, Wenjuan Wang, Lansheng Li, Yu Kang, Zhancheng Gao, Jun Yu, Zhenglin Du, Yanan Chu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7afaab9db2ee6172d70bbe5180a07660e6f1ac0" target='_blank'>
              Mitochondrial DNA content and methylation in sperm of patients with asthenozoospermia.
              </a>
            </td>
          <td>
            Qiang Geng, Ruifang Gao, Yuan Sun, Shaofeng Chen, Lili Sun, Wei Li, Zhong Li, Yu Zhao, Feng Zhao, Ying Zhang, Anwen Li, Hongbin Liu
          </td>
          <td>2024-08-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Since the mid-1970s, increasingly innovative methods to detect DNA methylation provided detailed information about its distribution, functions, and dynamics. As a result, new concepts were formulated and older ones were revised, transforming our understanding of the associated biology and catalyzing unprecedented advances in biomedical research, drug development, anthropology, and evolutionary biology. In this review, we discuss a few of the most notable advances, which are intimately intertwined with the study of DNA methylation, with a particular emphasis on the past three decades. Examples of these strides include elucidating the intricacies of 5-mC oxidation, which are at the core of the reversibility of this epigenetic modification; the 3D structural characterization of eukaryotic DNA methyltransferases, which offered insights into the mechanisms of several disease-associated mutations; a more in-depth understanding of DNA methylation in development and disease; the possibility to learn about the biology of extinct species; the development of epigenetic clocks and their use to interrogate aging and disease; and the emergence of epigenetic biomarkers and therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe028a64ba00f3b24fe4b38b8f6dd294810d717" target='_blank'>
              Then and Now in Eukaryotic DNA Methylation.
              </a>
            </td>
          <td>
            Richard A Stein, Faris E Gomaa, Pranaya Raparla, L. Riber
          </td>
          <td>2024-09-09</td>
          <td>Physiological genomics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a-RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect.
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Here, we introduce the highly versatile circular polymerase chain reaction (CiPCR) technique, propose a mechanism of action, and describe a number of examples demonstrating the versatility of this technique. CiPCR takes place between two fragments of dsDNA with two homologous regions, as long as one of the fragments carries said regions at its 3′- and 5′-ends. Upon hybridization, elongation by a polymerase occurs from all 3′-ends continuously until a 5′-end is reached, leading to stable circular dsDNA with staggered nicks. When both dsDNA fragments carry the homology at their 3′- and 5′-ends (Type I CiPCR), all four 3′-ends effectively prime amplification of the intervening region and CiPCR products can function as template during the reaction. In contrast, when only one of the two dsDNA fragments carries the homologous regions at its 3′- and 5′-ends and the other carries such regions internally (Type II CiPCR), only two 3′-ends can be amplified and CiPCR products possess no template activity. We demonstrate the applicability of both CiPCR types via well-illustrated experimental examples. CiPCR is well adapted to the quick resolution of most of the molecular cloning challenges faced by the biology/biomedicine laboratory, including the generation of insertions, deletions, and mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3cb2ea36d681e36d6cc2ddc27dd7f88df313b0b" target='_blank'>
              Circular PCR as an efficient and precise umbrella of methods for the generation of circular dsDNA with staggered nicks: Mechanism and types
              </a>
            </td>
          <td>
            Pedro Ferro-Gallego, Antón Vila-Sanjurjo, Andrea Katherine Valderrama Pereira, Gonzalo Porres Pérez, Lourdes Domínguez-Gerpe
          </td>
          <td>2024-07-19</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sequencing mitochondrial DNA (mtDNA) variants from single cells has resolved clonality and lineage in native human samples and clinical specimens. Prior work established that heteroplasmic mtDNA variants can be used to delineate clonality in hematopoiesis, but they have limited ability to reconstruct cellular phylogenies. However, a recent report by Weng et al. challenges the current paradigm by describing an unprecedented number of shared mtDNA variants between cells that reportedly resolve high-resolution phylogenetic trees. We re-examined the claims of Weng et al., and identified two major points of concern regarding this unprecedented connectedness. First, shared variants between cells are disproportionately detected in a single molecule per cell, and second, these variants are enriched 10–20-fold at the edges of mtDNA molecules, reminiscent of artifacts reported in other sequencing approaches. Further, our analyses show that pruning low support and likely artificial mtDNA variants removes nearly all of the reported phylogenetic structure. Thus, we strongly caution against using mtDNA variant calling workflows that rely on minimal evidence, including the computational pipeline introduced in Weng et al., as variants with high connectedness and low evidence are likely artifacts that lead to the construction of false phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae6d387965c56c26ee411bc842b91f11e043e8f3" target='_blank'>
              Artifacts in single-cell mitochondrial DNA mutation analyses misinform phylogenetic inference
              </a>
            </td>
          <td>
            Caleb A. Lareau, M. Chapman, Livius Penter, T. Nawy, D. Pe’er, Leif S. Ludwig
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="ABSTRACT The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Deciphering how nucleotides in genomes encode regulatory instructions and molecular machines is a long-standing goal in biology. DNA language models (LMs) implicitly capture functional elements and their organization from genomic sequences alone by modeling probabilities of each nucleotide given its sequence context. However, using DNA LMs for discovering functional genomic elements has been challenging due to the lack of interpretable methods. Here, we introduce nucleotide dependencies which quantify how nucleotide substitutions at one genomic position affect the probabilities of nucleotides at other positions. We generated genome-wide maps of pairwise nucleotide dependencies within kilobase ranges for animal, fungal, and bacterial species. We show that nucleotide dependencies indicate deleteriousness of human genetic variants more effectively than sequence alignment and DNA LM reconstruction. Regulatory elements appear as dense blocks in dependency maps, enabling the systematic identification of transcription factor binding sites as accurately as models trained on experimental binding data. Nucleotide dependencies also highlight bases in contact within RNA structures, including pseudoknots and tertiary structure contacts, with remarkable accuracy. This led to the discovery of four novel, experimentally validated RNA structures in Escherichia coli. Finally, using dependency maps, we reveal critical limitations of several DNA LM architectures and training sequence selection strategies by benchmarking and visual diagnosis. Altogether, nucleotide dependency analysis opens a new avenue for discovering and studying functional elements and their interactions in genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2048c84458f5c212aac5fcb83fafe24182dcd406" target='_blank'>
              Nucleotide dependency analysis of DNA language models reveals genomic functional elements
              </a>
            </td>
          <td>
            Pedro Tomaz da Silva, Alexander Karollus, Johannes Hingerl, Gihanna Galindez, Nils Wagner, Xavier Hernández-Alias, Danny Incarnato, Julien Gagneur
          </td>
          <td>2024-07-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028d055f673c0c4d806112c2a9d63a57da73eaef" target='_blank'>
              A brief review of noncoding RNA
              </a>
            </td>
          <td>
            Tina P. George, Suja Subramanian, M. H. Supriya
          </td>
          <td>2024-09-02</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Horizontal gene transfer is a key driver of bacterial evolution, but it also presents severe risks to bacteria by introducing invasive mobile genetic elements. To counter these threats, bacteria have developed various defense systems, including prokaryotic Argonautes (pAgo) and the DNA Defense Module DdmDE system. Through biochemical analysis, structural determination, and in vivo plasmid clearance assays, we elucidate the assembly and activation mechanisms of DdmDE, which eliminates small, multicopy plasmids. We demonstrate that DdmE, a pAgo-like protein, acts as a catalytically inactive, DNA-guided, DNA-targeting defense module. In the presence of guide DNA, DdmE targets plasmids and recruits a dimeric DdmD, which contains nuclease and helicase domains. Upon binding to DNA substrates, DdmD transitions from an autoinhibited dimer to an active monomer, which then translocates along and cleaves the plasmids. Together, our findings reveal the intricate mechanisms underlying DdmDE-mediated plasmid clearance, offering fundamental insights into bacterial defense systems against plasmid invasions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac230afd0ac05cd0569717ab193d8b0059c7437" target='_blank'>
              DdmDE eliminates plasmid invasion by DNA-guided DNA targeting
              </a>
            </td>
          <td>
            Xiao-Yuan Yang, Zhangfei Shen, Chen Wang, Kotaro Nakanishi, Tian-Min Fu
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Centrosome amplification is a feature of cancer cells associated with chromosome instability and invasiveness. Enhancing chromosome instability and subsequent cancer cell death via centrosome unclustering and multipolar divisions is an aimed-for therapeutic approach. Here, we show that centrosome amplification potentiates responses to conventional chemotherapy in addition to its effect on multipolar divisions and chromosome instability. We perform single-cell live imaging of chemotherapy responses in epithelial ovarian cancer cell lines and observe increased cell death when centrosome amplification is induced. By correlating cell fate with mitotic behaviors, we show that enhanced cell death can occur independently of chromosome instability. We identify that cells with centrosome amplification are primed for apoptosis. We show they are dependent on the apoptotic inhibitor BCL-XL and that this is not a consequence of mitotic stresses associated with centrosome amplification. Given the multiple mechanisms that promote chemotherapy responses in cells with centrosome amplification, we assess such a relationship in an epithelial ovarian cancer patient cohort. We show that high centrosome numbers associate with improved treatment responses and longer overall survival. Our work identifies apoptotic priming as a clinically relevant consequence of centrosome amplification, expanding our understanding of this pleiotropic cancer cell feature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9ce2e6325eaf9b5466f08fa222bbaf8fdd0b70" target='_blank'>
              Centrosome amplification primes ovarian cancer cells for apoptosis and potentiates the response to chemotherapy.
              </a>
            </td>
          <td>
            Frances Edwards, Giulia Fantozzi, Anthony Y Simon, Jean-Philippe Morretton, A. Herbette, Andrea E. Tijhuis, René Wardenaar, Stacy Foulane, Simon Gemble, D. Spierings, F. Foijer, Odette Mariani, A. Vincent-Salomon, S. Roman-Roman, X. Sastre-Garau, O. Goundiam, R. Basto
          </td>
          <td>2024-09-05</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNAs (eccDNAs), a type of double‐stranded DNAs (dsDNAs) that facilitate the activation of the DNA sensing machinery, have been implicated in the progression and prognosis of various diseases. While the roles of eccDNAs remain contentious, their significance in diffuse large B‐cell lymphoma (DLBCL) has not been reported. Methods Circular DNA sequencing (circle‐seq) was used to demonstrate the expression profile of eccDNAs in DLBCL, and atomic force microscopy to validate the presence of eccDNAs. CCK‐8 and scRNA‐seq techniques were employed to uncover the activation of eccDNA in the STING pathway, leading to enhanced cell proliferation. Chemotherapeutic drugs were used to test the hypothesis that DNA damage induces the production of eccDNA, thereby activating the STING pathway independent of cGAS. GEO databases were used for verification of the prognosis of the eccDNA‐related genes, and animal models were used to investigate the synergistic effects of DNA damage therapy in combination with STING inhibitors on anti‐tumour responses. Results EccDNAs were widely expressed in DLBCL and associated with the prognosis of patients. Elevated abundance of eccDNAs promoted the progression of DLBCL. Chemotherapeutic drugs‐induced DNA damage triggered the generation of eccDNAs, resulting in the activation of the STING signalling in a cGAS‐independent manner. Moreover, inhibition of STING exerted a synergistic anti‐tumour effect with cisplatin. Conclusions EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. This finding offers a rational therapeutic strategy combining chemotherapy with targeting STING. Highlights EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. The combined treatment of chemotherapeutic drugs with STING inhibitor significantly delayed the tumor progression, providing new insights into the therapeutic strategy for patients with DLBCL, particularly the relapsed and/or refractory (R/R) ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e573ea771bfdb5c81c97bedfb9abf4331d619e8" target='_blank'>
              Multi‐omics integration reveals the oncogenic role of eccDNAs in diffuse large B‐cell lymphoma through STING signalling
              </a>
            </td>
          <td>
            Zijuan Wu, Wei Zhang, Luqiao Wang, Jiayan Leng, Yongle Li, Zhou Fan, Mengtao Zhan, Lei Cao, Yongning Jiang, Yan Jiang, Bing Sun, Jianxin Fu, Jianyong Li, W. Shi, H. Jin
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="R-loops, structures that play a crucial role in various biological processes, are integral to gene expression, the maintenance of genome stability, and the formation of epigenomic signatures. When these R-loops are deregulated, they can contribute to the development of serious health conditions, including cancer and neurodegenerative diseases. The detection of R-loops is a complex process that involves several approaches. These include S9.6 antibody- or RNAse H-based immunoprecipitation, non-denaturing bisulfite footprinting, gel electrophoresis, and electron microscopy. Each of these methods offers unique insights into the nature and behavior of R-loops. In our study, we introduce a novel protocol that has been developed based on a single-molecule DNA combing assay. This innovative approach allows for the direct and simultaneous visualization of RNA:DNA hybrids and replication forks, providing a more comprehensive understanding of these structures. Our findings confirm the transcriptional origin of the hybrids, adding to the body of knowledge about their formation. Furthermore, we demonstrate that these hybrids have an inhibitory effect on the progression of replication forks, highlighting their potential impact on DNA replication and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d0ec1a89d3fa05ea164f315ebe78596d3a2da" target='_blank'>
              Simultaneous Visualization of R-Loops/RNA:DNA Hybrids and Replication Forks in a DNA Combing Assay
              </a>
            </td>
          <td>
            Miroslav Penchev Ivanov, Heather Zecchini, Petra Hamerlik
          </td>
          <td>2024-09-03</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is an aggressive primary intestinal malignancy with the third-highest incidence and second-highest mortality among all cancer types worldwide. Transcription factors (TFs) regulate cell development and differentiation owing to their ability to recognize specific DNA sequences upstream of genes. Numerous studies have demonstrated a strong correlation between TFs, the etiology of tumors, and therapeutic approaches. Here, we aimed to explore prognosis-related TFs and comprehend their carcinogenic mechanisms, thereby offering novel insights into the diagnosis and management of CRC. Materials and Methods Differentially expressed TFs between CRC and normal tissues were identified leveraging The Cancer Genome Atlas database, Weighted correlation network analysis and Cox regression analysis were performed to identify prognosis-related TFs. The cellular functions of hub TF zinc finger E-box binding homeobox 1 (ZEB1) were determined using by 5-ethynyl-2′-deoxyuridine and cell invasion assays in CRC cells. RNA-sequencing, Kyoto Encyclopedia of Genes and Genomes enrichment, and gene set enrichment analyses were used to identify the cellular processes in which ZEB1 participates. Immunoaffinity purification, silver staining mass spectrometry, and a chromatin immunoprecipitation assay were conducted to search for proteins that might interact with ZEB1 and the target genes they jointly regulate. Results Thirteen central TFs related to prognosis were identified through bioinformatics analysis techniques. Among these TFs, ZEB1 emerged as the TF most closely associated with CRC, as determined through a combination of regulatory network diagrams, survival curves, and phenotype analyses. ZEB1 promotes CRC cell growth by recruiting the NuRD(MTA1) complex, and the ZEB1/NuRD(MTA1) complex transcriptionally represses glycolysis-associated tumor suppressor genes. Conclusion Our study not only identified a hub biomarker related to CRC prognosis but also revealed the specific molecular mechanisms through which ZEB1 affects cancer progression. These insights provide crucial evidence for the diagnosis of CRC and potential treatment opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951b486f89f4fbb493f66d9c80785b0eee594498" target='_blank'>
              Transcription factor ZEB1 coordinating with NuRD complex to promote oncogenesis through glycolysis in colorectal cancer
              </a>
            </td>
          <td>
            T. Gao, Xinhui Hao, Jing-yao Zhang, Miaomiao Huo, Ting Hu, Tianyu Ma, Hefen Yu, X. Teng, Yong Wang, Yunkai Yang, Wei Huang, Yan Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92a939542e292edaa1bc766f527a0552465d51" target='_blank'>
              PTIP epigenetically regulates DNA damage-induced cell cycle arrest by upregulating PRDM1
              </a>
            </td>
          <td>
            Y. Nakata, Shion Nagasawa, Y. Sera, N. Yamasaki, A. Kanai, K. Kobatake, Takeshi Ueda, M. Koizumi, Ichiro Manabe, Osamu Kaminuma, Hiroaki Honda
          </td>
          <td>2024-08-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/addc7db29e74077945c05141098a7dc6f40ec3ae" target='_blank'>
              The role of ribosomal DNA methylation in embryonic development, aging and diseases
              </a>
            </td>
          <td>
            Fei Yang, Xutong Guo, Yiming Bao, Rujiao Li
          </td>
          <td>2024-07-31</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Summary DNA sequencing is becoming more affordable and faster through advances in high-throughput technologies. This rise in data availability has contributed to the development of novel algorithms to elucidate previously obscure features and led to an increased reliance on complex workflows to integrate such tools into analyses pipelines. To facilitate the analysis of DNA sequencing data, we created metapipeline-DNA, a highly configurable and extensible pipeline. It encompasses a broad range of processing including raw sequencing read alignment and recalibration, variant calling, quality control and subclonal reconstruction. Metapipeline-DNA also contains configuration options to select and tune analyses while being robust to failures. This standardizes and simplifies the ability to analyze large DNA sequencing in both clinical and research settings. Availability Metapipeline-DNA is an open-source Nextflow pipeline under the GPLv2 license and is freely available at https://github.com/uclahs-cds/metapipeline-DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce629f819b8f5832564713fdef250ffa1edd564" target='_blank'>
              Metapipeline-DNA: A Comprehensive Germline & Somatic Genomics Nextflow Pipeline
              </a>
            </td>
          <td>
            Yash Patel, Chenghao Zhu, Takafumi N. Yamaguchi, Nicholas K. Wang, Nicholas Wiltsie, Alfredo Gonzalez, Helena Winata, Nicole Zeltser, Yu Pan, M. F. E. Mootor, Timothy Sanders, C. Kandoth, Sorel T. Fitz-Gibbon, J. Livingstone, Lydia Y. Liu, Benjamin Carlin, Aaron Holmes, Jieun Oh, John Sahrmann, Shu Tao, Stefan E. Eng, Rupert Hugh-White, Kiarod Pashminehazar, Andrew Park, Arpi Beshlikyan, Madison Jordan, Selina Wu, Mao Tian, J. Arbet, B. Neilsen, Yuan Zhe Bugh, Gina Kim, Joseph Salmingo, Wenshu Zhang, Roni Haas, Aakarsh Anand, Edward Hwang, Anna Neiman-Golden, Philippa L. Steinberg, Wenyan Zhao, Prateek Anand, Brandon L. Tsai, P. Boutros
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The fidelity of replication, especially in the presence of DNA damage, is essential for the proper function of cells. Mutations that inactivate genes involved in DNA damage repair or bypass are enriched in several types of cancer cells. Thus, it is important to further our understanding of the mechanisms governing replication fidelity. PCNA is a ring-shaped complex that encircles DNA at the front of the replication fork, at the double-stranded/single-stranded DNA junction. It serves as a processivity factor for the different DNA replication polymerases, allowing them to replicate longer stretches of DNA by physically tethering them to the DNA and preventing their detachment. In addition, PCNA also regulates and coordinates different DNA damage bypass pathways meant to allow DNA replication in the presence of DNA damage. Due to its essentiality and the numerous functions it has in the cell, much is still unclear about PCNA. Here, we utilize PCNA mutants that lower the stability of the PCNA complex on the chromatin, and thus tend to disassociate and fall from the DNA. Using these mutants, we show that PCNA’s physical presence on the DNA can prevent DNA misalignment at repetitive sequences, leading to increased mutation formation. We also show that PCNA-interacting proteins play an important role in strengthening the ring’s stability on the chromatin. Such repetitive sequence-induced mutations are common in several human diseases and it is important to study their formation and the mechanisms guarding against them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4a21eb48fc762d33559ed0c2e7196cd781bc5" target='_blank'>
              Effects of PCNA Stability on the Formation of Mutations
              </a>
            </td>
          <td>
            Matan Arbel-Groissman, Batia Liefshitz, M. Kupiec
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22a32ba49d7731185468f66f4a9a15ddb1efe55" target='_blank'>
              Is cross-species horizontal gene transfer responsible for gallbladder carcinogenesis
              </a>
            </td>
          <td>
            Monika Rajput, Manoj Pandey, R. Dixit, Vijay K. Shukla
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d183f3062975c60ce5ba3ada1020c80a06fee4" target='_blank'>
              Prion-like domain mediated phase separation of ARID1A promotes oncogenic potential of Ewing’s sarcoma
              </a>
            </td>
          <td>
            Yong Ryoul Kim, Jaegeon Joo, Hee Jung Lee, Chaelim Kim, Ju-Chan Park, Young Suk Yu, Chang Rok Kim, Do Hui Lee, Joowon Cha, Hyemin Kwon, Kimberley M. Hanssen, T. G. Grünewald, Murim Choi, Ilkyu Han, Sangsu Bae, Inkyung Jung, Yongdae Shin, Sung Hee Baek
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is an epigenetic process that results in the transcriptional silencing of one X chromosome in the somatic cells of females. This phenomenon is common to both eutherian and marsupial mammals, but there are fundamental differences. In eutherians, the X chosen for silencing is random. DNA methylation on the eutherian inactive X is high at transcription start sites (TSSs) and their flanking regions, resulting in universally high DNA methylation. This contrasts XCI in marsupials where the paternally derived X is always silenced, and in which DNA methylation is low at TSSs and flanking regions. Here, we examined the DNA methylation status of the tammar wallaby X chromosome during spermatogenesis to determine the DNA methylation profile of the paternal X prior to and at fertilization. Whole genome enzymatic methylation sequencing was carried out on enriched flow-sorted populations of premeiotic, meiotic, and postmeiotic cells. We observed that the X displayed a pattern of DNA methylation from spermatogonia to mature sperm that reflected the inactive X in female somatic tissue. Therefore, the paternal X chromosome arrives at the egg with a DNA methylation profile that reflects the transcriptionally silent X in adult female somatic tissue. We present this epigenetic signature as a candidate for the long sought-after imprint for paternal XCI in marsupials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e12408d360f28982d362b3fdf85cb94edc8ed7" target='_blank'>
              Imprinted X chromosome inactivation in marsupials: The paternal X arrives at the egg with a silent DNA methylation profile.
              </a>
            </td>
          <td>
            A. M. Milton, Laia Marín-Gual, Nicholas C. Lister, Kim L. McIntyre, Patrick G. S. Grady, Melanie K Laird, Donna M Bond, T. Hore, Rachel J. O’Neill, A. Pask, Marilyn B Renfree, Aurora Ruiz-Herrera, Paul D. Waters
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261ea82053aaf73b2c3fb5ebe769c19bae07d5fa" target='_blank'>
              53BP1 loss elicits cGAS-STING-dependent antitumor immunity in ovarian and pancreatic cancer
              </a>
            </td>
          <td>
            Yajie Sun, Jeffrey Patterson-Fortin, Sen Han, Zhe Li, Zuzanna Nowicka, Yuna Hirohashi, Susan Kilgas, Jae Kyo Yi, A. Spektor, Wojciech Fendler, Panagiotis A Konstantinopoulos, Dipanjan Chowdhury
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Interleukin-12 (IL-12) is a critical cytokine with notable anticancer properties, including enhancing T-cell-mediated cancer cell killing, and curbing tumor angiogenesis. To date, many approaches are evaluated to achieve in situ overexpression of IL-12, minimizing leakage and the ensuing toxicity. Here, it is focused on circular single-stranded DNA (Css DNA), a type of DNA characterized by its unique structure, which could be expressed in mammals. It is discovered that Css DNA can induce sustained luciferase expression for half a year by intramuscular injection and showed effective antitumor results by intratumoral injection. Motivated by these findings, a folate-modified LNP system is now developed for the delivery of Css DNA expressing IL-12 for the therapy of 4T1 triple-negative breast cancer (TNBC). This delivery system effectively activates anti-cancer immune responses, slows tumor growth, significantly prolongs survival in animal models, and prevents tumor recurrence. After 6 months of long-term observation, the elevated level of IL-12 is still detectable in the lymph nodes and serum of the cured mice. This study highlights the long-term sustained expression capacity of Css DNA and its ability to inhibit recurrence, and the potential of tumor-targeted LNPs for Css DNA-based cancer therapy, providing a new insight into gene overexpression strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23279eaaa6ad21ff5b4d7a8ec371e59cc5dec26b" target='_blank'>
              Targeted Delivery of Circular Single-Stranded DNA Encoding IL-12 for the Treatment of Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Yijing Zhang, Xisen Cao, Guang Hu, Rui Ye, Li Zhang, Jie Song
          </td>
          <td>2024-09-03</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1fa9923fb71606ea55e83b6b8cbb7a9e56b764" target='_blank'>
              Competition shapes the landscape of X-chromosome-linked genetic diversity
              </a>
            </td>
          <td>
            Teresa Buenaventura, Hakan Bagci, Ilinca Patrascan, Joshua J Graham, Kelsey D Hipwell, Roel Oldenkamp, James W D King, Jesus Urtasun, G. Young, Daniel Mouzo, D. Gómez-Cabrero, B. Rowland, Daniel Panne, Amanda G Fisher, Matthias Merkenschlager
          </td>
          <td>2024-07-26</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2245c6c9be5dc1954f3bd2bc26c574b380581c40" target='_blank'>
              Comprehensive genomic analysis reveals clonal origin and subtype-specific evolution in a case of sporadic multiple meningiomas.
              </a>
            </td>
          <td>
            M. Sakaguchi, Masafumi Horie, Yukinobu Ito, Shingo Tanaka, Keishi Mizuguchi, Hiroko Ikeda, Etsuko Kiyokawa, Mitsutoshi Nakada, Daichi Maeda
          </td>
          <td>2024-07-27</td>
          <td>Brain tumor pathology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Unexplained stillbirth is hypothesised to occur due to premature placental ageing, with unexpected deterioration of placental function for gestational age. Circular RNAs (circRNAs) are enzyme resistant RNA molecules that accumulate in ageing tissues. Furthermore, circRNAs bind gDNA directly, forming circRNA:DNA complexes which can induce DNA breaks. This study investigated circRNA accumulation with gestational age in healthy and stillbirth placentae and determined whether circRNAs directly interact with placental DNA causing DNA damage. Placenta samples (n=60 term uncomplicated; n=4 unexplained stillbirth, 23, 26, 31, 34 weeks’ gestation) were assessed for DNA damage using an alkaline Comet Assay. Expression levels of 6 candidate circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277), and their linear transcripts, were quantified using qPCR. Physical interaction of candidate circRNAs with DNA was confirmed by DNA:RNA ImmunoPrecipitation (DRIP). The effect of circ_0009000 knockdown in HEK293T cells was assessed following transfection with either a siRNA (designed to knockdown circ_0009000) or a scrambled siRNA control, at 5, 10 and 20 nM final concentrations using Lipofectamine RNAiMax. DNA damage was assessed by Comet Assay. Compared with earlier gestations (37, 38, 39 and 40 weeks’), placental DNA damage and expression of all 6 candidate circRNAs, but not their linear transcripts, were increased in 40 and 41+ weeks’ gestation samples, and in stillbirth. DRIP-qPCR signal size was significantly larger in term placentae than in enzyme-treated controls, confirming that all candidate circRNA loci bind to placental DNA. Depletion of circ_0009000 by specific siRNA in HEK293T cells, significantly reduced DNA damage compared to control. Stillbirth placentae show accelerated ageing with premature accumulation of candidate circRNAs (first evidence in humans) at levels consistent with older gestation tissue. Importantly, these circRNAs bind to DNA and circ_0009000 causes DNA breaks in placenta. Therefore, circRNAs (circ_0009000, circ_0024157, circ_0061017, circ_0036877, circ_0054624 and circ_0111277) play a role in placental ageing and associate with stillbirth, likely via DNA damage. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9adf6d93429ec373710f176ef7b50a5c1efd5e40" target='_blank'>
              Circular RNAs correlate with DNA damage in ageing human placental tissue and in stillbirth
              </a>
            </td>
          <td>
            A. Arthurs, Matilda R. Jackson, Dylan McCullough, Hamish S. Scott, Christopher P. Barnett, Stuart T. Webb, Melanie D. Smith, Tanja Jankovic-Karasoulos, Gustaaf A. Dekker, Claire T. Roberts
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="During tumor development, promoter CpG islands that are normally silenced by Polycomb repressive complexes (PRCs) become DNA-hypermethylated. The molecular mechanism by which de novo DNA methyltransferase(s) [DNMT(s)] catalyze CpG methylation at PRC-regulated regions remains unclear. Here, we report a cryo–electron microscopy structure of the DNMT3A long isoform (DNMT3A1) amino-terminal region in complex with a nucleosome carrying PRC1-mediated histone H2A lysine-119 monoubiquitination (H2AK119Ub). We identify regions within the DNMT3A1 amino terminus that bind H2AK119Ub and the nucleosome acidic patch. This bidentate interaction is required for effective DNMT3A1 engagement with H2AK119Ub-modified chromatin in cells. Further, aberrant redistribution of DNMT3A1 to Polycomb target genes recapitulates the cancer-associated DNA hypermethylation signature and inhibits their transcriptional activation during cell differentiation. This effect is rescued by disruption of the DNMT3A1–acidic patch interaction. Together, our analyses reveal a binding interface critical for mediating promoter CpG island DNA hypermethylation, a major molecular hallmark of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f64e00b184c0e797697cd862ef32ddd61be317" target='_blank'>
              Cancer-associated DNA hypermethylation of Polycomb targets requires DNMT3A dual recognition of histone H2AK119 ubiquitination and the nucleosome acidic patch
              </a>
            </td>
          <td>
            Kristjan H. Gretarsson, S. Abini-Agbomson, Susan L Gloor, Daniel N. Weinberg, Jamie L McCuiston, V. S. Kumary, Allison Hickman, Varun Sahu, Rachel Lee, Xinjing Xu, Natalie Lipieta, Samuel Flashner, Oluwatobi A Adeleke, Irina K Popova, Hailey F Taylor, Kelsey Noll, C. L. Windham, Danielle N Maryanski, Bryan J Venters, Hiroshi Nakagawa, Michael-Christopher Keogh, K. Armache, Chao Lu
          </td>
          <td>2024-08-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neurons are central to lifelong learning and memory, but ageing disrupts their morphology and function, leading to cognitive decline. Although epigenetic mechanisms are known to play crucial roles in learning and memory, neuron-specific genome-wide epigenetic maps into old age remain scarce, often being limited to whole-brain homogenates and confounded by glial cells. Here, we mapped H3K4me3, H3K27ac, and H3K27me3 in mouse neurons across their lifespan. This revealed stable H3K4me3 and global losses of H3K27ac and H3K27me3 into old age. We observed patterns of synaptic function gene deactivation, regulated through the loss of the active mark H3K27ac, but not H3K4me3. Alongside this, embryonic development loci lost repressive H3K27me3 in old age. This suggests a loss of a highly refined neuronal cellular identity linked to global chromatin reconfiguration. Collectively, these findings indicate a key role for epigenetic regulation in neurons that is inextricably linked with ageing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abeefbff913f217ee52fd1f56e3ec31022929408" target='_blank'>
              Ageing-Related Changes to H3K4me3, H3K27ac, and H3K27me3 in Purified Mouse Neurons
              </a>
            </td>
          <td>
            Brandon Signal, Andrew J. Phipps, Katherine A Giles, Shannon N. Huskins, Timothy R Mercer, Mark D. Robinson, Adele Woodhouse, P. Taberlay
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ubiquitin modification and alternative polyadenylation play crucial roles in the onset and progression of cancer. Hence, this study aims to comprehensively and deeply understand gene regulation and associated biological processes in lung adenocarcinoma (LUAD) by integrating both mechanisms. Alternative polyadenylation (APA)-related E3 ubiquitin ligases in LUAD were identified through multiple databases, and the association between selected genetic loci influencing gene expression (apaQTL-SNPs) and LUAD risk were evaluated through the GWAS database of the Female Lung Cancer Consortium in Asia (FLCCA). Subsequently, the interaction between RNF213 and ZBTB20, as well as their functional mechanisms in LUAD, were investigated using bioinformatics analysis, Western blot, co-immunoprecipitation, and colony formation experiments. A total of five apaQTL-SNPs (rs41301932, rs4494603, rs9890400, rs56066320, and rs41301932), located on RNF213, were significantly associated with LUAD risk (p < 0.05), and they inhibit tumor growth through ubiquitin-mediated degradation of ZBTB20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c9426c96fac5af729404fd06cbe66dceed7c43" target='_blank'>
              Crosstalk among Alternative Polyadenylation, Genetic Variants and Ubiquitin Modification Contribute to Lung Adenocarcinoma Risk
              </a>
            </td>
          <td>
            Yutong Wu, Yanqiong Yuan, Huiwen Xu, Wendi Zhang, Anhui Ning, Siqi Li, Qiong Chen, Xiaobo Tao, Gongbu Pan, Tian Tian, Lei Zhang, Minjie Chu, Jiahua Cui
          </td>
          <td>2024-07-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In mammals, tail length is controlled by several genetic determinants, amongst which Hox13 genes located at the posterior extremities of Hox clusters, whose main function are to terminate the extension of the body axis. In this view, the precise timing in the transcriptional activation of these genes may impact upon body length. Unlike other Hox clusters, HoxB lacks all posterior genes between Hoxb9 and Hoxb13, two genes separated by a ca. 70 kb large DNA segment containing an unusually high number of CTCF sites, suggesting it isolates Hoxb13 from the rest of the cluster, thereby delaying its negative impact on trunk extension. We deleted the spacer DNA to induce a potential heterochronic gain of function of Hoxb13 at physiological concentration and observed a shortening of the tail as well as other abnormal phenotypes, which were all rescued by inactivating Hoxb13 in-cis with the deletion. A comparable gain of function was observed in mutant ES cells grown as pseudo-embryos in vitro, which allowed us to examine in details the importance of both the number and the orientation of CTCF sites in the insulating activity of the DNA spacer. A short cassette containing all the CTCF sites was sufficient to insulate Hoxb13 from the rest of HoxB and additional modifications of this CTCF cassette showed that two CTCF sites in convergent orientations are already capable of importantly delaying Hoxb13 activation in these conditions. We discuss the relative importance of genomic distance versus number and orientation of CTCF sites in preventing Hoxb13 to be activated too early during trunk extension and hence to modulate tail length.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bc27ccdd6c53e8dcd3d37c1bc58024dfc056eb7" target='_blank'>
              CTCF-DEPENDENT INSULATION OF Hoxb13 AND THE HETEROCHRONIC CONTROL OF TAIL LENGTH
              </a>
            </td>
          <td>
            Lucille Lopez-Delisle, J. Zákány, Célia Bochaton, P. Osteil, Alexandre Mayran, Fabrice Darbellay, Bénédicte Mascrez, Hocine Rekaik, Denis Duboule
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="RNA-binding proteins (RBPs) play critical cellular roles by mediating various stages of RNA life cycles. Ssd1, an RBP with pleiotropic effects, has been implicated in aneuploidy tolerance in Saccharomyces cerevisiae but its mechanistic role remains unclear. Here we used a network-based approach to inform on Ssd1’s role in aneuploidy tolerance, by identifying and experimentally perturbing a network of RBPs that share mRNA targets with Ssd1. We identified RBPs whose bound mRNA targets significantly overlap with Ssd1 targets. For 14 identified RBPs, we then used a genetic approach to generate all combinations of genotypes for euploid and aneuploid yeast with an extra copy of chromosome XII, with and without SSD1 and/or the RBP of interest. Deletion of 10 RBPs either exacerbated or alleviated the sensitivity of wild-type and/or ssd1Δ cells to chromosome XII duplication, in several cases indicating genetic interactions with SSD1 in the context of aneuploidy. We integrated these findings with results from a global over-expression screen that identified genes whose duplication complements ssd1Δ aneuploid sensitivity. The resulting network points to a sub-group of proteins with shared roles in translational repression and p-body formation, implicating these functions in aneuploidy tolerance. Our results reveal a role for new RBPs in aneuploidy tolerance and support a model in which Ssd1 mitigates translation-related stresses in aneuploid cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5529e825065bf2340ee2979d76bed7082bde2d" target='_blank'>
              Investigating the role of RNA-binding protein Ssd1 in aneuploidy tolerance through network analysis
              </a>
            </td>
          <td>
            H. A. Dutcher, A. Gasch
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628089f06d9782813abf7a0fd0e702990c7e28b0" target='_blank'>
              Chemoproteomic profiling unveils binding and functional diversity of endogenous proteins that interact with endogenous triplex DNA.
              </a>
            </td>
          <td>
            Hongzhan Xu, Jing Ye, Kui-Xing Zhang, Qingxi Hu, Tongxiao Cui, Chong Tong, Mengqi Wang, Huichao Geng, Kun-Ming Shui, Yan Sun, Jian Wang, Xiaomeng Hou, Kai Zhang, Ran Xie, Yafei Yin, Nan Chen, Jia-Yu Chen
          </td>
          <td>2024-09-02</td>
          <td>Nature chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1230bf0b0e23a37cb0ef069af27c70638d7b9578" target='_blank'>
              Cell-free DNA from germline TP53 mutation carriers reflect cancer-like fragmentation patterns
              </a>
            </td>
          <td>
            Derek Wong, Maha Tageldein, Ping Luo, Erik Ensminger, Jeffrey P. Bruce, L. Oldfield, Haifan Gong, N. Fischer, B. Laverty, Vallijah Subasri, Scott Davidson, Reem Khan, A. Villani, A. Shlien, R. Kim, David Malkin, Trevor J. Pugh
          </td>
          <td>2024-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cbd49dc4f182de3dedf2bd9170f18d407483728" target='_blank'>
              VolcanoSV enables accurate and robust structural variant calling in diploid genomes from single-molecule long read sequencing
              </a>
            </td>
          <td>
            Can Luo, Y. Liu, X. Zhou
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mitochondrial DNA (mtDNA) stability, essential for cellular energy production, relies on DNA polymerase gamma (POLγ). Here, we show that the POLγ Y951N disease causing mutation induces replication stalling and severe mtDNA depletion. However, unlike other POLγ disease causing mutations, Y951N does not directly impair exonuclease activity and only mildly affects polymerase activity. Instead, we found that Y951N compromises the enzyme’s ability to efficiently toggle between DNA synthesis and degradation, and is thus the first patient-derived mutation with impaired polymerase-exonuclease switching. These findings provide new insights into the intramolecular switch when POLγ proofreads the newly-synthesized DNA strand, and reveal a new mechanism for causing mitochondrial DNA instability. Significance Statement DNA polymerase gamma (POLγ) is essential for copying mitochondrial DNA (mtDNA), which is crucial for our energy production. POLγ must accurately switch between making new DNA (polymerase activity) and correcting errors (exonuclease activity). While it is known that mutations in POLγ can cause mitochondrial diseases by directly impairing these enzymatic functions, this study reveals a new mechanism. The Y951N mutation disrupts POLγ’s ability to switch between these activities, leading to severe blockages in DNA replication and a loss of mtDNA in human cells, even without significant direct impairment of polymerase or exonuclease activities. These findings provide new insights into the origins of mitochondrial diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0242e3e1606aa26c741b2c57aa12acc8d793383b" target='_blank'>
              The Y951N patient mutation inactivates the intramolecular switch in human mitochondrial DNA POLγ
              </a>
            </td>
          <td>
            Josefin M. E. Forslund, Tran V.H. Nguyen, V. Parkash, A. Berner, S. Goffart, J. Pohjoismäki, Paulina H. Wanrooij, Erik Johansson, Sjoerd Wanrooij
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The Target of Rapamycin Complex 1 (TORC1) integrates cellular cues and adapts cell growth and metabolism through substrate-specific phosphorylation. A genetic screen for suppressors of a conditional mutant of the cohesin loader Mis4 identified hypomorphic mutants of TORC1. Downregulation of TORC1 enhanced the binding of cohesin and its loader to their regular sites on chromosomes. In the context of impaired cohesin loading, TORC1 downregulation rescued chromosome segregation whereas upregulation had the opposite effect, suggesting environmental cues impinge on the robustness of chromosome segregation. TORC1 co-purified with cohesin from cellular extracts and the phosphorylation level of specific residues on Mis4 and cohesin were reduced in TORC1 mutants. Cohesin mutations mimicking the non-phosphorylated state mirrored the effects of TORC1 downregulation. Challenging cells with various conditions revealed that Mis4 and TORC1 regulate a common set of genes involved in the response to environmental changes. These genes are preferentially located far from centromeres and close to telomeres. We propose that cohesin is an effector of TORC1, orchestrating alterations in chromosome structure that facilitate cellular adaptation to environmental changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a2a1ace914b6006bf8337775d87c88bc0e0de3" target='_blank'>
              TORC1-dependent control of fission yeast cohesin
              </a>
            </td>
          <td>
            Dorian Besson, S. Vaur, Stéphanie Vazquez, S. Tournier, Y. Gachet, Adrien Birot, Stéphane Claverol, A. Marston, Anastasios Damdimopoulos, K. Ekwall, J. Javerzat
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdedb81bfc2b32fe60e68b41b7f434fc5731813" target='_blank'>
              Role of homeobox genes in cancer: immune system interactions, long non-coding RNAs, and tumor progression.
              </a>
            </td>
          <td>
            Saade Abdalkareem Jasim, Shireen Hamid Farhan, Irfan Ahmad, Ahmed Hjazi, Ashwani Kumar, Mohammed Abed Jawad, Atreyi Pramanik, Farag M. A. Altalbawy, Salim B. Alsaadi, M. Abosaoda
          </td>
          <td>2024-09-06</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Here, we present a method for enrichment of double-stranded cfDNA with an average length of ∼40 bp from cfDNA for high-throughput DNA sequencing. This class of cfDNA is enriched at gene promoters and binding sites of transcription factors or structural DNA-binding proteins, so that a genome-wide DNA footprint is directly captured from liquid biopsies. In short double-stranded cfDNA from healthy individuals, we find significant enrichment of 203 transcription factor motifs. Additionally, short double-stranded cfDNA signals at specific genomic regions correlate negatively with DNA methylation, positively with H3K4me3 histone modifications and gene transcription. The diagnostic potential of short double-stranded cell-free DNA (cfDNA) in blood plasma has not yet been recognized. When comparing short double-stranded cfDNA from patient samples of pancreatic ductal adenocarcinoma with colorectal carcinoma or septic with postoperative controls, we identify 136 and 241 differentially enriched loci, respectively. Using these differentially enriched loci, the disease types can be clearly distinguished by principal component analysis, demonstrating the diagnostic potential of short double-stranded cfDNA signals as a new class of biomarkers for liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/003bad913ef23b451e8afe9e333721e6d528290e" target='_blank'>
              A novel approach for in vivo DNA footprinting using short double-stranded cell-free DNA from plasma.
              </a>
            </td>
          <td>
            , Christina Hartwig, Mirko Sonntag, Lisa Bitzer, Christopher Adelmann, Y. Vainshtein, Karolina Glanz, S. Decker, Thorsten Brenner, Georg F. Weber, A. von Haeseler, Kai Sohn
          </td>
          <td>2024-09-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e1918f5013376fd130c9f594868a7f384598c2" target='_blank'>
              HMOX1: A pivotal regulator of prognosis and immune dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Jinfa Huang, Ruiwan Tan
          </td>
          <td>2024-08-30</td>
          <td>BMC Women's Health</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="“Telomere DNA and ribosomal DNA co-regulation model for cell senescence” suggests that cell senescence and Hayflick limits are co-regulated by telomere and rDNA. To this end, by partially knockout 45S rDNA, we found that common aging markers increased and cell passage times decreased, preliminarily proving that cell aging and Hayflick limits are co-regulated by 45S rDNA and telomeres. We also found that telomere length and 45S rDNA copy number of hESC and hiPSC are significantly increased, which may be the reason why hESC and hiPSC remain immortal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e1973b7270ad9e9b64dd522eff7ae60852ce9" target='_blank'>
              Experimental verification on “Telomere DNA and ribosomal DNA co-regulation model for cell senescence”
              </a>
            </td>
          <td>

          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The p53, a pivotal tumor suppressor, regulates various cellular responses, including DNA repair and apoptosis. Normally, p53 levels are low due to murine double minute clone 2 (MDM2) mediated polyubiquitination. However, stress signals disrupt p53-MDM2 interaction, stabilizing p53 and activating target genes. Dysfunctional p53 is common in cancers, especially colorectal cancer (CRC), with TP53 mutations in 43% of tumors. These mutations impair wild-type p53 function or confer novel activities, promoting cancer progression. Despite drugs targeting p53 entering trials, understanding wild-type and mutant p53 functions is crucial for novel CRC therapies. P53 mutations not only impact DNA repair and apoptosis but also play a crucial role in tumor immunotherapy. While rendering tumors resistant to chemotherapy, p53 mutations provide opportunities for immunotherapy due to neoantigen-rich tumors. Additionally, p53 mutations influence tumor microenvironment cells, such as fibroblasts and immunosuppressive cells, through p53-mediated signaling pathways. Investigating p53 mutations in tumor therapy is vital for personalized medicine and immunotherapy. In cancer treatment research, scientists explore drugs and strategies to restore or enhance p53 function. Targeting wild-type p53 aims to restore DNA repair and cell cycle control, while targeting mutant p53 seeks new drugs to inhibit its detrimental effects, advancing tumor treatment. Understanding p53 drugs and strategies is crucial for cancer therapy progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4492a2019f4f8e179322826eb5b657b70923676" target='_blank'>
              Unraveling the Role of TP53 in Colorectal Cancer Therapy: From Wild-Type Regulation to Mutant.
              </a>
            </td>
          <td>
            Wenshu Li, Longyuan Li, Huan Yang, Chenxi Shi, Zhe Lei, Lingchuan Guo, Yuhong Wang
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Humans are exposed to a variety of environmental toxicants which have been implicated in contributing to cancer development. A possible explanation for this link is that adaptive epigenetic changes occur in stem cell populations upon toxicant exposure. These cells are particularly vulnerable during developmental stages, leading to abnormal gene expression later in life. Environmental exposures have been associated with the initiation of sarcoma development. This study will focus on Ewing Sarcoma (ES), as this malignancy has a low mutational burden relative to other cancer types and is thought to be linked to agricultural chemicals. We are also focusing on stromal antigen 2 (STAG2), which is found to be frequently mutated in ES in addition to multiple other tumor types. Patients with STAG2-mutated ES have higher rates of metastatic disease and worse outcomes, but the reason for this remains poorly understood. STAG2 plays a role in the regulation of genomic organization, DNA repair, and modulation of replication stress (RS). It has also been shown that STAG2 loss may lead to dysregulation of the cGAS-STING pathway, which is critical for tumor immunosurveillance. We are testing the effects of environmental toxicants in mesenchymal stem cells (MSCs), the putative cell of origin for ES and many other sarcomas. We have successfully used CRISPR/Cas9 to generate STAG2 knockout MSCs. With this model, we are testing the effects of STAG2 loss and toxicant exposure on levels of RS, epigenomic instability, and innate immune suppression in MSCs. Our results indicate that these toxicants alter the cell cycle dynamics of MSCs, and that these alterations change depending on STAG2 status. We have also observed changes in the DNA damage response of STAG2-mutant MSCs when exposed to environmental toxicants. Interestingly, effects were seen even at sub-lethal doses that may reflect common, low-level exposures. We believe that our data will contribute to the understanding of the mechanism by which environmental toxicants facilitate malignant transformation of Ewing sarcoma and other pediatric sarcomas.
 Citation Format: Rachael Kohrn, Joyce Ohm. Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed252aa2969d787536854f4f2e71a66da32b809" target='_blank'>
              Abstract A029 Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis
              </a>
            </td>
          <td>
            Rachael Kohrn, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ancient environmental DNA (aeDNA) is now commonly used in paleoecology and evolutionary ecology, yet due to difficulties in gaining sufficient genome coverage on individual species from metagenome data, its genetic perspectives remain largely uninvestigated. Hybridization capture has proven as an effective approach for enriching the DNA of target species, thus increasing the genome coverage of sequencing data and enabling population and evolutionary genetics analysis. However, to date there is no tool available for designing capture probe sets tailored for aeDNA based population genetics. Here we present eProbe, an efficient, flexible and easy-to-use program toolkit that provides a complete workflow for capture probe design, assessment and validation. By benchmarking a probe set for foxtail millet, an annual grass, made by the eProbe workflow, we demonstrate a remarkable increase of capturing efficiency, with the target taxa recovery rate improved by 577-fold, and the genome coverage achieved by soil capture-sequencing data even higher than data directly shotgun sequenced from the plant tissues. Probes that underwent our filtering panels show notably higher efficiency. The capture sequencing data enabled accurate population and evolutionary genetic analysis, by effectively inferring the fine-scale genetic structures and patterns, as well as the genotypes on functional genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f415dd02d07f1d188d80e4a5c9368c04a940209a" target='_blank'>
              eProbe: a capture probe design toolkit for genetic diversity reconstructions from ancient environmental DNA
              </a>
            </td>
          <td>
            Zihao Huang, Zhengquan Gu, Yuanyang Cai, Ruairidh Macleod, Zhe Xue, Haoran Dong, Søren Overballe-Petersen, Shanlin Liu, Yu Gao, Hao Li, Sha Tang, Xianmin Diao, Morten Egevang Joergensen, Christoph Dockter, L. Vinner, E. Willerslev, Fahu Chen, Hongru Wang, Yucheng Wang
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In developing B cells, V(D)J gene recombination is initiated by the RAG1/2 endonuclease complex, introducing double-stranded DNA breaks (DSBs) in V, D, and J genes and resulting in the formation of the hypervariable parts of immunoglobulins (Ig). Persistent or aberrant RAG1/2 targeting is a potential threat to genome integrity. While RAG1 and RAG2 have been shown to bind various regions genome-wide, the in vivo off-target DNA damage instigated by RAG1/2 endonuclease remains less well understood. In the current study, we identified regions containing RAG1/2-induced DNA breaks in mouse pre-B cells on a genome-wide scale using a global DNA DSB detection strategy. We detected 1489 putative RAG1/2-dependent DSBs, most of which were located outside the Ig loci. DNA sequence motif analysis showed a specific enrichment of RAG1/2-induced DNA DSBs at GA- and CA-repeats and GC-rich motifs. These findings provide further insights into RAG1/2 off-target activity. The ability of RAG1/2 to introduce DSBs on the non-Ig loci during the endogenous V(D)J recombination emphasizes its genotoxic potential in developing lymphocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c3ee9eea0d95b181f8a61de27812d8926453b6" target='_blank'>
              RAG1/2 induces double-stranded DNA breaks at non-Ig loci in the proximity of single sequence repeats in developing B cells.
              </a>
            </td>
          <td>
            Katarina Ochodnicka-Mackovicova, Michal Mokry, Martin Haagmans, Ted E. Bradley, C. V. van Noesel, J. Guikema
          </td>
          <td>2024-07-24</td>
          <td>European journal of immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal fusions play an integral role in genome remodeling and karyotype evolution. Fusions that join a sex chromosome to an autosome are particularly abundant across the tree of life. However, previous models on the establishment of such fusions have not accounted for the physical structure of the chromosomes. We predict a fusion joining an autosome to the pseudoautosomal region (PAR) of a sex chromosome will not remain stable, and the fusion will switch from the X to the Y chromosome each generation due to recombination. We have produced a forward-time population genetic simulation to explore the outcomes of fusions to both the PAR and non-PAR of sex chromosomes. The model can simulate the fusion of an autosome containing a sexually antagonistic locus to either the PAR or non-PAR end of a sex chromosome. Our model is diploid, two-locus, and biallelic. Our results show a clear pattern where fusions to the non-PAR are favored in the presence of sexual antagonism, whereas fusions to the PAR are disfavored in the presence of sexual antagonism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417dce982562c863a7e38c28a474ba15591636ef" target='_blank'>
              Worse than nothing at all: the inequality of fusions joining autosomes to the PAR and non-PAR portions of sex chromosomes
              </a>
            </td>
          <td>
            Kayla T. Wilhoit, Emmarie P. Alexander, Heath Blackmon
          </td>
          <td>2024-07-23</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The tight control of the mitotic phase of differentiation is crucial to prevent tumourigenesis while securing tissue homeostasis. In the Drosophila female germline, differentiation involves precisely four mitotic divisions, and accumulating evidence suggests that bag-of- marbles (bam), the initiator of differentiation, is also involved in controlling the number of divisions. To test this hypothesis, we depleted Bam from differentiating cells and found a reduced number of mitotic divisions. We examined the regulation of Bam using RNA imaging methods and found that the bam 3’ UTR conveys instability to the transcript in the 8-cell cyst and early 16-cell cyst. We show that the RNA binding protein, Rbp9, is responsible for timing bam mRNA decay. Rbp9 itself is part of a sequential cascade of RNA binding proteins activated downstream of Bam, and we show that it is regulated through a change in transcription start site, driven by Rbfox1. Altogether, we propose a model in which Bam expression at the dawn of differentiation initiates a series of events that eventually terminates the Bam expression domain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ac40f2eb322caeece920877298412a69c4a5a9" target='_blank'>
              Destabilisation of bam transcripts terminates the mitotic phase of Drosophila female germline differentiation
              </a>
            </td>
          <td>
            Tamsin J. Samuels, Elizabeth J Torley, Emily L Naden, Phoebe E Blair, Frankjel A Hernandez Frometa, Felipe Karam Teixeira
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Like many pediatric cancers, Ewing sarcoma (EwS) is a genomically quiet disease. Driven by an oncogenic fusion protein, usually EWSR1-FLI1, the epigenome of this genetically homogenous cancer is a driver of clinical outcomes, where heightened epigenetic heterogeneity within tumors is associated with metastatic disease. While prognosis for localized disease is moderate, outcomes are dismal for patients who present with advanced, relapsed, or refractory disease and there is a lack of targeted therapies in this setting. We have previously described a targeted combination therapy harnessing DDK and WEE1 inhibition exploits elevated endogenous replication stress in EwS to force cells into mitotic catastrophe in vitro. While our experiments evaluating the efficacy of this combination in vivo have been extremely encouraging, the data have shown that individual mice receiving the same treatment have variable patterns of outgrowth after treatment cessation, with some mice demonstrating moderate increases in tumor volume and some mice demonstrating little to no tumor outgrowth. This differential response to therapy is most likely to occur through epigenetic dysregulation. Thus, we aim to interrogate intertumoral epigenetic heterogeneity and examine epigenetic profiles associated with treatment response using samples from in vivo studies. Samples were analyzed from a CDX model using TC32 cells injected subcutaneously. Reduced representation bisulfite sequencing was performed on a control arm (n = 5) and the following three treatment arms, where the DDK inhibitor is TAK-931 (simurosertib) and the WEE1 inhibitor is MK1775 (adavosertib): 1) Irinotecan + Temozolomide (n=9), 2) Irinotecan (full dose) + DDKi + WEE1i (n=7), 3) Irinotecan (half dose) + DDKi + WEE1i (n=9), for a total of 30 samples. Our analysis of this data evaluates epigenetic changes between treatment conditions and increases our understanding of epigenetic heterogeneity in EwS.
 Citation Format: Emily Isenhart, Ajay Gupta, Joyce Ohm. Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366bd5a5e10d37a86e17654dd6f774d1f8bf9952" target='_blank'>
              Abstract A056 Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma
              </a>
            </td>
          <td>
            E. Isenhart, Ajay Gupta, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The p53 genome is a gene that has truly taken the scientific world by storm. Once referred to as an oncogene, p53 has proved itself to be more complex and dubious time and time again. This paper will talk about the beginning doubts and queries revolving around p53 and the discoveries that led to it being referred to as a tumour suppressor gene, rather than being mislabeled as an oncogene. It will touch upon the importance that p53 brings to our daily lives such as repairing the DNA, maintaining the cell cycle and starting apoptosis, as well as highlight the differences between the gene’s two types: wild-type p53 and mutant p53. Furthermore, it will talk about how a slight mutation in p53, can cause it to form many protein isoforms, a leading contributing factor in cancers. It will also mention the effects post-translational modification will have on the two types of p53. In addition to this, it will delve into the negative and potentially harmful effects that p53 could have on our bodies. Finally, it will also talk about p53-mediated therapy and targeting mutant p53 with autophagy, which are 2 processes that have shown successful outcomes in reducing the complications caused by mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc70ab96424bdf601f1a4ba7f9b8f2ff8a5b550" target='_blank'>
              P53: Our Saviour and Our Destroyer
              </a>
            </td>
          <td>
            Narayani Haldar
          </td>
          <td>2024-07-23</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The emergence of cancers involves numerous coding and non-coding genes. Understanding the contribution of non-coding RNAs (ncRNAs) to the cancer neighborhood is crucial for interpreting the interaction between molecular markers of cancer. However, there is a lack of systematic studies on the involvement of ncRNAs in the cancer neighborhood. In this paper, we construct an interaction network which encompasses multiple genes. We focus on the fundamental topological indicator, namely connectivity, and evaluate its performance when applied to cancer-affected genes using statistical indices. Our findings reveal that ncRNAs significantly enhance the connectivity of affected genes and mediate the inclusion of more genes in the cancer module. To further explore the role of ncRNAs in the network, we propose a connectivity-based method which leverages the bridging function of ncRNAs across cancer-affected genes and reveals the non-coding RNAs extended omnigenic module (NeOModule). Topologically, this module promotes the formation of cancer patterns involving ncRNAs. Biologically, it is enriched with cancer pathways and treatment targets, providing valuable insights into disease relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c53c94851580faedc72696e1e649d6cf0d594b0" target='_blank'>
              Non-Coding RNAs Extended Omnigenic Module of Cancers
              </a>
            </td>
          <td>
            Jie Li, Bingbo Wang, Xiujuan Ma
          </td>
          <td>2024-07-27</td>
          <td>Entropy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Accurate haplotyping facilitates distinguishing allele-specific expression, identifying cis-regulatory elements, and characterizing genomic variations, which enables more precise investigations into the relationship between genotype and phenotype. Recent advances in third-generation single-molecule long read and synthetic co-barcoded read sequencing techniques have harnessed long-range information to simplify the assembly graph and improve assembly genomic sequence. However, it remains methodologically challenging to reconstruct the complete haplotypes due to high sequencing error rates of long reads and limited capturing efficiency of co-barcoded reads. We here present a pipeline, AsmMix, for generating both contiguous and accurate diploid genomes. It first assembles co-barcoded reads to generate accurate haplotype-resolved assemblies that may contain many gaps, while the long-read assembly is contiguous but susceptible to errors. Then two assembly sets are integrated into haplotype-resolved assemblies with reduced misassembles. Through extensive evaluation on multiple synthetic datasets, AsmMix consistently demonstrates high precision and recall rates for haplotyping across diverse sequencing platforms, coverage depths, read lengths, and read accuracies, significantly outperforming other existing tools in the field. Furthermore, we validate the effectiveness of our pipeline using a human whole genome dataset (HG002), and produce highly contiguous, accurate, and haplotype-resolved assemblies. These assemblies are evaluated using the GIAB benchmarks, confirming the accuracy of variant calling. Our results demonstrate that AsmMix offers a straightforward yet highly efficient approach that effectively leverages both long reads and co-barcoded reads for haplotype-resolved assembly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d167bf2512bd5bffeac2ae83dc689ad8663f8a0f" target='_blank'>
              AsmMix: an efficient haplotype-resolved hybrid de novo genome assembling pipeline
              </a>
            </td>
          <td>
            Chao Liu, Pei Wu, Xue Wu, Xia Zhao, Fang Chen, Xiaofang Cheng, Hongmei Zhu, Ou Wang, Mengyang Xu
          </td>
          <td>2024-07-26</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The deoxyribonucleic acid (DNA) carries hereditary codes which is translated by the cells to synthesize the ribonucleic acid (RNA) and polypeptides which can generate and perform vital function. The double helix structure is the most studied model of the DNA that was proposed by Watson and Crick. The capability of DNA to work as a genetic material can be stored and conducted during cell division to permit this information to be doubled and transmitted to the incoming generation. Any damage in the structure of DNA is an essential direct cause for the progression of cancer and other disorders. The factors for DNA damage can be classified as exogenous and endogenous factors. In this review article, we highlight the evidence-supported information about the structure, functions and clinical significance of DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ff4de2b8e4130d71de29efe87be0afb81a98a9" target='_blank'>
              Structure, Functions And Clinical Significance Of DNA: A Review Article
              </a>
            </td>
          <td>
            Ahmed M. Amshawee, Maryam A. Hussain, Seenaa Taqi Mansour Al Muhtaser, Ali A. Al- Faham
          </td>
          <td>2024-07-22</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA methylation is well-established as a major epigenetic mechanism that can control gene expression and is involved in both normal development and disease. Analysis of high-throughput-sequencing-based DNA methylation data is a step toward understanding the relationship between disease and phenotype. Analysis of CpG methylation at single-base resolution is routinely done by bisulfite sequencing, in which methylated Cs remain as C while unmethylated Cs are converted to U, subsequently seen as T nucleotides. Sequence reads are aligned to the reference genome using mapping tools that accept the C-T ambiguity. Then, various statistical packages are used to identify differences in methylation between (groups of) samples. We have previously developed the Differential Methylation Analysis Pipeline (DMAP) as an efficient, fast, and flexible tool for this work, both for whole-genome bisulfite sequencing (WGBS) and reduced-representation bisulfite sequencing (RRBS). The protocol described here includes a series of scripts that simplify the use of DMAP tools and that can accommodate the wider range of input formats now in use to perform analysis of whole-genome-scale DNA methylation sequencing data in various biological and clinical contexts. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol: DMAP2 workflow for whole-genome bisulfite sequencing (WGBS) and reduced-representation bisulfite sequencing (RRBS).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13fb6f53fc80d13d019e959851a23b4cd1b21" target='_blank'>
              DMAP2: A Pipeline for Analysis of Whole-Genome-Scale DNA Methylation Sequencing Data.
              </a>
            </td>
          <td>
            P. Stockwell, Euan J. Rodger, Gregory Gimenez, I. Morison, Aniruddha Chatterjee
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0c6be6efa212fbb68132ac9fcf2e7d1b9412f9" target='_blank'>
              Sumo-regulatory SENP2 controls the homeostatic squamous mitosis-differentiation checkpoint
              </a>
            </td>
          <td>
            Jesús Galán-Vidal, Lorena García-Gaipo, R. Molinuevo, Samantha Dias, Alex Tsoi, Javier Gómez-Román, James T. Elder, H. Hochegger, Alberto Gandarillas
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823edc7b7f2e48466de67d70b29a3e25297b1b88" target='_blank'>
              Cooperative genomic lesions in HRAS-mutant cancers predict resistance to farnesyltransferase inhibitors.
              </a>
            </td>
          <td>
            Arad Nigam, Gnana P. Krishnamoorthy, Walid K. Chatila, Katherine Berman, Mahesh Saqcena, H. Walch, Mandakini Venkatramani, Alan L. Ho, N. Schultz, James A. Fagin, B. Untch
          </td>
          <td>2024-08-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Bacterial DNA methylation is involved in diverse cellular functions ranging from modulation of gene expression, DNA repair, and restriction-modification systems for defense against viruses and other foreign DNA. Methylome analysis determines sites within a bacterial chromosome that are methylated, revealing motifs that may be targeted by native restriction enzymes. Identification of these motifs is therefore critical to making an organism genetically tractable, where mimicking the methylome patterns in Escherichia coli allows plasmid DNA to be protected from restriction in the target organism and can therefore drastically enhance transformation efficiency. Oxford Nanopore Technologies (ONT) sequencing can detect methylated bases during sequencing, but software is needed to identify the corresponding methylated motifs within the data. Here, we develop MIJAMP (MIJAMP Is Just A MethylBED Parser), a software package that was developed to discover methylated motifs from the output of ONT’s Modkit or other data in the methylBED format. MIJAMP employs a human-driven refinement strategy that empirically validates all motifs against genome-wide methylation data, thus eliminating false, under-, or overexplained motifs. MIJAMP can also report methylation data on a specific, user-defined motif. Using MIJAMP, we determined the methylated motifs both in a control strain (wild type E. coli) and in Picosynecococcus sp. strain PCC7002, laying the foundation for improved transformation in this organism. MIJAMP is available at https://code.ornl.gov/5g6/mijamp/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560305afe7d74c24c06221b94759147f1622ed98" target='_blank'>
              Discovering methylated DNA motifs in bacterial nanopore sequencing data with MIJAMP
              </a>
            </td>
          <td>
            Alyssa K. Tidwell, Evelyn Faust, Carrie A. Eckert, A. Guss, William G. Alexander
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Li-Fraumeni Syndrome (LFS) is a genetic predisposition disorder associated with pathogenic germline Tp53 mutations. A subset of LFS-associated neoplasms occurs at an increased risk within the first 5 years of life, with the most common being soft tissue sarcomas (STS). Histological characterization of STS is one of the primary approaches to guiding treatment. However, the heterogeneity of STS has necessitated a further understanding of the molecular etiology of these tumors. Examining sarcomas using a single-cell transcriptomic approach has shown unique subsets of cells with dysregulations in cell fate and differentiation. Within LFS-associated tumors, including LFS-STS a major event in tumorigenesis is the copy-neutral loss of heterozygosity (LOH). LOH was determined to occur much earlier than cancer onset in LFS, suggesting a dormancy period from the LOH until tumorigenesis; cancer-defining events in LFS are likely occurring prenatally. During embryogenesis, Tp53 is involved in differentiation and pluripotency regulation, as well as cell cycle control and apoptosis. Here we hypothesize that the emergence of STS in LFS mice is associated with distinct disturbances in the cell fate and differentiation pathways. Specifically, there will be deficits in cell fate determination and the organization of cells across multiple cell types in LFS-Sarcomas. Single-cell RNAseq was performed on spontaneous tumors and healthy matched muscle tissue from male and female C57BL/6J Trp53R172H/WT mice. Cell types were determined based on differential gene expression between clusters defined by principal component analysis. Despite tumor heterogeneity between mice, there were consistent disruptions of developmental pathways among tumor cells. Moreover, when comparing matched cell types across tumor and healthy muscle tissue we observed dysregulation of pathways promoting tumorigenesis, and cell fate determination in cells originating from tumor tissue. For the first time to our knowledge, this study examines the single-cell transcriptome of LFS-STS. Current literature has speculated on the presence of an early precancer state of LFS-associated tumors. This study will provide evidence to determine whether this precancer state can be determined in tumor samples, and identify elements that constitute this state. Moreover, we will determine changes in supporting non-tumor cells and whether they also exhibit dysregulations in cell state and differentiation. Together these findings will hopefully contribute to improvements in molecular subtyping across STS, and particularly within LFS individuals.
 Citation Format: Ashby Kissoondoyal, David Malkin. Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc62fb2bdb474c15b956e6aac5478623a2fc2613" target='_blank'>
              Abstract A009 Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 A hallmark of high-risk childhood medulloblastoma is the dysregulation of RNA translation.Dysregulation of ribosome activity not only influences known protein-coding genes, but also unannotated small open reading frames (sORFs) as well.We have recently shown that sORFs may produce functional microproteins in medulloblastoma.Here, we performed an integrative analysis of medulloblastoma patient samples and cell lines.We used ribosome profiling, RNAseq, and mass spectrometry to delineate dysregulation of microprotein production.We employed cell models of MYC overexpression to determine the role of MYC in selective translation of microproteins in medulloblastoma, finding specific associations with upstream open reading frames (uORFs) located in gene leader sequences.We then designed a step-wise approach to define functional microproteins in medulloblastoma through the use of CRISPR/Cas9 screens, saturation mutagenesis, and base editing.We observed distinctive signatures of microprotein bioactivity based on the molecular drivers of medulloblastoma according to disease subtypes.In the Group 3 subtype, we observed an enrichment for bioactive uORFs relative to other disease settings.We then characterized several key microprotein dependencies resulting from uORFs including a molecular mechanism for one candidate, ASNSD1-uORF, in mediating protein-RNA dyssynchrony in Group 3 medulloblastoma.Protein-RNA dyssynchrony is the divergent regulation of protein abundance from RNA abundance, and is a hallmark of Group 3 medulloblastoma.We probed the clinical and therapeutic implications of protein-RNA dyssynchrony and determined that this process may be associated with patterns of drug sensitivity and resistance in medulloblastoma.With this insight, we have now used ASNSD1-uORF to stratify medulloblastoma resistance patterns to inhibitors of RNA transcription.We determine that an interplay between RNA translational control and RNA transcriptional control, while driving resistance in some settings, may also reveal additional drug targets, such as cyclin-dependent kinases.Our findings underscore the fundamental importance of sORF translation in medulloblastoma and provide a rationale to include these ORFs in future studies seeking to define new cancer targets and therapeutic insights.
 Citation Format: John Prensner, Pratiti Bandopadhayay, Soumik Saha, Leslie Lupien, Seung H. Choi. Aberrant translation of microproteins as a source of new cancer dependencies in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84aa798346bd46032e7d3bf7e028fb5657ce73c4" target='_blank'>
              Abstract B017: Aberrant translation of microproteins as a source of new cancer dependencies in medulloblastoma
              </a>
            </td>
          <td>
            John R. Prensner, P. Bandopadhayay, Soumik Saha, Leslie E. Lupien, S. H. Choi
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Current microbial sequencing relies on short-read platforms like Illumina and DNBSEQ, favored for their low cost and high accuracy. However, these methods often produce fragmented draft genomes, hindering comprehensive bacterial function analysis. CycloneSEQ, a novel long-read sequencing platform developed by BGI-Research, its sequencing performance and assembly improvements has been evaluated. Findings Using CycloneSEQ long-read sequencing, the type strain produced long reads with an average length of 11.6 kbp and an average quality score of 14.4. After hybrid assembly with short reads data, the assembled genome exhibited an error rate of only 0.04 mismatches and 0.08 indels per 100 kbp compared to the reference genome. This method was validated across 9 diverse species, successfully assembling complete circular genomes. Hybrid assembly significantly enhances genome completeness by using long reads to fill gaps and accurately assemble multi-copy rRNA genes, which unable be achieved by short reads solely. Through data subsampling, we found that over 500 Mbp of short-read data combined with 100 Mbp of long-read data can result in a high-quality circular assembly. Additionally, using CycloneSEQ long reads effectively improves the assembly of circular complete genomes from mixed microbial communities. Conclusions CycloneSEQ’s read length is sufficient for circular bacterial genomes, but its base quality needs improvement. Integrating DNBSEQ short reads improved accuracy, resulting in complete and accurate assemblies. This efficient approach can be widely applied in microbial sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6673f8eadf6fe9dde2281d013056b4e16e52fe6" target='_blank'>
              Efficiently Constructing Complete Genomes with CycloneSEQ to Fill Gaps in Bacterial Draft Assemblies
              </a>
            </td>
          <td>
            Hewei Liang, Mengmeng Wang, Tongyuan Hu, Haoyu Wang, Wenxin He, Yanmei Ju, Ruijin Guo, Junyi Chen, Fei Guo, Tao Zeng, Yuliang Dong, Bo Wang, Chuanyu Liu, Xin Jin, Wenwei Zhang, Yuanqiang Zou, Xun Xu, Liang Xiao
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromosomal abnormalities are a common cause of human miscarriage but rarely reported in any other species. As a result, there are currently inadequate animal models available to study this condition. Horses present one potential model since mares receive intense gynecological care. This allowed us to investigate the prevalence of chromosomal copy number aberrations in 256 products of conception (POC) in a naturally occurring model of pregnancy loss (PL). Triploidy (three haploid sets of chromosomes) was the most common aberration, found in 42% of POCs following PL over the embryonic period. Over the same period, trisomies and monosomies were identified in 11.6% of POCs and subchromosomal aberrations in 4.2%. Whole and subchromosomal aberrations involved 17 autosomes, with chromosomes 3, 4, and 20 having the highest number of aberrations. Triploid fetuses had clear gross developmental anomalies of the brain. Collectively, data demonstrate that alterations in chromosome number contribute to PL similarly in women and mares, with triploidy the dominant ploidy type over the key period of organogenesis. These findings, along with highly conserved synteny between human and horse chromosomes, similar gestation lengths, and the shared single greatest risk for PL being advancing maternal age, provide strong evidence for the first animal model to truly recapitulate many key features of human miscarriage arising due to chromosomal aberrations, with shared benefits for humans and equids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b004c64eb472971d4f0ded1c47b545042aa9fc" target='_blank'>
              Naturally occurring horse model of miscarriage reveals temporal relationship between chromosomal aberration type and point of lethality.
              </a>
            </td>
          <td>
            Jessica M Lawson, S. E. Salem, Donald Miller, A. Kahler, Wilhelmina J van den Boer, Charlotte A Shilton, Tia Sever, R. Mouncey, Jenna Ward, Daniel J Hampshire, Alastair K Foote, Jill S Bryan, R. Juras, Oliver D Pynn, Brian W. Davis, R. R. Bellone, T. Raudsepp, A. D. de Mestre
          </td>
          <td>2024-08-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Aligning DNA sequences retrieved from fossils or other paleontological artifacts, referred to as ancient DNA, is particularly challenging due to the short sequence length and chemical damage which creates a specific pattern of substitution (C→T and G→A) in addition to the heightened divergence between the sample and the reference genome thus exacerbating reference bias. This bias can be mitigated by aligning to pangenome graphs to incorporate documented organismic variation, but this approach still suffers from substitution patterns due to chemical damage. We introduce a novel methodology introducing the RYmer index, a variant of the commonly-used minimizer index which represents purines (A,G) and pyrimidines (C,T) as R and Y respectively. This creates an indexing scheme robust to the aforementioned chemical damage. We implemented SAFARI, an ancient DNA damage-aware version of the pangenome aligner vg giraffe which uses RYmers to rescue alignments containing deaminated seeds. We show that our approach produces more correct alignments from ancient DNA sequences than current approaches while maintaining a tolerable rate of spurious alignments. In addition, we demonstrate that our algorithm improves the estimate of the rate of ancient DNA damage, especially for highly damaged samples. Crucially, we show that this improved alignment can directly translate into better insights gained from the data by showcasing its integration with a number of extant pangenome tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f60983de5dcd109f2475f2c8134a67852f35929" target='_blank'>
              SAFARI: Pangenome Alignment of Ancient DNA Using Purine/Pyrimidine Encodings
              </a>
            </td>
          <td>
            J. Rubin, J. van Waaij, Louis Kraft, Jouni Sirén, Peter Wad Sackett, Gabriel Renaud
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>