<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-05 12:47:22 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E. Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Phenotypic plasticity is a major factor of tumor heterogeneity and treatment resistance. In particular, cancer stem cells (CSCs) represent a small subpopulation within tumors with self-renewal and tumor-forming capabilities. Understanding reprogramming, maintenance, and lineage properties of CSCs requires dedicated tools to disentangle the respective influences of phenotypic inheritance and cell-cell interactions. Here we set up ultra-wide field microscopy of breast cancer cell lines expressing a stemness fluorescent reporter for several days. The fluorescent reporter distinguishes three phenotypes: cancer stem cells (CSCs), cancer differentiated cells (CDCs) and intermediate/transiting cancer cells (iCCs). Spatial statistics indicate significant zonation, aka phenotypic niches, with CSC clustering near each other but away from CDCs. Surprisingly, single cell time series reveal spontaneous reprogramming events from CDC to CSC even in unperturbed populations. We identify that such transitions are prone to arise during the cell cycle. Moreover, lineage analysis shows that the phenotype is partially inherited from ancestor cells. However, such heredity is not sufficient to explain the spatial properties of the cell population, which also depend on cell-cell interactions. Indeed, we identified that phenotypic transitions of cancer cells are influenced by the phenotypic state of neighboring cells. Reprogramming into CSCs is respectively promoted and inhibited by the presence of CSCs and CDCs in the neighborhood. Altogether, our results disentangle how phenotypic inheritance and intercellular interactions orchestrate the spatio-temporal self-organization of cancer cell heterogeneity, maintaining a subpopulation of CSCs within niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb7d353abe01263d529ea5810500bb13dbe6b8c4" target='_blank'>
              Spatial self-organization of cancer stem cell niches revealed by live single-cell imaging
              </a>
            </td>
          <td>
            Mathilde Brulé, Anaïs Horochowska, Emeline Fontaine, Raoul Torero-Ibad, Flavie Woesteland, Marie Denoulte, Jean Pesez, E. Adriaenssens, R. Toillon, X. Le Bourhis, Benjamin Pfeuty, Chann Lagadec, F. Anquez
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of medulloblastoma (MB) influences progression and therapy response, presenting a promising target for therapeutic advances. Prior single-cell analyses have characterized the cellular components of the TME but lack spatial context. To address this, we performed spatial transcriptomic sequencing on sixteen pediatric MB samples obtained at diagnosis, including two matched diagnosis-relapse pairs. Our analyses revealed inter- and intra-tumoral heterogeneity within the TME, comprised of tumor-associated astrocytes (TAAs), macrophages (TAMs), stromal components, and distinct subpopulations of MB cells at different stages of neuronal differentiation and cell cycle progression. We identified dense regions of quiescent progenitor-like MB cells enriched in patients with high-risk (HR) features and an increase in TAAs, TAMs, and dysregulated vascular endothelium following relapse. Our study presents novel insights into the spatial architecture and cellular landscape of the medulloblastoma TME, highlighting spatial patterns linked to HR features and relapse, which may serve as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b39423470039d57ba772c250043854baef4821" target='_blank'>
              Medulloblastoma Spatial Transcriptomics Reveals Tumor Microenvironment Heterogeneity with High-Density Progenitor Cell Regions Correlating with High-Risk Disease
              </a>
            </td>
          <td>
            Franklin Chien, Marina E. Michaud, Mojtaba Bakhtiari, Chanel Schroff, M. Snuderl, José E. Velázquez Vega, Tobey MacDonald, Manoj K. Bhasin
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b2c5c7d1cda920e03f84144f36e08d5e8bdd9f" target='_blank'>
              Genomic barcoding for clonal diversity monitoring and control in cell-based complex antibody production
              </a>
            </td>
          <td>
            Niels Bauer, Christoph Oberist, Michaela Poth, Julian Stingele, Oliver Popp, Simon Ausländer
          </td>
          <td>2024-06-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155d1a740aed9984732768efd341957b63aa39bb" target='_blank'>
              An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs
              </a>
            </td>
          <td>
            R. Chapple, Xueying Liu, Sivaraman Natarajan, Margaret I. M. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, C. Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, M. Dyer, Heather Sheppard, J. Easton, P. Geeleher
          </td>
          <td>2024-06-19</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) presents significant challenges due to its heterogeneity and late-stage diagnoses. Using single-cell and spatial transcriptomics to elucidate the complex landscape of HGSOC to understand its underlying mechanism. Our analysis reveals significant inter- and intra-tumoral diversity, manifested through distinct cellular subpopulations and varied microenvironmental niches. Notably, our findings highlight a widespread immunosuppressive environment, marked by complex networks of cell-cell interactions, particularly evident in areas of elevated tumor cell density within metastatic samples. We identify the exclusive presence of COL14A1+ neoplastic cells in metastatic specimens, alongside a strong correlation between CD8A+ NKT cells and poor prognosis, and elevated CHODL expression in HGSOC metastasis tissues. Furthermore, knockdown experiments targeting CHODL demonstrate its role in reducing migration and invasion abilities in HGSOC cells. A pivotal discovery of our study is the delineation of specific cellular signatures correlated with adverse outcomes, notably a subset of CHODL+ neoplastic cells characterized by a distinct metabolic phenotype with a predilection for lipid metabolism. The therapeutic targeting of this metabolic pathway with existing inhibitors appears promising in curbing tumor proliferation. These findings enhance our understanding of HGSOC heterogeneity and reveal potential therapeutic targets, promising more effective management strategies for this aggressive cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ff004a52301690996ef94c99200186510d19e37" target='_blank'>
              Integrating single-cell and spatial transcriptomic analysis to unveil heterogeneity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Haixia Luo, Kunyu Wang, Bin Li
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb8ba6a424922fdea28451ef81ee4284340b35a6" target='_blank'>
              Advances in spatial transcriptomics and its applications in cancer research
              </a>
            </td>
          <td>
            Yang Jin, Yuanli Zuo, Gang Li, Wenrong Liu, Yitong Pan, Ting Fan, Xin Fu, Xiaojun Yao, Yong Peng
          </td>
          <td>2024-06-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Adult-type diffuse gliomas, the most common primary brain tumors, pose significant clinical challenges due to limited treatment options, restricted anti-tumor immune response and dismal patient prognosis. In this study, we elucidate the immunological function and clinical relevance of intra-tumoral tertiary lymphoid structures (TLS) in adaptive anti-glioma immunity. We conducted a comprehensive, unbiased analysis of lymphoid aggregation in 642 gliomas using a multi-modal approach that combines RNA sequencing with spatial transcriptome and proteome profiling. Our findings reveal that TLS are present in 15% of tumors and correlate with improved overall survival. Gliomas with TLS exhibit a remodeled perivascular space, marked by transcriptional upregulation and spatial redistribution of collagens associated with barrier functions. Furthermore, we demonstrate that TLS maturation into sites of dynamic adaptive immune responses, characterized by clonal T and B cell expansion and IgA+ and IgG+ plasma-cell formation, is driven by efficient early T cell recruitment to the perivascular space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ba24c654b135fc550ac0b5abe83db49fb3978d" target='_blank'>
              Glioma-associated tertiary lymphoid structures are sites of lymphocyte clonal expansion and plasma cell formation
              </a>
            </td>
          <td>
            P. Cakmak, J. H. Lun, Aakanksha Singh, J. Macas, J. Schupp, M. Köhler, Tatjana Starzetz, Michael C. Burger, Eike Steidl, Lucie Marie Hasse, Elke Hattingen, Karlheinz Plate, Y. Reiss, K. Imkeller
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, , Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The patterns by which primary tumors spread to metastatic sites remain poorly understood. Here, we define patterns of metastatic seeding in prostate cancer (PCa) using a novel injection-based mouse model - EvoCaP (Evolution in Cancer of the Prostate), featuring aggressive metastatic cancer to bone, liver, lungs, and lymph nodes. To define migration histories between primary and metastatic sites, we used our EvoTraceR pipeline to track distinct tumor clones containing recordable barcodes. We detected widespread intratumoral heterogeneity from the primary tumor in metastatic seeding, with few clonal populations (CPs) instigating most migration. Metastasis-to-metastasis seeding was uncommon, as most cells remained confined within the tissue. Migration patterns in our model were congruent with human PCa seeding topologies. Our findings support the view of metastatic PCa as a systemic disease driven by waves of aggressive clones expanding their niche, infrequently overcoming constraints that otherwise keep them confined in the primary or metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7ba92880acaf1b934f559d90646f82a1f001d3" target='_blank'>
              Clonal Lineage Tracing with Somatic Delivery of Recordable Barcodes Reveals Migration Histories of Metastatic Prostate Cancer.
              </a>
            </td>
          <td>
            Ryan N Serio, Armin Scheben, Billy Lu, Domenic V Gargiulo, Lucrezia Patruno, Caroline L Buckholtz, Ryan J Chaffee, Megan C Jibilian, Steven G Persaud, Stephen J. Staklinski, Rebecca Hassett, Lise M Brault, Daniele Ramazzotti, Christopher E Barbieri, Adam Siepel, Dawid G Nowak
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dcce816f6a19dca7baf05569c687c99b3105e9" target='_blank'>
              Unbiasedly decoding the tumor microenvironment with single-cell multiomics analysis in pancreatic cancer
              </a>
            </td>
          <td>
            Yifan Fu, Jinxin Tao, Tao Liu, Yueze Liu, J. Qiu, Dan Su, Ruobing Wang, W. Luo, Zhe Cao, Guihu Weng, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-07-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The current challenge in analyzing sequencing-based spatial transcriptomics (ST) is to precisely resolve the spatial distribution of cell subpopulations with low abundance and detect context-dependent variation in cell states within complex tissue microenvironments. Here, we introduce an ultra-resolution ST deconvolution algorithm (UCASpatial) that improves the resolution of mapping cell subpopulations to spatial locations by leveraging the contribution of genes indicative of cell identity through entropy-based weighting. Using both in silico and real ST datasets, we demonstrate that UCASpatial improves the robustness and accuracy in identifying low-abundant cell subpopulations and distinguishing transcriptionally heterogeneous cell subpopulations. Applying UCASpatial to murine wound healing, we recapitulate known spatiotemporal dynamics of multiple cell subpopulations involved in wound healing and reveal the spatial segregation of distinct macrophage subpopulations embedded within different cellular communities at the wound center. In human colorectal cancer (CRC), we link genomic alterations in individual cancer clones to multi-cellular characteristics of the tumor immune microenvironment (TIME) and reveal the co-evolution of tumor cells and TIME within the same tumor. We show that the copy number gain on chromosome 20q (chr20q-gain) in tumor cells orchestrates a T cell-excluded TIME, indicative of resistance to immunotherapy in CRC, and is associated with tumor-intrinsic endogenous retrovirus silencing and impaired type I interferon response. Our findings present UCASpatial as a versatile tool for deciphering ultra-resolution cellular landscapes in ST and exploring intercellular mechanisms in complex and dynamic microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36aa9aa2d8c5c7dbae3c0976fedf2735c37f839" target='_blank'>
              Ultra-resolution Deconvolution of Spatial Transcriptomics Unveils Spatiotemporal Cellular Dynamics in Complex Microenvironments
              </a>
            </td>
          <td>
            Yin Xu, Zurui Huang, Yawei Zhang, Zhenghang Wang, Peijin Guo, Feifan Zhang, Wenxuan Gong, Guanghao Liang, Boyuan Mei, Lihui Dong, Renbao Chang, Minghui Gong, Yu Xia, Haochen Ni, Jing Yang, Yuan Gao, Zhaoqi Liu, Lin Shen, Jian Li, Meng Michelle Xu, Dali Han
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the spatial organization of the bone marrow (BM) microenvironment at single-cell resolution constitutes a challenge in multiple myeloma (MM). Using formalin-fixed paraffin-embedded (FFPE) samples from the MIcγ1 mouse model of MM and aged-matched healthy YFPcγ1 mice, we performed spatial transcriptional profiling with Visium Spatial Gene Expression. A custom data-analysis framework that combines spatial with single-cell transcriptomic profiling defined the BM cellular composition and established specific cell relations, visualizing the spatial distribution of transcriptionally heterogeneous MM plasma cells (MM-PC). MM pathogenesis transcriptional programs were spatially delineated within the BM microenvironment. A high-to-low MM-PC density gradient spatially correlated with effector-to-exhausted T cell phenotype abundance. In this context, MM cells in high-density MM-PC areas coexisted with dendritic cells while displaced neutrophils to the tumor border. Increased neutrophil extracellular trap formation, IL-17-driven inflammatory signaling, and osteoclast differentiation were spatially delineated within the BM microenvironment. The spatial identification of different areas of BM and the interaction between malignant cells and their microenvironment were validated in FFPE BM biopsies from MM patients with varying degrees of MM-PC infiltration. In summary, spatial transcriptomics depicts the BM cellular architecture of MM and reveals deregulated mechanisms underlying MM intercellular communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6b45562f865baeadce9704ad2edc7a5b178a26" target='_blank'>
              Spatial Transcriptomics Reveals a Myeloma Cell Architecture with Dysfunctional T-Cell Distribution, Neutrophil Traps, and Inflammatory Signaling
              </a>
            </td>
          <td>
            Laura Sudupe, Emma Muiños-López, A. Lopez-Perez, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, Paula Aguirre-Ruiz, Patxi San Martín-Úriz, Marta Larrayoz, Laura Alvarez-Gigli, Marta Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho González, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Solid carcinomas are often highly heterogenous cancers, arising from multiple epithelial cells of origin. Yet, how the cell of origin influences the response of the tumor microenvironment is poorly understood. Lung adenocarcinoma (LUAD) arises in the distal alveolar epithelium which is populated primarily by alveolar epithelial type I (AT1) and type II (AT2) cells. It has been previously reported that Gramd2+ AT1 cells can give rise to a histologically-defined LUAD that is distinct in pathology and transcriptomic identity from that arising from Sftpc+ AT2 cells1,2. To determine how cells of origin influence the tumor immune microenvironment (TIME) landscape, we comprehensively characterized transcriptomic, molecular, and cellular states within the TIME of Gramd2+ AT1 and Sftpc+ AT2-derived LUAD using KRASG12D oncogenic driver mouse models. Myeloid cells within the Gramd2+AT1-derived LUAD TIME were increased, specifically, immunoreactive monocytes and tumor associated macrophages (TAMs). In contrast, the Sftpc+ AT2 LUAD TIME was enriched for Arginase-1+ myeloid derived suppressor cells (MDSC) and TAMs expressing profiles suggestive of immunosuppressive function. Validation of immune infiltration was performed using flow cytometry, and intercellular interaction analysis between the cells of origin and major myeloid cell populations indicated that cell-type specific markers SFTPD in AT2 cells and CAV1 in AT1 cells mediated unique interactions with myeloid cells of the differential immunosuppressive states within each cell of origin mouse model. Taken together, Gramd2+ AT1-derived LUAD presents with an anti-tumor, immunoreactive TIME, while the TIME of Sftpc+ AT2-derived LUAD has hallmarks of immunosuppression. This study suggests that LUAD cell of origin influences the composition and suppression status of the TIME landscape and may hold critical implications for patient response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8e7b3edd38c2a1ddeaa53c137b22e2c605b672" target='_blank'>
              Cell of origin alters myeloid-mediated immunosuppression in lung adenocarcinoma
              </a>
            </td>
          <td>
            Minxiao Yang, Noah Shulkin, Edgar Gonzalez, Jonathan Castillo, Chunli Yan, Keqiang Zhang, Leonidas Arvanitis, Zea Borok, W. Wallace, Dan Raz, E. R. Torres, C. Marconett
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Adult-type diffuse gliomas are a family of aggressive brain tumours with few effective treatments. Their complex cellular makeup adds to the challenge of finding successful therapies. This intratumoural heterogeneity is fuelled by a subpopulation of glioma stem-like cells (GSCs) that drive tumour growth and resistance to standard treatments. Previous research focused on the three glioma types (astrocytoma, oligodendroglioma, glioblastoma) individually revealed malignant cells mimic the transcriptional profiles of normal brain cell types. Whether these diverse cellular states stem from a shared biological origin is unknown. Here, we show through single-cell RNA sequencing of 40 glioma tumours that all gliomas are described by seven recurring cell states. We also identify a shared astrocyte-like GSC population. Our unique method of identifying GSCs, based on reconstructed tumour phylogenies, repositions astrocyte-like cells at the apex of a differentiation hierarchy in glioma. Our findings indicate the transcriptional heterogeneity observed in gliomas stems from a GSC population recapitulating lineages of healthy adult neural stem cells. These results suggest a shared lineage drives the intratumoural heterogeneity observed in adult-type diffuse gliomas. We anticipate that a deeper understanding of the molecular mechanisms maintaining the GSC state will provide a new framework for future therapeutic development and research into glioma cell biology. Highlights Recurring cell states are shared across adult-type diffuse gliomas Reconstructed tumour phylogenies identify an astrocyte-like glioma stem cell population Tumour subclones are segregated non-randomly across cell states">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a7ceb66735b2467794675778803760e2c35252" target='_blank'>
              Adult-type diffuse gliomas share recurring cell states driven by a common astrocyte-like glioma stem cell population
              </a>
            </td>
          <td>
            Walter Muskovic, Ashraf Zaman, Madhavi Pandya, Emily Holton, Chia-Ling Chan, Rachael A McCloy, Himanshi Arora, Venessa T Chin, Joseph E Powell
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer poses a global health challenge, yet the influence of ethnicity on the tumor microenvironment (TME) remains understudied. In this investigation, we examined immune cell infiltration in 230 breast cancer samples, emphasizing diverse ethnic populations. Leveraging tissue microarrays (TMAs) and core samples, we applied multiplex immunofluorescence (mIF) to dissect immune cell subtypes across TME regions. Our analysis revealed distinct immune cell distribution patterns, particularly enriched in aggressive molecular subtypes triple-negative and HER2-positive tumors. We observed significant correlations between immune cell abundance and key clinicopathological parameters, including tumor size, lymph node involvement, and patient overall survival. Notably, immune cell location within different TME regions showed varying correlations with clinicopathologic parameters. Additionally, ethnicities exhibited diverse distributions of cells, with certain ethnicities showing higher abundance compared to others. In TMA samples, patients of Chinese and Caribbean origin displayed significantly lower numbers of B cells, TAMs, and FOXP3-positive cells. These findings highlight the intricate interplay between immune cells and breast cancer progression, with implications for personalized treatment strategies. Moving forward, integrating advanced imaging techniques, and exploring immune cell heterogeneity in diverse ethnic cohorts can uncover novel immune signatures and guide tailored immunotherapeutic interventions, ultimately improving breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f6a66ef05b8d07636d74bae111281d88087db13" target='_blank'>
              Detailed Profiling of the Tumor Microenvironment in Ethnic Breast Cancer, Using Tissue Microarrays and Multiplex Immunofluorescence
              </a>
            </td>
          <td>
            M. Zaakouk, A. Longworth, Kelly Hunter, Suhaib Jiman, Daniel Kearns, M. El-Deftar, Abeer M. Shaaban
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Motivation Cells exhibit a wide array of morphological features, enabling computer vision methods to identify and track relevant parameters. Morphological analysis has long been implemented to identify specific cell types and cell responses. Here we asked whether morphological features might also be used to classify transcriptomic subpopulations within in vitro cancer cell lines. Identifying cell subpopulations furthers our understanding of morphology as a reflection of underlying cell phenotype and could enable a better understanding of how subsets of cells compete and cooperate in disease progression and treatment. Results We demonstrate that cell morphology can reflect underlying transcriptomic differences in vitro using convolutional neural networks. First, we find that changes induced by chemotherapy treatment are highly identifiable in a breast cancer cell line. We then show that the intra cell line subpopulations that comprise breast cancer cell lines under standard growth conditions are also identifiable using cell morphology. We find that cell morphology is influenced by neighborhood effects beyond the cell boundary, and that including image information surrounding the cell can improve model discrimination ability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116b8716438d464b34fa34caed09100f7664c755" target='_blank'>
              Deep learning identifies heterogeneous subpopulations in breast cancer cell lines
              </a>
            </td>
          <td>
            T. Jost, Andrea L. Gardner, Daylin Morgan, Amy Brock
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The molecular underpinnings of High Grade Endometrial Carcinoma (HGEC) metastatic growth and survival are poorly understood. Here we show that ascites-derived and primary tumor HGEC cell lines in 3D spheroid culture faithfully recapitulate key features of malignant peritoneal effusion and exhibit fundamentally distinct transcriptomic, proteomic and metabolomic landscapes when compared with conventional 2D monolayers. Using genetic screening platform we identify MAPK14 (which encodes the protein kinase p38α) as a specific requirement for HGEC in spheroid culture. MAPK14/p38α has broad roles in programing the phosphoproteome, transcriptome and metabolome of HGEC spheroids, yet has negligible impact on monolayer cultures. MAPK14 promotes tumorigenicity in vivo and is specifically required to sustain a sub-population of spheroid cells that is enriched in cancer stemness markers. Therefore, spheroid growth of HGEC activates unique biological programs, including p38α signaling, that cannot be captured using 2D culture models and are highly relevant to malignant disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e7c989b30535b3174da390f7011080ff704c1d1" target='_blank'>
              MAPK14 /p38α Shapes the Molecular Landscape of Endometrial Cancer and promotes Tumorigenic Characteristics
              </a>
            </td>
          <td>
            Sayali Joseph, Xingyuan Zhang, G. Droby, Di Wu, V. Bae-Jump, Scott Lyons, Angie Mordant, Allie Mills, Laura Herring, Blake R Rushing, Jessica L. Bowser, Cyrus Vaziri
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Targeted therapies can induce strong tumor regression, but they typically fail to eradicate metastatic cancers. The elucidation of the causes that enable cancers to survive within a residual disease is essential for finding eradication strategies. The ability of cancers to survive eradication reflects not only cell-intrinsic sensitivities, but also microenvironmental effects. Paracrine signals produced by fibroblast, non-neoplastic cells that make tumor stroma, can provide strong but spatially limited therapy protection. Even though this phenomenon is well-established, its contribution to the ability of tumors to escape eradication remains poorly understood. To address this gap of knowledge, we developed an in silico model that recapitulates the effect of stroma on therapy responses in tumor tissues. This model enabled us to evaluate the contribution of spatial aspects of stroma-mediated resistance. Our analyses reveal that stroma dispersal might be the most important yet overlooked aspect of stromal resistance that determines the overall tumor responses to therapy. Abstract The response of tumors to anti-cancer therapies is defined not only by cell-intrinsic therapy sensitivities but also by local interactions with the tumor microenvironment. Fibroblasts that make tumor stroma have been shown to produce paracrine factors that can strongly reduce the sensitivity of tumor cells to many types of targeted therapies. Moreover, a high stroma/tumor ratio is generally associated with poor survival and reduced therapy responses. However, in contrast to advanced knowledge of the molecular mechanisms responsible for stroma-mediated resistance, its effect on the ability of tumors to escape therapeutic eradication remains poorly understood. To a large extent, this gap of knowledge reflects the challenge of accounting for the spatial aspects of microenvironmental resistance, especially over longer time frames. To address this problem, we integrated spatial inferences of proliferation-death dynamics from an experimental animal model of targeted therapy responses with spatial mathematical modeling. With this approach, we dissected the impact of tumor/stroma distribution, magnitude and distance of stromal effects. While all of the tested parameters affected the ability of tumor cells to resist elimination, spatial patterns of stroma distribution within tumor tissue had a particularly strong impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d34125347f89218581236d182c69f2cff0ff55a" target='_blank'>
              Dissecting the Spatially Restricted Effects of Microenvironment-Mediated Resistance on Targeted Therapy Responses
              </a>
            </td>
          <td>
            Tatiana Miti, Bina Desai, D. Miroshnychenko, D. Basanta, A. Marusyk
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Adenocarcinoma of the lung (LUAD) is a common and highly lethal disease. Clinical grading of disease strongly predicts recurrence and survival after surgery and is determined by morphological assessment of histological growth patterns in resected tumours. The molecular basis of growth pattern is poorly understood at present, as are the mechanisms linking growth pattern to recurrence and death. Interestingly, the two archetypal lethal morphologies, solid and micropapillary patterns, are characterised by their biphasic appearance. Both have an epithelial fraction which is in direct stromal contact, and a fraction which is not. This morphological variance seems likely to represent plasticity, and to be causally linked to mechanisms of virulence. To investigate the gene expression changes related to growth pattern both intra- and intertumoral, we applied spatial transcriptomics (Nanostring GeoMx DSP) to tissue microarray specimens of primary resected human lung adenocarcinoma. Using a variety of region-of-interest (ROI) selection strategies, we sampled 160 pure epithelial ROIs across 7 distinct morphological features of LUAD from 51 patients. Analyses of gene expression reveal fundamental trajectories connecting growth patterns, and crucial modes of plasticity which underly high-risk morphologies. These modes suggest mechanisms for the origins of growth pattern and mechanisms of virulence. Our work highlights dramatic divergence in gene expression programmes between highly lethal but morphologically diverse modes of tumour growth. Furthermore, it provides an explanation for how microscopically localised hypoxia in the primary tumour helps to establish and maintain survival strategies which ultimately determine morphology-specific mechanisms of tumour metastasis, suggesting new therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a08117cd59bda92ca8c6af39ca3279ca1ec800" target='_blank'>
              Spatial resolution of transcriptomic plasticity states underpinning lethal morphologies in lung adenocarcinoma
              </a>
            </td>
          <td>
            Hannah L Williams, Nicolas Poulain, Ian Powley, Silvia Martinelli, R. Bielik, H. Leslie, Colin Nixon, C. Wilson, M. Sereno, Zhangyi He, Leah Officer-Jones, Fiona Ballantyne, Rachel Pennie, C. Wood, David Y Lewis, Nigel B Jamieson, J. Le Quesne
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32ee169db3167501daf79f6fa134c84e68883d84" target='_blank'>
              Spatial transcriptomics in cancer research and potential clinical impact: a narrative review
              </a>
            </td>
          <td>
            Michael A Cilento, C. Sweeney, Lisa M. Butler
          </td>
          <td>2024-06-08</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, still most patients experience limited or no clinical benefit. The immune component dynamics during treatment and how it correlated with treatment efficacy remained largely known. We hypothesize that understanding the modulation dynamics of the TME upon ICI treatment would be key to elucidate the mechanisms of response and resistance. To systematically characterize these dynamics, we aggregated a large-scale single-cell RNA-sequencing (scRNA-seq) dataset of 141 patients across multiple cancer types and ICI treatment modalities, with longitudinally paired samples. We developed a robust, integration-free, and cluster-free deep phenotyping framework, leveraging multiple pan-cancer references to annotate 876,410 high-quality cells into 80 granular cell states. Our analysis revealed consistent compositional changes in 17 cell subtypes following ICI treatment, including an enhanced adaptive immune response such as CD8+ IL7R+ central memory T cells, GZMK+ effector memory T cells, germinal center B cells, precursor exhausted T cells, and CD4+ TfhTh1 cells, accompanied by a broad reduction in interferon responsive immune cells quiescent CD4+CREM-memory T cells. We uncovered co-regulated cell communities within the TME, highlighting the coordinated interplay between adaptive and innate immune cells, as well as immune and non-immune components. Furthermore, we identified two distinct patient groups exhibiting tightly correlated cellular dynamics within the TME post-treatment. The first group, enriched for responders, displayed a marked expansion of naive lymphocytes, while the second group, predominantly composed of non-responders, showed an increased abundance of immune experienced/suppressive cell states. This dichotomy in TME dynamics offers a potential predictive biomarker for patient stratification and personalized therapeutic strategies. Our study presents a comprehensive landscape of the cellular dynamics within the TME during ICI treatment, enabled by a powerful deep phenotyping approach showcasing the importance of a systems-level understanding of the TME dynamics in improving patient stratification and advancing personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e90093265f7348aad7b591e51e89915aa6d67b" target='_blank'>
              Cellular Dynamics Upon Immune Checkpoint Inhibition
              </a>
            </td>
          <td>
            Zhongyang Lin, Eketerina Petrenko, Mitalee Chandra, Nalini Srinivas, Selma Ugurel, Jürgen C. Becker, Dvir Aran
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have heralded a remarkable shift in cancer care, significantly extending survival for advanced cancer patients. However, despite their remarkable clinical successes, a substantial majority of patients fail to achieve a lasting response to ICIs. To address this challenge and gain insights into the complex landscape of the tumor microenvironment (TME), we conducted an extensive analysis using single-cell RNA sequencing (scRNA; ∼216K cells across 39 samples) and single-nucleus transposase-accessible chromatin sequencing (snATAC; ∼43K cells from 15 samples) in a metastatic melanoma cohort. This systematic approach delineates 14 distinct cell types and 55 cell subtypes, including the identification of 15 transcriptional hallmarks of malignant cells. Through correlation analysis of cell subtype proportions, we unveiled six distinct clusters associated with varying tumor responses. Particularly intriguing was the identification of the mature DC enriched in immunoregulatory molecules (mregDC) subtype exhibiting correlations with naive T and B cells, forming an anti-tumor program that underscores the importance of multiple immune cell types in mediating anti-tumor immunity. Notably, we found that mregDC abundance represents a good prognostic predictor of progression-free survival (PFS) in the context of ICI treatment, and when combined with the TCF7+/- CD8 T cell ratio, it reliably predicts patient PFS across treatments beyond ICI. We validated our findings using an independent cohort of 274 ICI-treated melanoma samples analyzed using tissue-level expression. We next compared mregDCs and conventional dendritic cell types 1 and 2 (cDC1 and cDC2) using transcriptome signature, differentiation trajectory, interactome, cytokine milieu, and epigenome landscape analyses. This comparative analysis shed light on the unique attributes of mregDCs within the TME. Finally, we investigated cell type/subtype-specific genes, pathways, immune response enrichment, and ligand-receptor interactions closely associated with the proportions of mregDCs within the TME. These molecular and cellular insights, with their critical roles in enhancing the immune response against cancer, offer valuable prospects for predicting the efficacy of ICI regimens, and potentially guiding the selection of rational combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd285dffb8067a1b036abc12a6336c80dd568b9" target='_blank'>
              Mature and migratory dendritic cells promote immune infiltration and response to anti-PD-1 checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Jie-Ying Yang, Cassia Wang, Doris Fu, Li-Lun Ho, K. Galani, Lee Chen, Dennie T. Frederick, Liang He, Amy Y. Huang, Emily J. Robitschek, Sandeep K. Yadav, Wentao Deng, Rahul Tacke, Kelly P. Burke, T. Sharova, Kunal Rai, David Liu, Genevieve M. Boland, M. Kellis
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Intratumor heterogeneity has been shown to play a role in the malignant progression of cancer. Although clonal evolution in primary cancer has been well studied, that in metastatic tumorigenesis is not fully understood. In this study, we established human colon cancer-derived organoids and investigated clonal dynamics during liver metastasis development by tracking barcode-labelled subclones. Long-term subclone co-cultures showed clonal drift, with a single subclone becoming dominant in the cell population. Interestingly, the selected subclones were not always the same, suggesting that clonal selection was not based on cell intrinsic properties. Furthermore, liver tumors developed by co-transplantation of organoid subclones into the immunodeficient mouse spleen showed a progressive drastic reduction in clonal diversity, and only one or two subclones predominated in the majority of large metastatic tumors. Importantly, selections were not limited to particular subclones but appeared to be random. A trend towards a reduction in clonal diversity was also found in liver metastases of multiple color-labeled organoids of mouse intestinal tumors. Based on these results, we propose a novel mechanism of metastasis development, i.e. a subclone population of the disseminated tumor cells in the liver is selected by neutral selection during colonization and constitutes large metastatic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2918909f5cedae4e48ddf4cd75fefc07c76ff3b" target='_blank'>
              Neutral selection and clonal expansion during the development of colon cancer metastasis.
              </a>
            </td>
          <td>
            Xuelian Lei, Daisuke Yamamoto, Hirotaka Kitamura, Kenji Kita, Noriyuki Inaki, Kazuhiro Murakami, M. Nakayama, H. Oshima, Masanobu Oshima
          </td>
          <td>2024-06-18</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Intracerebral hemorrhage (ICH) poses a formidable challenge in stroke management, with limited therapeutic options, particularly in the realm of immune‐targeted interventions. Clinical trials targeting immune responses post‐ICH have encountered setbacks, potentially attributable to the substantial cellular heterogeneity and intricate intercellular networks within the brain. Here, we present a pioneering investigation utilizing single‐cell RNA sequencing and spatial transcriptome profiling at hyperacute (1 h), acute (24 h), and subacute (7 days) intervals post‐ICH, aimed at unraveling the dynamic immunological landscape and spatial distributions within the cerebral tissue. Our comprehensive analysis revealed distinct cell differentiation patterns among myeloid and lymphocyte populations, along with delineated spatial distributions across various brain regions. Notably, we identified a subset of lymphocytes characterized by the expression of Spp1 and Lyz2, termed macrophage‐associated lymphocytes, which exhibited close interactions with myeloid cells. Specifically, we observed prominent interactions between Lgmn+Macro‐T cells and microglia through the spp1–cd44 pathway during the acute phase post‐ICH in the choroid plexus. These findings represent a significant advancement in our understanding of immune cell dynamics at single‐cell resolution across distinct post‐ICH time points, thereby laying the groundwork for exploring critical temporal windows and informing the development of targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af27d3cf37340430057062faa5cf04622e91eea" target='_blank'>
              Unraveling dynamic immunological landscapes in intracerebral hemorrhage: insights from single‐cell and spatial transcriptomic profiling
              </a>
            </td>
          <td>
            Lingui Gu, Hualin Chen, Mingjiang Sun, Yihao Chen, Qinglei Shi, Jianbo Chang, Junji Wei, Wenbin Ma, X. Bao, Renzhi Wang
          </td>
          <td>2024-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Coexistence of two cancer types in the same organ presents challenges for clinical decision-making, calling for personalized treatment strategies. Deep Visual Proteomics (DVP) combines AI driven single cell type analysis with laser microdissection and ultrasensitive mass spectrometry. In a composite case of classical Hodgkin lymphoma (cHL) and small lymphocytic lymphoma (SLL) in a single patient, we investigated the potential of DVP to inform precision oncology. We quantified the proteomic landscapes in the cHL and SLL to a depth of thousands of proteins. Our analysis revealed distinct proteome profiles in cHL and SLL populations, highlighting their clonal unrelatedness. Our data suggested standardized chemotherapy and interleukin-4 inhibition as potential strategies to manage chemo-resistance – instead of bone marrow transplantation. DVP highlighted minichromosome maintenance protein and proteasome inhibitors for cHL and H3K27 methylation and receptor tyrosine kinase inhibitors for SLL as subtype-specific treatments. Thus cell-type specific insights of DVP can guide personalized oncological treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5070048c8db349d0f00bc2f9584b60952c4497" target='_blank'>
              Deep Visual Proteomics Unveils Precision Medicine Insights in Composite Small Lymphocytic and Classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            Xiang Zheng, Lars Møller Pedersen, M. Bzorek, A. Mund, Lise Mette Rahbek Gjerdrum, Matthias Mann
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid growth and early metastasis and is susceptible to treatment resistance and recurrence. Understanding the intra-tumoral spatial heterogeneity in SCLC is crucial for improving patient outcomes and clinically relevant subtyping. In this study, a spatial whole transcriptome-wide analysis of 25 SCLC patients at sub-histological resolution using GeoMx Digital Spatial Profiling technology is performed. This analysis deciphered intra-tumoral multi-regional heterogeneity, characterized by distinct molecular profiles, biological functions, immune features, and molecular subtypes within spatially localized histological regions. Connections between different transcript-defined intra-tumoral phenotypes and their impact on patient survival and therapeutic response are also established. Finally, a gene signature, termed ITHtyper, based on the prevalence of intra-tumoral heterogeneity levels, which enables patient risk stratification from bulk RNA-seq profiles is identified. The prognostic value of ITHtyper is rigorously validated in independent multicenter patient cohorts. This study introduces a preliminary tumor-centric, regionally targeted spatial transcriptome resource that sheds light on previously unexplored intra-tumoral spatial heterogeneity in SCLC. These findings hold promise to improve tumor reclassification and facilitate the development of personalized treatments for SCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28041b5c4214338317866c30c837c28570a97950" target='_blank'>
              Spatial Transcriptome-Wide Profiling of Small Cell Lung Cancer Reveals Intra-Tumoral Molecular and Subtype Heterogeneity.
              </a>
            </td>
          <td>
            Zicheng Zhang, Xu-min Sun, Yutao Liu, Yibo Zhang, Zijian Yang, J. Dong, Nan Wang, Jianming Ying, Meng Zhou, Lin Yang
          </td>
          <td>2024-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Understanding transcriptional heterogeneity in cancer cells and its implication for treatment response is critical to identify how resistance occurs and may be targeted. Such heterogeneity can be captured by in vitro studies through clonal barcoding methods. We present TraCSED (Transformer-based modeling of Clonal Selection and Expression Dynamics), a dynamic deep learning approach for modeling clonal selection. Using single-cell gene expression and the fitness of barcoded clones, TraCSED identifies interpretable gene programs and the timepoints at which they are associated with clonal selection. When applied to cells treated with either giredestrant, an estrogen receptor (ER) antagonist and degrader, or palbociclib, a CDK4/6 inhibitor, time-dependent resistance pathways are revealed. For example, ER activity is associated with positive selection around day four under palbociclib treatment and this adaptive response can be suppressed by combining the drugs. Yet, in the combination treatment, one clone still emerged. Clustering based on partial least squares regression found that high baseline expression of both SNHG25 and SNCG genes was the primary marker of positive selection to co-treatment and thus potentially associated with innate resistance – an aspect that traditional differential analysis methods missed. In conclusion, TraCSED enables associating pathways with phenotypes in a time-dependent manner from scRNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb809968ef629bdd6c8b90e0032f40dbb52452b" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics (TraCSED) reveals resistance signatures in breast cancer
              </a>
            </td>
          <td>
            Nathan Maulding, Jun Zou, Wei Zhou, Ciara Metcalfe, Josh Stuart, Xin Ye, Marc Hafner
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Annotation of the cis-regulatory elements that drive transcriptional dysregulation in cancer cells is critical to improving our understanding of tumor biology. Herein, we present a compendium of matched chromatin accessibility (scATAC-seq) and transcriptome (scRNA-seq) profiles at single-cell resolution from human breast tumors and healthy mammary tissues processed immediately following surgical resection. We identify the most likely cell-of-origin for luminal breast tumors and basal breast tumors and then introduce a novel methodology that implements linear mixed-effects models to systematically quantify associations between regions of chromatin accessibility (i.e. regulatory elements) and gene expression in malignant cells versus normal mammary epithelial cells. These data unveil regulatory elements with that switch from silencers of gene expression in normal cells to enhancers of gene expression in cancer cells, leading to the upregulation of clinically relevant oncogenes. To translate the utility of this dataset into tractable models, we generated matched scATAC-seq and scRNA-seq profiles for breast cancer cell lines, revealing, for each subtype, a conserved oncogenic gene expression program between in vitro and in vivo cells. Together, this work highlights the importance of non-coding regulatory mechanisms that underlie oncogenic processes and the ability of single-cell multi-omics to define the regulatory logic of BC cells at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bf4929898d9a3caa32f9bca65e96287e05306d" target='_blank'>
              Defining the Regulatory Logic of Breast Cancer Using Single-Cell Epigenetic and Transcriptome Profiling
              </a>
            </td>
          <td>
            Matthew J. Regner, Susana Garcia-Recio, A. Thennavan, K. Wiśniewska, Raul Mendez-Giraldez, Brooke Felsheim, Philip M. Spanheimer, Joel S. Parker, C. Perou, Hector L. Franco
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>159</td>
        </tr>

        <tr id="The acidic tumor microenvironment favors cancer aggressiveness via incompletely understood pathways. Here, we asked whether acidic environments select for cancer stem cell (CSC) properties. Bulk RNA-seq of Panc-1 human pancreatic cancer cells adapted to extracellular pH 6.5 revealed upregulation of CSC markers including CD44, EpCam, Nestin and aldehyde dehydrogenases, and CSC pathway enrichment. We therefore assessed CSC characteristics of acid-adapted (AA) and non-adapted (Ctrl) PaTu8988s and MiaPaca-2 pancreatic cancer cells. Compared to Ctrl, AA cells exhibited increased ALDH- and β-catenin activity and pancreatosphere-forming efficiency, classical CSC characteristics. Panc-1, PaTu8988s and MiaPaCa-2 AA cells differed in CSC marker expression, and AA cells did not exhibit typical flow cytometric CSC populations. However, single-nucleus sequencing identified the acid adaptation-induced emergence of a population with clear CSC characteristics. Finally, in an orthotopic mouse model, AA Panc-1 cells drove strongly increased aggressiveness and liver metastasis compared to Ctrl cells. We conclude that acid-adaptation of pancreatic cancer cells leads to enrichment of a CSC phenotype with unusual traits, providing new insight into how acidic tumor microenvironments favor cancer aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87f31abeb1d47ff4ba195ad016a61c5c1131c375" target='_blank'>
              Tumor microenvironment acidosis favors pancreatic cancer stem cell properties and in vivo metastasis
              </a>
            </td>
          <td>
            Michala G Rolver, J. C. Roda, Yifan Dai, Mette Flinck, Renata Ialchina, Julie Hindkær, Rigmor T. Dyhr, August N. Bodilsen, N. S. Prasad, Jonathan Baldan, Jiayi Yao, Albin Sandelin, Luis Arnes, S. F. Pedersen
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Melanoma, a malignant skin cancer arising from melanocytes, exhibits rapid metastasis and a high mortality rate, especially in advanced stages. Current treatment modalities, including surgery, radiation, and immunotherapy, offer limited success, with immunotherapy using immune checkpoint inhibitors (ICIs) being the most promising. However, the high mortality rate underscores the urgent need for robust, non-invasive biomarkers to predict patient response to adjuvant therapies. The immune microenvironment of melanoma comprises various immune cells, which influence tumor growth and immune response. Melanoma cells employ multiple mechanisms for immune escape, including defects in immune recognition and epithelial-mesenchymal transition (EMT), which collectively impact treatment efficacy. Single-cell analysis technologies, such as single-cell RNA sequencing (scRNA-seq), have revolutionized the understanding of tumor heterogeneity and immune microenvironment dynamics. These technologies facilitate the identification of rare cell populations, co-expression patterns, and regulatory networks, offering deep insights into tumor progression, immune response, and therapy resistance. In the realm of biomarker discovery for melanoma, single-cell analysis has demonstrated significant potential. It aids in uncovering cellular composition, gene profiles, and novel markers, thus advancing diagnosis, treatment, and prognosis. Additionally, tumor-associated antibodies and specific genetic and cellular markers identified through single-cell analysis hold promise as predictive biomarkers. Despite these advancements, challenges such as RNA-protein expression discrepancies and tumor heterogeneity persist, necessitating further research. Nonetheless, single-cell analysis remains a powerful tool in elucidating the mechanisms underlying therapy response and resistance, ultimately contributing to the development of personalized melanoma therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824bd1e506d1d79fb980c0e85de9df9031024731" target='_blank'>
              Advancing immunotherapy for melanoma: the critical role of single-cell analysis in identifying predictive biomarkers
              </a>
            </td>
          <td>
            Ru He, Jiaan Lu, Jianglong Feng, Ziqing Lu, Kaixin Shen, Ke Xu, Huiyan Luo, Guanhu Yang, Hao Chi, Shangke Huang
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite sustained clinical responses to immune-checkpoint blockade (ICB) therapies in non-small-cell lung cancer (NSCLC), the majority of patients derive no clinical benefits, and the cellular and molecular underpinnings of such resistance remain incompletely understood. To identify cell types that may influence immunotherapy responses, we first integrated newly generated and previously published single-cell RNA sequencing data from 110 treatment-naïve patients with NSCLC. Among tumor-resident cell types, we identified MMP1+ cancer-associated fibroblasts (CAFs), which were inversely correlated with the level of tumor-reactive T cells—a key determinant of response to ICB. Further single-cell analysis for newly collected 21 tumor samples from NSCLC patients treated with anti-PD-1/PD-L1 agents revealed that MMP1+ fibroblasts were indeed enriched in treatment-refractory patients, and this observation was also validated in an independent dataset of bulk RNA sequencing from 344 NSCLC patients treated with PD-L1 agents. Examination of the spatial architecture showed that MMP1+ fibroblasts were located at the tumor-stroma boundary, forming a single-cell layer that encircled the cancer cell aggregates, and we hence defined MMP1+ fibroblasts as tumor-stroma boundary (tsb)CAFs. Such tsbCAFs likely promote resistance to ICB by functioning as a physical barrier that prevents tumor-reactive T cells from recognizing and killing cancer cells. Our study provides a new framework to identify cellular underpinnings of resistance to ICB and suggests new strategies to overcome ICB resistance. Highlights ◊ Identification and characterization of MMP1+ fibroblasts in lung cancer. ◊ Single-cell meta-analysis reveals cell populations impeding the accumulation of tumor-reactive T cells. ◊ MMP1+ fibroblasts correlate with the low infiltration of tumor-reactive T cells and the resistance to anti-PD-1/PD-L1 treatment. ◊ MMP1+ fibroblasts appear to form a space barrier between malignant and T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2d872e95a0ff58467c4e7529f81e90a6eb1891" target='_blank'>
              Defining cellular determinants of resistance to PD-1 pathway blockade in non-small-cell lung cancer
              </a>
            </td>
          <td>
            Baolin Liu, K. Feng, Kezhuo Yu, Ranran Gao, Xueda Hu, Boyu Qin, Jinliang Wang, Zhiqiang Xue, Weidong Han, Zemin Zhang
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The spatial arrangement of cells is vital in developmental processes and organogenesis in multicellular life forms. Deep learning models trained with spatial omics data uncover complex patterns and relationships among cells, genes, and proteins in a high-dimensional space, providing new insights into biological processes and diseases. State-of-the-art in silico spatial multi-cell gene expression methods using histological images of tissue stained with hematoxylin and eosin (H&E) to characterize cellular heterogeneity. These computational techniques offer the advantage of analyzing vast amounts of spatial data in a scalable and automated manner, thereby accelerating scientific discovery and enabling more precise medical diagnostics and treatments. In this work, we developed a vision transformer (ViT) framework to map histological signatures to spatial single-cell transcriptomic signatures, named SPiRiT (Spatial Omics Prediction and Reproducibility integrated Transformer). Our framework was enhanced by integrating cross validation with model interpretation during hyper-parameter tuning. SPiRiT predicts single-cell spatial gene expression using the matched histopathological image tiles of human breast cancer and whole mouse pup, evaluated by Xenium (10x Genomics) datasets. Furthermore, ViT model interpretation reveals the high-resolution, high attention area (HAR) that the ViT model uses to predict the gene expression, including marker genes for invasive cancer cells (FASN), stromal cells (POSTN), and lymphocytes (IL7R). In an apple-to-apple comparison with the ST-Net Convolutional Neural Network algorithm, SPiRiT improved predictive accuracy by 40% using human breast cancer Visium (10x Genomics) dataset. Cancer biomarker gene prediction and expression level are highly consistent with the tumor region annotation. In summary, our work highlights the feasibility to infer spatial single-cell gene expression using tissue morphology in multiple-species, i.e., human and mouse, and multi-organs, i.e., mouse whole body morphology. Importantly, incorporating model interpretation and vision transformer is expected to serve as a general-purpose framework for spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2dd7dd123b282d7d557858b4a296ca45cae841" target='_blank'>
              Inferring single-cell spatial gene expression with tissue morphology via explainable deep learning
              </a>
            </td>
          <td>
            Yue Zhao, Elaheh Alizadeh, Yang Liu, Ming Xu, J. M. Mahoney, Sheng Li
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Simple Summary This study investigates the role of visual intratumor heterogeneity (ITH) in breast cancer progression. By analyzing histologic images from the Carolina Breast Cancer Study (CBCS) and the Cancer Genome Atlas Breast Invasive Carcinoma (TCGA-BRCA) data using advanced image processing and machine learning techniques, we developed a measure of tumor heterogeneity based on visual features. Our findings indicate that tumors with low visual heterogeneity exhibited a higher risk of recurrence and were more likely to come from patients whose tumors comprised of only one subclone or had a TP53 mutation. Conversely, high visual heterogeneity was correlated with a more favorable prognosis. These results suggest that visual heterogeneity provides complementary information to molecular markers. A comprehensive understanding of both the visual and molecular aspects of heterogeneity has the potential to offer novel insights for treatment strategies. Abstract High intratumoral heterogeneity is thought to be a poor prognostic indicator. However, the source of heterogeneity may also be important, as genomic heterogeneity is not always reflected in histologic or ‘visual’ heterogeneity. We aimed to develop a predictor of histologic heterogeneity and evaluate its association with outcomes and molecular heterogeneity. We used VGG16 to train an image classifier to identify unique, patient-specific visual features in 1655 breast tumors (5907 core images) from the Carolina Breast Cancer Study (CBCS). Extracted features for images, as well as the epithelial and stromal image components, were hierarchically clustered, and visual heterogeneity was defined as a greater distance between images from the same patient. We assessed the association between visual heterogeneity, clinical features, and DNA-based molecular heterogeneity using generalized linear models, and we used Cox models to estimate the association between visual heterogeneity and tumor recurrence. Basal-like and ER-negative tumors were more likely to have low visual heterogeneity, as were the tumors from younger and Black women. Less heterogeneous tumors had a higher risk of recurrence (hazard ratio = 1.62, 95% confidence interval = 1.22–2.16), and were more likely to come from patients whose tumors were comprised of only one subclone or had a TP53 mutation. Associations were similar regardless of whether the image was based on stroma, epithelium, or both. Histologic heterogeneity adds complementary information to commonly used molecular indicators, with low heterogeneity predicting worse outcomes. Future work integrating multiple sources of heterogeneity may provide a more comprehensive understanding of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18cc90a062bf3cee9a0bd73529b68cc2b5c51dfd" target='_blank'>
              Visual Intratumor Heterogeneity and Breast Tumor Progression
              </a>
            </td>
          <td>
            Yao Li, Sarah C. Van Alsten, Dong Neuck Lee, Taebin Kim, Benjamin C. Calhoun, C. Perou, Sara E. Wobker, J. S. Marron, Katie Hoadley, M. A. Troester
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>159</td>
        </tr>

        <tr id="Melanomas are the deadliest skin cancers, in part due to cellular plasticity and heterogeneity. Intratumoral heterogeneity drives varied mutable phenotypes, specifically “melanocytic” and “mesenchymal” cell states, which result in differential functional properties and drug responses. Definitive and rigorous classification of these phenotypic states has been challenging with conventional biomarker-based methods, and high-parameter molecular methods are cell-destructive, labor-intensive, and time-consuming. To overcome these technical and practical limitations, we utilized label-free artificial intelligence-based morphological profiling to classify live melanoma cells into melanocytic and mesenchymal phenotypes based on high resolution imaging of single cells. To predict the phenotypes of single melanoma cells based on morphology alone, we developed the AI-based ‘Melanoma Phenotype Classifier’ trained with 19 patient-derived cell lines with known melanocytic or mesenchymal transcriptional profiles. To link phenotypic state with high-dimensional morphological profiles, cells were subjected to genetic and chemical perturbations known to shift phenotypic states. The AI classifier successfully predicted phenotypic shifts which were confirmed by single-cell RNA-Seq (scRNA-Seq). These results demonstrate that correlations between melanoma cell phenotypes and morphological changes are detectable by AI. Additionally, the Melanoma Phenotype Classifier was applied to dissociated tumor biopsy samples and characterization of phenotypic heterogeneity was supported by scRNA-Seq transcriptional profiles. This work establishes a link between cell morphology and melanoma phenotypes, laying the groundwork for the use of a label-free morphology-based method for phenotyping live melanoma cells combined with additional analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efb6b8eb45b844e5401bfcc2f8a0a6837a8825f0" target='_blank'>
              Label-Free Melanoma Phenotype Classification Using Artificial Intelligence-Based Morphological Profiling
              </a>
            </td>
          <td>
            Evelyn Lattmann, Andreja Jovic, Julie Kim, T. Pham, Christian Corona, Zhouyang Lian, K. Saini, Manisha Ray, Vivian Lu, A. Tastanova, Stephane C. Boutet, Mitchell P. Levesque
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary In our study, we investigated immune regulation in a subtype of lymphoma called mantle cell lymphoma (MCL). Our goal was to understand how different types of immune cells, specifically T cells, behave in MCL. To do this, we combined image analysis and a technology called digital spatial profiling. We looked at tumor tissue from 102 MCL patients and analyzed different T-cell subsets. Interestingly, we found that late-stage differentiated T cells (CD57+) were more common in tumor-rich areas. These T cells also showed an increased expression of immune suppressive markers. We identified potential targets for treatment, such as CD47, IDO1, and CTLA-4. Additionally, we found that patients with sparse T-cell infiltration had an increased amount of GITR, which tentatively can be therapeutically targeted. Our findings shed light on previously unknown features of T-cell behavior in MCL. Abstract With the aim to advance the understanding of immune regulation in MCL and to identify targetable T-cell subsets, we set out to combine image analysis and spatial omic technology focused on both early and late differentiation stages of T cells. MCL patient tissue (n = 102) was explored using image analysis and GeoMx spatial omics profiling of 69 proteins and 1812 mRNAs. Tumor cells, T helper (TH) cells and cytotoxic (TC) cells of early (CD57−) and late (CD57+) differentiation stage were analyzed. An image analysis workflow was developed based on fine-tuned Cellpose models for cell segmentation and classification. TC and CD57+ subsets of T cells were enriched in tumor-rich compared to tumor-sparse regions. Tumor-sparse regions had a higher expression of several key immune suppressive proteins, tentatively controlling T-cell expansion in regions close to the tumor. We revealed that T cells in late differentiation stages (CD57+) are enriched among MCL infiltrating T cells and are predictive of an increased expression of immune suppressive markers. CD47, IDO1 and CTLA-4 were identified as potential targets for patients with T-cell-rich MCL TIME, while GITR might be a feasible target for MCL patients with sparse T-cell infiltration. In subgroups of patients with a high degree of CD57+ TC-cell infiltration, several immune checkpoint inhibitors, including TIGIT, PD-L1 and LAG3 were increased, emphasizing the immune-suppressive features of this highly differentiated T-cell subset not previously described in MCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd671c1ae7f90cd22784b36ac1b15cb30c9901a3" target='_blank'>
              Quantification and Profiling of Early and Late Differentiation Stage T Cells in Mantle Cell Lymphoma Reveals Immunotherapeutic Targets in Subsets of Patients
              </a>
            </td>
          <td>
            L. Lokhande, Daniel Nilsson, Joana de Matos Rodrigues, May Hassan, L. Olsson, P. Pyl, Louella Vasquez, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0621945ca8b1b53d40c5be86757c2e25bfc05712" target='_blank'>
              Hypoxia within tumor microenvironment characterizes distinct genomic patterns and aids molecular subtyping for guiding individualized immunotherapy
              </a>
            </td>
          <td>
            Run Shi, Jing Sun, Hanyu Zhou, Tong Hu, Zhaojia Gao, Xin Wang, Minglun Li, Zhaokai Zhou, Yongqian Shu
          </td>
          <td>2024-06-07</td>
          <td>Journal of Big Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Therapeutics targeting tumor endothelial cells (TECs) have been explored for decades, with only suboptimal efficacy achieved, partly due to an insufficient understanding of the TEC heterogeneity across cancer patients. We integrated single-cell RNA-seq data of 575 cancer patients from 19 solid tumor types, comprehensively charting the TEC phenotypic diversities. Our analyses uncovered underappreciated compositional and functional heterogeneity in TECs from a pan-cancer perspective. Two subsets, CXCR4+ tip cells and SELE+ veins, represented the prominent angiogenic and proinflammatory phenotypes of TECs, respectively. They exhibited distinct spatial organization patterns, and compared to adjacent non-tumor tissues, tumor tissue showed an increased prevalence of CXCR4+ tip cells, yet with SELE+ veins depleted. Such functional and spatial characteristics underlie their differential associations with the response of anti-angiogenic therapies and immunotherapies. Our integrative resources and findings open new avenues to understand and clinically intervene in the tumor vasculature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc571fa1d1ff50ea9e24190fe0da0d50d2cd7b6f" target='_blank'>
              Pan-cancer integrative analyses dissect the remodeling of endothelial cells in human cancers
              </a>
            </td>
          <td>
            Jinhu Li, Dongfang Wang, Fei Tang, Xinnan Ling, Wenjie Zhang, Zemin Zhang
          </td>
          <td>2024-07-11</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology. In this study, we developed SpatialKNife (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. The SKNY algorithm identified tumour spatial domains from spatial transcriptomic data of breast cancer, followed by clustering of these domains, trajectory estimation, and spatial extension to the tumour microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed endothelial cell and macrophage infiltration into the TME at mid-stage progression. Our results suggest that analysis using the spatial domain as a unit reflects pathological mechanisms in the TME. This approach may be applicable to the biological estimation of diverse microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37e699643db70ef39e651b4f4b87acc5b96626c" target='_blank'>
              Spatial domain analysis to estimate spatiotemporal pathological mechanisms in microenvironment with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are a heterogeneous population of cells whose phenotypes and functions are shaped by factors that are incompletely understood. Herein, we asked when and where TAMs arise from blood monocytes and how they evolve during tumor development. We initiated pancreatic ductal adenocarcinoma (PDAC) in inducible monocyte fate-mapping mice and combined single-cell transcriptomics and high-dimensional flow cytometry to profile the monocyte-to-TAM transition. We revealed that monocytes differentiate first into a transient intermediate population of TAMs that generates two longer-lived lineages of terminally differentiated TAMs with distinct gene expression profiles, phenotypes, and intratumoral localization. Transcriptome datasets and tumor samples from patients with PDAC evidenced parallel TAM populations in humans and their prognostic associations. These insights will support the design of new therapeutic strategies targeting TAMs in PDAC. Intermediate tumor-associated macrophages (TAMs) emerge first and give rise to two more differentiated TAM populations in pancreatic cancer. Editor’s summary Tumor-associated macrophages (TAMs) promote tumor growth and immune suppression, but how TAMs develop from circulating monocytes and local tissue-resident macrophages remains incompletely understood. Using a model of pancreatic ductal adenocarcinoma (PDAC) in monocyte fate-mapping mice, Dunsmore et al. examined the dynamics of the monocyte-to-macrophage transition in tumors. A population of “intermediate” TAMs gave rise to two more differentiated TAM subsets, distinguishable by their distinct surface marker phenotype, dependence on the transcription factor Maf, migratory behavior, and localization in tumors. These findings provide a time- and space-resolved picture of TAM heterogeneity and development in PDAC, which could guide the future development of TAM-targeted therapeutic strategies. —Claire Olingy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a54c86dbe269d24ca8abdd22c4597319660f70" target='_blank'>
              Timing and location dictate monocyte fate and their transition to tumor-associated macrophages
              </a>
            </td>
          <td>
            G. Dunsmore, Wei Guo, Ziyi Li, D. Bejarano, Rhea Pai, Katharine Yang, Immanuel Kwok, Leonard Tan, M. Ng, Carlos De La Calle Fabregat, A. Yatim, A. Bougouin, Kevin Mulder, Jake Thomas, Javiera Villar, Mathilde Bied, Benoit Kloeckner, C. Dutertre, Grégoire Gessain, S. Chakarov, Zhaoyuan Liu, J. Scoazec, A. Lennon-Duménil, Thomas Marichal, C. Sautès-Fridman, W. Fridman, Ankur Sharma, Bing Su, A. Schlitzer, Lai Guan Ng, Camille Blériot, Florent Ginhoux
          </td>
          <td>2024-07-26</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="Background. Interactions among tumor, immune, and vascular niches play major roles in driving glioblastoma (GBM) malignancy and treatment responses. The composition, heterogeneity, and localization of extracellular core matrix proteins (CMPs) that mediate such interactions, however, are not well understood. Methods. Here, through computational genomics and proteomics approaches, we analyzed the functional and clinical relevance of CMP expression in GBM at bulk, single cell, and spatial anatomical resolution. Results. We identified genes encoding CMPs whose expression levels categorize GBM tumors into CMP expression-high (M-H) and CMP expression-low (M-L) groups. CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, infiltration of pro-tumor immune cells, and immune checkpoint gene expression. Anatomical and single-cell transcriptome analyses indicate that matrisome gene expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells driving GBM progression. Finally, we identified a 17-gene CMP expression signature, termed Matrisome 17 (M17) signature that further refines the prognostic value of CMP genes. The M17 signature is a significantly stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, potentially predicts responses to PD1 blockade. Conclusion. The matrisome gene expression signature provides a robust stratification of GBM patients by survival and potential biomarkers of functionally relevant GBM niches that can mediate mesenchymal-immune cross talk. Patient stratification based on matrisome profiles can contribute to selection and optimization of treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0881b040066587c281ed9ef82afae916e63793e" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Z. Dereli, Zheng Yin, Elisabeth K. Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach-Carson, Robert Rostomily, Anil Korkut
          </td>
          <td>2024-06-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Peripheral T-cell lymphomas (PTCL) are heterogeneous and highly aggressive hematologic malignancies with dismal outcomes, highlighting the need for novel targeted therapies. PTCLs are characterized by complex tumor-microenvironment (TME) ecosystems with low tumor cell content, admixed with rich immune infiltrates and stromal elements correlated with clinical outcomes. Direct targeting of the TME is an attractive strategy for PTCLs; however, the specific mechanisms that regulate the interdependence between lymphoma and TME remain poorly understood, hampering the development of TME-directed therapies. Our previous work based on driver genetic alterations in PTCL patients and genetic mouse models identified an important role of these driver mutations in remodeling the TME (Cortes et al., 2022). To address the specific role and mechanisms of tumor cell-TME interactions in the pathogenesis of PTCL, we developed a novel experimental platform and cutting-edge computational methodologies using single-nucleus RNAseq analysis to comprehensively profile lymphoma cells and their microenvironment from 27 PTCL patient samples of two major nodal pathologic subtypes, including 18 Nodal T follicular helper cell lymphomas (TFHcL) and 9 PTCL, not otherwise specified (PTCL, NOS). Our approach allowed the identification and in-depth characterization of tumor cells, lymphoid and myeloid TME, and vascular endothelium and non-endothelial stromal cells, which were challenging to capture with conventional single-cell RNAseq. The malignant T-cells in PTCL, NOS were grouped into patient-specific clusters demonstrating significant inter-tumor heterogeneity associated with diverse and frequently complex chromosomal abnormalities identified by copy number variation (CNV) analysis. In contrast, malignant TFHcL cells show a low degree to no apparent chromosomal abnormalities and divide into two major transcriptional subclusters, with tumor cells from the same patient falling into one of the two subclusters. Additionally, genomic profiles of TFHcL tumors revealed that the inter-tumor transcriptional heterogeneity of TFHcL tumor cells was strongly associated with the driver RHOA G17V mutation status. To profile oncogenic pathways critical for PTCL growth, we compared PTCL tumor cells to normal CD4 T-cells from reactive lymphoid tissue. PTCL, NOS and TFHcL tumor cells were equally enriched in the TCR signaling pathway, mitosis, and cell cycle related genes. However, exploitation of KRAS and cytokine-driven inflammatory signals was prevalent in TFHcL but not PTCL, NOS tumor cells. Analysis of the immune TME identified an increase in T regulatory cells, CD8 T-cells, immunoblasts, and macrophages in TFHcL. Furthermore, while dysregulated B-cells with heightened cytokine-driven inflammatory signals were unique to TFHcL, inflammatory macrophages and classic dendritic cells (cDC) were prominent in the TME of both subtypes. Our results uncover shared and subtype-specific pathogenic features in PTCL and pave the way for designing novel precision therapies for PTCL patients.
 Citation Format: Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig Soderquist, Ryan D Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero. Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17bf46cac83bb1af31141fb58102ffe6ac218af" target='_blank'>
              Abstract PR05: Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas
              </a>
            </td>
          <td>
            Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B. Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig R. Soderquist, Ryan Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) mortality rates continue to increase faster than those of other cancer types due to high heterogeneity, which limits diagnosis and treatment. Pathological and molecular subtyping have identified that HCC tumors with poor outcomes are characterized by intratumoral collagenous accumulation. However, the translational and post-translational regulation of tumor collagen, which is critical to the outcome, remains largely unknown. Here, we investigate the spatial extracellular proteome to understand the differences associated with HCC tumors defined by Hoshida transcriptomic subtypes of poor outcome (Subtype 1; S1; n = 12) and better outcome (Subtype 3; S3; n = 24) that show differential stroma-regulated pathways. Collagen-targeted mass spectrometry imaging (MSI) with the same-tissue reference libraries, built from untargeted and targeted LC-MS/MS was used to spatially define the extracellular microenvironment from clinically-characterized, formalin-fixed, paraffin-embedded tissue sections. Collagen α-1(I) chain domains for discoidin-domain receptor and integrin binding showed distinctive spatial distribution within the tumor microenvironment. Hydroxylated proline (HYP)-containing peptides from the triple helical regions of fibrillar collagens distinguished S1 from S3 tumors. Exploratory machine learning on multiple peptides extracted from the tumor regions could distinguish S1 and S3 tumors (with an area under the receiver operating curve of ≥0.98; 95% confidence intervals between 0.976 and 1.00; and accuracies above 94%). An overall finding was that the extracellular microenvironment has a high potential to predict clinically relevant outcomes in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53efde0c062c9d0f5b393a6e975a03778bb11cae" target='_blank'>
              The Spatial Extracellular Proteomic Tumor Microenvironment Distinguishes Molecular Subtypes of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Jade K Macdonald, Harrison B Taylor, Mengjun Wang, A. DelaCourt, Christin Edge, David N. Lewin, Naoto Kubota, Naoto Fujiwara, Fahmida Rasha, Cesia A. Marquez, A. Ono, Shiro Oka, K. Chayama, Sara Lewis, B. Taouli, Myron Schwartz, M. Fiel, Richard R Drake, Y. Hoshida, A. Mehta, Peggi M Angel
          </td>
          <td>2024-07-09</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Abstract Objectives Non‐small‐cell lung carcinoma (NSCLC) is the most prevalent and lethal form of lung cancer. The need for biomarker‐informed stratification of targeted therapies has underpinned the need to uncover the underlying properties of the tumor microenvironment (TME) through high‐plex quantitative assays. Methods In this study, we profiled resected NSCLC tissues from 102 patients by targeted spatial proteomics of 78 proteins across tumor, immune activation, immune cell typing, immune‐oncology, drug targets, cell death and PI3K/AKT modules to identify the tumor and stromal signatures associated with overall survival (OS). Results Survival analysis revealed that stromal CD56 (HR = 0.384, P = 0.06) and tumoral TIM3 (HR = 0.703, P = 0.05) were associated with better survival in univariate Cox models. In contrast, after adjusting for stage, BCLXL (HR = 2.093, P = 0.02) and cleaved caspase 9 (HR = 1.575, P = 0.1) negatively influenced survival. Delta testing indicated the protective effect of TIM‐3 (HR = 0.614, P = 0.04) on OS. In multivariate analysis, CD56 (HR = 0.172, P = 0.001) was associated with better survival in the stroma, while B7.H3 (HR = 1.72, P = 0.008) was linked to poorer survival in the tumor. Conclusions Deciphering the TME using high‐plex spatially resolved methods is giving us new insights into compartmentalised tumor and stromal protein signatures associated with clinical endpoints in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c86af101b13bc4f2f501f32d2e227ba48bb3c7a" target='_blank'>
              Spatial proteomic profiling of tumor and stromal compartments in non‐small‐cell lung cancer identifies signatures associated with overall survival
              </a>
            </td>
          <td>
            Vahid Yaghoubi Naei, J. Monkman, Habib Sadeghirad, Ahmed M Mehdi, Tony Blick, W. Mullally, K. O’Byrne, M. Warkiani, A. Kulasinghe
          </td>
          <td>2024-01-01</td>
          <td>Clinical & Translational Immunology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b06482a1333f1d8cf1f744297d83ab314bc5a5" target='_blank'>
              High-dimensional mapping of human CEACAM1 expression on immune cells and association with melanoma drug resistance
              </a>
            </td>
          <td>
            Yu‐Hwa Huang, Charles H Yoon, Amit Gandhi, Thomas Hanley, Carlos Castrillon, Yasuyuki Kondo, Xi Lin, Walter Kim, Chao Yang, Amine Driouchi, M. Carroll, S. Gray-Owen, D. Wesemann, Charles G. Drake, Monica M Bertagnolli, Nicole Beauchemin, Richard S Blumberg
          </td>
          <td>2024-07-02</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id=". The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest– derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2734d6282129acbb89725ff7878ceaacbfcfa19d" target='_blank'>
              Revealing the crucial roles of suppressive immune microenvironment in cardiac myxoma progression
              </a>
            </td>
          <td>
            Zedong Jiang, Qianlong Kang, Hong Qian, Zhijie Xu, Huan Tong, Jiaqing Yang, Li Li, Renwei Li, Guangqi Li, Fei Chen, Nan Lin, Yunuo Zhao, Huashan Shi, Juan Huang, Xuelei Ma
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

Targeted therapies have improved the clinical outcomes of most patients
with cancer. However, the heterogeneity of gastric cancer remains a major hurdle for precision
treatment. Further investigations into tumor microenvironment heterogeneity are required to resolve
these problems.



In this study, bioinformatic analyses, including metabolism analysis, pathway enrichment,
differentiation trajectory inference, regulatory network construction, and survival analysis,
were applied to gain a comprehensive understanding of tumor microenvironment biology within
gastric cancer using single-cell RNA-seq and public datasets and experiments were carried out to
confirm the conclusions of these analyses.



We profiled heterogeneous single-cell atlases and identified eight cell populations with
differential expression patterns. We identified two cancer-associated fibroblasts (CAFs) subtypes,
with particular emphasis on the role of inflammatory cancer-associated fibroblasts (iCAFs) in
EMT and lipid metabolic crosstalk within the tumor microenvironment. Notably, we detected two
differentiation states of iCAFs that existed in different tissues with discrepant expression of genes
involved in immuno-inflammation or ECM remodeling. Moreover, investigation of tumor-infiltrating
myeloid cells has revealed the functional diversity of myeloid cell lineages in gastric cancer.
Of which a proliferative cell lineage named C1QC+MKI67+TAMs was recognized with high immunosuppressive
capacities, suggesting it has immune suppression and cell proliferation functions
in the tumor niche. Finally, we explored regulatory networks based on ligand-receptor pairs and
found crucial pro-tumor crosstalk between CAFs and myeloid cells in the tumor microenvironment
(TME).



These findings provide insights for future cancer treatments and drug discovery
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a780498a7ac762e0b80f117551648fde3a8d7a" target='_blank'>
              Crosstalk Between Cancer-associated Fibroblasts and Myeloid Cells
Shapes the Heterogeneous Microenvironment of Gastric Cancer
              </a>
            </td>
          <td>
            Zhiwei Peng, Can Fang, Zhiwei Tong, Qiufan Rao, Zihao Ren, K. Hu
          </td>
          <td>2024-06-11</td>
          <td>Current Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Single-cell analysis has become an essential tool in modern biological research, providing unprecedented insights into cellular behavior and heterogeneity. By examining individual cells, this approach surpasses conventional population-based methods, revealing critical variations in cellular states, responses to environmental cues, and molecular signatures. In the context of cancer, with its diverse cell populations, single-cell analysis is critical for investigating tumor evolution, metastasis, and therapy resistance. Understanding the phenotype-genotype relationship at the single-cell level is crucial for deciphering the molecular mechanisms driving tumor development and progression. This review highlights innovative strategies for selective cell isolation based on desired phenotypes, including robotic aspiration, laser detachment, microraft arrays, optical traps, and droplet-based microfluidic systems. These advanced tools facilitate high-throughput single-cell phenotypic analysis and sorting, enabling the identification and characterization of specific cell subsets, thereby advancing therapeutic innovations in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1bb0534cf6f72176c966878e2924974c47306ed" target='_blank'>
              Advances in Single-Cell Techniques for Linking Phenotypes to Genotypes
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yushu Ma, Jinxiong Cheng, Yu-Chih Chen
          </td>
          <td>2024-07-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the spatial dynamics within tissue microenvironments is crucial for deciphering cellular interactions and molecular signaling in living systems. These spatial characteristics govern cell distribution, extracellular matrix components, and signaling molecules, influencing local biochemical and biophysical conditions. Decoding these features offers insights into physiological processes, disease progression, and clinical outcomes. By elucidating spatial relationships between cell types, researchers uncover tissue architecture, cell communication networks, and microenvironment dynamics, aiding in the identification of biomarkers and therapeutic targets. Digital pathology imaging, including Hematoxylin and Eosin (H&E) staining, provides high-resolution histological information that offer intricate insights into cell-cell spatial relationships with greater details. However, current methods for capturing cell-cell spatial interactions are constrained by either methodological scopes or implementations restricted to script-level access. This limitation undermines generalizability and standardization, crucial for ensuring reproducibility. To address these limitations, we introduce SpatialQPFs, an extendable R package designed for extraction of interpretable spatial features from digital pathology images. By leveraging segmented cell information, our package provides researchers with a comprehensive toolkit for applying a range of spatial statistical methods within a stochastic process framework which includes analysis of point pattern data, areal data, and geostatistical data. This allows for a thorough analysis of cell spatial relationships, enhancing the depth and accuracy of spatial insights derived from the tissue, thereby empowering researchers to conduct comprehensive spatial analyses efficiently and reproducibly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da34c04318b2cac3bb2622ebe7b27ef2fe040d" target='_blank'>
              SpatialQPFs: An R package for deciphering cell-cell spatial relationship
              </a>
            </td>
          <td>
            Xiao Li
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. This study enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained genomic features and histopathological traits of primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, correlating ex vivo drug sensitivity with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program is associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributes to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine. One Sentence Summary This study establishes patient-derived tumor organoids (PDOs) from 31 Head and Neck Squamous Cell Carcinoma (HNSCC) patients, faithfully recapitulating characteristics of primary tumors and accurately predicting clinical responses to cisplatin treatment. We reveal intertumoral heterogeneity within PDOs and a hybrid epithelial-mesenchymal transition (hEMT) program conferring cisplatin resistance, highlighting amphiregulin (AREG) as a regulator of cellular plasticity and potential therapeutic target for HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82266d25cc2972a0e3e79d403bbffccb3c4b2f76" target='_blank'>
              Genomic and single-cell characterization of patient-derived tumor organoid models of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Young Min Park, De-Chen Lin
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b652a22909b12491feaf9d1745a2154d89fbc8e0" target='_blank'>
              Immune profiling of mouse lung adenocarcinoma paraffin tissues using multiplex immunofluorescence panel: a pilot study
              </a>
            </td>
          <td>
            Jie Zhai, A. Tamegnon, Mei Jiang, R. Pandurengan, Edwin Roger Parra
          </td>
          <td>2024-06-14</td>
          <td>Laboratory Animal Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immunotherapies have shown promising results in treating patients with hematological malignancies like multiple myeloma, which is an incurable but treatable bone marrow-resident plasma cell cancer. Choosing the most efficacious treatment for a patient remains a challenge in such cancers. However, pre-clinical assays involving patient-derived tumor cells co-cultured in an ex vivo reconstruction of immune-tumor micro-environment have gained considerable notoriety over the past decade. Such assays can characterize a patient’s response to several therapeutic agents including immunotherapies in a high-throughput manner, where bright-field images of tumor (target) cells interacting with effector cells (T cells, Natural Killer (NK) cells, and macrophages) are captured once every 30 minutes for upto six days. Cell detection, tracking, and classification of thousands of cells of two or more types in each frame is bound to test the limits of some of the most advanced computer vision tools developed to date and requires a specialized approach. We propose TLCellClassifier (time-lapse cell classifier) for live cell detection, cell tracking, and cell type classification, with enhanced accuracy and efficiency obtained by integrating convolutional neural networks (CNN), metric learning, and long short-term memory (LSTM) networks, respectively. State-of-the-art computer vision software like KTH-SE and YOLOv8 are compared with TLCellClassifier, which shows improved accuracy in detection (CNN) and tracking (metric learning). A two-stage LSTM-based cell type classification method is implemented to distinguish between multiple myeloma (tumor/target) cells and macrophages/monocytes (immune/effector cells). Validation of cell type classification was done both using synthetic datasets and ex vivo experiments involving patient-derived tumor/immune cells. Availability and implementation https://github.com/QibingJiang/cell classification ml">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0af147d9156c5d559b6eeac8f7b6ba9f3178525c" target='_blank'>
              TLCellClassifier: Machine Learning Based Cell Classification for Bright-Field Time-Lapse Images
              </a>
            </td>
          <td>
            Qibing Jiang, P. Sudalagunta, M. Meads, Xiaohong Zhao, Alexandra Achille, David Noyes, Maria Silva, Rafael R. Canevarolo, Ken Shain, Ariosto Silva, Wei Zhang
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND AIMS
Surgical resection serves as the principal curative strategy for hepatocellular carcinoma (HCC), yet the incidence of postoperative recurrence remains alarmingly high. However, the spatial molecular structural alterations contributing to postoperative recurrence in HCC are still poorly understood.


APPROACH RESULTS
We employed imaging mass cytometry to profile the in-situ expression of 33 proteins within 358,729 single cells of 92 clinically annotated surgical specimens from 46 patients who were treated with surgical resections for primary and relapsed tumors. We revealed the recurrence progression of HCC was governed by the dynamic spatial distribution and functional interplay of diverse cell types across adjacent normal, tumor margin, and intratumor regions. Our exhaustive analyses revealed an aggressive, immunosuppression-related spatial ecosystem in relapsed HCC. Additionally, we illustrated the prominent implications of the TME of tumor margins in association with relapse HCC. Moreover, we identified a novel subpopulation of dendritic cells (PDL1+CD103+ DCs) enriched in the peritumoral area that correlated with early postoperative recurrence, which was further validated in an external cohort. Through the analysis of scRNA-seq data, we found the interaction of PDL1+CD103+ DCs with regulatory T cells and exhausted T cells enhanced immunosuppression and immune escape via multiple ligand-receptor pathways.


CONCLUSIONS
We comprehensively depicted the spatial landscape of single-cell dynamics and multicellular architecture within primary and relapsed HCC. Our findings highlight spatial organization is a prominent determinant of HCC recurrence and provide a valuable insight into the immune evasion mechanisms driving recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a63dde0bf07aaed3ddea67316c202ee6463051a" target='_blank'>
              Spatial proteomic landscape of primary and relapsed hepatocellular carcinoma reveals immune escape characteristics in early relapse.
              </a>
            </td>
          <td>
            Meilin Yang, Xiaoyi Song, Fan Zhang, Mingan Li, Wuguang Chang, Zheyan Wang, Man Li, Hong Shan, Dan Li
          </td>
          <td>2024-06-20</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Local inflammation in the pancreas is transient but imprints a durable epigenetic memory on epithelial cells, making them more amenable to oncogenic transformation. However, it is unclear whether epithelial cell heterogeneity is impacted by acute pancreatitis (AP) or whether population dynamics during regeneration contributes to the establishment of inflammation memory. To tackle those questions, we deployed experimental pancreatitis in mice and performed paired sequencing of transcriptomic and chromatin accessibility profiles at single nucleus resolution. We documented cell type abundance but also applied integrative analyses to infer phenotypically-distinct clusters of mesenchymal and exocrine cells. We found that AP perturbs a subset of “idling” acinar cells, which separate from more canonical “secretory” acini based on a more diversified proteome, which include elevated expression of signal transduction receptors. We linked acinar cell heterogeneity to epigenetic differences that also endow idling cells with superior plasticity. These constitute about 40% of acinar cells but can proliferate and skew their phenotype in response to AP. This leads to a remarkable recovery of pancreas histology and function, but also to the dissemination of idling-like features across the exocrine parenchyma. Mechanistically, idling acinar cells are characterized by enhanced transcriptional activity and protein synthesis. After recovery from pancreatitis, acini show elevation of both and establishment of chronic Unfolded Protein Response (UPR). We finally demonstrated that AP-primed pancreata show signs of elevated UPR and that ER stress promotes acinar cell metaplasia. Our data interrogate phenotypical dynamics during tissue regeneration to identify cell states amenable to epigenetic imprinting. They also suggest that UPR-alleviating strategies might curtail the risk of developing pancreatic cancer for individuals who experiences AP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0889f4e0ef737079e14a1df3e2a78bed4691d30d" target='_blank'>
              Population dynamics after pancreatitis dictates long-lasting epigenetic reprogramming and mediates tumor predisposition
              </a>
            </td>
          <td>
            Marco Fantuz, J. Liebig, Giovanni Fontana, Valerio Iebba, Carmine Carbone, Sören Lukassen, Christian Conrad, Alessandro Carrer
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e14b920d1cfc1c8266bd8525196b4f1f7d022cc" target='_blank'>
              Unveiling novel cell clusters and biomarkers in glioblastoma and its peritumoral microenvironment at the single-cell perspective
              </a>
            </td>
          <td>
            Liping Wang, Xinyi Li, Chengshi Xu, Danwen Wang, Chao Ma, Zefen Wang, Yirong Li, Zhiqiang Li
          </td>
          <td>2024-06-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The genomic landscape of clear cell renal cell carcinoma (ccRCC) has a considerable intra-tumor heterogeneity, which is a significant obstacle in the field of precision oncology and plays a pivotal role in metastasis, recurrence, and therapeutic resistance of cancer. The mechanisms of intra-tumor heterogeneity in ccRCC have yet to be fully established. We integrated single-cell RNA sequencing (scRNA-seq) and transposase-accessible chromatin sequencing (scATAC-seq) data from a single-cell multi-omics perspective. Based on consensus non-negative matrix factorization (cNMF) algorithm, functionally heterogeneous cancer cells were classified into metabolism, inflammatory, and EMT meta programs, with spatial transcriptomics sequencing (stRNA-seq) providing spatial information of such disparate meta programs of cancer cells. The bulk RNA sequencing (RNA-seq) data revealed high clinical prognostic values of functionally heterogeneous cancer cells of three meta programs, with transcription factor regulatory network and motif activities revealing the key transcription factors that regulate functionally heterogeneous ccRCC cells. The interactions between varying meta programs and other cell subpopulations in the microenvironment were investigated. Finally, we assessed the sensitivity of cancer cells of disparate meta programs to different anti-cancer agents. Our findings inform on the intra-tumor heterogeneity of ccRCC and its regulatory networks and offers new perspectives to facilitate the designs of rational therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56d844ad93caafe1a1fe2435935aca07cee4238" target='_blank'>
              Unfolding the mysteries of heterogeneity from a high-resolution perspective: integration analysis of single-cell multi-omics and spatial omics revealed functionally heterogeneous cancer cells in ccRCC
              </a>
            </td>
          <td>
            Jie Zheng, Wenhao Lu, Chengbang Wang, Shaohua Chen, Qingyun Zhang, Cheng Su
          </td>
          <td>2024-06-26</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Previous studies of hematopoietic stem cells (HSCs) primarily focused on single cell-based niche models, yielding fruitful but conflicting findings1–5. Here we report our investigation on the fetal liver (FL) as the primary fetal hematopoietic site using spatial transcriptomics. Our study reveals two distinct niches: the portal-vessel (PV) niche and the sinusoidal niche. The PV niche, composing N-cadherin (N-cad)HiPdgfrα+ mesenchymal stromal cells (MSCs), endothelial cells (ECs), and N-cadLoAlbumin+ hepatoblasts, maintains quiescent and multipotential FL-HSCs. Conversely, the sinusoidal niche, comprising ECs, hepatoblasts and hepatocytes, as well as potential macrophages and megakaryocytes, supports proliferative FL-HSCs biased towards myeloid lineages. Unlike prior reports on the role of Cxcl12, with its depletion from vessel-associated stromal cells leading to 80% of HSCs’ reduction in the adult bone marrow (BM)6,7, depletion of Cxcl12 via Cdh2CreERT (encoding N-cad) induces altered localization of HSCs from the PV to the sinusoidal niches, resulting in an increase of HSC number but with myeloid-bias. Similarly, we discovered that adult BM encompasses two niches within different zones, each composed of multi-cellular components: trabecular bone area (TBA, or metaphysis) supporting deep-quiescent HSCs, and central marrow (CM, or diaphysis) fostering heterogenous proliferative HSCs. This study transforms our understanding of niches by shifting from single cell-based to multicellular components within distinct zones, illuminating the intricate regulation of HSCs tailored to their different cycling states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bae823022e6e79b47c848c8afcb1e9ad22dac6d" target='_blank'>
              Characterization of Multicellular Niches Supporting Hematopoietic Stem Cells Within Distinct Zones
              </a>
            </td>
          <td>
            Ruochen Dong, Hua Li, Xi He, Chen Wang, Anoja Perera, Seth Malloy, Jonathon Russell, Wenting Li, Kaitlyn Petentler, Xinjian Mao, Zhe Yang, Michael Epp, Kate E. Hall, Allison Scott, M. McKinney, Shengping Huang, Sarah E Smith, Mark Hembree, Yongfu Wang, Zulin Yu, Jeff Haug, Jay Unruh, Brian Slaughter, XunLei Kang, Linheng Li
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Severe Coronavirus disease 2019 (COVID-19) induces heterogeneous and progressive diffuse alveolar damage (DAD) highly disrupting lung tissue architecture and homeostasis, hampering disease management leading to fatal outcomes. Characterizing DAD pathophysiology across disease progression is of ultimate importance to better understand the molecular and cellular features driving different DAD patterns and to optimize treatment strategies. To contextualize the interplay between cell types and assess their distribution, spatial transcriptomics (ST) techniques have emerged, allowing unprecedented resolution to investigate spatial architecture of tissues. To this end, post-mortem lung tissue provides valuable insights into cellular composition and their spatial relationships at the time of death. Here, we have leveraged VisumST technology in post-mortem COVID-19 induced acute and proliferative DAD lungs including control samples with normal morphological appearance, to unravel the immunopathological mechanisms underlying DAD, providing novel insights into cellular and molecular communication events driving DAD progression in fatal COVID-19. We report a progressive loss of endothelial cell types, pneumocytes type I and natural killer cells coupled with a continuous increase of myeloid and stromal cells, mostly peribronchial fibroblasts, over disease progression. Spatial organization analysis identified variable cellular compartments, ranging from major compartments defined by cell type lineages in control lungs to increased and more specific compartmentalization including immune-specific clusters across DAD spectrum. Importantly, spatially informed ligand-receptor interaction (LRI) analysis revealed an intercellular communication signature defining COVID-19 induced DAD lungs. Transcription factor (TF) activity enrichment analysis identified TGF-B pathway as DAD driver, highlighting SMAD3 and SMAD7 TFs activity role during lung fibrosis. Integration of deregulated LRIs and TFs activity allowed us to propose a downstream intracellular signaling pathway in peribronchial fibroblasts, suggesting potential novel therapeutic targets. Finally, spatio-temporal trajectories analysis provided insights into the alveolar epithelium regeneration program, characterizing markers of pneumocytes type II differentiation towards pneumocytes type I. In conclusion, we provide a spatial characterization of lung tissue architecture upon COVID-19 induced DAD progression, identifying molecular and cellular hallmarks that may help optimize treatment and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a4db747d52faae9abbf25e5031f73c4597d1cc" target='_blank'>
              Spatial transcriptomics unveils the in situ cellular and molecular hallmarks of the lung in fatal COVID-19
              </a>
            </td>
          <td>
            C. García-Prieto, Eva Musulen, V. Davalos, Gerardo Ferrer, D. Grases, E. Porta, Belén Pérez-Miés, T. Caniego-Casas, José Palacios, Xavier Saenz-Sardà, Elisabet Englund, Manel Esteller
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have become integral to treatment of non-small cell lung cancer (NSCLC). However, reliable biomarkers predictive of immunotherapy efficacy are limited. Here, we introduce HistoTME, a novel weakly supervised deep learning approach to infer the tumor microenvironment (TME) composition directly from histopathology images of NSCLC patients. We show that HistoTME accurately predicts the expression of 30 distinct cell type-specific molecular signatures directly from whole slide images, achieving an average Pearson correlation of 0.5 with the ground truth on independent tumor cohorts. Furthermore, we find that HistoTME-predicted microenvironment signatures and their underlying interactions improve prognostication of lung cancer patients receiving immunotherapy, achieving an AUROC of 0.75[95% CI: 0.61-0.88] for predicting treatment responses following first-line ICI treatment, utilizing an external clinical cohort of 652 patients. Collectively, HistoTME presents an effective approach for interrogating the TME and predicting ICI response, complementing PD-L1 expression, and bringing us closer to personalized immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08a8db8350cf11482e6ec8f05ba9999d5587863" target='_blank'>
              Predicting the Tumor Microenvironment Composition and Immunotherapy Response in Non-Small Cell Lung Cancer from Digital Histopathology Images
              </a>
            </td>
          <td>
            Sushant Patkar, Alex Chen, A. Basnet, Amber Bixby, Rahul Rajendran, Rachel Chernet, Susan Faso, Prashant A. Kumar, Devashish Desai, Ola El-Zammar, Christopher Curtiss, Saverio J. Carello, Michel Nasr, Peter Choyke, Stephanie Harmon, B. Turkbey, Tamara Jamaspishvili
          </td>
          <td>2024-06-12</td>
          <td>None</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Accumulating evidence suggests that the tumor immune microenvironment (TIME) significantly influences the response to immunotherapy, yet this complex relationship remains elusive. To address this issue, we developed TimiGP-Response (TIME Illustration based on Gene Pairing designed for immunotherapy Response), a computational framework leveraging single-cell and bulk transcriptomic data, along with response information, to construct cell-cell interaction networks associated with responders and estimate the role of immune cells in treatment response. This framework was showcased in triple-negative breast cancer treated with immune checkpoint inhibitors targeting the PD-1:PD-L1 interaction, and orthogonally validated with imaging mass cytometry. As a result, we identified CD8+ GZMB+ T cells associated with responders and its interaction with regulatory T cells emerged as a potential feature for selecting patients who may benefit from these therapies. Subsequently, we analyzed 3,410 patients with seven cancer types (melanoma, non-small cell lung cancer, renal cell carcinoma, metastatic urothelial carcinoma, hepatocellular carcinoma, breast cancer, and esophageal cancer) treated with various immunotherapies and combination therapies, as well as several chemo- and targeted therapies as controls. Using TimiGP-Response, we depicted the pan-cancer immune landscape associated with immunotherapy response at different resolutions. At the TIME level, CD8 T cells and CD4 memory T cells were associated with responders, while anti-inflammatory (M2) macrophages and mast cells were linked to non-responders across most cancer types and datasets. Given that T cells are the primary targets of these immunotherapies and our TIME analysis highlights their importance in response to treatment, we portrayed the pan-caner landscape on 40 T cell subtypes. Notably, CD8+ and CD4+ GZMK+ effector memory T cells emerged as crucial across all cancer types and treatments, while IL-17-producing CD8+ T cells were top candidates associated with immunotherapy non-responders. In summary, this study provides a computational method to study the association between TIME and response across the pan-cancer immune landscape, offering resources and insights into immune cell interactions and their impact on treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883be60621995b3d688f88f7d3f09de5740baf9e" target='_blank'>
              TimiGP-Response: the pan-cancer immune landscape associated with response to immunotherapy
              </a>
            </td>
          <td>
            Chenyang Li, Wei Hong, A. Reuben, Linghua Wang, Anirban Maitra, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins. Methods We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES- dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP- sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro. Results Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft. Conclusions Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states. Key Points Cellular plasticity influences the abundance of immunotherapeutic targets. Subtype-specific targets may be susceptible to epigenetically-mediated downregulation. Immunotherapeutic targets in development, B7-H3 and L1CAM, show “pan-subtype” expression. Importance of Study Neuroblastoma is a lethal childhood malignancy that shows cellular plasticity in response to anti-cancer therapies. Several plasma membrane proteins are being developed as immunotherapeutic targets in this disease. Here we define which cell surface proteins are susceptible to epigenetically regulated downregulation during an adrenergic to mesenchymal cell state switch and propose immunotherapeutic strategies to anticipate and circumvent acquired immunotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c43da3990dff1c35a4148b920ff54013c4ee8ec" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, F. Zammarchi, Patrick H van Berkel, Chi V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Interactions between mutant cells and their environment play a key role in determining cancer susceptibility. However, our understanding of how the pre-cancer microenvironment contributes to early tumorigenesis remains limited. Here, we show that newly emerging tumours at their most incipient stages shape their microenvironment in a critical process that determines their survival. Analysis of nascent squamous tumours in the upper gastrointestinal tract of the mouse reveals that the stress response of early tumour cells instructs the underlying mesenchyme to form a supportive “pre-cancer niche”, which dictates the long-term outcome of epithelial lesions. Stimulated fibroblasts beneath emerging tumours activate a wound healing response that triggers a dramatic remodelling of the underlying extracellular matrix, resulting in the formation of a fibronectin-rich stromal scaffold that promotes tumour growth. Functional heterotypic 3D culture assays and in vivo grafting experiments, combining carcinogen-free healthy epithelium and tumour-derived stroma, demonstrate that the pre-cancerous niche alone is sufficient to confer tumour properties to healthy epithelial cells. We propose a model where both mutations and the stromal response to genetic stress defines the likelihood of early tumours to survive and progress towards more advanced disease stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be23254ceedda48875f58bb5d700c5fd073374a7" target='_blank'>
              Pre-cancerous Niche Remodelling Dictates Nascent Tumour Survival
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. E. Rojo Arias, Y. Dang, S. Han, M. T. Bejar, B. Colom, J. C. Fowler, P. Jones, S. Rulands, B. D. Simons, M. P. Alcolea
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer/testis antigens (CTAs) are widely expressed in melanoma and lung cancer, emerging as promising targets for vaccination strategies and T-cell-based therapies in these malignancies. Despite recognizing the essential impact of intratumoral heterogeneity on clinical responses to immunotherapy, our understanding of intratumoral heterogeneity in CTA expression has remained limited. We employed single-cell mRNA sequencing to delineate the CTA expression profiles of cancer cells in clinically derived melanoma and lung cancer samples. Our findings reveal a high degree of intratumoral transcriptional heterogeneity in CTA expression. In melanoma, every cell expressed at least one CTA. However, most individual CTAs, including the widely used therapeutic targets NY-ESO-1 and MAGE, were confined to subpopulations of cells and were uncoordinated in their expression, resulting in mosaics of cancer cells with diverse CTA profiles. Coordinated expression was observed, however, mainly among highly structurally and evolutionarily related CTA genes. Importantly, a minor subset of CTAs, including PRAME and several members of the GAGE and MAGE-A families, were homogenously expressed in melanomas, highlighting their potential as therapeutic targets. Extensive heterogeneity in CTA expression was also observed in lung cancer. However, the frequency of CTA-positive cancer cells was notably lower and homogenously expressed CTAs were only identified in one of five tumors in this cancer type. Our findings underscore the need for careful CTA target selection in immunotherapy development and clinical testing and offer a rational framework for identifying the most promising candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59cb995a024b6b3c733ef9a23d2fbf82f85441" target='_blank'>
              Single-cell sequencing unveils extensive intratumoral heterogeneity of cancer/testis antigen expression in melanoma and lung cancer
              </a>
            </td>
          <td>
            S. Traynor, Mie K. Jakobsen, Tina M Green, Hana Komic, Y. Palarasah, C. B. Pedersen, Henrik J Ditzel, Fredrik B Thoren, Per Guldberg, M. Gjerstorff
          </td>
          <td>2024-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Immunotherapies have shown great promise in pleural mesothelioma (PM), yet most patients still do not achieve significant clinical response, highlighting the importance of improving understanding of the tumor microenvironment (TME). Here, we utilized high-throughput, single-cell RNA-sequencing to de novo identify 54 expression programs and construct a comprehensive cellular catalogue of the PM TME. We found four cancer-intrinsic programs associated with poor disease outcome and a novel fetal-like, endothelial cell population that likely responds to VEGF signaling and promotes angiogenesis. Throughout cellular compartments, we observe substantial difference in the TME associated with a cancer-intrinsic sarcomatoid signature, including enrichment in fetal-like endothelial cells, CXCL9+ macrophages, cytotoxic, exhausted, and regulatory T cells, which we validated using imaging and bulk deconvolution analyses on independent cohorts. Finally, we show, both computationally and experimentally, that NKG2A-HLA-E interaction between NK and tumor cells represents an important new therapeutic axis in PM, especially for epithelioid cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c84bfe3e9c66b621b1832492f79b2264c3e648" target='_blank'>
              Single cell view of tumor microenvironment gradients in pleural mesothelioma.
              </a>
            </td>
          <td>
            Bruno Giotti, Komal Dolasia, William Zhao, Peiwen Cai, Robert Sweeney, Elliot Merritt, Evgeny Kiner, Grace S Kim, Atharva Bhagwat, Thinh Nguyen, Samarth Hegde, Bailey G Fitzgerald, S. Shroff, T. Dawson, M. Garcia-Barros, Jamshid Abdul-Ghafar, Rachel Chen, Sacha Gnjatic, Alan Soto, Rachel Brody, S. Kim-Schulze, Zhihong Chen, Kristin G Beaumont, Miram Merad, R.M. Flores, R. Sebra, Amir Horowitz, Thomas U. Marron, Anna Tocheva, Andrea Wolf, A. Tsankov
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The incidence rate of intrahepatic cholangiocarcinoma (ICC), which has a poor prognosis, is rapidly increasing. To investigate the intratumor heterogeneity of ICC, we analyzed single-cell RNA sequencing data from the primary tumor and adjacent normal tissues of 14 treatment-naïve patients. We identified ten major cell types, along with 45 subclusters of cells. Notably, we identified a fibroblast cluster, Fibroblast_LUM+, which was preferably enriched in tumor tissues and actively interacted with cholangiocytes. LGALS1 was verified as a marker gene of Fibroblast_LUM+, contributing to the malignant phenotype of ICC. The higher amount of LGALS1 + fibroblasts were associated with poorer overall survival in ICC patients. LGALS1 + fibroblasts activated the proliferation and migration of tumor cells by upregulating the expression levels of CCR2, ADAM15, and β-integrin. Silencing LGALS1 in cancer-associated fibroblasts (CAFs) suppressed CAF-augmented tumor cell migration and invasion in vitro as well as tumor formation in vivo, suggesting that blockade of LGALS1 serves as a potential therapeutic approach for ICC. Taken together, our single-cell analysis provides insight into the interaction between malignant cells and specific subtypes of fibroblasts. Our work will further the understanding of the intratumor heterogeneity of ICC and provide novel strategies for the treatment of ICC by targeting fibroblasts in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e005ea2d0affd9b430e5c3563c36d159191c5026" target='_blank'>
              Single-cell analysis defines LGALS1 + fibroblasts that promote proliferation and migration of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Qiqi Cao, Jinxian Yang, Lixuan Jiang, Zhao Yang, Zhe-cai Fan, Shu-Zhen Chen, Sibo Zhu, Lei Yin, Hongyang Wang, Wen Wen
          </td>
          <td>2024-06-11</td>
          <td>Journal of molecular cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d31de7dddff96b795acb0a02ad70f6902f3e8f2" target='_blank'>
              Progress of single-cell RNA sequencing combined with spatial transcriptomics in tumour microenvironment and treatment of pancreatic cancer
              </a>
            </td>
          <td>
            Jie Zhu, Ke Zhang, Yuan Chen, Xinyu Ge, Junqing Wu, Peng Xu, Jie Yao
          </td>
          <td>2024-06-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86aae7f7f4b9ea46d5a7441fbb856a41092f79b4" target='_blank'>
              A YAP-centered mechanotransduction loop drives collective breast cancer cell invasion
              </a>
            </td>
          <td>
            Antoine A Khalil, D. Smits, Peter D. Haughton, T. Koorman, Karin A Jansen, Mathijs P. Verhagen, Mirjam C van der Net, Kitty van Zwieten, Lotte Enserink, Lisa Jansen, Abdelrahman G El-Gammal, Daan Visser, Milena Pasolli, Max Tak, Denise Westland, P. V. van Diest, C. Moelans, M. G. Roukens, Sandra Tavares, Anne-Marie Fortier, Morag Park, Riccardo Fodde, M. Gloerich, F. Zwartkruis, Patrick W B Derksen, J. de Rooij
          </td>
          <td>2024-06-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The interaction between gliomas and the immune system is poorly understood and thus hindering development of effective immunotherapies for glioma patients. The immune response is highly variable during tumor development, and affected by therapies such as surgery, radiation, and chemotherapy. Currently, analysis of these local changes is difficult due to poor accessibility of the tumor and high-morbidity of sampling. In this study, we developed a model for repeat-biopsy in mice to study these local immunological changes over time. Using fine needle biopsy we were able to safely and repeatedly collect cells from intracranial tumors in mice. Ultra-fast cycling technology (FAST) was used for multi-cycle immunofluorescence of retrieved cells, and provided insights in the changing immune response over time. The combination of these techniques can be utilized to study changes in the immune response in glioma or other intracranial diseases over time, and in response to treatment within the same animal. Teaser Fine-needle biopsy and ultra-fast cycling technology techniques were developed to allow for repeat sampling and analysis of glial tumors in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da1bd93129502cbdb331eecf978d2c9bf8be88b" target='_blank'>
              Probing the glioma micro-environment: analysis using biopsy in combination with ultra-fast cyclic immunolabeling
              </a>
            </td>
          <td>
            T. S. van Solinge, Juhyun Oh, E. Abels, Peter Koch, X. Breakefield, Ralph Weissleder, M. Broekman
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex ecosystem of diverse cell types whose interactions govern tumor growth and clinical outcome. While the TME’s impact on immunotherapy has been extensively studied, its role in chemotherapy response remains less explored. To address this, we developed DECODEM (DEcoupling Cell-type-specific Outcomes using DEconvolution and Machine learning), a generic computational framework leveraging cellular deconvolution of bulk transcriptomics to associate the gene expression of individual cell types in the TME with clinical response. Employing DECODEM to analyze the gene expression of breast cancer (BC) patients treated with neoadjuvant chemotherapy, we find that the gene expression of specific immune cells (myeloid, plasmablasts, B-cells) and stromal cells (endothelial, normal epithelial, CAFs) are highly predictive of chemotherapy response, going beyond that of the malignant cells. These findings are further tested and validated in a single-cell cohort of triple negative breast cancer. To investigate the possible role of immune cell-cell interactions (CCIs) in mediating chemotherapy response, we extended DECODEM to DECODEMi to identify such CCIs, validated in single-cell data. Our findings highlight the importance of active pre-treatment immune infiltration for chemotherapy success. The tools developed here are made publicly available and are applicable for studying the role of the TME in mediating response from readily available bulk tumor expression in a wide range of cancer treatments and indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec38ea4f715fa72ae5e3c095b32d63b6ede6629" target='_blank'>
              The expression patterns of different cell types and their interactions in the tumor microenvironment are predictive of breast cancer patient response to neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            S. R. Dhruba, S. Sahni, Binbin Wang, Di Wu, P. Rajagopal, Yael Schmidt, E. Shulman, Sanju Sinha, Stephen-John Sammut, Carlos Caldas, Kun Wang, E. Ruppin
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B. Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Simple Summary We utilized publicly available perturbation phosphoproteomic data to construct models elucidating cell state transitions across multiple breast cancer and normal breast tissue-derived cell lines. Employing a hybrid methodology, which integrates machine learning and mechanistic modeling, we separated luminal, basal, and normal cell states and revealed core networks that control cell state transitions. We determined causal connections within the core networks and developed interpretable mechanistic models that elucidated the drivers of cell phenotypes. Significantly, these models can predict synergistic drug combinations capable of potentially reversing oncogenic transformation in breast cancer cell lines. Our methodology will enable designer approaches to identify targeted perturbations that convert cell states and mechanistically underpin therapeutic interventions. Abstract Understanding signaling patterns of transformation and controlling cell phenotypes is a challenge of current biology. Here we applied a cell State Transition Assessment and Regulation (cSTAR) approach to a perturbation dataset of single cell phosphoproteomic patterns of multiple breast cancer (BC) and normal breast tissue-derived cell lines. Following a separation of luminal, basal, and normal cell states, we identified signaling nodes within core control networks, delineated causal connections, and determined the primary drivers underlying oncogenic transformation and transitions across distinct BC subtypes. Whereas cell lines within the same BC subtype have different mutational and expression profiles, the architecture of the core network was similar for all luminal BC cells, and mTOR was a main oncogenic driver. In contrast, core networks of basal BC were heterogeneous and segregated into roughly four major subclasses with distinct oncogenic and BC subtype drivers. Likewise, normal breast tissue cells were separated into two different subclasses. Based on the data and quantified network topologies, we derived mechanistic cSTAR models that serve as digital cell twins and allow the deliberate control of cell movements within a Waddington landscape across different cell states. These cSTAR models suggested strategies of normalizing phosphorylation networks of BC cell lines using small molecule inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/092c4f2c92e0be64eb32694e8a69506f512145ae" target='_blank'>
              Cell State Transition Models Stratify Breast Cancer Cell Phenotypes and Reveal New Therapeutic Targets
              </a>
            </td>
          <td>
            Oleksii S. Rukhlenko, Hiroaki Imoto, Ayush Tambde, Amy McGillycuddy, Philipp Junk, Anna Tuliakova, Walter Kolch, Boris N. Kholodenko
          </td>
          <td>2024-06-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Microscopic vascular invasion (VI) is predictive of recurrence and benefit from lobectomy in stage I lung adenocarcinoma (LUAD) but is difficult to assess in resection specimens and cannot be accurately predicted prior to surgery. Thus, new biomarkers are needed to identify this aggressive subset of stage I LUAD tumors. To assess molecular and microenvironment features associated with angioinvasive LUAD we profiled 162 resected stage I tumors with and without VI by RNA-seq and explored spatial patterns of gene expression in a subset of 15 samples by high-resolution spatial transcriptomics (stRNA-seq). Despite the small size of invaded blood vessels, we identified a gene expression signature of VI from the bulk RNA-seq discovery cohort (n=103) and found that it was associated with VI foci, desmoplastic stroma, and high-grade patterns in our stRNA-seq data. We observed a stronger association with high-grade patterns from VI+ compared with VI- tumors. Using the discovery cohort, we developed a transcriptomic predictor of VI, that in an independent validation cohort (n=60) was associated with VI (AUROC=0.86; p=5.42×10-6) and predictive of recurrence-free survival (HR=1.98; p=0.024), even in VI- LUAD (HR=2.76; p=0.003). To determine our VI predictor’s robustness to intra-tumor heterogeneity we used RNA-seq data from multi-region sampling of stage I LUAD cases in TRACERx, where the predictor scores showed high correlation (R=0.87, p<2.2×10-16) between two randomly sampled regions of the same tumor. Our study suggests that VI-associated gene expression changes are detectable beyond the site of intravasation and can be used to predict the presence of VI. This may enable the prediction of angioinvasive LUAD from biopsy specimens, allowing for more tailored medical and surgical management of stage I LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e11baff5192fc7ae49442b8a566204aad86b4c" target='_blank'>
              Identification of a gene expression signature of vascular invasion and recurrence in stage I lung adenocarcinoma via bulk and spatial transcriptomics
              </a>
            </td>
          <td>
            Dylan Steiner, Lila Sultan, Travis Sullivan, Hanqiao Liu, Sherry Zhang, Ashley LeClerc, Yuriy O Alekseyev, Gang Liu, Sarah A. Mazzilli, Jiarui Zhang, Kimberly Rieger-Christ, Eric J. Burks, Jennifer Beane, Marc E. Lenburg
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent advances in multi-modal algorithms have driven and been driven by the increasing availability of large image-text datasets, leading to significant strides in various fields, including computational pathology. However, in most existing medical image-text datasets, the text typically provides high-level summaries that may not sufficiently describe sub-tile regions within a large pathology image. For example, an image might cover an extensive tissue area containing cancerous and healthy regions, but the accompanying text might only specify that this image is a cancer slide, lacking the nuanced details needed for in-depth analysis. In this study, we introduce STimage-1K4M, a novel dataset designed to bridge this gap by providing genomic features for sub-tile images. STimage-1K4M contains 1,149 images derived from spatial transcriptomics data, which captures gene expression information at the level of individual spatial spots within a pathology image. Specifically, each image in the dataset is broken down into smaller sub-image tiles, with each tile paired with 15,000 – 30,000 dimensional gene expressions. With 4,293,195 pairs of sub-tile images and gene expressions, STimage-1K4M offers unprecedented granularity, paving the way for a wide range of advanced research in multi-modal data analysis an innovative applications in computational pathology, and beyond.1">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07b1c10691fed29957bacde1dd23164efd66218f" target='_blank'>
              STimage-1K4M: A histopathology image-gene expression dataset for spatial transcriptomics
              </a>
            </td>
          <td>
            Jiawen Chen, Muqing Zhou, Wenrong Wu, Jinwei Zhang, Yun Li, Didong Li
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Tumor heterogeneity is associated with poor prognosis and drug resistance, leading to therapeutic failure. Here, we aim to utilize tumor evolution analysis to decode the intra- and inter-tumoral heterogeneity of high-grade serous ovarian cancer (HGSOC), unraveling the correlation between tumor heterogeneity and prognosis as well as chemotherapy response through single-cell and spatial transcriptomic analysis. Methods: We collected and curated 28 HGSOC patients single-cell transcriptomic data from five datasets. Then, we developed a novel text mining-based machine learning approach to deconstruct the evolutionary patterns of tumor cell functions. This allowed us to identify key tumor-related genes within different evolutionary branches, elucidate the microenvironmental cell compositions that various functional tumor cells depend on, and analyze the intra- and inter-heterogeneity of tumors and their microenvironments in relation to prognosis and chemotherapy response in HGSOC patients. We further validated our findings in two spatial and seven bulk transcriptomic datasets, totally 1,030 patients. Results: By employing transcriptomic clusters as proxies for functional clonality, we identified a significant increase in tumor cell state heterogeneity, which was strongly correlated with patient prognosis and treatment response. Furthermore, increased intra- and inter-tumoral functional clonality was associated with the characteristics of cancer-associated fibroblast (CAF). We also found that the spatial proximity between CXCL12-positive CAF and tumor cells, mediated through the CXCL12/CXCR4 interaction, is highly positively correlated with poor prognosis and chemotherapy resistance in HGSOC. Finally, we constructed a panel of 24 genes through statistical modeling, that are highly correlated with CXCL12-positive fibroblasts and can predict both prognosis and chemotherapy response in HGSOC. Conclusions: Our study offers insights into the collective behavior of tumor cell communities in HGSOC, as well as potential drivers of tumor evolution in response to therapy. Functional analyses and experiments revealed a strong association between CXCL12-positive fibroblasts and tumor progression as well as treatment outcomes. Our findings provide an important theoretical basis for clinical HGSOC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87aba8d8692d6efde4883bed0b3dc632799a4739" target='_blank'>
              Decoding the impact of fibroblast heterogeneity on prognosis and drug resistance in high-grade serous ovarian cancer through tumor evolution analysis
              </a>
            </td>
          <td>
            Tingjie Wang, Lingxi Tian, Bing Wei, Jun Li, Cuiyun Zhang, Ruitao Long, Xiaofei Zhu, Yougai Zhang, Bo Wang, Jun Yang, Yongjun Guo
          </td>
          <td>2024-06-12</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a complex tumor microenvironment (TME) that promotes cancer progression. The tumor stroma is characterized by cancer-associated fibroblasts (CAFs) that secrete factors stimulating tumor growth, invasion, and immunosuppression. Macrophages tend to polarize toward a pro-tumor phenotype. Various other components, including endothelial cells, pericytes, and neural cells, likely contribute to the complexity of the PDAC TME. Epidemiological investigations have highlighted obesity as a risk factor for pancreatic cancer, suggesting that adipocytes may also play a significant role. This manuscript offers an overview of the main actors in TME, as well as dysregulated signaling pathways and gene mutations. It underlines how strategies modulating the TME hold promise for improving outcomes. Lastly, it describes how integrating multi-omic approaches may represent a future trajectory to explore novel therapeutic possibilities. Abstract Pancreatic ductal adenocarcinoma (PDAC) presents significant oncological challenges due to its aggressive nature and poor prognosis. The tumor microenvironment (TME) plays a critical role in progression and treatment resistance. Non-neoplastic cells, such as cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), contribute to tumor growth, angiogenesis, and immune evasion. Although immune cells infiltrate TME, tumor cells evade immune responses by secreting chemokines and expressing immune checkpoint inhibitors (ICIs). Vascular components, like endothelial cells and pericytes, stimulate angiogenesis to support tumor growth, while adipocytes secrete factors that promote cell growth, invasion, and treatment resistance. Additionally, perineural invasion, a characteristic feature of PDAC, contributes to local recurrence and poor prognosis. Moreover, key signaling pathways including Kirsten rat sarcoma viral oncogene (KRAS), transforming growth factor beta (TGF-β), Notch, hypoxia-inducible factor (HIF), and Wnt/β-catenin drive tumor progression and resistance. Targeting the TME is crucial for developing effective therapies, including strategies like inhibiting CAFs, modulating immune response, disrupting angiogenesis, and blocking neural cell interactions. A recent multi-omic approach has identified signature genes associated with anoikis resistance, which could serve as prognostic biomarkers and targets for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23cd29cbd6982ee3e4b132b1c93695ffca70902" target='_blank'>
              Decoding the Intricate Landscape of Pancreatic Cancer: Insights into Tumor Biology, Microenvironment, and Therapeutic Interventions
              </a>
            </td>
          <td>
            A. Argentiero, Alessandro Andriano, Ingrid Catalina Caradonna, Giulia de Martino, V. Desantis
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Durable serological memory following vaccination is critically dependent on the production and survival of long-lived plasma cells (LLPCs). Yet, the factors that control LLPC specification and survival remain poorly resolved. Using intravital two-photon imaging, we find that in contrast to most plasma cells (PCs) in the bone marrow (BM), LLPCs are uniquely sessile and organized into clusters that are dependent on APRIL, an important survival factor. Using deep, bulk RNA sequencing, and surface protein flow-based phenotyping, we find that LLPCs express a unique transcriptome and phenotype compared to bulk PCs, fine-tuning expression of key cell surface molecules, CD93, CD81, CXCR4, CD326, CD44, and CD48, important for adhesion and homing. Conditional deletion of Cxcr4 in PCs following immunization leads to rapid mobilization from the BM, reduced survival of antigen-specific PCs, and ultimately accelerated decay of antibody titer. In naïve mice, the endogenous LLPCs BCR repertoire exhibits reduced diversity, reduced somatic mutations, and increased public clones and IgM isotypes, particularly in young mice, suggesting LLPC specification is non-random. As mice age, the BM PC compartment becomes enriched in LLPCs, which may outcompete and limit entry of new PCs into the LLPC niche and pool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae2a0a86a1858977f32349e08a4ecc5beb84322" target='_blank'>
              Fine-tuning spatial-temporal dynamics and surface receptor expression support plasma cell-intrinsic longevity
              </a>
            </td>
          <td>
            Zhixin Jing, Phillip M. Galbo, Luis Ovando, Megan Demouth, Skylar Welte, Rosa Park, K. Chandran, Yinghao Wu, Thomas MacCarthy, Deyou Zheng, David R Fooksman
          </td>
          <td>2024-06-18</td>
          <td>eLife</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Efforts to cure BCR::ABL1 B cell acute lymphoblastic leukemia (Ph+ ALL) solely through inhibition of ABL1 kinase activity have thus far been insufficient despite the availability of tyrosine kinase inhibitors (TKIs) with broad activity against resistance mutants. The mechanisms that drive persistence within minimal residual disease (MRD) remain poorly understood and therefore untargeted. Utilizing 13 patient-derived xenograft (PDX) models and clinical trial specimens of Ph+ ALL, we examined how genetic and transcriptional features co-evolve to drive progression during prolonged TKI response. Our work reveals a landscape of cooperative mutational and transcriptional escape mechanisms that differ from those causing resistance to first generation TKIs. By analyzing MRD during remission, we show that the same resistance mutation can either increase or decrease cellular fitness depending on transcriptional state. We further demonstrate that directly targeting transcriptional state-associated vulnerabilities at MRD can overcome BCR::ABL1 independence, suggesting a new paradigm for rationally eradicating MRD prior to relapse. Finally, we illustrate how cell mass measurements of leukemia cells can be used to rapidly monitor dominant transcriptional features of Ph+ ALL to help rationally guide therapeutic selection from low-input samples. HIGHLIGHTS Relapse after remission on TKI can harbor mutations in ABL1, RAS, or neither Mutations and development-like cell state dictate fitness in residual disease Co-targeting cell state and ABL1 markedly reduces MRD Biophysical measurements provide an integrative, rapid measurement of cell state">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf85e0b581ae34776dbc829809704e7e0de66ff9" target='_blank'>
              Mutation and cell state compatibility is required and targetable in Ph+ acute lymphoblastic leukemia minimal residual disease
              </a>
            </td>
          <td>
            Peter S. Winter, Michelle L. Ramseier, Andrew W. Navia, Sachit D. Saksena, Haley Strouf, Nezha Senhaji, Alan DenAdel, Mahnoor Mirza, Hyun Hwan An, Laura L. Bilal, Peter Dennis, Catharine S. Leahy, Kay Shigemori, Jennyfer Galves-Reyes, Ye Zhang, Foster Powers, Nolawit Mulugeta, Alejandro J. Gupta, Nicholas L. Calistri, Alexandria van Scoyk, Kristen Jones, Huiyun Liu, Kristen Stevenson, Siyang Ren, M. Luskin, Charles P. Couturier, Ava P. Amini, Srivatsan Raghavan, Robert J. Kimmerling, Mark M. Stevens, Lorin Crawford, David M. Weinstock, S. Manalis, Alex K. Shalek, Mark A. Murakami
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d5310f7c7a3d332e8cba0823f8a46905b9d7fe7" target='_blank'>
              Utilizing human cerebral organoids to model breast cancer brain metastasis in culture
              </a>
            </td>
          <td>
            Chenran Wang, A. Nagayach, Harsh Patel, Lan Dao, Hui Zhu, Amanda R Wasylishen, Yanbo Fan, Ady Kendler, Ziyuan Guo
          </td>
          <td>2024-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The haematological malignancy multiple myeloma is associated with skewed T-cell activation and function. T-cell alterations are detectable in asymptomatic myeloma precursor conditions and have the potential to identify precursor patients at imminent risk of progression. However, what myeloma-associated T-cells alterations represent mechanistically, how they relate to tumour burden and gene expression, and what influences high inter-patient variability in immune composition remains unknown. Here, we assembled the largest ever dataset of published and newly-generated single-cell RNA and TCR sequencing of the marrow and blood from patients with myeloma, precursor conditions, and age-matched non-cancer controls. We show myeloma is not associated with T-cell exhaustion and instead defined by a pattern of T-cell differentiation resembling antigen-driven terminal memory differentiation. Myeloma-associated T-cell differentiation was dependent on tumour-intrinsic features including tumour burden and tumour expression of antigen-presentation genes. Expanded TCR clones accumulating in myeloma were not enriched for viral specificity and were detected in effector states in highly infiltrated marrows. Together, these results suggest anti-tumour immunity drives a novel form of cancer-associated T-cell memory differentiation in myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d674fe6f864f5fa613feeb41c61f5327c58bd541" target='_blank'>
              Tumour-intrinsic features shape T-cell differentiation through myeloma disease evolution
              </a>
            </td>
          <td>
            K. Foster, Elise Rees, L. Ainley, Eileen M. Boyle, Lydia Lee, Gwennan Ward, D. Galas-Filipowicz, Anna Mikolajczak, Emma J Lyon, Dylan Jankovic, Jasmine Rahman, Mahima Turakhia, Imran Uddin, Gordon Beattie, Yvette Hoade, Catherine Zhu, J. Reading, Ieuan G Walker, Michael Chapman, Karthik Ramasamy, Javier Herrero, B. Chain, S. Quezada, Kwee Yong
          </td>
          <td>2024-06-23</td>
          <td>None</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Achieving long-term disease control using therapeutic immunomodulation is a long-standing concept with a strong tradition in blood malignancies. Besides allogeneic hematopoietic stem cell transplantation that continues to provide potentially curative treatment for otherwise challenging diagnoses, recent years have seen impressive progress in immunotherapies for leukemias and lymphomas with immune checkpoint blockade, bispecific monoclonal antibodies, and CAR T cell therapies. Despite their success, non-response, relapse, and immune toxicities remain frequent, thus prioritizing the elucidation of the underlying mechanisms and identifying predictive biomarkers. The increasing availability of single-cell genomic tools now provides a system’s immunology view to resolve the molecular and cellular mechanisms of immunotherapies at unprecedented resolution. Here, we review recent studies that leverage these technological advancements for tracking immune responses, the emergence of immune resistance, and toxicities. As single-cell immune monitoring tools evolve and become more accessible, we expect their wide adoption for routine clinical applications to catalyze more precise therapeutic steering of personal immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a83dbce941baa359f9c950fa30e353ef39f9d564" target='_blank'>
              Single-cell genomics-based immune and disease monitoring in blood malignancies
              </a>
            </td>
          <td>
            Anja C Rathgeber, Leif S. Ludwig, Livius Penter
          </td>
          <td>2024-06-14</td>
          <td>Clinical Hematology International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Effective cancer immunotherapy relies on the clonal proliferation and expansion of CD8+ T cells in the tumor1,2. However, our insights into clonal expansions are limited, owing to an inability to track the same clones in tumors over time. Here, we developed a multi-tumor mouse model system to track hundreds of expanding and contracting CD8+ T cell clones over multiple timepoints in tumors of the same individual. Through coupling of clonal expansion dynamics and single-cell RNA/TCR-seq data, we identified a transcriptomic signature in PD-1+Ly108+ precursor exhausted cells3,4 that strongly predicts rates of intratumoral clone expansion in mice and humans. We found that expression of the signature successfully stratifies melanoma patient outcomes to PD-1/PD-L1 blockade5,6. Downregulation of the signature precedes clone contraction – a phase in which clones contract but maintain revivable precursor exhausted cells in the tumor. LAG-3 blockade – an FDA-approved therapy whose effects on CD8+ T cell responses are currently unclear7, re-activates the expansion signature, re-expanding pre-existing clones, including previously contracted clones. These findings reveal how the study of clonal expansion dynamics provide a powerful ‘pan-immunotherapy’ signature for monitoring immunotherapies, including PD-1/PD-L1 and LAG-3 blockade, with implications for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6681eced55a92fc5ceb65f99517fea6fa8caf1df" target='_blank'>
              LAG-3 blockade reactivates the CD8+ T cell expansion program to re-expand contracted clones in the tumor
              </a>
            </td>
          <td>
            Munetomo Takahashi, Mikiya Tsunoda, Hiroyasu Aoki, Masaki Kurosu, H. Ogiwara, S. Shichino, D. Bending, Shumpei Ishikawa, James E. D. Thaventhiran, Kouji Matsushima, S. Ueha
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors with dismal prognosis. One of the main bottlenecks for developing more effective therapies for glioblastoma stems from their histologic and molecular heterogeneity, leading to distinct tumor microenvironments and disease phenotypes. Effectively characterizing these features would improve the clinical management of glioblastoma. Glucose flux rates through glycolysis and mitochondrial oxidation have been recently shown to quantitatively depict glioblastoma proliferation in mouse models (GL261 and CT2A tumors, 38±3 mm3) using dynamic glucose-enhanced (DGE) deuterium spectroscopy. However, the spatial features of tumor microenvironment phenotypes remain hitherto unresolved. Here, we develop a DGE Deuterium Metabolic Imaging (DMI) approach for profiling tumor microenvironments through glucose conversion kinetics. Using a multimodal combination of tumor mouse models, novel strategies for spectroscopic imaging and noise attenuation, and histopathological correlations, we show that tumor lactate turnover mirrors phenotype differences between GL261 and CT2A mouse glioblastoma (59±7 mm3), whereas peritumoral glutamate-glutamine recycling is a potential marker of invasion capacity in pooled cohorts, linked to secondary brain lesions. Our findings were validated by histopathological characterization of each tumor, including cell density and proliferation, peritumoral infiltration, and distant migration. Our study bodes well for precision neuro-oncology, highlighting the importance of mapping glucose flux rates to better understand the metabolic heterogeneity of glioblastoma and its links to disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2927f90371bde6768c35753a6b358801ccb1fb15" target='_blank'>
              Deuterium Metabolic Imaging Phenotypes Mouse Glioblastoma Heterogeneity Through Glucose Turnover Kinetics
              </a>
            </td>
          <td>
            Rui V. Simões, Rafael N. Henriques, J. Olesen, Beatriz M Cardoso, Francisca F. Fernandes, Mariana A.V. Monteiro, S. Jespersen, Tânia Carvalho, N. Shemesh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution.
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding how cancer immunoediting sculpts tumor microenvironments is essential to disentangling tumor immune evasion mechanisms and developing immunotherapies. Here, we construct a comprehensive immunoediting map of human cancers via single-cell deconvolution of 11057 tumor-derived samples across 33 cancer types from TCGA and comparison with 17382 healthy samples across 30 tissues from GTEx. The map covers >1000 different cell states across all the major immune cell types. Mast cells, megakaryocytes, macrophages, neutrophils, plasma cells and T cells are up-regulated across a wide range of tumor types while natural killer cells and platelets are down-regulated in most tumor types, suggesting common cancer immunoediting events. While tumor heterogeneity is higher than the normal corresponding tissues, significant immune homogeneity exists among different tumor types compared with the distinct immune composition among normal tissues and organs. Our study provides a new holistic perspective to understanding cancer immunoediting. Our findings may provide important hints for developing novel cancer immunotherapies, and the high-resolution immunoediting map may serve as a rich resource for further pan-cancer investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfbdbbb07439c3b2cdbbf5a8ab406f56f64462e" target='_blank'>
              An immunoediting map of human cancers
              </a>
            </td>
          <td>
            Rui Gan, Xianwen Ren
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Melanoma, a highly malignant tumour, presents significant challenges due to its cellular heterogeneity, yet research on this aspect in cutaneous melanoma remains limited. In this study, we utilized single‐cell data from 92,521 cells to explore the tumour cell landscape. Through clustering analysis, we identified six distinct cell clusters and investigated their differentiation and metabolic heterogeneity using multi‐omics approaches. Notably, cytotrace analysis and pseudotime trajectories revealed distinct stages of tumour cell differentiation, which have implications for patient survival. By leveraging markers from these clusters, we developed a tumour cell‐specific machine learning model (TCM). This model not only predicts patient outcomes and responses to immunotherapy, but also distinguishes between genomically stable and unstable tumours and identifies inflamed (‘hot’) versus non‐inflamed (‘cold’) tumours. Intriguingly, the TCM score showed a strong association with TOMM40, which we experimentally validated as an oncogene promoting tumour proliferation, invasion and migration. Overall, our findings introduce a novel biomarker score that aids in selecting melanoma patients for improved prognoses and targeted immunotherapy, thereby guiding clinical treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceff46cb2d3d44eadc13e557a6116c6ae860759" target='_blank'>
              Unravelling tumour cell diversity and prognostic signatures in cutaneous melanoma through machine learning analysis
              </a>
            </td>
          <td>
            Wenhao Cheng, Ping Ni, Hao Wu, Xiaye Miao, Xiaodong Zhao, Dali Yan
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spatially resolved transcriptomics simultaneously measure the spatial location, histology images, and transcriptional profiles of the same cells or regions in undissociated tissues. Integrative analysis of multi-modal spatially resolved data holds immense potential for understanding the mechanisms of biology. Here we present a flexible multi-modal contrastive learning for the integration of spatially resolved transcriptomics (MuCST), which jointly perform denoising, elimination of heterogeneity, and compatible feature learning. We demonstrate that MuCST robustly and accurately identifies tissue subpopulations from simulated data with various types of perturbations. In cancer-related tissues, MuCST precisely identifies tumor-associated domains, reveals gene biomarkers for tumor regions, and exposes intra-tumoral heterogeneity. We also validate that MuCST is applicable to diverse datasets generated from various platforms, such as STARmap, Visium, and omsFISH for spatial transcriptomics, and hematoxylin and eosin or fluorescence microscopy for images. Overall, MuCST not only facilitates the integration of multi-modal spatially resolved data, but also serves as pre-processing for data restoration (Python software is available at https://github.com/xkmaxidian/MuCST).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e93e7b00fc6643c076f03733108be8401ccb4c7" target='_blank'>
              MuCST: restoring and integrating heterogeneous morphology images and spatial transcriptomics data with contrastive learning
              </a>
            </td>
          <td>
            Yu Wang, Xiaoke Ma
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Background: Mantle cell lymphoma (MCL) is an aggressive subtype of B-cell non-Hodgkin lymphoma that exhibits clinical, pathological, and genetic heterogeneity. While generally thought to be incurable, recently demonstrated clinical activity of both chimeric antigen receptor T (CAR-T) cells and bispecific antibodies suggest that immune-mediated therapies offer substantial benefit in this disease and underscores the value of understanding mechanisms of immune evasion in this disease.
 Aim: The aim of this project is to uncover distinct cellular networks driving immune evasion in MCL across disease sites.
 Methods: We analyzed 10 archival samples collected from 8 clinically annotated patients with untreated MCL, including mononuclear cells from peripheral blood (PB), lymph nodes (LN) and bone marrow (BM). After generating single-cell suspensions from tumors, we performed Cellular Indexing of Transcriptomes and Epitopes sequencing (CITE-seq) and identified malignant B cells based on the co-expression of cyclin D1 (CCND1) and B cell markers (MS4A1, CD79A, CD79B). B and T cells were then subclustered using a reference mapping geneset. Molecular signatures database was utilized to perform geneset enrichment analysis (GSEA).
 Results: Our study first confirms the heterogeneity of mantle cell lymphoma across compartments (i.e PB, LN, BM), showing 5 different clusters of malignant cells, with different abundance in cell proportion among patients. We found that malignant B cells expressed different levels of CCND1 expression across tissue compartments. Furthermore, we identified a specific cluster of malignant cells, enriched in the PB, marked by decreased expression of the costimulatory receptor CD40 and higher levels of B-cell receptor signaling. This may explain the rapid response of circulating MCL cells to Bruton Tyrosine Kinase (BTK) inhibition. Interestingly, the higher expression of CD40 on LN-resident MCL cells suggests that MCL cells may co-opt LN-resident T-cells to support their growth, as has been previously described. Consistent with this, T-cells from LN were phenotypically exhausted, suggesting that MCL cells within the LN leverage local interactions with T-cells to promote survival while avoiding T-cell-driven cytotoxicity. Finally, we also found that high level of Ki67 correlates with myeloid-cell abundance in the TME suggesting that monocytes may support tumor cell proliferation.
 Conclusion: These results provide a deeper understanding of tissue-specific heterogeneity in MCL, including distinct tumor-immune interactions at different tissue sites. In ongoing analyses, we will correlate these findings with patient outcomes and response to BTK inhibition and are currently analyzing samples from patients with relapsed disease to identify microenvironmental features associated with disease progression.
 Citation Format: Amira Marouf, Jahan Rahman, Ashlee Joseph, Irem Isgor, Ya-Hui Lin, Ronan Chaligne, Shenon Sethi, Ahmet Dogan, Gilles Salles, Anita Kumar, Santosha A Vardhana. Identifying tissue-specific drivers of immune evasion in Mantle Cell Lymphoma at single cell level [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93708d0eca850dbd52911bae7638726472d8fb06" target='_blank'>
              Abstract PO-025: Identifying tissue-specific drivers of immune evasion in Mantle Cell Lymphoma at single cell level
              </a>
            </td>
          <td>
            Amira Marouf, Jahan Rahman, A. Joseph, Irem Isgor, Ya-Hui Lin, R. Chaligné, Shenon Sethi, Ahmet Dogan, G. Salles, Anita Kumar, S. Vardhana
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The observation that certain therapeutic strategies for targeting inflammation benefit patients with distinct immune-mediated inflammatory diseases (IMIDs) is exemplified by the success of TNF blockade in conditions including rheumatoid arthritis, ulcerative colitis, and skin psoriasis, albeit only for subsets of individuals with each condition. This suggests intersecting “nodes” in inflammatory networks at a molecular and cellular level may drive and/or maintain IMIDs, being “shared” between traditionally distinct diagnoses without mapping neatly to a single clinical phenotype. In line with this proposition, integrative tumour tissue analyses in oncology have highlighted novel cell states acting across diverse cancers, with important implications for precision medicine. Drawing upon advances in the oncology field, this narrative review will first summarise learnings from the Human Cell Atlas in health as a platform for interrogating IMID tissues. It will then review cross-disease studies to date that inform this endeavour before considering future directions in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2514eae79e9acd2e8a9d4813c5ae46abe3ea1001" target='_blank'>
              Leveraging Multi-Tissue, Single-Cell Atlases as Tools to Elucidate Shared Mechanisms of Immune-Mediated Inflammatory Diseases
              </a>
            </td>
          <td>
            Anthony K. McLean, Gary Reynolds, Arthur G Pratt
          </td>
          <td>2024-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and aims The mitotic catastrophe (MC) pathway plays an important role in hepatocellular carcinoma (HCC) progression and tumor microenvironment (TME) regulation. However, the mechanisms linking MC heterogeneity to immune evasion and treatment response remain unclear. Methods Based on 94 previously published highly correlated genes for MC, HCC patients’ data from the Cancer Genome Atlas (TCGA) and changes in immune signatures and prognostic stratification were studied. Time and spatial-specific differences for MCGs were assessed by single-cell RNA sequencing and spatial transcriptome (ST) analysis. Multiple external databases (GEO, ICGC) were employed to construct an MC-related riskscore model. Results Identification of two MC-related subtypes in HCC patients from TCGA, with clear differences in immune signatures and prognostic risk stratification. Spatial mapping further associates low MC tumor regions with significant immune escape-related signaling. Nomogram combining MC riskscore and traditional indicators was validated great effect for early prediction of HCC patient outcomes. Conclusion MC heterogeneity enables immune escape and therapy resistance in HCC. The MC gene signature serves as a reliable prognostic indicator for liver cancer. By revealing clear immune and spatial heterogeneity of HCC, our integrated approach provides contextual therapeutic strategies for optimal clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c40a88d25b08814bc5a1ad112bbed77e719a8c" target='_blank'>
              Mitotic catastrophe heterogeneity: implications for prognosis and immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zun Mao, Zhixiang Gao, Ruyu Long, Huimin Guo, Long Chen, Sheng Huan, Guo-ping Yin
          </td>
          <td>2024-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In vitro models coupled with multimodal approaches are needed to dissect the dynamic response of local tumor immune microenvironment (TIME) to immunotherapy. Here the patient-derived primary lung cancer organoids (pLCOs) are generated by isolating tumor cell clusters, including the infiltrated immune cells. A function-associated single-cell RNA sequencing (FascRNA-seq) platform allowing both phenotypic evaluation and scRNA-seq at single-organoid level is developed to dissect the TIME of individual pLCOs. The analysis of 171 individual pLCOs derived from seven patients reveals that pLCOs retain the TIME heterogeneity in the parenchyma of parental tumor tissues, providing models with identical genetic background but various TIME. Linking the scRNA-seq data of individual pLCOs with their responses to anti-PD-1 (αPD-1) immune checkpoint blockade (ICB) allows to confirm the central role of CD8+ T cells in anti-tumor immunity, to identify potential tumor-reactive T cells with a set of 10 genes, and to unravel the factors regulating T cell activity, including CD99 gene. In summary, the study constructs a joint phenotypic and transcriptomic FascRNA-seq platform to dissect the dynamic response of local TIME under ICB treatment, providing a promising approach to evaluate novel immunotherapies and to understand the underlying molecular mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902a93c4a9cfc533a5004cf44836921bd67b326d" target='_blank'>
              Patient-Derived Tumor Organoids Combined with Function-Associated ScRNA-Seq for Dissecting the Local Immune Response of Lung Cancer.
              </a>
            </td>
          <td>
            Chang Liu, Kaiyi Li, X. Sui, Tian Zhao, Ting Zhang, Zhongyao Chen, Hainan Wu, Chao Li, Hao Li, Fan Yang, Zhidong Liu, You-Yong Lu, Jun Wang, Xiaofang Chen, Peng Liu
          </td>
          <td>2024-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Immune checkpoint (IC) blockade and adoptive transfer of tumor-specific T-cells (ACT) are two major strategies to treat metastatic melanoma. Their combination can potentiate T-cell activation in the suppressive tumor microenvironment, but the autoimmune adverse effects associated with systemic injection of IC blockers persist with this strategy. ACT of tumor-reactive T-cells defective for IC expression would overcome this issue. For this purpose, PD-1 and TIGIT appear to be relevant candidates, because their co-expression on highly tumor-reactive lymphocytes limits their therapeutic efficacy within the tumor microenvironme,nt. Our study compares the consequences of PDCD1 or TIGIT genetic deletion on anti-tumor properties and T-cell fitness of melanoma-specific T lymphocytes. Transcriptomic analyses revealed down-regulation of cell cycle-related genes in PD-1KO T-cells, consistent with biological observations, whereas proliferative pathways were preserved in TIGITKO T-cells. Functional analyses showed that PD-1KO and TIGITKO T-cells displayed superior antitumor reactivity than their wild-type counterpart in vitro and in a preclinical melanoma model using immunodeficient mice. Interestingly, it appears that TIGITKO T-cells were more effective at inhibiting tumor cell proliferation in vivo, and persist longer within tumors than PD-1KO T-cells, consistent with the absence of impact of TIGIT deletion on T-cell fitness. Taken together, these results suggest that TIGIT deletion, over PD-1 deletion, in melanoma-specific T-cells is a compelling option for future immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d39a7191a7050c3023b69888c21ce3d7915b91a" target='_blank'>
              Differential impact of genetic deletion of TIGIT or PD-1 on melanoma-specific T-lymphocytes
              </a>
            </td>
          <td>
            G. Cadiou, T. Beauvais, Lucine Marotte, Sylvia Lambot, C. Deleine, Caroline Vignes, Malika Gantier, Melanie Hussong, Samuel Rulli, Anne Jarry, Sylvain Simon, Bernard Malissen, N. Labarrière
          </td>
          <td>2024-07-08</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19b937a783d0083d4f357f86050f667df91ba2a" target='_blank'>
              ScRNA-seq revealed the tumor microenvironment heterogeneity related to the occurrence and metastasis in upper urinary tract urothelial carcinoma.
              </a>
            </td>
          <td>
            Shiyong Xin, Yanwei Zhang, Zhenhua Zhang, Ziyao Li, Xianchao Sun, Xiang Liu, L. Jin, Weiyi Li, Chaozhi Tang, Wangli Mei, Qiong Cao, Haojie Wang, Zhihao Wei, Zhen Zhou, Rongbing Li, Xiaofei Wen, Guosheng Yang, Weihua Chen, Junhua Zheng, L. Ye
          </td>
          <td>2024-06-14</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The functional heterogeneity and ecological niche of prostate cancer stem cells (PCSCs), which are major drivers of prostate cancer development and treatment resistance, have attracted considerable research attention. Cancer-associated fibroblasts (CAFs), which are crucial components of the tumor microenvironment (TME), substantially affect PCSC stemness. Additionally, CAFs promote PCSC growth and survival by releasing signaling molecules and modifying the surrounding environment. Conversely, PCSCs may affect the characteristics and behavior of CAFs by producing various molecules. This crosstalk mechanism is potentially crucial for prostate cancer progression and the development of treatment resistance. Using organoids to model the TME enables an in-depth study of CAF-PCSC interactions, providing a valuable preclinical tool to accurately evaluate potential target genes and design novel treatment strategies for prostate cancer. The objective of this review is to discuss the current research on the multilevel and multitarget regulatory mechanisms underlying CAF-PCSC interactions and crosstalk, aiming to inform therapeutic approaches that address challenges in prostate cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebc78492c002bb6ca79859426afb09d4c0e6ab75" target='_blank'>
              Cancer-associated fibroblasts and prostate cancer stem cells: crosstalk mechanisms and implications for disease progression
              </a>
            </td>
          <td>
            Haoran Chen, Suping Fang, Xudong Zhu, Hao Liu
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a61469c4cc1c16ace14928ca354598a7e35422ef" target='_blank'>
              Extracellular matrices of stromal cell subtypes regulate phenotype and contribute to the stromal microenvironment in vivo
              </a>
            </td>
          <td>
            A. Stone, Emma Rand, Gabriel Thornes, A. Kay, A. Barnes, Ian S. Hitchcock, P. Genever
          </td>
          <td>2024-06-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7f25a39411b25f30ce76ee209200149b2b9966" target='_blank'>
              Non-stem cell lineages as an alternative origin of intestinal tumorigenesis in the context of inflammation
              </a>
            </td>
          <td>
            Mathijs P. Verhagen, Rosalie Joosten, Mark Schmitt, Niko Välimäki, A. Sacchetti, K. Rajamäki, Jiahn Choi, Paola Procopio, Sara Silva, Berdine van der Steen, T. P. van den Bosch, Danielle Seinstra, Annemarie C de Vries, M. Doukas, Leonard H. Augenlicht, L. Aaltonen, Riccardo Fodde
          </td>
          <td>2024-06-20</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Throughout our lifetime, each beat of the heart requires the coordinated action of multiple cardiac cell types. Understanding cardiac cell biology, its intricate microenvironments, and the mechanisms that govern their function in health and disease are crucial to designing novel therapeutical and behavioral interventions. Recent advances in single-cell and spatial omics technologies have significantly propelled this understanding, offering novel insights into the cellular diversity and function and the complex interactions of cardiac tissue. This review provides a comprehensive overview of the cellular landscape of the heart, bridging the gap between suspension-based and emerging in situ approaches, focusing on the experimental and computational challenges, comparative analyses of mouse and human cardiac systems, and the rising contextualization of cardiac cells within their niches. As we explore the heart at this unprecedented resolution, integrating insights from both mouse and human studies will pave the way for novel diagnostic tools and therapeutic interventions, ultimately improving outcomes for patients with cardiovascular diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dd15f5fe35207f3835a17161c2591f154b337be" target='_blank'>
              Revisiting Cardiac Biology in the Era of Single Cell and Spatial Omics
              </a>
            </td>
          <td>
            Jack A. Palmer, Nadia Rosenthal, Sara A Teichmann, Monika Litvinukova
          </td>
          <td>2024-06-07</td>
          <td>Circulation Research</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Pericytes, recognized as mural cells, have long been described as components involved in blood vessel formation, playing a mere supporting role for endothelial cells (ECs). Emerging evidence strongly suggests their multifaceted roles in tissues and organs. Indeed, pericytes exhibit a remarkable ability to anticipate endothelial cell behavior and adapt their functions based on the specific cells they interact with. Pericytes can be activated by pro-inflammatory stimuli and crosstalk with immune cells, actively participating in their transmigration into blood vessels. Moreover, they can influence the immune response, often sustaining an immunosuppressive phenotype in most of the cancer types studied. In this review, we concentrate on the intricate crosstalk between pericytes and immune cells in cancer, highlighting the primary evidence regarding pericyte involvement in primary tumor mass dynamics, their contributions to tumor reprogramming for invasion and migration of malignant cells, and their role in the formation of pre-metastatic niches. Finally, we explored recent and emerging pharmacological approaches aimed at vascular normalization, including novel strategies to enhance the efficacy of immunotherapy through combined use with anti-angiogenic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76bcdf2dcb6f3841dc22f4ab784010532400194" target='_blank'>
              Pericytes: jack-of-all-trades in cancer-related inflammation
              </a>
            </td>
          <td>
            Marianna Moro, Federica Carolina Balestrero, A. A. Grolla
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastomas are known for their poor clinical prognosis, with recurrent tumors often exhibiting greater invasiveness and faster growth rates compared to primary tumors. To understand the intratumoral changes driving this phenomenon, we employed single-cell sequencing to analyze the differences between two pairs of primary and recurrent glioblastomas. Our findings revealed an upregulation of ferroptosis in endothelial cells within recurrent tumors, identified by the significant overexpression of the NOX4 gene. Further analysis indicated that knocking down NOX4 in endothelial cells reduced the activity of the ferroptosis pathway. Utilizing conditioned media from endothelial cells with lower ferroptosis activity, we observed a decrease in the growth rate of glioblastoma cells. These results highlighted the complex role of ferroptosis within tumors and suggested that targeting ferroptosis in the treatment of glioblastomas requires careful consideration of its effects on endothelial cells, as it may otherwise produce counterproductive outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb06c19dc2e1dad39392c5e5a73da1e9d0d09630" target='_blank'>
              Endothelial cell ferroptosis influences IDH wild-type glioblastoma growth in recurrent glioblastoma multiforme patients
              </a>
            </td>
          <td>
            Bo Liang, Xinghuan Ding, Siyuan Yang, Enshan Feng
          </td>
          <td>2024-07-08</td>
          <td>Brazilian Journal of Medical and Biological Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancers are commonly characterised by a complex pathology encompassing genetic, microscopic and macroscopic features, which can be probed individually using imaging and omics technologies. Integrating this data to obtain a full understanding of pathology remains challenging. We introduce a new method called Deep Latent Variable Path Modelling (DLVPM), which combines the representational power of deep learning with the capacity of path modelling to identify relationships between interacting elements in a complex system. To evaluate the capabilities of DLVPM, we initially trained a foundational model to map dependencies between SNV, Methylation, miRNA-Seq, RNA-Seq and Histological data using Breast Cancer data from The Cancer Genome Atlas (TCGA). This method exhibited superior performance in mapping associations between data types compared to classical path modelling. We additionally performed successful applications of the model to: stratify single-cell data, identify synthetic lethal interactions using CRISPR-Cas9 screens derived from cell-lines, and detect histologic-transcriptional associations using spatial transcriptomic data. Results from each of these data types can then be understood with reference to the same holistic model of illness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36a969a03ce741afe8ca32855f97d3ec725e814d" target='_blank'>
              Integrating Multi-Modal Cancer Data Using Deep Latent Variable Path Modelling
              </a>
            </td>
          <td>
            Alex Ing, Alvaro Andrades, M. R. Cosenza, J. Korbel
          </td>
          <td>2024-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Metastatic brain tumors are associated with significant morbidity and mortality. Recent trials have demonstrated the potential efficacy of immunotherapy for the management of brain metastasis (BrM), albeit in only a subset of patients. Understanding the BrM microenvironment and identifying the factors that shape this complex milieu can facilitate optimizing treatment strategies for this disease. The microbiome has been recognized as a hallmark of cancer with a major role in shaping response to immunotherapy. However, the role of the microbiome in BrM is largely unknown. We have previously reported that bacterial signals can be detected in BrM tumors via metagenomic analyses and staining. Here we present the spatial correlation between the intra-tumoral bacterial signal and the transcriptional changes in the tumor microenvironment. Tissue digital spatial profiling (DSP) and single-cell DSP were conducted on BrM tissue microarrays using the NanoString GeoMx® and CosMx® platforms, respectively. These assays incorporated a custom panel of pan-bacterial 16S rRNA probes, enabling a comparison between tumor regions and cells that showed a positive signal for bacterial 16S rRNA versus those that did not. DSP of BrM tumors revealed anti-bacterial responses and enrichment of innate immune activity in areas with 16S signal, suggesting an active host response to intra-tumoral 16S signals. Furthermore, single-cell DSP demonstrated a heterogenous distribution of cell types harboring the 16S signal in the tumor microenvironment. In tumor cells, intracellular 16S signal was associated with an upregulation of transcripts related to neuroinflammation, chemokine, and interleukin signaling. Furthermore, immune transcriptional changes were identified in the immediate neighborhood of 16S-positive cells. Overall, this study introduces bacterial signals as a novel element within metastatic brain tumors and demonstrates their association with anti-bacterial responses and immune activity. These findings set the foundation for future mechanistic and translational studies to evaluate microbiome modulation strategies in enhancing care for brain metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-06 SINGLE-CELL SPATIAL ANALYSIS OF METASTATIC BRAIN TUMORS REVEALS INTRACELLULAR BACTERIAL SIGNAL AND THEIR CORRELATION WITH THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            Golnaz Morad, Ashish Damani, Brenda Melendez, Bharat Singh, M. Wong, P. Sahasrabhojane, Yasmine Hoballah, Fabiana Veguilla, Jillian S. Losh, N. Ajami, Sherise Ferguson, Jennifer Wargo
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Since the introduction of the first immune checkpoint inhibitor (ICI), immunotherapy has changed the landscape of molecular therapeutics for cancers. However, ICIs do not work equally well on all cancers and for all patients. There has been a growing interest in using mathematical and computational models to optimize clinical responses. Ordinary differential equations (ODEs) have been widely used for mechanistic modeling in immuno-oncology and immunotherapy because they allow rapid simulations of temporal changes in the cellular and molecular populations involved. Nonetheless, ODEs cannot describe the spatial structure in the tumor microenvironment or quantify the influence of spatially-dependent characteristics of tumor-immune dynamics. For these reasons, agent-based models (ABMs) have gained popularity because they can model more detailed phenotypic and spatial heterogeneity that better reflect the complexity seen in vivo. In the context of anti-PD-1 ICIs, we compare treatment outcomes simulated from an ODE model and an ABM to show the importance of including spatial components in computational models of cancer immunotherapy. We consider tumor cells of high and low antigenicity and two distinct cytotoxic T lymphocyte (CTL) killing mechanisms. The preferred mechanism differs based on the antigenicity of tumor cells. Our ABM reveals varied phenotypic shifts within the tumor and spatial organization of tumor and CTLs, despite similarities in key immune parameters, initial conditions of simulation, and early temporal trajectories of the cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38a51133539694b984ad15aa283204f18aa1920" target='_blank'>
              Agent-Based Modeling of Virtual Tumors Reveals the Critical Influence of Microenvironmental Complexity on Immunotherapy Efficacy
              </a>
            </td>
          <td>
            Yixuan Wang, Daniel R. Bergman, Erica Trujillo, Anthony A. Fernald, Lie Li, Alexander T. Pearson, Randy F. Sweis, Trachette Jackson
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Understanding tumor–host immune interactions and the mechanisms of lung cancer response to immunotherapy is crucial. Current preclinical models used to study this often fall short of capturing the complexities of human lung cancer and lead to inconclusive results. To bridge the gap, we introduce two new murine monoclonal lung cancer cell lines for use in immunocompetent orthotopic models. We demonstrate how our cell lines exhibit immunohistochemical protein expression (TTF-1, NapA, PD-L1) and common driver mutations (KRAS, p53, and p110α) seen in human lung adenocarcinoma patients, and how our orthotopic models respond to combination immunotherapy in vivo in a way that closely mirrors current clinical outcomes. These new lung adenocarcinoma cell lines provide an invaluable, clinically relevant platform for investigating the intricate dynamics between tumor and the immune system, and thus potentially contributes to a deeper understanding of immunotherapeutic approaches to lung cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058e536d48922167d21bebd1081fee6dfb623837" target='_blank'>
              Orthotopic Models Using New, Murine Lung Adenocarcinoma Cell Lines Simulate Human Non-Small Cell Lung Cancer Treated with Immunotherapy
              </a>
            </td>
          <td>
            Eric P. Knott, Emily Y Kim, Edison Q Kim, Rochelle Freire, Justin A. Medina, Yujie Wang, Cheng-Bang Chen, Chunjing Wu, M. Wangpaichitr, Jose R Conejo-Garcia, Diane C. Lim
          </td>
          <td>2024-06-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are ectopic lymphoid aggregates associated with improved prognosis in numerous tumors. To date, their prognostic value in central nervous system cancers and the events underlying their formation remain unclear. Here, we find that TLS correlate with improved survival in glioblastoma patients. Furthermore, combining spatial transcriptomics of human tissues with longitudinal studies in murine glioma, we establish that the assembly of T cell-rich clusters is a prerequisite for the development of canonical TLS, and show that CD4 T cells play a central role in this process. Indeed, we provide evidence that IL7R+CCR7+ Th1 lymphocytes with lymphoid tissue-inducing potential are recruited to TLS nucleation sites, where they can initiate TLS assembly by expressing lymphotoxin β. Our work defines TLS as a potential prognostic factor in glioblastoma and identifies cellular and molecular mechanisms of TLS formation. These findings have broader implications for the development of TLS-inducing therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368cd006ed1ae7b496e449db1b6253ab2d80a3fe" target='_blank'>
              Cellular and molecular events organizing the assembly of tertiary lymphoid structures in glioblastoma
              </a>
            </td>
          <td>
            Alessandra Vaccaro, Fan Yang, Tiarne van de Walle, Saskia Franke, R. Lugano, A. Dénes, Anastasia Magoulopoulou, Inés Cid-Fariña, Anja Smits, Lene Uhrbom, Sylwia Libard, Francesco Latini, Magnus Essand, Mats Nilsson, Liqun He, T. Olsson Bontell, A. Jakola, Mohanraj Ramachandran, A. Dimberg
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Deciphering disease progression and the sophisticated tumor ecosystems is imperative for exploring tumorigenesis mechanisms and developing novel prevention strategies. Here, we dissected heterogeneous tissue microenvironments during malignant transitions by leveraging data from 4,972,145 single cells in 1396 samples from 62 datasets spanning 13 major tissue types. Within transitional stem-like subpopulations highly enriched in precancerous lesions and cancers, we identified 30 recurring cellular states, including hypoxia and epithelial senescence, revealing a high degree of plasticity in epithelial stem cells. By characterizing the dynamics of stem-cell crosstalk with the microenvironment along the pseudotime axis, we uncovered distinct roles of ANXA1 at different stages of tumor development. ANXA1 expression levels in stem cells were decreased from the healthy to the precancerous stages, which promoted inflammatory responses by recruiting neutrophils and regulating monocyte differentiation towards M1 macrophages. In contrast, during malignant progression, upregulated ANXA1 fostered M2 macrophage polarization and cancer-associated fibroblast transformation. Our spatiotemporal analysis further provided insights into mechanisms responsible for immunosuppression. Collectively, this study provided a systematic view of cancer origins, and suggested that restoring and maintaining the balance of inflammation and their mediators (e.g., AnxA1/FPRs signaling) may represent a novel approach to control the evolution of precancerous lesions and mitigate the risk for cancer development.
 Citation Format: Ruihan Luo. Single-cell landscape deciphering cancer cell-of-origin and cellular heterogeneity in malignant transformation of 13 major tissues [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3160fc9db342b50200de96ca8c057424be1898e" target='_blank'>
              Abstract A026: Single-cell landscape deciphering cancer Single-cell landscape deciphering cancer cell-of-origin and cellular heterogeneity in malignant transformation of 13 major tissues
              </a>
            </td>
          <td>
            Ruihan Luo
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The spatial composition and cellular heterogeneity of the tumor microenvironment plays a critical role in cancer development and progression. High-definition pathology imaging of tumor biopsies provide a high-resolution view of the spatial organization of different types of cells. This allows for systematic assessment of intra- and inter-patient spatial cellular interactions and heterogeneity by integrating accompanying patient-level genomics data. However, joint modeling across tumor biopsies presents unique challenges due to non-conformability (lack of a common spatial domain across biopsies) as well as high-dimensionality. To address this problem, we propose the Dual random effect and main effect selection model for Spatially structured regression model (DreameSpase). DreameSpase employs a Bayesian variable selection framework that facilitates the assessment of spatial heterogeneity with respect to covariates both within (through fixed effects) and between spaces (through spatial random effects) for non-conformable spatial domains. We demonstrate the efficacy of DreameSpase via simulations and integrative analyses of pathology imaging and gene expression data obtained from $335$ melanoma biopsies. Our findings confirm several existing relationships, e.g. neutrophil genes being associated with both inter- and intra-patient spatial heterogeneity, as well as discovering novel associations. We also provide freely available and computationally efficient software for implementing DreameSpase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f4788cc422b8751f7a8dd15c650e8347a66bb49" target='_blank'>
              Spatially Structured Regression for Non-conformable Spaces: Integrating Pathology Imaging and Genomics Data in Cancer
              </a>
            </td>
          <td>
            Nathaniel Osher, Jian Kang, Arvind Rao, Veerabhadran Baladandayuthapani
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A total of 104 patients with recurrent/metastatic SGC from the DKTK MASTER program were included in this study. Whole-exome or whole-genome sequencing and RNA-sequencing was performed on fresh frozen tumor tissue. The tumor immune microenvironment was analyzed using CIBERSORT deconvolution analysis and immune gene expression scores in bulk RNA-sequencing data. Single-nuclei sequencing and immunohistochemistry analyses were performed in selected samples. Results were validated in bulk RNA-sequencing data of a previously published independent dataset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39252ecb83125cd68db55b442ae5a705846bc745" target='_blank'>
              Integrative analysis reveals a macrophage-predominant, immunosuppressive immune microenvironment and subtype-specific therapeutic vulnerabilities in advanced salivary gland cancer
              </a>
            </td>
          <td>
            Erika Zuljan, Benjamin von der Emde, Iris Piwonski, Ana Cristina, Afonseca Pestana, K. Klinghammer, Andreas Mock, Peter Horak, Christoph Heining, F. Klauschen, Ina Pretzell, M. Boerries, Christian H. Brandts, S. Kreutzfeldt, M. Teleanu, Daniel Hübschmann, T. LucG, Morris, Ulrich Keller, H. Glimm, Stefan Fröhling, Sebastian Ochsenreither, Ulrich Keilholz, Eric Blanc, D. Beule, D. Rieke
          </td>
          <td>2024-06-13</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Over the last decade, Hippo signaling has emerged as a major tumor-suppressing pathway. Its dysregulation is associated with abnormal expression of YAP1 and TEAD-family genes. Recent works have highlighted the role of YAP1/TEAD activity in several cancers and its potential therapeutic implications. Therefore, identifying patients with a dysregulated Hippo pathway is key to enhancing treatment impact. Although recent studies have derived RNAseq-based signatures, there remains a need for a reproducible and cost-effective method to measure the pathway activation. In recent years, deep learning applied to histology slides have emerged as an effective way to predict molecular information from a data modality available in clinical routine. Here, we trained models to predict YAP1/TEAD activity from H&E-stained histology slides in multiple cancers. The robustness of our approach was assessed in seven independent validation cohorts. Finally, we showed that histological markers of disease aggressiveness were associated with dysfunctional Hippo signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9627581dee3ac77aa803a011621d196f8f2894" target='_blank'>
              Deep learning uncovers histological patterns of YAP1/TEAD activity related to disease aggressiveness in cancer patients
              </a>
            </td>
          <td>
            Benoît Schmauch, Vincent Cabeli, Omar Darwiche Domingues, Jean-Eudes Le Douget, Alexandra Hardy, Reda Belbahri, C. Maussion, Alberto Romagnoni, Markus Eckstein, Florian Fuchs, A. Swalduz, Sylvie Lantuejoul, Hugo Crochet, François Ghiringhelli, V. Derangère, C. Truntzer, H. Pass, Andre Moreira, Luis Chiriboga, Yuanning Zheng, Michael Ozawa, Brooke E. Howitt, Olivier Gevaert, N. Girard, E. Rexhepaj, I. Valtingojer, Laurent Debussche, Emanuele de Rinaldis, Frank Nestle, Emmanuel Spanakis, Valeria R. Fantin, Eric Y. Durand, Marion Classe, K. V. Loga, Elodie Pronier, Matteo Cesaroni
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background: Carotid atherosclerosis is orchestrated by cell-cell communication that drives progression along a clinical continuum (asymptomatic to symptomatic). Extracellular vesicles (EVs) are cell-derived nanoparticles representing a new paradigm in cellular communication. Little is known about their biological cargo, cellular origin/destination, and functional roles in human atherosclerotic plaque. Methods: EVs were enriched via size exclusion chromatography from human carotid endarterectomy samples dissected into paired plaque and marginal zones (symptomatic n=16, asymptomatic n=13). EV cargos were assessed via whole transcriptome miRNA sequencing and mass spectrometry-based proteomics. EV multi-omics were integrated with bulk and single cell RNA-sequencing (scRNA-seq) datasets to predict EV cellular origin and ligand-receptor interactions, and multi-modal biological network integration of EV-cargo was completed. EV functional impact was assessed with endothelial angiogenesis assays. Results: Carotid plaques contained more EVs than adjacent marginal zones, with differential enrichment for EV-miRNAs and EV-proteins in key atherogenic pathways. EV cellular origin analysis suggested that tissue EV signatures originated from endothelial cells (EC), smooth muscle cells (SMC), and immune cells. Integrated tissue vesiculomics and scRNA-seq indicated complex EV-vascular cell communication that changed with disease progression and plaque vulnerability (i.e., symptomatic disease). Plaques from symptomatic patients, but not asymptomatic patients, were characterized by increased involvement of endothelial pathways and more complex ligand-receptor interactions, relative to their marginal zones. Plaque-EVs were predicted to mediate communication with ECs. Pathway enrichment analysis delineated an endothelial signature with roles in angiogenesis and neovascularization – well-known indices of plaque instability. This was validated functionally, wherein human carotid symptomatic plaque EVs induced sprouting angiogenesis in comparison to their matched marginal zones. Conclusion: Our findings indicate that EVs may drive dynamic changes in plaques through EV- vascular cell communication and effector functions that typify vulnerability to rupture, precipitating symptomatic disease. The discovery of endothelial-directed angiogenic processes mediated by EVs creates new therapeutic avenues for atherosclerosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85baba07ce353659c168561fd588df1f617dc3d1" target='_blank'>
              Multiomics unveils extracellular vesicle-driven mechanisms of endothelial communication in human carotid atherosclerosis
              </a>
            </td>
          <td>
            S. Raju, Mandy E Turner, C. Cao, Majed Abdul-Samad, Neil Punwasi, M. Blaser, Rachel M E Cahalane, S. Botts, Kamal K. Prajapati, Sarvatit Patel, Ruilin Wu, D. Gustafson, Natalie J. Galant, Lindsey Fiddes, M. Chemaly, Ulf Hedin, L. Matic, Michael Seidman, Vallijah Subasri, Sasha A. Singh, Elena Aikawa, Jason E. Fish, Kathryn L. Howe
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex and dynamic ecosystem that adjoins the cancer cells within solid tumors and comprises distinct components such as extracellular matrix, stromal and immune cells, blood vessels, and an abundance of signaling molecules. In recent years, the mechanical properties of the TME have emerged as critical determinants of tumor progression and therapeutic response. Aberrant mechanical cues, including altered tissue architecture and stiffness, contribute to tumor progression, metastasis, and resistance to treatment. Moreover, burgeoning immunotherapies hold great promise for harnessing the immune system to target and eliminate solid malignancies; however, their success is hindered by the hostile mechanical landscape of the TME, which can impede immune cell infiltration, function, and persistence. Consequently, understanding TME mechanoimmunology - the interplay between mechanical forces and immune cell behavior - is essential for developing effective solid cancer therapies. Here, we review the role of TME mechanics in tumor immunology, focusing on recent therapeutic interventions aimed at modulating the mechanical properties of the TME to potentiate T cell immunotherapies, and innovative assays tailored to evaluate their clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b87a0621932f54045cb5dd1e27d16267699536f" target='_blank'>
              Mechanoimmunology in the solid tumor microenvironment.
              </a>
            </td>
          <td>
            Matteo Golo, Peter L H Newman, D. Kempe, M. Biro
          </td>
          <td>2024-06-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Functional precision oncology represents an emerging approach to genomic approaches by testing treatment options directly on patient-derived models. Current assays, including the use of patient-derived xenograft (PDX) and patient-derived organoid (PDOs), faces major barriers in clinical use due to technical challenges, such as standardization, cost, assay time, scalability, and faithful mimicry of patient tumor microenvironment (TME). Here, we introduce an Organ Chip (OC) device constructed entirely from thermoplastic materials, free of porous membrane or other barrier structures, and optimized for high-content imaging (HCI). This automation-compatible device supports tissue-specific extracellular matrices and coculture for a wide spectrum of organ and disease types, including the TME. As a proof-of-concept, we demonstrate the growth of pancreatic, lung, and colon cancer cell lines and primary lung cancer cells and the testing of cancer drugs in these models. HCI-based phenotypic profiling enabled accurate quantification of drug response, with better performance than traditional biochemical assays. Moreover, we developed a deep-learning method for assessing drug responses using bright field images. The integration of a low-cost, scalable, and faithful OC models with automatic high-content image analysis represents a significant stride towards functional precision oncology and cancer drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d0b12fd71d2c327aa8dbf5e25b30f3d0a306ae" target='_blank'>
              Combining Scalable Organ Chip Platform with Deep Learning-Based Imaging Analysis for Cancer Therapeutic Screening
              </a>
            </td>
          <td>
            Yu-Chieh Yuan, Beibei Xu, Jenna McCormack, XuHai Huang, Jingzhe Ma, Thomas Marshall, Yacong Sun, Hardeep Singh, Alyssa Fanelli, Paige Gilbride, Xiaohua Qian, Zhiyong Xie, Longlong Si, Xin Xie, H. Bai
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Glioma, a prevalent and deadly brain tumor, is marked by significant cellular heterogeneity and metabolic alterations. However, the comprehensive cell-of-origin and metabolic landscape in high-grade (Glioblastoma Multiforme, WHO grade IV) and low-grade (Oligoastrocytoma, WHO grade II) gliomas remains elusive. Methods In this study, we undertook single-cell transcriptome sequencing of these glioma grades to elucidate their cellular and metabolic distinctions. Following the identification of cell types, we compared metabolic pathway activities and gene expressions between high-grade and low-grade gliomas. Results Notably, astrocytes and oligodendrocyte progenitor cells (OPCs) exhibited the most substantial differences in both metabolic pathways and gene expression, indicative of their distinct origins. The comprehensive analysis identified the most altered metabolic pathways (MCPs) and genes across all cell types, which were further validated against TCGA and CGGA datasets for clinical relevance. Discussion Crucially, the metabolic enzyme phosphodiesterase 8B (PDE8B) was found to be exclusively expressed and progressively downregulated in astrocytes and OPCs in higher-grade gliomas. This decreased expression identifies PDE8B as a metabolism-related oncogene in IDH-mutant glioma, marking its dual role as both a protective marker for glioma grading and prognosis and as a facilitator in glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32a8dd10db0de50c204790d7bd0cb6e815520890" target='_blank'>
              Single-cell transcriptomic analysis identifies downregulated phosphodiesterase 8B as a novel oncogene in IDH-mutant glioma
              </a>
            </td>
          <td>
            Z. He, Yu Peng, Duo Wang, Chen Yang, Chengzhi Zhou, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is a highly aggressive cancer predominantly occurring in male adolescents and young adults. The lack of a comprehensive understanding on the biology of the disease is paralleled by its dismal survival rates (5–20%). To overcome this challenge, we first identified and prioritized urgently needed resources for clinicians and researchers. Thus, we established genome-wide single-cell RNA-sequencing and bulk proteomic data of in vitro and in vivo-generated knockdown models of the pathognomonic DSRCT fusion oncoprotein (EWSR1::WT1) and combined them with an original systems-biology-based pipeline including patient data and the largest histology collection of DSRCTs and morphological mimics available to date. These novel tools were enriched with curated public datasets including patient- and cell line-derived ChIP-seq, bulk and single-cell RNA-seq studies resulting in a multi-model and multi-omic toolbox for discovery analyses. As a proof of concept, our approach revealed the alpha-2/delta subunit of the voltage-dependent calcium channel complex, CACNA2D2, as a highly overexpressed, super-enhancer driven, direct target of EWSR1::WT1. Single-cell and bulk-level analyses of patient samples and xenografted cell lines highlighted CACNA2D2 as a critical component of our newly established EWSR1::WT1 oncogenic signature, that can be employed to robustly identify DSRCT in reference sets. Finally, we show that CACNA2D2 is a highly sensitive and specific single biomarker for fast, simple, and cost-efficient diagnosis of DSRCT. Collectively, we establish a large-scale multi-omics dataset for this devastating disease and provide a blueprint of how such toolbox can be used to identify new and clinically relevant diagnostic markers, which may significantly reduce misdiagnoses, and thus improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1adb9c7dca4062ab1492b77bcd56eb7a98c207ea" target='_blank'>
              Super-enhancer-driven CACNA2D2 is an EWSR1::WT1 signature gene encoding a diagnostic marker for desmoplastic small round cell tumor (DSRCT)
              </a>
            </td>
          <td>
            Florian H. Geyer, Alina Ritter, Seneca S Kinn-Gurzo, Tobias Faehling, Jing Li, Armin Jarosch, Carine Ngo, Endrit Vinca, Karim Aljakouch, Azhar Orynbek, S. Ohmura, Thomas Kirchner, R. Imle, L. Romero-Pérez, Stefanie Bertram, Enrique de Álava, Sophie Postel-Vilnay, A. Banito, M. Sill, Y. Versleijen-Jonkers, Benjamin F B Mayer, Martin Ebinger, M. Sparber-Sauer, Sabine Stegmaier, Daniel Baumhoer, W. Hartmann, Jeroen Krijgsveld, David Horst, Olivier Delattre, P. Grohar, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/709b9beacb699d72bb2f5c5e148e28301a1bdf47" target='_blank'>
              Label-free morpho-molecular phenotyping of living cancer cells by combined Raman spectroscopy and phase tomography
              </a>
            </td>
          <td>
            A. Bresci, Koseki J. Kobayashi-Kirschvink, Giulio Cerullo, R. Vanna, Peter T. C. So, Dario Polli, Jeon Woong Kang
          </td>
          <td>2024-06-29</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="During ontogeny, γδ T cells emerge from the thymus and directly seed peripheral tissues for in situ immunity. However, their functional role in humans has largely been defined from blood. Here, we analyzed the phenotype, transcriptome, function, and repertoire of human γδ T cells in blood and mucosal and lymphoid tissues from 176 donors across the life span, revealing distinct profiles in children compared with adults. In early life, clonally diverse Vδ1 subsets predominate across blood and tissues, comprising naïve and differentiated effector and tissue repair functions, whereas cytolytic Vδ2 subsets populate blood, spleen, and lungs. With age, Vδ1 and Vδ2 subsets exhibit clonal expansions and elevated cytolytic signatures, which are disseminated across sites. In adults, Vδ2 cells predominate in blood, whereas Vδ1 cells are enriched across tissues and express residency profiles. Thus, antigenic exposures over childhood drive the functional evolution and tissue compartmentalization of γδ T cells, leading to age-dependent roles in immunity. Human tissue γδ T cells are diverse and functionally heterogeneous in early life and differentiate into disseminated effectors in adults. Editor’s summary Our understanding of the functional role of human γδ T cells in tissues is very limited and has been primarily informed by peripheral blood studies. Gray et al. used a multiomics strategy to characterize human γδ T cells in the blood and various mucosal and lymphoid tissues from 176 donors whose ages ranged from just a few days to over 80 years. In contrast to the highly differentiated and cytotoxic γδ T cell populations found in adults, γδ T cells during infancy and childhood are clonally diverse, are tissue specific, and exhibit features of adaptive lymphocytes. —Seth Thomas Scanlon">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9606219ffcf7b81224b5f4d721cfafbeabdda7" target='_blank'>
              Human γδ T cells in diverse tissues exhibit site-specific maturation dynamics across the life span
              </a>
            </td>
          <td>
            Joshua I. Gray, Daniel P. Caron, Steven B. Wells, Rebecca S Guyer, P. Szabo, Daniel B. Rainbow, Can Ergen, Ksenia Rybkina, Marissa C Bradley, Rei Matsumoto, K. Pethe, Masaru Kubota, Sarah A. Teichmann, Joanne L. Jones, N. Yosef, Mark Atkinson, Maigan A. Brusko, Todd M. Brusko, T. Connors, Peter A. Sims, Donna L. Farber
          </td>
          <td>2024-06-07</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f58160d8cdc5bd6e2466eb7c04abf11234ebf3" target='_blank'>
              Patient-derived mini-colons enable long-term modeling of tumor-microenvironment complexity.
              </a>
            </td>
          <td>
            L. F. Lorenzo-Martín, N. Broguiere, Jakob Langer, Lucie Tillard, Mike Nikolaev, George Coukos, K. Homicsko, Matthias P. Lutolf
          </td>
          <td>2024-07-02</td>
          <td>Nature biotechnology</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Kidney allograft rejections are orchestrated by a variety of immune cells. Because of the complex histopathologic features, accurate pathological diagnosis poses challenges even for expert pathologists. The objective of this study was to unveil novel spatial indices associated with transplant rejection by using a spatial bioinformatic approach using 36-plex immunofluorescence image data.


METHODS
The image obtained from 11 T cell-mediated rejection (TCMR) and 12 antibody-mediated rejection (AMR) samples were segmented into 753 737 single cells using DeepCell's Mesmer algorithm. These cells were categorized into 13 distinct cell types through unsupervised clustering based on their biomarker expression profiles. Cell neighborhood analysis allowed us to stratify kidney tissue into 8 distinct neighborhood components consisting of unique cell type enrichment profiles.


RESULTS
In contrast to TCMR samples, AMR samples exhibited a higher frequency of neighborhood components that were characterized by an enrichment of CD31+ endothelial cells. Although the overall frequency of CD68+ macrophages in AMR samples was not significantly high, CD68+ macrophages within endothelial cell-rich lesions exhibited a significantly higher frequency in AMR samples than TCMR samples. Furthermore, the frequency of interactions between CD31+ cells and CD68+ cells was significantly increased in AMR samples, implying the pivotal role of macrophages in AMR pathogenesis. Importantly, patients demonstrating a high frequency of CD31:CD68 interactions experienced significantly poorer outcomes in terms of chronic AMR progression.


CONCLUSIONS
Collectively, these data indicate the potential of spatial bioinformatic as a valuable tool for aiding in pathological diagnosis and for uncovering new insights into the mechanisms underlying transplant rejection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e65530b2545156cbbf6161b36bc3af83e357d01" target='_blank'>
              Unveiling Spatial Immune Cell Profile in Kidney Allograft Rejections Using 36-plex Immunofluorescence Imaging.
              </a>
            </td>
          <td>
            Toshihito Hirai, Ayano Kondo, Tomokazu Shimizu, H. Fukuda, Daisuke Tokita, Toshio Takagi, Aaron T. Mayer, Hideki Ishida
          </td>
          <td>2024-06-24</td>
          <td>Transplantation</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), accounting for only a minor cell proportion (< 1%) within tumors, have profound implications in tumor initiation, metastasis, recurrence, and treatment resistance due to their inherent ability of self-renewal, multi-lineage differentiation, and tumor-initiating potential. In recent years, accumulating studies indicate that CSCs and tumor immune microenvironment act reciprocally in driving tumor progression and diminishing the efficacy of cancer therapies. Extracellular vesicles (EVs), pivotal mediators of intercellular communications, build indispensable biological connections between CSCs and immune cells. By transferring bioactive molecules, including proteins, nucleic acids, and lipids, EVs can exert mutual influence on both CSCs and immune cells. This interaction plays a significant role in reshaping the tumor immune microenvironment, creating conditions favorable for the sustenance and propagation of CSCs. Deciphering the intricate interplay between CSCs and immune cells would provide valuable insights into the mechanisms of CSCs being more susceptible to immune escape. This review will highlight the EV-mediated communications between CSCs and each immune cell lineage in the tumor microenvironment and explore potential therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b368dd6780df18350daa61429b592290175a99e" target='_blank'>
              Modulatory effects of cancer stem cell-derived extracellular vesicles on the tumor immune microenvironment
              </a>
            </td>
          <td>
            Xinyu Li, Cuilian Zhang, Wei Yue, Yuening Jiang
          </td>
          <td>2024-06-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab01bc978e62346ed66c2ee4492c4c5372e12a2c" target='_blank'>
              Characterization of cuproptosis signature in clear cell renal cell carcinoma by single cell and spatial transcriptome analysis
              </a>
            </td>
          <td>
            Xiaohong Zou, Xiaoqing Liu, Huiting Wang, Zhenhua Li, Chen Zhou
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rationale: The heterogeneity of tumor cells within the glioblastoma (GBM) microenvironment presents a complex challenge in curbing GBM progression. Understanding the specific mechanisms of interaction between different GBM cell subclusters and non-tumor cells is crucial. Methods: In this study, we utilized a comprehensive approach integrating glioma single-cell and spatial transcriptomics. This allowed us to examine the molecular interactions and spatial localization within GBM, focusing on a specific tumor cell subcluster, GBM subcluster 6, and M2-type tumor-associated macrophages (M2 TAMs). Results: Our analysis revealed a significant correlation between a specific tumor cell subcluster, GBM cluster 6, and M2-type TAMs. Further in vitro and in vivo experiments demonstrated the specific regulatory role of the CEBPB transcriptional network in GBM subcluster 6, which governs its tumorigenicity, recruitment of M2 TAMs, and polarization. This regulation involves molecules such as MCP1 for macrophage recruitment and the SPP1-Integrin αvβ1-Akt signaling pathway for M2 polarization. Conclusion: Our findings not only deepen our understanding of the formation of M2 TAMs, particularly highlighting the differential roles played by heterogeneous cells within GBM in this process, but also provided new insights for effectively controlling the malignant progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f32ae634b60547b99c952f2958aef6e7b9e1f2ae" target='_blank'>
              The CEBPB+ glioblastoma subcluster specifically drives the formation of M2 tumor-associated macrophages to promote malignancy growth
              </a>
            </td>
          <td>
            Yongchang Yang, Xingyu Jin, Yang Xie, Chunlan Ning, Yiding Ai, Haotian Wei, Xing Xu, Xianglian Ge, Tailong Yi, Qiang Huang, Xuejun Yang, Tao Jiang, Xiaoguang Wang, Yingzhe Piao, Xun Jin
          </td>
          <td>2024-07-02</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, , John S Lazo, Thomas Kislinger, Yu Lu, Kevin R Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Pro-inflammatory CD4+ T cells are major drivers of autoimmune diseases, yet therapies modulating T cell phenotypes to promote an anti-inflammatory state are lacking. Here, we identify T helper 17 (TH17) cell plasticity in the kidneys of patients with antineutrophil cytoplasmic antibody–associated glomerulonephritis on the basis of single-cell (sc) T cell receptor analysis and scRNA velocity. To uncover molecules driving T cell polarization and plasticity, we established an in vivo pooled scCRISPR droplet sequencing (iCROP-seq) screen and applied it to mouse models of glomerulonephritis and colitis. CRISPR-based gene targeting in TH17 cells could be ranked according to the resulting transcriptional perturbations, and polarization biases into T helper 1 (TH1) and regulatory T cells could be quantified. Furthermore, we show that iCROP-seq can facilitate the identification of therapeutic targets by efficient functional stratification of genes and pathways in a disease- and tissue-specific manner. These findings uncover TH17 to TH1 cell plasticity in the human kidney in the context of renal autoimmunity. A single-cell in vivo CRISPR screen highlights genes driving T cell tissue adaptation and plasticity in autoimmunity. Editor’s summary Modulating plasticity in pathogenic T cells to promote a tolerogenic phenotype is a promising therapeutic approach in autoimmunity and cancer. To explore how pathogenic TH17 cell plasticity is regulated in mouse models of kidney and gut inflammation, Enk et al. optimized an in vivo pooled single-cell CRISPR droplet sequencing (iCROP-seq) screen. iCROP-seq enabled the assessment and quantification of CRISPR-modified TH17 cell polarization states in vivo. Furthermore, T cells from patients with antineutrophil cytoplasmic antibody (ANCA)–associated glomerulonephritis exhibited plasticity. These findings indicate that iCROP-seq can be applied to preclinical mouse models of inflammation and suggest that targeting TH17 plasticity in patients with ANCA-glomerulonephritis is a promising therapeutic approach. —Hannah Isles">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dec04d242989fa1742f8f03f089c28ee5f90cc6" target='_blank'>
              Targeting T cell plasticity in kidney and gut inflammation by pooled single-cell CRISPR screening
              </a>
            </td>
          <td>
            Leon U. B. Enk, Malte Hellmig, K. Riecken, Christoph Kilian, Paul Datlinger, Saskia-L. Jauch-Speer, Tobias Neben, Zeba Sultana, Varshi Sivayoganathan, Alina Borchers, H. Paust, Yu Zhao, N. Asada, Shuya Liu, T. Agalioti, P. Pelczar, Thorsten Wiech, Christoph Bock, Tobias B. Huber, Samuel Huber, Stefan Bonn, Nicola Gagliani, B. Fehse, U. Panzer, Christian F. Krebs
          </td>
          <td>2024-06-14</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response.
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Clear Cell Renal Cell Carcinoma (ccRCC) is the most common type of kidney cancer, characterized by high heterogeneity and complexity. Recent studies have identified mitochondrial defects and autophagy as key players in the development of ccRCC. This study aims to delve into the changes in mitophagic activity within ccRCC and its impact on the tumor microenvironment, revealing its role in tumor cell metabolism, development, and survival strategies. Methods Comprehensive analysis of ccRCC tumor tissues using single cell sequencing and spatial transcriptomics to reveal the role of mitophagy in ccRCC. Mitophagy was determined to be altered among renal clear cells by gene set scoring. Key mitophagy cell populations and key prognostic genes were identified using NMF analysis and survival analysis approaches. The role of UBB in ccRCC was also demonstrated by in vitro experiments. Results Compared to normal kidney tissue, various cell types within ccRCC tumor tissues exhibited significantly increased levels of mitophagy, especially renal clear cells. Key genes associated with increased mitophagy levels, such as UBC, UBA52, TOMM7, UBB, MAP1LC3B, and CSNK2B, were identified, with their high expression closely linked to poor patient prognosis. Particularly, the ubiquitination process involving the UBB gene was found to be crucial for mitophagy and its quality control. Conclusion This study highlights the central role of mitophagy and its regulatory factors in the development of ccRCC, revealing the significance of the UBB gene and its associated ubiquitination process in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b8e19a6a697901ad3134462b0b24c7fad7d25a" target='_blank'>
              Mitophagy and clear cell renal cell carcinoma: insights from single-cell and spatial transcriptomics analysis
              </a>
            </td>
          <td>
            Lai Jiang, Xing Ren, Jinyan Yang, Haiqing Chen, Shengke Zhang, Xuancheng Zhou, Jinbang Huang, Chenglu Jiang, Yuheng Gu, Jingyi Tang, Guanhu Yang, Hao Chi, Jianhua Qin
          </td>
          <td>2024-06-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Conventional type 1 dendritic cells (cDC1) are critical regulators of anti-tumoral T-cell responses. The structure and abundance of intercellular contacts between cDC1 and CD8 T cells in cancer tissues is important to determine the outcome of the T-cell response. However, the molecular determinants controlling the stability of cDC1–CD8 interactions during cancer progression remain poorly investigated. Here, we generated a genetic model of non-small cell lung cancer crossed to a fluorescent cDC1 reporter (KP-XCR1venus) to allow the detection of cDC1-CD8T cell clusters in tumor tissues across tumor stages. We found that cDC1-CD8 clusters are abundant and productive at the early stages of tumor development but progressively diminish in advanced tumors. Transcriptional profiling and flow cytometry identified the adhesion molecule ALCAM/CD166 (Activated Leukocyte Cell Adhesion Molecule, ligand of CD6) as highly expressed by lung cDC1 and significantly downregulated in advanced tumors. Analysis of human datasets indicated that ALCAM is downregulated in non-small cell lung cancer and its expression correlates to better prognosis. Mechanistically, triggering ALCAM on lung cDC1 induces cytoskeletal remodeling and contact formation whereas its blockade prevents T-cell activation. Together, our results indicate that ALCAM is important to stabilize cDC1–CD8 interactions at early tumor stages, while its loss in advanced tumors contributes to immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd1ca4d0f78203d20e733a6f3ff393bf9fa0588" target='_blank'>
              ALCAM-mediated cDC1 CD8 T cells interactions are suppressed in advanced lung tumors
              </a>
            </td>
          <td>
            L. Morosi, G. M. Piperno, Lucía López, Roberto Amadio, Sonal Joshi, A. Rustighi, Giannino Del Sal, F. Benvenuti
          </td>
          <td>2024-06-14</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Application of molecular targeted therapies for esophageal adenocarcinoma (EAC) has been limited by a lack of druggable oncogenic drivers. We propose that synthetic lethal interactions may provide new opportunities for targeted therapies in EAC. We have taken an integrated multi-omics approach incorporating Perturb-Seq (CRISPR editing combined with single cell RNA sequencing) and in vivo tumorigenesis assays to perform high-throughput characterisation of >70 high-confidence EAC driver genes, and genome-wide CRISPR-Cas9 knockout screens in isogenic models of EAC tumorigenesis to identify disease relevant synthetic lethal genetic interactions. MS-based proteomics, reverse phase protein arrays, polysome profiling and bulk RNA-sequencing of isogenic models were utilised to interrogate the biology of bona fide EAC drivers and associated driver specific gene dependencies. The overall goals were to (i) enhance our understanding of EAC tumorigenesis, (ii) identify potential opportunities for therapeutic interventions targeting EAC drivers via synthetic lethal-like approaches, and (iii) reduce the complexity of genetic heterogeneity by categorising EAC drivers with similar phenotypic outcomes. Through our approach we have identified complex crosstalk between the tumor suppressor SMAD4 and regulation of mTOR signaling, with specific downstream effects on translational reprogramming in EAC. Mutation or loss of SMAD4 occurs in up to 20% of EAC, but not pre-malignant tissue (Barrett’s esophagus), and is sufficient to promote transformation of pre-malignant cells in our in vivo tumorigenesis model. In this model, xenotransplanted SMAD4-deficient (via CRISPR-Cas9 knockout or shRNA knockdown) Barrett’s metaplasia cells formed invasive, metastatic tumors after a period of latency. SMAD4 deficient cells had downregulated expression of 4E-BP1, which inhibits EIF4E, the cap-dependent translation initiation factor. This was accompanied by increased mTOR activity, including phosphorylation and inactivation of 4EBP1. Moreover, we found that SMAD4-deficient cells preferentially upregulate cap-dependent translation at the expense of IRES mediated translation. Furthermore, perturbation of additional negative regulators of mTOR signaling in combination with SMAD4 knockout exacerbated these effects and accelerated tumorigenesis in vivo. We have extended these findings to a model of Barrett’s esophagus patient-derived organoids (PDOs) and observed increased proliferative potential of our genetically modified PDOs. Finally, analysing gene ontologies for differentially expressed genes from Perturb-seq revealed that driver-dependent transcriptional changes can be categorized into a smaller number of functional pathways allowing us to potentially consider groups of drivers as functional units. This work advances our understanding of EAC tumorigenesis, provides new mechanistic insights into SMAD4-driven transformation as well as novel potential therapeutic avenues for SMAD4-deficient EAC.
 Citation Format: Julia V. Milne, Ebtihal Mustafa, Kenji Fujihara, Eric Kusnadi, Anna Trigos, Niko Thio, Maree Pechlivanis, Carlos Cabalag, Twishi Gulati, Kaylene Simpson, Cuong Duong, Luc Furic, Wayne Phillips, Nicholas Clemons. Delineating functional drivers of esophageal adenocarcinoma to identify synthetic lethal interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95f3d989c9a3fa8592c661128bc239a7c646772" target='_blank'>
              Abstract PR007: Delineating functional drivers of esophageal adenocarcinoma to identify synthetic lethal interactions
              </a>
            </td>
          <td>
            Julia V. Milne, Ebtihal Mustafa, Kenji Fujihara, Eric P. Kusnadi, A. Trigos, N. Thio, Maree Pechlivanis, C. Cabalag, Twishi Gulati, Kaylene Simpson, C. Duong, Luc Furic, Wayne Phillips, Nicholas Clemons
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Single cell and multiplex imaging modalities have opened new insights into the organization and architecture of the immune microenvironment of cancer. Our group was one of the first to demonstrate these methods in the analysis of Hodgkin [1,2] and non-Hodgkin lymphoma [3]. Most recently, we demonstrated that spatial biomarkers outperform gene-expression biomarkers in predicting relapse after autologous transplant [4]. Here we present previously unpublished data demonstrating single cell multiplexed spatial analysis, using the imaging mass cytometry platform, in a cohort of 426 patients with large cell lymphoma. Gene expression and mutational analysis are available for a majority of cases allowing us to correlate spatial signature with genetic classes such as LymphGen [5], and deconvolution methods such as EcoTyper [6]. Using this cohort, we identify spatial biomarkers that improve prediction of clinical outcomes beyond cell of origin or genetic subgroup and identify novel targets for clinical development. Finally, we will present our approaches to move our discovery level spatial profiling into clinical diagnostic space through application of automated multiplex immunofluorescence imaging and a machine learning based image analysis pipeline. [1] Aoki T, et al. Single-Cell Transcriptome Analysis Reveals Disease-Defining T-cell Subsets in the Tumor Microenvironment of Classic Hodgkin Lymphoma. Cancer Discov. 2020 doi: 10.1158/2159-8290.CD-19-0680. [2] Xu, A. et al. Single cell spatial analysis and biomarker discovery in Hodgkin lymphoma. Biorxiv, 2023. https://doi.org/10.1101/2023.05.24.542195. [3] Colombo AR, et al. Single-cell spatial analysis of tumor immune architecture in diffuse large B-cell lymphoma. Blood Adv. 2022 doi: 10.1182/bloodadvances.2022007493. [4] Aoki T, et al. Spatially Resolved Tumor Microenvironment Predicts Treatment Outcomes in Relapsed/Refractory Hodgkin Lymphoma. J Clin Oncol. 2024 doi: 10.1200/JCO.23.01115. [5] Wright GW, et al. A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications. Cancer Cell. 2020 doi: 10.1016/j.ccell.2020.03.015. [6] Steen CB, et al. The landscape of tumor cell states and ecosystems in diffuse large B cell lymphoma. Cancer Cell. 2021 doi: 10.1016/j.ccell.2021.08.011.
 Citation Format: Alexander Xu, Joseph Lownik, Anton Villamejor, Simeon Mahov, Ying Li, Yidan Xu-Monette, Ken Young, Akil Merchant. Single cell spatial analysis of large B cell lymphoma identifies biomarkers correlated with genetic subtypes and clincal outcomes [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127a30be3f605294a50171a3c1596c81524ecc96" target='_blank'>
              Abstract PO-030: Single cell spatial analysis of large B cell lymphoma identifies biomarkers correlated with genetic subtypes and clincal outcomes
              </a>
            </td>
          <td>
            Alexander M Xu, Joseph Lownik, A. Villamejor, Simeon Mahov, Ying Li, Yidan Xu-Monette, Ken Young, Akil Merchant
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive and fatal tumor in the eye, and due the special biology of UM, immunotherapy showed little effect in UM patients. To improve the efficacy of immunotherapy for UM patients is of great clinical importance. Single-cell RNA sequencing(scRNA-seq) provides a critical perspective for deciphering the complexity of intratumor heterogeneity and tumor microenvironment(TME). Combing the bioinformatics analysis, scRNA-seq could help to find prognosis-related molecular indicators, develop new therapeutic targets especially for immunotherapy, and finally to guide the clinical treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c4d47ceeecc5ed316ad19f6c8292032f2ac8a3" target='_blank'>
              Single cell RNA-sequencing in uveal melanoma: advances in heterogeneity, tumor microenvironment and immunotherapy
              </a>
            </td>
          <td>
            Shiyi Tang, Yun Zhang, Shengmei Huang, Tengfei Zhu, Xiaojing Huang
          </td>
          <td>2024-06-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors are the most aggressive primary brain tumors in adults that, despite maximum treatment, carry a dismal prognosis. GBM tumors exhibit tissue hypoxia, which promotes tumor aggressiveness and maintenance of glioma stem cells and creates an overall immunosuppressive landscape. This article reviews how hypoxic conditions overlap with inflammatory responses, favoring the proliferation of immunosuppressive cells and inhibiting cytotoxic T cell development. Immunotherapies, including vaccines, immune checkpoint inhibitors, and CAR-T cell therapy, represent promising avenues for GBM treatment. However, challenges such as tumor heterogeneity, immunosuppressive TME, and BBB restrictiveness hinder their effectiveness. Strategies to address these challenges, including combination therapies and targeting hypoxia, are actively being explored to improve outcomes for GBM patients. Targeting hypoxia in combination with immunotherapy represents a potential strategy to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01181d182e89bbea741de8c2380cb4e17ca4fac4" target='_blank'>
              Hypoxia within the glioblastoma tumor microenvironment: a master saboteur of novel treatments
              </a>
            </td>
          <td>
            Lisa Feldman
          </td>
          <td>2024-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction Intra-tumoral B cells mediate a plethora of immune effector mechanisms with key roles in anti-tumor immunity and serve as positive prognostic indicators in a variety of solid tumor types, including epithelial ovarian cancer (EOC). Several aspects of intra-tumoral B cells remain unclear, such as their state of activation, antigenic repertoires, and capacity to mature into plasma cells. Methods B lymphocytes were isolated from primary EOC tissue and malignant ascites and were maintained in cell culture medium. The stably maintained cell lines were profiled with flow cytometry and B cell receptor sequencing. Secreted antibodies were tested with a human proteome array comprising more than 21,000 proteins, followed by ELISA for validation. Originating tumor samples were used for spatial profiling with chip cytometry. Results Antibody-secreting B lymphocytes were isolated from the ovarian tumor microenvironment (TME) of four different EOC patients. The highly clonal cell populations underwent spontaneous immortalization in vitro, were stably maintained in an antibody-secreting state, and showed presence of Epstein-Barr viral (EBV) proteins. All originating tumors had high frequency of tumor-infiltrating B cells, present as lymphoid aggregates, or tertiary lymphoid structures. The antigens recognized by three of the four cell lines are coil-coil domain containing protein 155 (CCDC155), growth factor receptor-bound protein 2 (GRB2), and pyruvate dehydrogenase phosphatase2 (PDP2), respectively. Anti-CCDC155 circulating IgG antibodies were detected in 9 of 20 (45%) of EOC patients’ sera. Tissue analyses with multiparameter chip cytometry shows that the antibodies secreted by these novel human B cell lines engage their cognate antigens on tumor cells. Discussion These studies demonstrate that within the tumor-infiltrating lymphocyte population in EOC resides a low frequency population of antibody-secreting B cells that have been naturally exposed to EBV. Once stably maintained, these novel cell lines offer unique opportunities for future studies on intratumor B cell biology and new target antigen recognition, and for studies on EBV latency and/or viral reactivation in the TME of non-EBV related solid tumors such as the EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91568ddf7942710ae6ed5b0bc9abf380b3db7b97" target='_blank'>
              Characterization of latently infected EBV+ antibody-secreting B cells isolated from ovarian tumors and malignant ascites
              </a>
            </td>
          <td>
            Lixin Zhang, Mary Strange, E. Elishaev, Syed Zaidi, F. Modugno, M. Radolec, Robert P Edwards, Olivera J Finn, A. Vlad
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc563a244d4f94932a647caae41fe1f2ec7f8f77" target='_blank'>
              Dissecting tumor microenvironment from spatially resolved transcriptomics data by heterogeneous graph learning
              </a>
            </td>
          <td>
            Chunman Zuo, Junjie Xia, Luonan Chen
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Over decades, cancer understanding has advanced significantly at molecular and cellular levels, leading to various therapies based on intra-/inter-cellular networks. Despite this, cancer still remains a leading cause of death globally. The primary driver of cancer mortality is metastasis, responsible for about 90% of cancer deaths, due to unclear pathophysiological mechanisms that complicate treatment development. The Notch signaling pathway, a crucial intercellular network in many cancers, has been extensively studied and therapies targeting the Notch pathway also have been well-studied based on inhibiting various stages of Notch activation. On the other hand, Notch signaling’s role varies between cancers; for instance, in non-small cell lung cancer, Notch1 and Notch2 have opposing effects compared to their roles in embryonal brain tumors. In this study, we assumed a scenario of multiple cancers with contrasting Notch signaling pathways and explored optimal targeted therapies for reducing cancer cells by developing two mathematical models with contrasting Notch signaling pathways. The proposed therapies were compared with existing ones, and strategies were investigated to reduce cancer cell numbers for different stage of cancer. We found that that multiple cancers with contrasting Notch networks can be controlled by a common targeted signal network. Combination therapy enhancing Notch production may be most effective in early-stage cancer, while cleavage therapies may be more effective in late-stage cancer. Our study also suggests that optimal treatment should consider the cancer stage, with careful selection and ordering of medication therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0b0aca24e705cffc10843b8484f56eae84233b" target='_blank'>
              Optimal targeted therapy for multiple cancers based on contrastive Notch signaling networks
              </a>
            </td>
          <td>
            Tamaki Wakamoto, S. Seirin-Lee
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3b8fcb2d930ae2a4d496aebe20c735fc43c688" target='_blank'>
              Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping
              </a>
            </td>
          <td>
            Herman Netskar, Aline Pfefferle, J. Goodridge, Ebba Sohlberg, Olli Dufva, Sarah A. Teichmann, D. Brownlie, J. Michaëlsson, N. Marquardt, Trevor Clancy, Amir Horowitz, Karl-Johan Malmberg
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Poor prognosis and drug resistance in glioblastoma (GBM) can result from cellular heterogeneity and treatment-induced shifts in phenotypic states of tumor cells, including dedifferentiation into glioma stem-like cells (GSCs). This rare tumorigenic cell subpopulation resists temozolomide, undergoes proneural-to-mesenchymal transition (PMT) to evade therapy, and drives recurrence. Through inference of transcriptional regulatory networks (TRNs) of patient-derived GSCs (PD-GSCs) at single-cell resolution, we demonstrate how the topology of transcription factor interaction networks drives distinct trajectories of cell-state transitions in PD-GSCs resistant or susceptible to cytotoxic drug treatment. By experimentally testing predictions based on TRN simulations, we show that drug treatment drives surviving PD-GSCs along a trajectory of intermediate states, exposing vulnerability to potentiated killing by siRNA or a second drug targeting treatment-induced transcriptional programs governing nongenetic cell plasticity. Our findings demonstrate an approach to uncover TRN topology and use it to rationally predict combinatorial treatments that disrupt acquired resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad1f578ef16e0b9da3d9d4afd31f2cea877a074" target='_blank'>
              Gene regulatory network topology governs resistance and treatment escape in glioma stem-like cells
              </a>
            </td>
          <td>
            James H Park, P. Hothi, Adrián López García de Lomana, Min Pan, Rachel Calder, S. Turkarslan, Wei-Ju Wu, Hwahyung Lee, Anoop P. Patel, C. Cobbs, Sui Huang, N. S. Baliga
          </td>
          <td>2024-06-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive cancer with a dismal 5-year survival of < 7%, despite the addition of immunotherapy to first-line chemotherapy. Specific tumor biomarkers, such as delta-like ligand 3 (DLL3) and schlafen11 (SLFN11), may enable the selection of more efficacious, novel immunomodulating targeted treatments like bispecific T-cell engaging monoclonal antibodies (tarlatamab) and chemotherapy with PARP inhibitors. However, obtaining a tissue biopsy sample can be challenging in SCLC. Circulating tumor cells (CTCs) have the potential to provide molecular insights into a patient's cancer through a "simple" blood test. CTCs have been studied for their prognostic ability in SCLC; however, their value in guiding treatment decisions is yet to be elucidated. This review explores novel and promising targeted therapies in SCLC, summarizes current knowledge of CTCs in SCLC, and discusses how CTCs can be utilized for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6329cd4b280a0524de74c052643de178015b821" target='_blank'>
              Circulating tumor cells: advancing personalized therapy in small cell lung cancer patients.
              </a>
            </td>
          <td>
            Prajwol Shrestha, Steven Kao, Veronica K Cheung, Wendy A Cooper, Nico van Zandwijk, J. E. Rasko, D. Yeo
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Bone pain is a presenting feature of bone cancers such as osteosarcoma (OS), relayed by skeletal-innervating peripheral afferent neurons. Potential functions of tumor-associated sensory neurons in bone cancers beyond pain sensation are unknown. To uncover neural regulatory functions, a chemical-genetic approach in mice with a knock-in allele for TrkA was used to functionally perturb sensory nerve innervation during OS growth and disease progression. TrkA inhibition in transgenic mice led to significant reductions in sarcoma-associated sensory innervation and vascularization, tumor growth and metastasis, and prolonged overall survival. Single-cell transcriptomics revealed that sarcoma denervation was associated with phenotypic alterations in both OS tumor cells and cells within the tumor microenvironment, and with reduced calcitonin gene-related peptide (CGRP) and vascular endothelial growth factor (VEGF) signaling. Multimodal and multi-omics analyses of human OS bone samples and human dorsal root ganglia neurons further implicated peripheral innervation and neurotrophin signaling in OS tumor biology. In order to curb tumor-associated axonal ingrowth, we next leveraged FDA-approved bupivacaine liposomes leading to significant reductions in sarcoma growth, vascularity, as well as alleviation of pain. In sum, TrkA-expressing peripheral neurons positively regulate key aspects of OS progression and sensory neural inhibition appears to disrupt calcitonin receptor signaling (CALCR) and VEGF signaling within the sarcoma microenvironment leading to significantly reduced tumor growth and improved survival. These data suggest that interventions to prevent pathological innervation of osteosarcoma represent a novel adjunctive therapy to improve clinical outcomes and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38e8f176dcc8dd09eaf33b5874e7e585465ebfc" target='_blank'>
              TrkA+ sensory neurons regulate osteosarcoma proliferation and vascularization to promote disease progression
              </a>
            </td>
          <td>
            Q. Qin, Sowmya Ramesh, Zhao Li, Lingke Zhong, M. Cherief, Mary Archer, X. Xing, N. Thottappillil, M. Gomez-Salazar, Mingxin Xu, Manyu Zhu, Leslie Chang, Ankit Uniyal, Khadija Mazhar, Monisha Mittal, Edward F. McCarthy, Carol D. Morris, Benjamin Levi, Yun Guan, Thomas L. Clemens, Theodore J. Price, Aaron W. James
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Breast cancer is the second most diagnosed cancer, as well as the primary cause of cancer death in women worldwide. Of the different breast cancer subtypes, triple-negative breast cancer (TNBC) is particularly aggressive and is associated with poor prognosis. Black women are two to three times more likely to be diagnosed with TNBCs than white women. Recent experimental evidence suggests that basal-like TNBCs may derive from luminal cells which acquire basal characteristics through phenotypic plasticity, a newly recognized hallmark of cancer. Whether chemical exposures can promote phenotypic plasticity in breast cells is poorly understood. Methods To investigate further, we developed a high-content immunocytochemistry assay using normal human breast cells to test whether chemical exposures can impact luminal/basal plasticity by unbiased quantification of keratin 14 (KRT14), a basal-myoepithelial marker; keratin 8 (KRT8), a luminal-epithelial marker; and Hoechst 33342, a DNA marker. Six cell lines established from healthy tissue from donors to the Susan G. Komen Normal Tissue Bank were exposed for 48 hours to three different concentrations (0.1μM, 1μM, and 10μM) of eight ubiquitous chemicals (arsenic, BPA, BPS, cadmium, copper, DDE, lead, and PFNA), with documented exposure disparities in US Black women, in triplicate. Automated fluorescence image quantification was performed using Cell Profiler software, and a random-forest classifier was trained to classify individual cells as KRT8 positive, KRT14 positive, or hybrid (both KRT8 and KRT14 positive) using Cell Profiler Analyst. Results and discussion Results demonstrated significant concentration-dependent increases in hybrid populations in response to BPA, BPS, DDE, and PFNA. The increase in hybrid populations expressing both KRT14 and KRT8 is indicative of a phenotypically plastic progenitor-like population in line with known theories of carcinogenesis. Furthermore, BPA, BPS, DDE, and copper produced significant increases in cell proliferation, which could be indicative of a more malignant phenotype. These results further elucidate the relationship between chemical exposure and breast phenotypic plasticity and highlight potential environmental factors that may impact TNBC risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e98af388cf82a8a70746f82327ab464041f396" target='_blank'>
              Investigating phenotypic plasticity due to toxicants with exposure disparities in primary human breast cells in vitro
              </a>
            </td>
          <td>
            Jade Schroeder, Katelyn M. Polemi, Anagha Tapaswi, Laurie K. Svoboda, Jonathan Z. Sexton, Justin A. Colacino
          </td>
          <td>2024-06-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="CD36 expression in both immune and non-immune cells is known to be directly involved in cancer metastasis. Extracellular vesicles (EVs) secreted by malignant melanocytes play a vital role in developing tumor-promoting microenvironments, but it is unclear whether this is mediated through CD36. To understand the role of CD36 in melanoma, we first analyzed the SKCM dataset for clinical prognosis, evaluated the percentage of CD36 in lymphatic fluid-derived EVs (LEVs), and tested whether melanoma-derived EVs increase CD36 expression and induce M2-macrophage-like characteristics. Furthermore, we performed a multiplex immunofluorescence (MxIF) imaging analysis to evaluate the CD36 expression and its colocalization with various other cells in the lymph node (LN) of patients and control subjects. Our findings show that cutaneous melanoma patients have a worse clinical prognosis with high CD36 levels, and a higher percentage of CD36 in total LEVs were found at baseline in melanoma patients compared to control. We also found that monocytic and endothelial cells treated with melanoma EVs expressed more CD36 than untreated cells. Furthermore, melanoma-derived EVs can regulate immunosuppressive macrophage-like characteristics by upregulating CD36. The spatial imaging data show that cells in tumor-involved sentinel LNs exhibit a higher probability of CD36 expression than cells from control LNs, but this was not statistically significant. Conclusively, our findings demonstrated that CD36 plays a vital role in controlling the immunosuppressive microenvironment in the LN, which can promote the formation of a protumorigenic niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b61561168088d12ac63643c9b9a3c660069c384" target='_blank'>
              Melanoma-Derived Extracellular Vesicles Induce CD36-Mediated Pre-Metastatic Niche
              </a>
            </td>
          <td>
            Shankar Suman, W. Nevala, Alexey A. Leontovich, Caitlin Ward, James W Jakub, Yohan Kim, Liyi Geng, Noah A. Stueven, Chathu L. Atherton, Raymond M. Moore, Jill M. Schimke, Fabrice Lucien-Matteoni, Sarah A. McLaughlin, S. Markovic
          </td>
          <td>2024-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Glioblastoma poses a formidable challenge among primary brain tumors: its tumorigenic stem cells, capable of self-renewal, proliferation, and differentiation, contribute substantially to tumor initiation and therapy resistance. These glioblastoma stem cells (GSCs), resembling conventional stem and progenitor cells, adopt pathways critical for tissue development and repair, promoting uninterrupted tumor expansion. Long non-coding RNAs (lncRNAs), a substantial component of the human transcriptome, have garnered considerable interest for their pivotal roles in normal physiological processes and cancer pathogenesis. They display cell- or tissue-specific expression patterns, and extensive investigations have highlighted their impact on regulating GSC properties and cellular differentiation, thus offering promising avenues for therapeutic interventions. Consequently, lncRNAs, with their ability to exert regulatory control over tumor initiation and progression, have emerged as promising targets for innovative glioblastoma therapies. This review explores notable examples of GSC-associated lncRNAs and elucidates their functional roles in driving glioblastoma progression. Additionally, we delved deeper into utilizing a 3D in vitro model for investigating GSC biology and elucidated four primary methodologies for targeting lncRNAs as potential therapeutics in managing glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ed3ed8d6a1c37f2db5f6f03648c49bfe8a7955" target='_blank'>
              Glioblastoma stem cell long non-coding RNAs: therapeutic perspectives and opportunities
              </a>
            </td>
          <td>
            R. Hazra, Rinku Debnath, Arati Tuppad
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive neuroendocrine skin cancer with a ~50% response rate to immune checkpoint blockade (ICB) therapy. To identify predictive biomarkers, we integrated bulk and single-cell RNA-seq with spatial transcriptomics from a cohort of 186 samples from 116 patients, including bulk RNA-seq from 14 matched pairs pre- and post-ICB. In non-responders, tumors show evidence of increased tumor proliferation, neuronal stem cell markers, and IL-1. Responders have increased type I/II interferons and pre-existing tissue resident (Trm) CD8 or Vd1 gd T cells that functionally converge with overlapping antigen-specific transcriptional programs and clonal expansion of public TCRs. Spatial transcriptomics demonstrated co-localization of T cells with B and dendritic cells, which supply chemokines and co-stimulation. Lastly, ICB significantly increased clonal expansion or recruitment of Trm and Vd1 cells in tumors specifically in responders, underscoring their therapeutic importance. These data identify potential clinically actionable biomarkers and therapeutic targets for MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc8fff7c0c805192f9c9a67df2e29b038f02c52" target='_blank'>
              Pre-existing skin-resident CD8 and gd T cell circuits mediate immune response in Merkel cell carcinoma and predict immunotherapy efficacy.
              </a>
            </td>
          <td>
            Z. Reinstein, Yue Zhang, Oscar E. Ospina, Matt D Nichols, Victoria A Chu, Alvaro de Mingo Pulido, Karol Prieto, Jonathan V Nguyen, Rui Yin, Carlos Moran Segura, Ahmed Usman, B. Sell, Spencer Ng, Janis V de la Iglesia, Sunandana Chandra, Jeffrey A Sosman, Raymond J. Cho, Jeffrey B. Cheng, E. Ivanova, S. Koralov, R. Slebos, C. Chung, N. Khushalani, Jane L. Messina, A. Sarnaik, J. Zager, V. Sondak, Charles J. Vaske, Sungjune Kim, A. Brohl, Xinlei Mi, Brian G. Pierce, Xuefeng Wang, Brooke L. Fridley, Kenneth Y. Tsai, Jaehyuk Choi
          </td>
          <td>2024-07-26</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604ce3b0087ea6b1d65fa383e425e9a900dcd14b" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite extensive research efforts, cancer continues to stand as one of the leading causes of death on a global scale. To gain profound insights into the intricate mechanisms underlying cancer onset and progression, it is imperative to possess methodologies that allow the study of cancer cells at the single-cell level, focusing on critical parameters such as cell morphology, metabolism, and molecular characteristics. These insights are essential for effectively discerning between healthy and cancerous cells and comprehending tumoral progression. Recent advancements in microscopy techniques have significantly advanced the study of cancer cells, with Raman microspectroscopy (RM) emerging as a particularly powerful tool. Indeed, RM can provide both biochemical and spatial details at the single-cell level without the need for labels or causing disruptions to cell integrity. Moreover, RM can be correlated with other microscopy techniques, creating a synergy that offers a spectrum of complementary insights into cancer cell morphology and biology. This review aims to explore the correlation between RM and other microscopy techniques such as confocal fluoresce microscopy (CFM), atomic force microscopy (AFM), digital holography microscopy (DHM), and mass spectrometry imaging (MSI). Each of these techniques has their own strengths, providing different perspectives and parameters about cancer cell features. The correlation between information from these various analysis methods is a valuable tool for physicians and researchers, aiding in the comprehension of cancer cell morphology and biology, unraveling mechanisms underlying cancer progression, and facilitating the development of early diagnosis and/or monitoring cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/493ff8b7363d13b88cedadb4ac3e4389b6265366" target='_blank'>
              Correlative Raman Imaging: Development and Cancer Applications
              </a>
            </td>
          <td>
            Hossein Khadem, Maria Mangini, Somayeh Farazpour, Anna Chiara De Luca
          </td>
          <td>2024-06-28</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell type-specific actions of disease genes add a significant layer of complexity to the genetic architecture underlying diseases, obscuring our understanding of disease mechanisms. Single-cell omics have revealed the functional roles of genes at the cellular level, identifying cell types critical for disease progression. Often, a gene impact on disease through its altered network within specific cell types, rather than mere changes in expression levels. To explore the cell type-specific roles of disease genes, we developed HCNetlas (human cell network atlas), a resource cataloging cell type-specific gene networks (CGNs) for various healthy tissue cells. We also devised three network analysis methods to investigate cell type-specific functions of disease genes. These methods involve comparing HCNetlas CGNs with those derived from disease-affected tissue samples. These methods find that systemic lupus erythematosus genes predominantly function in myeloid cells, and Alzheimer’s disease genes mainly play roles in inhibitory and excitatory neurons. Moreover, they show many lung cancer genes exert their roles in immune cells. These findings suggest that HCNetlas has the potential to link disease-associated genes to cell types of action, facilitating development of cell type-resolved diagnostics and therapeutic strategies for complex human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a845b0d8454821603b2600b7feec0b82eddfab6" target='_blank'>
              HCNetlas: Human cell network atlas enabling cell type-resolved disease genetics
              </a>
            </td>
          <td>
            Jiwon Yu, Junha Cha, Geon Koh, Insuk Lee
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hypopharyngeal squamous cell carcinoma (SCC) is a highly aggressive cancer with a poor prognosis, particularly in advanced stages where concurrent chemoradiotherapy (CCRT) is used for treatment. However, resistance to CCRT poses a significant challenge, often leading to treatment failure and disease progression. This study explores the tumor microenvironment (TME) of hypopharyngeal SCC to understand the molecular mechanisms underlying CCRT resistance. Using spatial transcriptomics (ST), we analyzed tissue samples from patients with locally advanced hypopharyngeal SCC, distinguishing between those who were CCRT-resistant and those who were CCRT-naive. The analysis revealed six distinct cellular clusters within the TME, including a prominent epithelio-immune cellular area in CCRT-resistant tissues. SPP1 was identified as a key gene with significantly higher expression in CCRT-resistant samples, specifically within macrophages. Further investigation showed that SPP1+ macrophages interacted with malignant epithelial cells through SPP1-CD44 and SPP1-ITGB1 ligand-receptor pairs. These interactions were primarily localized in the peri-tumoral and intra-tumoral regions, highlighting their potential role in driving CCRT resistance. Our findings suggest that SPP1+ macrophages contribute to the resistant phenotype in hypopharyngeal SCC by modulating the TME and interacting with cancer cells. Understanding these interactions offers valuable insights into the mechanisms of CCRT resistance and may inform the development of targeted therapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d07108a9430fa63556b14b31c6b832a1633f5b4" target='_blank'>
              Spatial transcriptomics unveils landscape of resistance to concurrent chemo-radiotherapy in hypopharyngeal squamous cell carcinoma: the role of SPP1+ macrophages
              </a>
            </td>
          <td>
            J. Ohn, Sungwoo Bae, Hongyoon Choi, In Gul Kim, Kwon Joong Na, Eun-Jae Chung
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The rapid development of single-cell sequencing provides unparalleled opportunity for delineating the heterogeneous characteristics of individual cells, while current methodologies fall short in accurately deciphering the origins of cellular heterogeneity and pinpointing the states of cell fate decisions. In this study, we introduce a novel approach termed Single-cell Reinforcement Learning (scRL) by integrating a framework of reinforcement learning into single-cell data analysis with the assistance of an actor-critic architecture. scRL reveals that the intensity of fate decisions precedes the maximum potential of lineage differentiation and actual gene expression, which enables the forecasting and perturbation of cellular fate in an earlier stage. The superiority of scRL in assessing fate decisions is confirmed in several different patterns of single-cell datasets. By using scRL, we disclose a distinct role of Dapp1 in myeloid lineage decision during hematopoietic stem cells (HSCs) differentiation and uncover the priority of erythroid fate decision occurring post irradiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dcd7d2e6003e1ee97fadd6079d9c94bb4456e7d" target='_blank'>
              Reinforcement learning guides single-cell sequencing in decoding lineage and cell fate decisions
              </a>
            </td>
          <td>
            Zeyu Fu, Song Wang, Kangfu Sun, Baichuan Xu, Xianpeng Ye, Zhaoyang Wen, M. Shen, Mo Chen, Fang Chen, Yang Xu, Shilei Chen, Junping Wang
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a86fe03c5eeba9c5514e3529ad3773b3e2ecef7" target='_blank'>
              Three-dimensional environment sensitizes pancreatic cancer cells to the anti-proliferative effect of budesonide by reprogramming energy metabolism
              </a>
            </td>
          <td>
            Eduardo Ibello, Federica Saracino, Donatella Delle Cave, S. Buonaiuto, Filomena Amoroso, Gennaro Andolfi, Marco Corona, O. Guardiola, V. Colonna, Bruno Sainz, Lucia Altucci, Dario De Cesare, G. Cobellis, E. Lonardo, E. Patriarca, Cristina d’Aniello, G. Minchiotti
          </td>
          <td>2024-06-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba17e91584024423ac0c6e3cc0e42afd48ee9aa" target='_blank'>
              Emerging advances in defining the molecular and therapeutic landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, N. Takahashi, Subhamoy Chakraborty, Naoko Takebe, Amin H. Nassar, Nagla A Karim, S. Puri, Abdul Rafeh Naqash
          </td>
          <td>2024-07-04</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by a balance between basal and classical epithelial cancer cell states, with basal dominance associating with chemoresistance and a dismal prognosis. Targeting oncogenic KRAS, the primary driver of pancreatic cancer, shows early promise in clinical trials but efficacy is limited by acquired resistance. Using genetically engineered mouse models and patient-derived xenografts, we find that basal PDAC cells are highly sensitive to KRAS inhibitors. Employing fluorescent and bioluminescent reporter systems, we longitudinally track cell-state dynamics in vivo and reveal a rapid, KRAS inhibitor-induced enrichment of the classical state. Lineage-tracing identifies these enriched classical PDAC cells to be a reservoir for disease relapse. Genetic ablation of the classical cell-state is synergistic with KRAS inhibition, providing a pre-clinical proof-of-concept for this therapeutic strategy. Our findings motivate combining classical-state directed therapies with KRAS inhibitors to deepen responses and counteract resistance in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04662024790cb8192cdf2a7ce3c66046d31ffca3" target='_blank'>
              A classical epithelial state drives acute resistance to KRAS inhibition in pancreas cancer.
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Stefan R Torborg, Jung Yun Kim, Olivera Grbovic-Huezo, Huijin Feng, Z. Tarcan, Hulya Sahin Ozkan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, I. I. C. Chio, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-07-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is a multifaceted condition characterized by significant diversity in its molecular and pathological subtypes and clinical manifestation. Despite the progress made in the treatment of DLBCL through the development of novel drugs, an estimated one-third of patients encounter relapse or acquire refractory disease. The tumor microenvironment (TME) of DLBCL, a complex network consisting of cellular and noncellular components that engage in interactions with the tumor, is a parameter that is gaining increasing attention. The TME comprises both the immune and nonimmune microenvironments. The immune microenvironment comprises natural killer (NK) cells, dendritic cells (DCs), tumor-associated macrophages (TAMs), neutrophils, myeloid-derived suppressor cells (MDSCs), and T and B lymphocytes. The nonimmune microenvironment consists of the extracellular matrix (ECM), cancer-associated fibroblasts (CAFs), mesenchymal stromal cells, and other molecules that are secreted. Despite ongoing research, the exact impact of these components and their interaction on the progression of the disease remains elusive. A comprehensive review of significant discoveries concerning the cellular and noncellular constituents, molecular characteristics, and treatment response and prognosis of the TME in DLBCL, as well as the potential targeting of the TME with novel therapeutic approaches, is provided in this article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842f1b179cae9c35039714b759e7f43b1c1c5ae1" target='_blank'>
              Unraveling the Immune Microenvironment in Diffuse Large B-Cell Lymphoma: Prognostic and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            E. Koumpis, A. Papoudou-Bai, K. Papathanasiou, E. Kolettas, Panagiotis Kanavaros, E. Hatzimichael
          </td>
          <td>2024-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980719e09512c7538d00c8fd96d0dd2473599e42" target='_blank'>
              Loss of tumor suppressors promotes inflammatory tumor microenvironment and enhances LAG3+T cell mediated immune suppression
              </a>
            </td>
          <td>
            Sara Zahraeifard, Zhiguang Xiao, Jae Young So, Abdul Ahad, Selina Montoya, Woo-Yong Park, Trinadharao Sornapudi, Tiffany Andohkow, Abigail Read, Noemi Kedei, Vishal N. Koparde, H. Yang, M. Lee, Nathan Wong, Maggie Cam, Kun Wang, E. Ruppin, Ji Luo, C. Hollander, Li Yang
          </td>
          <td>2024-07-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2526497a55bfddb0a519406f7853c48bb5f37540" target='_blank'>
              Digital spatial profiling of the microenvironment of muscle invasive bladder cancer
              </a>
            </td>
          <td>
            Michael Eyers, J. Irlam, Gayle Marshall, V. Smith, Alexander Baker, Lucy Frost, P. Hoskin, Ananya Choudhury, C. West
          </td>
          <td>2024-06-18</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gastric cancer, the fifth most prevalent cancer worldwide, is often diagnosed in advanced stages with limited treatment options. Examining the tumor microenvironment (TME) and its metabolic reprogramming can provide insights for better diagnosis and treatment. This study investigates the link between TME factors and metabolic activity in gastric cancer using bulk and single-cell RNA-sequencing data. We identified two molecular subtypes in gastric cancer by analyzing the distinct expression patterns of 81 prognostic genes related to the TME and metabolism, which exhibited significant protein-level interactions. The high-risk subtype had increased stromal content, fibroblast and M2 macrophage infiltration, elevated glycosaminoglycans/glycosphingolipids biosynthesis, and fat metabolism, along with advanced clinicopathological features. It also exhibited low mutation rates and microsatellite instability, associating it with the mesenchymal phenotype. In contrast, the low-risk group showed higher tumor content and upregulated protein and sugar metabolism. We identified a 15-gene prognostic signature representing these characteristics, including CPVL, KYNU, CD36, and GPX3, strongly correlated with M2 macrophages, validated through single-cell analysis and an internal cohort. Despite resistance to immunotherapy, the high-risk group showed sensitivity to molecular targeted agents directed at IGF-1R (BMS-754807) and the PI3K-mTOR pathways (AZD8186, AZD8055). We experimentally validated these promising drugs for their inhibitory effects on MKN45 and MKN28 gastric cells. This study unveils the intricate interplay between TME and metabolic pathways in gastric cancer, offering potential for enhanced diagnosis, patient stratification, and personalized treatment. Understanding molecular features in each subtype enriches our comprehension of gastric cancer heterogeneity and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9428220090a4179bb1f023968e0100b63f3ef4" target='_blank'>
              Dissecting gastric cancer heterogeneity and exploring therapeutic strategies using bulk and single-cell transcriptomic analysis and experimental validation of tumor microenvironment and metabolic interplay
              </a>
            </td>
          <td>
            Xiantao Lin, Ping Yang, MingKun Wang, Xiuting Huang, Baiyao Wang, Chengcong Chen, Anan Xu, Jiazuo Cai, Muhammad Khan, Sha Liu, Jie Lin
          </td>
          <td>2024-06-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Substantial heterogeneity in molecular features, patient prognoses, and therapeutic responses in head and neck squamous cell carcinomas (HNSCC) highlights the urgent need to develop molecular classifications that reliably and accurately reflect tumor behavior and inform personalized therapy. Here, we leveraged the similarity network fusion bioinformatics approach to jointly analyze multi-omics datasets spanning copy number variations, somatic mutations, DNA methylation, and transcriptomic profiling and derived a prognostic classification system for HNSCC. The integrative model consistently identified three subgroups (IMC1-3) with specific genomic features, biological characteristics, and clinical outcomes across multiple independent cohorts. The IMC1 subgroup included proliferative, immune-activated tumors and exhibited a more favorable prognosis. The IMC2 subtype harbored activated EGFR signaling and an inflamed tumor microenvironment with cancer-associated fibroblast/vascular infiltrations. Alternatively, the IMC3 group featured highly aberrant metabolic activities and impaired immune infiltration and recruiting. Pharmacogenomics analyses from in silico predictions and from patient-derived xenograft model data unveiled subtype-specific therapeutic vulnerabilities including sensitivity to cisplatin and immunotherapy in IMC1 and EGFR inhibitors (EGFRi) in IMC2, which was experimentally validated in patient-derived organoid models. Two signatures for prognosis and EGFRi sensitivity were developed via machine learning. Together, this integrative multi-omics clustering for HNSCC improves current understanding of tumor heterogeneity and facilitates patient stratification and therapeutic development tailored to molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c3e275f858f75f81c9f658d4d03e003b3adc1b" target='_blank'>
              Integrative multi-omics analyses identify molecular subtypes of head and neck squamous cell carcinoma with distinct therapeutic vulnerabilities.
              </a>
            </td>
          <td>
            Pengfei Diao, Yibin Dai, An Wang, Xiaoxuan Bu, Ziyu Wang, Jin Li, Yaping Wu, Hongbing Jiang, Yanling Wang, Jie Cheng
          </td>
          <td>2024-07-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e877f4d47a5850d3ceab780a2d2a2472c654fb" target='_blank'>
              High-dimensional single-cell analysis of human natural killer cell heterogeneity
              </a>
            </td>
          <td>
            Lucas Rebuffet, J. Melsen, Bertrand Escalière, Daniela Basurto-Lozada, A. Bhandoola, N. Björkström, Y. Bryceson, R. Castriconi, Frank Cichocki, Marco Colonna, Daniel M Davis, Andreas Diefenbach, Yi Ding, M. Haniffa, Amir Horowitz, Lewis L. Lanier, Karl-Johan Malmberg, Jeffrey S Miller, Lorenzo Moretta, Émilie Narni-Mancinelli, L. A. O'Neill, C. Romagnani, Dylan G Ryan, S. Sivori, Dan Sun, Constance Vagne, Eric Vivier
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>3</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaac037eca190874316bd2b34e72ed4d604d818c" target='_blank'>
              Sulfatide imaging identifies tumor cells in colorectal cancer peritoneal metastases
              </a>
            </td>
          <td>
            G. M. Sarcinelli, L. Varinelli, S. Ghislanzoni, F. Padelli, D. Lorenzini, A. Vingiani, M. Milione, M. Guaglio, S. Kusamura, M. Deraco, G. Pruneri, M. Gariboldi, D. Baratti, I. Bongarzone
          </td>
          <td>2024-06-28</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e1810e2d03e004eeb58c6ca7cae3dc32ba5c9" target='_blank'>
              Single-cell transcriptional analysis of irradiated skin reveals changes in fibroblast subpopulations and variability in caveolin expression
              </a>
            </td>
          <td>
            Lionel E. Kameni, Michelle F. Griffin, Charlotte E. Berry, Siavash Shariatzadeh, Mauricio A. Downer, Caleb Valencia, Alexander Z. Fazilat, Rahim S Nazerali, Arash Momeni, Michael Januszyk, M. Longaker, Derrick C. Wan
          </td>
          <td>2024-06-26</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Mutations in polymerases Pold1 and Pole exonuclease domains in humans are associated with increased cancer incidence, elevated tumor mutation burden (TMB) and response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond. Here we generated Pold1 and Pole proofreading mutator mice and show that ICB treatment of mice with high TMB tumors did not improve survival as only a subset of tumors responded. Similarly, introducing the mutator alleles into mice with Kras/p53 lung cancer did not improve survival, however, passaging mutator tumor cells in vitro without immune editing caused rejection in immune-competent hosts, demonstrating the efficiency by which cells with antigenic mutations are eliminated. Finally, ICB treatment of mutator mice earlier, before observable tumors delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the use of ICB in individuals at high risk for cancer prevention. Highlights Germline somatic and conditional Pold1 and Pole exonuclease domain mutations in mice produce a mutator phenotype. Spontaneous cancers arise in mutator mice that have genomic features comparable to human tumors with these mutations. ICB treatment of mutator mice with tumors did not improve survival as only a subset of tumors respond. Introduction of the mutator alleles into an autochthonous mouse lung cancer model also did not produce immunogenic tumors, whereas passaging mutator tumor cells in vitro caused immune rejection indicating efficient selection against antigenic mutations in vivo. Prophylactic ICB treatment delayed cancer onset, improved survival, and selected for tumors with no aneuploidy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e6ba5a717c7dd3412aa2d3806c5dc666fb03f9" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer and Extends Survival in Murine DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. C. Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the continuous analysis of changes in phenotype, such as in vivo and ex vivo morphology and motion tracking of cellular proteins and organelles, over time-frames spanning the minute-hour scales, can provide important insights into patient treatment options. Despite improvements in the diagnosis and therapy of many types of breast cancer (BC), many aggressive forms, such as receptor triple-negative cancers, are associated with the worst patient outcomes; though initially effective in reducing tumor burden for some patients, acquired resistance to cytotoxic chemotherapy is almost universal, and there is no rationale for identifying intrinsically drug-resistant and drug-sensitive patient populations before initiating therapy. During cell division, the receptor triple-negative MDA-MB-231 mitotic spindles are the largest in comparison to other BC cell lines. Many of the MDA-MB-231 spindles exhibit rapid lateral twisting during metaphase, which remains unaffected by knockdown of the oncogene Myc and treatment with inhibitors of the serine/threonine-protein kinase B-Raf and the epidermal growth factor receptor (EGFR), alone or in any combination. In this manuscript, we outline a strategy for the selection of the most optimal tubulin inhibitor based on the ability to affect MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f15c7bb309b3d3ce06b26fa0c307848870d30" target='_blank'>
              Mitosis, Cytoskeleton Regulation, and Drug Resistance in Receptor Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Alexandre Matov
          </td>
          <td>2024-07-26</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30215d511cd6c83bd7fde1586962eb127268dc49" target='_blank'>
              Comprehensive analysis of single cell and bulk RNA sequencing reveals the heterogeneity of melanoma tumor microenvironment and predicts the response of immunotherapy.
              </a>
            </td>
          <td>
            Yuan Zhang, Cong Zhang, Jing He, Guichuan Lai, Wenlong Li, Haijiao Zeng, Xiaoni Zhong, Biao Xie
          </td>
          <td>2024-06-19</td>
          <td>Inflammation research : official journal of the European Histamine Research Society ... [et al.]</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7d40b34ce0a1270d80bd0970ece2da4263143d" target='_blank'>
              Integrative single-cell and bulk transcriptomes analyses reveals heterogeneity of serine-glycine-one-carbon metabolism with distinct prognoses and therapeutic vulnerabilities in HNSCC
              </a>
            </td>
          <td>
            Lixuan Wang, Rongchun Yang, Yue Kong, Jing Zhou, Yingyao Chen, Rui Li, Chuwen Chen, Xinran Tang, Xiaobing Chen, J. Xia, Xi-juan Chen, Bing-lin Cheng, Xianyue Ren
          </td>
          <td>2024-06-17</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Introduction Field cancerization is suggested to arise from imbalanced differentiation in individual basal progenitor cells leading to clonal expansion of mutant cells that eventually replace the epithelium, although without evidence. Methods We performed deep sequencing analyses to characterize the genomic and transcriptomic landscapes of field change in two patients with synchronous aerodigestive tract tumors. Results Our data support the emergence of numerous genetic alterations in cancer-associated genes but refutes the hypothesis that founder mutation(s) underpin this phenomenon. Mutational signature analysis identified defective homologous recombination as a common underlying mutational process unique to synchronous tumors. Discussion Our analyses suggest a common etiologic factor defined by mutational signatures and/or transcriptomic convergence, which could provide a therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae797c96e298d92f976b952e9d1ccd0f24f2a970" target='_blank'>
              Transcriptomic convergence despite genomic divergence drive field cancerization in synchronous squamous tumors
              </a>
            </td>
          <td>
            Qiu Xuan Tan, Nicholas B. Shannon, Weng Khong Lim, J. X. Teo, Daniel R. Y. Yap, Sze Min Lek, Joey W. S. Tan, Shih Jia, Josephine Hendrikson, Ying Liu, Gillian Ng, Clara Y. L. Chong, Wanyu Guo, Kelvin K. N. Koh, C. Ng, Vikneswari Rajasegaran, Jolene S.M. Wong, Chin Jin Seo, C. Ong, Tony K.H. Lim, Bin Tean Teh, Oi Lian Kon, C. Chia, Khee Chee, N. G. Iyer, C. Ong, M. Shammas, Sarah Mazzilli, Prajish Iyer
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="In the quest for improved therapeutics targeting immune checkpoints (ICs), we turn to spontaneously developing dog (canine) cancers, which are unique models that genetically and clinically mirror human equivalents. Despite its potential, canine cancer immunology remains largely unexplored. Here, we examine the RNA-seq-based expression of 44 ICs across 14 canine cancer types and an extensive human dataset. We unveil diverse canine IC expression patterns and unique human IC signatures that reflect the histological type and primary site of cancer. We uncover a striking similarity between canine brain cancers, osteosarcoma, and their human counterparts, identifying them as prospective immunotherapy models. Four ICs—CD160, A2AR, NKG2A, and OX40—are key to the differences observed between species. Moreover, individual patient IC signatures exhibit varying alignment with their respective cancer types, a finding with profound implications for personalized human therapy. This exploration illuminates new aspects of canine and human cancer immunology, setting the stage for discoveries at their crossroads.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f77ebd885f80c0d134fa31e9198696b90786b6" target='_blank'>
              Barking Up the Right Tree: Immune Checkpoint Signatures of Human and Dog Cancers
              </a>
            </td>
          <td>
            M. Kocikowski, M. Yébenes Mayordomo, Javier Alfaro, Maciej Parys
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Attenuated measles virus (MV) exerts its oncolytic activity in malignant pleural mesothelioma (MPM) cells that lack type-I interferon (IFN-I) production or responsiveness. However, other cells in the tumor microenvironment (TME), such as myeloid cells, possess functional antiviral pathways. In this study, we aimed to characterize the interplay between MV and the myeloid cells in human MPM. We cocultured MPM cell lines with monocytes or macrophages and infected them with MV. We analyzed the transcriptome of each cell type and studied their secretion and phenotypes by high-dimensional flow cytometry. We also measured transgene expression using an MV encoding GFP (MV-GFP). We show that MPM cells drive the differentiation of monocytes into M2-like macrophages. These macrophages inhibit GFP expression in tumor cells harboring a defect in IFN-I production and a functional signaling downstream of the IFN-I receptor, while having minimal effects on GFP expression in tumor cells with defect of responsiveness to IFN-I. Interestingly, inhibition of the IFN-I signaling by ruxolitinib restores GFP expression in tumor cells. Upon MV infection, cocultured macrophages express antiviral pro-inflammatory genes and induce the expression of IFN-stimulated genes in tumor cells. MV also increases the expression of HLA and costimulatory molecules on macrophages and their phagocytic activity. Finally, MV induces the secretion of inflammatory cytokines, especially IFN-I, and PD-L1 expression in tumor cells and macrophages. These results show that macrophages reduce viral proteins expression in some MPM cell lines through their IFN-I production and generate a pro-inflammatory interplay that may stimulate the patient’s anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42c6c72014e77858ecbe23e2ace88e24f5956005" target='_blank'>
              Interplay between oncolytic measles virus, macrophages and cancer cells induces a proinflammatory tumor microenvironment
              </a>
            </td>
          <td>
            Camille Chatelain, Laurine Berland, Marion Grard, Nicolas Jouand, Judith Fresquet, Joëlle Nader, Ugo Hirigoyen, Tacien Petithomme, C. Combredet, E. Pons-Tostivint, D. Fradin, Lucas Treps, C. Blanquart, Nicolas Boisgerault, Frédéric Tangy, J. Fonteneau
          </td>
          <td>2024-07-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58322ca71db2c109e772f2eb0f78a110f1a84ce4" target='_blank'>
              Senescent endothelial cells promote liver metastasis of uveal melanoma in single-cell resolution
              </a>
            </td>
          <td>
            Liang Ma, Xiaoyu He, Yidian Fu, Shengfang Ge, Zhi Yang
          </td>
          <td>2024-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9120ab8ec6281dd0ae5d345b0a1c2eb5d5dfca5c" target='_blank'>
              Mutation analysis in individual circulating tumor cells depicts intratumor heterogeneity in melanoma
              </a>
            </td>
          <td>
            Mark Sementsov, L. Ott, Julian Kött, A. Sartori, Amélie Lusque, Sarah Degenhardt, Bertille Segier, I. Heidrich, B. Volkmer, Rüdiger Greinert, Peter Mohr, R. Simon, J. Stadler, Darryl Irwin, Claudia Koch, Antje Andreas, Benjamin Deitert, V. Thewes, A. Trumpp, A. Schneeweiss, Yassine Belloum, S. Peine, H. Wikman, S. Riethdorf, Stefan W. Schneider, C. Gebhardt, Klaus Pantel, Laura Keller
          </td>
          <td>2024-06-19</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Tumor structure is heterogeneous and complex, and it is difficult to obtain complete characteristics by two‐dimensional analysis. The aim of this study was to visualize and characterize volumetric vascular information of clear cell renal cell carcinoma (ccRCC) tumors using whole tissue phenotyping and three‐dimensional light‐sheet microscopy. Here, we used the diagnosing immunolabeled paraffin‐embedded cleared organs pipeline for tissue clearing, immunolabeling, and three‐dimensional imaging. The spatial distributions of CD34, which targets blood vessels, and LYVE‐1, which targets lymphatic vessels, were examined by calculating three‐dimensional density, vessel length, vessel radius, and density curves, such as skewness, kurtosis, and variance of the expression. We then examined those associations with ccRCC outcomes and genetic alteration state. Formalin‐fixed paraffin‐embedded tumor samples from 46 ccRCC patients were included in the study. Receiver operating characteristic curve analyses revealed the associations between blood vessel and lymphatic vessel distributions and pathological factors such as a high nuclear grade, large tumor size, and the presence of venous invasion. Furthermore, three‐dimensional imaging parameters stratified ccRCC patients regarding survival outcomes. An analysis of genomic alterations based on volumetric vascular information parameters revealed that PI3K‐mTOR pathway mutations related to the blood vessel radius were significantly different. Collectively, we have shown that the spatial elucidation of volumetric vasculature information could be prognostic and may serve as a new biomarker for genomic alterations. High‐end tissue clearing techniques and volumetric immunohistochemistry enable three‐dimensional analysis of tumors, leading to a better understanding of the microvascular structure in the tumor space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63465b7766c5458fa29857848c92dc3c85ce6311" target='_blank'>
              Volumetric imaging of the tumor microvasculature reflects outcomes and genomic states of clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Yuta Kaneko, Tsukasa Masuda, Kimiharu Takamatsu, S. Mikami, Kohei Nakamura, Hiroshi Nishihara, R. Mizuno, Nobuyuki Tanaka, M. Oya
          </td>
          <td>2024-06-26</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="PURPOSE
In multiple myeloma (MM), therapy-induced clonal evolution is associated with treatment resistance and is one of the most important hindrances toward a cure for MM. To further understand the molecular mechanisms controlling the clonal evolution of MM, we applied single-cell RNA-sequencing (scRNA-seq) to paired diagnostic and post-treatment bone marrow (BM) samples.


EXPERIMENTAL DESIGN
scRNA-seq was performed on 38 BM samples from patients with monoclonal gammopathy of undetermined significance (MGUS) (n = 1), MM patients at diagnosis (n = 19), MM post-treatment (n = 17), and one healthy donor. The single-cell transcriptome data of malignant plasma cells and the surrounding immune microenvironment were analyzed.


RESULTS
Profiling by scRNA-seq data revealed three primary trajectories of transcriptional evolution after treatment: clonal elimination in patients with undetectable minimal residual disease (MRD-), as well as clonal stabilization and clonal selection in detectable MRD (MRD+) patients. We noted a metabolic shift towards fatty acid oxidation in cycling-resistant plasma cells (PCs), while selective PCs favored the NF-κB pathway. Intriguingly, when comparing the genetic and transcriptional dynamics, we found a significant correlation between genetic and non-genetic factors in driving the clonal evolution. Furthermore, we identified variations in cellular interactions between malignant plasma cells and the tumor microenvironment (TME). Selective PCs showed the most robust cellular interactions with the TME.


CONCLUSIONS
These data suggest that MM cells could rapidly adapt to induction treatment through transcriptional adaptation, metabolic adaptation, and specialized immune evasion. Targeting therapy-induced resistance mechanisms may help to avert refractory disease in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4967fd77a9456d257d33b503e573eb358b554d" target='_blank'>
              Identification of therapy-induced clonal evolution and resistance pathways in minimal residual clones in multiple myeloma through single-cell sequencing.
              </a>
            </td>
          <td>
            Jian Cui, Xiaoyun Li, S. Deng, C. Du, H. Fan, W. Yan, Jingyu Xu, Xiaoqing Li, Tengteng Yu, Shuaishuai Zhang, R. Lv, W. Sui, Mu Hao, Xin Du, Yan Xu, S. Yi, D. Zou, Tao Cheng, Lugui Qiu, Xin Gao, Gang An
          </td>
          <td>2024-06-20</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="CD8+ T cells play pivotal roles in combating intracellular pathogens and eliminating malignant cells in cancer. However, the prognostic role of CD8+ T cells in ovarian carcinoma is insufficiently exploited. Herein, through univariate Cox regression along with least absolute shrinkage and selection operator (LASSO) regression analyses, we developed a novel prognostic model based on CD8+ T cell markers identified by single-cell sequencing (scRNA-seq) analyses. Patient grouping by the median risk score reveals an excellent prognostic efficacy of this model in both training and validation cohorts. Of note, patients classified as low-risk group exhibit a dramatically improved prognosis. In addition, higher enrichment level of immune-related pathways and increased infiltration level of multiple immune cells are found in patients with lower risk score. Importantly, low-risk patients also exhibited higher response rate to immunotherapies. Summarily, this developed CD8+ T cell-associated prognostic model serves as an excellent predictor for clinical outcomes and aids in guiding therapeutic strategy choices for ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/409d1a4ad42cb1c888c3ed09d0a52f53075d8d47" target='_blank'>
              Comprehensive single-cell and bulk RNA-seq analyses reveal a novel CD8+ T cell-associated prognostic signature in ovarian cancer
              </a>
            </td>
          <td>
            Yi-Ling Han, Zhou Fang, Zhijie Gao, Wen-Ge Li, Jing Yang
          </td>
          <td>2024-06-25</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The ability to characterize immune cells and explore the molecular interactions that govern their functions has never been greater, fueled in recent years by the revolutionary advance of single-cell analysis platforms. However, precisely how immune cells respond to different stimuli and where differentiation processes and effector functions operate remain incompletely understood. Inferring cellular fate within single-cell transcriptomic analyses is now omnipresent, despite the assumptions typically required in such analyses. Recently developed experimental models support dynamic analyses of the immune response, providing insights into the temporal changes that occur within cells and the tissues in which such transitions occur. Here we will review these approaches and discuss how these can be combined with single-cell technologies to develop a deeper understanding of the immune responses that should support the development of better therapeutic options for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df359d3a21a0e0d9ae308793407ba6d01736c657" target='_blank'>
              Decoding changes in tumor-infiltrating leukocytes through dynamic experimental models and single-cell technologies.
              </a>
            </td>
          <td>
            Colin Y C Lee, M. Clatworthy, David R Withers
          </td>
          <td>2024-06-10</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the epidermal growth factor receptor (EGFR) are common in non-small cell lung cancer (NSCLC), particularly in never-smoker patients. However, these mutations are not always carcinogenic, and have recently been reported in histologically normal lung tissue from patients with and without lung cancer. To investigate the outcome of EGFR mutation in healthy lung stem cells, we grow murine alveolar type II organoids monoclonally in a three-dimensional Matrigel. Our experiments show that the EGFR-L858R mutation induces a change in organoid structure: mutated organoids display more ‘budding’, in comparison with non-mutant controls, which are nearly spherical. We perform on-lattice computational simulations, which suggest that this can be explained by the concentration of division among a small number of cells on the surface of the mutated organoids. We are currently unable to distinguish the cell-based mechanisms that lead to this spatial heterogeneity in growth, but suggest a number of future experiments which could be used to do so. We suggest that the likelihood of L858R-fuelled tumorigenesis is affected by whether the mutation arises in a spatial environment that allows the development of these surface protrusions. These data may have implications for cancer prevention strategies and for understanding NSCLC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e046dae2eb0578408d47ec2942e59fd61d21fe2" target='_blank'>
              An agent-based modelling framework to study growth mechanisms in EGFR-L858R mutant cell alveolar type II cells
              </a>
            </td>
          <td>
            Helena Coggan, C. Weeden, Philip Pearce, M. Dalwadi, Alastair Magness, C. Swanton, Karen M. Page
          </td>
          <td>2024-07-01</td>
          <td>Royal Society Open Science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BACKGROUND
Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain.


METHODS
Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package "Seurat" was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes.


RESULTS
Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA.


CONCLUSION
A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer.
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The global incidence of cancer is increasing, with estimates suggesting that there will be 26 million new cases and 17 million deaths per year by 2030. Cancer stem cells (CSCs) and extracellular vesicles (EVs) are key to the resistance and advancement of cancer. They play a crucial role in tumor dynamics and resistance to therapy. CSCs, initially discovered in acute myeloid leukemia, are well-known for their involvement in tumor initiation, progression, and relapse, mostly because of their distinct characteristics, such as resistance to drugs and the ability to self-renew. EVs, which include exosomes, microvesicles, and apoptotic bodies, play a vital role in facilitating communication between cells within the tumor microenvironment (TME). They have a significant impact on cellular behaviors and contribute to genetic and epigenetic changes. This paper analyzes the mutually beneficial association between CSCs and EVs, emphasizing their role in promoting tumor spread and developing resistance mechanisms. This review aims to investigate the interaction between these entities in order to discover new approaches for attacking the complex machinery of cancer cells. It highlights the significance of CSCs and EVs as crucial targets in the advancement of novel cancer treatments, which helps stimulate additional research, promote progress in ideas for cancer treatment, and provide renewed optimism in the effort to reduce the burden of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b17e9728aa76af6a02ab4a44623aab4c0608b6" target='_blank'>
              Extracellular vesicles and cancer stem cells: a deadly duo in tumor progression
              </a>
            </td>
          <td>
            Akram Tayanloo-Beik, Azin Eslami, Masoumeh Sarvari, H. Jalaeikhoo, Mohsen Rajaeinejad, Mohsen Nikandish, Ali Faridfar, Mostafa Rezaei-Tavirani, A. R. Mafi, Bagher Larijani, B. Arjmand
          </td>
          <td>2024-07-18</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Due to bladder tumors' contact with urine, urine-derived cells (UDCs) may serve as a surrogate for monitoring the tumor microenvironment (TME) in bladder cancer (BC). However, the composition of UDCs and the extent to which they mirror the tumor remain poorly characterized. We generated the first single-cell RNA-sequencing of BC patient UDCs with matched tumor and peripheral blood mononuclear cells (PBMC). BC urine was more cellular than healthy donor (HD) urine, containing multiple immune populations including myeloid cells, CD4+ and CD8+ T cells, natural killer (NK) cells, B cells, and dendritic cells (DCs) in addition to tumor and stromal cells. Immune UDCs were transcriptionally more similar to tumor than blood. UDCs encompassed cytotoxic and activated CD4+ T cells, exhausted and tissue-resident memory CD8+ T cells, macrophages, germinal-center-like B cells, tissue-resident and adaptive NK cells, and regulatory DCs found in tumor but lacking or absent in blood. Our findings suggest BC UDCs may be surrogates for the TME and serve as therapeutic biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5067f8b153d210fb128105c0d112c21eaf4fb125" target='_blank'>
              Urine scRNAseq reveals new insights into the bladder tumor immune microenvironment.
              </a>
            </td>
          <td>
            Michelle A Tran, Dina Youssef, S. Shroff, Disha Chowhan, Kristin G Beaumont, R. Sebra, Reza Mehrazin, P. Wiklund, Jenny J Lin, Amir Horowitz, A. Farkas, M. Galsky, J. Sfakianos, Nina Bhardwaj
          </td>
          <td>2024-06-07</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ae438e85d1f5a8bcba96886546fdec069966a7" target='_blank'>
              IRF8 deficiency-induced myeloid-derived suppressor cell promote immune evasion in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zhen-Yang Gao, Shang Liu, Han Xiao, Mengdi Li, Wan-gang Ren, Lin Xu, Zhong-Min Peng
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf3fb50dabab3c628e11b6f429b04114a9702d4" target='_blank'>
              Brain Chimeroids reveal individual susceptibility to neurotoxic triggers.
              </a>
            </td>
          <td>
            Noelia Antón-Bolaños, Irene Faravelli, Tyler Faits, Sophia Andreadis, Rahel Kastli, Sebastiano Trattaro, Xian Adiconis, Anqi Wei, Abhishek Sampath Kumar, Daniela J. Di Bella, M. Tegtmeyer, R. Nehme, J. Levin, Aviv Regev, Paola Arlotta
          </td>
          <td>2024-06-26</td>
          <td>Nature</td>
          <td>5</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Esophageal cancer (ESCA) is a form of malignant tumor associated with chronic inflammation and immune dysregulation. However, the specific immune status and key mechanisms of immune regulation in this disease require further exploration. Methods To investigate the features of the human ESCA tumor immune microenvironment and its possible regulation, we performed mass cytometry by time of flight, single-cell RNA sequencing, multicolor fluorescence staining of tissue, and flow cytometry analyses on tumor and paracancerous tissue from treatment-naïve patients. Results We depicted the immune landscape of the ESCA and revealed that CD8+ (tissue-resident memory CD8+ T cells (CD8+ TRMs) were closely related to disease progression. We also revealed the heterogeneity of CD8+ TRMs in the ESCA tumor microenvironment (TME), which was associated with their differentiation and function. Moreover, the subset of CD8+ TRMs in tumor (called tTRMs) that expressed high levels of granzyme B and immune checkpoints was markedly decreased in the TME of advanced ESCA. We showed that tTRMs are tumor effector cells preactivated in the TME. We then demonstrated that conventional dendritic cells (cDC2s) derived from intermediate monocytes (iMos) are essential for maintaining the proliferation of CD8+ TRMs in the TME. Our preliminary study showed that hypoxia can promote the apoptosis of iMos and impede the maturation of cDC2s, which in turn reduces the proliferative capacity of CD8+ TRMs, thereby contributing to the progression of cancer. Conclusions Our study revealed the essential antitumor roles of CD8+ TRMs and preliminarily explored the regulation of the iMo/cDC2/CD8+ TRM immune axis in the human ESCA TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f97387951043d32c32d4552a785f22b20a0ed2" target='_blank'>
              Hypoxia inhibits the iMo/cDC2/CD8+ TRMs immune axis in the tumor microenvironment of human esophageal cancer
              </a>
            </td>
          <td>
            Chuanqiang Wu, Huan Yu, Fuxiang Liang, Xiancong Huang, Bin Jiang, Zhiling Lou, Yafei Liu, Zixiang Wu, Qi Wang, Hong Shen, Ming Chen, Pin Wu, Ming Wu
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef1d96a67b79b906dd8d65b931e0cf0af5d2a6eb" target='_blank'>
              A single-cell chromatin accessibility dataset of human primed and naïve pluripotent stem cell-derived teratoma
              </a>
            </td>
          <td>
            Jinxiu Li, Lixin Fu, Yunpan Li, Wei Sun, Yao Yi, Wenqi Jia, Haiwei Li, Hao Liu, Pengcheng Guo, Yang Wang, Yue Shen, Xiuqing Zhang, Yuan Lv, Baoming Qin, Wenjuan Li, Chuanyu Liu, Longqi Liu, Md. Abdul Mazid, Yiwei Lai, Miguel A Esteban, Yu Jiang, Liang Wu
          </td>
          <td>2024-07-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Plexiform neurofibromas in neurofibromatosis type 1 (pNF1), characterized by aggressive growth and local invasiveness, are comprised primarily of Schwann cell-derived tumor cells (Nf1−/−) and the tumor microenvironment (TME). Fibroblasts (Nf1+/−), the most abundant cell type in the TME, are known to significantly contribute to pNF1 formation, but their precise role in the subsequent tumor progression is poorly understood. Using our established three-dimensional (3D) culture models with patient-derived pNF1 cells and fibroblasts, we observed that the fibroblast-derived secretome stimulates the growth and local invasiveness of pNF1 spheroids. Depletion of small extracellular vesicles (sEVs) from the fibroblast secretome completely eliminated the paracrine effect on pNF1 spheroid growth. These results suggest that targeting paracrine interactions between pNF1 tumor cells and fibroblasts may offer a potential therapeutic approach for reducing pNF1 tumor progression. Abstract Plexiform neurofibromas (PNs) occur in about a half of neurofibromatosis type 1 (NF1) patients and have garnered significant research attention due to their capacity for growth and potential for malignant transformation. NF1 plexiform neurofibroma (pNF1) is a complex tumor composed of Schwann cell-derived tumor cells (Nf1−/−) and the tumor microenvironment (TME). Although it has been widely demonstrated that the TME is involved in the formation of neurofibromas, little is known about the effects of the TME on the subsequent progression of human pNF1. Elucidating the molecular interactions between tumor cells and the TME may provide new therapeutic targets to reduce the progression of pNF1. In the present study, we focused on the contributions of fibroblasts, the most abundant cell types in the TME, to the growth of pNF1. To simulate the TME, we used a three-dimensional (3D) coculture model of immortalized pNF1 tumor cells (Nf1−/−) and primary fibroblasts (Nf1+/−) derived from pNF1 patients. We performed live-cell imaging of 3D/4D (3D in real-time) cultures through confocal microscopy followed by 3D quantitative analyses using advanced imaging software. The growth of pNF1 spheroids in 3D cocultures with fibroblasts was significantly greater than that of pNF1 spheroids in 3D monocultures. An increase in the growth of pNF1 spheroids also occurred when they were cultured with conditioned media (CM) from fibroblasts. Moreover, fibroblast-derived CM increased the invasive outgrowth and further local invasion of pNF1 spheroids. Interestingly, when small extracellular vesicles (sEVs) were depleted from the fibroblast-derived CM, the stimulation of the growth of pNF1 spheroids was lost. Our results suggest that fibroblast-derived sEVs are a therapeutic target for reducing the growth of pNF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab89f35d535a3eadd54db0a654a65465e448f0f4" target='_blank'>
              A Fibroblast-Derived Secretome Stimulates the Growth and Invasiveness of 3D Plexiform Neurofibroma Spheroids
              </a>
            </td>
          <td>
            Kyungmin Ji, George J. Schwenkel, Raymond R. Mattingly, H. Sundararaghavan, Z. G. Zhang, Michael Chopp
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cells show remarkable plasticity and can switch lineages in response to the tumor microenvironment. Cellular plasticity drives invasiveness and metastasis and helps cancer cells to evade therapy by developing resistance to radiation and cytotoxic chemotherapy. Increased understanding of cell fate determination through epigenetic reprogramming is critical to discover how cancer cells achieve transcriptomic and phenotypic plasticity. Glioblastoma is a perfect example of cancer evolution where cells retain an inherent level of plasticity through activation or maintenance of progenitor developmental programs. However, the principles governing epigenetic drivers of cellular plasticity in glioblastoma remain poorly understood. Here, using machine learning (ML) we employ cross-patient prediction of transcript expression using a combination of epigenetic features (ATAC-seq, CTCF ChIP-seq, RNAPII ChIP-seq, H3K27Ac ChIP-seq, and RNA-seq) of glioblastoma stem cells (GSCs). We investigate different ML and deep learning (DL) models for this task and build our final pipeline using XGBoost. The model trained on one patient generalizes to another one suggesting that the epigenetic signals governing gene transcription are consistent across patients even if GSCs can be very different. We demonstrate that H3K27Ac is the epigenetic feature providing the most significant contribution to cross-patient prediction of gene expression. In addition, using H3K27Ac signals from patients-derived GSCs, we can predict gene expression of human neural crest stem cells suggesting a shared developmental epigenetic trajectory between subpopulations of these malignant and benign stem cells. Our cross-patient ML/DL models determine weighted patterns of influence of epigenetic marks on gene expression across patients with glioblastoma and between GSCs and neural crest stem cells. We propose that broader application of this analysis could reshape our view of glioblastoma tumor evolution and inform the design of new epigenetic targeting therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f06de9705bc503e591aae9a45d9fab127b04d79" target='_blank'>
              Machine learning on multiple epigenetic features reveals H3K27Ac as a driver of gene expression prediction across patients with glioblastoma
              </a>
            </td>
          <td>
            Yusuke Suita, Hardy Bright, Yuan Pu, M. D. Toruner, Jordan Idehen, Nikos Tapinos, Ritambhara Singh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH), the relative expansion of mutant clones, is derived from hematopoietic stem cells (HSCs) with acquired somatic or cytogenetic alterations that improve cellular fitness. Individuals with CH have a higher risk for hematological and non-hematological diseases, such as cardiovascular disease, and have an overall higher mortality rate. Originally thought to be restricted to a small fraction of elderly people, recent advances in single-cell sequencing and bioinformatics have revealed that CH with multiple expanded mutant clones is universal in the elderly population. Just a few years ago, phylogenetic reconstruction across the human lifespan and novel sensitive sequencing techniques showed that CH can start earlier in life, decades before it was thought possible. These studies also suggest that environmental factors acting through aberrant inflammation might be a common theme promoting clonal expansion and disease progression. However, numerous aspects of this phenomenon remain to be elucidated and the precise mechanisms, context-specific drivers, and pathways of clonal expansion remain to be established. Here, we review our current understanding of the cellular mechanisms driving CH and specifically focus on how pro-inflammatory factors affect normal and mutant HSC fates to promote clonal selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2987594f9a59a1022a92c2e62d730f17ee567fcb" target='_blank'>
              Decoding Clonal Hematopoiesis: Emerging Themes and Novel Mechanistic Insights
              </a>
            </td>
          <td>
            S. Pendse, Dirk Loeffler
          </td>
          <td>2024-07-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Human and mouse natural killer (NK) cells have been shown to develop memory-like function after short-term exposure to the cocktail of IL-12/15/18 or to overnight co-culture with some tumor cell lines. The resulting cells retain enhanced lytic ability for up to 7 days as well as after cryopreservation, and memory-like NK cells (mlNK) have been shown to induce complete remissions in patients with hematological malignancies. No single phenotype has been described for mlNK and the physiological changes induced by the short-term cytokine or tumor-priming which are responsible for these enhanced functions have not been fully characterized. Here, we have generated mlNK by cytokine and tumor-priming to find commonalities to better define the nature of NK cell “memory” in vitro and, for the first time, in vivo. Methods We initiated mlNK in vitro from healthy donors with cytokines (initiated cytokine-induced memory-like (iCIML)-NK) and by tumor priming (TpNK) overnight and compared them by high-dimensional flow cytometry, proteomic and metabolomic profiling. As a potential mechanism of enhanced cytolytic function, we analyzed the avidity of binding of the mlNK to NK-resistant tumors (z-Movi). We generated TpNK from healthy donors and from cancer patients to determine whether mlNK generated by interaction with a single tumor type could enhance lytic activity. Finally, we used a replication-incompetent tumor cell line (INKmune) to treat patients with myeloid leukaemias to potentiate NK cell function in vivo. Results Tumor-primed mlNK from healthy donors and patients with cancer showed increased cytotoxicity against multiple tumor cell lines in vitro, analogous to iCIML-NK cells. Multidimensional cytometry identified distinct memory-like profiles of subsets of cells with memory-like characteristics; upregulation of CD57, CD69, CD25 and ICAM1. Proteomic profiling identified 41 proteins restricted to mlNK cells and we identified candidate molecules for the basis of NK memory which can explain how mlNK overcome inhibition by resistant tumors. Finally, of five patients with myelodysplastic syndrome or refractory acute myeloid leukemia treated with INKmune, three responded to treatment with measurable increases in NK lytic function and systemic cytokines. Conclusions NK cell “memory” is a physiological state associated with resistance to MHC-mediated inhibition, increased metabolic function, mitochondrial fitness and avidity to NK-resistant target cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc17a047cba9c227f6ae901a0377db375c35dbc" target='_blank'>
              Proteomic and phenotypic characteristics of memory-like natural killer cells for cancer immunotherapy
              </a>
            </td>
          <td>
            Helena Arellano-Ballestero, Agnieszka Zubiak, Chris Dally, Kim H. Orchard, Aljawharah Alrubayyi, X. Charalambous, Melina Michael, Rob Torrance, Trinity Eales, Kushal Das, M. Tran, May Sabry, D. Peppa, Mark W. Lowdell
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b564fa77c9ecb886651a32d892f7a470d9fe2e51" target='_blank'>
              Reconstructed colorectal cancer model to dissect the anti-tumor effect of mesenchymal stromal cells derived extracellular vesicles
              </a>
            </td>
          <td>
            E. D’Angelo, Sarah Tassinari, Andrea Biccari, S. Crotti, Francesca Sensi, Asia Marangio, Ombretta Repetto, Giuseppe Corona, Linda Bellucci, Federica Antico, Federico Caicci, G. Spolverato, Giovanni Montini, Benedetta Bussolati, Marco Agostini, Federica Collino
          </td>
          <td>2024-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Somatic mutations in T cells can cause cancer but also have implications for immunological diseases and cell therapies. The mutation spectrum in nonmalignant T cells is unclear. Here, we examined somatic mutations in CD4+ and CD8+ T cells from 90 patients with hematological and immunological disorders and used T cell receptor (TCR) and single-cell sequencing to link mutations with T cell expansions and phenotypes. CD8+ cells had a higher mutation burden than CD4+ cells. Notably, the biggest variant allele frequency (VAF) of non-synonymous variants was higher than synonymous variants in CD8+ T cells, indicating non-random occurrence. The non-synonymous VAF in CD8+ T cells strongly correlated with the TCR frequency, but not age. We identified mutations in pathways essential for T cell function and often affected lymphoid neoplasia. Single-cell sequencing revealed cytotoxic TEMRA phenotypes of mutated T cells. Our findings suggest that somatic mutations contribute to CD8+ T cell expansions without malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6cc1c82bbc03e28686df6931fd8499cc4ab7b1" target='_blank'>
              Somatic mutations associate with clonal expansion of CD8+ T cells
              </a>
            </td>
          <td>
            S. Lundgren, Mikko Myllymäki, Timo Järvinen, Mikko A. I. Keränen, J. Theodoropoulos, Johannes Smolander, Daehong Kim, U. Salmenniemi, G. Walldin, P. Savola, T. Kelkka, H. Rajala, Eva Hellström-Lindberg, M. Itälä-Remes, M. Kankainen, S. Mustjoki
          </td>
          <td>2024-06-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1c7c67a035b24ba72789a261c045fe83ce8bf3" target='_blank'>
              Combined SERS-Raman screening of HER2-overexpressing or silenced breast cancer cell lines
              </a>
            </td>
          <td>
            Sara Spaziani, Alessandro Esposito, Giovannina Barisciano, G. Quero, Satheeshkumar Elumalai, Manuela Leo, Vittorio Colantuoni, Maria Mangini, Marco Pisco, L. Sabatino, Anna Chiara De Luca, Andrea Cusano
          </td>
          <td>2024-06-20</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7c925865a9ee48af207a91531daee25f18e384" target='_blank'>
              Spatiotemporal heterogeneity of LMOD1 expression summarizes two modes of cell communication in colorectal cancer
              </a>
            </td>
          <td>
            Jie-Pin Li, Yuan-Jie Liu, Yang Li, Yimei Yin, Qian-wen Ye, Zhi-hua Lu, Yu-wei Dong, Jin-yong Zhou, Xi Zou, Yu-gen Chen
          </td>
          <td>2024-06-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 What are the molecular and cellular features that contribute to the transformation of myometrial stem cells into tumor-initiating cells within uterine leiomyoma?



 Transformative changes in differentiation and genetic processes, in a distinct subset of M/LM cells drives their progression into a tumorigenic state.



 Uterine leiomyomas (LM), which are benign smooth muscle-derived tumours of myometrium, impact approximately 75% of women, causing significant physical and psychological challenges and imposing substantial healthcare costs. The widespread consensus postulates a monoclonal origin for LM, implying their derivation from a dysregulated single multipotent stem cell that could give rise to tumor-initiating cells (TICs). While previous studies have shown cellular heterogeneity in both myometrium (M) and LM, the precise identity of the originating cells is unknown.



 Prospective, observational, and biomedical study conducted at Hospital La Fe (Valencia, Spain) for one year. Eight sample pairs of LM and M underwent single-nuclei RNAseq (snRNA-seq; n = 16) and single-cell proteomic (scP; n = 16) analyses, to generate a detailed transcriptomic and proteomic map decoupled from cell type, state, and spatial location. We further employed single-cell RNA velocity inference for differentiation trajectory [BR1] analysis of mesenchymal cells, followed by the identification of potential driver genes in LM tumorigenesis.



 Upon obtaining informed consent, we procured LM and M samples from eight patients aged 35-50 undergoing hysterectomies. Part of these samples were preserved in paraffin for spatial transcriptomics utilizing VISIUM (10x Genomics). The remaining tissues were dissociated into single-cell suspensions and underwent scRNA-seq and scP using Chromium Controller and Orbitrap Eclipse Tribid mass spectrometry, respectively. The analysis of the collected data was carried out using publicly accessible R/Python tools.



 Analysis of snRNAseq (∼52,000 cells) revealed similar cellular compositions in M and LM, including endothelial, perivascular, smooth muscle cells (SMC), fibroblasts, and immune cells. Further trajectory analysis unveiled a subset of myometrial cells with potential as TICs, exhibiting reduced expression of hormone receptors like PGR. Analysis of these cells [BR1] suggested differentiation into two distinct cell types -fibroblasts and SMCs- with elevated hormone expression over pseudotime. Differential analysis between M and LM also showed a dysregulated transcriptomic profile in the TICs within the tumor, as evidenced by upregulation of RAD51B and HMGA2 and downregulation PGR among other genes. These genes are indicative markers, underscoring their significant roles in the tumorigenesis of LM by influencing hormone response and disrupting DNA repair mechanisms. Spatial transcriptomic analysis confirmed these findings and suggested that the distribution of TICs could be ubiquitous, since the expression of potential myometrial-stem markers, was markedly increased only on certain spots within the tissue across both LM and M. Lastly, scProt profiling (∼5,000 cells) also allowed the identification of putative TICs, which showed upregulation in LM of specific proteins involved in replication, transcription, and translation.



 Our study sets the basis for TICs isolation in LM, yet functional validation studies need to be performed to address the clinical potential of our findings. Further studies including more patients, and addressing racial disparities will help to generalize these findings to a broader population.



 Our findings indicate the presence of a particular subset of M/LM cells whose changes in differentiation path and genetic processes seem to drive the transformation of these cells into a tumorigenic state to develop LM. Targeting these markers might show promise for effectively treating these tumors.



 NCT04214457
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e5f1c7a81c7b6baceb7a49d4c94f2c7bde8e8c" target='_blank'>
              O-264 Profiling Tumor-Initiating Cells in Uterine Leiomyoma through Single-Cell Multi-omics
              </a>
            </td>
          <td>
            A. Machado-Lopez, P. Punzón-Jiménez, J. Llera-Oyola, R. Perez-Moraga, M. Gálvez-Viedma, D. Grases, E. Satorres, E. Porta-Pardo, E. Schooff, B. Roson, C. Simon, A. Mas
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35eea075db8a8c1977ce6381cea9a663e6162ef" target='_blank'>
              PTGR1-mediated immune evasion mechanisms in late-stage triple-negative breast cancer: mechanisms of M2 macrophage infiltration and CD8+ T cell suppression.
              </a>
            </td>
          <td>
            Fang Huang, Fuhe Wang, Qilu Hu, Ying Li, Da Jiang
          </td>
          <td>2024-07-28</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Basal cell carcinoma (BCC) is the most frequently diagnosed skin cancer and the most common malignancy in humans. Different morphological subtypes of BCC are associated with low- or high-risk of recurrence and aggressiveness, but the underlying biology of how the individual subtypes arise remains largely unknown. Because the majority of BCCs appear to arise from mutations in the same pathway, we hypothesized that BCC development, growth and invasive potential is also influenced by the tumor microenvironment and in particular by cancer-associated fibroblasts (CAFs) and their secreted factors.


OBJECTIVE
We aimed to characterize the stroma of the different BCC subtypes with a focus on CAF populations.


METHODS
To investigate the stromal features of the different BCC subtypes, we applied laser-capture microdissection (LCM) followed by RNA sequencing. A cohort of 15 BCC samples from 5 different "pure" subtypes (superficial, nodular, micronodular, sclerosing and basosquamous; n=3 each) were selected and included in the analysis. Healthy skin was used as a control (n=6). We confirmed the results by immunohistochemistry. We validated our findings in two independent, public single-cell RNA sequencing (scRNAseq) datasets and by RNAscope.


RESULTS
The stroma of the different BCC subtypes have distinct gene expression signatures. Nodular and micronodular seem to have the most similar signatures, while superficial and sclerosing the most different. By comparing low- and high-risk BCC subtypes, we observed that Collagen 10A1 (COL10A1) is overexpressed in the stroma of sclerosing/infiltrative and basosquamous but not micronodular high-risk subtypes. Those findings were confirmed by immunohistochemistry in a cohort of 89 different BCC and 13 healthy skin samples. Moreover, scRNAseq analysis of BCCs of two independent datasets showed that the COL10A1-expressing population of cells is associated with the stroma adjacent to invasive BCC and shows extracellular matrix remodeling features.


CONCLUSION
We identified COL10A1 as a marker of high-risk BCC, in particular of the sclerosing/infiltrative and basosquamous subtypes. We demonstrated at the single cell level that COL10A1 is expressed by a specific CAF population associated with the stroma of invasive BCC. This opens up new tailored treatment options as well as a new prognostic biomarker for BCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5131e4d68fffbdfca6cced052102c01223f2d2d9" target='_blank'>
              COL10A1 expression distinguishes a subset of cancer-associated fibroblasts present in the stroma of high-risk basal cell carcinoma.
              </a>
            </td>
          <td>
            Mauro Esposito, L. Yerly, Prachi Shukla, Victoria Hermes, Federica Sella, Zsolt Balázs, Evelyn Lattmann, A. Tastanova, P. Turko, Ramon Lang, I. Kolm, Ramon Staeger, François Kuonen, Michael Krauthammer, J. Hafner, Mitchell P Levesque, G. Restivo
          </td>
          <td>2024-06-25</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac24e0e7a39e90599e20a198363cbf479bb3749d" target='_blank'>
              Using a pan-cancer atlas to investigate tumour associated macrophages as regulators of immunotherapy response
              </a>
            </td>
          <td>
            Alexander Coulton, Jun Murai, Danwen Qian, Krupa Thakkar, Claire E Lewis, Kevin Litchfield
          </td>
          <td>2024-07-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="PURPOSE
Classic Hodgkin lymphoma (cHL) is a B cell lymphoma that occurs primarily in young adults and, less frequently, in elderly individuals. A hallmark of cHL is the exceptional scarcity (1-5%) of the malignant Hodgkin Reed-Sternberg (HRS) cells within a network of non-malignant immune cells. Molecular determinants governing the relationship between HRS cells and their proximal microenvironment remain largely unknown.


EXPERIMENTAL DESIGN
We performed spatially resolved multiplexed protein imaging and transcriptomic sequencing to characterize HRS cell states, cellular neighborhoods, and gene expression signatures of 23.6 million cells from 36 newly diagnosed Epstein-Barr virus (EBV) positive and EBV-negative cHL tumors.


RESULTS
We show that MHC-I expression on HRS cells is associated with immune inflamed neighborhoods containing CD8+ T cells, MHC-II+ macrophages, and immune checkpoint expression (i.e., PD-1 and VISTA). We identified spatial clustering of HRS cells, consistent with the syncytial variant of cHL, and its association with T cell excluded neighborhoods in a subset of EBV-negative tumors. Finally, a subset of both EBV-positive and EBV-negative tumors contained regulatory T cells high neighborhoods harboring HRS cells with augmented proliferative capacity.


CONCLUSIONS
Our study links HRS cell properties with distinct immunophenotypes and potential immune escape mechanisms in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/645b8738be3b7a5dce2bff2635b7f7c7780b894d" target='_blank'>
              Multiplexed spatial profiling of Hodgkin Reed-Sternberg cell neighborhoods in classic Hodgkin lymphoma.
              </a>
            </td>
          <td>
            Maryam Pourmaleki, Caitlin J Jones, Sabrina D Mellinghoff, Bryan D. Greenstein, Priyadarshini Kumar, Miguel Foronda, Daniel A Navarrete, Carl Campos, M. Roshal, N. Schultz, Sohrab P. Shah, A. Schietinger, N. Socci, Travis J Hollmann, A. Dogan, I. Mellinghoff
          </td>
          <td>2024-07-01</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Targeted imaging of cancer lymphatic metastasis remains challenging due to its highly heterogeneous molecular and phenotypic diversity. Herein, we introduced triple-targeted protein nanoprobes capable of specifically binding to three targets for imaging cancer lymphatic metastasis, through a data-driven design approach combined with a synthetic biology-based assembly strategy. Specifically, to address the diversity of metastatic lymph nodes (LNs), a combination of three targets, including C-X-C motif chemokine receptor 4 (CXCR4), transferrin receptor protein 1 (TfR1) and vascular endothelial growth factor receptor 3 (VEGFR3) was identified, leveraging machine leaning-based bioinformatics analysis and examination of lymphatic metastatic tissues from patients with gastric cancer. Using this identified target combination, ferritin nanocage-based nanoprobes capable of specifically binding to all three targets were designed through the self-assembly of genetically engineered ferritin subunits using a synthetic biology approach. Multiplexed imaging of heterogeneous metastatic LNs using these nanoprobes resulted in approximately 80% binding with heterogeneous tumor cells, which was 60% higher than that of single-targeted nanoprobes in a polyclonal lymphatic metastasis animal model. In 19 freshly resected human gastric specimens, the signal from the triple-targeted nanoprobes significantly differentiated metastatic LNs from benign LNs, in high accordance with pathological examination. This study not only provides an effective nanoprobe for imaging highly heterogeneous lymphatic metastasis but also proposes a potential strategy for guiding the design of targeted nanomedicines for cancer lymphatic metastasis. This article is protected by copyright. All rights reserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/595024240b444e89cf21627f1500404247fe4401" target='_blank'>
              Data-Driven Design of Triple-Targeted Protein Nanoprobes for Multiplexed Imaging of Cancer Lymphatic Metastasis.
              </a>
            </td>
          <td>
            Guodong Shen, Xiao-hua Jia, Tianyi Qi, Zhenhua Hu, Anqi Xiao, Qiqi Liu, Keyu He, Weihong Guo, Dan Zhang, Wanjun Li, Genmao Cao, Guoxin Li, Jie Tian, Xinglu Huang, Yanfeng Hu
          </td>
          <td>2024-06-18</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small RNAs that are often dysregulated in many diseases, including cancers. They are highly tissue-specific and stable, thus, making them particularly useful as biomarkers. As the spatial transcriptomics field advances, protocols that enable highly sensitive and spatially resolved detection become necessary to maximize the information gained from samples. This is especially true of miRNAs where the location their expression within tissue can provide prognostic value with regard to patient outcome. Equally as important as detection are ways to assess and visualize the miRNA's spatial information in order to leverage the power of spatial transcriptomics over that of traditional nonspatial bulk assays. We present a highly sensitive methodology that simultaneously quantitates and spatially detects seven miRNAs in situ on formalin-fixed paraffin-embedded tissue sections. This method utilizes rolling circle amplification (RCA) in conjunction with a dual scanning approach in nanoliter well arrays with embedded hydrogel posts. The hydrogel posts are functionalized with DNA probes that enable the detection of miRNAs across a large dynamic range (4 orders of magnitude) and a limit of detection of 0.17 zeptomoles (1.7 × 10-4 attomoles). We applied our methodology coupled with a data analysis pipeline to K14-Cre Brca1f/fTp53f/f murine breast tumors to showcase the information gained from this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbaad333d7c14fb2504c61497f5a898eaa1ab830" target='_blank'>
              Sensitive Multiplexed MicroRNA Spatial Profiling and Data Classification Framework Applied to Murine Breast Tumors.
              </a>
            </td>
          <td>
            Omar N Mohd, Y. Heng, Lin Wang, Abhishek Thavamani, Erica S. Massicott, Gerburg M Wulf, Frank J Slack, Patrick S. Doyle
          </td>
          <td>2024-07-23</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9c5b5959b41a250a5ef58adb37088e34a07297" target='_blank'>
              Immuno-oncologic profiling of pediatric brain tumors reveals major clinical significance of the tumor immune microenvironment
              </a>
            </td>
          <td>
            Adrian B Levine, L. Nobre, Anirban Das, Scott Milos, V. Bianchi, Monique Johnson, Nick Fernandez, L. Stengs, S. Ryall, Michelle Ku, Mansuba Rana, Benjamin Laxer, J. Sheth, Stefanie-Grace Sbergio, Ivana Fedorakova, Vijay Ramaswamy, J. Bennett, R. Siddaway, U. Tabori, Cynthia Hawkins
          </td>
          <td>2024-07-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d4ba3cf7cf5603a608ca85287f8b1899486efd" target='_blank'>
              Targeting CERS6-AS1/FGFR1 axis as synthetic vulnerability to constrain stromal cells supported proliferation in Mantle cell lymphoma.
              </a>
            </td>
          <td>
            Udita Jindal, Mukesh Mamgain, U. Nath, Isha Sharma, Bhaskar Pant, Ankita Sharma, Archita Gupta, Khaliqur Rahman, Sunil Yadav, Manish Pratap Singh, Shaktiprasad Mishra, Chandra Praksah Chaturvedi, José Courty, Navin Singh, Seema Gupta, Sanjeev Kumar, S. Verma, S. Mallick, A. Gogia, Sunil K Raghav, Jayanta Sarkar, Kinshuk Raj Srivastava, Dipak Datta, Neeraj Jain
          </td>
          <td>2024-07-13</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Neuroblastoma is a childhood developmental cancer; however, its embryonic origins remain poorly understood. Moreover, in-depth studies of early tumor-driving events are limited because of the lack of appropriate models. Herein, we analyzed RNA sequencing data obtained from human neuroblastoma samples and found that loss of expression of trunk neural crest–enriched gene MOXD1 associates with advanced disease and worse outcome. Further, by using single-cell RNA sequencing data of human neuroblastoma cells and fetal adrenal glands and creating in vivo models of zebrafish, chick, and mouse, we show that MOXD1 is a determinate of tumor development. In addition, we found that MOXD1 expression is highly conserved and restricted to mesenchymal neuroblastoma cells and Schwann cell precursors during healthy development. Our findings identify MOXD1 as a lineage-restricted tumor-suppressor gene in neuroblastoma, potentiating further stratification of these tumors and development of novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4a1e60d743c1788421ba02de583dc1b69f419e" target='_blank'>
              MOXD1 is a lineage-specific gene and a tumor suppressor in neuroblastoma
              </a>
            </td>
          <td>
            Elina Fredlund, Stina Andersson, Elien Hilgert, Ezequiel Monferrer, Guadalupe Álvarez-Hernán, Sinan Karakaya, Siebe Loontiens, J. Bek, Tomas Gregor, Estelle Lecomte, Emma Magnusson, Enrika Miltenyte, Marie Cabirol, Michail Kyknas, Niklas Engström, Marie Arsenian Henriksson, Emma Hammarlund, J. Rosenblum, Rosa Noguera, Frank Speleman, J. van Nes, S. Mohlin
          </td>
          <td>2024-06-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung cancer is one of the most prevalent malignancies worldwide, contributing to over a million cancer-related deaths annually. Despite extensive research investigating the genetic factors associated with lung cancer susceptibility and prognosis, few studies have explored genetic predispositions regarding the immune system. This review discusses the most recent genomic findings related to the susceptibility to or protection against lung cancer, patient survival, and therapeutic responses. The results demonstrated the effect of immunogenetic variations in immune system-related genes associated with innate and adaptive immune responses, cytokine, and chemokine secretions, and signaling pathways. These genetic diversities may affect the crosstalk between tumor and immune cells within the tumor microenvironment, influencing cancer progression, invasion, and prognosis. Given the considerable variability in the individual immunegenomics profiles, future studies should prioritize large-scale analyses to identify potential genetic variations associated with lung cancer using highthroughput technologies across different populations. This approach will provide further information for predicting response to targeted therapy and promotes the development of new measures for individualized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/934ee629ca9ff19da207450ac61d6553ddf45bfe" target='_blank'>
              The genomic landscape of the immune system in lung cancer: present insights and continuing investigations
              </a>
            </td>
          <td>
            Mina Roshan-Zamir, Aida Khademolhosseini, Kavi Rajalingam, Abbas Ghaderi, Raja Rajalingam
          </td>
          <td>2024-06-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf72209777e452dc93a39bae0d7ff7fff3f49ae6" target='_blank'>
              Effects of tumor treating fields (TTFields) on human mesenchymal stromal cells.
              </a>
            </td>
          <td>
            M. Strack, Jan Kückelhaus, M. Diebold, Patrick Wuchter, Peter E Huber, Oliver Schnell, R. Sankowski, Marco Prinz, Anca-Ligia Grosu, D. Heiland, N. H. Nicolay, A. Rühle
          </td>
          <td>2024-06-20</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Most melanomas progress from radial to vertical growth phase before spreading locoregionally and distally. Much is still unknown about the metabolic changes in the tumor cells and their microenvironment during this metastatic progression. We aimed to gain new insight into the molecular characteristics of melanoma in regard to spatial lipidomics to deliver new knowledge regarding tumor metastatic progression. We included 10 fresh tumor samples from 10 patients including two in situ melanomas, two invasive primary melanomas, and six metastatic melanomas (four in-transit metastases and two distant metastases). In addition, we analyzed four healthy skin controls from the same patients. Time-of-flight imaging secondary ion mass spectrometry (ToF-SIMS) enabled detailed spatial-lipidomics that could be directly correlated with conventional histopathological analysis of consecutive H&E-stained tissue sections. Significant differences in the lipid profiles were found in primary compared to metastatic melanomas, notably an increase in phosphatidylethanolamine lipids relative to phosphatidylinositol lipids and an increase in GM3 gangliosides in the metastatic samples. Furthermore, analysis of the data from in transit versus distant metastases samples highlighted that specific phospholipids, and a difference in the long versus shorter chain GM3 gangliosides, discriminated the metastatic routes. Further studies are warranted to verify these preliminary findings. Lipidomic changes could serve as a novel biomarker for tumor progression and even serve as a target for novel treatments. Furthermore, analyzing the lipid profiles could help to differentiate between primary and metastatic melanomas in challenging cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd5da01fb93ce9422a464070818f23116949e9f" target='_blank'>
              ToF-SIMS imaging reveals changes in tumor cell lipids during metastatic progression of melanoma.
              </a>
            </td>
          <td>
            N. Neittaanmäki, O. Zaar, Kevin Sjögren Cehajic, Kelly Dimovska Nilsson, D. Katsarelias, R. O. Bagge, John Paoli, John S Fletcher
          </td>
          <td>2024-06-29</td>
          <td>Pigment cell & melanoma research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb93191d6e6b81b569e83a5d6ffc99c7c305e133" target='_blank'>
              Single-cell sequencing depicts tumor architecture and empowers clinical decision in metastatic conjunctival melanoma
              </a>
            </td>
          <td>
            Hanhan Shi, Hao Tian, Tianyu Zhu, Qili Liao, Chang Liu, Peng Yuan, Yongyun Li, Jie Yang, C. Zong, Shichong Jia, Jing Ruan, Shengfang Ge, Renbing Jia, P. Chai, Shiqiong Xu, Xianqun Fan
          </td>
          <td>2024-06-11</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Bispecific antibodies (bsAbs) that bind to two distinct surface antigens on cancer cells are emerging as an appealing therapeutic strategy in cancer immunotherapy. However, considering the vast number of surface proteins, experimental identification of potential antigen pairs that are selectively expressed in cancer cells and not in normal cells is both costly and time-consuming. Recent studies have utilized large bulk RNA-seq databases to propose bispecific targets for various cancers. However, co-expressed pairs derived from bulk RNA-seq do not necessarily indicate true co-expression of both markers in the malignant cell. Single-cell RNA-seq (scRNA-seq) can circumvent this issue but the issues in low-coverage of transcripts impede the large-scale characterization of co-expressed pairs.



 We present a computational pipeline for bsAbs target identification which combines the advantages of bulk and scRNA-seq while minimizing the issues associated with using these approaches separately. We select Hepatocellular Carcinoma (HCC) as a case study to demonstrate the utility of the approach. First, using the bulk RNA-seq samples in the OCTAD database, we identified target pairs that most distinctly differentiate tumor cases from healthy controls. Next, we confirmed our findings on the scRNA-seq database comprising 39,361 healthy cells from vital organs and 18,000 cells from HCC. The top pair was GPC3∼MUC13, where both genes are co-expressed on the surface of over 30% of malignant HCCs and have very low expression in other cells. Finally, we leveraged the emerging spatial transcriptomic to validate the co-expressed pair in situ.



 A standalone R package (https://github.com/Bin-Chen-Lab/bsAbsFinder)



 Supplementary data are available at Bioinformatics Advances online
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2059ebaa34bf1fd7c83bd94cb6ade0525ae3296b" target='_blank'>
              Computational discovery of co-expressed antigens as dual targeting candidates for cancer therapy through bulk, single-cell, and spatial transcriptomics
              </a>
            </td>
          <td>
            E. Chekalin, Shreya Paithankar, Rama Shankar, Jing Xing, Wenfeng Xu, Bin Chen
          </td>
          <td>2024-06-20</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="PURPOSE
Co-occurring mutations in KEAP1 and STK11KRAS have emerged as determinants of survival outcomes in non-small cell lung cancer (NSCLC) patients treated with immunotherapy. However, these mutational contexts identify a fraction of non-responders to immune checkpoint inhibitors. We hypothesized that KEAP1 wild-type tumors recapitulate the transcriptional footprint of KEAP1 mutations, and that this KEAPness phenotype can determine immune responsiveness with higher precision compared to mutation-based models.


EXPERIMENTAL DESIGN
The TCGA was used to infer the KEAPness phenotype and explore its immunological correlates at the pan-cancer level. The association between KEAPness and survival outcomes was tested in two independent cohorts of advanced NSCLC patients treated with immunotherapy and profiled by RNA-Seq (SU2C n=153; OAK/POPLAR n=439). The NSCLC TRACERx421 multi-region sequencing study (tumor regions n=947) was used to investigate evolutionary trajectories.


RESULTS
KEAPness-dominant tumors represented 50% of all NSCLCs and were associated with shorter progression-free survival (PFS) and overall survival (OS) compared to KEAPness-free cases in independent cohorts of NSCLC patients treated with immunotherapy (SU2C PFS P=0.042, OS P=0.008; OAK/POPLAR PFS P=0.0014, OS P<0.001). Patients with KEAPness tumors had survival outcomes comparable to those with KEAP1-mutant tumors. In the TRACERx421, KEAPness exhibited limited transcriptional intratumoral heterogeneity and immune exclusion, resembling the KEAP1-mutant disease. This phenotypic state occurred across genetically divergent tumors, exhibiting shared and private cancer genes under positive selection when compared to KEAP1-mutant tumors.


CONCLUSIONS
We identified a KEAPness phenotype across evolutionary divergent tumors. KEAPness outperforms mutation-based classifiers as a biomarker of inferior survival outcomes in NSCLC patients treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea8af08e6c600cf8a24f58a7981fb66121f005e" target='_blank'>
              Transcriptional phenocopies of deleterious KEAP1 mutations correlate with survival outcomes in lung cancer treated with immunotherapy.
              </a>
            </td>
          <td>
            S. Scalera, B. Ricciuti, D. Marinelli, M. Mazzotta, Laura Cipriani, Giulia Bon, Giulia Schiavoni, I. Terrenato, Alessandro Di Federico, J. Alessi, Maurizio Fanciulli, L. Ciuffreda, F. de Nicola, F. Goeman, G. Caravagna, Daniele Santini, R. De Maria, F. Cappuzzo, G. Ciliberto, M. Jamal-Hanjani, Mark M Awad, N. Mcgranahan, M. Maugeri-Saccà
          </td>
          <td>2024-07-09</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Computational methods that simulate tumors mathematically to describe cellular and molecular interactions are emerging as promising tools to simulate the impact of therapy entirely in silico, potentially greatly accelerating the delivery of new therapeutics to patients. To facilitate the design of dosing regimens and identification of potential biomarkers for immunotherapy, we developed a new computational model to track tumor progression at the organ scale while capturing the spatial heterogeneity of the tumor in HCC. This computational model of spatial quantitative systems pharmacology (spQSP) was designed to simulate the effects of combination immunotherapy. The model was initiated using literature-derived parameter values and fitted to the specifics of HCC. Model validation was done through comparison to spatial multi-omics data from a neoadjuvant HCC clinical trial combining anti-PD-1 immunotherapy and a multitargeted tyrosine kinase inhibitor (TKI) cabozantinib. Validation using spatial proteomics data from Imaging Mass Cytometry (IMC) demonstrated that closer proximity between CD8 T cells and macrophages correlated with non-response. We also compared the model output with Visium spatial transcriptomics (ST) profiling of samples from post-treatment tumor resections in the clinical trial and from another independent study of anti-PD1 monotherapy. ST data confirmed simulation results, suggesting the importance of spatial patterns of tumor vasculature and TGFβ in tumor and immune cell interactions. Our findings demonstrate that incorporating mathematical modeling and computer simulations with high-throughput spatial multi-omics data provides a novel approach for patient outcome prediction and biomarker discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15cf1462eb61e650b451d9180053af320c88c5a8" target='_blank'>
              Integration of Clinical Trial Spatial Multi-omics Analysis and Virtual Clinical Trials Enables Immunotherapy Response Prediction and Biomarker Discovery.
              </a>
            </td>
          <td>
            Shuming Zhang, Atul Deshpande, Babita K Verma, Hanwen Wang, Haoyang Mi, Long Yuan, W. Ho, Elizabeth M. Jaffee, Qingfeng Zhu, Robert A Anders, M. Yarchoan, Lusciane T. Kagohara, E. Fertig, Aleksander S. Popel
          </td>
          <td>2024-06-11</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly invasive, aggressive brain cancer that carries a median survival of 15 months and is resistant to standard therapeutics. Recent studies have demonstrated that intratumoral heterogeneity plays a critical role in promoting resistance by mediating tumor adaptation through microenvironmental cues. GBM can be separated into two distinct regions—a core and a rim, which are thought to drive specific aspects of tumor evolution. These differences in tumor progression are regulated by the diverse biomolecular and biophysical signals in these regions, but the acellular biophysical characteristics remain poorly described. This study investigates the mechanical and ultrastructural characteristics of the tumor extracellular matrix (ECM) in patient-matched GBM core and rim tissues. Seven patient-matched tumor core and rim samples and one non-neoplastic control were analyzed using atomic force microscopy, scanning electron microscopy, and immunofluorescence imaging to quantify mechanical, ultrastructural, and ECM composition changes. The results reveal significant differences in biophysical parameters between GBM core, rim, and non-neoplastic tissues. The GBM core is stiffer, denser, and is rich in ECM proteins hyaluronic acid and tenascin-C when compared to tumor rim and non-neoplastic tissues. These alterations are intimately related and have prognostic effect with stiff, dense tissue correlating with longer progression-free survival. These findings reveal new insights into the spatial heterogeneity of biophysical parameters in the GBM tumor microenvironment and identify a set of characteristics that may correlate with patient prognosis. In the long term, these characteristics may aid in the development of strategies to combat therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5313dd86c28916e3c6cd3d13c84d5b7be07567" target='_blank'>
              The prognostic effect of mechanical, ultrastructural, and ECM signatures in glioblastoma core and rim
              </a>
            </td>
          <td>
            Bradley J. Mahaffey, Zachary P. Fowler, Zoe Lung, Vivien Dang, Hyunchul Lee, Allison McKenzie Johnson, Marco A. Munoz, Dylan A. Goodin, Hermann Frieboes, Brian J. Williams, Joseph Chen
          </td>
          <td>2024-06-24</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer brain metastasis has a poor prognosis, is commonly observed in clinical practice, and the number of cases is increasing as overall cancer survival improves. However, experiments in mouse models have shown that brain metastasis itself is an inefficient process. One reason for this inefficiency is the brain microenvironment, which differs significantly from that of other organs, making it difficult for cancer cells to adapt. The brain microenvironment consists of unique resident cell types such as neurons, oligodendrocytes, astrocytes, and microglia. Accumulating evidence over the past decades suggests that the interactions between cancer cells and glial cells can positively or negatively influence the development of brain metastasis. Nevertheless, elucidating the complex interactions between cancer cells and glial cells remains challenging, in part due to the limitations of existing experimental models for glial cell culture. In this review, we first provide an overview of glial cell culture methods and then examine recent discoveries regarding the interactions between brain metastatic cancer cells and the surrounding glial cells, with a special focus on astrocytes and microglia. Finally, we discuss future perspectives for understanding the multifaceted interactions between cancer cells and glial cells for the treatment of metastatic brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b93951608ce8fdc53c244e85b3946f02850ef4e" target='_blank'>
              Multifaceted interactions between cancer cells and glial cells in brain metastasis.
              </a>
            </td>
          <td>
            Kojiro Ishibashi, E. Hirata
          </td>
          <td>2024-07-11</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, Chin L Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, Michael Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell-derived tumor organoids (STOs) possess a distinct genetic background, making them valuable tools for demonstrating tumor heterogeneity. In order to fulfill the high throughput demands of STO assays, we have developed a microfluidic chip containing 30 000 microwells, which is dedicated to a single cell culture approach for selective expansion and differential induction of cancer stem cells. The microwells are coated with a hydrophilic copolymer to eliminate cell adhesion, and the cell culture is supported by poly(ethylene glycol) (PEG) to establish a nonadhesive culture environment. By utilizing an input cell density of 7 × 103·mL-1, it is possible to construct a 4000 single cell culture system through stochastic cell occupation. We demonstrate that the addition of 15% PEG10000 in the cell culture medium effectively prevents cell loss while facilitating tumor stem cell expansion. As were demonstrated by HCT116, HT29, and SW480 colon cancer cells, the microfluidic approach achieved a STO formation rate of ∼20%, resulting in over 800 STOs generated from a single culture. Comprehensive analysis through histomorphology, immunohistochemistry, drug response evaluation, assessment of cell invasion, and biomarker detection reveals the heterogeneity among individual STOs. Specifically, the smaller STOs exhibited higher invasion and drug resistance capabilities compared with the larger ones. The developed microfluidic approach effectively facilitates STO formation and offers promising prospects for investigating tumor heterogeneity, as well as conducting personalized therapy-focused drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377864498ce498325a0e411b073919607dfc91f3" target='_blank'>
              Forming Single-Cell-Derived Colon Cancer Organoid Arrays on a Microfluidic Chip for High Throughput Tumor Heterogeneity Analysis.
              </a>
            </td>
          <td>
            Zihe Chen, Jueming Chen, Dongguo Lin, H. Kang, Yanzhang Luo, Xiaogang Wang, Lihui Wang, Dayu Liu
          </td>
          <td>2024-08-01</td>
          <td>ACS biomaterials science & engineering</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Early‐stage lung cancer is now more commonly identified in the form of ground‐glass nodules (GGNs). Presently, the treatment of lung cancer with GGNs mainly depends on surgery; however, issues still exist such as overtreatment and delayed treatment due to the nonuniform standard of follow‐up. Therefore, the discovery of a noninvasive treatment could expand the treatment repertoire of ground‐glass nodular lung cancer and benefit the prognosis of patients. Immunotherapy has recently emerged as a new promising approach in the field of lung cancer treatment. Thus, this study presents a comprehensive review of the immune microenvironment of lung cancer with GGNs and describes the functions and characteristics of various immune cells involved, aiming to provide guidance for the clinical identification of novel immunotherapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6576e613fd97c27c8c8dff51aeceb165db6a254d" target='_blank'>
              The immune microenvironment of lung adenocarcinoma featured with ground‐glass nodules
              </a>
            </td>
          <td>
            Changtai Zhao, R. Xiao, Hongming Jin, Xiao Li
          </td>
          <td>2024-06-24</td>
          <td>Thoracic Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is a highly aggressive tumor with unmet therapeutic needs, which can be explained by extensive intra-tumoral heterogeneity and plasticity. In this study, we aimed to investigate the specific metabolic features of Glioblastoma stem cells (GSC), a rare tumor subpopulation involved in tumor growth and therapy resistance.


METHODS
We conducted comprehensive analyses of primary patient-derived GBM cultures and GSC-enriched cultures of human GBM cell lines using state-of-the-art molecular, metabolic and phenotypic studies.


RESULTS
We showed that GSC-enriched cultures display distinct glycolytic profiles compared with differentiated tumor cells. Further analysis revealed that GSC relies on pyruvate carboxylase activity for survival and self-renewal capacity. Interestingly, inhibition of pyruvate carboxylase led to GSC death, particularly when the glutamine pool was low, and increased differentiation. Finally, while GSC displayed resistance to the chemotherapy drug etoposide, genetic or pharmacological inhibition of pyruvate carboxylase restored etoposide sensitivity in GSC, both in vitro and in orthotopic murine models.


CONCLUSION
Our findings demonstrate the critical role of pyruvate carboxylase in GSC metabolism, survival and escape to etoposide. They also highlight pyruvate carboxylase as a therapeutic target to overcome therapy resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd87fc69cc39f932ec1b38c0feef0134502734fd" target='_blank'>
              Metabolic Profiling of Glioblastoma Stem Cells Reveals Pyruvate Carboxylase as a Critical Survival Factor and Potential Therapeutic Target.
              </a>
            </td>
          <td>
            Ophélie Renoult, Mélanie Laurent-Blond, Hala Awada, Lisa Oliver, Noémie Joalland, Mikaël Croyal, François Paris, Catherine Gratas, C. Pecqueur
          </td>
          <td>2024-06-13</td>
          <td>Neuro-oncology</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Understanding the sepsis-induced immunological response can be facilitated by identifying phenotypic changes in immune cells at the single-cell level. Mass cytometry, a novel multiparametric single-cell analysis technique, offers considerable benefits in characterizing sepsis-induced phenotypic changes in peripheral blood mononuclear cells. Here, we analyzed peripheral blood mononuclear cells from 20 sepsis patients and 10 healthy donors using mass cytometry and employing 23 markers. Both manual gating and automated clustering approaches (PhenoGraph) were used for cell identification, complemented by uniform manifold approximation and projection (UMAP) for dimensionality reduction and visualization. Our study revealed that patients with sepsis exhibited a unique immune cell profile, marked by an increased presence of monocytes, B cells, and dendritic cells, alongside a reduction in natural killer (NK) cells and CD4/CD8 T cells. Notably, significant changes in the distributions of monocytes and B and CD4 T cells were observed. Clustering with PhenoGraph unveiled the subsets of each cell type and identified elevated CCR6 expression in sepsis patients' monocyte subset (PG#5), while further PhenoGraph clustering on manually gated T and B cells discovered sepsis-specific CD4 T cell subsets (CCR4low CD20low CD38low) and B cell subsets (HLA-DRlow CCR7low CCR6high), which could potentially serve as novel diagnostic markers for sepsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd72104c86bb90496756170307b21e6c5d5b9c2" target='_blank'>
              Phenotypic Landscape of Immune Cells in Sepsis: Insights from High-Dimensional Mass Cytometry.
              </a>
            </td>
          <td>
            Sehee Park, Haribalan Perumalsamy, Zayakhuu Gerelkhuu, Sneha Sunderraj, Yangsoon Lee, T. Yoon
          </td>
          <td>2024-06-08</td>
          <td>ACS infectious diseases</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis.
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cbb9745e3c5fd2457e27d5687d3c69baea8aa31" target='_blank'>
              Engineered matrices reveal stiffness-mediated chemoresistance in patient-derived pancreatic cancer organoids.
              </a>
            </td>
          <td>
            Bauer L. LeSavage, Daiyao Zhang, Carla Huerta-López, A. Gilchrist, B. Krajina, Kasper Karlsson, Amber R Smith, Kremena Karagyozova, Katarina C Klett, Michelle S Huang, Christopher Long, G. Kaber, C. Madl, P. Bollyky, Christina Curtis, Calvin J Kuo, S. Heilshorn
          </td>
          <td>2024-07-04</td>
          <td>Nature materials</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131fbeb07da63c8da0152ac32458fba6ab9aeb1b" target='_blank'>
              Chip collection of hepatocellular carcinoma based on O2 heterogeneity from patient tissue
              </a>
            </td>
          <td>
            Sewoom Baek, H. Ha, Jeong Su Park, Min Jeong Cho, Hye-Seon Kim, Seung Eun Yu, Seyong Chung, Chansik Kim, Jueun Kim, Ji Youn Lee, Yerin Lee, Hyunjae Kim, Yujin Nam, Sungwoo Cho, Kyubae Lee, Ja Kyung Yoon, Jin Sub Choi, Dai Hoon Han, Hak-Joon Sung
          </td>
          <td>2024-06-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a rare disease associated with a poor prognosis, primarily due to early recurrence and metastasis. An important feature of this condition is microvascular invasion (MVI). However, current predictive models based on imaging have limited efficacy in this regard. This study employed a random forest model to construct a predictive model for MVI identification and uncover its biological basis. Single-cell transcriptome sequencing, whole exome sequencing, and proteome sequencing were performed. The area under the curve of the prediction model in the validation set was 0.93. Further analysis indicated that MVI-associated tumor cells exhibited functional changes related to epithelial-mesenchymal transition and lipid metabolism due to alterations in the NF-kappa B and MAPK signaling pathways. Tumor cells were also differentially enriched for the IL-17 signaling pathway. There was less infiltration of SLC30A1+ CD8+ T cells expressing cytotoxic genes in MVI-associated ICC, whereas there was more infiltration of myeloid cells with attenuated expression of the MHC II pathway. Additionally, MVI-associated intercellular communication was closely related to the SPP1-CD44 and ANXA1-FPR1 pathways. These findings resulted in a brilliant predictive model and fresh insights into MVI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891c8bc5aef729b930179251eb7e060720eef85" target='_blank'>
              Machine learning based on biological context facilitates the identification of microvascular invasion in intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Shuaishuai Xu, Mingyu Wan, C. Ye, Ruyin Chen, Qiong Li, Xiaochen Zhang, Jian Ruan
          </td>
          <td>2024-08-01</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Multiple therapies with immune mechanisms of action such as CAR T-cells and bispecific antibodies are now approved for relapsed/refractory large B-cell lymphoma (rrLBCL). The efficacy of these therapies is likely influenced by lymphoma microenvironment (LME) characteristics, but these have not been directly characterized in a comprehensive fashion. We therefore performed single-nucleus multiome (RNA + ATAC), bulk RNA, and WES of 119 biopsies from 114 patients to capture both hematopoietic and non-hematopoietic cell types. After stringent quality control, 970,239 cells remained. Non-B-cell lineages were classified into 76 transcriptionally-distinct cell types using unsupervised clustering. LME archetypes were defined by non-negative matrix factorization (NMF) of non-B cell types. Associations with CAR T-cell outcomes were validated using published Nanostring data from the ZUMA7 clinical trial. NMF clustering identified 5 cellular modules: lymph-node 1 [LN1] and lymph-node 2 [LN2] that are characterized by lymph-node structural cell types, antigen presenting cells and naïve and memory T-cells; T-effector/exhausted [TEX] characterized by high frequencies of effector and exhausted CD8 T-cells; and fibroblast/macrophage 1 [FMAC1] and 2 [FMAC2] that were characterized by high frequencies of macrophage subsets and fibroblast subsets including cancer associated fibroblasts (CAFs). The pairs of LN1 and LN2 modules, and FMAC1 and FMAC2 modules, were correlated and therefore considered collectively in tumor archetype construction, resulting in three major archetypes: LN (33% of tumors), TEX (25% of tumors) and FMAC (42% of tumors). Among these biopsies, 17 were obtained immediately prior to, and 13 were subsequent to CD19 CAR T-cell therapy. Although numbers were limited, these cases suggested that the LN archetype was associated with greater benefit from CAR T cell therapy. We therefore leveraged our bulk RNA-sequencing data plus published Nanostring data from the ZUMA7 study of axicabtagene ciloleucel (axi-cel) in second line rrLBCL to develop a Naïve Bayes classifier for these archetypes. Evaluation of response data from ZUMA7 showed that the greatest benefit for axi-cel compared to chemotherapy was observed within the LN subtype (HR=0.18; P<0.0001), compared to the FMAC (HR=0.4; P<0.0001) and TEX (HR=0.71; P=0.23), with median PFS in the axi-cel arm of 30.16, 4.94, and 4.53 months respectively. Furthermore, LN subtype patients had significantly better outcomes compared to FMAC and TEX subtype patients in the axi-cel arm (HR=0.5, P=0.006) but not in the chemotherapy arm (HR=1.3; P=0.23). Therefore, accurate construction of the rrLBCL LME using direct cell measurements of both hematopoietic and non-hematopoietic cells with single-cell genomics permits the identification of cell types, cell modules, and LME archetypes with important implications for LBCL biology, and may present an opportunity for LME-guided selection of patients most likely to benefit from cellular therapy.
 Citation Format: Xubin Li, Kartik Singhal, Qing Deng, Dai Chihara, David Russler-Germain, Usama Hussein, Jennifer A. Foltz, Jared Henderson, Ashley Wilson, Evelyn Schmidt, Imran Nizamuddin, Tommy Dinh, Ryan Sun, Francisco Vega, Christopher Flowers, Obi Griffith, Todd Fehniger, Malachi Griffith, Michael R. Green. Comprehensive characterization of the non-tumor microenvironment of relapsed/refractory large B-cell lymphoma identifies patients with greatest benefit from CD19 CAR T-cell therapy [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731635a6f268d61f8670c7f3dbebaf34d1a0dda8" target='_blank'>
              Abstract PR06: Comprehensive characterization of the non-tumor microenvironment of relapsed/refractory large B-cell lymphoma identifies patients with greatest benefit from CD19 CAR T-cell therapy
              </a>
            </td>
          <td>
            Xubin Li, Kartik Singhal, Q. Deng, Dai Chihara, David A. Russler-Germain, Usama Hussein, J. Foltz, J. Henderson, Ashley Wilson, Evelyn Schmidt, Imran Nizamuddin, Tommy Dinh, Ryan Sun, F. Vega, C. Flowers, O. Griffith, T. Fehniger, M. Griffith, M. Green
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde19b1f96c1cfcaa9c403d17be06cabeee637d7" target='_blank'>
              Pan-cell death-related signature reveals tumor immune microenvironment and optimizes personalized therapy alternations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Linzhi Han, Jingyi He, Hongxin Xie, Yan Gong, Conghua Xie
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Pediatric solid and central nervous system tumors are the primary cause of cancer-related fatalities in children. Discovering novel targeted therapies requires utilizing pediatric cancer models that accurately mirror the patient's illness. However, the creation and evaluation of these models have significantly trailed adult cancer research, emphasizing the pressing demand for pediatric-centric cell line repositories. Here, we establish a centralized collection of over 450 childhood cancer cell lines. We subjected over 250 of these cell lines to comprehensive multi-omics analyses (including DNA sequencing, RNA sequencing, and DNA methylation analysis), while concurrently conducting pharmacological screenings and genetic CRISPR-Cas9 loss-of-function assays to unveil pediatric-specific treatment avenues and biomarkers. Machine learning approaches were then applied to uncover genotype-phenotype relationships and synthetic lethal interactions. Our endeavor sheds light on the specific vulnerabilities of pathways in molecularly characterized pediatric tumor subclasses and reveals clinically relevant therapeutic opportunities linked with biomarkers. We offer access to the cell line data and resources through an open-access portal.
 Citation Format: Ron Firestein. Generation and Mining of a Pediatric-focused Cancer Cell Line Atlas to Define Druggable Genetic Interactions in Childhood Malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f357133316507cc7783a7dde6bc84dc11f25508" target='_blank'>
              Abstract IA012: Generation and Mining of a Pediatric-focused Cancer Cell Line Atlas to Define Druggable Genetic Interactions in Childhood Malignancies
              </a>
            </td>
          <td>
            Ron Firestein
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Objective Hepatocellular carcinoma (HCC) poses a significant clinical challenge because the long-term benefits of immune checkpoint blockade therapy are limited. A comprehensive understanding of the mechanisms underlying immunotherapy resistance in HCC is imperative for improving patient prognosis. Design In this study, to systematically investigate the characteristics of cancer-associated fibroblast (CAF) subsets and the dynamic communication among the tumor microenvironment (TME) components regulated by CAF subsets, we generated an HCC atlas by compiling single-cell RNA sequencing (scRNA-seq) datasets on 220 samples from six datasets. We combined spatial transcriptomics with scRNA-seq and multiplexed immunofluorescence to identify the specific CAF subsets in the TME that determine the efficacy of immunotherapy in HCC patients. Results Our findings highlight the pivotal role of POSTN+ CAFs as potent immune response barriers at specific tumor locations, as they hinder effective T-cell infiltration and decrease the efficacy of immunotherapy. Additionally, we elucidated the interplay between POSTN+ CAFs and SPP1+ macrophages, whereby the former recruits the latter and triggers increased SPP1 expression via the IL-6/STAT3 signaling pathway. Moreover, we demonstrated a spatial correlation between POSTN+ CAFs and SPP1+ macrophages, revealing an immunosuppressive microenvironment that limits the immunotherapy response. Notably, we found that patients with elevated expression levels of both POSTN+ CAFs and SPP1+ macrophages achieved less therapeutic benefit in an immunotherapy cohort. Conclusion Our research elucidates light on the role of a particular subset of CAFs in immunotherapy resistance, emphasizing the potential benefits of targeting specific CAF subpopulations to improve clinical responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321d7431417f89b80a8af6ab7f043a0cb955e621" target='_blank'>
              POSTN+ cancer-associated fibroblasts determine the efficacy of immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hao Wang, Yuan Liang, Zheng Liu, Rui Zhang, Jiashuo Chao, Mingming Wang, Mu Liu, Lei Qiao, Zhengfeng Xuan, Haitao Zhao, Ling Lu
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common childhood soft tissue sarcoma, with two subtypes: Fusion-positive RMS (FPRMS), which has the PAX3-FOXO1 fusion gene, and fusion-negative RMS (FNRMS). Despite their distinct characteristics, treatments mainly rely on conventional chemotherapies without considering these differences. This study highlights that FNRMS cells exhibit significantly heightened interaction with the extracellular matrix (ECM) compared to FPRMS cells. Using single-cell RNA sequencing of skeletal muscle tissues and RNA sequencing of RMS samples, we identified the upregulation of genes related to cell-ECM interaction and TGFβ signaling in FNRMS compared to FPRMS. We also confirmed enhanced cell-ECM interaction stimulated by TGFβ signaling in FNRMS cells, using confocal reflection microscopy to monitor dynamic cell-ECM interaction and a live-cell sensor to quantitatively assess TGFβ signaling activity. Additionally, we discovered that the PAX3-FOXO1 fusion gene, characteristic of FPRMS, stimulated nitric oxide synthesis, which suppresses TGFβ signaling and reduces cell-ECM interaction. These findings suggest that PAX3-FOXO1 determines the diminished cell-ECM interactions in FPRMS. Experimental data show higher sensitivity of FNRMS to cell-ECM interaction disruption and TGFβ inhibition. Furthermore, the diminished cell-ECM interaction in FPRMS, allowing cells to survive in the ectopic environment through circulation, may partly explain its higher metastatic potential compared to FNRMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e3669a3b3b71d3313b2a73c513a3a0add64633" target='_blank'>
              Differential cell-ECM interaction of rhabdomyosarcoma subtypes regulated by PAX3-FOXO1
              </a>
            </td>
          <td>
            Antonios Chronopoulos, Ivan Chavez, Chandra Vemula, Nikhil Mittal, Victoria Zamloot, Sangyoon J. Han, Jinseok Park
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: The intratumoral microbiome is a tumor-specific component of the tumor microenvironment (TME), including microorganisms localized inside and around cancer cells and immune cells (Nejman Science2020), which was recently added to the “Hallmarks of Cancer” (Hanahan, Cancer Discovery 2022). Specific intratumoral microbiome signatures (IMS) can be detected using our targeted sequencing platform Memorial Sloan Kettering Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT) (Prashkin Nat Comm 2023) data through the implementation of a specific bioinformatic pipeline (Vanderbilt J Mol Diagn 2022, Abate Ann Surg 2022). In lymphoma, where antigen stimulation is known to promote lymphomagenesis, studying the intratumoral microbiome might inform lymphoma biology and potentially new treatment strategies. In anaplastic large cell lymphoma (ALCL), a specific histologic phenotype can be found in a very diverse spectrum of diseases, ranging from smoldering lymphomas such as breast-implant associated (BIA-ALCL) and primary cutaneous (pcALCL) to systemic aggressive lymphomas such as ALK-positive or ALK-negative ALCL. We hypothesized that the different IMS might be present in different ALCL clinical presentations.
 Methods: We used metagenomic techniques to analyze IMS on MSK-IMPACT-HEME as previously described (Vanderbilt J Mol Diagn 2022) of patients with ALCL in our lymphoma database. All patients signed informed consent. We calculated the odds ratio (OR) of each intra-lymphoma IMS in one ALCL subtype compared to that of the same IMS in the other ALCL subtypes. We used several different analyses to process the data for commensals and contaminants. In this regard, we built a database of potentially pathogenic or commensal microorganisms integrated in the decontam (R package) pipeline dataset. For BIA-ALCL, we compared the IMS results with those of known components of explanted textured implant biofilm.
 Results: We re-analyzed the MSK-IMPACT-HEME BAM files of 58 patients with ALCL, including 12 BIA-ALCL, 11 pcALCL, and 35 systemic ALK+/- ALCL. After reviewing the data for commensals and contaminants, we could identify more than one potentially lymphoma-specific or site-specific microorganism in pcALCL (n=2) and in BIA-ALCL (n=14) (slow-growing subtypes) compared with systemic ALCL. Fewer differences were noted between pcALCL and BIA-ALCL; however, Achromobacter and Rothia were significantly enriched in BIA-ALCL compared to systemic ALCL and pcALCL (OR 14.9 p=0.001; OR=5, p=0.025) and Micromonospora in pcALCL compared to BIA-ALCL and systemic ALCL (OR=5.5, p=0.05). No specific IMS was found in systemic ALCL, as the only potentially enriched IMS (Delftiae) is likely a contaminant.
 Conclusions: This analysis of IMS using targeted sequencing data, highlighted some possible associations between intratumoral microorganisms and lymphoma-specific subtypes, especially in indolent subtypes of ALCL (BIA- and pc-) compared to systemic ALCL. Further analyses with more cases and confirmatory techniques might confirm these initial findings.
 Citation Format: Paola Ghione, Horwitz Steven, Peter Cordeiro, Maria Arcila, Natasha Galasso, Ahmet Dogan, Gilles Salles, Chad Vanderbilt. Distinct intratumoral microbiome signatures enrichment in patients with different Anaplaplastic Large Cell Lymphoma (ALCL) subtypes [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fbcde97aa026f9feaac9c71a662b8d2a98bf026" target='_blank'>
              Abstract PO-022: Distinct intratumoral microbiome signatures enrichment in patients with different Anaplaplastic Large Cell Lymphoma (ALCL) subtypes
              </a>
            </td>
          <td>
            Paola Ghione, Horwitz Steven, Peter Cordeiro, M. Arcila, N. Galasso, Ahmet Dogan, G. Salles, Chad M Vanderbilt
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease.Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion.Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group.This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a597e04eb9dbff2e9d653b677b44484ada81b85" target='_blank'>
              Tumour microenvironment programming by an RNA–RNA-binding protein complex creates a druggable vulnerability in IDH-wild-type glioblastoma
              </a>
            </td>
          <td>
            Lele Wu, Zheng Zhao, Yong Jae Shin, Yiyun Yin, Anandhkumar Raju, T. Vaiyapuri, Khaireen Idzham, Miseol Son, Yeri Lee, Jason K. Sa, J. Chua, Bilal Unal, Y. Zhai, Wenhua Fan, Lijie Huang, Huimin Hu, J. Gunaratne, Do-Hyun Nam, T. Jiang, Vinay Tergaonkar
          </td>
          <td>2024-06-10</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Medulloblastoma (MB) stands as one of the prevalent malignant brain tumors among pediatric patients. Despite its prevalence, the intricate interplay between the regulatory program driving malignancy in MB cells and their interactions with the microenvironment remains insufficiently understood. Leveraging the capabilities of single-cell Assay for Transposase-Accessible Chromatin sequencing (scATAC-seq), the chromatin accessibility landscape is unveiled across 59,015 distinct MB cells. This expansive dataset encompasses cells belonging to discrete molecular subgroups, namely SHH, WNT, Group3, and Group4. Within these chromatin accessibility profiles, specific regulatory elements tied to individual subgroups are uncovered, shedding light on the distinct activities of transcription factors (TFs) that likely orchestrate the tumorigenesis process. Moreover, it is found that certain neurotransmitter receptors (NTRs) are subgroup-specific and can predict MB subgroup classification when combined with their associated transcription factors. Notably, targeting essential NTRs within tumors influences both the in vitro sphere-forming capability and the in vivo tumorigenic capacity of MB cells. These findings collectively provide fresh insights into comprehending the regulatory networks and cellular dynamics within MBs. Furthermore, the significance of the TF-NTR regulatory circuits is underscored as prospective biomarkers and viable therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b19ab545540365c4fc326bff03e9d828165b6" target='_blank'>
              Single-Cell Chromatin Accessibility Analysis Reveals Subgroup-Specific TF-NTR Regulatory Circuits in Medulloblastoma.
              </a>
            </td>
          <td>
            Xiaoyue Gao, Qiyuan Zhuang, Yun Li, Guochao Li, Zheng Huang, Shenzhi Chen, Shaoxing Sun, Hui Yang, Lan Jiang, Ying Mao
          </td>
          <td>2024-06-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e1b4639cfb922b131986ddda7afd728b72c924a" target='_blank'>
              Deciphering the molecular landscape: evolutionary progression from gynecomastia to aggressive male breast cancer.
              </a>
            </td>
          <td>
            Chuang Yang, Zhonglin Wang, Lijun Qian, Jingyue Fu, H. Sun
          </td>
          <td>2024-06-18</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b25961f9f4f14e47897a20148a37f2d8244f11" target='_blank'>
              Tumor cell enrichment by tissue suspension improves sensitivity to copy number variation in diffuse gastric cancer with low tumor content
              </a>
            </td>
          <td>
            K. Hatakeyama, Koji Muramatsu, T. Nagashima, Hiroyuki Ichida, Yuichi Kawanishi, Ryutaro Fukumura, K. Ohshima, Y. Shimoda, S. Ohnami, S. Ohnami, Koji Maruyama, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Takashi Sugino, Ken Yamaguchi
          </td>
          <td>2024-06-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bf390baec4f133e1159b69971ccead0d2deed57" target='_blank'>
              Establishment and characterization of the PDAC-X3 cell line: a novel Chinese-origin pancreatic ductal adenocarcinoma cell line.
              </a>
            </td>
          <td>
            Changpeng Chai, Huan-zhang Tang, Xin Miao, Yuanhui Su, Lu Li, Cheng Yu, Jian-Fen Yi, Zhenzhen Ye, L. Miao, Bo Zhang, Zhengfeng Wang, Wei Luo, Jinjing Hu, Hui Zhang, Wence Zhou, Hao Xu
          </td>
          <td>2024-07-16</td>
          <td>Human cell</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Preclinical models that replicate patient tumours as closely as possible are crucial for translational cancer research. While in vitro cancer models have many advantages in assessing tumour response therapy, in vivo systems are essential to enable evaluation of the role of the tumour cell extrinsic factors, such as the tumour microenvironment and host immune system. The requirement for a functional immune system is particularly important given the current focus on immunotherapies. Therefore, we set out to generate an immunocompetent, transplantable model of colorectal cancer suitable for in vivo assessment of immune-based therapeutic approaches. Intestinal tumours from a genetically engineered mouse model, driven by expression of a Pik3ca mutation and loss of Apc, were transplanted into wild type C57BL/6 host mice and subsequently passaged to form a novel syngeneic transplant model of colorectal cancer. Our work confirms the potential to develop a panel of mouse syngeneic grafts, akin to human PDX panels, from different genetically engineered, or carcinogen-induced, mouse models. Such panels would allow the in vivo testing of new pharmaceutical and immunotherapeutic treatment approaches across a range of tumours with a variety of genetic driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e96ac527246a723f0c584b8d2c966fbc703899" target='_blank'>
              Development of an in vivo syngeneic mouse transplant model of invasive intestinal adenocarcinoma driven by endogenous expression of Pik3caH1047R and Apc loss
              </a>
            </td>
          <td>
            Francesc de las Heras, Camilla B. Mitchell, William K. Murray, N. Clemons, Wayne A. Phillips
          </td>
          <td>2024-08-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac50133889cf2a95f1525e8512125ca2ba0a78b" target='_blank'>
              Comprehensive characterization of B7 family members in breast cancer: B7-H5 switch reverses breast cancer from “immuno-cold” into “immuno-hot” status
              </a>
            </td>
          <td>
            Jiayu Liu, Cenzhu Wang, Ying Jiang, Yunxu Zhou, Ling Chen, Zhiwen Qian, Lu Liu, Danping Wu, Yan Zhang
          </td>
          <td>2024-06-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03aef8e2ade23cd8110e6a9d02003c1b5ecd551f" target='_blank'>
              Nanoscale insights into hematology: super-resolved imaging on blood cell structure, function, and pathology
              </a>
            </td>
          <td>
            Jinghan Liu, Yuping Yolanda Tan, Wen Zheng, Yao Wang, L. Ju, Qian Peter Su
          </td>
          <td>2024-06-24</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [24],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>