<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-07 06:32:41 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa5087cd6813583e8724b35bca919eb32b35df3e" target='_blank'>
              Automated spatial omics landscape analysis approach reveals novel tissue architectures in ulcerative colitis
              </a>
            </td>
          <td>
            Derek R Holman, Samuel J S Rubin, Mariusz Ferenc, Elizabeth A Holman, Alexander N Koron, Robel Daniel, Brigid Boland, Garry P Nolan, John T Chang, Stephan Rogalla
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, S. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. We generated a spatial multi-omics atlas encompassing 26 PDAC tumors from patients treated with combination immunotherapies. Using machine learning-enabled H&E image classification models and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within TLS niches spanning across distinct morphologies and immunotherapies. Unsupervised learning generated a TLS-specific spatial gene expression signature that significantly associates with improved survival in PDAC patients. These analyses demonstrate TLS-associated intratumoral B cell maturation in pathological responders, confirmed with spatial proteomics and BCR profiling. Our study also identifies spatial features of pathologic immune responses, revealing TLS maturation colocalizing with IgG/IgA distribution and extracellular matrix remodeling. GRAPHICAL ABSTRACT HIGHLIGHTS Integrated multi-modal spatial profiling of human PDAC tumors from neoadjuvant immunotherapy clinical trials reveal diverse spatial niches enriched in TLS. TLS maturity is influenced by tumor location and the cellular neighborhoods in which TLS immune cells are recruited. Unsupervised machine learning of genome-wide signatures on spatial transcriptomics data characterizes the TLS-enriched TME and associates TLS transcriptomes with survival outcomes in PDAC. Interactions of spatially variable gene expression patterns showed TLS maturation is coupled with immunoglobulin distribution and ECM remodeling in pathologic responders. Intratumoral plasma cell and immunoglobin gene expression spatial dynamics demonstrate trafficking of TLS-driven humoral immunity in the PDAC TME. Significance We report a spatial multi-omics atlas of PDAC tumors from a series of immunotherapy neoadjuvant clinical trials. Intratumorally, pathologic responders exhibit mature TLS that propagate plasma cells into malignant niches. Our findings offer insights on the role of TLS-associated humoral immunity and stromal remodeling during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b4043873d18d3b758a620d9a4f90e32bea7034" target='_blank'>
              Spatial multi-omics reveal intratumoral humoral immunity niches associated with tertiary lymphoid structures in pancreatic cancer immunotherapy pathologic responders
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, James Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, Elizabeth M. Jaffee, Lei Zheng, W. Ho, Robert A Anders, E. Fertig, Lusciane T. Kagohara
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) resides in an immune rich microenvironment, yet, immune-based therapies have faltered in eliciting durable responses. Bridging this paradox requires a comprehensive understanding of leukemic interactions within the bone marrow microenvironment. We optimized a high-throughput tissue-microarray based pipeline for high-plex spatial immunofluorescence and mass cytometry imaging on a single slide, capturing immune, tumor, and structural components. Using unbiased clustering on the spatial K function, we unveiled the presence of tertiary lymphoid-like aggregates in bone marrow which we validated using spatial transcriptomics and an independent proteomics approach. We then found validated TLS signatures predictive of outcomes in AML using an integrated public 480 patient transcriptomic dataset. By harnessing high-plex spatial proteomics, we open the possibility of discovering of novel structures and interactions that underpin leukemic immune response. Further, our study’s methodologies and resources can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1b1bad3b36f52c6886c710f3a8b1e66619ef44" target='_blank'>
              Multimodal Spatial Proteomic Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, Sammy Ferri-Borgongo, Akshay Basi, Javier A. Gomez, Jessica L Root, P. Reville, Padmanee Sharma, Sreyashi Basu, Andres E. Quesada, Carlos Bueso-Ramos, T. Manshouri, Miriam Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Predicting the progression of solid cancers based solely on genetics is challenging due to the influence of the tumor microenvironment (TME). For colorectal cancer (CRC), tumors deficient in mismatch repair (dMMR) are more immune infiltrated than mismatch repair proficient (pMMR) tumors and have better prognosis following resection. Here we quantify features of the CRC TME by combining spatial profiling with genetic analysis and release our findings via a spatially enhanced version of cBioPortal that facilitates multi-modal data exploration and analysis. We find that ∼20% of pMMR tumors exhibit similar levels of T cell infiltration as dMMR tumors and that this is associated with better survival but not any specific somatic mutation. These T cell-infiltrated pMMR (tipMMR) tumors contain abundant cells expressing PD1 and PDL1 as well as T regulatory cells, consistent with a suppressed immune response. Thus, like dMMR CRC, tipMMR CRC may benefit from immune checkpoint inhibitor therapy. SIGNIFICANCE pMMR tumors with high T cell infiltration and active immunosuppression are identifiable with a mid-plex imaging assay whose clinical deployment might double the number of treatment-naïve CRCs eligible for ICIs. Moreover, the low tumor mutational burden in tipMMR CRC shows that MMR status is not the only factor promoting immune infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb91d144afd07ee13d239cb16bf5689a34f70783" target='_blank'>
              Integrating spatial profiles and cancer genomics to identify immune-infiltrated mismatch repair proficient colorectal cancers
              </a>
            </td>
          <td>
            Jeremiah Wala, Ino de Bruijn, S. Coy, Andréanne Gagné, Sabrina Chan, Yu-An Chen, J. Hoffer, J. Muhlich, N. Schultz, Sandro Santagata, P. Sorger
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The relative abundance of cancer-associated fibroblast (CAF) subtypes influences a tumor’s response to treatment, especially immunotherapy. However, the extent to which the underlying tumor composition associates with CAF subtype-specific gene expression is unclear. Here, we describe an interpretable machine learning (ML) approach, additive multiple instance learning (aMIL), to predict bulk gene expression signatures from H&E-stained whole slide images (WSI), focusing on an immunosuppressive LRRC15+ CAF-enriched TGFβ-CAF signature. aMIL models accurately predicted TGFβ-CAF across various cancer types. Tissue regions contributing most highly to slide-level predictions of TGFβ-CAF were evaluated by ML models characterizing spatial distributions of diverse cell and tissue types, stromal subtypes, and nuclear morphology. In breast cancer, regions contributing most to TGFβ-CAF-high predictions (“excitatory”) were localized to cancer stroma with high fibroblast density and mature collagen fibers. Regions contributing most to TGFβ-CAF-low predictions (“inhibitory”) were localized to cancer epithelium and densely inflamed stroma. Fibroblast and lymphocyte nuclear morphology also differed between excitatory and inhibitory regions. Thus, aMIL enables a data-driven link between tissue phenotype and transcription, offering biological interpretability beyond typical black-box models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dfe1e5aad3aff07bf10f8ea7b2dcf18d3178c7" target='_blank'>
              Spatial mapping of immunosuppressive CAF gene signatures in H&E-stained images using additive multiple instance learning
              </a>
            </td>
          <td>
            Miles Markey, Juhyun Kim, Zvi Goldstein, Y. Gerardin, Jacqueline Brosnan-Cashman, Syed Ashar Javed, Dinkar Juyal, Harshith Pagidela, Limin Yu, Bahar Rahsepar, J. Abel, S. Hennek, Archit Khosla, A. Taylor-Weiner, Chintan Parmar
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK) rearrangement is a major oncogenic driver in non-small cell lung cancer (NSCLC). While ALK tyrosine kinase inhibitors have shown promising therapeutic effects, overcoming resistance with immunotherapy becomes necessary when resistance develops. However, various clinical trials have revealed that their efficacies remain limited. To investigate the tumor microenvironment (TME) factors contributing to poor immune checkpoint blockade responses in ALK-positive patients, we performed single-cell RNA and ATAC sequencing on lung adenocarcinoma (LUAD) tumors with and without ALK rearrangements. Integrative analysis with additional public LUAD cohorts revealed distinct immune landscapes in ALK-positive tumors, marked by enriched innate immunity and depleted adaptive immunity. ALK-positive malignant cells exhibit higher stemness and aggressive phenotype. Tumor-associated macrophages (TAMs) in these tumors predominantly maintain pro-tumoral M2-like states, reinforcing immune suppression. B cells show reduced immune reactivity and impaired tertiary lymphoid structure formation, while CD8+ T cells display bystander-like signatures and reduced tumor reactivity. Single-cell chromatin accessibility profiles combined with regulatory network analysis suggest that differences in transcription factor activities, rather than chromatin accessibility, may underlie T cell dysfunction. These findings provide insights into the immunosuppressive TME of ALK-positive LUAD, potentially explaining the failure of recent immunotherapy trials and highlighting targets for improving efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb7277d768d3078fb9b8e8f161ef723f32126" target='_blank'>
              Single-cell multi-omics reveals tumor microenvironment factors underlying poor immunotherapy responses in ALK-positive lung cancer
              </a>
            </td>
          <td>
            S. Baek, Euijeong Sung, Gamin Kim, Min Hee Hong, Chang Young Lee, H. Shim, Seong Yong Park, Hye Ryun Kim, Insuk Lee
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, A. Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cell states are modulated by intrinsic driving forces such as gene expression noise and extrinsic signals from the tissue microenvironment. The distinction between intrinsic and extrinsic cell state determinants is essential for understanding the regulation of cell fate in tissues during development, homeostasis and disease. The rapidly growing availability of single-cell resolution spatial transcriptomics makes it possible to meet this challenge. However, available computational methods to infer topological tissue domains, spatially variable gene expression, or ligand-receptor interactions are limited in capturing cell state changes driven by crosstalk between individual cell types within the same niche. We present NiCo, a computational framework for integrating single-cell resolution spatial transcriptomics with matched single-cell RNA-sequencing reference data to infer the influence of the spatial niche on the cell state. By applying NiCo to mouse embryogenesis, adult small intestine and liver data, we demonstrate the capacity to predict novel niche interactions that govern cell state variation underlying tissue development and homeostasis. In particular, NiCo predicts a feedback mechanism between Kupffer cells and neighboring stellate cells limiting stellate cell activation in the normal liver. NiCo provides a powerful tool to elucidate tissue architecture and to identify drivers of cellular states in local niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330dd175c406437712c3044cd06a601fe89865f" target='_blank'>
              NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis
              </a>
            </td>
          <td>
            Ankit Agrawal, Stefan Thomann, Sukanya Basu, D. Grün
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying the key molecular pathways that enable metastasis by analyzing the eventual metastatic tumor is challenging because the state of the founder subclone likely changes following metastatic colonization. To address this challenge, we labeled primary mouse pancreatic ductal adenocarcinoma (PDAC) subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative in vivo metastatic potential prospectively. We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of IL-1 pathway genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC metastasis in the IL-1 pathway, including the NF-κB targets Fos and Il23a, and beyond the IL-1 pathway including Myo1b and Tmem40. We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies. This work establishes a strategy for linking cancer cell state to future behavior, reveals novel functional regulators of PDAC metastasis, and establishes a method for scoring human carcinomas based on metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130863ca923658f30d397c218a109254f4ad9093" target='_blank'>
              Identifying a gene signature of metastatic potential by linking pre-metastatic state to ultimate metastatic fate
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Weixiang Fang, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Faithful genetically engineered in vivo models of medulloblastoma (MB) are currently available only for some molecular subgroups, in keeping with recent studies showing the unique role of human-specific progenitors in the development of G3 and G4 MB subgroups. We generated human cerebellar organoids (CbO) from expanded potential stem cells (EPSC) and characterised in depth their epigenetic and transcriptomic profile. We show the presence of sub compartment-specific cerebellar lineages linked to MB formation, including populations expressing signature genes of G3 and G4 MB cells-of-origin in a 1 month-old CbO. Moreover, we demonstrate that CbO sustains MB cells proliferation and invasion in a 3D co-culture model (CbO-MB) while preserving their molecular identity, and that treatment of CbO-MB with anti-tumour compounds recapitulate the efficacy of in vivo drug testing in PDX models. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418ddee9c7f8cb6b4c5b80d2a93be6711b4eb2cf" target='_blank'>
              Modelling medulloblastoma pathogenesis and treatment in human cerebellar organoids
              </a>
            </td>
          <td>
            Thomas Willott, James G Nicholson, Yunchen Xiao, O. Ogunbiyi, Talisa Mistry, N. Zabet, A. Merve, S. Badodi, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Multi-modal spatial omics data are invaluable for exploring complex cellular behaviors in diseases from both morphological and molecular perspectives. Current analytical methods primarily focus on clustering and classification, and do not adequately examine the relationship between cell morphology and molecular dynamics. Here, we present MorphLink, a framework designed to systematically identify disease-related morphological-molecular interplays. MorphLink has been evaluated across a wide array of datasets, showcasing its effectiveness in extracting and linking interpretable morphological features with various molecular measurements in multi-modal spatial omics analyses. These linkages provide a transparent depiction of cellular behaviors that drive transcriptomic heterogeneity and immune diversity across different regions within diseased tissues, such as cancer. Additionally, MorphLink is scalable and robust against cross-sample batch effects, making it an efficient method for integrative spatial omics data analysis across samples, cohorts, and modalities, and enhancing the interpretation of results for large-scale studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/805521c4862696d06b9751f2a5958b90bb59ce21" target='_blank'>
              MorphLink: Bridging Cell Morphological Behaviors and Molecular Dynamics in Multi-modal Spatial Omics
              </a>
            </td>
          <td>
            Jing Huang, Chenyang Yuan, Jiahui Jiang, Jianfeng Chen, Sunil S. Badve, Y. Gokmen-Polar, R. Segura, Xinmiao Yan, Alexander Lazar, Jianjun Gao, Michael P. Epstein, Linghua Wang, Jian Hu
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMN) is one of the known precancerous lesions. Patients’ prognosis is aggravated as IPMN transforms into invasive Pancreatic Ductal Adenocarcinoma (PDAC). The molecular mechanisms underlying this progression lack effective experimental models and urgently need to be elaborated. We performed spatial transcriptomics (ST) on fresh tissue samples from the same patient including normal pancreas, high-grade IPMN, and invasive PDAC, and described the step-by-step development of transcriptional landscape including clone evolution and adjacent TME feature variation. Our findings identified the master transcript factors and critical signaling pathways promoting IPMN progression to invasive PDAC. Additionally, both IPMN and PDAC harbored the ELF3, MYC, and KLF4 amplification. The Spatial CNV profile demonstrated significant heterogeneity among PDAC in their spatial distribution compared to IPMN, with seven distinct subclones showing diverse functions, such as hypoxia, oxidative phosphorylation, and epithelial-mesenchymal Transition. We observed a marked shift in the immune landscape, with the depletion of CD4+ T-cells and dendritic cells and an increase in immune-suppressive M2 macrophages in invasive PDAC, indicating a transition to an immune-evasive microenvironment. Additionally, cancer-associated fibroblasts (CAFs), particularly myofibroblastic CAFs, were enriched adjacent to invasive PDAC, suggesting their active role in tumor progression. By leveraging spatial transcriptomic analysis, our study provides comprehensive insights into the intricate molecular landscape that underlies the progression of IPMNs to invasive PDAC. These findings not only enhance our understanding of this complex process but also offer valuable knowledge for early diagnosis and intervention. Highlights Spatial CNV analysis reveals clonal evolution and distinct subclones in PDAC. Key drivers like ELF3, MYC, and KLF4 are amplified in both IPMN and invasive PDAC. Immune landscape shifts from pro-inflammatory in IPMN to immune-evasive in PDAC. Enrichment of myofibroblastic CAFs suggests their role in tumor progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4caecc6532101fd5cd79dde472c73a99ffa9c7b0" target='_blank'>
              Evolution of the Spatial transcriptomic landscape during the progression of high-grade pancreatic intraductal papillary mucinous neoplasms to invasive cancer
              </a>
            </td>
          <td>
            Sirui Peng, Qiangxing Chen, Zixin Chen, Mengling Yao, Yu Cai, Du He, Yu Cai, K. Cheng, Jun Li, H. Cai, P. Gao, Xiafei Gu, Xin Wang, Yongbin Li, Man Zhang, L. Meng, Qi Xia, Panpan Xu, Jin Zhou, Zhong Wu, Bing Peng
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="State-of-the-art spatial proteomic and transcriptomic technologies can deeply pheno-type cells in their native tissue environment, providing a high throughput means to effectively quantify spatial relationships between diverse cell populations. However, the experimental design choice of which regions of a tissue will be imaged can greatly impact the interpretation of spatial quantifications. That is, spatial relationships identified in one region of interest may not be interpreted consistently across other regions. To address this challenge, we introduce Kontextual, a method which considers alternative frames of reference for contextualising spatial relationships. These contexts may represent landmarks, spatial domains, or groups of functionally similar cells which are consistent across regions. By modelling spatial relationships between cells relative to these contexts, Kontextual produces robust spatial quantifications that are not confounded by the region selected. We demonstrate in spatial proteomics and spatial transcriptomics datasets that modelling spatial relationships this way is biologically meaningful. We also demonstrate how this approach can be used in a classification setting to improve prediction of patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b328952e77a8a207acaab565457d22fab546aad" target='_blank'>
              Kontextual: Reframing analysis of spatial omics data reveals consistent cell relationships across images
              </a>
            </td>
          <td>
            Farhan Ameen, Nicholas Robertson, David M. Lin, S. Ghazanfar, E. Patrick
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The study presents a novel approach for quantifying cellular interactions in digital pathology using deep learning-based image cytometry. Traditional methods struggle with the diversity and heterogeneity of cells within tissues. To address this, we introduce the Spatial Interaction Potential (SIP) and the Co-Localization Index (CLI), leveraging deep learning classification probabilities. SIP assesses the potential for cell-to-cell interactions, similar to an electric field, while CLI incorporates distances between cells, accounting for dynamic cell movements. Our approach enhances traditional methods, providing a more sophisticated analysis of cellular interactions. We validate SIP and CLI through simulations and apply them to colorectal cancer specimens, demonstrating strong correlations with actual biological data. This innovative method offers significant improvements in understanding cellular interactions and has potential applications in various fields of digital pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949ba712b098fead7bc4bf83be6ad4882df65fb2" target='_blank'>
              Novel Methods for Analyzing Cellular Interactions in Deep Learning-Based Image Cytometry: Spatial Interaction Potential and Co-Localization Index
              </a>
            </td>
          <td>
            Toru Nagasaka, Kimihiro Yamashita, Mitsugu Fujita
          </td>
          <td>2024-08-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb73fa1be8db27e4aafae1a4087794eca77af2e" target='_blank'>
              Spatial interactions modulate tumor growth and immune infiltration
              </a>
            </td>
          <td>
            S. Marzban, Sonal Srivastava, Sharon Kartika, Rafael Bravo, Rachel Safriel, Aidan Zarski, Alexander R. A. Anderson, Christine H Chung, Antonio L. Amelio, Jeffrey West
          </td>
          <td>2024-09-30</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma is the most hypovascular solid tumor, imposing harsh constraints on tumor metabolism. Cancer cells adapt to reduced access to vascular-derived nutrients and O2 by cooperating with abundant stromal cells. Specifically, critical nutrients including alanine, proline, glycosaminoglycans, branched chain ketoacids, lysophosphatidic acid, and unsaturated fatty acids can be provided by fibroblasts in vitro and in vivo. However, the extent of metabolic support provided by other cell types in the PDAC TME is relatively unexplored. We hypothesized that cell types that are cooperating metabolically with cancer cells will be spatially positioned near cancer cells in these tumors. We performed Xenium spatial transcriptomics on 8 treatment-naive PDAC resections and generated a spatial atlas of nearly 700,000 cells across a range of expected cell types. We noted striking neutrophil aggregates, nucleating in the glands of classical-like malignant cells and elaborating extraglandularly. Cellular neighborhood analysis demonstrates neutrophil aggregation is strongly correlated with malignant cell proliferation and low endothelial content, suggesting neutrophils may be an important source of pro-tumoral, non-vascular nutrients in the PDAC TME. Reanalysis of published scRNAseq data revealed 3 neutrophil metabolic states in human PDAC tumors. Metabolic cluster 7 neutrophils were found only in PDAC patient samples but not healthy controls and were enriched for Type I interferon response as well as pentose phosphate pathway and iron uptake. Metabolic cluster 2 neutrophils were strikingly depleted from PDAC tumor samples compared to blood, suggesting decreased infiltration or increased cell death. Cluster 2 neutrophils were enriched for several catabolic pathways including glycogenolysis, autophagy, and proteasome, and selectively express the NETosis gene PADI4. Multiplexed immunofluorescence demonstrated that neutrophil aggregates in PDAC are citH3+/MPO+ large neutrophil extracellular traps (NETs). We reasoned pancreatic cancer cells could proliferate under tumor-like metabolic stress by repurposing amino acids liberated by autophagy, glucose liberated by glycogenolysis, or simply consuming bulk neutrophil dead cell debris via macropinocytosis. Using tumor interstitial fluid media (TIFM) and 0.5% O2 to mimic tumor-like metabolic stress, we cocultured pancreatic cancer cells with human primary neutrophils and HL60-derived neutrophil-like cells. However, under tumor-like metabolic conditions, coculture with human primary neutrophils failed to substantially increase cancer cell proliferation and NETotic neutrophils were cytotoxic. However, we noticed enhanced neutrophil viability in tumor-like metabolic conditions, suggesting metabolic regulation of neutrophil cell death. Together, these data shed light on the metabolic phenotypes and interactions with cancer cells of neutrophils in the PDAC tumor microenvironment.
 Citation Format: Carson D Poltorack, Sydney M Shaffer, M Celeste Simon. Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A065.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97afc3e27a38cdf1cbacb0a0b2dbe91b12facfe" target='_blank'>
              Abstract A065: Spatial and single cell transcriptomics uncovers metabolic reprogramming and NETosis of neutrophils in human pancreatic cancer
              </a>
            </td>
          <td>
            Carson D. Poltorack, Sydney M Shaffer, M. C. Simon
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Castleman disease (CD) is inflammatory lymphoproliferative disorder of unclear etiology. To determine the cellular and molecular basis of CD, we analyzed the spatial proteome of 4,485,009 single cells, transcriptome of 50,117 single nuclei, immune repertoire of 8187 single nuclei, and pathogenic mutations in Unicentric CD, idiopathic Multicentric CD, HHV8-associated MCD, and reactive lymph nodes. CD was characterized by increased non-lymphoid and stromal cells that formed unique microenvironments where they interacted with lymphoid cells. Interaction of activated follicular dendritic cell (FDC) cytoplasmic meshworks with mantle zone B cells was associated with B cell activation and differentiation. VEGF, IL-6, MAPK, and extracellular matrix pathways were elevated in stromal cells of CD. CXCL13+ FDCs, PDGFRA+ T-zone reticular cells (TRC), and ACTA2-positive perivascular reticular cells (PRC) were identified as the predominant source of increased VEGF expression and IL-6 signaling in CD. VEGF expression by FDCs was associated with peri-follicular neovascularization. FDC, TRC and PRC of CD activated JAK-STAT, TGFβ, and MAPK pathways via ligand-receptor interactions involving collagen, integrins, complement components, and VEGF receptors. T, B and plasma cells were polyclonal but showed class-switched and somatically hypermutated IgG1+ plasma cells consistent with stromal cell-driven germinal center activation. In conclusion, our findings show that stromal cell activation and associated B-cell activation and differentiation, neovascularization and stromal remodeling underlie CD and suggest new targets for treatment. Key points Castleman Disease is characterized by activation and proliferation of CXCL13+ FDCs, PDGFRA+ reticular cells, and ACTA2-positive PRCs. VEGF and IL-6 from lymph node stromal cells are associated with B-cell activation and differentiation, endothelial proliferation, and inflammation in CD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6effec836eaf8e7b0862b8cace8add3a480a246e" target='_blank'>
              Spatial and Single Cell Mapping of Castleman Disease Reveals Key Stromal Cell Types and Cytokine Pathways
              </a>
            </td>
          <td>
            David Smith, Anna Eichinger, Andrew Rech, Julia Wang, E. Esteva, Arta Seyedian, Xiaoxu Yang, Mei Zhang, Dan Martinez, Kai Tan, Minjie Luo, Christopher Y Park, Boris Reizis, Vinodh Pillai
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor heterogeneity impedes the development of biomarker assays suitable for early disease detection that would improve patient outcomes. The CA19-9 glycan is currently used as a standalone biomarker for PDAC. Furthermore, previous studies have shown that cancer cells may display aberrant membrane-associated glycans. We therefore hypothesized that PDAC cancer cell subpopulations could be distinguished by aberrant glycan signatures. We used multiplexed glycan immunofluorescence combined with pathologist annotation and automated image processing to distinguish between PDAC cancer cell subpopulations within tumor tissue. Using a training-set/test-set approach, we found that PDAC cancer cells may be identified by signatures comprising 4 aberrant glycans (VVL, CA19-9, sTRA, and GM2) and that there are three glycan-defined PDAC tumor types: sTRA type, CA19-9 type, and intermixed. To determine whether the aberrant glycan signatures could be detected in blood samples, we developed hybrid glycan sandwich assays for membrane-associated glycans. In both patient-matched tumor and blood samples, the proportion of aberrant glycans detected was consistent. Furthermore, our multiplexed glycan immunofluorescent approach proved to be more sensitive and more specific than CA19-9 alone. Our results provide proof of concept for a novel methodology to improve early PDAC detection and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6b226460c28415c919ed9fc15c5b9945dc42aa" target='_blank'>
              Multiplexed Glycan Immunofluorescence Identification of Pancreatic Cancer Cell Subpopulations in Both Tumor and Blood Samples
              </a>
            </td>
          <td>
            Braelyn Binkowski, Zachary L. Klamer, Chongfeng Gao, Ben Staal, Anna Repesh, Hoang-Le Tran, David M. Brass, Pamela Bartlett, Stephen Gallinger, Maria Blomqvist, J. B. Morrow, Peter Allen, Chanjuan Shi, A. Singhi, Randall Brand, Ying Huang, G. Hostetter, Brian Haab
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Trogocytosis is an underappreciated phenomenon that shapes the immune microenvironment surrounding many types of solid tumors. The consequences of membrane-bound proteins being deposited from a donor immune cell to a recipient cancer cell via trogocytosis are still unclear. Here, we report that human clear cell renal carcinoma tumors stably express the lymphoid markers CD45, CD56, CD14, and CD16. Flow cytometry performed on fresh kidney tumors revealed consistent CD45 expression on tumor cells, as well as varying levels of the other markers mentioned previously. These results were consistent with our immunofluorescent analysis, which also revealed colocalization of lymphoid markers with carbonic anhydrase 9 (CAIX), a standard kidney tumor marker. RNA analysis showed a significant upregulation of genes typically associated with immune cells in tumor cells following trogocytosis. Finally, we show evidence of chromosomal DNA being transferred from immune cells to tumor cells during trogocytosis. This horizontal gene transfer has transcriptional consequences in the recipient tumor cell, resulting in a fusion phenotype that expressed both immune and cancer specific proteins. This work demonstrates a novel mechanism by which tumor cell protein expression is altered through the acquisition of surface membrane fragments and genomic DNA from infiltrating lymphocytes. These results alter the way in which we understand tumor-immune cell interactions and may reveal new insights into the mechanisms by which tumors develop. Additionally, further studies into trogocytosis will help push the field towards the next generation of immunotherapies and biomarkers for treating renal cell carcinoma and other types of cancers. SIGNIFICANCE STATEMENT We have identified trogocytosis as a mechanism by which human clear cell renal carcinoma tumors acquire lymphocyte surface protein expression from tumor infiltrating immune cells. In addition to the transfer of membrane fragments, we have provided evidence to show that genomic DNA is transferred from a normal immune cell to a tumor cell during trogocytosis. This process alters the transcriptome of cancer cells such that they express significantly more mRNA for immune proteins such as the lymphocyte marker CD45 compared to tumor cells that have not undergone trogocytosis. This study provides an in-depth analysis of the interactions between cancer cells and tumor infiltrating lymphocytes, and how these interactions alter the development of human tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d8b30a74776c1d1ac2d3f7a393276d5557d985" target='_blank'>
              Renal cancer cells acquire immune surface protein through trogocytosis and horizontal gene transfer
              </a>
            </td>
          <td>
            Haley Q. Marcarian, Anutr Sivakoses, Anika M. Arias, Olivia C. Ihedioha, Benjamin R. Lee, Maria C. Bishop, A. L. Bothwell
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76207c3c4f2b4f2f11d67f0daf7aa41a3f2aa07" target='_blank'>
              Dysfunctional tumor-infiltrating Vδ1 + T lymphocytes in microsatellite-stable colorectal cancer
              </a>
            </td>
          <td>
            V. Stary, Ram V Pandey, Julia List, L. Kleissl, Florian Deckert, J. Kabiljo, J. Laengle, Vasileios Gerakopoulos, Rudolf Oehler, Lukas Watzke, Matthias Farlik, S. Lukowski, Anne B Vogt, Georg Stary, Hannes Stockinger, M. Bergmann, Nina Pilat
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background: HCC is a highly vascular tumor, and many effective drug regimens target the tumor blood vessels. Prior bulk HCC subtyping data used bulk transcriptomes, which contained a mixture of parenchymal and stromal contributions. Methods: We utilized computational deconvolution and cell-cell interaction analyses to cell type-specific (tumor-enriched and vessel-enriched) spatial transcriptomic data collected from 41 resected HCC tissue specimens. Results: We report that the prior Hoshida bulk transcriptional subtyping schema is driven largely by an endothelial fraction, show an alternative tumor-specific schema has potential prognostic value, and use spatially paired ligand-receptor analyses to identify known and novel (LGALS9 tumor-HAVCR2 vessel) signaling relationships that drive HCC biology in a subtype-specific and potentially targetable manner. Conclusions: Our study leverages spatial gene expression profiling technologies to dissect HCC heterogeneity and identify heterogeneous signaling relationships between cancer cells and their endothelial cells. Future validation and expansion of these findings may validate novel cancer-endothelial cell interactions and related drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5076806fef62fc7b47a9b3751a8891a984dc18" target='_blank'>
              HCC spatial transcriptomic profiling reveals significant and potentially targetable cancer-endothelial interactions
              </a>
            </td>
          <td>
            Chenyue Lu, Amaya Pankaj, Michael J. Raabe, Cole Nawrocki, Ann Liu, Nova Xu, Bidish Kumar Patel, Matthew J. Emmett, Avril K Coley, Cristina R. Ferrone, Vikram Deshpande, I. Bhan, Y. Hoshida, David T. Ting, Martin J. Aryee, Joseph W Franses
          </td>
          <td>2024-09-27</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Although localized prostate cancer is relatively indolent, advanced prostate cancer manifests with aggressive and often lethal variants, including neuroendocrine prostate cancer (NEPC). To identify drivers of aggressive prostate cancer, we leveraged Sleeping Beauty (SB) transposon mutagenesis in a mouse model based on prostate-specific loss-of-function of Pten and Tp53. Compared with control mice, SB mice developed more aggressive prostate tumors, with increased incidence of metastasis. Notably, a significant percentage of the SB prostate tumors display NEPC phenotypes, and the transcriptomic features of these SB mouse tumors recapitulated those of human NEPC. We identified common SB transposon insertion sites (CIS) and prioritized associated CIS-genes differentially expressed in NEPC versus non-NEPC SB tumors. Integrated analysis of CIS-genes encoding for proteins representing upstream, post-translational modulators of master regulators controlling the transcriptional state of SB-mouse and human NEPC tumors identified sirtuin 1 (Sirt1) as a candidate mechanistic determinant of NEPC. Gain-of-function studies in human prostate cancer cell lines confirmed that SIRT1 promotes NEPC, while its loss-of-function or pharmacological inhibition abrogates NEPC. This integrative analysis is generalizable and can be used to identify novel cancer drivers for other malignancies. Summary Using an unbiased forward mutagenesis screen in an autochthonous mouse model, we have investigated mechanistic determinants of aggressive prostate cancer. SIRT1 emerged as a key regulator of neuroendocrine prostate cancer differentiation and a potential target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef638b5be4a6bc97dbe24dca3c6171d01e593aa" target='_blank'>
              A forward genetic screen identifies Sirtuin1 as a driver of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Francisca Nunes de Almeida, A. Vasciaveo, Ainsley Mike Antao, Min Zou, Matteo Di Bernardo, Simone de Brot, A. Rodriguez-Calero, Alexander Chui, A. L. Wang, Nicolas Floc’h, Jaime Y Kim, Stephanie N Afari, Timur Mukhammadov, Juan M Arriaga, Jinqiu Lu, Michael M. Shen, Mark A. Rubin, Andrea Califano, Cory Abate-Shen
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Uterine carcinosarcoma (UCS) is a rare gynecologic cancer with unfavorable prognosis. Its biphasic histologic components are known to associated with outcomes. Dissecting the tumor heterogeneity within UCS is crucial to discover molecular biomarkers for patient stratification. The aim of this study was to utilize spatial profiling technology to analyze immune presentation in different histological components within UCS tumors. The study included 23 UCS patients whose tumor sections were subjected to nanoString GeoMx Digital Spatial Profiling (DSP) to profile 30 immune-related proteins in 491 segments. We identified distinct cluster-labeled subtypes showing varied immune marker expressions. The immune presentations within the carcinoma and sarcoma components showed significant differences. Markers indicating M2 macrophages (CD45+CD163+) and epithelial-mesenchymal transition (EMT) (SMA, fibronectin) were highly expressed in the sarcoma components. The immune presentation subtypes within each histological component also differed between patients with recurrence in 5 years and without recurrence over 5 years. The cluster-labeled subtype with high Neutrophil/PMN-MDSC marker expression (CD66b) was associated with longer survival, whereas the subtype with high NK cell marker expressions (CD56) was linked to poor prognosis. Our result suggested that immune features within the carcinoma component of UCS tumors would determine patient outcomes and should be considered as a strategy for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677b6a5b9abbf69e6cc6fe6fd37eee258299da27" target='_blank'>
              Inter- and intra-tumor heterogeneous immune presentation in primary uterine carcinosarcoma determined by digital spatial profiling
              </a>
            </td>
          <td>
            Ying-Xuan Li, Wei-Chou Lin, Ko-Chen Chen, Wen-Chun Chang, Chin-Jui Wu, Lin-Hung Wei, Ruby Yun-Ju Huang, Bor-Ching Sheu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute graft-versus-host disease (aGVHD) is a significant complication of allogeneic hematopoietic stem cell transplantation (aHSCT), driven by alloreactive donor T cells in the gut. However, the roles of additional donor and host cells in this process are not fully understood. We conducted multiplexed imaging on 59 biopsies from patients with gastrointestinal GVHD and 10 healthy controls, revealing key pathological changes, including fibrosis, crypt alterations, loss of Paneth cells, accumulation of endocrine cells, and disrupted immune organization, particularly a reduction in IgA-secreting plasma cells. Interestingly, CD8T cells were enriched only in a subset of patients, while others exhibited non-canonical enrichments of macrophages and neutrophils. Post-transplantation time significantly influenced immune composition, with host cells dominating plasma and T cell compartments long after transplantation. This spatial atlas of healthy duodenum and GVHD uncovers non-canonical immune dynamics, offering insights into disease pathophysiology and potential clinical applications in GVHD and other inflammatory bowel diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69689939e79be56e05c61106cc952ea2166e2eac" target='_blank'>
              A spatial atlas of human gastro-intestinal acute GVHD reveals epithelial and immune dynamics underlying disease pathophysiology
              </a>
            </td>
          <td>
            Nofar Azulay, Idan Milo, Yuval Bussi, Raz Ben Uri, T. Haran, Michal Eldar, O. Elhanani, Yotam Harnik, Oran Yakubovsky, I. Nachmany, T. Salame, Martin Wartenberg, Philippe Bertheau, D. Michonneau, Gérard Socié, Leeat Keren
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, S. Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Juvenile localized and systemic scleroderma are rare autoimmune diseases which cause significant disability and morbidity in children. The mechanisms driving juvenile scleroderma remain unclear, necessitating further cellular and molecular level studies. The Visium CytAssist spatial transcriptomics (ST) platform, which preserves the spatial location of cells and simultaneously sequences the whole transcriptome, was employed to profile the histopathological slides from skin lesions of juvenile scleroderma patients. (1) Spatial domains were identified from ST data and exhibited strong concordance with the pathologist’s annotations of anatomical structures. (2) The integration of paired ST data and single-cell RNA sequencing (scRNA-seq) from the same patients validated the comparable accuracy of the two platforms and facilitated the estimation of cell type composition in ST data. (3) The pathologist-annotated immune infiltrates, such as perivascular immune infiltrates, were clearly delineated by the ST analysis, underscoring the biological relevance of the findings. This is the first study utilizing spatial transcriptomics to investigate skin lesions in juvenile scleroderma patients. The validity of the ST data was corroborated by gene expression analyses and the pathologist’s assessments. Integration with scRNA-seq data facilitated the cell type-level analysis and validation. Analyses of immune infiltrates through combined ST data and pathological review enhances our understanding of the pathogenesis of juvenile scleroderma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d114ce60551b0d377557f894744cf39236d724d1" target='_blank'>
              Spatial Transcriptomics Identifies Cellular and Molecular Characteristics of Scleroderma Skin Lesions: Pilot Study in Juvenile Scleroderma
              </a>
            </td>
          <td>
            Tianhao Liu, Deren Esencan, Claudia M. Salgado, Chongyue Zhao, Ying-Ju Lai, Theresa Hutchins, Anwesha Sanyal, Wei Chen, Kathryn S Torok
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce58909f9de06504a3ce2e2bc45ebf1454164f" target='_blank'>
              Spatial single-cell protein landscape reveals vimentinhigh macrophages as immune-suppressive in the microenvironment of hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyao Qiu, Tao Zhou, Shuai Li, Jianmin Wu, Jing Tang, Guosheng Ma, Shuai Yang, Ji Hu, Kaiting Wang, Siyun Shen, Hongyang Wang, Lei Chen
          </td>
          <td>2024-09-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Increasing evidence suggests the tumor microenvironment (TME) governs solid tumor response to immune checkpoint inhibition (ICI). Decoding the relationship between the cell compositional diversity of the tumor microenvironment (TME) and the therapeutic outcomes has been a longstanding problem in solid tumor research. In this work, we develop a systems-theoretic formalism to decipher the key mechanisms of growth, proliferation, immune evasion, and drug resistance that run common across solid tumors in the context of Immune Checkpoint Inhibitors (ICI). We reconstructed a core TME network in common across most solid tumors, containing multiple tumor and non-tumor cell types in distinct functional states, and molecular agents mediating cellular signaling and cell-cell interactions. Our analysis shows that the core TME network is sufficient to yield a multiplicity of attractors corresponding to clinically observed TME subtypes namely, immune or fibro dominated, immune or fibro desert, and immune and fibro deficient. Importantly, the reachability around the pre-ICI attractors governs the response to ICI explaining the TME subtype-specific therapy outcomes. We analyzed the attractor transition network to identify subtype-specific combination therapies that can drive unresponsive TME to a responsive subtype. We derived mathematical conditions relating TME balances to determine the limits of the efficacy of combination therapies. Our results hold for a large class of smooth biochemical kinetics with monotone and bounded interactions and (semi-)concave proliferation rules. The analytical findings have been verified with extensive simulation of different TME sub-types. Overall, we propose a generalized systems formalism that accounts for the TME properties governing ICI response and can aid in designing intervention strategies for improved tumor prognosis. Significance Statement Non-responsive therapy outcomes have been a persistent problem in cancer treatment. Predicting the possibility of non-responsiveness to a particular therapy from the pre-treatment composition of the tumor microenvironment (TME) aids in designing appropriate combination treatment strategies toward an improved prognosis. The present work develops a systems-theoretic formalism that aims to unfold the mechanisms behind solid tumors’ growth, non-responsiveness, and recurrence. Unlike a single model, the proposed formalism does not assume any particular kinetics, barring some minimal assumptions. This enables us to explain some of the relevant observations made by the recent experimental studies. Finally, the closed-form conditions obtained for responsivity and recurrence can also guide novel therapeutic strategies that may be able to restore responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657fb6c08b911bd43c20c8cf26d56275f3634fc0" target='_blank'>
              From tumor microenvironment to immuno-therapeutic outcomes for solid tumors: A systems theoretic approach
              </a>
            </td>
          <td>
            Priyan Bhattacharya, R. Vadigepalli
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I. Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Regulatory T (Treg) cells are known to play critical roles in tissue repair via provision of growth factors such as amphiregulin (Areg). Areg-producing Treg cells have previously been difficult to study because of an inability to isolate live Areg-producing cells. In this report, we created a novel reporter mouse to detect Areg expression in live cells (AregThy1.1). We employed influenza A and bleomycin models of lung damage to sort Areg-producing and –non-producing Treg cells for transcriptomic analyses. Single cell RNA-seq revealed distinct subpopulations of Treg cells and allowed transcriptomic comparisons of damage-induced populations. Single cell TCR sequencing showed that Treg cell clonal expansion is biased towards Areg-producing Treg cells, and largely occurs within damage-induced subgroups. Gene module analysis revealed functional divergence of Treg cells into immunosuppression-oriented and tissue repair–oriented groups, leading to identification of candidate receptors for induction of repair activity in Treg cells. We tested these using an ex vivo assay for Treg cell–mediated tissue repair, identifying 4-1BB agonism as a novel mechanism for reparative activity induction. Overall, we demonstrate that the AregThy1.1mouse is a promising tool for investigating tissue repair activity in leukocytes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8109b78fe9a1acd23005c59f514a6302e62b3f" target='_blank'>
              An amphiregulin reporter mouse enables transcriptional and clonal expansion analysis of reparative lung Treg cells
              </a>
            </td>
          <td>
            Lucas F. Loffredo, Katherine A. Kaiser, Adam E. Kornberg, Samhita Rao, Kenia de los Santos-Alexis, Arnold Han, Nicholas Arpaia
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial in cancer development and therapeutic response. Immunotherapy is increasingly recognized as a critical component of cancer treatment. While immunotherapies have shown efficacy in various cancers, including breast cancer, patient responses vary widely. Some patients receive significant benefits, while others experience minimal or no improvement. This disparity underscores the complexity and diversity of the immune system. In this study, we investigated the immune landscape and cell–cell communication within the TME of breast cancer through integrated analysis of bulk and single-cell RNA sequencing data. We established profiles of tumor immune infiltration that span across a broad spectrum of adaptive and innate immune cells. Our clustering analysis of immune infiltration identified three distinct patient groups: high T cell abundance, moderate infiltration, and low infiltration. Patients with low immune infiltration exhibited the poorest survival rates, while those in the moderate infiltration group showed better outcomes than those with high T cell abundance. Moreover, the high cell abundance group was associated with a greater tumor burden and higher rates of TP53 mutations, whereas the moderate infiltration group was characterized by a lower tumor burden and elevated PIK3CA mutations. Analysis of an independent single-cell RNA-seq breast cancer dataset confirmed the presence of similar infiltration patterns. Further investigation into ligand–receptor interactions within the TME unveiled significant variations in cell–cell communication patterns among these groups. Notably, we found that the signaling pathways SPP1 and EGF were exclusively active in the low immune infiltration group, suggesting their involvement in immune suppression. This work comprehensively characterizes the composition and dynamic interplay in the breast cancer TME. Our findings reveal associations between the extent of immune infiltration and clinical outcomes, providing valuable prognostic information for patient stratification. The unique mutations and signaling pathways associated with different patient groups offer insights into the mechanisms underlying diverse tumor immune infiltration and the formation of an immunosuppressive tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff73b13094e5afc913b4266529d0100e73db7ee" target='_blank'>
              Characterizing the Tumor Microenvironment and Its Prognostic Impact in Breast Cancer
              </a>
            </td>
          <td>
            Wenjuan Zhang, Alex Lee, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e27ecfb76604dddc8d09da4d1bbaaf7b66725bf" target='_blank'>
              Spatial multiplexed immunofluorescence analysis reveals coordinated cellular networks associated with overall survival in metastatic osteosarcoma
              </a>
            </td>
          <td>
            Ryan A Lacinski, Sebastian A. Dziadowicz, Vincent K Melemai, Brody Fitzpatrick, John J. Pisquiy, Tanya Heim, Ines Lohse, Karen Schoedel, Nicolas J Llosa, Kurt R Weiss, Brock A Lindsey
          </td>
          <td>2024-09-27</td>
          <td>Bone Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Several methods for cell cycle inference from sequencing data exist and are widely adopted. In contrast, methods for classification of cell cycle state from imaging data are scarce. We have for the first time integrated sequencing and imaging derived cell cycle pseudo-times for assigning 449 imaged cells to 693 sequenced cells at an average resolution of 3.4 and 2.4 cells for sequencing and imaging data respectively. Data integration revealed thousands of pathways and organelle features that are correlated with each other, including several previously known interactions and novel associations. The ability to assign the transcriptome state of a profiled cell to its closest living relative, which is still actively growing and expanding opens the door for genotype-phenotype mapping at single cell resolution forward in time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73340e559ff9e02cb63ea7b7366f2d2d5be7e9ec" target='_blank'>
              Cell identity revealed by precise cell cycle state mapping links data modalities
              </a>
            </td>
          <td>
            Saeed Alahmari, Andrew Schultz, Jordan Albrecht, Vural Tagal, Zaid Siddiqui, Sandhya Prabhakaran, I. El Naqa, Alexander Anderson, Laura M. Heiser, Noemi Andor
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather M. O’Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell is important because of its potential to treat various diseases. As deep learning continues to advance, using unsupervised methods to classify medical images has become a significant focus because collecting high-quality labeled data for medical images is labor-intensive and time-consuming. Beyond the need for accurate labeling, there is a desire to explore the underlying characteristics of the data, even when labels may be ambiguous or uncertain. To address these challenges, we present a novel approach that combines image clustering with an insightful explanation of how these clusters are formed. Our method employs a U-net combined with a clustering algorithm to segment the dataset into different groups. After clustering, we use various techniques to interpret and elucidate the results. Moreover, our paper introduces a unique dataset focused on cell data, specifically highlighting the developmental patterns of cancer cells and T cells under various experimental conditions. This dataset offers a rich source of information and presents a complex challenge for image classification due to the diversity of conditions and cell behaviors involved. Our study thoroughly compares different architectural models on this new dataset, demonstrating the superior performance of our proposed architecture. Through experimental analysis and ablation studies, we provide substantial evidence of the benefits offered by our architecture, not only in terms of accuracy but also in its ability to reveal deeper insights into the data. This work advances the field of image classification and opens new possibilities for understanding complex biological processes through computer vision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b2222e0e50090ec7fb0ff692b055ed44f83f60" target='_blank'>
              High-throughput Image-based Clustering of CAR-T/Tumor Cocultures for Rapid and Facile Hit Identification
              </a>
            </td>
          <td>
            Zhi Xu, Xueqi Liu, Kirby Madden-Hennessey, Jordan Urbani, S. K. Nikkhoi, Anusuya Ramasubramanian, Kartika G. Venugopal, Qi Zhao, Eric L. Smith, Yun Fu
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Single-cell Hi-C (scHi-C) has provided unprecedented insights into the heterogeneity of 3D genome organization. However, its sparse and noisy nature poses challenges for computational analyses, such as chromatin architectural feature identification. Here, we introduce scCAFE, a deep learning model for the multi-scale detection of architectural features at the single-cell level. scCAFE provides a unified framework for annotating chromatin loops, TAD-like domains (TLDs), and compartments across individual cells. Our model outperforms previous scHi-C loop calling methods and delivers accurate predictions of TLDs and compartments that are biologically consistent with previous studies. The resulting single-cell annotations also offer a measure to characterize the heterogeneity of different levels of architectural features across cell types. We leverage this heterogeneity and identify a series of marker loop anchors, which demonstrate the potential of the 3D genome data to annotate cell identities without the aid of simultaneously sequenced omics data. Overall, scCAFE not only serves as a useful tool for analyzing single-cell genomic architecture, but also paves the way for precise cell-type annotations solely based on 3D genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9e853dc3e19f05f5603a9a9aa1ba89e02d8ae38" target='_blank'>
              Unveiling multi-scale architectural features in single-cell Hi-C data using scCAFE
              </a>
            </td>
          <td>
            Fuzhou Wang, Jiecong Lin, Hamid Alinejad-Rokny, Wenjing Ma, Lingkuan Meng, Lei Huang, Jixiang Yu, Nanjun Chen, Yuchen Wang, Zhongyu Yao, Weidun Xie, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The identification of gene regulatory networks (GRN) governing distinct cell fates in multilineage cellular differentiation systems is of critical importance for understanding cell fate decision. Single-cell RNA-sequencing (scRNA-seq) provides a powerful tool for the quantification of gene-level co-variation across the cell state manifold. However, accurate GRN reconstruction is hampered by the sparsity of scRNA-seq data introducing substantial technical noise. Moreover, the high dimensionality of typical scRNA-seq datasets limits the scalability of available approaches. To overcome these challenges, and to facilitate the inference of lineage-specific GRNs with directed regulator-target relations, we introduce NetID. This approach optimizes coverage of the cell state manifold by homogenous metacells and avoids spurious gene-gene correlations observed with available imputation methods. Benchmarking demonstrates superior performance of NetID compared to imputation-based GRN inference. By incorporating cell fate probability information, NetID facilitates prediction of lineage-specific GRNs and recovers known network motifs centered around lineage-determining transcription factors governing bone marrow hematopoiesis, making it a powerful toolkit for deciphering the gene regulatory control of cellular differentiation from large-scale single-cell transcriptome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/725296ffdf0a7cacc6e78577d9871b12ce8048c7" target='_blank'>
              Scalable identification of lineage-specific gene regulatory networks from metacells with NetID
              </a>
            </td>
          <td>
            Weixu Wang, Yichen Wang, Ruiqi Lyu, D. Grün
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We employed laser microdissection to selectively harvest tumor cells and stroma from the microenvironment of formalin-fixed, paraffin-embedded head and neck squamous cell carcinoma (HNSCC) tissues. The captured HNSCC tissue fractions were analyzed by quantitative mass spectrometry-based proteomics using a data independent analysis approach. In paired samples, we achieved excellent proteome coverage having quantified 6,668 proteins with a median quantitative coefficient of variation under 10%. We observed significant differences in relevant functional pathways between the spatially resolved tumor and stroma regions. Our results identified extracellular matrix (ECM) as a major component enriched in the stroma, including many cancer associated fibroblast signature proteins in this compartment. We demonstrate the potential for comparative deep proteome analysis from very low starting input in a scalable format that is useful to decipher the alterations in tumor and the stromal microenvironment. Correlating such results with clinical features or disease progression will likely enable identification of novel targets for disease classification and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81bad8ce1ede8240c16776044c3909a46ef259c" target='_blank'>
              dia-PASEF Proteomics of Tumor and Stroma LMD Enriched from Archived HNSCC Samples
              </a>
            </td>
          <td>
            Aswini Panigrahi, Allison L. Hunt, Diego Assis, Matthew Willetts, Bhaskar V Kallakury, Bruce Davidson, Thomas P. Conrads, R. Goldman
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Accurate phenotyping of cells in the tumor microenvironment is crucial for understanding cancer biology and developing effective therapies. However, current methods require precise cell segmentations and struggle to generalize across different imaging modalities, limiting their utility in digital pathology. Here, we show that Contrastive Learning Enabled Accurate Registration of Immune and Tumor Cells (CLEAR-IT) overcomes these limitations, providing a robust and versatile tool for cell phenotyping. CLEAR-IT accurately phenotypes cells comparable to state-of-the-art methods, generalizes across multiplex imaging modalities, maintains high performance even with limited number of labels, and enables the extraction of prognostic markers. Additionally, CLEAR-IT can be combined with existing methods to boost their performance, whereas its lack of need for precise cell segmentations significantly reduces training efforts. This method enhances the robustness and efficiency of digital pathology workflows, making it a valuable tool for cancer research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02cd2bfec0cff758ded2e68209475ad537d70fc" target='_blank'>
              CLEAR-IT: Contrastive Learning to Capture the Immune Composition of Tumor Microenvironments
              </a>
            </td>
          <td>
            Daniel Spengler, Serafim Korovin, Kirti Prakash, Peter Bankhead, R. Debets, Carlas S. Smith, H. E. Balcioglu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Pancreatic cancer remains one of the most deadly cancers, with over 450,000 fatalities in 2022 alone. The most lethal variation of this cancer is Pancreatic Ductal Adenocarcinoma (PDAC), with a 5-year survival rate of under 10%. Combination chemotherapy has provided major advances in this disease but it remains challenging for many reasons including the immunosuppressive nature of the tumor microenvironment (TME). PDAC has a highly desmoplastic microenvironment that suppresses antitumor immunity and promotes tumor cell invasion, metastasis, and resistance through epithelial to mesenchymal transition (EMT) of cancer cells. Cancer cells can undergo various forms of cell death, notably apoptosis, necrosis, autophagy, pyroptosis, and ferroptosis. These mechanisms are triggered by different cellular stressors and shape the microenvironment. Through the use of 15 Visium spatial transcriptomics samples from publicly available primary PDAC tumors and paired single-cell RNA (scRNA) data, we aim to understand how cell stress and cell death shape the tumor microenvironment (TME) and epithelial-mesenchymal transition (EMT). By leveraging information from known pathways in literature, we will identify cellular programs related to stress, death, and EMT. We will explore intracellular correlations within single-cell data and extracellular interactions within the TME using spatial data to uncover how different forms of cell death and stress signatures influence the TME and EMT spectrum. The initial phase involves identifying the types of stress that lead to various forms of cell death intracellularly and validating these gene signatures. Subsequently, we will apply these validated signatures to the spatial data to correlate each score and search for significant overlapping programs. This will extend beyond stress and death, incorporating TME signatures corresponding to different cell types and the EMT spectrum. By understanding the signals that promote the death of various cell populations, we aim to identify combination treatments that target both ends of the EMT spectrum. Additionally, this research will provide insights into the inflammatory impact of cell death on the TME. Our approach aims to inform the development of therapies that disrupt resistance mechanisms, ultimately improving treatment outcomes for patients with PDAC. This research can potentially reveal new therapeutic targets and strategies, enhancing our ability to combat this deadly disease.
 Citation Format: Rahul Bansal, Izabella Zamora, Samuel Wright, Nir Hacohen, Arnav Mehta. Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba425dbbc0b9c870572a7ccded079a29b6e67232" target='_blank'>
              Abstract C074: Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Rahul Bansal, Izabella Zamora, Samuel J Wright, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises a diverse array of cells, both cancerous and non-cancerous, including stromal cells and immune cells. Complex interactions among these cells play a central role in driving cancer progression, impacting critical aspects such as tumor initiation, growth, invasion, response to therapy, and the development of drug resistance. While targeting the TME has emerged as a promising therapeutic strategy, there is a critical need for innovative approaches that accurately replicate its complex cellular and non-cellular interactions; the goal being to develop targeted, personalized therapies that can effectively elicit anti-cancer responses in patients. Microfluidic systems present notable advantages over conventional in vitro 2D co-culture models and in vivo animal models, as they more accurately mimic crucial features of the TME and enable precise, controlled examination of the dynamic interactions among multiple human cell types at any time point. Combining these models with next-generation technologies, such as bioprinting, single cell sequencing and real-time biosensing, is a crucial next step in the advancement of microfluidic models. This review aims to emphasize the importance of this integrated approach to further our understanding of the TME by showcasing current microfluidic model systems that integrate next-generation technologies to dissect cellular intra-tumoral interactions across different tumor types. Carefully unraveling the complexity of the TME by leveraging next generation technologies will be pivotal for developing targeted therapies that can effectively enhance robust anti-tumoral responses in patients and address the limitations of current treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78b792b563d1d9bcfce83a027e1da609d44dc5" target='_blank'>
              Improving tumor microenvironment assessment in chip systems through next-generation technology integration
              </a>
            </td>
          <td>
            Daniela Gaebler, Stephanie J. Hachey, Christopher C. W. Hughes
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee31a2e42e286fd7d20df7672d92a049ecf6880" target='_blank'>
              An integrated single-cell reference atlas of the human endometrium
              </a>
            </td>
          <td>
            Magda Marečková, L. Garcia-Alonso, Marie Moullet, Valentina Lorenzi, R. Petryszak, Carmen Sancho-Serra, Agnes Oszlanczi, Cecilia Icoresi Mazzeo, Frederick C. K. Wong, Iva Kelava, Sophie Hoffman, M. Krassowski, K. Garbutt, Kezia Gaitskell, S. Yancheva, E. Woon, V. Male, I. Granne, Karin Hellner, Krishnaa T. Mahbubani, K. Saeb‐Parsy, M. Lotfollahi, E. Prigmore, J. Southcombe, Rebecca Dragovic, Christian M. Becker, K. Zondervan, R. Vento-Tormo
          </td>
          <td>2024-08-28</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69fde8318f7a677932512a6597cf3b791cc10829" target='_blank'>
              B-cell intrinsic RANK signaling cooperates with TCL1 to induce lineage-dependent B-cell transformation
              </a>
            </td>
          <td>
            Lisa Pfeuffer, Viola Siegert, Julia Frede, Leonie Rieger, Riccardo Trozzo, N. de Andrade Krätzig, Sandra Ring, Shamim Sarhadi, Nicole Beck, Stefan Niedermeier, Mar Abril-Gil, Mohamed Elbahloul, Marianne Remke, Katja Steiger, Ruth Eichner, Julia Jellusova, Roland Rad, Florian Bassermann, Christof Winter, J. Ruland, M. Buchner
          </td>
          <td>2024-08-28</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a typical highly heterogeneous solid tumor with high morbidity and mortality worldwide, especially in China; however, the immune microenvironment of HCC has not been clarified so far. Here, we employed single‐cell RNA sequencing (scRNA‐seq) on diethylnitrosamine (DEN)‐induced mouse HCC model to dissect the immune cell dynamics during tumorigenesis. Our findings reveal distinct immune profiles in both precancerous and cancerous lesions, indicating early tumor‐associated immunological alterations. Notably, specific T and B cell subpopulations are preferentially enriched in the HCC tumor microenvironment (TME). Furthermore, we identified a subpopulation of naïve B cells with high CD83 expression, correlating with improved prognosis in human HCC. These signature genes were validated in The Cancer Genome Atlas HCC RNA‐seq dataset. Moreover, cell interaction analysis revealed that subpopulations of B cells in both mouse and human samples are activated and may potentially contribute to oncogenic processes. In summary, our study provides insights into the dynamic immune microenvironment and cellular networks in HCC pathogenesis, with a specific emphasis on naïve B cells. These findings emphasize the significance of targeting TME in HCC patients to prevent HCC pathological progression, which may give a new perspective on the therapeutics for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76c814af70ac2f24272bcadf22a4dd9e85ffd9ec" target='_blank'>
              Leveraging single‐cell RNA‐seq for uncovering naïve B cells associated with better prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Qingjia Sun, Rui Gao, Yingxin Lin, Xianchao Zhou, Tao Wang, Jian He
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Ethnic disparities in breast cancer (BC) outcomes have been reported, with lower survival rates among black women compared to white. Additionally, the unfavorable triple negative (TNBC) sub-type is more prevalent in black women and within this population, risk of mortality is again higher compared to white women. Biological contributors to this disparity are less clearly understood than socio-economic factors. Recent studies recognize different breast tumour immune microenvironments between black and white BC patients, necessitating further research into ethnicity-related differences in anti-tumor immune responses. Such responses begin in regional lymph nodes (LNs) with tumor neoantigen presentation. Here, we generate single-cell spatial maps of involved TNBC LN cores in treatment naïve black and white women. To guide design of a phenotyping panel for LNs, we profiled blood samples (n=95) using high-dimensional flow cytometry from newly diagnosed black and white BC patients. We found differential skewing of NK cell subsets, lower memory B cell (identified as IgM+CD27+IgD- & IgM+CD27+IgD+ cells) abundance and higher CD4+ T effector memory presence in black patients compared to white. Additionally, higher antigen presentation capacity was observed with increased intermediate monocytes and conventional DCs, confirmed by MS-based proteomics. Using highly multiplexed imaging with a panel of 26 markers, we performed single-cell spatial immune profiling to characterize cellular phenotypes, neighborhoods and interactions within a cohort of LN core biopsies from black patients and white patients. A total of 536,113 cells were segmented for analysis. Cell abundance analysis revealed higher CD8+ T effector memory and CD4+ T cell subsets in white patients, except for CD4+ T effector memory cells which were higher in black patients, consistent with blood profiling. Cellular neighborhood analysis revealed black patient tumor-rich regions were enriched for M2 macrophages while white patients had no such enrichment. In these white patients, a neighborhood resembling canonical follicular structures almost exclusively containing B cells was identified. Interestingly in these black patients, analogous B cell rich regions also comprised a heterogeneous array of other immune cells. Further analysis to explore B cell zones is ongoing. Preliminary interaction analysis identified immunosuppressive crosstalk in black patients with enriched Treg & CD8+ T cell and Treg & NK interactions, not observed in white patients. These initial findings suggest ethnicity-based variation in LN architectures from treatment naive patients with TNBC, supporting the hypothesis that variation in immune programming could translate into differences in pro and anti-tumor immunity, contributing to disparities in outcome amongst black and white patients.
 Citation Format: Paul R. Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, Thanussuyah Alaguthurai, Priya Hossain, Patrycja Gazińska, Sheeba Irshad. Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C142.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c431f2f91ac1c49d9f19dea4541265da8c85d8d6" target='_blank'>
              Abstract C142: Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers
              </a>
            </td>
          <td>
            Paul Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, T. Alaguthurai, Priya Hossain, Patrycja Gazinska, S. Irshad
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Musculoskeletal sarcomas pose major challenges to researchers and clinicians due to their rarity and heterogeneity. Xenografting human cells or tumor fragments in rodents is a mainstay for the generation of cancer models and for the preclinical trial of novel drugs. Lately, though, technical, intrinsic and ethical concerns together with stricter regulations have significantly curbed the employment of murine patient-derived xenografts (mPDX). In alternatives to murine PDXs, researchers have focused on embryonal systems such as chorioallantoic membrane (CAM) and zebrafish embryos. These systems are time- and cost-effective hosts for tumor fragments and near-patient cells. The CAM of the chick embryo represents a unique vascularized environment to host xenografts with high engraftment rates, allowing for ease of visualization and molecular detection of metastatic cells. Thanks to the transparency of the larvae, zebrafish allow for the tracking of tumor development and metastatization, enabling high-throughput drug screening. This review will focus on xenograft models of musculoskeletal sarcomas to highlight the intrinsic and technically distinctive features of the different hosts, and how they can be exploited to elucidate biological mechanisms beneath the different phases of the tumor’s natural history and in drug development. Ultimately, the review suggests the combination of different models as an advantageous approach to boost basic and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616e3a054aa19b7341e12e48c8734d79d078b3d8" target='_blank'>
              Xenografting Human Musculoskeletal Sarcomas in Mice, Chick Embryo, and Zebrafish: How to Boost Translational Research
              </a>
            </td>
          <td>
            Veronica Giusti, G. Miserocchi, Giulia Sbanchi, Micaela Pannella, C. Hattinger, M. Cesari, Leonardo Fantoni, A. N. Guerrieri, C. Bellotti, A. De Vita, C. Spadazzi, D. Donati, M. Torsello, Enrico Lucarelli, T. Ibrahim, L. Mercatali
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Keratinocytes, the dominant cell type in the melanoma microenvironment during tumor initiation, exhibit diverse effects on melanoma progression. Using a zebrafish model of melanoma and human cell co-cultures, we observed that keratinocytes undergo an Epithelial–Mesenchymal Transition (EMT)-like transformation in the presence of melanoma, reminiscent of their behavior during wound healing. Surprisingly, overexpression of the EMT transcription factor Twist in keratinocytes led to improved overall survival in zebrafish melanoma models, despite no change in tumor initiation rates. This survival benefit was attributed to reduced melanoma invasion, as confirmed by human cell co-culture assays. Single-cell RNA-sequencing revealed a unique melanoma cell cluster in the Twist-overexpressing condition, exhibiting a more differentiated, less invasive phenotype. Further analysis nominated homotypic jam3b-jam3b and pgrn-sort1a interactions between Twist-overexpressing keratinocytes and melanoma cells as potential mediators of the invasive restraint. Our findings suggest that EMT in the tumor microenvironment (TME) may limit melanoma invasion through altered cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8055a65d64a36749562fed897a59067bde3fe48" target='_blank'>
              Restraint of melanoma progression by cells in the local skin environment
              </a>
            </td>
          <td>
            Yilun Ma, Mohita M. Tagore, Miranda V. Hunter, Ting-Hsiang Huang, Emily D. Montal, Joshua M. Weiss, Richard M. White
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. Vanderhyden, Hector C. Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="3D bioprinting has become a valuable tool for studying the biology of solid tumors, including glioblastoma multiforme (GBM). Our analysis of publicly available bulk RNA and single-cell sequencing data has allowed us to define the chemotactic profile of GBM tumors and identify the cell types that secrete particular chemokines in the GBM tumor microenvironment (TME). Our findings indicate that primary GBM tissues express multiple chemokines, whereas spherical monocultures of GBM cells significantly lose this diversity. Subsequently, the comparative analysis of GBM spherical monocultures vs. 3D-bioprinted multicultures of cells showed a restoration of chemokine profile diversity in 3D-bioprinted cultures. Furthermore, single-cell RNA-Seq analysis showed that cells of the perivascular niche (pericytes and endocytes) express multiple chemokines in the GBM TME. Next, we 3D-bioprinted cells from two glioblastoma cell lines, U-251 and DK-MG, alone and as co-cultures with mesenchymal stromal cells (representing cells of the perivascular niche) and assessed the chemokine secretome. The results clearly demonstrated that the interaction of tumors and mesenchymal cells leads to in a significant increase in the repertoire and levels of secreted chemokines under culture in 21% O2 and 1% O2. Our study indicates that cells of the perivascular niche may perform a substantial role in shaping the chemokine microenvironment in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae245299abc68f53e5e3fd8e4dc327ea2b0cb63" target='_blank'>
              3D-Bioprinted Co-Cultures of Glioblastoma Multiforme and Mesenchymal Stromal Cells Indicate a Role for Perivascular Niche Cells in Shaping Glioma Chemokine Microenvironment
              </a>
            </td>
          <td>
            K. Zielniok, Kinga Rusinek, A. Słysz, Mieszko Lachota, E. Bączyńska, Natalia Wiewiórska-Krata, Anna Szpakowska, Martyna Ciepielak, B. Foroncewicz, K. Mucha, Radosław Zagożdżon, Zygmunt Pojda
          </td>
          <td>2024-08-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite being the leading cause of cancer-related childhood mortality, pediatric gliomas have been relatively understudied, and the repurposing of immunotherapies has not been successful. Whole-transcriptome sequencing, single-cell sequencing, and sequential multiplex immunofluorescence were used to identify an immunotherapeutic strategy that could be applied to multiple preclinical glioma models. MAPK-driven pediatric gliomas have a higher IFN signature relative to other molecular subgroups. Single-cell sequencing identified an activated and cytotoxic microglia (MG) population designated MG-Act in BRAF-fused, MAPK-activated pilocytic astrocytoma (PA), but not in high-grade gliomas or normal brain. T cell immunoglobulin and mucin domain 3 (TIM3) was expressed on MG-Act and on the myeloid cells lining the tumor vasculature but not normal brain vasculature. TIM3 expression became upregulated on immune cells in the PA microenvironment, and anti-TIM3 reprogrammed ex vivo immune cells from human PAs to a proinflammatory cytotoxic phenotype. In a genetically engineered murine model of MAPK-driven, low-grade gliomas, anti-TIM3 treatment increased median survival over IgG- and anti–PD-1–treated mice. Single-cell RNA-Seq data during the therapeutic window of anti-TIM3 revealed enrichment of the MG-Act population. The therapeutic activity of anti-TIM3 was abrogated in mice on the CX3CR1 MG–KO background. These data support the use of anti-TIM3 in clinical trials of pediatric low-grade, MAPK-driven gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93c9fb9117641aac2f760a5c311e07768d3cba6" target='_blank'>
              Pediatric glioma immune profiling identifies TIM3 as a therapeutic target in BRAF fusion pilocytic astrocytoma
              </a>
            </td>
          <td>
            Shashwat Tripathi, Hinda Najem, Corey Dussold, Sebastian Pacheco, Ruochen Du, Moloud Sooreshjani, Lisa A Hurley, James P Chandler, R. Stupp, A. Sonabend, C. Horbinski, R. Lukas, Joanne Xiu, Giselle Y. López, Theodore Nicolaides, Valerie Brown, N. R. Wadhwani, Sandi K Lam, C. D. James, Ganesh Rao, Maria G Castro, Amy B. Heimberger, Michael DeCuypere
          </td>
          <td>2024-08-13</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Genetically engineered CD8+ T cells are being explored for the treatment of various cancers. Analytical characterization represents a major challenge in the development of genetically engineered cell therapies, especially assessing the potential off-target editing and product heterogeneity. As conventional sequencing techniques only provide information at the bulk level, they are unable to detect off-target CRISPR translocation or editing events occurring in minor cell subpopulations. In this study, we report the analytical development of a single-cell multi-omics DNA and protein assay to characterize genetically engineered cell products for safety and genotoxicity assessment. We were able to quantify on-target edits, off-target events, and potential translocations at the targeting loci with per-cell granularity, providing important characterization data of the final cell product. Conclusion: A single-cell multi-omics approach provides the resolution required to understand the composition of cellular products and identify critical quality attributes (CQAs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a403ecc7f675cd4ff2179f421e45809aa747f" target='_blank'>
              Assessing a single-cell multi-omic analytic platform to characterize ex vivo-engineered T-cell therapy products
              </a>
            </td>
          <td>
            Maryam Moshref, Jerry Hung-Hao Lo, Andrew McKay, Julien Camperi, Joseph Schroer, Norikiyo Ueno, Shu Wang, Saurabh Gulati, Somayeh Tarighat, Steffen Durinck, Ho Young Lee, Dayue Chen
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
The determinants of the response to checkpoint immunotherapy in hepatocellular carcinoma (HCC) remain poorly understood. The organisation of the immune response in the tumour microenvironment (TME) is expected to govern immunotherapy outcomes but spatial immunotypes remain poorly defined.


OBJECTIVE
We hypothesised that the deconvolution of spatial immune network architectures could identify clinically relevant immunotypes in HCC.


DESIGN
We conducted highly multiplexed imaging mass cytometry on HCC tissues from 101 patients. We performed in-depth spatial single-cell analysis in a discovery and validation cohort to deconvolute the determinants of the heterogeneity of HCC immune architecture and develop a spatial immune classification that was tested for the prediction of immune checkpoint inhibitor (ICI) therapy.


RESULTS
Bioinformatic analysis identified 23 major immune, stroma, parenchymal and tumour cell types in the HCC TME. Unsupervised neighbourhood detection based on the spatial interaction of immune cells identified three immune architectures with differing involvement of immune cells and immune checkpoints dominated by either CD8 T-cells, myeloid immune cells or B- and CD4 T-cells. We used these to define three major spatial HCC immunotypes that reflect a higher level of intratumour immune cell organisation: depleted, compartmentalised and enriched. Progression-free survival under ICI therapy differed significantly between the spatial immune types with improved survival of enriched patients. In patients with intratumour heterogeneity, the presence of one enriched area governed long-term survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/592e13ae9180b39b999935cfd28a102ececa8437" target='_blank'>
              Spatial single-cell profiling and neighbourhood analysis reveal the determinants of immune architecture connected to checkpoint inhibitor therapy outcome in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            H. Salié, Lara Wischer, A. D’Alessio, Ira Godbole, Yuan Suo, P. Otto-Mora, Juergen Beck, Olaf Neumann, A. Stenzinger, P. Schirmacher, C. Fulgenzi, Andreas Blaumeiser, Melanie Boerries, Natascha Roehlen, Michael Schultheiss, M. Hofmann, R. Thimme, D. Pinato, T. Longerich, B. Bengsch
          </td>
          <td>2024-09-30</td>
          <td>Gut</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Despite efforts to understand breast cancer biology, metastatic disease remains a clinical challenge. Identifying suppressors of breast cancer progression and mechanisms of transition to more invasive phenotypes could provide game changing therapeutic opportunities. Transcriptional deregulation is central to all malignancies, highlighted by the extensive reprogramming of regulatory elements that underlie oncogenic programs. Among these, super-enhancers (SEs) stand out due to their enrichment in genes controlling cancer hallmarks. To reveal novel breast cancer dependencies, we integrated the analysis of the SE landscape with master regulator activity inference for a series of breast cancer cell lines. As a result, we identified T-helper-inducing Poxviruses and Zinc-finger (POZ)/Krüppel-like factor (ThPOK, ZBTB7B), a CD4+ cell lineage commitment factor, as a breast cancer master regulator that is recurrently associated with a SE. ThPOK expression is highest in luminal breast cancer but is significantly reduced in the basal subtype. Manipulation of ThPOK levels in cell lines shows that its repressive function restricts breast cancer cells to an epithelial phenotype by suppressing the expression of genes involved in the epithelial-mesenchymal transition (EMT), WNT/β-catenin target genes, and the pro-metastatic TGFβ pathway. Our study reveals ThPOK as a master transcription factor that restricts the acquisition of metastatic features in breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bcac35bb061bf9ab262a029b66da149e294b402" target='_blank'>
              Super-enhancer profiling reveals ThPOK/ZBTB7B, a CD4+ cell lineage commitment factor, as a master regulator that restricts breast cancer cells to a luminal non-migratory phenotype
              </a>
            </td>
          <td>
            Camila D. Arcuschin, Kamin Kahrizi, Rosalyn W. Sayaman, Carolina DiBenedetto, Yizhuo Shen, Pedro J Salaberry, Ons Zakroui, Cecilia Schwarzer, Alessandro Scapozza, Paola Betancur, Julie D. Saba, Jean-Philippe Coppé, M. Barcellos-Hoff, Dietmar Kappes, L. J. van ’t Veer, Ignacio E Schor, Denise P Muñoz
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bca16ff9172b241a4b47443f7584ae07d099226" target='_blank'>
              Transcriptomic and proteomic spatial profiling of pediatric and adult diffuse midline glioma H3 K27-Altered
              </a>
            </td>
          <td>
            Sudarshawn Damodharan, Jack M Shireman, Elliot Xie, Emily Distler, Christina Kendziorski, Mahua Dey
          </td>
          <td>2024-09-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Metastasis is the major cause of cancer-related deaths. Emerging evidence has shown that epigenetic regulation plays a fundamental role in cancer metastasis. To better understand the epigenetic regulation of metastasis, we conducted an in vivo shRNA screen for vulnerabilities of brain metastasis and identified N-acetyltransferase 10 (NAT10) as a driver of brain metastasis. Knockdown of NAT10 significantly restrains cancer cell proliferation and migration in vitro, and tumor growth and brain metastasis in vivo. Structure-function analysis of NAT10 showed that its poorly characterized RNA helicase domain is critical for breast cancer cell growth in vitro, while its N-acetyltransferase domain is essential for primary tumor growth and brain metastasis in vivo. Integrative transcriptomic and proteomic analyses revealed key downstream effectors of NAT10, including PHGDH and PSAT1, two catalyzing enzymes for serine biosynthesis implicated in brain metastasis, and HSPA5, known to promote metastasis. We found that distant metastases of breast cancer, especially brain metastases express higher levels of NAT10, PHGDH, PSAT1, and HSPA5. Silencing PHGDH/PSAT1 or HSPA5 in metastatic breast cancer cells inhibits their ability to grow in the serine/glycine-limited condition or migrate, respectively, phenocopying the effects of NAT10 depletion. Moreover, NAT10 promotes the expression of PHGDH, PSAT1, and HSPA5 in its RNA helicase-dependent manner. These findings establish NAT10 as a master regulator of brain metastasis and shed light on the biological functions of its RNA helicase domain, nominating NAT10 as a target for treating metastatic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c4fb8d2bd13c9edb0f90c83cb38d7c4f111caf4" target='_blank'>
              An in vivo screen identifies NAT10 as a master regulator of brain metastasis
              </a>
            </td>
          <td>
            Jocelyn F Chen, Peng Xu, Wesley L Cai, Huacui Chen, E. Wingrove, Wenxue Li, Giulia Biancon, Meiling Zhang, Amer Balabaki, Ethan D Krop, Yangyi Zhang, Mingzhu Yin, T. Tebaldi, Jordan L. Meier, Thomas F. Westbrook, S. Halene, Yansheng Liu, Don X Nguyen, Q. Yan
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic cancer, one of the deadliest human malignancies, is characterized by a fibro-inflammatory tumor microenvironment and wide array of metabolic alterations. To comprehensively map metabolism in a cell type-specific manner, we harnessed a unique single-cell RNA-sequencing dataset of normal human pancreata. This was compared with human pancreatic cancer samples using a computational pipeline optimized for this study. In the cancer cells we observed enhanced biosynthetic programs. We identified downregulation of mitochondrial programs in several immune populations, relative to their normal counterparts in healthy pancreas. Although granulocytes, B cells, and CD8+ T cells all downregulated oxidative phosphorylation, the mechanisms by which this occurred were cell type specific. In fact, the expression pattern of the electron transport chain complexes was sufficient to identify immune cell types without the use of lineage markers. We also observed changes in tumor-associated macrophage (TAM) lipid metabolism, with increased expression of enzymes mediating unsaturated fatty acid synthesis and upregulation in cholesterol export. Concurrently, cancer cells exhibited upregulation of lipid/cholesterol receptor import. We thus identified a potential crosstalk whereby TAMs provide cholesterol to cancer cells. We suggest that this may be a new mechanism boosting cancer cell growth and a therapeutic target in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f304ad19d07747e1efedb9c7fa336ae96c4bd5a6" target='_blank'>
              Metabolic landscape of the healthy pancreas and pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Monica E Bonilla, Megan D Radyk, Matthew D Perricone, Ahmed M. Elhossiny, Alexis C Harold, Paola I Medina-Cabrera, Padma Kadiyala, Jiaqi Shi, Timothy L. Frankel, Eileen S Carpenter, Michael D. Green, Cristina Mitrea, C. Lyssiotis, M. Pasca di Magliano
          </td>
          <td>2024-08-13</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d5e47f21e09bedd3c2faf022dd2ac57af3858d" target='_blank'>
              Large field of view and spatial region of interest transcriptomics in fixed tissue
              </a>
            </td>
          <td>
            Xiaonan Cui, Xue Dong, Mengzhu Hu, Wenjian Zhou, Weiyang Shi
          </td>
          <td>2024-08-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related death in the US. This poor prognosis is related to the complex tumor microenvironment and tumor heterogeneity that drives resistance to chemotherapy. Here the heterogeneity of the stromal, immune, and malignant epithelial cell populations enables signaling and metabolic crosstalk mechanisms that drive immune suppression and resistance to therapy. Recently, our group unveiled that multiple distinct metabolic subclasses of cancer cells coexist within individual PDAC tumors. Among these, two PDAC clonal populations can engage in metabolic symbiosis that supports mitochondrial metabolism. One subtype has a constitutively high integrated stress response (ISR), enabling resistance to mitochondrial inhibitors. Further, these cells release asparagine that supports the mitochondrial metabolism of the other subtype that are more sensitive to mitochondrial inhibition. Given these observations, we are developing approaches to disrupt this metabolic symbiosis. Clonal PDAC cells with constitutively active ISR exhibit increased sensitivity to inhibition of the redox activity of Redox factor 1/Apurinic-apyrimidinic endonuclease 1 (Ref-1). Importantly, this can be leveraged to disrupt the support of the mitochondrial metabolism of clonal symbiotic pairs. Moreover, polyclonal PDAC cell lines representative of the heterogeneity seen in PDAC patients show a highly synergistic response to the combination of Ref-1 redox inhibitors with mitochondrial inhibitors. These data suggest this treatment combination will be synergistic in polyclonal tumors that exist in most PDAC patients. Overall, these data have the potential to lead to new avenues to directly target heterogenous populations of cancer cells to treat pancreatic cancer- an urgent clinical need.
 Citation Format: Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, Michelle Digman, Aimee Edinger, Claus Jorgensen, Melissa Fishel, Christopher J Halbrook. Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0412df26641d71ff1a5183fc1b222c74aa39cae6" target='_blank'>
              Abstract C045: Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, M. Digman, Aimee Edinger, Claus Jorgensen, M. Fishel, Christopher J. Halbrook
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55bf88cd2f361cbe0e78275013e0864970ee199" target='_blank'>
              In vivo CRISPR screens identify a dual function of MEN1 in regulating tumor–microenvironment interactions
              </a>
            </td>
          <td>
            Peiran Su, Yin Liu, Tianyi Chen, Yibo Xue, Yong Zeng, Guanghui Zhu, Sujun Chen, Mona Teng, Xinpei Ci, Mengdi Guo, Michael Y. He, Jun Hao, Vivian Chu, Wenxi Xu, Shiyan Wang, P. Mehdipour, Xin Xu, Sajid A. Marhon, F. Soares, Nhu-An Pham, Bell Xi Wu, Peter Hyunwuk Her, Shengrui Feng, Najd Alshamlan, Maryam Khalil, Rehna Krishnan, Fangyou Yu, Chang Chen, Francis Burrows, Razqallah Hakem, Mathieu Lupien, Shane Harding, Benjamin H. Lok, Catherine O'Brien, Alejandro Berlin, D. D. De Carvalho, David G Brooks, Daniel Schramek, Ming-Sound Tsao, Housheng Hansen He
          </td>
          <td>2024-09-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Although the combination of immunotherapy and radiotherapy (RT) for the treatment of malignant tumors has shown rapid development, the insight of how RT remodels the tumor microenvironment to prime antitumor immunity involves a complex interplay of cell types and signaling pathways, much of which remains to be elucidated. Four tumor samples were collected from the same abdominal wall metastasis site of the patient with gastric cancer at baseline and during fractionated RT for single-cell RNA and T-cell receptor sequencing. The Seurat analysis pipeline and immune receptor analysis were used to characterize the gastric cancer metastasis ecosystem and investigated its dynamic changes of cell proportion, cell functional profiles and cell-to-cell communication during RT. Immunohistochemical and immunofluorescent staining and bulk RNA sequencing were applied to validate the key results. We found tumor cells upregulated immune checkpoint genes in response to RT. The infiltration and clonal expansion of T lymphocytes declined within tumors undergoing irradiation. Moreover, RT led to the accumulation of proinflammatory macrophages and natural killer T cells with enhanced cytotoxic gene expression signature. In addition, subclusters of dendritic cells and endothelial cells showed decrease in the expression of antigen present features in post-RT samples. More ECM component secreted by myofibroblasts during RT. These findings indicate that RT induced the dynamics of the immune response that should be taken into consideration when designing and clinically implementing innovative multimodal cancer treatment regimens of different RT and immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523682bb135dcad73fb3cd605aea7d93bedb4b8c" target='_blank'>
              Dynamic single-cell sequencing unveils the tumor microenvironment evolution of gastric cancer abdominal wall metastases during radiotherapy.
              </a>
            </td>
          <td>
            Qianqian Mao, Zhenzhen Wu, Yonghong Lai, Ling Wang, Qiongzhi Zhao, Xi Xu, Xiansheng Lu, Wenjun Qiu, Zhi-Huan Zhang, Jiani Wu, Gaofeng Wang, R. Zhou, Jianhua Wu, Huiying Sun, N. Huang, Xiatong Huang, Luyang Jiang, Yiran Fang, Yuyun Kong, Li Liang, J. Bin, Yulin Liao, Min Shi, Wangjun Liao, D. Zeng
          </td>
          <td>2024-09-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Adenoid cystic carcinoma (ACC) is a malignant tumor primarily originating from the salivary glands, capable of affecting multiple organs. Although ACC typically exhibits slow growth, it is notorious for its propensity for neural invasion, local recurrence, and distant metastasis, making it a particularly challenging cancer to treat. The complexity of ACC's histological and molecular features poses significant challenges to current treatment modalities, which often show limited effectiveness. Recent advancements in single‐cell RNA‐sequencing (scRNA‐seq) have begun to unravel unprecedented insights into the heterogeneity and subpopulation diversity within ACC, revealing distinct cellular phenotypes and origins. This review delves into the intricate pathological and molecular characteristics of ACC, focusing on recent therapeutic advancements. We particularly emphasize the insights gained from scRNA‐seq studies that shed light on the cellular landscape of ACC, underscoring its heterogeneity and pathobiology. Moreover, by integrating analyses from public databases, this review proposes novel perspectives for advancing treatment strategies in ACC. This review contributes to the academic understanding of ACC by proposing novel therapeutic approaches informed by cutting‐edge molecular insights, paving the way for more effective, personalized therapeutic approaches for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e81e2fac0a605e61e6708a46d6b2502839fd4" target='_blank'>
              Adenoid cystic carcinoma: insights from molecular characterization and therapeutic advances
              </a>
            </td>
          <td>
            Yunxuan Jia, Yupeng Liu, Haitang Yang, Feng Yao
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a68f84c9fb8e36a5e9dc6d992ffaa775efbb85db" target='_blank'>
              SMAC mimetic drives microglia phenotype and glioblastoma immune microenvironment
              </a>
            </td>
          <td>
            E. Snacel-Fazy, A. Soubéran, M. Grange, Kevin Joseph, C. Colin, Philippe Morando, H. Luche, A. Pagano, S. Brustlein, F. Debarbieux, Soline Toutain, C. Siret, S. A. van de Pavert, G. Rougon, D. Figarella-Branger, V. Ravi, E. Tabouret, A. Tchoghandjian
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Molecular and cellular characterization of tumors is essential due to the complex and heterogeneous nature of cancer. In recent decades, many bioinformatic tools and experimental techniques have been developed to achieve personalized characterization of tumors. However, sample handling continues to be a major challenge as limitations such as prior treatments before sample acquisition, the amount of tissue obtained, transportation, or the inability to process fresh samples pose a hurdle for experimental strategies that require viable cell suspensions. Here, we present an optimized protocol that allows the recovery of highly viable cell suspensions from breast cancer primary tumor biopsies. Using these cell suspensions we have successfully characterized genome architecture through Hi-C. Also, we have evaluated single-cell gene expression and the tumor cellular microenvironment through single-cell RNAseq. Both technologies are key in the detailed and personalized molecular characterization of tumor samples. The protocol described here is a cost-effective alternative to obtain viable cell suspensions from biopsies simply and efficiently.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6963d9d1f4834d1881e0dfb691c6907687ec39cc" target='_blank'>
              Obtention of viable cell suspensions from breast cancer tumor biopsies for 3D chromatin conformation and single-cell transcriptome analysis
              </a>
            </td>
          <td>
            Aura Stephenson-Gussinye, Luis A. Rendón-Bautista, B. E. Ruiz-Medina, Eduardo Blanco-Olais, Rosario Pérez-Molina, Cleofas Marcial-Medina, Y. Chávarri-Guerra, Enrique Soto-Pérez-de-Celis, Andrea Morales-Alfaro, A. Esquivel-López, Fernando Candanedo-González, Armando Gamboa-Domínguez, Rubén Cortes-González, Alejandro Alfaro-Goldaracena, Sara E. Vázquez-Manjarrez, Guido Grajales-Figueroa, Beatriz Astudillo-Romero, Jesús Ruiz-Manriquez, A. C. Poot-Hernández, Paula Licona-Limón, Mayra Furlan-Magaril
          </td>
          <td>2024-08-22</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The concept of cancer stemness is undergoing rapid evolution, facilitated by technological advances allowing the tracking and elimination of stem cells in vivo. Following their identification in solid tumors, cancer stem cells were placed at the center of tumor initiation, growth and metastasis. However, increasing evidence suggests that stemness is a cellular state that can be acquired by differentiated cells in a process called plasticity. Here we show that CD44v6 may act as a molecular switch controlling plasticity of colorectal cancer cells. CD44v6/MET signaling controls the reappearance of Lgr5+ cells after ablation, in vitro and in vivo as demonstrated in tumor organoids derived from Lgr5DTR/eGFP mice using a CD44v6 blocking peptide. CD44v6 also affects the YAP/TAZ signaling pathway, involved in the very first steps of plasticity. In view of the essential role of plasticity for the establishment of metastases, blocking CD44v6 signaling may represent a pivotal and promising therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cb2eb85c74641a51555dede7b20eb7d8aa317e" target='_blank'>
              Breaking the cycle: How targeting CD44v6/MET signaling disrupts colorectal cancer cell plasticity
              </a>
            </td>
          <td>
            S. J. Sonnentag, S. Goldbach, D. Hoch, G. Andrieux, J. Becker, H. Fromm, L. Munoz-Sagredo, Y. Heneka, D. Grimm, F. R. Greten, V. Orian-Rousseau
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The presence of pre-existing or acquired drug-resistant cells within the tumor often leads to tumor relapse and metastasis. Single-cell RNA sequencing (scRNA-seq) enables to elucidate the subtle differences in drug responsiveness among distinct cell subpopulations within tumors. A few methods have employed scRNA-seq data to predict the drug response of individual cells to date, but their performance is far from satisfactory. In this study, we propose SSDA4Drug, a semi-supervised few-shot transfer learning method for inferring drug-resistant cancer cells. SSDA4Drug extracts pharmacogenomic features from both bulk and single-cell transcriptomic data by utilizing semi-supervised adversarial domain adaptation. This allows us to transfer knowledge of drug sensitivity from bulk-level cell lines to single cells. We conduct extensive performance evaluation experiments across multiple independent scRNA-seq datasets, and demonstrate the state-of-the-art performance of SSDA4Drug. Remarkably, with only one or two labeled target-domain samples, SSDA4Drug significantly boosts the predictive performance of single-cell drug responses. Moreover, SSDA4Drug accurately recapitulates the temporally dynamic changes of drug responses during continuous drug exposure of tumor cells, and successfully identifies reversible drug-responsive states in lung cancer cells, which initially acquired resistance through drug exposure but later restore sensitivity induced by drug holiday. Also, our predicted drug responses consistently align with the developmental patterns of drug sensitivity observed along the evolutionary trajectory of oral squamous cell carcinoma cells. In addition, our derived SHAP values and integrated gradients effectively pinpoint the key genes involved in drug resistance in prostate cancer cells. These findings highlight the exceptional performance of our method in determining single-cell drug responses. This powerful tool holds the potential for identifying drug-resistant tumor cell subpopulations, paving the way for strides in precision medicine and novel drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25abb010de95fa69c67d91f158879e6866ff5c92" target='_blank'>
              Identifying drug-resistant individual cells within tumors by semi-supervised domain adaptation
              </a>
            </td>
          <td>
            Kaishun Huang, Hui Liu
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Tumor‐associatedmacrophages (TAMs) exhibit remarkable heterogeneity in glioblastoma. Spatially resolved single‐cell transcriptomic studies identified a monocyte‐derived TAM subset localized in the peri‐necrotic niche, driven by hypoxic cues to acquire ahypoxia response signature. These hypoxia‐TAMs destabilize endothelial adherens junctions through adrenomedullin paracrine signaling, promoting the formation of hyperpermeable neovasculature that impedes drug delivery. Blocking adrenomedullin produced by hypoxia‐TAMs restores vascular integrity, increases drug deliveryinto tumors, and provides combinatorial therapeutic benefits. Here we discuss the heterogeneity of TAMs regarding functional states and locations in glioblastomas, and propose future directions for studying the temporospatial dynamics of multifaceted TAM. Highlights Single‐cell omics reveal a functionally and spatially distinct hypoxia‐TAM subset in glioblastoma. Adrenomedullin secreted by hypoxia‐TAM destabilizes tumor vasculature and its blockade enhances vessel integrity and drug delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fdc1e11579a80529898e9b0f1ec0aa652779e" target='_blank'>
              Identification of hypoxic macrophages in glioblastoma: Unveiling therapeutic insights from tumour microenvironment analysis
              </a>
            </td>
          <td>
            Zhen Qin, Xiu-Wu Bian, Yu Shi
          </td>
          <td>2024-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of spatially resolved transcriptomics (SRT) has greatly advanced our understanding of cellular microenvironments by integrating spatial information with molecular data collected from multiple tissue sections or individuals. However, methods for multi-sample spatial clustering are lacking, and existing methods primarily rely on molecular information alone. This paper introduces BayeSMART, a Bayesian statistical method designed to identify spatial domains across multiple samples. BayeSMART leverages artificial intelligence (AI)-reconstructed single-cell level information from the paired histology images of multi-sample SRT datasets while simultaneously considering the spatial context of gene expression. The AI integration enables BayeSMART to effectively interpret the spatial domains. We conducted case studies using four datasets from various tissue types and SRT platforms and compared BayeSMART with alternative multi-sample spatial clustering approaches and a number of state-of-the-art methods for single-sample SRT analysis, demonstrating that it surpasses existing methods in terms of clustering accuracy, interpretability, and computational efficiency. BayeSMART offers new insights into the spatial organization of cells in multi-sample SRT data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a670df0d414e493a2809eeaa5b1d8c9a56e43c" target='_blank'>
              BayeSMART: Bayesian Clustering of Multi-sample Spatially Resolved Transcriptomics Data
              </a>
            </td>
          <td>
            Yanghong Guo, Bencong Zhu, Chen Tang, Ruichen Rong, Ying Ma, Guanghua Xiao, Lin Xu, Qiwei Li
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Esophageal cancer (EC) is a major health issue, ranking seventh in incidence and sixth in mortality worldwide. Despite advancements in multidisciplinary treatment approaches, the 5-year survival rate for EC remains low at 21%. Challenges in EC treatment arise from late-stage diagnosis, high malignancy, and poor prognosis. Understanding the tumor microenvironment is critical, as it includes various cellular and extracellular components that influence tumor behavior and treatment response. Mast cells (MCs), as tissue-resident immune cells, play dual roles in tumor dynamics. High-throughput single-cell RNA sequencing offers a powerful tool for analyzing tumor heterogeneity and immune interactions, although its application in EC is limited.In this study, we investigated the immune microenvironment of EC using single-cell RNA sequencing and established a comprehensive immune profile. We also performed analysis of upstream transcription factors and downstream pathway enrichment to further comprehensively decipher MCs in EC. Besides, we performed knockdown experiments to explore the role of epidermal growth factor receptor (EGFR) signaling pathway in MCs-tumor cell interactions, highlighting its potential as a prognostic marker. Finally, we constructed a prognostic model for EC, which provided valuable suggestions for the diagnosis and prognosis of EC.Our analysis identified 11 major cell types, of which MCs were particularly present in pericarcinoma tissues. Further grouping of the 5,001 MCs identified 8 distinct subtypes, including SRSF7-highly expressed MCs, which showed strong tumor preference and potential tumor-promoting properties. Moreover, we identified the key signaling receptor EGFR and validated it by in vitro knockdown experiments, demonstrating its cancer-promoting effects. In addition, we established an independent prognostic indicator, SRSF7+ MCs risk score (SMRS), which showed a correlation between high SMRS group and poor prognosis.These findings illuminate the complex interactions within the tumor microenvironment of EC and suggest that targeting specific MCs subtypes, particularly via the EGFR signaling pathway, may present novel therapeutic strategies. This study establishes a comprehensive immune map of EC, offering insights for improved treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b52619ad1abc5e3af051c9a613c06b47617866c" target='_blank'>
              Unraveling the ecological landscape of mast cells in esophageal cancer through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Shengyi Zhang, Xinyi Zhang, Zhikai Xiahou, Shunqing Zuo, Jialong Xue, Yi Zhang
          </td>
          <td>2024-10-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>2</td>
          <td>39</td>
        </tr>

        <tr id="Background: Gastrointestinal cancer cells display both morphology and physiology diversity, thus posing a significant challenge for precise representation by a single data model. We conducted an in-depth study of gastrointestinal cancer heterogeneity by integrating and analyzing data from multiple modalities. Methods: We used a modified Canny algorithm to identify edges from tumor images, capturing intricate nonlinear interactions between pixels. These edge features were then combined with differentially expressed mRNA, miRNA, and immune cell data. Before data integration, we used the K-medoids algorithm to pre-cluster individual data types. The results of pre-clustering were used to construct the kernel matrix. Finally, we applied spectral clustering to the fusion matrix to identify different tumor subtypes. Furthermore, we identified hub genes linked to these subtypes and their biological roles through the application of Weighted Gene Co-expression Network Analysis (WGCNA) and Gene Ontology (GO) enrichment analysis. Results: Our investigation categorized patients into three distinct tumor subtypes and pinpointed hub genes associated with each. Genes MAGI2-AS3, MALAT1, and SPARC were identified as having a differential impact on the metastatic and invasive capabilities of cancer cells. Conclusion: By harnessing multimodal features, our study enhances the understanding of gastrointestinal tumor heterogeneity and identifies biomarkers for personalized medicine and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c68f2f427231b32eddf4811a28d710e5bb1cf" target='_blank'>
              Tumor Heterogeneity in Gastrointestinal Cancer Based on Multimodal Data Analysis
              </a>
            </td>
          <td>
            Dongmei Ai, Yang Du, Hongyu Duan, Juan Qi, Yuduo Wang
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="Gallbladder carcinoma (GBC) is a malignant hepatobiliary cancer characterized by an intricate tumor microenvironments (TME) and heterogeneity. The traditional GBC 2D culture models cannot faithfully recapitulate the characteristics of the TME. Three-dimensional (3D) bioprinting enables the establishment of high-throughput and high-fidelity multicellular GBC models. In this study, we designed a concentric cylindrical tetra-culture model to reconstitute the spatial distribution of cells in tumor tissue, with the inner portion containing GBC cells, and the outer ring containing a mixture of endothelial cells, fibroblasts, and macrophages. We confirmed the survival, proliferation, biomarker expression and gene expression profiles of GBC 3D tetra-culture models. Hematoxylin-eosin (HE) and immunofluorescence staining verified the morphology and robust expression of GBC/endothelial/fibroblast/macrophage biomarkers in GBC 3D tetra-culture models. Single-cell RNA sequencing revealed two distinct subtypes of GBC cells within the model, glandular epithelial and squamous epithelial cells, suggesting the mimicry of intratumoral heterogeneity. Comparative transcriptome profile analysis among various in vitro models revealed that cellular interactions and the TME in 3D tetra-culture models reshaped the biological processes of tumor cells to a more aggressive phenotype. GBC 3D tetra-culture models restored the characteristics of the TME as well as intratumoral heterogeneity. Therefore, this model is expected to have future applications in tumor biology research and antitumor drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc6d2cc903e389bd57ed19225ad293a9f8472b5b" target='_blank'>
              Multicellular 3D bioprinted human gallbladder carcinoma forin vitromimicry of tumor microenvironment and intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Yukai Jin, Jiangang Zhang, Jiali Xing, Yiran Li, Huiyu Yang, Liujian Ouyang, Zhiyuan Fang, Lejia Sun, Bao Jin, Pengyu Huang, Huayu Yang, Shunda Du, X. Sang, Yilei Mao
          </td>
          <td>2024-08-09</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background The rapid advancement of single-cell transcriptomic technologies has led to the curation of millions of cellular profiles, providing unprecedented insights into cellular heterogeneity across various tissues and developmental stages. This growing wealth of data presents an opportunity to uncover complex gene-gene relationships, yet also poses significant computational challenges. Results We present scEMB, a transformer-based deep learning model developed to capture context-aware gene embeddings from large-scale single-cell transcriptomics data. Trained on over 30 million single-cell transcriptomes, scEMB utilizes an innovative binning strategy that integrates data across multiple platforms, effectively preserving both gene expression hierarchies and cell-type specificity. In downstream tasks such as batch integration, clustering, and cell type annotation, scEMB demonstrates superior performance compared to existing models like scGPT and Geneformer. Notably, scEMB excels in silico correlation analysis, accurately predicting gene perturbation effects in CRISPR-edited datasets and microglia state transition, identifying a few known Alzheimer’s disease (AD) risks genes in top gene list. Additionally, scEMB offers robust fine-tuning capabilities for domain-specific applications, making it a versatile tool for tackling diverse biological problems such as therapeutic target discovery and disease modeling. Conclusions scEMB represents a powerful tool for extracting biologically meaningful insights from complex gene expression data. Its ability to model in silico perturbation effects and conduct correlation analyses in the embedding space highlights its potential to accelerate discoveries in precision medicine and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd379bbf385fbba692dbb10ae4d409bfe7e9c710" target='_blank'>
              scEMB: Learning context representation of genes based on large-scale single-cell transcriptomics
              </a>
            </td>
          <td>
            k.-l. Hsieh, Y. Chu, Xiaoyang Li, P. Pilié, Yulin Dai
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a7e2a6359020fadce11d67094c49cba1e4ed0b" target='_blank'>
              Multi-scale transcriptomics reveals that specific tumor cells promote lung adenocarcinoma metastasis through crosstalk with the microenvironment
              </a>
            </td>
          <td>
            Qi Cai, Lin Shi, Meng-ya Zhang, Peng Chen
          </td>
          <td>2024-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiform (GBM), a grade IV glioma, is the most common and aggressive cancer in the central nervous system. Current treatment for GBM includes surgical resection, radiation, and the frontline DNA alkylating drug, temozolomide (TMZ). The current median survival for GBM patients is about 14.5 months with 5% patients surviving up to 5 years. We propose that discerning distinct subsets within heterogeneous GBM will provide avenues for research to improve new therapies. We used different methods to isolate GBM cell subsets. These include stable transfectants of GBM cell lines with a lentiviral system in which green fluorescence protein (GFP) is regulated with tandem repeats of Oct4a and Sox2 response elements. Parallel studies with a plasmid using the full-length regulatory region of Oct4a indicated reduced efficiency in separating cell subsets, relative to SORE6-GFP lentivirus. Stem cell-linked gene expressions and function studies such as ALDH1, tumorsphere and in vivo passaging of GFP hi subsets confirmed the presence of cancer stem cells (CSCs). We also studied a more efficient method that could be relevant for primary GBM cells. We selected tumorspheres by plating heterogeneous GBM cells and then serially passaged the spheres. Studies for stem cell genes indicated that this method could be used for primary GBM cells. Overall, this study provided insights into methods to isolate GBM subsets, including primary GBM cells. The advantages of the methods are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e350655b514b418d1c77aa6fb2234fa8abba3" target='_blank'>
              A Platform to Establish a Working Hierarchy of Glioblastoma Multiforme Cells:
              </a>
            </td>
          <td>
            V. H. Savanur, Anushka Sarkar, Andrew Petryna, Ky Nguyen, Jesus Benites-Sandoval, Marina Gergues, Arash Hatefi, P. Rameshwar
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Translational Science</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c83627f2d1b572687cd5eed49bd4a2bf289b0" target='_blank'>
              Application of a single-cell-RNA-based biological-inspired graph neural network in diagnosis of primary liver tumors
              </a>
            </td>
          <td>
            Dao-Han Zhang, Chen Liang, Shu-Yang Hu, Xiao-yong Huang, Lei Yu, Xiangjing Meng, Xiaoen Guo, , Zhen Chen, Lv Zhang, Yan-Zi Pei, Mu Ye, , , Ying-Hong Shi, A. Ke, Yi Chen, Yuan Ji, Yujiang Geno Shi, Jian Zhou, Jia Fan, Guo-huan Yang, Qindong Sun, G. Shi, Jia-Cheng Lu
          </td>
          <td>2024-10-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) is a deadly cancer with poor response to targeted therapy, largely driven by an immunosuppressive tumor microenvironment (TME). Here we examine the immune-modulatory role of the receptor tyrosine kinase EphA4 in HNSCC progression. Within the TME, EphA4 is primarily expressed on regulatory T cells (Tregs) and macrophages. In contrast ephrinB2, an activating ligand of EphA4, is expressed in tumor blood vessels. Using genetically engineered mouse models, we show that EphA4 expressed in Tregs promotes tumor growth, whereas EphA4 expressed in monocytes inhibits tumor growth. In contrast, ephrinB2 knockout in blood vessels reduces both intratumoral Tregs and macrophages. A novel specific EphA4 inhibitor, APY-d3-PEG4, reverses the accelerated tumor growth we had previously reported with EphB4 cancer cell knockout. EphA4 knockout in macrophages not only enhanced their differentiation into M2 macrophage but also increased Treg suppressive activity. APY-d3-PEG4 reversed the accelerated growth seen in the EphA4 knockout of monocytes but conferred no additional benefit when EphA4 was knocked out on Tregs. Underscoring an EphA4-mediated interplay between Tregs and macrophages, we found that knockout of EphA4 in Tregs not only decreases their activation but also reduces tumor infiltration of pro-tumoral M2 macrophages. These data identify Tregs as a primary target of APY-d3-PEG4 and suggest a role for Tregs in regulating macrophage conversion. These data also support the possible anti-cancer therapeutic value of bispecific peptides or antibodies capable of promoting EphA4 blockade in Tregs but not macrophages. Significance EphA4 in regulatory T cells has a pro-tumoral effect while EphA4 in macrophages plays an anti-tumoral role underscoring the necessity of developing biologically rational therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99406a05b43999ff5eb6dc79db7091b1d866ebc8" target='_blank'>
              The pro-tumoral and anti-tumoral roles of EphA4 on T regulatory cells and tumor associated macrophages during HNSCC tumor progression
              </a>
            </td>
          <td>
            S. Corbo, D. Nguyen, Shilpa Bhatia, L. Darragh, K. Abdelazeem, B. V. Court, Nicholas A. Olimpo, J. Gadwa, Justin Yu, Chloe Hodgson, Von Samedi, Enrique Sanz Garcia, Lillian Siu, Anthony J. Saviola, Lynn E. Heasley, M. Knitz, Elena B. Pasquale, S. D. Karam
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Alpha-smooth muscle actin (α-SMA) expression in the stroma is linked to the presence of cancer-associated fibroblasts and is known to correlate with worse outcomes in various tumors. In this study, using a GeoMx digital spatial profiling approach, we characterized the gene expression of the tumor and α-SMA-expressing stromal cell compartments in pancreatic neuroendocrine tumors (PanNETs). The profiling was performed on tissues from eight retrospective cases (three grade 1, four grade 2, and one grade 3). Selected regions of interest were segmented geometrically based on tissue morphology and fluorescent signals from synaptophysin and α-SMA markers. The α-SMA-expressing stromal-cell-associated genes were involved in pathways of extracellular matrix modification, whereas, in tumor cells, the gene expression profiles were associated with pathways involved in cell proliferation. The comparison of gene expression profiles across all three PanNET grades revealed that the differences between grades are not only present at the level of the tumor but also in the α-SMA-expressing stromal cells. Furthermore, the tumor cells from regions with a rich presence of adjacent α-SMA-expressing stromal cells revealed an upregulation of matrix metalloproteinase-9 (MMP9) expression in grade 3 tumors. This study provides an in-depth characterization of gene expression profiles in α-SMA-expressing stromal and tumor cells, and outlines potential crosstalk mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fe06c12f5865d2f0fe11fa3ad5ad66746e58fe" target='_blank'>
              Tumor and α-SMA-expressing stromal cells in pancreatic neuroendocrine tumors have a distinct RNA profile depending on tumor grade.
              </a>
            </td>
          <td>
            H. Niedra, R. Peculis, Rihards Saksis, Ilona Mandrika, Sofija Vilisova, Jurijs Nazarovs, Austra Breikša, Aija Gerina, Julie Earl, Ignacio Ruz-Caracuel, M. Rosas, A. Puķītis, Natalja Senterjakova, V. Rovite
          </td>
          <td>2024-09-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The bone marrow microenvironment is critical for B-cell acute lymphoblastic leukemia (B-ALL) but its cellular heterogeneity remains poorly defined. Here, we employed single-cell RNA sequencing to comprehensively characterize the stromal and hematopoietic niches in pediatric B-ALL. Our analysis revealed two distinct mesenchymal stromal cell (MSC) populations as primary leukemia-supportive niches: early mesenchymal progenitors and adipogenic progenitors. Single-cell transcriptomic analysis infers that ALL blasts use distinct cell-cell interactions to communicate with the different stromal populations. Purified adipogenic progenitors from the bone of children with ALL support survival of the leukemic blasts ex vivo and their signature is enriched in relapse samples. Our data establish adipogenic progenitors as a distinct and novel component of the ALL niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f24fbb432c05f1e5ee3ed61b3b778c9a293df20" target='_blank'>
              Distinct Stromal Cell Populations Define the B-cell Acute Lymphoblastic Leukemia Microenvironment
              </a>
            </td>
          <td>
            Mauricio N. Ferrao Blanco, Bexultan Kazybay, M. Belderbos, Olaf Heidenreich, H. J. Vormoor
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Disruption of processes involved in tissue development and homeostatic self-renewal is increasingly implicated in cancer initiation, progression, and recurrence. The adrenal cortex is a dynamic tissue that undergoes life-long turnover. Here, using genetic fate mapping and murine adrenocortical carcinoma (ACC) models, we have identified a population of adrenocortical stem cells that express delta-like non-canonical Notch ligand 1 (DLK1). These cells are active during development, near dormant postnatally but are re-expressed in ACC. In a study of over 200 human ACC samples, we have shown DLK1 expression is ubiquitous and is an independent prognostic marker of recurrence-free survival. Paradoxically, despite its progenitor role, spatial transcriptomic analysis has identified DLK1 expressing cell populations to have increased steroidogenic potential in human ACC, a finding also observed in four human and one murine ACC cell lines. Finally, the cleavable DLK1 ectodomain is measurable in patients’ serum and can discriminate between ACC and other adrenal pathologies with high sensitivity and specificity to aid in diagnosis and follow-up of ACC patients. These data demonstrate a prognostic role for DLK1 in ACC, detail its hierarchical expression in homeostasis and oncogenic transformation and propose a role for its use as a biomarker in this malignancy. Graphical abstract Statement of significance This study presents DLK1 as a novel biomarker in ACC with opportunities for use in the diagnosis, prognosis and longitudinal follow up of patients. DLK1, a marker of adrenocortical stem cells, is re-expressed in ACC, is measurable in patients’ serum and is associated with increased malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9baab22c63f6f34cafaad1ee20a4c197797bfaf" target='_blank'>
              Dlk1 is a novel adrenocortical stem/progenitor cell marker that predicts malignancy in adrenocortical carcinoma
              </a>
            </td>
          <td>
            Katia Mariniello, J. Pittaway, B. Altieri, Kleiton S. Borges, Irene Hadjidemetriou, Claudio Ribeiro, G. Ruiz-Babot, Jiang A. Lim, Julie Foster, Julie Cleaver, Jane Sosabowski, Nafis Rahman, Milena Doroszko, C. Hantel, Sandra Sigala, A. Abate, M. Tamburello, Katja Kiseljak-Vassiliades, Margaret Wierman, L. Parvanta, Tarek E. Abdel-Aziz, Teng-Teng Chung, Aimee Di Marco, F. Palazzo, Celso E Gomez-Sanchez, David R. Taylor, Oliver Rayner, C. Ronchi, Carles Gaston-Massuet, S. Sbiera, William M. Drake, Emanuel Rognoni, Matthias Kroiss, D. T. Breault, M. Fassnacht, L. Guasti
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Tumor initiation and progression rely on intricate cellular pathways that promote proliferation while suppressing differentiation, yet the importance of pathways inhibiting differentiation in cancer remains incompletely understood. Here, we reveal a novel mechanism centered on the repression of the neuronal-specific transcription factor ARNT2 by the MYC oncogene that governs the balance between proliferation and differentiation. We found that MYC coordinates the transcriptional repression of ARNT2 through the activity of polycomb repressive complex 2 (PRC2). Notably, ARNT2, highly and specifically expressed in the central nervous system, is diminished in glioblastoma, inversely correlating with patient survival. Utilizing in vitro and in vivo models, we demonstrate that ARNT2 knockout (KO) exerts no discernible effect on the in vitro proliferation of glioblastoma cells, but significantly enhances the growth of glioblastoma cells in vivo. Conversely, ARNT2 overexpression severely dampens the growth of fully transformed glioblastoma cells subcutaneously or orthotopically xenografted in mice. Mechanistically, ARNT2 depletion diminishes differentiation and enhances stemness of glioblastoma cells. Our findings provide new insights into the complex mechanisms used by oncogenes to limit differentiation in cancer cells and define ARNT2 as a tumor suppressor in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fc2b3a844fbe1cf3418432eb71140217e74feb" target='_blank'>
              MYC-Mediated Inhibition of ARNT2 Uncovers a Key Tumor Suppressor in Glioblastoma
              </a>
            </td>
          <td>
            Maralice Conacci-Sorrell, Yi-Heng Hao, N. Borenstein-Auerbach, Anthony Grichuk, Li Li, M. Lafita-Navarro, Shun Fang, Pedro A S Nogueira, Jiwoong Kim, Lin Xu, J. Shay
          </td>
          <td>2024-08-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a31875ebe3aec993187c2c6c6d773a5efac6f4" target='_blank'>
              Human intermediate prostate cancer stem cells contribute to the initiation and development of prostate adenocarcinoma
              </a>
            </td>
          <td>
            Jie Mu, Ruizhi Li, Yu Zheng, Yi Lu, Lei Ma, Lin Yin, Miao Zhang, Wenyu Ma, Mengjia Chang, Aihua Liu, Jing Li, Haihui Zhu, Dong Wang
          </td>
          <td>2024-09-11</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial techniques such as spatial transcriptomes and MALDI-MSI, offering insights into both transcripts and metabolite of tissue sections. However, integrating them with high accuracy is challenge due to no shared spots or features. We present haCCA, a workflow designed to integrate spatial transcriptomes and metabolomes data using high-correlated feature pairs and modified spatial morphological alignment. This approach ensures high-resolution and accurate spot-to-spot data integration across neighbor tissue section. We applied haCCA to both publicly available 10X Visium and MALDI-MSI datasets from mouse brain tissue and a custom spatial transcriptome and MALDI-MSI dataset from an intrahepatic cholangiocarcinoma (ICC) model, exploring the metabolic alteration of NETs(neutrophil extracellular traps) on ICC, and finding a potential mechanism that NETs upregulated Scd1 to activate fatty acid metabolism. Providing new insights into the dynamic crosstalk between genes and metabolites that regulates the tumor biological behavior and drives the response to treatment. We developed and published an easy-to-use Python package to facilitate its use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a471f5555baf62c3ebcc010fc547002f0b3c386" target='_blank'>
              haCCA: Multi-module Integrating of spatial transcriptomes and metabolomes
              </a>
            </td>
          <td>
            Xiao-Tian Shen, Jing Xu, Chen Zhang, Xiaoyun Zhang, Zhou-Qing Chen, Hu-Liang Jiang, Lu-Yu Yang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, J. Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, Brendan C Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Y. Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Solid tumours exhibit a well‐defined architecture, comprising a differentiated core and a dynamic border that interfaces with the surrounding tissue. This border, characterised by distinct cellular morphology and molecular composition, serves as a critical determinant of the tumour's invasive behaviour. Notably, the invasive border of the primary tumour represents the principal site for intravasation of metastatic cells. These cells, known as circulating tumour cells (CTCs), function as ‘seeds’ for distant dissemination and display remarkable heterogeneity. Advancements in spatial sequencing technology are progressively unveiling the spatial biological features of tumours. However, systematic investigations specifically targeting the characteristics of the tumour border remain scarce. In this comprehensive review, we illuminate key biological insights along the tumour body‐border‐haematogenous metastasis axis over the past five years. We delineate the distinctive landscape of tumour invasion boundaries and delve into the intricate heterogeneity and phenotype of CTCs, which orchestrate haematogenous metastasis. These insights have the potential to explain the basis of tumour invasion and distant metastasis, offering new perspectives for the development of more complex and precise clinical interventions and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f6a6ffd5124276ada6cdb3c914a272e4284a04" target='_blank'>
              Single‐cell and spatial omics unravel the spatiotemporal biology of tumour border invasion and haematogenous metastasis
              </a>
            </td>
          <td>
            Xifu Cheng, Yuke Cao, Xiangyi Liu, Yuanheng Li, Qing Li, Dian Gao, Qiongfang Yu
          </td>
          <td>2024-09-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Hematopoietic stem and progenitor cells (HSPCs) rely on intercellular signaling to maintain and adjust their production of blood and immune cells. This process occurs in the semi-fluidic bone marrow, hosting dozens of cell types that constantly migrate and interact. To elucidate the dynamic network of cell-cell interaction and signaling transduction underlying hematopoiesis, we developed an algorithm to measure Cell-cell Spatial Interaction Probability (CellSIP) by integrating data on ligand and receptor expression, cell type abundance, and cellular spatial positioning. Using new and published mouse datasets, we validated CellSIP and uncovered signaling transductions indicating feedback mechanisms underlying hematopoiesis. Moreover, we identified significant correlations between signaling pathways across individual HSPCs at the same hematopoiesis stage. These pathway correlations illuminate the organization of cellular and signaling networks underlying hematopoiesis, revealing new regulators through their associations with established ones. The signaling quantification and correlation data are available through the Hematopoiesis Intercellular Signaling Explorer (HISE).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf26ff56c34b3ddada82e3d4951fade44feb8315" target='_blank'>
              Cellular and molecular networks of intercellular signaling in bone marrow hematopoiesis
              </a>
            </td>
          <td>
            Zachary Thomas, Bowen Wang, Wade R. Boohar, Mary Vergel, Jiya Eerdeng, Dayeon J. Shon, M. Elowitz, Rong Lu
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e014c815070452f107fe9ff31d1f8aa2306a82" target='_blank'>
              Molecular and functional landscape of malignant serous effusions for precision oncology
              </a>
            </td>
          <td>
            R. Wegmann, L. Bankel, Y. Festl, Kate Lau, Sohyon Lee, F. Arnold, V. Cappelletti, Aaron Fehr, Paola Picotti, Konstantin J Dedes, Daniel Franzen, D. Lenggenhager, Peter K. Bode, M. Zoche, H. Moch, Christian Britschgi, B. Snijder
          </td>
          <td>2024-10-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is a technique of measuring gene expression with spatial resolution on a tissue slide, which is especially useful in cancer research. This scoping study reviews 41 articles published by the end of 2023 to examine current trends for employed methods, applications, and data analysis approaches for spatial transcriptomics, identifying challenges and practical uses of this technique in studying cancer. The research shows that cancer is a major focus in spatial transcriptomics studies, with certain tools and methods being particularly popular among scientists. However, many studies do not fully explain their data processing methods, making it hard to reproduce their results. Emphasis on transparent sharing of analysis scripts and tailored single-cell analysis methods for ST is recommended to enhance study reproducibility and reliability in the domain. This work may stand as a spatial transcriptomics developmental snapshot and reference to contemporary neoplasm biology research conducted through this technique. Abstract Spatial transcriptomics (ST) examines gene expression within its spatial context on tissue, linking morphology and function. Advances in ST resolution and throughput have led to an increase in scientific interest, notably in cancer research. This scoping study reviews the challenges and practical applications of ST, summarizing current methods, trends, and data analysis techniques for ST in neoplasm research. We analyzed 41 articles published by the end of 2023 alongside public data repositories. The findings indicate cancer biology is an important focus of ST research, with a rising number of studies each year. Visium (10x Genomics, Pleasanton, CA, USA) is the leading ST platform, and SCTransform from Seurat R library is the preferred method for data normalization and integration. Many studies incorporate additional data types like single-cell sequencing and immunohistochemistry. Common ST applications include discovering the composition and function of tumor tissues in the context of their heterogeneity, characterizing the tumor microenvironment, or identifying interactions between cells, including spatial patterns of expression and co-occurrence. However, nearly half of the studies lacked comprehensive data processing protocols, hindering their reproducibility. By recommending greater transparency in sharing analysis methods and adapting single-cell analysis techniques with caution, this review aims to improve the reproducibility and reliability of future studies in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546384150762685b69a172f984b5438d683edf6a" target='_blank'>
              Scoping Review: Methods and Applications of Spatial Transcriptomics in Tumor Research
              </a>
            </td>
          <td>
            Kacper Maciejewski, Patrycja Czerwińska
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer- related deaths in the United States and is often not diagnosed until it has already metastasized. Our prior work established that calcium signaling pathways facilitate a pro-metastatic phenotype through the induction of an epithelial-to-mesenchymal transition (EMT) program in PDAC tumor cells, but the downstream signaling molecules mediating this phenotype have remained elusive. MethodsUsing a novel genetically engineered model of pancreatic cancer, we demonstrate that loss of Calcium/calmodulin-dependent kinase-II B (CaMK2B), a downstream effector kinase activated by calcium, enhances cellular plasticity, and increases metastasis. To this end, we generated a genetically engineered mouse model where Camk2b was knocked out in tumor cells in a lineage-labeled Kras- and p53-driven PDAC mouse model (herein KPCY-Camk2bcKO). Results Genetic deletion of Camk2b in the KPCY genetically engineered model accelerated tumor formation and dramatically reduced the overall median survival of KPCY- Camk2b cKO mice (78 days) relative to KPCY controls (136 days). Pathological and immunofluorescence analysis of KPCY-Camk2b cKO animals showed an increased number of poorly differentiated tumor regions and a higher metastatic burden, with >85% of KPCY- Camk2b cKO animals presenting with multi-focal metastases. Further analysis of tumor cells in KPCY-Camk2b cKO mice showed enhanced expression of markers of the highly aggressive squamous PDAC subtypes. Strikingly, Camk2b deletion combined with oncogenic Kras G12D (in the absence of p53 alterations), was sufficient to form lethal metastatic PDAC tumors, while age-matched control Kras G12D tumors with intact Camk2b only progressed to the acinar-to-ductal metaplasia (ADM) or Pancreatic Intraepithelial Neoplasia (PanIN) stage. Collectively, these results establish Camk2b loss as a new oncogenic and pro-metastatic mechanism in PDAC Conclusion This work demonstrates that Camk2b loss enhances tumor cell evolution and plasticity within PDAC to promote metastasis. In this context, we speculate that loss of Camk2b rewires calcium signaling to drive an aggressive subtype of PDAC with increased metastasis.
 Citation Format: Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason Pitarresi. Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7848880ff2d74aa871376f23a8aeef10bd0463c1" target='_blank'>
              Abstract C061: Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC
              </a>
            </td>
          <td>
            Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason R. Pitarresi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune evasion is a key phenomenon in understanding tumor recurrence, metastasis, and other critical steps in tumor progression. The tumor microenvironment (TME) is in constant flux due to the tumor’s ability to release signals that affect it, while immune cells within it can impact cancer cell behavior. Cancer cells undergo several changes, which can change the enrichment of different immune cells and modulate the activity of existing immune cells in the tumor microenvironment. Cancer cells can evade immune surveillance by downregulating antigen presentation or expressing immune checkpoint molecules. High levels of tumor-infiltrating lymphocytes (TILs) correlate with better outcomes, and robust immune responses can control tumor growth. On the contrary, increased enrichment of Tregs, myeloid-derived suppressor cells, and M2-like anti-inflammatory macrophages can hinder effective immune surveillance and predict poor prognosis. Overall, understanding these immune evasion mechanisms guides therapeutic strategies. Researchers aim to modulate the TME to enhance immune surveillance and improve patient outcomes. In this review article, we strive to summarize the composition of the tumor immune microenvironment, factors affecting the tumor immune microenvironment (TIME), and different therapeutic modalities targeting the immune cells. This review is a first-hand reference to understand the basics of immune surveillance and immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/906f18b6d340700979090863a508c55f86aa5360" target='_blank'>
              Cracking the Codes behind Cancer Cells’ Immune Evasion
              </a>
            </td>
          <td>
            Nikita Mundhara, Pritam Sadhukhan
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cbe85e7267d943e47fa9cdeee4b8e805b7a651" target='_blank'>
              CD24 flags anastasis in melanoma cells.
              </a>
            </td>
          <td>
            Martina H Vasileva, Anette Bennemann, Karolin Zachmann, Michael P Schön, Jorge Frank, Vijay Kumar Ulaganathan
          </td>
          <td>2024-08-13</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endometriosis, affecting 1 in 9 women, presents treatment and diagnostic challenges. To address these issues, we generated the biggest single-cell atlas of endometrial tissue to date, comprising 466,371 cells from 35 endometriosis and 25 non-endometriosis patients without exogenous hormonal treatment. Detailed analysis reveals significant gene expression changes and altered receptor-ligand interactions present in the endometrium of endometriosis patients, including increased inflammation, adhesion, proliferation, cell survival, and angiogenesis in various cell types. These alterations may enhance endometriosis lesion formation and offer novel therapeutic targets. Using ScaiVision, we developed neural network models predicting endometriosis of varying disease severity (median AUC = 0.83), including an 11-gene signature-based model (median AUC = 0.83) for hypothesis-generation without external validation. In conclusion, our findings illuminate numerous pathway and ligand-receptor changes in the endometrium of endometriosis patients, offering insights into pathophysiology, targets for novel treatments, and diagnostic models for enhanced outcomes in endometriosis management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9557348ef1327c4800ef29c1c6e1843f22fe83" target='_blank'>
              Tracing Endometriosis: Coupling deeply phenotyped, single-cell based Endometrial Differences and AI for disease pathology and prediction
              </a>
            </td>
          <td>
            Lea Duempelmann, Shaoline Sheppard, Angelo Duo, Jitka Skrabalova, Brett McKinnon, Thomas Andrieu, Dennis Goehlsdorf, S. Muzumdar, Cinzia Donato, Ryan Lusby, Wiebke Solass, Hans Bösmüller, Peter Nestorov, Michael D. Mueller
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Prostate cancer has substantial heterogeneity in clinical outcomes and therapeutic responses, posing challenges in predicting disease progression and tailoring treatment strategies. Recent studies have highlighted the potential prognostic value of evaluating the tumor microenvironment, including the presence of a histologically overt stromal response (HOST-response) characterized by peri-glandular stromal changes and architectural distortions. This retrospective study examined patient records from The Cancer Genome Atlas database to identify genomic alterations associated with the HOST-response in prostate cancer. Among 348 patients who underwent radical prostatectomy, 160 (45.98%) were identified as having a HOST-response. A gene expression analysis revealed 1263 genes with significantly higher expression in patients with a HOST-response. A protein–protein interaction network analysis identified seven hub genes (KIF2C, CENPA, CDC20, UBE2C, ESPL1, KIF23, and PLK1) highly interconnected in the network. A functional enrichment analysis revealed alterations in the cell division, cytoskeletal organization, cytokinesis, and interleukin-16 signaling pathways in patients with a HOST-response, suggesting dysregulated proliferation and inflammation. The distinct molecular signature associated with the HOST-response provides insights into the tumor–stroma interactions driving adverse outcomes and potential targets for tailored therapeutic interventions in this subset of patients with prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83f4d281e62fad2c54da06423aa969b3ab8b987f" target='_blank'>
              Molecular Alterations Associated with Histologically Overt Stromal Response in Patients with Prostate Cancer
              </a>
            </td>
          <td>
            M. Sayan, Y. Tuaç, Mahmut Akgul, Samet Kucukcolak, Elza Tjio, Dilara Akbulut, Luke W. Chen, David D. Yang, Shalini Moningi, J. E. Leeman, P. Orio, Paul L. Nguyen, Anthony V. D'Amico, Ç. Aktan
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Cancer-Associated Fibroblasts (CAFs) constitute a heterogeneous group of cells critical for the remodeling of the tumor microenvironment (TME). Given their significant impact on tumor progression, particularly in skin cancers, a deeper understanding of their characteristics and functions is essential.


METHODS
This study employed a single-cell transcriptomic analysis to explore the diversity of CAFs within three major types of skin cancer: basal cell carcinoma, melanoma, and head and neck squamous cell carcinoma. We applied analytical techniques, including Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), pseudotime tracking, metabolic profiling, and stemness assessment to delineate and define the functional attributes of identified CAF subgroups.


RESULTS
Our analysis successfully delineated nine distinct CAF subgroups across the studied tumor types. Of particular interest, we identified a novel CAF subtype, designated as C0, exclusive to basal cell carcinoma. This subtype exhibits phenotypic traits associated with invasive and destructive capabilities, significantly correlating with the progression of basal cell carcinoma. The identification of this subgroup provides new insights into the role of CAFs in cancer biology and opens avenues for targeted therapeutic strategies.


CONCLUSION
A pan-cancer analysis was performed on three cancers, BCC, MA, and HNSCC, focusing on tumor fibroblasts in TME. Unsupervised clustering categorized CAF into nine subpopulations, among which the C0 subpopulation had a strong correspondence with BCC-CAF and an invasive- destructive-related phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ae0c3c55b73c69971a741aaa77cb534354a3" target='_blank'>
              Pan-Cancer Single-Cell Analysis Revealing the Heterogeneity of Cancer-Associated Fibroblasts in Skin Tumors.
              </a>
            </td>
          <td>
            Yichi Zhang, Zhijie Zhao, Wenyi Huang, Byeong Seop Kim, Li Lin, Xin Li, Mengyuan Hou, Li Li, Yan Zhang, Wenjing Xi, G. Chai
          </td>
          <td>2024-09-25</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7102230cf5cc863ea65953d891599cf63c334991" target='_blank'>
              Profiling cell identity and tissue architecture with single-cell and spatial transcriptomics.
              </a>
            </td>
          <td>
            G. Gulati, Jeremy Philip D'Silva, Yunhe Liu, Linghua Wang, Aaron M. Newman
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344cd443c78245744a77b99c30e0d32165037249" target='_blank'>
              Mesenchymal stem cells in tumor microenvironment: drivers of bladder cancer progression through mitochondrial dynamics and energy production
              </a>
            </td>
          <td>
            Enguang Yang, Suoshi Jing, Fang Wang, Hanzhang Wang, Shengjun Fu, Li Yang, Junqiang Tian, Dragan J Golijanin, Wafik S. El-Deiry, Liang Cheng, Zhiping Wang
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8018e573a8779af2c11f1aae1832a3b88d066791" target='_blank'>
              Targeting SWI/SNF ATPases reduces neuroblastoma cell plasticity.
              </a>
            </td>
          <td>
            Man Xu, Jason J Hong, Xiyuan Zhang, Ming Sun, Xingyu Liu, Jeeyoun Kang, Hannah Stack, Wendy Fang, Haiyan Lei, Xavier Lacoste, Reona Okada, Raina Jung, Rosa Nguyen, J. Shern, Carol J Thiele, Zhihui Liu
          </td>
          <td>2024-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic cancer is a highly lethal and aggressive disease, commonly characterized by a unique tumor microenvironment, where mutant KRAS signaling plays a pivotal role in its pathophysiology. To deepen our understanding of KRAS-dependent signaling in earliest tumor development, oncogenic KRAS (KRASG12D) expression was induced in human stem cell-derived pancreatic ductal-like organoids (PDLOs), followed by time-resolved single-cell RNA and bulk ATAC sequencing. This unique precancerous state model facilitated a comprehensive characterization of early oncogenic events at cellular resolution. Furthermore, a machine learning approach was applied to accurately predict KRAS-dependent cell identities. Within seven days, mutated KRAS triggers distinct gene signature programs related to extracellular matrix remodeling and inflammation. Although derived from dysplastic yet non-cancerous states, these gene signatures negatively correlated with overall survival in PDAC (pancreatic ductal adenocarcinoma) patients. The KRAS-dependent signatures retrieved with the machine learning approach allowed the grouping of 24 published PDAC cases, showing differences in niche composition and communication patterns. Various co-culture systems demonstrated that KRASG12D-expressing PDLOs could activate pancreatic stellate cells, inducing a cancer- associated fibroblast (CAF)-like state. Simultaneously, these KRAS-activated organoids were protected from T cell infiltration in vitro. Furthermore, in silico approaches were employed to reconstruct a virtual pancreatic cancer space, demonstrating that our PDLOs transcriptionally bridge healthy and malignant ductal states. This also revealed potential KRAS-driven pathways mediating CAF activation and T cell shielding preceding cancer onset. In summary, our human PDLO model enables us to explore the transition from normal to cancerous states, addressing gaps in our understanding of tumor development and tumor niche preparation.
 Citation Format: Chantal Allgoewer, Markus Breunig, Meike Hohwieler, Medhanie Mulaw, Alexander Kleger. Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc4900be652d49c2d242f617b8845d915d29b" target='_blank'>
              Abstract C010: Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset
              </a>
            </td>
          <td>
            Chantal Allgoewer, M. Breunig, M. Hohwieler, Medhanie M Mulaw, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The glioblastoma (GBM) tumor microenvironment consists of a heterogeneous mixture of neoplastic and non-neoplastic cells, including immune cells. Tumor recurrence following standard-of-care therapy results in a rich landscape of inflammatory cells throughout the glioma-infiltrated cortex. Immune cells consisting of glioma-associated macrophages and microglia (GAMMs) overwhelmingly constitute the bulk of the recurrent glioblastoma (rGBM) microenvironment, in comparison to the highly cellular and proliferative tumor microenvironment characteristic of primary GBM. These immune cells dynamically interact within the tumor microenvironment and can contribute to disease progression and therapy resistance while also providing novel targets for emerging immunotherapies. Within these varying contexts, histological-based assessments of immune cells in rGBM, including immunohistochemistry (IHC) and immunofluorescence (IF), offer a critical way to visualize and examine the inflammatory landscape. Here, we exhaustively review the available body of literature on the inflammatory landscape in rGBM as identified through histological-based assessments. We highlight the heterogeneity of immune cells throughout the glioma-infiltrated cortex with a focus on microglia and macrophages, drawing insights from canonical and novel immune-cell histological markers to estimate cell phenotypes and function. Lastly, we discuss opportunities for immunomodulatory treatments aiming to harness the inflammatory landscape in rGBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4001488ad8c82d2b66d1744897ff8a71e5f6d40" target='_blank'>
              Unveiling the Inflammatory Landscape of Recurrent Glioblastoma through Histological-Based Assessments
              </a>
            </td>
          <td>
            Nicholas B. Dadario, D. Boyett, Damian E Teasley, Peter J. Chabot, Nathan J. Winans, Michael G. Argenziano, Colin P. Sperring, P. Canoll, Jeffrey N. Bruce
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b12faa181a1fd052137ca7b75f22bfa849402cf9" target='_blank'>
              GPR65 contributes to constructing immunosuppressive microenvironment in glioma
              </a>
            </td>
          <td>
            Jikang Fan, Jie Liu, Bin Zhang, Xuya Wang, Xisen Wang, Jianshen Liang, Yiming Li, Yu Zhang, Chen Zhang, Sheng-ping Yu, Tao Li, Xuejun Yang
          </td>
          <td>2024-08-10</td>
          <td>Neurosurgical Review</td>
          <td>2</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5edea2d9a83527bbebc5d48f3ba44b53a9b424d3" target='_blank'>
              Single-cell profiling uncovers proliferative cells as key determinants of survival outcomes in lower-grade glioma patients
              </a>
            </td>
          <td>
            Jian-Bing Peng, Qing Zhang, Xiaofeng Zhu, Zhu Yan, Meng Zhu
          </td>
          <td>2024-09-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9fca0c358cd3b69f5d15cee5978ab92d495cb1" target='_blank'>
              STIE: Single-cell level deconvolution, convolution, and clustering in in situ capturing-based spatial transcriptomics
              </a>
            </td>
          <td>
            Shijia Zhu, Naoto Kubota, Shidan Wang, Tao Wang, Guanghua Xiao, Y. Hoshida
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most common primary malignant brain tumor. The aim of this study was to elucidate the role of microenvironment and intrinsic T-type calcium channels (Cav3) in regulating tumor growth and progression. Methods We grafted syngeneic GBM cells into Cav3.2 knockout mice to assess the role of microenvironment T-Type calcium channels on GBM tumor growth. We performed single-cell RNA-seq (scRNA-seq) of tumors from WT and Cav3.2 KO mice to elucidate the regulation of tumors by the microenvironment. We used neurons from WT and Cav3.2 KO mice in co-culture with GBM stem cells (GSC) to assess the effects of Cav3.2 on neuron/GSC synaptic connections and tumor cell growth. Results Cav3.2 KO in the microenvironment led to significant reduction of GBM growth and prolongation of animal survival. scRNA-seq showed that microenvironment Cav3.2 regulates neuronal and glial biological processes. Microenvironment Cav3.2 downregulated numerous genes associated with regulating the OPC cell state in GBM tumors such as SOX10 and Olig2. Neuronal Cav3.2 promoted neuron/GSC synaptic connections and GSC growth. Treatment of GSCs with the Cav3 blocker mibefradil downregulated genes associated with neuronal processes. The Cav3 blocker drug mibefradil synergized with temozolomide (TMZ) and radiation to reduce in vivo tumor growth and prolong animal survival. Conclusions Together these data reveal a role for microenvironment Cav3 in promoting GBM tumor progression through regulating neuronal and glial processes particularly associated with the OPC-cell state. Targeting both intrinsic and microenvironment Cav3 with the inhibitor mibefradil significantly enhanced the anti-GBM effects of TMZ and radiation. Key Points Microenvironment Cav3.2 promotes GBM progression Microenvironment Cav3.2 promotes neuronal and glial processes Pharmacological targeting of intrinsic and microenvironment Cav3 synergizes with TMZ/radiation Importance of the Study In this study, we demonstrate for the first time that microenvironment Cav3.2 contributes to GBM progression and growth by regulating neuronal and glial processes. Our findings highlight the importance of T-type calcium channels in the microenvironment as well as the tumor and provides preclinically relevant data for the use of mibefradil to inhibit GBM growth in combination with standard of care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303b17198abd5c192529012a365daa1747ef06e0" target='_blank'>
              Microenvironment T-Type calcium channels regulate neuronal and glial processes to promote glioblastoma growth
              </a>
            </td>
          <td>
            Collin J. Dube, Ying Zhang, Shekhar Saha, Michelle Lai, Myron K. Gibert, Miguel Escalante, Kadie Hudson, Doris Wong, Pawel Marcinkiewicz, Ulas Yener, Yunan Sun, Esther Xu, Aditya Sorot, Elizabeth Mulcahy, B. Kefas, Farina Hanif, Fadilla Guessous, Ashley Vernon, Manoj K. Patel, David Schiff, Hui Zong, Benjamin Purow, Eric Holland, Swapnil Sonkusare, Harald Sontheimer, R. Abounader
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Galectin-9 is a multifaceted regulator of various pathophysiological processes that exerts positive or negative effects in a context-dependent manner. Here, we elucidated the distinctive functional properties of galectin-9 on myeloid cells within the brain tumor microenvironment. Galectin-9-expressing cells were abundant at the hypoxic tumor edge in the tumor-bearing ipsilateral hemisphere compared to the contralateral hemisphere in an intracranial mouse brain tumor model. Galectin-9 was highly expressed in microglia and macrophages in tumor-infiltrating cells. In primary glia, both the expression and secretion of galectin-9 were influenced by tumors. Analysis of a human glioblastoma bulk RNA-sequencing dataset and a single-cell RNA-sequencing dataset from a murine glioma model revealed a correlation between galectin-9 expression and glial cell activation. Notably, the galectin-9high microglial subset was functionally distinct from the galectin-9neg/low subset in the brain tumor microenvironment. Galectin-9high microglia exhibited properties of inflammatory activation and higher rates of cell death, whereas galectin-9neg/low microglia displayed a superior phagocytic ability against brain tumor cells. Blockade of galectin-9 suppressed tumor growth and altered the activity of glial and T cells in a mouse glioma model. Additionally, glial galectin-9 expression was regulated by Hif-2α in the hypoxic brain tumor microenvironment. Myeloid-specific Hif-2α deficiency led to attenuated tumor progression. Together, these findings reveal that galectin-9 on myeloid cells is an immunoregulator and putative therapeutic target in brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a40e47816f2629de4ffda7a5ee09bfdf58817c" target='_blank'>
              Galectin-9 Mediates the Functions of Microglia in the Hypoxic Brain Tumor Microenvironment.
              </a>
            </td>
          <td>
            Chanju Lee, Dahee Yu, Hyung-Seok Kim, Ki Sun Kim, Chi Young Chang, H. Yoon, Su Bin Won, Dae Yeon Kim, Eun Ah Goh, Yong Sun Lee, Jong Bae Park, Sang Soo Kim, Eun Jung Park
          </td>
          <td>2024-08-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf3da76b81a975c5e4f8d3102cc9399708ed2b0" target='_blank'>
              Mathematical modeling and quantitative analysis of phenotypic plasticity during tumor evolution based on single-cell data.
              </a>
            </td>
          <td>
            Yuyang Xiao, Xiufen Zou
          </td>
          <td>2024-08-20</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62fe0d4817b1cce8f7a8474cb3953d097a1d805f" target='_blank'>
              The spatial distribution of intermediate fibroblasts and myeloid-derived cells dictate lymph node metastasis dynamics in oral cancer
              </a>
            </td>
          <td>
            Soni Shaikh, Harsh Dhar, Manju Moorthy, Vijayalakshmi Bhat, Sangramjit Basu, Devmalya Banerjee, Deepak Kumar Mishra, Sourav Datta, Geetashree Mukherjee
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Gliomas are the most common primary intracranial tumors, known for their high invasiveness and destructiveness. Sialic acid-binding immunoglobulin-like lectin 7 (SIGLEC7) is present in various immune cells, especially macrophages, and significantly affects immune homeostasis and cancer cell response. However, research on the role and prognostic impact of SIGLEC7 in glioma patients is currently limited. Methods We utilized transcriptomic data from 702 glioma patients in The Cancer Genome Atlas (TCGA) and 693 glioma patients in the Chinese Glioma Genome Atlas (CGGA), along with clinical samples we collected, to comprehensively investigate the impact of SIGLEC7 on glioma expression patterns, biological functions, and prognostic value. We focused on its role in glioma-related immune responses and immune cell infiltration and analyzed its expression at the single-cell level. Finally, we validated the role of SIGLEC7 in gliomas through tissue and cell experiments. Results SIGLEC7 expression was significantly increased in glioma patients with malignant characteristics. Survival analysis indicated that glioma patients with high SIGLEC7 expression had significantly lower survival rates. Gene function analysis revealed that SIGLEC7 is primarily involved in immune and inflammatory responses and is strongly negatively correlated with tumor-associated immune regulation. Additionally, the expression of most immune checkpoints was positively correlated with SIGLEC7, and immune cell infiltration analysis clearly demonstrated a significant positive correlation between SIGLEC7 expression and M2 macrophage infiltration levels. Single-cell analysis, along with tissue and cell experiments, confirmed that SIGLEC7 enhances macrophage polarization towards the M2 phenotype, thereby promoting glioma invasiveness through the immunosuppressive effects of M2 macrophages. Cox regression analysis and the establishment of survival prediction models indicated that high SIGLEC7 expression is an unfavorable prognostic factor for glioma patients. Discussion High SIGLEC7 expression predicts poor prognosis in glioma patients and is closely associated with M2 macrophages in the tumor environment. In the future, SIGLEC7 may become a promising target for glioma immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92120f83830255b0d273a3b2691fe4fcad2cb604" target='_blank'>
              High expression of SIGLEC7 may promote M2-type macrophage polarization leading to adverse prognosis in glioma patients
              </a>
            </td>
          <td>
            Wenhao An, Changyuan Ren, Lei Yuan, Zhiqiang Qiu, Peishen Wang, Yanwen Cheng, Zi He, Xinye Han, Shouwei Li, Yihua An
          </td>
          <td>2024-08-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A comprehensive evaluation of the relationship between the densities of various cell types in the breast cancer tumor microenvironment and patient prognosis is currently lacking. Additionally, the absence of a large patch-level whole slide imaging (WSI) dataset of breast cancer with annotated cell types hinders the ability of artificial intelligence to evaluate cell density in breast cancer WSI. We first employed Lasso-Cox regression to build a breast cancer prognosis assessment model based on cell density in a population study. Pathology experts manually annotated a dataset containing over 70,000 patches and used transfer learning based on ResNet152 to develop an artificial intelligence model for identifying different cell types in these patches. The results showed that significant prognostic differences were observed among breast cancer patients stratified by cell density score (P = 0.0018), with the cell density score identified as an independent prognostic factor for breast cancer patients (P < 0.05). In the validation cohort, the predictive performance for overall survival (OS) was satisfactory, with area under the curve (AUC) values of 0.893 (OS) at 1-year, 0.823 (OS) at 3-year, and 0.861 (OS) at 5-year intervals. We trained a robust model based on ResNet152, achieving over 99% classification accuracy for different cell types in patches. These achievements offer new public resources and tools for personalized treatment and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17a923b124c2c0b21b7e052645a7723de6fefc4" target='_blank'>
              Detecting cell types and densities in the tumor microenvironment improves prognostic risk assessment for breast cancer.
              </a>
            </td>
          <td>
            Pu Liu, Xueli Zhang, Wenwen Wang, Yunping Zhu, Yongfang Xie, Yanhong Tai, Jie Ma
          </td>
          <td>2024-08-16</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e985501495b95f9eef3f50592dfe9ef56c04c3f" target='_blank'>
              Digital spatial profiling identifies the tumor center as a topological niche in prostate cancer characterized by an upregulation of BAD
              </a>
            </td>
          <td>
            Ann-Kathrin Huber, A. Kaczorowski, Felix Schneider, Sarah Böning, M. Görtz, David Langhoff, C. Schwab, Albrecht Stenzinger, Markus Hohenfellner, A. Duensing, Stefan Duensing
          </td>
          <td>2024-08-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ab10bbc0562e400afe55f02a7e1bbeddfbcff21" target='_blank'>
              Reinvigoration of cytotoxic T lymphocytes in microsatellite instability-high colon adenocarcinoma through lysosomal degradation of PD-L1
              </a>
            </td>
          <td>
            Dan Liu, Jin Yan, Fang Ma, Jingmei Wang, Siqi Yan, Wangxiao He
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="In the first pan-cancer analysis of the tumor vasculature, Pan and colleagues profile nearly 200,000 endothelial and mural cells (ECs and MCs), identifying novel subclusters and cell states using consensus trajectory inference. They identify differentiation trajectories in vascular and lymphatic endothelial cells, and subtype the pericyte population. During sprouting angiogenesis, venous cells dedifferentiate and transition to capillary and, finally, arterial ECs. Capillary ECs transition via "three angiogenic stages" (SI-SIII) during which APLN+ TipS1 cells were identified as potential modulators of tumor-induced neovascularization and anti-angiogenic therapy response. In lymphatic ECs, differentiation was inversely correlated between the lymphangiogenic (T1) and antigen-presenting (T2) trajectories, with T2 associated with a better prognosis. While several pericyte clusters were identified, BASP1+ matrix-associated pericytes were associated with APLN+ TipS1 cells and had a worse prognosis. These findings present transcriptional validation of previous experimental findings and serve as a resource to examine the tumor vascular microenvironment in detail.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5f776484aa47a465eea06a0fd10d87081b85a4" target='_blank'>
              The Pan-tumor Vasculature Under the Transcriptomic Magnifying Glass.
              </a>
            </td>
          <td>
            Krish Skandha Gopalan, G. Bergers
          </td>
          <td>2024-09-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95736f8e150d70ccc63e3057497b476be28ab1d9" target='_blank'>
              CCKBR+ cancer cells contribute to the intratumor heterogeneity of gastric cancer and confer sensitivity to FOXO inhibition.
              </a>
            </td>
          <td>
            Zhenya Tan, Ke Pan, Minqiong Sun, Xianzhu Pan, Zhi Yang, Zhiling Chang, Xue Yang, Jicheng Zhu, Li Zhan, Yakun Liu, Xiaofei Li, Keqiong Lin, Lin Chen, Hui Mo, Wei Luo, Chen Kan, Lunxi Duan, Hong Zheng
          </td>
          <td>2024-08-20</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Notch is a contact-dependent signaling pathway that plays critical roles in spatiotemporal patterning during embryogenesis. Beyond its role in development, Notch signaling can regulate tissue patterning in adult organs in response to tissue injury or cancer. In pancreatic cancer, Notch signaling drives pancreatic tumor epithelial growth and regulates myeloid function in the tumor microenvironment (TME). Its role in pancreatic cancer-associated fibroblasts (CAFs) is less clear. CAFs are a central component of the pancreatic TME and demonstrate significant phenotypic and functional differences depending on their spatial arrangement in pre-neoplastic and tumor tissue. Using Notch signaling reporter mice CBF: H2B-Venus bred into KC (Ptf1a cre/+;Kras LSL-G12D/+) and KPC (Ptf1a cre/+;Kras LSL-G12D/+;p53 flox/+) backgrounds, we demonstrate that canonical Notch signaling is activated in CAFs at different stages of neoplastic progression. Notably, Notch signaling appears to be most active in aSMA+ CAFs spatially located adjacent to pre-neoplastic/cancer cells, suggesting that Notch ligands expressed by pre-neoplastic/cancer cells are critical for this activation. To test the functional role of canonical Notch signaling in CAFs, we generated a fibroblast-specific, tamoxifen-inducible model of Notch inhibition that utilizes a dominant-negative form of the Notch transcriptional co-activator MAML1 (PDGFRa CreERT2/+ ,Rosa26 LSL-DNMAML). Using an orthotopic model of pancreatic cancer (50,000 cells; KPC) we demonstrate that inhibition of Notch signaling in CAFs results in smaller tumors. These CAFs expressed lower levels of the myofibroblastic marker aSMA+. Additionally, tumors from PDGFRa CreERT2/+, Rosa26 LSL-DNMAML mice had fewer Arg1+ macrophages and a dramatic influx of CD8+ T cells. Collectively, these results suggest that Notch signaling in CAFs regulates their phenotype and supports tumor progression by inducing an immunosuppressive tumor microenvironment.
 Citation Format: Allison Bischoff, Wei Yan, Filip Bednar, Marina Pasca di Magliano. Canonical Notch signaling in cancer-associated fibroblasts supports an immunosuppressive pancreatic tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb37f1ce69653da2f6a1ee7f86d4248295c4b114" target='_blank'>
              Abstract PR-03: Canonical Notch signaling in cancer-associated fibroblasts supports an immunosuppressive pancreatic tumor microenvironment
              </a>
            </td>
          <td>
            Allison C Bischoff, Wei Yan, Filip Bednar, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3fe31e6aa7a99673a4d6d194c90f42b74a1aa1" target='_blank'>
              Immunogenomic features of radiologically distinctive nodules in multiple primary lung cancer
              </a>
            </td>
          <td>
            Mei-Cheng Chen, Haolong Yang, Zhi Dong, Lu-Jie Li, Xiang-Min Li, Hong-He Luo, Qiong Li, Ying Zhu
          </td>
          <td>2024-09-05</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6029311c7be5cdea36f9e6f715640956c9f739" target='_blank'>
              Biophysically interpretable inference of cell types from multimodal sequencing data
              </a>
            </td>
          <td>
            T. Chari, Gennady Gorin, L. Pachter
          </td>
          <td>2024-09-01</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f337210da04933a9ae37820fdc6d7b7fe43d5d5" target='_blank'>
              CALB2 drives pancreatic cancer metastasis through inflammatory reprogramming of the tumor microenvironment
              </a>
            </td>
          <td>
            Jinxin Tao, Yani Gu, Zeyu Zhang, Guihu Weng, Yueze Liu, Jie Ren, Yanan Shi, J. Qiu, Yuanyang Wang, Dan Su, Ruobing Wang, Yifan Fu, Tao Liu, Liyuan Ye, Wenhao Luo, Hao Chen, Gang Yang, Zhe Cao, Hua Huang, Jianchun Xiao, Bo Ren, Lei You, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-10-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The ability of a cancer cell population to achieve heterogeneity in their phenotype distributions offers advantages in tumor invasiveness and drug resistance. Studying the mechanisms behind such observed heterogeneity in mammalian cells presents challenges due for instance to the prolonged proliferation times compared to widely studied unicellular organisms like bacteria and yeast. Here, we studied the response of leukemia cell populations to serum starvation via a protocol, we recently developed, that makes use of live cell fluorescence and flow cytometry in combination with a quantitative analytical model to follow the population proliferation while monitoring the dynamics of its phenotype distributions. We found that upon switching between a serum-rich to a serum-poor media, leukemia cells (i) maintain a memory of the previous environment up to one generation even in the presence of severe medium-depletion, before (ii) adapting their growth and division rates to the novel environment while preserving a sizer-like division strategy. Finally, looking at the mitochondria content of the proliferating vs non-proliferating cells, we found that the latter is characterized by a higher number of older mitochondria, suggesting a possible functional role of the observed asymmetric partitioning of (aged) mitochondria in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ae5c0661fc382b4dfefa11021ab88af1607a56" target='_blank'>
              Probing leukemia cells behavior under starvation
              </a>
            </td>
          <td>
            Simone Scalise, Giorgio Gosti, G. Ruocco, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-08-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3d4f61a2577f3255d5a96a1f62094418b1629b" target='_blank'>
              A cell state-specific metabolic vulnerability to GPX4-dependent ferroptosis in glioblastoma.
              </a>
            </td>
          <td>
            Matei A. Banu, Athanassios Dovas, Michael G. Argenziano, Wenting Zhao, Colin P. Sperring, Henar Cuervo Grajal, Zhouzerui Liu, Dominique Mo Higgins, Misha Amini, B. Pereira, Ling F Ye, A. Mahajan, N. Humala, J. Furnari, P. Upadhyayula, Fereshteh Zandkarimi, Trang Tt Nguyen, Damian E Teasley, Peter B Wu, Li Hai, Charles Karan, Tyrone Dowdy, Aida Razavilar, M. Siegelin, Jan Kitajewski, Mioara Larion, Jeffrey N. Bruce, Brent R. Stockwell, Peter A. Sims, Peter D Canoll
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="235 Background: Fibrosarcoma (FS) is a relatively uncommon subtype of soft tissue sarcoma, which predominantly occurs in the extremities and girdles of elderly individuals. FS are famous for its high recurrence and metastasis rate. However, the current treatment regimens have shown poor efficacy in treating these progressive lesions. Therefore, a deep understanding of the tumor microenvironment characteristics of fibrosarcoma is needed to develop more effective drugs.Therefore, a deeper understanding of the tumor microenvironment characteristics of fibrosarcoma is needed for further development of therapeutic drugs. Methods: We performed single-cell transcriptome sequencing for the first time on 6 patients bearing with FS. Bulk-seq data of soft tissue sarcoma was downloaded from public database including TCGA and GEO for prognostic analysis. What' s more, a cohort of 54 FS patients was retrospectively collected for protein detection. Results: The tumor microenvironment landscape was characterized by the presence of malignant fibroblast, diverse macrophages, limited T-cell presence, and high levels of vascular infiltration. The expression of siglec15 was significantly upregulated in the malignant cells. And the prognostic significance of siglec15 was confirmed by the TCGA public database. What’s more, the high expression of siglec15 protein was negatively related to overall survival of fibrosarcoma patients from our institution. We also explored the interactions among different cell subtypes, especially the interactions between malignant cells and endothelial cells, as well as macrophages and endothelial cells, which not only promote tumor progression but also inhibit T-cell infiltration and function. Conclusions: In total, our work has uncovered the low T cell infiltration landscape of FS. The primary reason of insufficient immune infiltration may be due to the blockage of T cell extravasation caused by the interaction between endothelial cells, malignant cells and macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca424f7bf6e590c667e5a83854870c663bc0d1f2" target='_blank'>
              The single cell transcriptional landscape of fibrosarcoma.
              </a>
            </td>
          <td>
            Juan Wang, Zhenfeng Li, Ning Zhang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bladder cancer (BCa) is one of the most challenging and costly cancers to treat, yet little progress has been made on the development of predictive biomarkers and targeted therapies. Here, we uncover a critical function of Numb as a tumor suppressor in the bladder, identifying loss of Numb expression as a causal alteration in BCa that underlies biological aggressiveness and disease progression. Through retrospective cohort studies, we established that a Numb-deficient tumor status correlates with worse overall survival in post-cystectomy muscle-invasive bladder cancer (MIBC) patients and increased risk of MIBC progression in non-muscle-invasive bladder cancer (NMIBC) patients. The prognostic value of Numb loss can be attributed to its crucial role as a determinant of aggressive bladder tumorigenesis, as demonstrated in mouse and human models. Targeted Numb ablation in the basal layer of the urothelium was alone sufficient to trigger spontaneous bladder tumorigenesis and drive progression from preneoplastic to preinvasive and, ultimately, overtly invasive tumors. Additionally, Numb ablation sensitized the urothelium to other oncogenic insults, accelerating tumor onset and progression. Using 3D-Matrigel organoid cultures to recapitulate bladder tumorigenesis in vitro, we found that Numb loss heightens the proliferative and invasive potential of both mouse and human BCa cells. Integrative transcriptomic and functional analyses revealed that downregulation of the canonical Hippo pathway, resulting in enhanced YAP transcriptional activity, underlies the biological aggressiveness of Numb-deficient BCa. These molecular events are dependent on the activation of RhoA/ROCK signaling subsequent to Numb loss. Thus, a dysfunctional Numb–RhoA/ROCK–Hippo/YAP regulatory network is at play in aggressive Numb-deficient BCa and represents a therapeutic vulnerability. A 27-gene prognostic signature capable of identifying high-risk Numb-deficient patients could provide the basis of a clinical tool to stratify patients for innovative RhoA/ROCK/YAP targeted therapies. One Sentence Summary Numb loss-directed hyperactivation of RhoA/ROCK/YAP underlies aggressive bladder cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72434e6e063f988f837149b1be62c13ce13452e2" target='_blank'>
              Loss of the tumor suppressor NUMB drives aggressive bladder cancer through hyperactivation of a RhoA/ROCK/YAP signaling circuitry
              </a>
            </td>
          <td>
            F. A. Tucci, R. Pennisi, D. Rigiracciolo, M. G. Filippone, R. Bonfanti, F. Romeo, S. Freddi, C. Soriani, S. Rodighiero, G. Jodice, F. Sanguedolce, G. Renne, N. Fusco, P. Di Fiore, G. Pruneri, G. Musi, G. Vago, D. Tosoni, S. Pece
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is the most aggressive subtype of breast cancer. TNBC displays exceptional transcriptional heterogeneity, both between patients and intratumorally, and disproportionately affects underserved women. Hypoxia, a state of low oxygen levels, is often present in the solid tumor microenvironment. In TNBC, hypoxia is a key driver of aggressiveness and therapeutic resistance. Past bulk RNA sequencing (RNAseq) studies in cancer research have lacked spatial resolution, failing to uncover precise transcriptional differences within tumor regions. This spatial limitation has been especially problematic for studying transcriptional heterogeneity of the cellular hypoxia response within tumors. By integrating single-cell RNA sequencing and spatial transcriptomic analyses in a TNBC xenograft tumor model, our research has uncovered a novel hypoxia gene signature, specific to cancer cells. Elevated expression of this hypoxia signature predicts poorer prognosis in breast cancer patients, especially within TNBC patients and those of African ancestry. This suggests that hypoxia may play a role in the racial disparities observed in TNBC prognosis, potentially informing treatment strategies. This workflow introduces a novel framework for translating general spatial transcriptomics data into clinically relevant and accurate gene signatures.
 Citation Format: Kent Schechter, Long C. Nguyen, Geetha P. Yerradoddi, Rania Bassiouni, John D. Carpten, Marsha R. Rosner. Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C105.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ba633337a4ee769f8bd92a6f7364c821869174" target='_blank'>
              Abstract C105: Novel cancer-specific hypoxia signature predicts poor prognosis and highlights racial disparities in breast cancer
              </a>
            </td>
          <td>
            Kent Schechter, Long C Nguyen, G. P. Yerradoddi, R. Bassiouni, John D. Carpten, M. Rosner
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The field of single-cell biology is growing rapidly and is generating large amounts of data from a variety of species, disease conditions, tissues, and organs. Coordinated efforts such as CZI CELLxGENE, HuBMAP, Broad Institute Single Cell Portal, and DISCO, allow researchers to access large volumes of curated datasets. Although the majority of the data is from scRNAseq experiments, a wide range of other modalities are represented as well. These resources have created an opportunity to build and expand the computational biology ecosystem to develop tools necessary for data reuse, and for extracting novel biological insights. Here, we highlight achievements made so far, areas where further development is needed, and specific challenges that need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/402d7d069a7187fa7df5e85d41594977c06e2e75" target='_blank'>
              Insights, opportunities and challenges provided by large cell atlases
              </a>
            </td>
          <td>
            M. Hemberg, Federico Marini, S. Ghazanfar, A. A. Ajami, Najla Abassi, Benedict Anchang, B. Benayoun, Yue Cao, Ken Chen, Yesid Cuesta-Astroz, Zach DeBruine, C. Dendrou, I. Vlaminck, K. Imkeller, I. Korsunsky, Alex R. Lederer, Pieter Meysman, Clint Miller, Kerry Mullan, Uwe Ohler, Nikolaos Patikas, Jonas Schuck, Jacqueline H.Y. Siu, Timothy Triche, Alex Tsankov, S. W. Laan, Masanao Yajima, Jean Y H Yang, Fabio Zanini, Ivana Jelic
          </td>
          <td>2024-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880d6fbe977b9fa0abf41b1330834beac1e2a6aa" target='_blank'>
              A patient-derived cell model for malignant transformation in IDH-mutant glioma
              </a>
            </td>
          <td>
            Olga Kim, Zach Sergi, Guangyang Yu, Kazu Yamamoto, Martha Quezado, Zied Abdullaev, Danel R Crooks, S. Kishimoto, Qi Li, Peng Lu, Burchelle N. Blackman, T. Andresson, Xiaolin Wu, Bao Tran, Jun S. Wei, Wei Zhang, Meili Zhang, Hua Song, Javed Khan, Murali C. Krishna, J. Brender, Jing Wu
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background FOXR2-activated central nervous system (CNS) neuroblastoma (CNS NB-FOXR2) is a recently identified subtype of brain tumor characterized by the elevated expression of the transcription factor FOXR2 mainly due to genomic rearrangements. However, the precise pathogenic mechanisms, including the cell type of origin, remain elusive. Methods A gene expression analysis of patient tumors was performed to identify putative cell types of origin. Based on this prediction, a new human embryonic stem cell–based model was developed to validate the origin and to examine the molecular and cellular mechanisms underlying the formation of CNS NB-FOXR2. Results Our data showed that CNS NB-FOXR2 tumors express a high level of lineage marker genes associated with the medial ganglionic eminence (MGE), a transient structure located in the developing ventral forebrain. Our model confirmed the cell-type-specific effect of FOXR2 on the proliferation and in vivo tumorigenicity. Additionally, we found that FOXR2 overexpression activated the MEK/ERK signaling pathway through a suppression of the endogenous RAS inhibitor DIRAS3. The MEK inhibitor trametinib suppressed the proliferation of FOXR2-expressing MGE progenitors more than nonexpressing cells. Conclusions Our study collectively demonstrates that MGE progenitors are the cell of origin of CNS NB-FOXR2 and that FOXR2 activates the MEK/ERK signaling pathway, providing a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8445a3e44a2783d48d762e8340d244130bab0b2f" target='_blank'>
              A human embryonic stem cell–based model reveals the cell of origin of FOXR2-activated CNS neuroblastoma
              </a>
            </td>
          <td>
            Hitomi N Royston, Autumn B Hampton, Dhruv Bhagat, Evonne F Pinto, Miriam D Emerson, Kosuke Funato
          </td>
          <td>2024-08-12</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Vulvar diseases are a critical yet often neglected area of women’s health, profoundly affecting patients’ quality of life and frequently resulting in long-term physical and psychological challenges. Lichen sclerosus (LS) is a chronic inflammatory skin disorder that predominantly affects the vulva, leading to severe itching, pain, scarring, and an increased risk of malignancy. Despite its profound impact on affected individuals, the molecular pathogenesis of vulvar LS (VLS) is not well understood, hindering the development of FDA-approved therapies. Here, we utilize single-cell and spatial transcriptomics to analyze lesional and non-lesional skin from VLS patients, as well as healthy control vulvar skin. Our findings demonstrate histologic, cellular, and molecular heterogeneities within VLS, yet highlight unifying molecular changes across keratinocytes, fibroblasts, immune cells, and melanocytes in lesional skin. They reveal cellular stress and damage in fibroblasts and keratinocytes, enhanced T cell activation and cytotoxicity, aberrant cell-cell signaling, and increased activation of the IFN, JAK/STAT, and p53 pathways in specific cell types. Using both monolayer and organotypic culture models, we also demonstrate that knockdown of select genes, which are downregulated in VLS lesional keratinocytes, partially recapitulates VLS-like stress-associated changes. Collectively, these data provide novel insights into the pathogenesis of VLS, identifying potential biomarkers and therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a1ede19b990ffc821eea644860eccc4ba0949" target='_blank'>
              Single-cell and spatial transcriptomics of vulvar lichen sclerosus reveal multi-compartmental alterations in gene expression and signaling cross-talk
              </a>
            </td>
          <td>
            Peng Sun, Christina N. Kraus, Wei Zhao, Jiahui Xu, Susie Suh, Quy Nguyen, Yunlong Jia, Arjun Nair, Melanie Oakes, Roberto Tinoco, Jessica Shiu, Bryan Sun, Ashley Elsensohn, Scott X. Atwood, Qing Nie, Xing Dai
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="High-grade gliomas are a major health challenge with poor prognosis and high morbidity. Immune-checkpoint inhibitors (ICI) have emerged as promising therapeutic options for several malignancies yet show little efficacy against central nervous system (CNS) tumors. CD200 is a newly recognized immune checkpoint that modulates immune homeostasis. CD200 protein is expressed by a variety of cells, including immune cells and stromal cells, and is overexpressed by many tumors. The shedding of CD200 from tumor cells can create an immunosuppressive environment that dampens anti-tumor immunity by modulating cytolytic activity and cytokine expression both within and outside the tumor microenvironment (TME). While it is well-accepted that CD200 induces a pro-tumorigenic environment through its ability to suppress the immune response, we sought to determine the role of glioma-specific expression of CD200. We show that CD200 is expressed across glioma types, is shed from tumor cells, and increases over time in the serum of patients undergoing immunotherapy. Using CD200 knockout (KO) glioma models, we demonstrated that glioma cell-derived CD200 promotes tumor growth in vivo and in vitro. Notably, CD200 KO gliomas are spontaneously rejected by their host, a process that required a fully functional immune system, including NK and T-cells. Moreover, we report that glioma-derived or brain-injected soluble CD200 contributes to the suppression of antigen-specific CD8 T-cells in the draining lymph nodes (dLNs). Our work provides new mechanistic insights regarding CD200-mediated immunosuppression by gliomas. Statement of significance We demonstrate mechanisms of the druggable glioma-derived CD200 checkpoint on tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b634d44ec918f6bd798f77ff4dc7cd21cc67c2e" target='_blank'>
              CD200 depletion in glioma enhances antitumor immunity and induces tumor rejection
              </a>
            </td>
          <td>
            Itay Raphael, Anzar A. Mujeeb, Elisabet Ampudia-Mesias, ReidAnn E. Sever, Brandon L. McClellan, Stephen C. Frederico, Chaim T. Sneiderman, Apoorva Mirji, Ali Daba, Francisco Puerta-Martínez, Michal Nisnboym, W. Edwards, Michael W. Graner, Christopher L. Moertel, Maria G. Castro, Gary Kohanbash, Michael Olin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
A specific type of prostate cancer (PC) that exhibits neuroendocrine (NE) differentiation is known as NEPC. NEPC has little to no response to androgen deprivation therapy and is associated with the development of metastatic castration-resistant PC (CRPC), which has an extremely poor prognosis. Our understanding of genetic drivers and activated pathways in NEPC is limited, which hinders precision medicine approaches. L1 cell adhesion molecule (L1CAM) is known to play an oncogenic role in metastatic cancers, including CRPC. However, the impact of L1CAM on NEPC progression remains elusive.


METHODS
L1CAM expression level was investigated using public gene expression databases of PC cohorts and patient-derived xenograft models. L1CAM knockdown was performed in different PC cells to study in vitro cell functions. A subline of CRPC cell line CWR22Rv1 was established after long-term exposure to abiraterone to induce NE differentiation. The androgen receptor-negative cell line PC3 was cultured under the tumor sphere-forming condition to enrich cancer stemness features. Several oxidative stress inducers were tested on PC cells to observe L1CAM-mediated gene expression and cell death.


RESULTS
L1CAM expression was remarkably high in NEPC compared to CRPC or adenocarcinoma tumors. L1CAM was also correlated with NE marker expressions and associated with the adenocarcinoma-to-NEPC progression in gene expression databases and CRPC cells with NE differentiation. L1CAM also promoted cancer stemness and NE phenotypes in PC3 cells under cancer stemness enrichment. L1CAM was also identified as a reactive oxygen species-induced gene, by which L1CAM counteracted CRPC cell death triggered by ionizing radiation.


CONCLUSIONS
Our results unveiled a new role of L1CAM in the acquisition of the NE phenotype in PC, contributing to the NE differentiation-related therapeutic resistance of CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058f0ff5a1eb85bfaf3568b71d602509408bcc46" target='_blank'>
              L1CAM mediates neuroendocrine phenotype acquisition in prostate cancer cells.
              </a>
            </td>
          <td>
            Chia-Ling Hsieh, Anh Duy Do, Chia-Yen Hsueh, Mafewu Olga Raboshakga, Tran Ngoc Thanh, Tran Tien Tai, Hsing-Jien Kung, Shain-Ying Sung
          </td>
          <td>2024-08-18</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Introduction: High-risk neuroblastoma (NBL) is a leading cause of pediatric cancer death and is characterized by substantial intratumoral heterogeneity leading to therapeutic resistance. Despite the significant insights from recent single-cell sequencing, the evolution of the tumor microenvironment under therapy remains obscure. We longitudinally profiled 22 high-risk NBL patients before and after induction chemotherapy through single-nucleus RNA and ATAC sequencing. We identified profound shifts in tumor and immune cell subpopulations. In particular, macrophages significantly expanded towards pro-angiogenic, immunosuppressive, and metabolic phenotypes. Through a prediction of paracrine signaling using our single-cell sequencing data, we predicted a role of the heparin-binding epidermal growth factor (HB-EGF)/Erb receptor tyrosine kinase 4 (ERBB4) signaling pathway between VCAN+ macrophages and neuroblasts. In this study, we sought to characterize the role of this signaling pathway in therapy response in high-risk neuroblastoma. Methods: We examined the interaction between malignant neuroblasts and macrophages using transwell co-culture assay. Macrophages were differentiated from the THP1 monocytes. Neuroblasts were cultured using NB1643, CHLA15, CHLA20, COG-N-297, and COG-N-590 neuroblastoma cell lines. To determine the source and activity of the HB-EGF/ERBB4/ERK signaling pathway, protein levels were determined using enzyme-linked immunosorbent assay and Western blot using co-culture supernatant and neuroblast and macrophage cell lysates. Pathway activity was modulated using CRM197 as the HB-EGF inhibitor and afatinib as the ERBB tyrosine kinase inhibitor. We examined the role of macrophage-derived HB-EGF on cell proliferation and migration by colony formation and transwell migration assays, respectively. Furthermore, we assessed the adrenergic (PHOX2B, GATA3, ALK, PHOX2A) and mesenchymal (YAP1 and PRRX1) marker expressions by real-time qPCR and Western blot. Results: We found that expression of HB-EGF in macrophages was significantly induced when co-cultured with multiple neuroblast cell lines. Binding of HB-EGF to ERBB4 in neuroblasts led to the activation of the kinase pathway and increased proliferation and migration of neuroblast cells, which can be inhibited by either HB-EGF or tyrosine kinase inhibitors. We examined adrenergic and mesenchymal marker expression in neuroblasts after prolonged co-culture with macrophages. While ALK and PHOX2B expressions were reduced, YAP1 and PRRX1 expressions increased in post-therapy CHLA20 cells but not in the paired diagnostic CHLA15 or NB1643 cells. Inhibition of the HB-EGF/ERBB pathway partially restored PHOX2B expression and reduced YAP1 and PRRX1 expression in macrophage co-cultured CHLA20 cells. Conclusion: We established the HB-EGF/ERBB4 axis between macrophage and neoplastic neuroblast, inducing ERK signaling, tumor cell proliferation, migration, and adrenergic-to-mesenchymal transition. These findings collectively reveal a novel extrinsic mechanism of therapy resistance in high-risk NBL.
 Citation Format: Rumeysa Biyik-Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea F. Surrey, Michael D. Hogarty, Kathrin Bernt, Nancy Zhang, John M. Maris, Kai Tan. VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf38a2142d9aa00bb31bb908d3c2ec97aaec27dc" target='_blank'>
              Abstract B062: VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis
              </a>
            </td>
          <td>
            Rumeysa Biyik‐Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea Surrey, M. Hogarty, Kathrin M Bernt, Nancy Zhang, John M. Maris, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Yvette Hayat, Sarah Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, Emmanuelle Huillard, Nina Pottier, Aurore Desmons, Inès Fayache, Garrett A. Kaas, Philip J. Kingsley, Lawrence J. Marnett, Eric Duplus, Elias El-Habr, Lucas Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, Michele Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, M. Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Alterations in Fibroblast growth factor receptor (FGFR)-family proteins frequently occur as oncogenes in many cancers, including a subset of pediatric gliomas. Here, we performed a genomic analysis of 11,635 gliomas across ages and found that 4.5% of all gliomas harbor FGFR alterations including structural variants (SV) and single nucleotide variants (SNV), with an incidence of almost 10% in pediatric gliomas. FGFR family members are differentially enriched by age, tumor grade, and histological subtype, with FGFR1-alterations associated with glioneuronal histologies and pediatric low-grade gliomas. Across development, we find FGFR1 expression in both neuronal and glial precursors, while FGFR3 expression is largely restricted to astrocytic lineages. Leveraging novel isogenic model systems, we confirm FGFR1 alterations to be sufficient to activate MAPK and mTOR signaling, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway inhibitors, including pan-FGFR inhibitors. Models driven by FGFR1 SVs exhibited different patterns of sensitivity compared to those driven by SNVs. Finally, we performed a retrospective analysis of clinical responses in children diagnosed with FGFR-driven gliomas and found that targeted MAPK or FGFR-inhibition with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc242f4fb57f142e32352b142311d05135cc1f" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations defines novel therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Eric Morin, April Apfelbaum, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe C Power, Margaret Cusick, Dana Novikov, Robert Jones, J. Vogelzang, Connor Bossi, Seth W. Malinowski, John Jeang, Jared Collins, Sehee Oh, Hyesung Jeon, A. Cameron, Patrick Rechter, Angela Deleon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, C. M. Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, T. Bale, Amy A. Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael Eck, Lissa Baird, Olaf Witt, Claudia L. Kleinman, Quang-De Nguyen, S. Alexandrescu, D. Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="PURPOSE
Malignant peripheral nerve sheath tumors (MPNST) are aggressive soft tissue sarcomas and the leading cause of mortality in individuals with neurofibromatosis type 1 (NF1). Despite many clinical trials, outcomes for patients with MPNST have remained stagnant and most succumb to their disease; thus, novel therapeutic approaches are needed. A better understanding of the MPNST immune ecosystem will aid in the development of strategies to activate the immune system against the tumor. Herein, we profile the tumor immune microenvironment (TIME) in NF1-associated peripheral nerve sheath tumors (PNST) to discover insights on the role that tumor-infiltrating immune cells play in malignant transformation.


EXPERIMENTAL DESIGN
Utilizing fresh and formalin-fixed, paraffin-embedded tissue from patients diagnosed with NF1-PNST, we dissected the TIME by using immunohistochemistry, multiparameter flow cytometry, and comparative transcriptomic studies.


RESULTS
Immunophenotyping confirmed increased immune cells infiltration during malignant progression, with a predominance of infiltrating myeloid cells, particularly CD163+ tumor-associated macrophages (TAM). The T cells within MPNST exhibited signs of tumor activation, characterized by high PD-1 expression. Additionally, MPNST specimens demonstrated elevated levels of immunosuppressive TAM, with heightened PD-L1 expression. The proportion of CD163+ myeloid cells within the TIME correlated with poorer progression free survival. Notably, loss of H3K27 trimethylation correlated with low immune cell infiltration in MPNST.


CONCLUSIONS
Malignant transformation of NF1-PNST is characterized by an immunosuppressive microenvironment comprising of TAM with high expression of PD-L1, which are associated with inferior outcomes. These findings suggest a clinical potential of immune modulating therapeutics that can unleash an anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45c2d277458a04b199f1d217e12500b372f16bc" target='_blank'>
              Multi-dimensional immunotyping of human NF1-associated peripheral nerve sheath tumors uncovers tumor-associated macrophages as key drivers of immune evasion in the tumor microenvironment.
              </a>
            </td>
          <td>
            Lindy Zhang, Alexandre Maalouf, Stavriani C Makri, Jineta Banerjee, Aditya Suru, Ada Tam, A. Calizo, K. Pollard, Jiawan Wang, Ludmila Danilova, Maria Ioannou, Adam S Levin, Carol D Morris, Daniel S Rhee, Allan J. Belzberg, J. Blakeley, Brian H Ladle, Drew M Pardoll, Calixto-Hope G Lucas, Fausto J Rodriguez, John M. Gross, Robert A Anders, C. Pratilas, Nicolas J Llosa
          </td>
          <td>2024-09-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc59d95e4da445f6a5103d3c621c07c257acc5d4" target='_blank'>
              Single-cell RNA transcriptomic analyses of tumor microenvironment of ovarian metastasis in gastric cancer.
              </a>
            </td>
          <td>
            Guoyu Chen, Mingda Zhang, Xiao-Zhu Lin, Qiqi Shi, Chenxin Xu, Bowen Sun, Xiuying Xiao, Haizhong Feng
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D'Amato, Yi Xue, Kristina Fontanez, Aaron A May-Zhang, Trinity Smithers, Yigal Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80a90a7fa373a0b5dc0cb49c622a37b283337d8" target='_blank'>
              Barriers and opportunities in pancreatic cancer immunotherapy
              </a>
            </td>
          <td>
            Yixin Ju, Dongzhi Xu, Miao-Miao Liao, Yutong Sun, Wen-Dai Bao, Fan Yao, Li Ma
          </td>
          <td>2024-09-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Data from cell viability assays, which measure cumulative division and death events in a population and reflect substantial cellular heterogeneity, are widely available. However, interpreting such data with mechanistic computational models is hindered because direct model/data comparison is often muddled. We developed an algorithm that tracks simulated division and death events in mechanistically detailed single-cell lineages to enable such a model/data comparison and suggest causes of cell-cell drug response variability. Using our previously developed model of mammalian single-cell proliferation and death signaling, we simulated drug dose response experiments for four targeted anti-cancer drugs (alpelisib, neratinib, trametinib and palbociclib) and compared them to experimental data. Simulations are consistent with data for strong growth inhibition by trametinib (MEK inhibitor) and overall lack of efficacy for alpelisib (PI-3K inhibitor), but are inconsistent with data for palbociclib (CDK4/6 inhibitor) and neratinib (EGFR inhibitor). Model/data inconsistencies suggest (i) the importance of CDK4/6 for driving the cell cycle may be overestimated, and (ii) that the cellular balance between basal (tonic) and ligand-induced signaling is a critical determinant of receptor inhibitor response. Simulations show subpopulations of rapidly and slowly dividing cells in both control and drug-treated conditions. Variations in mother cells prior to drug treatment all impinging on ERK pathway activity are associated with the rapidly dividing phenotype and trametinib resistance. This work lays a foundation for the application of mechanistic modeling to large-scale cell viability assay datasets and better understanding determinants of cellular heterogeneity in drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10d80f29cb012f285134e2b29378c29a5d6024bc" target='_blank'>
              Mechanistic modeling of cell viability assays with in silico lineage tracing
              </a>
            </td>
          <td>
            Arnab Mutsuddy, Jonah R. Huggins, Aurore Amrit, Cemal Erdem, Jon Calhoun, Marc R. Birtwistle
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7199657c55f0905635cd26958d5762b121b0a3f2" target='_blank'>
              Integrated multi-omics assessment of lineage plasticity in a prostate cancer patient with brain and dural metastases
              </a>
            </td>
          <td>
            Megan L Ludwig, David Moline, Alec Horrmann, Ella Boytim, Gabrianne Larson, Ali T Arafa, Masooma Sayeda, John R Lozada, Hannah E Bergom, Abderrahman Day, Sandhyarani Dasaraju, S. Dehm, Paari J. Murugan, Justin H. Hwang, J. Drake, Emmanuel S Antonarakis
          </td>
          <td>2024-09-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, P. Maini, Tyler Cassidy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) originates from precursor lesions, the most common of which are known as pancreatic intraepithelial neoplasia (PanIN). PanIN formation and progression is accompanied by accumulation of a precursor microenvironment (PME) composed of desmoplastic stroma that plays an important role in promoting tumor growth and immune evasion. The crosstalk between tumor cells and stromal cells is an area of active investigation. While PanIN and the PME have been extensively studied in mouse models, their nature in humans have remained elusive as there is no clinical indication to sample healthy pancreas. Through a unique collaboration with the Gift of Life Organ and Tissue donation program, we performed transcriptomic analysis on normal pancreata declined for transplant. We noted that epithelial cells in the PanIN and cancer microenvironment were transcriptomically similar; however, their surrounding stromal fibroblasts were markedly distinct. Given that most precursor lesions remain quiescent while some progress to tumor, we sought out to elucidate how the transcriptional programs in epithelial cells influence fibroblast cell fate. To do so, we developed a 3D ex-vivo organoid-fibroblast coculture system using primary organoid and fibroblast cultures successfully derived from Gift of Life pancreata or PDAC patients undergoing surgical resection. Our coculture system retains histological features of heterogeneity observed in vivo. Using single cell RNA-sequencing analysis of matched fibroblast-organoid cocultures we identified factors that were highly expressed in fibroblasts in tumor cocultures, but not in normal cocultures. We are currently interrogating candidate ligand/receptor interactions in our cocultures to determine the mediators of this epithelial-mediated fibroblast polarization. This model will allow us to gain an understanding about how components of the microenvironment restrain or promote malignant transformation and serve as a great tool to study therapies targeting tumor and stromal cells in a patient specific manner.
 Citation Format: Padma Kadiyala, Ahmed Elhossiny, Jianhua Liu, Alexander Bray, Katelyn Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy Frankel, Eileen Carpenter, Marina Pasca Di Magliano. Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9841941d0df90af0a793863b27fe1e8b8a9497" target='_blank'>
              Abstract A057: Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment
              </a>
            </td>
          <td>
            Padma Kadiyala, Ahmed M. Elhossiny, Jianhua Liu, Alexander W Bray, Katelyn L. Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy L. Frankel, Eileen S Carpenter, M. P. di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Curcumin is widely recognized for its diverse antitumor properties, ranging from breast cancer to many other types of cancers. However, its role in the tumor microenvironment remains to be elucidated. In this study, we established a 3D tumor spheroids model that can simulate the growth environment of tumor cells and visualized the antitumor metabolic alteration caused by curcumin using mass spectrometry imaging technology. Our results showed that curcumin not only exerts a profound impact on the growth and proliferation of breast cancer cells but in situ multivariate statistical analysis also reveals the significant effect on the overall metabolic profile of tumor spheroids. Meanwhile, our visualization map characterized curcumin metabolic processes of reduction and glucuronidation in tumor spheroids. More importantly, abnormal metabolic pathways related to lipid metabolism and polyamine metabolism were also remodeled at the metabolite and gene levels after curcumin intervention. These insights deepen our comprehension of the regulatory mechanism of curcumin on the tumor metabolic network, furnishing powerful references for antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9708e466bdcc061f5dd608fcbfa98f940da2834e" target='_blank'>
              Spatial Metabolomics Profiling Reveals Curcumin Induces Metabolic Reprogramming in Three-Dimensional Tumor Spheroids
              </a>
            </td>
          <td>
            Zihan Zhu, Yaqi Zhang, Lei Wang, Haoyuan Geng, Min Li, Shiping Chen, Xiao Wang, Panpan Chen, Chenglong Sun, Chao Zhang
          </td>
          <td>2024-09-01</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, P. Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="High-throughput sequencing of single-cell data can be used to rigorously evlauate cell specification and enable intricate variations between groups or conditions. Many popular existing methods for differential expression target differences in aggregate measurements (mean, median, sum) and limit their approaches to detect only global differential changes. We present a robust method for differential expression of single-cell data using a kernel-based score test, cytoKernel. cytoKernel is specifically designed to assess the differential expression of single cell RNA sequencing and high-dimensional flow or mass cytometry data using the full probability distribution pattern. cytoKernel is based on kernel embeddings which employs the probability distributions of the single cell data, by calculating the pairwise divergence/distance between distributions of subjects. It can detect both patterns involving aggregate changes, as well as more elusive variations that are often overlooked due to the multimodal characteristics of single cell data. We performed extensive benchmarks across both simulated and real data sets from mass cytometry data and single-cell RNA sequencing. The cytoKernel procedure effectively controls the False Discovery Rate (FDR) and shows favourable performance compared to existing methods. The method is able to identify more differential patterns than existing approaches. We apply cytoKernel to assess gene expression and protein marker expression differences from cell subpopulations in various publicly available single-cell RNAseq and mass cytometry data sets. The methods described in this paper are implemented in the open-source R package cytoKernel, which is freely available from Bioconductor at http://bioconductor.org/packages/cytoKernel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5266fa2edd1942a8b424629b41d58db895ce6a73" target='_blank'>
              cytoKernel: Robust kernel embeddings for assessing differential expression of single cell data
              </a>
            </td>
          <td>
            Tusharkanti Ghosh, Ryan M. Baxter, Souvik Seal, Victor G Lui, Pratyadipta Rudra, Thao Vu, Elena W Y Hsieh, Debashis Ghosh
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Despite extensive molecular stratification efforts, improvements in molecularly informed RT-based therapeutic strategies in the neoadjuvant setting have been limited. This study aims to elucidate the adaptive mechanisms and interactions within PDAC to identify new combinatorial therapeutic targets with RT. MethodsWe constructed a high-resolution molecular landscape of the cellular subtypes and spatial communities that constitute PDAC. Single-cell RNA sequencing (scRNA- seq) was performed on 16 PDAC biopsies of matched pre- and post-treatment samples from a Phase I/II dose-escalation clinical trial of pancreatic stereotactic body radiation therapy (SBRT). Additionally, Visium spatial transcriptomics (ST) was conducted on biopsies from primary PDAC patients collected at the time of surgery. These patients received either no neoadjuvant therapy (n = 8) or neoadjuvant chemoradiation (chemoRT) (n = 20). Cell type profiles were determined from scRNA-seq to compare changes in cell type proportions and signatures before and after RT. From these profiles, we employed robust cell type decomposition to analyze cell type signatures in our ST dataset. ResultsChemoRT induced several significant effects compared to controls within the tumor microenvironment, including an increase in the proportion of myofibroblasts, specifically senescent cancer-associated fibroblasts (CAFs), with signatures consistent with immune cell dysfunction. We observed an increase in CD8 and CD4 effector memory cells with RT; however, this occurred alongside a rise in immunosuppressive alternatively activated tumor-associated macrophages. Additionally, RT upregulated pathways related to HIF, cytokine production, B cell proliferation, and negative regulation of lymphocytes. Patients with increased infiltration of NK, memory B, and CD8 effector memory cells demonstrated improved overall survival outcomes. Inference of copy number variations enabled us to identify cancer subclones as either responsive or resistant in matched pre and post RT biopsies. Resistance was characterized by augmented hypoxia, KRAS activation, epithelial- mesenchymal transition, and interferon-gamma response signaling. Conversely, responders exhibited increased oxidative phosphorylation signaling and pathway enrichment associated with cell proliferation and cell cycle progression. Interestingly, RT induced a decrease in cancer cells characterized as harboring a chemoresistant basal-like molecular subtype. ConclusionThrough both longitudinal and therapy stratified tumor biopsies, we were able to observe significant selection pressures on stromal and cancer cells following RT. This includes upregulation of innate and adaptive immune pathways with a compensatory immunosuppressive response driven by myeloid cells and CAFs. This may provide opportunities for development of clinical trials to therapeutically target specific cellular phenotypes and their interactions.
 Citation Format: Vincent Bernard, Galia Jacobson, Kimal Rajapakshe, Jimin Min, Guangsheng Pei, Daniel Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, Manoop S Bhutani, Linghua Wang, Paola A Guerrero, Ethan B Ludmir, Albert C Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J Koay. A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee951a6470ee1fe16a77439ce9fbdb87aac3a94c" target='_blank'>
              Abstract B078: A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            V. Bernard, Galia Jacobson, K. Rajapakshe, Jimin Min, Guangsheng Pei, D. Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, M. Bhutani, Linghua Wang, Paola A Guerrero, E. Ludmir, Albert C. Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J. Koay
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Motivation The combination of single-cell sequencing with ligand-receptor analysis paves the way for the characterization of cell communication events in complex tissues. In particular, directed weighted graphs stand out as a natural representation of cell-cell communication events. However, current computational methods cannot analyze sample-specific cell-cell communication events, as measured in single-cell data produced in large patient cohorts. Cohort-based cell-cell communication analysis presents many challenges, such as the non-linear nature of cell-cell communication and the high variability presented by the patient-specific single-cell RNAseq datasets. Results Here, we present scACCorDiON (single-cell Analysis of Cell-Cell Communication in Disease clusters using Optimal transport in Directed Networks), an optimal transport algorithm exploring node distances on the Markov Chain as the ground metric between directed weighted graphs. Additionally, we derive a k-barycenter algorithm using the Wasserstein-based distance, which is able to cluster directed weighted graphs. We compare our approach with competing methods in several large cohorts of scRNA-seq data. Our results show that scACCorDiON can predict clusters better, matching the disease status of samples. Moreover, we show that barycenters provide a robust and explainable representation of cell cell communication events related to the detected clusters. We also provide a case study of pancreas adenocarcinoma, where scACCorDion detects a sub-cluster of disease samples associated with changes in the tumor microenvironment. Availability The code of scACCorDiON is available at https://scaccordion.readthedocs.io/en/latest Contact ivan.costa@rwth-aachen.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12408229b87ae8ceac3d6b2906ea4bd12adc6c6e" target='_blank'>
              scACCorDiON: A clustering approach for explainable patient level cell cell communication graph analysis
              </a>
            </td>
          <td>
            J. Nagai, Michael T. Schaub, Ivan G.Costa
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The expression of PD-L1 on tumor cells (TCs) is used as an immunotherapy biomarker in lung cancer, but heterogeneous intratumoral expression is often observed. Using a Digital Spatial Profiling, we performed proteomic and whole-transcriptomic analyses of TCs and immune cells (ICs) in spatially matched areas based on tumor PD-L1 expression and the status of the immune microenvironment. Our findings were validated using immunohistochemistry, The Cancer Genome Atlas, and immunotherapy cohorts. ICs in areas with high PD-L1 expression on TCs showed more features indicative of immunosuppression and exhaustion than ICs in areas with low PD-L1 expression on TCs. TCs highly expressing PD-L1 within immune-inflamed (IF) areas show up-regulation of pro-inflammatory processes, whereas TCs highly expressing PD-L1 within immune-deficient (ID) areas show up-regulation of various metabolic processes. Using differentially expressed genes of TCs between the IF and ID areas, we identified a novel prognostic gene signature for lung cancer. In addition, a high ratio of CD8+ cells to M2 macrophages was found to predict favorable outcomes in patients with PD-L1-expressing lung cancer after immune checkpoint inhibitor therapy. This study demonstrates that TCs and ICs have distinct spatial features within the tumor microenvironment that are related to tumor PD-L1 expression and IC infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc0a9207938d739ad9fb6635845dbbb9be7ec54" target='_blank'>
              Spatially resolved whole-transcriptomic and proteomic profiling of lung cancer and its immune-microenvironment according to PD-L1 expression.
              </a>
            </td>
          <td>
            Yoon Kyung Jeon, Jaemoon Koh, Dongjoo Lee, Sehui Kim, Seung Geun Song, B. Han, Hyein Jeong, Young A Kim, B. Keam, Se-Hoon Lee, K. Kim, Doo Hyun Chung
          </td>
          <td>2024-09-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b983ff02239e55a5a088e11ea44463f2a191dc" target='_blank'>
              Spatial computational modelling illuminates the role of the tumour microenvironment for treating glioblastoma with immunotherapies
              </a>
            </td>
          <td>
            Blanche Mongeon, Julien Hébert-Doutreloux, Anudeep Surendran, Elham Karimi, B. Fiset, D. Quail, Logan A Walsh, A. Jenner, Morgan Craig
          </td>
          <td>2024-08-18</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Patient-derived organoids (PDOs) have emerged as a powerful tool for modeling human cancers due to their ability to closely mirror in vivo architecture and preserve the genetic landscape of tumors. They offer a crucial platform for studying disease progression and treatment responses. In this study, we present a comprehensive characterization of a unique collection of PDO cultures derived from pancreatic ductal adenocarcinoma (PDAC), a particularly aggressive malignancy often diagnosed in advanced stages with limited treatment options and poor prognosis. Our investigation explores morphological and genetic profiling of 10 PDAC PDOs, specifically identifying distinct drug-response patterns. The morphological characteristics of these PDO cultures were investigated by microscopy, while the mutational status was acquired by panel sequencing. Subsequently, PDOs were treated with a panel of 10 inhibitors, either individually or in combination. The response to these treatments was assessed through live-cell imaging (Incucyte®) to capture and quantify changes in organoid morphology and endpoint measurements of cellular viability using a CellTiter-Glo® assay. Our detailed in vitro drug sensitivity testing revealed a remarkable diversity in the responses of the PDAC PDOs to different chemotherapeutic agents, including those currently under clinical trials. Of notable significance is the depth of understanding we have achieved, allowing us to establish potential correlations between PDO morphology, genetic mutations, and drug reactions. This finding provides a key insight into the complex interplay of these factors and their impact on treatment outcomes, offering valuable insights into understanding individual tumor subtypes' sensitivity or resistance to treatments. The PDOs exhibited a spectrum of cancer-associated mutations, including KRAS, TP53, SMAD4, CDKN2A, and ARID1A, reflecting the heterogeneity observed in PDAC patients. Morphologically, the organoids displayed various growth patterns, ranging from cystic and regular structures to filled or unfilled, granular, dark, and light membrane structures. Drug treatments underscored the diverse and unique drug-response profiles within the PDOs, with some demonstrating sensitivity to multiple treatments. In contrast, others exhibited resistance, highlighting the intricate nature of drug responses within this model. Overall, our findings emphasize the ability of PDAC organoids to faithfully capture the complexity and heterogeneity of the disease, making them promising platforms for guiding personalized oncology approaches. The diverse drug response profiles observed in PDOs advocate leveraging this model to develop precision medicine strategies tailored to patients' characteristics, ultimately paving the way for more effective and personalized treatment regimens in the fight against PDAC.
 Citation Format: Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, Silke Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, Elisabeth Heßmann, Volker Ellenrieder, Philipp Ströbel, Michael Ghadimi. Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a23581700d7eba1d7151f1cd1e85577199cc66e" target='_blank'>
              Abstract A023: Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications
              </a>
            </td>
          <td>
            Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, S. Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, E. Hessmann, V. Ellenrieder, Philipp Ströbel, Michael Ghadimi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Adrenocortical carcinoma (ACC) is a rare and highly heterogeneous disease with a notably poor prognosis due to significant challenges in diagnosis and treatment. Emphasizing on the importance of precision medicine, there is an increasing need for comprehensive genomic resources alongside well-developed experimental models to devise personalized therapeutic strategies. We present ACC_CellMinerCDB, a substantive genomic and drug sensitivity database (available at https://discover.nci.nih.gov/acc_cellminercdb) comprising ACC cell lines, patient-derived xenografts, surgical samples, and responses to more than 2,400 drugs examined by the NCI and National Center for Advancing Translational Sciences. This database exposes shared genomic pathways among ACC cell lines and surgical samples, thus authenticating the cell lines as research models. It also allows exploration of pertinent treatment markers such as MDR-1, SOAT1, MGMT, MMR, and SLFN11 and introduces the potential to repurpose agents like temozolomide for ACC therapy. ACC_CellMinerCDB provides the foundation for exploring larger preclinical ACC models. Significance: ACC_CellMinerCDB, a comprehensive database of cell lines, patient-derived xenografts, surgical samples, and drug responses, reveals shared genomic pathways and treatment-relevant markers in ACC. This resource offers insights into potential therapeutic targets and the opportunity to repurpose existing drugs for ACC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/086d906caed4e618b07f91a9727a62680855db77" target='_blank'>
              A Database Tool Integrating Genomic and Pharmacologic Data from Adrenocortical Carcinoma Cell Lines, PDX, and Patient Samples
              </a>
            </td>
          <td>
            Yashuhiro Arakawa, Fathi Elloumi, Sudhir Varma, Prashant Khandagale, Ukhyun Jo, Suresh Kumar, Nitin Roper, William C Reinhold, Robert W. Robey, Naoko Takebe, Michael M. Gottesman, Craig J Thomas, Valentina Boeva, A. Berruti, A. Abate, M. Tamburello, Sandra Sigala, C. Hantel, Isabel Weigand, M. Wierman, K. Kiseljak-Vassiliades, Jaydira Del Rivero, Y. Pommier
          </td>
          <td>2024-08-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="145 Background: Endemic NPC is invariably linked to Epstein-Barr virus infection, and often presents as locoregionally-advanced (LA-) disease. Nonetheless, it is acknowledged that LA-NPC is clinically heterogeneous, with disparate clinical presentations of ascending (A) and descending (D) subtypes that are characterized by T3-4N0-1 and T1-2N2-3 disease, respectively. Here, we investigated the TIME between these distinct NPC phenotypes using whole transcriptome sequencing and digital pathology. Methods: We utilized a cohort of 111 patients with NPC from 2 institutions – NCCS (n=68) and JXCH (n=43) as test and validation datasets, respectively. Whole transcriptome profiling using RNAseq (Illumina, CA) was performed in the NCCS cohort, while spatial analyses of hematoxylin and eosin-stained whole slide images (WSIs) were undertaken in the NCCS cohort (35 A- and 33 D-subtypes), and validated in the JXCH cohort (26 A- and 17 D-subtypes). A customized pipeline was developed to analyze the spatial relationships between tumor-rich and immune-rich regions (tiles) within the WSIs. Mean distances between each tumor-rich and its nearest immune-rich tiles were computed for each WSI. Highest and lowest quartiles of the mean distances in the test cohort were used as thresholds to classify NPC tumors into 3 immune classes of differing immune cell density. Results: Transcriptomic profiling revealed an enrichment of immune-related genesets and differentially expressed genes in D- relative to A-subtype tumors. This corresponded to observations of higher overall immune, T and B cell scores in D- than A-subtypes by deconvolution analyses (ESTIMATE and CIBERSORT). Through spatial analyses of our WSIs, we defined three immune classes – immune-dense, -sparse, and -desert, which corresponded to mean distances of <88 μm, 88-208 μm, and >208 μm between tumour- to immune-rich tiles, respectively. D-subtype NPC had significantly more immune-rich regions compared with A-subtypes in both NCCS and JXCH cohorts ( P=0.02 [NCCS]; P=0.01 [JXCH]). This corresponded to shorter median distances between tumor-rich to immune-rich tiles in D- than A-subtypes (103 μm vs 200 μm, P=0.01 [NCCS]; 123 μm vs250 μm, P=0.01 [JXCH]). Based on our novel spatial-based classification, we observed a higher proportion of immune dense tumors in D- than A-subtypes in the NCCS cohort, with only 3/33 (9%) D-subtypes being classified as immune desert. Using the same distance thresholds, we derived a consistent trend in the JXCH cohort, where a higher proportion of D-subtypes were classified as immune dense and sparse than A-subtypes (59% vs 31%, P=0.01). Conclusions: Our results support a model where the TIME delineates the biology underpinning the disparate clinical presentations of distinct A- and D-subtypes of LA-NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24cc4cf674b8d9159fd67ca9e00f76cac965bd2a" target='_blank'>
              Effect of tumor-immune microenvironment (TIME) on disparate ascending (A) and descending (D) subtypes of nasopharyngeal carcinoma (NPC).
              </a>
            </td>
          <td>
            E. Yeo, E. Ong, Boon Hao Hong, D. W. Lim, Joe Yeong, Sze Huey Tan, Melvin L K Chua, Jacqueline Siok Gek Hwang, Tianzhu Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869c734206e668652b18e8e50d0afc69d2273b39" target='_blank'>
              Intratumoral microbiota of pancreatic ductal adenocarcinoma impact patient prognosis by influencing tumor microenvironment
              </a>
            </td>
          <td>
            Jingze Leng, Hengyi Xu, Xiaoyu Liu, Yufan Yang, Chun Ning, Lejia Sun, Jiangming Qu, Xindi Ke, Xun Lan
          </td>
          <td>2024-09-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Immunotherapy response is associated with the presence of conventional dendritic cells (cDCs). cDC type 1 (cDC1) is critically important for CD8+ T cell activation, cDC type 2 (cDC2) regulates CD4+ T cell responses, and mature regulatory cDCs may dampen T cell responses in the tumor microenvironment (TME). However, we lack a clear understanding of cDC distribution in the human TME, cDC prevalence in metastatic sites, and cDC differences in early- versus late-stage disease. Rapid autopsy specimens of 10 patients with lung adenocarcinoma were evaluated to detect cDCs and immune cells via multiplex immunofluorescence using 18 markers and 42 tumors. First, we found that T cells, cDC1, and cDC2 were confined to stroma, whereas mature regulatory DCs were enriched in tumor, suggesting unique localization-specific functions. Second, lung and lymph node tumors were more enriched in T cells and cDCs than liver tumors, underscoring differences in the TME of metastatic sites. Third, although the proportion of T cells and cDC1 did not differ in different stages, an increase in the proportion of cDC2 and macrophages in late stage suggests potential differences in regulation of T cell responses in different stages. Collectively, these findings provide new, to our knowledge, insights into cDC biology in human cancer that may have important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8512ba6cd3679e5a0f0f3d1f8e432ba86900fbeb" target='_blank'>
              Rapid Autopsy to Define Dendritic Cell Spatial Distribution and T Cell Association in Lung Adenocarcinoma.
              </a>
            </td>
          <td>
            Hilal Ozakinci, Xiaofei Song, Gina S Nazario, Thomas Lila, Benjamin J Chen, Tyler Simpson, Jonathan V Nguyen, Carlos M Moran Segura, Zachary Thompson, Ram Thapa, Trevor A Rose, E. Haura, B. Pellini, Xiaoqing Yu, B. Ruffell, Dung-tsa Chen, Theresa A. Boyle, Amer A Beg
          </td>
          <td>2024-08-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is especially lethal among cancers because current treatments are few and ineffective. One key stromal factor thought to contribute to the dismal prognosis of PDAC is the presence of cancer-associated fibroblasts (CAFs). However, CAF-targeting therapeutic strategies have so far failed. Recently, providing an explanation for this apparent contrasting observations, phenotypically distinct PDAC CAF subtypes have been discovered. We previously described two main PDAC CAF populations that are defined by distinct targetable molecular mechanisms: TGF-beta-dependent myofibroblastic CAFs (myCAFs) and IL-1-dependent inflammatory CAFs (iCAFs) (J Exp Med 2017). We also demonstrated how the activation of one pathway (i.e., TGF-beta signaling) blocks the activation of another (i.e., IL-1 signaling), leading to this CAF heterogeneity (Cancer Discovery 2019). Despite our progress, the roles of distinct PDAC CAF subtypes remain to be largely elucidated. Indeed, the ability of myCAFs and iCAFs to at least partially interconvert and the lack of genetically engineered mouse models (GEMMs) for their in vivo manipulation have so far hindered careful investigation of their functions in PDAC progression. To overcome these challenges, we have engineered new mouse models and ‘CAF-locked’ fibroblast lines that enable to alter CAF composition in vitro and in vivo. Leveraging these new models, we are starting to understand the respective crosstalk of iCAFs and myCAFs with each other, with PDAC malignant cells, with immune cells and with other stromal cells. Improving PDAC survival for all patients will likely require a better understanding of the complex nature of the tumor microenvironment in these heterogeneous cancers. While further genetic manipulation of myCAF (e.g., EGFR-activated myCAFs – Cancer Cell 2024) and iCAF subsets will be required to achieve that, our studies are crucial to begin to decipher this complexity.
 Citation Format: Giulia Biffi. Functional heterogeneity of fibroblast populations in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr IA-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5159b911b55b7d105859836df1e2efc50a3e69f" target='_blank'>
              Abstract IA-03: Functional heterogeneity of fibroblast populations in pancreatic cancer
              </a>
            </td>
          <td>
            Giulia Biffi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Malignant rhabdoid tumors (MRT) are rare aggressive tumors of infancy characterized by the biallelic inactivation of SMARCB1. Their lineage of origin remains uncertain, but increasing evidence suggest a neural-crest origin. By analyzing the transcriptome of MRT occurring in peripheral and cranial nerves, we observe that their expression profiling was not distinct from any other tumor localization. We therefore investigated whether the development of Smarcb1-deficient tumors emerging from embryonic precursors of peripheral nerves in mice could recapitulate human MRT. we generated conditional Smarcb1 knockout in the Prss56-Cre mouse strain thereby targeting boundary cap cells which are neural crest derived Schwann cell precursor. All Smarcb1Flox/Flox;Prss56Cre+/+ showed ataxia within a median of two months, due to the development of massive intracranial tumor in the ponto-cerebellum angle. Smarcb1Flox/Flox;Prss56Cre+/- mainly developed paraplegia related to nerve root tumors, within a median delay of 7 months. Interestingly, transcriptome profiling confirmed the good correlation between mouse peripheral nerve tumors and human MRT; intracranial tumors in this new model correlated with MYC-ATRTs but with some specific features not observed in their human counterparts. All tumors harbored a significant immune infiltration, with some clonal T-cell expansion. Altogether, the Smarcb1Flox/Flox;Prss56-Cre mouse tumors offer a new model for human MRT and promising perspectives for immune-based innovative treatments.
 Citation Format: Zhi-Yan HAN, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, Arnault Tauziède-Espariat, Kevin Beccaria, Julien Masliah-Planchon, Christine Bourneix, Delphine Guillemot, Stéphanie Reynaud, Gaelle Pierron, Maëva Veyssiere, Owen Hoare, Céline Chauvin, Grégory Thomson, Didier Surdez, Pascale Varlet, Christelle Dufour, Volodia Dangouloff-Ros, Joshua J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, Franck Bourdeaut. Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624aa14765b3c481a1fc38ac19c8ad7677137eda" target='_blank'>
              Abstract A006 Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors
              </a>
            </td>
          <td>
            Zhi-Yan Han, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, A. Tauziède-Espariat, K. Beccaria, J. Masliah-Planchon, C. Bourneix, D. Guillemot, S. Reynaud, G. Pierron, Maëva Veyssiere, Owen Hoare, C. Chauvin, Grégory Thomson, D. Surdez, Pascale Varlet, Christelle Dufour, V. Dangouloff-Ros, J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim U. Rashid, Gary L. Johnson, J. J. Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Survivors of pediatric cancer face lifelong battles with severe morbidities, including a significant risk of recurrence. Pre-existing genetic variation within primary tumors give certain cell populations an evolutionary advantage, increasing the likelihood of treatment resistance and relapse. The rarity of pediatric cancer and limited clinical metadata have restricted comprehensive studies on chemoresistance development. I aim to characterize the timing of chemoresistance in common childhood cancers. I developed a pipeline combining clonal evolution reconstruction with mutational signature extraction at the subpopulation level to elucidate changes in mutational processes over time. Analyzing 1,886 tumor genomes from 10 pediatric cancer cohorts revealed a common pattern of parallel evolution in most pediatric cancers, regardless of tissue origin. After de novo signature extraction, mutational signatures linked to specific cancer treatments were detected in post-treatment tumor samples. In a subset of 611 samples with detailed clinical data, mutational signatures associated with alkylating agents like cisplatin were found in 37.8% of samples, accounting for a mean 30.6% of the mutation burden, whereas antimetabolite agents like fluorouracil were found in 14.9% of samples and contributed 13.8% of the mutation burden. The higher prevalence and burden of alkylating agent signatures suggested a mechanism of direct and continuous DNA damage, while antimetabolite agents depend on active cell cycling. The specific chemotherapy’s mechanism of action dictates its genomic presentation in treated tumors. When placed in the context of the phylogenetic tree, we identified specific subpopulations with therapy signatures, indicating subpopulation-level resistance. In cases with multiple tumor samples, resistant subpopulations in recurrences or metastases were traced to early ancestral clones already present at diagnosis, suggesting that early-stage chemotherapy-resistant subpopulations expanded post-treatment. Finally, the timing of resistance phenotype development varies between pediatric cancer types. In osteosarcoma, pulmonary metastases separate and expand independently before developing cisplatin-associated signatures. In contrast, gliomas show therapy effects earlier; shared subpopulations exhibit therapy-associated signatures before the bifurcation and expansion of progressive subpopulations. Thus, the mechanisms responsible for metastasis likely differ from those driving resistance in osteosarcoma, while in glioma, the effect of therapy coincides with transformation to high-grade subtypes. In summary, investigating mutational signatures at the subclonal level unveils new insights into the clonal dynamics and development of chemoresistance in pediatric cancers, contributing to our understanding of cancer evolution and advancing the field of personalized medicine.
 Citation Format: Sasha Blay, Mehdi Layeghifard, Scott Davidson, David Chen, Astra Schwertschkow, Vijay Ramaswamy, Michael Taylor, Elli Papaemmanuil, Anita Villani, David Malkin, Ludmil Alexandrov, Mark Cowley, Adam Shlien. Timing the development of chemoresistance in relapsed pediatric cancer with mutational signatures [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b0df81e16c18b865037513faadc43adba06012" target='_blank'>
              Abstract B001: Timing the development of chemoresistance in relapsed pediatric cancer with mutational signatures
              </a>
            </td>
          <td>
            S. Blay, M. Layeghifard, Scott Davidson, David Chen, Astra Schwertschkow, Vijay Ramaswamy, Michael D. Taylor, E. Papaemmanuil, A. Villani, D. Malkin, Ludmil B. Alexandrov, Mark J. Cowley, A. Shlien
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel R. Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Background Osteosarcoma (OS) survival rates and outcome have not improved in 50 years since the advent of modern chemotherapeutics. Thus, there is a critical need for an improved understanding of the tumor microenvironment to identify better therapies. Extracellular matrix (ECM) deposition and hypoxia are known to abrogate the efficacy of various chemical and cell‐based therapeutics. Here, we aim to mechanistically investigate the combinatorial effects of hypoxia and matrix deposition with the use of OS spheroids. Methods We use two murine OS cell lines with differential metastatic potential to form spheroids. We form spheroids of two sizes, use ascorbate‐2‐phosphate supplementation to enhance ECM deposition, and study cell response under standard (21% O2) and physiologic (5% O2) oxygen tensions. Finally, we examine chemotherapeutic responses to doxorubicin treatment. Results ECM production and oxygen tension are key determinants of spheroid size through cell organization based on nutrient and oxygen distribution. Interestingly, highly metastatic OS is more susceptible to chemotherapeutics compared to less metastatic OS when matrix production increases. Together, these data suggest that dynamic interactions between ECM production and oxygen diffusion may result in distinct chemotherapeutic responses despite inherent tumor aggressiveness. Conclusion This work establishes OS spheroids as a valuable tool for early OS tumor formation investigation and holds potential for novel therapeutic target and prognostic indicator discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5e2fb20e2a4e399455c45af17440bdc5ed9071" target='_blank'>
              Extracellular matrix production and oxygen diffusion regulate chemotherapeutic response in osteosarcoma spheroids
              </a>
            </td>
          <td>
            Isabel S Sagheb, Thomas P Coonan, R. L. Randall, Katherine H Griffin, J. K. Leach
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is characterized by several molecular alterations that contribute to its development and progression. These alterations include the epidermal growth factor receptor (EGFR), anaplastic lymphoma kinase (ALK), human epidermal growth factor receptor 2 (HER2), and mesenchymal–epithelial transition factor (c-MET). Among these, the hepatocyte growth factor (HGF)/c-MET signaling pathway plays a crucial role in NSCLC. In spite of this, the involvement of the HGF/c-MET signaling axis in remodeling the tumor microenvironment (TME) remains relatively unexplored. This review explores the biological functions of the HGF/c-MET signaling pathway in both normal and cancerous cells, examining its multifaceted roles in the NSCLC tumor microenvironment, including tumor cell proliferation, migration and invasion, angiogenesis, and immune evasion. Furthermore, we summarize the current progress and clinical applications of MET-targeted therapies in NSCLC and discuss future research directions, such as the development of novel MET inhibitors and the potential of combination immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7956b1a88c0d882a052d3a0639ac987956fb9aa3" target='_blank'>
              Unveiling the Role of HGF/c-Met Signaling in Non-Small Cell Lung Cancer Tumor Microenvironment
              </a>
            </td>
          <td>
            Shuxi Yao, Xinyue Liu, Ya-ling Feng, Yiming Li, Xiangtian Xiao, Yuelin Han, Shu Xia
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="25 Background: The brain tumor immune microenvironment (TiME) is characterised by suppression of tumor-infiltrating leukocytes. The lymphocyte niche activity is particularly downregulated within this TiME milieu. Methods: Here, using data-driven approaches in flow cytometry and RNA sequencing analysis, we aimed at dissecting leukocyte cell types infiltrating human brain tumor TiME. Results: Unsupervised clustering provided T lymphocyte subsets including double-negative (CD3+CD4- CD8-) T cells (DNT). Our DNT phenotype was validated with an independent dataset, suggesting DNT to be compatible with γδ T cell phenotype. In our cohort, myeloid immune cell frequencies were associated with malignancy type as previously reported, while a gliosarcoma notably presented unexpectedly high lymphocyte frequencies. Consistently, double-positive (CD4+ CD8+) T cells (DPT) had higher frequency in the gliosarcoma than other tumors. The flow cytometry results were supported by transcriptome patterns and the activity of immune-related genes in these tumors. Conclusions: Our findings feature the benefit of data-driven approaches for brain TiME lymphocyte analysis, and show that primary brain tumors can entail atypical lymphocyte subsets like DNTs and DPT cells. Unbiased lymphocyte characterisation can potentially outline novel targets for immune checkpoint blockade therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb1ae6998ef7cf2b065eaabbce0c558410319b8" target='_blank'>
              Unsupervised clustering of brain tumor leukocytes to reveal atypical lymphoid cell subsets.
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8fc05c94d1bb030c65261bd89103d4e78395190" target='_blank'>
              Caspase-1-dependent spatiality in triple-negative breast cancer and response to immunotherapy
              </a>
            </td>
          <td>
            Weiyue Zheng, Wanda Marini, K. Murakami, Valentin Sotov, Marcus O Butler, Chiara Gorrini, P. Ohashi, M. Reedijk
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Neuroblastoma is a devastating disease accounting for ~15% of all childhood cancer deaths. Collagen content and fiber association within the tumor stroma influence tumor progression and metastasis. High expression levels of collagen receptor kinase, Discoidin domain receptor II (DDR2), are associated with poor survival of neuroblastoma patients. Additionally, cancer cells generate and sustain mechanical forces within their enviroment as a part of their normal physiology. Despite this, whether collagen activated DDR2 signaling dysregulate these migration forces is still elusive. To address these questions, a shRNA DDR2 knockdown neuroblastoma cell line (SH-SY5Y) was engineered to evaluate the consequence of DDR2 on cellular mechanics. Atomic force microscopy and traction force microscopy were utlizing to unveil the biophysical altercations. DDR2 down-regulation was found to significantly reduce proliferation, cell stiffness, and cellular elongation. Aditionally, DDR2 down-regulated cells had decreased traction forces when plated on collagen coated elastic substrates. Together, these results highlight the important role that DDR2 has in reducing migration mechanics in neuroblastoma and might be a promising target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f707587725fd2ca416d376c5d57d87998251580c" target='_blank'>
              Investigation of cell mechanics and migration on DDR2-expressing neuroblastoma cell line
              </a>
            </td>
          <td>
            Theadora Vessella, Esteban J. Rozen, Jason Shohet, Qi Wen, Hong Susan Zhou
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Translocation renal cell carcinoma (tRCC) is an aggressive subtype of kidney cancer driven by TFE3 gene fusions, which act via poorly characterized downstream mechanisms. Here we report that TFE3 fusions transcriptionally rewire tRCCs toward oxidative phosphorylation (OXPHOS), contrasting with the highly glycolytic metabolism of most other renal cancers. This TFE3 fusion-driven OXPHOS program, together with heightened glutathione levels found in renal cancers, renders tRCCs sensitive to reductive stress – a metabolic stress state induced by an imbalance of reducing equivalents. Genome-scale CRISPR screening identifies tRCC-selective vulnerabilities linked to this metabolic state, including EGLN1, which hydroxylates HIF-1α and targets it for proteolysis. Inhibition of EGLN1 compromises tRCC cell growth by stabilizing HIF-1a and promoting metabolic reprogramming away from OXPHOS, thus representing a vulnerability to OXPHOS-dependent tRCC cells. Our study defines a distinctive tRCC-essential metabolic program driven by TFE3 fusions and nominates EGLN1 inhibition as a therapeutic strategy to counteract fusion-induced metabolic rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2568e01eb416e6b7b22ba26a8e1971911ede6d44" target='_blank'>
              TFE3 fusions direct an oncogenic transcriptional program that drives OXPHOS and unveils vulnerabilities in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Jiao Li, Kaimeng Huang, Fiona McBride, Ananthan Sadagopan, Daniel S. Gallant, Meha Thakur, Prateek Khanna, Bingchen Li, Maolin Ge, Cary N. Weiss, M. Achom, Qingru Xu, Kun Huang, Birgitta A. Ryback, Miao Gui, L. Bar-Peled, S. Viswanathan
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Introduction Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood. Methods We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques. Results We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001). Conclusion We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
 Citation Format: Adam Bryce, Stephan Dreyer, Fieke Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang. Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179192d42336b982da176e9aa5fe870344303cba" target='_blank'>
              Abstract A053: Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: One of the most significant unmet clinical challenges in paediatric oncology is the development of novel therapeutic strategies for recurrent medulloblastoma (R-MB). MYC-driven MBs are defined as classically cold tumours with a low incidence of infiltrating immune cells, resulting in a therapeutic challenge. The immune landscape of recurrent medulloblastoma is yet to explored. Understanding the complex interplay between the immune system and tumour biology is pivotal for comprehending disease progression and devising tailored therapeutic strategies that consider the distinct molecular and immunological profiles of both primary and recurrent MYC-driven tumours. The identification of cell-cell communications, particularly ligand–receptor pairs, allows for the inference of significant intercellular communications based on the expression of corresponding genes. Consequently, we hypothesised that the most influential cell-cell interactions within the tumour immune microenvironment (TME) of MYC-driven MB could unveil prevalent immune-suppressive interactions and potential vulnerabilities for therapeutic exploration. Methods/Results: Paired primary-recurrent bulk RNA-sequencing data, confirmed myeloid cells as the most infiltrating immune cell type in group3-MB and group4-MB. Comprehensive spatial phenotypic and cell-cell communication analyses corroborated this discovery, validating an increased incidence of macrophages in the matched-recurrent tumours and phenotypic markers of advanced immunosuppression. Subsequently, we used innovative algorithms for 10X MB single-cell data to predict interactions between tumour-cell ligands and immune-cell receptors within the TME; macrophages emerged as the core immune-cells involved in interactions throughout the TMEs, with the most significant ligand-receptor interaction and inflammatory response between MIF and CD74.In-depth immunohistochemistry analyses of primary and recurrent group3 and group4 tumours, and exhaustive tissue microarrays demonstrated expression of both CD74 and MIF, with limited expression of CD74 within the brain. To investigate the therapeutic potential of CD74, we developed recurrent, immune competent MYC-driven medulloblastoma mouse models. Comprehensive deconvolution analysis confirmed the TME integrity of our models to mirror that of the human disease. Locoregional delivery and repeat dosing of a bioactive-CD74 peptide demonstrated significant tumour reduction in our immune-competent mouse models, demonstrating the impact of our prediction algorithm and significant therapeutic potential of targeting the CD74-MIF axis in MYC-driven primary and recurrent MB. Conclusions: Essential cellular interactions and targetable therapeutic vulnerabilities have been identified in the tumour-microenvironment of MYC-driven primary-recurrent MB.
 Citation Format: Ben Draper, Dean Thompson, Alaide Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan. Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B063.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d2ace9063c026bc5c1042bf3f0c9b788241cdc" target='_blank'>
              Abstract B063: Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Benjamin Draper, Dean Thompson, A. Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Black persons have the highest mortality rates of both cervical cancer (CC) and pancreatic ductal adenocarcinoma (PDAC) in the United States. Differences in access to quality healthcare and socioeconomic status contribute to these cancer health disparities as well as ancestry and biology. In other cancer types, differences in the inflammatory tumor landscape across races and ancestral populations have been reported. However, the inflammatory tumor microenvironment (TME) as it relates to CC and PDAC health disparities is not well defined. Black patients are often underrepresented in PDAC studies. Meanwhile, CC health disparities studies primarily investigate socioeconomic status, screening, and vaccination rates. Therefore, we aim to characterize the inflammatory TME in these two cancer types with the long-term goal of informing prevention, prognostic, and therapeutic strategies. Clinicopathological characteristics were compared between Black (n = 144) and White (n = 198) patients with CC who received care at Johns Hopkins Medicine between 1995 and 2019. Surgically resected formalin-fixed paraffin embedded (FFPE) CC tissue samples were obtained from Non-Hispanic Black (n = 10), Non-Hispanic White (n = 10), Hispanic Black (n = 5), and Hispanic White (n = 5) women matched on grade, age, and subtype. Tissue microarrays (TMAs) were constructed using archival surgically resected FFPE PDAC tissues from Black (n = 45) and White (n = 45) patients, matched on sex, age, stage, and time of surgery. Multiplexing immunohistochemistry (mIHC) was used to assess markers for i) T cells (CD3, CD8, CD4, FoxP3), M2 macrophages (CD163), mast cells (Tryptase), and fibroblasts (FAP) in PDAC tissues and ii) B cells (CD79A), T cells (CD3, CD8, CD4, FoxP3), plasma cells (CD138/syndecan-1), programmed death-ligand 1 protein (PD-L1), macrophages (CD68, CD163), natural killer cells (NKP46/CD56/CD16), neutrophils (CD66ce), and mast cells (Tryptase) in CC tissues. HALO Image Analysis Platform (version 3.6.4134.137; Indica Labs) was used to quantify marker expression. GraphPad Prism Software (version 10.0.03), was used for graphing and statistical analysis. Squamous cell CC was the most frequent histological subtype in both Black (68%) and White (44%) women. Progression through treatment was more common among Black women (24%) compared to White women (7%). CD163+ M2 macrophages were significantly increased in PDAC tumors compared to benign tissues among Black patients (p = 0.0435), and in tumors from Black women compared to White women (p = 0.0067). FAP+ fibroblasts were also significantly increased in PDAC tumors compared to benign tissues among Black patients (p = 0.0001) and in tumors from Black women compared to White women (p = 0.0013). CD8+ cytotoxic T cells and FoxP3+ regulatory T cells were comparable between races. We observed differences in macrophage and fibroblast distribution between races supporting the effort for future studies that examine the significance of characterizing the TME to address cancer health disparities.
 Citation Format: Jelani Jarrett, Jae W Lee, Ali Dbouk, Vahinipriya Manoharan, Kennedy Rains, Terri Mason, Kimberly Levinson, Laura D Wood, Michael G Goggins, Janielle P Maynard. Characterization of the inflammatory tumor microenvironment and its association with cancer health disparities [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C120.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e342f00683797e507a82de7b3a64c9da9cf1e8e9" target='_blank'>
              Abstract C120: Characterization of the inflammatory tumor microenvironment and its association with cancer health disparities
              </a>
            </td>
          <td>
            Jelani Jarrett, Jae W Lee, Ali Dbouk, Vahinipriya Manoharan, Kennedy Rains, Terri Mason, Kimberly Levinson, Laura D. Wood, Michael G. Goggins, J. Maynard
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>11</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>