<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-30 06:32:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa5087cd6813583e8724b35bca919eb32b35df3e" target='_blank'>
              Automated spatial omics landscape analysis approach reveals novel tissue architectures in ulcerative colitis
              </a>
            </td>
          <td>
            Derek R Holman, Samuel J S Rubin, Mariusz Ferenc, Elizabeth A Holman, Alexander N Koron, Robel Daniel, Brigid Boland, Garry P Nolan, John T Chang, Stephan Rogalla
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, S. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PDAC) is a rapidly progressing cancer that responds poorly to immunotherapies. Intratumoral tertiary lymphoid structures (TLS) have been associated with rare long-term PDAC survivors, but the role of TLS in PDAC and their spatial relationships within the context of the broader tumor microenvironment remain unknown. We generated a spatial multi-omics atlas encompassing 26 PDAC tumors from patients treated with combination immunotherapies. Using machine learning-enabled H&E image classification models and unsupervised gene expression matrix factorization methods for spatial transcriptomics, we characterized cellular states within TLS niches spanning across distinct morphologies and immunotherapies. Unsupervised learning generated a TLS-specific spatial gene expression signature that significantly associates with improved survival in PDAC patients. These analyses demonstrate TLS-associated intratumoral B cell maturation in pathological responders, confirmed with spatial proteomics and BCR profiling. Our study also identifies spatial features of pathologic immune responses, revealing TLS maturation colocalizing with IgG/IgA distribution and extracellular matrix remodeling. GRAPHICAL ABSTRACT HIGHLIGHTS Integrated multi-modal spatial profiling of human PDAC tumors from neoadjuvant immunotherapy clinical trials reveal diverse spatial niches enriched in TLS. TLS maturity is influenced by tumor location and the cellular neighborhoods in which TLS immune cells are recruited. Unsupervised machine learning of genome-wide signatures on spatial transcriptomics data characterizes the TLS-enriched TME and associates TLS transcriptomes with survival outcomes in PDAC. Interactions of spatially variable gene expression patterns showed TLS maturation is coupled with immunoglobulin distribution and ECM remodeling in pathologic responders. Intratumoral plasma cell and immunoglobin gene expression spatial dynamics demonstrate trafficking of TLS-driven humoral immunity in the PDAC TME. Significance We report a spatial multi-omics atlas of PDAC tumors from a series of immunotherapy neoadjuvant clinical trials. Intratumorally, pathologic responders exhibit mature TLS that propagate plasma cells into malignant niches. Our findings offer insights on the role of TLS-associated humoral immunity and stromal remodeling during immunotherapy treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b4043873d18d3b758a620d9a4f90e32bea7034" target='_blank'>
              Spatial multi-omics reveal intratumoral humoral immunity niches associated with tertiary lymphoid structures in pancreatic cancer immunotherapy pathologic responders
              </a>
            </td>
          <td>
            Dimitri N. Sidiropoulos, Sarah M. Shin, Meredith Wetzel, Alexander Girgis, Daniel Bergman, Ludmila Danilova, Susheel Perikala, Daniel H. Shu, Janelle M. Montagne, Atul Deshpande, James Leatherman, Lucie Dequiedt, Victoria Jacobs, Aleksandra Ogurtsova, Guanglan Mo, Xuan Yuan, Dmitrijs Lvovs, G. Stein-O’Brien, M. Yarchoan, Qingfeng Zhu, Elizabeth I. Harper, A. Weeraratna, A. Kiemen, Elizabeth M. Jaffee, Lei Zheng, W. Ho, Robert A Anders, E. Fertig, Lusciane T. Kagohara
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) resides in an immune rich microenvironment, yet, immune-based therapies have faltered in eliciting durable responses. Bridging this paradox requires a comprehensive understanding of leukemic interactions within the bone marrow microenvironment. We optimized a high-throughput tissue-microarray based pipeline for high-plex spatial immunofluorescence and mass cytometry imaging on a single slide, capturing immune, tumor, and structural components. Using unbiased clustering on the spatial K function, we unveiled the presence of tertiary lymphoid-like aggregates in bone marrow which we validated using spatial transcriptomics and an independent proteomics approach. We then found validated TLS signatures predictive of outcomes in AML using an integrated public 480 patient transcriptomic dataset. By harnessing high-plex spatial proteomics, we open the possibility of discovering of novel structures and interactions that underpin leukemic immune response. Further, our study’s methodologies and resources can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1b1bad3b36f52c6886c710f3a8b1e66619ef44" target='_blank'>
              Multimodal Spatial Proteomic Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, Sammy Ferri-Borgongo, Akshay Basi, Javier A. Gomez, Jessica L Root, P. Reville, Padmanee Sharma, Sreyashi Basu, Andres E. Quesada, Carlos Bueso-Ramos, T. Manshouri, Miriam Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The relative abundance of cancer-associated fibroblast (CAF) subtypes influences a tumor’s response to treatment, especially immunotherapy. However, the extent to which the underlying tumor composition associates with CAF subtype-specific gene expression is unclear. Here, we describe an interpretable machine learning (ML) approach, additive multiple instance learning (aMIL), to predict bulk gene expression signatures from H&E-stained whole slide images (WSI), focusing on an immunosuppressive LRRC15+ CAF-enriched TGFβ-CAF signature. aMIL models accurately predicted TGFβ-CAF across various cancer types. Tissue regions contributing most highly to slide-level predictions of TGFβ-CAF were evaluated by ML models characterizing spatial distributions of diverse cell and tissue types, stromal subtypes, and nuclear morphology. In breast cancer, regions contributing most to TGFβ-CAF-high predictions (“excitatory”) were localized to cancer stroma with high fibroblast density and mature collagen fibers. Regions contributing most to TGFβ-CAF-low predictions (“inhibitory”) were localized to cancer epithelium and densely inflamed stroma. Fibroblast and lymphocyte nuclear morphology also differed between excitatory and inhibitory regions. Thus, aMIL enables a data-driven link between tissue phenotype and transcription, offering biological interpretability beyond typical black-box models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dfe1e5aad3aff07bf10f8ea7b2dcf18d3178c7" target='_blank'>
              Spatial mapping of immunosuppressive CAF gene signatures in H&E-stained images using additive multiple instance learning
              </a>
            </td>
          <td>
            Miles Markey, Juhyun Kim, Zvi Goldstein, Y. Gerardin, Jacqueline Brosnan-Cashman, Syed Ashar Javed, Dinkar Juyal, Harshith Pagidela, Limin Yu, Bahar Rahsepar, J. Abel, S. Hennek, Archit Khosla, A. Taylor-Weiner, Chintan Parmar
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Analyzing the local microenvironment of tumor cells can provide significant insights into their complex interactions with their cellular surroundings, including immune cells. By quantifying the prevalence and distances of certain immune cells in the vicinity of tumor cells through a neighborhood analysis, patterns may emerge that indicate specific associations between cell populations. Such analyses can reveal important aspects of tumor-immune dynamics, which may inform therapeutic strategies. This method enables an in-depth exploration of spatial interactions among different cell types, which is crucial for research in oncology, immunology, and developmental biology. Results We introduce an R Markdown script called SNAQTM (Single-cell Spatial Neighborhood Analysis and Quantification), which conducts a neighborhood analysis on immunofluorescent images without the need for extensive coding knowledge. As a demonstration, SNAQTM was used to analyze images of pancreatic ductal adenocarcinoma. Samples stained for DAPI, PanCK, CD68, and PD-L1 were segmented and classified using QuPath. The resulting CSV files were exported into RStudio for further analysis and visualization using SNAQTM. Visualizations include plots revealing the cellular composition of neighborhoods around multiple cell types within a customizable radius. Additionally, the analysis includes measuring the distances between cells of certain types relative to others across multiple regions of interest. Availability and implementation The R Markdown files that comprise the SNAQTM algorithm and the input data from this paper are freely available on the web at https://github.com/AryehSilver1/SNAQ. Visual Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/536ae41dfee8df569ec9d770e09c6f0416cb7a5b" target='_blank'>
              Get to know your neighbors with a SNAQ™: A framework for single cell spatial neighborhood analysis in immunohistochemical images
              </a>
            </td>
          <td>
            Aryeh Silver, Avirup Chakraborty, Avinash Pittu, Diana Feier, Miruna Anica, Illeana West, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, A. Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cell states are modulated by intrinsic driving forces such as gene expression noise and extrinsic signals from the tissue microenvironment. The distinction between intrinsic and extrinsic cell state determinants is essential for understanding the regulation of cell fate in tissues during development, homeostasis and disease. The rapidly growing availability of single-cell resolution spatial transcriptomics makes it possible to meet this challenge. However, available computational methods to infer topological tissue domains, spatially variable gene expression, or ligand-receptor interactions are limited in capturing cell state changes driven by crosstalk between individual cell types within the same niche. We present NiCo, a computational framework for integrating single-cell resolution spatial transcriptomics with matched single-cell RNA-sequencing reference data to infer the influence of the spatial niche on the cell state. By applying NiCo to mouse embryogenesis, adult small intestine and liver data, we demonstrate the capacity to predict novel niche interactions that govern cell state variation underlying tissue development and homeostasis. In particular, NiCo predicts a feedback mechanism between Kupffer cells and neighboring stellate cells limiting stellate cell activation in the normal liver. NiCo provides a powerful tool to elucidate tissue architecture and to identify drivers of cellular states in local niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330dd175c406437712c3044cd06a601fe89865f" target='_blank'>
              NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis
              </a>
            </td>
          <td>
            Ankit Agrawal, Stefan Thomann, Sukanya Basu, D. Grün
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Identifying the key molecular pathways that enable metastasis by analyzing the eventual metastatic tumor is challenging because the state of the founder subclone likely changes following metastatic colonization. To address this challenge, we labeled primary mouse pancreatic ductal adenocarcinoma (PDAC) subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative in vivo metastatic potential prospectively. We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of IL-1 pathway genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC metastasis in the IL-1 pathway, including the NF-κB targets Fos and Il23a, and beyond the IL-1 pathway including Myo1b and Tmem40. We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies. This work establishes a strategy for linking cancer cell state to future behavior, reveals novel functional regulators of PDAC metastasis, and establishes a method for scoring human carcinomas based on metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130863ca923658f30d397c218a109254f4ad9093" target='_blank'>
              Identifying a gene signature of metastatic potential by linking pre-metastatic state to ultimate metastatic fate
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Weixiang Fang, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterizing the transcriptional and translational gene expression patterns at the single-cell level within their three-dimensional (3D) tissue context is essential for revealing how genes shape tissue structure and function in health and disease. However, most existing spatial profiling techniques are limited to 5-20 µm thin tissue sections. Here, we developed Deep-STARmap and Deep-RIBOmap, which enable 3D in situ quantification of thousands of gene transcripts and their corresponding translation activities, respectively, within 200-µm thick tissue blocks. This is achieved through scalable probe synthesis, hydrogel embedding with efficient probe anchoring, and robust cDNA crosslinking. We first utilized Deep-STARmap in combination with multicolor fluorescent protein imaging for simultaneous molecular cell typing and 3D neuron morphology tracing in the mouse brain. We also demonstrate that 3D spatial profiling facilitates comprehensive and quantitative analysis of tumor-immune interactions in human skin cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f410940b56b1945c8670449d337f9efc5f10187d" target='_blank'>
              Scalable spatial single-cell transcriptomics and translatomics in 3D thick tissue blocks
              </a>
            </td>
          <td>
            Xin Sui, Jennifer A. Lo, Shuchen Luo, Yichun He, Zefang Tang, Zuwan Lin, Yiming Zhou, Wendy Xueyi Wang, Jia Liu, Xiao Wang
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Faithful genetically engineered in vivo models of medulloblastoma (MB) are currently available only for some molecular subgroups, in keeping with recent studies showing the unique role of human-specific progenitors in the development of G3 and G4 MB subgroups. We generated human cerebellar organoids (CbO) from expanded potential stem cells (EPSC) and characterised in depth their epigenetic and transcriptomic profile. We show the presence of sub compartment-specific cerebellar lineages linked to MB formation, including populations expressing signature genes of G3 and G4 MB cells-of-origin in a 1 month-old CbO. Moreover, we demonstrate that CbO sustains MB cells proliferation and invasion in a 3D co-culture model (CbO-MB) while preserving their molecular identity, and that treatment of CbO-MB with anti-tumour compounds recapitulate the efficacy of in vivo drug testing in PDX models. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418ddee9c7f8cb6b4c5b80d2a93be6711b4eb2cf" target='_blank'>
              Modelling medulloblastoma pathogenesis and treatment in human cerebellar organoids
              </a>
            </td>
          <td>
            Thomas Willott, James G Nicholson, Yunchen Xiao, O. Ogunbiyi, Talisa Mistry, N. Zabet, A. Merve, S. Badodi, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMN) is one of the known precancerous lesions. Patients’ prognosis is aggravated as IPMN transforms into invasive Pancreatic Ductal Adenocarcinoma (PDAC). The molecular mechanisms underlying this progression lack effective experimental models and urgently need to be elaborated. We performed spatial transcriptomics (ST) on fresh tissue samples from the same patient including normal pancreas, high-grade IPMN, and invasive PDAC, and described the step-by-step development of transcriptional landscape including clone evolution and adjacent TME feature variation. Our findings identified the master transcript factors and critical signaling pathways promoting IPMN progression to invasive PDAC. Additionally, both IPMN and PDAC harbored the ELF3, MYC, and KLF4 amplification. The Spatial CNV profile demonstrated significant heterogeneity among PDAC in their spatial distribution compared to IPMN, with seven distinct subclones showing diverse functions, such as hypoxia, oxidative phosphorylation, and epithelial-mesenchymal Transition. We observed a marked shift in the immune landscape, with the depletion of CD4+ T-cells and dendritic cells and an increase in immune-suppressive M2 macrophages in invasive PDAC, indicating a transition to an immune-evasive microenvironment. Additionally, cancer-associated fibroblasts (CAFs), particularly myofibroblastic CAFs, were enriched adjacent to invasive PDAC, suggesting their active role in tumor progression. By leveraging spatial transcriptomic analysis, our study provides comprehensive insights into the intricate molecular landscape that underlies the progression of IPMNs to invasive PDAC. These findings not only enhance our understanding of this complex process but also offer valuable knowledge for early diagnosis and intervention. Highlights Spatial CNV analysis reveals clonal evolution and distinct subclones in PDAC. Key drivers like ELF3, MYC, and KLF4 are amplified in both IPMN and invasive PDAC. Immune landscape shifts from pro-inflammatory in IPMN to immune-evasive in PDAC. Enrichment of myofibroblastic CAFs suggests their role in tumor progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4caecc6532101fd5cd79dde472c73a99ffa9c7b0" target='_blank'>
              Evolution of the Spatial transcriptomic landscape during the progression of high-grade pancreatic intraductal papillary mucinous neoplasms to invasive cancer
              </a>
            </td>
          <td>
            Sirui Peng, Qiangxing Chen, Zixin Chen, Mengling Yao, Yu Cai, Du He, Yu Cai, K. Cheng, Jun Li, H. Cai, P. Gao, Xiafei Gu, Xin Wang, Yongbin Li, Man Zhang, L. Meng, Qi Xia, Panpan Xu, Jin Zhou, Zhong Wu, Bing Peng
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Visium is a spatial sequencing technology that utilizes messenger RNA (mRNA) to spatially map gene expression within tissues. Despite its potential, research utilizing deconvolution tools and exploring microenvironment dynamics remains challenging. We address this gap by benchmarking deconvolution tools across diverse biological contexts, identifying optimal methodologies. Subsequently, we introduce a novel pipeline integrating advanced deconvolution techniques and novel tools for reproducible tissue microenvironment analysis. Through this approach, we uncover intricate immune aggregate biology, highlighting the power of our methodology in unraveling complex biological phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c49f5fb1c2a48f3d75e4d636c39c222777d2a03b" target='_blank'>
              Informing biologically relevant signal from spatial transcriptomic data
              </a>
            </td>
          <td>
            E. Radkevich, D. D'souza, Raphaël Mattiuz, Raphael Merand, Rachel Chen, Jarod Morgenroth-Rebin, K. Angeliadis, Daniella Nelson, Anis Kara, T. Dawson, Igor De Souza, K. Nie, Pauline Hamon, Samarth Hegde, Jesse Boumelha, R. Brody, Sinem Ozbey, C. Hennequin, Di Feng, Jie Dai, Edgar Gonzalez-Kozlova, Zhihong Chen, S. Kim-Schulze, Sacha Gnjatic, M. Merad, Vladimir Roudko
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd111ddc728a7e24176aee9fb44a9a80e5d7261a" target='_blank'>
              Tumor-agnostic cancer therapy using antibodies targeting oncofetal chondroitin sulfate
              </a>
            </td>
          <td>
            E. Vidal-Calvo, Anne Martin-Salazar, Swati Choudhary, R. Dagil, S. Raghavan, Lara Duvnjak, M. A. Nordmaj, T. M. Clausen, Ann Skafte, Jan Oberkofler, Kaituo Wang, Mette Ø. Agerbæk, Caroline Løppke, Amalie M Jørgensen, Daria Ropac, Joana Mujollari, Shona Willis, Agnès Garcias López, Rebecca L. Miller, Richard Karlsson, Felix Goerdeler, Yen-Hsi Chen, Ana R. Colaço, Yong Wang, T. Lavstsen, A. Martowicz, I. Nelepcu, M. Marzban, H. Oo, M. Ørum-Madsen, Yuzhuo Wang, Morten A Nielsen, Henrik Clausen, Michael Wierer, D. Wolf, Ismail Gögenur, T. Theander, Nader Al-Nakouzi, T. Gustavsson, M. Daugaard, A. Salanti
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="In clinical applications, spatial data collected under varying conditions, time points, or patients often lack discernible structural alignment. Computational tools designed to align adjacent tissue sections are unsuited for dealing with this structural heterogeneity. There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy by considering spatial context at various scales. Our approach outperforms existing mapping tools in dealing with heterogeneous samples and is flexible enough to map cells across samples, technologies, resolutions, developmental and regenerative time. Using our approach, we showed spatiotemporal decoupling of cells during development. We even performed alignment for a population of spatial data from cancer patients to identify sub- populations. Our interpretable mapping approach facilitates systemic comparison and analysis of heterogeneous spatial data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3efb8af02c2551620bc2f4404557c0e501a57546" target='_blank'>
              Multi-scale and multi-context interpretable mapping of cell states across heterogenous spatial samples
              </a>
            </td>
          <td>
            P. Martin, Wenqi Wang, Hyobin Kim, Henrietta Holze, Kyoung Jae Won
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The study presents a novel approach for quantifying cellular interactions in digital pathology using deep learning-based image cytometry. Traditional methods struggle with the diversity and heterogeneity of cells within tissues. To address this, we introduce the Spatial Interaction Potential (SIP) and the Co-Localization Index (CLI), leveraging deep learning classification probabilities. SIP assesses the potential for cell-to-cell interactions, similar to an electric field, while CLI incorporates distances between cells, accounting for dynamic cell movements. Our approach enhances traditional methods, providing a more sophisticated analysis of cellular interactions. We validate SIP and CLI through simulations and apply them to colorectal cancer specimens, demonstrating strong correlations with actual biological data. This innovative method offers significant improvements in understanding cellular interactions and has potential applications in various fields of digital pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949ba712b098fead7bc4bf83be6ad4882df65fb2" target='_blank'>
              Novel Methods for Analyzing Cellular Interactions in Deep Learning-Based Image Cytometry: Spatial Interaction Potential and Co-Localization Index
              </a>
            </td>
          <td>
            Toru Nagasaka, Kimihiro Yamashita, Mitsugu Fujita
          </td>
          <td>2024-08-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer for which few effective therapies exist. Immunotherapies specifically are ineffective in pancreatic cancer, in part due to its unique stromal and immune microenvironment. Pancreatic intraepithelial neoplasia, or PanIN, is the main precursor lesion to PDAC. Recently it was discovered that PanINs are remarkably abundant in the grossly normal pancreas, suggesting that the vast majority will never progress to cancer. Here, through construction of 48 samples of cm3-sized human pancreas tissue, we profiled the immune microenvironment of 1,476 PanINs in 3D and at single-cell resolution to better understand the early evolution of the pancreatic tumor microenvironment and to determine how inflammation may play a role in cancer progression. We found that bulk pancreatic inflammation strongly correlates to PanIN cell fraction. We found that the immune response around PanINs is highly heterogeneous, with distinct immune hotspots and cold spots that appear and disappear in a span of tens of microns. Immune hotspots generally mark locations of higher grade of dysplasia or locations near acinar atrophy. The immune composition at these hotspots is dominated by naïve, cytotoxic, and regulatory T cells, cancer associated fibroblasts, and tumor associated macrophages, with little similarity to the immune composition around less-inflamed PanINs. By mapping FOXP3+ cells in 3D, we found that regulatory T cells are present at higher density in larger PanIN lesions compared to smaller PanINs, suggesting that the early initiation of PanINs may not exhibit an immunosuppressive response. This analysis demonstrates that while PanINs are common in the pancreases of most individuals, inflammation may play a pivotal role, both at the bulk and the microscopic scale, in demarcating regions of significance in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/194ae57756ba7688c48e283a00a59550489311e0" target='_blank'>
              3D histology reveals that immune response to pancreatic precancers is heterogeneous and depends on global pancreas structure
              </a>
            </td>
          <td>
            A. Kiemen, Cristina Almagro-Pérez, Valentina Matos, André Forjaz, Alicia M. Braxton, Lucie Dequiedt, Jeeun Parksong, Courtney Cannon, Xuan Yuan, Sarah M. Shin, Jaanvi Mahesh Babu, Elizabeth D. Thompson, T. Cornish, W. Ho, Laura D. Wood, Pei-Hsun Wu, Arrate Muñoz Barrutia, Ralph H. Hruban, Denis Wirtz
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Image-based spatial transcriptomics (ST) offers spatial gene expression profile at the single-cell resolution and provides information to understand intercellular communication that is critical for maintaining tissue development and organ function. Disruption of normal cell-cell interactions (CCI) can lead to disease onset and progression. Current CCI analysis methods face several limitations, including subjection to the number of measured ligand-receptor genes in image-based spatial transcriptomics, limited graph encoding power, inadequate use of spatial information, and low interpretability. Here, we present GITIII, an interpretable self-supervised graph transformer- based language model that treats cells as words (nodes) and their cell neighborhood as a sentence to explore the communications among cells. Enhanced by multilayer perceptron-based distance scaler, physics-informed attention mechanism, and a state-of-the-art, expressive, and lightweight graph transformer model, GITIII infers CCI by investigating how the state of a cell is influenced by the spatial organization, ligand expression, cell types and states of neighboring cells. With its interpretable architecture, GITIII can be used to understand how the sender cell influences target genes in the receiver cell, visualize the spatial pattern and utility of CCI, identify significant CCI networks, perform CCI-informed cell subtyping, and compare CCI strength between disease groups. Applications to four ST datasets from several species, organs, and platforms, GITIII effectively identified and quantitatively interpreted key CCI patterns driving within-sample heterogeneity and disease progression, thus improving our understanding of brain structures, tumor microenvironments, and the interplay among different cell types responding to neighboring CCIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b394f3669761e67f7021ddd2a2bfa57c19a4a4e3" target='_blank'>
              Investigation of pair-wise single-cell interactions by statistically interpreting spatial cell state correlation learned by self-supervised graph inductive bias transformer
              </a>
            </td>
          <td>
            Xiao Xiao
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Castleman disease (CD) is inflammatory lymphoproliferative disorder of unclear etiology. To determine the cellular and molecular basis of CD, we analyzed the spatial proteome of 4,485,009 single cells, transcriptome of 50,117 single nuclei, immune repertoire of 8187 single nuclei, and pathogenic mutations in Unicentric CD, idiopathic Multicentric CD, HHV8-associated MCD, and reactive lymph nodes. CD was characterized by increased non-lymphoid and stromal cells that formed unique microenvironments where they interacted with lymphoid cells. Interaction of activated follicular dendritic cell (FDC) cytoplasmic meshworks with mantle zone B cells was associated with B cell activation and differentiation. VEGF, IL-6, MAPK, and extracellular matrix pathways were elevated in stromal cells of CD. CXCL13+ FDCs, PDGFRA+ T-zone reticular cells (TRC), and ACTA2-positive perivascular reticular cells (PRC) were identified as the predominant source of increased VEGF expression and IL-6 signaling in CD. VEGF expression by FDCs was associated with peri-follicular neovascularization. FDC, TRC and PRC of CD activated JAK-STAT, TGFβ, and MAPK pathways via ligand-receptor interactions involving collagen, integrins, complement components, and VEGF receptors. T, B and plasma cells were polyclonal but showed class-switched and somatically hypermutated IgG1+ plasma cells consistent with stromal cell-driven germinal center activation. In conclusion, our findings show that stromal cell activation and associated B-cell activation and differentiation, neovascularization and stromal remodeling underlie CD and suggest new targets for treatment. Key points Castleman Disease is characterized by activation and proliferation of CXCL13+ FDCs, PDGFRA+ reticular cells, and ACTA2-positive PRCs. VEGF and IL-6 from lymph node stromal cells are associated with B-cell activation and differentiation, endothelial proliferation, and inflammation in CD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6effec836eaf8e7b0862b8cace8add3a480a246e" target='_blank'>
              Spatial and Single Cell Mapping of Castleman Disease Reveals Key Stromal Cell Types and Cytokine Pathways
              </a>
            </td>
          <td>
            David Smith, Anna Eichinger, Andrew Rech, Julia Wang, E. Esteva, Arta Seyedian, Xiaoxu Yang, Mei Zhang, Dan Martinez, Kai Tan, Minjie Luo, Christopher Y Park, Boris Reizis, Vinodh Pillai
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor heterogeneity impedes the development of biomarker assays suitable for early disease detection that would improve patient outcomes. The CA19-9 glycan is currently used as a standalone biomarker for PDAC. Furthermore, previous studies have shown that cancer cells may display aberrant membrane-associated glycans. We therefore hypothesized that PDAC cancer cell subpopulations could be distinguished by aberrant glycan signatures. We used multiplexed glycan immunofluorescence combined with pathologist annotation and automated image processing to distinguish between PDAC cancer cell subpopulations within tumor tissue. Using a training-set/test-set approach, we found that PDAC cancer cells may be identified by signatures comprising 4 aberrant glycans (VVL, CA19-9, sTRA, and GM2) and that there are three glycan-defined PDAC tumor types: sTRA type, CA19-9 type, and intermixed. To determine whether the aberrant glycan signatures could be detected in blood samples, we developed hybrid glycan sandwich assays for membrane-associated glycans. In both patient-matched tumor and blood samples, the proportion of aberrant glycans detected was consistent. Furthermore, our multiplexed glycan immunofluorescent approach proved to be more sensitive and more specific than CA19-9 alone. Our results provide proof of concept for a novel methodology to improve early PDAC detection and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6b226460c28415c919ed9fc15c5b9945dc42aa" target='_blank'>
              Multiplexed Glycan Immunofluorescence Identification of Pancreatic Cancer Cell Subpopulations in Both Tumor and Blood Samples
              </a>
            </td>
          <td>
            Braelyn Binkowski, Zachary L. Klamer, Chongfeng Gao, Ben Staal, Anna Repesh, Hoang-Le Tran, David M. Brass, Pamela Bartlett, Stephen Gallinger, Maria Blomqvist, J. B. Morrow, Peter Allen, Chanjuan Shi, A. Singhi, Randall Brand, Ying Huang, G. Hostetter, Brian Haab
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
HCC is a highly vascular tumor, and many effective drug regimens target the tumor blood vessels. Prior bulk HCC subtyping data used bulk transcriptomes, which contained a mixture of parenchymal and stromal contributions.


METHODS
We utilized computational deconvolution and cell-cell interaction analyses to cell type-specific (tumor-enriched and vessel-enriched) spatial transcriptomic data collected from 41 resected HCC tissue specimens.


RESULTS
We report that the prior Hoshida bulk transcriptional subtyping schema is driven largely by an endothelial fraction, show an alternative tumor-specific schema has potential prognostic value, and use spatially paired ligand-receptor analyses to identify known and novel (LGALS9 tumor-HAVCR2 vessel) signaling relationships that drive HCC biology in a subtype-specific and potentially targetable manner.


CONCLUSIONS
Our study leverages spatial gene expression profiling technologies to dissect HCC heterogeneity and identify heterogeneous signaling relationships between cancer cells and their endothelial cells. Future validation and expansion of these findings may validate novel cancer-endothelial cell interactions and related drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e5076806fef62fc7b47a9b3751a8891a984dc18" target='_blank'>
              HCC spatial transcriptomic profiling reveals significant and potentially targetable cancer-endothelial interactions.
              </a>
            </td>
          <td>
            Chenyue Lu, Amaya Pankaj, Michael J. Raabe, Cole Nawrocki, Ann Liu, Nova Xu, Bidish Kumar Patel, Matthew J. Emmett, Avril K Coley, Cristina R. Ferrone, Vikram Deshpande, I. Bhan, Y. Hoshida, David T. Ting, Martin J. Aryee, Joseph W Franses
          </td>
          <td>2024-09-27</td>
          <td>Hepatology communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Trogocytosis is an underappreciated phenomenon that shapes the immune microenvironment surrounding many types of solid tumors. The consequences of membrane-bound proteins being deposited from a donor immune cell to a recipient cancer cell via trogocytosis are still unclear. Here, we report that human clear cell renal carcinoma tumors stably express the lymphoid markers CD45, CD56, CD14, and CD16. Flow cytometry performed on fresh kidney tumors revealed consistent CD45 expression on tumor cells, as well as varying levels of the other markers mentioned previously. These results were consistent with our immunofluorescent analysis, which also revealed colocalization of lymphoid markers with carbonic anhydrase 9 (CAIX), a standard kidney tumor marker. RNA analysis showed a significant upregulation of genes typically associated with immune cells in tumor cells following trogocytosis. Finally, we show evidence of chromosomal DNA being transferred from immune cells to tumor cells during trogocytosis. This horizontal gene transfer has transcriptional consequences in the recipient tumor cell, resulting in a fusion phenotype that expressed both immune and cancer specific proteins. This work demonstrates a novel mechanism by which tumor cell protein expression is altered through the acquisition of surface membrane fragments and genomic DNA from infiltrating lymphocytes. These results alter the way in which we understand tumor-immune cell interactions and may reveal new insights into the mechanisms by which tumors develop. Additionally, further studies into trogocytosis will help push the field towards the next generation of immunotherapies and biomarkers for treating renal cell carcinoma and other types of cancers. SIGNIFICANCE STATEMENT We have identified trogocytosis as a mechanism by which human clear cell renal carcinoma tumors acquire lymphocyte surface protein expression from tumor infiltrating immune cells. In addition to the transfer of membrane fragments, we have provided evidence to show that genomic DNA is transferred from a normal immune cell to a tumor cell during trogocytosis. This process alters the transcriptome of cancer cells such that they express significantly more mRNA for immune proteins such as the lymphocyte marker CD45 compared to tumor cells that have not undergone trogocytosis. This study provides an in-depth analysis of the interactions between cancer cells and tumor infiltrating lymphocytes, and how these interactions alter the development of human tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d8b30a74776c1d1ac2d3f7a393276d5557d985" target='_blank'>
              Renal cancer cells acquire immune surface protein through trogocytosis and horizontal gene transfer
              </a>
            </td>
          <td>
            Haley Q. Marcarian, Anutr Sivakoses, Anika M. Arias, Olivia C. Ihedioha, Benjamin R. Lee, Maria C. Bishop, A. L. Bothwell
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Although localized prostate cancer is relatively indolent, advanced prostate cancer manifests with aggressive and often lethal variants, including neuroendocrine prostate cancer (NEPC). To identify drivers of aggressive prostate cancer, we leveraged Sleeping Beauty (SB) transposon mutagenesis in a mouse model based on prostate-specific loss-of-function of Pten and Tp53. Compared with control mice, SB mice developed more aggressive prostate tumors, with increased incidence of metastasis. Notably, a significant percentage of the SB prostate tumors display NEPC phenotypes, and the transcriptomic features of these SB mouse tumors recapitulated those of human NEPC. We identified common SB transposon insertion sites (CIS) and prioritized associated CIS-genes differentially expressed in NEPC versus non-NEPC SB tumors. Integrated analysis of CIS-genes encoding for proteins representing upstream, post-translational modulators of master regulators controlling the transcriptional state of SB-mouse and human NEPC tumors identified sirtuin 1 (Sirt1) as a candidate mechanistic determinant of NEPC. Gain-of-function studies in human prostate cancer cell lines confirmed that SIRT1 promotes NEPC, while its loss-of-function or pharmacological inhibition abrogates NEPC. This integrative analysis is generalizable and can be used to identify novel cancer drivers for other malignancies. Summary Using an unbiased forward mutagenesis screen in an autochthonous mouse model, we have investigated mechanistic determinants of aggressive prostate cancer. SIRT1 emerged as a key regulator of neuroendocrine prostate cancer differentiation and a potential target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef638b5be4a6bc97dbe24dca3c6171d01e593aa" target='_blank'>
              A forward genetic screen identifies Sirtuin1 as a driver of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Francisca Nunes de Almeida, A. Vasciaveo, Ainsley Mike Antao, Min Zou, Matteo Di Bernardo, Simone de Brot, A. Rodriguez-Calero, Alexander Chui, A. L. Wang, Nicolas Floc’h, Jaime Y Kim, Stephanie N Afari, Timur Mukhammadov, Juan M Arriaga, Jinqiu Lu, Michael M. Shen, Mark A. Rubin, Andrea Califano, Cory Abate-Shen
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Uterine carcinosarcoma (UCS) is a rare gynecologic cancer with unfavorable prognosis. Its biphasic histologic components are known to associated with outcomes. Dissecting the tumor heterogeneity within UCS is crucial to discover molecular biomarkers for patient stratification. The aim of this study was to utilize spatial profiling technology to analyze immune presentation in different histological components within UCS tumors. The study included 23 UCS patients whose tumor sections were subjected to nanoString GeoMx Digital Spatial Profiling (DSP) to profile 30 immune-related proteins in 491 segments. We identified distinct cluster-labeled subtypes showing varied immune marker expressions. The immune presentations within the carcinoma and sarcoma components showed significant differences. Markers indicating M2 macrophages (CD45+CD163+) and epithelial-mesenchymal transition (EMT) (SMA, fibronectin) were highly expressed in the sarcoma components. The immune presentation subtypes within each histological component also differed between patients with recurrence in 5 years and without recurrence over 5 years. The cluster-labeled subtype with high Neutrophil/PMN-MDSC marker expression (CD66b) was associated with longer survival, whereas the subtype with high NK cell marker expressions (CD56) was linked to poor prognosis. Our result suggested that immune features within the carcinoma component of UCS tumors would determine patient outcomes and should be considered as a strategy for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677b6a5b9abbf69e6cc6fe6fd37eee258299da27" target='_blank'>
              Inter- and intra-tumor heterogeneous immune presentation in primary uterine carcinosarcoma determined by digital spatial profiling
              </a>
            </td>
          <td>
            Ying-Xuan Li, Wei-Chou Lin, Ko-Chen Chen, Wen-Chun Chang, Chin-Jui Wu, Lin-Hung Wei, Ruby Yun-Ju Huang, Bor-Ching Sheu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute graft-versus-host disease (aGVHD) is a significant complication of allogeneic hematopoietic stem cell transplantation (aHSCT), driven by alloreactive donor T cells in the gut. However, the roles of additional donor and host cells in this process are not fully understood. We conducted multiplexed imaging on 59 biopsies from patients with gastrointestinal GVHD and 10 healthy controls, revealing key pathological changes, including fibrosis, crypt alterations, loss of Paneth cells, accumulation of endocrine cells, and disrupted immune organization, particularly a reduction in IgA-secreting plasma cells. Interestingly, CD8T cells were enriched only in a subset of patients, while others exhibited non-canonical enrichments of macrophages and neutrophils. Post-transplantation time significantly influenced immune composition, with host cells dominating plasma and T cell compartments long after transplantation. This spatial atlas of healthy duodenum and GVHD uncovers non-canonical immune dynamics, offering insights into disease pathophysiology and potential clinical applications in GVHD and other inflammatory bowel diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69689939e79be56e05c61106cc952ea2166e2eac" target='_blank'>
              A spatial atlas of human gastro-intestinal acute GVHD reveals epithelial and immune dynamics underlying disease pathophysiology
              </a>
            </td>
          <td>
            Nofar Azulay, Idan Milo, Yuval Bussi, Raz Ben Uri, T. Haran, Michal Eldar, O. Elhanani, Yotam Harnik, Oran Yakubovsky, I. Nachmany, T. Salame, Martin Wartenberg, Philippe Bertheau, D. Michonneau, Gérard Socié, Leeat Keren
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, S. Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Juvenile localized and systemic scleroderma are rare autoimmune diseases which cause significant disability and morbidity in children. The mechanisms driving juvenile scleroderma remain unclear, necessitating further cellular and molecular level studies. The Visium CytAssist spatial transcriptomics (ST) platform, which preserves the spatial location of cells and simultaneously sequences the whole transcriptome, was employed to profile the histopathological slides from skin lesions of juvenile scleroderma patients. (1) Spatial domains were identified from ST data and exhibited strong concordance with the pathologist’s annotations of anatomical structures. (2) The integration of paired ST data and single-cell RNA sequencing (scRNA-seq) from the same patients validated the comparable accuracy of the two platforms and facilitated the estimation of cell type composition in ST data. (3) The pathologist-annotated immune infiltrates, such as perivascular immune infiltrates, were clearly delineated by the ST analysis, underscoring the biological relevance of the findings. This is the first study utilizing spatial transcriptomics to investigate skin lesions in juvenile scleroderma patients. The validity of the ST data was corroborated by gene expression analyses and the pathologist’s assessments. Integration with scRNA-seq data facilitated the cell type-level analysis and validation. Analyses of immune infiltrates through combined ST data and pathological review enhances our understanding of the pathogenesis of juvenile scleroderma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d114ce60551b0d377557f894744cf39236d724d1" target='_blank'>
              Spatial Transcriptomics Identifies Cellular and Molecular Characteristics of Scleroderma Skin Lesions: Pilot Study in Juvenile Scleroderma
              </a>
            </td>
          <td>
            Tianhao Liu, Deren Esencan, Claudia M. Salgado, Chongyue Zhao, Ying-Ju Lai, Theresa Hutchins, Anwesha Sanyal, Wei Chen, Kathryn S Torok
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce58909f9de06504a3ce2e2bc45ebf1454164f" target='_blank'>
              Spatial single-cell protein landscape reveals vimentinhigh macrophages as immune-suppressive in the microenvironment of hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xinyao Qiu, Tao Zhou, Shuai Li, Jianmin Wu, Jing Tang, Guosheng Ma, Shuai Yang, Ji Hu, Kaiting Wang, Siyun Shen, Hongyang Wang, Lei Chen
          </td>
          <td>2024-09-26</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c649f4e5bac69e4b666a1bcdc0297d323411c9" target='_blank'>
              Re-analysis of single cell and spatial transcriptomics data reveals B cell landscape in gastric cancer microenvironment and its potential crosstalk with tumor cells for clinical prognosis
              </a>
            </td>
          <td>
            Xing Cai, Jinru Yang, Yusheng Guo, Yanchao Yu, Chuansheng Zheng, Xiaofang Dai
          </td>
          <td>2024-08-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is an increasingly prevalent malignancy worldwide. This study aims to understand molecular alterations associated with lymph node metastasis of OSCC in order to improve treatment strategies. We analyzed a cohort of 46 patients with primary OSCC, including 10 with lymph node metastasis and 36 without. Using a comprehensive multi-omics approach—encompassing genomic, transcriptomic, proteomic, epigenetic, single-cell, and spatial analyses—we integrated data to delineate the molecular landscape of OSCC in the context of lymph node metastasis. Our genomic analysis identified significant mutations in key genes within the MAPK, TGF-β, and WNT signaling pathways, which are essential for tumor development. The proteogenomic analysis highlighted pathways critical for lymph node dissemination and factors contributing to an immunosuppressive tumor microenvironment. Elevated levels of POSTN were found to reorganize the extracellular matrix (ECM), interact with TGF-β, disrupt cell cycle regulation, and suppress the immune response by reducing VCAM1 activity. Integrated analyses of single-cell and spatial transcriptome data revealed that cancer-associated fibroblasts (CAFs) secrete TGF-β1/2, promoting cancer cell metastasis through epithelial-mesenchymal transition (EMT). Our integrated multi-omics analysis provides a detailed understanding of molecular mechanisms driving lymph node metastasis of OSCC. These insights could lead to more precise diagnostics and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bdd901aa91cd3ba2711b681f62cf92ddfab3b2" target='_blank'>
              Proteogenomic Characterization of Primary Oral Squamous Cell Carcinomas Unveils the Extracellular Matrix Remodeling and Immunosuppressive Microenvironment Linked with Lymph Node Metastasis
              </a>
            </td>
          <td>
            Yu Liu, Zhenyu Yang, J. Pu, Jie Zhong, U. Khoo, Yu-Xiong Su, Gao Zhang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I. Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Glioma is a highly heterogeneous malignancy of the central nervous system. This heterogeneity is driven by various molecular processes, including neoplastic transformation, cell cycle dysregulation, and angiogenesis. Among these biomolecular events, inflammation and stress pathways in the development and driving factors of glioma heterogeneity have been reported. However, the mechanisms of glioma heterogeneity under stress response remain unclear, especially from a spatial aspect. Methods This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to explore the impact of oxidative stress response genes in oligodendrocyte precursor cells (OPCs). Our analysis identified distinct pathways activated by oxidative stress in two different types of gliomas: high- and low- grade (HG and LG) gliomas. Results In HG gliomas, oxidative stress induced a metabolic shift from oxidative phosphorylation to glycolysis, promoting cell survival by preventing apoptosis. This metabolic reprogramming was accompanied by epithelial-to-mesenchymal transition (EMT) and an upregulation of stress response genes. Furthermore, SCENIC (Single-Cell rEgulatory Network Inference and Clustering) analysis revealed that oxidative stress activated the AP1 transcription factor in HG gliomas, thereby enhancing tumor cell survival and proliferation. Conclusion Our findings provide a novel perspective on the mechanisms of oxidative stress responses across various grades of gliomas. This insight enhances our comprehension of the evolutionary processes and heterogeneity within gliomas, potentially guiding future research and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56dee2729032b7749900d8cd04b93b8180eff50e" target='_blank'>
              Single-cell and spatial transcriptome assays reveal heterogeneity in gliomas through stress responses and pathway alterations
              </a>
            </td>
          <td>
            Z. He, Zheng Liu, Qi Wang, Xingjian Sima, Wei Zhao, Chunmei He, Wenjie Yang, Han Chen, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clinical trial data underscores the need to improve oncolytic virus (OV) distribution within tumors, a challenge compounded by the lack of predictive biomarkers and limited opportunities for post-treatment analysis. To decipher the factors influencing treatment outcomes, we employed multimodal bioluminescence imaging (MM-BLI) in conjunction with magnetic resonance imaging (MRI) to monitor OV infection, replication, and tumor viability in orthotopic brain glioma mouse models. This approach revealed two distinct therapeutic responses: “Responders” with tumor regression and “Non-Responders” with tumor progression. In-depth analysis of individual brains from both groups uncovered dynamic interactions between the OV and the tumor microenvironment, highlighting the involvement of Iba-1+ microglia and tumor necrosis in hindering OV distribution within the tumor. To address this, we incorporated a CSF-1 receptor inhibitor (PLX3397), which improved tumor control by enhancing OV’s direct cytopathic effects and reducing microglial interference. Our findings highlight microglia as a significant barrier to effective OV therapy, suggesting that targeting brain-resident immune cells could enhance the therapeutic efficacy of OVs in resistant brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45611a5fd1ccdd8bd8b5f63149ebbc79e2be6cc" target='_blank'>
              Longitudinal analysis of glioblastoma response to oncolytic HSV1 guides PLX3397 combination for enhanced viral permissiveness
              </a>
            </td>
          <td>
            Hirotaka Ito, Naoyuki Shono, Claudia Zagami, Min J Kim, Alayo A. Quazim, Eric McLaughlin, W. Goins, Soledad A. Fernandez, E. Chiocca, Hiroshi Nakashima
          </td>
          <td>2024-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e27ecfb76604dddc8d09da4d1bbaaf7b66725bf" target='_blank'>
              Spatial multiplexed immunofluorescence analysis reveals coordinated cellular networks associated with overall survival in metastatic osteosarcoma
              </a>
            </td>
          <td>
            Ryan A Lacinski, Sebastian A. Dziadowicz, Vincent K Melemai, Brody Fitzpatrick, John J. Pisquiy, Tanya Heim, Ines Lohse, Karen Schoedel, Nicolas J Llosa, Kurt R Weiss, Brock A Lindsey
          </td>
          <td>2024-09-27</td>
          <td>Bone Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite efforts to understand breast cancer biology, metastatic disease remains a clinical challenge. Identifying suppressors of breast cancer progression and mechanisms of transition to more invasive phenotypes could provide game changing therapeutic opportunities. Transcriptional deregulation is central to all malignancies, highlighted by the extensive reprogramming of regulatory elements that underlie oncogenic programs. Among these, super-enhancers (SEs) stand out due to their enrichment in genes controlling cancer hallmarks. To reveal novel breast cancer dependencies, we integrated the analysis of the SE landscape with master regulator activity inference for a series of breast cancer cell lines. As a result, we identified T-helper-inducing Poxviruses and Zinc-finger (POZ)/Krüppel-like factor (ThPOK, ZBTB7B), a CD4+ cell lineage commitment factor, as a breast cancer master regulator that is recurrently associated with a SE. ThPOK expression is highest in luminal breast cancer but is significantly reduced in the basal subtype. Manipulation of ThPOK levels in cell lines shows that its repressive function restricts breast cancer cells to an epithelial phenotype by suppressing the expression of genes involved in the epithelial-mesenchymal transition (EMT), WNT/β-catenin target genes, and the pro-metastatic TGFβ pathway. Our study reveals ThPOK as a master transcription factor that restricts the acquisition of metastatic features in breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf508abb336eb73c82bb427bf15d78980db2284" target='_blank'>
              Super-enhancer profiling reveals ThPOK/ZBTB7B, a CD4+ cell lineage commitment factor, as a master regulator that restricts breast cancer cells to a luminal non-migratory phenotype
              </a>
            </td>
          <td>
            Camila D. Arcuschin, Kamin Kahrizi, Rosalyn W. Sayaman, Carolina DiBenedetto, Yizhuo Shen, Pedro J Salaberry, Ons Zakroui, Cecilia Schwarzer, Alessandro Scapozza, Paola Betancur, Julie D. Saba, Jean-Philippe Coppé, M. Barcellos-Hoff, Dietmar Kappes, L. J. van ’t Veer, Ignacio E Schor, Denise P Muñoz
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather O'Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mutations inactivating TET2 are among the most common drivers of clonal hematopoiesis (CH). While TET2 inactivation is associated with monocyte-derived inflammation and improved chimeric antigen-receptor-T cell function, its impact on immunotherapy response is unknown. In our mouse model, hematopoietic Tet2 mutation enhanced immune checkpoint blockade (ICB) response. Enhanced ICB response with Tet2 mutation required phagocytes, CD4 and CD8 T cells. Mechanistically, in Tet2-mutant tumor-infiltrating leukocytes (TILs), ICB preferentially induced anti-tumor states and restricted cell states linked to tumor progression. Tet2-mutant monocytes activated costimulatory programs, while Tet2-mutant T cells showed enhanced T cell memory signatures, lesser exhaustion and decreased regulatory phenotype. Our murine data was clinically relevant, since we found that melanomas from patients with TET2 driver mutation-CH (TET2-CH) showed enhanced immune infiltration, T cell activation, and T cell memory programs. In melanoma patients treated with ICB, TET2-CH was associated with 6-fold greater odds of clinical benefit. Collectively, our data establishes that hematopoietic Tet2 inactivation primes leukocytes for anti-tumor states associated with immunotherapy response and provides a potential biomarker for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3e6160e702c2224ea8ff0222f9be539e313873" target='_blank'>
              Hematopoietic Tet2 inactivation enhances the response to checkpoint blockade immunotherapy
              </a>
            </td>
          <td>
            R. Vanner, Suraj Bansal, Marco M. Buttigeig, Andy G. X. Zeng, Vincent Rondeau, Darryl Y. Chan, Michelle A. Chan-Seng-Yue, Liqing Jin, Jessica Mcleod, E. Donato, Patrick Stelmach, C. Vlasschaert, Yitong Yang, Aarushi Gupta, Sofia Genta, Enrique Sanz Garcia, LI Shlush, Mauricio Ribeiro, Marcus O Butler, Sagi Abelson, Mark D. Minden, Steven M. Chan, Michael J. Rauh, A. Trumpp, John E. Dick
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell is important because of its potential to treat various diseases. As deep learning continues to advance, using unsupervised methods to classify medical images has become a significant focus because collecting high-quality labeled data for medical images is labor-intensive and time-consuming. Beyond the need for accurate labeling, there is a desire to explore the underlying characteristics of the data, even when labels may be ambiguous or uncertain. To address these challenges, we present a novel approach that combines image clustering with an insightful explanation of how these clusters are formed. Our method employs a U-net combined with a clustering algorithm to segment the dataset into different groups. After clustering, we use various techniques to interpret and elucidate the results. Moreover, our paper introduces a unique dataset focused on cell data, specifically highlighting the developmental patterns of cancer cells and T cells under various experimental conditions. This dataset offers a rich source of information and presents a complex challenge for image classification due to the diversity of conditions and cell behaviors involved. Our study thoroughly compares different architectural models on this new dataset, demonstrating the superior performance of our proposed architecture. Through experimental analysis and ablation studies, we provide substantial evidence of the benefits offered by our architecture, not only in terms of accuracy but also in its ability to reveal deeper insights into the data. This work advances the field of image classification and opens new possibilities for understanding complex biological processes through computer vision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b2222e0e50090ec7fb0ff692b055ed44f83f60" target='_blank'>
              High-throughput Image-based Clustering of CAR-T/Tumor Cocultures for Rapid and Facile Hit Identification
              </a>
            </td>
          <td>
            Zhi Xu, Xueqi Liu, Kirby Madden-Hennessey, Jordan Urbani, S. K. Nikkhoi, Anusuya Ramasubramanian, Kartika G. Venugopal, Qi Zhao, Eric L. Smith, Yun Fu
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Single-cell Hi-C (scHi-C) has provided unprecedented insights into the heterogeneity of 3D genome organization. However, its sparse and noisy nature poses challenges for computational analyses, such as chromatin architectural feature identification. Here, we introduce scCAFE, a deep learning model for the multi-scale detection of architectural features at the single-cell level. scCAFE provides a unified framework for annotating chromatin loops, TAD-like domains (TLDs), and compartments across individual cells. Our model outperforms previous scHi-C loop calling methods and delivers accurate predictions of TLDs and compartments that are biologically consistent with previous studies. The resulting single-cell annotations also offer a measure to characterize the heterogeneity of different levels of architectural features across cell types. We leverage this heterogeneity and identify a series of marker loop anchors, which demonstrate the potential of the 3D genome data to annotate cell identities without the aid of simultaneously sequenced omics data. Overall, scCAFE not only serves as a useful tool for analyzing single-cell genomic architecture, but also paves the way for precise cell-type annotations solely based on 3D genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9e853dc3e19f05f5603a9a9aa1ba89e02d8ae38" target='_blank'>
              Unveiling multi-scale architectural features in single-cell Hi-C data using scCAFE
              </a>
            </td>
          <td>
            Fuzhou Wang, Jiecong Lin, Hamid Alinejad-Rokny, Wenjing Ma, Lingkuan Meng, Lei Huang, Jixiang Yu, Nanjun Chen, Yuchen Wang, Zhongyu Yao, Weidun Xie, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The identification of gene regulatory networks (GRN) governing distinct cell fates in multilineage cellular differentiation systems is of critical importance for understanding cell fate decision. Single-cell RNA-sequencing (scRNA-seq) provides a powerful tool for the quantification of gene-level co-variation across the cell state manifold. However, accurate GRN reconstruction is hampered by the sparsity of scRNA-seq data introducing substantial technical noise. Moreover, the high dimensionality of typical scRNA-seq datasets limits the scalability of available approaches. To overcome these challenges, and to facilitate the inference of lineage-specific GRNs with directed regulator-target relations, we introduce NetID. This approach optimizes coverage of the cell state manifold by homogenous metacells and avoids spurious gene-gene correlations observed with available imputation methods. Benchmarking demonstrates superior performance of NetID compared to imputation-based GRN inference. By incorporating cell fate probability information, NetID facilitates prediction of lineage-specific GRNs and recovers known network motifs centered around lineage-determining transcription factors governing bone marrow hematopoiesis, making it a powerful toolkit for deciphering the gene regulatory control of cellular differentiation from large-scale single-cell transcriptome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/725296ffdf0a7cacc6e78577d9871b12ce8048c7" target='_blank'>
              Scalable identification of lineage-specific gene regulatory networks from metacells with NetID
              </a>
            </td>
          <td>
            Weixu Wang, Yichen Wang, Ruiqi Lyu, D. Grün
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We employed laser microdissection to selectively harvest tumor cells and stroma from the microenvironment of formalin-fixed, paraffin-embedded head and neck squamous cell carcinoma (HNSCC) tissues. The captured HNSCC tissue fractions were analyzed by quantitative mass spectrometry-based proteomics using a data independent analysis approach. In paired samples, we achieved excellent proteome coverage having quantified 6,668 proteins with a median quantitative coefficient of variation under 10%. We observed significant differences in relevant functional pathways between the spatially resolved tumor and stroma regions. Our results identified extracellular matrix (ECM) as a major component enriched in the stroma, including many cancer associated fibroblast signature proteins in this compartment. We demonstrate the potential for comparative deep proteome analysis from very low starting input in a scalable format that is useful to decipher the alterations in tumor and the stromal microenvironment. Correlating such results with clinical features or disease progression will likely enable identification of novel targets for disease classification and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81bad8ce1ede8240c16776044c3909a46ef259c" target='_blank'>
              dia-PASEF Proteomics of Tumor and Stroma LMD Enriched from Archived HNSCC Samples
              </a>
            </td>
          <td>
            Aswini Panigrahi, Allison L. Hunt, Diego Assis, Matthew Willetts, Bhaskar V Kallakury, Bruce Davidson, Thomas P. Conrads, R. Goldman
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Accurate phenotyping of cells in the tumor microenvironment is crucial for understanding cancer biology and developing effective therapies. However, current methods require precise cell segmentations and struggle to generalize across different imaging modalities, limiting their utility in digital pathology. Here, we show that Contrastive Learning Enabled Accurate Registration of Immune and Tumor Cells (CLEAR-IT) overcomes these limitations, providing a robust and versatile tool for cell phenotyping. CLEAR-IT accurately phenotypes cells comparable to state-of-the-art methods, generalizes across multiplex imaging modalities, maintains high performance even with limited number of labels, and enables the extraction of prognostic markers. Additionally, CLEAR-IT can be combined with existing methods to boost their performance, whereas its lack of need for precise cell segmentations significantly reduces training efforts. This method enhances the robustness and efficiency of digital pathology workflows, making it a valuable tool for cancer research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02cd2bfec0cff758ded2e68209475ad537d70fc" target='_blank'>
              CLEAR-IT: Contrastive Learning to Capture the Immune Composition of Tumor Microenvironments
              </a>
            </td>
          <td>
            Daniel Spengler, Serafim Korovin, Kirti Prakash, Peter Bankhead, R. Debets, Carlas S. Smith, H. E. Balcioglu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Pancreatic cancer remains one of the most deadly cancers, with over 450,000 fatalities in 2022 alone. The most lethal variation of this cancer is Pancreatic Ductal Adenocarcinoma (PDAC), with a 5-year survival rate of under 10%. Combination chemotherapy has provided major advances in this disease but it remains challenging for many reasons including the immunosuppressive nature of the tumor microenvironment (TME). PDAC has a highly desmoplastic microenvironment that suppresses antitumor immunity and promotes tumor cell invasion, metastasis, and resistance through epithelial to mesenchymal transition (EMT) of cancer cells. Cancer cells can undergo various forms of cell death, notably apoptosis, necrosis, autophagy, pyroptosis, and ferroptosis. These mechanisms are triggered by different cellular stressors and shape the microenvironment. Through the use of 15 Visium spatial transcriptomics samples from publicly available primary PDAC tumors and paired single-cell RNA (scRNA) data, we aim to understand how cell stress and cell death shape the tumor microenvironment (TME) and epithelial-mesenchymal transition (EMT). By leveraging information from known pathways in literature, we will identify cellular programs related to stress, death, and EMT. We will explore intracellular correlations within single-cell data and extracellular interactions within the TME using spatial data to uncover how different forms of cell death and stress signatures influence the TME and EMT spectrum. The initial phase involves identifying the types of stress that lead to various forms of cell death intracellularly and validating these gene signatures. Subsequently, we will apply these validated signatures to the spatial data to correlate each score and search for significant overlapping programs. This will extend beyond stress and death, incorporating TME signatures corresponding to different cell types and the EMT spectrum. By understanding the signals that promote the death of various cell populations, we aim to identify combination treatments that target both ends of the EMT spectrum. Additionally, this research will provide insights into the inflammatory impact of cell death on the TME. Our approach aims to inform the development of therapies that disrupt resistance mechanisms, ultimately improving treatment outcomes for patients with PDAC. This research can potentially reveal new therapeutic targets and strategies, enhancing our ability to combat this deadly disease.
 Citation Format: Rahul Bansal, Izabella Zamora, Samuel Wright, Nir Hacohen, Arnav Mehta. Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba425dbbc0b9c870572a7ccded079a29b6e67232" target='_blank'>
              Abstract C074: Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Rahul Bansal, Izabella Zamora, Samuel J Wright, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, Ondřej Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The tumor microenvironment (TME) comprises a diverse array of cells, both cancerous and non-cancerous, including stromal cells and immune cells. Complex interactions among these cells play a central role in driving cancer progression, impacting critical aspects such as tumor initiation, growth, invasion, response to therapy, and the development of drug resistance. While targeting the TME has emerged as a promising therapeutic strategy, there is a critical need for innovative approaches that accurately replicate its complex cellular and non-cellular interactions; the goal being to develop targeted, personalized therapies that can effectively elicit anti-cancer responses in patients. Microfluidic systems present notable advantages over conventional in vitro 2D co-culture models and in vivo animal models, as they more accurately mimic crucial features of the TME and enable precise, controlled examination of the dynamic interactions among multiple human cell types at any time point. Combining these models with next-generation technologies, such as bioprinting, single cell sequencing and real-time biosensing, is a crucial next step in the advancement of microfluidic models. This review aims to emphasize the importance of this integrated approach to further our understanding of the TME by showcasing current microfluidic model systems that integrate next-generation technologies to dissect cellular intra-tumoral interactions across different tumor types. Carefully unraveling the complexity of the TME by leveraging next generation technologies will be pivotal for developing targeted therapies that can effectively enhance robust anti-tumoral responses in patients and address the limitations of current treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78b792b563d1d9bcfce83a027e1da609d44dc5" target='_blank'>
              Improving tumor microenvironment assessment in chip systems through next-generation technology integration
              </a>
            </td>
          <td>
            Daniela Gaebler, Stephanie J. Hachey, Christopher C. W. Hughes
          </td>
          <td>2024-09-25</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a typical highly heterogeneous solid tumor with high morbidity and mortality worldwide, especially in China; however, the immune microenvironment of HCC has not been clarified so far. Here, we employed single‐cell RNA sequencing (scRNA‐seq) on diethylnitrosamine (DEN)‐induced mouse HCC model to dissect the immune cell dynamics during tumorigenesis. Our findings reveal distinct immune profiles in both precancerous and cancerous lesions, indicating early tumor‐associated immunological alterations. Notably, specific T and B cell subpopulations are preferentially enriched in the HCC tumor microenvironment (TME). Furthermore, we identified a subpopulation of naïve B cells with high CD83 expression, correlating with improved prognosis in human HCC. These signature genes were validated in The Cancer Genome Atlas HCC RNA‐seq dataset. Moreover, cell interaction analysis revealed that subpopulations of B cells in both mouse and human samples are activated and may potentially contribute to oncogenic processes. In summary, our study provides insights into the dynamic immune microenvironment and cellular networks in HCC pathogenesis, with a specific emphasis on naïve B cells. These findings emphasize the significance of targeting TME in HCC patients to prevent HCC pathological progression, which may give a new perspective on the therapeutics for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76c814af70ac2f24272bcadf22a4dd9e85ffd9ec" target='_blank'>
              Leveraging single‐cell RNA‐seq for uncovering naïve B cells associated with better prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Qingjia Sun, Rui Gao, Yingxin Lin, Xianchao Zhou, Tao Wang, Jian He
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite being the leading cause of childhood mortality, pediatric gliomas have been relatively understudied, and the repurposing of immunotherapies has not been successful. Whole transcriptome sequencing, single-cell sequencing, and sequential multiplex immunofluorescence were used to identify an immunotherapy strategy evaluated in multiple preclinical glioma models. MAPK-driven pediatric gliomas have a higher interferon signature relative to other molecular subgroups. Single-cell sequencing identified an activated and cytotoxic microglia population designated MG-Act in BRAF-fused MAPK-activated pilocytic astrocytoma (PA), but not in high-grade gliomas or normal brain. TIM3 is expressed on MG-Act and on the myeloid cells lining the tumor vasculature but not normal brain. TIM3 expression becomes upregulated on immune cells in the PA microenvironment and anti-TIM3 reprograms ex vivo immune cells from human PAs to a pro-inflammatory cytotoxic phenotype. In a genetically engineered murine model of MAPK-driven low-grade gliomas, anti-TIM3 treatment increased median survival over IgG and anti-PD1 treated mice. ScRNA sequencing data during the therapeutic window of anti-TIM3 demonstrates enrichment of the MG-Act population. The therapeutic activity of anti-TIM3 is abrogated in the CX3CR1 microglia knockout background. These data support the use of anti-TIM3 in clinical trials of pediatric low-grade MAPK-driven gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93c9fb9117641aac2f760a5c311e07768d3cba6" target='_blank'>
              Pediatric glioma immune profiling identifies TIM3 as a therapeutic target in BRAF-Fusion pilocytic astrocytoma.
              </a>
            </td>
          <td>
            Shashwat Tripathi, Hinda Najem, Corey Dussold, Sebastian Pacheco, Ruochen Du, Moloud Sooreshjani, Lisa A Hurley, James P Chandler, R. Stupp, A. Sonabend, C. Horbinski, R. Lukas, Joanne Xiu, Giselle Y López, Theodore Nicolaides, Valerie Brown, N. R. Wadhwani, Sandi K Lam, C. D. James, Ganesh Rao, Maria G Castro, Amy B. Heimberger, Michael DeCuypere
          </td>
          <td>2024-08-13</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d92fbde3dae7404c9f30bd22e44407b07e5261a" target='_blank'>
              Self-sustaining long-term 3D epithelioid cultures reveal drivers of clonal expansion in esophageal epithelium.
              </a>
            </td>
          <td>
            A. Herms, D. Fernandez-Antoran, M. Alcolea, Argyro Kalogeropoulou, Ujjwal Banerjee, G. Piedrafita, E. Abby, J. A. Valverde-Lopez, Inês S Ferreira, Irene Caseda, M. T. Bejar, S. Dentro, Sara Vidal-Notari, S. Ong, B. Colom, K. Murai, C. King, Krishnaa Mahbubani, K. Saeb‐Parsy, Alan R Lowe, M. Gerstung, Philip H Jones
          </td>
          <td>2024-09-23</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Ethnic disparities in breast cancer (BC) outcomes have been reported, with lower survival rates among black women compared to white. Additionally, the unfavorable triple negative (TNBC) sub-type is more prevalent in black women and within this population, risk of mortality is again higher compared to white women. Biological contributors to this disparity are less clearly understood than socio-economic factors. Recent studies recognize different breast tumour immune microenvironments between black and white BC patients, necessitating further research into ethnicity-related differences in anti-tumor immune responses. Such responses begin in regional lymph nodes (LNs) with tumor neoantigen presentation. Here, we generate single-cell spatial maps of involved TNBC LN cores in treatment naïve black and white women. To guide design of a phenotyping panel for LNs, we profiled blood samples (n=95) using high-dimensional flow cytometry from newly diagnosed black and white BC patients. We found differential skewing of NK cell subsets, lower memory B cell (identified as IgM+CD27+IgD- & IgM+CD27+IgD+ cells) abundance and higher CD4+ T effector memory presence in black patients compared to white. Additionally, higher antigen presentation capacity was observed with increased intermediate monocytes and conventional DCs, confirmed by MS-based proteomics. Using highly multiplexed imaging with a panel of 26 markers, we performed single-cell spatial immune profiling to characterize cellular phenotypes, neighborhoods and interactions within a cohort of LN core biopsies from black patients and white patients. A total of 536,113 cells were segmented for analysis. Cell abundance analysis revealed higher CD8+ T effector memory and CD4+ T cell subsets in white patients, except for CD4+ T effector memory cells which were higher in black patients, consistent with blood profiling. Cellular neighborhood analysis revealed black patient tumor-rich regions were enriched for M2 macrophages while white patients had no such enrichment. In these white patients, a neighborhood resembling canonical follicular structures almost exclusively containing B cells was identified. Interestingly in these black patients, analogous B cell rich regions also comprised a heterogeneous array of other immune cells. Further analysis to explore B cell zones is ongoing. Preliminary interaction analysis identified immunosuppressive crosstalk in black patients with enriched Treg & CD8+ T cell and Treg & NK interactions, not observed in white patients. These initial findings suggest ethnicity-based variation in LN architectures from treatment naive patients with TNBC, supporting the hypothesis that variation in immune programming could translate into differences in pro and anti-tumor immunity, contributing to disparities in outcome amongst black and white patients.
 Citation Format: Paul R. Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, Thanussuyah Alaguthurai, Priya Hossain, Patrycja Gazińska, Sheeba Irshad. Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C142.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c431f2f91ac1c49d9f19dea4541265da8c85d8d6" target='_blank'>
              Abstract C142: Differential single-cell spatial immune cell profiles and immune cell interactions identified within lymph nodes metastases of black and white women diagnosed with triple negative breast cancers
              </a>
            </td>
          <td>
            Paul Buckley, Melek Akay, Esme Carpenter, Rosalind Graham, Helen Kakkassery, T. Alaguthurai, Priya Hossain, Patrycja Gazinska, S. Irshad
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Keratinocytes, the dominant cell type in the melanoma microenvironment during tumor initiation, exhibit diverse effects on melanoma progression. Using a zebrafish model of melanoma and human cell co-cultures, we observed that keratinocytes undergo an Epithelial–Mesenchymal Transition (EMT)-like transformation in the presence of melanoma, reminiscent of their behavior during wound healing. Surprisingly, overexpression of the EMT transcription factor Twist in keratinocytes led to improved overall survival in zebrafish melanoma models, despite no change in tumor initiation rates. This survival benefit was attributed to reduced melanoma invasion, as confirmed by human cell co-culture assays. Single-cell RNA-sequencing revealed a unique melanoma cell cluster in the Twist-overexpressing condition, exhibiting a more differentiated, less invasive phenotype. Further analysis nominated homotypic jam3b-jam3b and pgrn-sort1a interactions between Twist-overexpressing keratinocytes and melanoma cells as potential mediators of the invasive restraint. Our findings suggest that EMT in the tumor microenvironment (TME) may limit melanoma invasion through altered cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8055a65d64a36749562fed897a59067bde3fe48" target='_blank'>
              Restraint of melanoma progression by cells in the local skin environment
              </a>
            </td>
          <td>
            Yilun Ma, Mohita M. Tagore, Miranda V. Hunter, Ting-Hsiang Huang, Emily D. Montal, Joshua M. Weiss, Richard M. White
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. Vanderhyden, Hector C. Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Summary Imagine if we could simultaneously predict spatial protein expression in tissues from their routine Hematoxylin and Eosin (H&E) stained images, and create tissue images given protein expression profiles thus enabling virtual simulations of how protein expression alterations impact histology in complex diseases like cancer. Such an approach could lead to more informed diagnostic and therapeutic decisions for precision medicine at lower costs and shorter turnaround times, more detailed insights into underlying disease pathology as well as improvement in predictive and generative performance. In this study, we investigate the intricate correlation between protein expressions obtained from Hyperion mass cytometry and histopathological microstructures in conventional H&E stained glioblastoma (GBM) samples, unveiling morphological patterns and cellular-level spatial alterations associated with protein expression changes. To model these complex relationships, we propose a novel generative-predictive framework called Ouroboros for producing H&E images from protein expressions and simultaneously predicting protein expressions from H&E images. Our comprehensive sample-independent validation over 9920 tissue spots from 4 GBM samples encompassing visual image analysis, quantitative analysis, subspace alignment and perturbation experiments shows that the proposed generative-predictive approach offers significant improvements in predicting protein expression from images in comparison to baseline methods as well as accurate generation of virtual GBM sample images. This proof of concept study can contribute to advancing our understanding of histological responses to protein expression perturbations and lays the foundations for further developments in this area. Availability and implementation Implementation and associated data for the proposed approach are available at the URL: https://github.com/Srijay/Ouroboros.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67ded4a0599c7810f90359134e15ada22ea589a" target='_blank'>
              Ouroboros: cross-linking protein expression perturbations and cancer histology imaging with generative-predictive modeling
              </a>
            </td>
          <td>
            Srijay Deshpande, Sokratia Georgaka, Michael Haley, Robert Sellers, James Minshull, J. Nallala, Martin Fergie, Nicholas Stone, N. Rajpoot, Syed-Murtuza Baker, Mudassar Iqbal, Kevin Couper, Federico Roncaroli, F. Minhas
          </td>
          <td>2024-09-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMNs) are critical precursors to pancreatic ductal adenocarcinoma (PDAC), a highly lethal cancer due to late detection and rapid progression. Using multi-region whole-genome and transcriptome sequencing, we traced the evolution of PDAC from IPMN, constructing detailed phylogenetic trees to provide insights into subclonal architectures and progression pathways. Our analysis identified two distinct evolutionary trajectories: one driven by a single ancestral clone, and another involving multiple independent ancestral clones, potentially influencing the timing and nature of PDAC onset. We further explored the roles of mutational signatures and structural variants (SVs) in promoting clonal evolution. Complementing these genomic findings, our transcriptomic analysis revealed unique gene expression profiles and variations in the immune landscape, correlating with the different progression stages from IPMN to PDAC. These insights reveal the complex molecular dynamics of IPMN progression to PDAC, highlighting the need to refine early detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15f2134ea7c52643fe484a9a257be6fc80940a8" target='_blank'>
              Clonal evolutionary analysis reveals patterns of malignant transformation in pancreatic cancer from Intraductal Papillary Mucinous IPMN Neoplasms (IPMN)
              </a>
            </td>
          <td>
            A. Pea, Xiaotong He, Rosie Upstill-Goddard, C. Luchini, Leonor Patricia Schubert Santana, Stephan B. Dreyer, F. Duthie, N. Jamieson, Colin J. McKay, E. Dickson, Alessandra Pulvirenti, Selma Rebus, Paola Piccoli, Nicola Sperandio, R. Lawlor, M. Milella, F. Froeling, Roberto Salvia, Aldo Scarpa, Andrew V. Bainkin, David C Wedge, David K. Chang
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Pancreatic cancer, one of the deadliest human malignancies, is characterized by a fibro-inflammatory tumor microenvironment and wide array of metabolic alterations. To comprehensively map metabolism in a cell type-specific manner, we harnessed a unique single-cell RNA-sequencing dataset of normal human pancreata. This was compared with human pancreatic cancer samples using a computational pipeline optimized for this study. In the cancer cells we observed enhanced biosynthetic programs. We identified downregulation of mitochondrial programs in several immune populations, relative to their normal counterparts in healthy pancreas. Although granulocytes, B cells, and CD8+ T cells all downregulated oxidative phosphorylation, the mechanisms by which this occurred were cell type specific. In fact, the expression pattern of the electron transport chain complexes was sufficient to identify immune cell types without the use of lineage markers. We also observed changes in tumor-associated macrophage (TAM) lipid metabolism, with increased expression of enzymes mediating unsaturated fatty acid synthesis and upregulation in cholesterol export. Concurrently, cancer cells exhibited upregulation of lipid/cholesterol receptor import. We thus identified a potential crosstalk whereby TAMs provide cholesterol to cancer cells. We suggest that this may be a new mechanism boosting cancer cell growth and a therapeutic target in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f304ad19d07747e1efedb9c7fa336ae96c4bd5a6" target='_blank'>
              Metabolic landscape of the healthy pancreas and pancreatic tumor microenvironment.
              </a>
            </td>
          <td>
            Monica E Bonilla, Megan D Radyk, Matthew D Perricone, Ahmed M. Elhossiny, Alexis C Harold, Paola I Medina-Cabrera, Padma Kadiyala, Jiaqi Shi, Timothy L. Frankel, Eileen S Carpenter, Michael D. Green, Cristina Mitrea, C. Lyssiotis, M. Pasca di Magliano
          </td>
          <td>2024-08-13</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d5e47f21e09bedd3c2faf022dd2ac57af3858d" target='_blank'>
              Large field of view and spatial region of interest transcriptomics in fixed tissue
              </a>
            </td>
          <td>
            Xiaonan Cui, Xue Dong, Mengzhu Hu, Wenjian Zhou, Weiyang Shi
          </td>
          <td>2024-08-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55bf88cd2f361cbe0e78275013e0864970ee199" target='_blank'>
              In vivo CRISPR screens identify a dual function of MEN1 in regulating tumor–microenvironment interactions
              </a>
            </td>
          <td>
            Peiran Su, Yin Liu, Tianyi Chen, Yibo Xue, Yong Zeng, Guanghui Zhu, Sujun Chen, Mona Teng, Xinpei Ci, Mengdi Guo, Michael Y. He, Jun Hao, Vivian Chu, Wenxi Xu, Shiyan Wang, P. Mehdipour, Xin Xu, Sajid A. Marhon, F. Soares, Nhu-An Pham, Bell Xi Wu, Peter Hyunwuk Her, Shengrui Feng, Najd Alshamlan, Maryam Khalil, Rehna Krishnan, Fangyou Yu, Chang Chen, Francis Burrows, Razqallah Hakem, Mathieu Lupien, Shane Harding, Benjamin H. Lok, Catherine O'Brien, Alejandro Berlin, D. D. De Carvalho, David G Brooks, Daniel Schramek, Ming-Sound Tsao, Housheng Hansen He
          </td>
          <td>2024-09-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Although the combination of immunotherapy and radiotherapy (RT) for the treatment of malignant tumors has shown rapid development, the insight of how RT remodels the tumor microenvironment to prime antitumor immunity involves a complex interplay of cell types and signaling pathways, much of which remains to be elucidated. Four tumor samples were collected from the same abdominal wall metastasis site of the patient with gastric cancer at baseline and during fractionated RT for single-cell RNA and T-cell receptor sequencing. The Seurat analysis pipeline and immune receptor analysis were used to characterize the gastric cancer metastasis ecosystem and investigated its dynamic changes of cell proportion, cell functional profiles and cell-to-cell communication during RT. Immunohistochemical and immunofluorescent staining and bulk RNA sequencing were applied to validate the key results. We found tumor cells upregulated immune checkpoint genes in response to RT. The infiltration and clonal expansion of T lymphocytes declined within tumors undergoing irradiation. Moreover, RT led to the accumulation of proinflammatory macrophages and natural killer T cells with enhanced cytotoxic gene expression signature. In addition, subclusters of dendritic cells and endothelial cells showed decrease in the expression of antigen present features in post-RT samples. More ECM component secreted by myofibroblasts during RT. These findings indicate that RT induced the dynamics of the immune response that should be taken into consideration when designing and clinically implementing innovative multimodal cancer treatment regimens of different RT and immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523682bb135dcad73fb3cd605aea7d93bedb4b8c" target='_blank'>
              Dynamic single-cell sequencing unveils the tumor microenvironment evolution of gastric cancer abdominal wall metastases during radiotherapy.
              </a>
            </td>
          <td>
            Qianqian Mao, Zhenzhen Wu, Yonghong Lai, Ling Wang, Qiongzhi Zhao, Xi Xu, Xiansheng Lu, Wenjun Qiu, Zhi-Huan Zhang, Jiani Wu, Gaofeng Wang, Rui Zhou, Jianhua Wu, Huiying Sun, N. Huang, Xiatong Huang, Luyang Jiang, Yiran Fang, Yuyun Kong, Li Liang, J. Bin, Yulin Liao, Min Shi, Wangjun Liao, D. Zeng
          </td>
          <td>2024-09-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Adenoid cystic carcinoma (ACC) is a malignant tumor primarily originating from the salivary glands, capable of affecting multiple organs. Although ACC typically exhibits slow growth, it is notorious for its propensity for neural invasion, local recurrence, and distant metastasis, making it a particularly challenging cancer to treat. The complexity of ACC's histological and molecular features poses significant challenges to current treatment modalities, which often show limited effectiveness. Recent advancements in single‐cell RNA‐sequencing (scRNA‐seq) have begun to unravel unprecedented insights into the heterogeneity and subpopulation diversity within ACC, revealing distinct cellular phenotypes and origins. This review delves into the intricate pathological and molecular characteristics of ACC, focusing on recent therapeutic advancements. We particularly emphasize the insights gained from scRNA‐seq studies that shed light on the cellular landscape of ACC, underscoring its heterogeneity and pathobiology. Moreover, by integrating analyses from public databases, this review proposes novel perspectives for advancing treatment strategies in ACC. This review contributes to the academic understanding of ACC by proposing novel therapeutic approaches informed by cutting‐edge molecular insights, paving the way for more effective, personalized therapeutic approaches for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e81e2fac0a605e61e6708a46d6b2502839fd4" target='_blank'>
              Adenoid cystic carcinoma: insights from molecular characterization and therapeutic advances
              </a>
            </td>
          <td>
            Yunxuan Jia, Yupeng Liu, Haitang Yang, Feng Yao
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Molecular and cellular characterization of tumors is essential due to the complex and heterogeneous nature of cancer. In recent decades, many bioinformatic tools and experimental techniques have been developed to achieve personalized characterization of tumors. However, sample handling continues to be a major challenge as limitations such as prior treatments before sample acquisition, the amount of tissue obtained, transportation, or the inability to process fresh samples pose a hurdle for experimental strategies that require viable cell suspensions. Here, we present an optimized protocol that allows the recovery of highly viable cell suspensions from breast cancer primary tumor biopsies. Using these cell suspensions we have successfully characterized genome architecture through Hi-C. Also, we have evaluated single-cell gene expression and the tumor cellular microenvironment through single-cell RNAseq. Both technologies are key in the detailed and personalized molecular characterization of tumor samples. The protocol described here is a cost-effective alternative to obtain viable cell suspensions from biopsies simply and efficiently.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6963d9d1f4834d1881e0dfb691c6907687ec39cc" target='_blank'>
              Obtention of viable cell suspensions from breast cancer tumor biopsies for 3D chromatin conformation and single-cell transcriptome analysis
              </a>
            </td>
          <td>
            Aura Stephenson-Gussinye, Luis A. Rendón-Bautista, B. E. Ruiz-Medina, Eduardo Blanco-Olais, Rosario Pérez-Molina, Cleofas Marcial-Medina, Y. Chávarri-Guerra, Enrique Soto-Pérez-de-Celis, Andrea Morales-Alfaro, A. Esquivel-López, Fernando Candanedo-González, Armando Gamboa-Domínguez, Rubén Cortes-González, Alejandro Alfaro-Goldaracena, Sara E. Vázquez-Manjarrez, Guido Grajales-Figueroa, Beatriz Astudillo-Romero, Jesús Ruiz-Manriquez, A. C. Poot-Hernández, Paula Licona-Limón, Mayra Furlan-Magaril
          </td>
          <td>2024-08-22</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Tumor‐associatedmacrophages (TAMs) exhibit remarkable heterogeneity in glioblastoma. Spatially resolved single‐cell transcriptomic studies identified a monocyte‐derived TAM subset localized in the peri‐necrotic niche, driven by hypoxic cues to acquire ahypoxia response signature. These hypoxia‐TAMs destabilize endothelial adherens junctions through adrenomedullin paracrine signaling, promoting the formation of hyperpermeable neovasculature that impedes drug delivery. Blocking adrenomedullin produced by hypoxia‐TAMs restores vascular integrity, increases drug deliveryinto tumors, and provides combinatorial therapeutic benefits. Here we discuss the heterogeneity of TAMs regarding functional states and locations in glioblastomas, and propose future directions for studying the temporospatial dynamics of multifaceted TAM. Highlights Single‐cell omics reveal a functionally and spatially distinct hypoxia‐TAM subset in glioblastoma. Adrenomedullin secreted by hypoxia‐TAM destabilizes tumor vasculature and its blockade enhances vessel integrity and drug delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fdc1e11579a80529898e9b0f1ec0aa652779e" target='_blank'>
              Identification of hypoxic macrophages in glioblastoma: Unveiling therapeutic insights from tumour microenvironment analysis
              </a>
            </td>
          <td>
            Zhen Qin, Xiu-Wu Bian, Yu Shi
          </td>
          <td>2024-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of spatially resolved transcriptomics (SRT) has greatly advanced our understanding of cellular microenvironments by integrating spatial information with molecular data collected from multiple tissue sections or individuals. However, methods for multi-sample spatial clustering are lacking, and existing methods primarily rely on molecular information alone. This paper introduces BayeSMART, a Bayesian statistical method designed to identify spatial domains across multiple samples. BayeSMART leverages artificial intelligence (AI)-reconstructed single-cell level information from the paired histology images of multi-sample SRT datasets while simultaneously considering the spatial context of gene expression. The AI integration enables BayeSMART to effectively interpret the spatial domains. We conducted case studies using four datasets from various tissue types and SRT platforms and compared BayeSMART with alternative multi-sample spatial clustering approaches and a number of state-of-the-art methods for single-sample SRT analysis, demonstrating that it surpasses existing methods in terms of clustering accuracy, interpretability, and computational efficiency. BayeSMART offers new insights into the spatial organization of cells in multi-sample SRT data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a670df0d414e493a2809eeaa5b1d8c9a56e43c" target='_blank'>
              BayeSMART: Bayesian Clustering of Multi-sample Spatially Resolved Transcriptomics Data
              </a>
            </td>
          <td>
            Yanghong Guo, Bencong Zhu, Chen Tang, Ruichen Rong, Ying Ma, Guanghua Xiao, Lin Xu, Qiwei Li
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature.Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature.A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo.This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="Gallbladder carcinoma (GBC) is a malignant hepatobiliary cancer characterized by an intricate tumor microenvironments (TME) and heterogeneity. The traditional GBC 2D culture models cannot faithfully recapitulate the characteristics of the TME. Three-dimensional (3D) bioprinting enables the establishment of high-throughput and high-fidelity multicellular GBC models. In this study, we designed a concentric cylindrical tetra-culture model to reconstitute the spatial distribution of cells in tumor tissue, with the inner portion containing GBC cells, and the outer ring containing a mixture of endothelial cells, fibroblasts, and macrophages. We confirmed the survival, proliferation, biomarker expression and gene expression profiles of GBC 3D tetra-culture models. Hematoxylin-eosin (HE) and immunofluorescence staining verified the morphology and robust expression of GBC/endothelial/fibroblast/macrophage biomarkers in GBC 3D tetra-culture models. Single-cell RNA sequencing revealed two distinct subtypes of GBC cells within the model, glandular epithelial and squamous epithelial cells, suggesting the mimicry of intratumoral heterogeneity. Comparative transcriptome profile analysis among various in vitro models revealed that cellular interactions and the TME in 3D tetra-culture models reshaped the biological processes of tumor cells to a more aggressive phenotype. GBC 3D tetra-culture models restored the characteristics of the TME as well as intratumoral heterogeneity. Therefore, this model is expected to have future applications in tumor biology research and antitumor drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc6d2cc903e389bd57ed19225ad293a9f8472b5b" target='_blank'>
              Multicellular 3D bioprinted human gallbladder carcinoma forin vitromimicry of tumor microenvironment and intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Yukai Jin, Jiangang Zhang, Jiali Xing, Yiran Li, Huiyu Yang, Liujian Ouyang, Zhiyuan Fang, Lejia Sun, Bao Jin, Pengyu Huang, Huayu Yang, Shunda Du, X. Sang, Yilei Mao
          </td>
          <td>2024-08-09</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The success of chimeric antigen receptor (CAR) T-cell therapy in treating hematologic malignancies has generated widespread interest in translating this technology to solid cancers. However, issues like tumor infiltration, the immunosuppressive tumor microenvironment, and tumor heterogeneity limit its efficacy in the solid tumor setting. Recent experimental and clinical studies propose local administration directly into the tumor or at the tumor site to increase CAR T-cell infiltration and improve treatment outcomes. Characteristics of the types of solid tumors that may be the most receptive to this treatment approach remain unclear. In this work, we develop a spatiotemporal model for CAR T-cell treatment of solid tumors, and use numerical simulations to compare the effect of introducing CAR T cells via intratumoral injection versus intracavitary administration in diverse cancer types. We demonstrate that the model can recapitulate tumor and CAR T-cell data from imaging studies of local administration of CAR T cells in mouse models. Our results suggest that locally administered CAR T cells will be most successful against slowly proliferating, highly diffusive tumors, which have the lowest average tumor cell density. These findings affirm the clinical observation that CAR T cells will not perform equally across different types of solid tumors, and suggest that measuring tumor density may be helpful when considering the feasibility of CAR T-cell therapy and planning dosages for a particular patient. We additionally find that local delivery of CAR T cells can result in deep tumor responses, provided that the initial CAR T-cell dose does not contain a significant fraction of exhausted cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7a6bf493c0b33ad1aad3083fdba685e7c59e1d" target='_blank'>
              Spatiotemporal dynamics of tumor - CAR T-cell interaction following local administration in solid cancers
              </a>
            </td>
          <td>
            Katherine Owens, Aminur Rahman, Ivana Bozic
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Alpha-smooth muscle actin (α-SMA) expression in the stroma is linked to the presence of cancer-associated fibroblasts and is known to correlate with worse outcomes in various tumors. In this study, using a GeoMx digital spatial profiling approach, we characterized the gene expression of the tumor and α-SMA-expressing stromal cell compartments in pancreatic neuroendocrine tumors (PanNETs). The profiling was performed on tissues from eight retrospective cases (three grade 1, four grade 2, and one grade 3). Selected regions of interest were segmented geometrically based on tissue morphology and fluorescent signals from synaptophysin and α-SMA markers. The α-SMA-expressing stromal-cell-associated genes were involved in pathways of extracellular matrix modification, whereas, in tumor cells, the gene expression profiles were associated with pathways involved in cell proliferation. The comparison of gene expression profiles across all three PanNET grades revealed that the differences between grades are not only present at the level of the tumor but also in the α-SMA-expressing stromal cells. Furthermore, the tumor cells from regions with a rich presence of adjacent α-SMA-expressing stromal cells revealed an upregulation of matrix metalloproteinase-9 (MMP9) expression in grade 3 tumors. This study provides an in-depth characterization of gene expression profiles in α-SMA-expressing stromal and tumor cells, and outlines potential crosstalk mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fe06c12f5865d2f0fe11fa3ad5ad66746e58fe" target='_blank'>
              Tumor and α-SMA-expressing stromal cells in pancreatic neuroendocrine tumors have a distinct RNA profile depending on tumor grade.
              </a>
            </td>
          <td>
            H. Niedra, R. Peculis, Rihards Saksis, Ilona Mandrika, Sofija Vilisova, Jurijs Nazarovs, Austra Breikša, Aija Gerina, Julie Earl, Ignacio Ruz-Caracuel, M. Rosas, A. Puķītis, Natalja Senterjakova, V. Rovite
          </td>
          <td>2024-09-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor-educated platelets (TEPs) are a potential method of liquid biopsy for the diagnosis and monitoring of cancer. However, the mechanism underlying tumor education of platelets is not known, and transcripts associated with TEPs are often not tumor-associated transcripts. We demonstrated that direct tumor transfer of transcripts to circulating platelets is an unlikely source of the TEP signal. We used CDSeq, a latent Dirichlet allocation algorithm, to deconvolute the TEP signal in blood samples from patients with glioblastoma. We demonstrated that a substantial proportion of transcripts in the platelet transcriptome are derived from non-platelet cells, and the use of this algorithm allows the removal of contaminant transcripts. Furthermore, we used the results of this algorithm to demonstrate that TEPs represent a subset of more activated platelets, which also contain transcripts normally associated with non-platelet inflammatory cells, suggesting that these inflammatory cells, possibly in the tumor microenvironment, transfer transcripts to platelets that are then found in circulation. Our analysis suggests a useful and efficient method of processing TEP transcriptomic data to enable the isolation of a unique TEP signal associated with specific tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1407d9261db4f9fb6a0492efe30995b31483fd" target='_blank'>
              Deconvolution of the tumor-educated platelet transcriptome reveals activated platelet and inflammatory cell transcript signatures.
              </a>
            </td>
          <td>
            Jerome M Karp, A. Modrek, R. Ezhilarasan, Ze-Yan Zhang, Yingwen Ding, Melanie Graciani, Ali Sahimi, Michele Silvestro, Ting Chen, Shuai Li, Kwok-Kin Wong, Bhama Ramkhelawon, Krishna Pl Bhat, Erik P Sulman
          </td>
          <td>2024-08-27</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The bone marrow microenvironment is critical for B-cell acute lymphoblastic leukemia (B-ALL) but its cellular heterogeneity remains poorly defined. Here, we employed single-cell RNA sequencing to comprehensively characterize the stromal and hematopoietic niches in pediatric B-ALL. Our analysis revealed two distinct mesenchymal stromal cell (MSC) populations as primary leukemia-supportive niches: early mesenchymal progenitors and adipogenic progenitors. Single-cell transcriptomic analysis infers that ALL blasts use distinct cell-cell interactions to communicate with the different stromal populations. Purified adipogenic progenitors from the bone of children with ALL support survival of the leukemic blasts ex vivo and their signature is enriched in relapse samples. Our data establish adipogenic progenitors as a distinct and novel component of the ALL niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f24fbb432c05f1e5ee3ed61b3b778c9a293df20" target='_blank'>
              Distinct Stromal Cell Populations Define the B-cell Acute Lymphoblastic Leukemia Microenvironment
              </a>
            </td>
          <td>
            Mauricio N. Ferrao Blanco, Bexultan Kazybay, Mirjam E Belderbos, Olaf Heidenreich, H. J. Vormoor
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Disruption of processes involved in tissue development and homeostatic self-renewal is increasingly implicated in cancer initiation, progression, and recurrence. The adrenal cortex is a dynamic tissue that undergoes life-long turnover. Here, using genetic fate mapping and murine adrenocortical carcinoma (ACC) models, we have identified a population of adrenocortical stem cells that express delta-like non-canonical Notch ligand 1 (DLK1). These cells are active during development, near dormant postnatally but are re-expressed in ACC. In a study of over 200 human ACC samples, we have shown DLK1 expression is ubiquitous and is an independent prognostic marker of recurrence-free survival. Paradoxically, despite its progenitor role, spatial transcriptomic analysis has identified DLK1 expressing cell populations to have increased steroidogenic potential in human ACC, a finding also observed in four human and one murine ACC cell lines. Finally, the cleavable DLK1 ectodomain is measurable in patients’ serum and can discriminate between ACC and other adrenal pathologies with high sensitivity and specificity to aid in diagnosis and follow-up of ACC patients. These data demonstrate a prognostic role for DLK1 in ACC, detail its hierarchical expression in homeostasis and oncogenic transformation and propose a role for its use as a biomarker in this malignancy. Graphical abstract Statement of significance This study presents DLK1 as a novel biomarker in ACC with opportunities for use in the diagnosis, prognosis and longitudinal follow up of patients. DLK1, a marker of adrenocortical stem cells, is re-expressed in ACC, is measurable in patients’ serum and is associated with increased malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9baab22c63f6f34cafaad1ee20a4c197797bfaf" target='_blank'>
              Dlk1 is a novel adrenocortical stem/progenitor cell marker that predicts malignancy in adrenocortical carcinoma
              </a>
            </td>
          <td>
            Katia Mariniello, J. Pittaway, B. Altieri, Kleiton S. Borges, Irene Hadjidemetriou, Claudio Ribeiro, G. Ruiz-Babot, Jiang A. Lim, Julie Foster, Julie Cleaver, Jane Sosabowski, Nafis Rahman, Milena Doroszko, C. Hantel, Sandra Sigala, A. Abate, M. Tamburello, Katja Kiseljak-Vassiliades, Margaret Wierman, L. Parvanta, Tarek E. Abdel-Aziz, Teng-Teng Chung, Aimee Di Marco, F. Palazzo, Celso E Gomez-Sanchez, David R. Taylor, Oliver Rayner, C. Ronchi, Carles Gaston-Massuet, S. Sbiera, William M. Drake, Emanuel Rognoni, Matthias Kroiss, D. T. Breault, M. Fassnacht, L. Guasti
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a31875ebe3aec993187c2c6c6d773a5efac6f4" target='_blank'>
              Human intermediate prostate cancer stem cells contribute to the initiation and development of prostate adenocarcinoma
              </a>
            </td>
          <td>
            Jie Mu, Ruizhi Li, Yu Zheng, Yi Lu, Lei Ma, Lin Yin, Miao Zhang, Wenyu Ma, Mengjia Chang, Aihua Liu, Jing Li, Haihui Zhu, Dong Wang
          </td>
          <td>2024-09-11</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatial techniques such as spatial transcriptomes and MALDI-MSI, offering insights into both transcripts and metabolite of tissue sections. However, integrating them with high accuracy is challenge due to no shared spots or features. We present haCCA, a workflow designed to integrate spatial transcriptomes and metabolomes data using high-correlated feature pairs and modified spatial morphological alignment. This approach ensures high-resolution and accurate spot-to-spot data integration across neighbor tissue section. We applied haCCA to both publicly available 10X Visium and MALDI-MSI datasets from mouse brain tissue and a custom spatial transcriptome and MALDI-MSI dataset from an intrahepatic cholangiocarcinoma (ICC) model, exploring the metabolic alteration of NETs(neutrophil extracellular traps) on ICC, and finding a potential mechanism that NETs upregulated Scd1 to activate fatty acid metabolism. Providing new insights into the dynamic crosstalk between genes and metabolites that regulates the tumor biological behavior and drives the response to treatment. We developed and published an easy-to-use Python package to facilitate its use.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a471f5555baf62c3ebcc010fc547002f0b3c386" target='_blank'>
              haCCA: Multi-module Integrating of spatial transcriptomes and metabolomes
              </a>
            </td>
          <td>
            Xiaotian Shen, Jing Xu, Chen Zhang, Xiaoyun Zhang, Zhou-Qing Chen, Hu-Liang Jiang, Lu-Yu Yang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, J. Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, Brendan C Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Y. Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Hematopoietic stem and progenitor cells (HSPCs) rely on intercellular signaling to maintain and adjust their production of blood and immune cells. This process occurs in the semi-fluidic bone marrow, hosting dozens of cell types that constantly migrate and interact. To elucidate the dynamic network of cell-cell interaction and signaling transduction underlying hematopoiesis, we developed an algorithm to measure Cell-cell Spatial Interaction Probability (CellSIP) by integrating data on ligand and receptor expression, cell type abundance, and cellular spatial positioning. Using new and published mouse datasets, we validated CellSIP and uncovered signaling transductions indicating feedback mechanisms underlying hematopoiesis. Moreover, we identified significant correlations between signaling pathways across individual HSPCs at the same hematopoiesis stage. These pathway correlations illuminate the organization of cellular and signaling networks underlying hematopoiesis, revealing new regulators through their associations with established ones. The signaling quantification and correlation data are available through the Hematopoiesis Intercellular Signaling Explorer (HISE).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf26ff56c34b3ddada82e3d4951fade44feb8315" target='_blank'>
              Cellular and molecular networks of intercellular signaling in bone marrow hematopoiesis
              </a>
            </td>
          <td>
            Zachary Thomas, Bowen Wang, Wade R. Boohar, Mary Vergel, Jiya Eerdeng, Dayeon J. Shon, M. Elowitz, Rong Lu
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Simple Summary Spatial transcriptomics is a technique of measuring gene expression with spatial resolution on a tissue slide, which is especially useful in cancer research. This scoping study reviews 41 articles published by the end of 2023 to examine current trends for employed methods, applications, and data analysis approaches for spatial transcriptomics, identifying challenges and practical uses of this technique in studying cancer. The research shows that cancer is a major focus in spatial transcriptomics studies, with certain tools and methods being particularly popular among scientists. However, many studies do not fully explain their data processing methods, making it hard to reproduce their results. Emphasis on transparent sharing of analysis scripts and tailored single-cell analysis methods for ST is recommended to enhance study reproducibility and reliability in the domain. This work may stand as a spatial transcriptomics developmental snapshot and reference to contemporary neoplasm biology research conducted through this technique. Abstract Spatial transcriptomics (ST) examines gene expression within its spatial context on tissue, linking morphology and function. Advances in ST resolution and throughput have led to an increase in scientific interest, notably in cancer research. This scoping study reviews the challenges and practical applications of ST, summarizing current methods, trends, and data analysis techniques for ST in neoplasm research. We analyzed 41 articles published by the end of 2023 alongside public data repositories. The findings indicate cancer biology is an important focus of ST research, with a rising number of studies each year. Visium (10x Genomics, Pleasanton, CA, USA) is the leading ST platform, and SCTransform from Seurat R library is the preferred method for data normalization and integration. Many studies incorporate additional data types like single-cell sequencing and immunohistochemistry. Common ST applications include discovering the composition and function of tumor tissues in the context of their heterogeneity, characterizing the tumor microenvironment, or identifying interactions between cells, including spatial patterns of expression and co-occurrence. However, nearly half of the studies lacked comprehensive data processing protocols, hindering their reproducibility. By recommending greater transparency in sharing analysis methods and adapting single-cell analysis techniques with caution, this review aims to improve the reproducibility and reliability of future studies in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546384150762685b69a172f984b5438d683edf6a" target='_blank'>
              Scoping Review: Methods and Applications of Spatial Transcriptomics in Tumor Research
              </a>
            </td>
          <td>
            Kacper Maciejewski, Patrycja Czerwińska
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Background Pancreatic ductal adenocarcinoma (PDAC) is the fourth leading cause of cancer- related deaths in the United States and is often not diagnosed until it has already metastasized. Our prior work established that calcium signaling pathways facilitate a pro-metastatic phenotype through the induction of an epithelial-to-mesenchymal transition (EMT) program in PDAC tumor cells, but the downstream signaling molecules mediating this phenotype have remained elusive. MethodsUsing a novel genetically engineered model of pancreatic cancer, we demonstrate that loss of Calcium/calmodulin-dependent kinase-II B (CaMK2B), a downstream effector kinase activated by calcium, enhances cellular plasticity, and increases metastasis. To this end, we generated a genetically engineered mouse model where Camk2b was knocked out in tumor cells in a lineage-labeled Kras- and p53-driven PDAC mouse model (herein KPCY-Camk2bcKO). Results Genetic deletion of Camk2b in the KPCY genetically engineered model accelerated tumor formation and dramatically reduced the overall median survival of KPCY- Camk2b cKO mice (78 days) relative to KPCY controls (136 days). Pathological and immunofluorescence analysis of KPCY-Camk2b cKO animals showed an increased number of poorly differentiated tumor regions and a higher metastatic burden, with >85% of KPCY- Camk2b cKO animals presenting with multi-focal metastases. Further analysis of tumor cells in KPCY-Camk2b cKO mice showed enhanced expression of markers of the highly aggressive squamous PDAC subtypes. Strikingly, Camk2b deletion combined with oncogenic Kras G12D (in the absence of p53 alterations), was sufficient to form lethal metastatic PDAC tumors, while age-matched control Kras G12D tumors with intact Camk2b only progressed to the acinar-to-ductal metaplasia (ADM) or Pancreatic Intraepithelial Neoplasia (PanIN) stage. Collectively, these results establish Camk2b loss as a new oncogenic and pro-metastatic mechanism in PDAC Conclusion This work demonstrates that Camk2b loss enhances tumor cell evolution and plasticity within PDAC to promote metastasis. In this context, we speculate that loss of Camk2b rewires calcium signaling to drive an aggressive subtype of PDAC with increased metastasis.
 Citation Format: Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason Pitarresi. Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7848880ff2d74aa871376f23a8aeef10bd0463c1" target='_blank'>
              Abstract C061: Loss of CaMK2B accelerates tumor formation and enhances metastatic competency in genetically engineered mouse models of PDAC
              </a>
            </td>
          <td>
            Jessica Peura, Nikita Bhalerao, Calvin Johnson, Yamini Ogoti, Faith Keller, Emma Watson, Zhen Zhao, Jason R. Pitarresi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G. Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cell type identity is determined by gene regulatory networks (GRNs), comprising the expression of specific transcription factors (TFs) regulating target genes (TGs) via binding to open chromatin regions (OCRs). The regulatory logic of differentiation includes factors specific to one or multiple cell types, functioning in a combinatorial fashion. Classic approaches of GRN discovery used perturbational data to elucidate TF-TG links, but are laborious and not scalable across the tree of life. Single cell transcriptomics has emerged as a revolutionary approach to study gene expression with cell type resolution, but incorporating perturbational data is challenging. Planarians, with their pluripotent neoblast stem cells continuously giving rise to all cell types, offer an ideal model to attempt this integration. Despite extensive single cell transcriptomic studies, the transcriptional and chromatin regulation at the cell type level remains unexplored. Here, we investigate the regulatory logic of planarian stem cell differentiation by obtaining an organism-level integration of single cell transcriptomics and single cell accessibility data. We identify specific open chromatin profiles for major differentiated cell types and analyse their transcriptomic landscape, revealing distinct gene modules expressed in individual types and combinations of them. Integrated analysis unveils gene networks reflecting known TF interactions in each type and identifies TFs potentially driving differentiation across multiple cell types. To validate our predictions, we combined TF knockdown RNAi experiments with single cell transcriptomics. We focus on hnf4, a TF known to be expressed in gut phagocytes, and confirm its influence on other types, including parenchymal cells. Our results demonstrate high overlap between predicted targets and experimentally-validated differentially-regulated genes. Overall, our study integrates TFs, TGs and OCRs to reveal the regulatory logic of planarian stem cell differentiation, showcasing that the combination of single cell methods and perturbational studies will be key for characterising GRNs widely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d812fbadba7ebc9a3898e7e95eae459989657669" target='_blank'>
              The Regulatory Logic of Planarian Stem Cell Differentiation
              </a>
            </td>
          <td>
            Alberto Pérez-Posada, Helena García-Castro, Elena Emili, Virginia Vanni, Cirenia Arias-Baldrich, S. Frölich, S. V. van Heeringen, N. Kenny, Jordi Solana
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab20abff3d5ab2e3023b99dd1dd6ab980ad6081" target='_blank'>
              Mutual exclusivity and co-occurrence patterns of immune checkpoints indicate NKG2A relates to anti-PD-1 resistance in gastric cancer
              </a>
            </td>
          <td>
            Guanjun Li, Xiaohan Liu, Chunhui Gu, Guichuang Ma, Shaowei Li, Zhenfeng Ma, Yanqiu Xiong, Yu Jiang, Qiong Huang, Jianhua Wu, Zhenzhen Wu, Wangjun Liao, Qijing Wu, Min Shi
          </td>
          <td>2024-08-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cbe85e7267d943e47fa9cdeee4b8e805b7a651" target='_blank'>
              CD24 flags anastasis in melanoma cells.
              </a>
            </td>
          <td>
            Martina H Vasileva, Anette Bennemann, Karolin Zachmann, Michael P Schön, Jorge Frank, Vijay Kumar Ulaganathan
          </td>
          <td>2024-08-13</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the analysis of changes in phenotype, such as in vivo and ex vivo morphology and localization of cellular proteins and organelles can provide important insights into patient treatment options. Novel therapies are needed to extend survival in metastatic castration-resistant prostate cancer (mCRPC). Prostate-specific membrane antigen (PSMA), a cell surface glycoprotein that is commonly overexpressed by prostate cancer (PC) cells relative to normal prostate cells, provides a validated target. We developed a software for image analysis designed to identify PSMA expression on the surface of epithelial cells in order to extract prognostic metrics. In addition, our software can deliver predictive information and inform clinicians regarding the efficacy of PC therapy. We can envisage additional applications of our software system, beyond PC, as PSMA is expressed in a variety of tissues. Our method is based on image denoising, topologic partitioning, and edge detection. These three steps allow to segment the area of each PSMA spot in an image of a coverslip with epithelial cells. Our objective has been to present the community with an integrated, easy to use by all, tool for resolving the complex cytoskeletal organization and it is our goal to have such software system approved for use in the clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926fb40317d30bcdb538d1f8f9766fb5c180eb87" target='_blank'>
              Quantitative Microscopy in Medicine
              </a>
            </td>
          <td>
            Alexandre Matov, Shayan Modiri
          </td>
          <td>2024-08-03</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Cancer-Associated Fibroblasts (CAFs) constitute a heterogeneous group of cells critical for the remodeling of the tumor microenvironment (TME). Given their significant impact on tumor progression, particularly in skin cancers, a deeper understanding of their characteristics and functions is essential.


METHODS
This study employed a single-cell transcriptomic analysis to explore the diversity of CAFs within three major types of skin cancer: basal cell carcinoma, melanoma, and head and neck squamous cell carcinoma. We applied analytical techniques, including Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), pseudotime tracking, metabolic profiling, and stemness assessment to delineate and define the functional attributes of identified CAF subgroups.


RESULTS
Our analysis successfully delineated nine distinct CAF subgroups across the studied tumor types. Of particular interest, we identified a novel CAF subtype, designated as C0, exclusive to basal cell carcinoma. This subtype exhibits phenotypic traits associated with invasive and destructive capabilities, significantly correlating with the progression of basal cell carcinoma. The identification of this subgroup provides new insights into the role of CAFs in cancer biology and opens avenues for targeted therapeutic strategies.


CONCLUSION
A pan-cancer analysis was performed on three cancers, BCC, MA, and HNSCC, focusing on tumor fibroblasts in TME. Unsupervised clustering categorized CAF into nine subpopulations, among which the C0 subpopulation had a strong correspondence with BCC-CAF and an invasive- destructive-related phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b65ae0c3c55b73c69971a741aaa77cb534354a3" target='_blank'>
              Pan-Cancer Single-Cell Analysis Revealing the Heterogeneity of Cancer-Associated Fibroblasts in Skin Tumors.
              </a>
            </td>
          <td>
            Yichi Zhang, Zhijie Zhao, Wenyi Huang, Byeong Seop Kim, Li Lin, Xin Li, Mengyuan Hou, Li Li, Yan Zhang, Wenjing Xi, G. Chai
          </td>
          <td>2024-09-25</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Effective drug discovery relies on combining target knowledge with functional assays and multi-omics data to understand chemicals’ molecular actions. However, the relationship between cell morphology and gene expression over time and across cell lines remains unclear. To explore this, we analyzed Cell Painting and L1000 data for 106 compounds across three cell lines from osteoblast, lung, and breast tumors (U2OS, A549, and MCF7) at three time points (6h, 24h, 48h) using a 10µM concentration. We found significant time and cell line effects in Cell Painting data, with less pronounced effects in transcriptomics. Using Weighted Gene Co-expression Network Analysis (WGCNA) and enrichment analysis, we identified connections between cell morphology and gene deregulation for chemicals with similar biological effects (e.g., HDAC and CDK inhibitors). These findings suggest that while Cell Painting shows distinct patterns, both technologies offer complementary insights into compound-induced cellular changes, enhancing drug discovery and chemical risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e8eb6d8184a42daa0a1c6534359ca9aca19c28" target='_blank'>
              Cell morphology and gene expression: tracking changes and complementarity across time and cell lines
              </a>
            </td>
          <td>
            Vanille Lejal, David Rouquié, Olivier Taboureau
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite improvements in patient outcomes, pediatric cancer remains a leading cause of non-accidental death in children. Recent genetic analysis of patients with pediatric cancers indicates an important role for both germline genetic predisposition and cancer-specific somatic driver mutations. Increasingly, evidence demonstrates that the developmental timepoint at which the cancer cell-of-origin transforms is critical to tumor identity and therapeutic response. Therefore, future therapeutic development would be bolstered by the use of disease models that faithfully recapitulate the genetic context, cell-of-origin, and developmental window of vulnerability in pediatric cancers. Human stem cells have the potential to incorporate all of these characteristics into a pediatric cancer model, while serving as a platform for rapid genetic and pharmacological testing. In this review, we describe how human stem cells have been used to model pediatric cancers and how these models compare to other pediatric cancer model modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5451f14ab5137f3c1ef0749fac76fd503e0d7a63" target='_blank'>
              Pediatric Tumors as Disorders of Development: The Case for In Vitro Modeling Based on Human Stem Cells
              </a>
            </td>
          <td>
            Cullen Clairmont, Joanna J Gell, Ching C Lau
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8018e573a8779af2c11f1aae1832a3b88d066791" target='_blank'>
              Targeting SWI/SNF ATPases reduces neuroblastoma cell plasticity.
              </a>
            </td>
          <td>
            Man Xu, Jason J Hong, Xiyuan Zhang, Ming Sun, Xingyu Liu, Jeeyoun Kang, Hannah Stack, Wendy Fang, Haiyan Lei, Xavier Lacoste, Reona Okada, Raina Jung, Rosa Nguyen, J. Shern, Carol J Thiele, Zhihui Liu
          </td>
          <td>2024-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251acda7df20c5eff2f9bfbbad717de02554a201" target='_blank'>
              Unlocking the potential of HHLA2: identifying functional immune infiltrating cells in the tumor microenvironment and predicting clinical outcomes in laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenjing Li, Jianqing You, Haixiang Xue, Yi Liu, Junjun Chen, Xiao Zheng, Lujun Chen, Changping Wu
          </td>
          <td>2024-08-06</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic cancer is a highly lethal and aggressive disease, commonly characterized by a unique tumor microenvironment, where mutant KRAS signaling plays a pivotal role in its pathophysiology. To deepen our understanding of KRAS-dependent signaling in earliest tumor development, oncogenic KRAS (KRASG12D) expression was induced in human stem cell-derived pancreatic ductal-like organoids (PDLOs), followed by time-resolved single-cell RNA and bulk ATAC sequencing. This unique precancerous state model facilitated a comprehensive characterization of early oncogenic events at cellular resolution. Furthermore, a machine learning approach was applied to accurately predict KRAS-dependent cell identities. Within seven days, mutated KRAS triggers distinct gene signature programs related to extracellular matrix remodeling and inflammation. Although derived from dysplastic yet non-cancerous states, these gene signatures negatively correlated with overall survival in PDAC (pancreatic ductal adenocarcinoma) patients. The KRAS-dependent signatures retrieved with the machine learning approach allowed the grouping of 24 published PDAC cases, showing differences in niche composition and communication patterns. Various co-culture systems demonstrated that KRASG12D-expressing PDLOs could activate pancreatic stellate cells, inducing a cancer- associated fibroblast (CAF)-like state. Simultaneously, these KRAS-activated organoids were protected from T cell infiltration in vitro. Furthermore, in silico approaches were employed to reconstruct a virtual pancreatic cancer space, demonstrating that our PDLOs transcriptionally bridge healthy and malignant ductal states. This also revealed potential KRAS-driven pathways mediating CAF activation and T cell shielding preceding cancer onset. In summary, our human PDLO model enables us to explore the transition from normal to cancerous states, addressing gaps in our understanding of tumor development and tumor niche preparation.
 Citation Format: Chantal Allgoewer, Markus Breunig, Meike Hohwieler, Medhanie Mulaw, Alexander Kleger. Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc4900be652d49c2d242f617b8845d915d29b" target='_blank'>
              Abstract C010: Single-cell profiling unleashes KRAS-driven redrafting of the tumor microenvironment before cancer onset
              </a>
            </td>
          <td>
            Chantal Allgoewer, M. Breunig, M. Hohwieler, Medhanie M Mulaw, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The glioblastoma (GBM) tumor microenvironment consists of a heterogeneous mixture of neoplastic and non-neoplastic cells, including immune cells. Tumor recurrence following standard-of-care therapy results in a rich landscape of inflammatory cells throughout the glioma-infiltrated cortex. Immune cells consisting of glioma-associated macrophages and microglia (GAMMs) overwhelmingly constitute the bulk of the recurrent glioblastoma (rGBM) microenvironment, in comparison to the highly cellular and proliferative tumor microenvironment characteristic of primary GBM. These immune cells dynamically interact within the tumor microenvironment and can contribute to disease progression and therapy resistance while also providing novel targets for emerging immunotherapies. Within these varying contexts, histological-based assessments of immune cells in rGBM, including immunohistochemistry (IHC) and immunofluorescence (IF), offer a critical way to visualize and examine the inflammatory landscape. Here, we exhaustively review the available body of literature on the inflammatory landscape in rGBM as identified through histological-based assessments. We highlight the heterogeneity of immune cells throughout the glioma-infiltrated cortex with a focus on microglia and macrophages, drawing insights from canonical and novel immune-cell histological markers to estimate cell phenotypes and function. Lastly, we discuss opportunities for immunomodulatory treatments aiming to harness the inflammatory landscape in rGBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4001488ad8c82d2b66d1744897ff8a71e5f6d40" target='_blank'>
              Unveiling the Inflammatory Landscape of Recurrent Glioblastoma through Histological-Based Assessments
              </a>
            </td>
          <td>
            Nicholas B. Dadario, D. Boyett, Damian E Teasley, Peter J. Chabot, Nathan J. Winans, Michael G. Argenziano, Colin P. Sperring, P. Canoll, Jeffrey N. Bruce
          </td>
          <td>2024-09-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9fca0c358cd3b69f5d15cee5978ab92d495cb1" target='_blank'>
              STIE: Single-cell level deconvolution, convolution, and clustering in in situ capturing-based spatial transcriptomics
              </a>
            </td>
          <td>
            Shijia Zhu, Naoto Kubota, Shidan Wang, Tao Wang, Guanghua Xiao, Y. Hoshida
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian clear cell carcinoma (CCC) has an East Asian preponderance. It is associated with endometriosis, a benign condition where endometrial (inner lining of the uterus) tissue is found outside the uterus and on the peritoneal surface, in the abdominal or pelvic space. CCC is relatively more resistant to conventional chemotherapy compared to other ovarian cancer subtypes and is associated with a poorer prognosis. In this study, we recruited and obtained tumour tissues from seven patients across the four stages of CCC. The tumour and the tumour microenvironment (TME) from 7 CCC patients spanning clinical stages 1-4 were transcriptionally profiled using high-resolution scRNA-seq to gain insight into CCC’s biological mechanisms. Firstly, we built a scRNA-seq resource for the CCC tumour microenvironment (TME). Secondly, we identified the different cell type proportions and found high levels of immune infiltration in CCC. Thirdly, since CCC is associated with endometriosis, we compared CCC with two publicly available endometriosis scRNA-seq datasets. The CCC malignant cells showed similarities with glandular secretory and ciliated epithelial cells found in endometriosis. Finally, we determined the differences in cell-cell communication between various cell types present in CCC TME and endometriosis conditions to gain insights into the transformations in CCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880aa346199a24fc3c757ae09dbee83e6a3ff0a7" target='_blank'>
              Deciphering tumour microenvironment and elucidating the origin of cancer cells in ovarian clear cell carcinoma
              </a>
            </td>
          <td>
            Uma S Kamaraj, Pradeep Gautam, Terence Cheng, Tham Su Chin, Sun Kuie Tay, Tew Hong Ho, R. Nadarajah, Ronald Chin Hong Goh, Shing Lih Wong, Sangeeta Mantoo, I. Busmanis, Hu Li, Minh TN Le, Qi-Jing Li, Elaine Hsuen Lim, Y. Loh
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemoradiation and immune therapies. There is limited biomarker to identify subsets of PDAC that responds to immunotherapy or targeted therapy. We have identified a subset of PDACs (approximately 25%) carrying COMPASS-like complex (CLC) gene mutations. Analysis of the clinicopathologic and molecular features of this PDAC subset revealed that these tumors are an aggressive group of PDACs with poor or squamous histologic differentiation, increased rates of TP53 mutations, and poor patient survival compared to control PDACs. Our preliminary data showed that PDACs carrying mutations in two members of the CLC genes (KDM6A and KMT2D) have immunosuppressive tumor microenvironment (TME), suggesting that CLC genes are candidate biomarkers for immunotherapy. However, a comprehensive analysis of the immune composition, immune checkpoint expression, and the relationship between tumor cells and immune cells in CLC gene mutated PDACs compared to controls has not been performed. In this study, we characterized the immune composition in CLC gene mutated PDACs compared to paired control PDACs using tyramide signal amplification multiplex fluorescent immunohistochemistry (mfIHC). Methods: Human PDAC tissue slides were stained with 2 mfIHC panels and cell types were defined by different markers: panel 1 (PanCK, CD163, PD-L1, CD3, CD8, FoxP3) and panel 2 (PanCK, CD163, CD3, CD8, TIGIT, TIM3). Random image sections were selected in each sample (5∼8 sections per sample) and divided into wild-type (WT) group (n=57) and mutant group (n=41) and analyzed using InForm Cell Analysis software. Tissue segmentation training was performed to separate stromal and epithelial elements. Cell percentage, cell density, and nearest neighbor distance of each cell type were calculated using R program. Results: The CLC gene mutation is associated with elevated epithelial cells (ECs) and PDL1+ ECs and increased infiltration of exhausted cytotoxic T cells (TIGIT+/TIM3+ CD8 T cells). CLC gene mutated PDACs are characterized by spatial proximity between exhausted T cells (TIGIT+) and epithelial cells/antigen presenting cells (APCs). Higher number of epithelial cells is associated with reduced infiltration of helper T cells in the mutant PDAC tissue. We also observed that high numbers of APC in proximity to CD4 T cells were positively correlated with the proximity of EC and CD8 cytotoxic T cells as well as the proximity of CD4 T cells and cytotoxic T cells in the WT group but not in the CLC gene mutant group. Conclusion: Our results provide insight into the effects of CLC gene mutations on shaping the immune microenvironment of PDAC and suggest that CLC gene mutated PDACs have an immunosuppressive immune microenvironment than other PDACs and may respond differently to immunotherapy.
 Citation Format: Shungang Zhang, Elaina Daniels, Jake McGue, Hong Sun Kim, Dafydd Thomas, Timothy Frankel, Jiaqi Shi. Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2790a085665b4a36a3c8b84d6bede9ac8c16c70" target='_blank'>
              Abstract A040: Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer
              </a>
            </td>
          <td>
            Shungang Zhang, Elaina Daniels, Jake McGue, H. Kim, Dafydd Thomas, Timothy L. Frankel, Jiaqi Shi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62fe0d4817b1cce8f7a8474cb3953d097a1d805f" target='_blank'>
              The spatial distribution of intermediate fibroblasts and myeloid-derived cells dictate lymph node metastasis dynamics in oral cancer
              </a>
            </td>
          <td>
            Soni Shaikh, Harsh Dhar, Manju Moorthy, Vijayalakshmi Bhat, Sangramjit Basu, Devmalya Banerjee, Deepak Kumar Mishra, Sourav Datta, Geetashree Mukherjee
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="A comprehensive evaluation of the relationship between the densities of various cell types in the breast cancer tumor microenvironment and patient prognosis is currently lacking. Additionally, the absence of a large patch-level whole slide imaging (WSI) dataset of breast cancer with annotated cell types hinders the ability of artificial intelligence to evaluate cell density in breast cancer WSI. We first employed Lasso-Cox regression to build a breast cancer prognosis assessment model based on cell density in a population study. Pathology experts manually annotated a dataset containing over 70,000 patches and used transfer learning based on ResNet152 to develop an artificial intelligence model for identifying different cell types in these patches. The results showed that significant prognostic differences were observed among breast cancer patients stratified by cell density score (P = 0.0018), with the cell density score identified as an independent prognostic factor for breast cancer patients (P < 0.05). In the validation cohort, the predictive performance for overall survival (OS) was satisfactory, with area under the curve (AUC) values of 0.893 (OS) at 1-year, 0.823 (OS) at 3-year, and 0.861 (OS) at 5-year intervals. We trained a robust model based on ResNet152, achieving over 99% classification accuracy for different cell types in patches. These achievements offer new public resources and tools for personalized treatment and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17a923b124c2c0b21b7e052645a7723de6fefc4" target='_blank'>
              Detecting cell types and densities in the tumor microenvironment improves prognostic risk assessment for breast cancer.
              </a>
            </td>
          <td>
            Pu Liu, Xueli Zhang, Wenwen Wang, Yunping Zhu, Yongfang Xie, Yanhong Tai, Jie Ma
          </td>
          <td>2024-08-16</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial in cancer development and therapeutic response. Immunotherapy is increasingly recognized as a critical component of cancer treatment. While immunotherapies have shown efficacy in various cancers, including breast cancer, patient responses vary widely. Some patients receive significant benefits, while others experience minimal or no improvement. This disparity underscores the complexity and diversity of the immune system. In this study, we investigated the immune landscape and cell–cell communication within the TME of breast cancer through integrated analysis of bulk and single-cell RNA sequencing data. We established profiles of tumor immune infiltration that span across a broad spectrum of adaptive and innate immune cells. Our clustering analysis of immune infiltration identified three distinct patient groups: high T cell abundance, moderate infiltration, and low infiltration. Patients with low immune infiltration exhibited the poorest survival rates, while those in the moderate infiltration group showed better outcomes than those with high T cell abundance. Moreover, the high cell abundance group was associated with a greater tumor burden and higher rates of TP53 mutations, whereas the moderate infiltration group was characterized by a lower tumor burden and elevated PIK3CA mutations. Analysis of an independent single-cell RNA-seq breast cancer dataset confirmed the presence of similar infiltration patterns. Further investigation into ligand–receptor interactions within the TME unveiled significant variations in cell–cell communication patterns among these groups. Notably, we found that the signaling pathways SPP1 and EGF were exclusively active in the low immune infiltration group, suggesting their involvement in immune suppression. This work comprehensively characterizes the composition and dynamic interplay in the breast cancer TME. Our findings reveal associations between the extent of immune infiltration and clinical outcomes, providing valuable prognostic information for patient stratification. The unique mutations and signaling pathways associated with different patient groups offer insights into the mechanisms underlying diverse tumor immune infiltration and the formation of an immunosuppressive tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff73b13094e5afc913b4266529d0100e73db7ee" target='_blank'>
              Characterizing the Tumor Microenvironment and Its Prognostic Impact in Breast Cancer
              </a>
            </td>
          <td>
            Wenjuan Zhang, Alex Lee, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2024-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="210 Background: Tissue-resident memory T cells (TRM) are a specialized subset of long-lived memory T cells that reside in peripheral tissues. However, the impact of TRM-related immunosurveillance on the tumor-immune microenvironment (TIME) and tumor progression across various non-small-cell lung cancer (NSCLC) patient populations is yet to be elucidated. Methods: Our comprehensive analysis of multiple independent single-cell and bulk RNA-seq datasets of patient NSCLC samples generated reliable, unique TRM signatures, through which we inferred the abundance of TRM in NSCLC. A machine learning model was developed to prognosticate survival, tested in multiple independent NSCLC cohorts. Model performance was corroborated through Kaplan-Meier survival plots, receiver operating characteristic (ROC) curves, principal component analysis, and t-SNE analyses. Results: We discovered that TRM abundance is consistently positively correlated with CD4+ T helper 1 cells, M1 macrophages, and resting dendritic cells in the TIME. In addition, TRM signatures are strongly associated with immune checkpoint genes and the prognosis of NSCLC patients. A TRM-based machine learning model to predict patient survival was validated and an 18-gene risk score was further developed to effectively stratify patients into low-risk and high-risk categories, wherein patients with high-risk scores had significantly lower overall survival than patients with low-risk. The prognostic value of the risk score was independently validated by the TCGA dataset and multiple independent NSCLC patient datasets. Notably, low-risk NSCLC patients with higher TRM infiltration exhibited enhanced T-cell immunity, nature killer cell activation, and other TIME immune responses related pathways, indicating a more active immune profile benefitting from immunotherapy. However, the TRM signature revealed low TRM abundance and a lack of prognostic association among lung squamous cell carcinoma patients in contrast to adenocarcinoma, indicating that the two NSCLC subtypes are driven by distinct TIMEs. Conclusions: Altogether, this study provides valuable insights into the complex interactions between TRM and TIME and their impact on NSCLC patient prognosis. The development of a simplified 18-gene risk score provides a practical prognostic marker for risk stratification. Keywords: Tissue resident memory T cell, non-small-cell lung cancer, prognosis, tumor immune microenvironment, machine learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f194e94cc8938f5d7330f21276b21d12f1c6b81d" target='_blank'>
              A comprehensive meta-analysis of tissue resident memory T cell shaping non-small-cell lung cancer immune microenvironment and patient prognosis.
              </a>
            </td>
          <td>
            Aidan Shen, Aliesha Garrett, Cheng-chi Chao, Dongliang Liu, Yangzhi Zhu, Junhua Mai, Chongming Jiang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95736f8e150d70ccc63e3057497b476be28ab1d9" target='_blank'>
              CCKBR+ cancer cells contribute to the intratumor heterogeneity of gastric cancer and confer sensitivity to FOXO inhibition.
              </a>
            </td>
          <td>
            Zhenya Tan, Ke Pan, Minqiong Sun, Xianzhu Pan, Zhi Yang, Zhiling Chang, Xue Yang, Jicheng Zhu, Li Zhan, Yakun Liu, Xiaofei Li, Keqiong Lin, Lin Chen, Hui Mo, Wei Luo, Chen Kan, Lunxi Duan, Hong Zheng
          </td>
          <td>2024-08-20</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3b4d74609d2880453dad8460ce60ad53367b365" target='_blank'>
              MerlinS13 phosphorylation regulates meningioma Wnt signaling and magnetic resonance imaging features
              </a>
            </td>
          <td>
            Charlotte D Eaton, Lauro Avalos, S. J. Liu, Zhenhong Chen, Naomi Zakimi, T. Casey-Clyde, Paola Bisignano, Calixto-Hope G Lucas, Erica Stevenson, A. Choudhury, H. Vasudevan, Stephen T. Magill, Jacob S Young, N. Krogan, J. Villanueva‐Meyer, D. Swaney, D. Raleigh
          </td>
          <td>2024-09-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3fe31e6aa7a99673a4d6d194c90f42b74a1aa1" target='_blank'>
              Immunogenomic features of radiologically distinctive nodules in multiple primary lung cancer
              </a>
            </td>
          <td>
            Mei-Cheng Chen, Haolong Yang, Zhi Dong, Lu-Jie Li, Xiang-Min Li, Hong-He Luo, Qiong Li, Ying Zhu
          </td>
          <td>2024-09-05</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6029311c7be5cdea36f9e6f715640956c9f739" target='_blank'>
              Biophysically interpretable inference of cell types from multimodal sequencing data.
              </a>
            </td>
          <td>
            T. Chari, Gennady Gorin, L. Pachter
          </td>
          <td>2024-09-01</td>
          <td>Nature computational science</td>
          <td>1</td>
          <td>74</td>
        </tr>

        <tr id="The ability of a cancer cell population to achieve heterogeneity in their phenotype distributions offers advantages in tumor invasiveness and drug resistance. Studying the mechanisms behind such observed heterogeneity in mammalian cells presents challenges due for instance to the prolonged proliferation times compared to widely studied unicellular organisms like bacteria and yeast. Here, we studied the response of leukemia cell populations to serum starvation via a protocol, we recently developed, that makes use of live cell fluorescence and flow cytometry in combination with a quantitative analytical model to follow the population proliferation while monitoring the dynamics of its phenotype distributions. We found that upon switching between a serum-rich to a serum-poor media, leukemia cells (i) maintain a memory of the previous environment up to one generation even in the presence of severe medium-depletion, before (ii) adapting their growth and division rates to the novel environment while preserving a sizer-like division strategy. Finally, looking at the mitochondria content of the proliferating vs non-proliferating cells, we found that the latter is characterized by a higher number of older mitochondria, suggesting a possible functional role of the observed asymmetric partitioning of (aged) mitochondria in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ae5c0661fc382b4dfefa11021ab88af1607a56" target='_blank'>
              Probing leukemia cells behavior under starvation
              </a>
            </td>
          <td>
            Simone Scalise, Giorgio Gosti, G. Ruocco, Giovanna Peruzzi, Mattia Miotto
          </td>
          <td>2024-08-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3d4f61a2577f3255d5a96a1f62094418b1629b" target='_blank'>
              A cell state-specific metabolic vulnerability to GPX4-dependent ferroptosis in glioblastoma.
              </a>
            </td>
          <td>
            Matei A. Banu, Athanassios Dovas, Michael G. Argenziano, Wenting Zhao, Colin P. Sperring, Henar Cuervo Grajal, Zhouzerui Liu, Dominique Mo Higgins, Misha Amini, B. Pereira, Ling F Ye, A. Mahajan, N. Humala, J. Furnari, P. Upadhyayula, Fereshteh Zandkarimi, Trang Tt Nguyen, Damian E Teasley, Peter B Wu, Li Hai, Charles Karan, Tyrone Dowdy, Aida Razavilar, M. Siegelin, Jan Kitajewski, Mioara Larion, Jeffrey N. Bruce, Brent R. Stockwell, Peter A. Sims, Peter D Canoll
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) is a type of blood and bone marrow cancer characterized by the proliferation of abnormal clonal haematopoietic cells in the bone marrow leading to bone marrow failure. Over the course of the disease, angiogenic factors released by leukaemic cells drastically alter the bone marrow vascular niches resulting in observable structural abnormalities. We use a technique from topological data analysis - persistent homology - to quantify the images and infer on the disease through the imaged morphological features. We find that persistent homology uncovers succinct dissimilarities between the control, early, and late stages of AML development. We then integrate persistent homology into stage-dependent Gaussian mixture models for the first time, proposing a new class of models which are applicable to persistent homology summaries and able to both infer patterns in morphological changes between different stages of progression as well as provide a basis for prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/908fdd9f4a85fe4603f9b7353c67f06a0aab3abc" target='_blank'>
              A Topological Gaussian Mixture Model for Bone Marrow Morphology in Leukaemia
              </a>
            </td>
          <td>
            Qiquan Wang, Anna Song, A. Batsivari, Dominique Bonnet, Anthea Monod
          </td>
          <td>2024-08-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Advances in omics technologies have allowed spatially resolved molecular profiling of single cells, providing a window not only into the diversity and distribution of cell types within a tissue but also into the effects of interactions between cells in shaping the transcriptional landscape. Cells send chemical and mechanical signals which are received by other cells, where they can subsequently initiate context-specific gene regulatory responses. These interactions and their responses shape the individual molecular phenotype of a cell in a given microenvironment. RNAs or proteins measured in individual cells, together with the cells' spatial distribution, provide invaluable information about these mechanisms and the regulation of genes beyond processes occurring independently in each individual cell. SpaCeNet is a method designed to elucidate both the intracellular molecular networks (how molecular variables affect each other within the cell) and the intercellular molecular networks (how cells affect molecular variables in their neighbors). This is achieved by estimating conditional independence relations between captured variables within individual cells and by disentangling these from conditional independence relations between variables of different cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ef214444a5271219b33a7d73c5e44038be1f04" target='_blank'>
              Spatial Cellular Networks from omics data with SpaCeNet.
              </a>
            </td>
          <td>
            Stefan Schrod, Niklas Lück, R. Lohmayer, S. Solbrig, Dennis Völkl, Tina Wipfler, K. Shutta, Marouen Ben Guebila, Andreas Schäfer, Tim Beißbarth, Helena U. Zacharias, Peter J Oefner, John Quackenbush, M. Altenbuchinger
          </td>
          <td>2024-09-04</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human tumor samples hold a plethora of information about their microenvironment and immune repertoire. Effective dissociation of human tissue samples into viable cell suspensions is a required input for the single-cell RNA sequencing (scRNAseq) pipeline. Unlike bulk RNA sequencing approaches, scRNAseq enables us to infer the transcriptional heterogeneity in tumor specimens at the single-cell level. Incorporating this approach in recent years has led to many discoveries, such as identifying immune and tumor cellular states and programs associated with clinical responses to immunotherapies and other types of treatments. Moreover, single-cell technologies applied to dissociated tissues can be used to identify accessible chromatin regions T and B cell receptor repertoire, and the expression of proteins, using DNA barcoded antibodies (CITEseq). The viability and quality of the dissociated sample are critical variables when using these technologies, as these can dramatically affect the cross-contamination of single cells with ambient RNA, the quality of the data, and interpretation. Moreover, long dissociation protocols can lead to the elimination of sensitive cell populations and the upregulation of a stress response gene signature. To overcome these limitations, we devised a rapid universal dissociation protocol, which has been validated on multiple types of human and murine tumors. The process begins with mechanical and enzymatic dissociation, followed by filtration, red blood lysis, and live dead enrichment, suitable for samples with a low input of cells (e.g., needle core biopsies). This protocol ensures a clean and viable single-cell suspension paramount to the successful generation of Gel Bead-In Emulsions (GEMs), barcoding, and sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe2078d45af3e9f2f3826e3ee2e17b4140e6605" target='_blank'>
              Dissociation of Human and Mouse Tumor Tissue Samples for Single-cell RNA Sequencing.
              </a>
            </td>
          <td>
            Jacy Fang, Isabella Salinas, Sarah San Vicente, Caroline Zielinski, M. Sade-Feldman
          </td>
          <td>2024-08-16</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880d6fbe977b9fa0abf41b1330834beac1e2a6aa" target='_blank'>
              A patient-derived cell model for malignant transformation in IDH-mutant glioma
              </a>
            </td>
          <td>
            Olga Kim, Zach Sergi, Guangyang Yu, Kazu Yamamoto, Martha Quezado, Zied Abdullaev, Danel R Crooks, Shun Kishimoto, Qi Li, Peng Lu, Burchelle N. Blackman, T. Andresson, Xiaolin Wu, Bao Tran, Jun S. Wei, Wei Zhang, Meili Zhang, Hua Song, Javed Khan, Murali C. Krishna, J. Brender, Jing Wu
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background FOXR2-activated central nervous system (CNS) neuroblastoma (CNS NB-FOXR2) is a recently identified subtype of brain tumor characterized by the elevated expression of the transcription factor FOXR2 mainly due to genomic rearrangements. However, the precise pathogenic mechanisms, including the cell type of origin, remain elusive. Methods A gene expression analysis of patient tumors was performed to identify putative cell types of origin. Based on this prediction, a new human embryonic stem cell–based model was developed to validate the origin and to examine the molecular and cellular mechanisms underlying the formation of CNS NB-FOXR2. Results Our data showed that CNS NB-FOXR2 tumors express a high level of lineage marker genes associated with the medial ganglionic eminence (MGE), a transient structure located in the developing ventral forebrain. Our model confirmed the cell-type-specific effect of FOXR2 on the proliferation and in vivo tumorigenicity. Additionally, we found that FOXR2 overexpression activated the MEK/ERK signaling pathway through a suppression of the endogenous RAS inhibitor DIRAS3. The MEK inhibitor trametinib suppressed the proliferation of FOXR2-expressing MGE progenitors more than nonexpressing cells. Conclusions Our study collectively demonstrates that MGE progenitors are the cell of origin of CNS NB-FOXR2 and that FOXR2 activates the MEK/ERK signaling pathway, providing a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8445a3e44a2783d48d762e8340d244130bab0b2f" target='_blank'>
              A human embryonic stem cell–based model reveals the cell of origin of FOXR2-activated CNS neuroblastoma
              </a>
            </td>
          <td>
            Hitomi N Royston, Autumn B Hampton, Dhruv Bhagat, Evonne F Pinto, Miriam D Emerson, Kosuke Funato
          </td>
          <td>2024-08-12</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Vulvar diseases are a critical yet often neglected area of women’s health, profoundly affecting patients’ quality of life and frequently resulting in long-term physical and psychological challenges. Lichen sclerosus (LS) is a chronic inflammatory skin disorder that predominantly affects the vulva, leading to severe itching, pain, scarring, and an increased risk of malignancy. Despite its profound impact on affected individuals, the molecular pathogenesis of vulvar LS (VLS) is not well understood, hindering the development of FDA-approved therapies. Here, we utilize single-cell and spatial transcriptomics to analyze lesional and non-lesional skin from VLS patients, as well as healthy control vulvar skin. Our findings demonstrate histologic, cellular, and molecular heterogeneities within VLS, yet highlight unifying molecular changes across keratinocytes, fibroblasts, immune cells, and melanocytes in lesional skin. They reveal cellular stress and damage in fibroblasts and keratinocytes, enhanced T cell activation and cytotoxicity, aberrant cell-cell signaling, and increased activation of the IFN, JAK/STAT, and p53 pathways in specific cell types. Using both monolayer and organotypic culture models, we also demonstrate that knockdown of select genes, which are downregulated in VLS lesional keratinocytes, partially recapitulates VLS-like stress-associated changes. Collectively, these data provide novel insights into the pathogenesis of VLS, identifying potential biomarkers and therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a1ede19b990ffc821eea644860eccc4ba0949" target='_blank'>
              Single-cell and spatial transcriptomics of vulvar lichen sclerosus reveal multi-compartmental alterations in gene expression and signaling cross-talk
              </a>
            </td>
          <td>
            Peng Sun, Christina N. Kraus, Wei Zhao, Jiahui Xu, Susie Suh, Quy Nguyen, Yunlong Jia, Arjun Nair, Melanie Oakes, Roberto Tinoco, Jessica Shiu, Bryan Sun, Ashley Elsensohn, Scott X. Atwood, Qing Nie, Xing Dai
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Immunotherapy represents a groundbreaking and monumental achievement in the field of cancer therapy, marking a significant advancement in fighting against this devastating disease. Lung cancer has showed consistent clinical improvements in response to immunotherapy treatments, yet, it is undeniable that challenges such as limited response rates acquire resistance, and the unclear fundamental mechanisms were inevitable problems.


METHODS
The cellular composition was defined and distinguished through single-cell RNA sequencing (scRNA-seq) analysis of MPR (major pathologic response) and NMPR (non-major pathologic response) samples in GSE207422, including four primary MPR samples and eight primary NMPR samples.


RESULTS
We found obvious difference in CD8+ T cell population between MPR and NMPR samples, with high expression of TYMS, RRM2, and BIRC5 in NPMR samples. Meanwhile, the proportion of macrophages and tumor epithelial cells infiltration increased in the NMPR samples. We discovered biomarkers (ACTN4, ATF3, BRD2, CDKN1A, and CHMP4B) in epithelial cells which were potentially represented worse outcomes.


CONCLUSIONS
By exploring the difference of tumor microenvironment (TME) in samples with different corresponding degrees of neoadjuvant immunotherapy, this research introduces a number of novel biomarkers for predicting the response of treatment and a theoretical basis for overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eb58f4e74cf2a065722c81b24475ac3132ac67a" target='_blank'>
              Single-cell RNA sequencing reveals microenvironmental infiltration in non-small cell lung cancer with different responses to immunotherapy.
              </a>
            </td>
          <td>
            Xinnan Hu, Yonghui Wu, Lixin Wang, Fujun Yang, Lingyun Ye, Xiaoxia Chen, Xiao Song, Ping Wei
          </td>
          <td>2024-09-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="High-grade gliomas are a major health challenge with poor prognosis and high morbidity. Immune-checkpoint inhibitors (ICI) have emerged as promising therapeutic options for several malignancies yet show little efficacy against central nervous system (CNS) tumors. CD200 is a newly recognized immune checkpoint that modulates immune homeostasis. CD200 protein is expressed by a variety of cells, including immune cells and stromal cells, and is overexpressed by many tumors. The shedding of CD200 from tumor cells can create an immunosuppressive environment that dampens anti-tumor immunity by modulating cytolytic activity and cytokine expression both within and outside the tumor microenvironment (TME). While it is well-accepted that CD200 induces a pro-tumorigenic environment through its ability to suppress the immune response, we sought to determine the role of glioma-specific expression of CD200. We show that CD200 is expressed across glioma types, is shed from tumor cells, and increases over time in the serum of patients undergoing immunotherapy. Using CD200 knockout (KO) glioma models, we demonstrated that glioma cell-derived CD200 promotes tumor growth in vivo and in vitro. Notably, CD200 KO gliomas are spontaneously rejected by their host, a process that required a fully functional immune system, including NK and T-cells. Moreover, we report that glioma-derived or brain-injected soluble CD200 contributes to the suppression of antigen-specific CD8 T-cells in the draining lymph nodes (dLNs). Our work provides new mechanistic insights regarding CD200-mediated immunosuppression by gliomas. Statement of significance We demonstrate mechanisms of the druggable glioma-derived CD200 checkpoint on tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b634d44ec918f6bd798f77ff4dc7cd21cc67c2e" target='_blank'>
              CD200 depletion in glioma enhances antitumor immunity and induces tumor rejection
              </a>
            </td>
          <td>
            Itay Raphael, Anzar A. Mujeeb, Elisabet Ampudia-Mesias, ReidAnn E. Sever, Brandon L. McClellan, Stephen C. Frederico, Chaim T. Sneiderman, Apoorva Mirji, Ali Daba, Francisco Puerta-Martínez, Michal Nisnboym, W. Edwards, Michael W. Graner, Christopher L. Moertel, Maria G. Castro, Gary Kohanbash, Michael Olin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Recent advances in single cell sequencing have led to an increased focus on the role of cell-type composition in phenotypic presentation and disease progression. Cell-type composition research in the heart is challenging due to large, frequently multinucleated cardiomyocytes that preclude most single cell approaches from obtaining accurate measurements of cell composition. Our in silico studies reveal that ignoring cell type composition when calculating differentially expressed genes (DEGs) can have significant consequences. For example, a relatively small change in cell abundance of only 10% can result in over 25% of DEGs being false positives. Methods We have implemented an algorithmic approach that uses snRNAseq datasets as a reference to accurately calculate cell type compositions from bulk RNAseq datasets through robust data cleaning, gene selection, and multi-sample cross-subject and cross-cell-type deconvolution. We applied our approach to cardiomyocyte-specific α1A adrenergic receptor (CM-α1A-AR) knockout mice. 8-12 week-old mice (either WT or CM-α1A-KO) were subjected to permanent left coronary artery (LCA) ligation or sham surgery (n=4 per group). Transcriptomes from the infarct border zones were collected 3 days later and analyzed using our algorithm to determine cell-type abundances, corrected differential expression calculations using DESeq2, and validated these findings using RNAscope. Results Uncorrected DEGs for the CM-α1A-KO X LCA interaction term featured many cell-type specific genes such as Timp4 (fibroblasts) and Aplnr (cardiomyocytes) and overall GO enrichment for terms pertaining to cardiomyocyte differentiation (P=3.1E-4). Using our algorithm, we observe a striking loss of cardiomyocytes and gain in fibroblasts in the α1A-KO + LCA mice that was not recapitulated in WT + LCA animals, although we did observe a similar increase in macrophage abundance in both conditions. This recapitulates prior results that showed a much more severe heart failure phenotype in CM-α1A-KO + LCA mice. Following correction for cell-type, our DEGs now highlight a novel set of genes enriched for GO terms such as cardiac contraction (P=3.7E-5) and actin filament organization (P=6.3E-5). Conclusions Our algorithm identifies and corrects for cell-type abundance in bulk RNAseq datasets opening new avenues for research on novel genes and pathways as well as an improved understanding of the role of cardiac cell types in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217ee4d32d0eb92d12b53e00c08af7cc8fb059d2" target='_blank'>
              Novel Insights into Post-Myocardial Infarction Cardiac Remodeling through Algorithmic Detection of Cell-Type Composition Shifts
              </a>
            </td>
          <td>
            Brian Gural, Logan Kirkland, Abbey Hockett, Peyton Sandroni, Jiandong Zhang, M. Rosa-Garrido, Samantha K. Swift, Douglas J. Chapski, Michael A. Flinn, Caitlin C. O’Meara, T. Vondriska, Michaela Patterson, Brian C. Jensen, Christoph D Rau
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="BACKGROUND
A specific type of prostate cancer (PC) that exhibits neuroendocrine (NE) differentiation is known as NEPC. NEPC has little to no response to androgen deprivation therapy and is associated with the development of metastatic castration-resistant PC (CRPC), which has an extremely poor prognosis. Our understanding of genetic drivers and activated pathways in NEPC is limited, which hinders precision medicine approaches. L1 cell adhesion molecule (L1CAM) is known to play an oncogenic role in metastatic cancers, including CRPC. However, the impact of L1CAM on NEPC progression remains elusive.


METHODS
L1CAM expression level was investigated using public gene expression databases of PC cohorts and patient-derived xenograft models. L1CAM knockdown was performed in different PC cells to study in vitro cell functions. A subline of CRPC cell line CWR22Rv1 was established after long-term exposure to abiraterone to induce NE differentiation. The androgen receptor-negative cell line PC3 was cultured under the tumor sphere-forming condition to enrich cancer stemness features. Several oxidative stress inducers were tested on PC cells to observe L1CAM-mediated gene expression and cell death.


RESULTS
L1CAM expression was remarkably high in NEPC compared to CRPC or adenocarcinoma tumors. L1CAM was also correlated with NE marker expressions and associated with the adenocarcinoma-to-NEPC progression in gene expression databases and CRPC cells with NE differentiation. L1CAM also promoted cancer stemness and NE phenotypes in PC3 cells under cancer stemness enrichment. L1CAM was also identified as a reactive oxygen species-induced gene, by which L1CAM counteracted CRPC cell death triggered by ionizing radiation.


CONCLUSIONS
Our results unveiled a new role of L1CAM in the acquisition of the NE phenotype in PC, contributing to the NE differentiation-related therapeutic resistance of CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058f0ff5a1eb85bfaf3568b71d602509408bcc46" target='_blank'>
              L1CAM mediates neuroendocrine phenotype acquisition in prostate cancer cells.
              </a>
            </td>
          <td>
            Chia-Ling Hsieh, Anh Duy Do, Chia-Yen Hsueh, Mafewu Olga Raboshakga, Tran Ngoc Thanh, Tran Tien Tai, Hsing-Jien Kung, Shain-Ying Sung
          </td>
          <td>2024-08-18</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/587d631ba6a320ce9960d57b667aeb776ca548db" target='_blank'>
              Development of 3D-iNET ORION: a novel, pre-clinical, three-dimensional in vitro cell model for modeling human metastatic neuroendocrine tumor of the pancreas
              </a>
            </td>
          <td>
            J. Strnadel, M. Valasek, Grace Y Lin, Huahui Lin, Ann M Ponsford Tipps, Sang Myung Woo, Ken Fujimura, Huawei Wang, Sunkyu Choi, Jack Bui, Christopher Hermosillo, Kristen Jepsen, Michael R Navarro, J. Kelber, R. Klemke, Michael Bouvet
          </td>
          <td>2024-08-05</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Yvette Hayat, Sarah Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, Emmanuelle Huillard, Nina Pottier, Aurore Desmons, Inès Fayache, Garrett A. Kaas, Philip J. Kingsley, Lawrence J. Marnett, Eric Duplus, Elias El-Habr, Lucas Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, Michele Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, Mirjam E Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Myelodysplastic syndrome and acute myeloid leukemia (AML) belong to a continuous disease spectrum of myeloid malignancies with poor prognosis in the relapsed/refractory setting necessitating novel therapies. Natural killer (NK) cells from patients with myeloid malignancies display global dysfunction with impaired killing capacity, altered metabolism, and an exhausted phenotype at the single-cell transcriptomic and proteomic levels. In this study, we identified that this dysfunction was mediated through a cross-talk between NK cells and myeloid blasts necessitating cell-cell contact. NK cell dysfunction could be prevented by targeting the αvβ-integrin/TGF-β/SMAD pathway but, once established, was persistent because of profound epigenetic reprogramming. We identified BATF as a core transcription factor and the main mediator of this NK cell dysfunction in AML. Mechanistically, we found that BATF was directly regulated and induced by SMAD2/3 and, in turn, bound to key genes related to NK cell exhaustion, such as HAVCR2, LAG3, TIGIT, and CTLA4. BATF deletion enhanced NK cell function against AML in vitro and in vivo. Collectively, our findings reveal a previously unidentified mechanism of NK immune evasion in AML manifested by epigenetic rewiring and inactivation of NK cells by myeloid blasts. This work highlights the importance of using healthy allogeneic NK cells as an adoptive cell therapy to treat patients with myeloid malignancies combined with strategies aimed at preventing the dysfunction by targeting the TGF-β pathway or BATF. BATF deletion prevents epigenetic rewiring and enhances the antitumor activity of NK cells in AML. Editor’s summary Patients with myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML), often relapse after standard of care. To improve outcomes, Kumar et al. have investigated the role of natural killer (NK) cells in antitumor immunity against myeloid blasts and have identified global dysfunction of NK cells in these patients. They identified BATF as responsible for this dysfunction, and by knocking out BATF downstream of the TGF-β canonical pathway in NK cells, they were able to enhance NK cell function against AML in vitro and in vivo. This has led to the initiation of a promising clinical trial using these TGFBR2 knockout NK cells against myeloid malignancies. —Dorothy Hallberg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1bf3fa2449127d2f41a09515bf651d31d92c80" target='_blank'>
              BATF is a major driver of NK cell epigenetic reprogramming and dysfunction in AML
              </a>
            </td>
          <td>
            Bijender Kumar, A. Singh, R. Başar, N. Uprety, Ye Li, Huihui Fan, Ana Karen Nunez Cortes, Mecit Kaplan, S. Acharya, Hila Shaim, Anna C Xu, Manrong Wu, Emily Ensley, Dexing Fang, P. Banerjee, L. M. Garcia, S. Tiberti, Paul Lin, H. Rafei, Maliha Nuzhat Munir, Madison Moore, M. Shanley, M. Mendt, L. Kerbauy, Bin Liu, Alexander Biederstädt, Elif Gokdemir, Susmita Ghosh, Kiran Kundu, Francia Reyes-Silva, X. Jiang, Xinhai Wan, April L. Gilbert, Merve Dede, V. Mohanty, J. Dou, Patrick Zhang, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Abhinav K. Jain, J. J. Rodríguez-Sevilla, S. Colla, Guillermo Garcia-Manero, Elizabeth J. Shpall, Ken Chen, Hussein A. Abbas, Kunal Rai, Katayoun Rezvani, M. Daher
          </td>
          <td>2024-09-11</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The primary function of the thyroid gland is the synthesis and release of thyroid hormones, which are essential for health from embryogenesis to adulthood. Thyroid disorders occur frequently and include congenital hypothyroidism, which occurs due to aberrant thyroid development (thyroid dysgenesis) or impaired hormone synthesis and is particularly prevalent in trisomy 21 (T21). In contrast, thyroid carcinoma, an acquired disorder, is the most common endocrine malignancy in both paediatric and adult populations. Understanding the molecular basis of thyroid dysgenesis and paediatric thyroid carcinoma remains challenging, and requires an improved understanding of foetal thyroid development. To address this, we generated a comprehensive spatiotemporal atlas of the human thyroid during the first and second trimesters of pregnancy. Profiling over 200,000 cells with single-cell sequencing revealed key cell types involved in thyroid gland development, including the hormone-producing thyrocytes. We discovered that foetal thyroid follicular cells are heterogeneous epithelial populations consisting of two main functional subtypes (fTFC1, fTFC2), with fTFC2 expressing increased levels of PAX8, and spatial transcriptomics revealed subtype co-occurrence within individual follicles. While both fTFC1 and fTFC2 persist in adult thyroid, fTFC2 is a minor population amongst additional PAX8-positive follicular cell subsets. We observed thyroid dysgenesis in T21 age-matched specimens, and T21 thyrocytes showed transcriptional signatures of cytoskeletal disorganisation and altered interactions with the extracellular matrix, as well as compensatory activation of metabolic stress gene programs and upregulation of thyroid biosynthetic genes. In line with the altered proportions of fTFC2 in healthy foetal and adult thyroid, papillary thyroid cancer in children is transcriptionally enriched for the fTFC2 signature compared to that in adults. All together, these findings reveal thyrocyte heterogeneity across the lifespan and provide insights into thyroid development in health and disease, informing potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c9750769f035ff50001aa051cc8485f708617a8" target='_blank'>
              A developmental cell atlas of the human thyroid gland
              </a>
            </td>
          <td>
            Hassan Massalha, Mi K. Trinh, Erick Armingol, Liz Tuck, Alexander V. Predeus, Pavel Mazin, Carmen Sancho-Serra, Agnes Oszlanczi, Yvette Wood, Conor Parks, Toochi Ogbonnah, Holly J. Whitfield, Iva Kelava, Sam Behjati, R. Vento-Tormo, Nadia Schoenmakers
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="One critical step in the metastatic cascade is the survival of circulating tumour cells (CTCs) within the bloodstream. While numerous interactions between CTCs and various hematopoietic cells have been described, the role of red blood cells (RBCs) in this process remains underexplored. This study investigates the interactions between tumour cells and RBCs from breast and lung cancer patients, revealing significant phenotypic and functional changes in the tumour cells, unlike when the contact is with RBCs from healthy donors. In vitro co-culture of cancer cell lines with RBCs from metastatic cancer patients resulted in increased tumour cell attachment accompanied by morphological changes. Additionally, RBCs-primed tumour cells showed increased adhesion and disruption of the endothelial barrier in vitro and increased invasiveness both in vitro and in vivo. Transcriptomic analysis showed that RBCs from metastatic breast cancer patients induce significant gene expression changes, notably upregulating PAK4, which enhances migration and epithelial-mesenchymal transition. PAK4 inhibition reduced these effects. Proteomic studies revealed substantial remodelling, including actin-related changes and the accumulation of VASP at cell edges, promoting directional migration. Clinically, higher RBC distribution width (RDW) in metastatic breast cancer patients is associated with increased CTC counts and worse outcome. This study highlights the previously unrecognized role of RBCs in promoting metastatic behaviours in cancer cells and suggests potential therapeutic targets, such as PAK4, to counteract these effects. Further exploration of RBCs-tumour cell interactions could provide new insights into metastatic mechanisms and improve cancer prognosis and treatment strategies. Key Points This study reveals the previously unknown role of RBCs in enhancing tumour cell invasiveness and metastatic potential. Tumour cells undergo significant phenotypic and functional changes after contact with RBCs from cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a5495cc85336152bb18294f77e87fc3e41607e1" target='_blank'>
              Red blood cell-tumour cell interactions promote tumour cell progression
              </a>
            </td>
          <td>
            Thais Pereira-Veiga, Celso Yáñez-Gómez, Aleksi Pekkarinen, C. Abuín, Christine Blechner, M. González-Conde, Alexander T. Bauer, S. Vidal-y-Sy, Ayham Moustafa, B. Siebels, A. B. Dávila-Ibáñez, P. Cabezas-Sáinz, Maider Santos, Laura Sánchez, Joao Gorgulho, Julian Götze, Kira Meißner, Juan Cueva, P. Palacios, Alexia Cortegoso, T. Curiel, Carmela Rodríguez, Marta Carmona, S. Peine, Milena Schmidt, N. Heuer-Olewinski, Martin Reck, M. Abdo, Rafael López-López, S. Windhorst, Klaus Pantel, H. Wikman, Clotilde Costa
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D'Amato, Yi Xue, Kristina Fontanez, Aaron A May-Zhang, Trinity Smithers, Yigal Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aging increases breast cancer risk while an early first pregnancy reduces a woman’s life-long risk. Several studies have explored the effect of either aging or pregnancy on mammary epithelial cells (MECs), but the combined effect of both remains unclear. Here, we interrogate the functional and transcriptomic changes at single cell resolution in the mammary gland of aged nulliparous and parous mice to discover that pregnancy normalizes age-related imbalances in lineage composition, while also inducing a differentiated cell state. Importantly, we uncover a minority population of Il33-expressing hybrid MECs with high cellular potency that accumulate in aged nulliparous mice but is significantly reduced in aged parous mice. Functionally, IL33 treatment of basal, but not luminal, epithelial cells from young mice phenocopies aged nulliparous MECs and promotes formation of organoids with Trp53 knockdown. Collectively, our study demonstrates that pregnancy blocks the age-associated loss of lineage integrity in the basal layer through a decrease in Il33+ hybrid MECs, potentially contributing to pregnancy-induced breast cancer protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3189ca70d80763a95a696e8eb95db69af76072ee" target='_blank'>
              Pregnancy Reduces Il33+ Hybrid Progenitor Accumulation in the Aged Mammary Gland
              </a>
            </td>
          <td>
            Andrew Olander, Cynthia M Ramirez, Veronica Haro Acosta, Paloma Medina, Sara Kaushik, Vanessa D. Jonsson, Shaheen S. Sikandar
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, P. Maini, Tyler Cassidy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Most patients with prostate adenocarcinoma develop resistance to therapies targeting the androgen receptor (AR). Consequently, a portion of these patients develop AR-independent neuroendocrine prostate cancer (NEPC), a rapidly progressing cancer with limited therapies and poor survival outcomes. Current research to understand the progression to NEPC suggests a model of lineage plasticity whereby AR-dependent luminal-like tumors progress towards an AR-independent NEPC state. Genetic analysis of human NEPC identified frequent loss of RB1 and TP53, and the loss of both genes in experimental models mediates the transition to a neuroendocrine lineage. Transcriptomics studies have shown that lineage transcription factors ASCL1 and NEUROD1 are present in NEPC. In this study, we modeled the progression of prostate adenocarcinoma to NEPC by establishing prostate organoids and subsequently generating subcutaneous allograft tumors from genetically-engineered mouse models harboring Cre-induced loss of Rb1 and Trp53 with Myc overexpression (RPM). These tumors were heterogeneous and displayed adenocarcinoma, squamous, and neuroendocrine features. ASCL1 and NEUROD1 were expressed within neuroendocrine-defined regions, with ASCL1 being predominant. Genetic loss of Ascl1 in this model did not decrease tumor incidence, growth, or metastasis; however, there was a notable decrease in neuroendocrine identity and an increase in basal-like identity. This study provides an in vivo model to study progression to NEPC and establishes the requirement for ASCL1 in driving neuroendocrine differentiation in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b06924a4ca0fcd63effebd03ee725e6bf5e8759" target='_blank'>
              Neuroendocrine Differentiation in Prostate Cancer Requires ASCL1.
              </a>
            </td>
          <td>
            Kathia E Rodarte, Shaked Nir Heyman, Lei Guo, Lydia Flores, T. Savage, Juan Villarreal, Su Deng, Lin Xu, Rajal B Shah, T. Oliver, Jane E Johnson
          </td>
          <td>2024-09-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Curcumin is widely recognized for its diverse antitumor properties, ranging from breast cancer to many other types of cancers. However, its role in the tumor microenvironment remains to be elucidated. In this study, we established a 3D tumor spheroids model that can simulate the growth environment of tumor cells and visualized the antitumor metabolic alteration caused by curcumin using mass spectrometry imaging technology. Our results showed that curcumin not only exerts a profound impact on the growth and proliferation of breast cancer cells but in situ multivariate statistical analysis also reveals the significant effect on the overall metabolic profile of tumor spheroids. Meanwhile, our visualization map characterized curcumin metabolic processes of reduction and glucuronidation in tumor spheroids. More importantly, abnormal metabolic pathways related to lipid metabolism and polyamine metabolism were also remodeled at the metabolite and gene levels after curcumin intervention. These insights deepen our comprehension of the regulatory mechanism of curcumin on the tumor metabolic network, furnishing powerful references for antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9708e466bdcc061f5dd608fcbfa98f940da2834e" target='_blank'>
              Spatial Metabolomics Profiling Reveals Curcumin Induces Metabolic Reprogramming in Three-Dimensional Tumor Spheroids
              </a>
            </td>
          <td>
            Zihan Zhu, Yaqi Zhang, Lei Wang, Haoyuan Geng, Min Li, Shiping Chen, Xiao Wang, Panpan Chen, Chenglong Sun, Chao Zhang
          </td>
          <td>2024-09-01</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, P. Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Gastrointestinal cancer cells display both morphology and physiology diversity, thus posing a significant challenge for precise representation by a single data model. We conducted an in-depth study of gastrointestinal cancer heterogeneity by integrating and analyzing data from multiple modalities. Methods: We used a modified Canny algorithm to identify edges from tumor images, capturing intricate nonlinear interactions between pixels. These edge features were then combined with differentially expressed mRNA, miRNA, and immune cell data. Before data integration, we used the K-medoids algorithm to pre-cluster individual data types. The results of pre-clustering were used to construct the kernel matrix. Finally, we applied spectral clustering to the fusion matrix to identify different tumor subtypes. Furthermore, we identified hub genes linked to these subtypes and their biological roles through the application of Weighted Gene Co-expression Network Analysis (WGCNA) and Gene Ontology (GO) enrichment analysis. Results: Our investigation categorized patients into three distinct tumor subtypes and pinpointed hub genes associated with each. Genes MAGI2-AS3, MALAT1, and SPARC were identified as having a differential impact on the metastatic and invasive capabilities of cancer cells. Conclusion: By harnessing multimodal features, our study enhances the understanding of gastrointestinal tumor heterogeneity and identifies biomarkers for personalized medicine and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c68f2f427231b32eddf4811a28d710e5bb1cf" target='_blank'>
              Tumor Heterogeneity in Gastrointestinal Cancer Based on Multimodal Data Analysis
              </a>
            </td>
          <td>
            Dongmei Ai, Yang Du, Hongyu Duan, Juan Qi, Yuduo Wang
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Despite extensive molecular stratification efforts, improvements in molecularly informed RT-based therapeutic strategies in the neoadjuvant setting have been limited. This study aims to elucidate the adaptive mechanisms and interactions within PDAC to identify new combinatorial therapeutic targets with RT. MethodsWe constructed a high-resolution molecular landscape of the cellular subtypes and spatial communities that constitute PDAC. Single-cell RNA sequencing (scRNA- seq) was performed on 16 PDAC biopsies of matched pre- and post-treatment samples from a Phase I/II dose-escalation clinical trial of pancreatic stereotactic body radiation therapy (SBRT). Additionally, Visium spatial transcriptomics (ST) was conducted on biopsies from primary PDAC patients collected at the time of surgery. These patients received either no neoadjuvant therapy (n = 8) or neoadjuvant chemoradiation (chemoRT) (n = 20). Cell type profiles were determined from scRNA-seq to compare changes in cell type proportions and signatures before and after RT. From these profiles, we employed robust cell type decomposition to analyze cell type signatures in our ST dataset. ResultsChemoRT induced several significant effects compared to controls within the tumor microenvironment, including an increase in the proportion of myofibroblasts, specifically senescent cancer-associated fibroblasts (CAFs), with signatures consistent with immune cell dysfunction. We observed an increase in CD8 and CD4 effector memory cells with RT; however, this occurred alongside a rise in immunosuppressive alternatively activated tumor-associated macrophages. Additionally, RT upregulated pathways related to HIF, cytokine production, B cell proliferation, and negative regulation of lymphocytes. Patients with increased infiltration of NK, memory B, and CD8 effector memory cells demonstrated improved overall survival outcomes. Inference of copy number variations enabled us to identify cancer subclones as either responsive or resistant in matched pre and post RT biopsies. Resistance was characterized by augmented hypoxia, KRAS activation, epithelial- mesenchymal transition, and interferon-gamma response signaling. Conversely, responders exhibited increased oxidative phosphorylation signaling and pathway enrichment associated with cell proliferation and cell cycle progression. Interestingly, RT induced a decrease in cancer cells characterized as harboring a chemoresistant basal-like molecular subtype. ConclusionThrough both longitudinal and therapy stratified tumor biopsies, we were able to observe significant selection pressures on stromal and cancer cells following RT. This includes upregulation of innate and adaptive immune pathways with a compensatory immunosuppressive response driven by myeloid cells and CAFs. This may provide opportunities for development of clinical trials to therapeutically target specific cellular phenotypes and their interactions.
 Citation Format: Vincent Bernard, Galia Jacobson, Kimal Rajapakshe, Jimin Min, Guangsheng Pei, Daniel Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, Manoop S Bhutani, Linghua Wang, Paola A Guerrero, Ethan B Ludmir, Albert C Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J Koay. A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee951a6470ee1fe16a77439ce9fbdb87aac3a94c" target='_blank'>
              Abstract B078: A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            V. Bernard, Galia Jacobson, K. Rajapakshe, Jimin Min, Guangsheng Pei, D. Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, M. Bhutani, Linghua Wang, Paola A Guerrero, E. Ludmir, Albert C. Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J. Koay
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M. Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The immune system is one of the first lines of defence against the emergence of cancer. When effector cells attempt to suppress the tumour, the cancer cells can respond in kind by evolving methods of escape or inhibition. Knowledge of this coevolutionary system and the selection taking place within it can help us understand tumour-immune dynamics both during tumorigenesis but also when treatments such as immunotherapies are applied. Here, we present an individual-based branching process model of mutation accumulation, where random mutations arising in cancer cells trigger corresponding specialised immune responses. Different from previous research, we explicitly model interactions between cancer and effector cells, while incorporating stochastic effects, which are especially important for the expansion and extinction of small populations. We find that the parameters governing interactions between the cancer and effector cells induce different outcomes of tumour progress, such as suppression and evasion. While it is hard to measure the cancer-immune dynamics directly in patients, genetic information of the cancer may indicate the presence of such interactions. Our model demonstrates signatures of selection in sequencing-derived summary statistics, such as the single-cell mutational burden. Thus, bulk and single-cell sequencing of a tumour may give information about the coevolutionary dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d348b418f4965c460fe85c654e3a6921ad317a0a" target='_blank'>
              Cancer-immune coevolution dictated by antigenic mutation accumulation
              </a>
            </td>
          <td>
            Long Wang, Christo Morison, Weini Huang
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinomas (RCC) are characterized by their largely diverse clinical outcomes. Leveraging genomic diversity among individual tumors, our research during the past decade has concentrated on developing personalized medicine approaches for the most common and the most aggressive type of RCC, clear cell renal carcinoma (ccRCC). We generated the largest dataset of ccRCC including somatic genomic and clinical annotations for over 940 ccRCC patients, and developed a genomic classifier, based on mutational status of 12 RCC-relevant genes, which is able to stratify patients according to their risk of relapse after nephrectomy and/or death due to RCC. Furthermore, we noticed the presence of mesenchymal-like cellular phenotypes in tumors of high-risk patients, representsing a de-differentiation process, known as sarcomatoic or rhabdoid (S/R) de-differentation. Methods The current knowledge about molecular mechanisms that may drive S/R de-differentiation is primarily generated from molecular profiling of bulk tumors, collecting data from a mixture of cells that co-exist in the tumor milieu. Therefore, high-resolution studies that precisely define molecular characteristics of different cellular phenotypes associated with S/R features are missing. We have used spatial transcriptomic profiling to investigate molecular mechanisms that underline S/R de-differentationin RCC. While preserving tissue context, we applied spatial whole human transcriptome profiling to areas exhibiting S/R or clear cell phenotypes within the same tumor specimens to generate phenotype-specific transcriptome profiles. In addition, we applied whole-exome sequencing (WES) to independent areas that exhibit different differentiation phenotype withing same tumors. Results Pathway and network analysis of genesets with upregulation in each area have revealed meaningful differences in cellular pathways which are active in each phenotype. Specifically, we observed that dysregulation of extracellular matrix (ECM) is a hallmark of S/R areas within RCC tumors. Furthermore, WES of phenotypically-distinct areas within each tumor has shed new light son the genome evoltuion of S/R phenotypes. Conclusions S/R dedifferentiation in RCC tumors is characterized by specific genomic evolutionary patterns and substantial dysregulation of ECM components. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad0d8a5a2c80baecbe7aa769327da96aacd5a05" target='_blank'>
              9 Delving into molecular underpinnings of sarcomatoid/rhabdoid dedifferentioation in renal cancers using spatial profiling
              </a>
            </td>
          <td>
            Y. Riazalhosseini
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6640869137efbf815e49406335626c0344fac69" target='_blank'>
              Age-associated clonal B cells drive B cell lymphoma in mice.
              </a>
            </td>
          <td>
            José P Castro, A. Shindyapina, Alessandro Barbieri, Kejun Ying, Olga S. Strelkova, Joao A. Paulo, A. Tyshkovskiy, Rico Meinl, C. Kerepesi, Anna P. Petrashen, Marco Mariotti, Margarita V Meer, Yan Hu, Alexander Karamyshev, Grigoriy Losyev, Mafalda Galhardo, Elsa Logarinho, Artur A. Indzhykulian, Steven P. Gygi, J. M. Sedivy, John P Manis, V. N. Gladyshev
          </td>
          <td>2024-08-08</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Motivation The combination of single-cell sequencing with ligand-receptor analysis paves the way for the characterization of cell communication events in complex tissues. In particular, directed weighted graphs stand out as a natural representation of cell-cell communication events. However, current computational methods cannot analyze sample-specific cell-cell communication events, as measured in single-cell data produced in large patient cohorts. Cohort-based cell-cell communication analysis presents many challenges, such as the non-linear nature of cell-cell communication and the high variability presented by the patient-specific single-cell RNAseq datasets. Results Here, we present scACCorDiON (single-cell Analysis of Cell-Cell Communication in Disease clusters using Optimal transport in Directed Networks), an optimal transport algorithm exploring node distances on the Markov Chain as the ground metric between directed weighted graphs. Additionally, we derive a k-barycenter algorithm using the Wasserstein-based distance, which is able to cluster directed weighted graphs. We compare our approach with competing methods in several large cohorts of scRNA-seq data. Our results show that scACCorDiON can predict clusters better, matching the disease status of samples. Moreover, we show that barycenters provide a robust and explainable representation of cell cell communication events related to the detected clusters. We also provide a case study of pancreas adenocarcinoma, where scACCorDion detects a sub-cluster of disease samples associated with changes in the tumor microenvironment. Availability The code of scACCorDiON is available at https://scaccordion.readthedocs.io/en/latest Contact ivan.costa@rwth-aachen.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12408229b87ae8ceac3d6b2906ea4bd12adc6c6e" target='_blank'>
              scACCorDiON: A clustering approach for explainable patient level cell cell communication graph analysis
              </a>
            </td>
          <td>
            J. Nagai, Michael T. Schaub, Ivan G.Costa
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The expression of PD-L1 on tumor cells (TCs) is used as an immunotherapy biomarker in lung cancer, but heterogeneous intratumoral expression is often observed. Using a Digital Spatial Profiling, we performed proteomic and whole-transcriptomic analyses of TCs and immune cells (ICs) in spatially matched areas based on tumor PD-L1 expression and the status of the immune microenvironment. Our findings were validated using immunohistochemistry, The Cancer Genome Atlas, and immunotherapy cohorts. ICs in areas with high PD-L1 expression on TCs showed more features indicative of immunosuppression and exhaustion than ICs in areas with low PD-L1 expression on TCs. TCs highly expressing PD-L1 within immune-inflamed (IF) areas show up-regulation of pro-inflammatory processes, whereas TCs highly expressing PD-L1 within immune-deficient (ID) areas show up-regulation of various metabolic processes. Using differentially expressed genes of TCs between the IF and ID areas, we identified a novel prognostic gene signature for lung cancer. In addition, a high ratio of CD8+ cells to M2 macrophages was found to predict favorable outcomes in patients with PD-L1-expressing lung cancer after immune checkpoint inhibitor therapy. This study demonstrates that TCs and ICs have distinct spatial features within the tumor microenvironment that are related to tumor PD-L1 expression and IC infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc0a9207938d739ad9fb6635845dbbb9be7ec54" target='_blank'>
              Spatially resolved whole-transcriptomic and proteomic profiling of lung cancer and its immune-microenvironment according to PD-L1 expression.
              </a>
            </td>
          <td>
            Yoon Kyung Jeon, Jaemoon Koh, Dongjoo Lee, Sehui Kim, Seung Geun Song, B. Han, Hyein Jeong, Young A Kim, B. Keam, Se-Hoon Lee, K. Kim, Doo Hyun Chung
          </td>
          <td>2024-09-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b983ff02239e55a5a088e11ea44463f2a191dc" target='_blank'>
              Spatial computational modelling illuminates the role of the tumour microenvironment for treating glioblastoma with immunotherapies
              </a>
            </td>
          <td>
            Blanche Mongeon, Julien Hébert-Doutreloux, Anudeep Surendran, Elham Karimi, B. Fiset, D. Quail, Logan A Walsh, A. Jenner, Morgan Craig
          </td>
          <td>2024-08-18</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Biomedical imaging and RNA sequencing with single-cell resolution improves our understanding of white blood cell diseases like leukemia. By combining morphological and transcriptomic data, we can gain insights into cellular functions and trajectoriess involved in blood cell differentiation. However, existing methodologies struggle with integrating morphological and transcriptomic data, leaving a significant research gap in comprehensively understanding the dynamics of cell differentiation. Here, we introduce an unsupervised method that explores and reconstructs these two modalities and uncovers the relationship between different subtypes of white blood cells from human peripheral blood smears in terms of morphology and their corresponding transcriptome. Our method is based on a beta-variational autoencoder ({\ss}-VAE) with a customized loss function, incorporating a R-CNN architecture to distinguish single-cell from background and to minimize any interference from artifacts. This implementation of {\ss}-VAE shows good reconstruction capability along with continuous latent embeddings, while maintaining clear differentiation between single-cell classes. Our novel approach is especially helpful to uncover the correlation of two latent features in complex biological processes such as formation of granules in the cell (granulopoiesis) with gene expression patterns. It thus provides a unique tool to improve the understanding of white blood cell maturation for biomedicine and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6118936e6de4528eea4409592a60efbf01c34e" target='_blank'>
              Multimodal Analysis of White Blood Cell Differentiation in Acute Myeloid Leukemia Patients using a β-Variational Autoencoder
              </a>
            </td>
          <td>
            Gizem Mert, Ario Sadafi, Raheleh Salehi, N. Navab, Carsten Marr
          </td>
          <td>2024-08-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Patient-derived organoids (PDOs) have emerged as a powerful tool for modeling human cancers due to their ability to closely mirror in vivo architecture and preserve the genetic landscape of tumors. They offer a crucial platform for studying disease progression and treatment responses. In this study, we present a comprehensive characterization of a unique collection of PDO cultures derived from pancreatic ductal adenocarcinoma (PDAC), a particularly aggressive malignancy often diagnosed in advanced stages with limited treatment options and poor prognosis. Our investigation explores morphological and genetic profiling of 10 PDAC PDOs, specifically identifying distinct drug-response patterns. The morphological characteristics of these PDO cultures were investigated by microscopy, while the mutational status was acquired by panel sequencing. Subsequently, PDOs were treated with a panel of 10 inhibitors, either individually or in combination. The response to these treatments was assessed through live-cell imaging (Incucyte®) to capture and quantify changes in organoid morphology and endpoint measurements of cellular viability using a CellTiter-Glo® assay. Our detailed in vitro drug sensitivity testing revealed a remarkable diversity in the responses of the PDAC PDOs to different chemotherapeutic agents, including those currently under clinical trials. Of notable significance is the depth of understanding we have achieved, allowing us to establish potential correlations between PDO morphology, genetic mutations, and drug reactions. This finding provides a key insight into the complex interplay of these factors and their impact on treatment outcomes, offering valuable insights into understanding individual tumor subtypes' sensitivity or resistance to treatments. The PDOs exhibited a spectrum of cancer-associated mutations, including KRAS, TP53, SMAD4, CDKN2A, and ARID1A, reflecting the heterogeneity observed in PDAC patients. Morphologically, the organoids displayed various growth patterns, ranging from cystic and regular structures to filled or unfilled, granular, dark, and light membrane structures. Drug treatments underscored the diverse and unique drug-response profiles within the PDOs, with some demonstrating sensitivity to multiple treatments. In contrast, others exhibited resistance, highlighting the intricate nature of drug responses within this model. Overall, our findings emphasize the ability of PDAC organoids to faithfully capture the complexity and heterogeneity of the disease, making them promising platforms for guiding personalized oncology approaches. The diverse drug response profiles observed in PDOs advocate leveraging this model to develop precision medicine strategies tailored to patients' characteristics, ultimately paving the way for more effective and personalized treatment regimens in the fight against PDAC.
 Citation Format: Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, Silke Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, Elisabeth Heßmann, Volker Ellenrieder, Philipp Ströbel, Michael Ghadimi. Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a23581700d7eba1d7151f1cd1e85577199cc66e" target='_blank'>
              Abstract A023: Patient-Derived Organoid Cultures for Personalized Therapies and Targeted Drug Screening Applications
              </a>
            </td>
          <td>
            Constanza Tapia Contreras, Günter Schneider, Denise Müller, Kimia Mirzakhani, Karly Conrads, S. Kaulfuß, Tim Beißbarth, Gabriela Salinas, Nils Beyer, Sophia Reinländer, Ulrix Sax, E. Hessmann, V. Ellenrieder, Philipp Ströbel, Michael Ghadimi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e0df7d22a45444a437c740c4db84a4c490aa49" target='_blank'>
              Membrane RRM2-positive cells represent a malignant population with cancer stem cell features in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yongzhi Zhao, Shuting Xue, Danduo Wei, Jianjuan Zhang, Nachuan Zhang, Liping Mao, Niya Liu, Lei Zhao, Jianing Yan, Yifan Wang, Xiujun Cai, Saiyong Zhu, Stephanie Roessler, Junfang Ji
          </td>
          <td>2024-09-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Non-Hispanic Black women (BW) are diagnosed with more aggressive breast cancer subtypes at younger ages and have higher mortality rates compared to non-Hispanic White women (WW). Within the breast microenvironment emergent data shows that stromal collagen plays a significant role in aggressiveness of breast cancer linked to ancestral traits. However, the molecular composition of collagen stroma has yet to be linked to ancestry and aggressiveness of breast cancer. The purpose of this study was to evaluate the microenvironment within ancestry-defined normal breast and triple-negative breast cancer (TNBC) to understand spatial regulation of collagen stroma that may link to ancestral differences in cancer. Multimodal, multiplexed spatial analyses were performed. Techniques included Second Harmonic Generation (SHG) to measure collagen fibers, mass spectrometry imaging to measure the collagen proteome, and mass-tagged antibody panels to investigate immune signatures in TNBC. Normal breast tissues were obtained from the Susan G. Komen Tissue Bank, African American (AA; n=20) and European American (EA; n=20) samples were annotated with risk scores, age, Body Mass Index (BMI), Breast Imaging-Reporting and Data System (BI-RADS), continental ancestry, and family history. Triple-negative breast cancer (TNBC, n=80) tissues from BW with extensive clinical annotations including age, BMI, stage, metastasis, and survival were evaluated in comparison to 20+ TNBC tissues from WW. In the normal breast cohort, evaluation of BMI (overweight and obese) within ancestry groups showed significant but opposing differences in fiber width measurements. As BMI increases, AA showed statistically thicker fibers while EA showed statistically thinner fibers. At the protein level, two types of collagens were significantly higher in EA and one type is higher in AA. Within the TNBC cohort, collagen fiber length and width measurements by SHG were significantly shorter and thinner in stage II compared to stage I but no other stage comparisons were significant. This may indicate that ECM remodeling occurs at earlier stages in TNBC. Initial analyses showed that collagen peptide intensities significantly vary by TNBC stage (75 peptides, ANOVA test FDR p-value < 0.05). In the cohort of women who survived, 90 ECM peptides significantly correlated with collagen fiber widths, indicating a relationship between collagen physical and molecular properties within a defined region. Seven immune markers defined differences in survival when grouped by high vs low expression within tumor regions. These markers include CD20 (Hazard Ratio HR = 7.99, p-value = 0.002) and PD-1 (HR = 3.68, p-value = 0.048), which have both been studied in breast cancer. Current work focuses on collagen peptide post-translational modifications that may differ in TNBC patients by vital status and are detected in normal breast at risk of cancer. These findings support that molecular composition in breast stroma contributes to aggressive ancestral traits of breast cancer risk and triple-negative breast cancer.
 Citation Format: Jaclyn B. Dunne, Heather Jensen-Smith, Laura Spruill, Taylor S. Hulahan, Jade K. Macdonald, George E. Sandusky, M. A. Hollingsworth, Anand S. Mehta, Richard R. Drake, Jeffrey R. Marks, Harikrishna Nakshatri, Marvella E. Ford, Peggi M. Angel. Multimodal and multiplexed spatial regulation of the breast collagen proteome: Comparative analysis in Black and White women from normal breast tissue and triple-negative breast cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d058388e9d844a6c66ca907f1790256ff73b1f8" target='_blank'>
              Abstract C003: Multimodal and multiplexed spatial regulation of the breast collagen proteome: Comparative analysis in Black and White women from normal breast tissue and triple-negative breast cancer
              </a>
            </td>
          <td>
            Jaclyn B. Dunne, Heather Jensen-Smith, Laura S Spruill, Taylor S. Hulahan, Jade K Macdonald, George E Sandusky, M. A. Hollingsworth, A. Mehta, Richard R. Drake, Jeffrey R. Marks, Hari Nakshatri, Marvella E. Ford, Peggi M. Angel
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d1e5e2f8fff698fa2e7aaaa422dd95deef960f" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis identifies a prognostic T-cell signature in colorectal cancer
              </a>
            </td>
          <td>
            Peng Cui, Haibo Wang, Zhigang Bai
          </td>
          <td>2024-08-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="145 Background: Endemic NPC is invariably linked to Epstein-Barr virus infection, and often presents as locoregionally-advanced (LA-) disease. Nonetheless, it is acknowledged that LA-NPC is clinically heterogeneous, with disparate clinical presentations of ascending (A) and descending (D) subtypes that are characterized by T3-4N0-1 and T1-2N2-3 disease, respectively. Here, we investigated the TIME between these distinct NPC phenotypes using whole transcriptome sequencing and digital pathology. Methods: We utilized a cohort of 111 patients with NPC from 2 institutions – NCCS (n=68) and JXCH (n=43) as test and validation datasets, respectively. Whole transcriptome profiling using RNAseq (Illumina, CA) was performed in the NCCS cohort, while spatial analyses of hematoxylin and eosin-stained whole slide images (WSIs) were undertaken in the NCCS cohort (35 A- and 33 D-subtypes), and validated in the JXCH cohort (26 A- and 17 D-subtypes). A customized pipeline was developed to analyze the spatial relationships between tumor-rich and immune-rich regions (tiles) within the WSIs. Mean distances between each tumor-rich and its nearest immune-rich tiles were computed for each WSI. Highest and lowest quartiles of the mean distances in the test cohort were used as thresholds to classify NPC tumors into 3 immune classes of differing immune cell density. Results: Transcriptomic profiling revealed an enrichment of immune-related genesets and differentially expressed genes in D- relative to A-subtype tumors. This corresponded to observations of higher overall immune, T and B cell scores in D- than A-subtypes by deconvolution analyses (ESTIMATE and CIBERSORT). Through spatial analyses of our WSIs, we defined three immune classes – immune-dense, -sparse, and -desert, which corresponded to mean distances of <88 μm, 88-208 μm, and >208 μm between tumour- to immune-rich tiles, respectively. D-subtype NPC had significantly more immune-rich regions compared with A-subtypes in both NCCS and JXCH cohorts ( P=0.02 [NCCS]; P=0.01 [JXCH]). This corresponded to shorter median distances between tumor-rich to immune-rich tiles in D- than A-subtypes (103 μm vs 200 μm, P=0.01 [NCCS]; 123 μm vs250 μm, P=0.01 [JXCH]). Based on our novel spatial-based classification, we observed a higher proportion of immune dense tumors in D- than A-subtypes in the NCCS cohort, with only 3/33 (9%) D-subtypes being classified as immune desert. Using the same distance thresholds, we derived a consistent trend in the JXCH cohort, where a higher proportion of D-subtypes were classified as immune dense and sparse than A-subtypes (59% vs 31%, P=0.01). Conclusions: Our results support a model where the TIME delineates the biology underpinning the disparate clinical presentations of distinct A- and D-subtypes of LA-NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24cc4cf674b8d9159fd67ca9e00f76cac965bd2a" target='_blank'>
              Effect of tumor-immune microenvironment (TIME) on disparate ascending (A) and descending (D) subtypes of nasopharyngeal carcinoma (NPC).
              </a>
            </td>
          <td>
            E. Yeo, E. Ong, Boon Hao Hong, D. W. Lim, Joe Yeong, Sze Huey Tan, Melvin L K Chua, Jacqueline Siok Gek Hwang, Tianzhu Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869c734206e668652b18e8e50d0afc69d2273b39" target='_blank'>
              Intratumoral microbiota of pancreatic ductal adenocarcinoma impact patient prognosis by influencing tumor microenvironment
              </a>
            </td>
          <td>
            Jingze Leng, Hengyi Xu, Xiaoyu Liu, Yufan Yang, Chun Ning, Lejia Sun, Jiangming Qu, Xindi Ke, Xun Lan
          </td>
          <td>2024-09-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Immunotherapy response is associated with the presence of conventional dendritic cells (cDCs). cDC type 1 (cDC1) is critically important for CD8+ T cell activation, cDC type 2 (cDC2) regulates CD4+ T cell responses, and mature regulatory cDCs may dampen T cell responses in the tumor microenvironment (TME). However, we lack a clear understanding of cDC distribution in the human TME, cDC prevalence in metastatic sites, and cDC differences in early- versus late-stage disease. Rapid autopsy specimens of 10 patients with lung adenocarcinoma were evaluated to detect cDCs and immune cells via multiplex immunofluorescence using 18 markers and 42 tumors. First, we found that T cells, cDC1, and cDC2 were confined to stroma, whereas mature regulatory DCs were enriched in tumor, suggesting unique localization-specific functions. Second, lung and lymph node tumors were more enriched in T cells and cDCs than liver tumors, underscoring differences in the TME of metastatic sites. Third, although the proportion of T cells and cDC1 did not differ in different stages, an increase in the proportion of cDC2 and macrophages in late stage suggests potential differences in regulation of T cell responses in different stages. Collectively, these findings provide new, to our knowledge, insights into cDC biology in human cancer that may have important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8512ba6cd3679e5a0f0f3d1f8e432ba86900fbeb" target='_blank'>
              Rapid Autopsy to Define Dendritic Cell Spatial Distribution and T Cell Association in Lung Adenocarcinoma.
              </a>
            </td>
          <td>
            Hilal Ozakinci, Xiaofei Song, Gina S Nazario, Thomas Lila, Benjamin J Chen, Tyler Simpson, Jonathan V Nguyen, Carlos M Moran Segura, Zachary Thompson, Ram Thapa, Trevor A Rose, E. Haura, B. Pellini, Xiaoqing Yu, B. Ruffell, Dung-tsa Chen, Theresa A. Boyle, Amer A Beg
          </td>
          <td>2024-08-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Background Sarcomatoid differentiation (SD) in renal cell carcinoma (RCC) is associated with poor survival and heightened response to immune checkpoint blockade. Detection of SD can be challenging due to spatial heterogeneity and sampling error. Herein, we introduce a novel tissue–informed epigenomic approach to noninvasively identify sarcomatoid differentiation in patients with RCC from cell-free DNA (cfDNA) using 1mL of plasma. Methods Chromatin immunoprecipitation and sequencing (ChIP-seq) for H3K27ac – a histone modification associated with active regulatory elements (REs) – was performed on pathologically reviewed clear cell RCC frozen tissue samples with and without SD (sarcomatoid-RCC and epithelioid-RCC, resp.) collected at the Dana-Farber Cancer Institute. Differentially marked REs between sarcomatoid and epithelioid subtypes were identified using DESeq2 (false discovery rate of q < 0.01). After establishing tissue signatures, ChIP-seq was then performed on cell-free chromatin (cfChIP-seq) in plasma from patients with sarc-RCC and epi-RCC. A Sarcomatoid Score was derived for each sample by aggregating the plasma H3K27ac signal at tissue-derived sarcomatoid-specific REs (sarc-REs), while normalizing to signal at epithelioid-specific REs (epi-REs). Scores were compared between the two groups using a Wilcoxon rank-sum test. A classifier was built to distinguish sarc-RCC from epi-RCC based on the Sarcomatoid Score and its performance was evaluated using the area under the receiver operating characteristic (AUROC) curve. Results We identified 25,919 differentially marked REs between 8 sarc-RCC and 8 epi-RCC tissue samples at a false discovery rate of q < 0.01. We selected 12,868 REs that are enriched in sarcomatoid vs. epithelioid. We generated cfChIP-seq profiles from plasma of 29 patients, 17 with sarc-RCC and 12 with epi-RCC. The Sarcomatoid Scores were significantly higher in sarc-RCC vs. epi-RCC plasma samples (p=1.6×10-4; Figure 1A). These scores achieved an AUROC curve of 0.9 for classifying patients with sarc-RCC from patients with epi-RCC (Figure 1B). Conclusions We present a proof-of-concept study in 1 cc of plasma for the detection of sarcomatoid differentiation in RCC based on the assessment of histone modification signals in cfDNA. This approach could help overcome the challenges of spatial heterogeneity and sampling error from tissue that make identification of sarc-RCC difficult. More generally, it establishes a paradigm for identifying histologic subtypes of cancer based on their epigenomic correlates from cfDNA, with possible therapeutic implications in real-time. Figure 1. (A) Sarcomatoid Score in plasma from patients with RCC at sarcomatoid-specific REs, comparing sarcomatoid (orange) and epithelioid RCC (blue). (B) ROC curves for distinguishing sarcomatoid from epithelioid RCC plasma samples using H3K27ac cell-free ChIP-seq signal at sarcomatoid-specific REs. ‘AUC’ indicates area under the ROC curve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03141a33629f618991474e63a226a8491b01de4" target='_blank'>
              44 Liquid biopsy epigenomic profiling for the detection of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Karl Semaan, T. Zarif, Marc Eid, J. Canniff, Noa Phillips, Bard Fortunato, Hunter Savignano, Matthew P Davidson, Razane El Hajj Chehade, S. Garinet, E. Saad, R. Saliby, M. Sun, J-H. Seo, Jacob E. Berchuck, David A Braun, M. Freedman, Toni K. Choueiri, S. Baca
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a8cbaf63e3462f27923725db4d9a627507061cb" target='_blank'>
              Radiogenomic profiling of global DNA methylation associated with molecular phenotypes and immune features in glioma
              </a>
            </td>
          <td>
            Z. Zhuang, Jinxin Lin, Zixiao Wan, Jingrong Weng, Ze Yuan, Yumo Xie, Zongchao Liu, Peiyi Xie, Siyue Mao, Zongming Wang, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2024-09-02</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Malignant rhabdoid tumors (MRT) are rare aggressive tumors of infancy characterized by the biallelic inactivation of SMARCB1. Their lineage of origin remains uncertain, but increasing evidence suggest a neural-crest origin. By analyzing the transcriptome of MRT occurring in peripheral and cranial nerves, we observe that their expression profiling was not distinct from any other tumor localization. We therefore investigated whether the development of Smarcb1-deficient tumors emerging from embryonic precursors of peripheral nerves in mice could recapitulate human MRT. we generated conditional Smarcb1 knockout in the Prss56-Cre mouse strain thereby targeting boundary cap cells which are neural crest derived Schwann cell precursor. All Smarcb1Flox/Flox;Prss56Cre+/+ showed ataxia within a median of two months, due to the development of massive intracranial tumor in the ponto-cerebellum angle. Smarcb1Flox/Flox;Prss56Cre+/- mainly developed paraplegia related to nerve root tumors, within a median delay of 7 months. Interestingly, transcriptome profiling confirmed the good correlation between mouse peripheral nerve tumors and human MRT; intracranial tumors in this new model correlated with MYC-ATRTs but with some specific features not observed in their human counterparts. All tumors harbored a significant immune infiltration, with some clonal T-cell expansion. Altogether, the Smarcb1Flox/Flox;Prss56-Cre mouse tumors offer a new model for human MRT and promising perspectives for immune-based innovative treatments.
 Citation Format: Zhi-Yan HAN, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, Arnault Tauziède-Espariat, Kevin Beccaria, Julien Masliah-Planchon, Christine Bourneix, Delphine Guillemot, Stéphanie Reynaud, Gaelle Pierron, Maëva Veyssiere, Owen Hoare, Céline Chauvin, Grégory Thomson, Didier Surdez, Pascale Varlet, Christelle Dufour, Volodia Dangouloff-Ros, Joshua J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, Franck Bourdeaut. Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624aa14765b3c481a1fc38ac19c8ad7677137eda" target='_blank'>
              Abstract A006 Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors
              </a>
            </td>
          <td>
            Zhi-Yan Han, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, A. Tauziède-Espariat, K. Beccaria, J. Masliah-Planchon, C. Bourneix, D. Guillemot, S. Reynaud, G. Pierron, Maëva Veyssiere, Owen Hoare, C. Chauvin, Grégory Thomson, D. Surdez, Pascale Varlet, Christelle Dufour, V. Dangouloff-Ros, J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim U. Rashid, Gary L. Johnson, J. J. Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing is a valuable technique for identifying diverse cell subtypes. A key challenge in this process is that the detection of rare cells is often missed by conventional methods due to low abundance and subtle features of these cells. To overcome this, we developed SCLCNF (Local Connectivity Network Feature Sharing in Single-Cell RNA sequencing), a novel approach that identifies rare cells by analyzing features uniquely expressed in these cells. SCLCNF creates a cellular connectivity network, considering how each cell relates to its neighbors. This network helps to pinpoint coexpression patterns unique to rare cells, utilizing a rarity score to confirm their presence. Our method performs better in detecting rare cells than existing techniques, offering enhanced robustness. It has proven to be effective in human gastrula data sets for accurately pinpointing rare cells, and in sepsis data sets where it uncovers previously unidentified rare cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41629b08b97b96e2a5087885fb3403f0009f9b9" target='_blank'>
              Connectivity Network Feature Sharing in Single-Cell RNA Sequencing Data Identifies Rare Cells
              </a>
            </td>
          <td>
            Shudong Wang, Hengxiao Li, Yahui Liu, Shanchen Pang, Sibo Qiao, Jionglong Su, Shaoqiang Wang, Yulin Zhang
          </td>
          <td>2024-08-03</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Preclinical models that replicate patient tumours as closely as possible are crucial for translational cancer research. While in vitro cancer models have many advantages in assessing tumour response therapy, in vivo systems are essential to enable evaluation of the role of the tumour cell extrinsic factors, such as the tumour microenvironment and host immune system. The requirement for a functional immune system is particularly important given the current focus on immunotherapies. Therefore, we set out to generate an immunocompetent, transplantable model of colorectal cancer suitable for in vivo assessment of immune-based therapeutic approaches. Intestinal tumours from a genetically engineered mouse model, driven by expression of a Pik3ca mutation and loss of Apc, were transplanted into wild type C57BL/6 host mice and subsequently passaged to form a novel syngeneic transplant model of colorectal cancer. Our work confirms the potential to develop a panel of mouse syngeneic grafts, akin to human PDX panels, from different genetically engineered, or carcinogen-induced, mouse models. Such panels would allow the in vivo testing of new pharmaceutical and immunotherapeutic treatment approaches across a range of tumours with a variety of genetic driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e96ac527246a723f0c584b8d2c966fbc703899" target='_blank'>
              Development of an in vivo syngeneic mouse transplant model of invasive intestinal adenocarcinoma driven by endogenous expression of Pik3caH1047R and Apc loss
              </a>
            </td>
          <td>
            Francesc de las Heras, Camilla B. Mitchell, William K. Murray, N. Clemons, Wayne A. Phillips
          </td>
          <td>2024-08-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2de571a905622de6c7a795e6f89203edad8fe14" target='_blank'>
              Tumor-derived GLI1 promotes remodeling of the immune tumor microenvironment in melanoma
              </a>
            </td>
          <td>
            Alessandro Giammona, Chiara De Vellis, Enrica Crivaro, Luisa Maresca, R. Amoriello, Federica Ricci, G. Anichini, Silvia Pietrobono, David R Pease, Martin E Fernandez-Zapico, Clara Ballerini, B. Stecca
          </td>
          <td>2024-08-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Background Osteosarcoma (OS) survival rates and outcome have not improved in 50 years since the advent of modern chemotherapeutics. Thus, there is a critical need for an improved understanding of the tumor microenvironment to identify better therapies. Extracellular matrix (ECM) deposition and hypoxia are known to abrogate the efficacy of various chemical and cell‐based therapeutics. Here, we aim to mechanistically investigate the combinatorial effects of hypoxia and matrix deposition with the use of OS spheroids. Methods We use two murine OS cell lines with differential metastatic potential to form spheroids. We form spheroids of two sizes, use ascorbate‐2‐phosphate supplementation to enhance ECM deposition, and study cell response under standard (21% O2) and physiologic (5% O2) oxygen tensions. Finally, we examine chemotherapeutic responses to doxorubicin treatment. Results ECM production and oxygen tension are key determinants of spheroid size through cell organization based on nutrient and oxygen distribution. Interestingly, highly metastatic OS is more susceptible to chemotherapeutics compared to less metastatic OS when matrix production increases. Together, these data suggest that dynamic interactions between ECM production and oxygen diffusion may result in distinct chemotherapeutic responses despite inherent tumor aggressiveness. Conclusion This work establishes OS spheroids as a valuable tool for early OS tumor formation investigation and holds potential for novel therapeutic target and prognostic indicator discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5e2fb20e2a4e399455c45af17440bdc5ed9071" target='_blank'>
              Extracellular matrix production and oxygen diffusion regulate chemotherapeutic response in osteosarcoma spheroids
              </a>
            </td>
          <td>
            Isabel S Sagheb, Thomas P Coonan, R. L. Randall, Katherine H Griffin, J. K. Leach
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) is characterized by several molecular alterations that contribute to its development and progression. These alterations include the epidermal growth factor receptor (EGFR), anaplastic lymphoma kinase (ALK), human epidermal growth factor receptor 2 (HER2), and mesenchymal–epithelial transition factor (c-MET). Among these, the hepatocyte growth factor (HGF)/c-MET signaling pathway plays a crucial role in NSCLC. In spite of this, the involvement of the HGF/c-MET signaling axis in remodeling the tumor microenvironment (TME) remains relatively unexplored. This review explores the biological functions of the HGF/c-MET signaling pathway in both normal and cancerous cells, examining its multifaceted roles in the NSCLC tumor microenvironment, including tumor cell proliferation, migration and invasion, angiogenesis, and immune evasion. Furthermore, we summarize the current progress and clinical applications of MET-targeted therapies in NSCLC and discuss future research directions, such as the development of novel MET inhibitors and the potential of combination immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7956b1a88c0d882a052d3a0639ac987956fb9aa3" target='_blank'>
              Unveiling the Role of HGF/c-Met Signaling in Non-Small Cell Lung Cancer Tumor Microenvironment
              </a>
            </td>
          <td>
            Shuxi Yao, Xinyue Liu, Ya-ling Feng, Yiming Li, Xiangtian Xiao, Yuelin Han, Shu Xia
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="25 Background: The brain tumor immune microenvironment (TiME) is characterised by suppression of tumor-infiltrating leukocytes. The lymphocyte niche activity is particularly downregulated within this TiME milieu. Methods: Here, using data-driven approaches in flow cytometry and RNA sequencing analysis, we aimed at dissecting leukocyte cell types infiltrating human brain tumor TiME. Results: Unsupervised clustering provided T lymphocyte subsets including double-negative (CD3+CD4- CD8-) T cells (DNT). Our DNT phenotype was validated with an independent dataset, suggesting DNT to be compatible with γδ T cell phenotype. In our cohort, myeloid immune cell frequencies were associated with malignancy type as previously reported, while a gliosarcoma notably presented unexpectedly high lymphocyte frequencies. Consistently, double-positive (CD4+ CD8+) T cells (DPT) had higher frequency in the gliosarcoma than other tumors. The flow cytometry results were supported by transcriptome patterns and the activity of immune-related genes in these tumors. Conclusions: Our findings feature the benefit of data-driven approaches for brain TiME lymphocyte analysis, and show that primary brain tumors can entail atypical lymphocyte subsets like DNTs and DPT cells. Unbiased lymphocyte characterisation can potentially outline novel targets for immune checkpoint blockade therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb1ae6998ef7cf2b065eaabbce0c558410319b8" target='_blank'>
              Unsupervised clustering of brain tumor leukocytes to reveal atypical lymphoid cell subsets.
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Demonstrating a combination of high prevalence, late detection, and low survival, Lung cancer is the leading cause of cancer related deaths worldwide (Sainz de Aja J, 2021) and Lung Adenocarcinoma (LUAD) is its most prevalent pathologic subtype (Zhou, 2021). Emerging from within the distal alveolar epithelium, LUAD presents with vast heterogeneity in histology, mutational spectra, and epigenomic dysregulation (Sainz de Aja J, 2021) (Zhou, 2021). Additionally, LUAD’s considerable intratumoral heterogeneity manifests vast genomic and phenotypic diversity across tumor cells and cellular components of the tumor microenvironment (TME) within the same tumor tissue. LUAD also reveals significant racial disparities in incidence, survival and mortality rates between White and Black men in the United States. The occurrence of LUAD is approximately 68.3 per capita for Black men and 61.5 per capita for White men, reflecting an 11% higher rate in the former. Further, survival outcomes indicate that Black men generally exhibit lower 5-year relative survival rates compared to White men, highlighting greater mortality risk and conspicuously poorer prognosis (Siegel RL et al., 2023). While addressing the social determinants of health is vital for the mitigation of health disparities, understanding the genetic and molecular foundations that underly racial health inequities is essential for developing comprehensive interventions and advancing more precise therapeutic modalities for more diverse patient populations. Single Nucleotide Polymorphisms (SNPs) can influence oncogenic transformation and may be associated with distinct molecular landscapes in tumors and adjacent TME. Furthermore, ancestry may play a pivotal role in shaping individual susceptibility and risk for mitochondrial dysfunction and ensuing oncogenic transformation. To elucidate the biological underpinnings of racial health disparities in LUAD, 10 Black and 10 White male LUAD patients’ tumor and normal corresponding adjacent tissues underwent molecular profiling for KRAS mutation status, Whole Exome Sequencing (WES) to determine ancestry, and analysis of associated LUAD induced Mitochondrial Somatic Nucleotide Variants (mtSNVs). We also employed the 10X Visium HD spatial transcriptomic platform to evaluate spatially resolved molecular and cellular differences in KRAS mediated LUAD between Black and White men. Hypothesis: Thus, we propose that global ancestry is associated with variations in intratumoral heterogeneity, TME composition, and mitochondrial dynamics, indicating a potential role in the observed disparities in susceptibility, malignancy and mortality in LUAD across diverse patient populations.
 Citation Format: Eun Kyu Sung, Yuxin Jin, Yonatan Amzaleg, John Carpten, William Dean Wallace, Dan Raz, Aaron M. Neely. Investigating the effects of ancestry in KRAS mediated lung adenocarcinoma: Exploring intratumoral heterogeneity and mitochondrial dynamics [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C169.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fd50ebcf66bd13578e33674dfa106c7a5fd8db" target='_blank'>
              Abstract C169: Investigating the effects of ancestry in KRAS mediated lung adenocarcinoma: Exploring intratumoral heterogeneity and mitochondrial dynamics
              </a>
            </td>
          <td>
            Eun Kyu Sung, Yuxin Jin, Yonatan Amzaleg, John D. Carpten, W. D. Wallace, Dan Raz, Aaron M. Neely
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, , W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc8bbed3fd4a0efe23da37bf78ebdb4d44abffa" target='_blank'>
              Focal adhesion kinase-mediated interaction between tumor and immune cells in the tumor microenvironment: implications for cancer-associated therapies and tumor progression.
              </a>
            </td>
          <td>
            Louis Boafo Kwantwi, T. Tandoh
          </td>
          <td>2024-09-13</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Introduction Cancer-associated fibroblasts (CAFs) are key players in the fibrotic tumour microenvironment (TME) of pancreatic cancer. Recent evidence has demonstrated the existence of multiple CAF subtypes, however there is relatively little evidence validating and corroborating these CAF subtypes in resected human pancreatic cancer. Furthermore, the spatial interactions between CAF subtypes and the epithelial and immune compartments of the pancreatic cancer TME is not well understood. Methods We designed and optimised a bespoke 8-plex immunofluorescence panel for the purposes of CAF subtype profiling and elucidation of CAF spatial biology: panCK, FAP, SMA, CTGF, S100A4, IL6, LIF, PDGFR. This was applied to a cohort of n=215 resected pancreatic cancer, well characterised with highly granular clinico-pathological and transcriptomic data through the Australian Pancreatic Cancer Genome Initiative. Spatial analysis was performed with novel computational biology techniques. Results We demonstrate seven CAF subtypes which cluster to form distinct, highly prognostic “enrichment profiles” (EPs). The cold EP represents quiescent CAFs in close proximity to immune cells (multivariate HR 0.45, 95% CI 0.30-0.68, p<0.001), and the fibrotic EP represents activated CAFs in close proximity to epithelial cells (multivariate HR 2.22, 95% CI 1.48-3.35, p<0.001). We also demonstrate the prognostic power of high spatial entropy (loss of tissue architecture, correlating with the fibrotic EP) vs low spatial entropy (preservation of tissue architecture, correlating with the cold EP) (HR 2.22, 95% CI 1.47-3.35, p<0.001). Conclusion We begin to unravel the elusive spatial biology of pancreatic CAFs in a large resected human cohort. We see changes in immune cell infiltrate, modulation of cellular relationships, and greater tissue disorganisation as a result of CAF activation. These features are highly prognostic and we hypothesise they are driven by changes in crosstalk between TME compartments. Current work focuses on application of these techniques to a large resected neoadjuvantly-treated cohort, allowing further interrogation of the notorious fibrotic post-neoadjuvant pancreatic cancer TME. If accepted for presentation we look forward to presenting both cohorts simultaneously.
 Citation Format: Adam Bryce, Stephan Dreyer, Fieke Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang. Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179192d42336b982da176e9aa5fe870344303cba" target='_blank'>
              Abstract A053: Interrogating the spatial biology of pancreatic cancer-associated fibroblasts in resected human pancreatic cancer
              </a>
            </td>
          <td>
            A. Bryce, S. Dreyer, F. Froeling, Silvia Martinelli, Rachel Pennie, Leah Officer-Jones, John Le Quesne, David Chang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: One of the most significant unmet clinical challenges in paediatric oncology is the development of novel therapeutic strategies for recurrent medulloblastoma (R-MB). MYC-driven MBs are defined as classically cold tumours with a low incidence of infiltrating immune cells, resulting in a therapeutic challenge. The immune landscape of recurrent medulloblastoma is yet to explored. Understanding the complex interplay between the immune system and tumour biology is pivotal for comprehending disease progression and devising tailored therapeutic strategies that consider the distinct molecular and immunological profiles of both primary and recurrent MYC-driven tumours. The identification of cell-cell communications, particularly ligand–receptor pairs, allows for the inference of significant intercellular communications based on the expression of corresponding genes. Consequently, we hypothesised that the most influential cell-cell interactions within the tumour immune microenvironment (TME) of MYC-driven MB could unveil prevalent immune-suppressive interactions and potential vulnerabilities for therapeutic exploration. Methods/Results: Paired primary-recurrent bulk RNA-sequencing data, confirmed myeloid cells as the most infiltrating immune cell type in group3-MB and group4-MB. Comprehensive spatial phenotypic and cell-cell communication analyses corroborated this discovery, validating an increased incidence of macrophages in the matched-recurrent tumours and phenotypic markers of advanced immunosuppression. Subsequently, we used innovative algorithms for 10X MB single-cell data to predict interactions between tumour-cell ligands and immune-cell receptors within the TME; macrophages emerged as the core immune-cells involved in interactions throughout the TMEs, with the most significant ligand-receptor interaction and inflammatory response between MIF and CD74.In-depth immunohistochemistry analyses of primary and recurrent group3 and group4 tumours, and exhaustive tissue microarrays demonstrated expression of both CD74 and MIF, with limited expression of CD74 within the brain. To investigate the therapeutic potential of CD74, we developed recurrent, immune competent MYC-driven medulloblastoma mouse models. Comprehensive deconvolution analysis confirmed the TME integrity of our models to mirror that of the human disease. Locoregional delivery and repeat dosing of a bioactive-CD74 peptide demonstrated significant tumour reduction in our immune-competent mouse models, demonstrating the impact of our prediction algorithm and significant therapeutic potential of targeting the CD74-MIF axis in MYC-driven primary and recurrent MB. Conclusions: Essential cellular interactions and targetable therapeutic vulnerabilities have been identified in the tumour-microenvironment of MYC-driven primary-recurrent MB.
 Citation Format: Ben Draper, Dean Thompson, Alaide Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan. Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B063.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d2ace9063c026bc5c1042bf3f0c9b788241cdc" target='_blank'>
              Abstract B063: Unveiling the immune landscape of recurrent, MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Benjamin Draper, Dean Thompson, A. Morcavallo, Bethany Remeniuk, John Anderson, Louis Chesler, Steve Clifford, Frank Huang, Laura K. Donovan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="RECK is a candidate tumor suppressor gene isolated as a gene that induces flat reversion in a cell line transformed by the KRAS oncogene. Since RECK knockout mice die in utero, they are not suitable for studying the effects of RECK on tumor formation. In this study, we found an increased incidence of spontaneous pulmonary adenomas in mice with reduced RECK expression (RECK-Hypo mice). To evaluate the effects of RECK expressed by either tumor cells or host cells on tumor growth, we established a tumorigenic cell line (MKER) from the kidney of a C57BL/6 mouse and performed syngeneic transplantation experiments. Our results indicate that when RECK expression is low in host cells, transplanted MKER cells grow faster and kill the animal more rapidly. Since RECK is required for the formation of proper fibrillin fibers that serve as a tissue reservoir for precursors of TGFβ-family cytokines, we assessed the levels of TGFβ1 in the peripheral blood. We found a significant increase in TGFβ1 in RECK-Hypo mice compared to wild-type mice. We also found that the proportion of FOXP3-positive regulatory T (Treg) cells among splenocytes was higher in RECK-Hypo mice compared to the control mice. Furthermore, the number of FOXP3-positive cells in spontaneous hematopoietic neoplasms in the lungs as well as tumors that formed after MKER transplantation was significantly higher in RECK-Hypo mice compared to the control mice. These findings indicate that RECK-mediated tumor suppression involves a non-cell-autonomous mechanism and that possible roles of TGFβ1 and Treg cells in such a mechanism warrant further study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2faede40c559e712f40528b041e79dd6dbd0c800" target='_blank'>
              Non-cell-autonomous suppression of tumor growth by RECK in immunocompetent mice.
              </a>
            </td>
          <td>
            T. Matsuzaki, Joe Inoue, Nagahiro Minato, Makoto Noda
          </td>
          <td>2024-08-05</td>
          <td>Journal of cellular physiology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba00ba75f2e9a1970e31667560ba965cf2ada42" target='_blank'>
              Metabolic reprogramming and immune evasion: the interplay in the tumor microenvironment
              </a>
            </td>
          <td>
            Haixia Zhang, Shizhen Li, Dan Wang, Siyang Liu, Tengfei Xiao, Wan-Yi Gu, Hongmin Yang, Hui Wang, Minghua Yang, Pan Chen
          </td>
          <td>2024-09-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, , S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e0460376460ba45ef4a6dae47bbf3b686cb2da" target='_blank'>
              Organismal metabolism regulates the expansion of oncogenic PIK3CA mutant clones in normal esophagus.
              </a>
            </td>
          <td>
            A. Herms, B. Colom, G. Piedrafita, Argyro Kalogeropoulou, Ujjwal Banerjee, C. King, E. Abby, K. Murai, Irene Caseda, D. Fernandez-Antoran, S. Ong, M. Hall, Christopher Bryant, R. Sood, J. Fowler, Albert Pol, C. Frezza, B. Vanhaesebroeck, P. Jones
          </td>
          <td>2024-08-21</td>
          <td>Nature genetics</td>
          <td>3</td>
          <td>24</td>
        </tr>

        <tr id="Deciphering cell identity genes is pivotal to understanding cell differentiation, development, and many diseases involving cell identity dysregulation. Here, we introduce SCIG, a machine-learning method to uncover cell identity genes in single cells. In alignment with recent reports that cell identity genes are regulated with unique epigenetic signatures, we found cell identity genes exhibit distinctive genetic sequence signatures, e.g., unique enrichment patterns of cis-regulatory elements. Using these genetic sequence signatures, along with gene expression information from single-cell RNA-seq data, enables SCIG to uncover the identity genes of a cell without a need for comparison to other cells. Cell identity gene score defined by SCIG surpassed expression value in network analysis to uncover master transcription factors regulating cell identity. Applying SCIG to the human endothelial cell atlas revealed that the tissue microenvironment is a critical supplement to master transcription factors for cell identity refinement. SCIG is publicly available at https://github.com/kaifuchenlab/SCIG, offering a valuable tool for advancing cell differentiation, development, and regenerative medicine research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4837e8fb2e900c0dddf8f66b8bf3476f75c9e1cf" target='_blank'>
              Machine Learning Uncovers Vascular Endothelial Cell Identity Genes by Expression Regulation Features in Single Cells
              </a>
            </td>
          <td>
            K. Arulsamy, Bo Xia, Hong Chen, Lili Zhang, Kaifu Chen
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0de2bb42c9d38bb212c4141ac747130ed1d8b87" target='_blank'>
              MYH6 suppresses tumor progression by downregulating KIT expression in human prostate cancer
              </a>
            </td>
          <td>
            Fei Wang, Hua Shen, Kai Li, Yanhong Ding, Jianqing Wang, Jian Sun
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 19],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>