<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-16 06:31:50 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Secondary lymphoid organs (SLOs), including tonsils (TS), lymph nodes (LN), and Peyer's Patches, exhibit complementary immune functions. However, little is known about the spatial organization of immune cells and extracellular matrix (ECM) in the SLOs. Traditional imaging is limited to a few markers, confining our understanding of the differences between the SLOs. Herein, imaging mass cytometry addressed this gap by simultaneously profiling 25-plex proteins in SLO tissues at subcellular resolution. The antibody panel targeted immune, stromal, chemokine, epigenetic, and functional markers. For robust cell identification, a computational workflow SpatialVizPheno was developed to spatially phenotype 999,970 cells using two approaches, including manual gating and semi-supervised gating, iterative clustering, and annotation. LN exhibited the highest density of B cells while the intestinal tissues contained the highest proportion of regulatory and follicular helper T cells. SpatialVizPheno identified the most prevalent interaction between follicular dendritic cells and stromal cells (SCs), plasmablasts/plasma cells, and the SCs across the lymphoid tissues. Collagen-enriched regions were associated with the spatial orientation of B cell follicles in both TS and LN tissues, but not in intestinal lymphoid tissues. Such spatial differences of immunophenotypes and ECM in different SLO tissues can be used to quantify the relationship between cellular organization and ultimate immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d9342b523b0b50e2fb3715a6d362425d2a168c" target='_blank'>
              Spatial immunophenotyping using multiplexed imaging of immune follicles in secondary lymphoid tissues
              </a>
            </td>
          <td>
            Mayar Allam, Thomas Hu, Zhou Fang, Michelle Pi, Ankur Singh, Ahmet F. Coskun
          </td>
          <td>2024-07-22</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, S. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            , Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, , Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Motivation Analyzing the local microenvironment of tumor cells can provide significant insights into their complex interactions with their cellular surroundings, including immune cells. By quantifying the prevalence and distances of certain immune cells in the vicinity of tumor cells through a neighborhood analysis, patterns may emerge that indicate specific associations between cell populations. Such analyses can reveal important aspects of tumor-immune dynamics, which may inform therapeutic strategies. This method enables an in-depth exploration of spatial interactions among different cell types, which is crucial for research in oncology, immunology, and developmental biology. Results We introduce an R Markdown script called SNAQTM (Single-cell Spatial Neighborhood Analysis and Quantification), which conducts a neighborhood analysis on immunofluorescent images without the need for extensive coding knowledge. As a demonstration, SNAQTM was used to analyze images of pancreatic ductal adenocarcinoma. Samples stained for DAPI, PanCK, CD68, and PD-L1 were segmented and classified using QuPath. The resulting CSV files were exported into RStudio for further analysis and visualization using SNAQTM. Visualizations include plots revealing the cellular composition of neighborhoods around multiple cell types within a customizable radius. Additionally, the analysis includes measuring the distances between cells of certain types relative to others across multiple regions of interest. Availability and implementation The R Markdown files that comprise the SNAQTM algorithm and the input data from this paper are freely available on the web at https://github.com/AryehSilver1/SNAQ. Visual Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/536ae41dfee8df569ec9d770e09c6f0416cb7a5b" target='_blank'>
              Get to know your neighbors with a SNAQ™: A framework for single cell spatial neighborhood analysis in immunohistochemical images
              </a>
            </td>
          <td>
            Aryeh Silver, Avirup Chakraborty, Avinash Pittu, Diana Feier, Miruna Anica, Illeana West, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite promising results with immune checkpoint blockade (ICB) therapy, outcomes of patients with brain metastasis (BrM) remain poor. Identifying determinant of resistance is crucial but has been hindered by limited access to patient samples and the scarcity of relevant preclinical models. Here, we developed two mouse melanoma BrM models that mirror the disparate response to ICB seen in patients. By characterizing their mutational landscape and performing single-cell phenotypic and transcriptomic analysis, we demonstrated that these models recapitulate both the cellular and molecular features of human disease. Through our comparative analysis, we identified key factors contributing to ICB response. In the responsive model, we found that tumor cells expressed inflammatory programs that polarized microglia toward reactive states, expressing high levels of MHC-I and -II and immunostimulatory molecules, eliciting robust T cell recruitment and activation, which favored anti-tumor immunity. Conversely, the resistant melanoma cells were characterized by neurological molecular signatures and expression of ligands that maintain microglia homeostatic states, resulting in poor T cell infiltration and therapy resistance. We validated our findings in BrM patients, showing that BR1 and BR3 signatures correlated with T cell infiltration and were associated with better and worse patient outcomes, respectively. Our study fills a critical gap by providing clinically relevant models and revealing mechanistic insights into BrM ICB responses, identifying potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b06309c3c54287af3d2a6158bdb011f7654e629" target='_blank'>
              Brain-Metastatic Melanoma Models Uncover Immune and Molecular Correlates of Response to Immunotherapy
              </a>
            </td>
          <td>
            Amélie Daugherty-Lopès, Eva Pérez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabe Needle, Isabella Church, George Kyriakopoulos, Yongmei Zhao, Charli Gruen, Antonella Sassano, Romina E. Araya, Andres Thorkelsson, Cari Smith, M. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, R. Goldszmid
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, Andreas Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cell states are modulated by intrinsic driving forces such as gene expression noise and extrinsic signals from the tissue microenvironment. The distinction between intrinsic and extrinsic cell state determinants is essential for understanding the regulation of cell fate in tissues during development, homeostasis and disease. The rapidly growing availability of single-cell resolution spatial transcriptomics makes it possible to meet this challenge. However, available computational methods to infer topological tissue domains, spatially variable gene expression, or ligand-receptor interactions are limited in capturing cell state changes driven by crosstalk between individual cell types within the same niche. We present NiCo, a computational framework for integrating single-cell resolution spatial transcriptomics with matched single-cell RNA-sequencing reference data to infer the influence of the spatial niche on the cell state. By applying NiCo to mouse embryogenesis, adult small intestine and liver data, we demonstrate the capacity to predict novel niche interactions that govern cell state variation underlying tissue development and homeostasis. In particular, NiCo predicts a feedback mechanism between Kupffer cells and neighboring stellate cells limiting stellate cell activation in the normal liver. NiCo provides a powerful tool to elucidate tissue architecture and to identify drivers of cellular states in local niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330dd175c406437712c3044cd06a601fe89865f" target='_blank'>
              NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis
              </a>
            </td>
          <td>
            Ankit Agrawal, Stefan Thomann, Sukanya Basu, D. Grün
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb667e3e1fbda176a1eb9bab50b39b93702041c" target='_blank'>
              STdGCN: spatial transcriptomic cell-type deconvolution using graph convolutional networks
              </a>
            </td>
          <td>
            Yawei Li, Yuan Luo
          </td>
          <td>2024-08-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Characterizing the transcriptional and translational gene expression patterns at the single-cell level within their three-dimensional (3D) tissue context is essential for revealing how genes shape tissue structure and function in health and disease. However, most existing spatial profiling techniques are limited to 5-20 µm thin tissue sections. Here, we developed Deep-STARmap and Deep-RIBOmap, which enable 3D in situ quantification of thousands of gene transcripts and their corresponding translation activities, respectively, within 200-µm thick tissue blocks. This is achieved through scalable probe synthesis, hydrogel embedding with efficient probe anchoring, and robust cDNA crosslinking. We first utilized Deep-STARmap in combination with multicolor fluorescent protein imaging for simultaneous molecular cell typing and 3D neuron morphology tracing in the mouse brain. We also demonstrate that 3D spatial profiling facilitates comprehensive and quantitative analysis of tumor-immune interactions in human skin cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f410940b56b1945c8670449d337f9efc5f10187d" target='_blank'>
              Scalable spatial single-cell transcriptomics and translatomics in 3D thick tissue blocks
              </a>
            </td>
          <td>
            Xin Sui, Jennifer A. Lo, Shuchen Luo, Yichun He, Zefang Tang, Zuwan Lin, Yiming Zhou, Wendy Xueyi Wang, Jia Liu, Xiao Wang
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor heterogeneity refers to the diversity observed among tumor cells: both between different tumors (inter-tumor heterogeneity) and within a single tumor (intra-tumor heterogeneity). These cells can display distinct morphological and phenotypic characteristics, including variations in cellular morphology, metastatic potential and variability treatment responses among patients. Therefore, a comprehensive understanding of such heterogeneity is necessary for deciphering tumor-specific mechanisms that may be diagnostically and therapeutically valuable. Innovative and multidisciplinary approaches are needed to understand this complex feature. In this context, proteogenomics has been emerging as a significant resource for integrating omics fields such as genomics and proteomics. By combining data obtained from both Next-Generation Sequencing (NGS) technologies and mass spectrometry (MS) analyses, proteogenomics aims to provide a comprehensive view of tumor heterogeneity. This approach reveals molecular alterations and phenotypic features related to tumor subtypes, potentially identifying therapeutic biomarkers. Many achievements have been made; however, despite continuous advances in proteogenomics-based methodologies, several challenges remain: in particular the limitations in sensitivity and specificity and the lack of optimal study models. This review highlights the impact of proteogenomics on characterizing tumor phenotypes, focusing on the critical challenges and current limitations of its use in different clinical and preclinical models for tumor phenotypic characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/958a7a3c1fd552ec9c7dc2019625a3d4361653d4" target='_blank'>
              Phenotyping Tumor Heterogeneity through Proteogenomics: Study Models and Challenges
              </a>
            </td>
          <td>
            Diletta Piana, Federica Iavarone, E. De Paolis, Gennaro Daniele, Federico Parisella, Angelo Minucci, V. Greco, Andrea Urbani
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMN) is one of the known precancerous lesions. Patients’ prognosis is aggravated as IPMN transforms into invasive Pancreatic Ductal Adenocarcinoma (PDAC). The molecular mechanisms underlying this progression lack effective experimental models and urgently need to be elaborated. We performed spatial transcriptomics (ST) on fresh tissue samples from the same patient including normal pancreas, high-grade IPMN, and invasive PDAC, and described the step-by-step development of transcriptional landscape including clone evolution and adjacent TME feature variation. Our findings identified the master transcript factors and critical signaling pathways promoting IPMN progression to invasive PDAC. Additionally, both IPMN and PDAC harbored the ELF3, MYC, and KLF4 amplification. The Spatial CNV profile demonstrated significant heterogeneity among PDAC in their spatial distribution compared to IPMN, with seven distinct subclones showing diverse functions, such as hypoxia, oxidative phosphorylation, and epithelial-mesenchymal Transition. We observed a marked shift in the immune landscape, with the depletion of CD4+ T-cells and dendritic cells and an increase in immune-suppressive M2 macrophages in invasive PDAC, indicating a transition to an immune-evasive microenvironment. Additionally, cancer-associated fibroblasts (CAFs), particularly myofibroblastic CAFs, were enriched adjacent to invasive PDAC, suggesting their active role in tumor progression. By leveraging spatial transcriptomic analysis, our study provides comprehensive insights into the intricate molecular landscape that underlies the progression of IPMNs to invasive PDAC. These findings not only enhance our understanding of this complex process but also offer valuable knowledge for early diagnosis and intervention. Highlights Spatial CNV analysis reveals clonal evolution and distinct subclones in PDAC. Key drivers like ELF3, MYC, and KLF4 are amplified in both IPMN and invasive PDAC. Immune landscape shifts from pro-inflammatory in IPMN to immune-evasive in PDAC. Enrichment of myofibroblastic CAFs suggests their role in tumor progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4caecc6532101fd5cd79dde472c73a99ffa9c7b0" target='_blank'>
              Evolution of the Spatial transcriptomic landscape during the progression of high-grade pancreatic intraductal papillary mucinous neoplasms to invasive cancer
              </a>
            </td>
          <td>
            Sirui Peng, Qiangxing Chen, Zixin Chen, Mengling Yao, Yu Cai, Du He, Yu Cai, K. Cheng, Jun Li, H. Cai, P. Gao, Xiafei Gu, Xin Wang, Yongbin Li, Man Zhang, L. Meng, Qi Xia, Panpan Xu, Jin Zhou, Zhong Wu, Bing Peng
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, Tae Kyung Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Spatiotemporal patterns of gene expression underlie embryogenesis. Despite progress in single-cell genomics, mapping these patterns across whole embryos with comprehensive gene coverage and at high resolution has remained elusive. Here, we introduce a whole-embryo imaging platform using multiplexed error-robust fluorescent in-situ hybridization (weMERFISH). We quantified the expression of 495 genes in whole-mount zebrafish embryos at subcellular resolution. Integration with single-cell multiomics data generated an atlas detailing the expression of 25,872 genes and the accessibility of 294,954 chromatin regions, explorable with an online interface MERFISHEYES (beta version). We found that temporal gene expression aligns with cellular maturation and morphogenetic movements, diverse expression patterns correspond to composites of tissue-specific accessible elements, and changes in gene expression generate sharp boundaries during gastrulation. These results establish a novel approach for whole-organism spatial transcriptomics, provide a comprehensive spatially resolved atlas of gene expression and chromatin accessibility, and reveal the diversity, precision and emergence of embryonic patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055645942fdf0308bdff4ae89217d6f4fdd3c819" target='_blank'>
              Whole-embryo Spatial Transcriptomics at Subcellular Resolution from Gastrulation to Organogenesis
              </a>
            </td>
          <td>
            Yinan Wan, Jakob El Kholtei, Ignatius Jenie, Mariona Colomer-Rosell, Jialin Liu, J. N. Acedo, Lucia Y. Du, Mireia Codina-Tobias, Mengfan Wang, Ahilya N. Sawh, Edward Lin, Tzy-Harn Chuang, S. Mango, Guoqiang Yu, Bogdan Bintu, Alexander F. Schier
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The study presents a novel approach for quantifying cellular interactions in digital pathology using deep learning-based image cytometry. Traditional methods struggle with the diversity and heterogeneity of cells within tissues. To address this, we introduce the Spatial Interaction Potential (SIP) and the Co-Localization Index (CLI), leveraging deep learning classification probabilities. SIP assesses the potential for cell-to-cell interactions, similar to an electric field, while CLI incorporates distances between cells, accounting for dynamic cell movements. Our approach enhances traditional methods, providing a more sophisticated analysis of cellular interactions. We validate SIP and CLI through simulations and apply them to colorectal cancer specimens, demonstrating strong correlations with actual biological data. This innovative method offers significant improvements in understanding cellular interactions and has potential applications in various fields of digital pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949ba712b098fead7bc4bf83be6ad4882df65fb2" target='_blank'>
              Novel Methods for Analyzing Cellular Interactions in Deep Learning-Based Image Cytometry: Spatial Interaction Potential and Co-Localization Index
              </a>
            </td>
          <td>
            Toru Nagasaka, Kimihiro Yamashita, Mitsugu Fujita
          </td>
          <td>2024-08-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Image-based spatial transcriptomics (ST) offers spatial gene expression profile at the single-cell resolution and provides information to understand intercellular communication that is critical for maintaining tissue development and organ function. Disruption of normal cell-cell interactions (CCI) can lead to disease onset and progression. Current CCI analysis methods face several limitations, including subjection to the number of measured ligand-receptor genes in image-based spatial transcriptomics, limited graph encoding power, inadequate use of spatial information, and low interpretability. Here, we present GITIII, an interpretable self-supervised graph transformer- based language model that treats cells as words (nodes) and their cell neighborhood as a sentence to explore the communications among cells. Enhanced by multilayer perceptron-based distance scaler, physics-informed attention mechanism, and a state-of-the-art, expressive, and lightweight graph transformer model, GITIII infers CCI by investigating how the state of a cell is influenced by the spatial organization, ligand expression, cell types and states of neighboring cells. With its interpretable architecture, GITIII can be used to understand how the sender cell influences target genes in the receiver cell, visualize the spatial pattern and utility of CCI, identify significant CCI networks, perform CCI-informed cell subtyping, and compare CCI strength between disease groups. Applications to four ST datasets from several species, organs, and platforms, GITIII effectively identified and quantitatively interpreted key CCI patterns driving within-sample heterogeneity and disease progression, thus improving our understanding of brain structures, tumor microenvironments, and the interplay among different cell types responding to neighboring CCIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b394f3669761e67f7021ddd2a2bfa57c19a4a4e3" target='_blank'>
              Investigation of pair-wise single-cell interactions by statistically interpreting spatial cell state correlation learned by self-supervised graph inductive bias transformer
              </a>
            </td>
          <td>
            Xiao Xiao
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Heterogeneity of tumor metabolism is an important, but still poorly understood aspect of tumor biology. Present work is focused on the visualization and quantification of cellular metabolic heterogeneity of colorectal cancer using fluorescence lifetime imaging (FLIM) of redox cofactor NAD(P)H. FLIM-microscopy of NAD(P)H was performed in vitro in four cancer cell lines (HT29, HCT116, CaCo2 and CT26), in vivo in the four types of colorectal tumors in mice and ex vivo in patients’ tumor samples. The dispersion and bimodality of the decay parameters were evaluated to quantify the intercellular metabolic heterogeneity. Our results demonstrate that patients’ colorectal tumors have significantly higher heterogeneity of energy metabolism compared with cultured cells and tumor xenografts, which was displayed as a wider and frequently bimodal distribution of a contribution of a free (glycolytic) fraction of NAD(P)H within a sample. Among patients’ tumors, the dispersion was larger in the high-grade and early stage ones, without, however, any association with bimodality. These results indicate that cell-level metabolic heterogeneity assessed from NAD(P)H FLIM has a potential to become a clinical prognostic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d39beee686da3cb9d1fa5f224388f8f6c20fccf" target='_blank'>
              Insights into metabolic heterogeneity of colorectal cancer gained from fluorescence lifetime imaging
              </a>
            </td>
          <td>
            A. D. Komarova, S. D. Sinyushkina, Ilia D Shchechkin, I. Druzhkova, Sofia A Smirnova, Vitaliy M Terekhov, A. Mozherov, Nadezhda I. Ignatova, Elena E. Nikonova, E. Shirshin, L. Shimolina, Sergey V Gamayunov, V. Shcheslavskiy, M. Shirmanova
          </td>
          <td>2024-08-28</td>
          <td>eLife</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor heterogeneity impedes the development of biomarker assays suitable for early disease detection that would improve patient outcomes. The CA19-9 glycan is currently used as a standalone biomarker for PDAC. Furthermore, previous studies have shown that cancer cells may display aberrant membrane-associated glycans. We therefore hypothesized that PDAC cancer cell subpopulations could be distinguished by aberrant glycan signatures. We used multiplexed glycan immunofluorescence combined with pathologist annotation and automated image processing to distinguish between PDAC cancer cell subpopulations within tumor tissue. Using a training-set/test-set approach, we found that PDAC cancer cells may be identified by signatures comprising 4 aberrant glycans (VVL, CA19-9, sTRA, and GM2) and that there are three glycan-defined PDAC tumor types: sTRA type, CA19-9 type, and intermixed. To determine whether the aberrant glycan signatures could be detected in blood samples, we developed hybrid glycan sandwich assays for membrane-associated glycans. In both patient-matched tumor and blood samples, the proportion of aberrant glycans detected was consistent. Furthermore, our multiplexed glycan immunofluorescent approach proved to be more sensitive and more specific than CA19-9 alone. Our results provide proof of concept for a novel methodology to improve early PDAC detection and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6b226460c28415c919ed9fc15c5b9945dc42aa" target='_blank'>
              Multiplexed Glycan Immunofluorescence Identification of Pancreatic Cancer Cell Subpopulations in Both Tumor and Blood Samples
              </a>
            </td>
          <td>
            Braelyn Binkowski, Zachary L. Klamer, ChongFeng Gao, Ben Staal, Anna Repesh, Hoang-Le Tran, David M. Brass, Pamela Bartlett, Steven Gallinger, Maria Blomqvist, J. B. Morrow, Peter Allen, Chanjuan Shi, A. Singhi, Randall Brand, Ying Huang, G. Hostetter, Brian Haab
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Trogocytosis is an underappreciated phenomenon that shapes the immune microenvironment surrounding many types of solid tumors. The consequences of membrane-bound proteins being deposited from a donor immune cell to a recipient cancer cell via trogocytosis are still unclear. Here, we report that human clear cell renal carcinoma tumors stably express the lymphoid markers CD45, CD56, CD14, and CD16. Flow cytometry performed on fresh kidney tumors revealed consistent CD45 expression on tumor cells, as well as varying levels of the other markers mentioned previously. These results were consistent with our immunofluorescent analysis, which also revealed colocalization of lymphoid markers with carbonic anhydrase 9 (CAIX), a standard kidney tumor marker. RNA analysis showed a significant upregulation of genes typically associated with immune cells in tumor cells following trogocytosis. Finally, we show evidence of chromosomal DNA being transferred from immune cells to tumor cells during trogocytosis. This horizontal gene transfer has transcriptional consequences in the recipient tumor cell, resulting in a fusion phenotype that expressed both immune and cancer specific proteins. This work demonstrates a novel mechanism by which tumor cell protein expression is altered through the acquisition of surface membrane fragments and genomic DNA from infiltrating lymphocytes. These results alter the way in which we understand tumor-immune cell interactions and may reveal new insights into the mechanisms by which tumors develop. Additionally, further studies into trogocytosis will help push the field towards the next generation of immunotherapies and biomarkers for treating renal cell carcinoma and other types of cancers. SIGNIFICANCE STATEMENT We have identified trogocytosis as a mechanism by which human clear cell renal carcinoma tumors acquire lymphocyte surface protein expression from tumor infiltrating immune cells. In addition to the transfer of membrane fragments, we have provided evidence to show that genomic DNA is transferred from a normal immune cell to a tumor cell during trogocytosis. This process alters the transcriptome of cancer cells such that they express significantly more mRNA for immune proteins such as the lymphocyte marker CD45 compared to tumor cells that have not undergone trogocytosis. This study provides an in-depth analysis of the interactions between cancer cells and tumor infiltrating lymphocytes, and how these interactions alter the development of human tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d8b30a74776c1d1ac2d3f7a393276d5557d985" target='_blank'>
              Renal cancer cells acquire immune surface protein through trogocytosis and horizontal gene transfer
              </a>
            </td>
          <td>
            Haley Q. Marcarian, Anutr Sivakoses, Anika M. Arias, Olivia C. Ihedioha, Benjamin R. Lee, Maria C. Bishop, A. L. Bothwell
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are a heterogeneous population of cells whose phenotypes and functions are shaped by factors that are incompletely understood. Herein, we asked when and where TAMs arise from blood monocytes and how they evolve during tumor development. We initiated pancreatic ductal adenocarcinoma (PDAC) in inducible monocyte fate-mapping mice and combined single-cell transcriptomics and high-dimensional flow cytometry to profile the monocyte-to-TAM transition. We revealed that monocytes differentiate first into a transient intermediate population of TAMs that generates two longer-lived lineages of terminally differentiated TAMs with distinct gene expression profiles, phenotypes, and intratumoral localization. Transcriptome datasets and tumor samples from patients with PDAC evidenced parallel TAM populations in humans and their prognostic associations. These insights will support the design of new therapeutic strategies targeting TAMs in PDAC. Intermediate tumor-associated macrophages (TAMs) emerge first and give rise to two more differentiated TAM populations in pancreatic cancer. Editor’s summary Tumor-associated macrophages (TAMs) promote tumor growth and immune suppression, but how TAMs develop from circulating monocytes and local tissue-resident macrophages remains incompletely understood. Using a model of pancreatic ductal adenocarcinoma (PDAC) in monocyte fate-mapping mice, Dunsmore et al. examined the dynamics of the monocyte-to-macrophage transition in tumors. A population of “intermediate” TAMs gave rise to two more differentiated TAM subsets, distinguishable by their distinct surface marker phenotype, dependence on the transcription factor Maf, migratory behavior, and localization in tumors. These findings provide a time- and space-resolved picture of TAM heterogeneity and development in PDAC, which could guide the future development of TAM-targeted therapeutic strategies. —Claire Olingy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a54c86dbe269d24ca8abdd22c4597319660f70" target='_blank'>
              Timing and location dictate monocyte fate and their transition to tumor-associated macrophages
              </a>
            </td>
          <td>
            G. Dunsmore, Wei Guo, Ziyi Li, D. Bejarano, Rhea Pai, Katharine Yang, Immanuel Kwok, Leonard Tan, M. Ng, Carlos De La Calle Fabregat, A. Yatim, A. Bougouin, Kevin Mulder, Jake Thomas, Javiera Villar, Mathilde Bied, Benoit Kloeckner, C. Dutertre, Grégoire Gessain, S. Chakarov, Zhaoyuan Liu, J. Scoazec, A. Lennon-Duménil, Thomas Marichal, C. Sautès-Fridman, W. Fridman, Ankur Sharma, Bing Su, A. Schlitzer, Lai Guan Ng, Camille Blériot, Florent Ginhoux
          </td>
          <td>2024-07-26</td>
          <td>Science Immunology</td>
          <td>1</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1affced3ef3c1684e9cf82520aac1e64770c30b6" target='_blank'>
              aKNNO: single-cell and spatial transcriptomics clustering with an optimized adaptive k-nearest neighbor graph
              </a>
            </td>
          <td>
            Jia Li, Y. Shyr, Qi Liu
          </td>
          <td>2024-08-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest–derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="We present Cell2Cell, a novel visual analytics approach for quantifying and visualizing networks of cell-cell interactions in three-dimensional (3D) multi-channel cancerous tissue data. By analyzing cellular interactions, biomedical experts can gain a more accurate understanding of the intricate relationships between cancer and immune cells. Recent methods have focused on inferring interaction based on the proximity of cells in low-resolution 2D multi-channel imaging data. By contrast, we analyze cell interactions by quantifying the presence and levels of specific proteins within a tissue sample (protein expressions) extracted from high-resolution 3D multi-channel volume data. Such analyses have a strong exploratory nature and require a tight integration of domain experts in the analysis loop to leverage their deep knowledge. We propose two complementary semi-automated approaches to cope with the increasing size and complexity of the data interactively: On the one hand, we interpret cell-to-cell interactions as edges in a cell graph and analyze the image signal (protein expressions) along those edges, using spatial as well as abstract visualizations. Complementary, we propose a cell-centered approach, enabling scientists to visually analyze polarized distributions of proteins in three dimensions, which also captures neighboring cells with biochemical and cell biological consequences. We evaluate our application in three case studies, where biologists and medical experts use Cell2Cell to investigate tumor micro-environments to identify and quantify T-cell activation in human tissue data. We confirmed that our tool can fully solve the use cases and enables a streamlined and detailed analysis of cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d89e79ae5d1df3c773e5ff1e1e269cd6c8d836f" target='_blank'>
              Cell2Cell: Explorative Cell Interaction Analysis in Multi-Volumetric Tissue Data.
              </a>
            </td>
          <td>
            Eric Morth, Kevin Sidak, Z. Maliga, Torsten Moller, Nils Gehlenborg, P. Sorger, Hanspeter Pfister, Johanna Beyer, Robert Kruger
          </td>
          <td>2024-09-10</td>
          <td>IEEE transactions on visualization and computer graphics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c649f4e5bac69e4b666a1bcdc0297d323411c9" target='_blank'>
              Re-analysis of single cell and spatial transcriptomics data reveals B cell landscape in gastric cancer microenvironment and its potential crosstalk with tumor cells for clinical prognosis
              </a>
            </td>
          <td>
            Xing Cai, Jinru Yang, Yusheng Guo, Yanchao Yu, Chuansheng Zheng, Xiaofang Dai
          </td>
          <td>2024-08-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Differential expression is a key application of imaging spatial transcriptomics, moving analysis beyond cell type localization to examining cell state responses to microenvironments. However, spatial data poses new challenges to differential expression: segmentation errors cause bias in fold-change estimates, and correlation among neighboring cells leads standard models to inflate statistical significance. We find that ignoring these issues can result in considerable false discoveries that greatly outnumber true findings. We present a suite of solutions to these fundamental challenges, and implement them in the R package smiDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fa7ba237194cebaf697e3482ebe6176d4e2160" target='_blank'>
              Differential Expression Analysis for Spatially Correlated Data
              </a>
            </td>
          <td>
            Ana Gabriela Vasconcelos, Daniel McGuire, Noah Simon, Patrick Danaher, A. Shojaie
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I. Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Uveal melanoma is the most common non-cutaneous melanoma and is an intraocular malignancy affecting nearly 7,000 individuals per year worldwide. Of these, approximately 50% will progress to metastatic disease for which there are currently no effective curative therapies. Despite advances in molecular profiling and metastatic stratification of uveal melanoma tumors, little is known regarding their underlying biology of metastasis. Our group has identified a disseminated neoplastic cell population characterized by co-expression of immune and melanoma proteins, circulating hybrid cells (hybrids), in patients with uveal melanoma. Compared to circulating tumor cells, which lack expression of immune proteins, hybrids are detected at an increased prevalence in peripheral blood and can be used as a non-invasive biomarker to predict metastatic progression. Methods To ascertain mechanisms underlying enhanced hybrid cell dissemination we identified hybrid cells within primary uveal melanoma tumors using single cell RNA sequencing ( n  = 8) and evaluated their gene expression and predicted ligand-receptor interactions in relation to other melanoma and immune cells within the primary tumor. We then verified expression of upregulated hybrid pathways within patient-matched tumor and peripheral blood hybrids ( n  = 4) using cyclic immunofluorescence and quantified their protein expression relative to other non-hybrid tumor and disseminated tumor cells. Results Among the top upregulated genes and pathways in hybrid cells were those involved in enhanced cell motility and cytoskeletal rearrangement, immune evasion, and altered cellular metabolism. In patient-matched tumor and peripheral blood, we verified gene expression by examining concordant protein expression for each pathway category: TMSB10 (cell motility), CD74 (immune evasion) and GPX1 (metabolism). Both TMSB10 and GPX1 were expressed on significantly higher numbers of disseminated hybrid cells compared to circulating tumor cells, and CD74 and GPX1 were expressed on more disseminated hybrids than tumor-resident hybrids. Lastly, we identified that hybrid cells express ligand-receptor signaling pathways implicated in promoting metastasis including GAS6-AXL, CXCL12-CXCR4, LGALS9-P4HB and IGF1-IGFR1. Conclusion These findings highlight the importance of TMSB10, GPX1 and CD74 for successful hybrid cell dissemination and survival in circulation. Our results contribute to the understanding of uveal melanoma tumor progression and interactions between tumor cells and immune cells in the tumor microenvironment that may promote metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20782b15e64557dc3e5252547de53160b7d85219" target='_blank'>
              Detection of neoplastic-immune hybrid cells with metastatic properties in uveal melanoma
              </a>
            </td>
          <td>
            Ashley N. Anderson, P. Conley, Christopher Klocke, Sidharth K. Sengupta, Amara Pang, Hannah C. Farley, Abigail R. Gillingham, Aubrey D. Dawson, Yichen Fan, Jocelyn A. Jones, Summer L. Gibbs, Alison H. Skalet, Guanming Wu, Melissa H. Wong
          </td>
          <td>2023-12-06</td>
          <td>Research Square</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Methods that predict fate potential or degree of differentiation from transcriptomic data have identified rare progenitor populations and uncovered developmental regulatory mechanisms. However, some state-of-the-art methods are too computationally burdensome for emerging large-scale data and all methods make inaccurate predictions in certain biological systems. We developed a method in R (stemFinder) that predicts single cell differentiation time based on heterogeneity in cell cycle gene expression. Our method is computationally tractable and is as good as or superior to competitors. As part of our benchmarking, we implemented four different performance metrics to assist potential users in selecting the tool that is most apt for their application. Finally, we explore the relationship between differentiation time and cell fate potential by analyzing a lineage tracing dataset with clonally labelled hematopoietic cells, revealing that metrics of differentiation time are correlated with the number of downstream lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fe7aece967bf7efe6ff7ba7d9a3f9b4b16c676a" target='_blank'>
              Cell cycle expression heterogeneity predicts degree of differentiation
              </a>
            </td>
          <td>
            K. Noller, Patrick Cahan
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gamma delta (γδ) T cells play a crucial role in anti-tumor immunity due to their cytotoxic properties. However, the role and extent of γδ T cells in production of pro-tumorigenic interleukin-17 (IL-17) within the tumor microenvironment (TME) of colorectal cancer (CRC) remains controversial. In this study, we re-analyzed nine published human CRC whole-tissue single-cell RNA sequencing (scRNA-seq) datasets, identifying 18,483 γδ T cells out of 951,785 total cells, in the neoplastic or adjacent normal tissue of 165 human CRC patients. Our results confirm that tumor-infiltrating γδ T cells exhibit high cytotoxicity-related transcription in both tumor and adjacent normal tissues, but critically, none of the γδ T cell clusters showed IL-17 production potential. We also identified various γδ T cell subsets, including Teff, TRM, Tpex, and Tex, and noted an increased expression of cytotoxic molecules in tumor-infiltrating γδ T cells compared to their normal area counterparts. Our work demonstrates that γδ T cells in CRC primarily function as cytotoxic effector cells rather than IL-17 producers, mitigating the concerns about their potential pro-tumorigenic roles in CRC, highlighting the importance of accurately characterizing these cells for cancer immunotherapy research and the unneglectable cross-species discrepancy between the mouse and human immune system in the study of cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d463d31ff3fa586402ec6f97a8a59e1f66cab1" target='_blank'>
              Systematic Analysis of Human Colorectal Cancer scRNA-seq Revealed Limited Pro-tumoral IL-17 Production Potential in Gamma Delta T Cells
              </a>
            </td>
          <td>
            Ran Ran, Martin Trapecar, Douglas K. Brubaker
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Microvascular invasion (MVI) stands as a pivotal pathological hallmark of hepatocellular carcinoma (HCC), closely linked to unfavorable prognosis, early recurrence, and metastatic progression. However, the precise mechanistic underpinnings governing its onset and advancement remain elusive. Methods In this research, we downloaded bulk RNA-seq data from the TCGA and HCCDB repositories, single-cell RNA-seq data from the GEO database, and spatial transcriptomics data from the CNCB database. Leveraging the Scissor algorithm, we delineated prognosis-related cell subpopulations and discerned a distinct MVI-related malignant cell subtype. A comprehensive exploration of these malignant cell subpopulations was undertaken through pseudotime analysis and cell-cell communication scrutiny. Furthermore, we engineered a prognostic model grounded in MVI-related genes, employing 101 algorithm combinations integrated by 10 machine-learning algorithms on the TCGA training set. Rigorous evaluation ensued on internal testing sets and external validation sets, employing C-index, calibration curves, and decision curve analysis (DCA). Results Pseudotime analysis indicated that malignant cells, showing a positive correlation with MVI, were primarily concentrated in the early to middle stages of differentiation, correlating with an unfavorable prognosis. Importantly, these cells showed significant enrichment in the MYC pathway and were involved in extensive interactions with diverse cell types via the MIF signaling pathway. The association of malignant cells with the MVI phenotype was corroborated through validation in spatial transcriptomics data. The prognostic model we devised demonstrated exceptional sensitivity and specificity, surpassing the performance of most previously published models. Calibration curves and DCA underscored the clinical utility of this model. Conclusions Through integrated multi-transcriptomics analysis, we delineated MVI-related malignant cells and elucidated their biological functions. This study provided novel insights for managing HCC, with the constructed prognostic model offering valuable support for clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f58d531e5dbaa5fd2fdfd9279935fd61dc3adf" target='_blank'>
              Multi-transcriptomics analysis of microvascular invasion-related malignant cells and development of a machine learning-based prognostic model in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Haoran Huang, Feifeng Wu, Yang Yu, Bo Xu, De-Hua Chen, Yuwei Huo, Shaoqiang Li
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mutations inactivating TET2 are among the most common drivers of clonal hematopoiesis (CH). While TET2 inactivation is associated with monocyte-derived inflammation and improved chimeric antigen-receptor-T cell function, its impact on immunotherapy response is unknown. In our mouse model, hematopoietic Tet2 mutation enhanced immune checkpoint blockade (ICB) response. Enhanced ICB response with Tet2 mutation required phagocytes, CD4 and CD8 T cells. Mechanistically, in Tet2-mutant tumor-infiltrating leukocytes (TILs), ICB preferentially induced anti-tumor states and restricted cell states linked to tumor progression. Tet2-mutant monocytes activated costimulatory programs, while Tet2-mutant T cells showed enhanced T cell memory signatures, lesser exhaustion and decreased regulatory phenotype. Our murine data was clinically relevant, since we found that melanomas from patients with TET2 driver mutation-CH (TET2-CH) showed enhanced immune infiltration, T cell activation, and T cell memory programs. In melanoma patients treated with ICB, TET2-CH was associated with 6-fold greater odds of clinical benefit. Collectively, our data establishes that hematopoietic Tet2 inactivation primes leukocytes for anti-tumor states associated with immunotherapy response and provides a potential biomarker for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3e6160e702c2224ea8ff0222f9be539e313873" target='_blank'>
              Hematopoietic Tet2 inactivation enhances the response to checkpoint blockade immunotherapy
              </a>
            </td>
          <td>
            R. Vanner, Suraj Bansal, Marco M. Buttigeig, Andy G. X. Zeng, Vincent Rondeau, Darryl Y. Chan, Michelle A. Chan-Seng-Yue, Liqing Jin, Jessica Mcleod, E. Donato, Patrick Stelmach, C. Vlasschaert, Yitong Yang, Aarushi Gupta, Sofia Genta, Enrique Sanz Garcia, LI Shlush, Mauricio Ribeiro, Marcus O Butler, Sagi Abelson, Mark Minden, Steven M. Chan, Michael J. Rauh, A. Trumpp, John E. Dick
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract The human adrenal gland is a complex endocrine tissue. Studies on adrenal renewal have been limited to animal models or human foetuses. Enhancing our understanding of adult human adrenal homeostasis is crucial for gaining insights into the pathogenesis of adrenal diseases, such as adrenocortical tumours. Here, we present a comprehensive cellular genomics analysis of the adult human normal adrenal gland, combining single‐nuclei RNA sequencing and spatial transcriptome data to reconstruct adrenal gland homeostasis. As expected, we identified primary cells of the various zones of the adrenal cortex and medulla, but we also uncovered additional cell types. They constitute the adrenal microenvironment, including immune cells, mostly composed of a large population of M2 macrophages, and new cell populations, including different subpopulations of vascular‐endothelial cells and cortical‐neuroendocrine cells. Utilizing spatial transcriptome and pseudotime trajectory analysis, we support evidence of the centripetal dynamics of adrenocortical cell maintenance and the essential role played by Wnt/β‐catenin, sonic hedgehog, and fibroblast growth factor pathways in the adult adrenocortical homeostasis. Furthermore, we compared single‐nuclei transcriptional profiles obtained from six healthy adrenal glands and twelve adrenocortical adenomas. This analysis unveiled a notable heterogeneity in cell populations within the adenoma samples. In addition, we identified six distinct adenoma‐specific clusters, each with varying distributions based on steroid profiles and tumour mutational status. Overall, our results provide novel insights into adrenal homeostasis and molecular mechanisms potentially underlying early adrenocortical tumorigenesis and/or autonomous steroid secretion. Our cell atlas represents a powerful resource to investigate other adrenal‐related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e76e6effd4abc2a2c7085002aaf49ccc7313fb5" target='_blank'>
              Single‐nucleus and spatial transcriptome reveal adrenal homeostasis in normal and tumoural adrenal glands
              </a>
            </td>
          <td>
            B. Altieri, A. K. Secener, Somesh Sai, Cornelius Fischer, S. Sbiera, Panagiota Arampatzi, S. Kircher, Sabine Herterich, Laura-Sophie Landwehr, Sarah N. Vitcetz, C. Braeuning, M. Fassnacht, C. Ronchi, Sascha Sauer, Ali Kerim
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86603b6c8f27f9a5a7d6a05553f0d41b941be5e5" target='_blank'>
              Spatial landscapes of cancers: insights and opportunities.
              </a>
            </td>
          <td>
            Julia Chen, L. Larsson, Alexander Swarbrick, J. Lundeberg
          </td>
          <td>2024-07-23</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="The identification of gene regulatory networks (GRN) governing distinct cell fates in multilineage cellular differentiation systems is of critical importance for understanding cell fate decision. Single-cell RNA-sequencing (scRNA-seq) provides a powerful tool for the quantification of gene-level co-variation across the cell state manifold. However, accurate GRN reconstruction is hampered by the sparsity of scRNA-seq data introducing substantial technical noise. Moreover, the high dimensionality of typical scRNA-seq datasets limits the scalability of available approaches. To overcome these challenges, and to facilitate the inference of lineage-specific GRNs with directed regulator-target relations, we introduce NetID. This approach optimizes coverage of the cell state manifold by homogenous metacells and avoids spurious gene-gene correlations observed with available imputation methods. Benchmarking demonstrates superior performance of NetID compared to imputation-based GRN inference. By incorporating cell fate probability information, NetID facilitates prediction of lineage-specific GRNs and recovers known network motifs centered around lineage-determining transcription factors governing bone marrow hematopoiesis, making it a powerful toolkit for deciphering the gene regulatory control of cellular differentiation from large-scale single-cell transcriptome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/725296ffdf0a7cacc6e78577d9871b12ce8048c7" target='_blank'>
              Scalable identification of lineage-specific gene regulatory networks from metacells with NetID
              </a>
            </td>
          <td>
            Weixu Wang, Yichen Wang, Ruiqi Lyu, D. Grün
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Accurate phenotyping of cells in the tumor microenvironment is crucial for understanding cancer biology and developing effective therapies. However, current methods require precise cell segmentations and struggle to generalize across different imaging modalities, limiting their utility in digital pathology. Here, we show that Contrastive Learning Enabled Accurate Registration of Immune and Tumor Cells (CLEAR-IT) overcomes these limitations, providing a robust and versatile tool for cell phenotyping. CLEAR-IT accurately phenotypes cells comparable to state-of-the-art methods, generalizes across multiplex imaging modalities, maintains high performance even with limited number of labels, and enables the extraction of prognostic markers. Additionally, CLEAR-IT can be combined with existing methods to boost their performance, whereas its lack of need for precise cell segmentations significantly reduces training efforts. This method enhances the robustness and efficiency of digital pathology workflows, making it a valuable tool for cancer research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02cd2bfec0cff758ded2e68209475ad537d70fc" target='_blank'>
              CLEAR-IT: Contrastive Learning to Capture the Immune Composition of Tumor Microenvironments
              </a>
            </td>
          <td>
            Daniel Spengler, Serafim Korovin, Kirti Prakash, Peter Bankhead, R. Debets, Carlas S. Smith, H. E. Balcioglu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, Ondřej Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract While paediatric blood cancers are deadly, modern medical advances have enabled clinicians to measure levels of residual cancer cells to manage therapeutic strategies for patients. However, blood cancers, including leukaemias and lymphomas, are highly heterogeneous and is comprised of complex clonal populations that can hinder efforts in detecting the cancer cells as well as managing treatments. Furthermore, the tumour microenvironment is comprised of heterogenous immune dynamics that may be different between patients. High‐throughput sequencing has constributed to new discoveries in genetic and transcriptomic alterations underpinning cancer, including blood cancers, and has changed how patients are monitored and managed. Here we discuss the recent efforts using single‐cell approach, particularly on efforts to track clonal heterogenity of paediatric blood cancer and the underlying immune response, highlighting avenues for novel biomarker discovery that may have significant impact on clinical oncology practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9b7aa16b92f8240bd156cad7d9dc00c572051c" target='_blank'>
              Measuring single‐cell immune clonality to track haematological cancers
              </a>
            </td>
          <td>
            Amos Choo, Z. Tuong
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background High-grade gliomas including glioblastoma (GBM) and diffuse midline gliomas (DMG) represent the most lethal and aggressive brain cancers where current treatment modalities offer limited efficacy. Chimeric antigen receptor (CAR) T cell therapies have emerged as a promising strategy, boasting tumor-specific targeting and the unique ability to penetrate the blood-brain barrier. However, the effective clinical application hinges on the optimal choice of antigen, with a limited number, currently under investigation. Methods We employed cell surface proteomic analysis of primary human high-grade glioma samples from both adult and pediatric patients. This led to the identification of Ephrin type-A receptor 3 (EphA3) as a prevalently expressed target. We engineered a second-generation EphA3-targeted CAR T cell and assessed function using in vitro and in vivo models of GBM and DMG. Results EphA3-targeted CAR T cells demonstrated robust antigen-specific killing of human GBM and DMG cell lines in vitro. In an orthotopic xenograft NSG mouse model, EphA3-targeted CAR T cells not only effectively eradicated tumors but also established a functional T cell population protective on rechallenge. Remarkably, mice rechallenged with a second contralateral orthotopic tumor implantation achieved complete tumor clearance and maintained a sustained complete response 6 months following initial treatment. Conclusion Building on the proven safety profile of EphA3 antibodies in clinical settings, our study provides compelling preclinical evidence supporting the efficacy of EphA3-targeted CAR T cells against high-grade gliomas. These findings underscore the potential for transitioning this innovative therapy into clinical trials, aiming to revolutionize the treatment landscape for patients afflicted with these formidable brain cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ccdf5eef190aeacc1b356281cbe1f1aeb78930" target='_blank'>
              EphA3-targeted chimeric antigen receptor T cells are effective in glioma and generate curative memory T cell responses
              </a>
            </td>
          <td>
            Leesa Lertsumitkul, Melinda Iliopoulos, Stacie S Wang, Sarah J McArthur, Lisa M Ebert, Alexander J Davenport, R. Endersby, Jordan R Hansford, Kate Drummond, Ryan S Cross, Misty R. Jenkins
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c025ffe7d90f79fbe29203a88da849228769c87c" target='_blank'>
              XENTURION is a population-level multidimensional resource of xenografts and tumoroids from metastatic colorectal cancer patients
              </a>
            </td>
          <td>
            S. Leto, Elena Grassi, Marco Avolio, V. Vurchio, F. Cottino, Martina Ferri, E. Zanella, Sofia Borgato, G. Corti, Laura di Blasio, D. Somale, M. Vara-Messler, Francesco Galimi, F. Sassi, B. Lupo, Irene Catalano, M. Pinnelli, Marco Viviani, Luca Sperti, A. Mellano, Alessandro Ferrero, C. C. Zingaretti, A. Puliafito, Luca Primo, A. Bertotti, L. Trusolino
          </td>
          <td>2024-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a typical highly heterogeneous solid tumor with high morbidity and mortality worldwide, especially in China; however, the immune microenvironment of HCC has not been clarified so far. Here, we employed single‐cell RNA sequencing (scRNA‐seq) on diethylnitrosamine (DEN)‐induced mouse HCC model to dissect the immune cell dynamics during tumorigenesis. Our findings reveal distinct immune profiles in both precancerous and cancerous lesions, indicating early tumor‐associated immunological alterations. Notably, specific T and B cell subpopulations are preferentially enriched in the HCC tumor microenvironment (TME). Furthermore, we identified a subpopulation of naïve B cells with high CD83 expression, correlating with improved prognosis in human HCC. These signature genes were validated in The Cancer Genome Atlas HCC RNA‐seq dataset. Moreover, cell interaction analysis revealed that subpopulations of B cells in both mouse and human samples are activated and may potentially contribute to oncogenic processes. In summary, our study provides insights into the dynamic immune microenvironment and cellular networks in HCC pathogenesis, with a specific emphasis on naïve B cells. These findings emphasize the significance of targeting TME in HCC patients to prevent HCC pathological progression, which may give a new perspective on the therapeutics for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76c814af70ac2f24272bcadf22a4dd9e85ffd9ec" target='_blank'>
              Leveraging single‐cell RNA‐seq for uncovering naïve B cells associated with better prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Qingjia Sun, Rui Gao, Yingxin Lin, Xianchao Zhou, Tao Wang, Jian He
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite being the leading cause of childhood mortality, pediatric gliomas have been relatively understudied, and the repurposing of immunotherapies has not been successful. Whole transcriptome sequencing, single-cell sequencing, and sequential multiplex immunofluorescence were used to identify an immunotherapy strategy evaluated in multiple preclinical glioma models. MAPK-driven pediatric gliomas have a higher interferon signature relative to other molecular subgroups. Single-cell sequencing identified an activated and cytotoxic microglia population designated MG-Act in BRAF-fused MAPK-activated pilocytic astrocytoma (PA), but not in high-grade gliomas or normal brain. TIM3 is expressed on MG-Act and on the myeloid cells lining the tumor vasculature but not normal brain. TIM3 expression becomes upregulated on immune cells in the PA microenvironment and anti-TIM3 reprograms ex vivo immune cells from human PAs to a pro-inflammatory cytotoxic phenotype. In a genetically engineered murine model of MAPK-driven low-grade gliomas, anti-TIM3 treatment increased median survival over IgG and anti-PD1 treated mice. ScRNA sequencing data during the therapeutic window of anti-TIM3 demonstrates enrichment of the MG-Act population. The therapeutic activity of anti-TIM3 is abrogated in the CX3CR1 microglia knockout background. These data support the use of anti-TIM3 in clinical trials of pediatric low-grade MAPK-driven gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93c9fb9117641aac2f760a5c311e07768d3cba6" target='_blank'>
              Pediatric glioma immune profiling identifies TIM3 as a therapeutic target in BRAF-Fusion pilocytic astrocytoma.
              </a>
            </td>
          <td>
            Shashwat Tripathi, Hinda Najem, Corey Dussold, Sebastian Pacheco, Ruochen Du, Moloud Sooreshjani, Lisa A Hurley, James P Chandler, R. Stupp, A. Sonabend, C. Horbinski, R. Lukas, Joanne Xiu, Giselle Y López, Theodore Nicolaides, Valerie Brown, N. R. Wadhwani, Sandi K Lam, C. D. James, Ganesh Rao, Maria G Castro, Amy B. Heimberger, Michael DeCuypere
          </td>
          <td>2024-08-13</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Keratinocytes, the dominant cell type in the melanoma microenvironment during tumor initiation, exhibit diverse effects on melanoma progression. Using a zebrafish model of melanoma and human cell co-cultures, we observed that keratinocytes undergo an Epithelial–Mesenchymal Transition (EMT)-like transformation in the presence of melanoma, reminiscent of their behavior during wound healing. Surprisingly, overexpression of the EMT transcription factor Twist in keratinocytes led to improved overall survival in zebrafish melanoma models, despite no change in tumor initiation rates. This survival benefit was attributed to reduced melanoma invasion, as confirmed by human cell co-culture assays. Single-cell RNA-sequencing revealed a unique melanoma cell cluster in the Twist-overexpressing condition, exhibiting a more differentiated, less invasive phenotype. Further analysis nominated homotypic jam3b-jam3b and pgrn-sort1a interactions between Twist-overexpressing keratinocytes and melanoma cells as potential mediators of the invasive restraint. Our findings suggest that EMT in the tumor microenvironment (TME) may limit melanoma invasion through altered cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8055a65d64a36749562fed897a59067bde3fe48" target='_blank'>
              Restraint of melanoma progression by cells in the local skin environment
              </a>
            </td>
          <td>
            Yilun Ma, Mohita M. Tagore, Miranda V. Hunter, Ting-Hsiang Huang, Emily D. Montal, Joshua M. Weiss, Richard M. White
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In the avian host, comprehensively cataloging immune cell types, their transcriptome profiles, and varying molecular responses to pathogen challenges are necessary steps toward a better understanding of the interplay between genetics and disease resilience. We present a first nuclei atlas of immune cell types derived from the three main immune organs of layer chickens, including spleen, bursa, and thymus. In bursa we also present, an accounting of cell type activation with the bacterial toxin lipopolysaccharide (LPS). Our analysis includes 36,370 total nuclei and 16, 12, and 12 transcriptionally distinct clusters for spleen, bursa, and thymus, respectively. We discover nuclei molecular profiles that uniquely distinguish states of the transcriptome within cell type that could serve as new means to characterize avian immune subtypes. We further subcluster refined immune cell type classifications, specifically highlighting the transcriptomic diversity of B and T cell subtypes. In the bursa, inferred intercellular communication and signaling pathway enrichment analyses across immune and non-immune cell types demonstrate the unappreciated complexity of the B cell repertoire in a model mimicking systemic bacterial infection. This census of all cell types in both primary and one major secondary avian immune organ system, although preliminary, provides a first review of how nuclei transcribe numerous genes, known and unknown, a critical prerequisite for the study avian immunogenetics by cell type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2ccbcf72e358c3e2b143655a8eda7729b28764e" target='_blank'>
              A single-nucleus census of immune and non-immune cell types for the major immune organ systems of chicken
              </a>
            </td>
          <td>
            Elaina R. Sculley, Edward S. Ricemeyer, Rachel C. Carroll, John Driver, Jacqueline Smith, Jim Kaufman, Cari J. Hearn, Adam Balic, Paula Chen, Susan Lamont, Skyler Kramer, Yvonne Drechsler, Hans Cheng, Wesley C. Warren
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. C. Vanderhyden, Hector C Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="3D bioprinting has become a valuable tool for studying the biology of solid tumors, including glioblastoma multiforme (GBM). Our analysis of publicly available bulk RNA and single-cell sequencing data has allowed us to define the chemotactic profile of GBM tumors and identify the cell types that secrete particular chemokines in the GBM tumor microenvironment (TME). Our findings indicate that primary GBM tissues express multiple chemokines, whereas spherical monocultures of GBM cells significantly lose this diversity. Subsequently, the comparative analysis of GBM spherical monocultures vs. 3D-bioprinted multicultures of cells showed a restoration of chemokine profile diversity in 3D-bioprinted cultures. Furthermore, single-cell RNA-Seq analysis showed that cells of the perivascular niche (pericytes and endocytes) express multiple chemokines in the GBM TME. Next, we 3D-bioprinted cells from two glioblastoma cell lines, U-251 and DK-MG, alone and as co-cultures with mesenchymal stromal cells (representing cells of the perivascular niche) and assessed the chemokine secretome. The results clearly demonstrated that the interaction of tumors and mesenchymal cells leads to in a significant increase in the repertoire and levels of secreted chemokines under culture in 21% O2 and 1% O2. Our study indicates that cells of the perivascular niche may perform a substantial role in shaping the chemokine microenvironment in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae245299abc68f53e5e3fd8e4dc327ea2b0cb63" target='_blank'>
              3D-Bioprinted Co-Cultures of Glioblastoma Multiforme and Mesenchymal Stromal Cells Indicate a Role for Perivascular Niche Cells in Shaping Glioma Chemokine Microenvironment
              </a>
            </td>
          <td>
            K. Zielniok, Kinga Rusinek, A. Słysz, Mieszko Lachota, Ewa Bączyńska, Natalia Wiewiórska-Krata, Anna Szpakowska, Martyna Ciepielak, B. Foroncewicz, K. Mucha, Radosław Zagożdżon, Zygmunt Pojda
          </td>
          <td>2024-08-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Metastases from gastric adenocarcinoma (GAC) lead to high morbidity and mortality. Developing innovative and effective therapies requires a comprehensive understanding of the tumor and immune biology of advanced GAC. Yet, collecting matched specimens from advanced, treatment-naïve GAC patients poses a significant challenge, limiting the scope of current research, which has predominantly focused on localized tumors. This gap hinders a deeper insight into the metastatic dynamics of GAC.


METHODS
We performed in-depth single-cell transcriptome and immune profiling on 68 paired, treatment-naïve, primary-metastatic tumors to delineate alterations in cancer cells and their tumor microenvironment (TME) during metastatic progression. To validate our observations, we conducted comprehensive functional studies both in vitro and in vivo, employing cell lines, multiple PDX and novel mouse models of GAC.


RESULTS
Liver and peritoneal metastases exhibited distinct properties in cancer cells and dynamics of TME phenotypes, supporting the notion that cancer cells and their local TMEs co-evolve at metastatic sites. Our study also revealed differential activation of cancer meta-programs across metastases. We observed evasion of cancer cell ferroptosis via GPX4 upregulation during GAC progression. Conditional depletion of Gpx4 or pharmacological inhibition of ferroptosis resistance significantly attenuated tumor growth and metastatic progression. Additionally, ferroptosis-resensitizing treatments augmented the efficacy of CAR T-cell therapy.


CONCLUSIONS
This study represents the largest single-cell dataset of metastatic GACs to date. High-resolution mapping of the molecular and cellular dynamics of GAC metastasis has revealed a rationale for targeting ferroptosis defense in combination with CAR T-cell therapy as a novel therapeutic strategy with potential immense clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c4774948f3c82306cdc21b3d5730989f11741b" target='_blank'>
              Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response.
              </a>
            </td>
          <td>
            Xiangdong Cheng, Enyu Dai, Jibo Wu, Natasha M. Flores, Yanshuo Chu, Ruiping Wang, M. Dang, Zhiyuan Xu, Guangchun Han, Yunhe Liu, D. Chatterjee, Can Hu, Jieer Ying, Yi-an Du, Litao Yang, Xiaoqing Guan, Shaowei Mo, Xuanye Cao, Guangsheng Pei, Jiahui Jiang, Xiaoyin Lu, Ana Morales Benitez, R. Waters, Melissa P Pizzi, N. Shanbhag, Yibo Fan, F. Peng, Samir Hanash, George A. Calin, Andrew Futreal, Shumei Song, Cassian Yee, Pawel K Mazur, Jiang-Jiang Qin, J. Ajani, Linghua Wang
          </td>
          <td>2024-08-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="Genetically engineered CD8+ T cells are being explored for the treatment of various cancers. Analytical characterization represents a major challenge in the development of genetically engineered cell therapies, especially assessing the potential off-target editing and product heterogeneity. As conventional sequencing techniques only provide information at the bulk level, they are unable to detect off-target CRISPR translocation or editing events occurring in minor cell subpopulations. In this study, we report the analytical development of a single-cell multi-omics DNA and protein assay to characterize genetically engineered cell products for safety and genotoxicity assessment. We were able to quantify on-target edits, off-target events, and potential translocations at the targeting loci with per-cell granularity, providing important characterization data of the final cell product. Conclusion: A single-cell multi-omics approach provides the resolution required to understand the composition of cellular products and identify critical quality attributes (CQAs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a403ecc7f675cd4ff2179f421e45809aa747f" target='_blank'>
              Assessing a single-cell multi-omic analytic platform to characterize ex vivo-engineered T-cell therapy products
              </a>
            </td>
          <td>
            Maryam Moshref, Jerry Hung-Hao Lo, Andrew McKay, Julien Camperi, Joseph Schroer, Norikiyo Ueno, Shu Wang, Saurabh Gulati, Somayeh Tarighat, Steffen Durinck, Ho Young Lee, Dayue Chen
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lymph nodes (LNs) are common sites of metastatic invasion in breast cancer, often preceding spread to distant organs and serving as key indicators of clinical disease progression. However, the mechanisms of cancer cell invasion into LNs are not well understood. Existing in vivo models struggle to isolate the specific impacts of the tumor-draining lymph node (TDLN) milieu on cancer cell invasion due to the co-evolving relationship between TDLNs and the upstream tumor. To address these limitations, we used live ex vivo LN tissue slices with intact chemotactic function to model cancer cell spread within a spatially organized microenvironment. After showing that BRPKp110 breast cancer cells were chemoattracted to factors secreted by naïve LN tissue in a 3D migration assay, we demonstrated that ex vivo LN slices could support cancer cell seeding, invasion, and spread. This novel approach revealed dynamic, preferential cancer cell invasion within specific anatomical regions of LNs, particularly the subcapsular sinus (SCS) and cortex, as well as chemokine-rich domains of immobilized CXCL13 and CCL1. While CXCR5 was necessary for a portion of BRPKp110 invasion into naïve LNs, disruption of CXCR5/CXCL13 signaling alone was insufficient to prevent invasion towards CXCL13-rich domains. Finally, we extended this system to pre-metastatic TDLNs, where the ex vivo model predicted a lower invasion of cancer cells. The reduced invasion was not due to diminished chemokine secretion, but it correlated with elevated intranodal IL-21. In summary, this innovative ex vivo model of cancer cell spread in live LN slices provides a platform to investigate cancer invasion within the intricate tissue microenvironment, supporting time-course analysis and parallel read-outs. We anticipate that this system will enable further research into cancer-immune interactions and allow isolation of specific factors that make TDLNs resistant to cancer cell invasion, which are challenging to dissect in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ae3a85a4da7cdcdafe63cb5499e61e44baef898" target='_blank'>
              Ex vivo model of breast cancer cell invasion in live lymph node tissue
              </a>
            </td>
          <td>
            Katerina Morgaenko, Abhinav Arneja, Alexander G. Ball, Audrey M Putelo, Jenny Munson, Melanie R. Rutkowski, R. Pompano
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor growth dynamics serve as a critical aspect of understanding cancer progression and treatment response to mitigate one of the most pressing challenges in healthcare. The in silico approach to understanding tumor behavior computationally provides an efficient, cost-effective alternative to wet-lab examinations and are adaptable to different environmental conditions, time scales, and unique patient parameters. As a result, this paper explored modeling of free tumor growth in cancer, surveying contemporary literature on continuum, discrete, and hybrid approaches. Factors like predictive power and high-resolution simulation competed against drawbacks like simulation load and parameter feasibility in these models. Understanding tumor behavior in different scenarios and contexts became the first step in advancing cancer research and revolutionizing clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5814b4312284d51679616c4dcd34ca6a0385e075" target='_blank'>
              Modeling free tumor growth: Discrete, continuum, and hybrid approaches to interpreting cancer development.
              </a>
            </td>
          <td>
            Dashmi Singh, Dana Paquin
          </td>
          <td>2024-07-19</td>
          <td>Mathematical biosciences and engineering : MBE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Various epigenetic regulations systematically govern gene expression in cells involving various biological processes. Dysregulation of the epigenome leads to aberrant transcriptional programs and subsequently results in diseases, such as cancer. Therefore, comprehensive profiling epigenomics is essential for exploring the mechanisms underlying gene expression regulation during development and disease. Methods In this study, we developed single‐cell chromatin proteins and accessibility tagmentation (scCPA‐Tag), a multi‐modal single‐cell epigenetic profile capturing technique based on barcoded Tn5 transposases and a droplet microfluidics platform. scCPA‐Tag enables the simultaneous capture of DNA profiles of histone modification and chromatin accessibility in the same cell. Results By applying scCPA‐Tag to K562 cells and a hepatocellular carcinoma (HCC) sample, we found that the silence of several chromatin‐accessible genes can be attributed to lysine‐27‐trimethylation of the histone H3 tail (H3K27me3) modification. We characterized the epigenetic features of the tumour cells and different immune cell types in the HCC tumour tissue by scCPA‐Tag. Besides, a tumour cell subtype (C2) with more aggressive features was identified and characterized by high chromatin accessibility and a lower abundance of H3K27me3 on tumour‐promoting genes. Conclusions Our multi‐modal scCPA‐Tag provides a comprehensive approach for exploring the epigenetic landscapes of heterogeneous cell types and revealing the mechanisms of gene expression regulation during developmental and pathological processes at the single‐cell level. Highlights scCPA‐Tag offers a highly efficient and high throughput technique to simultaneously profile histone modification and chromatin accessibility within a single cell. scCPA‐Tag enables to uncover multiple epigenetic modification features of cellular compositions within tumor tissues. scCPA‐Tag facilitates the exploration of the epigenetic landscapes of heterogeneous cell types and provides the mechanisms governing gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1d48ebd31bec69ee99957ebc5de9bec997e3ad" target='_blank'>
              Single‐cell multi‐modal chromatin profiles revealing epigenetic regulations of cells in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Chunqing Wang, Waidong Huang, Yu Zhong, Xuanxuan Zou, Shang Liu, Jie Li, Yunfan Sun, Kaiqian Zhou, Xi Chen, Zihao Li, Shanshan Wang, Yaling Huang, Yinqi Bai, Jianhua Yin, Xin Jin, Shiping Liu, Yue Yuan, Q. Deng, Miaomiao Jiang, Chuanyu Liu, Longqi Liu, Xun Xu, Liang Wu
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d5e47f21e09bedd3c2faf022dd2ac57af3858d" target='_blank'>
              Large field of view and spatial region of interest transcriptomics in fixed tissue
              </a>
            </td>
          <td>
            Xiaonan Cui, Xue Dong, Mengzhu Hu, Wenjian Zhou, Weiyang Shi
          </td>
          <td>2024-08-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Immune cells undergo cytokine-driven polarization in respond to diverse stimuli. This process significantly modulates their transcriptional profiles and functional states. Although single-cell RNA sequencing (scRNA-seq) has advanced our understanding of immune responses across various diseases or conditions, currently there lacks a method to systematically examine cytokine effects and immune cell polarization. To address this gap, we developed Single-cell unified polarization assessment (Scupa), the first computational method for comprehensive immune cell polarization analysis. Scupa is trained on data from the Immune Dictionary, which characterizes 66 cytokine-driven polarization states across 14 immune cell types. By leveraging the cell embeddings from the Universal Cell Embeddings model, Scupa effectively identifies polarized cells in new datasets generated from different species and experimental conditions. Applications of Scupa in independent datasets demonstrated its accuracy in classifying polarized cells and further revealed distinct polarization profiles in tumor-infiltrating myeloid cells across cancers. Scupa complements conventional single-cell data analysis by providing new insights into immune cell polarization, and it holds promise for assessing molecular effects or identifying therapeutic targets in cytokine-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4376cd46f3ca8bce1a4598eede7d5c23480496ac" target='_blank'>
              Scupa: Single-cell unified polarization assessment of immune cells using the single-cell foundation model
              </a>
            </td>
          <td>
            Wendao Liu, Zhongming Zhao
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Multimodal analysis of single-cell samples from healthy and diseased tissues at various stages provides a comprehensive view that identifies disease-specific cells, their molecular features and aids in patient stratification. Here, we present MultiMIL, a novel weakly-supervised multimodal model designed to construct multimodal single-cell references and prioritize phenotype-specific cells via patient classification. MultiMIL effectively integrates single-cell modalities, even when they only partially overlap, providing robust representations for downstream analyses such as phenotypic prediction and cell prioritization. Using a multiple-instance learning approach, MultiMIL aggregates cell-level measurements into sample-level representations and identifies disease-specific cell states through attention-based scoring. We demonstrate that MultiMIL accurately identifies disease-specific cell states in blood and lung samples, identifying novel disease-associated genes and achieving superior patient classification accuracy compared to existing methods. We anticipate MultiMIL will become an essential tool for querying single-cell multiomic atlases, enhancing our understanding of disease mechanisms and informing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8efc9b9ff38309e496bbcf1bd6af1cf5075840" target='_blank'>
              Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases
              </a>
            </td>
          <td>
            Anastasia Litinetskaya, Maiia Shulman, Soroor Hediyeh-zadeh, A. Moinfar, Fabiola Curion, Artur Szałata, Alireza Omidi, M. Lotfollahi, F. Theis
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55bf88cd2f361cbe0e78275013e0864970ee199" target='_blank'>
              In vivo CRISPR screens identify a dual function of MEN1 in regulating tumor–microenvironment interactions
              </a>
            </td>
          <td>
            Peiran Su, Yin Liu, Tianyi Chen, Yibo Xue, Yong Zeng, Guanghui Zhu, Sujun Chen, Mona Teng, Xinpei Ci, Mengdi Guo, Michael Y. He, Jun Hao, Vivian Chu, Wenxi Xu, Shiyan Wang, P. Mehdipour, Xin Xu, Sajid A. Marhon, F. Soares, Nhu-An Pham, Bell Xi Wu, Peter Hyunwuk Her, Shengrui Feng, Najd Alshamlan, Maryam Khalil, Rehna Krishnan, Fangyou Yu, Chang Chen, Francis Burrows, Razqallah Hakem, Mathieu Lupien, Shane Harding, Benjamin H. Lok, Catherine O'Brien, Alejandro Berlin, D. D. De Carvalho, David G Brooks, Daniel Schramek, Ming-Sound Tsao, Housheng Hansen He
          </td>
          <td>2024-09-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72034bf3ceec6cf3d3b2520c3a42de2885dca12a" target='_blank'>
              Tumor necrosis factor receptor 2 promotes endothelial cell-mediated suppression of CD8+ T cells through tuning glycolysis in chemoresistance of breast cancer
              </a>
            </td>
          <td>
            Yu Hu, Xiaohan Lou, Kaili Zhang, Longze Pan, Yue-Yu Bai, Linlin Wang, Ming Wang, Yan Yan, Jiajia Wan, Xiaohan Yao, Xixi Duan, Chen Ni, Zhihai Qin
          </td>
          <td>2024-07-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Skeletal muscle regeneration is mediated by resident muscle stem cells that produce progeny to repair or recreate muscle. Critical to this function is the ability to transition between states of proliferation and quiescence. This balancing act is disrupted with age, leading to eroded regenerative capacity. Notwistanding, mechanisms by which the regenerating niche directs MuSCs return to the dormant state are largely unknown. Since single-cell RNAseq methods exclude the analysis of multinucleated cells, we generated single-nuclei RNAseq datasets of regenerating muscle to capture the full breadth of myogenic progression. With this, we uncovered new transition states between differentiating myocytes and syncytial multinucleated cells. Using cell communication inference tools, we highlighted receptor-ligand interactions between MuSCs and fusing nuclei. We leveraged a bespoke biomimetic 3D niche that induces MuSC quiescence, to filter the predicted interactions using a Cas9-based functional genomics approach. We found the proteoglycan Perlecan (Hspg2) promotes MuSC re-quiescence. Hspg2 silencing in vivo, during muscle regeneration, induced an aging-like phenotype and perturbed MuSC re-quiescence. Notably, the temporal profile and overall levels of Perlecan were altered with age. Exogenous supplementation with Endorepellin (truncated Perlecan) rescued MuSC decline following aged muscle regeneration, offering a new therapeutic target. Thus, coupling in silico predictions with a 3D in vitro assay followed by in vivo investigations revealed a previously inaccessible window of biology; that spatiotemporal coordination of MuSC re-quiescence is directed by fusing myonuclei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8472bde10017480be8b7a03b85bf49bab5ad8ea6" target='_blank'>
              Integrating in silico predictions with an engineered tissue assay identifies Perlecan as an age-perturbed re-quiescence cue for muscle stem cells
              </a>
            </td>
          <td>
            Erik Jacques, Pauline Garcia, Orane Mercier, Yechen Hu, Cyril Dégletagne, Jade Ravent, Sidy Fall, Maira P. Almeida, Aaron R. Wheeler, S. Angers, P. Gilbert, F. Le Grand
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Molecular and cellular characterization of tumors is essential due to the complex and heterogeneous nature of cancer. In recent decades, many bioinformatic tools and experimental techniques have been developed to achieve personalized characterization of tumors. However, sample handling continues to be a major challenge as limitations such as prior treatments before sample acquisition, the amount of tissue obtained, transportation, or the inability to process fresh samples pose a hurdle for experimental strategies that require viable cell suspensions. Here, we present an optimized protocol that allows the recovery of highly viable cell suspensions from breast cancer primary tumor biopsies. Using these cell suspensions we have successfully characterized genome architecture through Hi-C. Also, we have evaluated single-cell gene expression and the tumor cellular microenvironment through single-cell RNAseq. Both technologies are key in the detailed and personalized molecular characterization of tumor samples. The protocol described here is a cost-effective alternative to obtain viable cell suspensions from biopsies simply and efficiently.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6963d9d1f4834d1881e0dfb691c6907687ec39cc" target='_blank'>
              Obtention of viable cell suspensions from breast cancer tumor biopsies for 3D chromatin conformation and single-cell transcriptome analysis
              </a>
            </td>
          <td>
            Aura Stephenson-Gussinye, Luis A. Rendón-Bautista, B. E. Ruiz-Medina, Eduardo Blanco-Olais, Rosario Pérez-Molina, Cleofas Marcial-Medina, Y. Chávarri-Guerra, Enrique Soto-Pérez-de-Celis, Andrea Morales-Alfaro, A. Esquivel-López, Fernando Candanedo-González, Armando Gamboa-Domínguez, Rubén Cortes-González, Alejandro Alfaro-Goldaracena, Sara E. Vázquez-Manjarrez, Guido Grajales-Figueroa, Beatriz Astudillo-Romero, Jesús Ruiz-Manriquez, A. C. Poot-Hernández, Paula Licona-Limón, Mayra Furlan-Magaril
          </td>
          <td>2024-08-22</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The presence of pre-existing or acquired drug-resistant cells within the tumor often leads to tumor relapse and metastasis. Single-cell RNA sequencing (scRNA-seq) enables to elucidate the subtle differences in drug responsiveness among distinct cell subpopulations within tumors. A few methods have employed scRNA-seq data to predict the drug response of individual cells to date, but their performance is far from satisfactory. In this study, we propose SSDA4Drug, a semi-supervised few-shot transfer learning method for inferring drug-resistant cancer cells. SSDA4Drug extracts pharmacogenomic features from both bulk and single-cell transcriptomic data by utilizing semi-supervised adversarial domain adaptation. This allows us to transfer knowledge of drug sensitivity from bulk-level cell lines to single cells. We conduct extensive performance evaluation experiments across multiple independent scRNA-seq datasets, and demonstrate the state-of-the-art performance of SSDA4Drug. Remarkably, with only one or two labeled target-domain samples, SSDA4Drug significantly boosts the predictive performance of single-cell drug responses. Moreover, SSDA4Drug accurately recapitulates the temporally dynamic changes of drug responses during continuous drug exposure of tumor cells, and successfully identifies reversible drug-responsive states in lung cancer cells, which initially acquired resistance through drug exposure but later restore sensitivity induced by drug holiday. Also, our predicted drug responses consistently align with the developmental patterns of drug sensitivity observed along the evolutionary trajectory of oral squamous cell carcinoma cells. In addition, our derived SHAP values and integrated gradients effectively pinpoint the key genes involved in drug resistance in prostate cancer cells. These findings highlight the exceptional performance of our method in determining single-cell drug responses. This powerful tool holds the potential for identifying drug-resistant tumor cell subpopulations, paving the way for strides in precision medicine and novel drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25abb010de95fa69c67d91f158879e6866ff5c92" target='_blank'>
              Identifying drug-resistant individual cells within tumors by semi-supervised domain adaptation
              </a>
            </td>
          <td>
            Kaishun Huang, Hui Liu
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Metastasis is a hallmark of cancer that is responsible for the greatest number of cancer-related deaths (~90%). For the metastasis to take place, the primary tumor mass engineers migratory cells that can invade adjacent tissues and circulate to distant organs where cancer cells have sufficient nutrients to grow and form new colonies. The migrating cell group typically consists of “LEADER” cells at the front, which navigate the path lead collective invasive packs, and “FOLLOWER” cells that trail behind and support the movement. Both leader and follower cells communicate and cooperate to achieve collective cancer migration and invasion. Recent studies showed that the elimination of a leader cell disrupts collective cell migration. However, The mechanisms controlling the synchronized migration of leader and follower cells are still unresolved. Therefore, the genetic and metabolic traits of leader and follower cells, enabling them to navigate their way into secondary nutrient-rich sites and escape the immune system surveillance, need further elucidation. Interestingly, the differential Multi-omic analyses of cancer leader and follower cells are an emerging area of research within the study of tumor heterogeneity and metastasis. This review highlights the recent findings about leader/follower cancer cells focusing on their unique proteomic, metabolic, and transcriptomic signatures rendering them their unique involvement in cancer metastasis and invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04fc44097d774157998b41ef30091e3fd7a1d4f3" target='_blank'>
              Differential Analyses of Leader and Follower Cancer Cells
              </a>
            </td>
          <td>
            Ashraf M. Omar, Amira A. Abdellatef
          </td>
          <td>2024-07-25</td>
          <td>Future Perspectives of Medical, Pharmaceutical and Environmental Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="CD4+Foxp3+ regulatory T cells (Tregs) are key to maintain peripheral self-tolerance and suppress immune responses to tumors. Their accumulation in the tumor microenvironment (TME) correlates with poor clinical outcome in several human cancers, including breast cancer (BC). However, the properties of intratumoral Tregs remain largely unknown. Here, we found that a functionally distinct subpopulation of tumor-infiltrating Tregs, which express the Foxp3 splicing variant retaining exon 2 (Foxp3E2), is prominent in the TME and peripheral blood of hormone receptor- positive (HR+) BC subjects with poor prognosis. Notably, a comprehensive examination of the Tumor Cell Genome Atlas (TCGA) validated Foxp3E2 as an independent prognostic marker in all other BC subtypes. We found that FOXP3E2 expression underlies BCs with highly immune suppressive landscape, defective mismatch repair and a stem-like signature thus highlighting pathways involved in tumor immune evasion. Finally, we confirmed the higher immunosuppressive capacity of BC patients-derived Foxp3E2+ Tregs by functional assays. Our study suggests Foxp3E2+ Tregs might be used as an independent biomarker to predict BC prognosis and recurrence, and to develop super-targeted depletion-based immunotherapies. One-sentence summaries Foxp3E2+ Treg enrichment reflects an increased tumor-immune suppression and predicts prognosis and recurrence in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562d7aed2bb35b4d998b16c0064b1c3932d34dac" target='_blank'>
              CD4+Foxp3E2+ regulatory T cell frequency predicts breast cancer prognosis and recurrence
              </a>
            </td>
          <td>
            Clorinda Fusco, Francesca di Rella, Antonietta Liotti, Alessandra Colamatteo, A. L. Ferrara, Vincenzo Gigantino, F. Collina, Emanuela Esposito, I. Donzelli, Antonio Porcellini, Antonia Feola, Teresa Micillo, Francesco Perna, Federica Garziano, G. Maniscalco, Gilda Varricchi, M. Mottola, B. Zuccarelli, Bruna De Simone, M. Bonito, G. Matarese, A. Accurso, Martina Pontillo, Daniela Russo, Luigi Insabato, Alessandra Spaziano, Irene Cantone, A. Pezone, V. Rosa
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5b4e38880804cee273ee49510e4f038a8ad036" target='_blank'>
              scPriorGraph: constructing biosemantic cell–cell graphs with prior gene set selection for cell type identification from scRNA-seq data
              </a>
            </td>
          <td>
            Xiyue Cao, Yu-An Huang, Zhu-Hong You, Xuequn Shang, Lun Hu, Peng-Wei Hu, Zhi-an Huang
          </td>
          <td>2024-08-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Heloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gallbladder carcinoma (GBC) is a malignant hepatobiliary cancer characterized by an intricate tumor microenvironments (TME) and heterogeneity. The traditional GBC 2D culture models cannot faithfully recapitulate the characteristics of the TME. Three-dimensional (3D) bioprinting enables the establishment of high-throughput and high-fidelity multicellular GBC models. In this study, we designed a concentric cylindrical tetra-culture model to reconstitute the spatial distribution of cells in tumor tissue, with the inner portion containing GBC cells, and the outer ring containing a mixture of endothelial cells, fibroblasts, and macrophages. We confirmed the survival, proliferation, biomarker expression and gene expression profiles of GBC 3D tetra-culture models. Hematoxylin-eosin (HE) and immunofluorescence staining verified the morphology and robust expression of GBC/endothelial/fibroblast/macrophage biomarkers in GBC 3D tetra-culture models. Single-cell RNA sequencing revealed two distinct subtypes of GBC cells within the model, glandular epithelial and squamous epithelial cells, suggesting the mimicry of intratumoral heterogeneity. Comparative transcriptome profile analysis among various in vitro models revealed that cellular interactions and the TME in 3D tetra-culture models reshaped the biological processes of tumor cells to a more aggressive phenotype. GBC 3D tetra-culture models restored the characteristics of the TME as well as intratumoral heterogeneity. Therefore, this model is expected to have future applications in tumor biology research and antitumor drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc6d2cc903e389bd57ed19225ad293a9f8472b5b" target='_blank'>
              Multicellular 3D bioprinted human gallbladder carcinoma forin vitromimicry of tumor microenvironment and intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Yukai Jin, Jiangang Zhang, Jiali Xing, Yiran Li, Huiyu Yang, Liujian Ouyang, Zhiyuan Fang, Lejia Sun, Bao Jin, Pengyu Huang, Huayu Yang, Shunda Du, X. Sang, Yilei Mao
          </td>
          <td>2024-08-09</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The success of chimeric antigen receptor (CAR) T-cell therapy in treating hematologic malignancies has generated widespread interest in translating this technology to solid cancers. However, issues like tumor infiltration, the immunosuppressive tumor microenvironment, and tumor heterogeneity limit its efficacy in the solid tumor setting. Recent experimental and clinical studies propose local administration directly into the tumor or at the tumor site to increase CAR T-cell infiltration and improve treatment outcomes. Characteristics of the types of solid tumors that may be the most receptive to this treatment approach remain unclear. In this work, we develop a spatiotemporal model for CAR T-cell treatment of solid tumors, and use numerical simulations to compare the effect of introducing CAR T cells via intratumoral injection versus intracavitary administration in diverse cancer types. We demonstrate that the model can recapitulate tumor and CAR T-cell data from imaging studies of local administration of CAR T cells in mouse models. Our results suggest that locally administered CAR T cells will be most successful against slowly proliferating, highly diffusive tumors, which have the lowest average tumor cell density. These findings affirm the clinical observation that CAR T cells will not perform equally across different types of solid tumors, and suggest that measuring tumor density may be helpful when considering the feasibility of CAR T-cell therapy and planning dosages for a particular patient. We additionally find that local delivery of CAR T cells can result in deep tumor responses, provided that the initial CAR T-cell dose does not contain a significant fraction of exhausted cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7a6bf493c0b33ad1aad3083fdba685e7c59e1d" target='_blank'>
              Spatiotemporal dynamics of tumor - CAR T-cell interaction following local administration in solid cancers
              </a>
            </td>
          <td>
            Katherine Owens, Aminur Rahman, Ivana Bozic
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab01bc978e62346ed66c2ee4492c4c5372e12a2c" target='_blank'>
              Characterization of cuproptosis signature in clear cell renal cell carcinoma by single cell and spatial transcriptome analysis
              </a>
            </td>
          <td>
            Xiaohong Zou, Xiaoqing Liu, Huiting Wang, Zhenhua Li, Chen Zhou
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Understanding interplay of breast cancer and microenvironment is critical. Here, we identified two transcriptomic subtypes and five immune infiltration patterns from RNA-seq and multiplex immunohistochemistry from 21 ER+/HER2- invasive lobular breast cancers. The proliferative subtype associated with increased immune infiltration especially by immunosuppressive regulatory T-cells and macrophages. We also defined a TAM-Low signature, which associated with lower infiltration of proliferative, pro-inflammatory TAM, and improved outcome in patients with ER+ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13ba7e3cf03c88a995520cfd413c5d66f042dd69" target='_blank'>
              Immune Infiltration Correlates with Transcriptomic Subtypes in Primary ER+ Invasive Lobular Breast Cancer
              </a>
            </td>
          <td>
            Fangyuan Chen, Sayali S Onkar, Jian Zou, Yujia Li, Haley Arbore, Sai Maley, George C. Tseng, Peter Lucas, Tullia Bruno, Dario Vignali, Julia Foldi, Marija Balic, Adrian Lee, S. Oesterreich
          </td>
          <td>2024-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, Jian-Qiang Lu, John S Lazo, T. Kislinger, Yu Lu, Kevin R. Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Human cytomegalovirus (HCMV) infection is common in tumor tissues across different types of cancer. While HCMV has not been recognized as a cancer-causing virus, numerous studies hint at its potential role in cancer development where its presence in various cancers corresponds with the hallmarks of cancer. Herein, we discuss and demonstrate that high-risk HCMV-DB and BL strains have the potential to trigger transformation in epithelial cells, including human mammary epithelial cells (HMECs), ovarian epithelial cells (OECs), and prostate epithelial cells (PECs), through the generation of polyploid giant cancer cells (PGCCs). A discussion is provided on how HCMV infection creates a cellular environment that promotes oncogenesis, supporting the continuous growth of CMV-transformed cells. The aforementioned transformed cells, named CTH, CTO, and CTP cells, underwent giant cell cycling with PGCC generation parallel to dedifferentiation, displaying stem-like characteristics and an epithelial–mesenchymal transition (EMT) phenotype. Furthermore, we propose that giant cell cycling through PGCCs, increased EZH2 expression, EMT, and the acquisition of malignant traits represent a deleterious response to the cellular stress induced by high-risk oncogenic HCMV strains, the latter being the origin of the transformation process in epithelial cells upon HCMV infection and leading to adenocarcinoma of poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139ca38ce75a83fdf75794021da421194cdfffd" target='_blank'>
              Polyploid Giant Cancer Cells: A Distinctive Feature in the Transformation of Epithelial Cells by High-Risk Oncogenic HCMV Strains
              </a>
            </td>
          <td>
            Georges Herbein, Ranim El Baba
          </td>
          <td>2024-07-31</td>
          <td>Viruses</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f2554beac8f2504f8636ae14089de94d4df8bc" target='_blank'>
              Deciphering the impact of aggregated autophagy-related genes TUBA1B and HSP90AA1 on colorectal cancer evolution: a single-cell sequencing study of the tumor microenvironment
              </a>
            </td>
          <td>
            Qianping Xu, Chao Liu, Hailin Wang, Shujuan Li, Hanshen Yan, Ziyang Liu, Kexin Chen, Yaoqin Xu, Runqin Yang, Jingfang Zhou, Xiaolin Yang, Jie Liu, Lexin Wang
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the deadliest human cancers with very limited treatment options available. The malignant behavior of GBM is manifested in a tumor which is highly invasive, resistant to standard cytotoxic chemotherapy, and strongly immunosuppressive. Immune checkpoint inhibitors have recently been introduced in the clinic and have yielded promising results in certain cancers. GBM, however, is largely refractory to these treatments. The immune checkpoint CD47 has recently gained attention as a potential target for intervention as it conveys a “don’t eat me” signal to tumor-associated macrophages (TAMs) via the inhibitory SIRP alpha protein. In preclinical models, the administration of anti-CD47 monoclonal antibodies has shown impressive results with GBM and other tumor models. Several well-characterized oncogenic pathways have recently been shown to regulate CD47 expression in GBM cells and glioma stem cells (GSCs) including Epidermal Growth Factor Receptor (EGFR) beta catenin. Other macrophage pathways involved in regulating phagocytosis including TREM2 and glycan binding proteins are discussed as well. Finally, chimeric antigen receptor macrophages (CAR-Ms) could be leveraged for greatly enhancing the phagocytosis of GBM and repolarization of the microenvironment in general. Here, we comprehensively review the mechanisms that regulate the macrophage phagocytosis of GBM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/531385db9d8c024f43256b737471afd7c402040f" target='_blank'>
              Phagocytosis Checkpoints in Glioblastoma: CD47 and Beyond
              </a>
            </td>
          <td>
            Amber Afzal, Zobia Afzal, Sophia Bizink, Amanda Davis, Sara Makahleh, Yara Mohamed, Salvatore J. Coniglio
          </td>
          <td>2024-07-23</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Tumor initiation and progression rely on intricate cellular pathways that promote proliferation while suppressing differentiation, yet the importance of pathways inhibiting differentiation in cancer remains incompletely understood. Here, we reveal a novel mechanism centered on the repression of the neuronal-specific transcription factor ARNT2 by the MYC oncogene that governs the balance between proliferation and differentiation. We found that MYC coordinates the transcriptional repression of ARNT2 through the activity of polycomb repressive complex 2 (PRC2). Notably, ARNT2, highly and specifically expressed in the central nervous system, is diminished in glioblastoma, inversely correlating with patient survival. Utilizing in vitro and in vivo models, we demonstrate that ARNT2 knockout (KO) exerts no discernible effect on the in vitro proliferation of glioblastoma cells, but significantly enhances the growth of glioblastoma cells in vivo. Conversely, ARNT2 overexpression severely dampens the growth of fully transformed glioblastoma cells subcutaneously or orthotopically xenografted in mice. Mechanistically, ARNT2 depletion diminishes differentiation and enhances stemness of glioblastoma cells. Our findings provide new insights into the complex mechanisms used by oncogenes to limit differentiation in cancer cells and define ARNT2 as a tumor suppressor in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fc2b3a844fbe1cf3418432eb71140217e74feb" target='_blank'>
              MYC-Mediated Inhibition of ARNT2 Uncovers a Key Tumor Suppressor in Glioblastoma
              </a>
            </td>
          <td>
            Maralice Conacci-Sorrell, Yi-Heng Hao, N. Borenstein-Auerbach, Anthony Grichuk, Li Li, M. Lafita-Navarro, Shun Fang, Pedro A S Nogueira, Jiwoong Kim, Lin Xu, J. Shay
          </td>
          <td>2024-08-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary The classification of breast cancer has been a complex subject, with diverse origins of normal breast epithelial cells and evolutionary pathways contributing to its extensive heterogeneity, which further complicates the treatment process. We developed a dynamic network model to simulate the emergence of different cellular states and explore the underlying biological mechanisms. This model revealed cellular states that align with four recognized molecular subtypes of breast cancer. It also identified feedback loops that either reinforce or facilitate phenotypic transitions, and allowed us to trace these transitions through the model’s topological structure. This approach provides new insights into the mechanisms driving breast cancer heterogeneity and may benefit the development of more targeted therapeutic strategies. Abstract Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Biological systems can be understood by perturbing individual components and studying the system’s response. Cell biology experiments are defined by the applied treatment, cellular state, and the assayed phenotype. Given the vast number of possible combinations, testing every scenario is impractical. We present Prophet, a transformer-based computational model for cellular phenotype prediction. Prophet learns a representation of the cell biology experiment space, enabling it to predict the outcomes of untested small molecule or genetic perturbations in new cellular contexts across diverse phenotypes including gene expression, cell viability, and cell morphology. Its scalable architecture facilitates training across independent assays, using transfer learning to enhance performance across phenotypes. In vitro validation shows Prophet’s potential to guide experimental design, making it a valuable tool for accelerating biological discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8123a1ada6d3520bdccd8d89073356381fcdd64" target='_blank'>
              Scalable and universal prediction of cellular phenotypes
              </a>
            </td>
          <td>
            Yuge Ji, Alejandro Tejada-Lapuerta, Niklas A. Schmacke, Zihe Zheng, Xinyue Zhang, Simrah Khan, I. Rothenaigner, Juliane Tschuck, K. Hadian, F. Theis
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="We performed multi-omic profiling of epidermal keratinocytes, precancerous actinic keratoses, and squamous cell carcinomas to understand the molecular transitions during skin carcinogenesis. Single-cell mutational analyses showed that most keratinocytes in normal skin had lower mutation burdens than melanocytes and fibroblasts, however keratinocytes with TP53 or NOTCH1 mutations had substantially higher mutation burdens, suggesting that these mutations prime keratinocytes for transformation by increasing their mutation rate. Mutational profiling and spatial transcriptomics on squamous cell carcinomas adjacent to actinic keratoses revealed TERT promoter and CDKN2A mutations emerging in actinic keratoses, whereas additional mutations inactivating ARID2 and activating the MAPK-pathway delineated the transition to squamous cell carcinomas. Spatial variation in gene expression patterns was common in both tumor and immune cells, with high expression of checkpoint molecules at the invasive front of tumors. In conclusion, this study documents key events during the evolution of cutaneous squamous cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6fcde4006164b184d1b142c905e9d0585954973" target='_blank'>
              Genetic evolution of keratinocytes to cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Bishal Tandukar, Delahny Deivendran, Limin Chen, Noel Cruz-Pacheco, Harsh Sharma, Albert Xu, Aravind K. Bandari, Daniel B. Chen, Christopher George, Annika Marty, Raymond J. Cho, Jeffrey Cheng, Drew K Saylor, P. Gerami, Sarah T. Arron, Boris C. Bastian, A. H. Shain
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Mismatch repair proficient (MMRp) tumors of colorectal origin are one of the prevalent yet unpredictable clinical challenges. Despite earnest efforts, optimal treatment modalities have yet to emerge for this class. The poor prognosis and limited actionability of MMRp are ascribed to a low neoantigen burden and a desert-like microenvironment. This review focuses on the critical roadblocks orchestrated by an immune evasive mechanistic milieu in the context of MMRp. The low density of effector immune cells, their weak spatiotemporal underpinnings, and the high-handedness of the IL-17-TGF-β signaling are intertwined and present formidable challenges for the existing therapies. Microbiome niche decorated by Fusobacterium nucleatum alters the metabolic program to maintain an immunosuppressive state. We also highlight the evolving strategies to repolarize and reinvigorate this microenvironment. Reconstruction of anti-tumor chemokine signaling, rational drug combinations eliciting T cell activation, and reprograming the maladapted microbiome are exciting developments in this direction. Alternative vulnerability of other DNA damage repair pathways is gaining momentum. Integration of liquid biopsy and ex vivo functional platforms provide precision oncology insights. We illustrated the perspectives and changing landscape of MMRp-CRC. The emerging opportunities discussed in this review can turn the tide in favor of fighting the treatment dilemma for this elusive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823c184571d6413de5f857c9ff7de13bd6953768" target='_blank'>
              Mismatch repair-proficient tumor footprints in the sands of immune desert: mechanistic constraints and precision platforms
              </a>
            </td>
          <td>
            Biswanath Majumder, N. Nataraj, Leela Maitreyi, Santanu Datta
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G. Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cbe85e7267d943e47fa9cdeee4b8e805b7a651" target='_blank'>
              CD24 flags anastasis in melanoma cells.
              </a>
            </td>
          <td>
            Martina H Vasileva, Anette Bennemann, Karolin Zachmann, Michael P Schön, Jorge Frank, Vijay Kumar Ulaganathan
          </td>
          <td>2024-08-13</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive neuroendocrine skin cancer with a ~50% response rate to immune checkpoint blockade (ICB) therapy. To identify predictive biomarkers, we integrated bulk and single-cell RNA-seq with spatial transcriptomics from a cohort of 186 samples from 116 patients, including bulk RNA-seq from 14 matched pairs pre- and post-ICB. In non-responders, tumors show evidence of increased tumor proliferation, neuronal stem cell markers, and IL-1. Responders have increased type I/II interferons and pre-existing tissue resident (Trm) CD8 or Vd1 gd T cells that functionally converge with overlapping antigen-specific transcriptional programs and clonal expansion of public TCRs. Spatial transcriptomics demonstrated co-localization of T cells with B and dendritic cells, which supply chemokines and co-stimulation. Lastly, ICB significantly increased clonal expansion or recruitment of Trm and Vd1 cells in tumors specifically in responders, underscoring their therapeutic importance. These data identify potential clinically actionable biomarkers and therapeutic targets for MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc8fff7c0c805192f9c9a67df2e29b038f02c52" target='_blank'>
              Pre-existing skin-resident CD8 and gd T cell circuits mediate immune response in Merkel cell carcinoma and predict immunotherapy efficacy.
              </a>
            </td>
          <td>
            Z. Reinstein, Yue Zhang, Oscar E. Ospina, Matt D Nichols, Victoria A Chu, Alvaro de Mingo Pulido, Karol Prieto, Jonathan V Nguyen, Rui Yin, Carlos M Moran Segura, Ahmed Usman, B. Sell, Spencer Ng, Janis V de la Iglesia, Sunandana Chandra, Jeffrey A Sosman, Raymond J. Cho, Jeffrey B. Cheng, E. Ivanova, S. Koralov, R. Slebos, C. Chung, N. Khushalani, Jane L. Messina, A. Sarnaik, J. Zager, V. Sondak, Charles J. Vaske, Sungjune Kim, A. Brohl, Xinlei Mi, Brian G. Pierce, Xuefeng Wang, Brooke L. Fridley, Kenneth Y. Tsai, Jaehyuk Choi
          </td>
          <td>2024-07-26</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the analysis of changes in phenotype, such as in vivo and ex vivo morphology and localization of cellular proteins and organelles can provide important insights into patient treatment options. Novel therapies are needed to extend survival in metastatic castration-resistant prostate cancer (mCRPC). Prostate-specific membrane antigen (PSMA), a cell surface glycoprotein that is commonly overexpressed by prostate cancer (PC) cells relative to normal prostate cells, provides a validated target. We developed a software for image analysis designed to identify PSMA expression on the surface of epithelial cells in order to extract prognostic metrics. In addition, our software can deliver predictive information and inform clinicians regarding the efficacy of PC therapy. We can envisage additional applications of our software system, beyond PC, as PSMA is expressed in a variety of tissues. Our method is based on image denoising, topologic partitioning, and edge detection. These three steps allow to segment the area of each PSMA spot in an image of a coverslip with epithelial cells. Our objective has been to present the community with an integrated, easy to use by all, tool for resolving the complex cytoskeletal organization and it is our goal to have such software system approved for use in the clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926fb40317d30bcdb538d1f8f9766fb5c180eb87" target='_blank'>
              Quantitative Microscopy in Medicine
              </a>
            </td>
          <td>
            Alexandre Matov, Shayan Modiri
          </td>
          <td>2024-08-03</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endometriosis, affecting 1 in 9 women, presents treatment and diagnostic challenges. To address these issues, we generated the biggest single-cell atlas of endometrial tissue to date, comprising 466,371 cells from 35 endometriosis and 25 non-endometriosis patients without exogenous hormonal treatment. Detailed analysis reveals significant gene expression changes and altered receptor-ligand interactions present in the endometrium of endometriosis patients, including increased inflammation, adhesion, proliferation, cell survival, and angiogenesis in various cell types. These alterations may enhance endometriosis lesion formation and offer novel therapeutic targets. Using ScaiVision, we developed neural network models predicting endometriosis of varying disease severity (median AUC = 0.83), including an 11-gene signature-based model (median AUC = 0.83) for hypothesis-generation without external validation. In conclusion, our findings illuminate numerous pathway and ligand-receptor changes in the endometrium of endometriosis patients, offering insights into pathophysiology, targets for novel treatments, and diagnostic models for enhanced outcomes in endometriosis management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9557348ef1327c4800ef29c1c6e1843f22fe83" target='_blank'>
              Tracing Endometriosis: Coupling deeply phenotyped, single-cell based Endometrial Differences and AI for disease pathology and prediction
              </a>
            </td>
          <td>
            Lea Duempelmann, Shaoline Sheppard, Angelo Duo, Jitka Skrabalova, Brett McKinnon, Thomas Andrieu, Dennis Goehlsdorf, S. Muzumdar, Cinzia Donato, Ryan Lusby, Wiebke Solass, Hans Bösmüller, Peter Nestorov, Michael D. Mueller
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="This study presents an interdisciplinary approach to analyse the distribution of cancer stem cell markers (CSCMs) across various cancer-affected organs using hypergraphs. Cancer stem cells (CSCs) play a crucial role in cancer initiation, progression, and metastasis. By employing hypergraphs, we model the relationships between CSCM locations and cancerous organs, providing a comprehensive representation of these interactions. Initially, we utilised an unweighted incidence matrix and its Markov transition matrices to gain a dynamic perspective on CSCM distributions. This method allows us to observe how these markers spread and influence cancer progression in a dynamical context. By calculating mutual information for each node and hyperedge, our analysis uncovers complex interaction patterns between CSCMs and organs, highlighting the critical roles of certain markers in cancer progression and metastasis. Our approach offers a detailed representation of cancer stem cell networks, enhancing our understanding of the mechanisms driving cancer heterogeneity and metastasis. By integrating hypergraph theory with cancer biology, this study provides valuable insights for developing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48368f26aa7447550426a623ac42529547196d96" target='_blank'>
              Mapping Cancer Stem Cell Markers Distribution:A Hypergraph Analysis Across Organs
              </a>
            </td>
          <td>
            D. H. Margarit, Gustavo Paccosi, M. V. Reale, Lilia M. Romanelli
          </td>
          <td>2024-07-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite improvements in patient outcomes, pediatric cancer remains a leading cause of non-accidental death in children. Recent genetic analysis of patients with pediatric cancers indicates an important role for both germline genetic predisposition and cancer-specific somatic driver mutations. Increasingly, evidence demonstrates that the developmental timepoint at which the cancer cell-of-origin transforms is critical to tumor identity and therapeutic response. Therefore, future therapeutic development would be bolstered by the use of disease models that faithfully recapitulate the genetic context, cell-of-origin, and developmental window of vulnerability in pediatric cancers. Human stem cells have the potential to incorporate all of these characteristics into a pediatric cancer model, while serving as a platform for rapid genetic and pharmacological testing. In this review, we describe how human stem cells have been used to model pediatric cancers and how these models compare to other pediatric cancer model modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5451f14ab5137f3c1ef0749fac76fd503e0d7a63" target='_blank'>
              Pediatric Tumors as Disorders of Development: The Case for In Vitro Modeling Based on Human Stem Cells
              </a>
            </td>
          <td>
            Cullen Clairmont, Joanna J Gell, Ching C Lau
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251acda7df20c5eff2f9bfbbad717de02554a201" target='_blank'>
              Unlocking the potential of HHLA2: identifying functional immune infiltrating cells in the tumor microenvironment and predicting clinical outcomes in laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenjing Li, Jianqing You, Haixiang Xue, Yi Liu, Junjun Chen, Xiao Zheng, Lujun Chen, Changping Wu
          </td>
          <td>2024-08-06</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) involves a complex interaction between tumor cells and immune cells, notably monocytes, leading to immunosuppression. This study explored these interactions using in vitro coculture systems of THP-1 cells and CRC cell lines, employing quantitative proteomics to analyze protein changes in monocytes. Multiple analytical methods were utilized to delineate the altered proteomic landscape, identify key proteins, and their associated functional pathways for comprehensive data analysis. Differentially expressed proteins (DEPs) were selected and validated by cross-referencing them with publicly available TCGA and GEO data sets to explore their potential clinical significance. Our analysis identified 161 up-regulated and 130 down-regulated DEPs. The enrichment results revealed impairments in adhesion and innate immune functions in monocytes, potentially facilitating cancer progression. The down-regulation of FN1, THSB1, and JUN may contribute to these impairments. Furthermore, the overexpression of ADAMTSL4, PRAM1, GPNMB, and NPC2 on monocytes was associated with unfavorable prognostic outcomes in CRC patients, suggesting potential biomarkers or therapeutic targets. This study illustrated the proteomic landscape of monocytes in response to CRC cells, providing clues for future investigations of the crosstalk between cancer cells and monocytes within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdfc88fbc9cbf99c3dfb122dc89a3992f904a45" target='_blank'>
              The Proteomic Landscape of Monocytes in Response to Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Yiran Wang, Luyao Zhang, Jing Xu, Jie Ma
          </td>
          <td>2024-08-06</td>
          <td>Journal of Proteome Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The immune response in breast tumors has an important role in prognosis, but the role of spatial localization of immune cells and of interaction between subtypes is not well characterized. We evaluated the association between spatially resolved tissue infiltrating immune cells (TIICs) and breast cancer specific survival (BCSS) in a large multicenter study. Patients and methods Tissue microarrays with tumor cores from 17,265 breast cancer patients of European descent were stained for CD8, FOXP3, CD20, and CD163. We developed a machine learning based tissue segmentation and immune cell detection algorithm using Halo to score each image for the percentage of marker positive cells by compartment (overall, stroma, or tumor). We assessed the association between log transformed TIIC scores and BCSS using Cox regression. Results Total CD8+ and CD20+ TIICs (stromal and intra-tumoral) were associated with better BCSS in women with ER-negative (HR per standard deviation = 0.91 [95% CI 0.85 - 0.98] and 0.89 [0.84 - 0.94] respectively) and ER-positive disease (HR = 0.92 [95% CI 0.87 - 0.98] and 0.93 [0.86 - 0.99] respectively) in multi-marker models. In contrast, CD163+ macrophages were associated with better BCSS in ER-negative disease (0.94 [0.87 - 1.00]) and a poorer BCSS in ER-positive disease 1.04 [0.99 - 1.10]. There was no association between FOXP3 and BCSS. The observed associations tended to be stronger for intra-tumoral than stromal compartments for all markers. However, the TIIC markers account for only 7.6 percent of the variation in BCSS explained by the multi-marker fully-adjusted model for ER-negative cases and 3.0 percent for ER-positive cases. Conclusions The presence of intra-tumoral and stromal TIICs is associated with better BCSS in both ER-negative and ER-positive breast cancer. This may have implications for the use of immunotherapy. However, the addition of TIICs to existing prognostic models would only result in a small improvement in model performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be0e809bb3aa75c428cf4b2997435d4686b9b36" target='_blank'>
              Spatially-resolved tumour infiltrating immune cells and prognosis in breast cancer
              </a>
            </td>
          <td>
            Aaron J. Bernstein, R. Keeman, Amber N. Hurson, F. Blows, M. Bolla, Jodi L. Miller, R. Milne, Hugo Horlings, A. J. Broek, Clara Bodelon, James M. Hodge, A. Patel, Lauren R. Teras, F. Canzian, Rudolf Kaaks, Hermann Brenner, B. Schöttker, S. Behrens, J. Chang-Claude, T. Maurer, Nadia Obi, F. Couch, H. R. Ali, C. Caldas, Irene Andrulis, G. Glendon, A. Mulligan, Wilma Mesker, Agnes Jager, A. Heemskerk-Gerritsen, P. Devilee, Scott Lawrence, J. Lissowska, Karun Mutreja, Thomas U Ahearn, S. Chanock, M. Duggan, D. Eccles, J. L. Jones, Will Tapper, A. Hollestelle, M. Hooning, John W. M. Martens, Carolien H. M. Deurzen, A. Cox, Simon S. Cross, Mikael Hartman, Jingmei Li, T. Putti, U. Hamann, M. Rashid, A. Jakubowska, Nicki Camp, Melissa H. Cessna, Amy Berrington de Gonzalez, Katarzyna Białkowska, J. Gronwald, J. Lubiński, Siddhartha Yadav, Pietro Lio, D. Easton, M. Abubakar, M. García-Closas, P. Pharoah, M. Schmidt
          </td>
          <td>2024-07-22</td>
          <td>None</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Mixed invasive ductal and lobular carcinoma (MDLC) is a rare histologic subtype of breast cancer displaying both E-cadherin positive ductal and E-cadherin negative lobular morphologies within the same tumor, posing challenges with regard to anticipated clinical management. It remains unclear whether these distinct morphologies also have distinct biology and risk of recurrence. Our spatially resolved transcriptomic, genomic, and single-cell profiling revealed clinically significant differences between ductal and lobular tumor regions including distinct intrinsic subtype heterogeneity - e.g., MDLC with triple-negative breast cancer (TNBC) or basal ductal and estrogen receptor positive (ER+) luminal lobular regions, distinct enrichment of cell cycle arrest/senescence and oncogenic (ER and MYC) signatures, genetic and epigenetic CDH1 inactivation in lobular but not ductal regions, and single-cell ductal and lobular subpopulations with unique oncogenic signatures further highlighting intraregional heterogeneity. Altogether, we demonstrated that the intratumoral morphological/histological heterogeneity within MDLC is underpinned by intrinsic subtype and oncogenic heterogeneity which may result in prognostic uncertainty and therapeutic dilemma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b946691678dc96aa33837697b37dc03e3a68321d" target='_blank'>
              Spatial molecular profiling of mixed invasive ductal and lobular breast cancers reveals heterogeneity in intrinsic molecular subtypes, oncogenic signatures, and mutations.
              </a>
            </td>
          <td>
            O. Shah, A. Nasrazadani, J. Foldi, J. Atkinson, Celina G. Kleer, Priscilla F McAuliffe, Tyler J Johnston, Wayne Stallaert, E. D. da Silva, P. Selenica, Higinio Dopeso, F. Pareja, D. Mandelker, B. Weigelt, Jorge S Reis-Filho, Rohit Bhargava, Peter C. Lucas, Adrian V. Lee, S. Oesterreich
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7a41ef98c35752ab8b215566e394e307d31a58" target='_blank'>
              PHF12 regulates HDAC1 to promote tumorigenesis via EGFR/AKT signaling pathway in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yiru Kong, Rongrong Jiang, Hui Zhou, M. Ge, Hao Lin, Yu Wang, Rongrong Yao, Qing Wang, Xiaohua Liang, Jing Li, Xinli Zhou
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The WDR5/MLL1-H3K4me3 epigenetic axis is often activated in both tumor cells and tumor-infiltrating immune cells to drive various cellular responses in the tumor microenvironment and has been extensively studied in hematopoietic cancer, but its respective functions in tumor cells and immune cells in the context of tumor growth regulation of solid tumor is still incompletely understood. We report here that WDR5 exhibits a higher expression level in human pancreatic tumor tissues compared with adjacent normal pancreas. Moreover, WDR5 expression is negatively correlated with patients’ response to chemotherapy or immunotherapy in human colon cancer and melanoma. However, WDR5 expression is positively correlated with the HLA level in human cancer cells, and H3K4me3 enrichment is observed at the promoter region of the HLA-A, HLA-B, and HLA-C genes in pancreatic cancer cells. Using mouse tumor cell lines and in vivo tumor models, we determined that WDR5 deficiency or inhibition significantly represses MHC I expression in vitro and in vivo in pancreatic tumor cells. Mechanistically, we determine that WDR5 deficiency inhibits H3K4me3 deposition at the MHC I (H2K) promoter region to repress MHC I (H2K) transcription. On the other hand, WDR5 depletion leads to the effective downregulation of immune checkpoints and immunosuppressive cytokines, including TGFβ and IL6, in the pancreatic tumor microenvironments. Our data determine that WDR5 not only regulates tumor cell immunogenicity to suppress tumor growth but also activates immune suppressive pathways to promote tumor immune evasion. Selective activation of the WDR5-MHC I pathway and/or selective inhibition of the WDR5–immune checkpoint and WDR5–cytokine pathways should be considered in WDR5-based epigenetic cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855836379cadca724defacc71dcd89c3fbbdd74b" target='_blank'>
              The Wdr5-H3K4me3 Epigenetic Axis Regulates Pancreatic Tumor Immunogenicity and Immune Suppression
              </a>
            </td>
          <td>
            Kaidi Deng, Liyan Liang, Yingcui Yang, Yanmin Wu, Yan Li, Rongrong Zhang, Yulin Tian, Chunwan Lu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd226a7cedc5fac02cecfc40140a6f89256a6e49" target='_blank'>
              Dynamics of clonal hematopoiesis and risk of hematologic malignancy.
              </a>
            </td>
          <td>
            C. M. Arends, S. Jaiswal
          </td>
          <td>2024-08-08</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7061e254307959081cb1b4acbe6296469e9930ab" target='_blank'>
              High throughput spatial immune mapping reveals an innate immune scar in post-COVID-19 brains
              </a>
            </td>
          <td>
            Marius Schwabenland, Dilara Hasavci, Sibylle Frase, K. Wolf, Nikolaus Deigendesch, Joerg M. Buescher, Kirsten D Mertz, B. Ondruschka, H. Altmeppen, Jakob Matschke, Markus Glatzel, S. Frank, R. Thimme, Juergen Beck, Jonas A. Hosp, Thomas Blank, B. Bengsch, M. Prinz
          </td>
          <td>2024-07-25</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most common primary malignant brain tumor. The aim of this study was to elucidate the role of microenvironment and intrinsic T-type calcium channels (Cav3) in regulating tumor growth and progression. Methods We grafted syngeneic GBM cells into Cav3.2 knockout mice to assess the role of microenvironment T-Type calcium channels on GBM tumor growth. We performed single-cell RNA-seq (scRNA-seq) of tumors from WT and Cav3.2 KO mice to elucidate the regulation of tumors by the microenvironment. We used neurons from WT and Cav3.2 KO mice in co-culture with GBM stem cells (GSC) to assess the effects of Cav3.2 on neuron/GSC synaptic connections and tumor cell growth. Results Cav3.2 KO in the microenvironment led to significant reduction of GBM growth and prolongation of animal survival. scRNA-seq showed that microenvironment Cav3.2 regulates neuronal and glial biological processes. Microenvironment Cav3.2 downregulated numerous genes associated with regulating the OPC cell state in GBM tumors such as SOX10 and Olig2. Neuronal Cav3.2 promoted neuron/GSC synaptic connections and GSC growth. Treatment of GSCs with the Cav3 blocker mibefradil downregulated genes associated with neuronal processes. The Cav3 blocker drug mibefradil synergized with temozolomide (TMZ) and radiation to reduce in vivo tumor growth and prolong animal survival. Conclusions Together these data reveal a role for microenvironment Cav3 in promoting GBM tumor progression through regulating neuronal and glial processes particularly associated with the OPC-cell state. Targeting both intrinsic and microenvironment Cav3 with the inhibitor mibefradil significantly enhanced the anti-GBM effects of TMZ and radiation. Key Points Microenvironment Cav3.2 promotes GBM progression Microenvironment Cav3.2 promotes neuronal and glial processes Pharmacological targeting of intrinsic and microenvironment Cav3 synergizes with TMZ/radiation Importance of the Study In this study, we demonstrate for the first time that microenvironment Cav3.2 contributes to GBM progression and growth by regulating neuronal and glial processes. Our findings highlight the importance of T-type calcium channels in the microenvironment as well as the tumor and provides preclinically relevant data for the use of mibefradil to inhibit GBM growth in combination with standard of care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303b17198abd5c192529012a365daa1747ef06e0" target='_blank'>
              Microenvironment T-Type calcium channels regulate neuronal and glial processes to promote glioblastoma growth
              </a>
            </td>
          <td>
            Collin J. Dube, Ying Zhang, Shekhar Saha, Michelle Lai, Myron K. Gibert, Miguel Escalante, Kadie Hudson, Doris Wong, Pawel Marcinkiewicz, Ulas Yener, Yunan Sun, Esther Xu, Aditya Sorot, Elizabeth Mulcahy, B. Kefas, Farina Hanif, Fadilla Guessous, Ashley Vernon, Manoj K. Patel, David Schiff, Hui Zong, Benjamin Purow, Eric Holland, Swapnil Sonkusare, Harald Sontheimer, R. Abounader
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common malignant primary brain tumor. GBM has an extremely poor prognosis and new treatments are badly needed. Efficient drug delivery to GBM is a major obstacle as the blood-brain barrier (BBB) prevents passage of the majority of cancer drugs into the brain. It is also recognized that the blood-brain tumor barrier (BTB) in the growing tumor represents a challenge. The BTB is heterogeneous and poorly characterized, but similar to the BBB it can prevent therapeutics from reaching effective intra-tumoral doses, dramatically hindering their potential. Here, we identified a 12-gene signature associated with the BTB, with functions related to vasculature development, morphogenesis and cell migration. We identified CDH5 as a core molecule in this set and confirmed its over-expression in GBM vasculature using spatial transcriptomics of GBM patient specimens. We found that the indirubin-derivative, 6-bromoindirubin acetoxime (BIA), could downregulate CDH5 and other BTB signature genes, causing endothelial barrier disruption in endothelial monolayers and BBB 3D spheroids in vitro. Treatment of tumor-bearing mice with BIA enabled increased intra-tumoral accumulation of the BBB non-penetrant chemotherapeutic drug cisplatin and potentiated cisplatin-mediated DNA damage by targeting DNA repair pathways. Finally, using an injectable BIA nanoparticle formulation, PPRX-1701, we significantly improved the efficacy of cisplatin in patient-derived GBM xenografts and prolonged their survival. Overall, our work reveals potential targets at the BTB for improved chemotherapy delivery and the bifunctional properties of BIA as a BTB modulator and potentiator of chemotherapy, supporting its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca2d6e3b8b9f2b3209e769ca054b665691b61645" target='_blank'>
              Modulation of blood-tumor barrier transcriptional programs improves intra-tumoral drug delivery and potentiates chemotherapy in GBM
              </a>
            </td>
          <td>
            J. Jimenez-Macias, Philippa Vaughn-Beaucaire, Ayush Bharati, Zheyun Xu, Megan Forrest, Jason Hong, Michael Sun, Andrea Schmidt, Jasmine S. Clark, William Hawkins, Noe Mercado, Jacqueline Real, Kelsey Huntington, M. Zdioruk, Michal O. Nowicki, C. Cho, Bin Wu, Weiyi Li, Theresa Logan, K. Manz, K. Pennell, B. Fedeles, Alexander S. Brodsky, Sean Lawler
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf3da76b81a975c5e4f8d3102cc9399708ed2b0" target='_blank'>
              Mathematical modeling and quantitative analysis of phenotypic plasticity during tumor evolution based on single-cell data.
              </a>
            </td>
          <td>
            Yuyang Xiao, Xiufen Zou
          </td>
          <td>2024-08-20</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9965a9579beaaf870afee3260a8af2cf6b7ea8" target='_blank'>
              IL-1R signaling drives enteric glia-macrophage interactions in colorectal cancer
              </a>
            </td>
          <td>
            L. van Baarle, V. De Simone, Linda Schneider, Sneha Santhosh, S. Abdurahiman, F. Biscu, Reiner Schneider, Lisa Zanoletti, Renata Siqueira de Mello, Sara Verbandt, Zedong Hu, M. Stakenborg, B. Ke, N. Stakenborg, Raquel Salvador Laureano, Balbina García-Reyes, Jonas Henn, Marieta Toma, M. Vanmechelen, G. Boeckxstaens, F. De Smet, A. Garg, S. Ibiza, Sabine Tejpar, Sven Wehner, G. Matteoli
          </td>
          <td>2024-07-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial in cancer development and therapeutic response. Immunotherapy is increasingly recognized as a critical component of cancer treatment. While immunotherapies have shown efficacy in various cancers, including breast cancer, patient responses vary widely. Some patients receive significant benefits, while others experience minimal or no improvement. This disparity underscores the complexity and diversity of the immune system. In this study, we investigated the immune landscape and cell–cell communication within the TME of breast cancer through integrated analysis of bulk and single-cell RNA sequencing data. We established profiles of tumor immune infiltration that span across a broad spectrum of adaptive and innate immune cells. Our clustering analysis of immune infiltration identified three distinct patient groups: high T cell abundance, moderate infiltration, and low infiltration. Patients with low immune infiltration exhibited the poorest survival rates, while those in the moderate infiltration group showed better outcomes than those with high T cell abundance. Moreover, the high cell abundance group was associated with a greater tumor burden and higher rates of TP53 mutations, whereas the moderate infiltration group was characterized by a lower tumor burden and elevated PIK3CA mutations. Analysis of an independent single-cell RNA-seq breast cancer dataset confirmed the presence of similar infiltration patterns. Further investigation into ligand–receptor interactions within the TME unveiled significant variations in cell–cell communication patterns among these groups. Notably, we found that the signaling pathways SPP1 and EGF were exclusively active in the low immune infiltration group, suggesting their involvement in immune suppression. This work comprehensively characterizes the composition and dynamic interplay in the breast cancer TME. Our findings reveal associations between the extent of immune infiltration and clinical outcomes, providing valuable prognostic information for patient stratification. The unique mutations and signaling pathways associated with different patient groups offer insights into the mechanisms underlying diverse tumor immune infiltration and the formation of an immunosuppressive tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff73b13094e5afc913b4266529d0100e73db7ee" target='_blank'>
              Characterizing the Tumor Microenvironment and Its Prognostic Impact in Breast Cancer
              </a>
            </td>
          <td>
            Wenjuan Zhang, Alex Lee, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2024-09-10</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Recent advances in spatial transcriptomics (ST) enable measurements of transcriptome within intact biological tissues by preserving spatial information, offering biologists unprecedented opportunities to comprehensively understand tissue micro-environment, where spatial domains are basic units of tissues. Although great efforts are devoted to this issue, they still have many shortcomings, such as ignoring local information and relations of spatial domains, requiring alternatives to solve these problems. Here, a novel algorithm for spatial domain identification in Spatial Transcriptomics data with Structure Correlation and Self-Representation (ST-SCSR), which integrates local information, global information, and similarity of spatial domains. Specifically, ST-SCSR utilzes matrix tri-factorization to simultaneously decompose expression profiles and spatial network of spots, where expressional and spatial features of spots are fused via the shared factor matrix that interpreted as similarity of spatial domains. Furthermore, ST-SCSR learns affinity graph of spots by manipulating expressional and spatial features, where local preservation and sparse constraints are employed, thereby enhancing the quality of graph. The experimental results demonstrate that ST-SCSR not only outperforms state-of-the-art algorithms in terms of accuracy, but also identifies many potential interesting patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a785908fa19d1b8473d1ec73348fbe37f91e24e" target='_blank'>
              ST-SCSR: identifying spatial domains in spatial transcriptomics data via structure correlation and self-representation
              </a>
            </td>
          <td>
            Min Zhang, Wensheng Zhang, Xiaoke Ma
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95736f8e150d70ccc63e3057497b476be28ab1d9" target='_blank'>
              CCKBR+ cancer cells contribute to the intratumor heterogeneity of gastric cancer and confer sensitivity to FOXO inhibition.
              </a>
            </td>
          <td>
            Zhenya Tan, Ke Pan, Minqiong Sun, Xianzhu Pan, Zhi Yang, Zhiling Chang, Xue Yang, Jicheng Zhu, Li Zhan, Yakun Liu, Xiaofei Li, Keqiong Lin, Lin Chen, Hui Mo, Wei Luo, Chen Kan, Lunxi Duan, Hong Zheng
          </td>
          <td>2024-08-20</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Renal cell carcinoma (RCC) is the most prevalent type of malignant kidney tumor in adults, with clear cell renal cell carcinoma (ccRCC) comprising about 75% of all cases. The SETD2 gene, which is involved in the modification of histone proteins, is often found to have alterations in ccRCC. Yet, our understanding of how these SETD2 mutations affect ccRCC characteristics and behavior within the tumor microenvironment is still not fully understood. Methods We conducted a detailed analysis of single-cell RNA sequencing (scRNA-seq) data from ccRCC. First, the data was preprocessed using the Python package, “scanpy.” High variability genes were pinpointed through Pearson’s correlation coefficient. Dimensionality reduction and clustering identification were performed using Principal Component Analysis (PCA) and the Leiden algorithm. Malignant cell identification was conducted with the “InferCNV” R package, while cell trajectories and intercellular communication were depicted using the Python packages “VIA” and “cellphoneDB.” We then employed the R package “Deseq2” to determine differentially expressed genes (DEGs) between groups. Using high-dimensional weighted gene correlation network analysis (hdWGCNA), co-expression modules were identified. We intersected these modules with DEGs to establish prognostic models through univariate Cox and the least absolute shrinkage and selection operator (LASSO) method. Results We identified 69 and 53 distinctive cell clusters, respectively. These were classified further into 12 unique cell types. This analysis highlighted the presence of an abnormal tumor sub-cluster (MT + group), identified by high mitochondrial-encoded protein gene expression and an indication of unfavorable prognosis. Investigation of cellular interactions spotlighted significant interactions between the MT + group and endothelial cells, macrophaes. In addition, we developed a prognostic model based on six characteristic genes. Notably, risk scores derived from these genes correlated significantly with various clinical features. Finally, a nomogram model was established to facilitate more accurate outcome prediction, incorporating four independent risk factors. Conclusion Our findings provide insight into the crucial transcriptomic characteristics of ccRCC associated with SETD2 mutation. We discovered that this mutation-induced subcluster could stimulate M2 polarization in macrophages, suggesting a heightened propensity for metastasis. Moreover, our prognostic model demonstrated effectiveness in forecasting overall survival for ccRCC patients, thus presenting a valuable clinical tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838967df027ef703d03c6fe47cf35883f45c841" target='_blank'>
              Revealing the characteristics of SETD2-mutated clear cell renal cell carcinoma through tumor heterogeneity analysis
              </a>
            </td>
          <td>
            Shansen Peng, Zhouzhou Xie, Huiming Jiang, Guihao Zhang, Nanhui Chen
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3fe31e6aa7a99673a4d6d194c90f42b74a1aa1" target='_blank'>
              Immunogenomic features of radiologically distinctive nodules in multiple primary lung cancer
              </a>
            </td>
          <td>
            Mei-Cheng Chen, Haolong Yang, Zhi Dong, Lu-Jie Li, Xiang-Min Li, Hong-He Luo, Qiong Li, Ying Zhu
          </td>
          <td>2024-09-05</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Glioblastoma is a highly aggressive brain cancer and, unlike many other cancers types, the median survival for patients after treatment (14.6 months) has barely improved in the last 20 years. Infiltrative growth into the surrounding brain parenchyma facilitates tumor recurrence and ultimately the death of the patient - novel therapies targeting this process are desperately needed. Lysyl oxidase inhibition has been shown to decrease invasive growth in a variety of solid tumours and is a potential therapy for glioblastoma patients. Methods Genes highly expressed in the mesenchymal subtype of glioblastoma were analyzed in a data set from the Cancer Genome Atlas and tissue microarrays. Two patient-derived human glioblastoma stem cell lines were used to assess the involvement of lysyl oxidase (LOX). The effect of LOX on infiltration was examined in an organotypic brain slice assay and in an orthotopic mouse model. Chemotactic assays, protease and cleavage arrays were used to assess the underlying mechanism behind LOX-mediated infiltration. The orthotopic model was used to evaluate potential clinical utility of targeting LOX in glioblastoma. Results LOX is overexpressed in the mesenchymal glioblastoma subtype and strongly associated with poor patient survival. LOX expression upregulates MMP7 expression, which subsequently cleaves the vascular matrix resulting in increased chemotaxis of glioblastoma cells. Conclusions We have uncovered a novel mechanism of glioblastoma infiltration and suggest that targeting LOX represent an effective therapeutic approach blocking glioblastoma infiltration. Importance of the study The ability of glioblastoma cells to infiltrate the surrounding normal brain tissue facilitates their evasion of current therapies, leading to tumor recurrence and ultimately the death of the patient. To improve targeted therapies for glioblastoma patients we need to understand the molecular mechanisms of glioblastoma cell infiltration and how cells interact with the unique microenvironment of the brain. We have identified a novel mechanism whereby tumor-derived LOX mediates chemotaxis of glioblastoma cells to the laminin rich perivascular niche, enabling infiltrative growth. Inhibiting this infiltrative pathway is a potential anti-invasive therapy that is desperately needed for glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb3e6c468837c0b0bfb72d57ecf46fe10ba327b" target='_blank'>
              Cleavage of the vascular matrix attracts glioblastoma cells to infiltrate the brain parenchyma
              </a>
            </td>
          <td>
            Lara Perryman, Anette M. Høye, T. Cox, Ann-Marie Baker, Jan E. Strøbech, Lidia Leonte, Lukram B Singh, Sergey Popov, Lasse G Lorentzen, Raphael Reuten, Hans Skovgaard Poulsen, Michael J Davies, Chris Jones, J. Erler
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3d4f61a2577f3255d5a96a1f62094418b1629b" target='_blank'>
              A cell state-specific metabolic vulnerability to GPX4-dependent ferroptosis in glioblastoma.
              </a>
            </td>
          <td>
            Matei A. Banu, Athanassios Dovas, Michael G. Argenziano, Wenting Zhao, Colin P. Sperring, Henar Cuervo Grajal, Zhouzerui Liu, Dominique Mo Higgins, Misha Amini, B. Pereira, Ling F Ye, A. Mahajan, N. Humala, J. Furnari, P. Upadhyayula, Fereshteh Zandkarimi, Trang Tt Nguyen, Damian E Teasley, Peter B Wu, Li Hai, Charles Karan, Tyrone Dowdy, Aida Razavilar, M. Siegelin, Jan Kitajewski, Mioara Larion, Jeffrey N Bruce, Brent R. Stockwell, Peter A. Sims, Peter D Canoll
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="235 Background: Fibrosarcoma (FS) is a relatively uncommon subtype of soft tissue sarcoma, which predominantly occurs in the extremities and girdles of elderly individuals. FS are famous for its high recurrence and metastasis rate. However, the current treatment regimens have shown poor efficacy in treating these progressive lesions. Therefore, a deep understanding of the tumor microenvironment characteristics of fibrosarcoma is needed to develop more effective drugs.Therefore, a deeper understanding of the tumor microenvironment characteristics of fibrosarcoma is needed for further development of therapeutic drugs. Methods: We performed single-cell transcriptome sequencing for the first time on 6 patients bearing with FS. Bulk-seq data of soft tissue sarcoma was downloaded from public database including TCGA and GEO for prognostic analysis. What' s more, a cohort of 54 FS patients was retrospectively collected for protein detection. Results: The tumor microenvironment landscape was characterized by the presence of malignant fibroblast, diverse macrophages, limited T-cell presence, and high levels of vascular infiltration. The expression of siglec15 was significantly upregulated in the malignant cells. And the prognostic significance of siglec15 was confirmed by the TCGA public database. What’s more, the high expression of siglec15 protein was negatively related to overall survival of fibrosarcoma patients from our institution. We also explored the interactions among different cell subtypes, especially the interactions between malignant cells and endothelial cells, as well as macrophages and endothelial cells, which not only promote tumor progression but also inhibit T-cell infiltration and function. Conclusions: In total, our work has uncovered the low T cell infiltration landscape of FS. The primary reason of insufficient immune infiltration may be due to the blockage of T cell extravasation caused by the interaction between endothelial cells, malignant cells and macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca424f7bf6e590c667e5a83854870c663bc0d1f2" target='_blank'>
              The single cell transcriptional landscape of fibrosarcoma.
              </a>
            </td>
          <td>
            Juan Wang, Zhenfeng Li, Ning Zhang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-dimensional single-cell data poses significant challenges in identifying underlying biological patterns due to the complexity and heterogeneity of cellular states. We propose a comprehensive gene-cell dependency visualization via unsupervised clustering, Growing Hierarchical Self-Organizing Map (GHSOM), specifically designed for analyzing high-dimensional single-cell data like single-cell sequencing and CRISPR screens. GHSOM is applied to cluster samples in a hierarchical structure such that the self-growth structure of clusters satisfies the required variations between and within. We propose a novel Significant Attributes Identification Algorithm to identify features that distinguish clusters. This algorithm pinpoints attributes with minimal variation within a cluster but substantial variation between clusters. These key attributes can then be used for targeted data retrieval and downstream analysis. Furthermore, we present two innovative visualization tools: Cluster Feature Map and Cluster Distribution Map. The Cluster Feature Map highlights the distribution of specific features across the hierarchical structure of GHSOM clusters. This allows for rapid visual assessment of cluster uniqueness based on chosen features. The Cluster Distribution Map depicts leaf clusters as circles on the GHSOM grid, with circle size reflecting cluster data size and color customizable to visualize features like cell type or other attributes. We apply our analysis to three single-cell datasets and one CRISPR dataset (cell-gene database) and evaluate clustering methods with internal and external CH and ARI scores. GHSOM performs well, being the best performer in internal evaluation (CH=4.2). In external evaluation, GHSOM has the third-best performance of all methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/140c0a81733fa80520acd13c32e97603d9f10e5c" target='_blank'>
              scGHSOM: Hierarchical clustering and visualization of single-cell and CRISPR data using growing hierarchical SOM
              </a>
            </td>
          <td>
            Shang-Jung Wen, Jia-Ming Chang, Fang Yu
          </td>
          <td>2024-07-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Waddington’s epigenetic landscape has long served as a conceptual framework for understanding cell fate decisions. The landscape’s geometry encodes the molecular mechanisms that guide the gene expression profiles of uncommitted cells toward terminally differentiated cell types. In this study, we demonstrate that applying the concept of intrinsic dimension to single-cell transcriptomic data can effectively capture trends in expression trajectories, supporting this framework. This approach allows us to define a robust cell potency score without relying on prior biological information. By analyzing an extensive collection of datasets from various species, experimental protocols, and differentiation processes, we validate our method and successfully reproduce established hierarchies of cell type potency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae38cc5f2491f9a800929bdb3d454ebd00a828eb" target='_blank'>
              The intrinsic dimension of gene expression during cell differentiation
              </a>
            </td>
          <td>
            Marta Biondo, Niccolò Cirone, Filippo Valle, Silvia Lazzardi, M. Caselle, Matteo Osella
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Patient-derived tumor organoids (PDTOs) are novel cellular models that maintain the genetic, phenotypic and structural features of patient tumor tissue and are useful for studying tumorigenesis and drug response. When integrated with advanced 3D imaging and analysis techniques, PDTOs can be used to establish physiologically relevant high-throughput and high-content drug screening platforms that support the development of patient-specific treatment strategies. However, in order to effectively leverage high-throughput PDTO observations for clinical predictions, it is critical to establish a quantitative understanding of the basic properties and variability of organoid growth dynamics. In this work, we introduced an innovative workflow for analyzing and understanding PDTO growth dynamics, by integrating a high-throughput imaging deep learning platform with mathematical modeling, incorporating flexible growth laws and variable dormancy times. We applied the workflow to colon cancer organoids and demonstrated that organoid growth is well-described by the Gompertz model of growth. Our analysis showed significant intrapatient heterogeneity in PDTO growth dynamics, with the initial exponential growth rate of an organoid following a lognormal distribution within each dataset. The level of intrapatient heterogeneity varied between patients, as did organoid growth rates and dormancy times of single seeded cells. Our work contributes to an emerging understanding of the basic growth characteristics of PDTOs, and it highlights the heterogeneity in organoid growth both within and between patients. These results pave the way for further modeling efforts aimed at predicting treatment response dynamics and drug resistance timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d868d7a68c9f23a73010e77ac074ffad6193b786" target='_blank'>
              Understanding patient-derived tumor organoid growth through an integrated imaging and mathematical modeling framework
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Seung-Jin Kim, Brandon Choi, J. K. Schmid, Karn Kaura, H. Lenz, S. Mumenthaler, Jasmine Foo
          </td>
          <td>2024-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880d6fbe977b9fa0abf41b1330834beac1e2a6aa" target='_blank'>
              A patient-derived cell model for malignant transformation in IDH-mutant glioma
              </a>
            </td>
          <td>
            Olga Kim, Zach Sergi, Guangyang Yu, Kazu Yamamoto, Martha Quezado, Zied Abdullaev, Danel R Crooks, Shun Kishimoto, Qi Li, Peng Lu, Burchelle N. Blackman, T. Andresson, Xiaolin Wu, Bao Tran, Jun S. Wei, Wei Zhang, Meili Zhang, Hua Song, Javed Khan, Murali C. Krishna, J. Brender, Jing Wu
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background FOXR2-activated central nervous system (CNS) neuroblastoma (CNS NB-FOXR2) is a recently identified subtype of brain tumor characterized by the elevated expression of the transcription factor FOXR2 mainly due to genomic rearrangements. However, the precise pathogenic mechanisms, including the cell type of origin, remain elusive. Methods A gene expression analysis of patient tumors was performed to identify putative cell types of origin. Based on this prediction, a new human embryonic stem cell–based model was developed to validate the origin and to examine the molecular and cellular mechanisms underlying the formation of CNS NB-FOXR2. Results Our data showed that CNS NB-FOXR2 tumors express a high level of lineage marker genes associated with the medial ganglionic eminence (MGE), a transient structure located in the developing ventral forebrain. Our model confirmed the cell-type-specific effect of FOXR2 on the proliferation and in vivo tumorigenicity. Additionally, we found that FOXR2 overexpression activated the MEK/ERK signaling pathway through a suppression of the endogenous RAS inhibitor DIRAS3. The MEK inhibitor trametinib suppressed the proliferation of FOXR2-expressing MGE progenitors more than nonexpressing cells. Conclusions Our study collectively demonstrates that MGE progenitors are the cell of origin of CNS NB-FOXR2 and that FOXR2 activates the MEK/ERK signaling pathway, providing a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8445a3e44a2783d48d762e8340d244130bab0b2f" target='_blank'>
              A human embryonic stem cell–based model reveals the cell of origin of FOXR2-activated CNS neuroblastoma
              </a>
            </td>
          <td>
            Hitomi N Royston, Autumn B Hampton, Dhruv Bhagat, Evonne F Pinto, Miriam D Emerson, Kosuke Funato
          </td>
          <td>2024-08-12</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3471e25fd3d93db955069ff8713da95ffc83f905" target='_blank'>
              Single-cell multi-omic and spatial profiling of human kidneys implicates the fibrotic microenvironment in kidney disease progression.
              </a>
            </td>
          <td>
            Amin Abedini, Jonathan Levinsohn, Konstantin A. Klötzer, Bernhard Dumoulin, Ziyuan Ma, Julia Frederick, Poonam Dhillon, Michael S. Balzer, Rojesh Shrestha, Hongbo Liu, Steven Vitale, Andi M. Bergeson, Kishor B. Devalaraja-Narashimha, Paola Grandi, Tanmoy Bhattacharyya, Erding Hu, Steven S. Pullen, Carine M. Boustany-Kari, Paolo Guarnieri, Anil Karihaloo, D. Traum, Hanying Yan, Kyle Coleman, Matthew Palmer, Lea Sarov-Blat, L. Morton, Christopher A. Hunter, Klaus H. Kaestner, Mingyao Li, K. Suszták
          </td>
          <td>2024-07-24</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Macrophages consist of a heterogeneous population of functionally distinct cells that participate in many physiological and pathological processes. They exhibit prominent plasticity by changing their different functional phenotypes represented by proinflammatory (M1) and anti-inflammatory (M2) in response to different environmental stimuli. Emerging evidence illustrates the importance of intracellular metabolic pathways in macrophage polarizations and functions. In the tumor microenvironment (TME), macrophages tend to M2 polarization, which promotes tumor growth and leads to adverse physiological effects. Due to the lack of highly specific antigens in M1 and M2 macrophages, significant challenges present in isolating these subtypes from clinical samples or in vitro coculture models of tumor-immune cells. In reverse, the single-cell technique provides the possibility to investigate the factors influencing macrophage polarization in the TME. In this research, we employed inertial microfluidic chip-mass spectrometry (IMC-MS) to conduct single-cell metabolomics analysis of macrophages polarized into the two major phenotypes, respectively, and 213 metabolites were identified in total. Subsequently, differential metabolites between macrophage phenotypes were analyzed using volcano plots and binary logistic regression models. Glutamine was pinpointed as a key metabolite for the M1 and M2 phenotypes. Experimental results from both monoculture and coculture cell models demonstrated that M1 polarization is more reliant on the presence of glutamine in the culture environment than M2 polarization. Glutamine deficiency resulted in failed M1 polarization, while its absence had a less pronounced effect on M2 polarization. Replenishing an appropriate amount of glutamine during the intermediate stages of coculture models significantly enhanced the proportion of M1 polarization and suppressed the growth of tumor cells. This research elucidated glutamine as a key factor influencing macrophage polarization in the TME via single-cell metabolomics based on IMC-MS, offering promising insights and targets for tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c30acba424dcd59708fd1e1a9d323eb89042a76" target='_blank'>
              Polarization of Macrophages in Tumor Microenvironment Using High-Throughput Single-Cell Metabolomics.
              </a>
            </td>
          <td>
            Xuesen Hu, Xinlin Liu, Disheng Feng, Tianrun Xu, Hang Li, Chunxiu Hu, Zhizhou Wang, Xinyu Liu, Peiyuan Yin, Xianzhe Shi, Dong Shang, Guowang Xu
          </td>
          <td>2024-09-02</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Vulvar diseases are a critical yet often neglected area of women’s health, profoundly affecting patients’ quality of life and frequently resulting in long-term physical and psychological challenges. Lichen sclerosus (LS) is a chronic inflammatory skin disorder that predominantly affects the vulva, leading to severe itching, pain, scarring, and an increased risk of malignancy. Despite its profound impact on affected individuals, the molecular pathogenesis of vulvar LS (VLS) is not well understood, hindering the development of FDA-approved therapies. Here, we utilize single-cell and spatial transcriptomics to analyze lesional and non-lesional skin from VLS patients, as well as healthy control vulvar skin. Our findings demonstrate histologic, cellular, and molecular heterogeneities within VLS, yet highlight unifying molecular changes across keratinocytes, fibroblasts, immune cells, and melanocytes in lesional skin. They reveal cellular stress and damage in fibroblasts and keratinocytes, enhanced T cell activation and cytotoxicity, aberrant cell-cell signaling, and increased activation of the IFN, JAK/STAT, and p53 pathways in specific cell types. Using both monolayer and organotypic culture models, we also demonstrate that knockdown of select genes, which are downregulated in VLS lesional keratinocytes, partially recapitulates VLS-like stress-associated changes. Collectively, these data provide novel insights into the pathogenesis of VLS, identifying potential biomarkers and therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a1ede19b990ffc821eea644860eccc4ba0949" target='_blank'>
              Single-cell and spatial transcriptomics of vulvar lichen sclerosus reveal multi-compartmental alterations in gene expression and signaling cross-talk
              </a>
            </td>
          <td>
            Peng Sun, Christina N. Kraus, Wei Zhao, Jiahui Xu, Susie Suh, Quy Nguyen, Yunlong Jia, Arjun Nair, Melanie Oakes, Roberto Tinoco, Jessica Shiu, Bryan Sun, Ashley Elsensohn, Scott X. Atwood, Qing Nie, Xing Dai
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="SF3B1 is the most recurrently mutated RNA splicing gene in cancer; However, the study of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, we compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV HDR editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5Max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. We further refined our approach by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia (CLL) cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes including altered splicing, copy number variations, and cell growth defect. Moreover, we uncovered that SF3B1 mutation can cause the loss of Y chromosome in CLL. Our results showcase PRECIS as an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering oncogenic hotspot mutations on SF3B1 via CRISPR-directed PRECIS mutagenesis.
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Systematic investigation of tumor-infiltrating immune (TII) cells is important to the development of immunotherapies, and the clinical response prediction in cancers. There exists complex transcriptional regulation within TII cells, and different immune cell types display specific regulation patterns. To dissect transcriptional regulation in TII cells, we first integrated the gene expression profiles from single-cell datasets, and proposed a computational pipeline to identify TII cell type-specific transcription factor (TF) mediated activity immune modules (TF-AIMs). Our analysis revealed key TFs, such as BACH2 and NFKB1 play important roles in B and NK cells, respectively. We also found some of these TF-AIMs may contribute to tumor pathogenesis. Based on TII cell type-specific TF-AIMs, we identified eight CD8+ T cell subtypes. In particular, we found the PD1 + CD8+ T cell subset and its specific TF-AIMs associated with immunotherapy response. Furthermore, the TII cell type-specific TF-AIMs displayed the potential to be used as predictive markers for immunotherapy response of cancer patients. At the pan-cancer level, we also identified and characterized six molecular subtypes across 9680 samples based on the activation status of TII cell type-specific TF-AIMs. Finally, we constructed a user-friendly web interface CellTF-AIMs (http://bio-bigdata.hrbmu.edu.cn/CellTF-AIMs/) for exploring transcriptional regulatory pattern in various TII cell types. Our study provides valuable implications and a rich resource for understanding the mechanisms involved in cancer microenvironment and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b9ff76c2393752409d847895c8cb3295294da" target='_blank'>
              Identifying cell type-specific transcription factor-mediated activity immune modules reveal implications for immunotherapy and molecular classification of pan-cancer
              </a>
            </td>
          <td>
            Feng Li, Jingwen Wang, Mengyue Li, Xiaomeng Zhang, Yongjuan Tang, Xinyu Song, Yifang Zhang, Liying Pei, Jiaqi Liu, Chunlong Zhang, Xia Li, Yanjun Xu, Yunpeng Zhang
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Aim The main focus of this study is to explore the molecular mechanism of IRF7 regulation on RPS18 transcription in M1‐type macrophages in pancreatic adenocarcinoma (PAAD) tissue, as well as the transfer of RPS18 by IRF7 via exosomes to PAAD cells and the regulation of ILF3 expression. Methods By utilising single‐cell RNA sequencing (scRNA‐seq) data and spatial transcriptomics (ST) data from the Gene Expression Omnibus database, we identified distinct cell types with significant expression differences in PAAD tissue. Among these cell types, we identified those closely associated with lipid metabolism. The differentially expressed genes within these cell types were analysed, and target genes relevant to prognosis were identified. Flow cytometry was employed to assess the expression levels of target genes in M1 and M2 macrophages. Cell lines with target gene knockout were constructed using CRISPR/Cas9 editing technology, and cell lines with target gene knockdown and overexpression were established using lentiviral vectors. Additionally, a co‐culture model of exosomes derived from M1 macrophages with PAAD cells was developed. The impact of M1 macrophage‐derived exosomes on the lipid metabolism of PAAD cells in the model was evaluated through metabolomics analysis. The effects of M1 macrophage‐derived exosomes on the viability, proliferation, division, migration and apoptosis of PAAD cells were assessed using MTT assay, flow cytometry, EdU assay, wound healing assay, Transwell assay and TUNEL staining. Furthermore, a mouse PAAD orthotopic implantation model was established, and bioluminescence imaging was utilised to assess the influence of M1 macrophage‐derived exosomes on the intratumoural formation capacity of PAAD cells, as well as measuring tumour weight and volume. The expression of proliferation‐associated proteins in tumour tissues was examined using immunohistochemistry. Results Through combined analysis of scRNA‐seq and ST technologies, we discovered a close association between M1 macrophages in PAAD samples and lipid metabolism signals, as well as a negative correlation between M1 macrophages and cancer cells. The construction of a prognostic risk score model identified RPS18 and IRF7 as two prognostically relevant genes in M1 macrophages, exhibiting negative and positive correlations, respectively. Mechanistically, it was found that IRF7 in M1 macrophages can inhibit the transcription of RPS18, reducing the transfer of RPS18 to PAAD cells via exosomes, consequently affecting the expression of ILF3 in PAAD cells. IRF7/RPS18 in M1 macrophages can also suppress lipid metabolism, cell viability, proliferation, migration, invasion and intratumoural formation capacity of PAAD cells, while promoting cell apoptosis. Conclusion Overexpression of IRF7 in M1 macrophages may inhibit RPS18 transcription, reduce the transfer of RPS18 from M1 macrophage‐derived exosomes to PAAD cells, thereby suppressing ILF3 expression in PAAD cells, inhibiting the lipid metabolism pathway, and curtailing the viability, proliferation, migration, invasion of PAAD cells, as well as enhancing cell apoptosis, ultimately inhibiting tumour formation in PAAD cells in vivo. Targeting IRF7/RPS18 in M1 macrophages could represent a promising immunotherapeutic approach for PAAD in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32285e4f384085d30139b0635486bb160ba3f7ac" target='_blank'>
              Single‐cell sequencing combined with spatial transcriptomics reveals that the IRF7 gene in M1 macrophages inhibits the occurrence of pancreatic cancer by regulating lipid metabolism‐related mechanisms
              </a>
            </td>
          <td>
            Ting Zhan, Yanli Zou, Zheng Han, Xia Tian, Mengge Chen, Jiaxi Liu, Xiulin Yang, Qingxi Zhu, Meng Liu, Wei Chen, Mingtao Chen, Xiaodong Huang, Jie Tan, Weijie Liu, Xia Tian
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Immunotherapy represents a groundbreaking and monumental achievement in the field of cancer therapy, marking a significant advancement in fighting against this devastating disease. Lung cancer has showed consistent clinical improvements in response to immunotherapy treatments, yet, it is undeniable that challenges such as limited response rates acquire resistance, and the unclear fundamental mechanisms were inevitable problems.


METHODS
The cellular composition was defined and distinguished through single-cell RNA sequencing (scRNA-seq) analysis of MPR (major pathologic response) and NMPR (non-major pathologic response) samples in GSE207422, including four primary MPR samples and eight primary NMPR samples.


RESULTS
We found obvious difference in CD8+ T cell population between MPR and NMPR samples, with high expression of TYMS, RRM2, and BIRC5 in NPMR samples. Meanwhile, the proportion of macrophages and tumor epithelial cells infiltration increased in the NMPR samples. We discovered biomarkers (ACTN4, ATF3, BRD2, CDKN1A, and CHMP4B) in epithelial cells which were potentially represented worse outcomes.


CONCLUSIONS
By exploring the difference of tumor microenvironment (TME) in samples with different corresponding degrees of neoadjuvant immunotherapy, this research introduces a number of novel biomarkers for predicting the response of treatment and a theoretical basis for overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eb58f4e74cf2a065722c81b24475ac3132ac67a" target='_blank'>
              Single-cell RNA sequencing reveals microenvironmental infiltration in non-small cell lung cancer with different responses to immunotherapy.
              </a>
            </td>
          <td>
            Xinnan Hu, Yonghui Wu, Lixin Wang, Fujun Yang, Lingyun Ye, Xiaoxia Chen, Xiao Song, Ping Wei
          </td>
          <td>2024-09-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="High-grade gliomas are a major health challenge with poor prognosis and high morbidity. Immune-checkpoint inhibitors (ICI) have emerged as promising therapeutic options for several malignancies yet show little efficacy against central nervous system (CNS) tumors. CD200 is a newly recognized immune checkpoint that modulates immune homeostasis. CD200 protein is expressed by a variety of cells, including immune cells and stromal cells, and is overexpressed by many tumors. The shedding of CD200 from tumor cells can create an immunosuppressive environment that dampens anti-tumor immunity by modulating cytolytic activity and cytokine expression both within and outside the tumor microenvironment (TME). While it is well-accepted that CD200 induces a pro-tumorigenic environment through its ability to suppress the immune response, we sought to determine the role of glioma-specific expression of CD200. We show that CD200 is expressed across glioma types, is shed from tumor cells, and increases over time in the serum of patients undergoing immunotherapy. Using CD200 knockout (KO) glioma models, we demonstrated that glioma cell-derived CD200 promotes tumor growth in vivo and in vitro. Notably, CD200 KO gliomas are spontaneously rejected by their host, a process that required a fully functional immune system, including NK and T-cells. Moreover, we report that glioma-derived or brain-injected soluble CD200 contributes to the suppression of antigen-specific CD8 T-cells in the draining lymph nodes (dLNs). Our work provides new mechanistic insights regarding CD200-mediated immunosuppression by gliomas. Statement of significance We demonstrate mechanisms of the druggable glioma-derived CD200 checkpoint on tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b634d44ec918f6bd798f77ff4dc7cd21cc67c2e" target='_blank'>
              CD200 depletion in glioma enhances antitumor immunity and induces tumor rejection
              </a>
            </td>
          <td>
            Itay Raphael, Anzar A. Mujeeb, Elisabet Ampudia-Mesias, ReidAnn E. Sever, Brandon L. McClellan, Stephen C. Frederico, Chaim T. Sneiderman, Apoorva Mirji, Ali Daba, Francisco Puerta-Martínez, Michal Nisnboym, W. Edwards, Michael W. Graner, Christopher L. Moertel, , Gary Kohanbash, Michael Olin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Recent advances in single cell sequencing have led to an increased focus on the role of cell-type composition in phenotypic presentation and disease progression. Cell-type composition research in the heart is challenging due to large, frequently multinucleated cardiomyocytes that preclude most single cell approaches from obtaining accurate measurements of cell composition. Our in silico studies reveal that ignoring cell type composition when calculating differentially expressed genes (DEGs) can have significant consequences. For example, a relatively small change in cell abundance of only 10% can result in over 25% of DEGs being false positives. Methods We have implemented an algorithmic approach that uses snRNAseq datasets as a reference to accurately calculate cell type compositions from bulk RNAseq datasets through robust data cleaning, gene selection, and multi-sample cross-subject and cross-cell-type deconvolution. We applied our approach to cardiomyocyte-specific α1A adrenergic receptor (CM-α1A-AR) knockout mice. 8-12 week-old mice (either WT or CM-α1A-KO) were subjected to permanent left coronary artery (LCA) ligation or sham surgery (n=4 per group). Transcriptomes from the infarct border zones were collected 3 days later and analyzed using our algorithm to determine cell-type abundances, corrected differential expression calculations using DESeq2, and validated these findings using RNAscope. Results Uncorrected DEGs for the CM-α1A-KO X LCA interaction term featured many cell-type specific genes such as Timp4 (fibroblasts) and Aplnr (cardiomyocytes) and overall GO enrichment for terms pertaining to cardiomyocyte differentiation (P=3.1E-4). Using our algorithm, we observe a striking loss of cardiomyocytes and gain in fibroblasts in the α1A-KO + LCA mice that was not recapitulated in WT + LCA animals, although we did observe a similar increase in macrophage abundance in both conditions. This recapitulates prior results that showed a much more severe heart failure phenotype in CM-α1A-KO + LCA mice. Following correction for cell-type, our DEGs now highlight a novel set of genes enriched for GO terms such as cardiac contraction (P=3.7E-5) and actin filament organization (P=6.3E-5). Conclusions Our algorithm identifies and corrects for cell-type abundance in bulk RNAseq datasets opening new avenues for research on novel genes and pathways as well as an improved understanding of the role of cardiac cell types in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217ee4d32d0eb92d12b53e00c08af7cc8fb059d2" target='_blank'>
              Novel Insights into Post-Myocardial Infarction Cardiac Remodeling through Algorithmic Detection of Cell-Type Composition Shifts
              </a>
            </td>
          <td>
            Brian Gural, Logan Kirkland, Abbey Hockett, Peyton Sandroni, Jiandong Zhang, M. Rosa-Garrido, Samantha K. Swift, Douglas J. Chapski, Michael A. Flinn, Caitlin C. O’Meara, T. Vondriska, Michaela Patterson, Brian C. Jensen, Christoph D Rau
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Yvette Hayat, Sarah Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, Emmanuelle Huillard, Nina Pottier, Aurore Desmons, Inès Fayache, Garrett A. Kaas, Philip J. Kingsley, Lawrence J. Marnett, Eric Duplus, Elias El-Habr, Lucas Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, Michele Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, Mirjam E Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cellular quiescence is a state of growth arrest or slowed proliferation that is described in normal and cancer stem cells (CSCs). Quiescence may protect CSCs from antiproliferative chemotherapy drugs. In T-cell acute lymphoblastic leukemia (T-ALL) patient-derived xenograft (PDX) mouse models, quiescent cells are associated with treatment resistance and stemness. Cell proliferation dyes are popular tools for the tracking of cell division. The fluorescent dye is covalently anchored into amine groups on the membrane and macromolecules inside the cell. This allows for the tracking of labeled cells for up to 10 divisions, which can be resolved by flow cytometry. Ultimately, cells with the highest proliferation rates will have low dye retention, as it will be diluted with each cell division, while dormant, slower-dividing cells will have the highest retention. The use of cell proliferation dyes to isolate dormant cells has been optimized and described in T-ALL mouse models. Complementary to the existing mouse models, the rag2:Myc-derived zebrafish T-ALL model provides an excellent venue to interrogate self-renewal in T-ALL due to the high frequency of leukemic stem cells (LSCs) and the convenience of zebrafish for large-scale transplant experiments. Here, we describe the workflow for the staining of zebrafish T-ALL cells with a cell proliferation dye, optimizing the concentration of the dye for zebrafish cells, passaging successfully stained cells in vivo, and the collection of cells with varying levels of dye retention by live cell sorting from transplanted animals. Given the absence of well-established cell surface makers for LSCs in T-ALL, this approach provides a functional means to interrogate quiescent cells in vivo. For representative results, we describe the engraftment efficiency and the LSC frequency of high and low dye-retaining cells. This method can help investigate additional properties of quiescent cells, including drug response, transcriptional profiles, and morphology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/056ac498044a7776dae6a3b947c4abe205cbe4c6" target='_blank'>
              Identification of Quiescent Cells in a Zebrafish T-Cell Acute Lymphoblastic Leukemia Model Using Cell Proliferation Staining.
              </a>
            </td>
          <td>
            Majd A. Al-Hamaly, Yelena Chernyavskaya, Jessica S. Blackburn
          </td>
          <td>2024-07-19</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Most patients with prostate adenocarcinoma develop resistance to therapies targeting the androgen receptor (AR). Consequently, a portion of these patients develop AR-independent neuroendocrine prostate cancer (NEPC), a rapidly progressing cancer with limited therapies and poor survival outcomes. Current research to understand the progression to NEPC suggests a model of lineage plasticity whereby AR-dependent luminal-like tumors progress towards an AR-independent NEPC state. Genetic analysis of human NEPC identified frequent loss of RB1 and TP53, and the loss of both genes in experimental models mediates the transition to a neuroendocrine lineage. Transcriptomics studies have shown that lineage transcription factors ASCL1 and NEUROD1 are present in NEPC. In this study, we modeled the progression of prostate adenocarcinoma to NEPC by establishing prostate organoids and subsequently generating subcutaneous allograft tumors from genetically-engineered mouse models harboring Cre-induced loss of Rb1 and Trp53 with Myc overexpression (RPM). These tumors were heterogeneous and displayed adenocarcinoma, squamous, and neuroendocrine features. ASCL1 and NEUROD1 were expressed within neuroendocrine-defined regions, with ASCL1 being predominant. Genetic loss of Ascl1 in this model did not decrease tumor incidence, growth, or metastasis; however, there was a notable decrease in neuroendocrine identity and an increase in basal-like identity. This study provides an in vivo model to study progression to NEPC and establishes the requirement for ASCL1 in driving neuroendocrine differentiation in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b06924a4ca0fcd63effebd03ee725e6bf5e8759" target='_blank'>
              Neuroendocrine Differentiation in Prostate Cancer Requires ASCL1.
              </a>
            </td>
          <td>
            Kathia E Rodarte, Shaked Nir Heyman, Lei Guo, Lydia Flores, T. Savage, Juan Villarreal, Su Deng, Lin Xu, Rajal B Shah, T. Oliver, Jane E Johnson
          </td>
          <td>2024-09-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The ultraviolet (UV) radiation triggers a pigmentation response in human skin, wherein, melanocytes rapidly activate divergent maturation and proliferation programs. Using single-cell sequencing, we demonstrate that these 2 programs are segregated in distinct subpopulations in melanocytes of human and zebrafish skin. The coexistence of these 2 cell states in cultured melanocytes suggests possible cell autonomy. Luria–Delbrück fluctuation test reveals that the initial establishment of these states is stochastic. Tracking of pigmenting cells ascertains that the stochastically acquired state is faithfully propagated in the progeny. A systemic approach combining single-cell multi-omics (RNA+ATAC) coupled to enhancer mapping with H3K27 acetylation successfully identified state-specific transcriptional networks. This comprehensive analysis led to the construction of a gene regulatory network (GRN) that under the influence of noise, establishes a bistable system of pigmentation and proliferation at the population level. This GRN recapitulates melanocyte behaviour in response to external cues that reinforce either of the states. Our work highlights that inherent stochasticity within melanocytes establishes dedicated states, and the mature state is sustained by selective enhancers mark through histone acetylation. While the initial cue triggers a proliferation response, the continued signal activates and maintains the pigmenting subpopulation via epigenetic imprinting. Thereby our study provides the basis of coexistence of distinct populations which ensures effective pigmentation response while preserving the self-renewal capacity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f2df7cd0fbd20ec135a875780520ab2994a139" target='_blank'>
              Distinct melanocyte subpopulations defined by stochastic expression of proliferation or maturation programs enable a rapid and sustainable pigmentation response
              </a>
            </td>
          <td>
            Ayush Aggarwal, Ayesha Nasreen, Babita Sharma, Sarthak Sahoo, Keerthic Aswin, Mohammed Faruq, Rajesh Pandey, M. Jolly, Abhyudai Singh, Rajesh S Gokhale, Vivek T. Natarajan
          </td>
          <td>2024-08-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Curcumin is widely recognized for its diverse antitumor properties, ranging from breast cancer to many other types of cancers. However, its role in the tumor microenvironment remains to be elucidated. In this study, we established a 3D tumor spheroids model that can simulate the growth environment of tumor cells and visualized the antitumor metabolic alteration caused by curcumin using mass spectrometry imaging technology. Our results showed that curcumin not only exerts a profound impact on the growth and proliferation of breast cancer cells but in situ multivariate statistical analysis also reveals the significant effect on the overall metabolic profile of tumor spheroids. Meanwhile, our visualization map characterized curcumin metabolic processes of reduction and glucuronidation in tumor spheroids. More importantly, abnormal metabolic pathways related to lipid metabolism and polyamine metabolism were also remodeled at the metabolite and gene levels after curcumin intervention. These insights deepen our comprehension of the regulatory mechanism of curcumin on the tumor metabolic network, furnishing powerful references for antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9708e466bdcc061f5dd608fcbfa98f940da2834e" target='_blank'>
              Spatial Metabolomics Profiling Reveals Curcumin Induces Metabolic Reprogramming in Three-Dimensional Tumor Spheroids
              </a>
            </td>
          <td>
            Zihan Zhu, Yaqi Zhang, Lei Wang, Haoyuan Geng, Min Li, Shiping Chen, Xiao Wang, Panpan Chen, Chenglong Sun, Chao Zhang
          </td>
          <td>2024-09-02</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, Philip Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b7af48216a0c84346ea77c18dc911a1e93afbf" target='_blank'>
              Endothelial cell heterogeneity in colorectal cancer: tip cells drive angiogenesis
              </a>
            </td>
          <td>
            Zhenyu Xie, Liaoran Niu, Kunli Du, Ling Chen, Gaozan Zheng, Songchen Dai, Han-Long Dan, Lili Duan, Xinyu Dou, Fan Feng, Jian Zhang, Jianyong Zheng
          </td>
          <td>2024-08-22</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="High-throughput sequencing of single-cell data can be used to rigorously evlauate cell specification and enable intricate variations between groups or conditions. Many popular existing methods for differential expression target differences in aggregate measurements (mean, median, sum) and limit their approaches to detect only global differential changes. We present a robust method for differential expression of single-cell data using a kernel-based score test, cytoKernel. cytoKernel is specifically designed to assess the differential expression of single cell RNA sequencing and high-dimensional flow or mass cytometry data using the full probability distribution pattern. cytoKernel is based on kernel embeddings which employs the probability distributions of the single cell data, by calculating the pairwise divergence/distance between distributions of subjects. It can detect both patterns involving aggregate changes, as well as more elusive variations that are often overlooked due to the multimodal characteristics of single cell data. We performed extensive benchmarks across both simulated and real data sets from mass cytometry data and single-cell RNA sequencing. The cytoKernel procedure effectively controls the False Discovery Rate (FDR) and shows favourable performance compared to existing methods. The method is able to identify more differential patterns than existing approaches. We apply cytoKernel to assess gene expression and protein marker expression differences from cell subpopulations in various publicly available single-cell RNAseq and mass cytometry data sets. The methods described in this paper are implemented in the open-source R package cytoKernel, which is freely available from Bioconductor at http://bioconductor.org/packages/cytoKernel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5266fa2edd1942a8b424629b41d58db895ce6a73" target='_blank'>
              cytoKernel: Robust kernel embeddings for assessing differential expression of single cell data
              </a>
            </td>
          <td>
            Tusharkanti Ghosh, Ryan M. Baxter, Souvik Seal, Victor G Lui, Pratyadipta Rudra, Thao Vu, Elena W Y Hsieh, Debashis Ghosh
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small RNAs that are often dysregulated in many diseases, including cancers. They are highly tissue-specific and stable, thus, making them particularly useful as biomarkers. As the spatial transcriptomics field advances, protocols that enable highly sensitive and spatially resolved detection become necessary to maximize the information gained from samples. This is especially true of miRNAs where the location their expression within tissue can provide prognostic value with regard to patient outcome. Equally as important as detection are ways to assess and visualize the miRNA's spatial information in order to leverage the power of spatial transcriptomics over that of traditional nonspatial bulk assays. We present a highly sensitive methodology that simultaneously quantitates and spatially detects seven miRNAs in situ on formalin-fixed paraffin-embedded tissue sections. This method utilizes rolling circle amplification (RCA) in conjunction with a dual scanning approach in nanoliter well arrays with embedded hydrogel posts. The hydrogel posts are functionalized with DNA probes that enable the detection of miRNAs across a large dynamic range (4 orders of magnitude) and a limit of detection of 0.17 zeptomoles (1.7 × 10-4 attomoles). We applied our methodology coupled with a data analysis pipeline to K14-Cre Brca1f/fTp53f/f murine breast tumors to showcase the information gained from this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbaad333d7c14fb2504c61497f5a898eaa1ab830" target='_blank'>
              Sensitive Multiplexed MicroRNA Spatial Profiling and Data Classification Framework Applied to Murine Breast Tumors.
              </a>
            </td>
          <td>
            Omar N Mohd, Y. Heng, Lin Wang, Abhishek Thavamani, Erica S. Massicott, Gerburg M Wulf, Frank J Slack, Patrick S. Doyle
          </td>
          <td>2024-07-23</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394c20d5ef3bbb71528bea2f3875ee00b262a370" target='_blank'>
              Glioblastoma cells increase expression of notch signaling and synaptic genes within infiltrated brain tissue.
              </a>
            </td>
          <td>
            D. Harwood, V. Pedersen, N. S. Bager, A. Schmidt, T. O. Stannius, A. Areškevičiūtė, Knud Josefsen, D. S. Nørøxe, David Scheie, Hannah E. Rostalski, Maya Jeje Schuang Lü, A. Locallo, Ulrik Lassen, F. O. Bagger, J. Weischenfeldt, D. H. Heiland, Kristoffer Vitting-Seerup, S. Michaelsen, BW Kristensen
          </td>
          <td>2024-09-09</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6640869137efbf815e49406335626c0344fac69" target='_blank'>
              Age-associated clonal B cells drive B cell lymphoma in mice.
              </a>
            </td>
          <td>
            José P Castro, A. Shindyapina, Alessandro Barbieri, Kejun Ying, Olga S. Strelkova, Joao A. Paulo, A. Tyshkovskiy, Rico Meinl, C. Kerepesi, Anna P. Petrashen, Marco Mariotti, Margarita V Meer, Yan Hu, Alexander Karamyshev, Grigoriy Losyev, Mafalda Galhardo, Elsa Logarinho, Artur A. Indzhykulian, Steven P. Gygi, J. M. Sedivy, John P Manis, V. N. Gladyshev
          </td>
          <td>2024-08-08</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Motivation The combination of single-cell sequencing with ligand-receptor analysis paves the way for the characterization of cell communication events in complex tissues. In particular, directed weighted graphs stand out as a natural representation of cell-cell communication events. However, current computational methods cannot analyze sample-specific cell-cell communication events, as measured in single-cell data produced in large patient cohorts. Cohort-based cell-cell communication analysis presents many challenges, such as the non-linear nature of cell-cell communication and the high variability presented by the patient-specific single-cell RNAseq datasets. Results Here, we present scACCorDiON (single-cell Analysis of Cell-Cell Communication in Disease clusters using Optimal transport in Directed Networks), an optimal transport algorithm exploring node distances on the Markov Chain as the ground metric between directed weighted graphs. Additionally, we derive a k-barycenter algorithm using the Wasserstein-based distance, which is able to cluster directed weighted graphs. We compare our approach with competing methods in several large cohorts of scRNA-seq data. Our results show that scACCorDiON can predict clusters better, matching the disease status of samples. Moreover, we show that barycenters provide a robust and explainable representation of cell cell communication events related to the detected clusters. We also provide a case study of pancreas adenocarcinoma, where scACCorDion detects a sub-cluster of disease samples associated with changes in the tumor microenvironment. Availability The code of scACCorDiON is available at https://scaccordion.readthedocs.io/en/latest Contact ivan.costa@rwth-aachen.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12408229b87ae8ceac3d6b2906ea4bd12adc6c6e" target='_blank'>
              scACCorDiON: A clustering approach for explainable patient level cell cell communication graph analysis
              </a>
            </td>
          <td>
            J. Nagai, Michael T. Schaub, Ivan G.Costa
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b983ff02239e55a5a088e11ea44463f2a191dc" target='_blank'>
              Spatial computational modelling illuminates the role of the tumour microenvironment for treating glioblastoma with immunotherapies
              </a>
            </td>
          <td>
            Blanche Mongeon, Julien Hébert-Doutreloux, Anudeep Surendran, Elham Karimi, B. Fiset, D. Quail, Logan A Walsh, A. Jenner, Morgan Craig
          </td>
          <td>2024-08-18</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Prostate cancer remains the most commonly diagnosed and the second leading cause of cancer-related deaths in men in the United States. The neoplastic transformation of prostate epithelia, concomitant with modulations in the stromal compartment, known as reactive stromal response, is critical for the growth, development, and progression of prostate cancer. Reactive stroma typifies an emergent response to disrupted tissue homeostasis commonly observed in wound repair and pathological conditions such as cancer. Despite the significance of reactive stroma in prostate cancer pathobiology, our understanding of the ontogeny, phenotypic and functional heterogeneity, and reactive stromal regulation of the immune microenvironment in prostate cancer remains limited. Traditionally characterized to have an immunologically "cold" tumor microenvironment, prostate cancer presents significant challenges for advancing immunotherapy compared to other solid tumors. This review explores the detrimental role of reactive stroma in prostate cancer, particularly its immunomodulatory function. Understanding the molecular characteristics and dynamic transcriptional program of the reactive stromal populations in tandem with tumor progression could offer insights into enhancing immunotherapy efficacy against prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd12cec3359f8ec93eb34750a8c77daa465b3613" target='_blank'>
              The reactive stroma response regulates the immune landscape in prostate cancer
              </a>
            </td>
          <td>
            Rintu Thomas, John Michael Jerome, Kimiko L. Krieger, Naghmana Ashraf, David R. Rowley
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf603a0e9cde601ac9006c72d111f317f71de81" target='_blank'>
              Single-cell profiling reveals altered immune landscape and impaired NK cell function in gastric cancer liver metastasis.
              </a>
            </td>
          <td>
            Xiaolong Tang, Lei Gao, Xingzhi Jiang, Zhenyu Hou, Yiwen Wang, Shiyang Hou, Hui Qu
          </td>
          <td>2024-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e0df7d22a45444a437c740c4db84a4c490aa49" target='_blank'>
              Membrane RRM2-positive cells represent a malignant population with cancer stem cell features in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yongzhi Zhao, Shuting Xue, Danduo Wei, Jianjuan Zhang, Nachuan Zhang, Liping Mao, Niya Liu, Lei Zhao, Jianing Yan, Yifan Wang, Xiujun Cai, Saiyong Zhu, Stephanie Roessler, Junfang Ji
          </td>
          <td>2024-09-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Simple Summary Cutaneous squamous cell carcinoma (cSCC) is a complex disease arising from an interplay of UV-induced DNA damage, genetic mutations, and alterations in the tumor microenvironment. While most cSCC cases can be treated with surgery, metastatic and locally advanced forms have a significantly worse prognosis and limited treatment options. The high mutational burden in cSCC makes it challenging to identify true driver alterations. Beyond genetics, the tumor microenvironment and its interaction with the immune system play a critical role in cSCC progression, evidenced by differences observed between immunocompromised and immunocompetent patients. In recent years, new therapeutic approaches have emerged, including immunotherapy and EGFR inhibitors, which have shown efficacy in treating advanced cSCC. Understanding the underlying mechanisms driving cSCC is crucial for developing novel and more effective therapies. This review aims to provide a comprehensive overview of the mutational landscape and microenvironmental factors in cSCC. Abstract Cutaneous squamous cell carcinoma (cSCC) manifests through the complex interactions of UV-induced DNA damage, genetic mutations, and alterations in the tumor microenvironment. A high mutational burden is present in cSCC, as well as both cSCC precursors and normal skin, making driver genes difficult to differentiate. Despite this, several key driver genes have been identified, including TP53, the NOTCH family, CDKN2A, PIK3CA, and EGFR. In addition to mutations, the tumor microenvironment and the manipulation and evasion of the immune system play a critical role in cSCC progression. Novel therapeutic approaches, such as immunotherapy and EGFR inhibitors, have been used to target these dysregulations, and have shown promise in treating advanced cSCC cases, emphasizing the need for targeted interventions considering both genetic and microenvironmental factors for improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064418ffcbe98e414731e1ea32740ff9d65441f2" target='_blank'>
              The Mutational and Microenvironmental Landscape of Cutaneous Squamous Cell Carcinoma: A Review
              </a>
            </td>
          <td>
            Tara M. Hosseini, Soo J. Park, Theresa W Guo
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, M. Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16df671c98a37a9e428d4261d351be144c2e34b" target='_blank'>
              Monitoring imatinib decreasing pericyte coverage and HIF-1α level in a colorectal cancer model by an ultrahigh-field multiparametric MRI approach
              </a>
            </td>
          <td>
            Xinpeng Hu, Kunlin Ye, Shaowei Bo, Zeyu Xiao, Mengjie Ma, Jinghua Pan, Xing Zhong, Dong Zhang, Xukai Mo, Xiaojun Yu, Minfeng Chen, L. Luo, Changzheng Shi
          </td>
          <td>2024-07-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d1e5e2f8fff698fa2e7aaaa422dd95deef960f" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis identifies a prognostic T-cell signature in colorectal cancer
              </a>
            </td>
          <td>
            Peng Cui, Haibo Wang, Zhigang Bai
          </td>
          <td>2024-08-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the intricate landscape of the tumor microenvironment, both cancer and stromal cells undergo rapid metabolic adaptations to support their growth. Given the relevant role of the metabolic secretome in fueling tumor progression, its unique metabolic characteristics have gained prominence as potential biomarkers and therapeutic targets. As a result, rapid and accurate tools have been developed to track metabolic changes in the tumor microenvironment with high sensitivity and resolution. Surface-enhanced Raman scattering (SERS) is a highly sensitive analytical technique and has been proven efficient toward the detection of metabolites in biological media. However, profiling secreted metabolites in complex cellular environments such as those in tumor–stroma 3D in vitro models remains challenging. To address this limitation, we employed a SERS-based strategy to investigate the metabolic secretome of pancreatic tumor models within 3D cultures. We aimed to monitor the immunosuppressive potential of stratified pancreatic cancer–stroma spheroids as compared to 3D cultures of either pancreatic cancer cells or cancer-associated fibroblasts, focusing on the metabolic conversion of tryptophan into kynurenine by the IDO-1 enzyme. We additionally sought to elucidate the dynamics of tryptophan consumption in correlation with the size, temporal evolution, and composition of the spheroids, as well as assessing the effects of different drugs targeting the IDO-1 machinery. As a result, we confirm that SERS can be a valuable tool toward the optimization of cancer spheroids, in connection with their tryptophan metabolizing capacity, potentially allowing high-throughput spheroid analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b277ed5165b24ce32a4bbc4ee2fa57b2e7a8af08" target='_blank'>
              Surface-Enhanced Raman Scattering Monitoring of Tryptophan Dynamics in 3D Pancreatic Tumor Models
              </a>
            </td>
          <td>
            Pablo S Valera, Margarida Henriques‐Pereira, Marita Wagner, V. Gaspar, João F. Mano, Luis M. Liz-Marzán
          </td>
          <td>2024-07-19</td>
          <td>ACS Sensors</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Dysregulation of the mitogen-activated protein kinase interacting kinases 1/2 (MNK1/2)–eukaryotic initiation factor 4E (eIF4E) signaling axis promotes breast cancer progression. MNK1 is known to influence cancer stem cells (CSCs); self-renewing populations that support metastasis, recurrence, and chemotherapeutic resistance, making them a clinically relevant target. The precise function of MNK1 in regulating CSCs, however, remains unexplored. Here, we generated MNK1 knockout cancer cell lines, resulting in diminished CSC properties in vitro and slowed tumor growth in vivo. Using a multiomics approach, we functionally demonstrated that loss of MNK1 restricts tumor cell metabolic adaptation by reducing glycolysis and increasing dependence on oxidative phosphorylation. Furthermore, MNK1-null breast and pancreatic tumor cells demonstrated suppressed metastasis to the liver, but not the lung. Analysis of The Cancer Genome Atlas (TCGA) data from breast cancer patients validated the positive correlation between MNK1 and glycolytic enzyme protein expression. This study defines metabolic perturbations as a previously unknown consequence of targeting MNK1/2, which may be therapeutically exploited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd46eff0abc3612dd21dd74ba5f7acbfb0e77403" target='_blank'>
              Blocking tumor-intrinsic MNK1 kinase restricts metabolic adaptation and diminishes liver metastasis
              </a>
            </td>
          <td>
            Samuel E. J. Preston, Michael S. Dahabieh, Raúl Ernesto Flores González, Christophe Gonçalves, Vincent R. Richard, M. Leibovitch, Eleanor Dakin, Theodore Papadopoulos, Carolina Lopez Naranjo, Paige McCallum, F. Huang, N. Gagnon, Stephanie Perrino, R. Zahedi, Christoph H. Borchers, Russell G. Jones, P. Brodt, Wilson H Miller, Sonia V. del Rincón
          </td>
          <td>2024-09-13</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3839aa0db41039893f6c2f9a46bbd7a1efcac73e" target='_blank'>
              Leveraging Cell Migration Dynamics to Discriminate Between Senescent and Presenescent Human Mesenchymal Stem Cells
              </a>
            </td>
          <td>
            Farshad Amiri, P. Mistriotis
          </td>
          <td>2024-07-20</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome editing has demonstrated its utility in generating isogenic cell-based disease models, enabling the precise introduction of genetic alterations into wild-type cells to mimic disease phenotypes and explore underlying mechanisms. However, its application in liver-related diseases has been limited by challenges in genetic modification of mature hepatocytes in a dish. Here, we conducted a systematic comparison of various methods for primary hepatocyte culture and gene delivery to achieve robust genome editing of hepatocytes ex vivo. Our efforts yielded editing efficiencies of up to 80% in primary murine hepatocytes cultured in monolayer and 20% in organoids. To model human hepatic tumorigenesis, we utilized hepatocytes differentiated from human pluripotent stem cells (hPSCs) as an alternative human hepatocyte source. We developed a series of cellular models by introducing various single or combined oncogenic alterations into hPSC-derived hepatocytes. Our findings demonstrated that distinct mutational patterns led to phenotypic variances, affecting both overgrowth and transcriptional profiles. Notably, we discovered that the PI3KCA E542K mutant, whether alone or in combination with exogenous c-MYC, significantly impaired hepatocyte functions and facilitated cancer metabolic reprogramming, highlighting the critical roles of these frequently mutated genes in driving liver neoplasia. In conclusion, our study demonstrates genome-engineered hepatocytes as valuable cellular models of hepatocarcinoma, providing insights into early tumorigenesis mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0098e14922960e96572df20d519a8a83fa8956" target='_blank'>
              Genome Engineering of Primary and Pluripotent Stem Cell-Derived Hepatocytes for Modeling Liver Tumor Formation
              </a>
            </td>
          <td>
            Lulu Zhang, Xunting Wang, Xuelian Yang, Yijia Chi, Yihang Chu, Yi Zhang, Yufan Gong, Fei Wang, Qian Zhao, Dongxin Zhao
          </td>
          <td>2024-09-02</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative apocrine breast carcinoma (TNAC) is a rare type of triple-negative breast cancer (TNBC) characterized by apocrine morphology. Due to its low clinical incidence, TNAC has not been well studied at the molecular level in terms of cellular mechanisms of tumorigenesis and treatment response. In this study, we collected formalin-fixed paraffin-embedded samples from 31 patients diagnosed with invasive TNAC. Mass spectrometry analysis and whole-exome sequencing were performed to systematically construct the proteome and mutation landscape of the TNAC. Comparing ductal carcinoma in situ (DCIS), invasive, and adjacent normal tissues, we observed increased cadherin binding and ligase activity during tumorigenesis, heightened extracellular matrix signaling at tumor initiation, and elevated GTPase activity during tumor progression. Besides, we discovered genes such as C3 and COL18A1 enriched in somatic mutations are also significantly dysregulated in invasive TNAC. Additionally, by assessing biopsies before neo-adjacent chemotherapy, we provided molecular evidence supporting clinical observations of chemotherapy insensitivity in TNAC patients. We further identified a protein panel (CAPN1, CORO1B, HK1, RAP1B, and VPS4B) differentiating TNAC from non-apocrine TNBC, which could potentially serve as diagnostic markers. Taken together, this study represents the first large-scale proteomics analysis of TNAC and may provide guidance on decisions regarding the chemotherapy treatment of TNAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0126eb87e77314e6e97348e0d5a5a4cb7c491972" target='_blank'>
              Proteomics Landscape of Triple-Negative Apocrine Breast Carcinoma Reveals Molecular Mechanisms of Tumorigenesis and Characteristics of Chemo-insensitivity
              </a>
            </td>
          <td>
            Yiying Zhu, M. Long, Wenhao Shi, Tianlong He, Fangzhou Xie, Annan Qian, Yuqiao Liu, Taobo Hu, Shaojun Tang
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Breast cancer stands as the most prevalent cancer diagnosed worldwide, often leading to brain metastasis, a challenging complication characterized by high mortality rates and a grim prognosis. Understanding the intricate mechanisms governing breast cancer brain metastasis (BCBM) remains an ongoing challenge. The unique microenvironment in the brain fosters an ideal setting for the colonization of breast cancer cells. The tumor microenvironment (TME) in brain metastases plays a pivotal role in the initiation and progression of BCBM, shaping the landscape for targeted therapeutic interventions. Current research primarily concentrates on unraveling the complexities of the TME in BCBM, with a particular emphasis on neuroglia and immune cells, such as microglia, monocyte-derived macrophages (MDMs), astrocytes and T cells. This comprehensive review delves deeply into these elements within the TME of BCBM, shedding light on their interplay, mechanisms, and potential as therapeutic targets to combat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f58be8a99bf67c919a1c80a20e8d34360b0f86" target='_blank'>
              Analysis of neuroglia and immune cells in the tumor microenvironment of breast cancer brain metastasis
              </a>
            </td>
          <td>
            Haixin Mo, Xin Zhang, Liangliang Ren
          </td>
          <td>2024-09-05</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Motivation Cancer is driven by somatic mutations that result in the expansion of genomically distinct sub-populations of cells called clones. Identifying the clonal composition of tumours and understanding the evolutionary relationships between clones is a crucial task in cancer genomics. Bulk DNA sequencing is commonly used for studying the clonal composition of tumours, but it is challenging to infer the genetic relationship between different clones due to the mixture of different cell populations. Results In this work, we introduce a new probabilistic model called PhyClone that can infer clonal phylogenies from bulk sequencing data. We demonstrate the performance of PhyClone on simulated and real-world datasets and show that it outperforms previous methods in terms of accuracy and scalability. Availability and implementation Source code is available on Github at: https://github.com/Roth-Lab/PhyClone under the GPL v3.0 license. Contact aroth@bccrc.ca">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2cba6bef4ca455a79d82a594e5b713c434402a" target='_blank'>
              PhyClone: Accurate Bayesian reconstruction of cancer phylogenies from bulk sequencing
              </a>
            </td>
          <td>
            Emilia Hurtado, A. Bouchard-Côté, Andrew Roth
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Intermediate cell carcinoma (Int-CA) is a rare and enigmatic primary liver cancer characterized by uniform tumor cells exhibiting mixed features of both HCC and intrahepatic cholangiocarcinoma. Despite the unique pathological features of int-CA, its molecular characteristics remain unclear yet. Methods: RNA sequencing and whole genome sequencing profiling were performed on int-CA tumors and compared with those of HCC and intrahepatic cholangiocarcinoma. Results: Int-CAs unveiled a distinct and intermediate transcriptomic feature that is strikingly different from both HCC and intrahepatic cholangiocarcinoma. The marked abundance of splicing events leading to intron retention emerged as a signature feature of int-CA, along with a prominent expression of Notch signaling. Further exploration revealed that METTL16 was suppressed within int-CA, showing a DNA copy number–dependent transcriptional deregulation. Notably, experimental investigations confirmed that METTL16 suppression facilitated invasive tumor characteristics through the activation of the Notch signaling cascade. Conclusions: Our results provide a molecular landscape of int-CA featured by METTL16 suppression and frequent intron retention events, which may play pivotal roles in the acquisition of the aggressive phenotype of Int-CA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab7b9d18c0e5e1be8d12c7361eff07994e2e2a" target='_blank'>
              Transcriptomic profiling of intermediate cell carcinoma of the liver
              </a>
            </td>
          <td>
            Byungchan Jang, S. Kwon, Jang Hyun Kim, Jung Mo Kim, Taek Chung, Jeong Eun Yoo, Haeryoung Kim, Julien Calderaro, Hyun Goo Woo, Young Nyun Park
          </td>
          <td>2024-08-01</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Islet transplantation is a promising therapy for diabetes treatment. However, the molecular underpinnings governing the immune response, particularly T-cell dynamics in syngeneic and allogeneic transplant settings, remain poorly understood. Understanding these T cell dynamics is crucial for enhancing graft acceptance and managing diabetes treatment more effectively. This study aimed to elucidate the molecular mechanisms, gene expression differences, biological pathway alterations, and intercellular communication patterns among T-cell subpopulations after syngeneic and allogeneic islet transplantation. Using single-cell RNA sequencing, we analyzed cellular heterogeneity and gene expression profiles using the Seurat package for quality control and dimensionality reduction through t-SNE. Differentially expressed genes (DEGs) were analyzed among different T cell subtypes. GSEA was conducted utilizing the HALLMARK gene sets from MSigDB, while CellChat was used to infer and visualize cell-cell communication networks. Our findings revealed genetic variations within T-cell subpopulations between syngeneic and allogeneic islet transplants. We identified significant DEGs across these conditions, highlighting molecular discrepancies that may underpin rejection or other immune responses. GSEA indicated activation of the interferon-alpha response in memory T cells and suppression in CD4+ helper and γδ T cells, whereas TNFα signaling via NFκB was particularly active in regulatory T cells, γδ T cells, proliferating T cells, and activated CD8+ T cells. CellChat analysis revealed complex communication patterns within T-cell subsets, notably between proliferating T cells and activated CD8+ T cells. In conclusion, our study provides a comprehensive molecular landscape of T-cell diversity in islet transplantation. The insights into specific gene upregulation in xenotransplants suggest potential targets for improving graft tolerance. The differential pathway activation across T-cell subsets underscores their distinct roles in immune responses posttransplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed58b0aa12d9ea27e89edfb2d6365a52a3801ea9" target='_blank'>
              Elucidating T cell dynamics and molecular mechanisms in syngeneic and allogeneic islet transplantation through single-cell RNA sequencing
              </a>
            </td>
          <td>
            Hairong Zhou, Zuhui Pu, Ying Lu, Peilin Zheng, Huizhen Yu, Lisha Mou
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Intercellular communication within the tumor microenvironment (TME) is essential for establishing, mediating, and synchronizing cancer cell invasion and metastasis. Cancer cells, individually and collectively, react at the cellular and molecular levels to insults from standard-of-care treatments used to treat patients with cancer. One form of cell communication that serves as a prime example of cellular phenotypic stress response is a type of cellular protrusion called tunneling nanotubes (TNTs). TNTs are ultrafine, actin-enriched contact-dependent forms of membrane protrusions that facilitate long distance cell communication through transfer of various cargo, including genetic materials, mitochondria, proteins, ions, and various other molecules. In the past 5-10 years, there has been a growing body of evidence that implicates TNTs as a novel mechanism of cell-cell communication in cancer that facilitates and propagates factors that drive or enhance chemotherapeutic resistance in a variety of cancer cell types. Notably, recent literature has highlighted the potential of TNTs to serve as cellular conduits and mediators of drug and nanoparticle delivery. Given that TNTs have also been shown to form in vivo in a variety of tumor types, disrupting TNT communication within the TME provides a novel strategy for enhancing the cytotoxic effect of existing chemotherapies while suppressing this form of cellular stress response. In this review, we examine current understanding of interplay between cancer cells occurring via TNTs, and even further, the implications of TNT-mediated tumor-stromal cross-talk and the potential to enhance chemoresistance. We then examine tumor microtubes, an analogous cell protrusion heavily implicated in mediating treatment resistance in glioblastoma multiforme, and end with a brief discussion of the effects of radiation and other emerging treatment modalities on TNT formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7135a9daa3ebfa1ca82ac734b9d04d57ff12fd6" target='_blank'>
              Do tunneling nanotubes drive chemoresistance in solid tumors and other malignancies?
              </a>
            </td>
          <td>
            A. Sarkari, Emil Lou
          </td>
          <td>2024-07-19</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b21391e9d7b4e36087d981b4ffc2b93179db8ea" target='_blank'>
              Combined microfluidic enrichment and staining workflow for single-cell analysis of circulating tumor cells in metastatic prostate cancer patients
              </a>
            </td>
          <td>
            Caroline Løppke, Amalie M Jørgensen, Nicolai T. Sand, Rasmus B Klitgaard, G. Daugaard, Mette Ø. Agerbæk
          </td>
          <td>2024-07-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim Rashid, Gary L. Johnson, J. J. Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing is a valuable technique for identifying diverse cell subtypes. A key challenge in this process is that the detection of rare cells is often missed by conventional methods due to low abundance and subtle features of these cells. To overcome this, we developed SCLCNF (Local Connectivity Network Feature Sharing in Single-Cell RNA sequencing), a novel approach that identifies rare cells by analyzing features uniquely expressed in these cells. SCLCNF creates a cellular connectivity network, considering how each cell relates to its neighbors. This network helps to pinpoint coexpression patterns unique to rare cells, utilizing a rarity score to confirm their presence. Our method performs better in detecting rare cells than existing techniques, offering enhanced robustness. It has proven to be effective in human gastrula data sets for accurately pinpointing rare cells, and in sepsis data sets where it uncovers previously unidentified rare cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41629b08b97b96e2a5087885fb3403f0009f9b9" target='_blank'>
              Connectivity Network Feature Sharing in Single-Cell RNA Sequencing Data Identifies Rare Cells
              </a>
            </td>
          <td>
            Shudong Wang, Hengxiao Li, Yahui Liu, Shanchen Pang, Sibo Qiao, Jionglong Su, Shaoqiang Wang, Yulin Zhang
          </td>
          <td>2024-08-03</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1b63b1c4f14a801f678ccab9f44f1b0b4a85ea" target='_blank'>
              Tumour-intrinsic endomembrane trafficking by ARF6 shapes an immunosuppressive microenvironment that drives melanomagenesis and response to checkpoint blockade therapy
              </a>
            </td>
          <td>
            Yinshen Wee, Junhua Wang, Emily C. Wilson, Coulson P. Rich, Aaron Rogers, Zongzhong Tong, Evelyn DeGroot, Y. N. Gopal, Michael A Davies, Huseyin Atakan Ekiz, Joshua K H Tay, Chris J. Stubben, Kenneth M. Boucher, Juan M. Oviedo, Keke C. Fairfax, Matthew A. Williams, Sheri L Holmen, Roger K. Wolff, Allie H. Grossmann
          </td>
          <td>2024-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Osteosarcoma is a common type of bone cancer characterized by a poor prognosis due to its metastatic nature. The tumor microenvironment (TME) plays a critical role in tumor metastasis and therapy response. Therefore, our study aims to explore the metastatic mechanism of osteosarcoma, potentially opening new avenues for cancer treatment.In this study, we collected data from the GSE152048, GSE14359, and GSE49003 datasets. Differentially expressed genes (DEGs) were identified in osteosarcoma cases with primary and metastatic features using R software and the limma package. Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses were performed to investigate metastasis-related genes. A protein–protein interaction (PPI) network was established using the STRING database to further analyze these metastasis-associated genes. The abundances of different cell types with a mixed cell population were estimated using the CIBERSORT approach. The scRNA-seq data were analyzed by the Seurat package in R software, and intercellular communications were elucidated using the CellChat R package.In this study, 92 DEGs related to metastasis were identified, including 41 upregulated and 51 downregulated genes in both the GSE14359 and GSE49003 datasets. Metastasis-associated pathways were identified, including those involving the cyclin-dependent protein kinase holoenzyme complex, transferase complex, transferring phosphorus-containing groups, SCF ubiquitin ligase complex, and the serine/threonine protein kinase complex. KEGG and PPI network analyses revealed 15 hub genes, including Skp2, KIF20A, CCNF, TROAP, PHB, CKS1B, MCM3, CCNA2, TRIP13, CENPM, Hsp90AB1, JUN, CKS2, TK1, and KIF4A. Skp2 has been known as an E3 ubiquitin ligase involved in osteosarcoma progression. The proportion of CD8+ T cells was found to be higher in metastatic osteosarcoma tissues, and high expression of PHB was associated with a favorable prognosis in osteosarcoma patients. Additionally, 23 cell clusters were classified into eight cell types, including chondrocytes, MSC, T cells, monocytes, tissue stem cells, neurons, endothelial cells, and macrophages. The 15 hub genes were expressed across various cell types, and interactions between different cell types were observed.Our study reveals the intricate communication between tumor microenvironment components and tumor metastasis in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff33ba2ff2e5acd8821d77a975c86bee16bbf2ba" target='_blank'>
              Single-cell RNA sequencing reveals the communications between tumor microenvironment components and tumor metastasis in osteosarcoma
              </a>
            </td>
          <td>
            Jiatong Li, Yang Bai, He Zhang, Ting Chen, Guanning Shang
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>