<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-02 06:31:02 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Spatial proteomics enable detailed analysis of tissue at single cell resolution. However, creating reliable segmentation masks and assigning accurate cell phenotypes to discrete cellular phenotypes can be challenging. We introduce IMmuneCite, a computational framework for comprehensive image pre-processing and single-cell dataset creation, focused on defining complex immune landscapes when using spatial proteomics platforms. We demonstrate that IMmuneCite facilitates the identification of 32 discrete immune cell phenotypes using data from human liver samples while substantially reducing nonbiological cell clusters arising from co-localization of markers for different cell lineages. We established its versatility and ability to accommodate any antibody panel and different species by applying IMmuneCite to data from murine liver tissue. This approach enabled deep characterization of different functional states in each immune compartment, uncovering key features of the immune microenvironment in clinical liver transplantation and murine hepatocellular carcinoma. In conclusion, we demonstrated that IMmuneCite is a user-friendly, integrated computational platform that facilitates investigation of the immune microenvironment across species, while ensuring the creation of an immune focused, spatially resolved single-cell proteomic dataset to provide high fidelity, biologically relevant analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7edc240d6b41584e3ef1ab2daeefdaa860e615e" target='_blank'>
              IMmuneCite: an integrated workflow for analysis of immune enriched spatial proteomic data
              </a>
            </td>
          <td>
            A. Barbetta, Sarah Bangerth, Jason T C Lee, B. Rocque, Evanthia T Roussos Torres, Rohit Kohli, Omid Akbari, J. Emamaullee
          </td>
          <td>2024-07-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="The multiplexed immunofluorescence (mIF) platform enables biomarker discovery through the simultaneous detection of multiple markers on a single tissue slide, offering detailed insights into intratumor heterogeneity and the tumor-immune microenvironment at spatially resolved single cell resolution. However, current mIF image analyses are labor-intensive, requiring specialized pathology expertise which limits their scalability and clinical application. To address this challenge, we developed CellGate, a deep-learning (DL) computational pipeline that provides streamlined, end-to-end whole-slide mIF image analysis including nuclei detection, cell segmentation, cell classification, and combined immuno-phenotyping across stacked images. The model was trained on over 750,000 single cell images from 34 melanomas in a retrospective cohort of patients using whole tissue sections stained for CD3, CD8, CD68, CK-SOX10, PD-1, PD-L1, and FOXP3 with manual gating and extensive pathology review. When tested on new whole mIF slides, the model demonstrated high precision-recall AUC. Further validation on whole-slide mIF images of 9 primary melanomas from an independent cohort confirmed that CellGate can reproduce expert pathology analysis with high accuracy. We show that spatial immuno-phenotyping results using CellGate provide deep insights into the immune cell topography and differences in T cell functional states and interactions with tumor cells in patients with distinct histopathology and clinical characteristics. This pipeline offers a fully automated and parallelizable computing process with substantially improved consistency for cell type classification across images, potentially enabling high throughput whole-slide mIF tissue image analysis for large-scale clinical and research applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17a151bdbdb300ecceb67002e8773c7471279841" target='_blank'>
              Spatial Immunophenotyping from Whole-Slide Multiplexed Tissue Imaging Using Convolutional Neural Networks
              </a>
            </td>
          <td>
            Mohammad Yosofvand, S. Edmiston, J. Smithy, Xiyu Peng, Caroline E Kostrzewa, Bridget Lin, Fiona D Ehrich, Allison Reiner, Jayson R. Miedema, A. P. Moy, I. Orlow, M. Postow, K. Panageas, V. Seshan, M. K. Callahan, Nancy E. Thomas, Ronglai Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Secondary lymphoid organs (SLOs), including tonsils (TS), lymph nodes (LN), and Peyer's Patches, exhibit complementary immune functions. However, little is known about the spatial organization of immune cells and extracellular matrix (ECM) in the SLOs. Traditional imaging is limited to a few markers, confining our understanding of the differences between the SLOs. Herein, imaging mass cytometry addressed this gap by simultaneously profiling 25-plex proteins in SLO tissues at subcellular resolution. The antibody panel targeted immune, stromal, chemokine, epigenetic, and functional markers. For robust cell identification, a computational workflow SpatialVizPheno was developed to spatially phenotype 999,970 cells using two approaches, including manual gating and semi-supervised gating, iterative clustering, and annotation. LN exhibited the highest density of B cells while the intestinal tissues contained the highest proportion of regulatory and follicular helper T cells. SpatialVizPheno identified the most prevalent interaction between follicular dendritic cells and stromal cells (SCs), plasmablasts/plasma cells, and the SCs across the lymphoid tissues. Collagen-enriched regions were associated with the spatial orientation of B cell follicles in both TS and LN tissues, but not in intestinal lymphoid tissues. Such spatial differences of immunophenotypes and ECM in different SLO tissues can be used to quantify the relationship between cellular organization and ultimate immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d9342b523b0b50e2fb3715a6d362425d2a168c" target='_blank'>
              Spatial immunophenotyping using multiplexed imaging of immune follicles in secondary lymphoid tissues
              </a>
            </td>
          <td>
            Mayar Allam, Thomas Hu, Zhou Fang, Michelle Pi, Ankur Singh, Ahmet F. Coskun
          </td>
          <td>2024-07-22</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, Svenja K. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, Sina Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong M. Nguyen, Alicia-Christina Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Notch signaling can have either an oncogenic or tumor-suppressive function in cancer depending on the cancer type and cellular context. While Notch can be oncogenic in early prostate cancer, we identified significant downregulation of the Notch pathway during prostate cancer progression from adenocarcinoma to neuroendocrine (NE) prostate cancer, where it functions as a tumor suppressor. Activation of Notch in NE and Rb1/Trp53-deficient prostate cancer models led to phenotypic conversion toward a more indolent, non-NE state with glandular features and expression of luminal lineage markers. This was accompanied by upregulation of MHC and type I IFN and immune cell infiltration. Overall, these data support Notch signaling as a suppressor of NE differentiation in advanced prostate cancer and provide insights into how Notch signaling influences lineage plasticity and the tumor microenvironment (TME).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de21afbb6c368d901e1bd4e6360df58ff3e7cece" target='_blank'>
              Notch signaling suppresses neuroendocrine differentiation and alters the immune microenvironment in advanced prostate cancer
              </a>
            </td>
          <td>
            S. Ku, Yanqing Wang, Maria Mica Garcia, Yasutaka Yamada, Kei Mizuno, M. Long, Spencer Rosario, M. Chinnam, Majd Al Assaad, L. Puca, Min Jin Kim, Martin K. Bakht, V. B. Venkadakrishnan, Brian D. Robinson, Andrés M Acosta, K. Wadosky, Juan Miguel Mosquera, David W. Goodrich, H. Beltran
          </td>
          <td>2024-07-18</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Adult-type diffuse gliomas, the most common primary brain tumors, pose significant clinical challenges due to limited treatment options, restricted anti-tumor immune response and dismal patient prognosis. In this study, we elucidate the immunological function and clinical relevance of intra-tumoral tertiary lymphoid structures (TLS) in adaptive anti-glioma immunity. We conducted a comprehensive, unbiased analysis of lymphoid aggregation in 642 gliomas using a multi-modal approach that combines RNA sequencing with spatial transcriptome and proteome profiling. Our findings reveal that TLS are present in 15% of tumors and correlate with improved overall survival. Gliomas with TLS exhibit a remodeled perivascular space, marked by transcriptional upregulation and spatial redistribution of collagens associated with barrier functions. Furthermore, we demonstrate that TLS maturation into sites of dynamic adaptive immune responses, characterized by clonal T and B cell expansion and IgA+ and IgG+ plasma-cell formation, is driven by efficient early T cell recruitment to the perivascular space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ba24c654b135fc550ac0b5abe83db49fb3978d" target='_blank'>
              Glioma-associated tertiary lymphoid structures are sites of lymphocyte clonal expansion and plasma cell formation
              </a>
            </td>
          <td>
            P. Cakmak, J. H. Lun, Aakanksha Singh, J. Macas, J. Schupp, M. Köhler, Tatjana Starzetz, Michael C. Burger, Eike Steidl, Lucie Marie Hasse, Elke Hattingen, Karlheinz Plate, Y. Reiss, K. Imkeller
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The patterns by which primary tumors spread to metastatic sites remain poorly understood. Here, we define patterns of metastatic seeding in prostate cancer (PCa) using a novel injection-based mouse model - EvoCaP (Evolution in Cancer of the Prostate), featuring aggressive metastatic cancer to bone, liver, lungs, and lymph nodes. To define migration histories between primary and metastatic sites, we used our EvoTraceR pipeline to track distinct tumor clones containing recordable barcodes. We detected widespread intratumoral heterogeneity from the primary tumor in metastatic seeding, with few clonal populations (CPs) instigating most migration. Metastasis-to-metastasis seeding was uncommon, as most cells remained confined within the tissue. Migration patterns in our model were congruent with human PCa seeding topologies. Our findings support the view of metastatic PCa as a systemic disease driven by waves of aggressive clones expanding their niche, infrequently overcoming constraints that otherwise keep them confined in the primary or metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7ba92880acaf1b934f559d90646f82a1f001d3" target='_blank'>
              Clonal Lineage Tracing with Somatic Delivery of Recordable Barcodes Reveals Migration Histories of Metastatic Prostate Cancer.
              </a>
            </td>
          <td>
            Ryan N Serio, Armin Scheben, Billy Lu, Domenic V Gargiulo, Lucrezia Patruno, Caroline L Buckholtz, Ryan J Chaffee, Megan C Jibilian, Steven G Persaud, Stephen J. Staklinski, Rebecca Hassett, Lise M Brault, Daniele Ramazzotti, Christopher E Barbieri, Adam Siepel, Dawid G Nowak
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dcce816f6a19dca7baf05569c687c99b3105e9" target='_blank'>
              Unbiasedly decoding the tumor microenvironment with single-cell multiomics analysis in pancreatic cancer
              </a>
            </td>
          <td>
            Yifan Fu, Jinxin Tao, Tao Liu, Yueze Liu, J. Qiu, Dan Su, Ruobing Wang, W. Luo, Zhe Cao, Guihu Weng, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-07-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The current challenge in analyzing sequencing-based spatial transcriptomics (ST) is to precisely resolve the spatial distribution of cell subpopulations with low abundance and detect context-dependent variation in cell states within complex tissue microenvironments. Here, we introduce an ultra-resolution ST deconvolution algorithm (UCASpatial) that improves the resolution of mapping cell subpopulations to spatial locations by leveraging the contribution of genes indicative of cell identity through entropy-based weighting. Using both in silico and real ST datasets, we demonstrate that UCASpatial improves the robustness and accuracy in identifying low-abundant cell subpopulations and distinguishing transcriptionally heterogeneous cell subpopulations. Applying UCASpatial to murine wound healing, we recapitulate known spatiotemporal dynamics of multiple cell subpopulations involved in wound healing and reveal the spatial segregation of distinct macrophage subpopulations embedded within different cellular communities at the wound center. In human colorectal cancer (CRC), we link genomic alterations in individual cancer clones to multi-cellular characteristics of the tumor immune microenvironment (TIME) and reveal the co-evolution of tumor cells and TIME within the same tumor. We show that the copy number gain on chromosome 20q (chr20q-gain) in tumor cells orchestrates a T cell-excluded TIME, indicative of resistance to immunotherapy in CRC, and is associated with tumor-intrinsic endogenous retrovirus silencing and impaired type I interferon response. Our findings present UCASpatial as a versatile tool for deciphering ultra-resolution cellular landscapes in ST and exploring intercellular mechanisms in complex and dynamic microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36aa9aa2d8c5c7dbae3c0976fedf2735c37f839" target='_blank'>
              Ultra-resolution Deconvolution of Spatial Transcriptomics Unveils Spatiotemporal Cellular Dynamics in Complex Microenvironments
              </a>
            </td>
          <td>
            Yin Xu, Zurui Huang, Yawei Zhang, Zhenghang Wang, Peijin Guo, Feifan Zhang, Wenxuan Gong, Guanghao Liang, Boyuan Mei, Lihui Dong, Renbao Chang, Minghui Gong, Yu Xia, Haochen Ni, Jing Yang, Yuan Gao, Zhaoqi Liu, Lin Shen, Jian Li, Meng Michelle Xu, Dali Han
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Understanding the spatial organization of the bone marrow (BM) microenvironment at single-cell resolution constitutes a challenge in multiple myeloma (MM). Using formalin-fixed paraffin-embedded (FFPE) samples from the MIcγ1 mouse model of MM and aged-matched healthy YFPcγ1 mice, we performed spatial transcriptional profiling with Visium Spatial Gene Expression. A custom data-analysis framework that combines spatial with single-cell transcriptomic profiling defined the BM cellular composition and established specific cell relations, visualizing the spatial distribution of transcriptionally heterogeneous MM plasma cells (MM-PC). MM pathogenesis transcriptional programs were spatially delineated within the BM microenvironment. A high-to-low MM-PC density gradient spatially correlated with effector-to-exhausted T cell phenotype abundance. In this context, MM cells in high-density MM-PC areas coexisted with dendritic cells while displaced neutrophils to the tumor border. Increased neutrophil extracellular trap formation, IL-17-driven inflammatory signaling, and osteoclast differentiation were spatially delineated within the BM microenvironment. The spatial identification of different areas of BM and the interaction between malignant cells and their microenvironment were validated in FFPE BM biopsies from MM patients with varying degrees of MM-PC infiltration. In summary, spatial transcriptomics depicts the BM cellular architecture of MM and reveals deregulated mechanisms underlying MM intercellular communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6b45562f865baeadce9704ad2edc7a5b178a26" target='_blank'>
              Spatial Transcriptomics Reveals a Myeloma Cell Architecture with Dysfunctional T-Cell Distribution, Neutrophil Traps, and Inflammatory Signaling
              </a>
            </td>
          <td>
            Laura Sudupe, Emma Muiños-López, A. Lopez-Perez, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, Paula Aguirre-Ruiz, Patxi San Martín-Úriz, Marta Larrayoz, Laura Alvarez-Gigli, Marta Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho González, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, Andreas Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb667e3e1fbda176a1eb9bab50b39b93702041c" target='_blank'>
              STdGCN: spatial transcriptomic cell-type deconvolution using graph convolutional networks
              </a>
            </td>
          <td>
            Yawei Li, Yuan Luo
          </td>
          <td>2024-08-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Faithful genetically engineered in vivo models of medulloblastoma (MB) are currently available only for some molecular subgroups, in keeping with recent studies showing the unique role of human-specific progenitors in the development of G3 and G4 MB subgroups. We generated human cerebellar organoids (CbO) from expanded potential stem cells (EPSC) and characterised in depth their epigenetic and transcriptomic profile. We show the presence of sub compartment-specific cerebellar lineages linked to MB formation, including populations expressing signature genes of G3 and G4 MB cells-of-origin in a 1 month-old CbO. Moreover, we demonstrate that CbO sustains MB cells proliferation and invasion in a 3D co-culture model (CbO-MB) while preserving their molecular identity, and that treatment of CbO-MB with anti-tumour compounds recapitulate the efficacy of in vivo drug testing in PDX models. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418ddee9c7f8cb6b4c5b80d2a93be6711b4eb2cf" target='_blank'>
              Modelling medulloblastoma pathogenesis and treatment in human cerebellar organoids
              </a>
            </td>
          <td>
            Thomas Willott, James G Nicholson, Yunchen Xiao, O. Ogunbiyi, Talisa Mistry, N. Zabet, A. Merve, S. Badodi, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor heterogeneity refers to the diversity observed among tumor cells: both between different tumors (inter-tumor heterogeneity) and within a single tumor (intra-tumor heterogeneity). These cells can display distinct morphological and phenotypic characteristics, including variations in cellular morphology, metastatic potential and variability treatment responses among patients. Therefore, a comprehensive understanding of such heterogeneity is necessary for deciphering tumor-specific mechanisms that may be diagnostically and therapeutically valuable. Innovative and multidisciplinary approaches are needed to understand this complex feature. In this context, proteogenomics has been emerging as a significant resource for integrating omics fields such as genomics and proteomics. By combining data obtained from both Next-Generation Sequencing (NGS) technologies and mass spectrometry (MS) analyses, proteogenomics aims to provide a comprehensive view of tumor heterogeneity. This approach reveals molecular alterations and phenotypic features related to tumor subtypes, potentially identifying therapeutic biomarkers. Many achievements have been made; however, despite continuous advances in proteogenomics-based methodologies, several challenges remain: in particular the limitations in sensitivity and specificity and the lack of optimal study models. This review highlights the impact of proteogenomics on characterizing tumor phenotypes, focusing on the critical challenges and current limitations of its use in different clinical and preclinical models for tumor phenotypic characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/958a7a3c1fd552ec9c7dc2019625a3d4361653d4" target='_blank'>
              Phenotyping Tumor Heterogeneity through Proteogenomics: Study Models and Challenges
              </a>
            </td>
          <td>
            Diletta Piana, Federica Iavarone, E. De Paolis, Gennaro Daniele, Federico Parisella, Angelo Minucci, Viviana Greco, Andrea Urbani
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer for which few effective therapies exist. Immunotherapies specifically are ineffective in pancreatic cancer, in part due to its unique stromal and immune microenvironment. Pancreatic intraepithelial neoplasia, or PanIN, is the main precursor lesion to PDAC. Recently it was discovered that PanINs are remarkably abundant in the grossly normal pancreas, suggesting that the vast majority will never progress to cancer. Here, through construction of 48 samples of cm3-sized human pancreas tissue, we profiled the immune microenvironment of 1,476 PanINs in 3D and at single-cell resolution to better understand the early evolution of the pancreatic tumor microenvironment and to determine how inflammation may play a role in cancer progression. We found that bulk pancreatic inflammation strongly correlates to PanIN cell fraction. We found that the immune response around PanINs is highly heterogeneous, with distinct immune hotspots and cold spots that appear and disappear in a span of tens of microns. Immune hotspots generally mark locations of higher grade of dysplasia or locations near acinar atrophy. The immune composition at these hotspots is dominated by naïve, cytotoxic, and regulatory T cells, cancer associated fibroblasts, and tumor associated macrophages, with little similarity to the immune composition around less-inflamed PanINs. By mapping FOXP3+ cells in 3D, we found that regulatory T cells are present at higher density in larger PanIN lesions compared to smaller PanINs, suggesting that the early initiation of PanINs may not exhibit an immunosuppressive response. This analysis demonstrates that while PanINs are common in the pancreases of most individuals, inflammation may play a pivotal role, both at the bulk and the microscopic scale, in demarcating regions of significance in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/194ae57756ba7688c48e283a00a59550489311e0" target='_blank'>
              3D histology reveals that immune response to pancreatic precancers is heterogeneous and depends on global pancreas structure
              </a>
            </td>
          <td>
            A. Kiemen, Cristina Almagro-Pérez, Valentina Matos, André Forjaz, Alicia M. Braxton, Lucie Dequiedt, Jeeun Parksong, Courtney Cannon, Xuan Yuan, Sarah M. Shin, Jaanvi Mahesh Babu, Elizabeth D. Thompson, T. Cornish, W. Ho, Laura D. Wood, Pei-Hsun Wu, Arrate Muñoz Barrutia, Ralph H. Hruban, Denis Wirtz
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Normal kidney development requires coordinated interactions between multiple progenitor cell lineages. The Foxd1+ stromal progenitors are critical for normal nephrogenesis and their heterogeneity is increasingly appreciated. However, the molecular mechanisms and trajectories that drive the differentiation of Foxd1+ cells toward the renal stroma, capsule, mesangial cells, renin cells, pericytes, and vascular smooth muscle cells (VSMCs) are poorly understood. Recent work has implicated Tcf21, a mesoderm-specific bHLH transcription factor critical for embryogenesis, in the development of the kidney stroma and perivascular cells. To investigate the role of Tcf21 in Foxd1+ cells, we performed single-cell RNA sequencing (scRNA-seq) on GFP+ cells from E14.5 Foxd1Cre;Rosa26mTmG;Tcf21f/f kidneys (Tcf21-cKO) and Foxd1Cre controls. Clustering of the entire dataset identified a large stromal population and a smaller representation of non-stromal lineages. Subclustering of stromal cells identified six populations associated with healthy kidney development: medullary/perivascular, proliferating, differentiating nephron, nephrogenic zone-associated, collecting duct-associated, and ureteric. Loss of Tcf21 resulted in a dramatic reduction in the medullary/perivascular, proliferating, nephrogenic zone-associated, and collecting duct-associated stromal subpopulations. Immunostaining confirmed that Tcf21-cKO has a severe constriction of the medullary and collecting duct-associated stromal space. We identified and validated a cluster unique to Tcf21-cKO kidneys exhibiting mosaic expression of genes from nephrogenic, proliferating, medullary, and perivascular stromal cells spanning across all pseudotime, suggesting cells halted in the midst of differentiation. These findings underscore a critical role for Tcf21 in the emergence of Foxd1+ derivatives, with loss of Tcf21 leading to a shift in stromal cell fates that results in abnormal kidney development. NEW & NOTEWORTHY The mechanisms responsible for the emergence of renal stromal heterogeneity has been unknown. Using scRNA-seq on Foxd1+ enriched cells from E14.5 kidneys, we identified seven molecularly distinct stromal populations and their regional association. The data suggest that the transcription factor Tcf21 regulates the adoption of fates by Foxd1+ cells that is required to form the normal milieu of stromal derivatives for the development of a kidney of normal size and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4605589abc5ae048f75a46e26a169fa6bf5a7c2" target='_blank'>
              The Transcription Factor TCF21 is necessary for adoption of cell fates by Foxd1+ stromal progenitors during kidney development
              </a>
            </td>
          <td>
            Gal Finer, Mohammad D. Khan, Yalu Zhou, G. Gadhvi, George S. Yacu, Joo-Seop Park, R. A. Gomez, M. Sequeira-Lopez, Susan E. Quaggin, Deborah R. Winter
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Recent lineage tracing single-cell techniques (LT-scSeq), e.g., the Lineage And RNA RecoverY (LARRY) barcoding system, have enabled clonally resolved interpretation of differentiation trajectories. However, the heterogeneity of clone-specific kinetics remains understudied, both quantitatively and in terms of interpretability, thus limiting the power of bar-coding systems to unravel how heterogeneous stem cell clones drive overall cell population dynamics. Here, we present CLADES, a NeuralODE-based framework to faithfully estimate clone-specific kinetics of cell states from newly generated and publicly available human cord blood LARRY LT-scSeq data. By incorporating a stochastic simulation algorithm (SSA) and differential expression gene (DEGs) analysis, CLADES yields cell division dynamics across differentiation timecourses and fate bias predictions for the early progenitor cells. Moreover, clone-level quantitative behaviours can be grouped into characteristic types by pooling individual clones into meta-clones. By benchmarking with CoSpar, we found that CLADES improves fate bias prediction accuracy at the meta-clone level. In conclusion, we report a broadly applicable approach to robustly quantify differentiation kinetics using meta-clones while providing valuable insights into the fate bias of cellular populations for any organ system maintained by a pool of heterogeneous stem and progenitor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181cc1f16a3c5b14fb171a92ae7dc42356fcbe11" target='_blank'>
              Unveiling Clonal Cell Fate and Differentiation Dynamics: A Hybrid NeuralODE-Gillespie Approach
              </a>
            </td>
          <td>
            Mingze Gao, Melania Barile, Shirom Chabra, M. Haltalli, Emily F. Calderbank, Yiming Chao, E. Laurenti, B. Gottgens, Yuanhua Huang
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 Therapeutics targeting tumor endothelial cells (TECs) have been explored for decades, with only suboptimal efficacy achieved, partly due to an insufficient understanding of the TEC heterogeneity across cancer patients. We integrated single-cell RNA-seq data of 575 cancer patients from 19 solid tumor types, comprehensively charting the TEC phenotypic diversities. Our analyses uncovered underappreciated compositional and functional heterogeneity in TECs from a pan-cancer perspective. Two subsets, CXCR4+ tip cells and SELE+ veins, represented the prominent angiogenic and proinflammatory phenotypes of TECs, respectively. They exhibited distinct spatial organization patterns, and compared to adjacent non-tumor tissues, tumor tissue showed an increased prevalence of CXCR4+ tip cells, yet with SELE+ veins depleted. Such functional and spatial characteristics underlie their differential associations with the response of anti-angiogenic therapies and immunotherapies. Our integrative resources and findings open new avenues to understand and clinically intervene in the tumor vasculature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc571fa1d1ff50ea9e24190fe0da0d50d2cd7b6f" target='_blank'>
              Pan-cancer integrative analyses dissect the remodeling of endothelial cells in human cancers
              </a>
            </td>
          <td>
            Jinhu Li, Dongfang Wang, Fei Tang, Xinnan Ling, Wenjie Zhang, Zemin Zhang
          </td>
          <td>2024-07-11</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Heterogeneity of tumor metabolism is an important, but still poorly understood aspect of tumor biology. Present work is focused on the visualization and quantification of cellular metabolic heterogeneity of colorectal cancer using fluorescence lifetime imaging (FLIM) of redox cofactor NAD(P)H. FLIM-microscopy of NAD(P)H was performed in vitro in four cancer cell lines (HT29, HCT116, CaCo2 and CT26), in vivo in the four types of colorectal tumors in mice and ex vivo in patients’ tumor samples. The dispersion and bimodality of the decay parameters were evaluated to quantify the intercellular metabolic heterogeneity. Our results demonstrate that patients’ colorectal tumors have significantly higher heterogeneity of energy metabolism compared with cultured cells and tumor xenografts, which was displayed as a wider and frequently bimodal distribution of a contribution of a free (glycolytic) fraction of NAD(P)H within a sample. Among patients’ tumors, the dispersion was larger in the high-grade and early stage ones, without, however, any association with bimodality. These results indicate that cell-level metabolic heterogeneity assessed from NAD(P)H FLIM has a potential to become a clinical prognostic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d39beee686da3cb9d1fa5f224388f8f6c20fccf" target='_blank'>
              Insights into metabolic heterogeneity of colorectal cancer gained from fluorescence lifetime imaging
              </a>
            </td>
          <td>
            A. D. Komarova, S. D. Sinyushkina, Ilia D Shchechkin, I. Druzhkova, Sofia A Smirnova, Vitaliy M Terekhov, A. Mozherov, Nadezhda I. Ignatova, Elena E. Nikonova, E. Shirshin, L. Shimolina, Sergey V Gamayunov, V. Shcheslavskiy, M. Shirmanova
          </td>
          <td>2024-08-28</td>
          <td>eLife</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are a heterogeneous population of cells whose phenotypes and functions are shaped by factors that are incompletely understood. Herein, we asked when and where TAMs arise from blood monocytes and how they evolve during tumor development. We initiated pancreatic ductal adenocarcinoma (PDAC) in inducible monocyte fate-mapping mice and combined single-cell transcriptomics and high-dimensional flow cytometry to profile the monocyte-to-TAM transition. We revealed that monocytes differentiate first into a transient intermediate population of TAMs that generates two longer-lived lineages of terminally differentiated TAMs with distinct gene expression profiles, phenotypes, and intratumoral localization. Transcriptome datasets and tumor samples from patients with PDAC evidenced parallel TAM populations in humans and their prognostic associations. These insights will support the design of new therapeutic strategies targeting TAMs in PDAC. Intermediate tumor-associated macrophages (TAMs) emerge first and give rise to two more differentiated TAM populations in pancreatic cancer. Editor’s summary Tumor-associated macrophages (TAMs) promote tumor growth and immune suppression, but how TAMs develop from circulating monocytes and local tissue-resident macrophages remains incompletely understood. Using a model of pancreatic ductal adenocarcinoma (PDAC) in monocyte fate-mapping mice, Dunsmore et al. examined the dynamics of the monocyte-to-macrophage transition in tumors. A population of “intermediate” TAMs gave rise to two more differentiated TAM subsets, distinguishable by their distinct surface marker phenotype, dependence on the transcription factor Maf, migratory behavior, and localization in tumors. These findings provide a time- and space-resolved picture of TAM heterogeneity and development in PDAC, which could guide the future development of TAM-targeted therapeutic strategies. —Claire Olingy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a54c86dbe269d24ca8abdd22c4597319660f70" target='_blank'>
              Timing and location dictate monocyte fate and their transition to tumor-associated macrophages
              </a>
            </td>
          <td>
            G. Dunsmore, Wei Guo, Ziyi Li, D. Bejarano, Rhea Pai, Katharine Yang, Immanuel Kwok, Leonard Tan, M. Ng, Carlos De La Calle Fabregat, A. Yatim, A. Bougouin, Kevin Mulder, Jake Thomas, Javiera Villar, Mathilde Bied, Benoit Kloeckner, C. Dutertre, Grégoire Gessain, S. Chakarov, Zhaoyuan Liu, J. Scoazec, A. Lennon-Duménil, Thomas Marichal, C. Sautès-Fridman, W. Fridman, Ankur Sharma, Bing Su, A. Schlitzer, Lai Guan Ng, Camille Blériot, Florent Ginhoux
          </td>
          <td>2024-07-26</td>
          <td>Science Immunology</td>
          <td>1</td>
          <td>86</td>
        </tr>

        <tr id="Although localized prostate cancer is relatively indolent, advanced prostate cancer manifests with aggressive and often lethal variants, including neuroendocrine prostate cancer (NEPC). To identify drivers of aggressive prostate cancer, we leveraged Sleeping Beauty (SB) transposon mutagenesis in a mouse model based on prostate-specific loss-of-function of Pten and Tp53. Compared with control mice, SB mice developed more aggressive prostate tumors, with increased incidence of metastasis. Notably, a significant percentage of the SB prostate tumors display NEPC phenotypes, and the transcriptomic features of these SB mouse tumors recapitulated those of human NEPC. We identified common SB transposon insertion sites (CIS) and prioritized associated CIS-genes differentially expressed in NEPC versus non-NEPC SB tumors. Integrated analysis of CIS-genes encoding for proteins representing upstream, post-translational modulators of master regulators controlling the transcriptional state of SB-mouse and human NEPC tumors identified sirtuin 1 (Sirt1) as a candidate mechanistic determinant of NEPC. Gain-of-function studies in human prostate cancer cell lines confirmed that SIRT1 promotes NEPC, while its loss-of-function or pharmacological inhibition abrogates NEPC. This integrative analysis is generalizable and can be used to identify novel cancer drivers for other malignancies. Summary Using an unbiased forward mutagenesis screen in an autochthonous mouse model, we have investigated mechanistic determinants of aggressive prostate cancer. SIRT1 emerged as a key regulator of neuroendocrine prostate cancer differentiation and a potential target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef638b5be4a6bc97dbe24dca3c6171d01e593aa" target='_blank'>
              A forward genetic screen identifies Sirtuin1 as a driver of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Francisca Nunes de Almeida, A. Vasciaveo, Ainsley Mike Antao, Min Zou, Matteo Di Bernardo, Simone de Brot, A. Rodriguez-Calero, Alexander Chui, Alexander L.E. Wang, Nicolas Floc’h, Jaime Y Kim, Stephanie N Afari, Timur Mukhammadov, Juan M Arriaga, Jinqiu Lu, , Mark A. Rubin, Andrea Califano, Cory Abate-Shen
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest-derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma.
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Single-cell transcriptomics (scRNA-seq) has enabled the characterization of cell state heterogeneity and recapitulation of differentiation trajectories. However, since proteins are the main functional entities in cells, the exclusive use of mRNA measurements comes at the risk of missing important biological information. Here we leverage recent technological advances in single-cell proteomics by Mass Spectrometry (scp-MS) to generate the first scp-MS dataset of an in vivo differentiation hierarchy encompassing over 2,500 human CD34+ hematopoietic stem and progenitor cells. Through integration with scRNA-seq, we identify proteins that are important for stem cell quiescence, which were not indicated by their mRNA transcripts, and demonstrate functional expression covariance during differentiation that is only detectable on protein level. Finally, we show that modeling translation dynamics can infer cell progression during differentiation and explain 45% more protein variation from mRNA than linear correlation. Our work serves as a framework for future single-cell multi-omics studies across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd77ec3528ae2b7fddb76ced051e72f33a867349" target='_blank'>
              Mapping the human hematopoietic stem and progenitor cell hierarchy through integrated single-cell proteomics and transcriptomics
              </a>
            </td>
          <td>
            B. Furtwängler, Nil Üresin, Sabrina Richter, M. Schuster, Despoina Barmpouri, Henrietta Holze, Anne Wenzel, Kirsten Grønbæk, Kim Theilgaard-Mönch, F. Theis, E. Schoof, Bo Porse
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Objectives Non‐small‐cell lung carcinoma (NSCLC) is the most prevalent and lethal form of lung cancer. The need for biomarker‐informed stratification of targeted therapies has underpinned the need to uncover the underlying properties of the tumor microenvironment (TME) through high‐plex quantitative assays. Methods In this study, we profiled resected NSCLC tissues from 102 patients by targeted spatial proteomics of 78 proteins across tumor, immune activation, immune cell typing, immune‐oncology, drug targets, cell death and PI3K/AKT modules to identify the tumor and stromal signatures associated with overall survival (OS). Results Survival analysis revealed that stromal CD56 (HR = 0.384, P = 0.06) and tumoral TIM3 (HR = 0.703, P = 0.05) were associated with better survival in univariate Cox models. In contrast, after adjusting for stage, BCLXL (HR = 2.093, P = 0.02) and cleaved caspase 9 (HR = 1.575, P = 0.1) negatively influenced survival. Delta testing indicated the protective effect of TIM‐3 (HR = 0.614, P = 0.04) on OS. In multivariate analysis, CD56 (HR = 0.172, P = 0.001) was associated with better survival in the stroma, while B7.H3 (HR = 1.72, P = 0.008) was linked to poorer survival in the tumor. Conclusions Deciphering the TME using high‐plex spatially resolved methods is giving us new insights into compartmentalised tumor and stromal protein signatures associated with clinical endpoints in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c86af101b13bc4f2f501f32d2e227ba48bb3c7a" target='_blank'>
              Spatial proteomic profiling of tumor and stromal compartments in non‐small‐cell lung cancer identifies signatures associated with overall survival
              </a>
            </td>
          <td>
            Vahid Yaghoubi Naei, J. Monkman, Habib Sadeghirad, Ahmed M Mehdi, Tony Blick, W. Mullally, K. O’Byrne, M. Warkiani, A. Kulasinghe
          </td>
          <td>2024-01-01</td>
          <td>Clinical & Translational Immunology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) mortality rates continue to increase faster than those of other cancer types due to high heterogeneity, which limits diagnosis and treatment. Pathological and molecular subtyping have identified that HCC tumors with poor outcomes are characterized by intratumoral collagenous accumulation. However, the translational and post-translational regulation of tumor collagen, which is critical to the outcome, remains largely unknown. Here, we investigate the spatial extracellular proteome to understand the differences associated with HCC tumors defined by Hoshida transcriptomic subtypes of poor outcome (Subtype 1; S1; n = 12) and better outcome (Subtype 3; S3; n = 24) that show differential stroma-regulated pathways. Collagen-targeted mass spectrometry imaging (MSI) with the same-tissue reference libraries, built from untargeted and targeted LC-MS/MS was used to spatially define the extracellular microenvironment from clinically-characterized, formalin-fixed, paraffin-embedded tissue sections. Collagen α-1(I) chain domains for discoidin-domain receptor and integrin binding showed distinctive spatial distribution within the tumor microenvironment. Hydroxylated proline (HYP)-containing peptides from the triple helical regions of fibrillar collagens distinguished S1 from S3 tumors. Exploratory machine learning on multiple peptides extracted from the tumor regions could distinguish S1 and S3 tumors (with an area under the receiver operating curve of ≥0.98; 95% confidence intervals between 0.976 and 1.00; and accuracies above 94%). An overall finding was that the extracellular microenvironment has a high potential to predict clinically relevant outcomes in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53efde0c062c9d0f5b393a6e975a03778bb11cae" target='_blank'>
              The Spatial Extracellular Proteomic Tumor Microenvironment Distinguishes Molecular Subtypes of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Jade K Macdonald, Harrison B Taylor, Mengjun Wang, A. DelaCourt, Christin Edge, David N. Lewin, Naoto Kubota, Naoto Fujiwara, F. Rasha, Cesia A. Marquez, A. Ono, Shiro Oka, K. Chayama, Sara Lewis, B. Taouli, Myron Schwartz, M. Fiel, Richard R Drake, Y. Hoshida, A. Mehta, Peggi M Angel
          </td>
          <td>2024-07-09</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Significance Treatment of prostate cancer is rapidly evolving with several promising drugs targeting different cell surface antigens. Selection of patients most likely to benefit from these therapies requires an understanding of how expression of these cell surface antigens varies across patients and how they change during disease progression, particularly in tumors that undergo lineage plasticity. Using immunohistochemistry and single-cell mRNA sequencing, we reveal heterogeneity of cell states across a cohort of advanced disease prostate cancer patients; this heterogeneity is not captured by conventional histology-based designations of adenocarcinoma and neuroendocrine prostate cancer. We show that these cell states can be identified by gene regulatory networks that could provide additional diagnostic precision based on their correlation with clinically relevant cell surface antigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c609499e96060bae9781113aade46c45aa614e3d" target='_blank'>
              Single-cell analysis of treatment-resistant prostate cancer: Implications of cell state changes for cell surface antigen–targeted therapies
              </a>
            </td>
          <td>
            S. Zaidi, Jooyoung Park, J. Chan, Martine P. Roudier, Jimmy L. Zhao, Anuradha Gopalan, K. Wadosky, Radhika A Patel, Erolcan Sayar, W. Karthaus, D. H. Kates, O. Chaudhary, Tianhao Xu, I. Masilionis, L. Mazutis, R. Chaligné, Aleksandar Z Obradovic, I. Linkov, A. Barlas, Achim A. Jungbluth, Natasha Rekhtman, J. Silber, Katia Manova-Todorova, Philip A Watson, Lawrence D. True, C. Morrissey, H. Scher, D. Rathkopf, Michael J Morris, David W. Goodrich, Jungmin Choi, Peter S Nelson, Michael C. Haffner, C. Sawyers
          </td>
          <td>2024-07-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, Sahar Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood.We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles.We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation.Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell analysis has become an essential tool in modern biological research, providing unprecedented insights into cellular behavior and heterogeneity. By examining individual cells, this approach surpasses conventional population-based methods, revealing critical variations in cellular states, responses to environmental cues, and molecular signatures. In the context of cancer, with its diverse cell populations, single-cell analysis is critical for investigating tumor evolution, metastasis, and therapy resistance. Understanding the phenotype-genotype relationship at the single-cell level is crucial for deciphering the molecular mechanisms driving tumor development and progression. This review highlights innovative strategies for selective cell isolation based on desired phenotypes, including robotic aspiration, laser detachment, microraft arrays, optical traps, and droplet-based microfluidic systems. These advanced tools facilitate high-throughput single-cell phenotypic analysis and sorting, enabling the identification and characterization of specific cell subsets, thereby advancing therapeutic innovations in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1bb0534cf6f72176c966878e2924974c47306ed" target='_blank'>
              Advances in Single-Cell Techniques for Linking Phenotypes to Genotypes.
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yushu Ma, Jinxiong Cheng, Yu-Chih Chen
          </td>
          <td>2024-07-25</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mechanistic models of dynamic, interacting cell populations have yielded many insights into the growth and resolution of immune responses. Historically these models have described the behavior of pre-defined cell types based on small numbers of phenotypic markers. The ubiquity of deep phenotyping therefore presents a new challenge; how do we confront tractable and interpretable mathematical models with high-dimensional data? To tackle this problem, we studied the development and persistence of lung-resident memory CD4 and CD8 T cells (TRM) in mice infected with influenza virus. We developed an approach in which dynamical model parameters and the population structure are inferred simultaneously. This method uses deep learning and stochastic variational inference and is trained on the single-cell flow-cytometry data directly, rather than on the kinetics of pre-identified clusters. We show that during the resolution phase of the immune response, memory CD4 and CD8 T cells within the lung are phenotypically diverse, with subsets exhibiting highly distinct and time-dependent dynamics. TRM heterogeneity is maintained long-term by ongoing differentiation of relatively persistent Bcl-2hi CD4 and CD8 TRM subsets which resolve into distinct functional populations. Our approach yields new insights into the dynamics of tissue-localized immune memory, and is a novel basis for interpreting time series of high-dimensional data, broadly applicable to diverse biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de98a7caad311fa21b6e5ae069bafe928ed0e792" target='_blank'>
              A variational deep-learning approach to modeling memory T cell dynamics
              </a>
            </td>
          <td>
            C. V. van Dorp, Joshua I. Gray, Daniel H. Paik, Donna L. Farber, Andrew J Yates
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the most common form of pancreatic cancer, a disease with dismal overall survival. Advances in treatment are hindered by a lack of preclinical models. Here, we show how a personalized organotypic “avatar” created from resected tissue allows spatial and temporal reporting on a complete in situ tumor microenvironment and mirrors clinical responses. Our perfusion culture method extends tumor slice viability, maintaining stable tumor content, metabolism, stromal composition, and immune cell populations for 12 days. Using multiplexed immunofluorescence and spatial transcriptomics, we identify immune neighborhoods and potential for immunotherapy. We used avatars to assess the impact of a preclinically validated metabolic therapy and show recovery of stromal and immune phenotypes and tumor redifferentiation. To determine clinical relevance, we monitored avatar response to gemcitabine treatment and identify a patient avatar-predictable response from clinical follow-up. Thus, avatars provide valuable information for syngeneic testing of therapeutics and a truly personalized therapeutic assessment platform for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361225cc3eb1a10b3d464aab534da9288640614c" target='_blank'>
              Development of human pancreatic cancer avatars as a model for dynamic immune landscape profiling and personalized therapy
              </a>
            </td>
          <td>
            Daniel Hughes, Alice Evans, Simei Go, Michael Eyres, Liuliu Pan, Somnath Mukherjee, Z. Soonawalla, Frances Willenbrock, Eric O’Neill
          </td>
          <td>2024-07-05</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Simple Summary Resistance to standard therapies represents a major challenge in melanoma treatment. This is mainly related to the high heterogeneity of this malignancy due to the presence of different types of cells in the tumor mass. Cancer stem cells (CSCs) are a small subset of cells endowed with self-renewal and tumorigenic capacity and the ability to escape anticancer therapies. Melanoma CSCs can be identified by the expression of biomarkers and intracellular pathways deeply involved in their malignant phenotype. Dissecting the molecular mechanisms underlying CSC biological functions might pave the way for novel therapeutic approaches. Herein, we highlight the intricate bidirectional communication between melanoma CSCs and their surrounding cells in the tumor microenvironment and discuss how this interplay might impact the processes of drug resistance and tumor relapse. We also address the potential benefits of CSC-targeted strategies, in combination with conventional drugs, for improving melanoma treatment strategies. Abstract Cutaneous melanoma still represents a significant health burden worldwide, being responsible for the majority of skin cancer deaths. Key advances in therapeutic strategies have significantly improved patient outcomes; however, most patients experience drug resistance and tumor relapse. Cancer stem cells (CSCs) are a small subpopulation of cells in different tumors, including melanoma, endowed with distinctive capacities of self-renewal and differentiation into bulk tumor cells. Melanoma CSCs are characterized by the expression of specific biomarkers and intracellular pathways; moreover, they play a pivotal role in tumor onset, progression and drug resistance. In recent years, great efforts have been made to dissect the molecular mechanisms underlying the protumor activities of melanoma CSCs to provide the basis for novel CSC-targeted therapies. Herein, we highlight the intricate crosstalk between melanoma CSCs and bystander cells in the tumor microenvironment (TME), including immune cells, endothelial cells and cancer-associated fibroblasts (CAFs), and its role in melanoma progression. Specifically, we discuss the peculiar capacities of melanoma CSCs to escape the host immune surveillance, to recruit immunosuppressive cells and to educate immune cells toward an immunosuppressive and protumor phenotype. We also address currently investigated CSC-targeted strategies that could pave the way for new promising therapeutic approaches for melanoma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7340de903da58d80075cbbac264d846e831d74b" target='_blank'>
              Unveiling the Dynamic Interplay between Cancer Stem Cells and the Tumor Microenvironment in Melanoma: Implications for Novel Therapeutic Strategies
              </a>
            </td>
          <td>
            Patrizia Limonta, Raffaella Chiaramonte, Lavinia Casati
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J. Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Clone assignment in single-cell genomics remains a challenge due to its diverse mutation macrostructures and many missing signals. Existing statistical methods, for the sake of numerical convergence, pose strong constraints on the form of predicted mutation patterns, so they easily identify sub-optimally fitted clones that overlook weak and rare mutations. To solve this problem, we developed SNPmanifold, a Python package that learns flexible mutation patterns using a shallow binomial variational autoencoder. The latent space of SNPmanifold can effectively represent and visualize complex mutations of SNPs (single-nucleotide polymorphisms) in the form of geometrical manifolds. Based on nuclear or mitochondrial SNPs, we demonstrated that SNPmanifold can effectively identify a large number of multiplexed donors of origin (k = 18) that all existing unsupervised methods fail and lineages of somatic clones with promising biological interpretation. Therefore, SNPmanifold can reveal insights into single-cell SNPs more comprehensively than other existing methods, especially in complex datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f5cea3d97421756c5e4a1a65f43ef0a97c212b" target='_blank'>
              Interpretable variational encoding of genotypes identifies comprehensive clonality and lineages in single cells geometrically
              </a>
            </td>
          <td>
            Hoi Man Chung, Yuanhua Huang
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Interactions between mutant cells and their environment play a key role in determining cancer susceptibility. However, our understanding of how the pre-cancer microenvironment contributes to early tumorigenesis remains limited. Here, we show that newly emerging tumours at their most incipient stages shape their microenvironment in a critical process that determines their survival. Analysis of nascent squamous tumours in the upper gastrointestinal tract of the mouse reveals that the stress response of early tumour cells instructs the underlying mesenchyme to form a supportive “pre-cancer niche”, which dictates the long-term outcome of epithelial lesions. Stimulated fibroblasts beneath emerging tumours activate a wound healing response that triggers a dramatic remodelling of the underlying extracellular matrix, resulting in the formation of a fibronectin-rich stromal scaffold that promotes tumour growth. Functional heterotypic 3D culture assays and in vivo grafting experiments, combining carcinogen-free healthy epithelium and tumour-derived stroma, demonstrate that the pre-cancerous niche alone is sufficient to confer tumour properties to healthy epithelial cells. We propose a model where both mutations and the stromal response to genetic stress defines the likelihood of early tumours to survive and progress towards more advanced disease stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be23254ceedda48875f58bb5d700c5fd073374a7" target='_blank'>
              Pre-cancerous Niche Remodelling Dictates Nascent Tumour Survival
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. E. Rojo Arias, Y. Dang, S. Han, M. T. Bejar, B. Colom, J. C. Fowler, P. Jones, S. Rulands, B. D. Simons, M. P. Alcolea
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immunotherapies have shown great promise in pleural mesothelioma (PM), yet most patients still do not achieve significant clinical response, highlighting the importance of improving understanding of the tumor microenvironment (TME). Here, we utilized high-throughput, single-cell RNA-sequencing to de novo identify 54 expression programs and construct a comprehensive cellular catalogue of the PM TME. We found four cancer-intrinsic programs associated with poor disease outcome and a novel fetal-like, endothelial cell population that likely responds to VEGF signaling and promotes angiogenesis. Throughout cellular compartments, we observe substantial difference in the TME associated with a cancer-intrinsic sarcomatoid signature, including enrichment in fetal-like endothelial cells, CXCL9+ macrophages, cytotoxic, exhausted, and regulatory T cells, which we validated using imaging and bulk deconvolution analyses on independent cohorts. Finally, we show, both computationally and experimentally, that NKG2A-HLA-E interaction between NK and tumor cells represents an important new therapeutic axis in PM, especially for epithelioid cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c84bfe3e9c66b621b1832492f79b2264c3e648" target='_blank'>
              Single cell view of tumor microenvironment gradients in pleural mesothelioma.
              </a>
            </td>
          <td>
            Bruno Giotti, Komal Dolasia, William Zhao, Peiwen Cai, Robert Sweeney, Elliot Merritt, Evgeny Kiner, Grace S Kim, Atharva Bhagwat, Thinh Nguyen, Samarth Hegde, Bailey G Fitzgerald, S. Shroff, T. Dawson, M. Garcia-Barros, Jamshid Abdul-Ghafar, Rachel Chen, Sacha Gnjatic, Alan Soto, Rachel Brody, S. Kim-Schulze, Zhihong Chen, Kristin G Beaumont, Miram Merad, R.M. Flores, R. Sebra, Amir Horowitz, Thomas U. Marron, Anna Tocheva, Andrea Wolf, A. Tsankov
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Granulomas (defining tuberculosis histopathological feature) are central to the host’s defense against Mycobacterium tuberculosis, critically influencing patient outcomes. However, knowledge of human granulomas’ structure and function are incomplete. This study employs single-cell and spatial transcriptomics to dissect human granuloma’s cellular composition, structure, communication and function from 19 pulmonary, lymphatic and skeletal samples. Results Our study identified nine key immune-activated/signaling-active cell clusters. Notably, we delineated a three-layered granuloma structure: a core with macrophages (Macro-c09, Macro-c10) and occasional fibroblasts (Fib-c03); a fibroblast-rich (Fib-c01) periphery; and an immune-infiltrated intermediate layer comprising diverse immune-cells recruited by strong signaling-molecules (SPP1/MIF) from core/periphery cells. This study also shows granuloma heterogeneity across individuals and tissues. Conclusions By merging scRNA-seq with ST-seq, we offer an intricate single-cell perspective of granulomas’ spatial-structure and formation mechanisms, identify signaling-molecules and significantly changed genes as potential targets for host-directed tuberculosis immunotherapy, highlight fibroblasts’ crucial role in granuloma formation, and provide an important reference/improved understanding of TB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c14b14dcc946c712193e06986c65772b8af84b9" target='_blank'>
              Single-Cell and Spatial Transcriptomic Analyses Deciphering the Three-Layer Architecture of Human Tuberculosis Granulomas
              </a>
            </td>
          <td>
            Xia Yu, Jie Wang, Peihan Wang, Xiaoqiang Liu, Cuidan Li, Yingjiao Ju, Sitong Liu, Yujie Dong, Jing Wang, Bahetibieke Tuohetaerbaike, Hao Wen, Wenbao Zhang, Haitao Niu, Sihong Xu, Chunlai Jiang, Xiaoyi Jiang, Jing Wu, Hairong Huang, Fei Chen
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Squamous cell carcinomas (SCCs) across different anatomical locations possess common molecular features. Recent studies showed that stromal cells may contribute to tumor progression and metastasis of SCCs. Limited by current sequencing technology and analysis methods, it has been difficult to combine stroma expression profiles with a large number of clinical information. Methods With the help of transfer learning on the cell line, single-cell, and bulk tumor sequencing data, we identified and validated 2 malignant gene patterns (V1 and V5) expressed by stromal cells of SCCs from head and neck (HNSCC), lung (LUSC), cervix (CESC), esophagus, and breast. Results Pattern V5 reflected a novel malignant feature that explained the mixed signals of HNSCC molecular subtypes. Higher expression of pattern V5 was related to shorter PFI with gender and cancer-type specificity. The other stromal gene pattern V1 was associated with poor PFI in patients after surgery in all the three squamous cancer types (HNSCC p = 0.0055, LUSC p = 0.0292, CESC p = 0.0451). Cancer-associated fibroblasts could induce HNSCC cancer cells to express pattern V1. Adjuvant radiotherapy may weaken the effect of high V1 on recurrence and metastasis, depending on the tumor radiosensitivity. Conclusion Considering the prognostic value of stromal gene patterns and its universality, we suggest that the genetic subtype classification of SCCs may be improved to a new system that integrates both malignant and non-malignant components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6008ecf1c1695bb535c0cacf60d9734a6a36e5" target='_blank'>
              Stromal cell-expressed malignant gene patterns contribute to the progression of squamous cell carcinomas across different sites
              </a>
            </td>
          <td>
            Kaiyan Qi, Guangqi Li, Yuanjun Jiang, Xuexin Tan, Qiao Qiao
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract The human adrenal gland is a complex endocrine tissue. Studies on adrenal renewal have been limited to animal models or human foetuses. Enhancing our understanding of adult human adrenal homeostasis is crucial for gaining insights into the pathogenesis of adrenal diseases, such as adrenocortical tumours. Here, we present a comprehensive cellular genomics analysis of the adult human normal adrenal gland, combining single‐nuclei RNA sequencing and spatial transcriptome data to reconstruct adrenal gland homeostasis. As expected, we identified primary cells of the various zones of the adrenal cortex and medulla, but we also uncovered additional cell types. They constitute the adrenal microenvironment, including immune cells, mostly composed of a large population of M2 macrophages, and new cell populations, including different subpopulations of vascular‐endothelial cells and cortical‐neuroendocrine cells. Utilizing spatial transcriptome and pseudotime trajectory analysis, we support evidence of the centripetal dynamics of adrenocortical cell maintenance and the essential role played by Wnt/β‐catenin, sonic hedgehog, and fibroblast growth factor pathways in the adult adrenocortical homeostasis. Furthermore, we compared single‐nuclei transcriptional profiles obtained from six healthy adrenal glands and twelve adrenocortical adenomas. This analysis unveiled a notable heterogeneity in cell populations within the adenoma samples. In addition, we identified six distinct adenoma‐specific clusters, each with varying distributions based on steroid profiles and tumour mutational status. Overall, our results provide novel insights into adrenal homeostasis and molecular mechanisms potentially underlying early adrenocortical tumorigenesis and/or autonomous steroid secretion. Our cell atlas represents a powerful resource to investigate other adrenal‐related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e76e6effd4abc2a2c7085002aaf49ccc7313fb5" target='_blank'>
              Single‐nucleus and spatial transcriptome reveal adrenal homeostasis in normal and tumoural adrenal glands
              </a>
            </td>
          <td>
            B. Altieri, A. K. Secener, Somesh Sai, Cornelius Fischer, S. Sbiera, Panagiota Arampatzi, S. Kircher, Sabine Herterich, Laura-Sophie Landwehr, Sarah N. Vitcetz, C. Braeuning, M. Fassnacht, C. Ronchi, Sascha Sauer, Ali Kerim
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86603b6c8f27f9a5a7d6a05553f0d41b941be5e5" target='_blank'>
              Spatial landscapes of cancers: insights and opportunities.
              </a>
            </td>
          <td>
            Julia Chen, L. Larsson, Alexander Swarbrick, J. Lundeberg
          </td>
          <td>2024-07-23</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are heterogenous in size, biogenesis, cargo and function. Beside small EVs, aggressive tumor cells release a population of particularly large EVs, namely large oncosomes (LO). This study provides the first resource of label-free quantitative proteomics of LO and small EVs obtained from distinct cancer cell types (prostate, breast, and glioma). This dataset was integrated with a SWATH Proteomic assay on LO, rigorously isolated from the plasma of patients with metastatic prostate cancer (PC). Proteins enriched in LO, which were identified also at the RNA level, and found in plasma LO significantly correlated with PC progression. Single EV RNA-Seq of the PC cell-derived LO confirmed some of the main findings from the bulk RNA-Seq, providing first evidence that single cell technologies can be successfully applied to EVs. This multiomics resource of cancer EVs can be leveraged for developing a multi-analyte approach for liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da254bd82830fbf0461d37b66c142c956fe3dc3" target='_blank'>
              Extracellular Vesicles heterogeneity through the lens of multiomics
              </a>
            </td>
          <td>
            Taylon F. Silva, E. Hutchins, Wenyan Zhao, Yari Ciani, Minhyung Kim, Emily Ko, Javier Mariscal, Zhuyu Qiu, Agnes Kittel, Bo Zhou, Yang Wang, Megan Hall, Francesca Galasso, Rebecca A. Reiman, Michael R Freeman, Sarah Parker, Jennifer Van Eyk, Wei Yang, Edwin Posadas, Jlenia Guarnerio, John Nolan, C. Théry, Andries Zijlstra, Shannon Stott, Sungyong You, F. Demichelis, P. Boutros, K. Van Keuren-Jensen, Dolores Di Vizio
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Spheroids have become principal three-dimensional models to study cancer, developmental processes, and drug efficacy. Single-cell analysis techniques have emerged as ideal tools to gauge the complexity of cellular responses in these models. However, the single-cell quantitative assessment based on 3D-microscopic data of the subcellular distribution of fluorescence markers, such as the nuclear/cytoplasm ratio of transcription factors, has largely remained elusive. For spheroid generation, ultra-low attachment plates are noteworthy due to their simplicity, compatibility with automation, and experimental and commercial accessibility. However, it is unknown whether and to what degree the plate type impacts spheroid formation and biology. This study developed a novel AI-based pipeline for the analysis of 3D-confocal data of optically cleared large spheroids at the wholemount, single-cell, and sub-cellular levels. To identify relevant samples for the pipeline, automated brightfield microscopy was employed to systematically compare the size and eccentricity of spheroids formed in six different plate types using four distinct human cell lines. This showed that all plate types exhibited similar spheroid-forming capabilities and the gross patterns of growth or shrinkage during 4 days after seeding were comparable. Yet, size and eccentricity varied systematically among specific cell lines and plate types. Based on this prescreen, spheroids of HaCaT keratinocytes and HT-29 cancer cells were further assessed. In HaCaT spheroids, the in-depth analysis revealed a correlation between spheroid size, cell proliferation, and the nuclear/cytoplasm ratio of the transcriptional coactivator, YAP1, as well as an inverse correlation with respect to cell differentiation. These findings, yielded with a spheroid model and at a single-cell level, corroborate earlier concepts of the role of YAP1 in cell proliferation and differentiation of keratinocytes in human skin. Further, the results show that the plate type may influence the outcome of experimental campaigns and that it is advisable to scan different plate types for the optimal configuration during a specific investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77683849dec4dce78290876016ad7ad74750130" target='_blank'>
              A multiparametric analysis including single-cell and subcellular feature assessment reveals differential behavior of spheroid cultures on distinct ultra-low attachment plate types
              </a>
            </td>
          <td>
            Mario Vitacolonna, Roman Bruch, Ane Agaçi, Elina Nürnberg, T. Cesetti, Florian Keller, F. Padovani, Simeon Sauer, K. Schmoller, Markus Reischl, Mathias Hafner, Rüdiger Rudolf
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Accurate phenotyping of cells in the tumor microenvironment is crucial for understanding cancer biology and developing effective therapies. However, current methods require precise cell segmentations and struggle to generalize across different imaging modalities, limiting their utility in digital pathology. Here, we show that Contrastive Learning Enabled Accurate Registration of Immune and Tumor Cells (CLEAR-IT) overcomes these limitations, providing a robust and versatile tool for cell phenotyping. CLEAR-IT accurately phenotypes cells comparable to state-of-the-art methods, generalizes across multiplex imaging modalities, maintains high performance even with limited number of labels, and enables the extraction of prognostic markers. Additionally, CLEAR-IT can be combined with existing methods to boost their performance, whereas its lack of need for precise cell segmentations significantly reduces training efforts. This method enhances the robustness and efficiency of digital pathology workflows, making it a valuable tool for cancer research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02cd2bfec0cff758ded2e68209475ad537d70fc" target='_blank'>
              CLEAR-IT: Contrastive Learning to Capture the Immune Composition of Tumor Microenvironments
              </a>
            </td>
          <td>
            Daniel Spengler, Serafim Korovin, Kirti Prakash, Peter Bankhead, R. Debets, Carlas S. Smith, H. E. Balcioglu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, Ondřej Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c2390b760037ef283de4d824348d1690331e4a" target='_blank'>
              Acidity and hypoxia of tumor microenvironment, a positive interplay in extracellular vesicle release by tumor cells.
              </a>
            </td>
          <td>
            S. Peppicelli, L. Calorini, F. Bianchini, L. Papucci, Lucia Magnelli, E. Andreucci
          </td>
          <td>2024-07-18</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract While paediatric blood cancers are deadly, modern medical advances have enabled clinicians to measure levels of residual cancer cells to manage therapeutic strategies for patients. However, blood cancers, including leukaemias and lymphomas, are highly heterogeneous and is comprised of complex clonal populations that can hinder efforts in detecting the cancer cells as well as managing treatments. Furthermore, the tumour microenvironment is comprised of heterogenous immune dynamics that may be different between patients. High‐throughput sequencing has constributed to new discoveries in genetic and transcriptomic alterations underpinning cancer, including blood cancers, and has changed how patients are monitored and managed. Here we discuss the recent efforts using single‐cell approach, particularly on efforts to track clonal heterogenity of paediatric blood cancer and the underlying immune response, highlighting avenues for novel biomarker discovery that may have significant impact on clinical oncology practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9b7aa16b92f8240bd156cad7d9dc00c572051c" target='_blank'>
              Measuring single‐cell immune clonality to track haematological cancers
              </a>
            </td>
          <td>
            Amos Choo, Z. Tuong
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Interactions between tumor and stromal cells are well known to play a prominent roles in progression of pancreatic ductal adenocarcinoma (PDAC). As knowledge of stromal crosstalk in PDAC has evolved, it has become clear that cancer associated fibroblasts can play both tumor promoting and tumor suppressive roles through a combination of paracrine crosstalk and juxtacrine interactions involving direct physical contact. Another major contributor to dismal survival statistics for PDAC is development of resistance to chemotherapy drugs. Though less is known about how the acquisition of chemoresistance impacts upon tumor-stromal crosstalk. Here, we use 3D co-culture geometries to recapitulate juxtacrine interactions between epithelial and stromal cells. In particular, extracellular matrix (ECM) overlay cultures in which stromal cells (pancreatic stellate cells, or normal human fibroblasts) are placed adjacent to PDAC cells (PANC1), result in direct heterotypic cell adhesions accompanied by dramatic fibroblast contractility which leads to highly condensed macroscopic multicellular aggregates as detected using particle image velocimetry (PIV) analysis to quantify cell velocities over the course of time lapse movie sequences. To investigate how drug resistance impacts these juxtacrine interactions we contrast cultures in which PANC1 are substituted with a drug resistant subline (PANC1-OR) previously established in our lab. We find that heterotypic cell-cell interactions are highly suppressed in drug-resistant cells relative to the parental PANC1 cells. To investigate further we conduct RNA-seq and bioinformatics analysis to identify differential gene expression in PANC1 and PANC1-OR, which shows that negative regulation of cell adhesion molecules, consistent with increased epithelial mesenchymal transition (EMT), is also consistent with loss of hetrotypic cell-cell contact necessary for the contractile behavior observed in drug naïve cultures. Overall these findings elucidate the role of drug-resistance in inhibiting an avenue of stromal crosstalk which is associated with tumor suppression and also help to establish cell culture conditions useful for further mechanistic investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a4fe450dafc0e09ef1ac6571627884db5342e41" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, J. Celli
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4b7d78b4a9a56b29e32a29b6b57cf4515f6ed0" target='_blank'>
              Panoramic tumor microenvironment in pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Xiaoying Li, Wanting Hou, Chaoxin Xiao, Heqi Yang, Chengjian Zhao, Dan Cao
          </td>
          <td>2024-07-15</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background High-grade gliomas including glioblastoma (GBM) and diffuse midline gliomas (DMG) represent the most lethal and aggressive brain cancers where current treatment modalities offer limited efficacy. Chimeric antigen receptor (CAR) T cell therapies have emerged as a promising strategy, boasting tumor-specific targeting and the unique ability to penetrate the blood-brain barrier. However, the effective clinical application hinges on the optimal choice of antigen, with a limited number, currently under investigation. Methods We employed cell surface proteomic analysis of primary human high-grade glioma samples from both adult and pediatric patients. This led to the identification of Ephrin type-A receptor 3 (EphA3) as a prevalently expressed target. We engineered a second-generation EphA3-targeted CAR T cell and assessed function using in vitro and in vivo models of GBM and DMG. Results EphA3-targeted CAR T cells demonstrated robust antigen-specific killing of human GBM and DMG cell lines in vitro. In an orthotopic xenograft NSG mouse model, EphA3-targeted CAR T cells not only effectively eradicated tumors but also established a functional T cell population protective on rechallenge. Remarkably, mice rechallenged with a second contralateral orthotopic tumor implantation achieved complete tumor clearance and maintained a sustained complete response 6 months following initial treatment. Conclusion Building on the proven safety profile of EphA3 antibodies in clinical settings, our study provides compelling preclinical evidence supporting the efficacy of EphA3-targeted CAR T cells against high-grade gliomas. These findings underscore the potential for transitioning this innovative therapy into clinical trials, aiming to revolutionize the treatment landscape for patients afflicted with these formidable brain cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ccdf5eef190aeacc1b356281cbe1f1aeb78930" target='_blank'>
              EphA3-targeted chimeric antigen receptor T cells are effective in glioma and generate curative memory T cell responses
              </a>
            </td>
          <td>
            Leesa Lertsumitkul, Melinda Iliopoulos, Stacie S Wang, Sarah J McArthur, Lisa M Ebert, Alexander J Davenport, R. Endersby, Jordan R Hansford, Kate Drummond, Ryan S Cross, Misty R. Jenkins
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c025ffe7d90f79fbe29203a88da849228769c87c" target='_blank'>
              XENTURION is a population-level multidimensional resource of xenografts and tumoroids from metastatic colorectal cancer patients
              </a>
            </td>
          <td>
            S. Leto, Elena Grassi, Marco Avolio, V. Vurchio, F. Cottino, Martina Ferri, E. Zanella, Sofia Borgato, Giorgio Corti, Laura di Blasio, D. Somale, M. Vara-Messler, Francesco Galimi, F. Sassi, B. Lupo, Irene Catalano, M. Pinnelli, Marco Viviani, Luca Sperti, A. Mellano, Alessandro Ferrero, C. C. Zingaretti, A. Puliafito, Luca Primo, A. Bertotti, L. Trusolino
          </td>
          <td>2024-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Tissue-resident macrophages (TRMs) are an integral part of the innate immune system, but their biology is not well understood in the context of cancer. Distinctive resident macrophage populations are identified in different organs in mice using fate mapping studies. They develop from the yolk sac and self-maintain themselves lifelong in specific tissular niches. Similarly, breast-resident macrophages are part of the mammary gland microenvironment. They reside in the breast adipose tissue stroma and close to the ductal epithelium and help in morphogenesis. In breast cancer, TRMs may promote disease progression and metastasis; however, precise mechanisms have not been elucidated. TRMs interact intimately with recruited macrophages, cytotoxic T cells, and other immune cells along with cancer cells, deciding further immunosuppressive or cytotoxic pathways. Moreover, triple-negative breast cancer (TNBC), which is generally associated with poor outcomes, can harbor specific TRM phenotypes. The influence of TRMs on adipose tissue stroma of the mammary gland also contributes to tumor progression. The complex crosstalk between TRMs with T cells, stroma, and breast cancer cells can establish a cascade of downstream events, understanding which can offer new insight for drug discovery and upcoming treatment choices. This review aims to acknowledge the previous research done in this regard while exploring existing research gaps and the future therapeutic potential of TRMs as a combination or single agent in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71fe245b89f20276baee1edbe89f69f9bb9748d9" target='_blank'>
              Understanding tissue-resident macrophages unlocks the potential for novel combinatorial strategies in breast cancer
              </a>
            </td>
          <td>
            Manjusha Biswas
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Keratinocytes, the dominant cell type in the melanoma microenvironment during tumor initiation, exhibit diverse effects on melanoma progression. Using a zebrafish model of melanoma and human cell co-cultures, we observed that keratinocytes undergo an Epithelial–Mesenchymal Transition (EMT)-like transformation in the presence of melanoma, reminiscent of their behavior during wound healing. Surprisingly, overexpression of the EMT transcription factor Twist in keratinocytes led to improved overall survival in zebrafish melanoma models, despite no change in tumor initiation rates. This survival benefit was attributed to reduced melanoma invasion, as confirmed by human cell co-culture assays. Single-cell RNA-sequencing revealed a unique melanoma cell cluster in the Twist-overexpressing condition, exhibiting a more differentiated, less invasive phenotype. Further analysis nominated homotypic jam3b-jam3b and pgrn-sort1a interactions between Twist-overexpressing keratinocytes and melanoma cells as potential mediators of the invasive restraint. Our findings suggest that EMT in the tumor microenvironment (TME) may limit melanoma invasion through altered cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8055a65d64a36749562fed897a59067bde3fe48" target='_blank'>
              Restraint of melanoma progression by cells in the local skin environment
              </a>
            </td>
          <td>
            Yilun Ma, Mohita M. Tagore, Miranda V. Hunter, Ting-Hsiang Huang, Emily D. Montal, Joshua M. Weiss, Richard M. White
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background CD24 plays a crucial role not only in promoting tumor progression and metastasis but also in modulating macrophage-mediated anti-tumor immunity. However, the impact of tumor CD24 on the immune landscape of the tumor microenvironment (TME) remains poorly explored. Here, we investigated the role of CD24a, murine CD24 gene, in the progression and immune dynamics of the tumor microenvironment (TME) in the 4T1 murine model of triple-negative breast cancer (TNBC). Methods: We employed Clustered Regularly Interspaced Short Palindromic Repeat (CRISPR)/Cas9 technology to perform a gene knockout of Cd24a in 4T1 cells. Flow cytometry was utilized to analyze the distribution and number of immune cells, including myeloid-derived suppressor cells (MDSCs), natural killer (NK) cells, T cells, and macrophages, within tumors, spleens, and bone marrow. Immunofluorescence (IF) staining was used to detect these immune cells in tumor sections. Additionally, ANDOR Dragonfly High-Speed Confocal was used to perform three-dimensional (3D) mapping of mouse tumors. Results Our study showed that knocking out CD24a significantly impeded tumor progression and prolonged mouse survival. Flow cytometry and IF analysis revealed that the loss of CD24a transformed tumor microenvironment from cold to hot by promoting the infiltration of M1 macrophages, cytotoxic CD8+ T cells, and CD49b+ natural killer (NK) cells while reducing the recruitment and expansion of granulocytic myeloid-derived suppressor cells (gMDSCs) in the TME. Additionally, the 3D mapping of TME further validated the “hot state” of CD24a knockout tumors. Conclusions Our study provides the first evidence that targeting CD24a could effectively reprogram the TME, enhancing its immunogenicity, and transforming immune cold tumors into hot tumors. This strategy may offer a promising therapeutic strategy for enhancing the immune response against poorly immunogenic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f888be94df7f979fefb9bdbfa437c0f898437cb" target='_blank'>
              CD24a knockout transforms the tumor microenvironment from cold to hot by promoting tumor-killing immune cell infiltration in a murine triple-negative breast cancer model
              </a>
            </td>
          <td>
            Shih-Hsuan Chan, Hsuan-Jung Tseng, Lu-Hai Wang
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer bone metastasis is the leading cause of mortality in patients with advanced breast cancer. Although decreased mineral density is a known risk factor for bone metastasis, the underlying mechanisms remain poorly understood because studying the isolated effect of bone mineral density on tumor heterogeneity is challenging with conventional approaches. Here, we investigate how bone mineral content affects tumor growth and microenvironmental complexity in vivo by combining single-cell RNA-sequencing with mineral-containing or mineral-free decellularized bone matrices. We discover that the absence of bone mineral significantly influences fibroblast and immune cell heterogeneity, promoting phenotypes that increase tumor growth and alter the response to injury or disease. Importantly, we observe that the stromal response to matrix mineral content depends on host immunocompetence and the murine tumor model used. Collectively, our findings suggest that bone mineral density affects tumor growth by altering microenvironmental complexity in an organism-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29d79f5245668a68eb4e20b21ea961a72baab370" target='_blank'>
              Bone mineral density affects tumor growth by shaping microenvironmental heterogeneity
              </a>
            </td>
          <td>
            Matthew A. Whitman, Madhav Mantri, Emmanuel Spanos, L. Estroff, I. De Vlaminck, Claudia Fischbach
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Metastases from gastric adenocarcinoma (GAC) lead to high morbidity and mortality. Developing innovative and effective therapies requires a comprehensive understanding of the tumor and immune biology of advanced GAC. Yet, collecting matched specimens from advanced, treatment-naïve GAC patients poses a significant challenge, limiting the scope of current research, which has predominantly focused on localized tumors. This gap hinders a deeper insight into the metastatic dynamics of GAC.


METHODS
We performed in-depth single-cell transcriptome and immune profiling on 68 paired, treatment-naïve, primary-metastatic tumors to delineate alterations in cancer cells and their tumor microenvironment (TME) during metastatic progression. To validate our observations, we conducted comprehensive functional studies both in vitro and in vivo, employing cell lines, multiple PDX and novel mouse models of GAC.


RESULTS
Liver and peritoneal metastases exhibited distinct properties in cancer cells and dynamics of TME phenotypes, supporting the notion that cancer cells and their local TMEs co-evolve at metastatic sites. Our study also revealed differential activation of cancer meta-programs across metastases. We observed evasion of cancer cell ferroptosis via GPX4 upregulation during GAC progression. Conditional depletion of Gpx4 or pharmacological inhibition of ferroptosis resistance significantly attenuated tumor growth and metastatic progression. Additionally, ferroptosis-resensitizing treatments augmented the efficacy of CAR T-cell therapy.


CONCLUSIONS
This study represents the largest single-cell dataset of metastatic GACs to date. High-resolution mapping of the molecular and cellular dynamics of GAC metastasis has revealed a rationale for targeting ferroptosis defense in combination with CAR T-cell therapy as a novel therapeutic strategy with potential immense clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c4774948f3c82306cdc21b3d5730989f11741b" target='_blank'>
              Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response.
              </a>
            </td>
          <td>
            Xiangdong Cheng, Enyu Dai, Jibo Wu, Natasha M. Flores, Yanshuo Chu, Ruiping Wang, M. Dang, Zhiyuan Xu, Guangchun Han, Yunhe Liu, D. Chatterjee, Can Hu, Jieer Ying, Yi-an Du, Litao Yang, Xiaoqing Guan, Shaowei Mo, Xuanye Cao, Guangsheng Pei, Jiahui Jiang, Xiaoyin Lu, Ana Morales Benitez, R. Waters, Melissa P Pizzi, N. Shanbhag, Yibo Fan, F. Peng, Samir Hanash, George A. Calin, Andrew Futreal, Shumei Song, Cassian Yee, Pawel K Mazur, Jiang-Jiang Qin, J. Ajani, Linghua Wang
          </td>
          <td>2024-08-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="Joint analysis of transcriptomic and T cell receptor (TCR) features at single-cell resolution provides a powerful approach for in-depth T cell immune function research. Here, we introduce a deep learning framework for single-T cell transcriptome and receptor analysis, MIST (Multi-Insight for T cell). MIST features three latent spaces: gene expression, TCR, and a joint latent space. Through analyses of antigen- specific T cells and T cells related to lung cancer immunotherapy, we demonstrate MIST’s interpretability and flexibility. MIST easily and accurately resolves cell function and antigen-specificity by vectorizing and integrating transcriptome and TCR data of T cells. In addition, using MIST, we identified the heterogeneity of CXCL13+ subsets in lung cancer infiltrating CD8+ T cells and their association with immunotherapy, providing additional insights into the functional transition of CXCL13+ T cells related to anti-PD-1 therapy that were not reported in the original study. MIST is available at https://github.com/aapupu/MIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4cb87f8bfc27d222d60016bd4d339891023ace7" target='_blank'>
              MIST: an interpretable and flexible deep learning framework for single-T cell transcriptome and receptor analysis
              </a>
            </td>
          <td>
            Wenpu Lai, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The functional heterogeneity and ecological niche of prostate cancer stem cells (PCSCs), which are major drivers of prostate cancer development and treatment resistance, have attracted considerable research attention. Cancer-associated fibroblasts (CAFs), which are crucial components of the tumor microenvironment (TME), substantially affect PCSC stemness. Additionally, CAFs promote PCSC growth and survival by releasing signaling molecules and modifying the surrounding environment. Conversely, PCSCs may affect the characteristics and behavior of CAFs by producing various molecules. This crosstalk mechanism is potentially crucial for prostate cancer progression and the development of treatment resistance. Using organoids to model the TME enables an in-depth study of CAF-PCSC interactions, providing a valuable preclinical tool to accurately evaluate potential target genes and design novel treatment strategies for prostate cancer. The objective of this review is to discuss the current research on the multilevel and multitarget regulatory mechanisms underlying CAF-PCSC interactions and crosstalk, aiming to inform therapeutic approaches that address challenges in prostate cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebc78492c002bb6ca79859426afb09d4c0e6ab75" target='_blank'>
              Cancer-associated fibroblasts and prostate cancer stem cells: crosstalk mechanisms and implications for disease progression
              </a>
            </td>
          <td>
            Haoran Chen, Suping Fang, Xudong Zhu, Hao Liu
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b25907582c98ee1f4e8b1fa5c49639650f918959" target='_blank'>
              Endosomal protein DENND10/FAM45A integrates extracellular vesicle release with cancer cell migration
              </a>
            </td>
          <td>
            Shenqing Sun, Qian Li, Ganggang Liu, Xiaoheng Huang, Aiqing Li, Haoran Guo, Lijuan Qi, Jie Zhang, Jianrui Song, Xiong Su, Yanling Zhang
          </td>
          <td>2024-07-10</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are ectopic lymphoid aggregates associated with improved prognosis in numerous tumors. To date, their prognostic value in central nervous system cancers and the events underlying their formation remain unclear. Here, we find that TLS correlate with improved survival in glioblastoma patients. Furthermore, combining spatial transcriptomics of human tissues with longitudinal studies in murine glioma, we establish that the assembly of T cell-rich clusters is a prerequisite for the development of canonical TLS, and show that CD4 T cells play a central role in this process. Indeed, we provide evidence that IL7R+CCR7+ Th1 lymphocytes with lymphoid tissue-inducing potential are recruited to TLS nucleation sites, where they can initiate TLS assembly by expressing lymphotoxin β. Our work defines TLS as a potential prognostic factor in glioblastoma and identifies cellular and molecular mechanisms of TLS formation. These findings have broader implications for the development of TLS-inducing therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368cd006ed1ae7b496e449db1b6253ab2d80a3fe" target='_blank'>
              Cellular and molecular events organizing the assembly of tertiary lymphoid structures in glioblastoma
              </a>
            </td>
          <td>
            Alessandra Vaccaro, Fan Yang, Tiarne van de Walle, Saskia Franke, R. Lugano, A. Dénes, Anastasia Magoulopoulou, Inés Cid-Fariña, Anja Smits, Lene Uhrbom, Sylwia Libard, Francesco Latini, Magnus Essand, Mats Nilsson, Liqun He, T. Olsson Bontell, A. Jakola, Mohanraj Ramachandran, A. Dimberg
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, E. Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Glioblastomas are known for their poor clinical prognosis, with recurrent tumors often exhibiting greater invasiveness and faster growth rates compared to primary tumors. To understand the intratumoral changes driving this phenomenon, we employed single-cell sequencing to analyze the differences between two pairs of primary and recurrent glioblastomas. Our findings revealed an upregulation of ferroptosis in endothelial cells within recurrent tumors, identified by the significant overexpression of the NOX4 gene. Further analysis indicated that knocking down NOX4 in endothelial cells reduced the activity of the ferroptosis pathway. Utilizing conditioned media from endothelial cells with lower ferroptosis activity, we observed a decrease in the growth rate of glioblastoma cells. These results highlighted the complex role of ferroptosis within tumors and suggested that targeting ferroptosis in the treatment of glioblastomas requires careful consideration of its effects on endothelial cells, as it may otherwise produce counterproductive outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb06c19dc2e1dad39392c5e5a73da1e9d0d09630" target='_blank'>
              Endothelial cell ferroptosis influences IDH wild-type glioblastoma growth in recurrent glioblastoma multiforme patients
              </a>
            </td>
          <td>
            Bo Liang, Xinghuan Ding, Siyuan Yang, Enshan Feng
          </td>
          <td>2024-07-08</td>
          <td>Brazilian Journal of Medical and Biological Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Single-cell sequencing technologies have revolutionized biomedical research by enabling deconvolution of cell type-specific properties in highly heterogeneous tissue. While robust tools have been developed to handle bioinformatic challenges posed by single-cell RNA and ATAC data, options for emergent modalities such as methylation are much more limited, impeding the utility of results. Here we present Amethyst, a comprehensive R package for atlas-scale single-cell methylation sequencing data analysis. Amethyst begins with base-level methylation calls and expedites batch integration, doublet detection, dimensionality reduction, clustering, cell type annotation, differentially methylated region calling, and interpretation of results, facilitating rapid data interaction in a local environment. We introduce the workflow using published single-cell methylation human peripheral blood mononuclear cell (PBMC) and human cortex data. We further leverage Amethyst on an atlas-scale brain dataset to discover a noncanonical methylation pattern in human astrocytes and oligodendrocytes, challenging prior assumptions that this form of methylation is only biologically relevant to neurons in the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/345bc3f744d769c3b98a5c6e3d9a3dfc3016fecd" target='_blank'>
              Single-cell DNA methylation analysis tool Amethyst reveals distinct noncanonical methylation patterns in human glial cells
              </a>
            </td>
          <td>
            Lauren E. Rylaarsdam, Ruth V. Nichols, Brendan L O'Connell, Stephen Coleman, G. Yardımcı, Andrew C Adey
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Since the introduction of the first immune checkpoint inhibitor (ICI), immunotherapy has changed the landscape of molecular therapeutics for cancers. However, ICIs do not work equally well on all cancers and for all patients. There has been a growing interest in using mathematical and computational models to optimize clinical responses. Ordinary differential equations (ODEs) have been widely used for mechanistic modeling in immuno-oncology and immunotherapy because they allow rapid simulations of temporal changes in the cellular and molecular populations involved. Nonetheless, ODEs cannot describe the spatial structure in the tumor microenvironment or quantify the influence of spatially-dependent characteristics of tumor-immune dynamics. For these reasons, agent-based models (ABMs) have gained popularity because they can model more detailed phenotypic and spatial heterogeneity that better reflect the complexity seen in vivo. In the context of anti-PD-1 ICIs, we compare treatment outcomes simulated from an ODE model and an ABM to show the importance of including spatial components in computational models of cancer immunotherapy. We consider tumor cells of high and low antigenicity and two distinct cytotoxic T lymphocyte (CTL) killing mechanisms. The preferred mechanism differs based on the antigenicity of tumor cells. Our ABM reveals varied phenotypic shifts within the tumor and spatial organization of tumor and CTLs, despite similarities in key immune parameters, initial conditions of simulation, and early temporal trajectories of the cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38a51133539694b984ad15aa283204f18aa1920" target='_blank'>
              Agent-Based Modeling of Virtual Tumors Reveals the Critical Influence of Microenvironmental Complexity on Immunotherapy Efficacy
              </a>
            </td>
          <td>
            Yixuan Wang, Daniel R. Bergman, Erica Trujillo, Anthony A. Fernald, Lie Li, Alexander T. Pearson, Randy F. Sweis, Trachette Jackson
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Targeting the tumor microenvironment (TME) is an attractive strategy for cancer therapy, as tumor cells in vivo are surrounded by many different influential cell types, with complex interactions strongly affecting tumor progression and therapeutic outcome. Cancer-associated fibroblasts (CAFs) represent an abundant stromal cell type in the TME that modulate tumor development by exerting an immunosuppressive effect to influence effector immune cell activation. One promising target for TME-directed therapy is the CAF marker fibroblast activation protein-α (FAP). In this study, we employ a multicellular three-dimensional (3D) spheroid model, including tumor cells, fibroblast cells, and naïve T cells and could observe a protective effect of fibroblasts on tumor cells. Subsequently, we demonstrate that fibroblasts express FAP at differing expression levels in two-dimensional (2D) versus 3D cells. Lastly, we show that in a triple-culture of tumor cells, T cells and fibroblasts, the simultaneous assembly of fibroblasts using the high-affinity ligand oncoFAP with an engineered α-CD3-scFv-Fc-dextran-oncoFAP construct resulted in effective T cell activation to augment immunogenicity. Overall, this model can be routinely used for preclinical screening to study the effects of fibroblasts on the TME in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded138adba947a8cd32556d8f5f10db81c5760a" target='_blank'>
              Mimicking the immunosuppressive impact of fibroblasts in a 3D multicellular spheroid model
              </a>
            </td>
          <td>
            Melanie Grotz, Lieke van Gijzel, Peter Bitsch, Stefania C. Carrara, Harald Kolmar, Sakshi Garg
          </td>
          <td>2024-07-26</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tumor microenvironment (TME) contains cells that regulate medication response and cancer growth in a major way. Tumor immunology research has been rejuvenated and cancer treatment has been changed by immunotherapy, a rapidly developing therapeutic approach. The growth patterns of tumor cells in vivo and the heterogeneity, complexity, and individuality of tumors produced from patients are not reflected in traditional two-dimensional tumor cell profiles. On the other hand, an in vitro three-dimensional (3D) model called the organoid model is gaining popularity. It can replicate the physiological and pathological properties of the original tissues in vivo. Tumor cells are the source of immune organoids. The TME characteristics can be preserved while preserving the variety of tumors by cultivating epithelial tumor cells with various stromal and immunological components. In addition to having genetic and physical similarities to human diseases and the ability to partially reconstruct the complex structure of tumors, these models are now widely used in research fields including cancer, developmental biology, regenerative mechanisms, drug development, disease modeling, and organ transplantation. This study reviews the function of organoids in immunotherapy and the tumor immune milieu. We also discuss current developments and suggest translational uses of tumor organoids in immuno-oncology research, immunotherapy modeling, and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe089d8eafbd615edce2fa5a415d30642e85d5cf" target='_blank'>
              Organoids: new frontiers in tumor immune microenvironment research
              </a>
            </td>
          <td>
            Yujia Yang, Jinlei Cui, Yajie Kong, Yu Hou, Cuiqing Ma
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multimodal analysis of single-cell samples from healthy and diseased tissues at various stages provides a comprehensive view that identifies disease-specific cells, their molecular features and aids in patient stratification. Here, we present MultiMIL, a novel weakly-supervised multimodal model designed to construct multimodal single-cell references and prioritize phenotype-specific cells via patient classification. MultiMIL effectively integrates single-cell modalities, even when they only partially overlap, providing robust representations for downstream analyses such as phenotypic prediction and cell prioritization. Using a multiple-instance learning approach, MultiMIL aggregates cell-level measurements into sample-level representations and identifies disease-specific cell states through attention-based scoring. We demonstrate that MultiMIL accurately identifies disease-specific cell states in blood and lung samples, identifying novel disease-associated genes and achieving superior patient classification accuracy compared to existing methods. We anticipate MultiMIL will become an essential tool for querying single-cell multiomic atlases, enhancing our understanding of disease mechanisms and informing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8efc9b9ff38309e496bbcf1bd6af1cf5075840" target='_blank'>
              Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases
              </a>
            </td>
          <td>
            Anastasia Litinetskaya, Maiia Shulman, Soroor Hediyeh-zadeh, A. Moinfar, Fabiola Curion, Artur Szałata, Alireza Omidi, M. Lotfollahi, F. Theis
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562977a8b24cea0e019cd93dcf3b43822ec9b974" target='_blank'>
              Leveraging neighborhood representations of single-cell data to achieve sensitive DE testing with miloDE
              </a>
            </td>
          <td>
            A. Missarova, Emma Dann, Leah Rosen, Rahul Satija, J. Marioni
          </td>
          <td>2024-07-18</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Predicting the transcriptional response of chemical perturbations is crucial to understanding gene function and developing drug candidates, promising a streamlined drug development process. Single-cell sequencing has provided an ideal data basis for training machine learning models for this task. Recent advances in deep learning have led to significant improvements in predictions of chemical as well as genetic perturbations at the single cell level. Experiments have shown that different cell types exhibit distinct transcriptional patterns and responses to perturbation. This poses a fundamental problem for predicting transcriptional responses of drugs or cell types outside the training data. Accordingly, existing methods lack cell-type-specific modeling or do not explicitly provide an interpretable mechanism for the gene features. In this study, we introduce a novel approach that employs a network representation of various cell types as an inductive bias, improving prediction performance in scenarios with limited data while acknowledging cellular differences. We applied our framework to four small-scale single-cell perturbation datasets and one large-scale screening experiment, demonstrating that this representation can inherently generalize to previously unseen cell types. Furthermore, our method outperforms the state-of-the-art methods in predicting the post-perturbation response in unobserved cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/800792362ef249f4ba2c5bd2c2590459a14303c2" target='_blank'>
              PrePR-CT: Predicting Perturbation Responses in Unseen Cell Types Using Cell-Type-Specific Graphs
              </a>
            </td>
          <td>
            Reem Alsulami, Robert Lehmann, S. Khan, Vincenzo Lagani, D. Gómez-Cabrero, N. Kiani, Jesper N. Tegnér
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e281f15c1b3ddbddd2f378f32dbc2fd4db79f0e" target='_blank'>
              Transcriptomic analysis unveils bona fide molecular signatures of microglia under conditions of homeostasis and viral encephalitis
              </a>
            </td>
          <td>
            Felix Mulenge, Olivia Luise Gern, Lena Mareike Busker, Angela Aringo, L. Ghita, I. Waltl, A. Pavlou, Ulrich Kalinke
          </td>
          <td>2024-08-17</td>
          <td>Journal of Neuroinflammation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Lung cancer is primarily a disease of the elderly. Despite shared molecular changes between aging and cancer 1 – such as permissive chromatin states and deregulated protein homeostasis – studies on physiologically aged models of human lung cancer are lacking. Here, we show that aging alters the progression of KRAS-driven non-small cell lung cancer (NSCLC), promoting metastasis while suppressing primary lung tumor growth. Clinically, a multicenter analysis of all consecutively diagnosed NSCLC cases in Western Sweden over a 3-year period confirmed increased metastasis and smaller primary tumor size with age in KRAS-driven NSCLC. In addition, primary lung tumor cultures derived from older mice demonstrated an increased metastatic phenotype. Unbiased transcriptomic and epigenomic analyses identified ATF4, a major arm of the unfolded protein response (UPR), as a driver of aging-induced lung cancer metastasis. Furthermore, we found that the age-associated increase in ATF4 fuels metastatic dissemination through metabolic rewiring, including increased glutaminolysis. Finally, we report that pharmacological inhibition of glutaminase effectively suppressed aging-induced metastasis. Our findings suggest a novel adjuvant therapy for human lung cancer by targeting aging-induced metabolic plasticity, highlighting the need to consider the biology of aging in the development of cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5969595822b915427a09087a832f53b7a65435" target='_blank'>
              Aging promotes lung cancer metastasis through epigenetic ATF4 induction
              </a>
            </td>
          <td>
            A. Patel, Jozefina Dzanan, Kevin X. Ali, E. Eklund, Samantha W. Alvarez, Ilayda Altinönder, Dorota Raj, A. E. El Zowalaty, Sama I. Sayin, Martin Dankis, Maria Schwarz, Emma Jonasson, Kristell Le Gal, Heba Albatrok, R. O. Bagge, Anetta Härtlova, Anders Ståhlberg, Andreas Hallqvist, Clotilde Wiel, V. Sayin
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Development of subtype heterogeneity in breast cancer requires stochastic transitions between cell types. Here, we sought to characterize how NF-κB contributes to breast cancer progression. We built a gene regulatory network model and calibrated it to the expression levels of NF-κB, TWIST1, SIP1, and SLUG in HER2+ and TNBC cell line, using patient and in vitro published data for validation. The model’s epigenetic landscape exhibits two attractor basins separated by a fluctuation-susceptible slow route. NF-κB stochastic fluctuations induce irreversible transitions from HER2+ to TNBC basins at different times, increasing heterogeneity. Mutations altering the availability of NF-κB change the size of the subtype basins, changing the transition probabilities. Our findings enhance the established attractor landscape formulation and deepen understanding of breast cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8337b6fd94c2dc5d7610f3b772efc6d10bcc115" target='_blank'>
              NF-κB Epigenetic Attractor Landscape Drives Breast Cancer Heterogeneity
              </a>
            </td>
          <td>
            Francisco Lopes, B. Pires, Alexandre A. B. Lima, R. Binato, Eliana Abdelhay
          </td>
          <td>2024-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e78446518da6c71cd38b60ad342558f8d623b65" target='_blank'>
              Medullary thyroid cancer: single-cell transcriptome and tumor evolution
              </a>
            </td>
          <td>
            Li-feng Wang, , Fang Yuan, Kai-wen Fu, Yongpeng He, Rui Chen
          </td>
          <td>2024-08-30</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gallbladder carcinoma (GBC) is a malignant hepatobiliary cancer characterized by an intricate tumor microenvironments (TME) and heterogeneity. The traditional GBC 2D culture models cannot faithfully recapitulate the characteristics of the TME. Three-dimensional (3D) bioprinting enables the establishment of high-throughput and high-fidelity multicellular GBC models. In this study, we designed a concentric cylindrical tetra-culture model to reconstitute the spatial distribution of cells in tumor tissue, with the inner portion containing GBC cells, and the outer ring containing a mixture of endothelial cells, fibroblasts, and macrophages. We confirmed the survival, proliferation, biomarker expression and gene expression profiles of GBC 3D tetra-culture models. Hematoxylin-eosin (HE) and immunofluorescence staining verified the morphology and robust expression of GBC/endothelial/fibroblast/macrophage biomarkers in GBC 3D tetra-culture models. Single-cell RNA sequencing revealed two distinct subtypes of GBC cells within the model, glandular epithelial and squamous epithelial cells, suggesting the mimicry of intratumoral heterogeneity. Comparative transcriptome profile analysis among various in vitro models revealed that cellular interactions and the TME in 3D tetra-culture models reshaped the biological processes of tumor cells to a more aggressive phenotype. GBC 3D tetra-culture models restored the characteristics of the TME as well as intratumoral heterogeneity. Therefore, this model is expected to have future applications in tumor biology research and antitumor drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc6d2cc903e389bd57ed19225ad293a9f8472b5b" target='_blank'>
              Multicellular 3D bioprinted human gallbladder carcinoma forin vitromimicry of tumor microenvironment and intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Yukai Jin, Jiangang Zhang, Jiali Xing, Yiran Li, Huiyu Yang, Liujian Ouyang, Zhiyuan Fang, Lejia Sun, Bao Jin, Pengyu Huang, Huayu Yang, Shunda Du, X. Sang, Yilei Mao
          </td>
          <td>2024-08-09</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Anticancer immunity is predicated on leukocyte migration into tumors. Once recruited, leukocytes undergo substantial reprogramming to adapt to the tumor microenvironment. A major challenge in the field is distinguishing recently recruited from resident leukocytes in tumors. In this study, we developed an intravascular Ab technique to label circulating mouse leukocytes before they migrate to tissues, providing unprecedented insight into the kinetics of recruitment. This approach unveiled the substantial role of leukocyte migration in tumor progression using a preclinical mouse model of lung adenocarcinoma. Regulatory T cells (Tregs), critical mediators of immunosuppression, were continuously and rapidly recruited into tumors throughout cancer progression. Moreover, leukocyte trafficking depended on the integrins CD11a/CD49d, and CD11a/CD49d blockade led to significant tumor burden reduction in mice. Importantly, preventing circulating Treg recruitment through depletion or sequestration in lymph nodes was sufficient to decrease tumor burden, indicating that Treg migration was crucial for suppressing antitumor immunity. These findings underscore the dynamic nature of the immune compartment within mouse lung tumors and demonstrate the relevance of a temporal map of leukocyte recruitment into tumors, thereby advancing our understanding of leukocyte migration in the context of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feab8b259d671cb0922cd43660c7992fa44d78d3" target='_blank'>
              Longitudinal Intravascular Antibody Labeling Identified Regulatory T Cell Recruitment as a Therapeutic Target in a Mouse Model of Lung Cancer.
              </a>
            </td>
          <td>
            Sean-Luc Shanahan, Nikesh Kunder, Charles Inaku, Natalie B Hagan, Grace Gibbons, Nicolas Mathey-Andrews, G. Anandappa, Shawn Soares, Kristen E. Pauken, Tyler Jacks, Jason M. Schenkel
          </td>
          <td>2024-07-31</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="3D bioprinting has become a valuable tool for studying the biology of solid tumors, including glioblastoma multiforme (GBM). Our analysis of publicly available bulk RNA and single-cell sequencing data has allowed us to define the chemotactic profile of GBM tumors and identify the cell types that secrete particular chemokines in the GBM tumor microenvironment (TME). Our findings indicate that primary GBM tissues express multiple chemokines, whereas spherical monocultures of GBM cells significantly lose this diversity. Subsequently, the comparative analysis of GBM spherical monocultures vs. 3D-bioprinted multicultures of cells showed a restoration of chemokine profile diversity in 3D-bioprinted cultures. Furthermore, single-cell RNA-Seq analysis showed that cells of the perivascular niche (pericytes and endocytes) express multiple chemokines in the GBM TME. Next, we 3D-bioprinted cells from two glioblastoma cell lines, U-251 and DK-MG, alone and as co-cultures with mesenchymal stromal cells (representing cells of the perivascular niche) and assessed the chemokine secretome. The results clearly demonstrated that the interaction of tumors and mesenchymal cells leads to in a significant increase in the repertoire and levels of secreted chemokines under culture in 21% O2 and 1% O2. Our study indicates that cells of the perivascular niche may perform a substantial role in shaping the chemokine microenvironment in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae245299abc68f53e5e3fd8e4dc327ea2b0cb63" target='_blank'>
              3D-Bioprinted Co-Cultures of Glioblastoma Multiforme and Mesenchymal Stromal Cells Indicate a Role for Perivascular Niche Cells in Shaping Glioma Chemokine Microenvironment
              </a>
            </td>
          <td>
            K. Zielniok, Kinga Rusinek, A. Słysz, Mieszko Lachota, Ewa Bączyńska, Natalia Wiewiórska-Krata, Anna Szpakowska, Martyna Ciepielak, B. Foroncewicz, K. Mucha, Radosław Zagożdżon, Zygmunt Pojda
          </td>
          <td>2024-08-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, Jian-Qiang Lu, John S Lazo, T. Kislinger, Yu Lu, Kevin R. Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Human cytomegalovirus (HCMV) infection is common in tumor tissues across different types of cancer. While HCMV has not been recognized as a cancer-causing virus, numerous studies hint at its potential role in cancer development where its presence in various cancers corresponds with the hallmarks of cancer. Herein, we discuss and demonstrate that high-risk HCMV-DB and BL strains have the potential to trigger transformation in epithelial cells, including human mammary epithelial cells (HMECs), ovarian epithelial cells (OECs), and prostate epithelial cells (PECs), through the generation of polyploid giant cancer cells (PGCCs). A discussion is provided on how HCMV infection creates a cellular environment that promotes oncogenesis, supporting the continuous growth of CMV-transformed cells. The aforementioned transformed cells, named CTH, CTO, and CTP cells, underwent giant cell cycling with PGCC generation parallel to dedifferentiation, displaying stem-like characteristics and an epithelial–mesenchymal transition (EMT) phenotype. Furthermore, we propose that giant cell cycling through PGCCs, increased EZH2 expression, EMT, and the acquisition of malignant traits represent a deleterious response to the cellular stress induced by high-risk oncogenic HCMV strains, the latter being the origin of the transformation process in epithelial cells upon HCMV infection and leading to adenocarcinoma of poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139ca38ce75a83fdf75794021da421194cdfffd" target='_blank'>
              Polyploid Giant Cancer Cells: A Distinctive Feature in the Transformation of Epithelial Cells by High-Risk Oncogenic HCMV Strains
              </a>
            </td>
          <td>
            Georges Herbein, Ranim El Baba
          </td>
          <td>2024-07-31</td>
          <td>Viruses</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity and tissue transcriptomic complexity. However, the high frequency of dropout events in scRNA-seq data complicates downstream analyses such as cell type identification and trajectory inference. Existing imputation methods address the dropout problem but face limitations such as high computational cost and risk of over-imputation. We present SmartImpute, a novel computational framework designed for targeted imputation of scRNA-seq data. SmartImpute focuses on a predefined set of marker genes, enhancing the biological relevance and computational efficiency of the imputation process while minimizing the risk of model misspecification. Utilizing a modified Generative Adversarial Imputation Network architecture, SmartImpute accurately imputes the missing gene expression and distinguishes between true biological zeros and missing values, preventing overfitting and preserving biologically relevant zeros. To ensure reproducibility, we also provide a function based on the GPT4 model to create target gene panels depending on the tissue types and research context. Our results, based on scRNA-seq data from head and neck squamous cell carcinoma and human bone marrow, demonstrate that SmartImpute significantly enhances cell type annotation and clustering accuracy while reducing computational burden. Benchmarking against other imputation methods highlights SmartImpute’s superior performance in terms of both accuracy and efficiency. Overall, SmartImpute provides a lightweight, efficient, and biologically relevant solution for addressing dropout events in scRNA-seq data, facilitating deeper insights into cellular heterogeneity and disease progression. Furthermore, SmartImpute’s targeted approach can be extended to spatial omics data, which also contain many missing values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6236ffea74eeb68e587d5adb9d7ee59d89926243" target='_blank'>
              SmartImpute: A Targeted Imputation Framework for Single-cell Transcriptome Data
              </a>
            </td>
          <td>
            Sijie Yao, Xiaoqing Yu, Xuefeng Wang
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction Intra-tumoral B cells mediate a plethora of immune effector mechanisms with key roles in anti-tumor immunity and serve as positive prognostic indicators in a variety of solid tumor types, including epithelial ovarian cancer (EOC). Several aspects of intra-tumoral B cells remain unclear, such as their state of activation, antigenic repertoires, and capacity to mature into plasma cells. Methods B lymphocytes were isolated from primary EOC tissue and malignant ascites and were maintained in cell culture medium. The stably maintained cell lines were profiled with flow cytometry and B cell receptor sequencing. Secreted antibodies were tested with a human proteome array comprising more than 21,000 proteins, followed by ELISA for validation. Originating tumor samples were used for spatial profiling with chip cytometry. Results Antibody-secreting B lymphocytes were isolated from the ovarian tumor microenvironment (TME) of four different EOC patients. The highly clonal cell populations underwent spontaneous immortalization in vitro, were stably maintained in an antibody-secreting state, and showed presence of Epstein-Barr viral (EBV) proteins. All originating tumors had high frequency of tumor-infiltrating B cells, present as lymphoid aggregates, or tertiary lymphoid structures. The antigens recognized by three of the four cell lines are coil-coil domain containing protein 155 (CCDC155), growth factor receptor-bound protein 2 (GRB2), and pyruvate dehydrogenase phosphatase2 (PDP2), respectively. Anti-CCDC155 circulating IgG antibodies were detected in 9 of 20 (45%) of EOC patients’ sera. Tissue analyses with multiparameter chip cytometry shows that the antibodies secreted by these novel human B cell lines engage their cognate antigens on tumor cells. Discussion These studies demonstrate that within the tumor-infiltrating lymphocyte population in EOC resides a low frequency population of antibody-secreting B cells that have been naturally exposed to EBV. Once stably maintained, these novel cell lines offer unique opportunities for future studies on intratumor B cell biology and new target antigen recognition, and for studies on EBV latency and/or viral reactivation in the TME of non-EBV related solid tumors such as the EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91568ddf7942710ae6ed5b0bc9abf380b3db7b97" target='_blank'>
              Characterization of latently infected EBV+ antibody-secreting B cells isolated from ovarian tumors and malignant ascites
              </a>
            </td>
          <td>
            Lixin Zhang, Mary Strange, E. Elishaev, Syed Zaidi, F. Modugno, M. Radolec, Robert P Edwards, Olivera J Finn, A. Vlad
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a3cd7f4d8e503d53aded204c426d8531003ab1" target='_blank'>
              State of the Art Modelling of the Breast Cancer Metastatic Microenvironment: Where Are We?
              </a>
            </td>
          <td>
            Mia Nuckhir, David Withey, Sara Cabral, H. Harrison, Robert B. Clarke
          </td>
          <td>2024-07-16</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Colorectal cancer is a predominant malignancy with a second mortality worldwide. Despite its prevalence, therapeutic options remain constrained and surgical operation is still the most useful therapy. In this regard, a comprehensive spatially resolved quantitative proteome atlas was constructed to explore the functional proteomic landscape of colorectal cancer. This strategy integrates histopathological analysis, laser capture microdissection, and proteomics. Spatial proteome profiling of 200 tissue section samples facilitated by the fully integrated sample preparation technology SISPROT enabled the identification of more than 4000 proteins on the Orbitrap Exploris 240 from 2 mm2 × 10 μm tissue sections. Compared with normal adjacent tissues, we identified a spectrum of cancer-associated proteins and dysregulated pathways across various regions of colorectal cancer including ascending colon, transverse colon, descending colon, sigmoid colon, and rectum. Additionally, we conducted proteomic analysis on tumoral epithelial cells and paracancerous epithelium from early to advanced stages in hallmark rectum cancer and sigmoid colon cancer. Bioinformatics analysis revealed functional proteins and cell-type signatures associated with different regions of colorectal tumors, suggesting potential clinical implications. Overall, this study provides a comprehensive spatially resolved functional proteome landscape of colorectal cancer, serving as a valuable resource for exploring potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3187a8d97cb9da8d288b849296a5e0819056a10" target='_blank'>
              Spatial Proteomic Profiling of Colorectal Cancer Revealed Its Tumor Microenvironment Heterogeneity.
              </a>
            </td>
          <td>
            Lifen Xie, Qian Kong, Meiling Ai, An He, Bin Yao, Luobin Zhang, Keren Zhang, Chaowei Zhu, Yangqiu Li, Ligang Xia, Ruijun Tian, Ruilian Xu
          </td>
          <td>2024-07-18</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary The classification of breast cancer has been a complex subject, with diverse origins of normal breast epithelial cells and evolutionary pathways contributing to its extensive heterogeneity, which further complicates the treatment process. We developed a dynamic network model to simulate the emergence of different cellular states and explore the underlying biological mechanisms. This model revealed cellular states that align with four recognized molecular subtypes of breast cancer. It also identified feedback loops that either reinforce or facilitate phenotypic transitions, and allowed us to trace these transitions through the model’s topological structure. This approach provides new insights into the mechanisms driving breast cancer heterogeneity and may benefit the development of more targeted therapeutic strategies. Abstract Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is characterized by malignant myeloid precursors that span a cellular hierarchy from dedifferentiated leukemic stem cells to mature blasts. While the diagnostic and prognostic importance of AML blast maturation is increasingly recognized, personalized therapies are currently not tailored to a patients individual makeup of this cellular hierarchy. In this study, we use multiplexed image-based ex vivo drug screening (pharmacoscopy) to systematically quantify the drug sensitivity across the cellular hierarchy of AML patients. We analyzed 174 prospective and longitudinal patient samples from 44 newly diagnosed AML patients, which indicated that differences in the AML hierarchy significantly identified poor responses to first-line therapy, outperforming European LeukemiaNet (ELN) criteria. Critically, drug response profiling across the AML hierarchy of each patient improved the accuracy of predicting patient response to first-line therapy (AUC 0.91), and revealed alternative individualized treatment options targeting the complete AML hierarchy of non-responding patients. We confirmed these findings in an independent cohort of 26 relapsed/refractory AML patients, for whom pan-hierarchy response profiling improved response predictions post hoc. Overall, our results quantify the clinical importance of therapeutically targeting the complete cellular hierarchy of newly diagnosed AML, and identify multiplexed image-based ex vivo drug screening to enable quantification and targeting of the AML maturation hierarchy for improved personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f9f644c5c3c331964c25fff55c3e4e0a44431d0" target='_blank'>
              The cellular hierarchy of acute myeloid leukemia informs personalized treatment
              </a>
            </td>
          <td>
            Y. Severin, Y. Festl, T. M. Benoit, R. Wegmann, B. D. Hale, M. Roiss, A.-K. Kienzler, T. Pabst, M. Scharl, S. Sunagawa, M. G. Manz, A. M. S. Mueller, B. Snijder
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db888cba55a8c9545864ee89cfee2a8c7e2a49ea" target='_blank'>
              Navigating glioblastoma complexity: the interplay of neurotransmitters and chromatin
              </a>
            </td>
          <td>
            Jessica Romero-Reyes, E. R. Vázquez-Martínez, Carlos-Camilo Silva, A. Molina-Hernández, Néstor Fabián Díaz, Ignacio Camacho-Arroyo
          </td>
          <td>2024-08-17</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer is the leading cancer diagnosed in women globally, with brain metastasis emerging as a major cause of death, particularly in human epidermal growth factor receptor 2 positive and triple-negative breast cancer subtypes. Comprehensive understanding of the molecular foundations of central nervous system metastases is imperative for the evolution of efficacious treatment strategies. Lipocalin-2 (LCN2), a secreted iron transport protein with multiple functions, has been linked to the progression of breast cancer brain metastasis (BCBM). In primary tumors, LCN2 promotes the proliferation and angiogenesis of breast cancer cells, triggers the epithelial-mesenchymal transition, interacts with matrix metalloproteinase-9, thereby facilitating the reorganization of the extracellular matrix and enhancing cancer cell invasion and migration. In brain microenvironment, LCN2 undermines the blood-brain barrier and facilitates tumor seeding in the brain by modulating the behavior of key cellular components. In summary, this review meticulously examines the fuel role of LCN2 in BCBM cascade, and investigates the potential mechanisms involved. It highlights the potential of LCN2 as both a therapeutic target and biomarker, indicating that interventions targeting LCN2 may offer improved outcomes for patients afflicted with BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec9c722834fa6f4f99791aa297a411fbbd161a74" target='_blank'>
              Lipocalin-2 promotes breast cancer brain metastasis by enhancing tumor invasion and modulating brain microenvironment
              </a>
            </td>
          <td>
            Yang Zhao, Xiaogen Tang, Tingting Lei, Dongwei Fu, Hongyi Zhang
          </td>
          <td>2024-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Head and neck cancer disparities in outcomes for Black Americans have been well recognized. However, the specific drivers of the inferior outcomes remain poorly understood. We investigated the biologic features of patient oral cancers and performed a follow-up study of the patient post-surgery recurrences and metastases aiming to explore potential mechanisms that might underpin the poorer outcomes among Black American patients. It was found that high levels of tumor stemness and tumor-promoting stromal characteristics were linked to patient recurrence and metastasis. There were more cases of Black American than White American exhibiting high stemness traits and strong tumor-promoting stromal features associated with tumor recurrences and metastases, although the investigated cases displayed comparable clinical diagnoses. Our findings revealed that the differences in tumor stemness and stromal property among cancers with similar diagnoses contribute to patient outcome disparities. Abstract Black Americans (BAs) with head and neck cancer (HNC) have worse survival outcomes compared to the White patients. While HNC disparities in patient outcomes for BAs have been well recognized, the specific drivers of the inferior outcomes remain poorly understood. Here, we investigated the biologic features of patient tumor specimens obtained during the surgical treatment of oral cancers and performed a follow-up study of the patients’ post-surgery recurrences and metastases with the aim to explore whether tumor biologic features could be associated with the poorer outcomes among BA patients compared with White American (WA) patients. We examined the tumor stemness traits and stromal properties as well as the post-surgery recurrence and metastasis of oral cancers among BA and WA patients. It was found that high levels of tumor self-renewal, invasion, tumorigenesis, metastasis, and tumor-promoting stromal characteristics were linked to post-surgery recurrence and metastasis. There were more BA than WA patients demonstrating high stemness traits and strong tumor-promoting stromal features in association with post-surgery tumor recurrences and metastases, although the investigated cases displayed clinically comparable TNM stages and histological grades. These findings demonstrated that the differences in tumor stemness and stromal property among cancers with comparable clinical diagnoses contribute to the outcome disparity in HNCs. More research is needed to understand the genetic and molecular basis of the biologic characteristics underlying the inferior outcomes among BA patients, so that targeting strategies can be developed to reduce HNC disparity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcad9291ce4667fd7a96a4d38826ce6e7b1c58" target='_blank'>
              Tumor Cell Stemness and Stromal Cell Features Contribute to Oral Cancer Outcome Disparity in Black Americans
              </a>
            </td>
          <td>
            S. Mirshahidi, Isabella J. Yuan, Zhong Chen, Alfred Simental, Steve C. Lee, Pedro A Andrade Filho, Thomas Murry, Feng Zeng, Penelope Duerksen-Hughes, Charles Wang, Xiangpeng Yuan
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cell type identity is determined by gene regulatory networks (GRNs), comprising the expression of specific transcription factors (TFs) regulating target genes (TGs) via binding to open chromatin regions (OCRs). The regulatory logic of differentiation includes factors specific to one or multiple cell types, functioning in a combinatorial fashion. Classic approaches of GRN discovery used perturbational data to elucidate TF-TG links, but are laborious and not scalable across the tree of life. Single cell transcriptomics has emerged as a revolutionary approach to study gene expression with cell type resolution, but incorporating perturbational data is challenging. Planarians, with their pluripotent neoblast stem cells continuously giving rise to all cell types, offer an ideal model to attempt this integration. Despite extensive single cell transcriptomic studies, the transcriptional and chromatin regulation at the cell type level remains unexplored. Here, we investigate the regulatory logic of planarian stem cell differentiation by obtaining an organism-level integration of single cell transcriptomics and single cell accessibility data. We identify specific open chromatin profiles for major differentiated cell types and analyse their transcriptomic landscape, revealing distinct gene modules expressed in individual types and combinations of them. Integrated analysis unveils gene networks reflecting known TF interactions in each type and identifies TFs potentially driving differentiation across multiple cell types. To validate our predictions, we combined TF knockdown RNAi experiments with single cell transcriptomics. We focus on hnf4, a TF known to be expressed in gut phagocytes, and confirm its influence on other types, including parenchymal cells. Our results demonstrate high overlap between predicted targets and experimentally-validated differentially-regulated genes. Overall, our study integrates TFs, TGs and OCRs to reveal the regulatory logic of planarian stem cell differentiation, showcasing that the combination of single cell methods and perturbational studies will be key for characterising GRNs widely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d812fbadba7ebc9a3898e7e95eae459989657669" target='_blank'>
              The Regulatory Logic of Planarian Stem Cell Differentiation
              </a>
            </td>
          <td>
            Alberto Pérez-Posada, Helena García-Castro, Elena Emili, Virginia Vanni, Cirenia Arias-Baldrich, S. Frölich, S. V. van Heeringen, N. Kenny, Jordi Solana
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The rapid development of single-cell sequencing provides unparalleled opportunity for delineating the heterogeneous characteristics of individual cells, while current methodologies fall short in accurately deciphering the origins of cellular heterogeneity and pinpointing the states of cell fate decisions. In this study, we introduce a novel approach termed Single-cell Reinforcement Learning (scRL) by integrating a framework of reinforcement learning into single-cell data analysis with the assistance of an actor-critic architecture. scRL reveals that the intensity of fate decisions precedes the maximum potential of lineage differentiation and actual gene expression, which enables the forecasting and perturbation of cellular fate in an earlier stage. The superiority of scRL in assessing fate decisions is confirmed in several different patterns of single-cell datasets. By using scRL, we disclose a distinct role of Dapp1 in myeloid lineage decision during hematopoietic stem cells (HSCs) differentiation and uncover the priority of erythroid fate decision occurring post irradiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dcd7d2e6003e1ee97fadd6079d9c94bb4456e7d" target='_blank'>
              Reinforcement learning guides single-cell sequencing in decoding lineage and cell fate decisions
              </a>
            </td>
          <td>
            Zeyu Fu, Song Wang, Kangfu Sun, Baichuan Xu, Xianpeng Ye, Zhaoyang Wen, M. Shen, Mo Chen, Fang Chen, Yang Xu, Shilei Chen, Jun-Ping Wang
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f459c742503a40a28067c79054343f2f4490725a" target='_blank'>
              Adhesion of pancreatic tumor cell clusters by desmosomal molecules enhances early liver metastases formation
              </a>
            </td>
          <td>
            Niclas Dietrich, Ramón Castellanos-Martínez, Julia Kemmling, A. Heuser, M. Schnoor, Camilla Schinner, Volker Spindler
          </td>
          <td>2024-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Endometriosis, affecting 1 in 9 women, presents treatment and diagnostic challenges. To address these issues, we generated the biggest single-cell atlas of endometrial tissue to date, comprising 466,371 cells from 35 endometriosis and 25 non-endometriosis patients without exogenous hormonal treatment. Detailed analysis reveals significant gene expression changes and altered receptor-ligand interactions present in the endometrium of endometriosis patients, including increased inflammation, adhesion, proliferation, cell survival, and angiogenesis in various cell types. These alterations may enhance endometriosis lesion formation and offer novel therapeutic targets. Using ScaiVision, we developed neural network models predicting endometriosis of varying disease severity (median AUC = 0.83), including an 11-gene signature-based model (median AUC = 0.83) for hypothesis-generation without external validation. In conclusion, our findings illuminate numerous pathway and ligand-receptor changes in the endometrium of endometriosis patients, offering insights into pathophysiology, targets for novel treatments, and diagnostic models for enhanced outcomes in endometriosis management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9557348ef1327c4800ef29c1c6e1843f22fe83" target='_blank'>
              Tracing Endometriosis: Coupling deeply phenotyped, single-cell based Endometrial Differences and AI for disease pathology and prediction
              </a>
            </td>
          <td>
            Lea Duempelmann, Shaoline Sheppard, Angelo Duo, Jitka Skrabalova, Brett McKinnon, Thomas Andrieu, Dennis Goehlsdorf, S. Muzumdar, Cinzia Donato, Ryan Lusby, Wiebke Solass, Hans Bösmüller, Peter Nestorov, Michael D. Mueller
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Stable isotope-resolved metabolomics delineates reprogrammed intersecting metabolic networks in human cancers. Knowledge gained from in vivo patient studies provides the "benchmark" for cancer models to recapitulate. It is particularly difficult to model patients' tumor microenvironment (TME) with its complex cell-cell/cell-matrix interactions, which shapes metabolic reprogramming crucial to cancer development/drug resistance. Patient-derived organotypic tissue cultures (PD-OTCs) represent a unique model that retains an individual patient's TME. PD-OTCs of non-small-cell lung cancer better recapitulated the in vivo metabolic reprogramming of patient tumors than the patient-derived tumor xenograft (PDTX), while enabling interrogation of immunometabolic response to modulators and TME-dependent resistance development. Patient-derived organoids (PDOs) are also good models for reconstituting TME-dependent metabolic reprogramming and for evaluating therapeutic responses. Single-cell based 'omics on combinations of PD-OTC and PDO models will afford an unprecedented understanding on TME dependence of human cancer metabolic reprogramming, which should translate into the identification of novel metabolic targets for regulating TME interactions and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6597c82613b1bff5869b53a6356a5e51a2359cc" target='_blank'>
              Metabolic Reprogramming in Human Cancer Patients and Patient-Derived Models.
              </a>
            </td>
          <td>
            T. W. Fan, R. Higashi, A. Lane
          </td>
          <td>2024-07-15</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hypopharyngeal squamous cell carcinoma (SCC) is a highly aggressive cancer with a poor prognosis, particularly in advanced stages where concurrent chemoradiotherapy (CCRT) is used for treatment. However, resistance to CCRT poses a significant challenge, often leading to treatment failure and disease progression. This study explores the tumor microenvironment (TME) of hypopharyngeal SCC to understand the molecular mechanisms underlying CCRT resistance. Using spatial transcriptomics (ST), we analyzed tissue samples from patients with locally advanced hypopharyngeal SCC, distinguishing between those who were CCRT-resistant and those who were CCRT-naive. The analysis revealed six distinct cellular clusters within the TME, including a prominent epithelio-immune cellular area in CCRT-resistant tissues. SPP1 was identified as a key gene with significantly higher expression in CCRT-resistant samples, specifically within macrophages. Further investigation showed that SPP1+ macrophages interacted with malignant epithelial cells through SPP1-CD44 and SPP1-ITGB1 ligand-receptor pairs. These interactions were primarily localized in the peri-tumoral and intra-tumoral regions, highlighting their potential role in driving CCRT resistance. Our findings suggest that SPP1+ macrophages contribute to the resistant phenotype in hypopharyngeal SCC by modulating the TME and interacting with cancer cells. Understanding these interactions offers valuable insights into the mechanisms of CCRT resistance and may inform the development of targeted therapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d07108a9430fa63556b14b31c6b832a1633f5b4" target='_blank'>
              Spatial transcriptomics unveils landscape of resistance to concurrent chemo-radiotherapy in hypopharyngeal squamous cell carcinoma: the role of SPP1+ macrophages
              </a>
            </td>
          <td>
            J. Ohn, Sungwoo Bae, Hongyoon Choi, In Gul Kim, Kwon Joong Na, Eun-Jae Chung
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/609bb25145d5246600ce4feb000ce7f09c4474b7" target='_blank'>
              Building a translational cancer dependency map for The Cancer Genome Atlas
              </a>
            </td>
          <td>
            Xu Shi, Christos Gekas, Daniel Verduzco, Sakina M. Petiwala, Cynthia Jeffries, Charles Lu, Erin Murphy, Tifani Anton, Andy H Vo, Zhiguang Xiao, Padmini Narayanan, Bee-Chun Sun, Aloma L D'Souza, J. M. Barnes, Somdutta Roy, Cyril Ramathal, Michael J. Flister, Zoltan Dezso
          </td>
          <td>2024-07-15</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="This study presents an interdisciplinary approach to analyse the distribution of cancer stem cell markers (CSCMs) across various cancer-affected organs using hypergraphs. Cancer stem cells (CSCs) play a crucial role in cancer initiation, progression, and metastasis. By employing hypergraphs, we model the relationships between CSCM locations and cancerous organs, providing a comprehensive representation of these interactions. Initially, we utilised an unweighted incidence matrix and its Markov transition matrices to gain a dynamic perspective on CSCM distributions. This method allows us to observe how these markers spread and influence cancer progression in a dynamical context. By calculating mutual information for each node and hyperedge, our analysis uncovers complex interaction patterns between CSCMs and organs, highlighting the critical roles of certain markers in cancer progression and metastasis. Our approach offers a detailed representation of cancer stem cell networks, enhancing our understanding of the mechanisms driving cancer heterogeneity and metastasis. By integrating hypergraph theory with cancer biology, this study provides valuable insights for developing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48368f26aa7447550426a623ac42529547196d96" target='_blank'>
              Mapping Cancer Stem Cell Markers Distribution:A Hypergraph Analysis Across Organs
              </a>
            </td>
          <td>
            D. H. Margarit, Gustavo Paccosi, M. V. Reale, Lilia M. Romanelli
          </td>
          <td>2024-07-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) exert clinical efficacy against various types of cancers by reinvigorating exhausted CD8+ T cells that can expand and directly attack cancer cells (cancer-specific T cells) among tumor-infiltrating lymphocytes (TILs). Although some reports have identified somatic mutations in TILs, their effect on antitumor immunity remains unclear. In this study, we successfully established 18 cancer-specific T cell clones, which have an exhaustion phenotype, from the TILs of four patients with melanoma. We conducted whole-genome sequencing for these T cell clones and identified various somatic mutations in them with high clonality. Among the somatic mutations, an SH2D2A loss-of-function frameshift mutation and TNFAIP3 deletion could activate T cell effector functions in vitro. Furthermore, we generated CD8+ T cell-specific Tnfaip3 knockout mice and showed that Tnfaip3 function loss in CD8+ T cell increased antitumor immunity, leading to remarkable response to PD-1 blockade in vivo. In addition, we analyzed bulk CD3+ T cells from TILs in additional 12 patients and identified an SH2D2A mutation in one patient through amplicon sequencing. These findings suggest that somatic mutations in TILs can affect antitumor immunity and suggest unique biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d5ca5b4ab22ea980ea6c4a3574f4132db1aef93" target='_blank'>
              Somatic mutations in tumor-infiltrating lymphocytes impact on antitumor immunity.
              </a>
            </td>
          <td>
            Fumiaki Mukohara, Kazuma Iwata, Takamasa Ishino, Takashi Inozume, J. Nagasaki, Youki Ueda, K. Suzawa, Toshihide Ueno, Hideki Ikeda, Katsushige Kawase, Yuka Saeki, Shusuke Kawashima, Kazuo Yamashita, Yu Kawahara, Yasuhiro Nakamura, Akiko Honobe-Tabuchi, Hiroko Watanabe, H. Dansako, T. Kawamura, Yutaka Suzuki, Hiroaki Honda, Hiroyuki Mano, Shinichi Toyooka, Masahito Kawazu, Yosuke Togashi
          </td>
          <td>2024-08-21</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by the selective expansion of hematopoietic stem and progenitor cells (HSPCs) carrying somatic mutations. While CHIP is typically asymptomatic, it has garnered substantial attention due to its association with the pathogenesis of multiple disease conditions, including cardiovascular disease (CVD) and hematological malignancies. In this Review, we will discuss seminal and recent studies that have advanced our understanding of mechanisms that drive selection for mutant HSPCs in the BM niche. Next, we will address recent studies evaluating potential relationships between the clonal dynamics of CHIP and hematopoietic development across the lifespan. Next, we will examine the roles of systemic factors that can influence hematopoietic stem cell (HSC) fitness, including inflammation, and exposures to cytotoxic agents in driving selection for CHIP clones. Furthermore, we will consider how — through their impact on the BM niche — lifestyle factors, including diet, exercise, and psychosocial stressors, might contribute to the process of somatic evolution in the BM that culminates in CHIP. Finally, we will review the role of old age as a major driver of selection in CHIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4403bef9498a4a5c2e26a32490bcbf4e8de6435a" target='_blank'>
              CHIP: a clonal odyssey of the bone marrow niche
              </a>
            </td>
          <td>
            Wolfgang E Schleicher, Bridget Hoag, Marco De Dominici, James DeGregori, Eric Pietras
          </td>
          <td>2024-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract As breast cancer migrates into the brain, it acquires distinct genomic, immunologic and phenotypic changes through interactions with the local microenvironment. We sought to determine how targetable gene expression changes mediate breast cancer extravasation into and proliferation within the brain. We created a brain metastatic cell line (“brain-seeking”) via serial intracardiac injection of mCherry+ 4T1 breast cancer cells in Balb/c mice and flow cytometry isolation of micrometastases. We conducted in vitro and in-vivo studies to uncover immunological and genetic differences between primary breast cancer and brain metastases. In culture, we observed significant upregulation in adhesion (p=0.01) and invasion (p=0.0004) along with upregulation of adhesion-promoting genes such as Sdc3, Serpine2 or ECM1 in the brain-seeking cells. We observed significant differences in survival of Balb/c mice injected intracranially with brain-seeking cells compared to the 4T1 breast cancer cells (p=0.04). In the mammary fat pad (MFP), parental 4T1 cells caused increased mortality compared to brain-seeking cells (p=0.003). This rejection of brain-seeking 4T1 cells in the MFP could be due to their prior adaptation to the brain microenvironment, while rejection of parental 4T1 cells in the brain microenvironment could be due to them lacking those adaptations. Consistent with this finding, of 20,382 analyzed genes in both cell lines, 108 related to immune regulation were significantly upregulated in brain-seeking cells including Cxcl1, Dusp1, C3. Flow cytometry showed increased cytotoxic and helper T cells in intracranially injected parental 4T1 cells vs. brain-seeking cells (p <0.001). In the MFP, however, 4T1 parental cells displayed CD3+ T cell depletion and an increased number of neutrophils (p <0.05) while the “brain-seeking cells” showed increased numbers of CD3+ and CD4+ cells (p <0.01) and macrophages (p<0.05). These findings suggest that brain metastases evolve from their primary tumor through adhesive and invasive adaptations promoting extravasation, along with unique immunological adaptations that allow them to proliferate free of immune surveillance in the brain microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a08cb54ecb10f18f98a481490dcd3d1898c9d25" target='_blank'>
              BSBM-03 TARGETABLE GENE EXPRESSION CHANGES MEDIATE EXTRAVASATION OF BREAST CANCER CELLS FROM THE CIRCULATION AND LOCAL PROLIFERATION IN THE BRAIN
              </a>
            </td>
          <td>
            Venina Kalistratova, R. Morshed, Eric Chalif, Sabraj A. Gill, Harmon Khela, Meeki K. Lad, Ankita Sati, Aakriti Mathur, Manish K Aghi
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma remains one of the most lethal solid tumors due to its local aggressiveness and metastatic potential, with a 5-year survival rate of only 13%. A robust connection between pancreatic cancer microenvironment and tumor progression exists, as well as resistance to current anticancer treatments. Pancreatic cancer has a complex tumor microenvironment, characterized by an intricate crosstalk between cancer cells, cancer-associated fibroblasts and immune cells. The complex composition of the tumor microenvironment is also reflected in the diversity of its acellular components, such as the extracellular matrix, cytokines, growth factors and secreted ligands involved in signaling pathways. Desmoplasia, the hallmark of the pancreatic cancer microenvironment, contributes by creating a dense and hypoxic environment that promotes further tumorigenesis, provides innate systemic resistance and suppresses anti-tumor immune invasion. We discuss the complex crosstalk among tumor microenvironment components and explore therapeutic strategies and opportunities in pancreatic cancer research. Better understanding of the tumor microenvironment and its influence on pancreatic cancer progression could lead to potential novel therapeutic options, such as integration of immunotherapy and cytokine-targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5be7a2581240dc6002801cdd72c3d6c7feb416a" target='_blank'>
              Focus on Pancreatic Cancer Microenvironment
              </a>
            </td>
          <td>
            Fabiana Pratticò, I. Garajová
          </td>
          <td>2024-07-26</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd5fd54ef5a030d3f344e0965a3022e2d90886e" target='_blank'>
              An image-based screen for secreted proteins involved in breast cancer G0 cell cycle arrest
              </a>
            </td>
          <td>
            William A. Weston, Jordan A. Holt, A. Wiecek, James Pilling, L. H. Schiavone, David M. Smith, M. Secrier, Alexis R Barr
          </td>
          <td>2024-08-10</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Introduction This study investigates the role of Fibroblast Activation Protein (FAP)-positive cancer-associated fibroblasts (FAP+CAF) in shaping the tumor immune microenvironment, focusing on its association with immune cell functionality and cytokine expression patterns. Methods Utilizing immunohistochemistry, we observed elevated FAP+CAF density in metastatic versus primary renal cell carcinoma (RCC) tumors, with higher FAP+CAF correlating with increased T cell infiltration in RCC, a unique phenomenon illustrating the complex interplay between tumor progression, FAP+CAF density, and immune response. Results Analysis of immune cell subsets in FAP+CAF-rich stromal areas further revealed significant correlations between FAP+ stroma and various T cell types, particularly in RCC and non-small cell lung cancer (NSCLC). This was complemented by transcriptomic analyses, expanding the range of stromal and immune cell subsets interrogated, as well as to additional tumor types. This enabled evaluating the association of these subsets with tumor infiltration, tumor vascularization and other components of the tumor microenvironment. Our comprehensive study also encompassed cytokine, angiogenesis, and inflammation gene signatures across different cancer types, revealing heterogeneous cellular composition, cytokine expressions and angiogenic profiles. Through cytokine pathway profiling, we explored the relationship between FAP+CAF density and immune cell states, uncovering potential immunosuppressive circuits that limit anti-tumor activity in tumor-resident immune cells. Conclusions These findings underscore the complexity of tumor biology and the necessity for personalized therapeutic and patient enrichment approaches. The insights gathered from FAP+CAF prevalence, immune infiltration, and gene signatures provide valuable perspectives on tumor microenvironments, aiding in future research and clinical strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340394b70e234f1340a6e340d3b01ec0cd62d31a" target='_blank'>
              Investigating the complex interplay between fibroblast activation protein α-positive cancer associated fibroblasts and the tumor microenvironment in the context of cancer immunotherapy
              </a>
            </td>
          <td>
            A. Kraxner, Franziska Braun, Wei-Yi Cheng, Tai-Hsien Ou Yang, Shweta V. Pipaliya, Marta Cañamero, Emilia Andersson, S. V. Harring, Sebastian Dziadek, Ann-Marie E. Bröske, Maurizio Ceppi, Tamara Tanos, Volker Teichgräber, Jehad Charo
          </td>
          <td>2024-07-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7a41ef98c35752ab8b215566e394e307d31a58" target='_blank'>
              PHF12 regulates HDAC1 to promote tumorigenesis via EGFR/AKT signaling pathway in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yiru Kong, Rongrong Jiang, Hui Zhou, M. Ge, Hao Lin, Yu Wang, Rongrong Yao, Qing Wang, Xiaohua Liang, Jing Li, Xinli Zhou
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The WDR5/MLL1-H3K4me3 epigenetic axis is often activated in both tumor cells and tumor-infiltrating immune cells to drive various cellular responses in the tumor microenvironment and has been extensively studied in hematopoietic cancer, but its respective functions in tumor cells and immune cells in the context of tumor growth regulation of solid tumor is still incompletely understood. We report here that WDR5 exhibits a higher expression level in human pancreatic tumor tissues compared with adjacent normal pancreas. Moreover, WDR5 expression is negatively correlated with patients’ response to chemotherapy or immunotherapy in human colon cancer and melanoma. However, WDR5 expression is positively correlated with the HLA level in human cancer cells, and H3K4me3 enrichment is observed at the promoter region of the HLA-A, HLA-B, and HLA-C genes in pancreatic cancer cells. Using mouse tumor cell lines and in vivo tumor models, we determined that WDR5 deficiency or inhibition significantly represses MHC I expression in vitro and in vivo in pancreatic tumor cells. Mechanistically, we determine that WDR5 deficiency inhibits H3K4me3 deposition at the MHC I (H2K) promoter region to repress MHC I (H2K) transcription. On the other hand, WDR5 depletion leads to the effective downregulation of immune checkpoints and immunosuppressive cytokines, including TGFβ and IL6, in the pancreatic tumor microenvironments. Our data determine that WDR5 not only regulates tumor cell immunogenicity to suppress tumor growth but also activates immune suppressive pathways to promote tumor immune evasion. Selective activation of the WDR5-MHC I pathway and/or selective inhibition of the WDR5–immune checkpoint and WDR5–cytokine pathways should be considered in WDR5-based epigenetic cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855836379cadca724defacc71dcd89c3fbbdd74b" target='_blank'>
              The Wdr5-H3K4me3 Epigenetic Axis Regulates Pancreatic Tumor Immunogenicity and Immune Suppression
              </a>
            </td>
          <td>
            Kaidi Deng, Liyan Liang, Yingcui Yang, Yanmin Wu, Yan Li, Rongrong Zhang, Yulin Tian, Chunwan Lu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most common primary malignant brain tumor. The aim of this study was to elucidate the role of microenvironment and intrinsic T-type calcium channels (Cav3) in regulating tumor growth and progression. Methods We grafted syngeneic GBM cells into Cav3.2 knockout mice to assess the role of microenvironment T-Type calcium channels on GBM tumor growth. We performed single-cell RNA-seq (scRNA-seq) of tumors from WT and Cav3.2 KO mice to elucidate the regulation of tumors by the microenvironment. We used neurons from WT and Cav3.2 KO mice in co-culture with GBM stem cells (GSC) to assess the effects of Cav3.2 on neuron/GSC synaptic connections and tumor cell growth. Results Cav3.2 KO in the microenvironment led to significant reduction of GBM growth and prolongation of animal survival. scRNA-seq showed that microenvironment Cav3.2 regulates neuronal and glial biological processes. Microenvironment Cav3.2 downregulated numerous genes associated with regulating the OPC cell state in GBM tumors such as SOX10 and Olig2. Neuronal Cav3.2 promoted neuron/GSC synaptic connections and GSC growth. Treatment of GSCs with the Cav3 blocker mibefradil downregulated genes associated with neuronal processes. The Cav3 blocker drug mibefradil synergized with temozolomide (TMZ) and radiation to reduce in vivo tumor growth and prolong animal survival. Conclusions Together these data reveal a role for microenvironment Cav3 in promoting GBM tumor progression through regulating neuronal and glial processes particularly associated with the OPC-cell state. Targeting both intrinsic and microenvironment Cav3 with the inhibitor mibefradil significantly enhanced the anti-GBM effects of TMZ and radiation. Key Points Microenvironment Cav3.2 promotes GBM progression Microenvironment Cav3.2 promotes neuronal and glial processes Pharmacological targeting of intrinsic and microenvironment Cav3 synergizes with TMZ/radiation Importance of the Study In this study, we demonstrate for the first time that microenvironment Cav3.2 contributes to GBM progression and growth by regulating neuronal and glial processes. Our findings highlight the importance of T-type calcium channels in the microenvironment as well as the tumor and provides preclinically relevant data for the use of mibefradil to inhibit GBM growth in combination with standard of care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303b17198abd5c192529012a365daa1747ef06e0" target='_blank'>
              Microenvironment T-Type calcium channels regulate neuronal and glial processes to promote glioblastoma growth
              </a>
            </td>
          <td>
            Collin J. Dube, Ying Zhang, Shekhar Saha, Michelle Lai, Myron K. Gibert, Miguel Escalante, Kadie Hudson, Doris Wong, Pawel Marcinkiewicz, Ulas Yener, Yunan Sun, Esther Xu, Aditya Sorot, Elizabeth Mulcahy, B. Kefas, Farina Hanif, Fadilla Guessous, Ashley Vernon, Manoj K. Patel, David Schiff, Hui Zong, Benjamin Purow, Eric Holland, Swapnil Sonkusare, Harald Sontheimer, R. Abounader
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common malignant primary brain tumor. GBM has an extremely poor prognosis and new treatments are badly needed. Efficient drug delivery to GBM is a major obstacle as the blood-brain barrier (BBB) prevents passage of the majority of cancer drugs into the brain. It is also recognized that the blood-brain tumor barrier (BTB) in the growing tumor represents a challenge. The BTB is heterogeneous and poorly characterized, but similar to the BBB it can prevent therapeutics from reaching effective intra-tumoral doses, dramatically hindering their potential. Here, we identified a 12-gene signature associated with the BTB, with functions related to vasculature development, morphogenesis and cell migration. We identified CDH5 as a core molecule in this set and confirmed its over-expression in GBM vasculature using spatial transcriptomics of GBM patient specimens. We found that the indirubin-derivative, 6-bromoindirubin acetoxime (BIA), could downregulate CDH5 and other BTB signature genes, causing endothelial barrier disruption in endothelial monolayers and BBB 3D spheroids in vitro. Treatment of tumor-bearing mice with BIA enabled increased intra-tumoral accumulation of the BBB non-penetrant chemotherapeutic drug cisplatin and potentiated cisplatin-mediated DNA damage by targeting DNA repair pathways. Finally, using an injectable BIA nanoparticle formulation, PPRX-1701, we significantly improved the efficacy of cisplatin in patient-derived GBM xenografts and prolonged their survival. Overall, our work reveals potential targets at the BTB for improved chemotherapy delivery and the bifunctional properties of BIA as a BTB modulator and potentiator of chemotherapy, supporting its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca2d6e3b8b9f2b3209e769ca054b665691b61645" target='_blank'>
              Modulation of blood-tumor barrier transcriptional programs improves intra-tumoral drug delivery and potentiates chemotherapy in GBM
              </a>
            </td>
          <td>
            J. Jimenez-Macias, Philippa Vaughn-Beaucaire, Ayush Bharati, Zheyun Xu, Megan Forrest, Jason Hong, Michael Sun, Andrea Schmidt, Jasmine S. Clark, William Hawkins, Noe Mercado, Jacqueline Real, Kelsey Huntington, M. Zdioruk, Michal O. Nowicki, C. Cho, Bin Wu, Weiyi Li, Theresa Logan, K. Manz, K. Pennell, B. Fedeles, Alexander S. Brodsky, Sean Lawler
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9965a9579beaaf870afee3260a8af2cf6b7ea8" target='_blank'>
              IL-1R signaling drives enteric glia-macrophage interactions in colorectal cancer
              </a>
            </td>
          <td>
            L. van Baarle, V. De Simone, Linda Schneider, Sneha Santhosh, S. Abdurahiman, F. Biscu, Reiner Schneider, Lisa Zanoletti, Renata Siqueira de Mello, Sara Verbandt, Zedong Hu, M. Stakenborg, B. Ke, N. Stakenborg, Raquel Salvador Laureano, Balbina García-Reyes, Jonas Henn, Marieta Toma, M. Vanmechelen, G. Boeckxstaens, F. De Smet, A. Garg, S. Ibiza, Sabine Tejpar, Sven Wehner, G. Matteoli
          </td>
          <td>2024-07-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinoma (RCC) with histological sarcomatoid de-differentiation (sRCC) has a historically poor prognosis across all RCC subtypes. Recently, immunotherapy with anti-PD1 + anti-CTLA4 combo has significantly improved disease-specific survival, and complete response are seen in upwards of 20% of patients. However, half of sRCC patients still have no response to this immunotherapy, and the mechanism of response remains unclear. Methods Our preliminary bulk RNA sequencing data reveal a distinct tumor microenvironment (TME) in human sRCC tumors, characterized by heightened immune cell infiltration, particularly enriched for regulatory T cells (Tregs). Subsequently, we delineated the spatial gene expression topography within sRCC and clear cell renal cell carcinoma (ccRCC) regions, culminating in the development of the sRCC signature. Concurrently, we established an innovative immunocompetent murine model of sRCC. Leveraging CRISPR technology, we replicated the human sRCC genotype by knocking out Vhl, Bap1, and Cdkn2a/2b genes in a mouse telomerase reverse transcriptase (mTERT) overexpressing renal proximal tubule epithelial cell line. This effort yielded a successfully engineered mouse implantable sRCC cell line bearing VhlMutBap1DelCdkn2aDelCdkn2bDel (HJRCC68N). Results The syngeneic mouse sRCC model we developed faithfully replicates human sRCC histology and tumor microenvironment and recapitulates the sRCC signature identified in human sRCC regions. Importantly, like in humans, mouse sRCC tumors exhibit a heterogeneous response to anti-PD1 + anti-CTLA4 combination, and anti-CTLA4 monotherapy, reflecting the clinical variability observed in human patients. Furthermore, transcriptomic analysis from our mouse model reveals the involvement of type 1 immunity in responder, providing insights into overcoming immunotherapy resistance. Conclusions In conclusion, our study unveiled distinct changes in the TME at the transcriptomic and spatial gene expression levels in human sRCC. Leveraging our novel murine model, which faithfully replicates human sRCC histology, TME characteristics, and mixed response to immunotherapy, we elucidated the involvement of type 1 immunity in treatment response. These insights offer valuable pathways for overcoming immunotherapy resistance and pave the way for the development of tailored treatment strategies aimed at improving outcomes for sRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5fbfe11a82cb5fed55f6641955ed49526e183d8" target='_blank'>
              18 Unraveling the Impact of Tumor Microenvironment on Immunotherapy Response/Resistance in Human Sarcomatoid Clear Cell Renal Cell Carcinoma: Insights from a Novel Mouse Model
              </a>
            </td>
          <td>
            Hui Jiang
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Tumor mutation burden (TMB) and VHL mutation play a crucial role in the management of patients with clear cell renal cell carcinoma (ccRCC), such as guiding adjuvant chemotherapy and improving clinical outcomes. However, the time‐consuming and expensive high‐throughput sequencing methods severely limit their clinical applicability. Predicting intratumoral heterogeneity poses significant challenges in biology and clinical settings. Our aimed to develop a self‐supervised attention‐based multiple instance learning (SSL‐ABMIL) model to predict TMB and VHL mutation status from hematoxylin and eosin‐stained histopathological images. Methods We obtained whole slide images (WSIs) and somatic mutation data of 350 ccRCC patients from The Cancer Genome Atlas for developing SSL‐ABMIL model. In parallel, 163 ccRCC patients from Clinical Proteomic Tumor Analysis Consortium cohort was used as independent external validation set. We systematically compared three different models (Wang‐ABMIL, Ciga‐ABMIL, and ImageNet‐MIL) for their ability to predict TMB and VHL alterations. Results We first identified two groups of populations with high‐ and low‐TMB (cut‐off point = 0.9). In two independent cohorts, the Wang‐ABMIL model achieved the highest performance with decent generalization performance (AUROC = 0.83 ± 0.02 and 0.8 ± 0.04 in predicting TMB and VHL, respectively). Attention heatmaps revealed that the Wang‐ABMIL model paid the highest attention to tumor regions in high‐TMB patients, while in VHL mutation prediction, non‐tumor regions were also assigned high attention, particularly the stromal regions infiltrated by lymphocytes. Conclusions Our results indicated that SSL‐ABMIL can effectively extract histological features for predicting TMB and VHL mutation, demonstrating promising results in linking tumor morphology and molecular biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef0da017ff2911ab8b44c1c00678b46071fd400" target='_blank'>
              Predicting tumor mutation burden and VHL mutation from renal cancer pathology slides with self‐supervised deep learning
              </a>
            </td>
          <td>
            Qingyuan Zheng, Xinyu Wang, Rui Yang, Junjie Fan, Jingping Yuan, Xiuheng Liu, Lei Wang, Zhuoni Xiao, Zhiyuan Chen
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer is one of the most lethal forms of malignancies; therefore, new treatment strategies are required to increase the patients’ survival. It has been established that different cell types that surround pancreatic cancer cells, thus forming the tumor microenvironment, are responsible for tumorigenicity and inefficacy of treatments, including immunotherapy. In the present review, we aim to summarize current knowledge regarding the molecular mechanisms underpinning the interaction of cancer cells with cells of their microenvironment and discuss associated strategies to improve treatment results. Abstract Pancreatic cancer demonstrates an ever-increasing incidence over the last years and represents one of the top causes of cancer-associated mortality. Cells of the tumor microenvironment (TME) interact with cancer cells in pancreatic ductal adenocarcinoma (PDAC) tumors to preserve cancer cells’ metabolism, inhibit drug delivery, enhance immune suppression mechanisms and finally develop resistance to chemotherapy and immunotherapy. New strategies target TME genetic alterations and specific pathways in cell populations of the TME. Complex molecular interactions develop between PDAC cells and TME cell populations including cancer-associated fibroblasts, myeloid-derived suppressor cells, pancreatic stellate cells, tumor-associated macrophages, tumor-associated neutrophils, and regulatory T cells. In the present review, we aim to fully explore the molecular landscape of the pancreatic cancer TME cell populations and discuss current TME targeting strategies to provide thoughts for further research and preclinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6521a90b4084dc31e666897cae7e9489d858db28" target='_blank'>
              Precision Targeting Strategies in Pancreatic Cancer: The Role of Tumor Microenvironment
              </a>
            </td>
          <td>
            Nikolaos Vitorakis, A. Gargalionis, K. Papavassiliou, Christos Adamopoulos, Athanasios G. Papavassiliou
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Amoeboid cells like leukocytes can enter and migrate within virtually every tissue of the body, even though tissues vary widely in their chemical and mechanical composition. Here, we imaged motile T cells as they colonized peripheral tissues during zebrafish development to ask if cells tailor their migration strategy to their local tissue environment. We found that T cells in most sites migrated with f-actin-rich leading-edge pseudopods, matching how they migrate in vitro. T cells notably deviated from this strategy in the epidermis, where they instead migrated using a rearward concentration of f-actin and stable leading-edge blebs. This mode of migration occurs under planar confinement in vitro, and we correspondingly found the stratified keratinocyte layers of the epidermis impose planar-like confinement on leukocytes in vivo. By imaging the same cell type across the body, our data collectively indicates that cells adapt their migration strategy to navigate different tissue geometries in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/544270565facfa5321cf63996e9656aca0dae665" target='_blank'>
              Live imaging in zebrafish reveals tissue-specific strategies for amoeboid migration
              </a>
            </td>
          <td>
            T. Robertson, Jonathan H. Schrope, Zoe Zwick, Julie K. Rindy, Adam Horn, Anna Huttenlocher
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Single-cell RNA sequencing (scRNA-seq) has helped to elucidate the cellular composition of cancer and its microenvironment. Recent scRNA-seq studies have highlighted the heterogeneity of glioblastoma (GBM). Moreover, single-cell GBM analyses have proposed resemblance of GBM cells to radial glia and outer radial glia supporting the hypothesis that remnants of developmental tissue get reactivated in cancer. A recent study isolated neural progenitor cells (NPCs) from developing fetal human brain (gestational week 17–19) and classified NPCs based on their expression of THY1, CD24 and EGFR. Ventricular radial glia are THY1-CD24-EGFR+ whereas outer radial glia are THY1-CD24-EGFR-. Early neuron precursors are CD24+THY1-EGFR+ and glial progenitor cells (GPCs) are THY1+EGFR+. GPCs give rise to THY1+EGFR+PDGFRA+ pre-oligodendrocyte progenitor cells. The importance of EGFR in NPCs again highlights the resemblance to glioma. Methods We aimed to apply the classification above in IDH mutant astrocytoma and oligodendroglioma as well as IDHwt glioblastoma samples. We used three publicly available datasets: Wang (paired 74 IDHwt primary and recurrent samples), Tirosh (6 primary oligodendroglioma samples) and Venteicher (10 primary IDH mutant astrocytoma). Results In IDH mutant astrocytoma, 82.63% of cells express THY1+ (mostly EGFR+PDGFRA+) and 10.76% of cells are THY1-CD24-EGFR+. In oligodendroglioma, 75% of cells are THY1+ (mostly EGFR+PDGFRA+) and 12.07% are THY1-CD24-EGFR+. In IDHwt EGFR amplified primary GBM samples, 87.5% of cells are THY1-CD24-EGFR+. This percentage drops to 70.4% in the recurrent setting. THY1-CD24-EGFR-cells increase from 9.7% to 23.1% at recurrence. In IDHwt EGFRwt primary GBM samples, 48.6% of cells are THY1-CD24-EGFR+ and 44.15% are THY1-CD24-EGFR-. In the recurrent setting, 43.26% of cells are THY1-CD24-EGFR+ and 49.58% are THY1-CD24-EGFR-. Conclusion IDH mutant gliomas and IDHwt glioblastoma express different progenitor cell markers. THY1 is highly expressed in IDH mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84519f9c31ef63e22be1710cc56f409181b65fe3" target='_blank'>
              The single-cell make-up of adult diffuse glioma based on the 2021 WHO classification
              </a>
            </td>
          <td>
            I. Alnahhas, Allison Kayne, Mehak M. Khan, Wenyin Shi
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Glioblastoma is a highly aggressive brain cancer and, unlike many other cancers types, the median survival for patients after treatment (14.6 months) has barely improved in the last 20 years. Infiltrative growth into the surrounding brain parenchyma facilitates tumor recurrence and ultimately the death of the patient - novel therapies targeting this process are desperately needed. Lysyl oxidase inhibition has been shown to decrease invasive growth in a variety of solid tumours and is a potential therapy for glioblastoma patients. Methods Genes highly expressed in the mesenchymal subtype of glioblastoma were analyzed in a data set from the Cancer Genome Atlas and tissue microarrays. Two patient-derived human glioblastoma stem cell lines were used to assess the involvement of lysyl oxidase (LOX). The effect of LOX on infiltration was examined in an organotypic brain slice assay and in an orthotopic mouse model. Chemotactic assays, protease and cleavage arrays were used to assess the underlying mechanism behind LOX-mediated infiltration. The orthotopic model was used to evaluate potential clinical utility of targeting LOX in glioblastoma. Results LOX is overexpressed in the mesenchymal glioblastoma subtype and strongly associated with poor patient survival. LOX expression upregulates MMP7 expression, which subsequently cleaves the vascular matrix resulting in increased chemotaxis of glioblastoma cells. Conclusions We have uncovered a novel mechanism of glioblastoma infiltration and suggest that targeting LOX represent an effective therapeutic approach blocking glioblastoma infiltration. Importance of the study The ability of glioblastoma cells to infiltrate the surrounding normal brain tissue facilitates their evasion of current therapies, leading to tumor recurrence and ultimately the death of the patient. To improve targeted therapies for glioblastoma patients we need to understand the molecular mechanisms of glioblastoma cell infiltration and how cells interact with the unique microenvironment of the brain. We have identified a novel mechanism whereby tumor-derived LOX mediates chemotaxis of glioblastoma cells to the laminin rich perivascular niche, enabling infiltrative growth. Inhibiting this infiltrative pathway is a potential anti-invasive therapy that is desperately needed for glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb3e6c468837c0b0bfb72d57ecf46fe10ba327b" target='_blank'>
              Cleavage of the vascular matrix attracts glioblastoma cells to infiltrate the brain parenchyma
              </a>
            </td>
          <td>
            Lara Perryman, Anette M. Høye, T. Cox, Ann-Marie Baker, Jan E. Strøbech, Lidia Leonte, Lukram B Singh, Sergey Popov, Lasse G Lorentzen, Raphael Reuten, Hans Skovgaard Poulsen, Michael J Davies, Chris Jones, J. Erler
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background: Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain. Methods: Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package “Seurat” was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes. Results: Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA. Conclusion: A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="235 Background: Fibrosarcoma (FS) is a relatively uncommon subtype of soft tissue sarcoma, which predominantly occurs in the extremities and girdles of elderly individuals. FS are famous for its high recurrence and metastasis rate. However, the current treatment regimens have shown poor efficacy in treating these progressive lesions. Therefore, a deep understanding of the tumor microenvironment characteristics of fibrosarcoma is needed to develop more effective drugs.Therefore, a deeper understanding of the tumor microenvironment characteristics of fibrosarcoma is needed for further development of therapeutic drugs. Methods: We performed single-cell transcriptome sequencing for the first time on 6 patients bearing with FS. Bulk-seq data of soft tissue sarcoma was downloaded from public database including TCGA and GEO for prognostic analysis. What' s more, a cohort of 54 FS patients was retrospectively collected for protein detection. Results: The tumor microenvironment landscape was characterized by the presence of malignant fibroblast, diverse macrophages, limited T-cell presence, and high levels of vascular infiltration. The expression of siglec15 was significantly upregulated in the malignant cells. And the prognostic significance of siglec15 was confirmed by the TCGA public database. What’s more, the high expression of siglec15 protein was negatively related to overall survival of fibrosarcoma patients from our institution. We also explored the interactions among different cell subtypes, especially the interactions between malignant cells and endothelial cells, as well as macrophages and endothelial cells, which not only promote tumor progression but also inhibit T-cell infiltration and function. Conclusions: In total, our work has uncovered the low T cell infiltration landscape of FS. The primary reason of insufficient immune infiltration may be due to the blockage of T cell extravasation caused by the interaction between endothelial cells, malignant cells and macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca424f7bf6e590c667e5a83854870c663bc0d1f2" target='_blank'>
              The single cell transcriptional landscape of fibrosarcoma.
              </a>
            </td>
          <td>
            Juan Wang, Zhenfeng Li, Ning Zhang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-dimensional single-cell data poses significant challenges in identifying underlying biological patterns due to the complexity and heterogeneity of cellular states. We propose a comprehensive gene-cell dependency visualization via unsupervised clustering, Growing Hierarchical Self-Organizing Map (GHSOM), specifically designed for analyzing high-dimensional single-cell data like single-cell sequencing and CRISPR screens. GHSOM is applied to cluster samples in a hierarchical structure such that the self-growth structure of clusters satisfies the required variations between and within. We propose a novel Significant Attributes Identification Algorithm to identify features that distinguish clusters. This algorithm pinpoints attributes with minimal variation within a cluster but substantial variation between clusters. These key attributes can then be used for targeted data retrieval and downstream analysis. Furthermore, we present two innovative visualization tools: Cluster Feature Map and Cluster Distribution Map. The Cluster Feature Map highlights the distribution of specific features across the hierarchical structure of GHSOM clusters. This allows for rapid visual assessment of cluster uniqueness based on chosen features. The Cluster Distribution Map depicts leaf clusters as circles on the GHSOM grid, with circle size reflecting cluster data size and color customizable to visualize features like cell type or other attributes. We apply our analysis to three single-cell datasets and one CRISPR dataset (cell-gene database) and evaluate clustering methods with internal and external CH and ARI scores. GHSOM performs well, being the best performer in internal evaluation (CH=4.2). In external evaluation, GHSOM has the third-best performance of all methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/140c0a81733fa80520acd13c32e97603d9f10e5c" target='_blank'>
              scGHSOM: Hierarchical clustering and visualization of single-cell and CRISPR data using growing hierarchical SOM
              </a>
            </td>
          <td>
            Shang-Jung Wen, Jia-Ming Chang, Fang Yu
          </td>
          <td>2024-07-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background and aims: Colorectal carcinomas (CRCs) in patients with Lynch syndrome (LS) exhibit heightened immunogenicity due to mismatch repair deficiency (MMR-d), often resulting in favorable responses to T cell immune checkpoint therapies. Recent studies indicate that the phenotype and genotype of LS-associated CRCs vary depending on the specific MMR gene mutated. Here, we investigated whether the immune profiles of LS-associated CRCs differ based on the MMR gene defects. Methods: Tissue material from 18 MLH1-, 16 MSH2-, 40 MSH6-, and 23 PMS2-mutated CRCs and 35 sporadic MMR-d CRCs were included in the study. Imaging mass cytometry (IMC) analysis, along with targeted multiplex immunofluorescence imaging (mIF) and immunohistochemistry, were applied to examine the tumor immune microenvironment, including Human Leukocyte Antigen (HLA) class I and programmed death-ligand 1 (PD-L1) expression. Results: Unsupervised hierarchical clustering of cell phenotypes identified by IMC, followed by mIF validation, revealed comparable lymphoid and myeloid cell infiltration levels across CRCs from all MMR groups. Infiltrating T cell levels negatively correlated with the number of mutations at coding microsatellite sequences, particularly in MLH1-mutated CRCs. HLA class I defects were observed in 76% of all CRCs. These defects were more frequently accompanied by {beta}2M defects in hereditary MMR-d CRCs (67%) compared to sporadic MMR-d CRCs (37%), and did not associate with the number of {gamma}{delta} T cells, which were present in CRCs from all MMR groups. PD-L1 expression in tumor cells was only detected in 8% of all CRCs. Conclusion: Our findings illustrate that, from an immunological perspective, there is no evidence of differing immunogenic features across MMR defects. This is important to consider when developing preventive vaccine strategies and evaluating immunotherapy for LS patients and those with MMR-d CRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0cdd08de3e8c7b38b9b131001cb730c9f4dade" target='_blank'>
              Immunological profiles in Lynch syndrome colorectal cancers are not specific to mismatch repair gene defects
              </a>
            </td>
          <td>
            Noah C. Helderman, M. Ijsselsteijn, Madalina Cabuta, Manon van der Ploeg, T. Wezel, A. Ahadova, Matthias Kloor, H. Morreau, Maartje Nielsen, N. F. D. Miranda
          </td>
          <td>2024-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Phylogenetic relationship of cells within tumours can help us to understand how cancer develops in space and time, iden-tify driver mutations and other evolutionary events that enable can-cer growth and spread. Numerous studies have reconstructed phylo-genies from single-cell DNA-seq data. Here we are looking into the problem of phylogenetic analysis of spatially resolved near single-cell RNA-seq data, which is a cost-efficient alternative (or complemen-tary) data source that integrates multiple sources of evolutionary information including point mutations, copy-number changes, and epimutations. Recent attempts to use such data, although promis-ing, raised many methodological challenges. Here, we explored data-preprocessing and modelling approaches for evolutionary analyses of Visium spatial transcriptomics data. We conclude that using only highly variable genes and accounting for heterogeneous RNA capture across tissue-covered spots improves the reconstructed topological relationships and influences estimated branch lengths.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d567ed01155056fa70bf3fe42ae8fbf32987b2be" target='_blank'>
              Filtering for highly variable genes and high quality spots improves phylogenetic analysis of cancer spatial transcriptomics Visium data
              </a>
            </td>
          <td>
            Alexandra “Sasha” Gavryushkina, H. R. Pinkney, Sarah D. Diermeier, A. Gavryushkin
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tissue-resident memory T cells (TRM) are a specialized subset of long-lived memory T cells that reside in peripheral tissues. However, the impact of TRM-related immunosurveillance on the tumor-immune microenvironment (TIME) and tumor progression across various non-small-cell lung cancer (NSCLC) patient populations is yet to be elucidated. Our comprehensive analysis of multiple independent single-cell and bulk RNA-seq datasets of patient NSCLC samples generated reliable, unique TRM signatures, through which we inferred the abundance of TRM in NSCLC. We discovered that TRM abundance is consistently positively correlated with CD4+ T helper 1 cells, M1 macrophages, and resting dendritic cells in the TIME. In addition, TRM signatures are strongly associated with immune checkpoint and stimulatory genes and the prognosis of NSCLC patients. A TRM-based machine learning model to predict patient survival was validated and an 18-gene risk score was further developed to effectively stratify patients into low-risk and high-risk categories, wherein patients with high-risk scores had significantly lower overall survival than patients with low-risk. The prognostic value of the risk score was independently validated by the Cancer Genome Atlas Program (TCGA) dataset and multiple independent NSCLC patient datasets. Notably, low-risk NSCLC patients with higher TRM infiltration exhibited enhanced T-cell immunity, nature killer cell activation, and other TIME immune responses related pathways, indicating a more active immune profile benefitting from immunotherapy. However, the TRM signature revealed low TRM abundance and a lack of prognostic association among lung squamous cell carcinoma patients in contrast to adenocarcinoma, indicating that the two NSCLC subtypes are driven by distinct TIMEs. Altogether, this study provides valuable insights into the complex interactions between TRM and TIME and their impact on NSCLC patient prognosis. The development of a simplified 18-gene risk score provides a practical prognostic marker for risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1e637bee54083c5c2fb1deaba9e34a97001048" target='_blank'>
              A comprehensive meta-analysis of tissue resident memory T cells and their roles in shaping immune microenvironment and patient prognosis in non-small cell lung cancer
              </a>
            </td>
          <td>
            Aidan Shen, Aliesha Garrett, Cheng-chi Chao, Dongliang Liu, Chao Cheng, Zhaohui Wang, Chen Qian, Yangzhi Zhu, Junhua Mai, Chongming Jiang
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The global incidence of cancer is increasing, with estimates suggesting that there will be 26 million new cases and 17 million deaths per year by 2030. Cancer stem cells (CSCs) and extracellular vesicles (EVs) are key to the resistance and advancement of cancer. They play a crucial role in tumor dynamics and resistance to therapy. CSCs, initially discovered in acute myeloid leukemia, are well-known for their involvement in tumor initiation, progression, and relapse, mostly because of their distinct characteristics, such as resistance to drugs and the ability to self-renew. EVs, which include exosomes, microvesicles, and apoptotic bodies, play a vital role in facilitating communication between cells within the tumor microenvironment (TME). They have a significant impact on cellular behaviors and contribute to genetic and epigenetic changes. This paper analyzes the mutually beneficial association between CSCs and EVs, emphasizing their role in promoting tumor spread and developing resistance mechanisms. This review aims to investigate the interaction between these entities in order to discover new approaches for attacking the complex machinery of cancer cells. It highlights the significance of CSCs and EVs as crucial targets in the advancement of novel cancer treatments, which helps stimulate additional research, promote progress in ideas for cancer treatment, and provide renewed optimism in the effort to reduce the burden of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b17e9728aa76af6a02ab4a44623aab4c0608b6" target='_blank'>
              Extracellular vesicles and cancer stem cells: a deadly duo in tumor progression
              </a>
            </td>
          <td>
            Akram Tayanloo-Beik, Azin Eslami, Masoumeh Sarvari, H. Jalaeikhoo, Mohsen Rajaeinejad, Mohsen Nikandish, Ali Faridfar, Mostafa Rezaei-Tavirani, A. R. Mafi, Bagher Larijani, B. Arjmand
          </td>
          <td>2024-07-18</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ability to genetically edit primary B cells via CRISPR/Cas9 technology represents a powerful tool to study molecular mechanisms of B ‐ cell pathogenesis. In this context, employing ribonucleoprotein complexes (RNPs), formed by recombinant Cas9 and genome ‐ targetting single guide RNA molecules, brings in advantage of accelerated set ‐ up and protocol robustness. Gene editing via RNP electroporation has been recently applied to primary tumor cells isolated from patients chronic lymphocytic leukemia (CLL), suggesting an efficient and valuable tool for studying leukemic cell biology and biomarker validation. 1,2 The work by Nardi et al. on this topic proposed to electroporate unmanipulated primary CLL cells that are subsequently put in culture with human CD40L ‐ expressing fibroblasts and soluble stimuli to promote CLL cell proliferation. In this context, cellular proliferation is required to achieve homozygous gene editing, whereas in unstimulated CLL cells it is possible to achieve only the heterozygous editing. 1 The method published by Mateos ‐ Jaimez et al. relies on the preactivation of CLL cells with CD40L/BAFF/IL ‐ 21 ‐ expressing stromal cells, followed by RNP electroporation and continuation of the stimulatory coculture. 2 Both methods approach 80% – 90% of editing efficiency and allow to perform downstream in vitro experiments on edited leukemic cells. Application of a similar RNP ‐ based editing approach to the widely used murine">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce4b4f9dae608c0b0ba780ba2b6869894794e80" target='_blank'>
              Robust and cost‐effective CRISPR/Cas9 gene editing of primary tumor B cells in Eµ‐TCL1 model of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Rosita Del Prete, Roberta Drago, Federica Nardi, Gaia Bartolini, Erika Bellini, Antonella De Rosa, Silvia Valensin, Anna Kabanova
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3471e25fd3d93db955069ff8713da95ffc83f905" target='_blank'>
              Single-cell multi-omic and spatial profiling of human kidneys implicates the fibrotic microenvironment in kidney disease progression.
              </a>
            </td>
          <td>
            Amin Abedini, Jonathan Levinsohn, Konstantin A. Klötzer, Bernhard Dumoulin, Ziyuan Ma, Julia Frederick, Poonam Dhillon, Michael S. Balzer, Rojesh Shrestha, Hongbo Liu, Steven Vitale, Andi M. Bergeson, Kishor B. Devalaraja-Narashimha, Paola Grandi, Tanmoy Bhattacharyya, Erding Hu, Steven S. Pullen, Carine M. Boustany-Kari, Paolo Guarnieri, Anil Karihaloo, D. Traum, Hanying Yan, Kyle Coleman, Matthew Palmer, Lea Sarov-Blat, L. Morton, Christopher A. Hunter, Klaus H. Kaestner, Mingyao Li, K. Suszták
          </td>
          <td>2024-07-24</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a219c571f234a60e0f1bec97d940766352b405a4" target='_blank'>
              The cross-talk between the macro and micro-environment in precursor lesions of pancreatic cancer leads to new and promising circulating biomarkers
              </a>
            </td>
          <td>
            Carla Mottini, Francesca Romana Auciello, I. Manni, Christian Pilarsky, D. Caputo, Giulio Caracciolo, Alessandro Rossetta, E. Di Gennaro, Alfredo Budillon, Giovanni Blandino, M. Roca, Giulia Piaggio
          </td>
          <td>2024-07-18</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Vulvar diseases are a critical yet often neglected area of women’s health, profoundly affecting patients’ quality of life and frequently resulting in long-term physical and psychological challenges. Lichen sclerosus (LS) is a chronic inflammatory skin disorder that predominantly affects the vulva, leading to severe itching, pain, scarring, and an increased risk of malignancy. Despite its profound impact on affected individuals, the molecular pathogenesis of vulvar LS (VLS) is not well understood, hindering the development of FDA-approved therapies. Here, we utilize single-cell and spatial transcriptomics to analyze lesional and non-lesional skin from VLS patients, as well as healthy control vulvar skin. Our findings demonstrate histologic, cellular, and molecular heterogeneities within VLS, yet highlight unifying molecular changes across keratinocytes, fibroblasts, immune cells, and melanocytes in lesional skin. They reveal cellular stress and damage in fibroblasts and keratinocytes, enhanced T cell activation and cytotoxicity, aberrant cell-cell signaling, and increased activation of the IFN, JAK/STAT, and p53 pathways in specific cell types. Using both monolayer and organotypic culture models, we also demonstrate that knockdown of select genes, which are downregulated in VLS lesional keratinocytes, partially recapitulates VLS-like stress-associated changes. Collectively, these data provide novel insights into the pathogenesis of VLS, identifying potential biomarkers and therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a1ede19b990ffc821eea644860eccc4ba0949" target='_blank'>
              Single-cell and spatial transcriptomics of vulvar lichen sclerosus reveal multi-compartmental alterations in gene expression and signaling cross-talk
              </a>
            </td>
          <td>
            Peng Sun, Christina N. Kraus, Wei Zhao, Jiahui Xu, Susie Suh, Quy Nguyen, Yunlong Jia, Arjun Nair, Melanie Oakes, Roberto Tinoco, Jessica Shiu, Bryan Sun, Ashley Elsensohn, Scott X. Atwood, Qing Nie, Xing Dai
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94635b24c0b480439c040348f97ec34458661c58" target='_blank'>
              Glioma actively orchestrate a self-advantageous extracellular matrix to promote recurrence and progression
              </a>
            </td>
          <td>
            Ruolun Wei, Jiasheng Zhou, Brandon Bui, Xianzhi Liu
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a malignancy with poor survival and high rates of chemoresistance. Temozolomide (TMZ), the standard-of-care chemotherapy for GBM patients, but GBM cells can be resistant to TMZ, resulting in limited clinical efficacy. Elucidating the complex mechanisms of TMZ chemoresistance in GBM requires novel in vitro models replicating the complex tumor microenvironment (TME). We present an multicellular 3D GBM model recapitulating the biomechanical characteristics of brain tissues and pericyte-mediated TMZ resistance. The composite hydrogel used to encapsulate GBM spheroids (U87, LN229, and PDM140), pericytes, or GBM spheroids with pericytes, mimics the rheological properties of brain tissues (G’∼800Pa and G”∼100Pa). When untreated, the GBM models remain viable and proliferative for 14 days. PDM140 spheroids were most sensitive to TMZ (IC50=73μM), followed by LN229 (IC50=278μM) and U87 (IC50=446μM). With pericytes, the viability of TMZ-treated GBM spheroids significantly increases by 22.7% for PDM140, 32.5% for LN229, and 22.1% for U87, confirming pericyte-induced GBM chemoresistance responses. The upregulation (380-fold) of C-C motif chemokine ligand 5 (CCL5) in pericytes upon TMZ treatment could explain the chemoresistance responses. This innovative brain-mimicking 3D GBM model represents a novel in vitro platform for testing the efficacy of TMZ and novel drugs targeting CCL5-mediated chemoresistance pathways in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e44682a4e27a5b6203cda330b0d2d84b547523eb" target='_blank'>
              Microphysiological system modeling pericyte-induced temozolomide resistance in glioblastoma
              </a>
            </td>
          <td>
            Surjendu Maity, Christopher Jewell, Can Yilgor, S. Kawakita, Saurabh Sharma, Alejandro Gomez, Marvin Mecwan, Natashya Falcone, M. Ermis, Mahsa Monirizad, Negar Hosseinzadeh Kouchehbaghi, Fatemeh Zehtabi, Danial Khorsandi, M. Dokmeci, Diogo Moniz-Garcia, Alfredo Quiñones-Hinojosa, Ali Khademhosseini, Vadim Jucaud
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b73c0404cfe6f4a410aecb14a07d1001cfb07164" target='_blank'>
              Astrocytes and the tumor microenvironment inflammatory state dictate the killing of glioblastoma cells by Smac mimetic compounds
              </a>
            </td>
          <td>
            Kyle Malone, Melanie Dugas, N. Earl, Tommy Alain, E. LaCasse, S. Beug
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, Mirjam E Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="One critical step in the metastatic cascade is the survival of circulating tumour cells (CTCs) within the bloodstream. While numerous interactions between CTCs and various hematopoietic cells have been described, the role of red blood cells (RBCs) in this process remains underexplored. This study investigates the interactions between tumour cells and RBCs from breast and lung cancer patients, revealing significant phenotypic and functional changes in the tumour cells, unlike when the contact is with RBCs from healthy donors. In vitro co-culture of cancer cell lines with RBCs from metastatic cancer patients resulted in increased tumour cell attachment accompanied by morphological changes. Additionally, RBCs-primed tumour cells showed increased adhesion and disruption of the endothelial barrier in vitro and increased invasiveness both in vitro and in vivo. Transcriptomic analysis showed that RBCs from metastatic breast cancer patients induce significant gene expression changes, notably upregulating PAK4, which enhances migration and epithelial-mesenchymal transition. PAK4 inhibition reduced these effects. Proteomic studies revealed substantial remodelling, including actin-related changes and the accumulation of VASP at cell edges, promoting directional migration. Clinically, higher RBC distribution width (RDW) in metastatic breast cancer patients is associated with increased CTC counts and worse outcome. This study highlights the previously unrecognized role of RBCs in promoting metastatic behaviours in cancer cells and suggests potential therapeutic targets, such as PAK4, to counteract these effects. Further exploration of RBCs-tumour cell interactions could provide new insights into metastatic mechanisms and improve cancer prognosis and treatment strategies. Key Points This study reveals the previously unknown role of RBCs in enhancing tumour cell invasiveness and metastatic potential. Tumour cells undergo significant phenotypic and functional changes after contact with RBCs from cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a5495cc85336152bb18294f77e87fc3e41607e1" target='_blank'>
              Red blood cell-tumour cell interactions promote tumour cell progression
              </a>
            </td>
          <td>
            Thais Pereira-Veiga, Celso Yáñez-Gómez, Aleksi Pekkarinen, C. Abuín, Christine Blechner, M. González-Conde, Alexander T. Bauer, S. Vidal-y-Sy, Ayham Moustafa, B. Siebels, A. B. Dávila-Ibáñez, P. Cabezas-Sáinz, Maider Santos, Laura Sánchez, Joao Gorgulho, Julian Götze, Kira Meißner, Juan Cueva, P. Palacios, Alexia Cortegoso, T. Curiel, Carmela Rodríguez, Marta Carmona, S. Peine, Milena Schmidt, N. Heuer-Olewinski, Martin Reck, M. Abdo, Rafael López-López, S. Windhorst, Klaus Pantel, H. Wikman, Clotilde Costa
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Autophagy is known to suppress tumor initiation by removing genotoxic stresses in normal cells. Conversely, autophagy is also known to support tumor progression by alleviating metabolic stresses in neoplastic cells. Centered on this pro-tumor role of autophagy, there have been many clinical trials to treat cancers through systemic blocking of autophagy. Such systemic inhibition affects both tumor cells and non-tumor cells, and the consequence of blocked autophagy in non-tumor cells in the context of tumor microenvironment is relatively understudied. Here, we examined the effect of autophagy-deficient myeloid cells on the progression of autophagy-competent tumors. We found that blocking autophagy only in myeloid cells modulated tumor progression markedly but such effects were context dependent. In a tumor implantation model, the growth of implanted tumor cells was substantially reduced in mice with autophagy-deficient myeloid cells; T cells infiltrated deeper into the tumors and were responsible for the reduced growth of the implanted tumor cells. In an oncogene-driven tumor induction model, however, tumors grew faster and metastasized more in mice with autophagy- deficient myeloid cells. These data demonstrate that the autophagy status of myeloid cells plays a critical role in tumor progression, promoting or suppressing tumor growth depending on the context of tumor-myeloid cell interactions. This study indicates that systemic use of autophagy inhibitors in cancer therapy may have differential effects on rates of tumor progression in patients due to effects on myeloid cells and that this warrants more targeted use of selective autophagy inhibitors in a cancer therapy in a clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671d32f402e6a6f65be574e029a3642f0e00b452" target='_blank'>
              Context-dependent roles for autophagy in myeloid cells in tumor progression
              </a>
            </td>
          <td>
            Jayoung Choi, Gayoung Park, Steve Seung-Young Lee, Erin M Dominici, Lev Becker, Kay F. Macleod, Stephen J. Kron, Seungmin Hwang
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gastric signet-ring cell carcinoma (GSRCC) is a subtype of gastric cancer with distinct phenotype and high risk of peritoneal metastasis. Studies have shown that early GSRCC has a good prognosis, while advanced GSRCC is insensitive to radiotherapy, chemotherapy or immune checkpoint blockade therapy. With technological advancement of single-cell RNA sequencing analysis and cytometry by time of flight mass cytometry, more detailed atlas of tumor microenvironment (TME) in GSRCC and its association with prognosis could be investigated extensively. Recently, two single-cell RNA sequencing studies revealed that GSRCC harbored a unique TME, manifested as highly immunosuppressive, leading to high immune escape. The TME of advanced GSRCC was enriched for immunosuppressive factors, including the loss of CXCL13 +-cluster of differentiation 8+-Tex cells and declined clonal crosstalk among populations of T and B cells. In addition, GSRCC was mainly infiltrated by follicular B cells. The increased proportion of SRCC was accompanied by a decrease in mucosa-associated lymphoid tissue-derived B cells and a significant increase in follicular B cells, which may be one of the reasons for the poor prognosis of GSRCC. By understanding the relationship between immunosuppressive TME and poor prognosis in GSRCC and the underlying mechanism, more effective immunotherapy strategies and improved treatment outcomes of GSRCC can be anticipated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca905db87222d85a29c55ed74cef0a1a4cac5a72" target='_blank'>
              Immunosuppressive tumor microenvironment in gastric signet-ring cell carcinoma
              </a>
            </td>
          <td>
            Yu-Qiong Xie, Chun-Chun Li, Mei-Rong Yu, Jiang Cao
          </td>
          <td>2024-09-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbea514dd68ac66e3d2deacc382cefe854e64feb" target='_blank'>
              Mechanism insights and therapeutic intervention of tumor metastasis: latest developments and perspectives
              </a>
            </td>
          <td>
            Xiaoli Shi, Xinyi Wang, Wentao Yao, Dongmin Shi, Xihuan Shao, Zhengqing Lu, Yue Chai, Jinhua Song, Weiwei Tang, Xuehao Wang
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A distinct feature of pancreatic ductal adenocarcinoma (PDAC) is a prominent tumor microenvironment (TME) with remarkable cellular and spatial heterogeneity that meaningfully impacts disease biology and treatment resistance. The dynamic crosstalk between cancer cells and the dense stromal compartment leads to spatially and temporally heterogeneous metabolic alterations, such as acidic pH that contributes to drug resistance in PDAC. Thus, monitoring the extracellular pH metabolic fluctuations within the TME is crucial to predict and to quantify anticancer drug efficacy. Here, a simple and reliable alginate-based 3D PDAC model embedding ratiometric optical pH sensors and cocultures of tumor (AsPC-1) and stromal cells for simultaneously monitoring metabolic pH variations and quantify drug response is presented. By means of time-lapse confocal laser scanning microscopy (CLSM) coupled with a fully automated computational analysis, the extracellular pH metabolic variations are monitored and quantified over time during drug testing with gemcitabine, folfirinox, and paclitaxel, commonly used in PDAC therapy. In particular, the extracellular acidification is more pronounced after drugs treatment, resulting in increased antitumor effect correlated with apoptotic cell death. These findings highlight the importance of studying the influence of cellular metabolic mechanisms on tumor response to therapy in 3D tumor models, this being crucial for the development of personalized medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dceab42d7b7c7a1f2eb735efea89f11000f608" target='_blank'>
              A 3D Pancreatic Cancer Model with Integrated Optical Sensors for Noninvasive Metabolism Monitoring and Drug Screening.
              </a>
            </td>
          <td>
            Anna Chiara Siciliano, S. Forciniti, V. Onesto, Helena Iuele, Donatella Delle Cave, Federica Carnevali, Giuseppe Gigli, E. Lonardo, L. L. del Mercato
          </td>
          <td>2024-07-08</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac24e0e7a39e90599e20a198363cbf479bb3749d" target='_blank'>
              Using a pan-cancer atlas to investigate tumour associated macrophages as regulators of immunotherapy response
              </a>
            </td>
          <td>
            Alexander Coulton, Jun Murai, Danwen Qian, Krupa Thakkar, Claire E Lewis, Kevin Litchfield
          </td>
          <td>2024-07-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, Philip Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b7af48216a0c84346ea77c18dc911a1e93afbf" target='_blank'>
              Endothelial cell heterogeneity in colorectal cancer: tip cells drive angiogenesis
              </a>
            </td>
          <td>
            Zhenyu Xie, Liaoran Niu, Kunli Du, Ling Chen, Gaozan Zheng, Songchen Dai, Han-Long Dan, Lili Duan, Xinyu Dou, Fan Feng, Jian Zhang, Jianyong Zheng
          </td>
          <td>2024-08-22</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="High-throughput sequencing of single-cell data can be used to rigorously evlauate cell specification and enable intricate variations between groups or conditions. Many popular existing methods for differential expression target differences in aggregate measurements (mean, median, sum) and limit their approaches to detect only global differential changes. We present a robust method for differential expression of single-cell data using a kernel-based score test, cytoKernel. cytoKernel is specifically designed to assess the differential expression of single cell RNA sequencing and high-dimensional flow or mass cytometry data using the full probability distribution pattern. cytoKernel is based on kernel embeddings which employs the probability distributions of the single cell data, by calculating the pairwise divergence/distance between distributions of subjects. It can detect both patterns involving aggregate changes, as well as more elusive variations that are often overlooked due to the multimodal characteristics of single cell data. We performed extensive benchmarks across both simulated and real data sets from mass cytometry data and single-cell RNA sequencing. The cytoKernel procedure effectively controls the False Discovery Rate (FDR) and shows favourable performance compared to existing methods. The method is able to identify more differential patterns than existing approaches. We apply cytoKernel to assess gene expression and protein marker expression differences from cell subpopulations in various publicly available single-cell RNAseq and mass cytometry data sets. The methods described in this paper are implemented in the open-source R package cytoKernel, which is freely available from Bioconductor at http://bioconductor.org/packages/cytoKernel.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5266fa2edd1942a8b424629b41d58db895ce6a73" target='_blank'>
              cytoKernel: Robust kernel embeddings for assessing differential expression of single cell data
              </a>
            </td>
          <td>
            Tusharkanti Ghosh, Ryan M. Baxter, Souvik Seal, Victor G Lui, Pratyadipta Rudra, Thao Vu, Elena W Y Hsieh, Debashis Ghosh
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9c5b5959b41a250a5ef58adb37088e34a07297" target='_blank'>
              Immuno-oncologic profiling of pediatric brain tumors reveals major clinical significance of the tumor immune microenvironment
              </a>
            </td>
          <td>
            Adrian B Levine, L. Nobre, Anirban Das, Scott Milos, V. Bianchi, Monique Johnson, Nick Fernandez, L. Stengs, S. Ryall, Michelle Ku, Mansuba Rana, Benjamin Laxer, J. Sheth, Stefanie-Grace Sbergio, Ivana Fedorakova, Vijay Ramaswamy, J. Bennett, R. Siddaway, U. Tabori, Cynthia Hawkins
          </td>
          <td>2024-07-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Elderly individuals have higher disease susceptibility and lower vaccine responsiveness, highlighting the need to better comprehend the aging immune system and its clinical associations. Here we conducted a deep immune profiling study of 550 elderly individuals (61–94 years) and 100 young adults (22–38 years). Utilizing high-dimensional spectral flow cytometry to identify 97 immune cell populations and 48-plex cytokine profiling, we detailed intricate age-and sex-related changes in the elderly immune system at an unprecedented depth. Synthesizing information from clinical, laboratory, and immunological data through an integrative multi-block analysis, we reveal overarching systems-level signatures of aging, such as increased concentrations of specific cytokines and frequencies of defined innate and adaptive immune cell subpopulations. Extending this approach, we identified unique immune signatures of smoking, obesity, and several diseases including osteoporosis, heart failure and gout. Our systems biology approach enables to uncover new relationships between clinical characteristics and immunological traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75977a9130047fe0c681122378f90c39329e640" target='_blank'>
              Integrative deep immune profiling of the elderly reveals systems-level signatures of aging, sex, smoking, and clinical traits
              </a>
            </td>
          <td>
            Lennart Riemann, Rodrigo Gutierrez, Iva Odak, Joana Barros-Martins, L. Roesner, Ximena Leon Lara, Christine Falk, T. F. Schulz, Gesine Hansen, T. Werfel, Reinhold Förster
          </td>
          <td>2024-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ex vivo 3D culture of human tissue explants addresses many limitations of traditional monolayer cell culture techniques, namely the lack of cellular heterogeneity and absence of 3D intercellular spatial relationships, but presents challenges with regard to repeatability due to the difficulty of acquiring multiple tissue samples from the same donor. In this study, we used a cryopreserved bank of human lung microexplants , ~1 mm3 fragments of peripheral lung from donors undergoing lung resection surgery, and a liquid-like-solid (LLS) 3D culture matrix to describe a method for the analysis of non-small cell lung cancer (NSCLC) adhesion to human lung tissue. H226 (squamous cell carcinoma), H441 (lung adenocarcinoma), and H460 (large cell carcinoma) cell lines were co-cultured with lung microexplants. Confocal fluorescence microscopy was used to visualize the adherence of each cell line to lung microexplants. Adherent cancer cells were quantified following filtration of non-adherent cells, digestion of cultured microexplants, and flow cytometry. This method was used to evaluate the role of integrins in cancer cell adherence. A statistically significant decrease in the adherence of H460 cells to lung microexplants was observed when anti-integrins were administered to H460 cells prior to co-culture with lung microexplants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452a0dd8222f8b60b6e4bae0fff63990ba8c2bc2" target='_blank'>
              3D Culture Analysis of Cancer Cell Adherence to Ex Vivo Lung Microexplants.
              </a>
            </td>
          <td>
            N. Diodati, Zadia Dupee, Felipe Lima, Jack E Famiglietti, R. Smolchek, Ganlin Qu, Yana Goddard, Duy Nguyen, Gregory Sawyer, Edward Phelps, B. Mehrad, Matthew Schaller
          </td>
          <td>2024-07-30</td>
          <td>Tissue engineering. Part C, Methods</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinomas (RCC) are characterized by their largely diverse clinical outcomes. Leveraging genomic diversity among individual tumors, our research during the past decade has concentrated on developing personalized medicine approaches for the most common and the most aggressive type of RCC, clear cell renal carcinoma (ccRCC). We generated the largest dataset of ccRCC including somatic genomic and clinical annotations for over 940 ccRCC patients, and developed a genomic classifier, based on mutational status of 12 RCC-relevant genes, which is able to stratify patients according to their risk of relapse after nephrectomy and/or death due to RCC. Furthermore, we noticed the presence of mesenchymal-like cellular phenotypes in tumors of high-risk patients, representsing a de-differentiation process, known as sarcomatoic or rhabdoid (S/R) de-differentation. Methods The current knowledge about molecular mechanisms that may drive S/R de-differentiation is primarily generated from molecular profiling of bulk tumors, collecting data from a mixture of cells that co-exist in the tumor milieu. Therefore, high-resolution studies that precisely define molecular characteristics of different cellular phenotypes associated with S/R features are missing. We have used spatial transcriptomic profiling to investigate molecular mechanisms that underline S/R de-differentationin RCC. While preserving tissue context, we applied spatial whole human transcriptome profiling to areas exhibiting S/R or clear cell phenotypes within the same tumor specimens to generate phenotype-specific transcriptome profiles. In addition, we applied whole-exome sequencing (WES) to independent areas that exhibit different differentiation phenotype withing same tumors. Results Pathway and network analysis of genesets with upregulation in each area have revealed meaningful differences in cellular pathways which are active in each phenotype. Specifically, we observed that dysregulation of extracellular matrix (ECM) is a hallmark of S/R areas within RCC tumors. Furthermore, WES of phenotypically-distinct areas within each tumor has shed new light son the genome evoltuion of S/R phenotypes. Conclusions S/R dedifferentiation in RCC tumors is characterized by specific genomic evolutionary patterns and substantial dysregulation of ECM components. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad0d8a5a2c80baecbe7aa769327da96aacd5a05" target='_blank'>
              9 Delving into molecular underpinnings of sarcomatoid/rhabdoid dedifferentioation in renal cancers using spatial profiling
              </a>
            </td>
          <td>
            Y. Riazalhosseini
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite improvements in vaccination, screening, and treatment, cervical cancer (CC) remains a major healthcare problem on a global scale. The tumor microenvironment (TME) plays an important and controversial role in cancer development, and the mechanism of the tumor’s escape from immunological surveillance is still not clearly defined. We aim to investigate the expression of CD68 and CD47 in patients with different histological variants of CC, tumor characteristics, and burden. This is a retrospective cohort study performed on paraffin-embedded tumor tissues from 191 patients diagnosed with CC between 2014 and 2021 at the Medical University Pleven, Bulgaria. Slides for immunohistochemical (IHC) evaluation were obtained, and the expression of CD68 was scored in intratumoral (IT) and stromal (ST) macrophages (CD68+cells) using a three-point scoring scale. The CD47 expression was reported as an H-score. All statistical analyses were performed using R v. 4.3.1 for Windows. Infiltration by CD68-IT cells in the tumor depended on histological type and the expression of CD47. Higher levels of the CD47 H-score were significantly more frequent among patients in the early stage. Higher levels of infiltration by CD68-ST cells were associated with worse prognosis, and the infiltration of CD68-IT cells was associated with reduced risk of death from neoplastic disease. TME is a complex ecosystem that has a major role in the growth and development of tumors. Macrophages are a major component of innate immunity and, when associated with a tumor process, are defined as TAM. Tumor cells try to escape immunological surveillance in three ways, and one of them is reducing immunogenicity by the overexpression of negative coreceptors by T-lymphocytes and their ligands on the surface of tumor cells. One such mechanism is the expression of CD47 in tumor cells, which sends a “don’t eat me” signal to the macrophages and, thus, prevents phagocytosis. To our knowledge, this is the first study that has tried to establish the relationship between the CD47 and CD68 expression levels and some clinicopathologic features in CC. We found that the only clinicopathological feature implicating the level of CD68 infiltration was the histological variant of the tumor, and only for CD68-IT–high levels were these observed in SCC. High levels of CD47 expression were seen more frequently in pT1B than pT2A and pT2B in the FIGO I stage than in the FIGO II and III stages. Infiltration by large numbers of CD68-IT cells was much more common among patients with a high expression of CD47 in tumor cells. A high level of infiltration by CD68-ST cells was associated with a worse prognosis, and a high level of infiltration by CD68-ST cells was associated with a lower risk of death from cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1d722e10db8e0eaed5ccb761e956968ed104a0" target='_blank'>
              Integrated Analysis of Phagocytic and Immunomodulatory Markers in Cervical Cancer Reveals Constellations of Potential Prognostic Relevance
              </a>
            </td>
          <td>
            Angel Yordanov, Polina Damyanova, Mariela Vasileva-Slaveva, Ihsan Hasan, S. Kostov, V. Shivarov
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6640869137efbf815e49406335626c0344fac69" target='_blank'>
              Age-associated clonal B cells drive B cell lymphoma in mice.
              </a>
            </td>
          <td>
            José P Castro, A. Shindyapina, Alessandro Barbieri, Kejun Ying, Olga S. Strelkova, Joao A. Paulo, A. Tyshkovskiy, Rico Meinl, C. Kerepesi, Anna P. Petrashen, Marco Mariotti, Margarita V Meer, Yan Hu, Alexander Karamyshev, Grigoriy Losyev, Mafalda Galhardo, Elsa Logarinho, Artur A. Indzhykulian, Steven P. Gygi, J. M. Sedivy, John P Manis, V. N. Gladyshev
          </td>
          <td>2024-08-08</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b983ff02239e55a5a088e11ea44463f2a191dc" target='_blank'>
              Spatial computational modelling illuminates the role of the tumour microenvironment for treating glioblastoma with immunotherapies
              </a>
            </td>
          <td>
            Blanche Mongeon, Julien Hébert-Doutreloux, Anudeep Surendran, Elham Karimi, B. Fiset, D. Quail, Logan A Walsh, A. Jenner, Morgan Craig
          </td>
          <td>2024-08-18</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Prostate cancer remains the most commonly diagnosed and the second leading cause of cancer-related deaths in men in the United States. The neoplastic transformation of prostate epithelia, concomitant with modulations in the stromal compartment, known as reactive stromal response, is critical for the growth, development, and progression of prostate cancer. Reactive stroma typifies an emergent response to disrupted tissue homeostasis commonly observed in wound repair and pathological conditions such as cancer. Despite the significance of reactive stroma in prostate cancer pathobiology, our understanding of the ontogeny, phenotypic and functional heterogeneity, and reactive stromal regulation of the immune microenvironment in prostate cancer remains limited. Traditionally characterized to have an immunologically "cold" tumor microenvironment, prostate cancer presents significant challenges for advancing immunotherapy compared to other solid tumors. This review explores the detrimental role of reactive stroma in prostate cancer, particularly its immunomodulatory function. Understanding the molecular characteristics and dynamic transcriptional program of the reactive stromal populations in tandem with tumor progression could offer insights into enhancing immunotherapy efficacy against prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd12cec3359f8ec93eb34750a8c77daa465b3613" target='_blank'>
              The reactive stroma response regulates the immune landscape in prostate cancer
              </a>
            </td>
          <td>
            Rintu Thomas, John Michael Jerome, Kimiko L. Krieger, Naghmana Ashraf, David R. Rowley
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf603a0e9cde601ac9006c72d111f317f71de81" target='_blank'>
              Single-cell profiling reveals altered immune landscape and impaired NK cell function in gastric cancer liver metastasis.
              </a>
            </td>
          <td>
            Xiaolong Tang, Lei Gao, Xingzhi Jiang, Zhenyu Hou, Yiwen Wang, Shiyang Hou, Hui Qu
          </td>
          <td>2024-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e22c8bf25d5562d254c88fd4d0c03d7186f26ff8" target='_blank'>
              Vascular galectins in tumor angiogenesis and cancer immunity
              </a>
            </td>
          <td>
            Victor Thijssen
          </td>
          <td>2024-07-01</td>
          <td>Seminars in Immunopathology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
Co-occurring mutations in KEAP1 and STK11KRAS have emerged as determinants of survival outcomes in non-small cell lung cancer (NSCLC) patients treated with immunotherapy. However, these mutational contexts identify a fraction of non-responders to immune checkpoint inhibitors. We hypothesized that KEAP1 wild-type tumors recapitulate the transcriptional footprint of KEAP1 mutations, and that this KEAPness phenotype can determine immune responsiveness with higher precision compared to mutation-based models.


EXPERIMENTAL DESIGN
The TCGA was used to infer the KEAPness phenotype and explore its immunological correlates at the pan-cancer level. The association between KEAPness and survival outcomes was tested in two independent cohorts of advanced NSCLC patients treated with immunotherapy and profiled by RNA-Seq (SU2C n=153; OAK/POPLAR n=439). The NSCLC TRACERx421 multi-region sequencing study (tumor regions n=947) was used to investigate evolutionary trajectories.


RESULTS
KEAPness-dominant tumors represented 50% of all NSCLCs and were associated with shorter progression-free survival (PFS) and overall survival (OS) compared to KEAPness-free cases in independent cohorts of NSCLC patients treated with immunotherapy (SU2C PFS P=0.042, OS P=0.008; OAK/POPLAR PFS P=0.0014, OS P<0.001). Patients with KEAPness tumors had survival outcomes comparable to those with KEAP1-mutant tumors. In the TRACERx421, KEAPness exhibited limited transcriptional intratumoral heterogeneity and immune exclusion, resembling the KEAP1-mutant disease. This phenotypic state occurred across genetically divergent tumors, exhibiting shared and private cancer genes under positive selection when compared to KEAP1-mutant tumors.


CONCLUSIONS
We identified a KEAPness phenotype across evolutionary divergent tumors. KEAPness outperforms mutation-based classifiers as a biomarker of inferior survival outcomes in NSCLC patients treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea8af08e6c600cf8a24f58a7981fb66121f005e" target='_blank'>
              Transcriptional phenocopies of deleterious KEAP1 mutations correlate with survival outcomes in lung cancer treated with immunotherapy.
              </a>
            </td>
          <td>
            S. Scalera, B. Ricciuti, D. Marinelli, M. Mazzotta, Laura Cipriani, Giulia Bon, Giulia Schiavoni, I. Terrenato, Alessandro Di Federico, J. Alessi, Maurizio Fanciulli, L. Ciuffreda, F. de Nicola, F. Goeman, G. Caravagna, Daniele Santini, R. De Maria, F. Cappuzzo, G. Ciliberto, M. Jamal-Hanjani, Mark M Awad, N. Mcgranahan, M. Maugeri-Saccà
          </td>
          <td>2024-07-09</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Simple Summary Cutaneous squamous cell carcinoma (cSCC) is a complex disease arising from an interplay of UV-induced DNA damage, genetic mutations, and alterations in the tumor microenvironment. While most cSCC cases can be treated with surgery, metastatic and locally advanced forms have a significantly worse prognosis and limited treatment options. The high mutational burden in cSCC makes it challenging to identify true driver alterations. Beyond genetics, the tumor microenvironment and its interaction with the immune system play a critical role in cSCC progression, evidenced by differences observed between immunocompromised and immunocompetent patients. In recent years, new therapeutic approaches have emerged, including immunotherapy and EGFR inhibitors, which have shown efficacy in treating advanced cSCC. Understanding the underlying mechanisms driving cSCC is crucial for developing novel and more effective therapies. This review aims to provide a comprehensive overview of the mutational landscape and microenvironmental factors in cSCC. Abstract Cutaneous squamous cell carcinoma (cSCC) manifests through the complex interactions of UV-induced DNA damage, genetic mutations, and alterations in the tumor microenvironment. A high mutational burden is present in cSCC, as well as both cSCC precursors and normal skin, making driver genes difficult to differentiate. Despite this, several key driver genes have been identified, including TP53, the NOTCH family, CDKN2A, PIK3CA, and EGFR. In addition to mutations, the tumor microenvironment and the manipulation and evasion of the immune system play a critical role in cSCC progression. Novel therapeutic approaches, such as immunotherapy and EGFR inhibitors, have been used to target these dysregulations, and have shown promise in treating advanced cSCC cases, emphasizing the need for targeted interventions considering both genetic and microenvironmental factors for improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064418ffcbe98e414731e1ea32740ff9d65441f2" target='_blank'>
              The Mutational and Microenvironmental Landscape of Cutaneous Squamous Cell Carcinoma: A Review
              </a>
            </td>
          <td>
            Tara M. Hosseini, Soo J. Park, Theresa W Guo
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, M. Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer brain metastasis has a poor prognosis, is commonly observed in clinical practice, and the number of cases is increasing as overall cancer survival improves. However, experiments in mouse models have shown that brain metastasis itself is an inefficient process. One reason for this inefficiency is the brain microenvironment, which differs significantly from that of other organs, making it difficult for cancer cells to adapt. The brain microenvironment consists of unique resident cell types such as neurons, oligodendrocytes, astrocytes, and microglia. Accumulating evidence over the past decades suggests that the interactions between cancer cells and glial cells can positively or negatively influence the development of brain metastasis. Nevertheless, elucidating the complex interactions between cancer cells and glial cells remains challenging, in part due to the limitations of existing experimental models for glial cell culture. In this review, we first provide an overview of glial cell culture methods and then examine recent discoveries regarding the interactions between brain metastatic cancer cells and the surrounding glial cells, with a special focus on astrocytes and microglia. Finally, we discuss future perspectives for understanding the multifaceted interactions between cancer cells and glial cells for the treatment of metastatic brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b93951608ce8fdc53c244e85b3946f02850ef4e" target='_blank'>
              Multifaceted interactions between cancer cells and glial cells in brain metastasis.
              </a>
            </td>
          <td>
            Kojiro Ishibashi, E. Hirata
          </td>
          <td>2024-07-11</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Solid tumors represent the most common type of cancer in humans and are classified into sarcomas, lymphomas, and carcinomas based on the originating cells. Among these, carcinomas, which arise from epithelial and glandular cells lining the body's tissues, are the most prevalent. Around the world, a significant increase in the incidence of solid tumors is observed during recent years. In this context, efforts to discover more effective cancer treatments have led to a deeper understanding of the tumor microenvironment and its components. Currently, the interactions between cancer cells and elements of the tumor microenvironment are being intensely investigated. Remarkable progress in research is noted, largely due to the development of advanced in vitro models, such as Tumor-on-a-Chip models that assist in understanding and ultimately discovering new effective treatments for a specific type of cancer. The purpose of this paper is to provide a review of the tumor microenvironment and cancer cell components, along with the advances on tumor-on-a-chip models designed to mimic tumors, offering a perspective on the current state-of-the-art. Recent studies using this kind of microdevices that reproduce the tumor microenvironment have allowed a better understanding of the cancer and its treatments. Nevertheless, current applications of this technology present some limitations that must be overcome in order to achieve a broad application by researchers looking for a deeper knowledge of cancer and new strategies to improve current therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d9d7fa6f0b9b61f8c31ffe7d59e62a370ae7ef" target='_blank'>
              The Microenvironment of Solid Tumors: Components and current challenges of Tumor-on-a-Chip models.
              </a>
            </td>
          <td>
            Ilva de Fátima Souza, João Paulo Jesus Vieira, E. Bonifácio, Bethânia Alves Avelar Freitas, Libardo Andrés González Torres
          </td>
          <td>2024-07-15</td>
          <td>Tissue engineering. Part B, Reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epithelial-mesenchymal transitions (EMTs) are thought to promote metastasis via downregulation of E-cadherin and upregulation of mesenchymal markers such as N-cadherin and vimentin. Contrary to this, E-cadherin is retained in many invasive carcinomas and promotes collective cell invasion. To investigate how E-cadherin regulates metastasis, we examined the highly metastatic, E-cadherin-positive murine 4T1 breast cancer model, together with the less metastatic, 4T1-related cell lines, 4T07, 168FARN, and 67NR. We found that 4T1 cells display a hybrid-E/M phenotype with co-expression of epithelial and mesenchymal markers, while 4T07, 168FARN, and 67NR display progressively more mesenchymal phenotypes in vitro that relate inversely to their metastatic capacity in vivo. Using RNA interference and constitutive expression, we demonstrate that the expression level of E-cadherin does not determine 4T1 or 4T07 cell metastatic capacity in mice. Mechanistically, 4T1 cells possess highly dynamic, unstable cell-cell junctions and can undergo collective invasion without E-cadherin downregulation. However, 4T1 orthotopic tumors in vivo also contain subregions of EMT-like loss of E-cadherin. Thus, 4T1 cells function as a model for carcinomas with a hybrid-E/M phenotype that promotes invasion and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6b8e8b2c86e54722dcd6899acc1d0f4a150bd0e" target='_blank'>
              The highly metastatic 4T1 breast carcinoma model possesses features of a hybrid epithelial-mesenchymal phenotype.
              </a>
            </td>
          <td>
            Mary E. Herndon, Mitchell Ayers, Katherine Gibson-Corley, Michael K Wendt, Lori L. Wallrath, Michael D. Henry, Christopher S. Stipp
          </td>
          <td>2024-08-06</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cells must adopt flexible regulatory strategies to make decisions regarding their fate, including differentiation, apoptosis, or survival in the face of various external stimuli. One key cellular strategy that enables these functions is stochastic gene expression programs. However, understanding how transcriptional bursting, and consequently, cell fate, responds to DNA damage on a genome-wide scale poses a challenge. In this study, we propose an interpretable and scalable inference framework, DeepTX, that leverages deep learning methods to connect mechanistic models and scRNA-seq data, thereby revealing genome-wide transcriptional burst kinetics. This framework enables rapid and accurate solutions to transcription models and the inference of transcriptional burst kinetics from scRNA-seq data. Applying this framework to several scRNA-seq datasets of DNA-damaging drug treatments, we observed that fluctuations in transcriptional bursting induced by different drugs could lead to distinct fate decisions: IdU treatment induces differentiation in mouse embryonic stem cells by increasing the burst size of gene expression, while 5FU treatment with low and high dose increases the burst frequency of gene expression to induce cell apoptosis and survival in human colon cancer cells. Together, these results show that DeepTX can be used to analyze single-cell transcriptomics data and can provide mechanistic insights into cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08e79a79f41500c7b1baabfdd652bb03e9ecc3e" target='_blank'>
              Deep learning linking mechanistic models to single-cell transcriptomics data reveals transcriptional bursting in response to DNA damage
              </a>
            </td>
          <td>
            Zhiwei Huang, Songhao Luo, Zihao Wang, Zhenquan Zhang, Benyuan Jiang, Qing Nie, Jiajun Zhang
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Motivation Cancer is driven by somatic mutations that result in the expansion of genomically distinct sub-populations of cells called clones. Identifying the clonal composition of tumours and understanding the evolutionary relationships between clones is a crucial task in cancer genomics. Bulk DNA sequencing is commonly used for studying the clonal composition of tumours, but it is challenging to infer the genetic relationship between different clones due to the mixture of different cell populations. Results In this work, we introduce a new probabilistic model called PhyClone that can infer clonal phylogenies from bulk sequencing data. We demonstrate the performance of PhyClone on simulated and real-world datasets and show that it outperforms previous methods in terms of accuracy and scalability. Availability and implementation Source code is available on Github at: https://github.com/Roth-Lab/PhyClone under the GPL v3.0 license. Contact aroth@bccrc.ca">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2cba6bef4ca455a79d82a594e5b713c434402a" target='_blank'>
              PhyClone: Accurate Bayesian reconstruction of cancer phylogenies from bulk sequencing
              </a>
            </td>
          <td>
            Emilia Hurtado, A. Bouchard-Côté, Andrew Roth
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a rare disease associated with a poor prognosis, primarily due to early recurrence and metastasis. An important feature of this condition is microvascular invasion (MVI). However, current predictive models based on imaging have limited efficacy in this regard. This study employed a random forest model to construct a predictive model for MVI identification and uncover its biological basis. Single-cell transcriptome sequencing, whole exome sequencing, and proteome sequencing were performed. The area under the curve of the prediction model in the validation set was 0.93. Further analysis indicated that MVI-associated tumor cells exhibited functional changes related to epithelial-mesenchymal transition and lipid metabolism due to alterations in the NF-kappa B and MAPK signaling pathways. Tumor cells were also differentially enriched for the IL-17 signaling pathway. There was less infiltration of SLC30A1+ CD8+ T cells expressing cytotoxic genes in MVI-associated ICC, whereas there was more infiltration of myeloid cells with attenuated expression of the MHC II pathway. Additionally, MVI-associated intercellular communication was closely related to the SPP1-CD44 and ANXA1-FPR1 pathways. These findings resulted in a brilliant predictive model and fresh insights into MVI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891c8bc5aef729b930179251eb7e060720eef85" target='_blank'>
              Machine learning based on biological context facilitates the identification of microvascular invasion in intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Shuaishuai Xu, Mingyu Wan, C. Ye, Ruyin Chen, Qiong Li, Xiaochen Zhang, Jian Ruan
          </td>
          <td>2024-08-01</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde19b1f96c1cfcaa9c403d17be06cabeee637d7" target='_blank'>
              Pan-cell death-related signature reveals tumor immune microenvironment and optimizes personalized therapy alternations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Linzhi Han, Jingyi He, Hongxin Xie, Yan Gong, Conghua Xie
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1324028a1d2d746e8a6e1979bfd5659c55c1e19" target='_blank'>
              Functional and clinical roles of stromal PDGF receptors in tumor biology.
              </a>
            </td>
          <td>
            Carina Strell, Elisabet Rodríguez-Tomàs, Arne Östman
          </td>
          <td>2024-07-09</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Sarcomatoid differentiation (SD) in renal cell carcinoma (RCC) is associated with poor survival and heightened response to immune checkpoint blockade. Detection of SD can be challenging due to spatial heterogeneity and sampling error. Herein, we introduce a novel tissue–informed epigenomic approach to noninvasively identify sarcomatoid differentiation in patients with RCC from cell-free DNA (cfDNA) using 1mL of plasma. Methods Chromatin immunoprecipitation and sequencing (ChIP-seq) for H3K27ac – a histone modification associated with active regulatory elements (REs) – was performed on pathologically reviewed clear cell RCC frozen tissue samples with and without SD (sarcomatoid-RCC and epithelioid-RCC, resp.) collected at the Dana-Farber Cancer Institute. Differentially marked REs between sarcomatoid and epithelioid subtypes were identified using DESeq2 (false discovery rate of q < 0.01). After establishing tissue signatures, ChIP-seq was then performed on cell-free chromatin (cfChIP-seq) in plasma from patients with sarc-RCC and epi-RCC. A Sarcomatoid Score was derived for each sample by aggregating the plasma H3K27ac signal at tissue-derived sarcomatoid-specific REs (sarc-REs), while normalizing to signal at epithelioid-specific REs (epi-REs). Scores were compared between the two groups using a Wilcoxon rank-sum test. A classifier was built to distinguish sarc-RCC from epi-RCC based on the Sarcomatoid Score and its performance was evaluated using the area under the receiver operating characteristic (AUROC) curve. Results We identified 25,919 differentially marked REs between 8 sarc-RCC and 8 epi-RCC tissue samples at a false discovery rate of q < 0.01. We selected 12,868 REs that are enriched in sarcomatoid vs. epithelioid. We generated cfChIP-seq profiles from plasma of 29 patients, 17 with sarc-RCC and 12 with epi-RCC. The Sarcomatoid Scores were significantly higher in sarc-RCC vs. epi-RCC plasma samples (p=1.6×10-4; Figure 1A). These scores achieved an AUROC curve of 0.9 for classifying patients with sarc-RCC from patients with epi-RCC (Figure 1B). Conclusions We present a proof-of-concept study in 1 cc of plasma for the detection of sarcomatoid differentiation in RCC based on the assessment of histone modification signals in cfDNA. This approach could help overcome the challenges of spatial heterogeneity and sampling error from tissue that make identification of sarc-RCC difficult. More generally, it establishes a paradigm for identifying histologic subtypes of cancer based on their epigenomic correlates from cfDNA, with possible therapeutic implications in real-time. Figure 1. (A) Sarcomatoid Score in plasma from patients with RCC at sarcomatoid-specific REs, comparing sarcomatoid (orange) and epithelioid RCC (blue). (B) ROC curves for distinguishing sarcomatoid from epithelioid RCC plasma samples using H3K27ac cell-free ChIP-seq signal at sarcomatoid-specific REs. ‘AUC’ indicates area under the ROC curve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03141a33629f618991474e63a226a8491b01de4" target='_blank'>
              44 Liquid biopsy epigenomic profiling for the detection of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Karl Semaan, T. Zarif, Marc Eid, John Canniff, Noa Phillips, Bard Fortunato, Hunter Savignano, Matthew P Davidson, Razane El Hajj Chehade, Simon Garinet, E. Saad, R. Saliby, M. Sun, J-H. Seo, Jacob E. Berchuck, David A Braun, M. Freedman, Toni K. Choueiri, S. Baca
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bf390baec4f133e1159b69971ccead0d2deed57" target='_blank'>
              Establishment and characterization of the PDAC-X3 cell line: a novel Chinese-origin pancreatic ductal adenocarcinoma cell line.
              </a>
            </td>
          <td>
            Changpeng Chai, Huan-zhang Tang, Xin Miao, Yuanhui Su, Lu Li, Cheng Yu, Jian-Fen Yi, Zhenzhen Ye, L. Miao, Bo Zhang, Zhengfeng Wang, Wei Luo, Jinjing Hu, Hui Zhang, Wence Zhou, Hao Xu
          </td>
          <td>2024-07-16</td>
          <td>Human cell</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b21391e9d7b4e36087d981b4ffc2b93179db8ea" target='_blank'>
              Combined microfluidic enrichment and staining workflow for single-cell analysis of circulating tumor cells in metastatic prostate cancer patients
              </a>
            </td>
          <td>
            Caroline Løppke, Amalie M Jørgensen, Nicolai T. Sand, Rasmus B Klitgaard, G. Daugaard, Mette Ø Agerbæk
          </td>
          <td>2024-07-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim Rashid, , Jen Jen Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) presents a significant clinical challenge due to its aggressive nature and extensive heterogeneity. Tumour purity, the proportion of malignant cells within a tumour, is an important covariate for understanding the disease, having direct clinical relevance or obscuring signal of the malignant portion in molecular analyses of bulk samples. However, current methods for estimating tumour purity are non-specific, unreliable or technically demanding. Therefore, we aimed to build a reliable and accessible purity estimator for GBM. Methods We developed GBMPurity, a deep learning model specifically designed to estimate the purity of IDH-wildtype primary GBM from bulk RNA-seq data. The model was trained using simulated pseudobulk tumours of known purity from labelled single-cell data acquired from the GBmap resource. The performance of GBMPurity was evaluated and compared to several existing tools using independent datasets. Results GBMPurity outperformed existing tools, achieving a mean absolute error of 0.15 and a concordance correlation coefficient of 0.88 on validation datasets. We demonstrate the utility of GBMPurity through inference on bulk RNA-seq samples and reveal reduced purity of the Proneural molecular subtype attributed to increased presence of healthy brain cells. Conclusions GBMPurity provides a reliable and accessible tool for estimating tumour purity from bulk RNA-seq data, enhancing the interpretation of bulk RNA-seq data and offering valuable insights into GBM biology. To facilitate the use of this tool by the wider research community, GBMPurity is available as a web-based tool at: https://gbmdeconvoluter.leeds.ac.uk/. Key Points GBMPurity is a glioblastoma-specific purity estimation tool. The model accurately estimates the purity of bulk RNA-seq data, outperforming existing tools. The model is available online at: https://gbmdeconvoluter.leeds.ac.uk/. Importance of the Study Glioblastoma (GBM) is a deadly brain tumour with a dismal prognosis. Research on this disease has lagged compared to other cancers, underscoring the need to streamline investigations. The cellular composition of the GBM tumour microenvironment significantly influences therapy resistance, prognosis, and the molecular state of neoplastic cells. Consequently, tumour purity (the proportion of malignant cells within a tumour) is a critical variable for understanding and contextualizing molecular and clinical analyses. We present GBMPurity (https://gbmdeconvoluter.leeds.ac.uk/), an accessible, GBM-specific tool that accurately predicts sample purity from bulk RNA-seq data. This tool can be used by the wider research community to support the interpretation of bulk omics data and accelerate the identification of more effective therapeutic strategies for treating GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79599e46ca487edcd98173e3e961858d87c8f9e" target='_blank'>
              GBMPurity: A Machine Learning Tool for Estimating Glioblastoma Tumour Purity from Bulk RNA-seq Data
              </a>
            </td>
          <td>
            Morgan P. H. Thomas, S. Ajaib, Georgette Tanner, Andrew J. Bulpitt, Lucy F. Stead
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Single-cell RNA sequencing is a valuable technique for identifying diverse cell subtypes. A key challenge in this process is that the detection of rare cells is often missed by conventional methods due to low abundance and subtle features of these cells. To overcome this, we developed SCLCNF (Local Connectivity Network Feature Sharing in Single-Cell RNA sequencing), a novel approach that identifies rare cells by analyzing features uniquely expressed in these cells. SCLCNF creates a cellular connectivity network, considering how each cell relates to its neighbors. This network helps to pinpoint coexpression patterns unique to rare cells, utilizing a rarity score to confirm their presence. Our method performs better in detecting rare cells than existing techniques, offering enhanced robustness. It has proven to be effective in human gastrula data sets for accurately pinpointing rare cells, and in sepsis data sets where it uncovers previously unidentified rare cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41629b08b97b96e2a5087885fb3403f0009f9b9" target='_blank'>
              Connectivity Network Feature Sharing in Single-Cell RNA Sequencing Data Identifies Rare Cells.
              </a>
            </td>
          <td>
            Shudong Wang, Hengxiao Li, Yahui Liu, Shanchen Pang, Sibo Qiao, Jionglong Su, Shaoqiang Wang, Yulin Zhang
          </td>
          <td>2024-08-03</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE
The molecular drivers underlying mucinous tumor pathogenicity are poorly understood. GNAS mutations predict metastatic burden and treatment resistance in mucinous appendiceal adenocarcinoma. We investigated the pan-cancer clinicopathologic relevance of GNAS variants.


METHODS
We assessed 58,043 patients with Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (IMPACT)-sequenced solid tumors to identify oncogenic variants, including GNAS, associated with mucinous tumor phenotype. We then performed comprehensive molecular analyses to compare GNAS-mutant (mut) and wild-type tumors across cancers. Gene expression patterns associated with GNAS-mut tumors were assessed in a The Cancer Genome Atlas cohort. Associations between GNAS variant status and peritoneal metastasis, first-line systemic therapy response, progression-free survival (PFS), and overall survival (OS) were determined using a propensity-matched subcohort of patients with metastatic disease.


RESULTS
Mucinous tumors were enriched for oncogenic GNAS variants. GNAS was mutated in >1% of small bowel, cervical, colorectal, pancreatic, esophagogastric, hepatobiliary, and GI neuroendocrine cancers. Across these cancers, GNAS-mut tumors exhibited a generally conserved C-to-T mutation-high, aneuploidy-low molecular profile with co-occurring prevalent KRAS variants (65% of GNAS-mut tumors) and fewer TP53 alterations. GNAS-mut tumors exhibited recurrently comutated alternative tumor suppressors (RBM10, INPPL1) and upregulation of MAPK and cell surface modulators. GNAS-mut tumors demonstrate an increased prevalence of peritoneal metastases (odds ratio [OR], 1.7 [95% CI, 1.1 to 2.5]; P = .006), worse response to first-line systemic therapy (OR, 2.2 [95% CI, 1.3 to 3.8]; P = .003), and shorter PFS (median, 5.6 v 7.0 months; P = .047). In a multivariable analysis, GNAS mutated status was independently prognostic of worse OS (hazard ratio, 1.25 [95% CI, 1.01 to 1.56]; adjusted P = .04).


CONCLUSION
Across the assessed cancers, GNAS-mut tumors exhibit a conserved molecular and clinical phenotype defined by mucinous tumor status, increased peritoneal metastasis, poor response to first-line systemic therapy, and worse survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a7fd5d8b4825c722728924fd719f0a0d9f3a8e" target='_blank'>
              Molecular and Clinicopathologic Impact of GNAS Variants Across Solid Tumors.
              </a>
            </td>
          <td>
            P. Johannet, Somer Abdelfattah, Callahan Wilde, Shrey Patel, H. Walch, B. Rousseau, Guillem Argiles, O. Artz, Miteshkumar Patel, Andrea Arfe, A. Cercek, R. Yaeger, K. Ganesh, N. Schultz, Luis A. Diaz, M. Foote
          </td>
          <td>2024-08-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="During developmental processes such as embryogenesis, how a group of cells fold into specific structures, is a central question in biology. However, it remains a major challenge to understand and predict the behavior of every cell within the living tissue over time during such intricate processes. Here we present a geometric deep-learning model that can accurately capture the highly convoluted interactions among cells. We demonstrate that multicellular data can be represented with both granular and foam-like physical pictures through a unified graph data structure, considering both cellular interactions and cell junction networks. Using this model, we achieve interpretable 4-D morphological sequence alignment, and predicting cell rearrangements before they occur at single-cell resolution. Furthermore, using neural activation map and ablation studies, we demonstrate cell geometries and cell junction networks together regulate morphogenesis at single-cell precision. This approach offers a pathway toward a unified dynamic atlas for a variety of developmental processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce21f827f84dd8500ab0740daa37b4f870f6a662" target='_blank'>
              Multicell-Fold: geometric learning in folding multicellular life
              </a>
            </td>
          <td>
            Haiqian Yang, Anh Q. Nguyen, Dapeng Bi, Markus J. Buehler, Ming Guo
          </td>
          <td>2024-07-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) poses a significant health burden globally, with high mortality rates despite various treatment options. Immunotherapy, particularly immune-checkpoint inhibitors (ICIs), has shown promise, but resistance and metastasis remain major challenges. Understanding the intricacies of the tumor microenvironment (TME) is imperative for optimizing HCC management strategies and enhancing patient prognosis.This study employed a comprehensive approach integrating multi-omics approaches, including single-cell RNA sequencing (scRNA-seq), bulk RNA sequencing (Bulk RNA-seq), and validation in clinical samples using spatial transcriptomics (ST) and multiplex immunohistochemistry (mIHC). The analysis aimed to identify key factors influencing the immunosuppressive microenvironment associated with HCC metastasis and immunotherapy resistance.HMGB2 is significantly upregulated in HCCTrans, a transitional subgroup associated with aggressive metastasis. Furthermore, HMGB2 expression positively correlates with an immunosuppressive microenvironment, particularly evident in exhausted T cells. Notably, HMGB2 expression correlated positively with immunosuppressive markers and poor prognosis in HCC patients across multiple cohorts. ST combined with mIHC validated the spatial expression patterns of HMGB2 within the TME, providing additional evidence of its role in HCC progression and immune evasion.HMGB2 emerges as a critical player of HCC progression, metastasis, and immunosuppression. Its elevated expression correlates with aggressive tumor behavior and poor patient outcomes, suggesting its potential as both a therapeutic target and a prognostic indicator in HCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1807de7a88d55a53a45a87ce21f525cc4ca5c475" target='_blank'>
              HMGB2 drives tumor progression and shapes the immunosuppressive microenvironment in hepatocellular carcinoma: insights from multi-omics analysis
              </a>
            </td>
          <td>
            Yan-zhu Chen, , Zuo-lin Xiang
          </td>
          <td>2024-08-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2de571a905622de6c7a795e6f89203edad8fe14" target='_blank'>
              Tumor-derived GLI1 promotes remodeling of the immune tumor microenvironment in melanoma
              </a>
            </td>
          <td>
            Alessandro Giammona, Chiara De Vellis, Enrica Crivaro, Luisa Maresca, R. Amoriello, Federica Ricci, G. Anichini, Silvia Pietrobono, David R Pease, Martin E Fernandez-Zapico, Clara Ballerini, B. Stecca
          </td>
          <td>2024-08-02</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Brain tumors, particularly glioblastoma (GBM), are devastating and challenging to treat, with a low 5-year survival rate of only 6.6%. Mouse models are established to understand tumorigenesis and develop new therapeutic strategies. Large-scale genomic studies have facilitated the identification of genetic alterations driving human brain tumor development and progression. Genetically engineered mouse models (GEMMs) with clinically relevant genetic alterations are widely used to investigate tumor origin. Additionally, syngeneic implantation models, utilizing cell lines derived from GEMMs or other sources, are popular for their consistent and relatively short latency period, addressing various brain cancer research questions. In recent years, the success of immunotherapy in specific cancer types has led to a surge in cancer immunology-related research which specifically necessitates the utilization of immunocompetent mouse models. In this review, we provide a comprehensive summary of GEMMs and syngeneic mouse models for adult brain tumors, emphasizing key features such as model origin, genetic alteration background, oncogenic mechanisms, and immune-related characteristics. Our review serves as a valuable resource for the brain tumor research community, aiding in the selection of appropriate models to study cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff748c1f4b0e36e21dc90ffb3f8ae15d403a0e7" target='_blank'>
              In Vivo Mouse Models for Adult Brain Tumors: Exploring Tumorigenesis and Advancing Immunotherapy Development.
              </a>
            </td>
          <td>
            John Figg, Dongjiang Chen, Laura Falceto Font, Catherine Flores, Dan Jin
          </td>
          <td>2024-07-11</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastoma is a devastating disease accounting for ~15% of all childhood cancer deaths. Collagen content and fiber association within the tumor stroma influence tumor progression and metastasis. High expression levels of collagen receptor kinase, Discoidin domain receptor II (DDR2), are associated with poor survival of neuroblastoma patients. Additionally, cancer cells generate and sustain mechanical forces within their enviroment as a part of their normal physiology. Despite this, whether collagen activated DDR2 signaling dysregulate these migration forces is still elusive. To address these questions, a shRNA DDR2 knockdown neuroblastoma cell line (SH-SY5Y) was engineered to evaluate the consequence of DDR2 on cellular mechanics. Atomic force microscopy and traction force microscopy were utlizing to unveil the biophysical altercations. DDR2 down-regulation was found to significantly reduce proliferation, cell stiffness, and cellular elongation. Aditionally, DDR2 down-regulated cells had decreased traction forces when plated on collagen coated elastic substrates. Together, these results highlight the important role that DDR2 has in reducing migration mechanics in neuroblastoma and might be a promising target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f707587725fd2ca416d376c5d57d87998251580c" target='_blank'>
              Investigation of cell mechanics and migration on DDR2-expressing neuroblastoma cell line
              </a>
            </td>
          <td>
            Theadora Vessella, Esteban J. Rozen, Jason Shohet, Qi Wen, Hong Susan Zhou
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab20abff3d5ab2e3023b99dd1dd6ab980ad6081" target='_blank'>
              Mutual exclusivity and co-occurrence patterns of immune checkpoints indicate NKG2A relates to anti-PD-1 resistance in gastric cancer
              </a>
            </td>
          <td>
            Guanjun Li, Xiaohan Liu, Chunhui Gu, Guichuang Ma, Shaowei Li, Zhenfeng Ma, Yanqiu Xiong, Yu Jiang, Qiong Huang, Jianhua Wu, Zhenzhen Wu, Wangjun Liao, Qijing Wu, Min Shi
          </td>
          <td>2024-08-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Most head and neck squamous cell carcinoma (HNSCC) patients are diagnosed at an advanced local stage. While immunotherapy has improved survival rates, only a minority of patients respond durably to targeted immunotherapies, posing substantial clinical challenges. We investigated the heterogeneity of the tumor microenvironment in HNSCC cohorts before and after immunotherapy by analyzing single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing datasets retrieved from public databases.We constructed a single-cell transcriptome landscape of HNSCC patients before and after immunotherapy and analyzed the cellular composition, developmental trajectories, gene regulatory networks, and communication patterns of different cell type subpopulations. Additionally, we assessed the expression levels of relevant indicators in HNSCC cells via western blot, ELISA, and fluorescent probe techniques.At the single-cell level, we identified a subpopulation of TP63+ SLC7A5+ HNSCC that exhibited a ferroptosis-resistant phenotype. This subpopulation suppresses ferroptosis in malignant cells through the transcriptional upregulation of SLC7A5 mediated by high TP63 expression, thereby promoting tumor growth and resistance to immunotherapy. The experimental results demonstrated that the overexpression of TP63 upregulated the expression of SLC7A5 and suppressed the concentrations of Fe2+ and ROS in HNSCC cells. By integrating bulk transcriptome data, we developed a clinical scoring model based on TP63 and SLC7A5, which are closely associated with tumor stage, revealing the significant prognostic efficacy of the TP63+ SLC7A5+ HNSCC-mediated ferroptosis mechanism in HNSCC patients.Our research elucidates the TME in HNSCC before and after immunotherapy, revealing a novel mechanism by which TP63+ SLC7A5+ HNSCC inhibits ferroptosis and enhances tumor resistance via TP63-induced SLC7A5 upregulation. These insights lay the foundation for the development of more effective treatments for HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea63ddf8e02932d2e36f67e9aabda84b6c8ce55c" target='_blank'>
              TP63 transcriptionally regulates SLC7A5 to suppress ferroptosis in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Zilong Chen, Haoxi Cai, Weiwei Ye, Junming Wu, Jing Liu, Yun Xie, Shiqiang Feng, Yuanpei Jin, Yunxia Lv, Hui Ye, Chengfu Cai, Gengming Cai
          </td>
          <td>2024-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Carcinomas are common in humans but rare among closely related “great apes.” Plausible explanations, including human-specific genomic alterations affecting the biology of sialic acids, are proposed, but causality remains unproven. Here, an integrated evolutionary genetics-phenome-transcriptome approach studied the role of SIGLEC12 gene (encoding Siglec-XII) in epithelial transformation and cancer. Exogenous expression of the protein in cell lines and genetically engineered mice recapitulated approximately 30% of the human population in whom the protein is expressed in a form that cannot bind ligand because of a fixed, homozygous, human-universal missense mutation. Siglec-XII–null cells/mice recapitulated the remaining approximately 70% of the human population in whom an additional polymorphic frameshift mutation eliminates the entire protein. Siglec-XII expression drove several pro-oncogenic phenotypes in cell lines and increased tumor burden in mice challenged with chemical carcinogen and inflammation. Transcriptomic studies yielded a 29-gene signature of Siglec-XII–positive disease and when used as a computational tool for navigating human data sets, pinpointed with surprising precision that SIGLEC12 expression (model) recapitulates a very specific type of colorectal carcinomas (disease) that is associated with mismatch-repair defects and inflammation, disproportionately affects European Americans, and carries a favorable prognosis. They revealed a hitherto-unknown evolutionary genetic mechanism for an ethnic/environmental predisposition of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a026fde004925d82bafb73d8c39f7bda40aeeb" target='_blank'>
              Human-specific elimination of epithelial Siglec-XII suppresses the risk of inflammation-driven colorectal cancers
              </a>
            </td>
          <td>
            Hector A Cuello, Saptarshi Sinha, Andrea L Verhagen, N. Varki, A. Varki, Pradipta Ghosh
          </td>
          <td>2024-07-11</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Glioma is a highly heterogeneous malignancy of the central nervous system. This heterogeneity is driven by various molecular processes, including neoplastic transformation, cell cycle dysregulation, and angiogenesis. Among these biomolecular events, inflammation and stress pathways in the development and driving factors of glioma heterogeneity have been reported. However, the mechanisms of glioma heterogeneity under stress response remain unclear, especially from a spatial aspect.This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to explore the impact of oxidative stress response genes in oligodendrocyte precursor cells (OPCs). Our analysis identified distinct pathways activated by oxidative stress in two different types of gliomas: high- and low- grade (HG and LG) gliomas.In HG gliomas, oxidative stress induced a metabolic shift from oxidative phosphorylation to glycolysis, promoting cell survival by preventing apoptosis. This metabolic reprogramming was accompanied by epithelial-to-mesenchymal transition (EMT) and an upregulation of stress response genes. Furthermore, SCENIC (Single-Cell rEgulatory Network Inference and Clustering) analysis revealed that oxidative stress activated the AP1 transcription factor in HG gliomas, thereby enhancing tumor cell survival and proliferation.Our findings provide a novel perspective on the mechanisms of oxidative stress responses across various grades of gliomas. This insight enhances our comprehension of the evolutionary processes and heterogeneity within gliomas, potentially guiding future research and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56dee2729032b7749900d8cd04b93b8180eff50e" target='_blank'>
              Single-cell and spatial transcriptome assays reveal heterogeneity in gliomas through stress responses and pathway alterations
              </a>
            </td>
          <td>
            Z. He, Zheng Liu, Qi Wang, Xingjian Sima, Wei Zhao, Chunmei He, Wenjie Yang, Han Chen, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Prostate cancer is a complex and heterogeneous disease with multiple tumor foci, each potentially harboring distinct driver molecular aberrations. This complexity poses challenges to effective management. We took an innovative approach to gain a comprehensive understanding of the genetic underpinnings of each tumor focus and avoid overlooking more minor yet clinically significant foci. Instead of relying solely on a systematic sampling of dominant foci, we conducted molecular analysis on whole-mount radical prostatectomy specimens. Our study aimed to find distinct molecular subsets of prostate cancer and assess their correlation with clinical outcomes, focusing on Caucasians (CA) and African Americans (AA). Method: We randomly selected 2201 whole-mount radical prostatectomy cases, with 1207 (54.8%) from CA and 994 (45.1%) from AA patients evaluated for a 5-year biochemical recurrence-free survival rate (BCR). Of these 2201 cases, 834 (463 -56% were from CA and 371 -44% from AA patients) were subjected to molecular analysis using dual immunohistochemistry (IHC) for ERG and SPINK1, along with dual RNA in-situ hybridization (RNA-ISH) for ETV1 and ETV4 to evaluate tumor molecular heterogeneity on whole-mount specimens. The Chi-squared test examined racial disparities in aberrant oncogene expression. To assess BCR-free survival, we employed the Kaplan-Meier method and Cox-PH models for patients with distinct molecular subsets of prostate cancer. Additionally, Gleason Grade groups of prostate biopsies were summarized using a spaghetti plot and compared using linear mixed models. Results: Analysis of the 2201 cases revealed that AA with localized prostate cancer behaved differently with better 5-year BCR-free survival than CA after radical prostatectomy (AA: 0.82, 95% CI 0.80-0.85; CA: 0.71, 95% CI 0.68-0.75; p<.001). Molecular profiling of whole-mount specimens from 834 cases revealed that 16.4%, 58.4%, 21.7%, and 3.5% of patients with localized prostate cancer expressed none, one, two, and three of the four oncogenes, respectively. This finding identified new molecular subsets of prostate cancer with more than one driver mutation in a mutually exclusive manner within the multifocal disease. ERG and SPINK1 expression showed a negative correlation (p<.001). Notably, AA patients exhibited a lower incidence of ERG (38.8% vs. 60.3%) but a higher incidence of SPINK1 (63.3% vs. 35.6%) than CA patients. The incidences of ETV1 (9.4% vs. 9.3%) and ETV4 (4.6% vs. 3.9%) were not statistically significant between the two racial groups. However, significantly, ETV1 expression was associated with worse BCR-free survival in CA patients (hazard ratio [HR]=2.36, 95% CI 1.22-4.57, p=.02), while ETV4 expression was linked to worse BCR-free survival in AA patients (HR=2.65, 95% CI 1.15-6.09, p=.02). Moreover, ETV4 expression was associated with regional lymph node metastasis in AA patients (odds ratio [OR]=5.14, 95% CI 1.3-17.4, p=.01) but not in CA patients (OR=0.60, 95% CI 0.03-3.17, p=.63) at the time of radical prostatectomy. Additionally, in patients who underwent multiple biopsies before radical prostatectomy, the Gleason Grade group increased over time in AA patients (0.25 per year, p<.001) but remained unchanged in CA patients. ERG expression was associated with a lower Gleason Grade group (-0.20, p=.03), while ETV4 expression was linked to a higher Gleason Grade (0.54, p=.01). Conclusions: Our study reveals that AA with localized prostate cancer behaves differently and has better BCR-free survival than CA after radical prostatectomy, even after adjusting for known prognostic factors. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion within a multifocal prostate shows significant molecular heterogeneity between localized prostate cancer in CA and AA patients. Importantly, given the association of ETV1 and ETV4 expression with worse BCR-free survival in CA and AA, respectively, ETV1 and ETV4 emerge as potential prognostic markers, offering insights for clinical practice to predict prostate cancer recurrence after radical prostatectomy. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion and SPINK1 in a mutually exclusive pattern indicates the clonal origin of independent tumor foci, which is a rare and unique phenomenon in prostate cancer hitherto unidentified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863b31d824c9eeb2a43dee5a1d15a9728d0743ea" target='_blank'>
              Unveiling Prostate Cancer's Molecular Tapestry: Ethnic Disparities and Prognostic Insights from Whole-Mount Prostatectomy Tissue Analysis
              </a>
            </td>
          <td>
            Wei Zhao, Pin Li, Shannon Carskadon, Sunita Ghosh, Craig G. Rogers, James Peabody, D. Chitale, Mani Menon, Sean Williamson, Nilesh Gupta, N. Palanisamy
          </td>
          <td>2024-07-16</td>
          <td>None</td>
          <td>0</td>
          <td>65</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>