<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-26 06:30:13 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E. Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Spatial proteomics enable detailed analysis of tissue at single cell resolution. However, creating reliable segmentation masks and assigning accurate cell phenotypes to discrete cellular phenotypes can be challenging. We introduce IMmuneCite, a computational framework for comprehensive image pre-processing and single-cell dataset creation, focused on defining complex immune landscapes when using spatial proteomics platforms. We demonstrate that IMmuneCite facilitates the identification of 32 discrete immune cell phenotypes using data from human liver samples while substantially reducing nonbiological cell clusters arising from co-localization of markers for different cell lineages. We established its versatility and ability to accommodate any antibody panel and different species by applying IMmuneCite to data from murine liver tissue. This approach enabled deep characterization of different functional states in each immune compartment, uncovering key features of the immune microenvironment in clinical liver transplantation and murine hepatocellular carcinoma. In conclusion, we demonstrated that IMmuneCite is a user-friendly, integrated computational platform that facilitates investigation of the immune microenvironment across species, while ensuring the creation of an immune focused, spatially resolved single-cell proteomic dataset to provide high fidelity, biologically relevant analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7edc240d6b41584e3ef1ab2daeefdaa860e615e" target='_blank'>
              IMmuneCite: an integrated workflow for analysis of immune enriched spatial proteomic data
              </a>
            </td>
          <td>
            A. Barbetta, Sarah Bangerth, Jason T C Lee, B. Rocque, Evanthia T Roussos Torres, Rohit Kohli, Omid Akbari, J. Emamaullee
          </td>
          <td>2024-07-09</td>
          <td>Research Square</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of medulloblastoma (MB) influences progression and therapy response, presenting a promising target for therapeutic advances. Prior single-cell analyses have characterized the cellular components of the TME but lack spatial context. To address this, we performed spatial transcriptomic sequencing on sixteen pediatric MB samples obtained at diagnosis, including two matched diagnosis-relapse pairs. Our analyses revealed inter- and intra-tumoral heterogeneity within the TME, comprised of tumor-associated astrocytes (TAAs), macrophages (TAMs), stromal components, and distinct subpopulations of MB cells at different stages of neuronal differentiation and cell cycle progression. We identified dense regions of quiescent progenitor-like MB cells enriched in patients with high-risk (HR) features and an increase in TAAs, TAMs, and dysregulated vascular endothelium following relapse. Our study presents novel insights into the spatial architecture and cellular landscape of the medulloblastoma TME, highlighting spatial patterns linked to HR features and relapse, which may serve as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b39423470039d57ba772c250043854baef4821" target='_blank'>
              Medulloblastoma Spatial Transcriptomics Reveals Tumor Microenvironment Heterogeneity with High-Density Progenitor Cell Regions Correlating with High-Risk Disease
              </a>
            </td>
          <td>
            Franklin Chien, Marina E. Michaud, Mojtaba Bakhtiari, Chanel Schroff, M. Snuderl, José E. Velázquez Vega, Tobey J. MacDonald, Manoj K. Bhasin
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Secondary lymphoid organs (SLOs), including tonsils (TS), lymph nodes (LN), and Peyer's Patches, exhibit complementary immune functions. However, little is known about the spatial organization of immune cells and extracellular matrix (ECM) in the SLOs. Traditional imaging is limited to a few markers, confining our understanding of the differences between the SLOs. Herein, imaging mass cytometry addressed this gap by simultaneously profiling 25-plex proteins in SLO tissues at subcellular resolution. The antibody panel targeted immune, stromal, chemokine, epigenetic, and functional markers. For robust cell identification, a computational workflow SpatialVizPheno was developed to spatially phenotype 999,970 cells using two approaches, including manual gating and semi-supervised gating, iterative clustering, and annotation. LN exhibited the highest density of B cells while the intestinal tissues contained the highest proportion of regulatory and follicular helper T cells. SpatialVizPheno identified the most prevalent interaction between follicular dendritic cells and stromal cells (SCs), plasmablasts/plasma cells, and the SCs across the lymphoid tissues. Collagen-enriched regions were associated with the spatial orientation of B cell follicles in both TS and LN tissues, but not in intestinal lymphoid tissues. Such spatial differences of immunophenotypes and ECM in different SLO tissues can be used to quantify the relationship between cellular organization and ultimate immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d9342b523b0b50e2fb3715a6d362425d2a168c" target='_blank'>
              Spatial immunophenotyping using multiplexed imaging of immune follicles in secondary lymphoid tissues
              </a>
            </td>
          <td>
            Mayar Allam, Thomas Hu, Zhou Fang, Michelle Pi, Ankur Singh, Ahmet F. Coskun
          </td>
          <td>2024-07-22</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G. Danielli, K. Maass, David R. Ghasemi, Svenja K. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, Sina Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong M. Nguyen, Alicia-Christina Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, Amy Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Adult-type diffuse gliomas, the most common primary brain tumors, pose significant clinical challenges due to limited treatment options, restricted anti-tumor immune response and dismal patient prognosis. In this study, we elucidate the immunological function and clinical relevance of intra-tumoral tertiary lymphoid structures (TLS) in adaptive anti-glioma immunity. We conducted a comprehensive, unbiased analysis of lymphoid aggregation in 642 gliomas using a multi-modal approach that combines RNA sequencing with spatial transcriptome and proteome profiling. Our findings reveal that TLS are present in 15% of tumors and correlate with improved overall survival. Gliomas with TLS exhibit a remodeled perivascular space, marked by transcriptional upregulation and spatial redistribution of collagens associated with barrier functions. Furthermore, we demonstrate that TLS maturation into sites of dynamic adaptive immune responses, characterized by clonal T and B cell expansion and IgA+ and IgG+ plasma-cell formation, is driven by efficient early T cell recruitment to the perivascular space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ba24c654b135fc550ac0b5abe83db49fb3978d" target='_blank'>
              Glioma-associated tertiary lymphoid structures are sites of lymphocyte clonal expansion and plasma cell formation
              </a>
            </td>
          <td>
            P. Cakmak, J. H. Lun, Aakanksha Singh, J. Macas, J. Schupp, M. Köhler, Tatjana Starzetz, Michael C. Burger, Eike Steidl, Lucie Marie Hasse, Elke Hattingen, Karlheinz Plate, Y. Reiss, K. Imkeller
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The patterns by which primary tumors spread to metastatic sites remain poorly understood. Here, we define patterns of metastatic seeding in prostate cancer (PCa) using a novel injection-based mouse model - EvoCaP (Evolution in Cancer of the Prostate), featuring aggressive metastatic cancer to bone, liver, lungs, and lymph nodes. To define migration histories between primary and metastatic sites, we used our EvoTraceR pipeline to track distinct tumor clones containing recordable barcodes. We detected widespread intratumoral heterogeneity from the primary tumor in metastatic seeding, with few clonal populations (CPs) instigating most migration. Metastasis-to-metastasis seeding was uncommon, as most cells remained confined within the tissue. Migration patterns in our model were congruent with human PCa seeding topologies. Our findings support the view of metastatic PCa as a systemic disease driven by waves of aggressive clones expanding their niche, infrequently overcoming constraints that otherwise keep them confined in the primary or metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7ba92880acaf1b934f559d90646f82a1f001d3" target='_blank'>
              Clonal Lineage Tracing with Somatic Delivery of Recordable Barcodes Reveals Migration Histories of Metastatic Prostate Cancer.
              </a>
            </td>
          <td>
            Ryan N Serio, Armin Scheben, Billy Lu, Domenic V Gargiulo, Lucrezia Patruno, Caroline L Buckholtz, Ryan J Chaffee, Megan C Jibilian, Steven G Persaud, Stephen J. Staklinski, Rebecca Hassett, Lise M Brault, Daniele Ramazzotti, Christopher E Barbieri, Adam Siepel, Dawid G Nowak
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dcce816f6a19dca7baf05569c687c99b3105e9" target='_blank'>
              Unbiasedly decoding the tumor microenvironment with single-cell multiomics analysis in pancreatic cancer
              </a>
            </td>
          <td>
            Yifan Fu, Jinxin Tao, Tao Liu, Yueze Liu, J. Qiu, Dan Su, Ruobing Wang, W. Luo, Zhe Cao, Guihu Weng, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-07-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The current challenge in analyzing sequencing-based spatial transcriptomics (ST) is to precisely resolve the spatial distribution of cell subpopulations with low abundance and detect context-dependent variation in cell states within complex tissue microenvironments. Here, we introduce an ultra-resolution ST deconvolution algorithm (UCASpatial) that improves the resolution of mapping cell subpopulations to spatial locations by leveraging the contribution of genes indicative of cell identity through entropy-based weighting. Using both in silico and real ST datasets, we demonstrate that UCASpatial improves the robustness and accuracy in identifying low-abundant cell subpopulations and distinguishing transcriptionally heterogeneous cell subpopulations. Applying UCASpatial to murine wound healing, we recapitulate known spatiotemporal dynamics of multiple cell subpopulations involved in wound healing and reveal the spatial segregation of distinct macrophage subpopulations embedded within different cellular communities at the wound center. In human colorectal cancer (CRC), we link genomic alterations in individual cancer clones to multi-cellular characteristics of the tumor immune microenvironment (TIME) and reveal the co-evolution of tumor cells and TIME within the same tumor. We show that the copy number gain on chromosome 20q (chr20q-gain) in tumor cells orchestrates a T cell-excluded TIME, indicative of resistance to immunotherapy in CRC, and is associated with tumor-intrinsic endogenous retrovirus silencing and impaired type I interferon response. Our findings present UCASpatial as a versatile tool for deciphering ultra-resolution cellular landscapes in ST and exploring intercellular mechanisms in complex and dynamic microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36aa9aa2d8c5c7dbae3c0976fedf2735c37f839" target='_blank'>
              Ultra-resolution Deconvolution of Spatial Transcriptomics Unveils Spatiotemporal Cellular Dynamics in Complex Microenvironments
              </a>
            </td>
          <td>
            Yin Xu, Zurui Huang, Yawei Zhang, Zhenghang Wang, Peijin Guo, Feifan Zhang, Wenxuan Gong, Guanghao Liang, Boyuan Mei, Lihui Dong, Renbao Chang, Minghui Gong, Yu Xia, Haochen Ni, Jing Yang, Yuan Gao, Zhaoqi Liu, Lin Shen, Jian Li, Meng Michelle Xu, Dali Han
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the spatial organization of the bone marrow (BM) microenvironment at single-cell resolution constitutes a challenge in multiple myeloma (MM). Using formalin-fixed paraffin-embedded (FFPE) samples from the MIcγ1 mouse model of MM and aged-matched healthy YFPcγ1 mice, we performed spatial transcriptional profiling with Visium Spatial Gene Expression. A custom data-analysis framework that combines spatial with single-cell transcriptomic profiling defined the BM cellular composition and established specific cell relations, visualizing the spatial distribution of transcriptionally heterogeneous MM plasma cells (MM-PC). MM pathogenesis transcriptional programs were spatially delineated within the BM microenvironment. A high-to-low MM-PC density gradient spatially correlated with effector-to-exhausted T cell phenotype abundance. In this context, MM cells in high-density MM-PC areas coexisted with dendritic cells while displaced neutrophils to the tumor border. Increased neutrophil extracellular trap formation, IL-17-driven inflammatory signaling, and osteoclast differentiation were spatially delineated within the BM microenvironment. The spatial identification of different areas of BM and the interaction between malignant cells and their microenvironment were validated in FFPE BM biopsies from MM patients with varying degrees of MM-PC infiltration. In summary, spatial transcriptomics depicts the BM cellular architecture of MM and reveals deregulated mechanisms underlying MM intercellular communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6b45562f865baeadce9704ad2edc7a5b178a26" target='_blank'>
              Spatial Transcriptomics Reveals a Myeloma Cell Architecture with Dysfunctional T-Cell Distribution, Neutrophil Traps, and Inflammatory Signaling
              </a>
            </td>
          <td>
            Laura Sudupe, Emma Muiños-López, A. Lopez-Perez, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, Paula Aguirre-Ruiz, Patxi San Martín-Úriz, Marta Larrayoz, Laura Alvarez-Gigli, Marta Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho González, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="The molecular underpinnings of High Grade Endometrial Carcinoma (HGEC) metastatic growth and survival are poorly understood. Here we show that ascites-derived and primary tumor HGEC cell lines in 3D spheroid culture faithfully recapitulate key features of malignant peritoneal effusion and exhibit fundamentally distinct transcriptomic, proteomic and metabolomic landscapes when compared with conventional 2D monolayers. Using genetic screening platform we identify MAPK14 (which encodes the protein kinase p38α) as a specific requirement for HGEC in spheroid culture. MAPK14/p38α has broad roles in programing the phosphoproteome, transcriptome and metabolome of HGEC spheroids, yet has negligible impact on monolayer cultures. MAPK14 promotes tumorigenicity in vivo and is specifically required to sustain a sub-population of spheroid cells that is enriched in cancer stemness markers. Therefore, spheroid growth of HGEC activates unique biological programs, including p38α signaling, that cannot be captured using 2D culture models and are highly relevant to malignant disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e7c989b30535b3174da390f7011080ff704c1d1" target='_blank'>
              MAPK14 /p38α Shapes the Molecular Landscape of Endometrial Cancer and promotes Tumorigenic Characteristics
              </a>
            </td>
          <td>
            Sayali Joseph, Xingyuan Zhang, G. Droby, Di Wu, V. Bae-Jump, Scott Lyons, Angie Mordant, Allie Mills, Laura Herring, Blake R Rushing, Jessica L. Bowser, Cyrus Vaziri
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rationale: The heterogeneity of tumor cells within the glioblastoma (GBM) microenvironment presents a complex challenge in curbing GBM progression. Understanding the specific mechanisms of interaction between different GBM cell subclusters and non-tumor cells is crucial. Methods: In this study, we utilized a comprehensive approach integrating glioma single-cell and spatial transcriptomics. This allowed us to examine the molecular interactions and spatial localization within GBM, focusing on a specific tumor cell subcluster, GBM subcluster 6, and M2-type tumor-associated macrophages (M2 TAMs). Results: Our analysis revealed a significant correlation between a specific tumor cell subcluster, GBM cluster 6, and M2-type TAMs. Further in vitro and in vivo experiments demonstrated the specific regulatory role of the CEBPB transcriptional network in GBM subcluster 6, which governs its tumorigenicity, recruitment of M2 TAMs, and polarization. This regulation involves molecules such as MCP1 for macrophage recruitment and the SPP1-Integrin αvβ1-Akt signaling pathway for M2 polarization. Conclusion: Our findings not only deepen our understanding of the formation of M2 TAMs, particularly highlighting the differential roles played by heterogeneous cells within GBM in this process, but also provided new insights for effectively controlling the malignant progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f32ae634b60547b99c952f2958aef6e7b9e1f2ae" target='_blank'>
              The CEBPB+ glioblastoma subcluster specifically drives the formation of M2 tumor-associated macrophages to promote malignancy growth
              </a>
            </td>
          <td>
            Yongchang Yang, Xingyu Jin, Yang Xie, Chunlan Ning, Yiding Ai, Haotian Wei, Xing Xu, Xianglian Ge, Tailong Yi, Qiang Huang, Xuejun Yang, Tao Jiang, Xiaoguang Wang, Yingzhe Piao, Xun Jin
          </td>
          <td>2024-07-02</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Motivation Cells exhibit a wide array of morphological features, enabling computer vision methods to identify and track relevant parameters. Morphological analysis has long been implemented to identify specific cell types and cell responses. Here we asked whether morphological features might also be used to classify transcriptomic subpopulations within in vitro cancer cell lines. Identifying cell subpopulations furthers our understanding of morphology as a reflection of underlying cell phenotype and could enable a better understanding of how subsets of cells compete and cooperate in disease progression and treatment. Results We demonstrate that cell morphology can reflect underlying transcriptomic differences in vitro using convolutional neural networks. First, we find that changes induced by chemotherapy treatment are highly identifiable in a breast cancer cell line. We then show that the intra cell line subpopulations that comprise breast cancer cell lines under standard growth conditions are also identifiable using cell morphology. We find that cell morphology is influenced by neighborhood effects beyond the cell boundary, and that including image information surrounding the cell can improve model discrimination ability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116b8716438d464b34fa34caed09100f7664c755" target='_blank'>
              Deep learning identifies heterogeneous subpopulations in breast cancer cell lines
              </a>
            </td>
          <td>
            T. Jost, Andrea L. Gardner, Daylin Morgan, Amy Brock
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Targeted therapies can induce strong tumor regression, but they typically fail to eradicate metastatic cancers. The elucidation of the causes that enable cancers to survive within a residual disease is essential for finding eradication strategies. The ability of cancers to survive eradication reflects not only cell-intrinsic sensitivities, but also microenvironmental effects. Paracrine signals produced by fibroblast, non-neoplastic cells that make tumor stroma, can provide strong but spatially limited therapy protection. Even though this phenomenon is well-established, its contribution to the ability of tumors to escape eradication remains poorly understood. To address this gap of knowledge, we developed an in silico model that recapitulates the effect of stroma on therapy responses in tumor tissues. This model enabled us to evaluate the contribution of spatial aspects of stroma-mediated resistance. Our analyses reveal that stroma dispersal might be the most important yet overlooked aspect of stromal resistance that determines the overall tumor responses to therapy. Abstract The response of tumors to anti-cancer therapies is defined not only by cell-intrinsic therapy sensitivities but also by local interactions with the tumor microenvironment. Fibroblasts that make tumor stroma have been shown to produce paracrine factors that can strongly reduce the sensitivity of tumor cells to many types of targeted therapies. Moreover, a high stroma/tumor ratio is generally associated with poor survival and reduced therapy responses. However, in contrast to advanced knowledge of the molecular mechanisms responsible for stroma-mediated resistance, its effect on the ability of tumors to escape therapeutic eradication remains poorly understood. To a large extent, this gap of knowledge reflects the challenge of accounting for the spatial aspects of microenvironmental resistance, especially over longer time frames. To address this problem, we integrated spatial inferences of proliferation-death dynamics from an experimental animal model of targeted therapy responses with spatial mathematical modeling. With this approach, we dissected the impact of tumor/stroma distribution, magnitude and distance of stromal effects. While all of the tested parameters affected the ability of tumor cells to resist elimination, spatial patterns of stroma distribution within tumor tissue had a particularly strong impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d34125347f89218581236d182c69f2cff0ff55a" target='_blank'>
              Dissecting the Spatially Restricted Effects of Microenvironment-Mediated Resistance on Targeted Therapy Responses
              </a>
            </td>
          <td>
            Tatiana Miti, Bina Desai, D. Miroshnychenko, D. Basanta, A. Marusyk
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Intracerebral hemorrhage (ICH) poses a formidable challenge in stroke management, with limited therapeutic options, particularly in the realm of immune‐targeted interventions. Clinical trials targeting immune responses post‐ICH have encountered setbacks, potentially attributable to the substantial cellular heterogeneity and intricate intercellular networks within the brain. Here, we present a pioneering investigation utilizing single‐cell RNA sequencing and spatial transcriptome profiling at hyperacute (1 h), acute (24 h), and subacute (7 days) intervals post‐ICH, aimed at unraveling the dynamic immunological landscape and spatial distributions within the cerebral tissue. Our comprehensive analysis revealed distinct cell differentiation patterns among myeloid and lymphocyte populations, along with delineated spatial distributions across various brain regions. Notably, we identified a subset of lymphocytes characterized by the expression of Spp1 and Lyz2, termed macrophage‐associated lymphocytes, which exhibited close interactions with myeloid cells. Specifically, we observed prominent interactions between Lgmn+Macro‐T cells and microglia through the spp1–cd44 pathway during the acute phase post‐ICH in the choroid plexus. These findings represent a significant advancement in our understanding of immune cell dynamics at single‐cell resolution across distinct post‐ICH time points, thereby laying the groundwork for exploring critical temporal windows and informing the development of targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af27d3cf37340430057062faa5cf04622e91eea" target='_blank'>
              Unraveling dynamic immunological landscapes in intracerebral hemorrhage: insights from single‐cell and spatial transcriptomic profiling
              </a>
            </td>
          <td>
            Lingui Gu, Hualin Chen, Mingjiang Sun, Yihao Chen, Qinglei Shi, Jianbo Chang, Junji Wei, Wenbin Ma, X. Bao, Renzhi Wang
          </td>
          <td>2024-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, still most patients experience limited or no clinical benefit. The immune component dynamics during treatment and how it correlated with treatment efficacy remained largely known. We hypothesize that understanding the modulation dynamics of the TME upon ICI treatment would be key to elucidate the mechanisms of response and resistance. To systematically characterize these dynamics, we aggregated a large-scale single-cell RNA-sequencing (scRNA-seq) dataset of 141 patients across multiple cancer types and ICI treatment modalities, with longitudinally paired samples. We developed a robust, integration-free, and cluster-free deep phenotyping framework, leveraging multiple pan-cancer references to annotate 876,410 high-quality cells into 80 granular cell states. Our analysis revealed consistent compositional changes in 17 cell subtypes following ICI treatment, including an enhanced adaptive immune response such as CD8+ IL7R+ central memory T cells, GZMK+ effector memory T cells, germinal center B cells, precursor exhausted T cells, and CD4+ TfhTh1 cells, accompanied by a broad reduction in interferon responsive immune cells quiescent CD4+CREM-memory T cells. We uncovered co-regulated cell communities within the TME, highlighting the coordinated interplay between adaptive and innate immune cells, as well as immune and non-immune components. Furthermore, we identified two distinct patient groups exhibiting tightly correlated cellular dynamics within the TME post-treatment. The first group, enriched for responders, displayed a marked expansion of naive lymphocytes, while the second group, predominantly composed of non-responders, showed an increased abundance of immune experienced/suppressive cell states. This dichotomy in TME dynamics offers a potential predictive biomarker for patient stratification and personalized therapeutic strategies. Our study presents a comprehensive landscape of the cellular dynamics within the TME during ICI treatment, enabled by a powerful deep phenotyping approach showcasing the importance of a systems-level understanding of the TME dynamics in improving patient stratification and advancing personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e90093265f7348aad7b591e51e89915aa6d67b" target='_blank'>
              Cellular Dynamics Upon Immune Checkpoint Inhibition
              </a>
            </td>
          <td>
            Zhongyang Lin, Eketerina Petrenko, Mitalee Chandra, Nalini Srinivas, Selma Ugurel, Jürgen C. Becker, Dvir Aran
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cancers evolve in a dynamic ecosystem. Thus, characterizing cancer’s ecological dynamics is crucial to understanding cancer evolution and can lead to discovering novel biomarkers to predict disease progression. Ductal carcinoma in situ (DCIS) is an early-stage breast cancer characterized by abnormal epithelial cell growth confined within the milk ducts. Although there has been extensive research on genetic and epigenetic causes of breast carcinogenesis, none of these studies have successfully identified a biomarker for the progression and/or upstaging of DCIS. In this study, we show that ecological habitat analysis of hypoxia and acidosis biomarkers can significantly improve prediction of DCIS upstaging. First, we developed a novel eco-evolutionary designed approach to define habitats in the tumor intra-ductal microenvironment based on oxygen diffusion distance in our DCIS cohort of 84 patients. Then, we identify cancer cells with metabolic phenotypes attributed to their habitat conditions, such as the expression of CA9 indicating hypoxia responding phenotype, and LAMP2b indicating a hypoxia-induced acid adaptation. Traditionally these markers have shown limited predictive capabilities for DCIS upstaging, if any. However, when analyzed from an ecological perspective, their power to differentiate between indolent and upstaged DCIS increased significantly. Second, using eco-evolutionary guided computational and digital pathology techniques, we discovered distinct spatial patterns of these biomarkers and used the distribution of such patterns to predict patient upstaging. The patterns were characterized by both cellular features and spatial features. With a 5-fold validation on the biopsy cohort, we trained a random forest classifier to achieve the area under curve(AUC) of 0.74. Our results affirm the importance of using eco-evolutionary-designed approaches in biomarkers discovery studies in the era of digital pathology by demonstrating the role of eco-evolution dynamics in predicting cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786cf3e3e85927b01c56064b452b334cb91ffed1" target='_blank'>
              Eco-evolutionary Guided Pathomic Analysis to Predict DCIS Upstaging
              </a>
            </td>
          <td>
            Yujie Xiao, Manal Elmasry, Ji Dong K. Bai, Andrew Chen, Yuzhu Chen, Brooke Jackson, Joseph O. Johnson, Robert J. Gillies, Prateek Prasanna, Chao Chen, Mehdi Damaghi
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recent lineage tracing single-cell techniques (LT-scSeq), e.g., the Lineage And RNA RecoverY (LARRY) barcoding system, have enabled clonally resolved interpretation of differentiation trajectories. However, the heterogeneity of clone-specific kinetics remains understudied, both quantitatively and in terms of interpretability, thus limiting the power of bar-coding systems to unravel how heterogeneous stem cell clones drive overall cell population dynamics. Here, we present CLADES, a NeuralODE-based framework to faithfully estimate clone-specific kinetics of cell states from newly generated and publicly available human cord blood LARRY LT-scSeq data. By incorporating a stochastic simulation algorithm (SSA) and differential expression gene (DEGs) analysis, CLADES yields cell division dynamics across differentiation timecourses and fate bias predictions for the early progenitor cells. Moreover, clone-level quantitative behaviours can be grouped into characteristic types by pooling individual clones into meta-clones. By benchmarking with CoSpar, we found that CLADES improves fate bias prediction accuracy at the meta-clone level. In conclusion, we report a broadly applicable approach to robustly quantify differentiation kinetics using meta-clones while providing valuable insights into the fate bias of cellular populations for any organ system maintained by a pool of heterogeneous stem and progenitor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181cc1f16a3c5b14fb171a92ae7dc42356fcbe11" target='_blank'>
              Unveiling Clonal Cell Fate and Differentiation Dynamics: A Hybrid NeuralODE-Gillespie Approach
              </a>
            </td>
          <td>
            Mingze Gao, Melania Barile, Shirom Chabra, M. Haltalli, Emily F. Calderbank, Yiming Chao, E. Laurenti, B. Gottgens, Yuanhua Huang
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Melanoma, a malignant skin cancer arising from melanocytes, exhibits rapid metastasis and a high mortality rate, especially in advanced stages. Current treatment modalities, including surgery, radiation, and immunotherapy, offer limited success, with immunotherapy using immune checkpoint inhibitors (ICIs) being the most promising. However, the high mortality rate underscores the urgent need for robust, non-invasive biomarkers to predict patient response to adjuvant therapies. The immune microenvironment of melanoma comprises various immune cells, which influence tumor growth and immune response. Melanoma cells employ multiple mechanisms for immune escape, including defects in immune recognition and epithelial-mesenchymal transition (EMT), which collectively impact treatment efficacy. Single-cell analysis technologies, such as single-cell RNA sequencing (scRNA-seq), have revolutionized the understanding of tumor heterogeneity and immune microenvironment dynamics. These technologies facilitate the identification of rare cell populations, co-expression patterns, and regulatory networks, offering deep insights into tumor progression, immune response, and therapy resistance. In the realm of biomarker discovery for melanoma, single-cell analysis has demonstrated significant potential. It aids in uncovering cellular composition, gene profiles, and novel markers, thus advancing diagnosis, treatment, and prognosis. Additionally, tumor-associated antibodies and specific genetic and cellular markers identified through single-cell analysis hold promise as predictive biomarkers. Despite these advancements, challenges such as RNA-protein expression discrepancies and tumor heterogeneity persist, necessitating further research. Nonetheless, single-cell analysis remains a powerful tool in elucidating the mechanisms underlying therapy response and resistance, ultimately contributing to the development of personalized melanoma therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824bd1e506d1d79fb980c0e85de9df9031024731" target='_blank'>
              Advancing immunotherapy for melanoma: the critical role of single-cell analysis in identifying predictive biomarkers
              </a>
            </td>
          <td>
            Ru He, Jiaan Lu, Jianglong Feng, Ziqing Lu, Kaixin Shen, Ke Xu, Huiyan Luo, Guanhu Yang, Hao Chi, Shangke Huang
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Melanomas are the deadliest skin cancers, in part due to cellular plasticity and heterogeneity. Intratumoral heterogeneity drives varied mutable phenotypes, specifically “melanocytic” and “mesenchymal” cell states, which result in differential functional properties and drug responses. Definitive and rigorous classification of these phenotypic states has been challenging with conventional biomarker-based methods, and high-parameter molecular methods are cell-destructive, labor-intensive, and time-consuming. To overcome these technical and practical limitations, we utilized label-free artificial intelligence-based morphological profiling to classify live melanoma cells into melanocytic and mesenchymal phenotypes based on high resolution imaging of single cells. To predict the phenotypes of single melanoma cells based on morphology alone, we developed the AI-based ‘Melanoma Phenotype Classifier’ trained with 19 patient-derived cell lines with known melanocytic or mesenchymal transcriptional profiles. To link phenotypic state with high-dimensional morphological profiles, cells were subjected to genetic and chemical perturbations known to shift phenotypic states. The AI classifier successfully predicted phenotypic shifts which were confirmed by single-cell RNA-Seq (scRNA-Seq). These results demonstrate that correlations between melanoma cell phenotypes and morphological changes are detectable by AI. Additionally, the Melanoma Phenotype Classifier was applied to dissociated tumor biopsy samples and characterization of phenotypic heterogeneity was supported by scRNA-Seq transcriptional profiles. This work establishes a link between cell morphology and melanoma phenotypes, laying the groundwork for the use of a label-free morphology-based method for phenotyping live melanoma cells combined with additional analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efb6b8eb45b844e5401bfcc2f8a0a6837a8825f0" target='_blank'>
              Label-Free Melanoma Phenotype Classification Using Artificial Intelligence-Based Morphological Profiling
              </a>
            </td>
          <td>
            Evelyn Lattmann, Andreja Jovic, Julie Kim, T. Pham, Christian Corona, Zhouyang Lian, K. Saini, Manisha Ray, Vivian Lu, A. Tastanova, Stephane C. Boutet, Mitchell P. Levesque
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 Therapeutics targeting tumor endothelial cells (TECs) have been explored for decades, with only suboptimal efficacy achieved, partly due to an insufficient understanding of the TEC heterogeneity across cancer patients. We integrated single-cell RNA-seq data of 575 cancer patients from 19 solid tumor types, comprehensively charting the TEC phenotypic diversities. Our analyses uncovered underappreciated compositional and functional heterogeneity in TECs from a pan-cancer perspective. Two subsets, CXCR4+ tip cells and SELE+ veins, represented the prominent angiogenic and proinflammatory phenotypes of TECs, respectively. They exhibited distinct spatial organization patterns, and compared to adjacent non-tumor tissues, tumor tissue showed an increased prevalence of CXCR4+ tip cells, yet with SELE+ veins depleted. Such functional and spatial characteristics underlie their differential associations with the response of anti-angiogenic therapies and immunotherapies. Our integrative resources and findings open new avenues to understand and clinically intervene in the tumor vasculature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc571fa1d1ff50ea9e24190fe0da0d50d2cd7b6f" target='_blank'>
              Pan-cancer integrative analyses dissect the remodeling of endothelial cells in human cancers
              </a>
            </td>
          <td>
            Jinhu Li, Dongfang Wang, Fei Tang, Xinnan Ling, Wenjie Zhang, Zemin Zhang
          </td>
          <td>2024-07-11</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76207c3c4f2b4f2f11d67f0daf7aa41a3f2aa07" target='_blank'>
              Dysfunctional tumor-infiltrating Vδ1 + T lymphocytes in microsatellite-stable colorectal cancer
              </a>
            </td>
          <td>
            V. Stary, Ram V Pandey, Julia List, L. Kleissl, Florian Deckert, J. Kabiljo, J. Laengle, Vasileios Gerakopoulos, Rudolf Oehler, Lukas Watzke, Matthias Farlik, S. Lukowski, Anne B Vogt, Georg Stary, Hannes Stockinger, M. Bergmann, Nina Pilat
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest-derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma.
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Objectives Non‐small‐cell lung carcinoma (NSCLC) is the most prevalent and lethal form of lung cancer. The need for biomarker‐informed stratification of targeted therapies has underpinned the need to uncover the underlying properties of the tumor microenvironment (TME) through high‐plex quantitative assays. Methods In this study, we profiled resected NSCLC tissues from 102 patients by targeted spatial proteomics of 78 proteins across tumor, immune activation, immune cell typing, immune‐oncology, drug targets, cell death and PI3K/AKT modules to identify the tumor and stromal signatures associated with overall survival (OS). Results Survival analysis revealed that stromal CD56 (HR = 0.384, P = 0.06) and tumoral TIM3 (HR = 0.703, P = 0.05) were associated with better survival in univariate Cox models. In contrast, after adjusting for stage, BCLXL (HR = 2.093, P = 0.02) and cleaved caspase 9 (HR = 1.575, P = 0.1) negatively influenced survival. Delta testing indicated the protective effect of TIM‐3 (HR = 0.614, P = 0.04) on OS. In multivariate analysis, CD56 (HR = 0.172, P = 0.001) was associated with better survival in the stroma, while B7.H3 (HR = 1.72, P = 0.008) was linked to poorer survival in the tumor. Conclusions Deciphering the TME using high‐plex spatially resolved methods is giving us new insights into compartmentalised tumor and stromal protein signatures associated with clinical endpoints in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c86af101b13bc4f2f501f32d2e227ba48bb3c7a" target='_blank'>
              Spatial proteomic profiling of tumor and stromal compartments in non‐small‐cell lung cancer identifies signatures associated with overall survival
              </a>
            </td>
          <td>
            Vahid Yaghoubi Naei, J. Monkman, Habib Sadeghirad, Ahmed M Mehdi, Tony Blick, W. Mullally, K. O’Byrne, M. Warkiani, A. Kulasinghe
          </td>
          <td>2024-01-01</td>
          <td>Clinical & Translational Immunology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) mortality rates continue to increase faster than those of other cancer types due to high heterogeneity, which limits diagnosis and treatment. Pathological and molecular subtyping have identified that HCC tumors with poor outcomes are characterized by intratumoral collagenous accumulation. However, the translational and post-translational regulation of tumor collagen, which is critical to the outcome, remains largely unknown. Here, we investigate the spatial extracellular proteome to understand the differences associated with HCC tumors defined by Hoshida transcriptomic subtypes of poor outcome (Subtype 1; S1; n = 12) and better outcome (Subtype 3; S3; n = 24) that show differential stroma-regulated pathways. Collagen-targeted mass spectrometry imaging (MSI) with the same-tissue reference libraries, built from untargeted and targeted LC-MS/MS was used to spatially define the extracellular microenvironment from clinically-characterized, formalin-fixed, paraffin-embedded tissue sections. Collagen α-1(I) chain domains for discoidin-domain receptor and integrin binding showed distinctive spatial distribution within the tumor microenvironment. Hydroxylated proline (HYP)-containing peptides from the triple helical regions of fibrillar collagens distinguished S1 from S3 tumors. Exploratory machine learning on multiple peptides extracted from the tumor regions could distinguish S1 and S3 tumors (with an area under the receiver operating curve of ≥0.98; 95% confidence intervals between 0.976 and 1.00; and accuracies above 94%). An overall finding was that the extracellular microenvironment has a high potential to predict clinically relevant outcomes in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53efde0c062c9d0f5b393a6e975a03778bb11cae" target='_blank'>
              The Spatial Extracellular Proteomic Tumor Microenvironment Distinguishes Molecular Subtypes of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Jade K Macdonald, Harrison B Taylor, Mengjun Wang, A. DelaCourt, Christin Edge, David N. Lewin, Naoto Kubota, Naoto Fujiwara, Fahmida Rasha, Cesia A. Marquez, A. Ono, Shiro Oka, K. Chayama, Sara Lewis, B. Taouli, Myron Schwartz, M. Fiel, Richard R Drake, Y. Hoshida, A. Mehta, Peggi M Angel
          </td>
          <td>2024-07-09</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b06482a1333f1d8cf1f744297d83ab314bc5a5" target='_blank'>
              High-dimensional mapping of human CEACAM1 expression on immune cells and association with melanoma drug resistance
              </a>
            </td>
          <td>
            Yu‐Hwa Huang, Charles H Yoon, Amit Gandhi, Thomas Hanley, Carlos Castrillon, Yasuyuki Kondo, Xi Lin, Walter Kim, Chao Yang, Amine Driouchi, M. Carroll, S. Gray-Owen, D. Wesemann, Charles G. Drake, Monica M Bertagnolli, Nicole Beauchemin, Richard S Blumberg
          </td>
          <td>2024-07-02</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2734d6282129acbb89725ff7878ceaacbfcfa19d" target='_blank'>
              Revealing the crucial roles of suppressive immune microenvironment in cardiac myxoma progression
              </a>
            </td>
          <td>
            Zedong Jiang, Qianlong Kang, Hong Qian, Zhijie Xu, Huan Tong, Jiaqing Yang, Li Li, Renwei Li, Guangqi Li, Fei Chen, Nan Lin, Yunuo Zhao, Huashan Shi, Juan Huang, Xuelei Ma
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors have a complex metabolism that differs from most metabolic processes in healthy tissues. It is highly dynamic and driven by the tumor cells themselves, as well as by the non‐transformed stromal infiltrates and immune components. Each of these cell populations has a distinct metabolism that depends on both their cellular state and the availability of nutrients. Consequently, to fully understand the individual metabolic states of all tumor‐forming cells, correlative mass spectrometric imaging (MSI) up to cellular resolution with minimal metabolite shift needs to be achieved. By using a secondary ion mass spectrometer (SIMS) equipped with an Orbitrap mass analyzer, we present a workflow to image primary murine tumor tissues up to cellular resolution and correlate these ion images with post acquisition immunofluorescence or histological staining. In a murine breast cancer model, we could identify metabolic profiles that clearly distinguish tumor tissue from stromal cells and immune infiltrates. We demonstrate the robustness of the classification by applying the same profiles to an independent murine model of lung cancer, which is accurately segmented by histological traits. Our pipeline allows metabolic segmentation with simultaneous cell identification, which in the future will enable the design of subpopulation‐targeted metabolic interventions for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e6d2390afe35044a275be6c42d6f73bf5b238a" target='_blank'>
              Orbi‐SIMS Mediated Metabolomics Analysis of Pathogenic Tissue up to Cellular Resolution
              </a>
            </td>
          <td>
            Christine Kern, Astrid Scherer, Laura Gambs, Mariia Yuneva, H. Walczak, G. Liccardi, J. Saggau, Peter Kreuzaler, M. Rohnke
          </td>
          <td>2024-07-12</td>
          <td>Chemistry–Methods</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. This study enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained genomic features and histopathological traits of primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, correlating ex vivo drug sensitivity with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program is associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributes to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine. One Sentence Summary This study establishes patient-derived tumor organoids (PDOs) from 31 Head and Neck Squamous Cell Carcinoma (HNSCC) patients, faithfully recapitulating characteristics of primary tumors and accurately predicting clinical responses to cisplatin treatment. We reveal intertumoral heterogeneity within PDOs and a hybrid epithelial-mesenchymal transition (hEMT) program conferring cisplatin resistance, highlighting amphiregulin (AREG) as a regulator of cellular plasticity and potential therapeutic target for HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82266d25cc2972a0e3e79d403bbffccb3c4b2f76" target='_blank'>
              Genomic and single-cell characterization of patient-derived tumor organoid models of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Young Min Park, De-Chen Lin
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Local inflammation in the pancreas is transient but imprints a durable epigenetic memory on epithelial cells, making them more amenable to oncogenic transformation. However, it is unclear whether epithelial cell heterogeneity is impacted by acute pancreatitis (AP) or whether population dynamics during regeneration contributes to the establishment of inflammation memory. To tackle those questions, we deployed experimental pancreatitis in mice and performed paired sequencing of transcriptomic and chromatin accessibility profiles at single nucleus resolution. We documented cell type abundance but also applied integrative analyses to infer phenotypically-distinct clusters of mesenchymal and exocrine cells. We found that AP perturbs a subset of “idling” acinar cells, which separate from more canonical “secretory” acini based on a more diversified proteome, which include elevated expression of signal transduction receptors. We linked acinar cell heterogeneity to epigenetic differences that also endow idling cells with superior plasticity. These constitute about 40% of acinar cells but can proliferate and skew their phenotype in response to AP. This leads to a remarkable recovery of pancreas histology and function, but also to the dissemination of idling-like features across the exocrine parenchyma. Mechanistically, idling acinar cells are characterized by enhanced transcriptional activity and protein synthesis. After recovery from pancreatitis, acini show elevation of both and establishment of chronic Unfolded Protein Response (UPR). We finally demonstrated that AP-primed pancreata show signs of elevated UPR and that ER stress promotes acinar cell metaplasia. Our data interrogate phenotypical dynamics during tissue regeneration to identify cell states amenable to epigenetic imprinting. They also suggest that UPR-alleviating strategies might curtail the risk of developing pancreatic cancer for individuals who experiences AP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0889f4e0ef737079e14a1df3e2a78bed4691d30d" target='_blank'>
              Population dynamics after pancreatitis dictates long-lasting epigenetic reprogramming and mediates tumor predisposition
              </a>
            </td>
          <td>
            Marco Fantuz, J. Liebig, Giovanni Fontana, Valerio Iebba, Carmine Carbone, Sören Lukassen, Christian Conrad, Alessandro Carrer
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the most common form of pancreatic cancer, a disease with dismal overall survival. Advances in treatment are hindered by a lack of preclinical models. Here, we show how a personalized organotypic “avatar” created from resected tissue allows spatial and temporal reporting on a complete in situ tumor microenvironment and mirrors clinical responses. Our perfusion culture method extends tumor slice viability, maintaining stable tumor content, metabolism, stromal composition, and immune cell populations for 12 days. Using multiplexed immunofluorescence and spatial transcriptomics, we identify immune neighborhoods and potential for immunotherapy. We used avatars to assess the impact of a preclinically validated metabolic therapy and show recovery of stromal and immune phenotypes and tumor redifferentiation. To determine clinical relevance, we monitored avatar response to gemcitabine treatment and identify a patient avatar-predictable response from clinical follow-up. Thus, avatars provide valuable information for syngeneic testing of therapeutics and a truly personalized therapeutic assessment platform for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361225cc3eb1a10b3d464aab534da9288640614c" target='_blank'>
              Development of human pancreatic cancer avatars as a model for dynamic immune landscape profiling and personalized therapy
              </a>
            </td>
          <td>
            Daniel Hughes, Alice Evans, Simei Go, Michael Eyres, Liuliu Pan, Somnath Mukherjee, Z. Soonawalla, Frances Willenbrock, Eric O’Neill
          </td>
          <td>2024-07-05</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Clone assignment in single-cell genomics remains a challenge due to its diverse mutation macrostructures and many missing signals. Existing statistical methods, for the sake of numerical convergence, pose strong constraints on the form of predicted mutation patterns, so they easily identify sub-optimally fitted clones that overlook weak and rare mutations. To solve this problem, we developed SNPmanifold, a Python package that learns flexible mutation patterns using a shallow binomial variational autoencoder. The latent space of SNPmanifold can effectively represent and visualize complex mutations of SNPs (single-nucleotide polymorphisms) in the form of geometrical manifolds. Based on nuclear or mitochondrial SNPs, we demonstrated that SNPmanifold can effectively identify a large number of multiplexed donors of origin (k = 18) that all existing unsupervised methods fail and lineages of somatic clones with promising biological interpretation. Therefore, SNPmanifold can reveal insights into single-cell SNPs more comprehensively than other existing methods, especially in complex datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f5cea3d97421756c5e4a1a65f43ef0a97c212b" target='_blank'>
              Interpretable variational encoding of genotypes identifies comprehensive clonality and lineages in single cells geometrically
              </a>
            </td>
          <td>
            Hoi Man Chung, Yuanhua Huang
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Severe Coronavirus disease 2019 (COVID-19) induces heterogeneous and progressive diffuse alveolar damage (DAD) highly disrupting lung tissue architecture and homeostasis, hampering disease management leading to fatal outcomes. Characterizing DAD pathophysiology across disease progression is of ultimate importance to better understand the molecular and cellular features driving different DAD patterns and to optimize treatment strategies. To contextualize the interplay between cell types and assess their distribution, spatial transcriptomics (ST) techniques have emerged, allowing unprecedented resolution to investigate spatial architecture of tissues. To this end, post-mortem lung tissue provides valuable insights into cellular composition and their spatial relationships at the time of death. Here, we have leveraged VisumST technology in post-mortem COVID-19 induced acute and proliferative DAD lungs including control samples with normal morphological appearance, to unravel the immunopathological mechanisms underlying DAD, providing novel insights into cellular and molecular communication events driving DAD progression in fatal COVID-19. We report a progressive loss of endothelial cell types, pneumocytes type I and natural killer cells coupled with a continuous increase of myeloid and stromal cells, mostly peribronchial fibroblasts, over disease progression. Spatial organization analysis identified variable cellular compartments, ranging from major compartments defined by cell type lineages in control lungs to increased and more specific compartmentalization including immune-specific clusters across DAD spectrum. Importantly, spatially informed ligand-receptor interaction (LRI) analysis revealed an intercellular communication signature defining COVID-19 induced DAD lungs. Transcription factor (TF) activity enrichment analysis identified TGF-B pathway as DAD driver, highlighting SMAD3 and SMAD7 TFs activity role during lung fibrosis. Integration of deregulated LRIs and TFs activity allowed us to propose a downstream intracellular signaling pathway in peribronchial fibroblasts, suggesting potential novel therapeutic targets. Finally, spatio-temporal trajectories analysis provided insights into the alveolar epithelium regeneration program, characterizing markers of pneumocytes type II differentiation towards pneumocytes type I. In conclusion, we provide a spatial characterization of lung tissue architecture upon COVID-19 induced DAD progression, identifying molecular and cellular hallmarks that may help optimize treatment and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a4db747d52faae9abbf25e5031f73c4597d1cc" target='_blank'>
              Spatial transcriptomics unveils the in situ cellular and molecular hallmarks of the lung in fatal COVID-19
              </a>
            </td>
          <td>
            C. García-Prieto, Eva Musulen, V. Davalos, Gerardo Ferrer, D. Grases, E. Porta, Belén Pérez-Miés, T. Caniego-Casas, José Palacios, Xavier Saenz-Sardà, Elisabet Englund, Manel Esteller
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins. Methods We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES- dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP- sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro. Results Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft. Conclusions Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states. Key Points Cellular plasticity influences the abundance of immunotherapeutic targets. Subtype-specific targets may be susceptible to epigenetically-mediated downregulation. Immunotherapeutic targets in development, B7-H3 and L1CAM, show “pan-subtype” expression. Importance of Study Neuroblastoma is a lethal childhood malignancy that shows cellular plasticity in response to anti-cancer therapies. Several plasma membrane proteins are being developed as immunotherapeutic targets in this disease. Here we define which cell surface proteins are susceptible to epigenetically regulated downregulation during an adrenergic to mesenchymal cell state switch and propose immunotherapeutic strategies to anticipate and circumvent acquired immunotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c43da3990dff1c35a4148b920ff54013c4ee8ec" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, F. Zammarchi, Patrick H van Berkel, Chi V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Interactions between mutant cells and their environment play a key role in determining cancer susceptibility. However, our understanding of how the pre-cancer microenvironment contributes to early tumorigenesis remains limited. Here, we show that newly emerging tumours at their most incipient stages shape their microenvironment in a critical process that determines their survival. Analysis of nascent squamous tumours in the upper gastrointestinal tract of the mouse reveals that the stress response of early tumour cells instructs the underlying mesenchyme to form a supportive “pre-cancer niche”, which dictates the long-term outcome of epithelial lesions. Stimulated fibroblasts beneath emerging tumours activate a wound healing response that triggers a dramatic remodelling of the underlying extracellular matrix, resulting in the formation of a fibronectin-rich stromal scaffold that promotes tumour growth. Functional heterotypic 3D culture assays and in vivo grafting experiments, combining carcinogen-free healthy epithelium and tumour-derived stroma, demonstrate that the pre-cancerous niche alone is sufficient to confer tumour properties to healthy epithelial cells. We propose a model where both mutations and the stromal response to genetic stress defines the likelihood of early tumours to survive and progress towards more advanced disease stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be23254ceedda48875f58bb5d700c5fd073374a7" target='_blank'>
              Pre-cancerous Niche Remodelling Dictates Nascent Tumour Survival
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. E. Rojo Arias, Y. Dang, S. Han, M. T. Bejar, B. Colom, J. C. Fowler, P. Jones, S. Rulands, B. D. Simons, M. P. Alcolea
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Microvascular invasion (MVI) stands as a pivotal pathological hallmark of hepatocellular carcinoma (HCC), closely linked to unfavorable prognosis, early recurrence, and metastatic progression. However, the precise mechanistic underpinnings governing its onset and advancement remain elusive. Methods In this research, we downloaded bulk RNA-seq data from the TCGA and HCCDB repositories, single-cell RNA-seq data from the GEO database, and spatial transcriptomics data from the CNCB database. Leveraging the Scissor algorithm, we delineated prognosis-related cell subpopulations and discerned a distinct MVI-related malignant cell subtype. A comprehensive exploration of these malignant cell subpopulations was undertaken through pseudotime analysis and cell-cell communication scrutiny. Furthermore, we engineered a prognostic model grounded in MVI-related genes, employing 101 algorithm combinations integrated by 10 machine-learning algorithms on the TCGA training set. Rigorous evaluation ensued on internal testing sets and external validation sets, employing C-index, calibration curves, and decision curve analysis (DCA). Results Pseudotime analysis indicated that malignant cells, showing a positive correlation with MVI, were primarily concentrated in the early to middle stages of differentiation, correlating with an unfavorable prognosis. Importantly, these cells showed significant enrichment in the MYC pathway and were involved in extensive interactions with diverse cell types via the MIF signaling pathway. The association of malignant cells with the MVI phenotype was corroborated through validation in spatial transcriptomics data. The prognostic model we devised demonstrated exceptional sensitivity and specificity, surpassing the performance of most previously published models. Calibration curves and DCA underscored the clinical utility of this model. Conclusions Through integrated multi-transcriptomics analysis, we delineated MVI-related malignant cells and elucidated their biological functions. This study provided novel insights for managing HCC, with the constructed prognostic model offering valuable support for clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f58d531e5dbaa5fd2fdfd9279935fd61dc3adf" target='_blank'>
              Multi-transcriptomics analysis of microvascular invasion-related malignant cells and development of a machine learning-based prognostic model in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Haoran Huang, Feifeng Wu, Yang Yu, Bo Xu, De-Hua Chen, Yuwei Huo, Shaoqiang Li
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immunotherapies have shown great promise in pleural mesothelioma (PM), yet most patients still do not achieve significant clinical response, highlighting the importance of improving understanding of the tumor microenvironment (TME). Here, we utilized high-throughput, single-cell RNA-sequencing to de novo identify 54 expression programs and construct a comprehensive cellular catalogue of the PM TME. We found four cancer-intrinsic programs associated with poor disease outcome and a novel fetal-like, endothelial cell population that likely responds to VEGF signaling and promotes angiogenesis. Throughout cellular compartments, we observe substantial difference in the TME associated with a cancer-intrinsic sarcomatoid signature, including enrichment in fetal-like endothelial cells, CXCL9+ macrophages, cytotoxic, exhausted, and regulatory T cells, which we validated using imaging and bulk deconvolution analyses on independent cohorts. Finally, we show, both computationally and experimentally, that NKG2A-HLA-E interaction between NK and tumor cells represents an important new therapeutic axis in PM, especially for epithelioid cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c84bfe3e9c66b621b1832492f79b2264c3e648" target='_blank'>
              Single cell view of tumor microenvironment gradients in pleural mesothelioma.
              </a>
            </td>
          <td>
            Bruno Giotti, Komal Dolasia, William Zhao, Peiwen Cai, Robert Sweeney, Elliot Merritt, Evgeny Kiner, Grace S Kim, Atharva Bhagwat, Thinh Nguyen, Samarth Hegde, Bailey G Fitzgerald, S. Shroff, T. Dawson, M. Garcia-Barros, Jamshid Abdul-Ghafar, Rachel Chen, Sacha Gnjatic, Alan Soto, Rachel Brody, S. Kim-Schulze, Zhihong Chen, Kristin G Beaumont, Miram Merad, R.M. Flores, R. Sebra, Amir Horowitz, Thomas U. Marron, Anna Tocheva, Andrea Wolf, A. Tsankov
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Granulomas (defining tuberculosis histopathological feature) are central to the host’s defense against Mycobacterium tuberculosis, critically influencing patient outcomes. However, knowledge of human granulomas’ structure and function are incomplete. This study employs single-cell and spatial transcriptomics to dissect human granuloma’s cellular composition, structure, communication and function from 19 pulmonary, lymphatic and skeletal samples. Results Our study identified nine key immune-activated/signaling-active cell clusters. Notably, we delineated a three-layered granuloma structure: a core with macrophages (Macro-c09, Macro-c10) and occasional fibroblasts (Fib-c03); a fibroblast-rich (Fib-c01) periphery; and an immune-infiltrated intermediate layer comprising diverse immune-cells recruited by strong signaling-molecules (SPP1/MIF) from core/periphery cells. This study also shows granuloma heterogeneity across individuals and tissues. Conclusions By merging scRNA-seq with ST-seq, we offer an intricate single-cell perspective of granulomas’ spatial-structure and formation mechanisms, identify signaling-molecules and significantly changed genes as potential targets for host-directed tuberculosis immunotherapy, highlight fibroblasts’ crucial role in granuloma formation, and provide an important reference/improved understanding of TB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c14b14dcc946c712193e06986c65772b8af84b9" target='_blank'>
              Single-Cell and Spatial Transcriptomic Analyses Deciphering the Three-Layer Architecture of Human Tuberculosis Granulomas
              </a>
            </td>
          <td>
            Xia Yu, Jie Wang, Peihan Wang, Xiaoqiang Liu, Cuidan Li, Yingjiao Ju, Sitong Liu, Yujie Dong, Jing Wang, Bahetibieke Tuohetaerbaike, Hao Wen, Wenbao Zhang, Haitao Niu, Sihong Xu, Chunlai Jiang, Xiaoyi Jiang, Jing Wu, Hairong Huang, Fei Chen
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The involvement of cytotoxic CD4+ T cells (CD4+ CTLs) and their potential role in dictating the response to immune checkpoint inhibitors (ICIs) in patients with metastatic renal cell carcinoma (mRCC) remains an unexplored area of research. Methods Utilizing single-cell RNA sequencing, we analyzed the immunophenotype and expression patterns of CD4+ T lymphocyte subtypes in mRCC patients, followed by preliminary validation via multi-immunofluorescent staining. In addition, we obtained a comprehensive immunotherapy dataset encompassing single-cell RNA sequencing datasets and bulk RNA-seq cohorts from the European Genome-Phenome Archive and ArrayExpress database. Utilizing the CIBERSORTx deconvolution algorithms, we derived a signature score for CD4+ CTLs from the bulk-RNA-seq datasets of the CheckMate 009/025 clinical trials. Results Single-cell analysis of CD4+ T lymphocytes in mRCC reveals several cancer-specific states, including diverse phenotypes of regulatory T cells. Remarkably, we observe that CD4+ CTLs cells constitute a substantial proportion of all CD4+ T lymphocyte sub-clusters in mRCC patients, highlighting their potential significance in the disease. Furthermore, within mRCC patients, we identify two distinct cytotoxic states of CD4+ T cells: CD4+GZMK+ T cells, which exhibit a weaker cytotoxic potential, and CD4+GZMB+ T cells, which demonstrate robust cytotoxic activity. Both regulatory T cells and CD4+ CTLs originate from proliferating CD4+ T cells within mRCC tissues. Intriguingly, our trajectory analysis indicates that the weakly cytotoxic CD4+GZMK+ T cells differentiate from their more cytotoxic CD4+GZMB+ counterparts. In comparing patients with lower CD4+ CTLs levels to those with higher CD4+ CTLs abundance in the CheckMate 009 and 25 immunotherapy cohorts, the latter group exhibited significantly improved OS and PFS probability. Conclusion Our study underscores the pivotal role that intratumoral CD4+ CTLs may play in bolstering anti-tumor immunity, suggesting their potential as a promising biomarker for predicting response to ICIs in patients with mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f2bc88c864cde26033332d6b179b9c1f5c8c941" target='_blank'>
              Single-Cell Analysis Identifies Distinct Populations of Cytotoxic CD4+ T Cells Linked to the Therapeutic Efficacy of Immune Checkpoint Inhibitors in Metastatic Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Xu Yang, Jianwei Wu, Longlong Fan, Binghua Chen, Shiqiang Zhang, Wenzhong Zheng
          </td>
          <td>2024-07-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86603b6c8f27f9a5a7d6a05553f0d41b941be5e5" target='_blank'>
              Spatial landscapes of cancers: insights and opportunities.
              </a>
            </td>
          <td>
            Julia Chen, L. Larsson, Alexander Swarbrick, J. Lundeberg
          </td>
          <td>2024-07-23</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Adoptive T-cell therapy targeting antigens expressed in glioblastoma has emerged as a potential therapeutic strategy to prevent or delay recurrence and prolong overall survival in this aggressive disease setting. Ephrin receptor A3 (EphA3), which is highly expressed in glioblastoma; in particular, on the tumor vasculature and brain cancer stem cells, is an ideal target for immune-based therapies. Methods We have designed an EphA3-targeted chimeric antigen receptor (CAR) using the single chain variable fragment of a novel monoclonal antibody, and assessed its therapeutic potential against EphA3-expressing patient-derived glioblastoma neurospheres, organoids and xenografted glioblastoma tumors in immunodeficient mice. Results In vitro expanded EphA3 CAR T cells from healthy individuals efficiently recognize and kill EphA3-positive glioblastoma cells in vitro. Furthermore, these effector cells demonstrated curative efficacy in an orthotopic xenograft model of glioblastoma. EphA3 CAR T cells were equally effective in targeting patient-derived neurospheres and infiltrate, disaggregate, and induce apoptosis in glioblastoma-derived organoids. Conclusions This study provides compelling evidence supporting the therapeutic potential of EphA3 CAR T-cell therapy against glioblastoma by targeting EphA3 associated with brain cancer stem cells and the tumor vasculature. The ability to target patient-derived glioblastoma underscores the translational significance of this EphA3 CAR T-cell therapy in the pursuit of effective and targeted glioblastoma treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d8e4bbabec48b8e92d18d0d80291e093d16ba77" target='_blank'>
              EphA3 CAR T cells are effective against glioblastoma in preclinical models
              </a>
            </td>
          <td>
            Paulo Martins, R. D'Souza, N. Skarne, L. Lekieffre, Shane Horsefield, M. Ranjankumar, Xiang Li, Thuy T Le, Fiona Smith, Corey Smith, Jacqueline Burrows, Bryan Day, R. Khanna
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, O. Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a complex tumor microenvironment (TME) that promotes cancer progression. The tumor stroma is characterized by cancer-associated fibroblasts (CAFs) that secrete factors stimulating tumor growth, invasion, and immunosuppression. Macrophages tend to polarize toward a pro-tumor phenotype. Various other components, including endothelial cells, pericytes, and neural cells, likely contribute to the complexity of the PDAC TME. Epidemiological investigations have highlighted obesity as a risk factor for pancreatic cancer, suggesting that adipocytes may also play a significant role. This manuscript offers an overview of the main actors in TME, as well as dysregulated signaling pathways and gene mutations. It underlines how strategies modulating the TME hold promise for improving outcomes. Lastly, it describes how integrating multi-omic approaches may represent a future trajectory to explore novel therapeutic possibilities. Abstract Pancreatic ductal adenocarcinoma (PDAC) presents significant oncological challenges due to its aggressive nature and poor prognosis. The tumor microenvironment (TME) plays a critical role in progression and treatment resistance. Non-neoplastic cells, such as cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), contribute to tumor growth, angiogenesis, and immune evasion. Although immune cells infiltrate TME, tumor cells evade immune responses by secreting chemokines and expressing immune checkpoint inhibitors (ICIs). Vascular components, like endothelial cells and pericytes, stimulate angiogenesis to support tumor growth, while adipocytes secrete factors that promote cell growth, invasion, and treatment resistance. Additionally, perineural invasion, a characteristic feature of PDAC, contributes to local recurrence and poor prognosis. Moreover, key signaling pathways including Kirsten rat sarcoma viral oncogene (KRAS), transforming growth factor beta (TGF-β), Notch, hypoxia-inducible factor (HIF), and Wnt/β-catenin drive tumor progression and resistance. Targeting the TME is crucial for developing effective therapies, including strategies like inhibiting CAFs, modulating immune response, disrupting angiogenesis, and blocking neural cell interactions. A recent multi-omic approach has identified signature genes associated with anoikis resistance, which could serve as prognostic biomarkers and targets for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23cd29cbd6982ee3e4b132b1c93695ffca70902" target='_blank'>
              Decoding the Intricate Landscape of Pancreatic Cancer: Insights into Tumor Biology, Microenvironment, and Therapeutic Interventions
              </a>
            </td>
          <td>
            A. Argentiero, Alessandro Andriano, Ingrid Catalina Caradonna, Giulia de Martino, V. Desantis
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract While paediatric blood cancers are deadly, modern medical advances have enabled clinicians to measure levels of residual cancer cells to manage therapeutic strategies for patients. However, blood cancers, including leukaemias and lymphomas, are highly heterogeneous and is comprised of complex clonal populations that can hinder efforts in detecting the cancer cells as well as managing treatments. Furthermore, the tumour microenvironment is comprised of heterogenous immune dynamics that may be different between patients. High‐throughput sequencing has constributed to new discoveries in genetic and transcriptomic alterations underpinning cancer, including blood cancers, and has changed how patients are monitored and managed. Here we discuss the recent efforts using single‐cell approach, particularly on efforts to track clonal heterogenity of paediatric blood cancer and the underlying immune response, highlighting avenues for novel biomarker discovery that may have significant impact on clinical oncology practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9b7aa16b92f8240bd156cad7d9dc00c572051c" target='_blank'>
              Measuring single‐cell immune clonality to track haematological cancers
              </a>
            </td>
          <td>
            Amos Choo, Z. Tuong
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d5310f7c7a3d332e8cba0823f8a46905b9d7fe7" target='_blank'>
              Utilizing human cerebral organoids to model breast cancer brain metastasis in culture
              </a>
            </td>
          <td>
            Chenran Wang, A. Nagayach, Harsh Patel, Lan Dao, Hui Zhu, Amanda R Wasylishen, Yanbo Fan, Ady Kendler, Ziyuan Guo
          </td>
          <td>2024-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b4b7d78b4a9a56b29e32a29b6b57cf4515f6ed0" target='_blank'>
              Panoramic tumor microenvironment in pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Xiaoying Li, Wanting Hou, Chaoxin Xiao, Heqi Yang, Chengjian Zhao, Dan Cao
          </td>
          <td>2024-07-15</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background High-grade gliomas including glioblastoma (GBM) and diffuse midline gliomas (DMG) represent the most lethal and aggressive brain cancers where current treatment modalities offer limited efficacy. Chimeric antigen receptor (CAR) T cell therapies have emerged as a promising strategy, boasting tumor-specific targeting and the unique ability to penetrate the blood-brain barrier. However, the effective clinical application hinges on the optimal choice of antigen, with a limited number, currently under investigation. Methods We employed cell surface proteomic analysis of primary human high-grade glioma samples from both adult and pediatric patients. This led to the identification of Ephrin type-A receptor 3 (EphA3) as a prevalently expressed target. We engineered a second-generation EphA3-targeted CAR T cell and assessed function using in vitro and in vivo models of GBM and DMG. Results EphA3-targeted CAR T cells demonstrated robust antigen-specific killing of human GBM and DMG cell lines in vitro. In an orthotopic xenograft NSG mouse model, EphA3-targeted CAR T cells not only effectively eradicated tumors but also established a functional T cell population protective on rechallenge. Remarkably, mice rechallenged with a second contralateral orthotopic tumor implantation achieved complete tumor clearance and maintained a sustained complete response 6 months following initial treatment. Conclusion Building on the proven safety profile of EphA3 antibodies in clinical settings, our study provides compelling preclinical evidence supporting the efficacy of EphA3-targeted CAR T cells against high-grade gliomas. These findings underscore the potential for transitioning this innovative therapy into clinical trials, aiming to revolutionize the treatment landscape for patients afflicted with these formidable brain cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ccdf5eef190aeacc1b356281cbe1f1aeb78930" target='_blank'>
              EphA3-targeted chimeric antigen receptor T cells are effective in glioma and generate curative memory T cell responses
              </a>
            </td>
          <td>
            Leesa Lertsumitkul, Melinda Iliopoulos, Stacie S Wang, Sarah J McArthur, Lisa M Ebert, Alexander J Davenport, R. Endersby, Jordan R Hansford, Kate Drummond, Ryan S Cross, Misty R. Jenkins
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Melanoma, a highly malignant tumour, presents significant challenges due to its cellular heterogeneity, yet research on this aspect in cutaneous melanoma remains limited. In this study, we utilized single‐cell data from 92,521 cells to explore the tumour cell landscape. Through clustering analysis, we identified six distinct cell clusters and investigated their differentiation and metabolic heterogeneity using multi‐omics approaches. Notably, cytotrace analysis and pseudotime trajectories revealed distinct stages of tumour cell differentiation, which have implications for patient survival. By leveraging markers from these clusters, we developed a tumour cell‐specific machine learning model (TCM). This model not only predicts patient outcomes and responses to immunotherapy, but also distinguishes between genomically stable and unstable tumours and identifies inflamed (‘hot’) versus non‐inflamed (‘cold’) tumours. Intriguingly, the TCM score showed a strong association with TOMM40, which we experimentally validated as an oncogene promoting tumour proliferation, invasion and migration. Overall, our findings introduce a novel biomarker score that aids in selecting melanoma patients for improved prognoses and targeted immunotherapy, thereby guiding clinical treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceff46cb2d3d44eadc13e557a6116c6ae860759" target='_blank'>
              Unravelling tumour cell diversity and prognostic signatures in cutaneous melanoma through machine learning analysis
              </a>
            </td>
          <td>
            Wenhao Cheng, Ping Ni, Hao Wu, Xiaye Miao, Xiaodong Zhao, Dali Yan
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors with dismal prognosis. One of the main bottlenecks for developing more effective therapies for glioblastoma stems from their histologic and molecular heterogeneity, leading to distinct tumor microenvironments and disease phenotypes. Effectively characterizing these features would improve the clinical management of glioblastoma. Glucose flux rates through glycolysis and mitochondrial oxidation have been recently shown to quantitatively depict glioblastoma proliferation in mouse models (GL261 and CT2A tumors, 38±3 mm3) using dynamic glucose-enhanced (DGE) deuterium spectroscopy. However, the spatial features of tumor microenvironment phenotypes remain hitherto unresolved. Here, we develop a DGE Deuterium Metabolic Imaging (DMI) approach for profiling tumor microenvironments through glucose conversion kinetics. Using a multimodal combination of tumor mouse models, novel strategies for spectroscopic imaging and noise attenuation, and histopathological correlations, we show that tumor lactate turnover mirrors phenotype differences between GL261 and CT2A mouse glioblastoma (59±7 mm3), whereas peritumoral glutamate-glutamine recycling is a potential marker of invasion capacity in pooled cohorts, linked to secondary brain lesions. Our findings were validated by histopathological characterization of each tumor, including cell density and proliferation, peritumoral infiltration, and distant migration. Our study bodes well for precision neuro-oncology, highlighting the importance of mapping glucose flux rates to better understand the metabolic heterogeneity of glioblastoma and its links to disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2927f90371bde6768c35753a6b358801ccb1fb15" target='_blank'>
              Deuterium Metabolic Imaging Phenotypes Mouse Glioblastoma Heterogeneity Through Glucose Turnover Kinetics
              </a>
            </td>
          <td>
            Rui V. Simões, Rafael N. Henriques, J. Olesen, Beatriz M Cardoso, Francisca F. Fernandes, Mariana A.V. Monteiro, S. Jespersen, Tânia Carvalho, N. Shemesh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Spatially resolved transcriptomics simultaneously measure the spatial location, histology images, and transcriptional profiles of the same cells or regions in undissociated tissues. Integrative analysis of multi-modal spatially resolved data holds immense potential for understanding the mechanisms of biology. Here we present a flexible multi-modal contrastive learning for the integration of spatially resolved transcriptomics (MuCST), which jointly perform denoising, elimination of heterogeneity, and compatible feature learning. We demonstrate that MuCST robustly and accurately identifies tissue subpopulations from simulated data with various types of perturbations. In cancer-related tissues, MuCST precisely identifies tumor-associated domains, reveals gene biomarkers for tumor regions, and exposes intra-tumoral heterogeneity. We also validate that MuCST is applicable to diverse datasets generated from various platforms, such as STARmap, Visium, and omsFISH for spatial transcriptomics, and hematoxylin and eosin or fluorescence microscopy for images. Overall, MuCST not only facilitates the integration of multi-modal spatially resolved data, but also serves as pre-processing for data restoration (Python software is available at https://github.com/xkmaxidian/MuCST).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e93e7b00fc6643c076f03733108be8401ccb4c7" target='_blank'>
              MuCST: restoring and integrating heterogeneous morphology images and spatial transcriptomics data with contrastive learning
              </a>
            </td>
          <td>
            Yu Wang, Xiaoke Ma
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Metastases from gastric adenocarcinoma (GAC) lead to high morbidity and mortality. Developing innovative and effective therapies requires a comprehensive understanding of the tumor and immune biology of advanced GAC. Yet, collecting matched specimens from advanced, treatment-naïve GAC patients poses a significant challenge, limiting the scope of current research, which has predominantly focused on localized tumors. This gap hinders a deeper insight into the metastatic dynamics of GAC.


METHODS
We performed in-depth single-cell transcriptome and immune profiling on 68 paired, treatment-naïve, primary-metastatic tumors to delineate alterations in cancer cells and their tumor microenvironment (TME) during metastatic progression. To validate our observations, we conducted comprehensive functional studies both in vitro and in vivo, employing cell lines, multiple PDX and novel mouse models of GAC.


RESULTS
Liver and peritoneal metastases exhibited distinct properties in cancer cells and dynamics of TME phenotypes, supporting the notion that cancer cells and their local TMEs co-evolve at metastatic sites. Our study also revealed differential activation of cancer meta-programs across metastases. We observed evasion of cancer cell ferroptosis via GPX4 upregulation during GAC progression. Conditional depletion of Gpx4 or pharmacological inhibition of ferroptosis resistance significantly attenuated tumor growth and metastatic progression. Additionally, ferroptosis-resensitizing treatments augmented the efficacy of CAR T-cell therapy.


CONCLUSIONS
This study represents the largest single-cell dataset of metastatic GACs to date. High-resolution mapping of the molecular and cellular dynamics of GAC metastasis has revealed a rationale for targeting ferroptosis defense in combination with CAR T-cell therapy as a novel therapeutic strategy with potential immense clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c4774948f3c82306cdc21b3d5730989f11741b" target='_blank'>
              Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response.
              </a>
            </td>
          <td>
            Xiangdong Cheng, Enyu Dai, Jibo Wu, Natasha M. Flores, Yanshuo Chu, Ruiping Wang, M. Dang, Zhiyuan Xu, Guangchun Han, Yunhe Liu, D. Chatterjee, Can Hu, Jieer Ying, Yi-an Du, Litao Yang, Xiaoqing Guan, Shaowei Mo, Xuanye Cao, Guangsheng Pei, Jiahui Jiang, Xiaoyin Lu, Ana Morales Benitez, R. Waters, Melissa P Pizzi, N. Shanbhag, Yibo Fan, F. Peng, Samir Hanash, G. Calin, Andrew Futreal, Shumei Song, Cassian Yee, Pawel K Mazur, Jiang-Jiang Qin, J. Ajani, Linghua Wang
          </td>
          <td>2024-08-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="Tumor growth dynamics serve as a critical aspect of understanding cancer progression and treatment response to mitigate one of the most pressing challenges in healthcare. The in silico approach to understanding tumor behavior computationally provides an efficient, cost-effective alternative to wet-lab examinations and are adaptable to different environmental conditions, time scales, and unique patient parameters. As a result, this paper explored modeling of free tumor growth in cancer, surveying contemporary literature on continuum, discrete, and hybrid approaches. Factors like predictive power and high-resolution simulation competed against drawbacks like simulation load and parameter feasibility in these models. Understanding tumor behavior in different scenarios and contexts became the first step in advancing cancer research and revolutionizing clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5814b4312284d51679616c4dcd34ca6a0385e075" target='_blank'>
              Modeling free tumor growth: Discrete, continuum, and hybrid approaches to interpreting cancer development.
              </a>
            </td>
          <td>
            Dashmi Singh, Dana Paquin
          </td>
          <td>2024-07-19</td>
          <td>Mathematical biosciences and engineering : MBE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pericytes, recognized as mural cells, have long been described as components involved in blood vessel formation, playing a mere supporting role for endothelial cells (ECs). Emerging evidence strongly suggests their multifaceted roles in tissues and organs. Indeed, pericytes exhibit a remarkable ability to anticipate endothelial cell behavior and adapt their functions based on the specific cells they interact with. Pericytes can be activated by pro-inflammatory stimuli and crosstalk with immune cells, actively participating in their transmigration into blood vessels. Moreover, they can influence the immune response, often sustaining an immunosuppressive phenotype in most of the cancer types studied. In this review, we concentrate on the intricate crosstalk between pericytes and immune cells in cancer, highlighting the primary evidence regarding pericyte involvement in primary tumor mass dynamics, their contributions to tumor reprogramming for invasion and migration of malignant cells, and their role in the formation of pre-metastatic niches. Finally, we explored recent and emerging pharmacological approaches aimed at vascular normalization, including novel strategies to enhance the efficacy of immunotherapy through combined use with anti-angiogenic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76bcdf2dcb6f3841dc22f4ab784010532400194" target='_blank'>
              Pericytes: jack-of-all-trades in cancer-related inflammation
              </a>
            </td>
          <td>
            Marianna Moro, Federica Carolina Balestrero, A. A. Grolla
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are ectopic lymphoid aggregates associated with improved prognosis in numerous tumors. To date, their prognostic value in central nervous system cancers and the events underlying their formation remain unclear. Here, we find that TLS correlate with improved survival in glioblastoma patients. Furthermore, combining spatial transcriptomics of human tissues with longitudinal studies in murine glioma, we establish that the assembly of T cell-rich clusters is a prerequisite for the development of canonical TLS, and show that CD4 T cells play a central role in this process. Indeed, we provide evidence that IL7R+CCR7+ Th1 lymphocytes with lymphoid tissue-inducing potential are recruited to TLS nucleation sites, where they can initiate TLS assembly by expressing lymphotoxin β. Our work defines TLS as a potential prognostic factor in glioblastoma and identifies cellular and molecular mechanisms of TLS formation. These findings have broader implications for the development of TLS-inducing therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368cd006ed1ae7b496e449db1b6253ab2d80a3fe" target='_blank'>
              Cellular and molecular events organizing the assembly of tertiary lymphoid structures in glioblastoma
              </a>
            </td>
          <td>
            Alessandra Vaccaro, Fan Yang, Tiarne van de Walle, Saskia Franke, R. Lugano, A. Dénes, Anastasia Magoulopoulou, Inés Cid-Fariña, Anja Smits, Lene Uhrbom, Sylwia Libard, Francesco Latini, Magnus Essand, Mats Nilsson, Liqun He, T. Olsson Bontell, A. Jakola, Mohanraj Ramachandran, A. Dimberg
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, Elisabeth Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Single-cell sequencing technologies have revolutionized biomedical research by enabling deconvolution of cell type-specific properties in highly heterogeneous tissue. While robust tools have been developed to handle bioinformatic challenges posed by single-cell RNA and ATAC data, options for emergent modalities such as methylation are much more limited, impeding the utility of results. Here we present Amethyst, a comprehensive R package for atlas-scale single-cell methylation sequencing data analysis. Amethyst begins with base-level methylation calls and expedites batch integration, doublet detection, dimensionality reduction, clustering, cell type annotation, differentially methylated region calling, and interpretation of results, facilitating rapid data interaction in a local environment. We introduce the workflow using published single-cell methylation human peripheral blood mononuclear cell (PBMC) and human cortex data. We further leverage Amethyst on an atlas-scale brain dataset to discover a noncanonical methylation pattern in human astrocytes and oligodendrocytes, challenging prior assumptions that this form of methylation is only biologically relevant to neurons in the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/345bc3f744d769c3b98a5c6e3d9a3dfc3016fecd" target='_blank'>
              Single-cell DNA methylation analysis tool Amethyst reveals distinct noncanonical methylation patterns in human glial cells
              </a>
            </td>
          <td>
            Lauren E. Rylaarsdam, Ruth V. Nichols, Brendan L O'Connell, Stephen Coleman, G. Yardımcı, Andrew C Adey
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding tumor–host immune interactions and the mechanisms of lung cancer response to immunotherapy is crucial. Current preclinical models used to study this often fall short of capturing the complexities of human lung cancer and lead to inconclusive results. To bridge the gap, we introduce two new murine monoclonal lung cancer cell lines for use in immunocompetent orthotopic models. We demonstrate how our cell lines exhibit immunohistochemical protein expression (TTF-1, NapA, PD-L1) and common driver mutations (KRAS, p53, and p110α) seen in human lung adenocarcinoma patients, and how our orthotopic models respond to combination immunotherapy in vivo in a way that closely mirrors current clinical outcomes. These new lung adenocarcinoma cell lines provide an invaluable, clinically relevant platform for investigating the intricate dynamics between tumor and the immune system, and thus potentially contributes to a deeper understanding of immunotherapeutic approaches to lung cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058e536d48922167d21bebd1081fee6dfb623837" target='_blank'>
              Orthotopic Models Using New, Murine Lung Adenocarcinoma Cell Lines Simulate Human Non-Small Cell Lung Cancer Treated with Immunotherapy
              </a>
            </td>
          <td>
            Eric P. Knott, Emily Kim, Edison Q Kim, Rochelle Freire, Justin A. Medina, Yujie Wang, Cheng-Bang Chen, Chunjing Wu, M. Wangpaichitr, Jose R Conejo-Garcia, Diane C. Lim
          </td>
          <td>2024-06-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Immune cell subsets in the tumor predict prognosis. Identifying reliable subsets consistently in multiple patients is clinically important. What’s more advantageous to the field is if such subsets can be a target of immunotherapy. Here we analyzed single-cell RNA-sequencing datasets of patients with triple-negative breast cancer and identified APOE or FABP5-expressing macrophages correlated with poor prognosis. Further validation with TCGA-BRCA cohort detailed molecular signatures of these macrophages as lipidic and senescent. Our receptor-ligand mapping identified lipidic and senescent macrophages both suppress T-lymphoid and myeloid immunogenicity. Finally, we discovered that immune checkpoint therapy combined with chemotherapy reprogrammed anti-tumor microenvironment enriched with FOLR2+ macrophages facilitated T-cell activations. This suggests that lipidic and senescent macrophages could be a therapeutic target of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da8ec9405ed3a63b54dbe2fe2bb33c7cd24fe33d" target='_blank'>
              Lipidic and senescent macrophages predict progression and response to combinatorial immunotherapy in triple-negative breast cancer
              </a>
            </td>
          <td>
            Chun Lai Chan, Alex To, Shihui Zhang, J. Wong, Yuanhua Huang, Yiming Chao, Ryohichi Sugimura
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Multimodal analysis of single-cell samples from healthy and diseased tissues at various stages provides a comprehensive view that identifies disease-specific cells, their molecular features and aids in patient stratification. Here, we present MultiMIL, a novel weakly-supervised multimodal model designed to construct multimodal single-cell references and prioritize phenotype-specific cells via patient classification. MultiMIL effectively integrates single-cell modalities, even when they only partially overlap, providing robust representations for downstream analyses such as phenotypic prediction and cell prioritization. Using a multiple-instance learning approach, MultiMIL aggregates cell-level measurements into sample-level representations and identifies disease-specific cell states through attention-based scoring. We demonstrate that MultiMIL accurately identifies disease-specific cell states in blood and lung samples, identifying novel disease-associated genes and achieving superior patient classification accuracy compared to existing methods. We anticipate MultiMIL will become an essential tool for querying single-cell multiomic atlases, enhancing our understanding of disease mechanisms and informing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8efc9b9ff38309e496bbcf1bd6af1cf5075840" target='_blank'>
              Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases
              </a>
            </td>
          <td>
            Anastasia Litinetskaya, Maiia Shulman, Soroor Hediyeh-zadeh, A. Moinfar, Fabiola Curion, Artur Szałata, Alireza Omidi, M. Lotfollahi, F. Theis
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Immune cell interactions and metabolic changes are crucial in determining the tumor microenvironment and affecting various clinical outcomes. However, the clinical significance of metabolism evolution of immune cell evolution in colorectal cancer (CRC) remains unexplored. Methods Single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing data were acquired from TCGA and GEO datasets. For the analysis of macrophage differentiation trajectories, we employed the R packages Seurat and Monocle. Consensus clustering was further applied to identify the molecular classification. Immunohistochemical results from AOM and AOM/DSS models were used to validate macrophage expression. Subsequently, GSEA, ESTIMATE scores, prognosis, clinical characteristics, mutational burden, immune cell infiltration, and the variance in gene expression among different clusters were compared. We constructed a prognostic model and nomograms based on metabolic gene signatures identified through the MEGENA framework. Results We found two heterogeneous groups of M2 macrophages with various clinical outcomes through the evolutionary process. The prognosis of Cluster 2 was poorer. Further investigation showed that Cluster 2 constituted a metabolically active group while Cluster 1 was comparatively metabolically inert. Metabolic variations in M2 macrophages during tumor development are related to tumor prognosis. Additionally, Cluster 2 showed the most pronounced genomic instability and had highly elevated metabolic pathways, notably those associated with the ECM. We identified eight metabolic genes (PRELP, NOTCH3, CNOT6, ASRGL1, SRSF1, PSMD4, RPL31, and CNOT7) to build a predictive model validated in CRC datasets. Then, a nomogram based on the M2 risk score improved predictive performance. Furthermore, our study demonstrated that immune checkpoint inhibitor therapy may benefit patients with low-risk. Discussion Our research reveals underlying relationships between metabolic phenotypes and immunological profiles and suggests a unique M2 classification technique for CRC. The identified gene signatures may be key factors linking immunity and tumor metabolism, warranting further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b77333ce5794bf43c82ae5c8985b3dc9293d6dd" target='_blank'>
              M2 Macrophage Classification of Colorectal Cancer Reveals Intrinsic Connections with Metabolism Reprogramming and Clinical Characteristics
              </a>
            </td>
          <td>
            Fengxing Huang, Youwei Wang, Yu Shao, Runan Zhang, Mengting Li, Lan Liu, Qiu-xia Zhao
          </td>
          <td>2024-07-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background The heterogeneity and dynamic changes of endometrial cells have a significant impact on health as they determine the normal function of the endometrium during the menstrual cycle. Dysfunction of the endometrium can lead to the occurrence of various gynecological diseases. Therefore, deconvolution of immune microenvironment that drives transcriptional programs throughout the menstrual cycle is key to understand regulatory biology of endometrium. Methods Herein, we comprehensively analyzed single-cell transcriptome of 59,397 cells across ten human endometrium samples and revealed the dynamic cellular heterogeneity throughout the menstrual cycle. Results We identified two perivascular cell subtypes, four epithelial subtypes and four fibroblast cell types in endometrium. Moreover, we inferred the cell type-specific transcription factor (TF) activities and linked critical TFs to transcriptional output of diverse immune cell types, highlighting the importance of transcriptional regulation in endometrium. Dynamic interactions between various types of cells in endometrium contribute to a range of biological pathways regulating differentiation of secretory. Integration of the molecular biomarkers identified in endometrium and bulk transcriptome of 535 endometrial cancers (EC), we revealed five RNA-based molecular subtypes of EC with highly intratumoral heterogeneity and different clinical manifestations. Mechanism analysis uncovered clinically relevant pathways for pathogenesis of EC. Conclusion In summary, our results revealed the dynamic immune microenvironment of endometrium and provided novel insights into future development of RNA-based treatments for endometriosis and endometrial carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ccb0924331867e645195c52b43129abb7308e5" target='_blank'>
              Mapping Single-Cell Transcriptomes of Endometrium Reveals Potential Biomarkers in Endometrial Cancer
              </a>
            </td>
          <td>
            Gang Xu, Tao Pan, Si Li, Jing Guo, Ya Zhang, Qi Xu, Renwei Chen, Yanlin Ma, Yongsheng Li
          </td>
          <td>2024-07-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72034bf3ceec6cf3d3b2520c3a42de2885dca12a" target='_blank'>
              Tumor necrosis factor receptor 2 promotes endothelial cell-mediated suppression of CD8+ T cells through tuning glycolysis in chemoresistance of breast cancer
              </a>
            </td>
          <td>
            Yu Hu, Xiaohan Lou, Kaili Zhang, Longze Pan, Yue-Yu Bai, Linlin Wang, Ming Wang, Yan Yan, Jiajia Wan, Xiaohan Yao, Xixi Duan, Chen Ni, Zhihai Qin
          </td>
          <td>2024-07-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="As a key factor in tumorigenesis, progression, recurrence and metastasis, the biological properties, metabolic adaptations and immune escape mechanisms of CSCs are the focus of current oncological research. CSCs possess self-renewal, multidirectional differentiation and tumorigenicity, and their mechanisms of action can be elucidated by the clonal evolution, hierarchical model and the dynamic CSCs model, of which the dynamic model is widely recognized due to its better explanation of the function and origin of CSCs. The origin hypothesis of CSCs involves cell-cell fusion, horizontal gene transfer, genomic instability and microenvironmental regulation, which together shape the diversity of CSCs. In terms of classification, CSCs include primary CSCs (pri-CSCs), precancerous stem cells (pre-CSCs), migratory CSCs (mig-CSCs), and chemo-radiotherapy-resistant CSCs (cr-CSCs and rr-CSCs), with each type playing a specific role in tumor progression. Surface markers of CSCs, such as CD24, CD34, CD44, CD90, CD133, CD166, EpCAM, and LGR5, offer the possibility of identifying, isolating, and targeting CSCs, but the instability and heterogeneity of their expression increase the difficulty of treatment. CSCs have adapted to their survival needs through metabolic reprogramming, showing the ability to flexibly switch between glycolysis and oxidative phosphorylation (OXPHOS), as well as adjustments to amino acid and lipid metabolism. The Warburg effect typifies their metabolic profiles, and altered glutamine and fatty acid metabolism further contributes to the rapid proliferation and survival of CSCs. CSCs are able to maintain their stemness by regulating the metabolic networks to maintain their stemness characteristics, enhance antioxidant defences, and adapt to therapeutic stress. Immune escape is another strategy for CSCs to maintain their survival, and CSCs can effectively evade immune surveillance through mechanisms such as up-regulating PD-L1 expression and promoting the formation of an immunosuppressive microenvironment. Together, these properties reveal the multidimensional complexity of CSCs, underscoring the importance of a deeper understanding of the biology of CSCs for the development of more effective tumor therapeutic strategies. In the future, therapies targeting CSCs will focus on precise identification of surface markers, intervention of metabolic pathways, and overcoming immune escape, with the aim of improving the relevance and efficacy of cancer treatments, and ultimately improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d151893cfe9d17714dccbbfae0e7272d874a9" target='_blank'>
              Multidimensional analysis of tumor stem cells: from biological properties, metabolic adaptations to immune escape mechanisms
              </a>
            </td>
          <td>
            Han Han, Ting He, Yingfan Wu, Tianmei He, Weiqiang Zhou
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e281f15c1b3ddbddd2f378f32dbc2fd4db79f0e" target='_blank'>
              Transcriptomic analysis unveils bona fide molecular signatures of microglia under conditions of homeostasis and viral encephalitis
              </a>
            </td>
          <td>
            Felix Mulenge, Olivia Luise Gern, Lena Mareike Busker, Angela Aringo, L. Ghita, I. Waltl, A. Pavlou, Ulrich Kalinke
          </td>
          <td>2024-08-17</td>
          <td>Journal of Neuroinflammation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Starting from one totipotent cell, complex multicellular organisms form through a series of differentiation and morphogenetic events, culminating in a multitude of cell types arranged in a functional and intricate spatial pattern. To do so, cells coordinate with each other, resulting in dynamics which follow a precise developmental trajectory, constraining the space of possible embryo-to-embryo variation. Using recent single-cell sequencing data of early ascidian embryos, we leverage natural variation together with modeling and inference techniques from statistical physics to investigate development at the level of a complete interconnected embryo – an embryonic transcriptome. After developing a robust and biophysically motivated approach to identifying distinct transcriptomic states or cell types, a statistical analysis reveals correlations within embryos and across cell types demonstrating the presence of collective variation. From these intra-embryo correlations, we infer minimal networks of cell-cell interactions, which reveal the collective modes of gene expression. Our work demonstrates how the existence and nature of spatial interactions along with the collective modes of expression that they give rise to can be inferred from single-cell gene expression measurements, opening up a wider range of biological questions that can be addressed using sequencing-based modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917441574a1989eb6aea19dc5d9b06ebec4e3fa8" target='_blank'>
              Physical modeling of embryonic transcriptomes identifies collective modes of gene expression
              </a>
            </td>
          <td>
            Dominic J. Skinner, Patrick Lemaire, Madhav Mani
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f58160d8cdc5bd6e2466eb7c04abf11234ebf3" target='_blank'>
              Patient-derived mini-colons enable long-term modeling of tumor-microenvironment complexity.
              </a>
            </td>
          <td>
            L. F. Lorenzo-Martín, N. Broguiere, Jakob Langer, Lucie Tillard, Mike Nikolaev, George Coukos, K. Homicsko, Matthias P. Lutolf
          </td>
          <td>2024-07-02</td>
          <td>Nature biotechnology</td>
          <td>3</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Development of subtype heterogeneity in breast cancer requires stochastic transitions between cell types. Here, we sought to characterize how NF-κB contributes to breast cancer progression. We built a gene regulatory network model and calibrated it to the expression levels of NF-κB, TWIST1, SIP1, and SLUG in HER2+ and TNBC cell line, using patient and in vitro published data for validation. The model’s epigenetic landscape exhibits two attractor basins separated by a fluctuation-susceptible slow route. NF-κB stochastic fluctuations induce irreversible transitions from HER2+ to TNBC basins at different times, increasing heterogeneity. Mutations altering the availability of NF-κB change the size of the subtype basins, changing the transition probabilities. Our findings enhance the established attractor landscape formulation and deepen understanding of breast cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8337b6fd94c2dc5d7610f3b772efc6d10bcc115" target='_blank'>
              NF-κB Epigenetic Attractor Landscape Drives Breast Cancer Heterogeneity
              </a>
            </td>
          <td>
            Francisco Lopes, B. Pires, Alexandre A. B. Lima, R. Binato, Eliana Abdelhay
          </td>
          <td>2024-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Ovarian and other peritoneal cancers have a strong tendency to metastasize into the surrounding adipose tissue. This study describes an effect of the adipose microenvironment on upregulation of sialic acid-containing glycans in ovarian cancer (OC). Heterogeneous populations of glycosylated OC tumors converged to a highly sialylated cell state that regulates tumorigenesis in an immune-dependent manner. Methods We modeled the adipose microenvironment by conditioning growth media with human patient-derived adipose tissue. OC cell lines grown in the presence vs. absence of adipose conditioned media (ACM) were characterized by transcriptomics, western blotting, and chemical biology glycan labeling methods. Fluorescence-activated cell sorting was used to separate adipose-driven upregulation of hypersialylated (“SNA-high”) vs. hyposialylated (“SNA-low”) OC subpopulations. The two subpopulations were characterized by further transcriptomic and quantitative polymerase chain reaction analyses, then injected into a syngeneic mouse model. Immune system involvement was implicated using wild type and athymic nude mice with a primary endpoint of overall survival. Results Adipose conditioning resulted in upregulation of sialyltransferases ST3GAL1, ST6GAL1, ST6GALNAC3, and ST8Sia1. In culture, OC cells displayed two distinct sialylated subpopulations that were stable for up to 9 passages, suggesting inherent heterogeneity in sialylation that is maintained throughout cell division and media changes. OC tumors that implanted in the omental adipose tissue exclusively reprogrammed to the highly sialylated subpopulation. In wild type C57BL/6 mice, only the hypersialylated SNA-high subpopulation implanted in the adipose, whereas the hyposialylated SNA-low subpopulation failed to be tumorigenic (p=0.023, n=5). In the single case where SNA-low established a tumor, post-mortem analysis revealed reprogramming of the tumor to the SNA-high state in vivo. In athymic nude mice, both subpopulations rapidly formed tumors, implicating a role of the adaptive immune system. Conclusions These findings suggest a model of glycan-dependent tumor evolution wherein the adipose microenvironment reprograms OC to a tumorigenic state that resists the adaptive immune system. Mechanistically, adipose factors upregulate sialyltransferases. To our knowledge, this is the first demonstration of the effect of adipose microenvironment on OC tumor sialylation. Our results set the stage for translational applications targeting sialic acid pathways in OC and other peritoneal cancer tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ade0f16e59c010463533d8ecb04aec54d965a7b" target='_blank'>
              Adipose microenvironment promotes hypersialylation of ovarian cancer cells
              </a>
            </td>
          <td>
            Alexandra Fox, Garry D. Leonard, Nicholas K Adzibolosu, Terrence Wong, Roslyn Tedja, Sapna Sharma, Radhika Gogoi, Robert Morris, Gil Mor, C. Fehl, A. Alvero
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, Jian-Qiang Lu, John S Lazo, T. Kislinger, Yu Lu, Kevin R. Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cellular biomechanics plays critical roles in cancer metastasis and tumor progression. Existing studies on cancer cell biomechanics are mostly conducted in flat 2D conditions, where cells’ behavior can differ considerably from those in 3D physiological environments. Despite great advances in developing 3D in vitro models, probing cellular elasticity in 3D conditions remains a major challenge for existing technologies. In this work, we utilize optical Brillouin microscopy to longitudinally acquire mechanical images of growing cancerous spheroids over the period of eight days. The dense mechanical mapping from Brillouin microscopy enables us to extract spatially resolved and temporally evolving mechanical features that were previously inaccessible. Using an established machine learning algorithm, we demonstrate that incorporating these extracted mechanical features significantly improves the classification accuracy of cancer cells, from 74% to 95%. Building on this finding, we have developed a deep learning pipeline capable of accurately differentiating cancerous spheroids from normal ones solely using Brillouin images, suggesting the mechanical features of cancer cells could potentially serve as a new biomarker in cancer classification and detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872d3dafd293741f9c83632444cfe12e1fdf3df8" target='_blank'>
              Mechanical evolution of metastatic cancer cells in three-dimensional microenvironment
              </a>
            </td>
          <td>
            Karlin Hilai, Daniil Grubich, Marcus Akrawi, Hui Zhu, Razanne Zaghloul, Chenjun Shi, Man Do, Dongxiao Zhu, Jitao Zhang
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Drugs that target immune checkpoint have become the most popular weapon in cancer immunotherapy, yet only have practical benefits for a small percentage of patients. Tumor cells constantly interact with their microenvironment, which is made up of a variety of immune cells as well as endothelial cells and fibroblasts. Immune checkpoint expression and blocked signaling of immune cells in the tumor microenvironment (TME) are key to tumor progression. In this study, we perform deliberation convolution on the TCGA database for human lung, breast, and colorectal cancer to infer crosstalk between immune checkpoint receptors (ICRs) and ligands (ICLs) in TME of pan-carcinogenic solid tumor types, validated by flow cytometry. Analysis of immune checkpoints showed that there was little variation between different tumor types. It showed that CD160, LAG3, TIGIT were found to be highly expressed in CD8+ T cells instead of CD4+ T cells, PD-L1, PD-L2, CD86, LGALS9, TNFRSF14, LILRB4 and other ligands were highly expressed on macrophages, FVR, NECTIN2, FGL1 were highly expressed on Epithelial cells, CD200 was highly expressed in Endothelial cells, and CD80 was highly expressed in CD8 High expression on T cells. Overall, our study provides a new resource for the expression of immune checkpoints in TME on various types of cells. Significance: This study provides immune checkpoint expression of immune cells of multiple cancer types to infer immune mechanisms in the tumor microenvironment and provide ideas for the development of new immune checkpoint-blocking drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610c4b6d799289ca37d42a629ff0e158a4a9bf80" target='_blank'>
              Pan-cancer analysis of immune checkpoint receptors and ligands in various cells in the tumor immune microenvironment.
              </a>
            </td>
          <td>
            Jiahuan Jiang, Yazhang Xu, Di Chen, Jiaxin Li, Xiaoling Zhu, Jun Pan, Leyi Zhang, Pu Cheng, Jian Huang
          </td>
          <td>2024-08-07</td>
          <td>Aging</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding interplay of breast cancer and microenvironment is critical. Here, we identified two transcriptomic subtypes and five immune infiltration patterns from RNA-seq and multiplex immunohistochemistry from 21 ER+/HER2- invasive lobular breast cancers. The proliferative subtype associated with increased immune infiltration especially by immunosuppressive regulatory T-cells and macrophages. We also defined a TAM-Low signature, which associated with lower infiltration of proliferative, pro-inflammatory TAM, and improved outcome in patients with ER+ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13ba7e3cf03c88a995520cfd413c5d66f042dd69" target='_blank'>
              Immune Infiltration Correlates with Transcriptomic Subtypes in Primary ER+ Invasive Lobular Breast Cancer
              </a>
            </td>
          <td>
            Fangyuan Chen, Sayali S Onkar, Jian Zou, Yujia Li, Haley Arbore, Sai Maley, George C. Tseng, Peter Lucas, Tullia Bruno, Dario Vignali, Julia Foldi, Marija Balic, Adrian Lee, S. Oesterreich
          </td>
          <td>2024-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity and tissue transcriptomic complexity. However, the high frequency of dropout events in scRNA-seq data complicates downstream analyses such as cell type identification and trajectory inference. Existing imputation methods address the dropout problem but face limitations such as high computational cost and risk of over-imputation. We present SmartImpute, a novel computational framework designed for targeted imputation of scRNA-seq data. SmartImpute focuses on a predefined set of marker genes, enhancing the biological relevance and computational efficiency of the imputation process while minimizing the risk of model misspecification. Utilizing a modified Generative Adversarial Imputation Network architecture, SmartImpute accurately imputes the missing gene expression and distinguishes between true biological zeros and missing values, preventing overfitting and preserving biologically relevant zeros. To ensure reproducibility, we also provide a function based on the GPT4 model to create target gene panels depending on the tissue types and research context. Our results, based on scRNA-seq data from head and neck squamous cell carcinoma and human bone marrow, demonstrate that SmartImpute significantly enhances cell type annotation and clustering accuracy while reducing computational burden. Benchmarking against other imputation methods highlights SmartImpute’s superior performance in terms of both accuracy and efficiency. Overall, SmartImpute provides a lightweight, efficient, and biologically relevant solution for addressing dropout events in scRNA-seq data, facilitating deeper insights into cellular heterogeneity and disease progression. Furthermore, SmartImpute’s targeted approach can be extended to spatial omics data, which also contain many missing values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6236ffea74eeb68e587d5adb9d7ee59d89926243" target='_blank'>
              SmartImpute: A Targeted Imputation Framework for Single-cell Transcriptome Data
              </a>
            </td>
          <td>
            Sijie Yao, Xiaoqing Yu, Xuefeng Wang
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Introduction Intra-tumoral B cells mediate a plethora of immune effector mechanisms with key roles in anti-tumor immunity and serve as positive prognostic indicators in a variety of solid tumor types, including epithelial ovarian cancer (EOC). Several aspects of intra-tumoral B cells remain unclear, such as their state of activation, antigenic repertoires, and capacity to mature into plasma cells. Methods B lymphocytes were isolated from primary EOC tissue and malignant ascites and were maintained in cell culture medium. The stably maintained cell lines were profiled with flow cytometry and B cell receptor sequencing. Secreted antibodies were tested with a human proteome array comprising more than 21,000 proteins, followed by ELISA for validation. Originating tumor samples were used for spatial profiling with chip cytometry. Results Antibody-secreting B lymphocytes were isolated from the ovarian tumor microenvironment (TME) of four different EOC patients. The highly clonal cell populations underwent spontaneous immortalization in vitro, were stably maintained in an antibody-secreting state, and showed presence of Epstein-Barr viral (EBV) proteins. All originating tumors had high frequency of tumor-infiltrating B cells, present as lymphoid aggregates, or tertiary lymphoid structures. The antigens recognized by three of the four cell lines are coil-coil domain containing protein 155 (CCDC155), growth factor receptor-bound protein 2 (GRB2), and pyruvate dehydrogenase phosphatase2 (PDP2), respectively. Anti-CCDC155 circulating IgG antibodies were detected in 9 of 20 (45%) of EOC patients’ sera. Tissue analyses with multiparameter chip cytometry shows that the antibodies secreted by these novel human B cell lines engage their cognate antigens on tumor cells. Discussion These studies demonstrate that within the tumor-infiltrating lymphocyte population in EOC resides a low frequency population of antibody-secreting B cells that have been naturally exposed to EBV. Once stably maintained, these novel cell lines offer unique opportunities for future studies on intratumor B cell biology and new target antigen recognition, and for studies on EBV latency and/or viral reactivation in the TME of non-EBV related solid tumors such as the EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91568ddf7942710ae6ed5b0bc9abf380b3db7b97" target='_blank'>
              Characterization of latently infected EBV+ antibody-secreting B cells isolated from ovarian tumors and malignant ascites
              </a>
            </td>
          <td>
            Lixin Zhang, Mary Strange, E. Elishaev, Syed Zaidi, F. Modugno, M. Radolec, Robert P Edwards, Olivera J Finn, A. Vlad
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Colorectal cancer (CRC) is among the most hackneyed malignancies. Even patients with identical clinical symptoms and the same TNM stage still exhibit radically different clinical outcomes after receiving equivalent treatment regimens, indicating extensive heterogeneity of CRC. Myriad molecular subtypes of CRC have been exploited for decades, including the most compelling consensus molecular subtype (CMS) classification that has been broadly applied for patient stratification and biomarker‐drug combination formulation. Encountering barriers to clinical translation, however, CMS classification fails to fully reflect inter‐ or intra‐tumor heterogeneity of CRC. As a consequence, addressing heterogeneity and precisely managing CRC patients with unique characteristics remain arduous tasks for clinicians. Review In this review, we systematically summarize molecular subtypes of CRC and further elaborate on their clinical applications, limitations, and future orientations. Conclusion In recent years, exploration of subtypes through cell lines, animal models, patient‐derived xenografts (PDXs), organoids, and clinical trials contributes to refining biological insights and unraveling subtype‐specific therapies in CRC. Therapeutic interventions including nanotechnology, clustered regulatory interspaced short palindromic repeat/CRISPR‐associated nuclease 9 (CRISPR/Cas9), gut microbiome, and liquid biopsy are powerful tools with the possibility to shift the immunologic landscape and outlook for CRC precise medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96246eef67244edae07307aa0c9832bfae113b1" target='_blank'>
              Molecular subtypes of colorectal cancer in the era of precision oncotherapy: Current inspirations and future challenges
              </a>
            </td>
          <td>
            Q. Dang, Lulu Zuo, Xinru Hu, Zhaokai Zhou, Shuang Chen, Shutong Liu, Yuhao Ba, Anning Zuo, Hui-Xian Xu, Siyuan Weng, Yuyuan Zhang, Peng Luo, Q. Cheng, Zaoqu Liu, Xinwei Han
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Gastric cancer (GC) is a major cause of cancer-related mortality worldwide. GC is determined by multiple (epi)genetic and environmental factors; can occur at distinct anatomic positions of the stomach; and displays high heterogeneity, with different cellular origins and diverse histological and molecular features. This heterogeneity has hindered efforts to fully understand the pathology of GC and develop efficient therapeutics. In the past decade, great progress has been made in the study of GC, particularly in molecular subtyping, investigation of the immune microenvironment, and defining the evolutionary path and dynamics. Preclinical mouse models, particularly immunocompetent models that mimic the cellular and molecular features of human GC, in combination with organoid culture and clinical studies, have provided powerful tools for elucidating the molecular and cellular mechanisms underlying GC pathology and immune evasion, and the development of novel therapeutic strategies. Herein, we first briefly introduce current progress and challenges in GC study and subsequently summarize immunocompetent GC mouse models, emphasizing the potential application of genetically engineered mouse models in antitumor immunity and immunotherapy studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa70abd8cf5628d82d68f47668cbf85178b0b2d" target='_blank'>
              Modeling human gastric cancers in immunocompetent mice
              </a>
            </td>
          <td>
            Weihong Zhang, Shilong Wang, Hui Zhang, Yan Meng, Shi Jiao, Liwei An, Zhaocai Zhou
          </td>
          <td>2024-06-28</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3b8fcb2d930ae2a4d496aebe20c735fc43c688" target='_blank'>
              Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping
              </a>
            </td>
          <td>
            Herman Netskar, Aline Pfefferle, J. Goodridge, Ebba Sohlberg, Olli Dufva, Sarah A. Teichmann, D. Brownlie, J. Michaëlsson, N. Marquardt, Trevor Clancy, Amir Horowitz, Karl-Johan Malmberg
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Motivation Spatial proteomics can reveal the spatial organization of immune cells in the tumor immune microenvironment. Relating measures of spatial clustering, such as Ripley’s K or Besag’s L, to patient outcomes may offer important clinical insights. However, these measures require pre-specifying a radius in which to quantify clustering, yet no consensus exists on the optimal radius which may be context-specific. Results We propose a SPatial Omnibus Test (SPOT) which conducts this analysis across a range of candidate radii. At each radius, SPOT evaluates the association between the spatial summary and outcome, adjusting for confounders. SPOT then aggregates results across radii using the Cauchy combination test, yielding an omnibus P-value characterizing the overall degree of association. Using simulations, we verify that the type I error rate is controlled and show SPOT can be more powerful than alternatives. We also apply SPOT to ovarian and lung cancer studies. Availability and implementation An R package and tutorial are provided at https://github.com/sarahsamorodnitsky/SPOT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec30fce6fd25e1c87e31190798a50855db79eb0" target='_blank'>
              A Spatial Omnibus Test (SPOT) for Spatial Proteomic Data
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Antoni Ribas, Michael C. Wu
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary Head and neck cancer disparities in outcomes for Black Americans have been well recognized. However, the specific drivers of the inferior outcomes remain poorly understood. We investigated the biologic features of patient oral cancers and performed a follow-up study of the patient post-surgery recurrences and metastases aiming to explore potential mechanisms that might underpin the poorer outcomes among Black American patients. It was found that high levels of tumor stemness and tumor-promoting stromal characteristics were linked to patient recurrence and metastasis. There were more cases of Black American than White American exhibiting high stemness traits and strong tumor-promoting stromal features associated with tumor recurrences and metastases, although the investigated cases displayed comparable clinical diagnoses. Our findings revealed that the differences in tumor stemness and stromal property among cancers with similar diagnoses contribute to patient outcome disparities. Abstract Black Americans (BAs) with head and neck cancer (HNC) have worse survival outcomes compared to the White patients. While HNC disparities in patient outcomes for BAs have been well recognized, the specific drivers of the inferior outcomes remain poorly understood. Here, we investigated the biologic features of patient tumor specimens obtained during the surgical treatment of oral cancers and performed a follow-up study of the patients’ post-surgery recurrences and metastases with the aim to explore whether tumor biologic features could be associated with the poorer outcomes among BA patients compared with White American (WA) patients. We examined the tumor stemness traits and stromal properties as well as the post-surgery recurrence and metastasis of oral cancers among BA and WA patients. It was found that high levels of tumor self-renewal, invasion, tumorigenesis, metastasis, and tumor-promoting stromal characteristics were linked to post-surgery recurrence and metastasis. There were more BA than WA patients demonstrating high stemness traits and strong tumor-promoting stromal features in association with post-surgery tumor recurrences and metastases, although the investigated cases displayed clinically comparable TNM stages and histological grades. These findings demonstrated that the differences in tumor stemness and stromal property among cancers with comparable clinical diagnoses contribute to the outcome disparity in HNCs. More research is needed to understand the genetic and molecular basis of the biologic characteristics underlying the inferior outcomes among BA patients, so that targeting strategies can be developed to reduce HNC disparity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcad9291ce4667fd7a96a4d38826ce6e7b1c58" target='_blank'>
              Tumor Cell Stemness and Stromal Cell Features Contribute to Oral Cancer Outcome Disparity in Black Americans
              </a>
            </td>
          <td>
            S. Mirshahidi, Isabella J. Yuan, Zhong Chen, Alfred Simental, Steve C. Lee, Pedro A Andrade Filho, Thomas Murry, Feng Zeng, Penelope Duerksen-Hughes, Charles Wang, Xiangpeng Yuan
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over decades, cancer understanding has advanced significantly at molecular and cellular levels, leading to various therapies based on intra-/inter-cellular networks. Despite this, cancer still remains a leading cause of death globally. The primary driver of cancer mortality is metastasis, responsible for about 90% of cancer deaths, due to unclear pathophysiological mechanisms that complicate treatment development. The Notch signaling pathway, a crucial intercellular network in many cancers, has been extensively studied and therapies targeting the Notch pathway also have been well-studied based on inhibiting various stages of Notch activation. On the other hand, Notch signaling’s role varies between cancers; for instance, in non-small cell lung cancer, Notch1 and Notch2 have opposing effects compared to their roles in embryonal brain tumors. In this study, we assumed a scenario of multiple cancers with contrasting Notch signaling pathways and explored optimal targeted therapies for reducing cancer cells by developing two mathematical models with contrasting Notch signaling pathways. The proposed therapies were compared with existing ones, and strategies were investigated to reduce cancer cell numbers for different stage of cancer. We found that that multiple cancers with contrasting Notch networks can be controlled by a common targeted signal network. Combination therapy enhancing Notch production may be most effective in early-stage cancer, while cleavage therapies may be more effective in late-stage cancer. Our study also suggests that optimal treatment should consider the cancer stage, with careful selection and ordering of medication therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0b0aca24e705cffc10843b8484f56eae84233b" target='_blank'>
              Optimal targeted therapy for multiple cancers based on contrastive Notch signaling networks
              </a>
            </td>
          <td>
            Tamaki Wakamoto, S. Seirin-Lee
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="T cell acute lymphoblastic leukemia (T-ALL) is an aggressive leukemia predominantly affecting adolescents and young adults. T-ALL relapses are characterized by chemotherapy resistance, cellular heterogeneity and dismal outcome. To gain a deeper understanding of the cellular heterogeneity and mechanisms driving relapse, we conducted single-cell full-length RNA sequencing of 13 matched pediatric T-ALL patient-derived xenografts (PDX) samples obtained at initial diagnosis and relapse, generating the to date most comprehensive longitudinal single cell study in paired T-ALL samples, along with 5 non-relapsing PDX samples collected at initial diagnosis. This dataset identifies considerable transcriptomic diversity among individual T-ALL cell populations. Notably however, 11 of the 18 patients exhibit a small T-ALL cell subpopulation with a shared set of gene regulatory networks characterized by a common set of active regulons, expression patterns and splice isoforms. This profile involves the upregulation of a stem-like cell signature with enrichment of cell adhesion, and inhibition of the cell cycle and metabolic activity. Comprehensive investigations of these networks identify transcripts enforcing drug resistance through NF-κB expression and TGF-β signaling and of anti-apoptotic T cell signaling. Longitudinal monitoring of these stem-like cells demonstrates this subpopulation to account for only a small proportion of leukemia cells initially with a substantial expansion at relapse suggesting resistance to first line therapy. Chemotherapy resistance is functionally corroborated through in vitro and in vivo drug testing. We thus report the discovery of inherently treatment-resistant stem-like T-ALL cell populations underscoring the potential for devising future therapeutic strategies aimed at targeting stemness pathways in pediatric T-ALL. Key Points Single-cell full-length RNA sequencing reveals dormant, treatment-resistant cells expanding upon T cell acute lymphoblastic leukemia relapse. Stem-like T-ALL cells exhibit a shared gene regulatory network conferring chemotherapy resistance and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88dbfdc32eaada9aa463b5fbbffd8e6808956ec" target='_blank'>
              Role of Stem-Like Cells in Chemotherapy Resistance and Relapse in pediatric T Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Julia Costea, Kerstin K. Rauwolf, Pietro Zafferani, T. Rausch, A. Mathioudaki, Judith B. Zaugg, M. Schrappe, C. Eckert, G. Escherich, Jean-Pierre Bourquin, B. Bornhauser, Andreas E. Kulozik, J. Korbel
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma poses a formidable challenge among primary brain tumors: its tumorigenic stem cells, capable of self-renewal, proliferation, and differentiation, contribute substantially to tumor initiation and therapy resistance. These glioblastoma stem cells (GSCs), resembling conventional stem and progenitor cells, adopt pathways critical for tissue development and repair, promoting uninterrupted tumor expansion. Long non-coding RNAs (lncRNAs), a substantial component of the human transcriptome, have garnered considerable interest for their pivotal roles in normal physiological processes and cancer pathogenesis. They display cell- or tissue-specific expression patterns, and extensive investigations have highlighted their impact on regulating GSC properties and cellular differentiation, thus offering promising avenues for therapeutic interventions. Consequently, lncRNAs, with their ability to exert regulatory control over tumor initiation and progression, have emerged as promising targets for innovative glioblastoma therapies. This review explores notable examples of GSC-associated lncRNAs and elucidates their functional roles in driving glioblastoma progression. Additionally, we delved deeper into utilizing a 3D in vitro model for investigating GSC biology and elucidated four primary methodologies for targeting lncRNAs as potential therapeutics in managing glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ed3ed8d6a1c37f2db5f6f03648c49bfe8a7955" target='_blank'>
              Glioblastoma stem cell long non-coding RNAs: therapeutic perspectives and opportunities
              </a>
            </td>
          <td>
            R. Hazra, Rinku Debnath, Arati Tuppad
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d92421193d9e49ae77ced5b407ab809b8e467e3" target='_blank'>
              Single-cell transcriptomic profiling unveils insights into ovarian fibrosis in obese mice
              </a>
            </td>
          <td>
            Bang Xiao, Zhihui Dai, Zhixuan Li, Dabing Xu, Haozan Yin, Fu Yang, Ningxia Sun
          </td>
          <td>2024-07-02</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hypopharyngeal squamous cell carcinoma (SCC) is a highly aggressive cancer with a poor prognosis, particularly in advanced stages where concurrent chemoradiotherapy (CCRT) is used for treatment. However, resistance to CCRT poses a significant challenge, often leading to treatment failure and disease progression. This study explores the tumor microenvironment (TME) of hypopharyngeal SCC to understand the molecular mechanisms underlying CCRT resistance. Using spatial transcriptomics (ST), we analyzed tissue samples from patients with locally advanced hypopharyngeal SCC, distinguishing between those who were CCRT-resistant and those who were CCRT-naive. The analysis revealed six distinct cellular clusters within the TME, including a prominent epithelio-immune cellular area in CCRT-resistant tissues. SPP1 was identified as a key gene with significantly higher expression in CCRT-resistant samples, specifically within macrophages. Further investigation showed that SPP1+ macrophages interacted with malignant epithelial cells through SPP1-CD44 and SPP1-ITGB1 ligand-receptor pairs. These interactions were primarily localized in the peri-tumoral and intra-tumoral regions, highlighting their potential role in driving CCRT resistance. Our findings suggest that SPP1+ macrophages contribute to the resistant phenotype in hypopharyngeal SCC by modulating the TME and interacting with cancer cells. Understanding these interactions offers valuable insights into the mechanisms of CCRT resistance and may inform the development of targeted therapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d07108a9430fa63556b14b31c6b832a1633f5b4" target='_blank'>
              Spatial transcriptomics unveils landscape of resistance to concurrent chemo-radiotherapy in hypopharyngeal squamous cell carcinoma: the role of SPP1+ macrophages
              </a>
            </td>
          <td>
            J. Ohn, Sungwoo Bae, Hongyoon Choi, In Gul Kim, Kwon Joong Na, Eun-Jae Chung
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) exert clinical efficacy against various types of cancers by reinvigorating exhausted CD8+ T cells that can expand and directly attack cancer cells (cancer-specific T cells) among tumor-infiltrating lymphocytes (TILs). Although some reports have identified somatic mutations in TILs, their effect on antitumor immunity remains unclear. In this study, we successfully established 18 cancer-specific T cell clones, which have an exhaustion phenotype, from the TILs of four patients with melanoma. We conducted whole-genome sequencing for these T cell clones and identified various somatic mutations in them with high clonality. Among the somatic mutations, an SH2D2A loss-of-function frameshift mutation and TNFAIP3 deletion could activate T cell effector functions in vitro. Furthermore, we generated CD8+ T cell-specific Tnfaip3 knockout mice and showed that Tnfaip3 function loss in CD8+ T cell increased antitumor immunity, leading to remarkable response to PD-1 blockade in vivo. In addition, we analyzed bulk CD3+ T cells from TILs in additional 12 patients and identified an SH2D2A mutation in one patient through amplicon sequencing. These findings suggest that somatic mutations in TILs can affect antitumor immunity and suggest unique biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d5ca5b4ab22ea980ea6c4a3574f4132db1aef93" target='_blank'>
              Somatic mutations in tumor-infiltrating lymphocytes impact on antitumor immunity.
              </a>
            </td>
          <td>
            Fumiaki Mukohara, Kazuma Iwata, Takamasa Ishino, Takashi Inozume, J. Nagasaki, Youki Ueda, K. Suzawa, Toshihide Ueno, Hideki Ikeda, Katsushige Kawase, Yuka Saeki, Shusuke Kawashima, Kazuo Yamashita, Yu Kawahara, Yasuhiro Nakamura, Akiko Honobe-Tabuchi, Hiroko Watanabe, H. Dansako, T. Kawamura, Yutaka Suzuki, Hiroaki Honda, Hiroyuki Mano, Shinichi Toyooka, Masahito Kawazu, Yosuke Togashi
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by the selective expansion of hematopoietic stem and progenitor cells (HSPCs) carrying somatic mutations. While CHIP is typically asymptomatic, it has garnered substantial attention due to its association with the pathogenesis of multiple disease conditions, including cardiovascular disease (CVD) and hematological malignancies. In this Review, we will discuss seminal and recent studies that have advanced our understanding of mechanisms that drive selection for mutant HSPCs in the BM niche. Next, we will address recent studies evaluating potential relationships between the clonal dynamics of CHIP and hematopoietic development across the lifespan. Next, we will examine the roles of systemic factors that can influence hematopoietic stem cell (HSC) fitness, including inflammation, and exposures to cytotoxic agents in driving selection for CHIP clones. Furthermore, we will consider how — through their impact on the BM niche — lifestyle factors, including diet, exercise, and psychosocial stressors, might contribute to the process of somatic evolution in the BM that culminates in CHIP. Finally, we will review the role of old age as a major driver of selection in CHIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4403bef9498a4a5c2e26a32490bcbf4e8de6435a" target='_blank'>
              CHIP: a clonal odyssey of the bone marrow niche
              </a>
            </td>
          <td>
            Wolfgang E Schleicher, Bridget Hoag, Marco De Dominici, James DeGregori, Eric Pietras
          </td>
          <td>2024-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251acda7df20c5eff2f9bfbbad717de02554a201" target='_blank'>
              Unlocking the potential of HHLA2: identifying functional immune infiltrating cells in the tumor microenvironment and predicting clinical outcomes in laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenjing Li, Jianqing You, Haixiang Xue, Yi Liu, Junjun Chen, Xiao Zheng, Lujun Chen, Changping Wu
          </td>
          <td>2024-08-06</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/609bb25145d5246600ce4feb000ce7f09c4474b7" target='_blank'>
              Building a translational cancer dependency map for The Cancer Genome Atlas.
              </a>
            </td>
          <td>
            Xu Shi, Christos Gekas, Daniel Verduzco, Sakina M. Petiwala, Cynthia Jeffries, Charles Lu, Erin Murphy, Tifani Anton, Andy H Vo, Zhiguang Xiao, Padmini Narayanan, Bee-Chun Sun, Aloma L D'Souza, J. M. Barnes, Somdutta Roy, Cyril Ramathal, Michael J. Flister, Zoltan Dezso
          </td>
          <td>2024-07-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma remains one of the most lethal solid tumors due to its local aggressiveness and metastatic potential, with a 5-year survival rate of only 13%. A robust connection between pancreatic cancer microenvironment and tumor progression exists, as well as resistance to current anticancer treatments. Pancreatic cancer has a complex tumor microenvironment, characterized by an intricate crosstalk between cancer cells, cancer-associated fibroblasts and immune cells. The complex composition of the tumor microenvironment is also reflected in the diversity of its acellular components, such as the extracellular matrix, cytokines, growth factors and secreted ligands involved in signaling pathways. Desmoplasia, the hallmark of the pancreatic cancer microenvironment, contributes by creating a dense and hypoxic environment that promotes further tumorigenesis, provides innate systemic resistance and suppresses anti-tumor immune invasion. We discuss the complex crosstalk among tumor microenvironment components and explore therapeutic strategies and opportunities in pancreatic cancer research. Better understanding of the tumor microenvironment and its influence on pancreatic cancer progression could lead to potential novel therapeutic options, such as integration of immunotherapy and cytokine-targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5be7a2581240dc6002801cdd72c3d6c7feb416a" target='_blank'>
              Focus on Pancreatic Cancer Microenvironment
              </a>
            </td>
          <td>
            Fabiana Pratticò, I. Garajová
          </td>
          <td>2024-07-26</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction This study investigates the role of Fibroblast Activation Protein (FAP)-positive cancer-associated fibroblasts (FAP+CAF) in shaping the tumor immune microenvironment, focusing on its association with immune cell functionality and cytokine expression patterns. Methods Utilizing immunohistochemistry, we observed elevated FAP+CAF density in metastatic versus primary renal cell carcinoma (RCC) tumors, with higher FAP+CAF correlating with increased T cell infiltration in RCC, a unique phenomenon illustrating the complex interplay between tumor progression, FAP+CAF density, and immune response. Results Analysis of immune cell subsets in FAP+CAF-rich stromal areas further revealed significant correlations between FAP+ stroma and various T cell types, particularly in RCC and non-small cell lung cancer (NSCLC). This was complemented by transcriptomic analyses, expanding the range of stromal and immune cell subsets interrogated, as well as to additional tumor types. This enabled evaluating the association of these subsets with tumor infiltration, tumor vascularization and other components of the tumor microenvironment. Our comprehensive study also encompassed cytokine, angiogenesis, and inflammation gene signatures across different cancer types, revealing heterogeneous cellular composition, cytokine expressions and angiogenic profiles. Through cytokine pathway profiling, we explored the relationship between FAP+CAF density and immune cell states, uncovering potential immunosuppressive circuits that limit anti-tumor activity in tumor-resident immune cells. Conclusions These findings underscore the complexity of tumor biology and the necessity for personalized therapeutic and patient enrichment approaches. The insights gathered from FAP+CAF prevalence, immune infiltration, and gene signatures provide valuable perspectives on tumor microenvironments, aiding in future research and clinical strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340394b70e234f1340a6e340d3b01ec0cd62d31a" target='_blank'>
              Investigating the complex interplay between fibroblast activation protein α-positive cancer associated fibroblasts and the tumor microenvironment in the context of cancer immunotherapy
              </a>
            </td>
          <td>
            A. Kraxner, Franziska Braun, Wei-Yi Cheng, Tai-Hsien Ou Yang, Shweta V. Pipaliya, Marta Cañamero, Emilia Andersson, S. V. Harring, Sebastian Dziadek, Ann-Marie E. Bröske, Maurizio Ceppi, Tamara Tanos, Volker Teichgräber, Jehad Charo
          </td>
          <td>2024-07-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is a multifaceted condition characterized by significant diversity in its molecular and pathological subtypes and clinical manifestation. Despite the progress made in the treatment of DLBCL through the development of novel drugs, an estimated one-third of patients encounter relapse or acquire refractory disease. The tumor microenvironment (TME) of DLBCL, a complex network consisting of cellular and noncellular components that engage in interactions with the tumor, is a parameter that is gaining increasing attention. The TME comprises both the immune and nonimmune microenvironments. The immune microenvironment comprises natural killer (NK) cells, dendritic cells (DCs), tumor-associated macrophages (TAMs), neutrophils, myeloid-derived suppressor cells (MDSCs), and T and B lymphocytes. The nonimmune microenvironment consists of the extracellular matrix (ECM), cancer-associated fibroblasts (CAFs), mesenchymal stromal cells, and other molecules that are secreted. Despite ongoing research, the exact impact of these components and their interaction on the progression of the disease remains elusive. A comprehensive review of significant discoveries concerning the cellular and noncellular constituents, molecular characteristics, and treatment response and prognosis of the TME in DLBCL, as well as the potential targeting of the TME with novel therapeutic approaches, is provided in this article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842f1b179cae9c35039714b759e7f43b1c1c5ae1" target='_blank'>
              Unraveling the Immune Microenvironment in Diffuse Large B-Cell Lymphoma: Prognostic and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            E. Koumpis, A. Papoudou-Bai, K. Papathanasiou, E. Kolettas, Panagiotis Kanavaros, E. Hatzimichael
          </td>
          <td>2024-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e877f4d47a5850d3ceab780a2d2a2472c654fb" target='_blank'>
              High-dimensional single-cell analysis of human natural killer cell heterogeneity
              </a>
            </td>
          <td>
            Lucas Rebuffet, J. Melsen, Bertrand Escalière, Daniela Basurto-Lozada, A. Bhandoola, N. Björkström, Y. Bryceson, R. Castriconi, Frank Cichocki, Marco Colonna, Daniel M Davis, Andreas Diefenbach, Yi Ding, M. Haniffa, Amir Horowitz, Lewis L. Lanier, Karl-Johan Malmberg, Jeffrey S Miller, Lorenzo Moretta, Émilie Narni-Mancinelli, L. A. O'Neill, C. Romagnani, Dylan G Ryan, S. Sivori, Dan Sun, Constance Vagne, Eric Vivier
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>4</td>
          <td>67</td>
        </tr>

        <tr id="Lack of diversity and proportionate representation in genomics datasets and databases contributes to inequity in healthcare outcomes globally1,2. The relationships of human diversity with biological and biomedical phenotypes are pervasive3, yet remain understudied, particularly in a single-cell genomics context. Here we present the Asian Immune Diversity Atlas (AIDA), a multi-national single-cell RNA-sequencing (scRNA-seq) healthy reference atlas of human immune cells. AIDA comprises 1,265,624 circulating immune cells from 619 healthy donors and 6 controls, spanning 7 population groups across 5 countries. AIDA is one of the largest healthy blood datasets in terms of number of cells, and also the most diverse in terms of number of population groups. Though population groups are frequently compared at the continental level, we identified a pervasive impact of sub-continental diversity on cellular and molecular properties of immune cells. These included cell populations and genes implicated in disease risk and pathogenesis as well as those relevant for diagnostics. We detected single-cell signatures of human diversity not apparent at the level of cell types, as well as modulation of the effects of age and sex by self-reported ethnicity. We discovered functional genetic variants influencing cell type-specific gene expression, including context-dependent effects, which were under-represented in analyses of non-Asian population groups, and which helped contextualise disease-associated variants. We validated our findings using multiple independent datasets and cohorts. AIDA provides fundamental insights into the relationships of human diversity with immune cell phenotypes, enables analyses of multi-ancestry disease datasets, and facilitates the development of precision medicine efforts in Asia and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05ebb7ee11200fe5a0a3e32e6a6a32456bcea51e" target='_blank'>
              Single-cell analysis of human diversity in circulating immune cells
              </a>
            </td>
          <td>
            K. Kock, Le Min Tan, Kyung Yeon Han, Yoshinari Ando, Damita Jevapatarakul, Ankita Chatterjee, Quy Xiao Xuan Lin, Eliora Violain Buyamin, Radhika Sonthalia, Deepa Rajagopalan, Yoshihiko Tomofuji, Shvetha Sankaran, Mi-So Park, Mai Abe, Juthamard Chantaraamporn, Seiko Furukawa, Supratim Ghosh, Gyo Inoue, Miki Kojima, Tsukasa Kouno, Jinyeong Lim, K. Myouzen, Sarintip Nguantad, Jin-Mi Oh, N. A. Rayan, S. Sarkar, A. Suzuki, Narita Thungsatianpun, P. Venkatesh, Jonathan Moody, Masahiro Nakano, Ziyue Chen, Chi Tian, Yuntian Zhang, Yihan Tong, C. Tan, Anteneh Mehari Tizazu, Marie Loh, You Yi Hwang, R. Ho, A. Larbi, Tze Pin Ng, Hong-Hee Won, Fred A. Wright, A. Villani, Jong-Eun Park, Murim Choi, Boxiang Liu, Arindam Maitra, Manop Pithukpakorn, B. Suktitipat, K. Ishigaki, Yukinori Okada, Kazuhiko Yamamoto, P. Carninci, John C. Chambers, C. Hon, P. Matangkasombut, Varodom Charoensawan, Partha P. Majumder, Jay W. Shin, Woong-Yang Park, Shyam Prabhakar
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>48</td>
        </tr>

        <tr id="Substantial heterogeneity in molecular features, patient prognoses, and therapeutic responses in head and neck squamous cell carcinomas (HNSCC) highlights the urgent need to develop molecular classifications that reliably and accurately reflect tumor behavior and inform personalized therapy. Here, we leveraged the similarity network fusion bioinformatics approach to jointly analyze multi-omics datasets spanning copy number variations, somatic mutations, DNA methylation, and transcriptomic profiling and derived a prognostic classification system for HNSCC. The integrative model consistently identified three subgroups (IMC1-3) with specific genomic features, biological characteristics, and clinical outcomes across multiple independent cohorts. The IMC1 subgroup included proliferative, immune-activated tumors and exhibited a more favorable prognosis. The IMC2 subtype harbored activated EGFR signaling and an inflamed tumor microenvironment with cancer-associated fibroblast/vascular infiltrations. Alternatively, the IMC3 group featured highly aberrant metabolic activities and impaired immune infiltration and recruiting. Pharmacogenomics analyses from in silico predictions and from patient-derived xenograft model data unveiled subtype-specific therapeutic vulnerabilities including sensitivity to cisplatin and immunotherapy in IMC1 and EGFR inhibitors (EGFRi) in IMC2, which was experimentally validated in patient-derived organoid models. Two signatures for prognosis and EGFRi sensitivity were developed via machine learning. Together, this integrative multi-omics clustering for HNSCC improves current understanding of tumor heterogeneity and facilitates patient stratification and therapeutic development tailored to molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c3e275f858f75f81c9f658d4d03e003b3adc1b" target='_blank'>
              Integrative multi-omics analyses identify molecular subtypes of head and neck squamous cell carcinoma with distinct therapeutic vulnerabilities.
              </a>
            </td>
          <td>
            Pengfei Diao, Yibin Dai, An Wang, Xiaoxuan Bu, Ziyu Wang, Jin Li, Yaping Wu, Hongbing Jiang, Yanling Wang, Jie Cheng
          </td>
          <td>2024-07-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e7e1fc16e696eaffc0fa02c1362c0f5c07aa1" target='_blank'>
              The complex interplay of tumor-infiltrating cells in driving therapeutic resistance pathways
              </a>
            </td>
          <td>
            Dengxiong Li, Fanglin Shao, Qingxin Yu, Rui-cheng Wu, Zhou-ting Tuo, Jie Wang, Luxia Ye, Yiqing Guo, Koo Han Yoo, Mang Ke, U. Okoli, Chaipanichkul Premkamon, Yubo Yang, Wuran Wei, Susan Heavey, William C. Cho, Dechao Feng
          </td>
          <td>2024-08-19</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM), a malignant melanocyte-derived skin cancer, potentially leads to fatal outcomes without effective treatment. The variability in immunotherapy responses among melanoma patients is significantly influenced by the intricate immune microenvironment, particularly due to the status of tumor T cells, encompassing their activity, exhaustion levels, and antigen recognition capabilities. This study utilized single-cell RNA sequencing (scRNA-seq) to analyze 34 melanoma samples from two public datasets (GSE215120 and GSE115978). Herein, we extracted 706 marker genes associated with immune checkpoint (ICP) therapy from these T cells, 509 markers of T cells from 11 melanoma tissues, and eventually identified 33 candidate genes. These genes underwent LASSO and COX regression analyses to identify the signature genes. Of the initial 33 candidate genes, we successfully isolated six distinct T cell-associated immunotherapy-related genes (IRTGs). Additionally, the computation of each patient risk score proved beneficial in evaluating the immune cell infiltration level and functions as an independent prognostic factor for melanoma patient survival. The risk score results revealed promising predictive outcomes in determining the response of melanoma patients to immunotherapy. Notably, our study is the first to reveal the potential correlation between signature gene PEB4B and the immune microenvironment in melaoma, which was explored with multiple immunofluorescence (IF) and Immune Infiltration Assessment. In a conclusion, our findings demonstrate the potential utility of a risk score dependent on signature genes as a predictive tool for assessing the prognosis and response to immunotherapeutic interventions in melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4480d7b0dac6fe343faaffd214e0f795e255eda" target='_blank'>
              The combination of single-cell and RNA sequencing analysis decodes the melanoma tumor microenvironment and identifies novel T cell-associated signature genes
              </a>
            </td>
          <td>
            Sihan Luo, Daiyue Wang, Jiajie Chen, Shaocheng Hong, Yuanyuan Fang, Lu Cao, Liang Yong, Shengxiu Liu
          </td>
          <td>2024-08-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaac037eca190874316bd2b34e72ed4d604d818c" target='_blank'>
              Sulfatide imaging identifies tumor cells in colorectal cancer peritoneal metastases
              </a>
            </td>
          <td>
            G. M. Sarcinelli, L. Varinelli, S. Ghislanzoni, F. Padelli, D. Lorenzini, A. Vingiani, M. Milione, M. Guaglio, S. Kusamura, M. Deraco, G. Pruneri, M. Gariboldi, D. Baratti, I. Bongarzone
          </td>
          <td>2024-06-28</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Background Triple-negative breast cancer (TNBC) poses unique challenges due to its complex nature and the need for more effective treatments. Recent studies showed encouraging outcomes from combining paclitaxel (PTX) with programmed cell death protein-1 (PD-1) blockade in treating TNBC, although the exact mechanisms behind the improved results are unclear. Methods We employed an integrated approach, analyzing spatial transcriptomics and single-cell RNA sequencing data from TNBC patients to understand why the combination of PTX and PD-1 blockade showed better response in TNBC patients. We focused on toll-like receptor 4 (TLR4), a receptor of PTX, and its role in modulating the cross-presentation signaling pathways in tumor-associated macrophages (TAMs) within the tumor microenvironment. Leveraging insights obtained from patient-derived data, we conducted in vitro experiments using immunosuppressive bone marrow-derived macrophages (iBMDMs) to validate if PTX could augment the cross-presentation and phagocytosis activities. Subsequently, we extended our study to an in vivo murine model of TNBC to ascertain the effects of PTX on the cross-presentation capabilities of TAMs and its downstream impact on CD8+ T cell-mediated immune responses. Results Data analysis from TNBC patients revealed that the activation of TLR4 and cross-presentation signaling pathways are crucial for the antitumor efficacy of PTX. In vitro studies showed that PTX treatment enhances the cross-presentation ability of iBMDMs. In vivo experiments demonstrated that PTX activates TLR4-dependent cross-presentation in TAMs, improving CD8+ T cell-mediated antitumor responses. The efficacy of PTX in promoting antitumor immunity was elicited when combined with PD-1 blockade, suggesting a complementary interaction. Conclusions This study reveals how PTX boosts the effectiveness of PD-1 inhibitors in treating TNBC. We found that PTX activates TLR4 signaling in TAMs. This activation enhances their ability to present antigens, thereby boosting CD8+ T cell antitumor responses. These findings not only shed light on PTX’s immunomodulatory role in TNBC but also underscore the potential of targeting TAMs’ antigen presentation capabilities in immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f44d6bfbc504dc1dddbf87d82153d8097bc8bd" target='_blank'>
              Novel insights into paclitaxel’s role on tumor-associated macrophages in enhancing PD-1 blockade in breast cancer treatment
              </a>
            </td>
          <td>
            Yoonjeong Choi, Seong A. Kim, Hanul Jung, Eunhae Kim, Yoon Kyoung Kim, Seohyun Kim, Jaehyun Kim, Yeji Lee, Min Kyoung Jo, Jiwan Woo, Yakdol Cho, Dongjoo Lee, Hongyoon Choi, Cherlhyun Jeong, Gi-Hoon Nam, Mi-Kyung Kwon, In-San Kim
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9965a9579beaaf870afee3260a8af2cf6b7ea8" target='_blank'>
              IL-1R signaling drives enteric glia-macrophage interactions in colorectal cancer
              </a>
            </td>
          <td>
            L. van Baarle, V. De Simone, Linda Schneider, Sneha Santhosh, S. Abdurahiman, F. Biscu, Reiner Schneider, Lisa Zanoletti, Renata Siqueira de Mello, Sara Verbandt, Zedong Hu, M. Stakenborg, B. Ke, N. Stakenborg, Raquel Salvador Laureano, Balbina García-Reyes, Jonas Henn, Marieta Toma, M. Vanmechelen, G. Boeckxstaens, F. De Smet, A. Garg, S. Ibiza, Sabine Tejpar, Sven Wehner, G. Matteoli
          </td>
          <td>2024-07-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinoma (RCC) with histological sarcomatoid de-differentiation (sRCC) has a historically poor prognosis across all RCC subtypes. Recently, immunotherapy with anti-PD1 + anti-CTLA4 combo has significantly improved disease-specific survival, and complete response are seen in upwards of 20% of patients. However, half of sRCC patients still have no response to this immunotherapy, and the mechanism of response remains unclear. Methods Our preliminary bulk RNA sequencing data reveal a distinct tumor microenvironment (TME) in human sRCC tumors, characterized by heightened immune cell infiltration, particularly enriched for regulatory T cells (Tregs). Subsequently, we delineated the spatial gene expression topography within sRCC and clear cell renal cell carcinoma (ccRCC) regions, culminating in the development of the sRCC signature. Concurrently, we established an innovative immunocompetent murine model of sRCC. Leveraging CRISPR technology, we replicated the human sRCC genotype by knocking out Vhl, Bap1, and Cdkn2a/2b genes in a mouse telomerase reverse transcriptase (mTERT) overexpressing renal proximal tubule epithelial cell line. This effort yielded a successfully engineered mouse implantable sRCC cell line bearing VhlMutBap1DelCdkn2aDelCdkn2bDel (HJRCC68N). Results The syngeneic mouse sRCC model we developed faithfully replicates human sRCC histology and tumor microenvironment and recapitulates the sRCC signature identified in human sRCC regions. Importantly, like in humans, mouse sRCC tumors exhibit a heterogeneous response to anti-PD1 + anti-CTLA4 combination, and anti-CTLA4 monotherapy, reflecting the clinical variability observed in human patients. Furthermore, transcriptomic analysis from our mouse model reveals the involvement of type 1 immunity in responder, providing insights into overcoming immunotherapy resistance. Conclusions In conclusion, our study unveiled distinct changes in the TME at the transcriptomic and spatial gene expression levels in human sRCC. Leveraging our novel murine model, which faithfully replicates human sRCC histology, TME characteristics, and mixed response to immunotherapy, we elucidated the involvement of type 1 immunity in treatment response. These insights offer valuable pathways for overcoming immunotherapy resistance and pave the way for the development of tailored treatment strategies aimed at improving outcomes for sRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5fbfe11a82cb5fed55f6641955ed49526e183d8" target='_blank'>
              18 Unraveling the Impact of Tumor Microenvironment on Immunotherapy Response/Resistance in Human Sarcomatoid Clear Cell Renal Cell Carcinoma: Insights from a Novel Mouse Model
              </a>
            </td>
          <td>
            Hui Jiang
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Tumor mutation burden (TMB) and VHL mutation play a crucial role in the management of patients with clear cell renal cell carcinoma (ccRCC), such as guiding adjuvant chemotherapy and improving clinical outcomes. However, the time‐consuming and expensive high‐throughput sequencing methods severely limit their clinical applicability. Predicting intratumoral heterogeneity poses significant challenges in biology and clinical settings. Our aimed to develop a self‐supervised attention‐based multiple instance learning (SSL‐ABMIL) model to predict TMB and VHL mutation status from hematoxylin and eosin‐stained histopathological images. Methods We obtained whole slide images (WSIs) and somatic mutation data of 350 ccRCC patients from The Cancer Genome Atlas for developing SSL‐ABMIL model. In parallel, 163 ccRCC patients from Clinical Proteomic Tumor Analysis Consortium cohort was used as independent external validation set. We systematically compared three different models (Wang‐ABMIL, Ciga‐ABMIL, and ImageNet‐MIL) for their ability to predict TMB and VHL alterations. Results We first identified two groups of populations with high‐ and low‐TMB (cut‐off point = 0.9). In two independent cohorts, the Wang‐ABMIL model achieved the highest performance with decent generalization performance (AUROC = 0.83 ± 0.02 and 0.8 ± 0.04 in predicting TMB and VHL, respectively). Attention heatmaps revealed that the Wang‐ABMIL model paid the highest attention to tumor regions in high‐TMB patients, while in VHL mutation prediction, non‐tumor regions were also assigned high attention, particularly the stromal regions infiltrated by lymphocytes. Conclusions Our results indicated that SSL‐ABMIL can effectively extract histological features for predicting TMB and VHL mutation, demonstrating promising results in linking tumor morphology and molecular biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef0da017ff2911ab8b44c1c00678b46071fd400" target='_blank'>
              Predicting tumor mutation burden and VHL mutation from renal cancer pathology slides with self‐supervised deep learning
              </a>
            </td>
          <td>
            Qingyuan Zheng, Xinyu Wang, Rui Yang, Junjie Fan, Jingping Yuan, Xiuheng Liu, Lei Wang, Zhuoni Xiao, Zhiyuan Chen
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oral squamous cell carcinomas (OSCC), the predominant head and neck cancer, pose significant challenges due to late-stage diagnoses and low five-year survival rates. Spatial transcriptomics offers a promising avenue to decipher the genetic intricacies of OSCC tumor microenvironments. In spatial transcriptomics, Cell-type deconvolution is a crucial inferential goal; however, current methods fail to consider the high zero-inflation present in OSCC data. To address this, we develop a novel zero-inflated version of the hierarchical generalized transformation model (ZI-HGT) and apply it to the Conditional AutoRegressive Deconvolution (CARD) for cell-type deconvolution. The ZI-HGT serves as an auxiliary Bayesian technique for CARD, reconciling the highly zero-inflated OSCC spatial transcriptomics data with CARD’s normality assumption. The combined ZI-HGT + CARD framework achieves enhanced cell-type deconvolution accuracy and quantifies uncertainty in the estimated cell-type proportions. We demonstrate the superior performance through simulations and analysis of the OSCC data. Furthermore, our approach enables the determination of the locations of the diverse fibroblast population in the tumor microenvironment, critical for understanding tumor growth and immunosuppression in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/090b060bd9dd44f20620bc9cc982e8c6875f2751" target='_blank'>
              A Zero-Inflated Hierarchical Generalized Transformation Model to Address Non-Normality in Spatially-Informed Cell-Type Deconvolution
              </a>
            </td>
          <td>
            Hunter J. Melton, Jonathan R. Bradley, Chong Wu
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Single-cell RNA sequencing (scRNA-seq) has helped to elucidate the cellular composition of cancer and its microenvironment. Recent scRNA-seq studies have highlighted the heterogeneity of glioblastoma (GBM). Moreover, single-cell GBM analyses have proposed resemblance of GBM cells to radial glia and outer radial glia supporting the hypothesis that remnants of developmental tissue get reactivated in cancer. A recent study isolated neural progenitor cells (NPCs) from developing fetal human brain (gestational week 17–19) and classified NPCs based on their expression of THY1, CD24 and EGFR. Ventricular radial glia are THY1-CD24-EGFR+ whereas outer radial glia are THY1-CD24-EGFR-. Early neuron precursors are CD24+THY1-EGFR+ and glial progenitor cells (GPCs) are THY1+EGFR+. GPCs give rise to THY1+EGFR+PDGFRA+ pre-oligodendrocyte progenitor cells. The importance of EGFR in NPCs again highlights the resemblance to glioma. Methods We aimed to apply the classification above in IDH mutant astrocytoma and oligodendroglioma as well as IDHwt glioblastoma samples. We used three publicly available datasets: Wang (paired 74 IDHwt primary and recurrent samples), Tirosh (6 primary oligodendroglioma samples) and Venteicher (10 primary IDH mutant astrocytoma). Results In IDH mutant astrocytoma, 82.63% of cells express THY1+ (mostly EGFR+PDGFRA+) and 10.76% of cells are THY1-CD24-EGFR+. In oligodendroglioma, 75% of cells are THY1+ (mostly EGFR+PDGFRA+) and 12.07% are THY1-CD24-EGFR+. In IDHwt EGFR amplified primary GBM samples, 87.5% of cells are THY1-CD24-EGFR+. This percentage drops to 70.4% in the recurrent setting. THY1-CD24-EGFR-cells increase from 9.7% to 23.1% at recurrence. In IDHwt EGFRwt primary GBM samples, 48.6% of cells are THY1-CD24-EGFR+ and 44.15% are THY1-CD24-EGFR-. In the recurrent setting, 43.26% of cells are THY1-CD24-EGFR+ and 49.58% are THY1-CD24-EGFR-. Conclusion IDH mutant gliomas and IDHwt glioblastoma express different progenitor cell markers. THY1 is highly expressed in IDH mutant gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84519f9c31ef63e22be1710cc56f409181b65fe3" target='_blank'>
              The single-cell make-up of adult diffuse glioma based on the 2021 WHO classification
              </a>
            </td>
          <td>
            I. Alnahhas, Allison Kayne, Mehak M. Khan, Wenyin Shi
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58322ca71db2c109e772f2eb0f78a110f1a84ce4" target='_blank'>
              Senescent endothelial cells promote liver metastasis of uveal melanoma in single-cell resolution
              </a>
            </td>
          <td>
            Liang Ma, Xiaoyu He, Yidian Fu, Shengfang Ge, Zhi Yang
          </td>
          <td>2024-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Glioblastoma is a highly aggressive brain cancer and, unlike many other cancers types, the median survival for patients after treatment (14.6 months) has barely improved in the last 20 years. Infiltrative growth into the surrounding brain parenchyma facilitates tumor recurrence and ultimately the death of the patient - novel therapies targeting this process are desperately needed. Lysyl oxidase inhibition has been shown to decrease invasive growth in a variety of solid tumours and is a potential therapy for glioblastoma patients. Methods Genes highly expressed in the mesenchymal subtype of glioblastoma were analyzed in a data set from the Cancer Genome Atlas and tissue microarrays. Two patient-derived human glioblastoma stem cell lines were used to assess the involvement of lysyl oxidase (LOX). The effect of LOX on infiltration was examined in an organotypic brain slice assay and in an orthotopic mouse model. Chemotactic assays, protease and cleavage arrays were used to assess the underlying mechanism behind LOX-mediated infiltration. The orthotopic model was used to evaluate potential clinical utility of targeting LOX in glioblastoma. Results LOX is overexpressed in the mesenchymal glioblastoma subtype and strongly associated with poor patient survival. LOX expression upregulates MMP7 expression, which subsequently cleaves the vascular matrix resulting in increased chemotaxis of glioblastoma cells. Conclusions We have uncovered a novel mechanism of glioblastoma infiltration and suggest that targeting LOX represent an effective therapeutic approach blocking glioblastoma infiltration. Importance of the study The ability of glioblastoma cells to infiltrate the surrounding normal brain tissue facilitates their evasion of current therapies, leading to tumor recurrence and ultimately the death of the patient. To improve targeted therapies for glioblastoma patients we need to understand the molecular mechanisms of glioblastoma cell infiltration and how cells interact with the unique microenvironment of the brain. We have identified a novel mechanism whereby tumor-derived LOX mediates chemotaxis of glioblastoma cells to the laminin rich perivascular niche, enabling infiltrative growth. Inhibiting this infiltrative pathway is a potential anti-invasive therapy that is desperately needed for glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb3e6c468837c0b0bfb72d57ecf46fe10ba327b" target='_blank'>
              Cleavage of the vascular matrix attracts glioblastoma cells to infiltrate the brain parenchyma
              </a>
            </td>
          <td>
            Lara Perryman, Anette M. Høye, T. Cox, Ann-Marie Baker, Jan E. Strøbech, Lidia Leonte, Lukram B Singh, Sergey Popov, Lasse G Lorentzen, Raphael Reuten, Hans Skovgaard Poulsen, Michael J Davies, Chris Jones, J. Erler
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Modest response rates to immunotherapy observed in advanced lung cancer patients underscore the need to identify reliable biomarkers and targets, enhancing both treatment decision-making and efficacy. Factors such as PD-L1 expression, tumor mutation burden, and a ‘hot’ tumor microenvironment with heightened effector T cell infiltration have consistently been associated with positive responses. In contrast, the predictive role of the abundantly present tumor-infiltrating myeloid cell (TIMs) fraction remains somewhat uncertain, partly explained by their towering variety in terms of ontogeny, phenotype, location, and function. Nevertheless, numerous preclinical and clinical studies established a clear link between lung cancer progression and alterations in intra- and extramedullary hematopoiesis, leading to emergency myelopoiesis at the expense of megakaryocyte/erythroid and lymphoid differentiation. These observations affirm that a continuous crosstalk between solid cancers such as lung cancer and the bone marrow niche (BMN) must take place. However, the BMN, encompassing hematopoietic stem and progenitor cells, differentiated immune and stromal cells, remains inadequately explored in solid cancer patients. Subsequently, no clear consensus has been reached on the exact breadth of tumor installed hematopoiesis perturbing cues nor their predictive power for immunotherapy. As the current era of single-cell omics is reshaping our understanding of the hematopoietic process and the subcluster landscape of lung TIMs, we aim to present an updated overview of the hierarchical differentiation process of TIMs within the BMN of solid cancer bearing subjects. Our comprehensive overview underscores that lung cancer should be regarded as a systemic disease in which the cues governing the lung tumor-BMN crosstalk might bolster the definition of new biomarkers and druggable targets, potentially mitigating the high attrition rate of leading immunotherapies for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09f4d97d42cb2976ecd4a13f91ebddf956195779" target='_blank'>
              The crosstalk between lung cancer and the bone marrow niche fuels emergency myelopoiesis
              </a>
            </td>
          <td>
            Evelyn Calderon-Espinosa, K. De Ridder, Thomas Benoot, Y. Jansen, Domien Vanhonacker, Robbe Heestermans, A. De Becker, Ivan Van Riet, Lore Decoster, C. Goyvaerts
          </td>
          <td>2024-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain. Methods: Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package “Seurat” was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes. Results: Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA. Conclusion: A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tissue-resident memory T cells (TRM) are a specialized subset of long-lived memory T cells that reside in peripheral tissues. However, the impact of TRM-related immunosurveillance on the tumor-immune microenvironment (TIME) and tumor progression across various non-small-cell lung cancer (NSCLC) patient populations is yet to be elucidated. Our comprehensive analysis of multiple independent single-cell and bulk RNA-seq datasets of patient NSCLC samples generated reliable, unique TRM signatures, through which we inferred the abundance of TRM in NSCLC. We discovered that TRM abundance is consistently positively correlated with CD4+ T helper 1 cells, M1 macrophages, and resting dendritic cells in the TIME. In addition, TRM signatures are strongly associated with immune checkpoint and stimulatory genes and the prognosis of NSCLC patients. A TRM-based machine learning model to predict patient survival was validated and an 18-gene risk score was further developed to effectively stratify patients into low-risk and high-risk categories, wherein patients with high-risk scores had significantly lower overall survival than patients with low-risk. The prognostic value of the risk score was independently validated by the Cancer Genome Atlas Program (TCGA) dataset and multiple independent NSCLC patient datasets. Notably, low-risk NSCLC patients with higher TRM infiltration exhibited enhanced T-cell immunity, nature killer cell activation, and other TIME immune responses related pathways, indicating a more active immune profile benefitting from immunotherapy. However, the TRM signature revealed low TRM abundance and a lack of prognostic association among lung squamous cell carcinoma patients in contrast to adenocarcinoma, indicating that the two NSCLC subtypes are driven by distinct TIMEs. Altogether, this study provides valuable insights into the complex interactions between TRM and TIME and their impact on NSCLC patient prognosis. The development of a simplified 18-gene risk score provides a practical prognostic marker for risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1e637bee54083c5c2fb1deaba9e34a97001048" target='_blank'>
              A comprehensive meta-analysis of tissue resident memory T cells and their roles in shaping immune microenvironment and patient prognosis in non-small cell lung cancer
              </a>
            </td>
          <td>
            Aidan Shen, Aliesha Garrett, Cheng-Chi Chao, Dongliang Liu, Chao Cheng, Zhaohui Wang, Chen Qian, Yangzhi Zhu, Junhua Mai, Chongming Jiang
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this study, we explore the possibility of inferring characteristics of the tumor-immune microenvironment (TIME) from the blood. Specifically, we investigate two datasets of head and neck squamous cell carcinoma (HNSCC) patients with matched scRNA-Seq from peripheral blood mononuclear cells (PBMCs) and tumor tissues. Our analysis shows that the immune cell fractions and gene expression profiles of various immune cells within the tumor microenvironment can be inferred from the matched PBMC scRNA-Seq data. We find that the established exhausted T-cell signature can be predicted from the blood and serve as a valuable prognostic blood biomarker of immunotherapy response. Additionally, our study reveals that the inferred ratio between tumor memory B and regulatory T cell fractions is predictive of immunotherapy response and is superior to the well-established cytolytic and exhausted T-cell signatures. These results highlight the promising potential of PBMC scRNA-Seq in cancer immunotherapy and warrant, and will hopefully facilitate, further investigations on a larger scale. The code for predicting tumor immune microenvironment from PBMC scRNA-Seq, TIMEP, is provided, offering other researchers the opportunity to investigate its prospective applications in various other indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ade53f587544d91914fb929dd7efe4a6e7cd50" target='_blank'>
              Inferring characteristics of the tumor-immune microenvironment of HNSCC patients from single-cell transcriptomics of peripheral blood.
              </a>
            </td>
          <td>
            Yingying Cao, Tiangen Chang, Fiorella Schischlik, Kun Wang, Sanju Sinha, Sridhar Hannenhalli, Peng Jiang, E. Ruppin
          </td>
          <td>2024-08-08</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in the epidermal growth factor receptor (EGFR) are common in non-small cell lung cancer (NSCLC), particularly in never-smoker patients. However, these mutations are not always carcinogenic, and have recently been reported in histologically normal lung tissue from patients with and without lung cancer. To investigate the outcome of EGFR mutation in healthy lung stem cells, we grow murine alveolar type II organoids monoclonally in a three-dimensional Matrigel. Our experiments show that the EGFR-L858R mutation induces a change in organoid structure: mutated organoids display more ‘budding’, in comparison with non-mutant controls, which are nearly spherical. We perform on-lattice computational simulations, which suggest that this can be explained by the concentration of division among a small number of cells on the surface of the mutated organoids. We are currently unable to distinguish the cell-based mechanisms that lead to this spatial heterogeneity in growth, but suggest a number of future experiments which could be used to do so. We suggest that the likelihood of L858R-fuelled tumorigenesis is affected by whether the mutation arises in a spatial environment that allows the development of these surface protrusions. These data may have implications for cancer prevention strategies and for understanding NSCLC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e046dae2eb0578408d47ec2942e59fd61d21fe2" target='_blank'>
              An agent-based modelling framework to study growth mechanisms in EGFR-L858R mutant cell alveolar type II cells
              </a>
            </td>
          <td>
            Helena Coggan, C. Weeden, Philip Pearce, M. Dalwadi, Alastair Magness, C. Swanton, Karen M. Page
          </td>
          <td>2024-07-01</td>
          <td>Royal Society Open Science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cancer cells show remarkable plasticity and can switch lineages in response to the tumor microenvironment. Cellular plasticity drives invasiveness and metastasis and helps cancer cells to evade therapy by developing resistance to radiation and cytotoxic chemotherapy. Increased understanding of cell fate determination through epigenetic reprogramming is critical to discover how cancer cells achieve transcriptomic and phenotypic plasticity. Glioblastoma is a perfect example of cancer evolution where cells retain an inherent level of plasticity through activation or maintenance of progenitor developmental programs. However, the principles governing epigenetic drivers of cellular plasticity in glioblastoma remain poorly understood. Here, using machine learning (ML) we employ cross-patient prediction of transcript expression using a combination of epigenetic features (ATAC-seq, CTCF ChIP-seq, RNAPII ChIP-seq, H3K27Ac ChIP-seq, and RNA-seq) of glioblastoma stem cells (GSCs). We investigate different ML and deep learning (DL) models for this task and build our final pipeline using XGBoost. The model trained on one patient generalizes to another one suggesting that the epigenetic signals governing gene transcription are consistent across patients even if GSCs can be very different. We demonstrate that H3K27Ac is the epigenetic feature providing the most significant contribution to cross-patient prediction of gene expression. In addition, using H3K27Ac signals from patients-derived GSCs, we can predict gene expression of human neural crest stem cells suggesting a shared developmental epigenetic trajectory between subpopulations of these malignant and benign stem cells. Our cross-patient ML/DL models determine weighted patterns of influence of epigenetic marks on gene expression across patients with glioblastoma and between GSCs and neural crest stem cells. We propose that broader application of this analysis could reshape our view of glioblastoma tumor evolution and inform the design of new epigenetic targeting therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f06de9705bc503e591aae9a45d9fab127b04d79" target='_blank'>
              Machine learning on multiple epigenetic features reveals H3K27Ac as a driver of gene expression prediction across patients with glioblastoma
              </a>
            </td>
          <td>
            Yusuke Suita, Hardy Bright, Yuan Pu, M. D. Toruner, Jordan Idehen, Nikos Tapinos, Ritambhara Singh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The global incidence of cancer is increasing, with estimates suggesting that there will be 26 million new cases and 17 million deaths per year by 2030. Cancer stem cells (CSCs) and extracellular vesicles (EVs) are key to the resistance and advancement of cancer. They play a crucial role in tumor dynamics and resistance to therapy. CSCs, initially discovered in acute myeloid leukemia, are well-known for their involvement in tumor initiation, progression, and relapse, mostly because of their distinct characteristics, such as resistance to drugs and the ability to self-renew. EVs, which include exosomes, microvesicles, and apoptotic bodies, play a vital role in facilitating communication between cells within the tumor microenvironment (TME). They have a significant impact on cellular behaviors and contribute to genetic and epigenetic changes. This paper analyzes the mutually beneficial association between CSCs and EVs, emphasizing their role in promoting tumor spread and developing resistance mechanisms. This review aims to investigate the interaction between these entities in order to discover new approaches for attacking the complex machinery of cancer cells. It highlights the significance of CSCs and EVs as crucial targets in the advancement of novel cancer treatments, which helps stimulate additional research, promote progress in ideas for cancer treatment, and provide renewed optimism in the effort to reduce the burden of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b17e9728aa76af6a02ab4a44623aab4c0608b6" target='_blank'>
              Extracellular vesicles and cancer stem cells: a deadly duo in tumor progression
              </a>
            </td>
          <td>
            Akram Tayanloo-Beik, Azin Eslami, Masoumeh Sarvari, H. Jalaeikhoo, Mohsen Rajaeinejad, Mohsen Nikandish, Ali Faridfar, Mostafa Rezaei-Tavirani, A. R. Mafi, Bagher Larijani, B. Arjmand
          </td>
          <td>2024-07-18</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ability to genetically edit primary B cells via CRISPR/Cas9 technology represents a powerful tool to study molecular mechanisms of B ‐ cell pathogenesis. In this context, employing ribonucleoprotein complexes (RNPs), formed by recombinant Cas9 and genome ‐ targetting single guide RNA molecules, brings in advantage of accelerated set ‐ up and protocol robustness. Gene editing via RNP electroporation has been recently applied to primary tumor cells isolated from patients chronic lymphocytic leukemia (CLL), suggesting an efficient and valuable tool for studying leukemic cell biology and biomarker validation. 1,2 The work by Nardi et al. on this topic proposed to electroporate unmanipulated primary CLL cells that are subsequently put in culture with human CD40L ‐ expressing fibroblasts and soluble stimuli to promote CLL cell proliferation. In this context, cellular proliferation is required to achieve homozygous gene editing, whereas in unstimulated CLL cells it is possible to achieve only the heterozygous editing. 1 The method published by Mateos ‐ Jaimez et al. relies on the preactivation of CLL cells with CD40L/BAFF/IL ‐ 21 ‐ expressing stromal cells, followed by RNP electroporation and continuation of the stimulatory coculture. 2 Both methods approach 80% – 90% of editing efficiency and allow to perform downstream in vitro experiments on edited leukemic cells. Application of a similar RNP ‐ based editing approach to the widely used murine">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce4b4f9dae608c0b0ba780ba2b6869894794e80" target='_blank'>
              Robust and cost‐effective CRISPR/Cas9 gene editing of primary tumor B cells in Eµ‐TCL1 model of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Rosita Del Prete, Roberta Drago, Federica Nardi, Gaia Bartolini, Erika Bellini, Antonella De Rosa, Silvia Valensin, Anna Kabanova
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03562858743a9cb635c7fdc4ef5409e1ded279d8" target='_blank'>
              Single-cell atlas profiling revealed cellular characteristics and dynamic changes after PD-1 blockade therapy of brain metastases from laryngeal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Yunzhi Zou, H. Duan, Zekun Deng, Rong Xiang, Jixiang Zhao, Zhenhua Zhang, Wanming Hu, Yuanzhong Yang, Zeming Yan, Shujuan Wen, Zexian Liu, Gao Zhang, Y. Mou, Depei Li, Xiaobing Jiang
          </td>
          <td>2024-08-01</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 What are the molecular and cellular features that contribute to the transformation of myometrial stem cells into tumor-initiating cells within uterine leiomyoma?



 Transformative changes in differentiation and genetic processes, in a distinct subset of M/LM cells drives their progression into a tumorigenic state.



 Uterine leiomyomas (LM), which are benign smooth muscle-derived tumours of myometrium, impact approximately 75% of women, causing significant physical and psychological challenges and imposing substantial healthcare costs. The widespread consensus postulates a monoclonal origin for LM, implying their derivation from a dysregulated single multipotent stem cell that could give rise to tumor-initiating cells (TICs). While previous studies have shown cellular heterogeneity in both myometrium (M) and LM, the precise identity of the originating cells is unknown.



 Prospective, observational, and biomedical study conducted at Hospital La Fe (Valencia, Spain) for one year. Eight sample pairs of LM and M underwent single-nuclei RNAseq (snRNA-seq; n = 16) and single-cell proteomic (scP; n = 16) analyses, to generate a detailed transcriptomic and proteomic map decoupled from cell type, state, and spatial location. We further employed single-cell RNA velocity inference for differentiation trajectory [BR1] analysis of mesenchymal cells, followed by the identification of potential driver genes in LM tumorigenesis.



 Upon obtaining informed consent, we procured LM and M samples from eight patients aged 35-50 undergoing hysterectomies. Part of these samples were preserved in paraffin for spatial transcriptomics utilizing VISIUM (10x Genomics). The remaining tissues were dissociated into single-cell suspensions and underwent scRNA-seq and scP using Chromium Controller and Orbitrap Eclipse Tribid mass spectrometry, respectively. The analysis of the collected data was carried out using publicly accessible R/Python tools.



 Analysis of snRNAseq (∼52,000 cells) revealed similar cellular compositions in M and LM, including endothelial, perivascular, smooth muscle cells (SMC), fibroblasts, and immune cells. Further trajectory analysis unveiled a subset of myometrial cells with potential as TICs, exhibiting reduced expression of hormone receptors like PGR. Analysis of these cells [BR1] suggested differentiation into two distinct cell types -fibroblasts and SMCs- with elevated hormone expression over pseudotime. Differential analysis between M and LM also showed a dysregulated transcriptomic profile in the TICs within the tumor, as evidenced by upregulation of RAD51B and HMGA2 and downregulation PGR among other genes. These genes are indicative markers, underscoring their significant roles in the tumorigenesis of LM by influencing hormone response and disrupting DNA repair mechanisms. Spatial transcriptomic analysis confirmed these findings and suggested that the distribution of TICs could be ubiquitous, since the expression of potential myometrial-stem markers, was markedly increased only on certain spots within the tissue across both LM and M. Lastly, scProt profiling (∼5,000 cells) also allowed the identification of putative TICs, which showed upregulation in LM of specific proteins involved in replication, transcription, and translation.



 Our study sets the basis for TICs isolation in LM, yet functional validation studies need to be performed to address the clinical potential of our findings. Further studies including more patients, and addressing racial disparities will help to generalize these findings to a broader population.



 Our findings indicate the presence of a particular subset of M/LM cells whose changes in differentiation path and genetic processes seem to drive the transformation of these cells into a tumorigenic state to develop LM. Targeting these markers might show promise for effectively treating these tumors.



 NCT04214457
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e5f1c7a81c7b6baceb7a49d4c94f2c7bde8e8c" target='_blank'>
              O-264 Profiling Tumor-Initiating Cells in Uterine Leiomyoma through Single-Cell Multi-omics
              </a>
            </td>
          <td>
            A. Machado-Lopez, P. Punzón-Jiménez, J. Llera-Oyola, R. Perez-Moraga, M. Gálvez-Viedma, D. Grases, E. Satorres, E. Porta-Pardo, E. Schooff, B. Roson, C. Simon, A. Mas
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
Classic Hodgkin lymphoma (cHL) is a B cell lymphoma that occurs primarily in young adults and, less frequently, in elderly individuals. A hallmark of cHL is the exceptional scarcity (1-5%) of the malignant Hodgkin Reed-Sternberg (HRS) cells within a network of non-malignant immune cells. Molecular determinants governing the relationship between HRS cells and their proximal microenvironment remain largely unknown.


EXPERIMENTAL DESIGN
We performed spatially resolved multiplexed protein imaging and transcriptomic sequencing to characterize HRS cell states, cellular neighborhoods, and gene expression signatures of 23.6 million cells from 36 newly diagnosed Epstein-Barr virus (EBV) positive and EBV-negative cHL tumors.


RESULTS
We show that MHC-I expression on HRS cells is associated with immune inflamed neighborhoods containing CD8+ T cells, MHC-II+ macrophages, and immune checkpoint expression (i.e., PD-1 and VISTA). We identified spatial clustering of HRS cells, consistent with the syncytial variant of cHL, and its association with T cell excluded neighborhoods in a subset of EBV-negative tumors. Finally, a subset of both EBV-positive and EBV-negative tumors contained regulatory T cells high neighborhoods harboring HRS cells with augmented proliferative capacity.


CONCLUSIONS
Our study links HRS cell properties with distinct immunophenotypes and potential immune escape mechanisms in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/645b8738be3b7a5dce2bff2635b7f7c7780b894d" target='_blank'>
              Multiplexed spatial profiling of Hodgkin Reed-Sternberg cell neighborhoods in classic Hodgkin lymphoma.
              </a>
            </td>
          <td>
            Maryam Pourmaleki, Caitlin J Jones, Sabrina D Mellinghoff, Bryan D. Greenstein, Priyadarshini Kumar, Miguel Foronda, Daniel A Navarrete, Carl Campos, M. Roshal, N. Schultz, Sohrab P. Shah, A. Schietinger, N. Socci, Travis J Hollmann, A. Dogan, I. Mellinghoff
          </td>
          <td>2024-07-01</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Motivation Recently developed spatial lineage tracing technologies induce somatic mutations at specific genomic loci in a population of growing cells and then measure these mutations in the sampled cells along with the physical locations of the cells. These technologies enable high-throughput studies of developmental processes over space and time. However, these applications rely on accurate reconstruction of a spatial cell lineage tree describing both past cell divisions and cell locations. Spatial lineage trees are related to phylogeographic models that have been well-studied in the phylogenetics literature. We demonstrate that standard phylogeographic models based on Brownian motion are inadequate to describe the spatial symmetric displacement (SD) of cells during cell division. Results We introduce a new model—the SD model for cell motility that includes symmetric displacements of daughter cells from the parental cell followed by independent diffusion of daughter cells. We show that this model more accurately describes the locations of cells in a real spatial lineage tracing of mouse embryonic stem cells. Combining the spatial SD model with an evolutionary model of DNA mutations, we obtain a phylogeographic model for spatial lineage tracing. Using this model, we devise a maximum likelihood framework—MOLLUSC (Maximum Likelihood Estimation Of Lineage and Location Using Single-Cell Spatial Lineage tracing Data)—to co-estimate time-resolved branch lengths, spatial diffusion rate, and mutation rate. On both simulated and real data, we show that MOLLUSC accurately estimates all parameters. In contrast, the Brownian motion model overestimates spatial diffusion rate in all test cases. In addition, the inclusion of spatial information improves accuracy of branch length estimation compared to sequence data alone. On real data, we show that spatial information has more signal than sequence data for branch length estimation, suggesting augmenting lineage tracing technologies with spatial information is useful to overcome the limitations of genome-editing in developmental systems. Availability and Implementation The python implementation of MOLLUSC is available at https://github.com/raphael-group/MOLLUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93b1c0eaf33d8e0ce3646746a6a5177fc9543531" target='_blank'>
              Maximum likelihood phylogeographic inference of cell motility and cell division from spatial lineage tracing data
              </a>
            </td>
          <td>
            Uyen Mai, Gary Hu, Benjamin J. Raphael
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/228f6ee044df6edc918c18217ff468ea90e78c6b" target='_blank'>
              EMBER creates a unified space for independent breast cancer transcriptomic datasets enabling precision oncology
              </a>
            </td>
          <td>
            Carlos Ronchi, Syed Haider, C. Brisken
          </td>
          <td>2024-07-09</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac24e0e7a39e90599e20a198363cbf479bb3749d" target='_blank'>
              Using a pan-cancer atlas to investigate tumour associated macrophages as regulators of immunotherapy response
              </a>
            </td>
          <td>
            Alexander Coulton, Jun Murai, Danwen Qian, Krupa Thakkar, Claire E Lewis, Kevin Litchfield
          </td>
          <td>2024-07-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70504aeb9b8ca67b917dcc2474590fbeb0bd285d" target='_blank'>
              Single-cell transcriptomic analysis reveals a decrease in the frequency of macrophage-RGS1high subsets in patients with osteoarticular tuberculosis
              </a>
            </td>
          <td>
            Ying Jiang, Xinqiang Zhang, Bo Wang, Liping Tang, Xin Liu, Xiudong Ding, Yueming Dong, Hong Lei, Di Wang, Huicheng Feng
          </td>
          <td>2024-08-10</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For triple-negative breast cancer (TNBC), the most aggressive subset of breast cancer, immune cell infiltrates have prognostic implications. The presence of myeloid-derived suppressor cells supports tumor progression, while tumor-infiltrating lymphocytes (TILs) correlate with improved survival and responsiveness to immunotherapy. Manipulating the abundance of these populations may enhance tumor immunity. Gemcitabine (GEM), a clinically employed chemotherapeutic, is reported to be systemically myeloablative, and thus it is a potentially useful adjunct therapy for promoting anti-tumor immunity. However, knowledge about the immunological effects of GEM intratumorally is limited. Thus, we directly compared the impact of systemic GEM on immune cell presence and functionality in the tumor microenvironment (TME) to its effects in the periphery. We found that GEM is not myeloablative in the TME; rather, we observed sustained, significant reductions in TILs and dendritic cells—crucial components in initiating an adaptive immune response. We also performed bulk-RNA sequencing to identify immunological alterations transcriptionally induced by GEM. While we found evidence of upregulation in the interferon-gamma (IFN-γ) response pathway, we determined that GEM-mediated growth control is not dependent on IFN-γ. Overall, our findings yield new insights into the tissue- and temporal-dependent immune ablative effects of GEM, contrasting the paradigm that this therapy is specifically myeloablative.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8b369c3cc0cc0f15b653495833002d9af4d298" target='_blank'>
              Tissue- and Temporal-Dependent Dynamics of Myeloablation in Response to Gemcitabine Chemotherapy
              </a>
            </td>
          <td>
            Lydia E. Kitelinger, E. A. Thim, Sarah Y. Zipkowitz, Richard J. Price, Timothy N. J. Bullock
          </td>
          <td>2024-08-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, Philip Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b7af48216a0c84346ea77c18dc911a1e93afbf" target='_blank'>
              Endothelial cell heterogeneity in colorectal cancer: tip cells drive angiogenesis
              </a>
            </td>
          <td>
            Zhenyu Xie, Liaoran Niu, Kunli Du, Ling Chen, Gaozan Zheng, Songchen Dai, Han-Long Dan, Lili Duan, Xinyu Dou, Fan Feng, Jian Zhang, Jianyong Zheng
          </td>
          <td>2024-08-22</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small RNAs that are often dysregulated in many diseases, including cancers. They are highly tissue-specific and stable, thus, making them particularly useful as biomarkers. As the spatial transcriptomics field advances, protocols that enable highly sensitive and spatially resolved detection become necessary to maximize the information gained from samples. This is especially true of miRNAs where the location their expression within tissue can provide prognostic value with regard to patient outcome. Equally as important as detection are ways to assess and visualize the miRNA's spatial information in order to leverage the power of spatial transcriptomics over that of traditional nonspatial bulk assays. We present a highly sensitive methodology that simultaneously quantitates and spatially detects seven miRNAs in situ on formalin-fixed paraffin-embedded tissue sections. This method utilizes rolling circle amplification (RCA) in conjunction with a dual scanning approach in nanoliter well arrays with embedded hydrogel posts. The hydrogel posts are functionalized with DNA probes that enable the detection of miRNAs across a large dynamic range (4 orders of magnitude) and a limit of detection of 0.17 zeptomoles (1.7 × 10-4 attomoles). We applied our methodology coupled with a data analysis pipeline to K14-Cre Brca1f/fTp53f/f murine breast tumors to showcase the information gained from this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbaad333d7c14fb2504c61497f5a898eaa1ab830" target='_blank'>
              Sensitive Multiplexed MicroRNA Spatial Profiling and Data Classification Framework Applied to Murine Breast Tumors.
              </a>
            </td>
          <td>
            Omar N Mohd, Y. Heng, Lin Wang, Abhishek Thavamani, Erica S. Massicott, Gerburg M Wulf, Frank J Slack, Patrick S. Doyle
          </td>
          <td>2024-07-23</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9c5b5959b41a250a5ef58adb37088e34a07297" target='_blank'>
              Immuno-oncologic profiling of pediatric brain tumors reveals major clinical significance of the tumor immune microenvironment
              </a>
            </td>
          <td>
            Adrian B Levine, L. Nobre, Anirban Das, Scott Milos, V. Bianchi, Monique Johnson, Nick Fernandez, L. Stengs, S. Ryall, Michelle Ku, Mansuba Rana, Benjamin Laxer, J. Sheth, Stefanie-Grace Sbergio, Ivana Fedorakova, Vijay Ramaswamy, J. Bennett, R. Siddaway, U. Tabori, Cynthia Hawkins
          </td>
          <td>2024-07-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Motivation Cell–cell interactions (CCIs) consist of cells exchanging signals with themselves and neighboring cells by expressing ligand and receptor molecules and play a key role in cellular development, tissue homeostasis, and other critical biological functions. Since direct measurement of CCIs is challenging, multiple methods have been developed to infer CCIs by quantifying correlations between the gene expression of the ligands and receptors that mediate CCIs, originally from bulk RNA-sequencing data and more recently from single-cell or spatially resolved transcriptomics (SRT) data. SRT has a particular advantage over single-cell approaches, since ligand–receptor correlations can be computed between cells or spots that are physically close in the tissue. However, the transcript counts of individual ligands and receptors in SRT data are generally low, complicating the inference of CCIs from expression correlations. Results We introduce Copulacci, a count-based model for inferring CCIs from SRT data. Copulacci uses a Gaussian copula to model dependencies between the expression of ligands and receptors from nearby spatial locations even when the transcript counts are low. On simulated data, Copulacci outperforms existing CCI inference methods based on the standard Spearman and Pearson correlation coefficients. Using several real SRT datasets, we show that Copulacci discovers biologically meaningful ligand–receptor interactions that are lowly expressed and undiscoverable by existing CCI inference methods. Availability and implementation Copulacci is implemented in Python and available at https://github.com/raphael-group/copulacci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7018d88732147a5c725f1a06701187686f85ae3c" target='_blank'>
              A count-based model for delineating cell–cell interactions in spatial transcriptomics data
              </a>
            </td>
          <td>
            Hirak Sarkar, Uthsav Chitra, Julian Gold, Benjamin J. Raphael
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Abstract Head and neck squamous cell carcinoma (HNSCC) is a global cancer burden with a 5‐year overall survival rate of around 50%, stagnant for decades. A tumour‐induced immunosuppressive microenvironment contributes to HNSCC progression, with the adenosine (ADO) pathway and an upregulated expression of inhibitory immune checkpoint regulators playing a key role in this context. The correlation between high neutrophil‐to‐lymphocyte ratio (NLR) with advanced tumour staging suggests involvement of neutrophils (NØ) in cancer progression. Interestingly, we associated a high NLR with an increased intracellular PD‐L1 localization in primary HNSCC samples, potentially mediating more aggressive tumour characteristics and therefore synergistically favouring tumour progression. Still, further research is needed to harness this knowledge for effective treatments and overcome resistance. Since it is hypothesized that the tumour microenvironment (TME) may be influenced by small extracellular vesicles (sEVs) secreted by tumours (TEX), this study aims to investigate the impact of HNSCC‐derived TEX on NØ and blockade of ADO receptors as a potential strategy to reverse the pro‐tumour phenotype of NØ. UMSCC47‐TEX exhibited CD73 enzymatic activity involved in ADO signalling, as well as the immune checkpoint inhibitor PD‐L1. Data revealed that TEX induce chemotaxis of NØ and the sustained interaction promotes a shift into a pro‐tumour phenotype, dependent on ADO receptors (P1R), increasing CD170high subpopulation, CD73 and PD‐L1 expression, followed by an immunosuppressive secretome. Blocking A3R reduced CD73 and PD‐L1 expression. Co‐culture experiments with HNSCC cells demonstrated that TEX‐modulated NØ increase the CD73/PD‐L1 axis, through Cyclin D‐CDK4/6 signalling. To support these findings, the CAM model with primary tumour was treated with NØ supernatant. Moreover, these NØ promoted an increase in migration, invasion, and reduced cell death. Targeting P1R on NØ, particularly A3R, exhibited potential therapeutic strategy to counteract immunosuppression in HNSCC. Understanding the TEX‐mediated crosstalk between tumours and NØ offers insights into immunomodulation for improving cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f83d69cf6f6cf4f49191e11cc12ba0dc0477471" target='_blank'>
              The immunomodulatory ballet of tumour‐derived extracellular vesicles and neutrophils orchestrating the dynamic CD73/PD‐L1 pathway in cancer
              </a>
            </td>
          <td>
            Dominique S. Rubenich, Jordana L Domagalski, Gabriela F S Gentil, Jonas Eichberger, Mathias Fiedler, Florian Weber, Marianne Federlin, H. Poeck, Torsten E Reichert, T. Ettl, Richard J Bauer, Elizandra Braganhol, D. Schulz
          </td>
          <td>2024-07-01</td>
          <td>Journal of Extracellular Vesicles</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinomas (RCC) are characterized by their largely diverse clinical outcomes. Leveraging genomic diversity among individual tumors, our research during the past decade has concentrated on developing personalized medicine approaches for the most common and the most aggressive type of RCC, clear cell renal carcinoma (ccRCC). We generated the largest dataset of ccRCC including somatic genomic and clinical annotations for over 940 ccRCC patients, and developed a genomic classifier, based on mutational status of 12 RCC-relevant genes, which is able to stratify patients according to their risk of relapse after nephrectomy and/or death due to RCC. Furthermore, we noticed the presence of mesenchymal-like cellular phenotypes in tumors of high-risk patients, representsing a de-differentiation process, known as sarcomatoic or rhabdoid (S/R) de-differentation. Methods The current knowledge about molecular mechanisms that may drive S/R de-differentiation is primarily generated from molecular profiling of bulk tumors, collecting data from a mixture of cells that co-exist in the tumor milieu. Therefore, high-resolution studies that precisely define molecular characteristics of different cellular phenotypes associated with S/R features are missing. We have used spatial transcriptomic profiling to investigate molecular mechanisms that underline S/R de-differentationin RCC. While preserving tissue context, we applied spatial whole human transcriptome profiling to areas exhibiting S/R or clear cell phenotypes within the same tumor specimens to generate phenotype-specific transcriptome profiles. In addition, we applied whole-exome sequencing (WES) to independent areas that exhibit different differentiation phenotype withing same tumors. Results Pathway and network analysis of genesets with upregulation in each area have revealed meaningful differences in cellular pathways which are active in each phenotype. Specifically, we observed that dysregulation of extracellular matrix (ECM) is a hallmark of S/R areas within RCC tumors. Furthermore, WES of phenotypically-distinct areas within each tumor has shed new light son the genome evoltuion of S/R phenotypes. Conclusions S/R dedifferentiation in RCC tumors is characterized by specific genomic evolutionary patterns and substantial dysregulation of ECM components. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad0d8a5a2c80baecbe7aa769327da96aacd5a05" target='_blank'>
              9 Delving into molecular underpinnings of sarcomatoid/rhabdoid dedifferentioation in renal cancers using spatial profiling
              </a>
            </td>
          <td>
            Y. Riazalhosseini
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Most melanomas progress from radial to vertical growth phase before spreading locoregionally and distally. Much is still unknown about the metabolic changes in the tumor cells and their microenvironment during this metastatic progression. We aimed to gain new insight into the molecular characteristics of melanoma in regard to spatial lipidomics to deliver new knowledge regarding tumor metastatic progression. We included 10 fresh tumor samples from 10 patients including two in situ melanomas, two invasive primary melanomas, and six metastatic melanomas (four in-transit metastases and two distant metastases). In addition, we analyzed four healthy skin controls from the same patients. Time-of-flight imaging secondary ion mass spectrometry (ToF-SIMS) enabled detailed spatial-lipidomics that could be directly correlated with conventional histopathological analysis of consecutive H&E-stained tissue sections. Significant differences in the lipid profiles were found in primary compared to metastatic melanomas, notably an increase in phosphatidylethanolamine lipids relative to phosphatidylinositol lipids and an increase in GM3 gangliosides in the metastatic samples. Furthermore, analysis of the data from in transit versus distant metastases samples highlighted that specific phospholipids, and a difference in the long versus shorter chain GM3 gangliosides, discriminated the metastatic routes. Further studies are warranted to verify these preliminary findings. Lipidomic changes could serve as a novel biomarker for tumor progression and even serve as a target for novel treatments. Furthermore, analyzing the lipid profiles could help to differentiate between primary and metastatic melanomas in challenging cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd5da01fb93ce9422a464070818f23116949e9f" target='_blank'>
              ToF-SIMS imaging reveals changes in tumor cell lipids during metastatic progression of melanoma.
              </a>
            </td>
          <td>
            N. Neittaanmäki, O. Zaar, Kevin Sjögren Cehajic, Kelly Dimovska Nilsson, D. Katsarelias, R. O. Bagge, John Paoli, John S Fletcher
          </td>
          <td>2024-06-29</td>
          <td>Pigment cell & melanoma research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf603a0e9cde601ac9006c72d111f317f71de81" target='_blank'>
              Single-cell profiling reveals altered immune landscape and impaired NK cell function in gastric cancer liver metastasis.
              </a>
            </td>
          <td>
            Xiaolong Tang, Lei Gao, Xingzhi Jiang, Zhenyu Hou, Yiwen Wang, Shiyang Hou, Hui Qu
          </td>
          <td>2024-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="PURPOSE
Co-occurring mutations in KEAP1 and STK11KRAS have emerged as determinants of survival outcomes in non-small cell lung cancer (NSCLC) patients treated with immunotherapy. However, these mutational contexts identify a fraction of non-responders to immune checkpoint inhibitors. We hypothesized that KEAP1 wild-type tumors recapitulate the transcriptional footprint of KEAP1 mutations, and that this KEAPness phenotype can determine immune responsiveness with higher precision compared to mutation-based models.


EXPERIMENTAL DESIGN
The TCGA was used to infer the KEAPness phenotype and explore its immunological correlates at the pan-cancer level. The association between KEAPness and survival outcomes was tested in two independent cohorts of advanced NSCLC patients treated with immunotherapy and profiled by RNA-Seq (SU2C n=153; OAK/POPLAR n=439). The NSCLC TRACERx421 multi-region sequencing study (tumor regions n=947) was used to investigate evolutionary trajectories.


RESULTS
KEAPness-dominant tumors represented 50% of all NSCLCs and were associated with shorter progression-free survival (PFS) and overall survival (OS) compared to KEAPness-free cases in independent cohorts of NSCLC patients treated with immunotherapy (SU2C PFS P=0.042, OS P=0.008; OAK/POPLAR PFS P=0.0014, OS P<0.001). Patients with KEAPness tumors had survival outcomes comparable to those with KEAP1-mutant tumors. In the TRACERx421, KEAPness exhibited limited transcriptional intratumoral heterogeneity and immune exclusion, resembling the KEAP1-mutant disease. This phenotypic state occurred across genetically divergent tumors, exhibiting shared and private cancer genes under positive selection when compared to KEAP1-mutant tumors.


CONCLUSIONS
We identified a KEAPness phenotype across evolutionary divergent tumors. KEAPness outperforms mutation-based classifiers as a biomarker of inferior survival outcomes in NSCLC patients treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea8af08e6c600cf8a24f58a7981fb66121f005e" target='_blank'>
              Transcriptional phenocopies of deleterious KEAP1 mutations correlate with survival outcomes in lung cancer treated with immunotherapy.
              </a>
            </td>
          <td>
            S. Scalera, B. Ricciuti, D. Marinelli, M. Mazzotta, Laura Cipriani, Giulia Bon, Giulia Schiavoni, I. Terrenato, Alessandro Di Federico, J. Alessi, Maurizio Fanciulli, L. Ciuffreda, F. de Nicola, F. Goeman, G. Caravagna, Daniele Santini, R. De Maria, F. Cappuzzo, G. Ciliberto, M. Jamal-Hanjani, Mark M Awad, N. Mcgranahan, M. Maugeri-Saccà
          </td>
          <td>2024-07-09</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, M. Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Gastric cancer (GC) is characterized by significant intratumoral heterogeneity, and stem cells are promising therapeutic targets. Despite advancements in spatial transcriptome analyses, unexplored targets for addressing cancer stemness remain unknown. This study aimed to identify Nuclear Factor IX (NFIX) as a critical regulator of cancer stemness in GC and evaluate its clinicopathological significance and function. Spatial transcriptome analysis of GC was conducted. The correlation between NFIX expression, clinicopathological factors, and prognosis was assessed using immunostaining in 127 GC cases. Functional analyses of cancer cell lines validated these findings. Spatial transcriptome analysis stratified GC tissues based on genetic profiles, identified CSC-like cells, and further refined the classification to identify and highlight the significance of NFIX, as validated by Monocle 3 and CytoTRACE analyses. Knockdown experiments in cancer cell lines have demonstrated the involvement of NFIX in cancer cell proliferation and kinase activity. This study underscores the role of spatial transcriptome analysis in refining GC tissue classification and identifying therapeutic targets, highlighting NFIX as a pivotal factor. NFIX expression is correlated with poor prognosis and drives GC progression, suggesting its potential as a novel therapeutic target for personalized GC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f19e2ccd74245f8115ff0d1d1d57356414aa99d" target='_blank'>
              Discovering cancer stem-like molecule, nuclear factor I X, using spatial transcriptome in gastric cancer.
              </a>
            </td>
          <td>
            Akira Ishikawa, Takafumi Fukui, Aya Kido, Narutaka Katsuya, K. Kuraoka, N. Uraoka, Takahisa Suzuki, Shiro Oka, Takahiro Kotachi, Hassan Ashktorab, D. Smoot, Wataru Yasui
          </td>
          <td>2024-07-17</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="The genome is partitioned into distinct chromatin compartments with at least two main classes, a transcriptionally active A and an inactive B compartment, corresponding mostly to the segregation of euchromatin and heterochromatin. Chromatin within the same compartment has a higher tendency to interact with itself than with regions in opposing compartments. A/B compartments are traditionally derived from ensemble Hi-C contact matrices through principal component analysis of their covariance matrices. However, defining compartments in single cells from single-cell Hi-C maps is non trivial due to sparsity of the data and the fact that homologous copies are typically not resolved. Here we present an unsupervised approach, named MaxComp, to determine single-cell A/B compartments from geometric considerations in 3D chromosome structures, either from multiplexed FISH imaging or from models derived from Hi-C data. By representing each single-cell structure as an undirected graph with edge-weights encoding structural information, the problem of predicting chromosome compartments can be transformed to an alternative form of the Max-cut problem, a semidefinite graph programming method (SPD) to determine an optimal division of a chromosome structure graph into two structural compartments. Our results show that compartment annotations from principal component analysis of ensemble Hi-C data can be perfectly reproduced as population averages of our single-cell compartment predictions. We therefore prove that compartment predictions can be achieved from geometric considerations alone using 3D coordinates of chromatin regions together with information about their nuclear microenvironment. Our results reveal substantial cell-to-cell heterogeneity of compartments in a cell population, which substantially differs between individual genomic regions. Moreover, by applying our approach to multiplexed FISH tracing experiments, our method sheds light on the relationship between single-cell compartment annotations and gene transcriptional activity in single cells. Overall our approach provides new insights into single-cell chromatin condensation, relationship between population and single-cell chromatin compartmentalization, the cell-to-cell variations of chromatin compartments and its impact on gene transcription. Author Summary Chromosome conformation capture and imaging techniques revealed the segregation of genomic chromatin into at least two functional compartments. Hi-C contact frequency matrices show checkerboard-like patterns indicating that chromatin regions are divided into at least two states, possibly a result of phase separation. Chromatin regions in the same state have preferential interactions with each other, often over extended sequence distances, while interactions to regions in the opposing state are minimized. Principal component analysis (PCA) on ensemble Hi-C contact frequency matrices can identify these compartment states. However, because the compartment annotations are derived from a cell population, this method cannot provide information about compartments in single cells. Here in this study, we introduce an unsupervised method to predict single-cell compartments using graph-based programming, which utilizes only structural information in single cells. Our results demonstrate that PCA-based ensemble compartment annotations can be reproduced as population averages of our single-cell compartment predictions. Moreover, our results reveal the cell-to-cell heterogeneity of compartments in a cell population, which shows significant disparities among different chromatin regions. Moreover, by applying our approach to multiplexed FISH tracing experiments, our method reveals the relationship between single-cell compartment annotations and gene transcriptional activity in single cells. Finally, our approach also allows us to relate chromatin structural features in single cells with compartment properties. Comparison with other existing approaches showed that our method produces overall better compartmentalization scores in single cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5153cd191eb013027c48e1d56d079848585fef6" target='_blank'>
              Prediction of single-cell chromatin compartments from single-cell chromosome structures by MaxComp
              </a>
            </td>
          <td>
            Yuxiang Zhan, Francesco Musella, Frank Alber
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer brain metastasis has a poor prognosis, is commonly observed in clinical practice, and the number of cases is increasing as overall cancer survival improves. However, experiments in mouse models have shown that brain metastasis itself is an inefficient process. One reason for this inefficiency is the brain microenvironment, which differs significantly from that of other organs, making it difficult for cancer cells to adapt. The brain microenvironment consists of unique resident cell types such as neurons, oligodendrocytes, astrocytes, and microglia. Accumulating evidence over the past decades suggests that the interactions between cancer cells and glial cells can positively or negatively influence the development of brain metastasis. Nevertheless, elucidating the complex interactions between cancer cells and glial cells remains challenging, in part due to the limitations of existing experimental models for glial cell culture. In this review, we first provide an overview of glial cell culture methods and then examine recent discoveries regarding the interactions between brain metastatic cancer cells and the surrounding glial cells, with a special focus on astrocytes and microglia. Finally, we discuss future perspectives for understanding the multifaceted interactions between cancer cells and glial cells for the treatment of metastatic brain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b93951608ce8fdc53c244e85b3946f02850ef4e" target='_blank'>
              Multifaceted interactions between cancer cells and glial cells in brain metastasis.
              </a>
            </td>
          <td>
            Kojiro Ishibashi, E. Hirata
          </td>
          <td>2024-07-11</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Uveal melanoma (UM) is the most common intraocular tumor in adults, and nearly 50% of patients develop metastatic disease with a high mortality rate. Therefore, the development of relevant preclinical in vivo models that accurately recapitulate the metastatic cascade is crucial. We exploited the chick embryo chorioallantoic membrane (CAM) xenograft model to quantify both experimental and spontaneous metastasis by qPCR analysis. Our study found that the transplanted UM cells spread predominantly and early in the liver, reflecting the primary site of metastasis in patients. Visible signs of pigmented metastasis were observed in the eyes, liver, and distal CAM. Lung metastases occurred rarely and brain metastases progressed more slowly. However, UM cell types of different origins and genetic profiles caused an individual spectrum of organ metastases. Metastasis to multiple organs, including the liver, was often associated with risk factors such as high proliferation rate, hyperpigmentation, and epithelioid cell type. The severity of liver metastasis was related to the hepatic metastatic origin and chromosome 8 abnormalities rather than monosomy 3 and BAP1 deficiency. The presented CAM xenograft model may prove useful to study the metastatic potential of patients or to test individualized therapeutic options for metastasis in different organs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f8730b08ae8b6d195c62a52a5e4d930fdafe25" target='_blank'>
              The Chick Chorioallantoic Membrane as a Xenograft Model for the Quantitative Analysis of Uveal Melanoma Metastasis in Multiple Organs
              </a>
            </td>
          <td>
            Hongtao Liu, Theodora Tsimpaki, Ralitsa Anastasova, N. E. Bechrakis, M. Fiorentzis, U. Berchner‐Pfannschmidt
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cells must adopt flexible regulatory strategies to make decisions regarding their fate, including differentiation, apoptosis, or survival in the face of various external stimuli. One key cellular strategy that enables these functions is stochastic gene expression programs. However, understanding how transcriptional bursting, and consequently, cell fate, responds to DNA damage on a genome-wide scale poses a challenge. In this study, we propose an interpretable and scalable inference framework, DeepTX, that leverages deep learning methods to connect mechanistic models and scRNA-seq data, thereby revealing genome-wide transcriptional burst kinetics. This framework enables rapid and accurate solutions to transcription models and the inference of transcriptional burst kinetics from scRNA-seq data. Applying this framework to several scRNA-seq datasets of DNA-damaging drug treatments, we observed that fluctuations in transcriptional bursting induced by different drugs could lead to distinct fate decisions: IdU treatment induces differentiation in mouse embryonic stem cells by increasing the burst size of gene expression, while 5FU treatment with low and high dose increases the burst frequency of gene expression to induce cell apoptosis and survival in human colon cancer cells. Together, these results show that DeepTX can be used to analyze single-cell transcriptomics data and can provide mechanistic insights into cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08e79a79f41500c7b1baabfdd652bb03e9ecc3e" target='_blank'>
              Deep learning linking mechanistic models to single-cell transcriptomics data reveals transcriptional bursting in response to DNA damage
              </a>
            </td>
          <td>
            Zhiwei Huang, Songhao Luo, Zihao Wang, Zhenquan Zhang, Benyuan Jiang, Qing Nie, Jiajun Zhang
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background
Intrahepatic cholangiocarcinoma (ICC) is a highly desmoplastic tumor with poor prognosis even after curative resection. We investigated the associations between the composition of the ICC stroma and immune cell infiltration and aimed to develop a stromal-immune signature to predict prognosis in surgically treated ICC.


Patients and methods
We recruited 359 ICC patients and performed immunohistochemistry to detect α-smooth muscle actin (α-SMA), CD3, CD4, CD8, Foxp3, CD68, and CD66b. Aniline was used to stain collagen deposition. Survival analyses were performed to detect prognostic values of these markers. Recursive partitioning for a discrete-time survival tree was applied to define a stromal-immune signature with distinct prognostic value. We delineated an integrated stromal-immune signature based on immune cell subpopulations and stromal composition to distinguish subgroups with different recurrence-free survival (RFS) and overall survival (OS) time.


Results
We defined four major patterns of ICC stroma composition according to the distributions of α-SMA and collagen: dormant (α-SMAlow/collagenhigh), fibrogenic (α-SMAhigh/collagenhigh), inert (α-SMAlow/collagenlow), and fibrolytic (α-SMAhigh/collagenlow). The stroma types were characterized by distinct patterns of infiltration by immune cells. We divided patients into six classes. Class I, characterized by high CD8 expression and dormant stroma, displayed the longest RFS and OS, whereas Class VI, characterized by low CD8 expression and high CD66b expression, displayed the shortest RFS and OS. The integrated stromal-immune signature was consolidated in a validation cohort.


Conclusion
We developed and validated a stromal-immune signature to predict prognosis in surgically treated ICC. These findings provide new insights into the stromal-immune response to ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38208917a49e6f4d85e269cabc0575476c4f3e3e" target='_blank'>
              Development and Validation of a Stromal-Immune Signature to Predict Prognosis in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yu-Hang Ye, Haoyang Xin, Jia-Li Li, Ning Li, Si-yuan Pan, Long Chen, Jing-Yue Pan, Zhi-qiang Hu, Peng-Cheng Wang, Chubin Luo, Rongrong Sun, Jia Fan, Jian Zhou, Zheng-jun Zhou, Shaolai Zhou
          </td>
          <td>2024-08-06</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Isolation of tumor-specific T cells and their antigen receptors (TCRs) from malignant pleural effusions (MPE) may facilitate the development of TCR-transduced adoptive cellular immunotherapy products for advanced lung cancer patients. However, the characteristics and markers of tumor-specific T-cells in MPE are largely undefined. To this end, to establish the phenotypes and antigen specificities of CD8+ T cells, we performed single-cell RNA and TCR sequencing of samples from three advanced lung cancer patients. Dimensionality reduction on a total of 4,983 CD8+ T cells revealed 10 clusters including naïve, memory, and exhausted phenotypes. We focused particularly on exhausted T cell clusters and tested their TCR reactivity against neoantigens predicted from autologous cancer cell lines. Four different TCRs specific for the same neoantigen and one orphan TCR specific for the autologous cell line were identified from one of the patients. Differential gene expression analysis in tumor-specific T cells relative to the other T cells identified CXCL13, as a candidate gene expressed by tumor-specific T cells. In addition to expressing CXCL13, tumor-specific T cells were present in a higher proportion of T cells co-expressing PDCD1(PD-1)/TNFRSF9(4-1BB). Furthermore, flow cytometric analyses in advanced lung cancer patients with MPE documented that those with high PD-1/4-1BB expression have a better prognosis in the subset of 57 adenocarcinoma patients (p = .039). These data suggest that PD-1/4-1BB co-expression might identify tumor-specific CD8+ T cells in MPE, which are associated with patients’ prognosis. (233 words)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a773a20ede56ea68c7d829514c6ec846aa69141" target='_blank'>
              Candidate tumor-specific CD8+ T cell subsets identified in the malignant pleural effusion of advanced lung cancer patients by single-cell analysis
              </a>
            </td>
          <td>
            Yusuke Sugita, Daisuke Muraoka, Ayako Demachi‐Okamura, Hiroyasu Komuro, K. Masago, E. Sasaki, Yasunori Fukushima, Takuya Matsui, Shuichi Shinohara, Yusuke Takahashi, Reina Nishida, Chieko Takashima, T. Yamaguchi, Y. Horio, Kana Hashimoto, I. Tanaka, Hiroshi Hamana, Hiroyuki Kishi, Daiki Miura, Yuki Tanaka, K. Onoue, K. Onoguchi, Yoshiko Yamashita, R. Stratford, Trevor Clancy, Rui Yamaguchi, Hiroaki Kuroda, H. Ishibashi, K. Okubo, Hirokazu Matsushita
          </td>
          <td>2024-06-28</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Background Cervical cancer has the second-highest mortality rate among malignant tumors of the female reproductive system. Immune checkpoint inhibitors such as programmed cell death protein 1 (PD-1) blockade are promising therapeutic agents, but their efficacy when combined with neoadjuvant chemotherapy (NACT) has not been fully tested, and how they alter the tumor microenvironment has not been comprehensively elucidated. Methods In this study, we conducted single-cell RNA sequencing using 46,950 cells from nine human cervical cancer tissues representing sequential different stages of NACT and PD-1 blockade combination therapy. We delineated the trajectory of cervical epithelial cells and identified the crucial factors involved in combination therapy. Cell–cell communication analysis was performed between tumor and immune cells. In addition, THP-1-derived and primary monocyte-derived macrophages were cocultured with cervical cancer cells and phagocytosis was detected by flow cytometry. The antitumor activity of blocking CD74 was validated in vivo using a CD74 humanized subcutaneous tumor model. Results Pathway enrichment analysis indicated that NACT activated cytokine and complement-related immune responses. Cell–cell communication analysis revealed that after NACT therapy, interaction strength between T cells and cancer cells decreased, but intensified between macrophages and cancer cells. We verified that macrophages were necessary for the PD-1 blockade to exert antitumor effects in vitro. Additionally, CD74-positive macrophages frequently interacted with the most immunoreactive epithelial subgroup 3 (Epi3) cancer subgroup during combination NACT. We found that CD74 upregulation limited phagocytosis and stimulated M2 polarization, whereas CD74 blockade enhanced macrophage phagocytosis, decreasing cervical cancer cell viability in vitro and in vivo. Conclusions Our study reveals the dynamic cell–cell interaction network in the cervical cancer microenvironment influenced by combining NACT and PD-1 blockade. Furthermore, blocking tumor-associated macrophage-derived CD74 could augment neoadjuvant therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24fd531eb948f7ca4c4500c29ee671a5c0536dba" target='_blank'>
              Targeting tumor-associated macrophage-derived CD74 improves efficacy of neoadjuvant chemotherapy in combination with PD-1 blockade for cervical cancer
              </a>
            </td>
          <td>
            Zixiang Wang, Bingyu Wang, Yuan Feng, Jinwen Ye, Zhonghao Mao, Teng Zhang, Meining Xu, Wenjing Zhang, Xinlin Jiao, Qing Zhang, Youzhong Zhang, Baoxia Cui
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a rare disease associated with a poor prognosis, primarily due to early recurrence and metastasis. An important feature of this condition is microvascular invasion (MVI). However, current predictive models based on imaging have limited efficacy in this regard. This study employed a random forest model to construct a predictive model for MVI identification and uncover its biological basis. Single-cell transcriptome sequencing, whole exome sequencing, and proteome sequencing were performed. The area under the curve of the prediction model in the validation set was 0.93. Further analysis indicated that MVI-associated tumor cells exhibited functional changes related to epithelial-mesenchymal transition and lipid metabolism due to alterations in the NF-kappa B and MAPK signaling pathways. Tumor cells were also differentially enriched for the IL-17 signaling pathway. There was less infiltration of SLC30A1+ CD8+ T cells expressing cytotoxic genes in MVI-associated ICC, whereas there was more infiltration of myeloid cells with attenuated expression of the MHC II pathway. Additionally, MVI-associated intercellular communication was closely related to the SPP1-CD44 and ANXA1-FPR1 pathways. These findings resulted in a brilliant predictive model and fresh insights into MVI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891c8bc5aef729b930179251eb7e060720eef85" target='_blank'>
              Machine learning based on biological context facilitates the identification of microvascular invasion in intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Shuaishuai Xu, Mingyu Wan, C. Ye, Ruyin Chen, Qiong Li, Xiaochen Zhang, Jian Ruan
          </td>
          <td>2024-08-01</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Intermediate cell carcinoma (Int-CA) is a rare and enigmatic primary liver cancer characterized by uniform tumor cells exhibiting mixed features of both HCC and intrahepatic cholangiocarcinoma. Despite the unique pathological features of int-CA, its molecular characteristics remain unclear yet. Methods: RNA sequencing and whole genome sequencing profiling were performed on int-CA tumors and compared with those of HCC and intrahepatic cholangiocarcinoma. Results: Int-CAs unveiled a distinct and intermediate transcriptomic feature that is strikingly different from both HCC and intrahepatic cholangiocarcinoma. The marked abundance of splicing events leading to intron retention emerged as a signature feature of int-CA, along with a prominent expression of Notch signaling. Further exploration revealed that METTL16 was suppressed within int-CA, showing a DNA copy number–dependent transcriptional deregulation. Notably, experimental investigations confirmed that METTL16 suppression facilitated invasive tumor characteristics through the activation of the Notch signaling cascade. Conclusions: Our results provide a molecular landscape of int-CA featured by METTL16 suppression and frequent intron retention events, which may play pivotal roles in the acquisition of the aggressive phenotype of Int-CA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab7b9d18c0e5e1be8d12c7361eff07994e2e2a" target='_blank'>
              Transcriptomic profiling of intermediate cell carcinoma of the liver
              </a>
            </td>
          <td>
            Byungchan Jang, S. Kwon, Jang Hyun Kim, Jung Mo Kim, Taek Chung, Jeong Eun Yoo, Haeryoung Kim, Julien Calderaro, Hyun Goo Woo, Young Nyun Park
          </td>
          <td>2024-08-01</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde19b1f96c1cfcaa9c403d17be06cabeee637d7" target='_blank'>
              Pan-cell death-related signature reveals tumor immune microenvironment and optimizes personalized therapy alternations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Linzhi Han, Jingyi He, Hongxin Xie, Yan Gong, Conghua Xie
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Objective Hepatocellular carcinoma (HCC) poses a significant clinical challenge because the long-term benefits of immune checkpoint blockade therapy are limited. A comprehensive understanding of the mechanisms underlying immunotherapy resistance in HCC is imperative for improving patient prognosis. Design In this study, to systematically investigate the characteristics of cancer-associated fibroblast (CAF) subsets and the dynamic communication among the tumor microenvironment (TME) components regulated by CAF subsets, we generated an HCC atlas by compiling single-cell RNA sequencing (scRNA-seq) datasets on 220 samples from six datasets. We combined spatial transcriptomics with scRNA-seq and multiplexed immunofluorescence to identify the specific CAF subsets in the TME that determine the efficacy of immunotherapy in HCC patients. Results Our findings highlight the pivotal role of POSTN+ CAFs as potent immune response barriers at specific tumor locations, as they hinder effective T-cell infiltration and decrease the efficacy of immunotherapy. Additionally, we elucidated the interplay between POSTN+ CAFs and SPP1+ macrophages, whereby the former recruits the latter and triggers increased SPP1 expression via the IL-6/STAT3 signaling pathway. Moreover, we demonstrated a spatial correlation between POSTN+ CAFs and SPP1+ macrophages, revealing an immunosuppressive microenvironment that limits the immunotherapy response. Notably, we found that patients with elevated expression levels of both POSTN+ CAFs and SPP1+ macrophages achieved less therapeutic benefit in an immunotherapy cohort. Conclusion Our research elucidates light on the role of a particular subset of CAFs in immunotherapy resistance, emphasizing the potential benefits of targeting specific CAF subpopulations to improve clinical responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321d7431417f89b80a8af6ab7f043a0cb955e621" target='_blank'>
              POSTN+ cancer-associated fibroblasts determine the efficacy of immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hao Wang, Yuan Liang, Zheng Liu, Rui Zhang, Jiashuo Chao, Mingming Wang, Mu Liu, Lei Qiao, Zhengfeng Xuan, Haitao Zhao, Ling Lu
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Background Sarcomatoid differentiation (SD) in renal cell carcinoma (RCC) is associated with poor survival and heightened response to immune checkpoint blockade. Detection of SD can be challenging due to spatial heterogeneity and sampling error. Herein, we introduce a novel tissue–informed epigenomic approach to noninvasively identify sarcomatoid differentiation in patients with RCC from cell-free DNA (cfDNA) using 1mL of plasma. Methods Chromatin immunoprecipitation and sequencing (ChIP-seq) for H3K27ac – a histone modification associated with active regulatory elements (REs) – was performed on pathologically reviewed clear cell RCC frozen tissue samples with and without SD (sarcomatoid-RCC and epithelioid-RCC, resp.) collected at the Dana-Farber Cancer Institute. Differentially marked REs between sarcomatoid and epithelioid subtypes were identified using DESeq2 (false discovery rate of q < 0.01). After establishing tissue signatures, ChIP-seq was then performed on cell-free chromatin (cfChIP-seq) in plasma from patients with sarc-RCC and epi-RCC. A Sarcomatoid Score was derived for each sample by aggregating the plasma H3K27ac signal at tissue-derived sarcomatoid-specific REs (sarc-REs), while normalizing to signal at epithelioid-specific REs (epi-REs). Scores were compared between the two groups using a Wilcoxon rank-sum test. A classifier was built to distinguish sarc-RCC from epi-RCC based on the Sarcomatoid Score and its performance was evaluated using the area under the receiver operating characteristic (AUROC) curve. Results We identified 25,919 differentially marked REs between 8 sarc-RCC and 8 epi-RCC tissue samples at a false discovery rate of q < 0.01. We selected 12,868 REs that are enriched in sarcomatoid vs. epithelioid. We generated cfChIP-seq profiles from plasma of 29 patients, 17 with sarc-RCC and 12 with epi-RCC. The Sarcomatoid Scores were significantly higher in sarc-RCC vs. epi-RCC plasma samples (p=1.6×10-4; Figure 1A). These scores achieved an AUROC curve of 0.9 for classifying patients with sarc-RCC from patients with epi-RCC (Figure 1B). Conclusions We present a proof-of-concept study in 1 cc of plasma for the detection of sarcomatoid differentiation in RCC based on the assessment of histone modification signals in cfDNA. This approach could help overcome the challenges of spatial heterogeneity and sampling error from tissue that make identification of sarc-RCC difficult. More generally, it establishes a paradigm for identifying histologic subtypes of cancer based on their epigenomic correlates from cfDNA, with possible therapeutic implications in real-time. Figure 1. (A) Sarcomatoid Score in plasma from patients with RCC at sarcomatoid-specific REs, comparing sarcomatoid (orange) and epithelioid RCC (blue). (B) ROC curves for distinguishing sarcomatoid from epithelioid RCC plasma samples using H3K27ac cell-free ChIP-seq signal at sarcomatoid-specific REs. ‘AUC’ indicates area under the ROC curve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f03141a33629f618991474e63a226a8491b01de4" target='_blank'>
              44 Liquid biopsy epigenomic profiling for the detection of sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Karl Semaan, T. Zarif, Marc Eid, John Canniff, Noa Phillips, Bard Fortunato, Hunter Savignano, Matthew P Davidson, Razane El Hajj Chehade, Simon Garinet, E. Saad, R. Saliby, M. Sun, J-H. Seo, Jacob E. Berchuck, David A Braun, M. Freedman, Toni K. Choueiri, S. Baca
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c26de2e8d9a710cffe1117e49d0f50cd225d69" target='_blank'>
              Cancer organoids 2.0: modelling the complexity of the tumour immune microenvironment.
              </a>
            </td>
          <td>
            Roel Polak, Elisa T. Zhang, Calvin J Kuo
          </td>
          <td>2024-07-08</td>
          <td>Nature reviews. Cancer</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
Therapy resistance is a major clinical hurdle in bone cancer treatment and seems to be largely driven by poorly understood microenvironmental factors. Recent evidence suggests a critical role for a unique subpopulation of mesenchymal stem cells with inflammatory features (iMSCs), though their origin and function remained unexplored. We demonstrate that cancer-secreted extracellular vesicles (EVs) trigger the development of iMSCs, which hinder therapy response in vivo, and set out to identify strategies to counteract their function.


EXPERIMENTAL DESIGN
The role of iMSCs in therapy resistance was evaluated in an orthotopic xenograft mouse model of osteosarcoma. EV-induced alterations of the MSC transcriptome were analyzed and compared with scRNA-seq data of osteosarcoma and multiple myeloma patient biopsies. Functional assays identified EV components driving iMSC development. We assessed the efficacy of clinical drugs in blocking iMSC-induced resistance in vivo.


RESULTS
We found that iMSCs are induced by interaction with cancer EVs and completely abrogate the antimetastatic effect of TGFb signaling inhibition. Importantly, EV-induced iMSCs faithfully recapitulate the inflammatory single-cell RNA signature of stromal cells enriched in multiple myeloma and osteosarcoma patient biopsies. Mechanistically, cancer EVs act through two distinct mechanisms. EV-associated TGFb induces IL6 production, while the EV-RNA cargo enhances TLR3-mediated chemokine production. We reveal that simultaneous blockade of downstream EV-activated pathways with ladarixin and tocilizumab disrupts metastasis formation and overcomes iMSC-induced resistance.


CONCLUSIONS
Our observations establish iMSCs as major contributors to drug resistance, reveal EVs as physiological triggers of iMSC development and highlight a promising combination strategy to improve therapy response in bone cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fb87f3a9aca9bc675c26582efc89f032b59273" target='_blank'>
              Tumor-secreted extracellular vesicles counteract therapy response by triggering inflammatory mesenchymal stem cell development.
              </a>
            </td>
          <td>
            Crescenzo Massaro, Hilal N Sensoy, Manon Mulders, Celine De Schrijver, C. Gómez-Martín, Juan Simon-Nieto, T. Lagerweij, Alisha Atmopawiro, Jennifer Pérez-Boza, Maarten P. Bebelman, Leontien Bosch, Simone Foderaro, Mafalda Neves Ferreira, Monique A J van Eijndhoven, Jan R. T. van Weering, C. Dell'Aversana, L. Altucci, C. D. Savci-Heijink, Niels W C J van de Donk, Cristina Giorgio, L. Brandolini, M. Allegretti, D. M. Pegtel, S. R. Baglio
          </td>
          <td>2024-08-08</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="PURPOSE
The molecular drivers underlying mucinous tumor pathogenicity are poorly understood. GNAS mutations predict metastatic burden and treatment resistance in mucinous appendiceal adenocarcinoma. We investigated the pan-cancer clinicopathologic relevance of GNAS variants.


METHODS
We assessed 58,043 patients with Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (IMPACT)-sequenced solid tumors to identify oncogenic variants, including GNAS, associated with mucinous tumor phenotype. We then performed comprehensive molecular analyses to compare GNAS-mutant (mut) and wild-type tumors across cancers. Gene expression patterns associated with GNAS-mut tumors were assessed in a The Cancer Genome Atlas cohort. Associations between GNAS variant status and peritoneal metastasis, first-line systemic therapy response, progression-free survival (PFS), and overall survival (OS) were determined using a propensity-matched subcohort of patients with metastatic disease.


RESULTS
Mucinous tumors were enriched for oncogenic GNAS variants. GNAS was mutated in >1% of small bowel, cervical, colorectal, pancreatic, esophagogastric, hepatobiliary, and GI neuroendocrine cancers. Across these cancers, GNAS-mut tumors exhibited a generally conserved C-to-T mutation-high, aneuploidy-low molecular profile with co-occurring prevalent KRAS variants (65% of GNAS-mut tumors) and fewer TP53 alterations. GNAS-mut tumors exhibited recurrently comutated alternative tumor suppressors (RBM10, INPPL1) and upregulation of MAPK and cell surface modulators. GNAS-mut tumors demonstrate an increased prevalence of peritoneal metastases (odds ratio [OR], 1.7 [95% CI, 1.1 to 2.5]; P = .006), worse response to first-line systemic therapy (OR, 2.2 [95% CI, 1.3 to 3.8]; P = .003), and shorter PFS (median, 5.6 v 7.0 months; P = .047). In a multivariable analysis, GNAS mutated status was independently prognostic of worse OS (hazard ratio, 1.25 [95% CI, 1.01 to 1.56]; adjusted P = .04).


CONCLUSION
Across the assessed cancers, GNAS-mut tumors exhibit a conserved molecular and clinical phenotype defined by mucinous tumor status, increased peritoneal metastasis, poor response to first-line systemic therapy, and worse survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a7fd5d8b4825c722728924fd719f0a0d9f3a8e" target='_blank'>
              Molecular and Clinicopathologic Impact of GNAS Variants Across Solid Tumors.
              </a>
            </td>
          <td>
            P. Johannet, Somer Abdelfattah, Callahan Wilde, Shrey Patel, H. Walch, B. Rousseau, Guillem Argiles, O. Artz, Miteshkumar Patel, Andrea Arfe, A. Cercek, R. Yaeger, K. Ganesh, N. Schultz, Luis A. Diaz, M. Foote
          </td>
          <td>2024-08-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Despite advances in the genomic classification of breast cancer, current clinical tests and treatment decisions are commonly based on protein-level information. Nowadays breast cancer clinical treatment selection is based on the immunohistochemical (IHC) determination of four protein biomarkers: Estrogen Receptor 1 (ESR1), Progesterone Receptor (PGR), Human Epidermal Growth Factor Receptor 2 (HER2), and proliferation marker Ki-67. The prognostic correlation of tumor-infiltrating T cells has been widely studied in breast cancer, but tumor-infiltrating B cells have not received so much attention. We aimed to find a correlation between immunohistochemical results and a proteomic approach in measuring the expression of proteins isolated from B-cell lymphocytes in peripheral blood samples. Shotgun proteomic analysis was chosen for its key advantage over other proteomic methods, which is its comprehensive and untargeted approach to analyzing proteins. This approach facilitates better characterization of disease-associated changes at the protein level. We identified 18 proteins in B cell lymphocytes with a significant fold change of more than 2, which have promising potential to serve as breast cancer biomarkers in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc2c686ab3ce63d751f38765244b443a0deeaae" target='_blank'>
              B Cell Lymphocytes as a Potential Source of Breast Carcinoma Marker Candidates
              </a>
            </td>
          <td>
            S. Tkáčiková, Miroslav Marcin, P. Bober, Mária Kacírová, Michaela Šuliková, Jozef Parnica, Dávid Tóth, Marek Lenárt, Jozef Radoňák, P. Urdzík, Ján Fedačko, Ján Sabo
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Immunotherapy has brought significant advancements in the treatment of lung adenocarcinoma (LUAD), but identifying suitable candidates remains challenging. In this study, we investigated tumour cell heterogeneity using extensive single-cell data and explored the impact of different tumour cell cluster abundances on immunotherapy in the POPLAR and OAK immunotherapy cohorts. Notably, we found a significant correlation between CKS1B+ tumour cell abundance and treatment response, as well as stemness potential. Leveraging marker genes from the CKS1B+ tumour cell cluster, we employed machine learning algorithms to establish a prognostic and immunotherapeutic signature (PIS) for LUAD. In multiple cohorts, PIS outperformed 144 previously published signatures in predicting LUAD prognosis. Importantly, PIS reliably predicted genomic alterations, chemotherapy sensitivity and immunotherapy responses. Immunohistochemistry validated lower expression of immune markers in the low-PIS group, while in vitro experiments underscored the role of the key gene PSMB7 in LUAD progression. In conclusion, PIS represents a novel biomarker facilitating the selection of suitable LUAD patients for immunotherapy, ultimately improving prognosis and guiding clinical decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ecfb4cbb07b59d7d596d4bed49de4c7820b19d3" target='_blank'>
              Exploring cellular diversity in lung adenocarcinoma epithelium: Advancing prognostic methods and immunotherapeutic strategies.
              </a>
            </td>
          <td>
            Lianmin Zhang, Yanan Cui, Jie Mei, Zhenfa Zhang, Pengpeng Zhang
          </td>
          <td>2024-06-30</td>
          <td>Cell proliferation</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Brain tumors, particularly glioblastoma (GBM), are devastating and challenging to treat, with a low 5-year survival rate of only 6.6%. Mouse models are established to understand tumorigenesis and develop new therapeutic strategies. Large-scale genomic studies have facilitated the identification of genetic alterations driving human brain tumor development and progression. Genetically engineered mouse models (GEMMs) with clinically relevant genetic alterations are widely used to investigate tumor origin. Additionally, syngeneic implantation models, utilizing cell lines derived from GEMMs or other sources, are popular for their consistent and relatively short latency period, addressing various brain cancer research questions. In recent years, the success of immunotherapy in specific cancer types has led to a surge in cancer immunology-related research which specifically necessitates the utilization of immunocompetent mouse models. In this review, we provide a comprehensive summary of GEMMs and syngeneic mouse models for adult brain tumors, emphasizing key features such as model origin, genetic alteration background, oncogenic mechanisms, and immune-related characteristics. Our review serves as a valuable resource for the brain tumor research community, aiding in the selection of appropriate models to study cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff748c1f4b0e36e21dc90ffb3f8ae15d403a0e7" target='_blank'>
              In Vivo Mouse Models for Adult Brain Tumors: Exploring Tumorigenesis and Advancing Immunotherapy Development.
              </a>
            </td>
          <td>
            John Figg, Dongjiang Chen, Laura Falceto Font, Catherine Flores, Dan Jin
          </td>
          <td>2024-07-11</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background PIK3CA mutations are implicated in various cancers, but the implications of multiple concurrent mutations and their orientations within the gene have not been fully explored. Methods In this study, we analyzed multi‐PIK3CA mutations across a diverse pan‐cancer cohort comprising 3564 tumors. Results Multi‐PIK3CA mutations were present in 10.3% of all PIK3CA‐mutant tumors, predominantly occurring in breast and gynecological cancers. Notably, mutations within the helical domain (E542:E545) exclusively occurred in the trans‐orientation, contrasting with mutations in the kinase ABD and C2 domains, which mainly appeared in the cis orientation. Conclusions The distinct pattern of mutation orientations in PIK3CA suggests variable oncogenic potential, with helical domain mutations in the trans‐orientation potentially being less oncogenic. These findings highlight the importance of mutation orientation in the PIK3CA gene as potential biomarkers for targeted therapy. This understanding is crucial for designing clinical trials that leverage PI3K inhibitors, aiming for more effective and precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c039fa2d1ed75da420038d18b0f0cfdcf748c0d" target='_blank'>
              Characterizing multi‐PIK3CA mutations across cancer types: Toward precision oncology
              </a>
            </td>
          <td>
            Kohei Nakamura, Marin Ishikawa, R. Kawano, E. Aimono, Takaaki Mizuno, S. Nohara, Shigeki Tanishima, Hideyuki Hayashi, H. Nishihara
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The rise of single-cell RNA-sequencing (scRNA-seq) and evolved computational algorithms have significantly advanced biomedical science by revealing and visualizing the multifaceted and diverse nature of single cells. These technical advancements have also highlighted the pivotal role of cell clusters as representations of biologically universal entities such as cell types and cell states. However, to some extent, these clusterings remain dataset-specific and method-dependent. To improve comparability across different datasets or compositions, we previously introduced a graph-based representation of cell collections that captures the statistical dependencies of their characteristic genes. While our earlier work focused on theoretical insights, it was not sufficiently adapted and fine-tuned for practical implementation. To address this, the present paper introduces an improved practice to define and evaluate cellular identities based on our theory. First, we provide a concise summary of our previous theory and workflow. Then, point-by-point, we highlight the issues that needed fixing and propose solutions. The framework’s utility was enhanced by leveraging alternative formats of cellular features such as gene ontology (GO) terms and effectively handling dropouts. Supplemental techniques are offered to reinforce the versatility and robustness of our method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da477f921cb27ea88031ef50da4dab1e779172ca" target='_blank'>
              A graph-based practice of evaluating collective identities of cell clusters
              </a>
            </td>
          <td>
            Yuji Okano, Y. Kase, Hideyuki Okano
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/372929d4770d0ba8c1a928418b60ee5ffa7b960b" target='_blank'>
              Comprehensive investigation of malignant epithelial cell-related genes in clear cell renal cell carcinoma: development of a prognostic signature and exploration of tumor microenvironment interactions
              </a>
            </td>
          <td>
            Songyang Liu, Ge Li, Xiaomao Yin, Yihan Zhou, Dongmei Luo, Zhenggang Yang, Jin Zhang, Jianfeng Wang
          </td>
          <td>2024-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f9c45ead2d22a430930d445e02bcc5ccb61da5" target='_blank'>
              The proteogenomic landscape of multiple myeloma reveals insights into disease biology and therapeutic opportunities.
              </a>
            </td>
          <td>
            Evelyn Ramberger, Valeriia Sapozhnikova, Yuen Lam Dora Ng, A. Dolnik, M. Ziehm, Oliver Popp, E. Sträng, M. Kull, F. Grünschläger, Josefine Krüger, Manuela Benary, S. Müller, Xiang Gao, Arunima Murgai, Mohamed Haji, Annika Schmidt, R. Lutz, A. Nogai, Jan Braune, Dominik Laue, Christian Langer, C. Khandanpour, Florian Bassermann, H. Döhner, M. Engelhardt, Christian Straka, M. Hundemer, D. Beule, Simon Haas, Ulrich Keller, H. Einsele, Lars Bullinger, Stefan Knop, P. Mertins, J. Krönke
          </td>
          <td>2024-06-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/571475d254bb27cb6d2937910baaa56e8534b72e" target='_blank'>
              Refining the optimal CAF cluster marker for predicting TME-dependent survival expectancy and treatment benefits in NSCLC patients
              </a>
            </td>
          <td>
            Kai Li, Rui Wang, Guo-Wei Liu, Zi-Yang Peng, Ji-Chang Wang, G. Xiao, Shou-Ching Tang, Ning Du, Jia Zhang, Jing Zhang, Hong Ren, Xin Sun, Yiping Yang, Da-Peng Liu
          </td>
          <td>2024-07-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Prostate cancer is a complex and heterogeneous disease with multiple tumor foci, each potentially harboring distinct driver molecular aberrations. This complexity poses challenges to effective management. We took an innovative approach to gain a comprehensive understanding of the genetic underpinnings of each tumor focus and avoid overlooking more minor yet clinically significant foci. Instead of relying solely on a systematic sampling of dominant foci, we conducted molecular analysis on whole-mount radical prostatectomy specimens. Our study aimed to find distinct molecular subsets of prostate cancer and assess their correlation with clinical outcomes, focusing on Caucasians (CA) and African Americans (AA). Method: We randomly selected 2201 whole-mount radical prostatectomy cases, with 1207 (54.8%) from CA and 994 (45.1%) from AA patients evaluated for a 5-year biochemical recurrence-free survival rate (BCR). Of these 2201 cases, 834 (463 -56% were from CA and 371 -44% from AA patients) were subjected to molecular analysis using dual immunohistochemistry (IHC) for ERG and SPINK1, along with dual RNA in-situ hybridization (RNA-ISH) for ETV1 and ETV4 to evaluate tumor molecular heterogeneity on whole-mount specimens. The Chi-squared test examined racial disparities in aberrant oncogene expression. To assess BCR-free survival, we employed the Kaplan-Meier method and Cox-PH models for patients with distinct molecular subsets of prostate cancer. Additionally, Gleason Grade groups of prostate biopsies were summarized using a spaghetti plot and compared using linear mixed models. Results: Analysis of the 2201 cases revealed that AA with localized prostate cancer behaved differently with better 5-year BCR-free survival than CA after radical prostatectomy (AA: 0.82, 95% CI 0.80-0.85; CA: 0.71, 95% CI 0.68-0.75; p<.001). Molecular profiling of whole-mount specimens from 834 cases revealed that 16.4%, 58.4%, 21.7%, and 3.5% of patients with localized prostate cancer expressed none, one, two, and three of the four oncogenes, respectively. This finding identified new molecular subsets of prostate cancer with more than one driver mutation in a mutually exclusive manner within the multifocal disease. ERG and SPINK1 expression showed a negative correlation (p<.001). Notably, AA patients exhibited a lower incidence of ERG (38.8% vs. 60.3%) but a higher incidence of SPINK1 (63.3% vs. 35.6%) than CA patients. The incidences of ETV1 (9.4% vs. 9.3%) and ETV4 (4.6% vs. 3.9%) were not statistically significant between the two racial groups. However, significantly, ETV1 expression was associated with worse BCR-free survival in CA patients (hazard ratio [HR]=2.36, 95% CI 1.22-4.57, p=.02), while ETV4 expression was linked to worse BCR-free survival in AA patients (HR=2.65, 95% CI 1.15-6.09, p=.02). Moreover, ETV4 expression was associated with regional lymph node metastasis in AA patients (odds ratio [OR]=5.14, 95% CI 1.3-17.4, p=.01) but not in CA patients (OR=0.60, 95% CI 0.03-3.17, p=.63) at the time of radical prostatectomy. Additionally, in patients who underwent multiple biopsies before radical prostatectomy, the Gleason Grade group increased over time in AA patients (0.25 per year, p<.001) but remained unchanged in CA patients. ERG expression was associated with a lower Gleason Grade group (-0.20, p=.03), while ETV4 expression was linked to a higher Gleason Grade (0.54, p=.01). Conclusions: Our study reveals that AA with localized prostate cancer behaves differently and has better BCR-free survival than CA after radical prostatectomy, even after adjusting for known prognostic factors. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion within a multifocal prostate shows significant molecular heterogeneity between localized prostate cancer in CA and AA patients. Importantly, given the association of ETV1 and ETV4 expression with worse BCR-free survival in CA and AA, respectively, ETV1 and ETV4 emerge as potential prognostic markers, offering insights for clinical practice to predict prostate cancer recurrence after radical prostatectomy. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion and SPINK1 in a mutually exclusive pattern indicates the clonal origin of independent tumor foci, which is a rare and unique phenomenon in prostate cancer hitherto unidentified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863b31d824c9eeb2a43dee5a1d15a9728d0743ea" target='_blank'>
              Unveiling Prostate Cancer's Molecular Tapestry: Ethnic Disparities and Prognostic Insights from Whole-Mount Prostatectomy Tissue Analysis
              </a>
            </td>
          <td>
            Wei Zhao, Pin Li, Shannon Carskadon, Sunita Ghosh, Craig G. Rogers, James Peabody, D. Chitale, Mani Menon, Sean Williamson, Nilesh Gupta, N. Palanisamy
          </td>
          <td>2024-07-16</td>
          <td>None</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley
          </td>
          <td>2024-08-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary The expression of the mechanosensory Piezo2 channel has already been described in different malignant tumors. There is discordance in the literature regarding breast carcinoma, with its expression described either as decreased or increased in neoplasms with respect to benign tissue. A retrospective cohort of 125 patients whose breasts were resected for carcinoma was chosen to determine the relationship between Piezo2 and different clinical and histological variables. A significant association was found with the Ki67 proliferation index, with a tendency for most proliferative tumors to be positive for Piezo2. Abstract In the last decade, a group of Ca2+ channels called Piezo were discovered, demonstrating a decisive role in the cellular response to mechanical stimuli and being essential in the biological behavior of cells regarding the extracellular compartment. Several investigations have suggested a potential role in carcinogenesis, with a tumor suppressor role in some cases but increased expression in several high-grade neoplasms. Regarding Piezo2 expression in mammary gland neoplasms, a protective role for Piezo2 was initially suggested, but a subsequent study demonstrated a relationship between Piezo2 expression and the highly aggressive triple-negative phenotype of breast carcinoma. A cohort of 125 patients with clinical follow-up was chosen to study Piezo2 expression and clarify its clinical implications using the same immunohistochemical evaluation performed for other breast carcinoma parameters. Fisher’s exact test was chosen to identify potential relationships between the different variables. A significant association was found with the Ki67 proliferation index, but not with mitoses. The tendency of most proliferative tumors was to have an increased score for Piezo2. A similar association was found between Piezo2 expression and perineural invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6531181900e59a16e97bcf0664ec51d36a032225" target='_blank'>
              Prognostic Evaluation of Piezo2 Channels in Mammary Gland Carcinoma
              </a>
            </td>
          <td>
            R. Martín-Sanz, Aline Rodrigues-Françoso, Y. García-Mesa, F. J. García-Alonso, María Asunción Gómez-Muñoz, Sandra Malmierca-González, Rocío Salazar-Blázquez, Olivia García-Suárez, J. Feito
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) is a heterogenous disease that disproportionately affects Black women. TNBC outcomes among Black women are dismal secondary to multiple factors, such as poor healthcare accessibility resulting in delays in diagnosis, and aggressive disease biology in addition to a pro-tumor immune microenvironment (TME). Black women with breast cancer exhibit elevated levels of serum pro-inflammatory cytokines, and a pro-tumorigenic TME with higher immunosuppressive regulatory T cells (Tregs), M2 macrophages and exhausted CD8+ T cells. We have shown that the combined use of toll-like receptor 3 (TLR3) ligands with interferon-α (chemokine modulation: CKM) is able to enrich the tumor with CD8+ T cells, while not increasing immunosuppressive cells. Recent clinical trials have revealed the efficacy of immune checkpoint inhibitors (ICI) in rejuvenizing exhausted CD8+ T cells. We hypothesize that strategies to modulate the TME by enriching chemokines that attract CD8+T cells followed by reversal of CD8+ T cell exhaustion (ICI), when added to standard treatment, could potentially improve clinical outcomes, and mitigate the racial disparities in TNBC outcomes between Black and White Women.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a19fbb259d7a213132cc235d155d56ac9120b99" target='_blank'>
              Targeting the tumor microenvironment to improve clinical outcomes in triple negative breast cancer patients and bridge the current disparity gap
              </a>
            </td>
          <td>
            Malak Alharbi, A. M. Roy, Jayasree Krishnan, Pawel Kalinski, Song Yao, Shipra Gandhi
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aaa6f91eb9665f5c96c7d24846ea516080ae96e" target='_blank'>
              Power-law scaling in intratumoral microbiota of colorectal cancer
              </a>
            </td>
          <td>
            N. Dovrolis, M. Gazouli, F. Rigal, Robert J. Whittaker, Thomas J. Matthews, Konstantinos Georgiou, George Theodoropoulos, K. Triantis
          </td>
          <td>2024-07-07</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) is dependent on a supportive tumor immune microenvironment (TIME), where infiltration of CD163+ macrophages has a negative prognostic impact. This study explores how abundance and spatial localization of CD163+ cells are associated with the biology of the MCL TIME. This is achieved through spatial multi-omic investigations of tumor and infiltrating CD163+ and CD3+ cells, respectively. We analyzed diagnostic MCL tissue from 100 patients. Sixty-three proteins were measured by GeoMx® digital spatial profiling in tissue microarrays. Regions of interests (ROIs) were selected in tumor-rich and tumor-sparse tissue regions. Molecular profiling of CD163+ macrophage segments, CD20+ MCL tumor cell segments and CD3+ T-cell segments was performed. To validate protein profiles, 1811 mRNAs were measured in CD20+ cells and two subsets of T-cells. Image analysis was used to extract the phenotype and position of each targeted cell allowing exploration of cell frequencies and cellular neighborhoods. Proteomic investigations revealed that CD163+ cells modulate their immune profile depending on the localization and that the immune inhibitory molecules VISTA and B7-H3 have higher expression in tumor-sparse versus tumor-rich tissue regions and targeting should be explored. We show that MCL tissues with more abundant infiltration of CD163+ cells have a higher expression of key components of the mitogen-activated protein kinase (MAPK) pathway, which was validated by complementary mRNA analyses. Thus, the MAPK pathway may be a feasible therapeutic target in MCL patients with CD163+ cell infiltration. We further show the independent and combined prognostic value of CD11c and CD163 beyond established risk factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d71801d630d06d650258878ed1d5d14b266a8e5" target='_blank'>
              CD163+ macrophages in mantle cell lymphoma induce activation of pro-survival pathways and immune suppression.
              </a>
            </td>
          <td>
            J. Rodrigues, L. Lokhande, L. Olsson, May Hassan, Angelica Johansson, Anna Janská, Darshan Kumar, Lina Schmidt, A. Nikkarinen, P. Hollander, I. Glimelius, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-07-03</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
Human gliomas are classified using isocitrate dehydrogenase (IDH) status as a prognosticator; however, the influence of genetic differences and treatment effects on ensuing immunity remains unclear.


METHODS
In this study, we used sequential single-cell transcriptomics on 144,678 and spectral cytometry on over two million immune cells encompassing 48 human gliomas to decipher their immune landscape.


RESULTS
We identified 22 distinct immune cell types that contribute to glioma immunity. Specifically, brain-resident microglia (MG) were reduced with a concomitant increase in CD8+ T lymphocytes during glioma recurrence independent of IDH status. In contrast, IDH-wild-type-associated patterns, such as an abundance of antigen-presenting cell-like MG and cytotoxic CD8+ T cells, were observed. Beyond elucidating the differences in IDH, relapse, and treatment-associated immunity, we discovered novel inflammatory MG subpopulations expressing granulysin, a cytotoxic peptide, which is otherwise expressed in lymphocytes only. Furthermore, we provide a robust genomic framework for defining macrophage polarization beyond M1/M2 paradigm and reference signatures of glioma-specific tumor immune microenvironment (termed Glio-TIME-36) for deconvoluting transcriptomic datasets.


CONCLUSIONS
This study provides advanced optics of the human pan-glioma immune contexture as a valuable guide for translational and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620d79ca9e285578dd6bf3575fec8b24ea7ccde9" target='_blank'>
              Immune landscape of isocitrate dehydrogenase-stratified primary and recurrent human gliomas.
              </a>
            </td>
          <td>
            Pravesh Gupta, M. Dang, Shivangi Oberai, Simona Migliozzi, Rakesh Trivedi, Gayatri Kumar, Mekenzie M Peshoff, Nancy Milam, Aml Ahmed, Krishna Bojja, Tuan M. Tran, J. Gumin, C. Kamiya-Matsuoka, J. Huse, Kathryn Cox, Jianzhuo Li, H. Shehwana, Sameer A. Sheth, Rodriguez Saxon, Baohua Sun, Brittany Parker-Kerrigan, A. Maheshwari, E. P. Parra Cuentas, N. Navin, A. Heimberger, Frederick F Lang, A. Iavarone, K. Clise-Dwyer, Linghua Wang, Krishna P. Bhat
          </td>
          <td>2024-08-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="While often represented as static entities, gene networks are highly context-dependent. Here, we developed a multi-task learning strategy to yield context-specific representations of gene network dynamics. We assembled a corpus comprising ∼103 million human single-cell transcriptomes from a broad range of tissues and diseases and performed a two stage pretraining, first with non-malignant cells to generate a foundational model and then with continual learning on cancer cells to tune the model to the cancer domain. We performed multi-task learning with the foundational model to learn context-specific representations of a broad range of cell types, tissues, developmental stages, and diseases. We then leveraged the cancer-tuned model to jointly learn cell states and predict tumor-restricting factors within the colorectal tumor microenvironment. Model quantization allowed resource-efficient fine-tuning and inference while preserving biological knowledge. Overall, multi-task learning enables context-specific disease modeling that can yield contextual predictions of candidate therapeutic targets for human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9913f0b488b5e3e74870d991a16be130e495ac98" target='_blank'>
              Quantized multi-task learning for context-specific representations of gene network dynamics
              </a>
            </td>
          <td>
            Han Chen, Madhavan S. Venkatesh, Javier Gómez Ortega, Siddharth V. Mahesh, T. Nandi, R. Madduri, Karin Pelka, Christina V. Theodoris
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Brain metastases (BrMs) constitute the most common type of brain cancer with abysmal prognosis. Given the high incidence and poor prognosis of BrM, understanding the tumor microenvironment (TME) architecture across BrM subtypes is increasingly crucial, especially with the emergence of novel immunotherapeutic agents like immune checkpoint inhibitors, which have demonstrated efficacy in melanoma and lung cancer BrM. Tertiary lymphoid structures (TLS) are spatially organized lymphoid aggregates in the TME that mediate antitumor immunity. Despite being established as prognostic and predictive markers in various cancers, their significance in BrM lacks extensive investigation. We performed gene expression profiling using paired-end RNA sequencing and multiplex immunofluorescence combined with spatial imaging on 149 human BrM samples. Cell type deconvolution analysis revealed that BrMs from non-small cell lung carcinoma (NSCLC) and malignant melanoma exhibited significantly higher immune cell infiltration compared to BrMs from breast carcinoma. Furthermore, B cell infiltration was detected in some patients, suggesting potential TLS formation. We created a metagene signature comprising TLS hallmarks in BrMs using transcriptomic profiling, allowing us to categorize BrM patients into TLS-positive and TLS-negative groups. This finding was corroborated by multiplex immunofluorescence staining of corresponding BrM tissue sections, which confirmed TLS-like aggregate formation in 40% of the BrM patients. Differential gene expression analysis highlighted the presence of activated lymphocyte-mediated immunity as well as increased immunoglobulin production signature in TLS-positive tumors. Additionally, a higher TLS signature score was associated with improved overall survival of lung carcinoma patients after BrM diagnosis. Our results underscore the heterogeneity of the tumor microenvironment in BrM, and the identification of intratumoral TLS may guide patient stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f904325c9c40fd220a1c52e4c6b7474d7dea43" target='_blank'>
              BMRK-09 PROGNOSTIC ROLE OF TUMOR-INFILTRATING LYMPHOCYTES AND TERTIARY LYMPHOID STRUCTURES IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            S. S. Mughal, Y. Reiss, J. Felsberg, L. Meyer, J. Macas, S. Schlue, Tatjana Starzetz, K. Köhrer, Tanja N Fehm, Volkmar Müller, K. Lamszus, Iris Helfrich, H. Wikman, A. Berghoff, B. Brors, Karlheinz Plate, G. Reifenberger
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cell surface receptors play crucial roles in cellular responses to extracellular ligands, helping to modulate the functions of a cell based on information coming from outside the cell. Syndecan refers to a family of cell adhesion receptors that regulate both extracellular and cytosolic events. Alteration of syndecan expression disrupts regulatory mechanisms in a cell type-specific fashion, often leading to serious diseases, notably cancer. Given the multifaceted functions and distinct tissue distributions of syndecan, it will be important to unravel the gene-level intricacies of syndecan expression and thereby further understand its involvement in various carcinogenic processes. Although accumulating evidence indicates that the protein expression patterns of syndecan family members are significantly altered in cancer cells, the underlying gene-level mechanisms remain largely unknown. This review endeavors to explore syndecan gene expression levels across different cancer types by scrutinizing extensive cancer genome datasets utilizing tools such as cBioPortal. Our analysis unveils that somatic mutations in SDC genes are rare occurrences, whereas copy number alterations are frequently observed across diverse cancers, particularly in SDC2 and SDC4. Notably, amplifications of SDC2 and SDC4 correlate with heightened metastatic potential and dismal prognosis. This underscores the recurrent nature of SDC2 and SDC4 amplifications during carcinogenesis and sheds light on their role in promoting cancer activity through augmented protein expression. The identification of these amplifications not only enriches our understanding of carcinogenic mechanisms but also hints at the potential therapeutic avenue of targeting SDC2 and SDC4 to curb cancer cell proliferation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f6d496d5667d6bd159572b85ae76a0360e15b9" target='_blank'>
              Correlation of syndecan gene amplification with metastatic potential and clinical outcomes in carcinomas.
              </a>
            </td>
          <td>
            Sewoon Kim, Hyeonju Yang, Subin Cho, Yunjung Jang, I. Han, Eok-Soo Oh
          </td>
          <td>2024-07-02</td>
          <td>American journal of physiology. Cell physiology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadefc00f25039c8bfe4e59370aaa7e3b2efe161" target='_blank'>
              Anoikis-Related Genes Impact Prognosis and Tumor Microenvironment in Bladder Cancer.
              </a>
            </td>
          <td>
            Tao Zhang, Shaojun Chen, Shanna Qu, Longsheng Wang
          </td>
          <td>2024-08-22</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Chimeric antigen receptor (CAR) T-cells have emerged as a ground-breaking therapy for the treatment of hematological malignancies due to their capacity for rapid tumor-specific killing and long-lasting tumor immunity. However, the same success has not been observed in patients with solid tumors. Largely, this is due to the additional challenges imposed by safe and uniform target selection, inefficient CAR T-cell access to sites of disease and the presence of a hostile immunosuppressive tumor microenvironment.


AREAS COVERED
Literature was reviewed on the PubMed database from the first description of a CAR by Kuwana, Kurosawa and colleagues in December 1987 through to the present day. This literature indicates that in order to tackle solid tumors, CAR T-cells can be further engineered with additional armoring strategies that facilitate trafficking to and infiltration of malignant lesions together with reversal of suppressive immune checkpoints that operate within solid tumor lesions.


EXPERT OPINION
In this review, we describe a number of recent advances in CAR T-cell technology that set out to combat the problems imposed by solid tumors including tumor recruitment, infiltration, immunosuppression, metabolic compromise, and hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c957915bda93838ef155e042bc947afb3392ab3" target='_blank'>
              CAR-T cell technologies that interact with the tumour microenvironment in solid tumours.
              </a>
            </td>
          <td>
            Chelsea Alice Taylor, Maya Glover, John Maher
          </td>
          <td>2024-07-17</td>
          <td>Expert review of clinical immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Immune checkpoint-inhibitors have become a standard treatment for clear cell renal cell carcinoma (ccRCC). However, responses to checkpoint-inhibition are heterogeneous and many patients are resistant or eventually develop resistance. A better understanding of functional mechanisms governing immune responses in ccRCC is needed to identify biomarkers for therapeutic responses and to unravel novel therapeutic targets to overcome primary or acquired resistance to therapies. Methods We developed a fully human explant model derived from freshly resected ccRCC tissue (Fig. 1A). Our tissue explant model preserves all components of the tumor microenvironment including not only tumor cells, but also the immune cell compartment as well as tumor-stroma and -vasculature in culture (Fig. 1B). Following treatment with combined checkpoint-inhibition (Nivolumab plus Ipilimumab) or an anti-CCR5 inhibitor (Maraviroc), the tissue explants were analyzed by immunohistochemistry staining and multiplex cytokine profiling of 50 cytokines. Spatial and functional insights were incorporated in an agent-based in silico model (PhysiCell, Ghaffarizadeh et al., PLoS Comput. Biol., 2018) for unlimited exploration of functional cellular dynamics in the tumor microenvironment (Fig. 1D). Results Immunohistochemical analyses and cytokine profiling of ccRCC-tissue explants showed heterogeneous immune responses among different patients and indicated an impaired cytotoxic T cell response following checkpoint-inhibition (Fig. 1C). Spatial analyses of immune cell populations revealed clusters of CD8+ T cells and CD163+ macrophages localized closely to CD31+ endothelial cells. Further characterization of the tumor microenvironment revealed high CCR5-expression in the tumor, particularly on the tumor blood vessels (Fig. 1B). Treatment of the tissue explants with the anti-CCR5 inhibitor Maraviroc led to an increase of CD8+ T cells and cytotoxic cytokines (Granzyme B, IFNgamma, TNFalpha, IFNalpha2) in the tumor in comparison to checkpoint-inhibition (Fig. 1C). Spatial and functional information from cultured ccRCC-tissue explants were integrated into an agent-based in silico model of the ccRCC tumor microenvironment, which includes tumor cells, T cells, macrophages and endothelial cells (Fig. 1D). In silico simulation of different immune cell modulating treatment conditions showed an increase of T cells and cytotoxic cytokines upon blockade of the interaction between T cells with endothelial cells or macrophages (Fig. 1E). Conclusions Our combined ex vivo and in silico analyses provide evidence for immunosuppression in the ccRCC tumor microenvironment mediated by macrophages and endothelial cells. CCR5 was shown to be a potential target to overcome immune resistance to checkpoint-inhibition in ccRCC. Further characterization of immune cell compositions and functional mechanisms in the perivascular area is imperative to enhance therapeutic responses to immunotherapies for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78af278232c241a08a9645b6c5f42c4fea1d9f55" target='_blank'>
              30 Synergistic systematic analysis with fully human tissue models andin silico modeling of “copycats” reveals mechanisms of T cell suppression in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xin-Wen Zhang, Alexandra Pöchmann, Xu Holtkotte, M. Suarez-Carmona, Prof Stefan Duensing, S. Zschäbitz, Prof Dirk Jäger, Prof Matthias Gaida, Prof Niels Halama
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma is the most common and malignant primary brain tumor, with high morbidity and mortality. Despite an aggressive, multimodal treatment regimen, including surgical resection followed by chemotherapy and radiotherapy, the prognosis of glioblastoma patients remains poor. One formidable challenge to advancing glioblastoma therapy is the complexity of the tumor microenvironment. The tumor microenvironment of glioblastoma is a highly dynamic and heterogeneous system that consists of not only cancerous cells but also various resident or infiltrating inflammatory cells. These inflammatory cells not only provide a unique tumor environment for glioblastoma cells to develop and grow but also play important roles in regulating tumor aggressiveness and treatment resistance. Targeting the tumor microenvironment, especially neuroinflammation, has increasingly been recognized as a novel therapeutic approach in glioblastoma. In this review, we discuss the components of the tumor microenvironment in glioblastoma, focusing on neuroinflammation. We discuss the interactions between different tumor microenvironment components as well as their functions in regulating glioblastoma pathogenesis and progression. We will also discuss the anti-tumor microenvironment interventions that can be employed as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670eaa472170b926b6f530ad1202e654791b0745" target='_blank'>
              Neuroinflammation in Glioblastoma: Progress and Perspectives
              </a>
            </td>
          <td>
            Xin Li, Wenting Gou, Xiaoqin Zhang
          </td>
          <td>2024-07-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer arises through the accumulation of somatic mutations in key biological pathways. This paper aims to develop a probabilistic approach to delineate the temporal order of mutations during cancer development based on mutation profile data from a cohort of patients. A unique feature of our method is that it incorporates intra-tumor heterogeneity (ITH) information, which refers to the heterogeneous cell populations within a tumor and characterizes the evolutionary history of the tumor. We showed that by integrating ITH, pathways, and functional annotation information, our method yielded high accuracy in inferring the temporal order of pathway mutations during carcinogenesis. Abstract The development of cancer involves the accumulation of somatic mutations in several essential biological pathways. Delineating the temporal order of pathway mutations during tumorigenesis is crucial for comprehending the biological mechanisms underlying cancer development and identifying potential targets for therapeutic intervention. Several computational and statistical methods have been introduced for estimating the order of somatic mutations based on mutation profile data from a cohort of patients. However, one major issue of current methods is that they do not take into account intra-tumor heterogeneity (ITH), which limits their ability to accurately discern the order of pathway mutations. To address this problem, we propose PATOPAI, a probabilistic approach to estimate the temporal order of mutations at the pathway level by incorporating ITH information as well as pathway and functional annotation information of mutations. PATOPAI uses a maximum likelihood approach to estimate the probability of pathway mutational events occurring in a specific sequence, wherein it focuses on the orders that are consistent with the phylogenetic structure of the tumors. Applications to whole exome sequencing data from The Cancer Genome Atlas (TCGA) illustrate our method’s ability to recover the temporal order of pathway mutations in several cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296d55a31b561e1788fdfc0790cecd99495dadae" target='_blank'>
              A Probabilistic Approach to Estimate the Temporal Order of Pathway Mutations Accounting for Intra-Tumor Heterogeneity
              </a>
            </td>
          <td>
            Menghan Wang, Yanqi Xie, Jinpeng Liu, Austin Li, Li Chen, Arnold Stromberg, Susanne M. Arnold, Chunming Liu, Chi Wang
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a82685ebf391bea616973c3440cc99b04c6536" target='_blank'>
              Differential transcriptional invasion signatures from patient derived organoid models define a functional prognostic tool for head and neck cancer
              </a>
            </td>
          <td>
            Peter D. Haughton, W. Haakma, Theofilos Chalkiadakis, G. Breimer, E. Driehuis, Hans Clevers, Stefan M. Willems, Stefan Prekovic, Patrick W B Derksen
          </td>
          <td>2024-06-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genomic, epigenomic, and transcriptomic alterations are hallmarks of cancer cells, and are closely connected. Especially, epigenetic regulation plays a critical role in tumorigenesis and progression. The growing single-cell epigenome data in cancer research provide new opportunities for data mining from a more comprehensive perspective. However, there is still a lack of databases designed for interactively exploring the single-cell multi-omics data of human pan-cancer, especially for the single-cell epigenome data. To fill in the gap, we developed scCancerExplorer, a comprehensive and user-friendly database to facilitate the exploration of the single-cell genome, epigenome (chromatin accessibility and DNA methylation), and transcriptome data of 50 cancer types. Five major modules were provided to explore those data interactively, including "integrated multi-omics analysis", "single-cell transcriptome", "single-cell epigenome", "single-cell genome", and "TCGA analysis". By simple clicking, users can easily investigate gene expression features, chromatin accessibility patterns, transcription factor activities, DNA methylation states, copy number variations, and TCGA survival analysis results. Taken together, scCancerExplorer is distinguished from previous databases with rich and interactive functions for exploring the single-cell multi-omics data of human pan-cancer. It bridges the gap between single-cell multi-omics data and the end-users, and will facilitate progress in the field of cancer research. scCancerExplorer is freely accessible via https://bianlab.cn/scCancerExplorer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0bfe81970da09d476152acc53e069683a992ff" target='_blank'>
              scCancerExplorer: a comprehensive database for interactively exploring single-cell multi-omics data of human pan-cancer
              </a>
            </td>
          <td>
            Changzhi Huang, Zekai Liu, Yunlei Guo, Wanchu Wang, Zhen Yuan, Yusheng Guan, Deng Pan, Zhibin Hu, Linhua Sun, Zan Fu, Shuhui Bian
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer characterized by late diagnosis, rapid progression, and a high mortality rate. Its complex biology, characterized by a dense, stromal tumor environment with an immunosuppressive milieu, contributes to resistance against standard treatments like chemotherapy and radiation. This comprehensive review explores the dynamic role of the microbiome in modulating chemotherapy efficacy and outcomes in PDAC. It delves into the microbiome’s impact on drug metabolism and resistance, and the interaction between microbial elements, drugs, and human biology. We also highlight the significance of specific bacterial species and microbial enzymes in influencing drug action and the immune response in the tumor microenvironment. Cutting-edge methodologies, including artificial intelligence, low-biomass microbiome analysis and patient-derived organoid models, are discussed, offering insights into the nuanced interactions between microbes and cancer cells. The potential of microbiome-based interventions as adjuncts to conventional PDAC treatments are discussed, paving the way for personalized therapy approaches. This review synthesizes recent research to provide an in-depth understanding of how the microbiome affects chemotherapy efficacy. It focuses on elucidating key mechanisms and identifying existing knowledge gaps. Addressing these gaps is crucial for enhancing personalized medicine and refining cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33468af8b48c7327c4b215eb7932772751d9192f" target='_blank'>
              Microbiome and pancreatic cancer: time to think about chemotherapy
              </a>
            </td>
          <td>
            J. de Castilhos, Katharina Tillmanns, Jana Blessing, Arnelyn Laraño, Vadim Borisov, Christoph K Stein-Thoeringer
          </td>
          <td>2024-07-18</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background co-targeting of immune checkpoint inhibitors (ICI) CTLA-4 and PD-1 has recently become the new first-line standard of care therapy of pleural mesothelioma (PM) patients, with a significant improvement of overall survival over conventional chemotherapy. The analysis by tumor histotype demonstrated a greater efficacy of ICI therapy in non-epithelioid (non-E) vs epithelioid (E) PM; although some E PM patients also benefit from treatment. This evidence suggests that molecular tumor features, beyond histotype, could be relevant to improve the efficacy of ICI therapy in PM. Among these, tumor DNA methylation emerges as a promising factor to explore, due to its potential role in driving the immune phenotype of cancer cells. Thus, we utilized a panel of cultured PM cells of different histotype, to provide preclinical evidence supporting the role of the tumor methylation landscape and of its pharmacologic modulation, to prospectively improve the efficacy of ICI therapy of PM patients. Methods the methylome profile (EPIC array) of distinct E (#5) and non-E (#9) PM cell lines was analyzed, followed by integrated analysis with their associated transcriptomic profile (Clariom S array), before and after in vitro treatment with the DNA hypomethylating agent (DHA) guadecitabine. The most variable methylated probes were selected to calculate the methylation score (CIMP index) for each cell line at baseline. Genes that were differentially expressed and methylated were then selected for gene ontology analysis. Results the CIMP index stratified PM cell lines in two distinct classes, CIMP (hyper-methylated; #7) and LOW (hypo-methylated; #7), regardless of their E or non-E histotype. Integrated analyses of methylome and transcriptome data revealed that CIMP PM cells had a substantial number of hyper-methylated, silenced genes, which negatively impacted their immune phenotype compared to LOW PM cells. Treatment with DHA reverted the methylation-driven immune-compromised profile of CIMP PM cells and enhanced the constitutive immune-favorable profile of LOW PM cells. Conclusion the study highlighted the relevance of DNA methylation in shaping the constitutive immune classification of PM cells, that is independent from their histological subtypes. The identified role of DHA in shifting the phenotype of PM cells towards an immune-favorable state supports its role in clinical trials of precision epigenetic therapy combined with ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b753d8055100836c4c184dd41cdeb41298408ffc" target='_blank'>
              DNA methylation status classifies pleural mesothelioma cells according to their immune profile: implication for precision epigenetic therapy
              </a>
            </td>
          <td>
            M. F. Lofiego, Rossella Tufano, Emma Bello, Laura Solmonese, Francesco Marzani, Francesca Piazzini, Fabrizio Celesti, F. Caruso, Teresa Maria Rosaria Noviello, R. Mortarini, A. Anichini, Michele Ceccarelli, Luana Calabrò, Michele Maio, S. Coral, A. D. Di Giacomo, A. Covre
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d38ab5074cc28a8e5b738a908dc6864353dab59" target='_blank'>
              Optimizing immunofluorescence with high-dynamic-range imaging to enhance PD-L1 expression evaluation for 3D pathology assessment from NSCLC tumor tissue
              </a>
            </td>
          <td>
            Hsien-Neng Huang, Chun-Wei Kuo, Yu-Ling Hung, Chia-Hung Yang, Y. Hsieh, Yu-Chieh Lin, M. D. Chang, Yen-Yin Lin, Jen-Chung Ko
          </td>
          <td>2024-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cell division drives multicellular growth and its dysregulation can cause disease. While approximately 44 divisions are needed to produce without death all 1013 cells in the human body, current methods are limited to count 10 cell divisions in vivo across diverse mammalian cell types. Here we introduce the DivisionCounter, a method to count cell division in vivo over large division ranges (∼70 divisions) using an easy fluorescence readout. We demonstrate that the DivisionCounter accurately measures the average cell division count of cells both in vitro and in vivo. Its use revealed that leukemia tumor division rates are independent of the organ’s specific microenvironment and CAR-T cell treatment, providing an estimate of tumor death rates in vivo. The DivisionCounter method holds unique potential for quantifying contributions of cell division, death, and migration to the growth of healthy and pathological mammalian tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c8066076dbc5591bb50c9646e744b74ab90f62" target='_blank'>
              The DivisionCounter, a method for counting large ranges of cell divisions in vivo, reveals cell dynamics of leukemic cell killing via CAR-T therapy
              </a>
            </td>
          <td>
            Lucie Hustin, Cecile Conrad, Chang Liu, Jaime R Fuentealba, Silvia Menegatti, S. Shneer, A. Battistella, Fanny Tabarin, Tom S Weber, Sebastian Amigorena, Ken R Duffy, Leïla Perié
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16db7fe8f3bc4329cf344c224d8e623bae436c6d" target='_blank'>
              Elucidating the pan-oncologic landscape of S100A9: prognostic and therapeutic corollaries from an integrative bioinformatics and Mendelian randomization analysis
              </a>
            </td>
          <td>
            Yingying Chen, Zixuan Wu, Xingxing Yi
          </td>
          <td>2024-08-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aggrephagy, a type of autophagy, degrades the aggregation of misfolded protein in cells. However, the role of aggrephagy in multiple myeloma (MM) has not been fully demonstrated. In this study, we first investigated the correlation between aggrephagy signaling, MM immune microenvironment composition and disease prognosis. Single-cell RNA-seq data, including the expression profiles of 12,187 single cells from seven MM bone marrow (BM) and seven healthy BM samples, were analyzed by non-negative matrix factorization for 44 aggrephagy-related genes. Bulk RNA-seq cohorts from the Gene Expression Omnibus database were used to evaluate the prognostic value of aggrephagy-related immune cell subtypes and predict immune checkpoint blockade immunotherapeutic response in MM. Compared with healthy BM, MM BM exhibited different patterns of aggrephagy-related gene expression. In MM BM, macrophages, CD8+ T cells, B cells and natural killer cells could be grouped into four to nine aggrephagy-related subclusters. The signature of aggrephagy signaling molecule expression in the immune cells correlates with the patient's prognosis. Our investigation provides a novel view of aggrephagy signaling in MM tumor microenvironment cells, which might be a prognostic indicator and potential target for MM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7f506c9ce79ae57845d0dafd18ceb978803a7d" target='_blank'>
              Single-cell sequencing reveals the correlation of aggrephagy signaling and multiple myeloma immune microenvironment composition.
              </a>
            </td>
          <td>
            Xin Wang, Yu Feng, Fangfang Wang, Zhimei Lin, Jingcao Huang, Qian Li, Hongmei Luo, Xiang Liu, Xinyu Zhai, Qianwen Gao, Linfeng Li, Yue Zhang, Jingjing Wen, Li Zhang, Ting Niu, Yuhuan Zheng
          </td>
          <td>2024-07-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cell migration requires the interplay among diverse migration patterns. The molecular basis of distinct migration programs is undoubtedly vital but not fully explored. Meanwhile, the lack of tools for investigating spontaneous migratory plasticity in a single living cell also adds to the hindrance. Here, we developed a micro/nanotechnology-enabled single-cell analytical platform to achieve coherent monitoring of spontaneous migratory pattern and signaling molecules. Via the platform, we unveiled a previously unappreciated STAT3 regionalization on the multifunctional regulations of migration. Specifically, nuclear STAT3 is associated with amoeboid migration, while cytoplasmic STAT3 promotes mesenchymal movement. Opposing effects of JAK2 multisite phosphorylation shape its response to STAT3 distribution in a dynamic and antagonistic manner, eventually triggering a reversible amoeboid-mesenchymal transition. Based on the above results, bioinformatics further revealed a possible downstream regulator of nucleocytoplasmic STAT3. Thus, our platform, as an exciting technological advance in single-cell migration research, can provide in-depth mechanism interpretations of tumor metastasis and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7aa9042cbc5c64cbf53c127a1f559066cb19d" target='_blank'>
              Single Living Cell "Observation-Analysis" Integrated Platform Decodes Cell Migration Plasticity Orchestrated by Nucleocytoplasmic STAT3.
              </a>
            </td>
          <td>
            Anqi Zhang, Wanting Qu, Peixin Guan, Ying Li, Zhen Liu
          </td>
          <td>2024-06-28</td>
          <td>Nano letters</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f902f0c016306dc82479eae5dce47565c7056a" target='_blank'>
              Integrated immuno-transcriptomic analysis of ovarian cancer identifies a four-chemokine-dominated subtype with antitumor immune-active phenotype and favorable prognosis.
              </a>
            </td>
          <td>
            Lili Zhuo, Fanling Meng, Kaidi Sun, Meng Zhou, Jie Sun
          </td>
          <td>2024-08-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Advances in understanding the pathological mechanisms of breast cancer have resulted in the emergence of novel therapeutic strategies. However, triple-negative breast cancer (TNBC), a molecular subtype of breast cancer with a poor prognosis, lacks classical and general therapeutic targets, hindering the clinical application of several therapies to breast cancer. As insights into the unique immunity and molecular mechanisms of TNBC have become more extensive, immunotherapy has gradually become a valuable complementary approach to classical radiotherapy and chemotherapy. CD8+ cells are significant actors in the tumor immunity cycle; thus, research on TNBC immunotherapy is increasingly focused in this direction. Recently, CD8+ tissue-resident memory (TRM) cells, a subpopulation of CD8+ cells, have been explored in relation to breast cancer and found to seemingly play an undeniably important role in tumor surveillance and lymphocytic infiltration. In this review, we summarize the recent advances in the mechanisms and relative targets of CD8+ T cells, and discuss the features and potential applications of CD8+ TRM cells in non-luminal breast cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a9050f4d78074037b77f761a64e95aa5938b134" target='_blank'>
              The potential role of CD8+ cytotoxic T lymphocytes and one branch connected with tissue-resident memory in non-luminal breast cancer
              </a>
            </td>
          <td>
            Ziqi Zhao, Xinyu Ma, Zhengang Cai
          </td>
          <td>2024-07-10</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 How does the ovarian microenvironment, involving neural fibers, vascular networks, and immune cells, interplay with follicular dynamics in the human adult ovary?



 Neural fibers growing into the ovarian cortex and vascular network expansion were associated with primordial follicle activation and growth. Macrophages associated with antral follicular atresia.



 The ovarian follicles, responsible for preserving and supplying oocytes, play a crucial role in fertility, with their fate influenced by the ovarian microenvironment. However, the three-dimensional (3D), irregular, and dynamic structural changes of the ovarian microenvironment are difficult to interpret using two-dimensional analysis. 3D-imaging technologies have been developed to reveal intricate structural and functional relationships particularly in rodents, but these approaches have not been widely adopted for human ovarian research.



 Our study aims to explore the follicular microenvironment dynamics in the human adult ovary. In this work, we optimized whole-mount immunofluorescence using TUBB3, PODXL, PECAM1, ACTA2 and CD68, and conducted 3D imaging on intact adult ovarian tissue, offering a visualization of the 3D structures and dynamic changes within ovarian follicles, neural fibers, vascular networks, and immune cells in human adult ovary at the single-cell level.



 The human ovarian samples were collected from donors (N = 5) who underwent oophorectomy after testosterone treatment. Part of the intact ovary, including cortex and medulla, was fixed and used for whole-mount immunofluorescence using markers for neural fibers (TUBB3), blood vessels (PODXL, PECAM1, ACTA2), and immune cells (CD68). Subsequently, an enhanced iDiSCO method was employed for tissue clearance. Finally, 3D images were acquired using a lightsheet microscope, followed by image analysis and 3D reconstruction using Imaris software.



 Our results showed that TUBB3-labeled nerve fibers predominantly distributed in the ovarian cortex, with few presented in the medullary region. Notably, their presence surrounding secondary and antral follicles was limited. We observed a rich distribution of PODXL-labeled capillaries surrounding primordial and primary follicles in the cortical layer of the adult ovary. As follicles progressed to the secondary stage, PECAM1-labeled vasculature contributed to the construction of a vascular network enveloping the secondary follicles and connecting extensively to larger-caliber arteries containing (ACTA2+) smooth muscle cells. In antral follicles, large larger-caliber arteries were also detected, maintaining proximity to the theca cell layer. Interestingly, in cases of follicular atresia, vascular invasion within the follicular cavity was observed. Additionally, clusters of  CD68+ macrophages exhibited a scattered distribution around the follicles at various stages, with a significant accumulation in atretic small-antral follicles.



 Our study was conducted on ovaries from a limited number of donors after gender-affirming surgery. Although we aimed to provide insights into the ovarian microenvironment, we were so far unable to obtain ovaries with large antral follicles, ovulatory follicles and the formation of the corpus luteum.



 The views provided by 3D-image analysis of the human ovary will augment the current knowledge of the complex interactions between the follicles and the ovarian microenvironment. 3D ovarian mapping will also provide a roadmap for generating oocytes in vitro for fertility preservation and ovarian disease modelling.



 N/A.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54a1b1980fee1b20f1a25c06eb846532c2e418d" target='_blank'>
              P-693 Three-dimensional modeling of follicular microenvironment dynamics in the human adult ovary
              </a>
            </td>
          <td>
            F. Wei, H. Cheng, X. Fan, J. . S. D. Valle, J. Asseler, T. Stolk, W. . V. Vugt, L. . E. V. D. Meeren, N. . M. V. Mello, S. . M. C. D. S. Lopes
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="High-grade gliomas (HGGs) are the commonest primary brain cancers. They are characterized by a pattern of aggressive growth and diffuse infiltration of the host brain that severely limits the efficacy of conventional treatments and patient outcomes, which remain generally poor. Recent work has described a suite of mechanisms via which HGGs interact, predominantly bidirectionally, with various cell types in the host brain including neurons, glial cells, immune cells, and vascular elements to drive tumor growth and invasion. These insights have the potential to inspire novel approaches to HGG therapy that are critically needed. This review explores HGG-host brain interactions and considers whether and how they might be exploited for therapeutic gain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3db42f40290dec089f04b45fb5296f6f414506" target='_blank'>
              Breaking Down Glioma-Microenvironment Crosstalk.
              </a>
            </td>
          <td>
            Raghavskandhan Ramachandran, Alexander F Jeans
          </td>
          <td>2024-07-26</td>
          <td>The Neuroscientist : a review journal bringing neurobiology, neurology and psychiatry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
We describe the fibrotic rim formed in the desmoplastic growth pattern (DHGP) of colorectal cancer liver metastasis (CLM) using in situ sequencing (ISS). The origin of the desmoplastic rim is still a matter of debate, and the detailed cellular organization has not yet been fully elucidated. Understanding the biology of the DHGP in CLM can explore targeted treatment to improve survival.


EXPERIMENTAL DESIGN
We used ISS, targeting 150 genes, to characterize the desmoplastic rim by unsupervised clustering of gene co-expression patterns. The cohort comprised 10 chemo-naïve liver metastasis resection samples with a DHGP.


RESULTS
Unsupervised clustering of spatially mapped genes revealed molecular and cellular diversity within the desmoplastic rim. We confirmed the presence of the ductular reaction and cancer-associated fibroblasts. Importantly, we discovered angiogenesis and outer and inner zonation in the rim, characterized by NGFR and POSTN expression.


CONCLUSIONS
ISS enabled the analysis of the cellular organization of the fibrous rim surrounding CLM with a DHGP and suggests a transition from the outer part of the rim, with nonspecific liver injury response, into the inner part, with gene expression indicating collagen synthesis and extracellular matrix remodeling influenced by the interaction with cancer cells, creating a cancer cell supportive environment. Moreover, we found angiogenic processes in the rim. Our results provide a potential explanation of the origin of the rim in DHGP and lead to exploring novel targeted treatments for patients with CLM to improve survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9982955fd0cffba29cc733d363ff61eae9c26ea5" target='_blank'>
              Spatial transcriptome mapping of the desmoplastic growth pattern of colorectal liver metastases by in situ sequencing reveals a biologically relevant zonation of the desmoplastic rim.
              </a>
            </td>
          <td>
            Axel Andersson, Maria Escriva Conde, Olga Surova, Peter Vermeulen, C. Wählby, Mats Nilsson, Hanna Nyström
          </td>
          <td>2024-07-25</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Breast cancer (BC) represents a major public health issue. The effectiveness of neoadjuvant chemotherapy (NAC) varies among breast cancer patients, with some experiencing incomplete pathological responses. This variability in treatment response may be attributed to differences in tumor heterogeneity and its microenvironment (TME). This study investigates gene expression patterns associated to non-response to NAC in invasive BC patients, emphasizing the role of genes, pathways and the variability among BC subtypes. A transcriptomic study analyzed 58 baseline samples from women with advanced breast cancer at the Colombian National Cancer Institute, categorizing them into 29 NAC responders and 29 non-responders. The study comprised gene expression comparison, enrichment analysis, tumor microenvironment estimation via xCell, and therapeutic efficacy of targeted drugs using PrRophetic package. Different gene expression profiles distinguished responders from non-responders among various breast cancer subtypes, highlighting immune-related pathways like IL-17 and TNF signaling, B and T cell receptor signaling, complement and coagulation cascades, natural killer cell-mediated cytotoxicity, and NF-kB signaling. Non-responders in Luminal B HER2- subtype exhibited increased endothelial cells and immune and microenvironment scores, while Luminal B HER2+ non-responders showed higher levels of CD4 Tcm, CD4 Tem, and megakaryocytes. Sensitivity to multiple potential therapeutic drugs (Lestaurtinib, Avagacestat, GSK3 inhibitor, Veliparib, Tretinoin, Afatinib, Vinorelbine, RSK1 inhibitor, Motesanib) varied distinctly among non-responders. Immune-related genes and diverse immune cell subtypes within the TME correlate with response to NAC, with significant changes observed between response groups and subtypes. Comprehensive detection and evaluation of TME components are crucial for predicting NAC efficacy and preventing disease relapse. These findings underscore the importance of studying mixed populations to uncover novel insights and address disparities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3405f06f64129be498cc163c7032b460fe8bf47e" target='_blank'>
              Transcriptomic Features Associated to Neoadjuvant Chemotherapy Response in Four Molecular Breast Cancer Subtypes
              </a>
            </td>
          <td>
            Hedda Michelle Guevara-Nieto, Rafael Parra-Medina, Carlos Alberto Orozco, Alejandro Mejia-Garcia, Jovanny Zabaleta, L. López-Kleine, Alba-Lucia Combita-Rojas
          </td>
          <td>2024-07-01</td>
          <td>JCO Global Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: The standard of care for stage 1 NSCLC is upfront surgery followed by surveillance. However, 20–30% of stage 1 NSCLC recur. There is an unmet need to identify individuals likely to recur who would benefit from frequent monitoring and aggressive cancer treatments. Collagen 1 (Col1) fibers detected by second harmonic generation (SHG) microscopy are a major structural component of the extracellular matrix (ECM) of tumors that play a role in cancer progression. Method: We characterized Col1 fibers with SHG microscopy imaging of surgically resected stage 1 NSCLC. Gene expression from RNA sequencing data was used to validate the SHG microscopy findings. Results: We identified a significant (p ≤ 0.05) increase in the Col1 fiber volume in stage 1 NSCLC that recurred. The increase in Col1 fiber volume was supported by significant increases in the gene expression of Col1 in invasive, compared to noninvasive, lung adenocarcinoma. Significant differences were identified in the gene expression of other ECM proteins, as well as CAFs, immune checkpoint markers, immune cytokines, and T-cell markers. Conclusion: Col1 fiber analysis can provide a companion diagnostic test to evaluate the likelihood of tumor recurrence following stage 1 NSCLC. The studies expand our understanding of the role of the ECM in NSCLC recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44725624f1f075bfc0e7e921ca6f6dac14d40057" target='_blank'>
              Collagen 1 Fiber Volume Predicts for Recurrence of Stage 1 Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            S. Kakkad, B. Krishnamachary, Nadege Fackche, Matthew Garner, Malcom Brock, Peng Huang, Z. Bhujwalla
          </td>
          <td>2024-07-01</td>
          <td>Tomography</td>
          <td>0</td>
          <td>62</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [21],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>