<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-19 06:29:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E. Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of medulloblastoma (MB) influences progression and therapy response, presenting a promising target for therapeutic advances. Prior single-cell analyses have characterized the cellular components of the TME but lack spatial context. To address this, we performed spatial transcriptomic sequencing on sixteen pediatric MB samples obtained at diagnosis, including two matched diagnosis-relapse pairs. Our analyses revealed inter- and intra-tumoral heterogeneity within the TME, comprised of tumor-associated astrocytes (TAAs), macrophages (TAMs), stromal components, and distinct subpopulations of MB cells at different stages of neuronal differentiation and cell cycle progression. We identified dense regions of quiescent progenitor-like MB cells enriched in patients with high-risk (HR) features and an increase in TAAs, TAMs, and dysregulated vascular endothelium following relapse. Our study presents novel insights into the spatial architecture and cellular landscape of the medulloblastoma TME, highlighting spatial patterns linked to HR features and relapse, which may serve as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b39423470039d57ba772c250043854baef4821" target='_blank'>
              Medulloblastoma Spatial Transcriptomics Reveals Tumor Microenvironment Heterogeneity with High-Density Progenitor Cell Regions Correlating with High-Risk Disease
              </a>
            </td>
          <td>
            Franklin Chien, Marina E. Michaud, Mojtaba Bakhtiari, Chanel Schroff, M. Snuderl, José E. Velázquez Vega, Tobey J. MacDonald, Manoj K. Bhasin
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b2c5c7d1cda920e03f84144f36e08d5e8bdd9f" target='_blank'>
              Genomic barcoding for clonal diversity monitoring and control in cell-based complex antibody production
              </a>
            </td>
          <td>
            Niels Bauer, Christoph Oberist, Michaela Poth, Julian Stingele, Oliver Popp, Simon Ausländer
          </td>
          <td>2024-06-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) presents significant challenges due to its heterogeneity and late-stage diagnoses. Using single-cell and spatial transcriptomics to elucidate the complex landscape of HGSOC to understand its underlying mechanism. Our analysis reveals significant inter- and intra-tumoral diversity, manifested through distinct cellular subpopulations and varied microenvironmental niches. Notably, our findings highlight a widespread immunosuppressive environment, marked by complex networks of cell-cell interactions, particularly evident in areas of elevated tumor cell density within metastatic samples. We identify the exclusive presence of COL14A1+ neoplastic cells in metastatic specimens, alongside a strong correlation between CD8A+ NKT cells and poor prognosis, and elevated CHODL expression in HGSOC metastasis tissues. Furthermore, knockdown experiments targeting CHODL demonstrate its role in reducing migration and invasion abilities in HGSOC cells. A pivotal discovery of our study is the delineation of specific cellular signatures correlated with adverse outcomes, notably a subset of CHODL+ neoplastic cells characterized by a distinct metabolic phenotype with a predilection for lipid metabolism. The therapeutic targeting of this metabolic pathway with existing inhibitors appears promising in curbing tumor proliferation. These findings enhance our understanding of HGSOC heterogeneity and reveal potential therapeutic targets, promising more effective management strategies for this aggressive cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ff004a52301690996ef94c99200186510d19e37" target='_blank'>
              Integrating single-cell and spatial transcriptomic analysis to unveil heterogeneity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Haixia Luo, Kunyu Wang, Bin Li
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The patterns by which primary tumors spread to metastatic sites remain poorly understood. Here, we define patterns of metastatic seeding in prostate cancer (PCa) using a novel injection-based mouse model - EvoCaP (Evolution in Cancer of the Prostate), featuring aggressive metastatic cancer to bone, liver, lungs, and lymph nodes. To define migration histories between primary and metastatic sites, we used our EvoTraceR pipeline to track distinct tumor clones containing recordable barcodes. We detected widespread intratumoral heterogeneity from the primary tumor in metastatic seeding, with few clonal populations (CPs) instigating most migration. Metastasis-to-metastasis seeding was uncommon, as most cells remained confined within the tissue. Migration patterns in our model were congruent with human PCa seeding topologies. Our findings support the view of metastatic PCa as a systemic disease driven by waves of aggressive clones expanding their niche, infrequently overcoming constraints that otherwise keep them confined in the primary or metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7ba92880acaf1b934f559d90646f82a1f001d3" target='_blank'>
              Clonal Lineage Tracing with Somatic Delivery of Recordable Barcodes Reveals Migration Histories of Metastatic Prostate Cancer.
              </a>
            </td>
          <td>
            Ryan N Serio, Armin Scheben, Billy Lu, Domenic V Gargiulo, Lucrezia Patruno, Caroline L Buckholtz, Ryan J Chaffee, Megan C Jibilian, Steven G Persaud, Stephen J. Staklinski, Rebecca Hassett, Lise M Brault, Daniele Ramazzotti, Christopher E Barbieri, Adam Siepel, Dawid G Nowak
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dcce816f6a19dca7baf05569c687c99b3105e9" target='_blank'>
              Unbiasedly decoding the tumor microenvironment with single-cell multiomics analysis in pancreatic cancer
              </a>
            </td>
          <td>
            Yifan Fu, Jinxin Tao, Tao Liu, Yueze Liu, J. Qiu, Dan Su, Ruobing Wang, W. Luo, Zhe Cao, Guihu Weng, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-07-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The current challenge in analyzing sequencing-based spatial transcriptomics (ST) is to precisely resolve the spatial distribution of cell subpopulations with low abundance and detect context-dependent variation in cell states within complex tissue microenvironments. Here, we introduce an ultra-resolution ST deconvolution algorithm (UCASpatial) that improves the resolution of mapping cell subpopulations to spatial locations by leveraging the contribution of genes indicative of cell identity through entropy-based weighting. Using both in silico and real ST datasets, we demonstrate that UCASpatial improves the robustness and accuracy in identifying low-abundant cell subpopulations and distinguishing transcriptionally heterogeneous cell subpopulations. Applying UCASpatial to murine wound healing, we recapitulate known spatiotemporal dynamics of multiple cell subpopulations involved in wound healing and reveal the spatial segregation of distinct macrophage subpopulations embedded within different cellular communities at the wound center. In human colorectal cancer (CRC), we link genomic alterations in individual cancer clones to multi-cellular characteristics of the tumor immune microenvironment (TIME) and reveal the co-evolution of tumor cells and TIME within the same tumor. We show that the copy number gain on chromosome 20q (chr20q-gain) in tumor cells orchestrates a T cell-excluded TIME, indicative of resistance to immunotherapy in CRC, and is associated with tumor-intrinsic endogenous retrovirus silencing and impaired type I interferon response. Our findings present UCASpatial as a versatile tool for deciphering ultra-resolution cellular landscapes in ST and exploring intercellular mechanisms in complex and dynamic microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36aa9aa2d8c5c7dbae3c0976fedf2735c37f839" target='_blank'>
              Ultra-resolution Deconvolution of Spatial Transcriptomics Unveils Spatiotemporal Cellular Dynamics in Complex Microenvironments
              </a>
            </td>
          <td>
            Yin Xu, Zurui Huang, Yawei Zhang, Zhenghang Wang, Peijin Guo, Feifan Zhang, Wenxuan Gong, Guanghao Liang, Boyuan Mei, Lihui Dong, Renbao Chang, Minghui Gong, Yu Xia, Haochen Ni, Jing Yang, Yuan Gao, Zhaoqi Liu, Lin Shen, Jian Li, Meng Michelle Xu, Dali Han
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the spatial organization of the bone marrow (BM) microenvironment at single-cell resolution constitutes a challenge in multiple myeloma (MM). Using formalin-fixed paraffin-embedded (FFPE) samples from the MIcγ1 mouse model of MM and aged-matched healthy YFPcγ1 mice, we performed spatial transcriptional profiling with Visium Spatial Gene Expression. A custom data-analysis framework that combines spatial with single-cell transcriptomic profiling defined the BM cellular composition and established specific cell relations, visualizing the spatial distribution of transcriptionally heterogeneous MM plasma cells (MM-PC). MM pathogenesis transcriptional programs were spatially delineated within the BM microenvironment. A high-to-low MM-PC density gradient spatially correlated with effector-to-exhausted T cell phenotype abundance. In this context, MM cells in high-density MM-PC areas coexisted with dendritic cells while displaced neutrophils to the tumor border. Increased neutrophil extracellular trap formation, IL-17-driven inflammatory signaling, and osteoclast differentiation were spatially delineated within the BM microenvironment. The spatial identification of different areas of BM and the interaction between malignant cells and their microenvironment were validated in FFPE BM biopsies from MM patients with varying degrees of MM-PC infiltration. In summary, spatial transcriptomics depicts the BM cellular architecture of MM and reveals deregulated mechanisms underlying MM intercellular communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6b45562f865baeadce9704ad2edc7a5b178a26" target='_blank'>
              Spatial Transcriptomics Reveals a Myeloma Cell Architecture with Dysfunctional T-Cell Distribution, Neutrophil Traps, and Inflammatory Signaling
              </a>
            </td>
          <td>
            Laura Sudupe, Emma Muiños-López, A. Lopez-Perez, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, Paula Aguirre-Ruiz, Patxi San Martín-Úriz, Marta Larrayoz, Laura Alvarez-Gigli, Marta Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho González, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Solid carcinomas are often highly heterogenous cancers, arising from multiple epithelial cells of origin. Yet, how the cell of origin influences the response of the tumor microenvironment is poorly understood. Lung adenocarcinoma (LUAD) arises in the distal alveolar epithelium which is populated primarily by alveolar epithelial type I (AT1) and type II (AT2) cells. It has been previously reported that Gramd2+ AT1 cells can give rise to a histologically-defined LUAD that is distinct in pathology and transcriptomic identity from that arising from Sftpc+ AT2 cells1,2. To determine how cells of origin influence the tumor immune microenvironment (TIME) landscape, we comprehensively characterized transcriptomic, molecular, and cellular states within the TIME of Gramd2+ AT1 and Sftpc+ AT2-derived LUAD using KRASG12D oncogenic driver mouse models. Myeloid cells within the Gramd2+AT1-derived LUAD TIME were increased, specifically, immunoreactive monocytes and tumor associated macrophages (TAMs). In contrast, the Sftpc+ AT2 LUAD TIME was enriched for Arginase-1+ myeloid derived suppressor cells (MDSC) and TAMs expressing profiles suggestive of immunosuppressive function. Validation of immune infiltration was performed using flow cytometry, and intercellular interaction analysis between the cells of origin and major myeloid cell populations indicated that cell-type specific markers SFTPD in AT2 cells and CAV1 in AT1 cells mediated unique interactions with myeloid cells of the differential immunosuppressive states within each cell of origin mouse model. Taken together, Gramd2+ AT1-derived LUAD presents with an anti-tumor, immunoreactive TIME, while the TIME of Sftpc+ AT2-derived LUAD has hallmarks of immunosuppression. This study suggests that LUAD cell of origin influences the composition and suppression status of the TIME landscape and may hold critical implications for patient response to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8e7b3edd38c2a1ddeaa53c137b22e2c605b672" target='_blank'>
              Cell of origin alters myeloid-mediated immunosuppression in lung adenocarcinoma
              </a>
            </td>
          <td>
            Minxiao Yang, Noah Shulkin, Edgar Gonzalez, Jonathan Castillo, Chunli Yan, Keqiang Zhang, Leonidas Arvanitis, Zea Borok, W. Wallace, Dan Raz, E. R. Torres, C. Marconett
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Biliary tract cancers (BTCs) are a group of deadly malignancies encompassing intrahepatic and extrahepatic cholangiocarcinoma, gallbladder carcinoma, and ampullary carcinoma. Here, we present the integrative analysis of 63 BTC cell lines via multi-omics clustering and genome- scale CRISPR screens, providing a platform to illuminate BTC biology and inform therapeutic development. We identify dependencies broadly enriched in BTC compared to other cancers as well as dependencies selective to the anatomic subtypes. Notably, cholangiocarcinoma cell lines are stratified into distinct lineage subtypes based on biliary or dual biliary/hepatocyte marker signatures, associated with dependency on specific lineage survival factors. Transcriptional analysis of patient specimens demonstrates the prognostic significance of these lineage subtypes. Additionally, we delineate strategies to enhance targeted therapies or to overcome resistance in cell lines with key driver gene mutations. Furthermore, clustering based on dependencies and proteomics data elucidates unexpected functional relationships, including a BTC subgroup with partial squamous differentiation. Thus, this cell line atlas reveals potential therapeutic targets in molecularly defined BTCs, unveils biologically distinct disease subtypes, and offers a vital resource for BTC research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef975abc103c925058ab977fa358ad6a654baecf" target='_blank'>
              Generation of a biliary tract cancer cell line atlas reveals molecular subtypes and therapeutic targets
              </a>
            </td>
          <td>
            Vindhya Vijay, Negin Karisani, Lei Shi, Yu-Han Hung, Phuong Vu, Prabhat Kattel, Lauren Kenney, J. Merritt, Ramzi Adil, Qibiao Wu, Yuanli Zhen, Robert Morris, J. Kreuzer, Meena Kathiresan, Xcanda Ixchel Herrera Lopez, Haley Ellis, Ilaria Gritti, Lilian Lecorgne, Ines Farag, Alexandra Popa, William Shen, Hiroyuki Kato, Qin Xu, E. Balasooriya, Meng-Ju Wu, Saireudee Chaturantabut, R. Kate Kelley, James M. Cleary, Michael S. Lawrence, David Root, C. Benes, Vikram Deshpande, D. Juric, William R. Sellers, Cristina R. Ferrone, Wilhelm Haas, Francisca Vazquez, G. Getz, N. Bardeesy
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>195</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rationale: The heterogeneity of tumor cells within the glioblastoma (GBM) microenvironment presents a complex challenge in curbing GBM progression. Understanding the specific mechanisms of interaction between different GBM cell subclusters and non-tumor cells is crucial. Methods: In this study, we utilized a comprehensive approach integrating glioma single-cell and spatial transcriptomics. This allowed us to examine the molecular interactions and spatial localization within GBM, focusing on a specific tumor cell subcluster, GBM subcluster 6, and M2-type tumor-associated macrophages (M2 TAMs). Results: Our analysis revealed a significant correlation between a specific tumor cell subcluster, GBM cluster 6, and M2-type TAMs. Further in vitro and in vivo experiments demonstrated the specific regulatory role of the CEBPB transcriptional network in GBM subcluster 6, which governs its tumorigenicity, recruitment of M2 TAMs, and polarization. This regulation involves molecules such as MCP1 for macrophage recruitment and the SPP1-Integrin αvβ1-Akt signaling pathway for M2 polarization. Conclusion: Our findings not only deepen our understanding of the formation of M2 TAMs, particularly highlighting the differential roles played by heterogeneous cells within GBM in this process, but also provided new insights for effectively controlling the malignant progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f32ae634b60547b99c952f2958aef6e7b9e1f2ae" target='_blank'>
              The CEBPB+ glioblastoma subcluster specifically drives the formation of M2 tumor-associated macrophages to promote malignancy growth
              </a>
            </td>
          <td>
            Yongchang Yang, Xingyu Jin, Yang Xie, Chunlan Ning, Yiding Ai, Haotian Wei, Xing Xu, Xianglian Ge, Tailong Yi, Qiang Huang, Xuejun Yang, Tao Jiang, Xiaoguang Wang, Yingzhe Piao, Xun Jin
          </td>
          <td>2024-07-02</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Motivation Cells exhibit a wide array of morphological features, enabling computer vision methods to identify and track relevant parameters. Morphological analysis has long been implemented to identify specific cell types and cell responses. Here we asked whether morphological features might also be used to classify transcriptomic subpopulations within in vitro cancer cell lines. Identifying cell subpopulations furthers our understanding of morphology as a reflection of underlying cell phenotype and could enable a better understanding of how subsets of cells compete and cooperate in disease progression and treatment. Results We demonstrate that cell morphology can reflect underlying transcriptomic differences in vitro using convolutional neural networks. First, we find that changes induced by chemotherapy treatment are highly identifiable in a breast cancer cell line. We then show that the intra cell line subpopulations that comprise breast cancer cell lines under standard growth conditions are also identifiable using cell morphology. We find that cell morphology is influenced by neighborhood effects beyond the cell boundary, and that including image information surrounding the cell can improve model discrimination ability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116b8716438d464b34fa34caed09100f7664c755" target='_blank'>
              Deep learning identifies heterogeneous subpopulations in breast cancer cell lines
              </a>
            </td>
          <td>
            T. Jost, Andrea L. Gardner, Daylin Morgan, Amy Brock
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Targeted therapies can induce strong tumor regression, but they typically fail to eradicate metastatic cancers. The elucidation of the causes that enable cancers to survive within a residual disease is essential for finding eradication strategies. The ability of cancers to survive eradication reflects not only cell-intrinsic sensitivities, but also microenvironmental effects. Paracrine signals produced by fibroblast, non-neoplastic cells that make tumor stroma, can provide strong but spatially limited therapy protection. Even though this phenomenon is well-established, its contribution to the ability of tumors to escape eradication remains poorly understood. To address this gap of knowledge, we developed an in silico model that recapitulates the effect of stroma on therapy responses in tumor tissues. This model enabled us to evaluate the contribution of spatial aspects of stroma-mediated resistance. Our analyses reveal that stroma dispersal might be the most important yet overlooked aspect of stromal resistance that determines the overall tumor responses to therapy. Abstract The response of tumors to anti-cancer therapies is defined not only by cell-intrinsic therapy sensitivities but also by local interactions with the tumor microenvironment. Fibroblasts that make tumor stroma have been shown to produce paracrine factors that can strongly reduce the sensitivity of tumor cells to many types of targeted therapies. Moreover, a high stroma/tumor ratio is generally associated with poor survival and reduced therapy responses. However, in contrast to advanced knowledge of the molecular mechanisms responsible for stroma-mediated resistance, its effect on the ability of tumors to escape therapeutic eradication remains poorly understood. To a large extent, this gap of knowledge reflects the challenge of accounting for the spatial aspects of microenvironmental resistance, especially over longer time frames. To address this problem, we integrated spatial inferences of proliferation-death dynamics from an experimental animal model of targeted therapy responses with spatial mathematical modeling. With this approach, we dissected the impact of tumor/stroma distribution, magnitude and distance of stromal effects. While all of the tested parameters affected the ability of tumor cells to resist elimination, spatial patterns of stroma distribution within tumor tissue had a particularly strong impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d34125347f89218581236d182c69f2cff0ff55a" target='_blank'>
              Dissecting the Spatially Restricted Effects of Microenvironment-Mediated Resistance on Targeted Therapy Responses
              </a>
            </td>
          <td>
            Tatiana Miti, Bina Desai, D. Miroshnychenko, D. Basanta, A. Marusyk
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have heralded a remarkable shift in cancer care, significantly extending survival for advanced cancer patients. However, despite their remarkable clinical successes, a substantial majority of patients fail to achieve a lasting response to ICIs. To address this challenge and gain insights into the complex landscape of the tumor microenvironment (TME), we conducted an extensive analysis using single-cell RNA sequencing (scRNA; ∼216K cells across 39 samples) and single-nucleus transposase-accessible chromatin sequencing (snATAC; ∼43K cells from 15 samples) in a metastatic melanoma cohort. This systematic approach delineates 14 distinct cell types and 55 cell subtypes, including the identification of 15 transcriptional hallmarks of malignant cells. Through correlation analysis of cell subtype proportions, we unveiled six distinct clusters associated with varying tumor responses. Particularly intriguing was the identification of the mature DC enriched in immunoregulatory molecules (mregDC) subtype exhibiting correlations with naive T and B cells, forming an anti-tumor program that underscores the importance of multiple immune cell types in mediating anti-tumor immunity. Notably, we found that mregDC abundance represents a good prognostic predictor of progression-free survival (PFS) in the context of ICI treatment, and when combined with the TCF7+/- CD8 T cell ratio, it reliably predicts patient PFS across treatments beyond ICI. We validated our findings using an independent cohort of 274 ICI-treated melanoma samples analyzed using tissue-level expression. We next compared mregDCs and conventional dendritic cell types 1 and 2 (cDC1 and cDC2) using transcriptome signature, differentiation trajectory, interactome, cytokine milieu, and epigenome landscape analyses. This comparative analysis shed light on the unique attributes of mregDCs within the TME. Finally, we investigated cell type/subtype-specific genes, pathways, immune response enrichment, and ligand-receptor interactions closely associated with the proportions of mregDCs within the TME. These molecular and cellular insights, with their critical roles in enhancing the immune response against cancer, offer valuable prospects for predicting the efficacy of ICI regimens, and potentially guiding the selection of rational combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd285dffb8067a1b036abc12a6336c80dd568b9" target='_blank'>
              Mature and migratory dendritic cells promote immune infiltration and response to anti-PD-1 checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Jie-Ying Yang, Cassia Wang, Doris Fu, Li-Lun Ho, K. Galani, Lee Chen, Dennie T. Frederick, Liang He, Amy Y. Huang, Emily J. Robitschek, Sandeep K. Yadav, Wentao Deng, Rahul Tacke, Kelly P. Burke, T. Sharova, Kunal Rai, David Liu, Genevieve M. Boland, M. Kellis
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Intracerebral hemorrhage (ICH) poses a formidable challenge in stroke management, with limited therapeutic options, particularly in the realm of immune‐targeted interventions. Clinical trials targeting immune responses post‐ICH have encountered setbacks, potentially attributable to the substantial cellular heterogeneity and intricate intercellular networks within the brain. Here, we present a pioneering investigation utilizing single‐cell RNA sequencing and spatial transcriptome profiling at hyperacute (1 h), acute (24 h), and subacute (7 days) intervals post‐ICH, aimed at unraveling the dynamic immunological landscape and spatial distributions within the cerebral tissue. Our comprehensive analysis revealed distinct cell differentiation patterns among myeloid and lymphocyte populations, along with delineated spatial distributions across various brain regions. Notably, we identified a subset of lymphocytes characterized by the expression of Spp1 and Lyz2, termed macrophage‐associated lymphocytes, which exhibited close interactions with myeloid cells. Specifically, we observed prominent interactions between Lgmn+Macro‐T cells and microglia through the spp1–cd44 pathway during the acute phase post‐ICH in the choroid plexus. These findings represent a significant advancement in our understanding of immune cell dynamics at single‐cell resolution across distinct post‐ICH time points, thereby laying the groundwork for exploring critical temporal windows and informing the development of targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af27d3cf37340430057062faa5cf04622e91eea" target='_blank'>
              Unraveling dynamic immunological landscapes in intracerebral hemorrhage: insights from single‐cell and spatial transcriptomic profiling
              </a>
            </td>
          <td>
            Lingui Gu, Hualin Chen, Mingjiang Sun, Yihao Chen, Qinglei Shi, Jianbo Chang, Junji Wei, Wenbin Ma, X. Bao, Renzhi Wang
          </td>
          <td>2024-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, still most patients experience limited or no clinical benefit. The immune component dynamics during treatment and how it correlated with treatment efficacy remained largely known. We hypothesize that understanding the modulation dynamics of the TME upon ICI treatment would be key to elucidate the mechanisms of response and resistance. To systematically characterize these dynamics, we aggregated a large-scale single-cell RNA-sequencing (scRNA-seq) dataset of 141 patients across multiple cancer types and ICI treatment modalities, with longitudinally paired samples. We developed a robust, integration-free, and cluster-free deep phenotyping framework, leveraging multiple pan-cancer references to annotate 876,410 high-quality cells into 80 granular cell states. Our analysis revealed consistent compositional changes in 17 cell subtypes following ICI treatment, including an enhanced adaptive immune response such as CD8+ IL7R+ central memory T cells, GZMK+ effector memory T cells, germinal center B cells, precursor exhausted T cells, and CD4+ TfhTh1 cells, accompanied by a broad reduction in interferon responsive immune cells quiescent CD4+CREM-memory T cells. We uncovered co-regulated cell communities within the TME, highlighting the coordinated interplay between adaptive and innate immune cells, as well as immune and non-immune components. Furthermore, we identified two distinct patient groups exhibiting tightly correlated cellular dynamics within the TME post-treatment. The first group, enriched for responders, displayed a marked expansion of naive lymphocytes, while the second group, predominantly composed of non-responders, showed an increased abundance of immune experienced/suppressive cell states. This dichotomy in TME dynamics offers a potential predictive biomarker for patient stratification and personalized therapeutic strategies. Our study presents a comprehensive landscape of the cellular dynamics within the TME during ICI treatment, enabled by a powerful deep phenotyping approach showcasing the importance of a systems-level understanding of the TME dynamics in improving patient stratification and advancing personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e90093265f7348aad7b591e51e89915aa6d67b" target='_blank'>
              Cellular Dynamics Upon Immune Checkpoint Inhibition
              </a>
            </td>
          <td>
            Zhongyang Lin, Eketerina Petrenko, Mitalee Chandra, Nalini Srinivas, Selma Ugurel, Jürgen C. Becker, Dvir Aran
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cancers evolve in a dynamic ecosystem. Thus, characterizing cancer’s ecological dynamics is crucial to understanding cancer evolution and can lead to discovering novel biomarkers to predict disease progression. Ductal carcinoma in situ (DCIS) is an early-stage breast cancer characterized by abnormal epithelial cell growth confined within the milk ducts. Although there has been extensive research on genetic and epigenetic causes of breast carcinogenesis, none of these studies have successfully identified a biomarker for the progression and/or upstaging of DCIS. In this study, we show that ecological habitat analysis of hypoxia and acidosis biomarkers can significantly improve prediction of DCIS upstaging. First, we developed a novel eco-evolutionary designed approach to define habitats in the tumor intra-ductal microenvironment based on oxygen diffusion distance in our DCIS cohort of 84 patients. Then, we identify cancer cells with metabolic phenotypes attributed to their habitat conditions, such as the expression of CA9 indicating hypoxia responding phenotype, and LAMP2b indicating a hypoxia-induced acid adaptation. Traditionally these markers have shown limited predictive capabilities for DCIS upstaging, if any. However, when analyzed from an ecological perspective, their power to differentiate between indolent and upstaged DCIS increased significantly. Second, using eco-evolutionary guided computational and digital pathology techniques, we discovered distinct spatial patterns of these biomarkers and used the distribution of such patterns to predict patient upstaging. The patterns were characterized by both cellular features and spatial features. With a 5-fold validation on the biopsy cohort, we trained a random forest classifier to achieve the area under curve(AUC) of 0.74. Our results affirm the importance of using eco-evolutionary-designed approaches in biomarkers discovery studies in the era of digital pathology by demonstrating the role of eco-evolution dynamics in predicting cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786cf3e3e85927b01c56064b452b334cb91ffed1" target='_blank'>
              Eco-evolutionary Guided Pathomic Analysis to Predict DCIS Upstaging
              </a>
            </td>
          <td>
            Yujie Xiao, Manal Elmasry, Ji Dong K. Bai, Andrew Chen, Yuzhu Chen, Brooke Jackson, Joseph O. Johnson, Robert J. Gillies, Prateek Prasanna, Chao Chen, Mehdi Damaghi
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma, a malignant skin cancer arising from melanocytes, exhibits rapid metastasis and a high mortality rate, especially in advanced stages. Current treatment modalities, including surgery, radiation, and immunotherapy, offer limited success, with immunotherapy using immune checkpoint inhibitors (ICIs) being the most promising. However, the high mortality rate underscores the urgent need for robust, non-invasive biomarkers to predict patient response to adjuvant therapies. The immune microenvironment of melanoma comprises various immune cells, which influence tumor growth and immune response. Melanoma cells employ multiple mechanisms for immune escape, including defects in immune recognition and epithelial-mesenchymal transition (EMT), which collectively impact treatment efficacy. Single-cell analysis technologies, such as single-cell RNA sequencing (scRNA-seq), have revolutionized the understanding of tumor heterogeneity and immune microenvironment dynamics. These technologies facilitate the identification of rare cell populations, co-expression patterns, and regulatory networks, offering deep insights into tumor progression, immune response, and therapy resistance. In the realm of biomarker discovery for melanoma, single-cell analysis has demonstrated significant potential. It aids in uncovering cellular composition, gene profiles, and novel markers, thus advancing diagnosis, treatment, and prognosis. Additionally, tumor-associated antibodies and specific genetic and cellular markers identified through single-cell analysis hold promise as predictive biomarkers. Despite these advancements, challenges such as RNA-protein expression discrepancies and tumor heterogeneity persist, necessitating further research. Nonetheless, single-cell analysis remains a powerful tool in elucidating the mechanisms underlying therapy response and resistance, ultimately contributing to the development of personalized melanoma therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824bd1e506d1d79fb980c0e85de9df9031024731" target='_blank'>
              Advancing immunotherapy for melanoma: the critical role of single-cell analysis in identifying predictive biomarkers
              </a>
            </td>
          <td>
            Ru He, Jiaan Lu, Jianglong Feng, Ziqing Lu, Kaixin Shen, Ke Xu, Huiyan Luo, Guanhu Yang, Hao Chi, Shangke Huang
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Simple Summary This study investigates the role of visual intratumor heterogeneity (ITH) in breast cancer progression. By analyzing histologic images from the Carolina Breast Cancer Study (CBCS) and the Cancer Genome Atlas Breast Invasive Carcinoma (TCGA-BRCA) data using advanced image processing and machine learning techniques, we developed a measure of tumor heterogeneity based on visual features. Our findings indicate that tumors with low visual heterogeneity exhibited a higher risk of recurrence and were more likely to come from patients whose tumors comprised of only one subclone or had a TP53 mutation. Conversely, high visual heterogeneity was correlated with a more favorable prognosis. These results suggest that visual heterogeneity provides complementary information to molecular markers. A comprehensive understanding of both the visual and molecular aspects of heterogeneity has the potential to offer novel insights for treatment strategies. Abstract High intratumoral heterogeneity is thought to be a poor prognostic indicator. However, the source of heterogeneity may also be important, as genomic heterogeneity is not always reflected in histologic or ‘visual’ heterogeneity. We aimed to develop a predictor of histologic heterogeneity and evaluate its association with outcomes and molecular heterogeneity. We used VGG16 to train an image classifier to identify unique, patient-specific visual features in 1655 breast tumors (5907 core images) from the Carolina Breast Cancer Study (CBCS). Extracted features for images, as well as the epithelial and stromal image components, were hierarchically clustered, and visual heterogeneity was defined as a greater distance between images from the same patient. We assessed the association between visual heterogeneity, clinical features, and DNA-based molecular heterogeneity using generalized linear models, and we used Cox models to estimate the association between visual heterogeneity and tumor recurrence. Basal-like and ER-negative tumors were more likely to have low visual heterogeneity, as were the tumors from younger and Black women. Less heterogeneous tumors had a higher risk of recurrence (hazard ratio = 1.62, 95% confidence interval = 1.22–2.16), and were more likely to come from patients whose tumors were comprised of only one subclone or had a TP53 mutation. Associations were similar regardless of whether the image was based on stroma, epithelium, or both. Histologic heterogeneity adds complementary information to commonly used molecular indicators, with low heterogeneity predicting worse outcomes. Future work integrating multiple sources of heterogeneity may provide a more comprehensive understanding of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18cc90a062bf3cee9a0bd73529b68cc2b5c51dfd" target='_blank'>
              Visual Intratumor Heterogeneity and Breast Tumor Progression
              </a>
            </td>
          <td>
            Yao Li, Sarah C. Van Alsten, Dong Neuck Lee, Taebin Kim, Benjamin C. Calhoun, C. Perou, Sara E. Wobker, J. S. Marron, Katie Hoadley, M. A. Troester
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>160</td>
        </tr>

        <tr id="Melanomas are the deadliest skin cancers, in part due to cellular plasticity and heterogeneity. Intratumoral heterogeneity drives varied mutable phenotypes, specifically “melanocytic” and “mesenchymal” cell states, which result in differential functional properties and drug responses. Definitive and rigorous classification of these phenotypic states has been challenging with conventional biomarker-based methods, and high-parameter molecular methods are cell-destructive, labor-intensive, and time-consuming. To overcome these technical and practical limitations, we utilized label-free artificial intelligence-based morphological profiling to classify live melanoma cells into melanocytic and mesenchymal phenotypes based on high resolution imaging of single cells. To predict the phenotypes of single melanoma cells based on morphology alone, we developed the AI-based ‘Melanoma Phenotype Classifier’ trained with 19 patient-derived cell lines with known melanocytic or mesenchymal transcriptional profiles. To link phenotypic state with high-dimensional morphological profiles, cells were subjected to genetic and chemical perturbations known to shift phenotypic states. The AI classifier successfully predicted phenotypic shifts which were confirmed by single-cell RNA-Seq (scRNA-Seq). These results demonstrate that correlations between melanoma cell phenotypes and morphological changes are detectable by AI. Additionally, the Melanoma Phenotype Classifier was applied to dissociated tumor biopsy samples and characterization of phenotypic heterogeneity was supported by scRNA-Seq transcriptional profiles. This work establishes a link between cell morphology and melanoma phenotypes, laying the groundwork for the use of a label-free morphology-based method for phenotyping live melanoma cells combined with additional analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efb6b8eb45b844e5401bfcc2f8a0a6837a8825f0" target='_blank'>
              Label-Free Melanoma Phenotype Classification Using Artificial Intelligence-Based Morphological Profiling
              </a>
            </td>
          <td>
            Evelyn Lattmann, Andreja Jovic, Julie Kim, T. Pham, Christian Corona, Zhouyang Lian, K. Saini, Manisha Ray, Vivian Lu, A. Tastanova, Stephane C. Boutet, Mitchell P. Levesque
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Therapeutics targeting tumor endothelial cells (TECs) have been explored for decades, with only suboptimal efficacy achieved, partly due to an insufficient understanding of the TEC heterogeneity across cancer patients. We integrated single-cell RNA-seq data of 575 cancer patients from 19 solid tumor types, comprehensively charting the TEC phenotypic diversities. Our analyses uncovered underappreciated compositional and functional heterogeneity in TECs from a pan-cancer perspective. Two subsets, CXCR4+ tip cells and SELE+ veins, represented the prominent angiogenic and proinflammatory phenotypes of TECs, respectively. They exhibited distinct spatial organization patterns, and compared to adjacent non-tumor tissues, tumor tissue showed an increased prevalence of CXCR4+ tip cells, yet with SELE+ veins depleted. Such functional and spatial characteristics underlie their differential associations with the response of anti-angiogenic therapies and immunotherapies. Our integrative resources and findings open new avenues to understand and clinically intervene in the tumor vasculature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc571fa1d1ff50ea9e24190fe0da0d50d2cd7b6f" target='_blank'>
              Pan-cancer integrative analyses dissect the remodeling of endothelial cells in human cancers
              </a>
            </td>
          <td>
            Jinhu Li, Dongfang Wang, Fei Tang, Xinnan Ling, Wenjie Zhang, Zemin Zhang
          </td>
          <td>2024-07-11</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary In our study, we investigated immune regulation in a subtype of lymphoma called mantle cell lymphoma (MCL). Our goal was to understand how different types of immune cells, specifically T cells, behave in MCL. To do this, we combined image analysis and a technology called digital spatial profiling. We looked at tumor tissue from 102 MCL patients and analyzed different T-cell subsets. Interestingly, we found that late-stage differentiated T cells (CD57+) were more common in tumor-rich areas. These T cells also showed an increased expression of immune suppressive markers. We identified potential targets for treatment, such as CD47, IDO1, and CTLA-4. Additionally, we found that patients with sparse T-cell infiltration had an increased amount of GITR, which tentatively can be therapeutically targeted. Our findings shed light on previously unknown features of T-cell behavior in MCL. Abstract With the aim to advance the understanding of immune regulation in MCL and to identify targetable T-cell subsets, we set out to combine image analysis and spatial omic technology focused on both early and late differentiation stages of T cells. MCL patient tissue (n = 102) was explored using image analysis and GeoMx spatial omics profiling of 69 proteins and 1812 mRNAs. Tumor cells, T helper (TH) cells and cytotoxic (TC) cells of early (CD57−) and late (CD57+) differentiation stage were analyzed. An image analysis workflow was developed based on fine-tuned Cellpose models for cell segmentation and classification. TC and CD57+ subsets of T cells were enriched in tumor-rich compared to tumor-sparse regions. Tumor-sparse regions had a higher expression of several key immune suppressive proteins, tentatively controlling T-cell expansion in regions close to the tumor. We revealed that T cells in late differentiation stages (CD57+) are enriched among MCL infiltrating T cells and are predictive of an increased expression of immune suppressive markers. CD47, IDO1 and CTLA-4 were identified as potential targets for patients with T-cell-rich MCL TIME, while GITR might be a feasible target for MCL patients with sparse T-cell infiltration. In subgroups of patients with a high degree of CD57+ TC-cell infiltration, several immune checkpoint inhibitors, including TIGIT, PD-L1 and LAG3 were increased, emphasizing the immune-suppressive features of this highly differentiated T-cell subset not previously described in MCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd671c1ae7f90cd22784b36ac1b15cb30c9901a3" target='_blank'>
              Quantification and Profiling of Early and Late Differentiation Stage T Cells in Mantle Cell Lymphoma Reveals Immunotherapeutic Targets in Subsets of Patients
              </a>
            </td>
          <td>
            L. Lokhande, Daniel Nilsson, Joana de Matos Rodrigues, May Hassan, L. Olsson, P. Pyl, Louella Vasquez, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology. In this study, we developed SpatialKNife (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. The SKNY algorithm identified tumour spatial domains from spatial transcriptomic data of breast cancer, followed by clustering of these domains, trajectory estimation, and spatial extension to the tumour microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed endothelial cell and macrophage infiltration into the TME at mid-stage progression. Our results suggest that analysis using the spatial domain as a unit reflects pathological mechanisms in the TME. This approach may be applicable to the biological estimation of diverse microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37e699643db70ef39e651b4f4b87acc5b96626c" target='_blank'>
              Spatial domain analysis to estimate spatiotemporal pathological mechanisms in microenvironment with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ef25fe7aea98e6c411ef8acff16e5305282d18" target='_blank'>
              Deciphering the tumor immune microenvironment from a multidimensional omics perspective: insight into next-generation CAR-T cell immunotherapy and beyond
              </a>
            </td>
          <td>
            Zhaokai Zhou, Jiahui Wang, Jiaojiao Wang, Shuai Yang, Ruizhi Wang, Ge Zhang, Zhengrui Li, Run Shi, Zhan Wang, Qiong Lu
          </td>
          <td>2024-06-26</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76207c3c4f2b4f2f11d67f0daf7aa41a3f2aa07" target='_blank'>
              Dysfunctional tumor-infiltrating Vδ1 + T lymphocytes in microsatellite-stable colorectal cancer
              </a>
            </td>
          <td>
            V. Stary, Ram V Pandey, Julia List, L. Kleissl, Florian Deckert, J. Kabiljo, J. Laengle, Vasileios Gerakopoulos, Rudolf Oehler, Lukas Watzke, Matthias Farlik, S. Lukowski, Anne B Vogt, Georg Stary, Hannes Stockinger, M. Bergmann, Nina Pilat
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background. Interactions among tumor, immune, and vascular niches play major roles in driving glioblastoma (GBM) malignancy and treatment responses. The composition, heterogeneity, and localization of extracellular core matrix proteins (CMPs) that mediate such interactions, however, are not well understood. Methods. Here, through computational genomics and proteomics approaches, we analyzed the functional and clinical relevance of CMP expression in GBM at bulk, single cell, and spatial anatomical resolution. Results. We identified genes encoding CMPs whose expression levels categorize GBM tumors into CMP expression-high (M-H) and CMP expression-low (M-L) groups. CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, infiltration of pro-tumor immune cells, and immune checkpoint gene expression. Anatomical and single-cell transcriptome analyses indicate that matrisome gene expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells driving GBM progression. Finally, we identified a 17-gene CMP expression signature, termed Matrisome 17 (M17) signature that further refines the prognostic value of CMP genes. The M17 signature is a significantly stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, potentially predicts responses to PD1 blockade. Conclusion. The matrisome gene expression signature provides a robust stratification of GBM patients by survival and potential biomarkers of functionally relevant GBM niches that can mediate mesenchymal-immune cross talk. Patient stratification based on matrisome profiles can contribute to selection and optimization of treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0881b040066587c281ed9ef82afae916e63793e" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Z. Dereli, Zheng Yin, Elisabeth K. Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach-Carson, Robert Rostomily, Anil Korkut
          </td>
          <td>2024-06-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Objectives Non‐small‐cell lung carcinoma (NSCLC) is the most prevalent and lethal form of lung cancer. The need for biomarker‐informed stratification of targeted therapies has underpinned the need to uncover the underlying properties of the tumor microenvironment (TME) through high‐plex quantitative assays. Methods In this study, we profiled resected NSCLC tissues from 102 patients by targeted spatial proteomics of 78 proteins across tumor, immune activation, immune cell typing, immune‐oncology, drug targets, cell death and PI3K/AKT modules to identify the tumor and stromal signatures associated with overall survival (OS). Results Survival analysis revealed that stromal CD56 (HR = 0.384, P = 0.06) and tumoral TIM3 (HR = 0.703, P = 0.05) were associated with better survival in univariate Cox models. In contrast, after adjusting for stage, BCLXL (HR = 2.093, P = 0.02) and cleaved caspase 9 (HR = 1.575, P = 0.1) negatively influenced survival. Delta testing indicated the protective effect of TIM‐3 (HR = 0.614, P = 0.04) on OS. In multivariate analysis, CD56 (HR = 0.172, P = 0.001) was associated with better survival in the stroma, while B7.H3 (HR = 1.72, P = 0.008) was linked to poorer survival in the tumor. Conclusions Deciphering the TME using high‐plex spatially resolved methods is giving us new insights into compartmentalised tumor and stromal protein signatures associated with clinical endpoints in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c86af101b13bc4f2f501f32d2e227ba48bb3c7a" target='_blank'>
              Spatial proteomic profiling of tumor and stromal compartments in non‐small‐cell lung cancer identifies signatures associated with overall survival
              </a>
            </td>
          <td>
            Vahid Yaghoubi Naei, J. Monkman, Habib Sadeghirad, Ahmed M Mehdi, Tony Blick, W. Mullally, K. O’Byrne, M. Warkiani, A. Kulasinghe
          </td>
          <td>2024-01-01</td>
          <td>Clinical & Translational Immunology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) mortality rates continue to increase faster than those of other cancer types due to high heterogeneity, which limits diagnosis and treatment. Pathological and molecular subtyping have identified that HCC tumors with poor outcomes are characterized by intratumoral collagenous accumulation. However, the translational and post-translational regulation of tumor collagen, which is critical to the outcome, remains largely unknown. Here, we investigate the spatial extracellular proteome to understand the differences associated with HCC tumors defined by Hoshida transcriptomic subtypes of poor outcome (Subtype 1; S1; n = 12) and better outcome (Subtype 3; S3; n = 24) that show differential stroma-regulated pathways. Collagen-targeted mass spectrometry imaging (MSI) with the same-tissue reference libraries, built from untargeted and targeted LC-MS/MS was used to spatially define the extracellular microenvironment from clinically-characterized, formalin-fixed, paraffin-embedded tissue sections. Collagen α-1(I) chain domains for discoidin-domain receptor and integrin binding showed distinctive spatial distribution within the tumor microenvironment. Hydroxylated proline (HYP)-containing peptides from the triple helical regions of fibrillar collagens distinguished S1 from S3 tumors. Exploratory machine learning on multiple peptides extracted from the tumor regions could distinguish S1 and S3 tumors (with an area under the receiver operating curve of ≥0.98; 95% confidence intervals between 0.976 and 1.00; and accuracies above 94%). An overall finding was that the extracellular microenvironment has a high potential to predict clinically relevant outcomes in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53efde0c062c9d0f5b393a6e975a03778bb11cae" target='_blank'>
              The Spatial Extracellular Proteomic Tumor Microenvironment Distinguishes Molecular Subtypes of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Jade K Macdonald, Harrison B Taylor, Mengjun Wang, A. DelaCourt, Christin Edge, David N. Lewin, Naoto Kubota, Naoto Fujiwara, Fahmida Rasha, Cesia A. Marquez, A. Ono, Shiro Oka, K. Chayama, Sara Lewis, B. Taouli, Myron Schwartz, M. Fiel, Richard R Drake, Y. Hoshida, A. Mehta, Peggi M Angel
          </td>
          <td>2024-07-09</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2734d6282129acbb89725ff7878ceaacbfcfa19d" target='_blank'>
              Revealing the crucial roles of suppressive immune microenvironment in cardiac myxoma progression
              </a>
            </td>
          <td>
            Zedong Jiang, Qianlong Kang, Hong Qian, Zhijie Xu, Huan Tong, Jiaqing Yang, Li Li, Renwei Li, Guangqi Li, Fei Chen, Nan Lin, Yunuo Zhao, Huashan Shi, Juan Huang, Xuelei Ma
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. This study enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained genomic features and histopathological traits of primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, correlating ex vivo drug sensitivity with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program is associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributes to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine. One Sentence Summary This study establishes patient-derived tumor organoids (PDOs) from 31 Head and Neck Squamous Cell Carcinoma (HNSCC) patients, faithfully recapitulating characteristics of primary tumors and accurately predicting clinical responses to cisplatin treatment. We reveal intertumoral heterogeneity within PDOs and a hybrid epithelial-mesenchymal transition (hEMT) program conferring cisplatin resistance, highlighting amphiregulin (AREG) as a regulator of cellular plasticity and potential therapeutic target for HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82266d25cc2972a0e3e79d403bbffccb3c4b2f76" target='_blank'>
              Genomic and single-cell characterization of patient-derived tumor organoid models of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Young Min Park, De-Chen Lin
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the spatial dynamics within tissue microenvironments is crucial for deciphering cellular interactions and molecular signaling in living systems. These spatial characteristics govern cell distribution, extracellular matrix components, and signaling molecules, influencing local biochemical and biophysical conditions. Decoding these features offers insights into physiological processes, disease progression, and clinical outcomes. By elucidating spatial relationships between cell types, researchers uncover tissue architecture, cell communication networks, and microenvironment dynamics, aiding in the identification of biomarkers and therapeutic targets. Digital pathology imaging, including Hematoxylin and Eosin (H&E) staining, provides high-resolution histological information that offer intricate insights into cell-cell spatial relationships with greater details. However, current methods for capturing cell-cell spatial interactions are constrained by either methodological scopes or implementations restricted to script-level access. This limitation undermines generalizability and standardization, crucial for ensuring reproducibility. To address these limitations, we introduce SpatialQPFs, an extendable R package designed for extraction of interpretable spatial features from digital pathology images. By leveraging segmented cell information, our package provides researchers with a comprehensive toolkit for applying a range of spatial statistical methods within a stochastic process framework which includes analysis of point pattern data, areal data, and geostatistical data. This allows for a thorough analysis of cell spatial relationships, enhancing the depth and accuracy of spatial insights derived from the tissue, thereby empowering researchers to conduct comprehensive spatial analyses efficiently and reproducibly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da34c04318b2cac3bb2622ebe7b27ef2fe040d" target='_blank'>
              SpatialQPFs: An R package for deciphering cell-cell spatial relationship
              </a>
            </td>
          <td>
            Xiao Li
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Local inflammation in the pancreas is transient but imprints a durable epigenetic memory on epithelial cells, making them more amenable to oncogenic transformation. However, it is unclear whether epithelial cell heterogeneity is impacted by acute pancreatitis (AP) or whether population dynamics during regeneration contributes to the establishment of inflammation memory. To tackle those questions, we deployed experimental pancreatitis in mice and performed paired sequencing of transcriptomic and chromatin accessibility profiles at single nucleus resolution. We documented cell type abundance but also applied integrative analyses to infer phenotypically-distinct clusters of mesenchymal and exocrine cells. We found that AP perturbs a subset of “idling” acinar cells, which separate from more canonical “secretory” acini based on a more diversified proteome, which include elevated expression of signal transduction receptors. We linked acinar cell heterogeneity to epigenetic differences that also endow idling cells with superior plasticity. These constitute about 40% of acinar cells but can proliferate and skew their phenotype in response to AP. This leads to a remarkable recovery of pancreas histology and function, but also to the dissemination of idling-like features across the exocrine parenchyma. Mechanistically, idling acinar cells are characterized by enhanced transcriptional activity and protein synthesis. After recovery from pancreatitis, acini show elevation of both and establishment of chronic Unfolded Protein Response (UPR). We finally demonstrated that AP-primed pancreata show signs of elevated UPR and that ER stress promotes acinar cell metaplasia. Our data interrogate phenotypical dynamics during tissue regeneration to identify cell states amenable to epigenetic imprinting. They also suggest that UPR-alleviating strategies might curtail the risk of developing pancreatic cancer for individuals who experiences AP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0889f4e0ef737079e14a1df3e2a78bed4691d30d" target='_blank'>
              Population dynamics after pancreatitis dictates long-lasting epigenetic reprogramming and mediates tumor predisposition
              </a>
            </td>
          <td>
            Marco Fantuz, J. Liebig, Giovanni Fontana, Valerio Iebba, Carmine Carbone, Sören Lukassen, Christian Conrad, Alessandro Carrer
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the most common form of pancreatic cancer, a disease with dismal overall survival. Advances in treatment are hindered by a lack of preclinical models. Here, we show how a personalized organotypic “avatar” created from resected tissue allows spatial and temporal reporting on a complete in situ tumor microenvironment and mirrors clinical responses. Our perfusion culture method extends tumor slice viability, maintaining stable tumor content, metabolism, stromal composition, and immune cell populations for 12 days. Using multiplexed immunofluorescence and spatial transcriptomics, we identify immune neighborhoods and potential for immunotherapy. We used avatars to assess the impact of a preclinically validated metabolic therapy and show recovery of stromal and immune phenotypes and tumor redifferentiation. To determine clinical relevance, we monitored avatar response to gemcitabine treatment and identify a patient avatar-predictable response from clinical follow-up. Thus, avatars provide valuable information for syngeneic testing of therapeutics and a truly personalized therapeutic assessment platform for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361225cc3eb1a10b3d464aab534da9288640614c" target='_blank'>
              Development of human pancreatic cancer avatars as a model for dynamic immune landscape profiling and personalized therapy
              </a>
            </td>
          <td>
            Daniel Hughes, Alice Evans, Simei Go, Michael Eyres, Liuliu Pan, Somnath Mukherjee, Z. Soonawalla, Frances Willenbrock, Eric O’Neill
          </td>
          <td>2024-07-05</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The genomic landscape of clear cell renal cell carcinoma (ccRCC) has a considerable intra-tumor heterogeneity, which is a significant obstacle in the field of precision oncology and plays a pivotal role in metastasis, recurrence, and therapeutic resistance of cancer. The mechanisms of intra-tumor heterogeneity in ccRCC have yet to be fully established. We integrated single-cell RNA sequencing (scRNA-seq) and transposase-accessible chromatin sequencing (scATAC-seq) data from a single-cell multi-omics perspective. Based on consensus non-negative matrix factorization (cNMF) algorithm, functionally heterogeneous cancer cells were classified into metabolism, inflammatory, and EMT meta programs, with spatial transcriptomics sequencing (stRNA-seq) providing spatial information of such disparate meta programs of cancer cells. The bulk RNA sequencing (RNA-seq) data revealed high clinical prognostic values of functionally heterogeneous cancer cells of three meta programs, with transcription factor regulatory network and motif activities revealing the key transcription factors that regulate functionally heterogeneous ccRCC cells. The interactions between varying meta programs and other cell subpopulations in the microenvironment were investigated. Finally, we assessed the sensitivity of cancer cells of disparate meta programs to different anti-cancer agents. Our findings inform on the intra-tumor heterogeneity of ccRCC and its regulatory networks and offers new perspectives to facilitate the designs of rational therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56d844ad93caafe1a1fe2435935aca07cee4238" target='_blank'>
              Unfolding the mysteries of heterogeneity from a high-resolution perspective: integration analysis of single-cell multi-omics and spatial omics revealed functionally heterogeneous cancer cells in ccRCC
              </a>
            </td>
          <td>
            Jie Zheng, Wenhao Lu, Chengbang Wang, Shaohua Chen, Qingyun Zhang, Cheng Su
          </td>
          <td>2024-06-26</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Clone assignment in single-cell genomics remains a challenge due to its diverse mutation macrostructures and many missing signals. Existing statistical methods, for the sake of numerical convergence, pose strong constraints on the form of predicted mutation patterns, so they easily identify sub-optimally fitted clones that overlook weak and rare mutations. To solve this problem, we developed SNPmanifold, a Python package that learns flexible mutation patterns using a shallow binomial variational autoencoder. The latent space of SNPmanifold can effectively represent and visualize complex mutations of SNPs (single-nucleotide polymorphisms) in the form of geometrical manifolds. Based on nuclear or mitochondrial SNPs, we demonstrated that SNPmanifold can effectively identify a large number of multiplexed donors of origin (k = 18) that all existing unsupervised methods fail and lineages of somatic clones with promising biological interpretation. Therefore, SNPmanifold can reveal insights into single-cell SNPs more comprehensively than other existing methods, especially in complex datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f5cea3d97421756c5e4a1a65f43ef0a97c212b" target='_blank'>
              Interpretable variational encoding of genotypes identifies comprehensive clonality and lineages in single cells geometrically
              </a>
            </td>
          <td>
            Hoi Man Chung, Yuanhua Huang
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Severe Coronavirus disease 2019 (COVID-19) induces heterogeneous and progressive diffuse alveolar damage (DAD) highly disrupting lung tissue architecture and homeostasis, hampering disease management leading to fatal outcomes. Characterizing DAD pathophysiology across disease progression is of ultimate importance to better understand the molecular and cellular features driving different DAD patterns and to optimize treatment strategies. To contextualize the interplay between cell types and assess their distribution, spatial transcriptomics (ST) techniques have emerged, allowing unprecedented resolution to investigate spatial architecture of tissues. To this end, post-mortem lung tissue provides valuable insights into cellular composition and their spatial relationships at the time of death. Here, we have leveraged VisumST technology in post-mortem COVID-19 induced acute and proliferative DAD lungs including control samples with normal morphological appearance, to unravel the immunopathological mechanisms underlying DAD, providing novel insights into cellular and molecular communication events driving DAD progression in fatal COVID-19. We report a progressive loss of endothelial cell types, pneumocytes type I and natural killer cells coupled with a continuous increase of myeloid and stromal cells, mostly peribronchial fibroblasts, over disease progression. Spatial organization analysis identified variable cellular compartments, ranging from major compartments defined by cell type lineages in control lungs to increased and more specific compartmentalization including immune-specific clusters across DAD spectrum. Importantly, spatially informed ligand-receptor interaction (LRI) analysis revealed an intercellular communication signature defining COVID-19 induced DAD lungs. Transcription factor (TF) activity enrichment analysis identified TGF-B pathway as DAD driver, highlighting SMAD3 and SMAD7 TFs activity role during lung fibrosis. Integration of deregulated LRIs and TFs activity allowed us to propose a downstream intracellular signaling pathway in peribronchial fibroblasts, suggesting potential novel therapeutic targets. Finally, spatio-temporal trajectories analysis provided insights into the alveolar epithelium regeneration program, characterizing markers of pneumocytes type II differentiation towards pneumocytes type I. In conclusion, we provide a spatial characterization of lung tissue architecture upon COVID-19 induced DAD progression, identifying molecular and cellular hallmarks that may help optimize treatment and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a4db747d52faae9abbf25e5031f73c4597d1cc" target='_blank'>
              Spatial transcriptomics unveils the in situ cellular and molecular hallmarks of the lung in fatal COVID-19
              </a>
            </td>
          <td>
            C. García-Prieto, Eva Musulen, V. Davalos, Gerardo Ferrer, D. Grases, E. Porta, Belén Pérez-Miés, T. Caniego-Casas, José Palacios, Xavier Saenz-Sardà, Elisabet Englund, Manel Esteller
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins. Methods We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES- dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP- sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro. Results Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft. Conclusions Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states. Key Points Cellular plasticity influences the abundance of immunotherapeutic targets. Subtype-specific targets may be susceptible to epigenetically-mediated downregulation. Immunotherapeutic targets in development, B7-H3 and L1CAM, show “pan-subtype” expression. Importance of Study Neuroblastoma is a lethal childhood malignancy that shows cellular plasticity in response to anti-cancer therapies. Several plasma membrane proteins are being developed as immunotherapeutic targets in this disease. Here we define which cell surface proteins are susceptible to epigenetically regulated downregulation during an adrenergic to mesenchymal cell state switch and propose immunotherapeutic strategies to anticipate and circumvent acquired immunotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c43da3990dff1c35a4148b920ff54013c4ee8ec" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, F. Zammarchi, Patrick H van Berkel, Chi V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Simple Summary Immune checkpoint blockade has shown remarkable efficacy across various cancers but has failed to improve outcomes in patients with relapsed medulloblastoma (MB). While it is thought that the cold, immunosuppressive tumor microenvironment of MB accounts for this poor efficacy, the precise mechanisms contributing to immune suppression in this context remain unclear. In this study, we explore the immune landscape of MB using a previously unexplored syngeneic mouse model. Our study demonstrates that this model faithfully recapitulates the cold, immunosuppressive tumor microenvironment observed in human disease. Importantly, our research uncovers mechanisms employed by myeloid cells and tumor cells to evade immune detection while highlighting the therapeutic potential of targeting V-domain Ig Suppressor of T-cell Activation (VISTA), a novel inhibitory immune checkpoint, to enhance anti-tumor immunity. Abstract Background: Relapsed medulloblastoma (MB) poses a significant therapeutic challenge due to its highly immunosuppressive tumor microenvironment. Immune checkpoint inhibitors (ICIs) have struggled to mitigate this challenge, largely due to low T-cell infiltration and minimal PD-L1 expression. Identifying the mechanisms driving low T-cell infiltration is crucial for developing more effective immunotherapies. Methods: We utilize a syngeneic mouse model to investigate the tumor immune microenvironment of MB and compare our findings to transcriptomic and proteomic data from human MB. Results: Flow cytometry reveals a notable presence of CD45hi/CD11bhi macrophage-like and CD45int/CD11bint microglia-like tumor-associated macrophages (TAMs), alongside regulatory T-cells (Tregs), expressing high levels of the inhibitory checkpoint molecule VISTA. Compared to sham control mice, the CD45hi/CD11bhi compartment significantly expands in tumor-bearing mice and exhibits a myeloid-specific signature composed of VISTA, CD80, PD-L1, CTLA-4, MHCII, CD40, and CD68. These findings are corroborated by proteomic and transcriptomic analyses of human MB samples. Immunohistochemistry highlights an abundance of VISTA-expressing myeloid cells clustering at the tumor–cerebellar border, while T-cells are scarce and express FOXP3. Additionally, tumor cells exhibit immunosuppressive properties, inhibiting CD4 T-cell proliferation in vitro. Identification of VISTA’s binding partner, VSIG8, on tumor cells, and its correlation with increased VISTA expression in human transcriptomic analyses suggests a potential therapeutic target. Conclusions: This study underscores the multifaceted mechanisms of immune evasion in MB and highlights the therapeutic potential of targeting the VISTA–VSIG axis to enhance anti-tumor responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8972b71fd9c803882e0f815586aa8205257f7159" target='_blank'>
              VISTA Emerges as a Promising Target against Immune Evasion Mechanisms in Medulloblastoma
              </a>
            </td>
          <td>
            S. Harrabi, A. Hormigo, Natalia Muñoz Perez, Juliana Pensabene, Jr Phillip M. Galbo, Negar Sadeghipour, Joanne Xiu, Kirsten Moziak, Rita M. Yazejian, Rachel L. Welch, W. R. Bell, Soma Sengupta, Sonikpreet Aulakh, Charles G Eberhart, David M Loeb, Emad Eskandar, Deyou Zheng, Xingxing Zang, Allison M. Martin
          </td>
          <td>2024-07-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Accumulating evidence suggests that the tumor immune microenvironment (TIME) significantly influences the response to immunotherapy, yet this complex relationship remains elusive. To address this issue, we developed TimiGP-Response (TIME Illustration based on Gene Pairing designed for immunotherapy Response), a computational framework leveraging single-cell and bulk transcriptomic data, along with response information, to construct cell-cell interaction networks associated with responders and estimate the role of immune cells in treatment response. This framework was showcased in triple-negative breast cancer treated with immune checkpoint inhibitors targeting the PD-1:PD-L1 interaction, and orthogonally validated with imaging mass cytometry. As a result, we identified CD8+ GZMB+ T cells associated with responders and its interaction with regulatory T cells emerged as a potential feature for selecting patients who may benefit from these therapies. Subsequently, we analyzed 3,410 patients with seven cancer types (melanoma, non-small cell lung cancer, renal cell carcinoma, metastatic urothelial carcinoma, hepatocellular carcinoma, breast cancer, and esophageal cancer) treated with various immunotherapies and combination therapies, as well as several chemo- and targeted therapies as controls. Using TimiGP-Response, we depicted the pan-cancer immune landscape associated with immunotherapy response at different resolutions. At the TIME level, CD8 T cells and CD4 memory T cells were associated with responders, while anti-inflammatory (M2) macrophages and mast cells were linked to non-responders across most cancer types and datasets. Given that T cells are the primary targets of these immunotherapies and our TIME analysis highlights their importance in response to treatment, we portrayed the pan-caner landscape on 40 T cell subtypes. Notably, CD8+ and CD4+ GZMK+ effector memory T cells emerged as crucial across all cancer types and treatments, while IL-17-producing CD8+ T cells were top candidates associated with immunotherapy non-responders. In summary, this study provides a computational method to study the association between TIME and response across the pan-cancer immune landscape, offering resources and insights into immune cell interactions and their impact on treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883be60621995b3d688f88f7d3f09de5740baf9e" target='_blank'>
              TimiGP-Response: the pan-cancer immune landscape associated with response to immunotherapy
              </a>
            </td>
          <td>
            Chenyang Li, Wei Hong, A. Reuben, Linghua Wang, Anirban Maitra, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Interactions between mutant cells and their environment play a key role in determining cancer susceptibility. However, our understanding of how the pre-cancer microenvironment contributes to early tumorigenesis remains limited. Here, we show that newly emerging tumours at their most incipient stages shape their microenvironment in a critical process that determines their survival. Analysis of nascent squamous tumours in the upper gastrointestinal tract of the mouse reveals that the stress response of early tumour cells instructs the underlying mesenchyme to form a supportive “pre-cancer niche”, which dictates the long-term outcome of epithelial lesions. Stimulated fibroblasts beneath emerging tumours activate a wound healing response that triggers a dramatic remodelling of the underlying extracellular matrix, resulting in the formation of a fibronectin-rich stromal scaffold that promotes tumour growth. Functional heterotypic 3D culture assays and in vivo grafting experiments, combining carcinogen-free healthy epithelium and tumour-derived stroma, demonstrate that the pre-cancerous niche alone is sufficient to confer tumour properties to healthy epithelial cells. We propose a model where both mutations and the stromal response to genetic stress defines the likelihood of early tumours to survive and progress towards more advanced disease stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be23254ceedda48875f58bb5d700c5fd073374a7" target='_blank'>
              Pre-cancerous Niche Remodelling Dictates Nascent Tumour Survival
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. E. Rojo Arias, Y. Dang, S. Han, M. T. Bejar, B. Colom, J. C. Fowler, P. Jones, S. Rulands, B. D. Simons, M. P. Alcolea
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immunotherapies have shown great promise in pleural mesothelioma (PM), yet most patients still do not achieve significant clinical response, highlighting the importance of improving understanding of the tumor microenvironment (TME). Here, we utilized high-throughput, single-cell RNA-sequencing to de novo identify 54 expression programs and construct a comprehensive cellular catalogue of the PM TME. We found four cancer-intrinsic programs associated with poor disease outcome and a novel fetal-like, endothelial cell population that likely responds to VEGF signaling and promotes angiogenesis. Throughout cellular compartments, we observe substantial difference in the TME associated with a cancer-intrinsic sarcomatoid signature, including enrichment in fetal-like endothelial cells, CXCL9+ macrophages, cytotoxic, exhausted, and regulatory T cells, which we validated using imaging and bulk deconvolution analyses on independent cohorts. Finally, we show, both computationally and experimentally, that NKG2A-HLA-E interaction between NK and tumor cells represents an important new therapeutic axis in PM, especially for epithelioid cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c84bfe3e9c66b621b1832492f79b2264c3e648" target='_blank'>
              Single cell view of tumor microenvironment gradients in pleural mesothelioma.
              </a>
            </td>
          <td>
            Bruno Giotti, Komal Dolasia, William Zhao, Peiwen Cai, Robert Sweeney, Elliot Merritt, Evgeny Kiner, Grace S Kim, Atharva Bhagwat, Thinh Nguyen, Samarth Hegde, Bailey G Fitzgerald, S. Shroff, T. Dawson, M. Garcia-Barros, Jamshid Abdul-Ghafar, Rachel Chen, Sacha Gnjatic, Alan Soto, Rachel Brody, S. Kim-Schulze, Zhihong Chen, Kristin G Beaumont, Miram Merad, R.M. Flores, R. Sebra, Amir Horowitz, Thomas U. Marron, Anna Tocheva, Andrea Wolf, A. Tsankov
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The involvement of cytotoxic CD4+ T cells (CD4+ CTLs) and their potential role in dictating the response to immune checkpoint inhibitors (ICIs) in patients with metastatic renal cell carcinoma (mRCC) remains an unexplored area of research. Methods Utilizing single-cell RNA sequencing, we analyzed the immunophenotype and expression patterns of CD4+ T lymphocyte subtypes in mRCC patients, followed by preliminary validation via multi-immunofluorescent staining. In addition, we obtained a comprehensive immunotherapy dataset encompassing single-cell RNA sequencing datasets and bulk RNA-seq cohorts from the European Genome-Phenome Archive and ArrayExpress database. Utilizing the CIBERSORTx deconvolution algorithms, we derived a signature score for CD4+ CTLs from the bulk-RNA-seq datasets of the CheckMate 009/025 clinical trials. Results Single-cell analysis of CD4+ T lymphocytes in mRCC reveals several cancer-specific states, including diverse phenotypes of regulatory T cells. Remarkably, we observe that CD4+ CTLs cells constitute a substantial proportion of all CD4+ T lymphocyte sub-clusters in mRCC patients, highlighting their potential significance in the disease. Furthermore, within mRCC patients, we identify two distinct cytotoxic states of CD4+ T cells: CD4+GZMK+ T cells, which exhibit a weaker cytotoxic potential, and CD4+GZMB+ T cells, which demonstrate robust cytotoxic activity. Both regulatory T cells and CD4+ CTLs originate from proliferating CD4+ T cells within mRCC tissues. Intriguingly, our trajectory analysis indicates that the weakly cytotoxic CD4+GZMK+ T cells differentiate from their more cytotoxic CD4+GZMB+ counterparts. In comparing patients with lower CD4+ CTLs levels to those with higher CD4+ CTLs abundance in the CheckMate 009 and 25 immunotherapy cohorts, the latter group exhibited significantly improved OS and PFS probability. Conclusion Our study underscores the pivotal role that intratumoral CD4+ CTLs may play in bolstering anti-tumor immunity, suggesting their potential as a promising biomarker for predicting response to ICIs in patients with mRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f2bc88c864cde26033332d6b179b9c1f5c8c941" target='_blank'>
              Single-Cell Analysis Identifies Distinct Populations of Cytotoxic CD4+ T Cells Linked to the Therapeutic Efficacy of Immune Checkpoint Inhibitors in Metastatic Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Xu Yang, Jianwei Wu, Longlong Fan, Binghua Chen, Shiqiang Zhang, Wenzhong Zheng
          </td>
          <td>2024-07-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Flow cytometry is a widely used technique for immune cell analysis, offering insights into cell composition and function. Spectral flow cytometry allows for high-dimensional analysis of immune cells, overcoming limitations of conventional flow cytometry. However, analyzing data from large antibody panels can be challenging using traditional bi-axial gating strategies. Here, we present a novel analysis pipeline designed to improve analysis of spectral flow cytometry. We employ this method to identify rare T cell populations in aging. We isolated splenocytes from young (2–3 months) and aged (18–19 months) female mice then stained these with a panel of 20 fluorescently labeled antibodies. Spectral flow cytometry was performed, followed by data processing and analysis using Python within a Jupyter Notebook environment to perform batch correction, unsupervised clustering, dimensionality reduction, and differential expression analysis. Our analysis of 3,776,804 T cells from 11 spleens revealed 34 distinct T cell clusters identified by surface marker expression. We observed significant differences between young and aged mice, with certain clusters enriched in one age group over the other. Naïve, effector memory, and central memory CD8+ and CD4+ T cell subsets exhibited age-associated changes in abundance and marker expression. Additionally, γδ T cell clusters showed differential abundance between age groups. By leveraging high-dimensional analysis methods borrowed from single-cell RNA sequencing analysis, we identified age-related differences in T cell subsets, providing insights into the immune aging process. This approach offers a robust, free, and easily implemented analysis pipeline for spectral flow cytometry data that may facilitate the discovery of novel therapeutic targets for age-related immune dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddf83cb7ddc45db95ea02054cc7fa44ba7e00a8" target='_blank'>
              Development of a Spectral Flow Cytometry Analysis Pipeline for High-Dimensional Immune Cell Characterization
              </a>
            </td>
          <td>
            Donald Vardaman, Md Akkas Ali, Chase Bolding, Harrison Tidwell, Holly Stephens, Daniel J. Tyrrell
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary We utilized publicly available perturbation phosphoproteomic data to construct models elucidating cell state transitions across multiple breast cancer and normal breast tissue-derived cell lines. Employing a hybrid methodology, which integrates machine learning and mechanistic modeling, we separated luminal, basal, and normal cell states and revealed core networks that control cell state transitions. We determined causal connections within the core networks and developed interpretable mechanistic models that elucidated the drivers of cell phenotypes. Significantly, these models can predict synergistic drug combinations capable of potentially reversing oncogenic transformation in breast cancer cell lines. Our methodology will enable designer approaches to identify targeted perturbations that convert cell states and mechanistically underpin therapeutic interventions. Abstract Understanding signaling patterns of transformation and controlling cell phenotypes is a challenge of current biology. Here we applied a cell State Transition Assessment and Regulation (cSTAR) approach to a perturbation dataset of single cell phosphoproteomic patterns of multiple breast cancer (BC) and normal breast tissue-derived cell lines. Following a separation of luminal, basal, and normal cell states, we identified signaling nodes within core control networks, delineated causal connections, and determined the primary drivers underlying oncogenic transformation and transitions across distinct BC subtypes. Whereas cell lines within the same BC subtype have different mutational and expression profiles, the architecture of the core network was similar for all luminal BC cells, and mTOR was a main oncogenic driver. In contrast, core networks of basal BC were heterogeneous and segregated into roughly four major subclasses with distinct oncogenic and BC subtype drivers. Likewise, normal breast tissue cells were separated into two different subclasses. Based on the data and quantified network topologies, we derived mechanistic cSTAR models that serve as digital cell twins and allow the deliberate control of cell movements within a Waddington landscape across different cell states. These cSTAR models suggested strategies of normalizing phosphorylation networks of BC cell lines using small molecule inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/092c4f2c92e0be64eb32694e8a69506f512145ae" target='_blank'>
              Cell State Transition Models Stratify Breast Cancer Cell Phenotypes and Reveal New Therapeutic Targets
              </a>
            </td>
          <td>
            Oleksii S. Rukhlenko, Hiroaki Imoto, Ayush Tambde, Amy McGillycuddy, Philipp Junk, Anna Tuliakova, Walter Kolch, Boris N. Kholodenko
          </td>
          <td>2024-06-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, O. Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The haematological malignancy multiple myeloma is associated with skewed T-cell activation and function. T-cell alterations are detectable in asymptomatic myeloma precursor conditions and have the potential to identify precursor patients at imminent risk of progression. However, what myeloma-associated T-cells alterations represent mechanistically, how they relate to tumour burden and gene expression, and what influences high inter-patient variability in immune composition remains unknown. Here, we assembled the largest ever dataset of published and newly-generated single-cell RNA and TCR sequencing of the marrow and blood from patients with myeloma, precursor conditions, and age-matched non-cancer controls. We show myeloma is not associated with T-cell exhaustion and instead defined by a pattern of T-cell differentiation resembling antigen-driven terminal memory differentiation. Myeloma-associated T-cell differentiation was dependent on tumour-intrinsic features including tumour burden and tumour expression of antigen-presentation genes. Expanded TCR clones accumulating in myeloma were not enriched for viral specificity and were detected in effector states in highly infiltrated marrows. Together, these results suggest anti-tumour immunity drives a novel form of cancer-associated T-cell memory differentiation in myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d674fe6f864f5fa613feeb41c61f5327c58bd541" target='_blank'>
              Tumour-intrinsic features shape T-cell differentiation through myeloma disease evolution
              </a>
            </td>
          <td>
            K. Foster, Elise Rees, L. Ainley, Eileen M. Boyle, Lydia Lee, Gwennan Ward, D. Galas-Filipowicz, Anna Mikolajczak, Emma J Lyon, Dylan Jankovic, Jasmine Rahman, Mahima Turakhia, Imran Uddin, Gordon Beattie, Yvette Hoade, Catherine Zhu, J. Reading, Ieuan G Walker, Michael Chapman, Karthik Ramasamy, Javier Herrero, B. Chain, S. Quezada, Kwee Yong
          </td>
          <td>2024-06-23</td>
          <td>None</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a complex tumor microenvironment (TME) that promotes cancer progression. The tumor stroma is characterized by cancer-associated fibroblasts (CAFs) that secrete factors stimulating tumor growth, invasion, and immunosuppression. Macrophages tend to polarize toward a pro-tumor phenotype. Various other components, including endothelial cells, pericytes, and neural cells, likely contribute to the complexity of the PDAC TME. Epidemiological investigations have highlighted obesity as a risk factor for pancreatic cancer, suggesting that adipocytes may also play a significant role. This manuscript offers an overview of the main actors in TME, as well as dysregulated signaling pathways and gene mutations. It underlines how strategies modulating the TME hold promise for improving outcomes. Lastly, it describes how integrating multi-omic approaches may represent a future trajectory to explore novel therapeutic possibilities. Abstract Pancreatic ductal adenocarcinoma (PDAC) presents significant oncological challenges due to its aggressive nature and poor prognosis. The tumor microenvironment (TME) plays a critical role in progression and treatment resistance. Non-neoplastic cells, such as cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), contribute to tumor growth, angiogenesis, and immune evasion. Although immune cells infiltrate TME, tumor cells evade immune responses by secreting chemokines and expressing immune checkpoint inhibitors (ICIs). Vascular components, like endothelial cells and pericytes, stimulate angiogenesis to support tumor growth, while adipocytes secrete factors that promote cell growth, invasion, and treatment resistance. Additionally, perineural invasion, a characteristic feature of PDAC, contributes to local recurrence and poor prognosis. Moreover, key signaling pathways including Kirsten rat sarcoma viral oncogene (KRAS), transforming growth factor beta (TGF-β), Notch, hypoxia-inducible factor (HIF), and Wnt/β-catenin drive tumor progression and resistance. Targeting the TME is crucial for developing effective therapies, including strategies like inhibiting CAFs, modulating immune response, disrupting angiogenesis, and blocking neural cell interactions. A recent multi-omic approach has identified signature genes associated with anoikis resistance, which could serve as prognostic biomarkers and targets for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23cd29cbd6982ee3e4b132b1c93695ffca70902" target='_blank'>
              Decoding the Intricate Landscape of Pancreatic Cancer: Insights into Tumor Biology, Microenvironment, and Therapeutic Interventions
              </a>
            </td>
          <td>
            A. Argentiero, Alessandro Andriano, Ingrid Catalina Caradonna, Giulia de Martino, V. Desantis
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Effective cancer immunotherapy relies on the clonal proliferation and expansion of CD8+ T cells in the tumor1,2. However, our insights into clonal expansions are limited, owing to an inability to track the same clones in tumors over time. Here, we developed a multi-tumor mouse model system to track hundreds of expanding and contracting CD8+ T cell clones over multiple timepoints in tumors of the same individual. Through coupling of clonal expansion dynamics and single-cell RNA/TCR-seq data, we identified a transcriptomic signature in PD-1+Ly108+ precursor exhausted cells3,4 that strongly predicts rates of intratumoral clone expansion in mice and humans. We found that expression of the signature successfully stratifies melanoma patient outcomes to PD-1/PD-L1 blockade5,6. Downregulation of the signature precedes clone contraction – a phase in which clones contract but maintain revivable precursor exhausted cells in the tumor. LAG-3 blockade – an FDA-approved therapy whose effects on CD8+ T cell responses are currently unclear7, re-activates the expansion signature, re-expanding pre-existing clones, including previously contracted clones. These findings reveal how the study of clonal expansion dynamics provide a powerful ‘pan-immunotherapy’ signature for monitoring immunotherapies, including PD-1/PD-L1 and LAG-3 blockade, with implications for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6681eced55a92fc5ceb65f99517fea6fa8caf1df" target='_blank'>
              LAG-3 blockade reactivates the CD8+ T cell expansion program to re-expand contracted clones in the tumor
              </a>
            </td>
          <td>
            Munetomo Takahashi, Mikiya Tsunoda, Hiroyasu Aoki, Masaki Kurosu, H. Ogiwara, S. Shichino, D. Bending, Shumpei Ishikawa, James E. D. Thaventhiran, Kouji Matsushima, S. Ueha
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d5310f7c7a3d332e8cba0823f8a46905b9d7fe7" target='_blank'>
              Utilizing human cerebral organoids to model breast cancer brain metastasis in culture
              </a>
            </td>
          <td>
            Chenran Wang, A. Nagayach, Harsh Patel, Lan Dao, Hui Zhu, Amanda R Wasylishen, Yanbo Fan, Ady Kendler, Ziyuan Guo
          </td>
          <td>2024-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Melanoma, a highly malignant tumour, presents significant challenges due to its cellular heterogeneity, yet research on this aspect in cutaneous melanoma remains limited. In this study, we utilized single‐cell data from 92,521 cells to explore the tumour cell landscape. Through clustering analysis, we identified six distinct cell clusters and investigated their differentiation and metabolic heterogeneity using multi‐omics approaches. Notably, cytotrace analysis and pseudotime trajectories revealed distinct stages of tumour cell differentiation, which have implications for patient survival. By leveraging markers from these clusters, we developed a tumour cell‐specific machine learning model (TCM). This model not only predicts patient outcomes and responses to immunotherapy, but also distinguishes between genomically stable and unstable tumours and identifies inflamed (‘hot’) versus non‐inflamed (‘cold’) tumours. Intriguingly, the TCM score showed a strong association with TOMM40, which we experimentally validated as an oncogene promoting tumour proliferation, invasion and migration. Overall, our findings introduce a novel biomarker score that aids in selecting melanoma patients for improved prognoses and targeted immunotherapy, thereby guiding clinical treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceff46cb2d3d44eadc13e557a6116c6ae860759" target='_blank'>
              Unravelling tumour cell diversity and prognostic signatures in cutaneous melanoma through machine learning analysis
              </a>
            </td>
          <td>
            Wenhao Cheng, Ping Ni, Hao Wu, Xiaye Miao, Xiaodong Zhao, Dali Yan
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors with dismal prognosis. One of the main bottlenecks for developing more effective therapies for glioblastoma stems from their histologic and molecular heterogeneity, leading to distinct tumor microenvironments and disease phenotypes. Effectively characterizing these features would improve the clinical management of glioblastoma. Glucose flux rates through glycolysis and mitochondrial oxidation have been recently shown to quantitatively depict glioblastoma proliferation in mouse models (GL261 and CT2A tumors, 38±3 mm3) using dynamic glucose-enhanced (DGE) deuterium spectroscopy. However, the spatial features of tumor microenvironment phenotypes remain hitherto unresolved. Here, we develop a DGE Deuterium Metabolic Imaging (DMI) approach for profiling tumor microenvironments through glucose conversion kinetics. Using a multimodal combination of tumor mouse models, novel strategies for spectroscopic imaging and noise attenuation, and histopathological correlations, we show that tumor lactate turnover mirrors phenotype differences between GL261 and CT2A mouse glioblastoma (59±7 mm3), whereas peritumoral glutamate-glutamine recycling is a potential marker of invasion capacity in pooled cohorts, linked to secondary brain lesions. Our findings were validated by histopathological characterization of each tumor, including cell density and proliferation, peritumoral infiltration, and distant migration. Our study bodes well for precision neuro-oncology, highlighting the importance of mapping glucose flux rates to better understand the metabolic heterogeneity of glioblastoma and its links to disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2927f90371bde6768c35753a6b358801ccb1fb15" target='_blank'>
              Deuterium Metabolic Imaging Phenotypes Mouse Glioblastoma Heterogeneity Through Glucose Turnover Kinetics
              </a>
            </td>
          <td>
            Rui V. Simões, Rafael N. Henriques, J. Olesen, Beatriz M Cardoso, Francisca F. Fernandes, Mariana A.V. Monteiro, S. Jespersen, Tânia Carvalho, N. Shemesh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables interrogating the molecular composition of tissue with ever-increasing resolution, depth, and sensitivity. However, costs, rapidly evolving technology, and lack of standards have constrained computational methods in ST to narrow tasks and small cohorts. In addition, the underlying tissue morphology as reflected by H&E-stained whole slide images (WSIs) encodes rich information often overlooked in ST studies. Here, we introduce HEST-1k, a collection of 1,108 spatial transcriptomic profiles, each linked to a WSI and metadata. HEST-1k was assembled using HEST-Library from 131 public and internal cohorts encompassing 25 organs, two species (Homo Sapiens and Mus Musculus), and 320 cancer samples from 25 cancer types. HEST-1k processing enabled the identification of 1.5 million expression--morphology pairs and 60 million nuclei. HEST-1k is tested on three use cases: (1) benchmarking foundation models for histopathology (HEST-Benchmark), (2) biomarker identification, and (3) multimodal representation learning. HEST-1k, HEST-Library, and HEST-Benchmark can be freely accessed via https://github.com/mahmoodlab/hest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/351b175485ca22362ace3258ab42fe3b708be864" target='_blank'>
              HEST-1k: A Dataset for Spatial Transcriptomics and Histology Image Analysis
              </a>
            </td>
          <td>
            Guillaume Jaume, Paul Doucet, Andrew H. Song, Ming Y. Lu, Cristina Almagro-P'erez, Sophia J. Wagner, Anurag J. Vaidya, Richard J. Chen, Drew F. K. Williamson, Ahrong Kim, Faisal Mahmood
          </td>
          <td>2024-06-23</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Metastases from gastric adenocarcinoma (GAC) lead to high morbidity and mortality. Developing innovative and effective therapies requires a comprehensive understanding of the tumor and immune biology of advanced GAC. Yet, collecting matched specimens from advanced, treatment-naïve GAC patients poses a significant challenge, limiting the scope of current research, which has predominantly focused on localized tumors. This gap hinders a deeper insight into the metastatic dynamics of GAC.


METHODS
We performed in-depth single-cell transcriptome and immune profiling on 68 paired, treatment-naïve, primary-metastatic tumors to delineate alterations in cancer cells and their tumor microenvironment (TME) during metastatic progression. To validate our observations, we conducted comprehensive functional studies both in vitro and in vivo, employing cell lines, multiple PDX and novel mouse models of GAC.


RESULTS
Liver and peritoneal metastases exhibited distinct properties in cancer cells and dynamics of TME phenotypes, supporting the notion that cancer cells and their local TMEs co-evolve at metastatic sites. Our study also revealed differential activation of cancer meta-programs across metastases. We observed evasion of cancer cell ferroptosis via GPX4 upregulation during GAC progression. Conditional depletion of Gpx4 or pharmacological inhibition of ferroptosis resistance significantly attenuated tumor growth and metastatic progression. Additionally, ferroptosis-resensitizing treatments augmented the efficacy of CAR T-cell therapy.


CONCLUSIONS
This study represents the largest single-cell dataset of metastatic GACs to date. High-resolution mapping of the molecular and cellular dynamics of GAC metastasis has revealed a rationale for targeting ferroptosis defense in combination with CAR T-cell therapy as a novel therapeutic strategy with potential immense clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c4774948f3c82306cdc21b3d5730989f11741b" target='_blank'>
              Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response.
              </a>
            </td>
          <td>
            Xiangdong Cheng, Enyu Dai, Jibo Wu, Natasha M. Flores, Yanshuo Chu, Ruiping Wang, M. Dang, Zhiyuan Xu, Guangchun Han, Yunhe Liu, D. Chatterjee, Can Hu, Jieer Ying, Yi-an Du, Litao Yang, Xiaoqing Guan, Shaowei Mo, Xuanye Cao, Guangsheng Pei, Jiahui Jiang, Xiaoyin Lu, Ana Morales Benitez, R. Waters, Melissa P Pizzi, N. Shanbhag, Yibo Fan, F. Peng, Samir Hanash, G. Calin, Andrew Futreal, Shumei Song, Cassian Yee, Pawel K Mazur, Jiang-Jiang Qin, J. Ajani, Linghua Wang
          </td>
          <td>2024-08-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pericytes, recognized as mural cells, have long been described as components involved in blood vessel formation, playing a mere supporting role for endothelial cells (ECs). Emerging evidence strongly suggests their multifaceted roles in tissues and organs. Indeed, pericytes exhibit a remarkable ability to anticipate endothelial cell behavior and adapt their functions based on the specific cells they interact with. Pericytes can be activated by pro-inflammatory stimuli and crosstalk with immune cells, actively participating in their transmigration into blood vessels. Moreover, they can influence the immune response, often sustaining an immunosuppressive phenotype in most of the cancer types studied. In this review, we concentrate on the intricate crosstalk between pericytes and immune cells in cancer, highlighting the primary evidence regarding pericyte involvement in primary tumor mass dynamics, their contributions to tumor reprogramming for invasion and migration of malignant cells, and their role in the formation of pre-metastatic niches. Finally, we explored recent and emerging pharmacological approaches aimed at vascular normalization, including novel strategies to enhance the efficacy of immunotherapy through combined use with anti-angiogenic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76bcdf2dcb6f3841dc22f4ab784010532400194" target='_blank'>
              Pericytes: jack-of-all-trades in cancer-related inflammation
              </a>
            </td>
          <td>
            Marianna Moro, Federica Carolina Balestrero, A. A. Grolla
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The spatial composition and cellular heterogeneity of the tumor microenvironment plays a critical role in cancer development and progression. High-definition pathology imaging of tumor biopsies provide a high-resolution view of the spatial organization of different types of cells. This allows for systematic assessment of intra- and inter-patient spatial cellular interactions and heterogeneity by integrating accompanying patient-level genomics data. However, joint modeling across tumor biopsies presents unique challenges due to non-conformability (lack of a common spatial domain across biopsies) as well as high-dimensionality. To address this problem, we propose the Dual random effect and main effect selection model for Spatially structured regression model (DreameSpase). DreameSpase employs a Bayesian variable selection framework that facilitates the assessment of spatial heterogeneity with respect to covariates both within (through fixed effects) and between spaces (through spatial random effects) for non-conformable spatial domains. We demonstrate the efficacy of DreameSpase via simulations and integrative analyses of pathology imaging and gene expression data obtained from $335$ melanoma biopsies. Our findings confirm several existing relationships, e.g. neutrophil genes being associated with both inter- and intra-patient spatial heterogeneity, as well as discovering novel associations. We also provide freely available and computationally efficient software for implementing DreameSpase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f4788cc422b8751f7a8dd15c650e8347a66bb49" target='_blank'>
              Spatially Structured Regression for Non-conformable Spaces: Integrating Pathology Imaging and Genomics Data in Cancer
              </a>
            </td>
          <td>
            Nathaniel Osher, Jian Kang, Arvind Rao, Veerabhadran Baladandayuthapani
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune cell subsets in the tumor predict prognosis. Identifying reliable subsets consistently in multiple patients is clinically important. What’s more advantageous to the field is if such subsets can be a target of immunotherapy. Here we analyzed single-cell RNA-sequencing datasets of patients with triple-negative breast cancer and identified APOE or FABP5-expressing macrophages correlated with poor prognosis. Further validation with TCGA-BRCA cohort detailed molecular signatures of these macrophages as lipidic and senescent. Our receptor-ligand mapping identified lipidic and senescent macrophages both suppress T-lymphoid and myeloid immunogenicity. Finally, we discovered that immune checkpoint therapy combined with chemotherapy reprogrammed anti-tumor microenvironment enriched with FOLR2+ macrophages facilitated T-cell activations. This suggests that lipidic and senescent macrophages could be a therapeutic target of immune checkpoint therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da8ec9405ed3a63b54dbe2fe2bb33c7cd24fe33d" target='_blank'>
              Lipidic and senescent macrophages predict progression and response to combinatorial immunotherapy in triple-negative breast cancer
              </a>
            </td>
          <td>
            Chun Lai Chan, Alex To, Shihui Zhang, J. Wong, Yuanhua Huang, Yiming Chao, Ryohichi Sugimura
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Carotid atherosclerosis is orchestrated by cell-cell communication that drives progression along a clinical continuum (asymptomatic to symptomatic). Extracellular vesicles (EVs) are cell-derived nanoparticles representing a new paradigm in cellular communication. Little is known about their biological cargo, cellular origin/destination, and functional roles in human atherosclerotic plaque. Methods: EVs were enriched via size exclusion chromatography from human carotid endarterectomy samples dissected into paired plaque and marginal zones (symptomatic n=16, asymptomatic n=13). EV cargos were assessed via whole transcriptome miRNA sequencing and mass spectrometry-based proteomics. EV multi-omics were integrated with bulk and single cell RNA-sequencing (scRNA-seq) datasets to predict EV cellular origin and ligand-receptor interactions, and multi-modal biological network integration of EV-cargo was completed. EV functional impact was assessed with endothelial angiogenesis assays. Results: Carotid plaques contained more EVs than adjacent marginal zones, with differential enrichment for EV-miRNAs and EV-proteins in key atherogenic pathways. EV cellular origin analysis suggested that tissue EV signatures originated from endothelial cells (EC), smooth muscle cells (SMC), and immune cells. Integrated tissue vesiculomics and scRNA-seq indicated complex EV-vascular cell communication that changed with disease progression and plaque vulnerability (i.e., symptomatic disease). Plaques from symptomatic patients, but not asymptomatic patients, were characterized by increased involvement of endothelial pathways and more complex ligand-receptor interactions, relative to their marginal zones. Plaque-EVs were predicted to mediate communication with ECs. Pathway enrichment analysis delineated an endothelial signature with roles in angiogenesis and neovascularization – well-known indices of plaque instability. This was validated functionally, wherein human carotid symptomatic plaque EVs induced sprouting angiogenesis in comparison to their matched marginal zones. Conclusion: Our findings indicate that EVs may drive dynamic changes in plaques through EV- vascular cell communication and effector functions that typify vulnerability to rupture, precipitating symptomatic disease. The discovery of endothelial-directed angiogenic processes mediated by EVs creates new therapeutic avenues for atherosclerosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85baba07ce353659c168561fd588df1f617dc3d1" target='_blank'>
              Multiomics unveils extracellular vesicle-driven mechanisms of endothelial communication in human carotid atherosclerosis
              </a>
            </td>
          <td>
            S. Raju, Mandy E Turner, C. Cao, Majed Abdul-Samad, Neil Punwasi, M. Blaser, Rachel M E Cahalane, S. Botts, Kamal K. Prajapati, Sarvatit Patel, Ruilin Wu, D. Gustafson, Natalie J. Galant, Lindsey Fiddes, M. Chemaly, Ulf Hedin, L. Matic, Michael Seidman, Vallijah Subasri, Sasha A. Singh, Elena Aikawa, Jason E. Fish, Kathryn L. Howe
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Introduction Immune cell interactions and metabolic changes are crucial in determining the tumor microenvironment and affecting various clinical outcomes. However, the clinical significance of metabolism evolution of immune cell evolution in colorectal cancer (CRC) remains unexplored. Methods Single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing data were acquired from TCGA and GEO datasets. For the analysis of macrophage differentiation trajectories, we employed the R packages Seurat and Monocle. Consensus clustering was further applied to identify the molecular classification. Immunohistochemical results from AOM and AOM/DSS models were used to validate macrophage expression. Subsequently, GSEA, ESTIMATE scores, prognosis, clinical characteristics, mutational burden, immune cell infiltration, and the variance in gene expression among different clusters were compared. We constructed a prognostic model and nomograms based on metabolic gene signatures identified through the MEGENA framework. Results We found two heterogeneous groups of M2 macrophages with various clinical outcomes through the evolutionary process. The prognosis of Cluster 2 was poorer. Further investigation showed that Cluster 2 constituted a metabolically active group while Cluster 1 was comparatively metabolically inert. Metabolic variations in M2 macrophages during tumor development are related to tumor prognosis. Additionally, Cluster 2 showed the most pronounced genomic instability and had highly elevated metabolic pathways, notably those associated with the ECM. We identified eight metabolic genes (PRELP, NOTCH3, CNOT6, ASRGL1, SRSF1, PSMD4, RPL31, and CNOT7) to build a predictive model validated in CRC datasets. Then, a nomogram based on the M2 risk score improved predictive performance. Furthermore, our study demonstrated that immune checkpoint inhibitor therapy may benefit patients with low-risk. Discussion Our research reveals underlying relationships between metabolic phenotypes and immunological profiles and suggests a unique M2 classification technique for CRC. The identified gene signatures may be key factors linking immunity and tumor metabolism, warranting further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b77333ce5794bf43c82ae5c8985b3dc9293d6dd" target='_blank'>
              M2 Macrophage Classification of Colorectal Cancer Reveals Intrinsic Connections with Metabolism Reprogramming and Clinical Characteristics
              </a>
            </td>
          <td>
            Fengxing Huang, Youwei Wang, Yu Shao, Runan Zhang, Mengting Li, Lan Liu, Qiu-xia Zhao
          </td>
          <td>2024-07-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Glioma, a prevalent and deadly brain tumor, is marked by significant cellular heterogeneity and metabolic alterations. However, the comprehensive cell-of-origin and metabolic landscape in high-grade (Glioblastoma Multiforme, WHO grade IV) and low-grade (Oligoastrocytoma, WHO grade II) gliomas remains elusive. Methods In this study, we undertook single-cell transcriptome sequencing of these glioma grades to elucidate their cellular and metabolic distinctions. Following the identification of cell types, we compared metabolic pathway activities and gene expressions between high-grade and low-grade gliomas. Results Notably, astrocytes and oligodendrocyte progenitor cells (OPCs) exhibited the most substantial differences in both metabolic pathways and gene expression, indicative of their distinct origins. The comprehensive analysis identified the most altered metabolic pathways (MCPs) and genes across all cell types, which were further validated against TCGA and CGGA datasets for clinical relevance. Discussion Crucially, the metabolic enzyme phosphodiesterase 8B (PDE8B) was found to be exclusively expressed and progressively downregulated in astrocytes and OPCs in higher-grade gliomas. This decreased expression identifies PDE8B as a metabolism-related oncogene in IDH-mutant glioma, marking its dual role as both a protective marker for glioma grading and prognosis and as a facilitator in glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32a8dd10db0de50c204790d7bd0cb6e815520890" target='_blank'>
              Single-cell transcriptomic analysis identifies downregulated phosphodiesterase 8B as a novel oncogene in IDH-mutant glioma
              </a>
            </td>
          <td>
            Z. He, Yu Peng, Duo Wang, Chen Yang, Chengzhi Zhou, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f58160d8cdc5bd6e2466eb7c04abf11234ebf3" target='_blank'>
              Patient-derived mini-colons enable long-term modeling of tumor-microenvironment complexity.
              </a>
            </td>
          <td>
            L. F. Lorenzo-Martín, N. Broguiere, Jakob Langer, Lucie Tillard, Mike Nikolaev, George Coukos, K. Homicsko, Matthias P. Lutolf
          </td>
          <td>2024-07-02</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Kidney allograft rejections are orchestrated by a variety of immune cells. Because of the complex histopathologic features, accurate pathological diagnosis poses challenges even for expert pathologists. The objective of this study was to unveil novel spatial indices associated with transplant rejection by using a spatial bioinformatic approach using 36-plex immunofluorescence image data.


METHODS
The image obtained from 11 T cell-mediated rejection (TCMR) and 12 antibody-mediated rejection (AMR) samples were segmented into 753 737 single cells using DeepCell's Mesmer algorithm. These cells were categorized into 13 distinct cell types through unsupervised clustering based on their biomarker expression profiles. Cell neighborhood analysis allowed us to stratify kidney tissue into 8 distinct neighborhood components consisting of unique cell type enrichment profiles.


RESULTS
In contrast to TCMR samples, AMR samples exhibited a higher frequency of neighborhood components that were characterized by an enrichment of CD31+ endothelial cells. Although the overall frequency of CD68+ macrophages in AMR samples was not significantly high, CD68+ macrophages within endothelial cell-rich lesions exhibited a significantly higher frequency in AMR samples than TCMR samples. Furthermore, the frequency of interactions between CD31+ cells and CD68+ cells was significantly increased in AMR samples, implying the pivotal role of macrophages in AMR pathogenesis. Importantly, patients demonstrating a high frequency of CD31:CD68 interactions experienced significantly poorer outcomes in terms of chronic AMR progression.


CONCLUSIONS
Collectively, these data indicate the potential of spatial bioinformatic as a valuable tool for aiding in pathological diagnosis and for uncovering new insights into the mechanisms underlying transplant rejection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e65530b2545156cbbf6161b36bc3af83e357d01" target='_blank'>
              Unveiling Spatial Immune Cell Profile in Kidney Allograft Rejections Using 36-plex Immunofluorescence Imaging.
              </a>
            </td>
          <td>
            Toshihito Hirai, Ayano Kondo, Tomokazu Shimizu, H. Fukuda, Daisuke Tokita, Toshio Takagi, Aaron T. Mayer, Hideki Ishida
          </td>
          <td>2024-06-24</td>
          <td>Transplantation</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf457125c2dfc7b5223e6aad5f2e990ae804842e" target='_blank'>
              Breast cancer patient-derived organoids for the investigation of patient-specific tumour evolution
              </a>
            </td>
          <td>
            S. Mazzucchelli, L. Signati, L. Messa, Alma Franceschini, A. Bonizzi, L. Castagnoli, P. Gasparini, C. Consolandi, E. Mangano, P. Pelucchi, I. Cifola, Tania Camboni, M. Severgnini, Laura Villani, Barbara Tagliaferri, S. Carelli, S. Pupa, Cristina Cereda, Fabio Corsi
          </td>
          <td>2024-06-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Ovarian and other peritoneal cancers have a strong tendency to metastasize into the surrounding adipose tissue. This study describes an effect of the adipose microenvironment on upregulation of sialic acid-containing glycans in ovarian cancer (OC). Heterogeneous populations of glycosylated OC tumors converged to a highly sialylated cell state that regulates tumorigenesis in an immune-dependent manner. Methods We modeled the adipose microenvironment by conditioning growth media with human patient-derived adipose tissue. OC cell lines grown in the presence vs. absence of adipose conditioned media (ACM) were characterized by transcriptomics, western blotting, and chemical biology glycan labeling methods. Fluorescence-activated cell sorting was used to separate adipose-driven upregulation of hypersialylated (“SNA-high”) vs. hyposialylated (“SNA-low”) OC subpopulations. The two subpopulations were characterized by further transcriptomic and quantitative polymerase chain reaction analyses, then injected into a syngeneic mouse model. Immune system involvement was implicated using wild type and athymic nude mice with a primary endpoint of overall survival. Results Adipose conditioning resulted in upregulation of sialyltransferases ST3GAL1, ST6GAL1, ST6GALNAC3, and ST8Sia1. In culture, OC cells displayed two distinct sialylated subpopulations that were stable for up to 9 passages, suggesting inherent heterogeneity in sialylation that is maintained throughout cell division and media changes. OC tumors that implanted in the omental adipose tissue exclusively reprogrammed to the highly sialylated subpopulation. In wild type C57BL/6 mice, only the hypersialylated SNA-high subpopulation implanted in the adipose, whereas the hyposialylated SNA-low subpopulation failed to be tumorigenic (p=0.023, n=5). In the single case where SNA-low established a tumor, post-mortem analysis revealed reprogramming of the tumor to the SNA-high state in vivo. In athymic nude mice, both subpopulations rapidly formed tumors, implicating a role of the adaptive immune system. Conclusions These findings suggest a model of glycan-dependent tumor evolution wherein the adipose microenvironment reprograms OC to a tumorigenic state that resists the adaptive immune system. Mechanistically, adipose factors upregulate sialyltransferases. To our knowledge, this is the first demonstration of the effect of adipose microenvironment on OC tumor sialylation. Our results set the stage for translational applications targeting sialic acid pathways in OC and other peritoneal cancer tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ade0f16e59c010463533d8ecb04aec54d965a7b" target='_blank'>
              Adipose microenvironment promotes hypersialylation of ovarian cancer cells
              </a>
            </td>
          <td>
            Alexandra Fox, Garry D. Leonard, Nicholas K Adzibolosu, Terrence Wong, Roslyn Tedja, Sapna Sharma, Radhika Gogoi, Robert Morris, Gil Mor, C. Fehl, A. Alvero
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cellular biomechanics plays critical roles in cancer metastasis and tumor progression. Existing studies on cancer cell biomechanics are mostly conducted in flat 2D conditions, where cells’ behavior can differ considerably from those in 3D physiological environments. Despite great advances in developing 3D in vitro models, probing cellular elasticity in 3D conditions remains a major challenge for existing technologies. In this work, we utilize optical Brillouin microscopy to longitudinally acquire mechanical images of growing cancerous spheroids over the period of eight days. The dense mechanical mapping from Brillouin microscopy enables us to extract spatially resolved and temporally evolving mechanical features that were previously inaccessible. Using an established machine learning algorithm, we demonstrate that incorporating these extracted mechanical features significantly improves the classification accuracy of cancer cells, from 74% to 95%. Building on this finding, we have developed a deep learning pipeline capable of accurately differentiating cancerous spheroids from normal ones solely using Brillouin images, suggesting the mechanical features of cancer cells could potentially serve as a new biomarker in cancer classification and detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872d3dafd293741f9c83632444cfe12e1fdf3df8" target='_blank'>
              Mechanical evolution of metastatic cancer cells in three-dimensional microenvironment
              </a>
            </td>
          <td>
            Karlin Hilai, Daniil Grubich, Marcus Akrawi, Hui Zhu, Razanne Zaghloul, Chenjun Shi, Man Do, Dongxiao Zhu, Jitao Zhang
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Intra-tumoral B cells mediate a plethora of immune effector mechanisms with key roles in anti-tumor immunity and serve as positive prognostic indicators in a variety of solid tumor types, including epithelial ovarian cancer (EOC). Several aspects of intra-tumoral B cells remain unclear, such as their state of activation, antigenic repertoires, and capacity to mature into plasma cells. Methods B lymphocytes were isolated from primary EOC tissue and malignant ascites and were maintained in cell culture medium. The stably maintained cell lines were profiled with flow cytometry and B cell receptor sequencing. Secreted antibodies were tested with a human proteome array comprising more than 21,000 proteins, followed by ELISA for validation. Originating tumor samples were used for spatial profiling with chip cytometry. Results Antibody-secreting B lymphocytes were isolated from the ovarian tumor microenvironment (TME) of four different EOC patients. The highly clonal cell populations underwent spontaneous immortalization in vitro, were stably maintained in an antibody-secreting state, and showed presence of Epstein-Barr viral (EBV) proteins. All originating tumors had high frequency of tumor-infiltrating B cells, present as lymphoid aggregates, or tertiary lymphoid structures. The antigens recognized by three of the four cell lines are coil-coil domain containing protein 155 (CCDC155), growth factor receptor-bound protein 2 (GRB2), and pyruvate dehydrogenase phosphatase2 (PDP2), respectively. Anti-CCDC155 circulating IgG antibodies were detected in 9 of 20 (45%) of EOC patients’ sera. Tissue analyses with multiparameter chip cytometry shows that the antibodies secreted by these novel human B cell lines engage their cognate antigens on tumor cells. Discussion These studies demonstrate that within the tumor-infiltrating lymphocyte population in EOC resides a low frequency population of antibody-secreting B cells that have been naturally exposed to EBV. Once stably maintained, these novel cell lines offer unique opportunities for future studies on intratumor B cell biology and new target antigen recognition, and for studies on EBV latency and/or viral reactivation in the TME of non-EBV related solid tumors such as the EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91568ddf7942710ae6ed5b0bc9abf380b3db7b97" target='_blank'>
              Characterization of latently infected EBV+ antibody-secreting B cells isolated from ovarian tumors and malignant ascites
              </a>
            </td>
          <td>
            Lixin Zhang, Mary Strange, E. Elishaev, Syed Zaidi, F. Modugno, M. Radolec, Robert P Edwards, Olivera J Finn, A. Vlad
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The purpose of this study was to elucidate the relationship between distinct brain regions and molecular subtypes in glioblastoma (GB), focusing on integrating modern statistical tools and molecular profiling to better understand the heterogeneity of Isocitrate Dehydrogenase wild-type (IDH-wt) gliomas. Methods This retrospective study comprised 441 patients diagnosed with new IDH-wt glioma between 2009 and 2020 at Heidelberg University Hospital. The diagnostic process included preoperative magnetic resonance imaging and molecular characterization, encompassing IDH-status determination and subclassification, through DNA-methylation profiling. To discern and map distinct brain regions associated with specific methylation subtypes, a support-vector regression-based lesion-symptom mapping (SVR-LSM) was employed. Lesion maps were adjusted to 2 mm³ resolution. Significance was assessed with beta maps, using a threshold of P < .005, with 10 000 permutations and a cluster size minimum of 100 voxels. Results Of 441 initially screened glioma patients, 423 (95.9%) met the inclusion criteria. Following DNA-methylation profiling, patients were classified into RTK II (40.7%), MES (33.8%), RTK I (18%), and other methylation subclasses (7.6%). Between molecular subtypes, there was no difference in tumor volume. Using SVR-LSM, distinct brain regions correlated with each subclass were identified: MES subtypes were associated with left-hemispheric regions involving the superior temporal gyrus and insula cortex, RTK I with right frontal regions, and RTK II with 3 clusters in the left hemisphere. Conclusions This study linked molecular diversity and spatial features in glioblastomas using SVR-LSM. Future studies should validate these findings in larger, independent cohorts to confirm the observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61055e06d68add13a86ef99666992a593250b06" target='_blank'>
              Unraveling glioblastoma diversity: Insights into methylation subtypes and spatial relationships
              </a>
            </td>
          <td>
            Martha Foltyn-Dumitru, Haidar Alzaid, Aditya Rastogi, U. Neuberger, Felix Sahm, T. Kessler, Wolfgang Wick, M. Bendszus, Phillipp Vollmuth, Marianne Schell
          </td>
          <td>2024-06-28</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3b8fcb2d930ae2a4d496aebe20c735fc43c688" target='_blank'>
              Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping
              </a>
            </td>
          <td>
            Herman Netskar, Aline Pfefferle, J. Goodridge, Ebba Sohlberg, Olli Dufva, Sarah A. Teichmann, D. Brownlie, J. Michaëlsson, N. Marquardt, Trevor Clancy, Amir Horowitz, Karl-Johan Malmberg
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Bone pain is a presenting feature of bone cancers such as osteosarcoma (OS), relayed by skeletal-innervating peripheral afferent neurons. Potential functions of tumor-associated sensory neurons in bone cancers beyond pain sensation are unknown. To uncover neural regulatory functions, a chemical-genetic approach in mice with a knock-in allele for TrkA was used to functionally perturb sensory nerve innervation during OS growth and disease progression. TrkA inhibition in transgenic mice led to significant reductions in sarcoma-associated sensory innervation and vascularization, tumor growth and metastasis, and prolonged overall survival. Single-cell transcriptomics revealed that sarcoma denervation was associated with phenotypic alterations in both OS tumor cells and cells within the tumor microenvironment, and with reduced calcitonin gene-related peptide (CGRP) and vascular endothelial growth factor (VEGF) signaling. Multimodal and multi-omics analyses of human OS bone samples and human dorsal root ganglia neurons further implicated peripheral innervation and neurotrophin signaling in OS tumor biology. In order to curb tumor-associated axonal ingrowth, we next leveraged FDA-approved bupivacaine liposomes leading to significant reductions in sarcoma growth, vascularity, as well as alleviation of pain. In sum, TrkA-expressing peripheral neurons positively regulate key aspects of OS progression and sensory neural inhibition appears to disrupt calcitonin receptor signaling (CALCR) and VEGF signaling within the sarcoma microenvironment leading to significantly reduced tumor growth and improved survival. These data suggest that interventions to prevent pathological innervation of osteosarcoma represent a novel adjunctive therapy to improve clinical outcomes and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38e8f176dcc8dd09eaf33b5874e7e585465ebfc" target='_blank'>
              TrkA+ sensory neurons regulate osteosarcoma proliferation and vascularization to promote disease progression
              </a>
            </td>
          <td>
            Q. Qin, Sowmya Ramesh, Zhao Li, Lingke Zhong, M. Cherief, Mary Archer, X. Xing, N. Thottappillil, M. Gomez-Salazar, Mingxin Xu, Manyu Zhu, Leslie Chang, Ankit Uniyal, Khadija Mazhar, Monisha Mittal, Edward F. McCarthy, Carol D. Morris, Benjamin Levi, Yun Guan, Thomas L. Clemens, Theodore J. Price, Aaron W. James
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Head and neck cancer disparities in outcomes for Black Americans have been well recognized. However, the specific drivers of the inferior outcomes remain poorly understood. We investigated the biologic features of patient oral cancers and performed a follow-up study of the patient post-surgery recurrences and metastases aiming to explore potential mechanisms that might underpin the poorer outcomes among Black American patients. It was found that high levels of tumor stemness and tumor-promoting stromal characteristics were linked to patient recurrence and metastasis. There were more cases of Black American than White American exhibiting high stemness traits and strong tumor-promoting stromal features associated with tumor recurrences and metastases, although the investigated cases displayed comparable clinical diagnoses. Our findings revealed that the differences in tumor stemness and stromal property among cancers with similar diagnoses contribute to patient outcome disparities. Abstract Black Americans (BAs) with head and neck cancer (HNC) have worse survival outcomes compared to the White patients. While HNC disparities in patient outcomes for BAs have been well recognized, the specific drivers of the inferior outcomes remain poorly understood. Here, we investigated the biologic features of patient tumor specimens obtained during the surgical treatment of oral cancers and performed a follow-up study of the patients’ post-surgery recurrences and metastases with the aim to explore whether tumor biologic features could be associated with the poorer outcomes among BA patients compared with White American (WA) patients. We examined the tumor stemness traits and stromal properties as well as the post-surgery recurrence and metastasis of oral cancers among BA and WA patients. It was found that high levels of tumor self-renewal, invasion, tumorigenesis, metastasis, and tumor-promoting stromal characteristics were linked to post-surgery recurrence and metastasis. There were more BA than WA patients demonstrating high stemness traits and strong tumor-promoting stromal features in association with post-surgery tumor recurrences and metastases, although the investigated cases displayed clinically comparable TNM stages and histological grades. These findings demonstrated that the differences in tumor stemness and stromal property among cancers with comparable clinical diagnoses contribute to the outcome disparity in HNCs. More research is needed to understand the genetic and molecular basis of the biologic characteristics underlying the inferior outcomes among BA patients, so that targeting strategies can be developed to reduce HNC disparity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcad9291ce4667fd7a96a4d38826ce6e7b1c58" target='_blank'>
              Tumor Cell Stemness and Stromal Cell Features Contribute to Oral Cancer Outcome Disparity in Black Americans
              </a>
            </td>
          <td>
            S. Mirshahidi, Isabella J. Yuan, Zhong Chen, Alfred Simental, Steve C. Lee, Pedro A Andrade Filho, Thomas Murry, Feng Zeng, Penelope Duerksen-Hughes, Charles Wang, Xiangpeng Yuan
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over decades, cancer understanding has advanced significantly at molecular and cellular levels, leading to various therapies based on intra-/inter-cellular networks. Despite this, cancer still remains a leading cause of death globally. The primary driver of cancer mortality is metastasis, responsible for about 90% of cancer deaths, due to unclear pathophysiological mechanisms that complicate treatment development. The Notch signaling pathway, a crucial intercellular network in many cancers, has been extensively studied and therapies targeting the Notch pathway also have been well-studied based on inhibiting various stages of Notch activation. On the other hand, Notch signaling’s role varies between cancers; for instance, in non-small cell lung cancer, Notch1 and Notch2 have opposing effects compared to their roles in embryonal brain tumors. In this study, we assumed a scenario of multiple cancers with contrasting Notch signaling pathways and explored optimal targeted therapies for reducing cancer cells by developing two mathematical models with contrasting Notch signaling pathways. The proposed therapies were compared with existing ones, and strategies were investigated to reduce cancer cell numbers for different stage of cancer. We found that that multiple cancers with contrasting Notch networks can be controlled by a common targeted signal network. Combination therapy enhancing Notch production may be most effective in early-stage cancer, while cleavage therapies may be more effective in late-stage cancer. Our study also suggests that optimal treatment should consider the cancer stage, with careful selection and ordering of medication therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0b0aca24e705cffc10843b8484f56eae84233b" target='_blank'>
              Optimal targeted therapy for multiple cancers based on contrastive Notch signaling networks
              </a>
            </td>
          <td>
            Tamaki Wakamoto, S. Seirin-Lee
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="T cell acute lymphoblastic leukemia (T-ALL) is an aggressive leukemia predominantly affecting adolescents and young adults. T-ALL relapses are characterized by chemotherapy resistance, cellular heterogeneity and dismal outcome. To gain a deeper understanding of the cellular heterogeneity and mechanisms driving relapse, we conducted single-cell full-length RNA sequencing of 13 matched pediatric T-ALL patient-derived xenografts (PDX) samples obtained at initial diagnosis and relapse, generating the to date most comprehensive longitudinal single cell study in paired T-ALL samples, along with 5 non-relapsing PDX samples collected at initial diagnosis. This dataset identifies considerable transcriptomic diversity among individual T-ALL cell populations. Notably however, 11 of the 18 patients exhibit a small T-ALL cell subpopulation with a shared set of gene regulatory networks characterized by a common set of active regulons, expression patterns and splice isoforms. This profile involves the upregulation of a stem-like cell signature with enrichment of cell adhesion, and inhibition of the cell cycle and metabolic activity. Comprehensive investigations of these networks identify transcripts enforcing drug resistance through NF-κB expression and TGF-β signaling and of anti-apoptotic T cell signaling. Longitudinal monitoring of these stem-like cells demonstrates this subpopulation to account for only a small proportion of leukemia cells initially with a substantial expansion at relapse suggesting resistance to first line therapy. Chemotherapy resistance is functionally corroborated through in vitro and in vivo drug testing. We thus report the discovery of inherently treatment-resistant stem-like T-ALL cell populations underscoring the potential for devising future therapeutic strategies aimed at targeting stemness pathways in pediatric T-ALL. Key Points Single-cell full-length RNA sequencing reveals dormant, treatment-resistant cells expanding upon T cell acute lymphoblastic leukemia relapse. Stem-like T-ALL cells exhibit a shared gene regulatory network conferring chemotherapy resistance and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88dbfdc32eaada9aa463b5fbbffd8e6808956ec" target='_blank'>
              Role of Stem-Like Cells in Chemotherapy Resistance and Relapse in pediatric T Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Julia Costea, Kerstin K. Rauwolf, Pietro Zafferani, T. Rausch, A. Mathioudaki, Judith B. Zaugg, M. Schrappe, C. Eckert, G. Escherich, Jean-Pierre Bourquin, B. Bornhauser, Andreas E. Kulozik, J. Korbel
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma poses a formidable challenge among primary brain tumors: its tumorigenic stem cells, capable of self-renewal, proliferation, and differentiation, contribute substantially to tumor initiation and therapy resistance. These glioblastoma stem cells (GSCs), resembling conventional stem and progenitor cells, adopt pathways critical for tissue development and repair, promoting uninterrupted tumor expansion. Long non-coding RNAs (lncRNAs), a substantial component of the human transcriptome, have garnered considerable interest for their pivotal roles in normal physiological processes and cancer pathogenesis. They display cell- or tissue-specific expression patterns, and extensive investigations have highlighted their impact on regulating GSC properties and cellular differentiation, thus offering promising avenues for therapeutic interventions. Consequently, lncRNAs, with their ability to exert regulatory control over tumor initiation and progression, have emerged as promising targets for innovative glioblastoma therapies. This review explores notable examples of GSC-associated lncRNAs and elucidates their functional roles in driving glioblastoma progression. Additionally, we delved deeper into utilizing a 3D in vitro model for investigating GSC biology and elucidated four primary methodologies for targeting lncRNAs as potential therapeutics in managing glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ed3ed8d6a1c37f2db5f6f03648c49bfe8a7955" target='_blank'>
              Glioblastoma stem cell long non-coding RNAs: therapeutic perspectives and opportunities
              </a>
            </td>
          <td>
            R. Hazra, Rinku Debnath, Arati Tuppad
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive neuroendocrine skin cancer with a ~50% response rate to immune checkpoint blockade (ICB) therapy. To identify predictive biomarkers, we integrated bulk and single-cell RNA-seq with spatial transcriptomics from a cohort of 186 samples from 116 patients, including bulk RNA-seq from 14 matched pairs pre- and post-ICB. In non-responders, tumors show evidence of increased tumor proliferation, neuronal stem cell markers, and IL-1. Responders have increased type I/II interferons and pre-existing tissue resident (Trm) CD8 or Vd1 gd T cells that functionally converge with overlapping antigen-specific transcriptional programs and clonal expansion of public TCRs. Spatial transcriptomics demonstrated co-localization of T cells with B and dendritic cells, which supply chemokines and co-stimulation. Lastly, ICB significantly increased clonal expansion or recruitment of Trm and Vd1 cells in tumors specifically in responders, underscoring their therapeutic importance. These data identify potential clinically actionable biomarkers and therapeutic targets for MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc8fff7c0c805192f9c9a67df2e29b038f02c52" target='_blank'>
              Pre-existing skin-resident CD8 and gd T cell circuits mediate immune response in Merkel cell carcinoma and predict immunotherapy efficacy.
              </a>
            </td>
          <td>
            Z. Reinstein, Yue Zhang, Oscar E. Ospina, Matt D Nichols, Victoria A Chu, Alvaro de Mingo Pulido, Karol Prieto, Jonathan V Nguyen, Rui Yin, Carlos M Moran Segura, Ahmed Usman, B. Sell, Spencer Ng, Janis V de la Iglesia, Sunandana Chandra, Jeffrey A Sosman, Raymond J. Cho, Jeffrey B. Cheng, E. Ivanova, S. Koralov, R. Slebos, C. Chung, N. Khushalani, Jane L. Messina, A. Sarnaik, J. Zager, V. Sondak, Charles J. Vaske, Sungjune Kim, A. Brohl, Xinlei Mi, Brian G. Pierce, Xuefeng Wang, Brooke L. Fridley, Kenneth Y. Tsai, Jaehyuk Choi
          </td>
          <td>2024-07-26</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Hypopharyngeal squamous cell carcinoma (SCC) is a highly aggressive cancer with a poor prognosis, particularly in advanced stages where concurrent chemoradiotherapy (CCRT) is used for treatment. However, resistance to CCRT poses a significant challenge, often leading to treatment failure and disease progression. This study explores the tumor microenvironment (TME) of hypopharyngeal SCC to understand the molecular mechanisms underlying CCRT resistance. Using spatial transcriptomics (ST), we analyzed tissue samples from patients with locally advanced hypopharyngeal SCC, distinguishing between those who were CCRT-resistant and those who were CCRT-naive. The analysis revealed six distinct cellular clusters within the TME, including a prominent epithelio-immune cellular area in CCRT-resistant tissues. SPP1 was identified as a key gene with significantly higher expression in CCRT-resistant samples, specifically within macrophages. Further investigation showed that SPP1+ macrophages interacted with malignant epithelial cells through SPP1-CD44 and SPP1-ITGB1 ligand-receptor pairs. These interactions were primarily localized in the peri-tumoral and intra-tumoral regions, highlighting their potential role in driving CCRT resistance. Our findings suggest that SPP1+ macrophages contribute to the resistant phenotype in hypopharyngeal SCC by modulating the TME and interacting with cancer cells. Understanding these interactions offers valuable insights into the mechanisms of CCRT resistance and may inform the development of targeted therapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d07108a9430fa63556b14b31c6b832a1633f5b4" target='_blank'>
              Spatial transcriptomics unveils landscape of resistance to concurrent chemo-radiotherapy in hypopharyngeal squamous cell carcinoma: the role of SPP1+ macrophages
              </a>
            </td>
          <td>
            J. Ohn, Sungwoo Bae, Hongyoon Choi, In Gul Kim, Kwon Joong Na, Eun-Jae Chung
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dendritic cells (DCs) play a pivotal role in immune surveillance, acting as sentinels that coordinate immune responses within tissues. Although differences in the identity and functional states of DC subpopulations have been identified through multiparametric flow cytometry and single-cell RNA sequencing, these methods do not provide information about the spatial context in which the cells are located. This knowledge is crucial for understanding tissue organisation and cellular cross-talk. Recent developments in multiplex imaging techniques can now offer insights into this complex spatial and functional landscape. This review provides a concise overview of these imaging methodologies, emphasising their application in identifying DCs to delineate their tissue-specific functions and aiding newcomers in navigating this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3753794a82ec4a0185d0a53fde9fcef8846c138" target='_blank'>
              Multiplexed imaging to reveal tissue dendritic cell spatial localisation and function.
              </a>
            </td>
          <td>
            Giuseppe Rocca, Marco Galli, Anna Celant, Giulia Stucchi, L. Marongiu, Stefano Cozzi, M. Innocenti, Francesca Granucci
          </td>
          <td>2024-07-05</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Chromobox 2 (CBX2), an epigenetic reader and component of polycomb repressor complex 1, is highly expressed in >75% of high-grade serous carcinoma. Increased CBX2 expression is associated with poorer survival, whereas CBX2 knockdown leads to improved chemotherapy sensitivity. In a high-grade serous carcinoma immune-competent murine model, knockdown of CBX2 decreased tumor progression. We sought to explore the impact of modulation of CBX2 on the tumor immune microenvironment (TIME), understanding that the TIME plays a critical role in disease progression and development of therapy resistance. Exploration of existing datasets demonstrated that elevated CBX2 expression significantly correlated with specific immune cell types in the TIME. RNA sequencing and pathway analysis of differentially expressed genes demonstrated immune signature enrichment. Confocal microscopy and co-culture experiments found that modulation of CBX2 leads to increased recruitment and infiltration of macrophages. Flow cytometry of macrophages cultured with CBX2-overexpressing cells showed increased M2-like macrophages and decreased phagocytosis activity. Cbx2 knockdown in the Trp53-null, Brca2-null ID8 syngeneic murine model (ID8 Trp53−/−Brca2−/−) led to decreased tumor progression compared with the control. NanoString immuno-oncology panel analysis suggested that knockdown in Cbx2 shifts immune cell composition, with an increase in macrophages. Multispectral immunohistochemistry (mIHC) further confirmed an increase in macrophage infiltration. Increased CBX2 expression leads to recruitment and polarization of protumor macrophages, and targeting CBX2 may serve to modulate the TIME to enhance the efficacy of immune therapies. Significance: CBX2 expression correlates with the TIME. CBX2 modulation shifts the macrophage population, potentially leading to an immunosuppressive microenvironment, highlighting CBX2 as a target to improve efficacy of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e63e5432f45d9d8f73521a642c20ec768fe66f0" target='_blank'>
              Tumor-Intrinsic Activity of Chromobox 2 Remodels the Tumor Microenvironment in High-grade Serous Carcinoma
              </a>
            </td>
          <td>
            Ritsuko Iwanaga, Tomomi M Yamamoto, Karina Gomez, Lily L. Nguyen, Elizabeth R. Woodruff, Miriam D Post, Railey G Mikeska, Etienne Danis, T. Danhorn, M. Boorgula, Siddhartha S Mitra, Nicole A. Marjon, Benjamin G Bitler, Lindsay W Brubaker
          </td>
          <td>2024-07-10</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251acda7df20c5eff2f9bfbbad717de02554a201" target='_blank'>
              Unlocking the potential of HHLA2: identifying functional immune infiltrating cells in the tumor microenvironment and predicting clinical outcomes in laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenjing Li, Jianqing You, Haixiang Xue, Yi Liu, Junjun Chen, Xiao Zheng, Lujun Chen, Changping Wu
          </td>
          <td>2024-08-06</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction This study investigates the role of Fibroblast Activation Protein (FAP)-positive cancer-associated fibroblasts (FAP+CAF) in shaping the tumor immune microenvironment, focusing on its association with immune cell functionality and cytokine expression patterns. Methods Utilizing immunohistochemistry, we observed elevated FAP+CAF density in metastatic versus primary renal cell carcinoma (RCC) tumors, with higher FAP+CAF correlating with increased T cell infiltration in RCC, a unique phenomenon illustrating the complex interplay between tumor progression, FAP+CAF density, and immune response. Results Analysis of immune cell subsets in FAP+CAF-rich stromal areas further revealed significant correlations between FAP+ stroma and various T cell types, particularly in RCC and non-small cell lung cancer (NSCLC). This was complemented by transcriptomic analyses, expanding the range of stromal and immune cell subsets interrogated, as well as to additional tumor types. This enabled evaluating the association of these subsets with tumor infiltration, tumor vascularization and other components of the tumor microenvironment. Our comprehensive study also encompassed cytokine, angiogenesis, and inflammation gene signatures across different cancer types, revealing heterogeneous cellular composition, cytokine expressions and angiogenic profiles. Through cytokine pathway profiling, we explored the relationship between FAP+CAF density and immune cell states, uncovering potential immunosuppressive circuits that limit anti-tumor activity in tumor-resident immune cells. Conclusions These findings underscore the complexity of tumor biology and the necessity for personalized therapeutic and patient enrichment approaches. The insights gathered from FAP+CAF prevalence, immune infiltration, and gene signatures provide valuable perspectives on tumor microenvironments, aiding in future research and clinical strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340394b70e234f1340a6e340d3b01ec0cd62d31a" target='_blank'>
              Investigating the complex interplay between fibroblast activation protein α-positive cancer associated fibroblasts and the tumor microenvironment in the context of cancer immunotherapy
              </a>
            </td>
          <td>
            A. Kraxner, Franziska Braun, Wei-Yi Cheng, Tai-Hsien Ou Yang, Shweta V. Pipaliya, Marta Cañamero, Emilia Andersson, S. V. Harring, Sebastian Dziadek, Ann-Marie E. Bröske, Maurizio Ceppi, Tamara Tanos, Volker Teichgräber, Jehad Charo
          </td>
          <td>2024-07-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is a multifaceted condition characterized by significant diversity in its molecular and pathological subtypes and clinical manifestation. Despite the progress made in the treatment of DLBCL through the development of novel drugs, an estimated one-third of patients encounter relapse or acquire refractory disease. The tumor microenvironment (TME) of DLBCL, a complex network consisting of cellular and noncellular components that engage in interactions with the tumor, is a parameter that is gaining increasing attention. The TME comprises both the immune and nonimmune microenvironments. The immune microenvironment comprises natural killer (NK) cells, dendritic cells (DCs), tumor-associated macrophages (TAMs), neutrophils, myeloid-derived suppressor cells (MDSCs), and T and B lymphocytes. The nonimmune microenvironment consists of the extracellular matrix (ECM), cancer-associated fibroblasts (CAFs), mesenchymal stromal cells, and other molecules that are secreted. Despite ongoing research, the exact impact of these components and their interaction on the progression of the disease remains elusive. A comprehensive review of significant discoveries concerning the cellular and noncellular constituents, molecular characteristics, and treatment response and prognosis of the TME in DLBCL, as well as the potential targeting of the TME with novel therapeutic approaches, is provided in this article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842f1b179cae9c35039714b759e7f43b1c1c5ae1" target='_blank'>
              Unraveling the Immune Microenvironment in Diffuse Large B-Cell Lymphoma: Prognostic and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            E. Koumpis, A. Papoudou-Bai, K. Papathanasiou, E. Kolettas, Panagiotis Kanavaros, E. Hatzimichael
          </td>
          <td>2024-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e877f4d47a5850d3ceab780a2d2a2472c654fb" target='_blank'>
              High-dimensional single-cell analysis of human natural killer cell heterogeneity
              </a>
            </td>
          <td>
            Lucas Rebuffet, J. Melsen, Bertrand Escalière, Daniela Basurto-Lozada, A. Bhandoola, N. Björkström, Y. Bryceson, R. Castriconi, Frank Cichocki, Marco Colonna, Daniel M Davis, Andreas Diefenbach, Yi Ding, M. Haniffa, Amir Horowitz, Lewis L. Lanier, Karl-Johan Malmberg, Jeffrey S Miller, Lorenzo Moretta, Émilie Narni-Mancinelli, L. A. O'Neill, C. Romagnani, Dylan G Ryan, S. Sivori, Dan Sun, Constance Vagne, Eric Vivier
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>4</td>
          <td>67</td>
        </tr>

        <tr id="Lack of diversity and proportionate representation in genomics datasets and databases contributes to inequity in healthcare outcomes globally1,2. The relationships of human diversity with biological and biomedical phenotypes are pervasive3, yet remain understudied, particularly in a single-cell genomics context. Here we present the Asian Immune Diversity Atlas (AIDA), a multi-national single-cell RNA-sequencing (scRNA-seq) healthy reference atlas of human immune cells. AIDA comprises 1,265,624 circulating immune cells from 619 healthy donors and 6 controls, spanning 7 population groups across 5 countries. AIDA is one of the largest healthy blood datasets in terms of number of cells, and also the most diverse in terms of number of population groups. Though population groups are frequently compared at the continental level, we identified a pervasive impact of sub-continental diversity on cellular and molecular properties of immune cells. These included cell populations and genes implicated in disease risk and pathogenesis as well as those relevant for diagnostics. We detected single-cell signatures of human diversity not apparent at the level of cell types, as well as modulation of the effects of age and sex by self-reported ethnicity. We discovered functional genetic variants influencing cell type-specific gene expression, including context-dependent effects, which were under-represented in analyses of non-Asian population groups, and which helped contextualise disease-associated variants. We validated our findings using multiple independent datasets and cohorts. AIDA provides fundamental insights into the relationships of human diversity with immune cell phenotypes, enables analyses of multi-ancestry disease datasets, and facilitates the development of precision medicine efforts in Asia and beyond.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05ebb7ee11200fe5a0a3e32e6a6a32456bcea51e" target='_blank'>
              Single-cell analysis of human diversity in circulating immune cells
              </a>
            </td>
          <td>
            K. Kock, Le Min Tan, Kyung Yeon Han, Yoshinari Ando, Damita Jevapatarakul, Ankita Chatterjee, Quy Xiao Xuan Lin, Eliora Violain Buyamin, Radhika Sonthalia, Deepa Rajagopalan, Yoshihiko Tomofuji, Shvetha Sankaran, Mi-So Park, Mai Abe, Juthamard Chantaraamporn, Seiko Furukawa, Supratim Ghosh, Gyo Inoue, Miki Kojima, Tsukasa Kouno, Jinyeong Lim, K. Myouzen, Sarintip Nguantad, Jin-Mi Oh, N. A. Rayan, S. Sarkar, A. Suzuki, Narita Thungsatianpun, P. Venkatesh, Jonathan Moody, Masahiro Nakano, Ziyue Chen, Chi Tian, Yuntian Zhang, Yihan Tong, C. Tan, Anteneh Mehari Tizazu, Marie Loh, You Yi Hwang, R. Ho, A. Larbi, Tze Pin Ng, Hong-Hee Won, Fred A. Wright, A. Villani, Jong-Eun Park, Murim Choi, Boxiang Liu, Arindam Maitra, Manop Pithukpakorn, B. Suktitipat, K. Ishigaki, Yukinori Okada, Kazuhiko Yamamoto, P. Carninci, John C. Chambers, C. Hon, P. Matangkasombut, Varodom Charoensawan, Partha P. Majumder, Jay W. Shin, Woong-Yang Park, Shyam Prabhakar
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>48</td>
        </tr>

        <tr id="Substantial heterogeneity in molecular features, patient prognoses, and therapeutic responses in head and neck squamous cell carcinomas (HNSCC) highlights the urgent need to develop molecular classifications that reliably and accurately reflect tumor behavior and inform personalized therapy. Here, we leveraged the similarity network fusion bioinformatics approach to jointly analyze multi-omics datasets spanning copy number variations, somatic mutations, DNA methylation, and transcriptomic profiling and derived a prognostic classification system for HNSCC. The integrative model consistently identified three subgroups (IMC1-3) with specific genomic features, biological characteristics, and clinical outcomes across multiple independent cohorts. The IMC1 subgroup included proliferative, immune-activated tumors and exhibited a more favorable prognosis. The IMC2 subtype harbored activated EGFR signaling and an inflamed tumor microenvironment with cancer-associated fibroblast/vascular infiltrations. Alternatively, the IMC3 group featured highly aberrant metabolic activities and impaired immune infiltration and recruiting. Pharmacogenomics analyses from in silico predictions and from patient-derived xenograft model data unveiled subtype-specific therapeutic vulnerabilities including sensitivity to cisplatin and immunotherapy in IMC1 and EGFR inhibitors (EGFRi) in IMC2, which was experimentally validated in patient-derived organoid models. Two signatures for prognosis and EGFRi sensitivity were developed via machine learning. Together, this integrative multi-omics clustering for HNSCC improves current understanding of tumor heterogeneity and facilitates patient stratification and therapeutic development tailored to molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c3e275f858f75f81c9f658d4d03e003b3adc1b" target='_blank'>
              Integrative multi-omics analyses identify molecular subtypes of head and neck squamous cell carcinoma with distinct therapeutic vulnerabilities.
              </a>
            </td>
          <td>
            Pengfei Diao, Yibin Dai, An Wang, Xiaoxuan Bu, Ziyu Wang, Jin Li, Yaping Wu, Hongbing Jiang, Yanling Wang, Jie Cheng
          </td>
          <td>2024-07-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980719e09512c7538d00c8fd96d0dd2473599e42" target='_blank'>
              Loss of tumor suppressors promotes inflammatory tumor microenvironment and enhances LAG3+T cell mediated immune suppression
              </a>
            </td>
          <td>
            Sara Zahraeifard, Zhiguang Xiao, Jae Young So, Abdul Ahad, Selina Montoya, Woo-Yong Park, Trinadharao Sornapudi, Tiffany Andohkow, Abigail Read, Noemi Kedei, Vishal N. Koparde, H. Yang, M. Lee, Nathan Wong, Maggie Cam, Kun Wang, E. Ruppin, Ji Luo, C. Hollander, Li Yang
          </td>
          <td>2024-07-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM), a malignant melanocyte-derived skin cancer, potentially leads to fatal outcomes without effective treatment. The variability in immunotherapy responses among melanoma patients is significantly influenced by the intricate immune microenvironment, particularly due to the status of tumor T cells, encompassing their activity, exhaustion levels, and antigen recognition capabilities. This study utilized single-cell RNA sequencing (scRNA-seq) to analyze 34 melanoma samples from two public datasets (GSE215120 and GSE115978). Herein, we extracted 706 marker genes associated with immune checkpoint (ICP) therapy from these T cells, 509 markers of T cells from 11 melanoma tissues, and eventually identified 33 candidate genes. These genes underwent LASSO and COX regression analyses to identify the signature genes. Of the initial 33 candidate genes, we successfully isolated six distinct T cell-associated immunotherapy-related genes (IRTGs). Additionally, the computation of each patient risk score proved beneficial in evaluating the immune cell infiltration level and functions as an independent prognostic factor for melanoma patient survival. The risk score results revealed promising predictive outcomes in determining the response of melanoma patients to immunotherapy. Notably, our study is the first to reveal the potential correlation between signature gene PEB4B and the immune microenvironment in melaoma, which was explored with multiple immunofluorescence (IF) and Immune Infiltration Assessment. In a conclusion, our findings demonstrate the potential utility of a risk score dependent on signature genes as a predictive tool for assessing the prognosis and response to immunotherapeutic interventions in melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4480d7b0dac6fe343faaffd214e0f795e255eda" target='_blank'>
              The combination of single-cell and RNA sequencing analysis decodes the melanoma tumor microenvironment and identifies novel T cell-associated signature genes
              </a>
            </td>
          <td>
            Sihan Luo, Daiyue Wang, Jiajie Chen, Shaocheng Hong, Yuanyuan Fang, Lu Cao, Liang Yong, Shengxiu Liu
          </td>
          <td>2024-08-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e1810e2d03e004eeb58c6ca7cae3dc32ba5c9" target='_blank'>
              Single-cell transcriptional analysis of irradiated skin reveals changes in fibroblast subpopulations and variability in caveolin expression
              </a>
            </td>
          <td>
            Lionel E. Kameni, Michelle F. Griffin, Charlotte E. Berry, Siavash Shariatzadeh, Mauricio A. Downer, Caleb Valencia, Alexander Z. Fazilat, Rahim S Nazerali, Arash Momeni, Michael Januszyk, M. Longaker, Derrick C. Wan
          </td>
          <td>2024-06-26</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaac037eca190874316bd2b34e72ed4d604d818c" target='_blank'>
              Sulfatide imaging identifies tumor cells in colorectal cancer peritoneal metastases
              </a>
            </td>
          <td>
            G. M. Sarcinelli, L. Varinelli, S. Ghislanzoni, F. Padelli, D. Lorenzini, A. Vingiani, M. Milione, M. Guaglio, S. Kusamura, M. Deraco, G. Pruneri, M. Gariboldi, D. Baratti, I. Bongarzone
          </td>
          <td>2024-06-28</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Genome-level clonal decomposition of a single specimen has been widely studied; however, it is mostly limited to cancer research. In this study, we developed a new algorithm CLEMENT, which conducts accurate decomposition and reconstruction of multiple subclones in genome sequencing of non-tumor (normal) samples. CLEMENT employs the Expectation-Maximization (EM) algorithm with optimization strategies specific to non-tumor subclones, including false variant call identification, non-disparate clone fuzzy clustering, and clonal allele fraction confinement. In the simulation and in vitro cell line mixture data, CLEMENT outperformed current cancer decomposition algorithms in estimating the number of clones (root-mean-square-error = 0.58–0.78 versus 1.43–3.34) and in the variant-clone membership agreement (∼85.5% versus 70.1–76.7%). Additional testing on human multi-clonal normal tissue sequencing confirmed the accurate identification of subclones that originated from different cell types. Clone-level analysis, including mutational burden and signatures, provided a new understanding of normal-tissue composition. We expect that CLEMENT will serve as a crucial tool in the currently emerging field of non-tumor genome analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b10b8eaeda7c6b37ca2e0e9728849f940c0c059" target='_blank'>
              CLEMENT: genomic decomposition and reconstruction of non-tumor subclones
              </a>
            </td>
          <td>
            Young-Soo Chung, Seungseok Kang, Jisu Kim, Sangbo Lee, Sangwoo Kim
          </td>
          <td>2024-06-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Triple-negative breast cancer (TNBC) poses unique challenges due to its complex nature and the need for more effective treatments. Recent studies showed encouraging outcomes from combining paclitaxel (PTX) with programmed cell death protein-1 (PD-1) blockade in treating TNBC, although the exact mechanisms behind the improved results are unclear. Methods We employed an integrated approach, analyzing spatial transcriptomics and single-cell RNA sequencing data from TNBC patients to understand why the combination of PTX and PD-1 blockade showed better response in TNBC patients. We focused on toll-like receptor 4 (TLR4), a receptor of PTX, and its role in modulating the cross-presentation signaling pathways in tumor-associated macrophages (TAMs) within the tumor microenvironment. Leveraging insights obtained from patient-derived data, we conducted in vitro experiments using immunosuppressive bone marrow-derived macrophages (iBMDMs) to validate if PTX could augment the cross-presentation and phagocytosis activities. Subsequently, we extended our study to an in vivo murine model of TNBC to ascertain the effects of PTX on the cross-presentation capabilities of TAMs and its downstream impact on CD8+ T cell-mediated immune responses. Results Data analysis from TNBC patients revealed that the activation of TLR4 and cross-presentation signaling pathways are crucial for the antitumor efficacy of PTX. In vitro studies showed that PTX treatment enhances the cross-presentation ability of iBMDMs. In vivo experiments demonstrated that PTX activates TLR4-dependent cross-presentation in TAMs, improving CD8+ T cell-mediated antitumor responses. The efficacy of PTX in promoting antitumor immunity was elicited when combined with PD-1 blockade, suggesting a complementary interaction. Conclusions This study reveals how PTX boosts the effectiveness of PD-1 inhibitors in treating TNBC. We found that PTX activates TLR4 signaling in TAMs. This activation enhances their ability to present antigens, thereby boosting CD8+ T cell antitumor responses. These findings not only shed light on PTX’s immunomodulatory role in TNBC but also underscore the potential of targeting TAMs’ antigen presentation capabilities in immunotherapy approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f44d6bfbc504dc1dddbf87d82153d8097bc8bd" target='_blank'>
              Novel insights into paclitaxel’s role on tumor-associated macrophages in enhancing PD-1 blockade in breast cancer treatment
              </a>
            </td>
          <td>
            Yoonjeong Choi, Seong A. Kim, Hanul Jung, Eunhae Kim, Yoon Kyoung Kim, Seohyun Kim, Jaehyun Kim, Yeji Lee, Min Kyoung Jo, Jiwan Woo, Yakdol Cho, Dongjoo Lee, Hongyoon Choi, Cherlhyun Jeong, Gi-Hoon Nam, Mi-Kyung Kwon, In-San Kim
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinoma (RCC) with histological sarcomatoid de-differentiation (sRCC) has a historically poor prognosis across all RCC subtypes. Recently, immunotherapy with anti-PD1 + anti-CTLA4 combo has significantly improved disease-specific survival, and complete response are seen in upwards of 20% of patients. However, half of sRCC patients still have no response to this immunotherapy, and the mechanism of response remains unclear. Methods Our preliminary bulk RNA sequencing data reveal a distinct tumor microenvironment (TME) in human sRCC tumors, characterized by heightened immune cell infiltration, particularly enriched for regulatory T cells (Tregs). Subsequently, we delineated the spatial gene expression topography within sRCC and clear cell renal cell carcinoma (ccRCC) regions, culminating in the development of the sRCC signature. Concurrently, we established an innovative immunocompetent murine model of sRCC. Leveraging CRISPR technology, we replicated the human sRCC genotype by knocking out Vhl, Bap1, and Cdkn2a/2b genes in a mouse telomerase reverse transcriptase (mTERT) overexpressing renal proximal tubule epithelial cell line. This effort yielded a successfully engineered mouse implantable sRCC cell line bearing VhlMutBap1DelCdkn2aDelCdkn2bDel (HJRCC68N). Results The syngeneic mouse sRCC model we developed faithfully replicates human sRCC histology and tumor microenvironment and recapitulates the sRCC signature identified in human sRCC regions. Importantly, like in humans, mouse sRCC tumors exhibit a heterogeneous response to anti-PD1 + anti-CTLA4 combination, and anti-CTLA4 monotherapy, reflecting the clinical variability observed in human patients. Furthermore, transcriptomic analysis from our mouse model reveals the involvement of type 1 immunity in responder, providing insights into overcoming immunotherapy resistance. Conclusions In conclusion, our study unveiled distinct changes in the TME at the transcriptomic and spatial gene expression levels in human sRCC. Leveraging our novel murine model, which faithfully replicates human sRCC histology, TME characteristics, and mixed response to immunotherapy, we elucidated the involvement of type 1 immunity in treatment response. These insights offer valuable pathways for overcoming immunotherapy resistance and pave the way for the development of tailored treatment strategies aimed at improving outcomes for sRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5fbfe11a82cb5fed55f6641955ed49526e183d8" target='_blank'>
              18 Unraveling the Impact of Tumor Microenvironment on Immunotherapy Response/Resistance in Human Sarcomatoid Clear Cell Renal Cell Carcinoma: Insights from a Novel Mouse Model
              </a>
            </td>
          <td>
            Hui Jiang
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the quest for improved therapeutics targeting immune checkpoints (ICs), we turn to spontaneously developing dog (canine) cancers, which are unique models that genetically and clinically mirror human equivalents. Despite its potential, canine cancer immunology remains largely unexplored. Here, we examine the RNA-seq-based expression of 44 ICs across 14 canine cancer types and an extensive human dataset. We unveil diverse canine IC expression patterns and unique human IC signatures that reflect the histological type and primary site of cancer. We uncover a striking similarity between canine brain cancers, osteosarcoma, and their human counterparts, identifying them as prospective immunotherapy models. Four ICs—CD160, A2AR, NKG2A, and OX40—are key to the differences observed between species. Moreover, individual patient IC signatures exhibit varying alignment with their respective cancer types, a finding with profound implications for personalized human therapy. This exploration illuminates new aspects of canine and human cancer immunology, setting the stage for discoveries at their crossroads.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f77ebd885f80c0d134fa31e9198696b90786b6" target='_blank'>
              Barking Up the Right Tree: Immune Checkpoint Signatures of Human and Dog Cancers
              </a>
            </td>
          <td>
            M. Kocikowski, M. Yébenes Mayordomo, Javier Alfaro, Maciej Parys
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2a1e8a0b040b1a9b17302a42daea756ed498f33" target='_blank'>
              Cancer cell plasticity defines response to immunotherapy in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Laura Lorenzo-Sanz, Marta Lopez-Cerda, Victoria da Silva-Diz, Marta H Artés, Sandra Llop, Rosa M. Penín, Josep Oriol Bermejo, Eva Gonzalez-Suarez, Manel Esteller, Francesc Viñals, Enrique Espinosa, Marc Oliva, J. Piulats, J. Martín-Liberal, Purificación Muñoz
          </td>
          <td>2024-06-24</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58322ca71db2c109e772f2eb0f78a110f1a84ce4" target='_blank'>
              Senescent endothelial cells promote liver metastasis of uveal melanoma in single-cell resolution
              </a>
            </td>
          <td>
            Liang Ma, Xiaoyu He, Yidian Fu, Shengfang Ge, Zhi Yang
          </td>
          <td>2024-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Tumor structure is heterogeneous and complex, and it is difficult to obtain complete characteristics by two‐dimensional analysis. The aim of this study was to visualize and characterize volumetric vascular information of clear cell renal cell carcinoma (ccRCC) tumors using whole tissue phenotyping and three‐dimensional light‐sheet microscopy. Here, we used the diagnosing immunolabeled paraffin‐embedded cleared organs pipeline for tissue clearing, immunolabeling, and three‐dimensional imaging. The spatial distributions of CD34, which targets blood vessels, and LYVE‐1, which targets lymphatic vessels, were examined by calculating three‐dimensional density, vessel length, vessel radius, and density curves, such as skewness, kurtosis, and variance of the expression. We then examined those associations with ccRCC outcomes and genetic alteration state. Formalin‐fixed paraffin‐embedded tumor samples from 46 ccRCC patients were included in the study. Receiver operating characteristic curve analyses revealed the associations between blood vessel and lymphatic vessel distributions and pathological factors such as a high nuclear grade, large tumor size, and the presence of venous invasion. Furthermore, three‐dimensional imaging parameters stratified ccRCC patients regarding survival outcomes. An analysis of genomic alterations based on volumetric vascular information parameters revealed that PI3K‐mTOR pathway mutations related to the blood vessel radius were significantly different. Collectively, we have shown that the spatial elucidation of volumetric vasculature information could be prognostic and may serve as a new biomarker for genomic alterations. High‐end tissue clearing techniques and volumetric immunohistochemistry enable three‐dimensional analysis of tumors, leading to a better understanding of the microvascular structure in the tumor space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63465b7766c5458fa29857848c92dc3c85ce6311" target='_blank'>
              Volumetric imaging of the tumor microvasculature reflects outcomes and genomic states of clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Yuta Kaneko, Tsukasa Masuda, Kimiharu Takamatsu, S. Mikami, Kohei Nakamura, Hiroshi Nishihara, R. Mizuno, Nobuyuki Tanaka, M. Oya
          </td>
          <td>2024-06-26</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Modest response rates to immunotherapy observed in advanced lung cancer patients underscore the need to identify reliable biomarkers and targets, enhancing both treatment decision-making and efficacy. Factors such as PD-L1 expression, tumor mutation burden, and a ‘hot’ tumor microenvironment with heightened effector T cell infiltration have consistently been associated with positive responses. In contrast, the predictive role of the abundantly present tumor-infiltrating myeloid cell (TIMs) fraction remains somewhat uncertain, partly explained by their towering variety in terms of ontogeny, phenotype, location, and function. Nevertheless, numerous preclinical and clinical studies established a clear link between lung cancer progression and alterations in intra- and extramedullary hematopoiesis, leading to emergency myelopoiesis at the expense of megakaryocyte/erythroid and lymphoid differentiation. These observations affirm that a continuous crosstalk between solid cancers such as lung cancer and the bone marrow niche (BMN) must take place. However, the BMN, encompassing hematopoietic stem and progenitor cells, differentiated immune and stromal cells, remains inadequately explored in solid cancer patients. Subsequently, no clear consensus has been reached on the exact breadth of tumor installed hematopoiesis perturbing cues nor their predictive power for immunotherapy. As the current era of single-cell omics is reshaping our understanding of the hematopoietic process and the subcluster landscape of lung TIMs, we aim to present an updated overview of the hierarchical differentiation process of TIMs within the BMN of solid cancer bearing subjects. Our comprehensive overview underscores that lung cancer should be regarded as a systemic disease in which the cues governing the lung tumor-BMN crosstalk might bolster the definition of new biomarkers and druggable targets, potentially mitigating the high attrition rate of leading immunotherapies for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09f4d97d42cb2976ecd4a13f91ebddf956195779" target='_blank'>
              The crosstalk between lung cancer and the bone marrow niche fuels emergency myelopoiesis
              </a>
            </td>
          <td>
            Evelyn Calderon-Espinosa, K. De Ridder, Thomas Benoot, Yanina Jansen, Domien Vanhonacker, Robbe Heestermans, A. De Becker, Ivan Van Riet, Lore Decoster, C. Goyvaerts
          </td>
          <td>2024-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain. Methods: Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package “Seurat” was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes. Results: Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA. Conclusion: A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Immunotherapy, particularly immune checkpoint blockade (ICB) therapy, is a promising treatment modality in oncology that involves augmenting the tumor-attacking capabilities of the immune system. Several preclinical and clinical studies have underscored the transformative potential of ICBs in the context of malignancies. 1 However, the efficacy of ICB therapy is often impeded by intrinsic and acquired resistance mechanisms in solid tumors, which are commonly due to a paucity of tumor neoantigens, checkpoint targets, and the inflammatory presence of cytotoxic T lymphocytes. 2 The tumor microenvironment (TME) is profoundly immunosuppressive, which is the key reason for the limited clinical efficacy of ICB therapy. Within this milieu, tumor-associated macrophages (TAMs) are the predominant myeloid cell subset, and their high infiltration levels have been implicated in mediating resistance to ICB therapy, thereby correlating with adverse prognosis across a spectrum of cancers. 3 Recent advances reported in Nature Cancer have shed light on specific TAM subpopulations that are instrumental in facilitating tumor immune evasion and resistance to ICB therapy. Two recent studies have delineated the roles of Sirp α + TAMs in colorectal cancer (CRC) 4 and Siglec-9 + TAMs in glioblastoma multiforme (GBM">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253268c89e8e487a8b93e72b5360f684908a756a" target='_blank'>
              Targeting macrophages to reprogram the tumor immune microenvironment
              </a>
            </td>
          <td>
            Zhidong Xie, Jing Liao, Jun Chen
          </td>
          <td>2024-08-15</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CD8+ T cells play pivotal roles in combating intracellular pathogens and eliminating malignant cells in cancer. However, the prognostic role of CD8+ T cells in ovarian carcinoma is insufficiently exploited. Herein, through univariate Cox regression along with least absolute shrinkage and selection operator (LASSO) regression analyses, we developed a novel prognostic model based on CD8+ T cell markers identified by single-cell sequencing (scRNA-seq) analyses. Patient grouping by the median risk score reveals an excellent prognostic efficacy of this model in both training and validation cohorts. Of note, patients classified as low-risk group exhibit a dramatically improved prognosis. In addition, higher enrichment level of immune-related pathways and increased infiltration level of multiple immune cells are found in patients with lower risk score. Importantly, low-risk patients also exhibited higher response rate to immunotherapies. Summarily, this developed CD8+ T cell-associated prognostic model serves as an excellent predictor for clinical outcomes and aids in guiding therapeutic strategy choices for ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/409d1a4ad42cb1c888c3ed09d0a52f53075d8d47" target='_blank'>
              Comprehensive single-cell and bulk RNA-seq analyses reveal a novel CD8+ T cell-associated prognostic signature in ovarian cancer
              </a>
            </td>
          <td>
            Yi-Ling Han, Zhou Fang, Zhijie Gao, Wen-Ge Li, Jing Yang
          </td>
          <td>2024-06-25</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Tissue-resident memory T cells (TRM) are a specialized subset of long-lived memory T cells that reside in peripheral tissues. However, the impact of TRM-related immunosurveillance on the tumor-immune microenvironment (TIME) and tumor progression across various non-small-cell lung cancer (NSCLC) patient populations is yet to be elucidated. Our comprehensive analysis of multiple independent single-cell and bulk RNA-seq datasets of patient NSCLC samples generated reliable, unique TRM signatures, through which we inferred the abundance of TRM in NSCLC. We discovered that TRM abundance is consistently positively correlated with CD4+ T helper 1 cells, M1 macrophages, and resting dendritic cells in the TIME. In addition, TRM signatures are strongly associated with immune checkpoint and stimulatory genes and the prognosis of NSCLC patients. A TRM-based machine learning model to predict patient survival was validated and an 18-gene risk score was further developed to effectively stratify patients into low-risk and high-risk categories, wherein patients with high-risk scores had significantly lower overall survival than patients with low-risk. The prognostic value of the risk score was independently validated by the Cancer Genome Atlas Program (TCGA) dataset and multiple independent NSCLC patient datasets. Notably, low-risk NSCLC patients with higher TRM infiltration exhibited enhanced T-cell immunity, nature killer cell activation, and other TIME immune responses related pathways, indicating a more active immune profile benefitting from immunotherapy. However, the TRM signature revealed low TRM abundance and a lack of prognostic association among lung squamous cell carcinoma patients in contrast to adenocarcinoma, indicating that the two NSCLC subtypes are driven by distinct TIMEs. Altogether, this study provides valuable insights into the complex interactions between TRM and TIME and their impact on NSCLC patient prognosis. The development of a simplified 18-gene risk score provides a practical prognostic marker for risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1e637bee54083c5c2fb1deaba9e34a97001048" target='_blank'>
              A comprehensive meta-analysis of tissue resident memory T cells and their roles in shaping immune microenvironment and patient prognosis in non-small cell lung cancer
              </a>
            </td>
          <td>
            Aidan Shen, Aliesha Garrett, Cheng-Chi Chao, Dongliang Liu, Chao Cheng, Zhaohui Wang, Chen Qian, Yangzhi Zhu, Junhua Mai, Chongming Jiang
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this study, we explore the possibility of inferring characteristics of the tumor-immune microenvironment (TIME) from the blood. Specifically, we investigate two datasets of head and neck squamous cell carcinoma (HNSCC) patients with matched scRNA-Seq from peripheral blood mononuclear cells (PBMCs) and tumor tissues. Our analysis shows that the immune cell fractions and gene expression profiles of various immune cells within the tumor microenvironment can be inferred from the matched PBMC scRNA-Seq data. We find that the established exhausted T-cell signature can be predicted from the blood and serve as a valuable prognostic blood biomarker of immunotherapy response. Additionally, our study reveals that the inferred ratio between tumor memory B and regulatory T cell fractions is predictive of immunotherapy response and is superior to the well-established cytolytic and exhausted T-cell signatures. These results highlight the promising potential of PBMC scRNA-Seq in cancer immunotherapy and warrant, and will hopefully facilitate, further investigations on a larger scale. The code for predicting tumor immune microenvironment from PBMC scRNA-Seq, TIMEP, is provided, offering other researchers the opportunity to investigate its prospective applications in various other indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ade53f587544d91914fb929dd7efe4a6e7cd50" target='_blank'>
              Inferring characteristics of the tumor-immune microenvironment of HNSCC patients from single-cell transcriptomics of peripheral blood.
              </a>
            </td>
          <td>
            Yingying Cao, Tiangen Chang, Fiorella Schischlik, Kun Wang, Sanju Sinha, Sridhar Hannenhalli, Peng Jiang, E. Ruppin
          </td>
          <td>2024-08-08</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in the epidermal growth factor receptor (EGFR) are common in non-small cell lung cancer (NSCLC), particularly in never-smoker patients. However, these mutations are not always carcinogenic, and have recently been reported in histologically normal lung tissue from patients with and without lung cancer. To investigate the outcome of EGFR mutation in healthy lung stem cells, we grow murine alveolar type II organoids monoclonally in a three-dimensional Matrigel. Our experiments show that the EGFR-L858R mutation induces a change in organoid structure: mutated organoids display more ‘budding’, in comparison with non-mutant controls, which are nearly spherical. We perform on-lattice computational simulations, which suggest that this can be explained by the concentration of division among a small number of cells on the surface of the mutated organoids. We are currently unable to distinguish the cell-based mechanisms that lead to this spatial heterogeneity in growth, but suggest a number of future experiments which could be used to do so. We suggest that the likelihood of L858R-fuelled tumorigenesis is affected by whether the mutation arises in a spatial environment that allows the development of these surface protrusions. These data may have implications for cancer prevention strategies and for understanding NSCLC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e046dae2eb0578408d47ec2942e59fd61d21fe2" target='_blank'>
              An agent-based modelling framework to study growth mechanisms in EGFR-L858R mutant cell alveolar type II cells
              </a>
            </td>
          <td>
            Helena Coggan, C. Weeden, Philip Pearce, M. Dalwadi, Alastair Magness, C. Swanton, Karen M. Page
          </td>
          <td>2024-07-01</td>
          <td>Royal Society Open Science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf3fb50dabab3c628e11b6f429b04114a9702d4" target='_blank'>
              Brain Chimeroids reveal individual susceptibility to neurotoxic triggers.
              </a>
            </td>
          <td>
            Noelia Antón-Bolaños, Irene Faravelli, Tyler Faits, Sophia Andreadis, Rahel Kastli, Sebastiano Trattaro, Xian Adiconis, Anqi Wei, Abhishek Sampath Kumar, Daniela J. Di Bella, M. Tegtmeyer, R. Nehme, J. Levin, Aviv Regev, Paola Arlotta
          </td>
          <td>2024-06-26</td>
          <td>Nature</td>
          <td>5</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ae438e85d1f5a8bcba96886546fdec069966a7" target='_blank'>
              IRF8 deficiency-induced myeloid-derived suppressor cell promote immune evasion in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zhen-Yang Gao, Shang Liu, Han Xiao, Mengdi Li, Wan-gang Ren, Lin Xu, Zhong-Min Peng
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The global incidence of cancer is increasing, with estimates suggesting that there will be 26 million new cases and 17 million deaths per year by 2030. Cancer stem cells (CSCs) and extracellular vesicles (EVs) are key to the resistance and advancement of cancer. They play a crucial role in tumor dynamics and resistance to therapy. CSCs, initially discovered in acute myeloid leukemia, are well-known for their involvement in tumor initiation, progression, and relapse, mostly because of their distinct characteristics, such as resistance to drugs and the ability to self-renew. EVs, which include exosomes, microvesicles, and apoptotic bodies, play a vital role in facilitating communication between cells within the tumor microenvironment (TME). They have a significant impact on cellular behaviors and contribute to genetic and epigenetic changes. This paper analyzes the mutually beneficial association between CSCs and EVs, emphasizing their role in promoting tumor spread and developing resistance mechanisms. This review aims to investigate the interaction between these entities in order to discover new approaches for attacking the complex machinery of cancer cells. It highlights the significance of CSCs and EVs as crucial targets in the advancement of novel cancer treatments, which helps stimulate additional research, promote progress in ideas for cancer treatment, and provide renewed optimism in the effort to reduce the burden of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b17e9728aa76af6a02ab4a44623aab4c0608b6" target='_blank'>
              Extracellular vesicles and cancer stem cells: a deadly duo in tumor progression
              </a>
            </td>
          <td>
            Akram Tayanloo-Beik, Azin Eslami, Masoumeh Sarvari, H. Jalaeikhoo, Mohsen Rajaeinejad, Mohsen Nikandish, Ali Faridfar, Mostafa Rezaei-Tavirani, A. R. Mafi, Bagher Larijani, B. Arjmand
          </td>
          <td>2024-07-18</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ability to genetically edit primary B cells via CRISPR/Cas9 technology represents a powerful tool to study molecular mechanisms of B ‐ cell pathogenesis. In this context, employing ribonucleoprotein complexes (RNPs), formed by recombinant Cas9 and genome ‐ targetting single guide RNA molecules, brings in advantage of accelerated set ‐ up and protocol robustness. Gene editing via RNP electroporation has been recently applied to primary tumor cells isolated from patients chronic lymphocytic leukemia (CLL), suggesting an efficient and valuable tool for studying leukemic cell biology and biomarker validation. 1,2 The work by Nardi et al. on this topic proposed to electroporate unmanipulated primary CLL cells that are subsequently put in culture with human CD40L ‐ expressing fibroblasts and soluble stimuli to promote CLL cell proliferation. In this context, cellular proliferation is required to achieve homozygous gene editing, whereas in unstimulated CLL cells it is possible to achieve only the heterozygous editing. 1 The method published by Mateos ‐ Jaimez et al. relies on the preactivation of CLL cells with CD40L/BAFF/IL ‐ 21 ‐ expressing stromal cells, followed by RNP electroporation and continuation of the stimulatory coculture. 2 Both methods approach 80% – 90% of editing efficiency and allow to perform downstream in vitro experiments on edited leukemic cells. Application of a similar RNP ‐ based editing approach to the widely used murine">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce4b4f9dae608c0b0ba780ba2b6869894794e80" target='_blank'>
              Robust and cost‐effective CRISPR/Cas9 gene editing of primary tumor B cells in Eµ‐TCL1 model of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Rosita Del Prete, Roberta Drago, Federica Nardi, Gaia Bartolini, Erika Bellini, Antonella De Rosa, Silvia Valensin, Anna Kabanova
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 What are the molecular and cellular features that contribute to the transformation of myometrial stem cells into tumor-initiating cells within uterine leiomyoma?



 Transformative changes in differentiation and genetic processes, in a distinct subset of M/LM cells drives their progression into a tumorigenic state.



 Uterine leiomyomas (LM), which are benign smooth muscle-derived tumours of myometrium, impact approximately 75% of women, causing significant physical and psychological challenges and imposing substantial healthcare costs. The widespread consensus postulates a monoclonal origin for LM, implying their derivation from a dysregulated single multipotent stem cell that could give rise to tumor-initiating cells (TICs). While previous studies have shown cellular heterogeneity in both myometrium (M) and LM, the precise identity of the originating cells is unknown.



 Prospective, observational, and biomedical study conducted at Hospital La Fe (Valencia, Spain) for one year. Eight sample pairs of LM and M underwent single-nuclei RNAseq (snRNA-seq; n = 16) and single-cell proteomic (scP; n = 16) analyses, to generate a detailed transcriptomic and proteomic map decoupled from cell type, state, and spatial location. We further employed single-cell RNA velocity inference for differentiation trajectory [BR1] analysis of mesenchymal cells, followed by the identification of potential driver genes in LM tumorigenesis.



 Upon obtaining informed consent, we procured LM and M samples from eight patients aged 35-50 undergoing hysterectomies. Part of these samples were preserved in paraffin for spatial transcriptomics utilizing VISIUM (10x Genomics). The remaining tissues were dissociated into single-cell suspensions and underwent scRNA-seq and scP using Chromium Controller and Orbitrap Eclipse Tribid mass spectrometry, respectively. The analysis of the collected data was carried out using publicly accessible R/Python tools.



 Analysis of snRNAseq (∼52,000 cells) revealed similar cellular compositions in M and LM, including endothelial, perivascular, smooth muscle cells (SMC), fibroblasts, and immune cells. Further trajectory analysis unveiled a subset of myometrial cells with potential as TICs, exhibiting reduced expression of hormone receptors like PGR. Analysis of these cells [BR1] suggested differentiation into two distinct cell types -fibroblasts and SMCs- with elevated hormone expression over pseudotime. Differential analysis between M and LM also showed a dysregulated transcriptomic profile in the TICs within the tumor, as evidenced by upregulation of RAD51B and HMGA2 and downregulation PGR among other genes. These genes are indicative markers, underscoring their significant roles in the tumorigenesis of LM by influencing hormone response and disrupting DNA repair mechanisms. Spatial transcriptomic analysis confirmed these findings and suggested that the distribution of TICs could be ubiquitous, since the expression of potential myometrial-stem markers, was markedly increased only on certain spots within the tissue across both LM and M. Lastly, scProt profiling (∼5,000 cells) also allowed the identification of putative TICs, which showed upregulation in LM of specific proteins involved in replication, transcription, and translation.



 Our study sets the basis for TICs isolation in LM, yet functional validation studies need to be performed to address the clinical potential of our findings. Further studies including more patients, and addressing racial disparities will help to generalize these findings to a broader population.



 Our findings indicate the presence of a particular subset of M/LM cells whose changes in differentiation path and genetic processes seem to drive the transformation of these cells into a tumorigenic state to develop LM. Targeting these markers might show promise for effectively treating these tumors.



 NCT04214457
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e5f1c7a81c7b6baceb7a49d4c94f2c7bde8e8c" target='_blank'>
              O-264 Profiling Tumor-Initiating Cells in Uterine Leiomyoma through Single-Cell Multi-omics
              </a>
            </td>
          <td>
            A. Machado-Lopez, P. Punzón-Jiménez, J. Llera-Oyola, R. Perez-Moraga, M. Gálvez-Viedma, D. Grases, E. Satorres, E. Porta-Pardo, E. Schooff, B. Roson, C. Simon, A. Mas
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Summary Neoplastic tumors originate from a single cell, and their evolution can be traced through lineages characterized by mutations, copy number alterations, and structural variants. These lineages are reconstructed and mapped onto evolutionary trees with algorithmic approaches. However, without ground truth benchmark sets, the validity of an algorithm remains uncertain, limiting potential clinical applicability. With a growing number of algorithms available, there is urgent need for standardized benchmark sets to evaluate their merits. Benchmark sets rely on in silico simulations of tumor sequence, but there are no accepted standards for simulation tools, presenting a major obstacle to progress in this field. Availability and implementation All analysis done in the paper was based on publicly available data from the publication of each accessed tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92deba8c2972c7062a1ed2f5dce7cd80ceb128bf" target='_blank'>
              Assessing the merits: an opinion on the effectiveness of simulation techniques in tumor subclonal reconstruction
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Yunzhou Liu, Robert B Scharpf, Rachel Karchin
          </td>
          <td>2024-06-26</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/228f6ee044df6edc918c18217ff468ea90e78c6b" target='_blank'>
              EMBER creates a unified space for independent breast cancer transcriptomic datasets enabling precision oncology
              </a>
            </td>
          <td>
            Carlos Ronchi, Syed Haider, C. Brisken
          </td>
          <td>2024-07-09</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="BACKGROUND
Basal cell carcinoma (BCC) is the most frequently diagnosed skin cancer and the most common malignancy in humans. Different morphological subtypes of BCC are associated with low- or high-risk of recurrence and aggressiveness, but the underlying biology of how the individual subtypes arise remains largely unknown. Because the majority of BCCs appear to arise from mutations in the same pathway, we hypothesized that BCC development, growth and invasive potential is also influenced by the tumor microenvironment and in particular by cancer-associated fibroblasts (CAFs) and their secreted factors.


OBJECTIVE
We aimed to characterize the stroma of the different BCC subtypes with a focus on CAF populations.


METHODS
To investigate the stromal features of the different BCC subtypes, we applied laser-capture microdissection (LCM) followed by RNA sequencing. A cohort of 15 BCC samples from 5 different "pure" subtypes (superficial, nodular, micronodular, sclerosing and basosquamous; n=3 each) were selected and included in the analysis. Healthy skin was used as a control (n=6). We confirmed the results by immunohistochemistry. We validated our findings in two independent, public single-cell RNA sequencing (scRNAseq) datasets and by RNAscope.


RESULTS
The stroma of the different BCC subtypes have distinct gene expression signatures. Nodular and micronodular seem to have the most similar signatures, while superficial and sclerosing the most different. By comparing low- and high-risk BCC subtypes, we observed that Collagen 10A1 (COL10A1) is overexpressed in the stroma of sclerosing/infiltrative and basosquamous but not micronodular high-risk subtypes. Those findings were confirmed by immunohistochemistry in a cohort of 89 different BCC and 13 healthy skin samples. Moreover, scRNAseq analysis of BCCs of two independent datasets showed that the COL10A1-expressing population of cells is associated with the stroma adjacent to invasive BCC and shows extracellular matrix remodeling features.


CONCLUSION
We identified COL10A1 as a marker of high-risk BCC, in particular of the sclerosing/infiltrative and basosquamous subtypes. We demonstrated at the single cell level that COL10A1 is expressed by a specific CAF population associated with the stroma of invasive BCC. This opens up new tailored treatment options as well as a new prognostic biomarker for BCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5131e4d68fffbdfca6cced052102c01223f2d2d9" target='_blank'>
              COL10A1 expression distinguishes a subset of cancer-associated fibroblasts present in the stroma of high-risk basal cell carcinoma.
              </a>
            </td>
          <td>
            Mauro Esposito, L. Yerly, Prachi Shukla, Victoria Hermes, Federica Sella, Zsolt Balázs, Evelyn Lattmann, A. Tastanova, P. Turko, Ramon Lang, I. Kolm, Ramon Staeger, François Kuonen, Michael Krauthammer, J. Hafner, Mitchell P Levesque, G. Restivo
          </td>
          <td>2024-06-25</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="A distinct feature of pancreatic ductal adenocarcinoma (PDAC) is a prominent tumor microenvironment (TME) with remarkable cellular and spatial heterogeneity that meaningfully impacts disease biology and treatment resistance. The dynamic crosstalk between cancer cells and the dense stromal compartment leads to spatially and temporally heterogeneous metabolic alterations, such as acidic pH that contributes to drug resistance in PDAC. Thus, monitoring the extracellular pH metabolic fluctuations within the TME is crucial to predict and to quantify anticancer drug efficacy. Here, a simple and reliable alginate-based 3D PDAC model embedding ratiometric optical pH sensors and cocultures of tumor (AsPC-1) and stromal cells for simultaneously monitoring metabolic pH variations and quantify drug response is presented. By means of time-lapse confocal laser scanning microscopy (CLSM) coupled with a fully automated computational analysis, the extracellular pH metabolic variations are monitored and quantified over time during drug testing with gemcitabine, folfirinox, and paclitaxel, commonly used in PDAC therapy. In particular, the extracellular acidification is more pronounced after drugs treatment, resulting in increased antitumor effect correlated with apoptotic cell death. These findings highlight the importance of studying the influence of cellular metabolic mechanisms on tumor response to therapy in 3D tumor models, this being crucial for the development of personalized medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dceab42d7b7c7a1f2eb735efea89f11000f608" target='_blank'>
              A 3D Pancreatic Cancer Model with Integrated Optical Sensors for Noninvasive Metabolism Monitoring and Drug Screening.
              </a>
            </td>
          <td>
            Anna Chiara Siciliano, S. Forciniti, V. Onesto, Helena Iuele, Donatella Delle Cave, Federica Carnevali, Giuseppe Gigli, E. Lonardo, L. L. del Mercato
          </td>
          <td>2024-07-08</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac24e0e7a39e90599e20a198363cbf479bb3749d" target='_blank'>
              Using a pan-cancer atlas to investigate tumour associated macrophages as regulators of immunotherapy response
              </a>
            </td>
          <td>
            Alexander Coulton, Jun Murai, Danwen Qian, Krupa Thakkar, Claire E Lewis, Kevin Litchfield
          </td>
          <td>2024-07-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35eea075db8a8c1977ce6381cea9a663e6162ef" target='_blank'>
              PTGR1-mediated immune evasion mechanisms in late-stage triple-negative breast cancer: mechanisms of M2 macrophage infiltration and CD8+ T cell suppression.
              </a>
            </td>
          <td>
            Fang Huang, Fuhe Wang, Qilu Hu, Ying Li, Da Jiang
          </td>
          <td>2024-07-28</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small RNAs that are often dysregulated in many diseases, including cancers. They are highly tissue-specific and stable, thus, making them particularly useful as biomarkers. As the spatial transcriptomics field advances, protocols that enable highly sensitive and spatially resolved detection become necessary to maximize the information gained from samples. This is especially true of miRNAs where the location their expression within tissue can provide prognostic value with regard to patient outcome. Equally as important as detection are ways to assess and visualize the miRNA's spatial information in order to leverage the power of spatial transcriptomics over that of traditional nonspatial bulk assays. We present a highly sensitive methodology that simultaneously quantitates and spatially detects seven miRNAs in situ on formalin-fixed paraffin-embedded tissue sections. This method utilizes rolling circle amplification (RCA) in conjunction with a dual scanning approach in nanoliter well arrays with embedded hydrogel posts. The hydrogel posts are functionalized with DNA probes that enable the detection of miRNAs across a large dynamic range (4 orders of magnitude) and a limit of detection of 0.17 zeptomoles (1.7 × 10-4 attomoles). We applied our methodology coupled with a data analysis pipeline to K14-Cre Brca1f/fTp53f/f murine breast tumors to showcase the information gained from this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbaad333d7c14fb2504c61497f5a898eaa1ab830" target='_blank'>
              Sensitive Multiplexed MicroRNA Spatial Profiling and Data Classification Framework Applied to Murine Breast Tumors.
              </a>
            </td>
          <td>
            Omar N Mohd, Y. Heng, Lin Wang, Abhishek Thavamani, Erica S. Massicott, Gerburg M Wulf, Frank J Slack, Patrick S. Doyle
          </td>
          <td>2024-07-23</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma is a childhood developmental cancer; however, its embryonic origins remain poorly understood. Moreover, in-depth studies of early tumor-driving events are limited because of the lack of appropriate models. Herein, we analyzed RNA sequencing data obtained from human neuroblastoma samples and found that loss of expression of trunk neural crest–enriched gene MOXD1 associates with advanced disease and worse outcome. Further, by using single-cell RNA sequencing data of human neuroblastoma cells and fetal adrenal glands and creating in vivo models of zebrafish, chick, and mouse, we show that MOXD1 is a determinate of tumor development. In addition, we found that MOXD1 expression is highly conserved and restricted to mesenchymal neuroblastoma cells and Schwann cell precursors during healthy development. Our findings identify MOXD1 as a lineage-restricted tumor-suppressor gene in neuroblastoma, potentiating further stratification of these tumors and development of novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4a1e60d743c1788421ba02de583dc1b69f419e" target='_blank'>
              MOXD1 is a lineage-specific gene and a tumor suppressor in neuroblastoma
              </a>
            </td>
          <td>
            Elina Fredlund, Stina Andersson, Elien Hilgert, Ezequiel Monferrer, Guadalupe Álvarez-Hernán, Sinan Karakaya, Siebe Loontiens, J. Bek, Tomas Gregor, Estelle Lecomte, Emma Magnusson, Enrika Miltenyte, Marie Cabirol, Michail Kyknas, Niklas Engström, Marie Arsenian Henriksson, Emma Hammarlund, J. Rosenblum, Rosa Noguera, Frank Speleman, J. van Nes, S. Mohlin
          </td>
          <td>2024-06-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9c5b5959b41a250a5ef58adb37088e34a07297" target='_blank'>
              Immuno-oncologic profiling of pediatric brain tumors reveals major clinical significance of the tumor immune microenvironment
              </a>
            </td>
          <td>
            Adrian B Levine, L. Nobre, Anirban Das, Scott Milos, V. Bianchi, Monique Johnson, Nick Fernandez, L. Stengs, S. Ryall, Michelle Ku, Mansuba Rana, Benjamin Laxer, J. Sheth, Stefanie-Grace Sbergio, Ivana Fedorakova, Vijay Ramaswamy, J. Bennett, R. Siddaway, U. Tabori, Cynthia Hawkins
          </td>
          <td>2024-07-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Most melanomas progress from radial to vertical growth phase before spreading locoregionally and distally. Much is still unknown about the metabolic changes in the tumor cells and their microenvironment during this metastatic progression. We aimed to gain new insight into the molecular characteristics of melanoma in regard to spatial lipidomics to deliver new knowledge regarding tumor metastatic progression. We included 10 fresh tumor samples from 10 patients including two in situ melanomas, two invasive primary melanomas, and six metastatic melanomas (four in-transit metastases and two distant metastases). In addition, we analyzed four healthy skin controls from the same patients. Time-of-flight imaging secondary ion mass spectrometry (ToF-SIMS) enabled detailed spatial-lipidomics that could be directly correlated with conventional histopathological analysis of consecutive H&E-stained tissue sections. Significant differences in the lipid profiles were found in primary compared to metastatic melanomas, notably an increase in phosphatidylethanolamine lipids relative to phosphatidylinositol lipids and an increase in GM3 gangliosides in the metastatic samples. Furthermore, analysis of the data from in transit versus distant metastases samples highlighted that specific phospholipids, and a difference in the long versus shorter chain GM3 gangliosides, discriminated the metastatic routes. Further studies are warranted to verify these preliminary findings. Lipidomic changes could serve as a novel biomarker for tumor progression and even serve as a target for novel treatments. Furthermore, analyzing the lipid profiles could help to differentiate between primary and metastatic melanomas in challenging cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd5da01fb93ce9422a464070818f23116949e9f" target='_blank'>
              ToF-SIMS imaging reveals changes in tumor cell lipids during metastatic progression of melanoma.
              </a>
            </td>
          <td>
            N. Neittaanmäki, O. Zaar, Kevin Sjögren Cehajic, Kelly Dimovska Nilsson, D. Katsarelias, R. O. Bagge, John Paoli, John S Fletcher
          </td>
          <td>2024-06-29</td>
          <td>Pigment cell & melanoma research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf603a0e9cde601ac9006c72d111f317f71de81" target='_blank'>
              Single-cell profiling reveals altered immune landscape and impaired NK cell function in gastric cancer liver metastasis.
              </a>
            </td>
          <td>
            Xiaolong Tang, Lei Gao, Xingzhi Jiang, Zhenyu Hou, Yiwen Wang, Shiyang Hou, Hui Qu
          </td>
          <td>2024-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="PURPOSE
Co-occurring mutations in KEAP1 and STK11KRAS have emerged as determinants of survival outcomes in non-small cell lung cancer (NSCLC) patients treated with immunotherapy. However, these mutational contexts identify a fraction of non-responders to immune checkpoint inhibitors. We hypothesized that KEAP1 wild-type tumors recapitulate the transcriptional footprint of KEAP1 mutations, and that this KEAPness phenotype can determine immune responsiveness with higher precision compared to mutation-based models.


EXPERIMENTAL DESIGN
The TCGA was used to infer the KEAPness phenotype and explore its immunological correlates at the pan-cancer level. The association between KEAPness and survival outcomes was tested in two independent cohorts of advanced NSCLC patients treated with immunotherapy and profiled by RNA-Seq (SU2C n=153; OAK/POPLAR n=439). The NSCLC TRACERx421 multi-region sequencing study (tumor regions n=947) was used to investigate evolutionary trajectories.


RESULTS
KEAPness-dominant tumors represented 50% of all NSCLCs and were associated with shorter progression-free survival (PFS) and overall survival (OS) compared to KEAPness-free cases in independent cohorts of NSCLC patients treated with immunotherapy (SU2C PFS P=0.042, OS P=0.008; OAK/POPLAR PFS P=0.0014, OS P<0.001). Patients with KEAPness tumors had survival outcomes comparable to those with KEAP1-mutant tumors. In the TRACERx421, KEAPness exhibited limited transcriptional intratumoral heterogeneity and immune exclusion, resembling the KEAP1-mutant disease. This phenotypic state occurred across genetically divergent tumors, exhibiting shared and private cancer genes under positive selection when compared to KEAP1-mutant tumors.


CONCLUSIONS
We identified a KEAPness phenotype across evolutionary divergent tumors. KEAPness outperforms mutation-based classifiers as a biomarker of inferior survival outcomes in NSCLC patients treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea8af08e6c600cf8a24f58a7981fb66121f005e" target='_blank'>
              Transcriptional phenocopies of deleterious KEAP1 mutations correlate with survival outcomes in lung cancer treated with immunotherapy.
              </a>
            </td>
          <td>
            S. Scalera, B. Ricciuti, D. Marinelli, M. Mazzotta, Laura Cipriani, Giulia Bon, Giulia Schiavoni, I. Terrenato, Alessandro Di Federico, J. Alessi, Maurizio Fanciulli, L. Ciuffreda, F. de Nicola, F. Goeman, G. Caravagna, Daniele Santini, R. De Maria, F. Cappuzzo, G. Ciliberto, M. Jamal-Hanjani, Mark M Awad, N. Mcgranahan, M. Maugeri-Saccà
          </td>
          <td>2024-07-09</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, Michael Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Gastric cancer (GC) is characterized by significant intratumoral heterogeneity, and stem cells are promising therapeutic targets. Despite advancements in spatial transcriptome analyses, unexplored targets for addressing cancer stemness remain unknown. This study aimed to identify Nuclear Factor IX (NFIX) as a critical regulator of cancer stemness in GC and evaluate its clinicopathological significance and function. Spatial transcriptome analysis of GC was conducted. The correlation between NFIX expression, clinicopathological factors, and prognosis was assessed using immunostaining in 127 GC cases. Functional analyses of cancer cell lines validated these findings. Spatial transcriptome analysis stratified GC tissues based on genetic profiles, identified CSC-like cells, and further refined the classification to identify and highlight the significance of NFIX, as validated by Monocle 3 and CytoTRACE analyses. Knockdown experiments in cancer cell lines have demonstrated the involvement of NFIX in cancer cell proliferation and kinase activity. This study underscores the role of spatial transcriptome analysis in refining GC tissue classification and identifying therapeutic targets, highlighting NFIX as a pivotal factor. NFIX expression is correlated with poor prognosis and drives GC progression, suggesting its potential as a novel therapeutic target for personalized GC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f19e2ccd74245f8115ff0d1d1d57356414aa99d" target='_blank'>
              Discovering cancer stem-like molecule, nuclear factor I X, using spatial transcriptome in gastric cancer.
              </a>
            </td>
          <td>
            Akira Ishikawa, Takafumi Fukui, Aya Kido, Narutaka Katsuya, K. Kuraoka, N. Uraoka, Takahisa Suzuki, Shiro Oka, Takahiro Kotachi, Hassan Ashktorab, D. Smoot, Wataru Yasui
          </td>
          <td>2024-07-17</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly invasive, aggressive brain cancer that carries a median survival of 15 months and is resistant to standard therapeutics. Recent studies have demonstrated that intratumoral heterogeneity plays a critical role in promoting resistance by mediating tumor adaptation through microenvironmental cues. GBM can be separated into two distinct regions—a core and a rim, which are thought to drive specific aspects of tumor evolution. These differences in tumor progression are regulated by the diverse biomolecular and biophysical signals in these regions, but the acellular biophysical characteristics remain poorly described. This study investigates the mechanical and ultrastructural characteristics of the tumor extracellular matrix (ECM) in patient-matched GBM core and rim tissues. Seven patient-matched tumor core and rim samples and one non-neoplastic control were analyzed using atomic force microscopy, scanning electron microscopy, and immunofluorescence imaging to quantify mechanical, ultrastructural, and ECM composition changes. The results reveal significant differences in biophysical parameters between GBM core, rim, and non-neoplastic tissues. The GBM core is stiffer, denser, and is rich in ECM proteins hyaluronic acid and tenascin-C when compared to tumor rim and non-neoplastic tissues. These alterations are intimately related and have prognostic effect with stiff, dense tissue correlating with longer progression-free survival. These findings reveal new insights into the spatial heterogeneity of biophysical parameters in the GBM tumor microenvironment and identify a set of characteristics that may correlate with patient prognosis. In the long term, these characteristics may aid in the development of strategies to combat therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5313dd86c28916e3c6cd3d13c84d5b7be07567" target='_blank'>
              The prognostic effect of mechanical, ultrastructural, and ECM signatures in glioblastoma core and rim
              </a>
            </td>
          <td>
            Bradley J. Mahaffey, Zachary P. Fowler, Zoe Lung, Vivien Dang, Hyunchul Lee, Allison McKenzie Johnson, Marco A. Munoz, Dylan A. Goodin, Hermann Frieboes, Brian J. Williams, Joseph Chen
          </td>
          <td>2024-06-24</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a rare disease associated with a poor prognosis, primarily due to early recurrence and metastasis. An important feature of this condition is microvascular invasion (MVI). However, current predictive models based on imaging have limited efficacy in this regard. This study employed a random forest model to construct a predictive model for MVI identification and uncover its biological basis. Single-cell transcriptome sequencing, whole exome sequencing, and proteome sequencing were performed. The area under the curve of the prediction model in the validation set was 0.93. Further analysis indicated that MVI-associated tumor cells exhibited functional changes related to epithelial-mesenchymal transition and lipid metabolism due to alterations in the NF-kappa B and MAPK signaling pathways. Tumor cells were also differentially enriched for the IL-17 signaling pathway. There was less infiltration of SLC30A1+ CD8+ T cells expressing cytotoxic genes in MVI-associated ICC, whereas there was more infiltration of myeloid cells with attenuated expression of the MHC II pathway. Additionally, MVI-associated intercellular communication was closely related to the SPP1-CD44 and ANXA1-FPR1 pathways. These findings resulted in a brilliant predictive model and fresh insights into MVI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891c8bc5aef729b930179251eb7e060720eef85" target='_blank'>
              Machine learning based on biological context facilitates the identification of microvascular invasion in intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Shuaishuai Xu, Mingyu Wan, C. Ye, Ruyin Chen, Qiong Li, Xiaochen Zhang, Jian Ruan
          </td>
          <td>2024-08-01</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea95b5d32d4f493d8d481804d513d4259db2149" target='_blank'>
              Deep Learning-Based Automated Analysis of NK Cell Cytotoxicity in Single Cancer Cell Arrays
              </a>
            </td>
          <td>
            Dowon Moon, Seong-Eun Kim, Chuangqi Wang, Kwonmoo Lee, Junsang Doh
          </td>
          <td>2024-06-27</td>
          <td>BioChip Journal</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: Intermediate cell carcinoma (Int-CA) is a rare and enigmatic primary liver cancer characterized by uniform tumor cells exhibiting mixed features of both HCC and intrahepatic cholangiocarcinoma. Despite the unique pathological features of int-CA, its molecular characteristics remain unclear yet. Methods: RNA sequencing and whole genome sequencing profiling were performed on int-CA tumors and compared with those of HCC and intrahepatic cholangiocarcinoma. Results: Int-CAs unveiled a distinct and intermediate transcriptomic feature that is strikingly different from both HCC and intrahepatic cholangiocarcinoma. The marked abundance of splicing events leading to intron retention emerged as a signature feature of int-CA, along with a prominent expression of Notch signaling. Further exploration revealed that METTL16 was suppressed within int-CA, showing a DNA copy number–dependent transcriptional deregulation. Notably, experimental investigations confirmed that METTL16 suppression facilitated invasive tumor characteristics through the activation of the Notch signaling cascade. Conclusions: Our results provide a molecular landscape of int-CA featured by METTL16 suppression and frequent intron retention events, which may play pivotal roles in the acquisition of the aggressive phenotype of Int-CA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab7b9d18c0e5e1be8d12c7361eff07994e2e2a" target='_blank'>
              Transcriptomic profiling of intermediate cell carcinoma of the liver
              </a>
            </td>
          <td>
            Byungchan Jang, S. Kwon, Jang Hyun Kim, Jung Mo Kim, Taek Chung, Jeong Eun Yoo, Haeryoung Kim, Julien Calderaro, Hyun Goo Woo, Young Nyun Park
          </td>
          <td>2024-08-01</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde19b1f96c1cfcaa9c403d17be06cabeee637d7" target='_blank'>
              Pan-cell death-related signature reveals tumor immune microenvironment and optimizes personalized therapy alternations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Linzhi Han, Jingyi He, Hongxin Xie, Yan Gong, Conghua Xie
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Objective Hepatocellular carcinoma (HCC) poses a significant clinical challenge because the long-term benefits of immune checkpoint blockade therapy are limited. A comprehensive understanding of the mechanisms underlying immunotherapy resistance in HCC is imperative for improving patient prognosis. Design In this study, to systematically investigate the characteristics of cancer-associated fibroblast (CAF) subsets and the dynamic communication among the tumor microenvironment (TME) components regulated by CAF subsets, we generated an HCC atlas by compiling single-cell RNA sequencing (scRNA-seq) datasets on 220 samples from six datasets. We combined spatial transcriptomics with scRNA-seq and multiplexed immunofluorescence to identify the specific CAF subsets in the TME that determine the efficacy of immunotherapy in HCC patients. Results Our findings highlight the pivotal role of POSTN+ CAFs as potent immune response barriers at specific tumor locations, as they hinder effective T-cell infiltration and decrease the efficacy of immunotherapy. Additionally, we elucidated the interplay between POSTN+ CAFs and SPP1+ macrophages, whereby the former recruits the latter and triggers increased SPP1 expression via the IL-6/STAT3 signaling pathway. Moreover, we demonstrated a spatial correlation between POSTN+ CAFs and SPP1+ macrophages, revealing an immunosuppressive microenvironment that limits the immunotherapy response. Notably, we found that patients with elevated expression levels of both POSTN+ CAFs and SPP1+ macrophages achieved less therapeutic benefit in an immunotherapy cohort. Conclusion Our research elucidates light on the role of a particular subset of CAFs in immunotherapy resistance, emphasizing the potential benefits of targeting specific CAF subpopulations to improve clinical responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321d7431417f89b80a8af6ab7f043a0cb955e621" target='_blank'>
              POSTN+ cancer-associated fibroblasts determine the efficacy of immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hao Wang, Yuan Liang, Zheng Liu, Rui Zhang, Jiashuo Chao, Mingming Wang, Mu Liu, Lei Qiao, Zhengfeng Xuan, Haitao Zhao, Ling Lu
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Therapy resistance is a major clinical hurdle in bone cancer treatment and seems to be largely driven by poorly understood microenvironmental factors. Recent evidence suggests a critical role for a unique subpopulation of mesenchymal stem cells with inflammatory features (iMSCs), though their origin and function remained unexplored. We demonstrate that cancer-secreted extracellular vesicles (EVs) trigger the development of iMSCs, which hinder therapy response in vivo, and set out to identify strategies to counteract their function.


EXPERIMENTAL DESIGN
The role of iMSCs in therapy resistance was evaluated in an orthotopic xenograft mouse model of osteosarcoma. EV-induced alterations of the MSC transcriptome were analyzed and compared with scRNA-seq data of osteosarcoma and multiple myeloma patient biopsies. Functional assays identified EV components driving iMSC development. We assessed the efficacy of clinical drugs in blocking iMSC-induced resistance in vivo.


RESULTS
We found that iMSCs are induced by interaction with cancer EVs and completely abrogate the antimetastatic effect of TGFb signaling inhibition. Importantly, EV-induced iMSCs faithfully recapitulate the inflammatory single-cell RNA signature of stromal cells enriched in multiple myeloma and osteosarcoma patient biopsies. Mechanistically, cancer EVs act through two distinct mechanisms. EV-associated TGFb induces IL6 production, while the EV-RNA cargo enhances TLR3-mediated chemokine production. We reveal that simultaneous blockade of downstream EV-activated pathways with ladarixin and tocilizumab disrupts metastasis formation and overcomes iMSC-induced resistance.


CONCLUSIONS
Our observations establish iMSCs as major contributors to drug resistance, reveal EVs as physiological triggers of iMSC development and highlight a promising combination strategy to improve therapy response in bone cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fb87f3a9aca9bc675c26582efc89f032b59273" target='_blank'>
              Tumor-secreted extracellular vesicles counteract therapy response by triggering inflammatory mesenchymal stem cell development.
              </a>
            </td>
          <td>
            Crescenzo Massaro, Hilal N Sensoy, Manon Mulders, Celine De Schrijver, C. Gómez-Martín, Juan Simon-Nieto, T. Lagerweij, Alisha Atmopawiro, Jennifer Pérez-Boza, Maarten P. Bebelman, Leontien Bosch, Simone Foderaro, Mafalda Neves Ferreira, Monique A J van Eijndhoven, Jan R. T. van Weering, C. Dell'Aversana, L. Altucci, C. D. Savci-Heijink, Niels W C J van de Donk, Cristina Giorgio, L. Brandolini, M. Allegretti, D. M. Pegtel, S. R. Baglio
          </td>
          <td>2024-08-08</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="PURPOSE
The molecular drivers underlying mucinous tumor pathogenicity are poorly understood. GNAS mutations predict metastatic burden and treatment resistance in mucinous appendiceal adenocarcinoma. We investigated the pan-cancer clinicopathologic relevance of GNAS variants.


METHODS
We assessed 58,043 patients with Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (IMPACT)-sequenced solid tumors to identify oncogenic variants, including GNAS, associated with mucinous tumor phenotype. We then performed comprehensive molecular analyses to compare GNAS-mutant (mut) and wild-type tumors across cancers. Gene expression patterns associated with GNAS-mut tumors were assessed in a The Cancer Genome Atlas cohort. Associations between GNAS variant status and peritoneal metastasis, first-line systemic therapy response, progression-free survival (PFS), and overall survival (OS) were determined using a propensity-matched subcohort of patients with metastatic disease.


RESULTS
Mucinous tumors were enriched for oncogenic GNAS variants. GNAS was mutated in >1% of small bowel, cervical, colorectal, pancreatic, esophagogastric, hepatobiliary, and GI neuroendocrine cancers. Across these cancers, GNAS-mut tumors exhibited a generally conserved C-to-T mutation-high, aneuploidy-low molecular profile with co-occurring prevalent KRAS variants (65% of GNAS-mut tumors) and fewer TP53 alterations. GNAS-mut tumors exhibited recurrently comutated alternative tumor suppressors (RBM10, INPPL1) and upregulation of MAPK and cell surface modulators. GNAS-mut tumors demonstrate an increased prevalence of peritoneal metastases (odds ratio [OR], 1.7 [95% CI, 1.1 to 2.5]; P = .006), worse response to first-line systemic therapy (OR, 2.2 [95% CI, 1.3 to 3.8]; P = .003), and shorter PFS (median, 5.6 v 7.0 months; P = .047). In a multivariable analysis, GNAS mutated status was independently prognostic of worse OS (hazard ratio, 1.25 [95% CI, 1.01 to 1.56]; adjusted P = .04).


CONCLUSION
Across the assessed cancers, GNAS-mut tumors exhibit a conserved molecular and clinical phenotype defined by mucinous tumor status, increased peritoneal metastasis, poor response to first-line systemic therapy, and worse survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a7fd5d8b4825c722728924fd719f0a0d9f3a8e" target='_blank'>
              Molecular and Clinicopathologic Impact of GNAS Variants Across Solid Tumors.
              </a>
            </td>
          <td>
            P. Johannet, Somer Abdelfattah, Callahan Wilde, Shrey Patel, H. Walch, B. Rousseau, Guillem Argiles, O. Artz, Miteshkumar Patel, Andrea Arfe, A. Cercek, R. Yaeger, K. Ganesh, N. Schultz, Luis A. Diaz, M. Foote
          </td>
          <td>2024-08-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1b63b1c4f14a801f678ccab9f44f1b0b4a85ea" target='_blank'>
              Tumour-intrinsic endomembrane trafficking by ARF6 shapes an immunosuppressive microenvironment that drives melanomagenesis and response to checkpoint blockade therapy
              </a>
            </td>
          <td>
            Yinshen Wee, Junhua Wang, Emily C. Wilson, Coulson P. Rich, Aaron Rogers, Zongzhong Tong, Evelyn DeGroot, Y. N. Gopal, Michael A Davies, Huseyin Atakan Ekiz, Joshua K H Tay, Chris J. Stubben, Kenneth M. Boucher, Juan M. Oviedo, Keke C. Fairfax, Matthew A. Williams, Sheri L Holmen, Roger K. Wolff, Allie H. Grossmann
          </td>
          <td>2024-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Preclinical models that replicate patient tumours as closely as possible are crucial for translational cancer research. While in vitro cancer models have many advantages in assessing tumour response therapy, in vivo systems are essential to enable evaluation of the role of the tumour cell extrinsic factors, such as the tumour microenvironment and host immune system. The requirement for a functional immune system is particularly important given the current focus on immunotherapies. Therefore, we set out to generate an immunocompetent, transplantable model of colorectal cancer suitable for in vivo assessment of immune-based therapeutic approaches. Intestinal tumours from a genetically engineered mouse model, driven by expression of a Pik3ca mutation and loss of Apc, were transplanted into wild type C57BL/6 host mice and subsequently passaged to form a novel syngeneic transplant model of colorectal cancer. Our work confirms the potential to develop a panel of mouse syngeneic grafts, akin to human PDX panels, from different genetically engineered, or carcinogen-induced, mouse models. Such panels would allow the in vivo testing of new pharmaceutical and immunotherapeutic treatment approaches across a range of tumours with a variety of genetic driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e96ac527246a723f0c584b8d2c966fbc703899" target='_blank'>
              Development of an in vivo syngeneic mouse transplant model of invasive intestinal adenocarcinoma driven by endogenous expression of Pik3caH1047R and Apc loss
              </a>
            </td>
          <td>
            Francesc de las Heras, Camilla B. Mitchell, William K. Murray, N. Clemons, Wayne A. Phillips
          </td>
          <td>2024-08-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Brain tumors, particularly glioblastoma (GBM), are devastating and challenging to treat, with a low 5-year survival rate of only 6.6%. Mouse models are established to understand tumorigenesis and develop new therapeutic strategies. Large-scale genomic studies have facilitated the identification of genetic alterations driving human brain tumor development and progression. Genetically engineered mouse models (GEMMs) with clinically relevant genetic alterations are widely used to investigate tumor origin. Additionally, syngeneic implantation models, utilizing cell lines derived from GEMMs or other sources, are popular for their consistent and relatively short latency period, addressing various brain cancer research questions. In recent years, the success of immunotherapy in specific cancer types has led to a surge in cancer immunology-related research which specifically necessitates the utilization of immunocompetent mouse models. In this review, we provide a comprehensive summary of GEMMs and syngeneic mouse models for adult brain tumors, emphasizing key features such as model origin, genetic alteration background, oncogenic mechanisms, and immune-related characteristics. Our review serves as a valuable resource for the brain tumor research community, aiding in the selection of appropriate models to study cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff748c1f4b0e36e21dc90ffb3f8ae15d403a0e7" target='_blank'>
              In Vivo Mouse Models for Adult Brain Tumors: Exploring Tumorigenesis and Advancing Immunotherapy Development.
              </a>
            </td>
          <td>
            John Figg, Dongjiang Chen, Laura Falceto Font, Catherine Flores, Dan Jin
          </td>
          <td>2024-07-11</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background PIK3CA mutations are implicated in various cancers, but the implications of multiple concurrent mutations and their orientations within the gene have not been fully explored. Methods In this study, we analyzed multi‐PIK3CA mutations across a diverse pan‐cancer cohort comprising 3564 tumors. Results Multi‐PIK3CA mutations were present in 10.3% of all PIK3CA‐mutant tumors, predominantly occurring in breast and gynecological cancers. Notably, mutations within the helical domain (E542:E545) exclusively occurred in the trans‐orientation, contrasting with mutations in the kinase ABD and C2 domains, which mainly appeared in the cis orientation. Conclusions The distinct pattern of mutation orientations in PIK3CA suggests variable oncogenic potential, with helical domain mutations in the trans‐orientation potentially being less oncogenic. These findings highlight the importance of mutation orientation in the PIK3CA gene as potential biomarkers for targeted therapy. This understanding is crucial for designing clinical trials that leverage PI3K inhibitors, aiming for more effective and precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c039fa2d1ed75da420038d18b0f0cfdcf748c0d" target='_blank'>
              Characterizing multi‐PIK3CA mutations across cancer types: Toward precision oncology
              </a>
            </td>
          <td>
            Kohei Nakamura, Marin Ishikawa, R. Kawano, E. Aimono, Takaaki Mizuno, S. Nohara, Shigeki Tanishima, Hideyuki Hayashi, H. Nishihara
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/571475d254bb27cb6d2937910baaa56e8534b72e" target='_blank'>
              Refining the optimal CAF cluster marker for predicting TME-dependent survival expectancy and treatment benefits in NSCLC patients
              </a>
            </td>
          <td>
            Kai Li, Rui Wang, Guo-Wei Liu, Zi-Yang Peng, Ji-Chang Wang, G. Xiao, Shou-Ching Tang, Ning Du, Jia Zhang, Jing Zhang, Hong Ren, Xin Sun, Yiping Yang, Da-Peng Liu
          </td>
          <td>2024-07-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Previous characterization of the genome and transcriptome of glioblastoma (GBM) has revealed molecular alterations that potentially drive GBM pathogenesis and heterogeneity 1-6. These open-resources are evolving, such as The Cancer Genome Atlas (TCGA) and The Cancer Imaging Atlas (TCIA) at the National Institute of Health comprising a large cohort of molecular and MRI data. Yet, no report deciphers the link between molecular signatures and MRI-classified GBM. The necessity to re-form molecular and imaging data motivated our computational approach to integrate TCIA and TCGA datasets derived from GBM. We uncovered common and distinct molecular signatures across GBM patients and specific to tumor locations, respectively. Despite heterogeneity in GBM, the top 12 genes from our analysis highlights that the dysregulation of a subset of neurotransmitter receptor or transporter and synaptic activity is common across GBM patients. The coherent layer of imaging and molecular information would help us stratify precision neuro-oncology and treatment options in ways that are not possible through MRI or genomic data alone. Our findings provide molecular targets in the disrupted neurocircuit of GBM, suggesting imbalanced excitation and inhibition. Given the fact that GBM patients exhibit similar symptoms resembling patients with neurodegenerative diseases and seizures, our results supported the hypothesis-GBM in the context of neurological disorders beyond a solely cancerous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bcee33160d7d19048e98fc72e314300b743ed5" target='_blank'>
              Differential Gene Expression in MRI-classified Glioblastoma
              </a>
            </td>
          <td>
            Christopher T. Rhodes, Yufeng Wang, Chin-Hsing Annie Lin
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley, E. Tyler
          </td>
          <td>2024-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary The expression of the mechanosensory Piezo2 channel has already been described in different malignant tumors. There is discordance in the literature regarding breast carcinoma, with its expression described either as decreased or increased in neoplasms with respect to benign tissue. A retrospective cohort of 125 patients whose breasts were resected for carcinoma was chosen to determine the relationship between Piezo2 and different clinical and histological variables. A significant association was found with the Ki67 proliferation index, with a tendency for most proliferative tumors to be positive for Piezo2. Abstract In the last decade, a group of Ca2+ channels called Piezo were discovered, demonstrating a decisive role in the cellular response to mechanical stimuli and being essential in the biological behavior of cells regarding the extracellular compartment. Several investigations have suggested a potential role in carcinogenesis, with a tumor suppressor role in some cases but increased expression in several high-grade neoplasms. Regarding Piezo2 expression in mammary gland neoplasms, a protective role for Piezo2 was initially suggested, but a subsequent study demonstrated a relationship between Piezo2 expression and the highly aggressive triple-negative phenotype of breast carcinoma. A cohort of 125 patients with clinical follow-up was chosen to study Piezo2 expression and clarify its clinical implications using the same immunohistochemical evaluation performed for other breast carcinoma parameters. Fisher’s exact test was chosen to identify potential relationships between the different variables. A significant association was found with the Ki67 proliferation index, but not with mitoses. The tendency of most proliferative tumors was to have an increased score for Piezo2. A similar association was found between Piezo2 expression and perineural invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6531181900e59a16e97bcf0664ec51d36a032225" target='_blank'>
              Prognostic Evaluation of Piezo2 Channels in Mammary Gland Carcinoma
              </a>
            </td>
          <td>
            R. Martín-Sanz, Aline Rodrigues-Françoso, Y. García-Mesa, F. J. García-Alonso, María Asunción Gómez-Muñoz, Sandra Malmierca-González, Rocío Salazar-Blázquez, Olivia García-Suárez, J. Feito
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ff0f7d3a22a2ccb36635a3877c0f50c26a651" target='_blank'>
              MHC-I-presented non-canonical antigens expand the cancer immunotherapy targets in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yangyang Cai, Donghao Li, Dezhong Lv, Jiaxin Yu, Yingying Ma, Tiantongfei Jiang, Na Ding, Zhigang Liu, Yongsheng Li, Juan Xu
          </td>
          <td>2024-08-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary This study investigates the impact of the BRAFV600E mutation, present in half of melanoma cases, on immune microenvironment and therapeutic response. Analyzing the Cancer Genome Atlas data, we uncover that the mutation correlates with reduced tumor mutation burden, indicating a lower generation of immune-stimulating neopeptides. Examination of immune subtypes reveal heightened immunosuppression in BRAFV600E-mutated tumors. Using melanoma cell lines harboring different genomic profiles, we observed an enhanced response to immunogenic cell death (ICD)—a crucial process eliciting anti-tumor immune responses—mediated by photodynamic therapy (PDT). Transcriptomic analysis highlights upregulation of IFNAR1, IFNAR2, and CXCL10 genes in BRAFV600E-mutated cells, suggesting their role in ICD. Our results describe the intricate relationship between the BRAFV600E mutation and immune responses, hinting at a potential link between this mutation and responsiveness to ICD-inducing therapies, particularly PDT, putatively linked to increased IFN-1 pathway activation. Abstract The BRAFV600E mutation, found in approximately 50% of melanoma cases, plays a crucial role in the activation of the MAPK/ERK signaling pathway, which promotes tumor cell proliferation. This study aimed to evaluate its impact on the melanoma immune microenvironment and therapeutic responses, particularly focusing on immunogenic cell death (ICD), a pivotal cytotoxic process triggering anti-tumor immune responses. Through comprehensive in silico analysis of the Cancer Genome Atlas data, we explored the association between the BRAFV600E mutation, immune subtype dynamics, and tumor mutation burden (TMB). Our findings revealed that the mutation correlated with a lower TMB, indicating a reduced generation of immunogenic neoantigens. Investigation into immune subtypes reveals an exacerbation of immunosuppression mechanisms in BRAFV600E-mutated tumors. To assess the response to ICD inducers, including doxorubicin and Me-ALA-based photodynamic therapy (PDT), compared to the non-ICD inducer cisplatin, we used distinct melanoma cell lines with wild-type BRAF (SK-MEL-2) and BRAFV600E mutation (SK-MEL-28, A375). We demonstrated a differential response to PDT between the WT and BRAFV600E cell lines. Further transcriptomic analysis revealed upregulation of IFNAR1, IFNAR2, and CXCL10 genes associated with the BRAFV600E mutation, suggesting their involvement in ICD. Using a gene reporter assay, we showed that PDT robustly activated the IFN-1 pathway through cGAS-STING signaling. Collectively, our results underscore the complex interplay between the BRAFV600E mutation and immune responses, suggesting a putative correlation between tumors carrying the mutation and their responsiveness to therapies inducing the IFN-1 pathway, such as the ICD inducer PDT, possibly mediated by the elevated expression of IFNAR1/2 receptors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ed26388b396064bd1f63bcbdf14dd7a6adc97c" target='_blank'>
              Impact of Genomic Mutation on Melanoma Immune Microenvironment and IFN-1 Pathway-Driven Therapeutic Responses
              </a>
            </td>
          <td>
            F. Mentucci, Elisa Ayelén Romero Nuñez, Agustina Ercole, Valentina Silvetti, J. Dal Col, M. Lamberti
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Here, we investigate if microproteins encoded by noncanonical open reading frames (ncORFs) are a relevant source of tumor-specific antigens. We analyze RNA sequencing data from 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue together with ribosome profiling and immunopeptidomics data. Combining human leukocyte antigen–epitope binding predictions and experimental validation experiments, we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs, including two peptides that can trigger an immune response in humanized mice. We identify a subset of 33 tumor-specific long noncoding RNAs expressing novel cancer antigens shared by more than 10% of the HCC samples analyzed, which, when combined, cover a large proportion of the patients. The results of the study open avenues for extending the range of anticancer vaccines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537c5fce6ae2d9f52e6619d811174a75e25d4f1f" target='_blank'>
              Microproteins encoded by noncanonical ORFs are a major source of tumor-specific antigens in a liver cancer patient meta-cohort
              </a>
            </td>
          <td>
            Marta E. Camarena, Patrick Theunissen, Marta Ruiz, Jorge Ruiz-Orera, Beatriz Calvo-Serra, Robert Castelo, Carla Castro, Pablo Sarobe, Puri Fortes, J. Perera-Bel, M. M. Albà
          </td>
          <td>2024-07-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) is dependent on a supportive tumor immune microenvironment (TIME), where infiltration of CD163+ macrophages has a negative prognostic impact. This study explores how abundance and spatial localization of CD163+ cells are associated with the biology of the MCL TIME. This is achieved through spatial multi-omic investigations of tumor and infiltrating CD163+ and CD3+ cells, respectively. We analyzed diagnostic MCL tissue from 100 patients. Sixty-three proteins were measured by GeoMx® digital spatial profiling in tissue microarrays. Regions of interests (ROIs) were selected in tumor-rich and tumor-sparse tissue regions. Molecular profiling of CD163+ macrophage segments, CD20+ MCL tumor cell segments and CD3+ T-cell segments was performed. To validate protein profiles, 1811 mRNAs were measured in CD20+ cells and two subsets of T-cells. Image analysis was used to extract the phenotype and position of each targeted cell allowing exploration of cell frequencies and cellular neighborhoods. Proteomic investigations revealed that CD163+ cells modulate their immune profile depending on the localization and that the immune inhibitory molecules VISTA and B7-H3 have higher expression in tumor-sparse versus tumor-rich tissue regions and targeting should be explored. We show that MCL tissues with more abundant infiltration of CD163+ cells have a higher expression of key components of the mitogen-activated protein kinase (MAPK) pathway, which was validated by complementary mRNA analyses. Thus, the MAPK pathway may be a feasible therapeutic target in MCL patients with CD163+ cell infiltration. We further show the independent and combined prognostic value of CD11c and CD163 beyond established risk factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d71801d630d06d650258878ed1d5d14b266a8e5" target='_blank'>
              CD163+ macrophages in mantle cell lymphoma induce activation of pro-survival pathways and immune suppression.
              </a>
            </td>
          <td>
            J. Rodrigues, L. Lokhande, L. Olsson, May Hassan, Angelica Johansson, Anna Janská, Darshan Kumar, Lina Schmidt, A. Nikkarinen, P. Hollander, I. Glimelius, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-07-03</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Single-cell technology has allowed researchers to probe tissue complexity and dynamics at unprecedented depth in health and disease. However, the generation of high-dimensionality single-cell atlases and virtual three-dimensional tissues requires integrated reference maps that harmonize disparate experimental designs, analytical pipelines, and taxonomies. Here, we present a comprehensive single-cell transcriptome integration map of cardiac fibrosis, which underpins pathophysiology in most cardiovascular diseases. Our findings reveal similarity between cardiac fibroblast (CF) identities and dynamics in ischemic versus pressure overload models of cardiomyopathy. We also describe timelines for commitment of activated CFs to proliferation and myofibrogenesis, profibrotic and antifibrotic polarization of myofibroblasts and matrifibrocytes, and CF conservation across mouse and human healthy and diseased hearts. These insights have the potential to inform knowledge-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39d1bacb1f7ac6282e099a55f3c47a0b78634e" target='_blank'>
              Integration mapping of cardiac fibroblast single-cell transcriptomes elucidates cellular principles of fibrosis in diverse pathologies
              </a>
            </td>
          <td>
            R. Patrick, Vaibhao Janbandhu, Vikram Tallapragada, Shannon S M Tan, Emily E. McKinna, Osvaldo Contreras, S. Ghazanfar, David T. Humphreys, Nicholas J. Murray, Yen T. H. Tran, Robert D Hume, James J. H. Chong, Richard P. Harvey
          </td>
          <td>2024-06-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Organoid tumor models have emerged as a powerful tool in the fields of biology and medicine as such 3D structures grown from tumor cells recapitulate better tumor characteristics, making these tumoroids unique for personalized cancer research. Assessment of their functional behavior, particularly protein secretion, is of significant importance to provide comprehensive insights. Here, a label-free spectroscopic imaging platform is presented with advanced integrated optofluidic nanoplasmonic biosensor that enables real-time secretion analysis from single tumoroids. A novel two-layer microwell design isolates tumoroids, preventing signal interference, and the microarray configuration allows concurrent analysis of multiple tumoroids. The dual imaging capability combining time-lapse plasmonic spectroscopy and bright-field microscopy facilitates simultaneous observation of secretion dynamics, motility, and morphology. The integrated biosensor is demonstrated with colorectal tumoroids derived from both cell lines and patient samples to investigate their vascular endothelial growth factor A (VEGF-A) secretion, growth, and movement under various conditions, including normoxia, hypoxia, and drug treatment. This platform, by offering a label-free approach with nanophotonics to monitor tumoroids, can pave the way for new applications in fundamental biological studies, drug screening, and the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738f082d29ba965763765f12b51207081cea62bd" target='_blank'>
              Nanoplasmonic Single-Tumoroid Microarray for Real-Time Secretion Analysis.
              </a>
            </td>
          <td>
            Yen‐Cheng Liu, Saeid Ansaryan, Jiayi Tan, N. Broguiere, L. F. Lorenzo-Martín, K. Homicsko, G. Coukos, Matthias P. Lütolf, Hatice Altug
          </td>
          <td>2024-06-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8c4d0ef06e7916deea0b92c894ed2b387bbc58" target='_blank'>
              Unraveling the Drug Response Heterogeneity with Single-Cell Vibrational Phenomics.
              </a>
            </td>
          <td>
            Yue Wang, Yadi Wang, Junhong Lü, Xueling Li
          </td>
          <td>2024-06-24</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Brain metastases (BrMs) constitute the most common type of brain cancer with abysmal prognosis. Given the high incidence and poor prognosis of BrM, understanding the tumor microenvironment (TME) architecture across BrM subtypes is increasingly crucial, especially with the emergence of novel immunotherapeutic agents like immune checkpoint inhibitors, which have demonstrated efficacy in melanoma and lung cancer BrM. Tertiary lymphoid structures (TLS) are spatially organized lymphoid aggregates in the TME that mediate antitumor immunity. Despite being established as prognostic and predictive markers in various cancers, their significance in BrM lacks extensive investigation. We performed gene expression profiling using paired-end RNA sequencing and multiplex immunofluorescence combined with spatial imaging on 149 human BrM samples. Cell type deconvolution analysis revealed that BrMs from non-small cell lung carcinoma (NSCLC) and malignant melanoma exhibited significantly higher immune cell infiltration compared to BrMs from breast carcinoma. Furthermore, B cell infiltration was detected in some patients, suggesting potential TLS formation. We created a metagene signature comprising TLS hallmarks in BrMs using transcriptomic profiling, allowing us to categorize BrM patients into TLS-positive and TLS-negative groups. This finding was corroborated by multiplex immunofluorescence staining of corresponding BrM tissue sections, which confirmed TLS-like aggregate formation in 40% of the BrM patients. Differential gene expression analysis highlighted the presence of activated lymphocyte-mediated immunity as well as increased immunoglobulin production signature in TLS-positive tumors. Additionally, a higher TLS signature score was associated with improved overall survival of lung carcinoma patients after BrM diagnosis. Our results underscore the heterogeneity of the tumor microenvironment in BrM, and the identification of intratumoral TLS may guide patient stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f904325c9c40fd220a1c52e4c6b7474d7dea43" target='_blank'>
              BMRK-09 PROGNOSTIC ROLE OF TUMOR-INFILTRATING LYMPHOCYTES AND TERTIARY LYMPHOID STRUCTURES IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            S. S. Mughal, Y. Reiss, J. Felsberg, L. Meyer, J. Macas, S. Schlue, Tatjana Starzetz, K. Köhrer, Tanja N Fehm, Volkmar Müller, K. Lamszus, Iris Helfrich, H. Wikman, A. Berghoff, B. Brors, Karlheinz Plate, G. Reifenberger
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="INTRODUCTION
Chimeric antigen receptor (CAR) T-cells have emerged as a ground-breaking therapy for the treatment of hematological malignancies due to their capacity for rapid tumor-specific killing and long-lasting tumor immunity. However, the same success has not been observed in patients with solid tumors. Largely, this is due to the additional challenges imposed by safe and uniform target selection, inefficient CAR T-cell access to sites of disease and the presence of a hostile immunosuppressive tumor microenvironment.


AREAS COVERED
Literature was reviewed on the PubMed database from the first description of a CAR by Kuwana, Kurosawa and colleagues in December 1987 through to the present day. This literature indicates that in order to tackle solid tumors, CAR T-cells can be further engineered with additional armoring strategies that facilitate trafficking to and infiltration of malignant lesions together with reversal of suppressive immune checkpoints that operate within solid tumor lesions.


EXPERT OPINION
In this review, we describe a number of recent advances in CAR T-cell technology that set out to combat the problems imposed by solid tumors including tumor recruitment, infiltration, immunosuppression, metabolic compromise, and hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c957915bda93838ef155e042bc947afb3392ab3" target='_blank'>
              CAR-T cell technologies that interact with the tumour microenvironment in solid tumours.
              </a>
            </td>
          <td>
            Chelsea Alice Taylor, Maya Glover, John Maher
          </td>
          <td>2024-07-17</td>
          <td>Expert review of clinical immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c323e127988809654aaf5a547e1a3c8242712a2" target='_blank'>
              Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
              </a>
            </td>
          <td>
            Ian A. Mellis, Madeline E. Melzer, Nicholas Bodkin, Yogesh Goyal
          </td>
          <td>2024-08-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Paget's "Seed and Soil" theory, proposed in 1889, emphasizes the importance of the microenvironment where cancer cells grow in metastatic sites. Over a century later, this concept remains a cornerstone in comprehending cancer biology and devising treatment strategies. The "Seed and Soil" theory, which initially explained how cancer spreads to distant organs, now also applies to the tumor microenvironment (TME) within primary tumors. This theory emphasizes the critical interaction between cancer cells ("seeds") and their surrounding environment ("soil") and how this interaction affects both tumor progression within the primary site and at metastatic sites. An important point to note is that the characteristics of the TME are not static but dynamic, undergoing substantial changes during tumor progression and after treatment with therapeutic drugs. Cancer-associated fibroblasts (CAFs), recognized as the principal noncancerous cellular component within the TME, play multifaceted roles in tumor progression including promoting angiogenesis, remodeling the extracellular matrix, and regulating immune responses. In this comprehensive review, we focus on the findings regarding how the dynamics of CAFs contribute to cancer progression and drug sensitivity. Understanding the dynamics of CAFs could provide new insights into cancer pathology and lead to important advancements in cancer research and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17013ae4cfc9ae5c5a61d1e465d0fff7154b7a9" target='_blank'>
              New insights into cancer pathology learned from the dynamics of cancer-associated fibroblasts.
              </a>
            </td>
          <td>
            Genichiro Ishii
          </td>
          <td>2024-06-25</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma is the most common and malignant primary brain tumor, with high morbidity and mortality. Despite an aggressive, multimodal treatment regimen, including surgical resection followed by chemotherapy and radiotherapy, the prognosis of glioblastoma patients remains poor. One formidable challenge to advancing glioblastoma therapy is the complexity of the tumor microenvironment. The tumor microenvironment of glioblastoma is a highly dynamic and heterogeneous system that consists of not only cancerous cells but also various resident or infiltrating inflammatory cells. These inflammatory cells not only provide a unique tumor environment for glioblastoma cells to develop and grow but also play important roles in regulating tumor aggressiveness and treatment resistance. Targeting the tumor microenvironment, especially neuroinflammation, has increasingly been recognized as a novel therapeutic approach in glioblastoma. In this review, we discuss the components of the tumor microenvironment in glioblastoma, focusing on neuroinflammation. We discuss the interactions between different tumor microenvironment components as well as their functions in regulating glioblastoma pathogenesis and progression. We will also discuss the anti-tumor microenvironment interventions that can be employed as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670eaa472170b926b6f530ad1202e654791b0745" target='_blank'>
              Neuroinflammation in Glioblastoma: Progress and Perspectives
              </a>
            </td>
          <td>
            Xin Li, Wenting Gou, Xiaoqin Zhang
          </td>
          <td>2024-07-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cancer arises through the accumulation of somatic mutations in key biological pathways. This paper aims to develop a probabilistic approach to delineate the temporal order of mutations during cancer development based on mutation profile data from a cohort of patients. A unique feature of our method is that it incorporates intra-tumor heterogeneity (ITH) information, which refers to the heterogeneous cell populations within a tumor and characterizes the evolutionary history of the tumor. We showed that by integrating ITH, pathways, and functional annotation information, our method yielded high accuracy in inferring the temporal order of pathway mutations during carcinogenesis. Abstract The development of cancer involves the accumulation of somatic mutations in several essential biological pathways. Delineating the temporal order of pathway mutations during tumorigenesis is crucial for comprehending the biological mechanisms underlying cancer development and identifying potential targets for therapeutic intervention. Several computational and statistical methods have been introduced for estimating the order of somatic mutations based on mutation profile data from a cohort of patients. However, one major issue of current methods is that they do not take into account intra-tumor heterogeneity (ITH), which limits their ability to accurately discern the order of pathway mutations. To address this problem, we propose PATOPAI, a probabilistic approach to estimate the temporal order of mutations at the pathway level by incorporating ITH information as well as pathway and functional annotation information of mutations. PATOPAI uses a maximum likelihood approach to estimate the probability of pathway mutational events occurring in a specific sequence, wherein it focuses on the orders that are consistent with the phylogenetic structure of the tumors. Applications to whole exome sequencing data from The Cancer Genome Atlas (TCGA) illustrate our method’s ability to recover the temporal order of pathway mutations in several cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296d55a31b561e1788fdfc0790cecd99495dadae" target='_blank'>
              A Probabilistic Approach to Estimate the Temporal Order of Pathway Mutations Accounting for Intra-Tumor Heterogeneity
              </a>
            </td>
          <td>
            Menghan Wang, Yanqi Xie, Jinpeng Liu, Austin Li, Li Chen, Arnold Stromberg, Susanne M. Arnold, Chunming Liu, Chi Wang
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a82685ebf391bea616973c3440cc99b04c6536" target='_blank'>
              Differential transcriptional invasion signatures from patient derived organoid models define a functional prognostic tool for head and neck cancer
              </a>
            </td>
          <td>
            Peter D. Haughton, W. Haakma, Theofilos Chalkiadakis, G. Breimer, E. Driehuis, Hans Clevers, Stefan M. Willems, Stefan Prekovic, Patrick W B Derksen
          </td>
          <td>2024-06-28</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Medullary thyroid carcinoma (MTC) is driven by a small number of pathogenic genetic variants and tumours usually exhibit a correspondingly low tumour mutational burden. This reduces tumour visibility to the immune system and impacts the immune cell profile of the tumour microenvironment. In the last decade targeted pathway inhibitors have revolutionized the therapeutic landscape for patients with advanced disease, with increasing options for systemic therapy tailored to the molecular signature of the tumour. Therefore, understanding the molecular basis of disease, pathogenesis of immune evasion and mechanisms of escape of pathway inhibition is of paramount importance. Here, we summarize genetic and molecular drivers of MTC and their relevance to tumour immunogenicity, the cellular milieu of the tumour microenvironment, and response to targeted therapy. Abstract In this review, we explore the underlying molecular biology of medullary thyroid carcinoma (MTC) and its interplay with the host immune system. MTC is consistently driven by a small number of specific pathogenic variants, beyond which few additional genetic events are required for tumorigenesis. This explains the exceedingly low tumour mutational burden seen in most MTC, in contrast to other cancers. However, because of the low tumour mutational burden (TMB), there is a correspondingly low level of tumour-associated neoantigens that are presented to the host immune system. This reduces tumour visibility and vigour of the anti-tumour immune response and suggests the efficacy of immunotherapy in MTC is likely to be poor, acknowledging this inference is largely based on the extrapolation of data from other tumour types. The dominance of specific RET (REarranged during Transfection) pathogenic variants in MTC tumorigenesis rationalizes the observed efficacy of the targeted RET-specific tyrosine kinase inhibitors (TKIs) in comparison to multi-kinase inhibitors (MKIs). Therapeutic durability of pathway inhibitors is an ongoing research focus. It may be limited by the selection pressure TKI treatment creates, promoting survival of resistant tumour cell clones that can escape pathway inhibition through binding-site mutations, activation of alternate pathways, and modulation of the cellular and cytokine milieu of the tumour microenvironment (TME).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0d9ed357e13a4c1b8c8631e5ca7b2fb2abcfd" target='_blank'>
              Medullary Thyroid Cancer: Molecular Drivers and Immune Cellular Milieu of the Tumour Microenvironment—Implications for Systemic Treatment
              </a>
            </td>
          <td>
            A. Papachristos, Hazel Serrao-Brown, Anthony J Gill, R. Clifton-Bligh, Stanley B Sidhu
          </td>
          <td>2024-06-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background co-targeting of immune checkpoint inhibitors (ICI) CTLA-4 and PD-1 has recently become the new first-line standard of care therapy of pleural mesothelioma (PM) patients, with a significant improvement of overall survival over conventional chemotherapy. The analysis by tumor histotype demonstrated a greater efficacy of ICI therapy in non-epithelioid (non-E) vs epithelioid (E) PM; although some E PM patients also benefit from treatment. This evidence suggests that molecular tumor features, beyond histotype, could be relevant to improve the efficacy of ICI therapy in PM. Among these, tumor DNA methylation emerges as a promising factor to explore, due to its potential role in driving the immune phenotype of cancer cells. Thus, we utilized a panel of cultured PM cells of different histotype, to provide preclinical evidence supporting the role of the tumor methylation landscape and of its pharmacologic modulation, to prospectively improve the efficacy of ICI therapy of PM patients. Methods the methylome profile (EPIC array) of distinct E (#5) and non-E (#9) PM cell lines was analyzed, followed by integrated analysis with their associated transcriptomic profile (Clariom S array), before and after in vitro treatment with the DNA hypomethylating agent (DHA) guadecitabine. The most variable methylated probes were selected to calculate the methylation score (CIMP index) for each cell line at baseline. Genes that were differentially expressed and methylated were then selected for gene ontology analysis. Results the CIMP index stratified PM cell lines in two distinct classes, CIMP (hyper-methylated; #7) and LOW (hypo-methylated; #7), regardless of their E or non-E histotype. Integrated analyses of methylome and transcriptome data revealed that CIMP PM cells had a substantial number of hyper-methylated, silenced genes, which negatively impacted their immune phenotype compared to LOW PM cells. Treatment with DHA reverted the methylation-driven immune-compromised profile of CIMP PM cells and enhanced the constitutive immune-favorable profile of LOW PM cells. Conclusion the study highlighted the relevance of DNA methylation in shaping the constitutive immune classification of PM cells, that is independent from their histological subtypes. The identified role of DHA in shifting the phenotype of PM cells towards an immune-favorable state supports its role in clinical trials of precision epigenetic therapy combined with ICI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b753d8055100836c4c184dd41cdeb41298408ffc" target='_blank'>
              DNA methylation status classifies pleural mesothelioma cells according to their immune profile: implication for precision epigenetic therapy
              </a>
            </td>
          <td>
            M. F. Lofiego, Rossella Tufano, Emma Bello, Laura Solmonese, Francesco Marzani, Francesca Piazzini, Fabrizio Celesti, F. Caruso, Teresa Maria Rosaria Noviello, R. Mortarini, A. Anichini, Michele Ceccarelli, Luana Calabrò, Michele Maio, S. Coral, A. D. Di Giacomo, A. Covre
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Aggrephagy, a type of autophagy, degrades the aggregation of misfolded protein in cells. However, the role of aggrephagy in multiple myeloma (MM) has not been fully demonstrated. In this study, we first investigated the correlation between aggrephagy signaling, MM immune microenvironment composition and disease prognosis. Single-cell RNA-seq data, including the expression profiles of 12,187 single cells from seven MM bone marrow (BM) and seven healthy BM samples, were analyzed by non-negative matrix factorization for 44 aggrephagy-related genes. Bulk RNA-seq cohorts from the Gene Expression Omnibus database were used to evaluate the prognostic value of aggrephagy-related immune cell subtypes and predict immune checkpoint blockade immunotherapeutic response in MM. Compared with healthy BM, MM BM exhibited different patterns of aggrephagy-related gene expression. In MM BM, macrophages, CD8+ T cells, B cells and natural killer cells could be grouped into four to nine aggrephagy-related subclusters. The signature of aggrephagy signaling molecule expression in the immune cells correlates with the patient's prognosis. Our investigation provides a novel view of aggrephagy signaling in MM tumor microenvironment cells, which might be a prognostic indicator and potential target for MM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7f506c9ce79ae57845d0dafd18ceb978803a7d" target='_blank'>
              Single-cell sequencing reveals the correlation of aggrephagy signaling and multiple myeloma immune microenvironment composition.
              </a>
            </td>
          <td>
            Xin Wang, Yu Feng, Fangfang Wang, Zhimei Lin, Jingcao Huang, Qian Li, Hongmei Luo, Xiang Liu, Xinyu Zhai, Qianwen Gao, Linfeng Li, Yue Zhang, Jingjing Wen, Li Zhang, Ting Niu, Yuhuan Zheng
          </td>
          <td>2024-07-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="APOBEC3A and APOBEC3B genome mutators, key drivers of tumor evolution and drug resistance, may also contribute to generation of neoepitopes for cytotoxic T cells (CTLs). Given the vast polymorphism of Class I HLA, the CTL immunopeptidome, comprised of all possible 8-11mer peptides presented by several thousand HLA class I alleles is highly variable in the population based on individual HLA class I genotype. We developed a genome-wide immunogenicity scanning platform to systematically explore how APOBEC3A/B-driven mutational landscapes reshape the immunogenic potential of the human immunopeptidome. We evaluated all possible APOBEC3-mediated mutations, numbering in the billions, for their potential impact on peptide:MHC and T cell receptor binding. We found that HLA class I alleles vary markedly in their immunopeptidome’s susceptibility to APOBEC3A/B-induced mutational changes, with fewer alleles exhibiting significant increase in neoepitope immunogenicity. Mutations within APOBEC3B hotspots displayed the greatest capacity for enhanced immunogenicity and neo-epitope formation. We further found that the cumulative potential of an individual’s HLA haplotype’s immunopeptidome to gain immunogenicity upon APOBEC3-mediated mutation is a robust predictor of survival in APOBEC3-mutated tumors and correlates with enhanced CD8+ T cell activation. Thus, HLA haplotype is a prognostic marker in APOBEC3-mutated tumors, offering insights into the interplay between the source of cancer genome instability and immune surveillance. Significance statement This work is the first to conduct a genome-wide scan to quantify the impact of A3A- and A3B-mediated mutations on the immunogenicity changes in the human immunopeptidome presented by several thousand HLA class I alleles. We found that the effects of APOBEC3-mediated mutations vary significantly across the immunopeptidomes of different HLA alleles, and that HLA genotype is a predictor of survival and immune activation status in APOBEC3-mutated tumors, reflecting each individual’s unique HLA class I immunopeptidome’s propensity for either increased or decreased immunogenicity when altered by A3A and/or A3B. The identification of HLA genotype as a biomarker in APOBEC3-mutated tumors presents a promising opportunity for developing personalized immunotherapy strategies and exploring potential A3 inhibition. This study highlights that the effects of APOBEC3-mediated mutations on immunogenicity are not random; rather, individual HLA class I polymorphisms determine whether these mutations enhance or suppress anti-tumor immunity, which has direct prognostic implications for solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aed813291723dff8c62fb49014cfcd5f3563f33b" target='_blank'>
              HLA polymorphism impacts immune response to neoepitopes and survival in APOBEC-mutated cancers
              </a>
            </td>
          <td>
            Faezeh Borzooee, Alireza Heravi-Moussavi, Mani Larijani
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell migration requires the interplay among diverse migration patterns. The molecular basis of distinct migration programs is undoubtedly vital but not fully explored. Meanwhile, the lack of tools for investigating spontaneous migratory plasticity in a single living cell also adds to the hindrance. Here, we developed a micro/nanotechnology-enabled single-cell analytical platform to achieve coherent monitoring of spontaneous migratory pattern and signaling molecules. Via the platform, we unveiled a previously unappreciated STAT3 regionalization on the multifunctional regulations of migration. Specifically, nuclear STAT3 is associated with amoeboid migration, while cytoplasmic STAT3 promotes mesenchymal movement. Opposing effects of JAK2 multisite phosphorylation shape its response to STAT3 distribution in a dynamic and antagonistic manner, eventually triggering a reversible amoeboid-mesenchymal transition. Based on the above results, bioinformatics further revealed a possible downstream regulator of nucleocytoplasmic STAT3. Thus, our platform, as an exciting technological advance in single-cell migration research, can provide in-depth mechanism interpretations of tumor metastasis and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7aa9042cbc5c64cbf53c127a1f559066cb19d" target='_blank'>
              Single Living Cell "Observation-Analysis" Integrated Platform Decodes Cell Migration Plasticity Orchestrated by Nucleocytoplasmic STAT3.
              </a>
            </td>
          <td>
            Anqi Zhang, Wanting Qu, Peixin Guan, Ying Li, Zhen Liu
          </td>
          <td>2024-06-28</td>
          <td>Nano letters</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f902f0c016306dc82479eae5dce47565c7056a" target='_blank'>
              Integrated immuno-transcriptomic analysis of ovarian cancer identifies a four-chemokine-dominated subtype with antitumor immune-active phenotype and favorable prognosis.
              </a>
            </td>
          <td>
            Lili Zhuo, Fanling Meng, Kaidi Sun, Meng Zhou, Jie Sun
          </td>
          <td>2024-08-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Advances in understanding the pathological mechanisms of breast cancer have resulted in the emergence of novel therapeutic strategies. However, triple-negative breast cancer (TNBC), a molecular subtype of breast cancer with a poor prognosis, lacks classical and general therapeutic targets, hindering the clinical application of several therapies to breast cancer. As insights into the unique immunity and molecular mechanisms of TNBC have become more extensive, immunotherapy has gradually become a valuable complementary approach to classical radiotherapy and chemotherapy. CD8+ cells are significant actors in the tumor immunity cycle; thus, research on TNBC immunotherapy is increasingly focused in this direction. Recently, CD8+ tissue-resident memory (TRM) cells, a subpopulation of CD8+ cells, have been explored in relation to breast cancer and found to seemingly play an undeniably important role in tumor surveillance and lymphocytic infiltration. In this review, we summarize the recent advances in the mechanisms and relative targets of CD8+ T cells, and discuss the features and potential applications of CD8+ TRM cells in non-luminal breast cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a9050f4d78074037b77f761a64e95aa5938b134" target='_blank'>
              The potential role of CD8+ cytotoxic T lymphocytes and one branch connected with tissue-resident memory in non-luminal breast cancer
              </a>
            </td>
          <td>
            Ziqi Zhao, Xinyu Ma, Zhengang Cai
          </td>
          <td>2024-07-10</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 How does the ovarian microenvironment, involving neural fibers, vascular networks, and immune cells, interplay with follicular dynamics in the human adult ovary?



 Neural fibers growing into the ovarian cortex and vascular network expansion were associated with primordial follicle activation and growth. Macrophages associated with antral follicular atresia.



 The ovarian follicles, responsible for preserving and supplying oocytes, play a crucial role in fertility, with their fate influenced by the ovarian microenvironment. However, the three-dimensional (3D), irregular, and dynamic structural changes of the ovarian microenvironment are difficult to interpret using two-dimensional analysis. 3D-imaging technologies have been developed to reveal intricate structural and functional relationships particularly in rodents, but these approaches have not been widely adopted for human ovarian research.



 Our study aims to explore the follicular microenvironment dynamics in the human adult ovary. In this work, we optimized whole-mount immunofluorescence using TUBB3, PODXL, PECAM1, ACTA2 and CD68, and conducted 3D imaging on intact adult ovarian tissue, offering a visualization of the 3D structures and dynamic changes within ovarian follicles, neural fibers, vascular networks, and immune cells in human adult ovary at the single-cell level.



 The human ovarian samples were collected from donors (N = 5) who underwent oophorectomy after testosterone treatment. Part of the intact ovary, including cortex and medulla, was fixed and used for whole-mount immunofluorescence using markers for neural fibers (TUBB3), blood vessels (PODXL, PECAM1, ACTA2), and immune cells (CD68). Subsequently, an enhanced iDiSCO method was employed for tissue clearance. Finally, 3D images were acquired using a lightsheet microscope, followed by image analysis and 3D reconstruction using Imaris software.



 Our results showed that TUBB3-labeled nerve fibers predominantly distributed in the ovarian cortex, with few presented in the medullary region. Notably, their presence surrounding secondary and antral follicles was limited. We observed a rich distribution of PODXL-labeled capillaries surrounding primordial and primary follicles in the cortical layer of the adult ovary. As follicles progressed to the secondary stage, PECAM1-labeled vasculature contributed to the construction of a vascular network enveloping the secondary follicles and connecting extensively to larger-caliber arteries containing (ACTA2+) smooth muscle cells. In antral follicles, large larger-caliber arteries were also detected, maintaining proximity to the theca cell layer. Interestingly, in cases of follicular atresia, vascular invasion within the follicular cavity was observed. Additionally, clusters of  CD68+ macrophages exhibited a scattered distribution around the follicles at various stages, with a significant accumulation in atretic small-antral follicles.



 Our study was conducted on ovaries from a limited number of donors after gender-affirming surgery. Although we aimed to provide insights into the ovarian microenvironment, we were so far unable to obtain ovaries with large antral follicles, ovulatory follicles and the formation of the corpus luteum.



 The views provided by 3D-image analysis of the human ovary will augment the current knowledge of the complex interactions between the follicles and the ovarian microenvironment. 3D ovarian mapping will also provide a roadmap for generating oocytes in vitro for fertility preservation and ovarian disease modelling.



 N/A.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e54a1b1980fee1b20f1a25c06eb846532c2e418d" target='_blank'>
              P-693 Three-dimensional modeling of follicular microenvironment dynamics in the human adult ovary
              </a>
            </td>
          <td>
            F. Wei, H. Cheng, X. Fan, J. . S. D. Valle, J. Asseler, T. Stolk, W. . V. Vugt, L. . E. V. D. Meeren, N. . M. V. Mello, S. . M. C. D. S. Lopes
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e729c6639ea5afb16aac5bfb652b95ef0aced008" target='_blank'>
              Single cell transcriptome analysis identified a unique neutrophil type associated with Alzheimer’s disease
              </a>
            </td>
          <td>
            Xiaolin Zhang, Guiqin He, Yixuan Hu, Boren Liu, Yuliang Xu, Xia Li, Xinyou Lv, Jin Li
          </td>
          <td>2024-06-25</td>
          <td>Immunity & Ageing : I & A</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Testicular germ cell tumor (TGCT) is the most common type of tumor in young men. Type II germ cell tumors including postpubertal-type teratomas are derived from the germ cell neoplasia in situ (GCNIS), whereas prepubertal-type teratomas arise independently of the GCNIS. The consomic mouse strain 129.MOLF-Chr19 (M19) is a suitable murine model of such tumors, but its characterization remains incomplete.


OBJECTIVE
Here, we interrogated the suitability of testicular tumors in M19 mice as a model of human TGCT by analyzing their histological features and gene expression signature.


MATERIAL AND METHODS
Testes collected from M19 mice of different ages were categorized by macroscopic appearance based on size and the degree of suspected tumorigenesis. Histological sections from selected tumors were stained with Hematoxylin and Eosin, and expression of genes associated with tumorigenesis was determined in frozen tissue samples from a large range of tumors of different subclasses using RT-qPCR and Fluidigm Dynamic Arrays.


RESULTS
Macroscopically, testicular specimens appeared very heterogeneous concerning size and signs indicating the presence of a tumor. Histological analysis confirmed the development of teratomas with areas of cells corresponding to all three germ cell layers. Gene expression analyses indicated upregulation of markers related to proliferation, vascular invasive potential and pluripotency, and revealed a strong correlation of gene expression with tumor size and a significant intercorrelation of individual genes.


DISCUSSION AND CONCLUSION
TGCT in M19 mice is reminiscent of human testicular teratomas presenting with areas of cells derived from all germ layers and showing a typical gene signature. We thus confirm that these mice can serve as a suitable murine model of pure teratomas for preclinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f25d522aad60939acf94fbdd77bbef077e1e806" target='_blank'>
              Phenotype and gene signature of testicular tumors in 129.MOLF-Chr19 mice resemble human teratomas.
              </a>
            </td>
          <td>
            F. Gayer, Lucas Klaus, Sybille D. Reichardt, A. Fichtner, Holger M Reichardt
          </td>
          <td>2024-07-29</td>
          <td>Andrology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: The standard of care for stage 1 NSCLC is upfront surgery followed by surveillance. However, 20–30% of stage 1 NSCLC recur. There is an unmet need to identify individuals likely to recur who would benefit from frequent monitoring and aggressive cancer treatments. Collagen 1 (Col1) fibers detected by second harmonic generation (SHG) microscopy are a major structural component of the extracellular matrix (ECM) of tumors that play a role in cancer progression. Method: We characterized Col1 fibers with SHG microscopy imaging of surgically resected stage 1 NSCLC. Gene expression from RNA sequencing data was used to validate the SHG microscopy findings. Results: We identified a significant (p ≤ 0.05) increase in the Col1 fiber volume in stage 1 NSCLC that recurred. The increase in Col1 fiber volume was supported by significant increases in the gene expression of Col1 in invasive, compared to noninvasive, lung adenocarcinoma. Significant differences were identified in the gene expression of other ECM proteins, as well as CAFs, immune checkpoint markers, immune cytokines, and T-cell markers. Conclusion: Col1 fiber analysis can provide a companion diagnostic test to evaluate the likelihood of tumor recurrence following stage 1 NSCLC. The studies expand our understanding of the role of the ECM in NSCLC recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44725624f1f075bfc0e7e921ca6f6dac14d40057" target='_blank'>
              Collagen 1 Fiber Volume Predicts for Recurrence of Stage 1 Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            S. Kakkad, B. Krishnamachary, Nadege Fackche, Matthew Garner, Malcom Brock, Peng Huang, Z. Bhujwalla
          </td>
          <td>2024-07-01</td>
          <td>Tomography</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b2d6a96eaf6f8e360e8cc647966e437f298c78" target='_blank'>
              Biological determinants of PSMA expression, regulation and heterogeneity in prostate cancer.
              </a>
            </td>
          <td>
            Martin K. Bakht, H. Beltran
          </td>
          <td>2024-07-08</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Recent advancements in understanding clear cell renal cell carcinoma (ccRCC) have underscored the critical role of the BAP1 gene in its pathogenesis and prognosis. While the von Hippel–Lindau (VHL) mutation has been extensively studied, emerging evidence suggests that mutations in BAP1 and other genes significantly impact patient outcomes. Radiogenomics with and without texture analysis based on CT imaging holds promise in predicting BAP1 mutation status and overall survival outcomes. However, prospective studies with larger cohorts and standardized imaging protocols are needed to validate these findings and translate them into clinical practice effectively, paving the way for personalized treatment strategies in ccRCC. This review aims to summarize the current knowledge on the role of BAP1 mutation in ccRCC pathogenesis and prognosis, as well as the potential of radiogenomics in predicting mutation status and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5336d663e7fd20b2dee53d827284e098335f0a7b" target='_blank'>
              Advancements in Radiogenomics for Clear Cell Renal Cell Carcinoma: Understanding the Impact of BAP1 Mutation
              </a>
            </td>
          <td>
            F. Greco, Valerio D’Andrea, Andrea Buoso, Laura Cea, C. Bernetti, B. Beomonte Zobel, C. Mallio
          </td>
          <td>2024-07-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background CD8+ T cells have been recognized as crucial factors in the prognosis of melanoma. However, there is currently a lack of gene markers that accurately describe their characteristics and functions in acral melanoma (AM), which hinders the development of personalized medicine. Methods Firstly, we explored the composition differences of immune cells in AM using single‐cell RNA sequencing (scRNA‐seq) data and comprehensively characterized the immune microenvironment of AM in terms of composition, developmental differentiation, function, and cell communication. Subsequently, we constructed and validated a prognostic risk scoring model based on differentially expressed genes (DEGs) of CD8+ T cells using the TCGA‐SKCM cohort through Lasso‐Cox method. Lastly, immunofluorescence staining was performed to validate the expression of four genes (ISG20, CCL4, LPAR6, DDIT3) in AM and healthy skin tissues as included in the prognostic model. Results The scRNA‐seq data revealed that memory CD8+ T cells accounted for the highest proportion in the immune microenvironment of AM, reaching 70.5%. Cell–cell communication analysis showed extensive communication relationships among effector CD8+ T cells. Subsequently, we constructed a prognostic scoring model based on DEGs derived from CD8+ T cell sources. Four CD8+ T cell‐related genes were included in the construction and validation of the prognostic model. Additionally, immunofluorescence results demonstrated that ISG20 and CCL4 were downregulated, while LPAR6 and DDIT3 were upregulated in AM tissues compared to normal skin tissues. Conclusion Identifying biomarkers based on the expression levels of CD8+ T cell‐related genes may be an effective approach for establishing prognostic models in AM patients. The independently prognostic risk evaluation model we constructed provides new insights and theoretical support for immunotherapy in AM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c05a7475876a47bf216bccfccce932865e0e239b" target='_blank'>
              Establishment of a CD8+ T cells‐related prognostic risk model for acral melanoma based on single‐cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Wenwen Wang, Pu Liu, Jie Ma, Jun Li, Ling Leng
          </td>
          <td>2024-08-01</td>
          <td>Skin Research and Technology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary We studied canine mammary tumors to better understand similar human breast cancer using a technique called liquid biopsy, which analyzes blood samples to detect disease, focusing on the detection of tiny particles called small extracellular vesicles. These structures are very interesting because they can carry proteins that may indicate the presence of cancer. In this study, we collected blood from healthy dogs, dogs with benign and malignant CMTs, and those in remission and also with recurrence. We found no differences in the size or amount of the vesicles among the groups but identified specific proteins that could serve as markers for cancer. These proteins could potentially help in the diagnosis, prognosis and monitoring of mammary cancer. Abstract (Background). Canine mammary tumors (CMTs) have emerged as an important model for understanding pathophysiological aspects of human disease. Liquid biopsy (LB), which relies on blood-borne biomarkers and offers minimal invasiveness, holds promise for reflecting the disease status of patients. Small extracellular vesicles (SEVs) and their protein cargo have recently gained attention as potential tools for disease screening and monitoring. (Objectives). This study aimed to isolate SEVs from canine patients and analyze their proteomic profile to assess their diagnostic and prognostic potential. (Methods). Plasma samples were collected from female dogs grouped into CMT (malignant and benign), healthy controls, relapse, and remission groups. SEVs were isolated and characterized using ultracentrifugation (UC), nanoparticle tracking analysis (NTA) and transmission electron microscopy (TEM). Proteomic analysis of circulating SEVs was conducted using liquid chromatography–mass spectrometry (LC–MS). (Results). While no significant differences were observed in the concentration and size of exosomes among the studied groups, proteomic profiling revealed important variations. Mass spectrometry identified exclusive proteins that could serve as potential biomarkers for mammary cancer. These included Inter-alpha-trypsin inhibitor heavy chain (ITIH2 and ITI4), phosphopyruvate hydratase or alpha enolase (ENO1), eukaryotic translation elongation factor 2 (eEF2), actin (ACTB), transthyretin (TTR), beta-2-glycoprotein 1 (APOH) and gelsolin (GSN) found in female dogs with malignant tumors. Additionally, vitamin D-binding protein (VDBP), also known as group-specific component (GC), was identified as a protein present during remission. (Conclusions). The results underscore the potential of proteins found in SEVs as valuable biomarkers in CMTs. Despite the lack of differences in vesicle concentration and size between the groups, the analysis of protein content revealed promising markers with potential applications in CMT diagnosis and monitoring. These findings suggest a novel approach in the development of more precise and effective diagnostic tools for this challenging clinical condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9e2b284c1b44e1af7ff555a60e9babfbb59bc7" target='_blank'>
              Exploring Canine Mammary Cancer through Liquid Biopsy: Proteomic Profiling of Small Extracellular Vesicles
              </a>
            </td>
          <td>
            A. A. Novais, G. Tamarindo, Luryan Mikaelly Minotti Melo, Beatriz Castilho Balieiro, Daniela Nóbrega, Gislaine dos Santos, Schaienni Fontoura Saldanha, Fabiana Ferreira de Souza, L. Chuffa, S. Bracha, D. A. Zuccari
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f72da82131133699afff9e553d0c3f1deec5b4" target='_blank'>
              Single-cell transcriptional profiling of clear cell renal cell carcinoma reveals a tumor-associated endothelial tip cell phenotype
              </a>
            </td>
          <td>
            Justina Zvirblyte, J. Nainys, S. Juzenas, Karolis Goda, R. Kubiliute, D. Dasevičius, Marius Kincius, Albertas Ulys, S. Jarmalaitė, L. Mazutis
          </td>
          <td>2024-06-28</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1505c77f8051a5655ab22cbceab5a23bb9f797" target='_blank'>
              Inflammatory breast cancer microenvironment repertoire based on DNA methylation data deconvolution reveals actionable targets to enhance the treatment efficacy
              </a>
            </td>
          <td>
            Naiade Calanca, F. L. C. Faldoni, C. P. Souza, Jeferson Santos Souza, Bianca Elen de Souza Alves, Milena Botelho Pereira Soares, Deysi Viviana Tenazoa Wong, R. C. Lima-Júnior, F. Marchi, C. Rainho, S. Rogatto
          </td>
          <td>2024-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Motivation Understanding the genomic heterogeneity of tumors is an important task in computational oncology, especially in the context of finding personalized treatments based on the genetic profile of each patient’s tumor. Tumor clustering that takes into account the temporal order of genetic events, as represented by tumor mutation trees, is a powerful approach for grouping together patients with genetically and evolutionarily similar tumors and can provide insights into discovering tumor subtypes, for more accurate clinical diagnosis and prognosis. Results Here, we propose oncotree2vec, a method for clustering tumor mutation trees by learning vector representations of mutation trees that capture the different relationships between subclones in an unsupervised manner. Learning low-dimensional tree embeddings facilitates the visualization of relations between trees in large cohorts and can be used for downstream analyses, such as deep learning approaches for single-cell multi-omics data integration. We assessed the performance and the usefulness of our method in three simulation studies and on two real datasets: a cohort of 43 trees from six cancer types with different branching patterns corresponding to different modes of spatial tumor evolution and a cohort of 123 AML mutation trees. Availability and implementation https://github.com/cbg-ethz/oncotree2vec.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79d339874d3f1c60cc3bdf81392bdced5cfc7f01" target='_blank'>
              Oncotree2vec — a method for embedding and clustering of tumor mutation trees
              </a>
            </td>
          <td>
            Monica-Andreea Baciu-Drăgan, N. Beerenwinkel
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinoma (RCC) comprises various histological subtypes, with clear cell RCC (ccRCC) being the most prevalent histotype. RCC with sarcomatoid features (sRCC) is a unique kidney cancer subtype associated with aggressive biological features and poor clinical outcomes that can arise from multiple RCC histologies, most commonly ccRCC. While clinically aggressive, sRCC paradoxically has also demonstrated preferential responsiveness to immune checkpoint blockade (ICB) therapies in subgroup analyses of multiple phase III trials. However, the mediators of this immune sensitivity are largely unknown. We therefore applied transcriptomic techniques to identify orchestrators of immune activity within the sRCC tumor microenvironment (TME). Methods Nephrectomy specimens from patients with sRCC and ccRCC were procured for single cell RNA sequencing (scRNAseq). Clustering and dimensionality reduction were performed, and cell populations were annotated based on expression of canonical lineage markers. Immune populations (CD8+ T cells, CD4+ T cells, B/plasma cells, myeloid cells) were computationally extracted and differential gene expression between sRCC- and ccRCC-derived cells within each subpopulation was performed. Gene expression programs enriched in sRCC samples by scRNAseq were validated on publicly available bulk gene expression data comparing sRCC to ccRCC. Spatial transcriptomics were performed on sRCC tumor sections using the 10X Visium platform. Results Across 18 RCC specimens (10 sRCC; 8 ccRCC), 73,123 cells were analyzed by scRNAseq. Within the CD8+ T cell compartment, CXCL13 was the most significantly enriched nuclear-encoded gene in sRCC samples (log 2-fold change=1.29; q<0.001). CXCL13 was also significantly enriched in CD4+ T cells from sRCC (q<0.001), suggesting enhanced presence of follicular T cells within the sRCC TME. As follicular T cells function in support of B cells, we next interrogated the B lymphocyte population. sRCC samples were enriched for mature B cell and plasma cells, with a five-fold increase in the relative abundance of plasma cells compared to ccRCC samples. Immune deconvolution of patient-derived RNA sequencing from the IMmotion 151 trial revealed a significant increase in the predicted proportion of B lymphocytes (including plasma cells) within the sRCC TME relative to ccRCC (p=0.034). Further, over 20 B lymphocyte activation and maturation pathways were consistently enriched (q<0.25) in sRCC across clinical datasets (Javelin101, CheckMate, and TCGA KIPAN), including Signaling by the B Cell Receptor, Positive Regulation of B Cell Activation, and Immunoglobulin Production Involved in Immunoglobulin Mediated Immune Response, amongst others. B lymphocytes mediate anti-tumor immunity through antibody dependent cellular cytotoxicity (ADCC), and thus the phagocytic effectors of ADCC were interrogated. Myeloid populations differentially expressed FCγR3A in sRCC vs ccRCC (q<0.001), which was recapitulated in bulk gene expression populations (q=0.002, 0.12, and 1.08x10-4 in Javelin101, CheckMate, and TCGA cohorts, respectively). Given the enrichment of follicular T cell and differentiated B lymphocyte programs, we explored the presence of tertiary lymphoid structures (TLS) in sRCC. Two distinct TLS signatures – the 12 Chemokine Score and the TLS Imprint Signature – were significantly enriched in sRCC vs ccRCC across RCC patient datasets (Figure 1). Furthermore, spatial transcriptomics were applied to H&E slides of sRCC to successfully identify the presence of TLS by expression of TLS-associated genes adjacent to sarcomatoid regions. Conclusions TLS, which have previously been associated with response to ICB in RCC (Meylan et al. Immunity. 2022), are transcriptomically enriched for in sRCC, paralleling an observed increase in CXCL13-expressing T cells and differentiated B lymphocytes. Together, TLS and their constituents offer a previously unexplored mediator of immunosurveillance in the sRCC TME that may underlie the paradoxical responsiveness to ICB seen within this population clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d1a1ac8d02565dbe4858b8d2a9dcb1f3479f41" target='_blank'>
              25 Enrichment of tertiary lymphoid structures provides novel insight into mediators of anti-tumor immune activity in sarcomatoid renal cell carcinoma
              </a>
            </td>
          <td>
            Mr Nicholas Salgia, Mr Wilhelm Aubrecht, Kristopher Attwood, Jacky Chow, Kevin H Eng, Jianmin Wang, Bo Xu, Mark Long, Eric C. Kauffman, J. Muhitch
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Cerebral vascular malformations (CCMs) are primarily found within the brain, where they result in increased risk for stroke, seizures, and focal neurological deficits. The unique feature of the brain vasculature is the blood-brain barrier formed by the brain neurovascular unit. Recent studies suggest that loss of CCM genes causes disruptions of blood-brain barrier integrity as the inciting events for CCM development. CCM lesions are proposed to be initially derived from a single clonal expansion of a subset of angiogenic venous capillary endothelial cells (ECs) and respective resident endothelial progenitor cells (EPCs). However, the critical signaling events in the subclass of brain ECs/EPCs for CCM lesion initiation and progression are unclear.


METHODS
Brain EC-specific CCM3-deficient (Pdcd10BECKO) mice were generated by crossing Pdcd10fl/fl mice with Mfsd2a-CreERT2 mice. Single-cell RNA-sequencing analyses were performed by the chromium single-cell platform (10× genomics). Cell clusters were annotated into EC subtypes based on visual inspection and GO analyses. Cerebral vessels were visualized by 2-photon in vivo imaging and tissue immunofluorescence analyses. Regulation of mTOR (mechanistic target of rapamycin) signaling by CCM3 and Cav1 (caveolin-1) was performed by cell biology and biochemical approaches.


RESULTS
Single-cell RNA-sequencing analyses from P10 Pdcd10BECKO mice harboring visible CCM lesions identified upregulated CCM lesion signature and mitotic EC clusters but decreased blood-brain barrier-associated EC clusters. However, a unique EPC cluster with high expression levels of stem cell markers enriched with mTOR signaling was identified from early stages of the P6 Pdcd10BECKO brain. Indeed, mTOR signaling was upregulated in both mouse and human CCM lesions. Genetic deficiency of Raptor (regulatory-associated protein of mTOR), but not of Rictor (rapamycin-insensitive companion of mTOR), prevented CCM lesion formation in the Pdcd10BECKO model. Importantly, the mTORC1 (mTOR complex 1) pharmacological inhibitor rapamycin suppressed EPC proliferation and ameliorated CCM pathogenesis in Pdcd10BECKO mice. Mechanistic studies suggested that Cav1/caveolae increased in CCM3-depleted EPC-mediated intracellular trafficking and complex formation of the mTORC1 signaling proteins.


CONCLUSIONS
CCM3 is critical for maintaining blood-brain barrier integrity and CCM3 loss-induced mTORC1 signaling in brain EPCs initiates and facilitates CCM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a66e8694fff1176e979e3a762243ced5ee4e69e" target='_blank'>
              mTORC1 Signaling in Brain Endothelial Progenitors Contributes to CCM Pathogenesis.
              </a>
            </td>
          <td>
            Wang Min, Lingfeng Qin, Haifeng Zhang, Francesc López-Giráldez, Ning Jiang, Yeaji Kim, Varsha K Mohan, Minhong Su, Katie N Murray, J. Grutzendler, J. H. Zhou
          </td>
          <td>2024-07-03</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="INTRODUCTION
Tertiary lymphoid structures (TLS) arise at chronic inflammatory sites where they function as miniature lymph nodes to generate immune responses, which can be beneficial or detrimental, in diseases as diverse as autoimmunity, chronic infections and cancer. A growing number of studies show that a TLS presence in tumors from cancer patients treated with immune checkpoint inhibitors is closely linked with improved clinical outcomes. TLS may foster the generation of specific anti-tumor immune responses and immunological memory that recognizes a patient's own tumor. Due to repeated rounds of chronic inflammation, some tumor-associated TLS may be immunologically inactive, with immune checkpoint inhibitors functioning to revitalize them through pathway activation.This review summarizes work on TLS and how they mediate immune responses in human tumors. We also explore TLS as potential prognostic and predictive biomarkers for immunotherapy.


EXPERT OPINION
The presence of TLS in human tumors has been linked with a better clinical prognosis, response to treatment(s) and overall survival. TLS provide a structured microenvironment for the activation, expansion and maturation of immune cells at the tumor site. These activities can enhance the efficacy of immunotherapeutic treatments such as checkpoint inhibitors and cancer vaccines by revitalizing local anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefcc1b2935f50bd99b3fd04fc429dffde315a40" target='_blank'>
              Tumor-associated tertiary lymphoid structures in cancer: implications for immunotherapy.
              </a>
            </td>
          <td>
            M. Langouo Fontsa, Francine Padonou, K. Willard-Gallo
          </td>
          <td>2024-07-15</td>
          <td>Expert review of clinical immunology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Osteosarcoma (OS) is an aggressive and highly lethal bone tumor, highlighting the urgent need for further exploration of its underlying mechanisms. In this study, we conducted analyses utilizing bulk transcriptome sequencing data of OS and healthy control samples, as well as single cell sequencing data, obtained from public databases. Initially, we evaluated the differential expression of four tumor microenvironment (TME)-related gene sets between tumor and control groups. Subsequently, unsupervised clustering analysis of tumor tissues identified two significantly distinct clusters. We calculated the differential scores of the four TME-related gene sets for Clusters 1 (C1) and 2 (C2), using Gene Set Variation Analysis (GSVA, followed by single-variable Cox analysis. For the two clusters, we performed survival analysis, examined disparities in clinical-pathological distribution, analyzed immune cell infiltration and immune evasion prediction, assessed differences in immune infiltration abundance, and evaluated drug sensitivity. Differentially expressed genes (DEGs) between the two clusters were subjected to Gene Ontology (GO) and Gene Set Enrichment Analysis (GSEA). We conducted Weighted Gene Co-expression Network Analysis (WGCNA) on the TARGET-OS dataset to identify key genes, followed by GO enrichment analysis. Using LASSO and multiple regression analysis we conducted a prognostic model comprising eleven genes (ALOX5AP, CD37, BIN2, C3AR1, HCLS1, ACSL5, CD209, FCGR2A, CORO1A, CD74, CD163) demonstrating favorable diagnostic efficacy and prognostic potential in both training and validation cohorts. Using the model, we conducted further immune, drug sensitivity and enrichment analysis. We performed dimensionality reduction and annotation of cell subpopulations in single cell sequencing analysis, with expression profiles of relevant genes in each subpopulation analyzed. We further substantiated the role of ACSL5 in OS through a variety of wet lab experiments. Our study provides new insights and theoretical foundations for the prognosis, treatment, and drug development for OS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d966c1117b1e9f9eefc9ae5fd9c28b7f2675ce52" target='_blank'>
              Comprehensive investigation of tumor immune microenvironment and prognostic biomarkers in osteosarcoma through integrated bulk and single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Shao-Bo Shi, Li Zhang, Xiaohua Guo
          </td>
          <td>2024-07-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Summary Phenotype-based drug screening emerges as a powerful approach for identifying compounds that actively interact with cells. Transcriptional and proteomic profiling of cell lines and individual cells provide insights into the cellular state alterations that occur at the molecular level in response to external perturbations, such as drugs or genetic manipulations. In this paper, we propose cycleCDR, a novel deep learning framework to predict cellular response to external perturbations. We leverage the autoencoder to map the unperturbed cellular states to a latent space, in which we postulate the effects of drug perturbations on cellular states follow a linear additive model. Next, we introduce the cycle consistency constraints to ensure that unperturbed cellular state subjected to drug perturbation in the latent space would produces the perturbed cellular state through the decoder. Conversely, removal of perturbations from the perturbed cellular states can restore the unperturbed cellular state. The cycle consistency constraints and linear modeling in the latent space enable to learn transferable representations of external perturbations, so that our model can generalize well to unseen drugs during training stage. We validate our model on four different types of datasets, including bulk transcriptional responses, bulk proteomic responses, and single-cell transcriptional responses to drug/gene perturbations. The experimental results demonstrate that our model consistently outperforms existing state-of-the-art methods, indicating our method is highly versatile and applicable to a wide range of scenarios. Availability and implementation The source code is available at: https://github.com/hliulab/cycleCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1c2b23f3deb29435692d8627c8b4e0b406ec968" target='_blank'>
              Predicting single-cell cellular responses to perturbations using cycle consistency learning
              </a>
            </td>
          <td>
            Wei Huang, Hui Liu
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer, a complex disease with a significant prevalence to form metastases, necessitates novel therapeutic strategies to improve treatment outcomes. Here, we present the results of a comparative molecular study of primary breast tumours, their metastases, and the corresponding primary cell lines using Desorption Electrospray Ionisation (DESI) and Laser-Assisted Rapid Evaporative Ionisation Mass Spectrometry (LA-REIMS) imaging. Our results show that ambient ionisation mass spectrometry technology is suitable for rapid characterisation of samples, providing a lipid- and metabolite-rich spectrum within seconds. Our study demonstrates that the lipidomic fingerprint of the primary tumour is not significantly distinguishable from that of its metastasis, in parallel with the similarity observed between their respective primary cell lines. While significant differences were observed between tumours and the corresponding cell lines, distinct lipidomic signatures and several phospholipids such as PA(36:2), PE(36:1), and PE(P-38:4)/PE(O-38:5) for LA-REIMS imaging and PE(P-38:4)/PE(O-38:5), PS(36:1), and PI(38:4) for DESI-MSI were identified in both tumours and cells. We show that the tumours’ characteristics can be found in the corresponding primary cell lines, offering a promising avenue for assessing tumour responsiveness to therapeutic interventions. A comparative analysis by DESI-MSI and LA-REIMS imaging revealed complementary information, demonstrating the utility of LA-REIMS in the molecular imaging of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c685f0f46b0129a7e331bdb7fd66a1b6f6c2ec" target='_blank'>
              Characterisation of Canine and Feline Breast Tumours, Their Metastases, and Corresponding Primary Cell Lines Using LA-REIMS and DESI-MS Imaging
              </a>
            </td>
          <td>
            Adrienn Molnár, Gabriel Stefan Horkovics-Kováts, Nóra Kucsma, Zsuzsanna Szegő, Boglárka Tauber, Attila Egri, Zoltán Szkupien, Bálint András Deák, J. McKenzie, J. Thuróczy, Richard Schäffer, Gitta Schlosser, G. Szakács, Júlia Balog
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="This study aims to explore the relationship between radiological imaging and genomic characteristics in clear cell renal cell carcinoma (ccRCC), focusing on the expression of adipose differentiation-related protein (ADFP) detected through computed tomography (CT). The goal is to establish a radiogenomic lipid profile and understand its association with tumor characteristics. Data from The Cancer Genome Atlas (TCGA) and the Cancer Imaging Archive (TCIA) were utilized to correlate imaging features with adipose differentiation-related protein (ADFP) expression in ccRCC. CT scans assessed various tumor features, including size, composition, margin, necrosis, and growth pattern, alongside measurements of tumoral Hounsfield units (HU) and abdominal adipose tissue compartments. Statistical analyses compared demographics, clinical–pathological features, adipose tissue quantification, and tumoral HU between groups. Among 197 patients, 22.8% exhibited ADFP expression significantly associated with hydronephrosis. Low-grade ccRCC patients expressing ADFP had higher quantities of visceral and subcutaneous adipose tissue and lower tumoral HU values compared to their high-grade counterparts. Similar trends were observed in low-grade ccRCC patients without ADFP expression. ADFP expression in ccRCC correlates with specific imaging features such as hydronephrosis and altered adipose tissue distribution. Low-grade ccRCC patients with ADFP expression display a distinct lipid metabolic profile, emphasizing the relationship between radiological features, genomic expression, and tumor metabolism. These findings suggest potential for personalized diagnostic and therapeutic strategies targeting tumor lipid metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296ff4689b5854d8f25f6651e5b4a53381b1ffca" target='_blank'>
              The Radiogenomic Landscape of Clear Cell Renal Cell Carcinoma: Insights into Lipid Metabolism through Evaluation of ADFP Expression
              </a>
            </td>
          <td>
            F. Greco, A. Panunzio, C. Bernetti, Alessandro Tafuri, B. Beomonte Zobel, C. Mallio
          </td>
          <td>2024-08-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b832000fd344b4eefe68bc90f1e3f23a719de81" target='_blank'>
              Single-nucleus RNA sequencing reveals glial cell type-specific responses to ischemic stroke in male rodents
              </a>
            </td>
          <td>
            D. Bormann, M. Knoflach, Emilia Poreba, Christian J. Riedl, Giulia Testa, Cyrille Orset, Anthony Levilly, Andréa Cottereau, Philipp Jauk, Simon Hametner, Nadine Stranzl, B. Golabi, D. Copic, K. Klas, M. Direder, Hannes Kühtreiber, Melanie Salek, Stephanie Zur Nedden, Gabriele Baier-Bitterlich, S. Kiechl, Carmen Haider, V. Endmayr, R. Höftberger, H. Ankersmit, M. Mildner
          </td>
          <td>2024-07-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9c7bf2e9973b7ebd586377b2c893dc19b5c3ce" target='_blank'>
              Fast and efficient isolation of murine circulating tumor cells using screencell technology for pre-clinical analyzes
              </a>
            </td>
          <td>
            Fei Ye, Janine Wechsler, Amira Bouzidi, Georges Uzan, S. Naserian
          </td>
          <td>2024-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aging is one of the biggest risk factors for cancer development. Over 85% of all cancers occur in individuals aged over 55 years, often accompanied by age-associated immune defects. Previous studies on the tumor microenvironment (TME) during aging have identified several factors, such as the roles of fibroblasts, immunosuppression, and metastasis. However, the aging-associated defects in anti-tumor immunity, particularly regarding T cells, remain underexplored. Recent findings by Zhivaki and colleagues suggest that age-related immune defects affecting anti-tumor responses involve reduced levels of CD8+ T cells and compromised dendritic cell (DC) functions such as antigen presentation and migration. Their study demonstrates that a hyperactive DC vaccine can restore DC functions in older mice. Furthermore, these hyperactive DCs, characterized by increased IL-1β production and better migratory capability to the lymph node, promote the development of cytolytic CD4+ T cells exhibiting Th1-like phenotypes. This research reveals mechanisms underlying the response to hyperactive DC vaccines in older mice and highlights the critical role of cytolytic CD4+ T cells as substitutes for CD8+ T cells in driving anti-tumor immunity and achieving long-term tumor control in older mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0873254f5c053e156ab3fe19bd0e7a94f7dda772" target='_blank'>
              Hyperactive DCs redirect aged anti-tumor immunity.
              </a>
            </td>
          <td>
            Alex C Y Chen, Debattama R. Sen
          </td>
          <td>2024-07-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c3b0853114104d3c4fec95736d64e395739f71" target='_blank'>
              Single-cell atlas of the human brain vasculature across development, adulthood and disease
              </a>
            </td>
          <td>
            T. Wälchli, Moheb Ghobrial, Marc Schwab, S. Takada, Hang Zhong, Samuel Suntharalingham, Sandra Vetiska, D. Gonzalez, Rui-Qi Wu, H. Rehrauer, Anuroopa Dinesh, Kai Yu, Edward L Y Chen, Jeroen Bisschop, Fiona Farnhammer, Ann Mansur, Joanna Kalucka, I. Tirosh, Luca Regli, K. Schaller, Karl Frei, Troy Ketela, Mark Bernstein, P. Kongkham, Peter Carmeliet, T. Valiante, Peter B Dirks, M. Suvà, Gelareh Zadeh, Viviane Tabar, Ralph Schlapbach, H. Jackson, Katrien De Bock, Jason E Fish, Philippe P Monnier, Gary D. Bader, Ivan Radovanovic
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e4d778cae3d7e3203dfad059efb7f4865240dc4" target='_blank'>
              Insights into treatment-specific prognostic somatic mutations in NSCLC from the AACR NSCLC GENIE BPC cohort analysis
              </a>
            </td>
          <td>
            Yi Liu, Sindhu Yalavarthi, Fan Yang, Yusif Abdul-Rashid, Shenkun Tang, Zihe Long, Yongkai Qin, Kerui Wu, Zhifei Wang
          </td>
          <td>2024-07-02</td>
          <td>BMC Pulmonary Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
The impact of the intratumoral microbiome on immune checkpoint inhibitor (ICI) efficacy in patients with non-small-cell lung cancer (NSCLC) is unknown. Preclinically, intratumoral Escherichia is associated with a proinflammatory tumor microenvironment and decreased metastases. We sought to determine whether intratumoral Escherichia is associated with outcome to ICI in patients with NSCLC.


PATIENTS AND METHODS
We examined the intratumoral microbiome in 958 patients with advanced NSCLC treated with ICI by querying unmapped next-generation sequencing reads against a bacterial genome database. Putative environmental contaminants were filtered using no-template controls (n = 2,378). The impact of intratumoral Escherichia detection on overall survival (OS) was assessed using univariable and multivariable analyses. The findings were further validated in an external independent cohort of 772 patients. Escherichia fluorescence in situ hybridization (FISH) and transcriptomic profiling were performed.


RESULTS
In the discovery cohort, read mapping to intratumoral Escherichia was associated with significantly longer OS (16 v 11 months; hazard ratio, 0.73 [95% CI, 0.59 to 0.92]; P = .0065) in patients treated with single-agent ICI, but not combination chemoimmunotherapy. The association with OS in the single-agent ICI cohort remained statistically significant in multivariable analysis adjusting for prognostic features including PD-L1 expression (P = .023). Analysis of an external validation cohort confirmed the association with improved OS in univariable and multivariable analyses of patients treated with single-agent ICI, and not in patients treated with chemoimmunotherapy. Escherichia localization within tumor cells was supported by coregistration of FISH staining and serial hematoxylin and eosin sections. Transcriptomic analysis correlated Escherichia-positive samples with expression signatures of immune cell infiltration.


CONCLUSION
Read mapping to potential intratumoral Escherichia was associated with survival to single-agent ICI in two independent cohorts of patients with NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f72d8a122524814e51f93f33b1af3c864f0ed52" target='_blank'>
              Intratumoral Escherichia Is Associated With Improved Survival to Single-Agent Immune Checkpoint Inhibition in Patients With Advanced Non-Small-Cell Lung Cancer.
              </a>
            </td>
          <td>
            A. Elkrief, M. Montesion, S. Sivakumar, Caryn Hale, A. Bowman, Ayyüce Begüm Bektaş, M. Bradic, Wenfei Kang, E. Chan, Pooja Gogia, Katia Manova-Todorova, Douglas A Mata, J. Egger, H. Rizvi, N. Socci, Daniel W. Kelly, Eric Rosiek, Fanli Meng, Grittney K. Tam, Ning Fan, A. Drilon, Helena A Yu, Gregory J Riely, N. Rekhtman, Álvaro Quintanal Villalonga, S. Dogan, Umesh Bhanot, M. Gönen, Brian Loomis, M. Hellmann, A. Schoenfeld, Marc Ladanyi, Charles M. Rudin, Chad M Vanderbilt
          </td>
          <td>2024-07-22</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="Kataegis, the focal hypermutation of single base substitutions (SBS) in tumour genomes, has received little attention with respect to prostate cancer (PCa) associated molecular and clinical features. Most notably, data is lacking with regards to this tumour evolutionary phenomenon and PCa racial disparities, with African men disproportionately impacted. Here through comparison between African (n = 109) and non-African (n = 79) whole genome sequenced treatment naïve primary tumours, using a single analytical workflow we assessed for shared and unique features of kataegis. Linking kataegis to aggressive presentation, structural variant burden and copy number loss, we attributed APOBEC3 activity through higher rates of SBS2 to high-risk African tumours. While kataegis positive African patients presented with elevated prostate specific antigen levels, their tumours showed evolutionary unique trajectories marked by increased subclonal and structural variant-independent kataegis. The potential to exacerbate tumour heterogeneity emphases the significance of continued exploration of biological behaviours and environmental exposures for African patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffb66593d089a1d0dd173ec540151b58770a9dc" target='_blank'>
              Kataegis associated mutational processes linked to adverse prostate cancer presentation in African men
              </a>
            </td>
          <td>
            Vanessa Hayes, Jue Jiang, Avraam Tapinos, Ruotian Huang, R. Bornman, Phillip Stricker, S. Mutambirwa, David C Wedge, Weerachai Jaratlerdsiri
          </td>
          <td>2024-06-28</td>
          <td>Research Square</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Identifying biomarkers to predict immune checkpoint inhibitor (ICI) efficacy is warranted. Considering that somatic mutation-derived neoantigens induce strong immune responses, patients with a high tumor mutational burden reportedly tend to respond to ICIs. Therefore, the original function of neoantigenic mutations and their impact on the tumor microenvironment (TME) require attention. RNF43 is a type of RING E3 ubiquitin ligase, and long-term survivors in most cancers had conserved patterns of mutations of RNF43. Also, high microsatellite instability patients had a higher RNF43 mutation rate compared with microsatellite stability tumor patients, who were more sensitive to ICI treatment. Therefore, RNF43 has become a promising biomarker of immunotherapy in a wide range of cancers. This review focuses on the up-to-date knowledge of RNF43 mutation in cancer. We summarize the cancer hallmarks involving activities regulated by RNF43 and highlight its extremely sophisticated regulation of WNT signaling and tumor microenvironment. The key genes interacting with RNF43 have also been summarized and discussed. Additionally, we highlight and propose new strategies of targeting RNF43 and RNF43-based combinations with established immunotherapy and combination therapy. These efforts may provide new perspectives for RNF43-based target therapy in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c419259554c9a116f2d42d506d7f1a73316659d7" target='_blank'>
              RNF43 in cancer: Molecular understanding and clinical significance in immunotherapy.
              </a>
            </td>
          <td>
            Xingfa Huo, Weizhong Han, Zhen Yang, Yongzhi Lu, Ning Liu, H. Hou
          </td>
          <td>2024-08-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor-associated antigens (TAAs) are important targets for cancer vaccines. However, TAA-based vaccines have not yet achieved their full potential in clinical trials. In contrast, immune checkpoint blockade (ICB) has emerged as an effective therapy, leading to durable responses in selected cancer patients. To date, few generalizable associations between TAAs and ICB benefit have been reported, with most studies focusing on melanoma that has the highest mutation rate in cancer. In this study, we developed a TAA burden (TAB) algorithm based on known and putative TAAs and investigated the association of TAB with ICB benefit. Analysis of the IMVigor210 patient cohort of urothelial carcinoma treated with anti-PD-L1 revealed that high tumor mutation burden (TMB) weakened the association of TAB with ICB benefit. Furthermore, TAB correlated with ICB efficacy in tumors characterized by negative PD-L1 staining on immune cells, while high levels of PD-L1 staining on immune cells were linked to T-cell exhaustion. Validation across independent clinical datasets-including urothelial carcinoma cohorts treated with anti-PD1/PD-L1 agents and neoadjuvant anti-PD1 trials for head and neck cancers-corroborated the finding that TAB correlates with ICB benefit in tumors with low T-cell exhaustion. Pan-cancer analyses revealed that in most cancer entities, tumors with higher T-cell exhaustion exhibited lower TAB levels, implying possible immunoediting of TAAs in tumors with established antitumor immunity. Our study challenges the prevailing notion of a lack of association between TAAs and ICB response. It also underscores the need for future investigations into the immunogenicity of TAAs and TAA-based vaccine strategies in tumors with low levels of T-cell exhaustion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b935b3c0dbea7bfa6f10bcf62b6f57eefc92e90" target='_blank'>
              Tumor-Associated Antigen Burden Correlates with Immune Checkpoint Blockade Benefit in Tumors with Low Levels of T-cell Exhaustion.
              </a>
            </td>
          <td>
            Yue Wang, Mengying Hu, Olivera J Finn, Xiao-Song Wang
          </td>
          <td>2024-08-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [25],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>