<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-23 06:32:17 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa5087cd6813583e8724b35bca919eb32b35df3e" target='_blank'>
              Automated spatial omics landscape analysis approach reveals novel tissue architectures in ulcerative colitis
              </a>
            </td>
          <td>
            Derek R Holman, Samuel J S Rubin, Mariusz Ferenc, Elizabeth A Holman, Alexander N Koron, Robel Daniel, Brigid Boland, Garry P Nolan, John T Chang, Stephan Rogalla
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, S. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is heterogeneous with low tumor purity, prominent microenvironment and complex architecture, which preclude identification of shared tumor intrinsic biology within and across patients. We overcame these challenges by applying complementary spatial omics approaches – providing necessary resolution and context – to primary untreated PDAC tumors from 39 patients capturing 341,949 low-bulk and 531,718 single-cell spatial transcriptomes. Of these, 59,403 low-bulk profiles and 205,665 single-cells represented tumor cells. We leveraged this data to build on prior reports of tumor cell subtypes in PDAC. We identify 5 distinct malignant subtypes, excluding ductal-like and intraepithelial neoplasia cells, that span the classical-to-basal spectrum. One tumor subtype is highly proliferative and enriched for the DNA synthesis-condensation-mitosis transcriptional network suggesting this subset disproportionately contribute to tumor growth. Strikingly, the spectrum of tumor cell subtypes was present within all patients, suggesting that subtype-based treatment stratification may need refinement. We find that each subtype has its own distinct regulators and histology. The classical subtype has extensive cell intrinsic regulation, higher cell density and lower extracellular matrix content. In contrast, the basal subtype phenotype results from a combination of cell intrinsic and tumor microenvironment (TME) interactions, wherein the basal subtype has a lower cell density, is surrounded by activated stroma and dense collagen matrix, and is poised for hypoxic adaptation. We derive the composition of cellular neighborhoods providing an explanation for the complex architecture seen in PDAC. Tumor cell neighborhoods stratified based on which side of the classical-to-basal spectrum they fell. For example, cells towards the classical-side of the spectrum had more homogenous tumor neighborhoods that generally lacked intermediate and basal tumor cells. In contrast, cells towards the basal-side of the spectrum were likely to contain classical neighbors. More broadly, neighborhoods along this continuum also contained different proportions of immune and stromal cells, cell densities, collagen deposition, and TME adaptation. These observations support a model where tumor subtypes exist within disparate niches. Lastly, our atlas unifies the catalog of previously reported stromal cells (fibroblasts and stellate cells versus myofibroblastic and inflammatory cancer associated fibroblasts (CAF)) and describes a novel interferon stimulated gene (ISG) resistance signature (ISG.RS) fibroblast. We map the location of these various stromal cells relative to tumor subtypes. For example, myofibroblast CAFs and ISG.RS fibroblasts are enriched near the basal tumor subtype whereas inflammatory CAF abundance increases with distance away from tumor cells. In sum, our atlas provides a lens for stratifying tumor complexity in a cancer cell centric manner and provides a path for testing therapeutic hypothesis in the right cell subtype and context.
 Citation Format: Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara Heij, Mike Mason, Ryan Golhar, Isaac Neuhaus, Enas Abu Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky. In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B081.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566e9b3495ee9764c389c226c9c17e824614540e" target='_blank'>
              Abstract B081: In situ multi-modal characterization of pancreatic ductal adenocarcinoma reveals tumor cell identity as a defining factor of the surrounding microenvironment
              </a>
            </td>
          <td>
            Brian Rabe, Anna Lyubetskaya, Andrew Kavran, Yulong Bai, Andrew Fisher, Hannah Pliner, Alba Font-Tello, Anne Lewin, Ruifeng Hu, Alexandre P. Alloy, Yelena Cheng, Chao Dai, Yunfan Fan, Constance Brett, Todd Brett, Lauren Giampapa, Soeren Stahlschmidt, Fayaz Seifuddin, Steven Vasquez Grinnell, Daniel Carrera, Carlos Rios, Tom Lila, Pradeep Kar, Abhishek Shukla, Rachael Bashford Rogers, Lara R. Heij, Mike Mason, R. Golhar, Isaac Neuhaus, E. A. Shah, Shivan Sivakumar, Jimena Trillo Tinoco, Benjamin J Chen, Konstantinos J Mavrakis, Eugene Drokhlyansky
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases, but their role in tumorigenesis is unknown. We investigated whether and how SMC diversity and plasticity plays a role in tumor angiogenesis and the tumor microenvironment. Methods and Results We use SMC-specific lineage-tracing mouse models and single cell RNA sequencing to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that a significant proportion of SMCs adopt a phenotype traditionally associated with macrophage-like cells. These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Computationally predicted by the ligand-receptor algorithm CellChat, signaling from BST2 on the surface of tumor cells to PIRA2 on SMCs promote this phenotypic transition; in vitro SMC assays demonstrate upregulation of macrophage transcriptional programs, and increased proliferation, migration, and phagocytic ability when exposed to BST2. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and cells that do transition have a comparatively higher inflammatory signal typically associated with anti-tumor effect. Conclusion As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. Further translational work is needed to understand how this phenotypic switch could influence the response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcad9e00e00b0b1e9b4d096ed61477beb04a0767" target='_blank'>
              BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, N. G. Lopez, Wei Feng Ma, Hua Gao, Fudi Wang, Sharika Bamezai, Changhao Fu, Yoko Kojima, Shaunak S. Adkar, Lingfeng Luo, Clint L. Miller, N. Leeper
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignant tumor among women with high heterogeneity. Traditional techniques frequently struggle to comprehensively capture the intricacy and variety of cellular states and interactions within breast cancer. As global precision medicine rapidly advances, single-cell RNA sequencing (scRNA-seq) has become a highly effective technique, revolutionizing breast cancer research by offering unprecedented insights into the cellular heterogeneity and complexity of breast cancer. This cutting-edge technology facilitates the analysis of gene expression profiles at the single-cell level, uncovering diverse cell types and states within the tumor microenvironment. By dissecting the cellular composition and transcriptional signatures of breast cancer cells, scRNA-seq provides new perspectives for understanding the mechanisms behind tumor therapy, drug resistance and metastasis in breast cancer. In this review, we summarized the working principle and workflow of scRNA-seq and emphasized the major applications and discoveries of scRNA-seq in breast cancer research, highlighting its impact on our comprehension of breast cancer biology and its potential for guiding personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d20b5e78664c0194ad04247bdfc15101c7c5f63" target='_blank'>
              Single-Cell RNA-Sequencing: Opening New Horizons for Breast Cancer Research
              </a>
            </td>
          <td>
            Lingyan Xiang, Jie Rao, Jingping Yuan, Ting Xie, Honglin Yan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The relative abundance of cancer-associated fibroblast (CAF) subtypes influences a tumor’s response to treatment, especially immunotherapy. However, the extent to which the underlying tumor composition associates with CAF subtype-specific gene expression is unclear. Here, we describe an interpretable machine learning (ML) approach, additive multiple instance learning (aMIL), to predict bulk gene expression signatures from H&E-stained whole slide images (WSI), focusing on an immunosuppressive LRRC15+ CAF-enriched TGFβ-CAF signature. aMIL models accurately predicted TGFβ-CAF across various cancer types. Tissue regions contributing most highly to slide-level predictions of TGFβ-CAF were evaluated by ML models characterizing spatial distributions of diverse cell and tissue types, stromal subtypes, and nuclear morphology. In breast cancer, regions contributing most to TGFβ-CAF-high predictions (“excitatory”) were localized to cancer stroma with high fibroblast density and mature collagen fibers. Regions contributing most to TGFβ-CAF-low predictions (“inhibitory”) were localized to cancer epithelium and densely inflamed stroma. Fibroblast and lymphocyte nuclear morphology also differed between excitatory and inhibitory regions. Thus, aMIL enables a data-driven link between tissue phenotype and transcription, offering biological interpretability beyond typical black-box models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dfe1e5aad3aff07bf10f8ea7b2dcf18d3178c7" target='_blank'>
              Spatial mapping of immunosuppressive CAF gene signatures in H&E-stained images using additive multiple instance learning
              </a>
            </td>
          <td>
            Miles Markey, Juhyun Kim, Zvi Goldstein, Y. Gerardin, Jacqueline Brosnan-Cashman, Syed Ashar Javed, Dinkar Juyal, Harshith Pagidela, Limin Yu, Bahar Rahsepar, J. Abel, S. Hennek, Archit Khosla, A. Taylor-Weiner, Chintan Parmar
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Analyzing the local microenvironment of tumor cells can provide significant insights into their complex interactions with their cellular surroundings, including immune cells. By quantifying the prevalence and distances of certain immune cells in the vicinity of tumor cells through a neighborhood analysis, patterns may emerge that indicate specific associations between cell populations. Such analyses can reveal important aspects of tumor-immune dynamics, which may inform therapeutic strategies. This method enables an in-depth exploration of spatial interactions among different cell types, which is crucial for research in oncology, immunology, and developmental biology. Results We introduce an R Markdown script called SNAQTM (Single-cell Spatial Neighborhood Analysis and Quantification), which conducts a neighborhood analysis on immunofluorescent images without the need for extensive coding knowledge. As a demonstration, SNAQTM was used to analyze images of pancreatic ductal adenocarcinoma. Samples stained for DAPI, PanCK, CD68, and PD-L1 were segmented and classified using QuPath. The resulting CSV files were exported into RStudio for further analysis and visualization using SNAQTM. Visualizations include plots revealing the cellular composition of neighborhoods around multiple cell types within a customizable radius. Additionally, the analysis includes measuring the distances between cells of certain types relative to others across multiple regions of interest. Availability and implementation The R Markdown files that comprise the SNAQTM algorithm and the input data from this paper are freely available on the web at https://github.com/AryehSilver1/SNAQ. Visual Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/536ae41dfee8df569ec9d770e09c6f0416cb7a5b" target='_blank'>
              Get to know your neighbors with a SNAQ™: A framework for single cell spatial neighborhood analysis in immunohistochemical images
              </a>
            </td>
          <td>
            Aryeh Silver, Avirup Chakraborty, Avinash Pittu, Diana Feier, Miruna Anica, Illeana West, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, 
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite promising results with immune checkpoint blockade (ICB) therapy, outcomes of patients with brain metastasis (BrM) remain poor. Identifying determinant of resistance is crucial but has been hindered by limited access to patient samples and the scarcity of relevant preclinical models. Here, we developed two mouse melanoma BrM models that mirror the disparate response to ICB seen in patients. By characterizing their mutational landscape and performing single-cell phenotypic and transcriptomic analysis, we demonstrated that these models recapitulate both the cellular and molecular features of human disease. Through our comparative analysis, we identified key factors contributing to ICB response. In the responsive model, we found that tumor cells expressed inflammatory programs that polarized microglia toward reactive states, expressing high levels of MHC-I and -II and immunostimulatory molecules, eliciting robust T cell recruitment and activation, which favored anti-tumor immunity. Conversely, the resistant melanoma cells were characterized by neurological molecular signatures and expression of ligands that maintain microglia homeostatic states, resulting in poor T cell infiltration and therapy resistance. We validated our findings in BrM patients, showing that BR1 and BR3 signatures correlated with T cell infiltration and were associated with better and worse patient outcomes, respectively. Our study fills a critical gap by providing clinically relevant models and revealing mechanistic insights into BrM ICB responses, identifying potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b06309c3c54287af3d2a6158bdb011f7654e629" target='_blank'>
              Brain-Metastatic Melanoma Models Uncover Immune and Molecular Correlates of Response to Immunotherapy
              </a>
            </td>
          <td>
            Amélie Daugherty-Lopès, Eva Pérez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabe Needle, Isabella Church, George Kyriakopoulos, Yongmei Zhao, Charli Gruen, Antonella Sassano, Romina E. Araya, Andres Thorkelsson, Cari Smith, M. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, R. Goldszmid
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, Andreas Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cell states are modulated by intrinsic driving forces such as gene expression noise and extrinsic signals from the tissue microenvironment. The distinction between intrinsic and extrinsic cell state determinants is essential for understanding the regulation of cell fate in tissues during development, homeostasis and disease. The rapidly growing availability of single-cell resolution spatial transcriptomics makes it possible to meet this challenge. However, available computational methods to infer topological tissue domains, spatially variable gene expression, or ligand-receptor interactions are limited in capturing cell state changes driven by crosstalk between individual cell types within the same niche. We present NiCo, a computational framework for integrating single-cell resolution spatial transcriptomics with matched single-cell RNA-sequencing reference data to infer the influence of the spatial niche on the cell state. By applying NiCo to mouse embryogenesis, adult small intestine and liver data, we demonstrate the capacity to predict novel niche interactions that govern cell state variation underlying tissue development and homeostasis. In particular, NiCo predicts a feedback mechanism between Kupffer cells and neighboring stellate cells limiting stellate cell activation in the normal liver. NiCo provides a powerful tool to elucidate tissue architecture and to identify drivers of cellular states in local niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330dd175c406437712c3044cd06a601fe89865f" target='_blank'>
              NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis
              </a>
            </td>
          <td>
            Ankit Agrawal, Stefan Thomann, Sukanya Basu, D. Grün
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Identifying the key molecular pathways that enable metastasis by analyzing the eventual metastatic tumor is challenging because the state of the founder subclone likely changes following metastatic colonization. To address this challenge, we labeled primary mouse pancreatic ductal adenocarcinoma (PDAC) subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative in vivo metastatic potential prospectively. We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of IL-1 pathway genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC metastasis in the IL-1 pathway, including the NF-κB targets Fos and Il23a, and beyond the IL-1 pathway including Myo1b and Tmem40. We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies. This work establishes a strategy for linking cancer cell state to future behavior, reveals novel functional regulators of PDAC metastasis, and establishes a method for scoring human carcinomas based on metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130863ca923658f30d397c218a109254f4ad9093" target='_blank'>
              Identifying a gene signature of metastatic potential by linking pre-metastatic state to ultimate metastatic fate
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Weixiang Fang, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb667e3e1fbda176a1eb9bab50b39b93702041c" target='_blank'>
              STdGCN: spatial transcriptomic cell-type deconvolution using graph convolutional networks
              </a>
            </td>
          <td>
            Yawei Li, Yuan Luo
          </td>
          <td>2024-08-05</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor heterogeneity refers to the diversity observed among tumor cells: both between different tumors (inter-tumor heterogeneity) and within a single tumor (intra-tumor heterogeneity). These cells can display distinct morphological and phenotypic characteristics, including variations in cellular morphology, metastatic potential and variability treatment responses among patients. Therefore, a comprehensive understanding of such heterogeneity is necessary for deciphering tumor-specific mechanisms that may be diagnostically and therapeutically valuable. Innovative and multidisciplinary approaches are needed to understand this complex feature. In this context, proteogenomics has been emerging as a significant resource for integrating omics fields such as genomics and proteomics. By combining data obtained from both Next-Generation Sequencing (NGS) technologies and mass spectrometry (MS) analyses, proteogenomics aims to provide a comprehensive view of tumor heterogeneity. This approach reveals molecular alterations and phenotypic features related to tumor subtypes, potentially identifying therapeutic biomarkers. Many achievements have been made; however, despite continuous advances in proteogenomics-based methodologies, several challenges remain: in particular the limitations in sensitivity and specificity and the lack of optimal study models. This review highlights the impact of proteogenomics on characterizing tumor phenotypes, focusing on the critical challenges and current limitations of its use in different clinical and preclinical models for tumor phenotypic characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/958a7a3c1fd552ec9c7dc2019625a3d4361653d4" target='_blank'>
              Phenotyping Tumor Heterogeneity through Proteogenomics: Study Models and Challenges
              </a>
            </td>
          <td>
            Diletta Piana, Federica Iavarone, E. De Paolis, Gennaro Daniele, Federico Parisella, Angelo Minucci, V. Greco, Andrea Urbani
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT The Hodgkin and Reed – Sternberg (HRS) cells in classical Hodgkin Lymphoma (cHL) actively modify the immune tumor microenvironment (TME) attracting immunosuppressive cells and expressing inhibitory molecules. A high frequency of myeloid cells in the TME is correlated with an unfavorable prognosis, but more specific and rare cell populations lack precise markers. Myeloid-derived suppressor cells (MDSCs) have been identified in the peripheral blood of cHL patients, where they appear to be correlated with disease aggressiveness. TNFRSF9 (CD137) is a T cell co-stimulator expressed by monocytic and dendritic cells. Its expression has also been described in HRS cells, where it is thought to play a role in reducing antitumor responses. Here, we perform qualitative and quantitative analyses of lymphocytic and MDSC subtypes and determine the CD137 cell distribution in cHL primary tumors using multiplex immunofluorescence and automated multispectral imaging. The results were correlated with patients’ clinical features. Cells were stained with specific panels of immune checkpoint markers (PD-1, PD-L1, CD137), tumor-infiltrating T lymphocytes (CD3, PD-1), and monocytic cells/MDSCs (CD68, CD14, CD33, Arg-1, CD11b). This approach allowed us to identify distinct phenotypes and to analyze spatial interactions between immune subpopulations and tumor cells. The results confirm CD137 expression by T, monocytic and HRS cells. In addition, the expression of CD137, T exhausted cells, and monocytic MDSCs (m-MDSCs) in the vicinity of malignant HRS cells were associated with a worse prognosis. Our findings reveal new elements of the TME that mediate immune escape, and confirm CD137 as a candidate target for immunotherapy in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51358572d89656eb6e1d345b3d0aa534886ca6c1" target='_blank'>
              Multiplex spatial analysis reveals increased CD137 expression and m-MDSC neighboring tumor cells in refractory classical Hodgkin Lymphoma
              </a>
            </td>
          <td>
            José L. Solórzano, Victoria Menendez Garcia, E. Parra, Luisa Solis, Ruth Salazar, M. García‐Cosío, F. Climent, Sara Fernández, Eva Díaz, A. Francisco-Cruz, Joseph Khoury, Mei Jiang, Auriole Tamegnon, Carlos Montalbán, Ignacio Melero, I. Wistuba, Carlos E. De Andrea, Juan F. García
          </td>
          <td>2024-08-10</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Digital pathology is a rapidly advancing field where deep learning methods can be employed to extract meaningful imaging features. However, the efficacy of training deep learning models is often hindered by the scarcity of annotated pathology images, particularly images with detailed annotations for small image patches or tiles. To overcome this challenge, we propose an innovative approach that leverages paired spatially resolved transcriptomic data to annotate pathology images. We demonstrate the feasibility of this approach and introduce a novel transfer-learning neural network model, STpath (Spatial Transcriptomics and pathology images), designed to predict cell type proportions or classify tumor microenvironments. Our findings reveal that the features from pre-trained deep learning models are associated with cell type identities in pathology image patches. Evaluating STpath using three distinct breast cancer datasets, we observe its promising performance despite the limited training data. STpath excels in samples with variable cell type proportions and high-resolution pathology images. As the influx of spatially resolved transcriptomic data continues, we anticipate ongoing updates to STpath, evolving it into an invaluable AI tool for assisting pathologists in various diagnostic tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c676b24c002770712f317d9415e7742d47b5ad9" target='_blank'>
              Exploit Spatially Resolved Transcriptomic Data to Infer Cellular Features from Pathology Imaging Data
              </a>
            </td>
          <td>
            Zhining Sui, Ziyi Li, Wei Sun
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd111ddc728a7e24176aee9fb44a9a80e5d7261a" target='_blank'>
              Tumor-agnostic cancer therapy using antibodies targeting oncofetal chondroitin sulfate
              </a>
            </td>
          <td>
            E. Vidal-Calvo, Anne Martin-Salazar, Swati Choudhary, R. Dagil, S. Raghavan, Lara Duvnjak, M. A. Nordmaj, T. M. Clausen, Ann Skafte, Jan Oberkofler, Kaituo Wang, Mette Ø. Agerbæk, Caroline Løppke, Amalie M Jørgensen, Daria Ropac, Joana Mujollari, Shona Willis, Agnès Garcias López, Rebecca L. Miller, Richard Karlsson, Felix Goerdeler, Yen-Hsi Chen, Ana R. Colaço, Yong Wang, T. Lavstsen, A. Martowicz, I. Nelepcu, M. Marzban, H. Oo, M. Ørum-Madsen, Yuzhuo Wang, Morten A Nielsen, Henrik Clausen, Michael Wierer, D. Wolf, Ismail Gögenur, T. Theander, Nader Al-Nakouzi, T. Gustavsson, M. Daugaard, A. Salanti
          </td>
          <td>2024-08-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The study presents a novel approach for quantifying cellular interactions in digital pathology using deep learning-based image cytometry. Traditional methods struggle with the diversity and heterogeneity of cells within tissues. To address this, we introduce the Spatial Interaction Potential (SIP) and the Co-Localization Index (CLI), leveraging deep learning classification probabilities. SIP assesses the potential for cell-to-cell interactions, similar to an electric field, while CLI incorporates distances between cells, accounting for dynamic cell movements. Our approach enhances traditional methods, providing a more sophisticated analysis of cellular interactions. We validate SIP and CLI through simulations and apply them to colorectal cancer specimens, demonstrating strong correlations with actual biological data. This innovative method offers significant improvements in understanding cellular interactions and has potential applications in various fields of digital pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949ba712b098fead7bc4bf83be6ad4882df65fb2" target='_blank'>
              Novel Methods for Analyzing Cellular Interactions in Deep Learning-Based Image Cytometry: Spatial Interaction Potential and Co-Localization Index
              </a>
            </td>
          <td>
            Toru Nagasaka, Kimihiro Yamashita, Mitsugu Fujita
          </td>
          <td>2024-08-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1374088281d4f8b66ab8cd65f25ae26c49ad31" target='_blank'>
              Spatial transcriptomics reveals tumor-derived SPP1 induces fibroblast chemotaxis and activation in the hepatocellular carcinoma microenvironment
              </a>
            </td>
          <td>
            W. Tong, Tianze Wang, Yi Bai, Xingpeng Yang, Pinsheng Han, Liuyang Zhu, Yamin Zhang, Zhongyang Shen
          </td>
          <td>2024-09-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Castleman disease (CD) is inflammatory lymphoproliferative disorder of unclear etiology. To determine the cellular and molecular basis of CD, we analyzed the spatial proteome of 4,485,009 single cells, transcriptome of 50,117 single nuclei, immune repertoire of 8187 single nuclei, and pathogenic mutations in Unicentric CD, idiopathic Multicentric CD, HHV8-associated MCD, and reactive lymph nodes. CD was characterized by increased non-lymphoid and stromal cells that formed unique microenvironments where they interacted with lymphoid cells. Interaction of activated follicular dendritic cell (FDC) cytoplasmic meshworks with mantle zone B cells was associated with B cell activation and differentiation. VEGF, IL-6, MAPK, and extracellular matrix pathways were elevated in stromal cells of CD. CXCL13+ FDCs, PDGFRA+ T-zone reticular cells (TRC), and ACTA2-positive perivascular reticular cells (PRC) were identified as the predominant source of increased VEGF expression and IL-6 signaling in CD. VEGF expression by FDCs was associated with peri-follicular neovascularization. FDC, TRC and PRC of CD activated JAK-STAT, TGFβ, and MAPK pathways via ligand-receptor interactions involving collagen, integrins, complement components, and VEGF receptors. T, B and plasma cells were polyclonal but showed class-switched and somatically hypermutated IgG1+ plasma cells consistent with stromal cell-driven germinal center activation. In conclusion, our findings show that stromal cell activation and associated B-cell activation and differentiation, neovascularization and stromal remodeling underlie CD and suggest new targets for treatment. Key points Castleman Disease is characterized by activation and proliferation of CXCL13+ FDCs, PDGFRA+ reticular cells, and ACTA2-positive PRCs. VEGF and IL-6 from lymph node stromal cells are associated with B-cell activation and differentiation, endothelial proliferation, and inflammation in CD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6effec836eaf8e7b0862b8cace8add3a480a246e" target='_blank'>
              Spatial and Single Cell Mapping of Castleman Disease Reveals Key Stromal Cell Types and Cytokine Pathways
              </a>
            </td>
          <td>
            David Smith, Anna Eichinger, Andrew Rech, Julia Wang, E. Esteva, Arta Seyedian, Xiaoxu Yang, Mei Zhang, Dan Martinez, Kai Tan, Minjie Luo, Christopher Y Park, Boris Reizis, Vinodh Pillai
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are a heterogeneous population of cells whose phenotypes and functions are shaped by factors that are incompletely understood. Herein, we asked when and where TAMs arise from blood monocytes and how they evolve during tumor development. We initiated pancreatic ductal adenocarcinoma (PDAC) in inducible monocyte fate-mapping mice and combined single-cell transcriptomics and high-dimensional flow cytometry to profile the monocyte-to-TAM transition. We revealed that monocytes differentiate first into a transient intermediate population of TAMs that generates two longer-lived lineages of terminally differentiated TAMs with distinct gene expression profiles, phenotypes, and intratumoral localization. Transcriptome datasets and tumor samples from patients with PDAC evidenced parallel TAM populations in humans and their prognostic associations. These insights will support the design of new therapeutic strategies targeting TAMs in PDAC. Intermediate tumor-associated macrophages (TAMs) emerge first and give rise to two more differentiated TAM populations in pancreatic cancer. Editor’s summary Tumor-associated macrophages (TAMs) promote tumor growth and immune suppression, but how TAMs develop from circulating monocytes and local tissue-resident macrophages remains incompletely understood. Using a model of pancreatic ductal adenocarcinoma (PDAC) in monocyte fate-mapping mice, Dunsmore et al. examined the dynamics of the monocyte-to-macrophage transition in tumors. A population of “intermediate” TAMs gave rise to two more differentiated TAM subsets, distinguishable by their distinct surface marker phenotype, dependence on the transcription factor Maf, migratory behavior, and localization in tumors. These findings provide a time- and space-resolved picture of TAM heterogeneity and development in PDAC, which could guide the future development of TAM-targeted therapeutic strategies. —Claire Olingy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4a54c86dbe269d24ca8abdd22c4597319660f70" target='_blank'>
              Timing and location dictate monocyte fate and their transition to tumor-associated macrophages
              </a>
            </td>
          <td>
            Garett Dunsmore, Wei Guo, Ziyi Li, D. Bejarano, Rhea Pai, Katharine Yang, Immanuel Kwok, Leonard Tan, M. Ng, Carlos De La Calle Fabregat, A. Yatim, A. Bougouin, Kevin Mulder, Jake Thomas, Javiera Villar, Mathilde Bied, Benoit Kloeckner, C. Dutertre, Grégoire Gessain, S. Chakarov, Zhaoyuan Liu, J. Scoazec, A. Lennon-Duménil, Thomas Marichal, C. Sautès-Fridman, W. Fridman, Ankur Sharma, Bing Su, A. Schlitzer, Lai Guan Ng, Camille Blériot, Florent Ginhoux
          </td>
          <td>2024-07-26</td>
          <td>Science Immunology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76207c3c4f2b4f2f11d67f0daf7aa41a3f2aa07" target='_blank'>
              Dysfunctional tumor-infiltrating Vδ1 + T lymphocytes in microsatellite-stable colorectal cancer
              </a>
            </td>
          <td>
            V. Stary, Ram V Pandey, Julia List, L. Kleissl, Florian Deckert, J. Kabiljo, J. Laengle, Vasileios Gerakopoulos, Rudolf Oehler, Lukas Watzke, Matthias Farlik, S. Lukowski, Anne B Vogt, Georg Stary, Hannes Stockinger, M. Bergmann, Nina Pilat
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1affced3ef3c1684e9cf82520aac1e64770c30b6" target='_blank'>
              aKNNO: single-cell and spatial transcriptomics clustering with an optimized adaptive k-nearest neighbor graph
              </a>
            </td>
          <td>
            Jia Li, Y. Shyr, Qi Liu
          </td>
          <td>2024-08-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Uterine carcinosarcoma (UCS) is a rare gynecologic cancer with unfavorable prognosis. Its biphasic histologic components are known to associated with outcomes. Dissecting the tumor heterogeneity within UCS is crucial to discover molecular biomarkers for patient stratification. The aim of this study was to utilize spatial profiling technology to analyze immune presentation in different histological components within UCS tumors. The study included 23 UCS patients whose tumor sections were subjected to nanoString GeoMx Digital Spatial Profiling (DSP) to profile 30 immune-related proteins in 491 segments. We identified distinct cluster-labeled subtypes showing varied immune marker expressions. The immune presentations within the carcinoma and sarcoma components showed significant differences. Markers indicating M2 macrophages (CD45+CD163+) and epithelial-mesenchymal transition (EMT) (SMA, fibronectin) were highly expressed in the sarcoma components. The immune presentation subtypes within each histological component also differed between patients with recurrence in 5 years and without recurrence over 5 years. The cluster-labeled subtype with high Neutrophil/PMN-MDSC marker expression (CD66b) was associated with longer survival, whereas the subtype with high NK cell marker expressions (CD56) was linked to poor prognosis. Our result suggested that immune features within the carcinoma component of UCS tumors would determine patient outcomes and should be considered as a strategy for patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677b6a5b9abbf69e6cc6fe6fd37eee258299da27" target='_blank'>
              Inter- and intra-tumor heterogeneous immune presentation in primary uterine carcinosarcoma determined by digital spatial profiling
              </a>
            </td>
          <td>
            Ying-Xuan Li, Wei-Chou Lin, Ko-Chen Chen, Wen-Chun Chang, Chin-Jui Wu, Lin-Hung Wei, Ruby Yun-Ju Huang, Bor-Ching Sheu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, S. Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2734d6282129acbb89725ff7878ceaacbfcfa19d" target='_blank'>
              Revealing the crucial roles of suppressive immune microenvironment in cardiac myxoma progression
              </a>
            </td>
          <td>
            Zedong Jiang, Qianlong Kang, Hong Qian, Zhijie Xu, Huan Tong, Jiaqing Yang, Li Li, Renwei Li, Guangqi Li, Fei Chen, Nan Lin, Yunuo Zhao, Huashan Shi, Juan Huang, Xuelei Ma
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Juvenile localized and systemic scleroderma are rare autoimmune diseases which cause significant disability and morbidity in children. The mechanisms driving juvenile scleroderma remain unclear, necessitating further cellular and molecular level studies. The Visium CytAssist spatial transcriptomics (ST) platform, which preserves the spatial location of cells and simultaneously sequences the whole transcriptome, was employed to profile the histopathological slides from skin lesions of juvenile scleroderma patients. (1) Spatial domains were identified from ST data and exhibited strong concordance with the pathologist’s annotations of anatomical structures. (2) The integration of paired ST data and single-cell RNA sequencing (scRNA-seq) from the same patients validated the comparable accuracy of the two platforms and facilitated the estimation of cell type composition in ST data. (3) The pathologist-annotated immune infiltrates, such as perivascular immune infiltrates, were clearly delineated by the ST analysis, underscoring the biological relevance of the findings. This is the first study utilizing spatial transcriptomics to investigate skin lesions in juvenile scleroderma patients. The validity of the ST data was corroborated by gene expression analyses and the pathologist’s assessments. Integration with scRNA-seq data facilitated the cell type-level analysis and validation. Analyses of immune infiltrates through combined ST data and pathological review enhances our understanding of the pathogenesis of juvenile scleroderma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d114ce60551b0d377557f894744cf39236d724d1" target='_blank'>
              Spatial Transcriptomics Identifies Cellular and Molecular Characteristics of Scleroderma Skin Lesions: Pilot Study in Juvenile Scleroderma
              </a>
            </td>
          <td>
            Tianhao Liu, Deren Esencan, Claudia M. Salgado, Chongyue Zhao, Ying-Ju Lai, Theresa Hutchins, Anwesha Sanyal, Wei Chen, Kathryn S. Torok
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We present Cell2Cell, a novel visual analytics approach for quantifying and visualizing networks of cell-cell interactions in three-dimensional (3D) multi-channel cancerous tissue data. By analyzing cellular interactions, biomedical experts can gain a more accurate understanding of the intricate relationships between cancer and immune cells. Recent methods have focused on inferring interaction based on the proximity of cells in low-resolution 2D multi-channel imaging data. By contrast, we analyze cell interactions by quantifying the presence and levels of specific proteins within a tissue sample (protein expressions) extracted from high-resolution 3D multi-channel volume data. Such analyses have a strong exploratory nature and require a tight integration of domain experts in the analysis loop to leverage their deep knowledge. We propose two complementary semi-automated approaches to cope with the increasing size and complexity of the data interactively: On the one hand, we interpret cell-to-cell interactions as edges in a cell graph and analyze the image signal (protein expressions) along those edges, using spatial as well as abstract visualizations. Complementary, we propose a cell-centered approach, enabling scientists to visually analyze polarized distributions of proteins in three dimensions, which also captures neighboring cells with biochemical and cell biological consequences. We evaluate our application in three case studies, where biologists and medical experts use Cell2Cell to investigate tumor micro-environments to identify and quantify T-cell activation in human tissue data. We confirmed that our tool can fully solve the use cases and enables a streamlined and detailed analysis of cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d89e79ae5d1df3c773e5ff1e1e269cd6c8d836f" target='_blank'>
              Cell2Cell: Explorative Cell Interaction Analysis in Multi-Volumetric Tissue Data.
              </a>
            </td>
          <td>
            Eric Morth, Kevin Sidak, Z. Maliga, Torsten Moller, Nils Gehlenborg, P. Sorger, Hanspeter Pfister, Johanna Beyer, Robert Kruger
          </td>
          <td>2024-09-10</td>
          <td>IEEE transactions on visualization and computer graphics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c649f4e5bac69e4b666a1bcdc0297d323411c9" target='_blank'>
              Re-analysis of single cell and spatial transcriptomics data reveals B cell landscape in gastric cancer microenvironment and its potential crosstalk with tumor cells for clinical prognosis
              </a>
            </td>
          <td>
            Xing Cai, Jinru Yang, Yusheng Guo, Yanchao Yu, Chuansheng Zheng, Xiaofang Dai
          </td>
          <td>2024-08-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/134c6a39946a95594365892da7bfa505a4ad41ab" target='_blank'>
              The prognostic effect of infiltrating immune cells is shaped by proximal M2 macrophages in lung adenocarcinoma
              </a>
            </td>
          <td>
            Jian-Rong Li, Vikram R Shaw, Yupei Lin, Xiang Wang, M. Aminu, Yong Li, Jia Wu, Jianjun Zhang, Christopher I. Amos, Chao Cheng
          </td>
          <td>2024-09-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Activating and inhibitory receptors of natural killer (NK) cells such as NKp, NKG2, or CLEC are highly relevant to cold tumors including glioblastoma (GBM). Here, we aimed to characterize the expression of these receptors in GBM to gain insight into their potential role as modulators of the intratumoral microenvironment. Methods We performed a transcriptomic analysis of several NK receptors with a focus on the activating receptor encoded by KLRC2, NKG2C, among bulk and single-cell RNA sequencing GBM data sets. We also evaluated the effects of KLRC2-overexpressing GL261 cells in mice treated with or without programmed cell death protein-1 (PD-1) monoclonal antibody (mAb). Finally, we analyzed samples from two clinical trials evaluating PD-1 mAb effects in patients with GBM to determine the potential of NKG2C to serve as a biomarker of response. Results We observed significant expression of several inhibitory NK receptors on GBM-infiltrating NK and T cells, which contrasts with the strong expression of KLRC2 on tumor cells, mainly at the infiltrative margin. Neoplastic KLRC2 expression was associated with a reduction in the number of myeloid-derived suppressor cells and with a higher level of tumor-resident lymphocytes. A stronger antitumor activity after PD-1 mAb treatment was observed in NKG2Chigh-expressing tumors both in mouse models and patients with GBM whereas the expression of inhibitory NK receptors showed an inverse association. Conclusions This study explored the role of neoplastic NKG2C/KLRC2 expression in shaping the immune profile of GBM and suggests that it is a predictive biomarker for positive responses to immune checkpoint inhibitor treatment in patients with GBM. Future studies could further validate this finding in prospective trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48d3c8dd11805fd767549ac9579d883d6708fa68" target='_blank'>
              NKG2C/KLRC2 tumor cell expression enhances immunotherapeutic efficacy against glioblastoma
              </a>
            </td>
          <td>
            Olaya de Dios, M. A. Ramírez-González, Irene Gómez-Soria, Berta Segura-Collar, Juliana Manosalva, Diego Megías, Carlos E. De Andrea, Leticia Fernández-Rubio, Aurelio Hernández-Laín, J. Sepúlveda-Sánchez, M. Rodriguez-Ruiz, Á. Pérez-Núñez, Derek A Wainwright, R. Gargini, P. Sánchez-Gómez
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Simple Summary Resistance to standard therapies represents a major challenge in melanoma treatment. This is mainly related to the high heterogeneity of this malignancy due to the presence of different types of cells in the tumor mass. Cancer stem cells (CSCs) are a small subset of cells endowed with self-renewal and tumorigenic capacity and the ability to escape anticancer therapies. Melanoma CSCs can be identified by the expression of biomarkers and intracellular pathways deeply involved in their malignant phenotype. Dissecting the molecular mechanisms underlying CSC biological functions might pave the way for novel therapeutic approaches. Herein, we highlight the intricate bidirectional communication between melanoma CSCs and their surrounding cells in the tumor microenvironment and discuss how this interplay might impact the processes of drug resistance and tumor relapse. We also address the potential benefits of CSC-targeted strategies, in combination with conventional drugs, for improving melanoma treatment strategies. Abstract Cutaneous melanoma still represents a significant health burden worldwide, being responsible for the majority of skin cancer deaths. Key advances in therapeutic strategies have significantly improved patient outcomes; however, most patients experience drug resistance and tumor relapse. Cancer stem cells (CSCs) are a small subpopulation of cells in different tumors, including melanoma, endowed with distinctive capacities of self-renewal and differentiation into bulk tumor cells. Melanoma CSCs are characterized by the expression of specific biomarkers and intracellular pathways; moreover, they play a pivotal role in tumor onset, progression and drug resistance. In recent years, great efforts have been made to dissect the molecular mechanisms underlying the protumor activities of melanoma CSCs to provide the basis for novel CSC-targeted therapies. Herein, we highlight the intricate crosstalk between melanoma CSCs and bystander cells in the tumor microenvironment (TME), including immune cells, endothelial cells and cancer-associated fibroblasts (CAFs), and its role in melanoma progression. Specifically, we discuss the peculiar capacities of melanoma CSCs to escape the host immune surveillance, to recruit immunosuppressive cells and to educate immune cells toward an immunosuppressive and protumor phenotype. We also address currently investigated CSC-targeted strategies that could pave the way for new promising therapeutic approaches for melanoma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7340de903da58d80075cbbac264d846e831d74b" target='_blank'>
              Unveiling the Dynamic Interplay between Cancer Stem Cells and the Tumor Microenvironment in Melanoma: Implications for Novel Therapeutic Strategies
              </a>
            </td>
          <td>
            Patrizia Limonta, Raffaella Chiaramonte, Lavinia Casati
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4161d5e91ced9145566548eee008e1ecc6c40ac" target='_blank'>
              Revealing the crosstalk between LOX+ fibroblast and M2 macrophage in gastric cancer by single-cell sequencing
              </a>
            </td>
          <td>
            Dapeng Chen, W. Tong, Bing Ang, Yi Bai, Wenhui Dong, Xiyue Deng, Chunjiong Wang, Yamin Zhang
          </td>
          <td>2024-09-09</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45dc8e7e8f1a911b36ee4955cc44ccd6cbb40670" target='_blank'>
              IGF1R signaling induces epithelial-mesenchymal plasticity via ITGAV in cutaneous carcinoma
              </a>
            </td>
          <td>
            Marta Lopez-Cerda, Laura Lorenzo-Sanz, Victoria da Silva-Diz, Sandra Llop, Rosa M. Penín, Josep Oriol Bermejo, Richard de Goeij-de Haas, S. Piersma, T. Pham, Connie R. Jimenez, J. Martín-Liberal, Purificación Muñoz
          </td>
          <td>2024-07-29</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eadaab3d293ff876f51403f3620a7ad83b9f29" target='_blank'>
              Spatial transcriptomic analysis of primary and metastatic pancreatic cancers highlights tumor microenvironmental heterogeneity.
              </a>
            </td>
          <td>
            A. Khaliq, Meenakshi Rajamohan, Omer Saeed, Kimia Mansouri, Asif Adil, Chi Zhang, Anita Turk, Julienne L. Carstens, Michael House, Sikander Hayat, G. Nagaraju, Sam G Pappas, Y. A. Wang, N. Zyromski, Mateusz Opyrchal, Kelvin P Lee, Heather O'Hagan, Bassel El Rayes, Ashiq Masood
          </td>
          <td>2024-09-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mutations inactivating TET2 are among the most common drivers of clonal hematopoiesis (CH). While TET2 inactivation is associated with monocyte-derived inflammation and improved chimeric antigen-receptor-T cell function, its impact on immunotherapy response is unknown. In our mouse model, hematopoietic Tet2 mutation enhanced immune checkpoint blockade (ICB) response. Enhanced ICB response with Tet2 mutation required phagocytes, CD4 and CD8 T cells. Mechanistically, in Tet2-mutant tumor-infiltrating leukocytes (TILs), ICB preferentially induced anti-tumor states and restricted cell states linked to tumor progression. Tet2-mutant monocytes activated costimulatory programs, while Tet2-mutant T cells showed enhanced T cell memory signatures, lesser exhaustion and decreased regulatory phenotype. Our murine data was clinically relevant, since we found that melanomas from patients with TET2 driver mutation-CH (TET2-CH) showed enhanced immune infiltration, T cell activation, and T cell memory programs. In melanoma patients treated with ICB, TET2-CH was associated with 6-fold greater odds of clinical benefit. Collectively, our data establishes that hematopoietic Tet2 inactivation primes leukocytes for anti-tumor states associated with immunotherapy response and provides a potential biomarker for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3e6160e702c2224ea8ff0222f9be539e313873" target='_blank'>
              Hematopoietic Tet2 inactivation enhances the response to checkpoint blockade immunotherapy
              </a>
            </td>
          <td>
            R. Vanner, Suraj Bansal, Marco M. Buttigeig, Andy G. X. Zeng, Vincent Rondeau, Darryl Y. Chan, Michelle A. Chan-Seng-Yue, Liqing Jin, Jessica Mcleod, E. Donato, Patrick Stelmach, C. Vlasschaert, Yitong Yang, Aarushi Gupta, Sofia Genta, Enrique Sanz Garcia, LI Shlush, Mauricio Ribeiro, Marcus O Butler, Sagi Abelson, Mark Minden, Steven M. Chan, Michael J. Rauh, A. Trumpp, John E. Dick
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The human adrenal gland is a complex endocrine tissue. Studies on adrenal renewal have been limited to animal models or human foetuses. Enhancing our understanding of adult human adrenal homeostasis is crucial for gaining insights into the pathogenesis of adrenal diseases, such as adrenocortical tumours. Here, we present a comprehensive cellular genomics analysis of the adult human normal adrenal gland, combining single‐nuclei RNA sequencing and spatial transcriptome data to reconstruct adrenal gland homeostasis. As expected, we identified primary cells of the various zones of the adrenal cortex and medulla, but we also uncovered additional cell types. They constitute the adrenal microenvironment, including immune cells, mostly composed of a large population of M2 macrophages, and new cell populations, including different subpopulations of vascular‐endothelial cells and cortical‐neuroendocrine cells. Utilizing spatial transcriptome and pseudotime trajectory analysis, we support evidence of the centripetal dynamics of adrenocortical cell maintenance and the essential role played by Wnt/β‐catenin, sonic hedgehog, and fibroblast growth factor pathways in the adult adrenocortical homeostasis. Furthermore, we compared single‐nuclei transcriptional profiles obtained from six healthy adrenal glands and twelve adrenocortical adenomas. This analysis unveiled a notable heterogeneity in cell populations within the adenoma samples. In addition, we identified six distinct adenoma‐specific clusters, each with varying distributions based on steroid profiles and tumour mutational status. Overall, our results provide novel insights into adrenal homeostasis and molecular mechanisms potentially underlying early adrenocortical tumorigenesis and/or autonomous steroid secretion. Our cell atlas represents a powerful resource to investigate other adrenal‐related pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e76e6effd4abc2a2c7085002aaf49ccc7313fb5" target='_blank'>
              Single‐nucleus and spatial transcriptome reveal adrenal homeostasis in normal and tumoural adrenal glands
              </a>
            </td>
          <td>
            B. Altieri, A. K. Secener, Somesh Sai, Cornelius Fischer, S. Sbiera, Panagiota Arampatzi, S. Kircher, Sabine Herterich, Laura-Sophie Landwehr, Sarah N. Vitcetz, C. Braeuning, M. Fassnacht, C. Ronchi, Sascha Sauer, Ali Kerim
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chimeric antigen receptor T cell is important because of its potential to treat various diseases. As deep learning continues to advance, using unsupervised methods to classify medical images has become a significant focus because collecting high-quality labeled data for medical images is labor-intensive and time-consuming. Beyond the need for accurate labeling, there is a desire to explore the underlying characteristics of the data, even when labels may be ambiguous or uncertain. To address these challenges, we present a novel approach that combines image clustering with an insightful explanation of how these clusters are formed. Our method employs a U-net combined with a clustering algorithm to segment the dataset into different groups. After clustering, we use various techniques to interpret and elucidate the results. Moreover, our paper introduces a unique dataset focused on cell data, specifically highlighting the developmental patterns of cancer cells and T cells under various experimental conditions. This dataset offers a rich source of information and presents a complex challenge for image classification due to the diversity of conditions and cell behaviors involved. Our study thoroughly compares different architectural models on this new dataset, demonstrating the superior performance of our proposed architecture. Through experimental analysis and ablation studies, we provide substantial evidence of the benefits offered by our architecture, not only in terms of accuracy but also in its ability to reveal deeper insights into the data. This work advances the field of image classification and opens new possibilities for understanding complex biological processes through computer vision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b2222e0e50090ec7fb0ff692b055ed44f83f60" target='_blank'>
              High-throughput Image-based Clustering of CAR-T/Tumor Cocultures for Rapid and Facile Hit Identification
              </a>
            </td>
          <td>
            Zhi Xu, Xueqi Liu, Kirby Madden-Hennessey, Jordan Urbani, S. K. Nikkhoi, Anusuya Ramasubramanian, Kartika G. Venugopal, Qi Zhao, Eric L. Smith, Yun Fu
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Single-cell Hi-C (scHi-C) has provided unprecedented insights into the heterogeneity of 3D genome organization. However, its sparse and noisy nature poses challenges for computational analyses, such as chromatin architectural feature identification. Here, we introduce scCAFE, a deep learning model for the multi-scale detection of architectural features at the single-cell level. scCAFE provides a unified framework for annotating chromatin loops, TAD-like domains (TLDs), and compartments across individual cells. Our model outperforms previous scHi-C loop calling methods and delivers accurate predictions of TLDs and compartments that are biologically consistent with previous studies. The resulting single-cell annotations also offer a measure to characterize the heterogeneity of different levels of architectural features across cell types. We leverage this heterogeneity and identify a series of marker loop anchors, which demonstrate the potential of the 3D genome data to annotate cell identities without the aid of simultaneously sequenced omics data. Overall, scCAFE not only serves as a useful tool for analyzing single-cell genomic architecture, but also paves the way for precise cell-type annotations solely based on 3D genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9e853dc3e19f05f5603a9a9aa1ba89e02d8ae38" target='_blank'>
              Unveiling multi-scale architectural features in single-cell Hi-C data using scCAFE
              </a>
            </td>
          <td>
            Fuzhou Wang, Jiecong Lin, Hamid Alinejad-Rokny, Wenjing Ma, Lingkuan Meng, Lei Huang, Jixiang Yu, Nanjun Chen, Yuchen Wang, Zhongyu Yao, Weidun Xie, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="
 Pancreatic Ductal Adenocarcinoma (PDAC) is one of the most lethal cancers, with a 5-year survival rate of only 13%. PDAC develops from precursor lesions, most commonly Pancreatic Intraepithelial Neoplasia (PanIN). Studying PanIN in human patients is difficult due to its microscopic nature and the lack of indication for sampling the pancreas in the absence of disease. In collaboration with Gift of Life - Michigan, a local organ and tissue donor organization, we obtained over 80 pancreata for research. Surprisingly, we found PanIN in the vast majority of the organs, regardless of the donor’s age; the prevalence of PanIN compared to the relatively low incidence of PDAC indicates that most lesions are destined not to progress. To gather insight into factors that might distinguish lesions designed to progress from lesions that are likely to remain dormant, we sought to characterize both epithelial cells and their supporting microenvironment. Using spatial transcriptomics, we determined that PanIN epithelial cells are transcriptionaally similar to pancreatic cancer; further, limited genetic characterization revealed that they contain similar oncogenic KRAS mutations as pancreatic malignancy. We then sought to determine whether factors in the microenvironment distinguish sporadic PanIN from those associated with overt malignancy. We used 10x Genomic CytAssist Visium technology and profiled an initial batch of 7 tissue sections from donor pancreata and an equal number from tumors. We employed the BayesPrism algorithm to deconvolute the Visium data (which lacked single-cell resolution), using our single-cell RNAseq atlas to define stroma- and epithelial-specific expression profiles. We then segmented the tissue into epithelial and stromal compartments. We further identified various stromal subpopulations, with macrophages and fibroblasts being the most abundant cell types. We defined subpopulations of epithelial cells, fibroblasts, and macrophages through unsupervised clustering. In both donor samples and tumors, we identified heterogeneous populations of stromal cells present in specific geographic patterns, with PanIN-adjacent stroma clearly distinct from stroma surrounding healthy ducts. Cell-cell neighborhood analysis revealed a distinct geographic arrangement of epithelial and stromal compartments, indicating potential communication patterns. Independent unsupervised statistical analysis using Non-negative Matrix Factorization (NMF) using Coordinated Gene Association in Pattern Sets (CoGAPS) R package also revealed similar patterns. Our next steps involve validating gene expression signatures that define each subpopulation and identifying tumor-specific or PanIN-specific genes. Ultimately, we aim to investigate interactions between epithelial cells, fibroblasts, and macrophages to identify potential tumor-promoting or tumor-restraining pathways.
 Citation Format: Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah Watkoske, Atul Deshpande, Elana Fertig, Arvind Rao, Eileen S. Carpenter, Timothy L. Frankel, Marina Pasca di Magliano. Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A046.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f587649d06619c2f34a308f750ab4e8707c4ec42" target='_blank'>
              Abstract A046: Spatial transcriptomics reveals heterogeneity of epithelial and stromal compartments in healthy and tumor pancreas
              </a>
            </td>
          <td>
            Ahmed M. Elhossiny, Jude Okoye, Imade Williams, Yaqing Zhang, Hannah R Watkoske, Atul Deshpande, E. Fertig, Arvind Rao, Eileen S Carpenter, Timothy L. Frankel, M. Pasca di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="During the development from epithelial neoplasms to invasive carcinoma, uncontrolled growth and proliferation in a confined space cause tumor cells survive in a prolonged overcrowding condition. However, the role of this aberrant physical stimulus induced by overcrowding in the invasiveness acquisition of epithelial cells remains unclear. Here, we first identified overcrowded cancer cells were more abundant at invasive foci in human cancer samples and mouse xenograft models. Combining with mechanical modelling, genetic manipulations and biophysical measurements, we revealed that prolonged overcrowding induces retentive invasiveness in cancer cells by triggering nanocorrugated topographic transition of plasma membrane. Further study demonstrated that the nanocorrugated topographic transition is induced by overcrowding-increased Laplace pressure and disrupts lipid raft-like domains organization to promote tumor invasion. Finally, suppressing nanocorrugated topographical transition of plasma membrane by strengthening membrane-cortex-attachment resists to tumor cell invasion in a mouse model of oncogenesis. This work reveals the profound effect of tissue-scale mechanical revolution-induced biophysical transition of cancer cells in driving tumor invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96309d0c656c868fef684ed435f46b0a78a1517e" target='_blank'>
              Overcrowding Drives Tumor Invasion by Inducing Nanotopographical Transition of Plasma Membrane
              </a>
            </td>
          <td>
            Xinbin Zhao, Min Tan, Long Li, Xubo Lin, Zekun Li, Xiaohuan Wang, Bingqi Song, Zheng Guo, Tailin Chen, Sen Hou, Jiehou Fan, Shijiang Wang, Yun Zhang, Yubo Fan, Jing Du
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The identification of gene regulatory networks (GRN) governing distinct cell fates in multilineage cellular differentiation systems is of critical importance for understanding cell fate decision. Single-cell RNA-sequencing (scRNA-seq) provides a powerful tool for the quantification of gene-level co-variation across the cell state manifold. However, accurate GRN reconstruction is hampered by the sparsity of scRNA-seq data introducing substantial technical noise. Moreover, the high dimensionality of typical scRNA-seq datasets limits the scalability of available approaches. To overcome these challenges, and to facilitate the inference of lineage-specific GRNs with directed regulator-target relations, we introduce NetID. This approach optimizes coverage of the cell state manifold by homogenous metacells and avoids spurious gene-gene correlations observed with available imputation methods. Benchmarking demonstrates superior performance of NetID compared to imputation-based GRN inference. By incorporating cell fate probability information, NetID facilitates prediction of lineage-specific GRNs and recovers known network motifs centered around lineage-determining transcription factors governing bone marrow hematopoiesis, making it a powerful toolkit for deciphering the gene regulatory control of cellular differentiation from large-scale single-cell transcriptome data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/725296ffdf0a7cacc6e78577d9871b12ce8048c7" target='_blank'>
              Scalable identification of lineage-specific gene regulatory networks from metacells with NetID
              </a>
            </td>
          <td>
            Weixu Wang, Yichen Wang, Ruiqi Lyu, D. Grün
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, R. Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We employed laser microdissection to selectively harvest tumor cells and stroma from the microenvironment of formalin-fixed, paraffin-embedded head and neck squamous cell carcinoma (HNSCC) tissues. The captured HNSCC tissue fractions were analyzed by quantitative mass spectrometry-based proteomics using a data independent analysis approach. In paired samples, we achieved excellent proteome coverage having quantified 6,668 proteins with a median quantitative coefficient of variation under 10%. We observed significant differences in relevant functional pathways between the spatially resolved tumor and stroma regions. Our results identified extracellular matrix (ECM) as a major component enriched in the stroma, including many cancer associated fibroblast signature proteins in this compartment. We demonstrate the potential for comparative deep proteome analysis from very low starting input in a scalable format that is useful to decipher the alterations in tumor and the stromal microenvironment. Correlating such results with clinical features or disease progression will likely enable identification of novel targets for disease classification and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81bad8ce1ede8240c16776044c3909a46ef259c" target='_blank'>
              dia-PASEF Proteomics of Tumor and Stroma LMD Enriched from Archived HNSCC Samples
              </a>
            </td>
          <td>
            Aswini Panigrahi, Allison L. Hunt, Diego Assis, Matthew Willetts, Bhaskar V Kallakury, Bruce Davidson, Thomas P. Conrads, R. Goldman
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Pancreatic cancer remains one of the most deadly cancers, with over 450,000 fatalities in 2022 alone. The most lethal variation of this cancer is Pancreatic Ductal Adenocarcinoma (PDAC), with a 5-year survival rate of under 10%. Combination chemotherapy has provided major advances in this disease but it remains challenging for many reasons including the immunosuppressive nature of the tumor microenvironment (TME). PDAC has a highly desmoplastic microenvironment that suppresses antitumor immunity and promotes tumor cell invasion, metastasis, and resistance through epithelial to mesenchymal transition (EMT) of cancer cells. Cancer cells can undergo various forms of cell death, notably apoptosis, necrosis, autophagy, pyroptosis, and ferroptosis. These mechanisms are triggered by different cellular stressors and shape the microenvironment. Through the use of 15 Visium spatial transcriptomics samples from publicly available primary PDAC tumors and paired single-cell RNA (scRNA) data, we aim to understand how cell stress and cell death shape the tumor microenvironment (TME) and epithelial-mesenchymal transition (EMT). By leveraging information from known pathways in literature, we will identify cellular programs related to stress, death, and EMT. We will explore intracellular correlations within single-cell data and extracellular interactions within the TME using spatial data to uncover how different forms of cell death and stress signatures influence the TME and EMT spectrum. The initial phase involves identifying the types of stress that lead to various forms of cell death intracellularly and validating these gene signatures. Subsequently, we will apply these validated signatures to the spatial data to correlate each score and search for significant overlapping programs. This will extend beyond stress and death, incorporating TME signatures corresponding to different cell types and the EMT spectrum. By understanding the signals that promote the death of various cell populations, we aim to identify combination treatments that target both ends of the EMT spectrum. Additionally, this research will provide insights into the inflammatory impact of cell death on the TME. Our approach aims to inform the development of therapies that disrupt resistance mechanisms, ultimately improving treatment outcomes for patients with PDAC. This research can potentially reveal new therapeutic targets and strategies, enhancing our ability to combat this deadly disease.
 Citation Format: Rahul Bansal, Izabella Zamora, Samuel Wright, Nir Hacohen, Arnav Mehta. Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba425dbbc0b9c870572a7ccded079a29b6e67232" target='_blank'>
              Abstract C074: Influence of Cell Death and Stress on the Tumor Microenvironment and EMT in Pancreatic Ductal Adenocarcinoma Through Single Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Rahul Bansal, Izabella Zamora, Samuel J Wright, N. Hacohen, Arnav Mehta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, Ondřej Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Tumor heterogeneity and plasticity, driven by Epithelial-Mesenchymal Transition (EMT), enable cancer therapeutic resistance. We previously showed that EMT promotes primary cilia formation, which enables stemness and tumorigenesis in triple-negative breast cancer (TNBC). Here, we establish a role for primary cilia in human TNBC chemotherapeutic resistance. We developed patient-derived organoids, and showed that these recapitulated the cellular heterogeneity of TNBC biopsies. Notably, one of the identified cell states bore a quasi-mesenchymal phenotype, primary cilia, and stemness signatures. We treated our TNBC organoids with chemotherapeutics and observed partial killing. The surviving cells with organoid-reconstituting capacity showed selective enrichment for the quasi-mesenchymal ciliated cell subpopulation. Genomic analyses argue that this enrichment reflects a combination of pre-existing cells and ones that arose through drug-induced cellular plasticity. We developed a family of small-molecule inhibitors of ciliogenesis and show that these, or genetic ablation of primary cilia, suppress chemoresistance. We conclude that primary cilia help TNBC to evade chemotherapy. Significance Cancer cells that activate EMT to acquire a quasi-mesenchymal state form primary cilia to evade chemotherapy in human triple-negative breast cancer. Pharmacological inhibition of primary ciliogenesis counteracts EMT-induced chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dd368daa4ef108513a9f36d1e4d1835f96618d2" target='_blank'>
              Primary cilia promote EMT-induced triple-negative breast tumor heterogeneity and resistance to therapy
              </a>
            </td>
          <td>
            Camille E. Tessier, J. Derrien, Aurore M M Dupuy, Thomas Pelé, Martin Moquet, Julie Roul, E. Douillard, Camille El Harrif, Xavier Pinson, Matthieu Le Gallo, Florence Godey, Patrick Tas, Roselyne Viel, Claude Prigent, Éric Letouzé, Peggy Suzanne, Patrick Dallemagne, M. Campone, Robert A. Weinberg, Jacqueline A. Lees, Philippe P. Juin, Vincent J Guen
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Background High-grade gliomas including glioblastoma (GBM) and diffuse midline gliomas (DMG) represent the most lethal and aggressive brain cancers where current treatment modalities offer limited efficacy. Chimeric antigen receptor (CAR) T cell therapies have emerged as a promising strategy, boasting tumor-specific targeting and the unique ability to penetrate the blood-brain barrier. However, the effective clinical application hinges on the optimal choice of antigen, with a limited number, currently under investigation. Methods We employed cell surface proteomic analysis of primary human high-grade glioma samples from both adult and pediatric patients. This led to the identification of Ephrin type-A receptor 3 (EphA3) as a prevalently expressed target. We engineered a second-generation EphA3-targeted CAR T cell and assessed function using in vitro and in vivo models of GBM and DMG. Results EphA3-targeted CAR T cells demonstrated robust antigen-specific killing of human GBM and DMG cell lines in vitro. In an orthotopic xenograft NSG mouse model, EphA3-targeted CAR T cells not only effectively eradicated tumors but also established a functional T cell population protective on rechallenge. Remarkably, mice rechallenged with a second contralateral orthotopic tumor implantation achieved complete tumor clearance and maintained a sustained complete response 6 months following initial treatment. Conclusion Building on the proven safety profile of EphA3 antibodies in clinical settings, our study provides compelling preclinical evidence supporting the efficacy of EphA3-targeted CAR T cells against high-grade gliomas. These findings underscore the potential for transitioning this innovative therapy into clinical trials, aiming to revolutionize the treatment landscape for patients afflicted with these formidable brain cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ccdf5eef190aeacc1b356281cbe1f1aeb78930" target='_blank'>
              EphA3-targeted chimeric antigen receptor T cells are effective in glioma and generate curative memory T cell responses
              </a>
            </td>
          <td>
            Leesa Lertsumitkul, Melinda Iliopoulos, Stacie S Wang, Sarah J McArthur, Lisa M Ebert, Alexander J Davenport, R. Endersby, Jordan R Hansford, Kate Drummond, Ryan S Cross, Misty R. Jenkins
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69fde8318f7a677932512a6597cf3b791cc10829" target='_blank'>
              B-cell intrinsic RANK signaling cooperates with TCL1 to induce lineage-dependent B-cell transformation
              </a>
            </td>
          <td>
            Lisa Pfeuffer, Viola Siegert, Julia Frede, Leonie Rieger, Riccardo Trozzo, N. de Andrade Krätzig, Sandra Ring, Shamim Sarhadi, Nicole Beck, Stefan Niedermeier, Mar Abril-Gil, Mohamed Elbahloul, Marianne Remke, Katja Steiger, Ruth Eichner, Julia Jellusova, Roland Rad, Florian Bassermann, Christof Winter, J. Ruland, M. Buchner
          </td>
          <td>2024-08-28</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a typical highly heterogeneous solid tumor with high morbidity and mortality worldwide, especially in China; however, the immune microenvironment of HCC has not been clarified so far. Here, we employed single‐cell RNA sequencing (scRNA‐seq) on diethylnitrosamine (DEN)‐induced mouse HCC model to dissect the immune cell dynamics during tumorigenesis. Our findings reveal distinct immune profiles in both precancerous and cancerous lesions, indicating early tumor‐associated immunological alterations. Notably, specific T and B cell subpopulations are preferentially enriched in the HCC tumor microenvironment (TME). Furthermore, we identified a subpopulation of naïve B cells with high CD83 expression, correlating with improved prognosis in human HCC. These signature genes were validated in The Cancer Genome Atlas HCC RNA‐seq dataset. Moreover, cell interaction analysis revealed that subpopulations of B cells in both mouse and human samples are activated and may potentially contribute to oncogenic processes. In summary, our study provides insights into the dynamic immune microenvironment and cellular networks in HCC pathogenesis, with a specific emphasis on naïve B cells. These findings emphasize the significance of targeting TME in HCC patients to prevent HCC pathological progression, which may give a new perspective on the therapeutics for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76c814af70ac2f24272bcadf22a4dd9e85ffd9ec" target='_blank'>
              Leveraging single‐cell RNA‐seq for uncovering naïve B cells associated with better prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Qingjia Sun, Rui Gao, Yingxin Lin, Xianchao Zhou, Tao Wang, Jian He
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite being the leading cause of childhood mortality, pediatric gliomas have been relatively understudied, and the repurposing of immunotherapies has not been successful. Whole transcriptome sequencing, single-cell sequencing, and sequential multiplex immunofluorescence were used to identify an immunotherapy strategy evaluated in multiple preclinical glioma models. MAPK-driven pediatric gliomas have a higher interferon signature relative to other molecular subgroups. Single-cell sequencing identified an activated and cytotoxic microglia population designated MG-Act in BRAF-fused MAPK-activated pilocytic astrocytoma (PA), but not in high-grade gliomas or normal brain. TIM3 is expressed on MG-Act and on the myeloid cells lining the tumor vasculature but not normal brain. TIM3 expression becomes upregulated on immune cells in the PA microenvironment and anti-TIM3 reprograms ex vivo immune cells from human PAs to a pro-inflammatory cytotoxic phenotype. In a genetically engineered murine model of MAPK-driven low-grade gliomas, anti-TIM3 treatment increased median survival over IgG and anti-PD1 treated mice. ScRNA sequencing data during the therapeutic window of anti-TIM3 demonstrates enrichment of the MG-Act population. The therapeutic activity of anti-TIM3 is abrogated in the CX3CR1 microglia knockout background. These data support the use of anti-TIM3 in clinical trials of pediatric low-grade MAPK-driven gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93c9fb9117641aac2f760a5c311e07768d3cba6" target='_blank'>
              Pediatric glioma immune profiling identifies TIM3 as a therapeutic target in BRAF-Fusion pilocytic astrocytoma.
              </a>
            </td>
          <td>
            Shashwat Tripathi, Hinda Najem, Corey Dussold, Sebastian Pacheco, Ruochen Du, Moloud Sooreshjani, Lisa A Hurley, James P Chandler, R. Stupp, A. Sonabend, C. Horbinski, R. Lukas, Joanne Xiu, Giselle Y López, Theodore Nicolaides, Valerie Brown, N. R. Wadhwani, Sandi K Lam, C. D. James, Ganesh Rao, Maria G Castro, Amy B. Heimberger, Michael DeCuypere
          </td>
          <td>2024-08-13</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Mass spectrometry imaging (MSI) is a powerful technique for label-free spatial mapping of multiple classes of biomolecules in tissue sections. However, differences in desorption and ionization efficiency of different classes of molecules make it challenging to simultaneously map biomolecules at each omics layer in the same tissue sample. Herein, we present a correlative imaging method using nanospray desorption electrospray ionization (nano-DESI) MSI, which enables the spatial mapping of lipids, metabolites, peptides, and proteins with cellular-level spatial resolution in a single tissue section. We demonstrate the molecular profiling of specific cell types and identify truncated peptides in mouse pancreatic tissue. Distinct chemical gradients of peptides and lipids extending from endocrine cells to exocrine cells indicate their different roles in endocrine-exocrine crosstalk and intracellular signaling. The results underscore the power of the developed imaging approach for spatial multi-omics analysis that provides deep insights into cellular diversity and the intricate molecular interactions that occur within heterogenous biological tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89cce4615599d97524d4a853c849900f6428c4f4" target='_blank'>
              Correlative Imaging for Comprehensive Molecular Mapping of Individual Cell Types in Biological Tissues
              </a>
            </td>
          <td>
            Manxi Yang, Mushfeqa Iqfath, Frederick Nguele Meke, Zihan Qu, Emerson Hernly, Pei Su, Zhong-Yin Zhang, Julia Laskin
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Even in its localized stages, pancreatic ductal adenocarcinoma (PDAC) is virtually incurable, characterized by early dissemination and disease recurrence. The liver is the most common site of metastasis, and in most cases hepatic recurrences drive survival outcomes. This underscores the critical need to target and prevent liver metastases. We leveraged a novel biobanking strategy to perform paired single-cell RNA (scRNA-seq) and TCR sequencing (scTCR-seq) to profile multisite immunity in patients with localized PDAC across the tumor, peripheral blood, and premetastatic liver. This analysis of three patients included three tumor, three blood, and nine liver (three per patient from separate segments) samples, and generated 85,748 cells for downstream analysis. We evaluated populations of tumor-expanded T cells as potential indicators of antitumoral immunity, and utilized validated computational approaches to identify tumor-reactive T cells based on key markers of dysfunction and exhaustion. Gene set enrichment analysis was performed on tumor infiltrating lymphocytes (TILs) using signatures specific to tumor-reactive CD4 and CD8 T cells (“NeoTCR4” and “NeoTCR8,” respectively), and scored these using the AUCell package in R. Overall, 153 CD4 and 22 CD8 T cells fit candidate NeoTCR4 and NeoTCR8 signature states, representing 5.6% and 0.8% of the total TIL population respectively. These nominated T cells expressed key marker genes associated with tumor specificity such as CXCL13, TIGIT, PDCD1, ENTPD1, and TOX. Thirteen of these predicted tumor-reactive T cell clones identified within the tumor were also detected within our liver samples, suggestive of a potential antitumoral immune response in the premetastatic liver space. We next performed 10x Xenium in situ spatial transcriptomic analysis of primary tumor, normal adjacent pancreas, and premetastatic liver from two of these patients, using tissue microarrays to incorporate replicate 2mm core biopsies from each of these sites. A custom designed gene panel, guided by our scRNA-seq dataset, targeted tumor-specific mutations (e.g. KRAS G12V, G12D, G12R, and Q61) and immune related genes with a specific focus on markers of T cell activity and exhaustion. Leveraging these latter marker genes, which represent a subset of the NeoTCR4 and NeoTCR8 signatures, we identified a population of exhausted TILs within the tumor microenvironment, representing potentially tumor-reactive T cells whose interactions can now be explored at spatial resolution. Future directions are aimed at experimentally validating the tumor reactivity of the predicted neoantigen-specific TILs within our dataset and characterizing the immune neighborhoods that exist in both the tumor and liver of PDAC patients. This work establishes exciting opportunities for novel T-cell therapy in the treatment of PDAC, and we have already implemented these experimental findings in a forthcoming clinical trial investigating the applicability of this predictive signature score for the generation of TCR-engineered lymphocyte therapy in patients with PDAC.
 Citation Format: Elishama N Kanu, Ashley A Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, Austin M Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, Michael E Lidsky, Sabino Zani, Dan G Blazer III, Peter J Allen, Nicholas D Klemen, Zhicheng Ji, Daniel P Nussbaum, Erika J Crosby. Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb49745088796a4e1e8fb6d2e9a57e189de99728" target='_blank'>
              Abstract B052: Single-cell and spatial analysis of the immune landscape unveils a subset of potentially tumor-reactive T cells in patients with localized PDAC
              </a>
            </td>
          <td>
            Elishama N. Kanu, Ashley A. Fletcher, Sri Krishna, Frank J Lowery, Haotian Zhuang, Ethan S Agritelley, Jiayin Bao, A. Eckhoff, Karrie Comatas, Tao Wang, Bin-Jin Hwang, M. Lidsky, S. Zani, D. Blazer, Peter J. Allen, Nicholas D. Klemen, Zhicheng Ji, Daniel P. Nussbaum, Erika J. Crosby
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Gliomas comprise a diverse spectrum of related tumor subtypes with varying biological and molecular features and clinical outcomes. Advances in detailed genetic and epigenetic characterizations along with an appreciation that subtypes associated with developmental origins, including brain location and patient age, have shifted glioma classification from the historical reliance on histopathological features to updated categories incorporating molecular signatures and spatiotemporal incidence. Within a subtype, individual gliomas show cellular heterogeneity, generally containing subpopulations resembling different types of normal glial and progenitor cells. In addition to tumor-autonomous mechanisms of aberrant growth regulation driven by genetic mutations and signaling between tumor cells, interactions with the tumor microenvironment, including neurons, astrocytes, oligodendrocyte precursor cells, and the immune microenvironment play important roles in driving glioma growth and influencing response to treatment. The emerging understanding of the complex contributions of normal brain to glioma growth represents new opportunities for therapeutic advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4b46271e9cdc90000ffc4d7028f66e1ab4920c" target='_blank'>
              Glial Malignancies.
              </a>
            </td>
          <td>
            Suzanne J Baker, Hui Zong, Michelle Monje
          </td>
          <td>2024-09-16</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. Vanderhyden, Hector C Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Summary Imagine if we could simultaneously predict spatial protein expression in tissues from their routine Hematoxylin and Eosin (H&E) stained images, and create tissue images given protein expression profiles thus enabling virtual simulations of how protein expression alterations impact histology in complex diseases like cancer. Such an approach could lead to more informed diagnostic and therapeutic decisions for precision medicine at lower costs and shorter turnaround times, more detailed insights into underlying disease pathology as well as improvement in predictive and generative performance. In this study, we investigate the intricate correlation between protein expressions obtained from Hyperion mass cytometry and histopathological microstructures in conventional H&E stained glioblastoma (GBM) samples, unveiling morphological patterns and cellular-level spatial alterations associated with protein expression changes. To model these complex relationships, we propose a novel generative-predictive framework called Ouroboros for producing H&E images from protein expressions and simultaneously predicting protein expressions from H&E images. Our comprehensive sample-independent validation over 9920 tissue spots from 4 GBM samples encompassing visual image analysis, quantitative analysis, subspace alignment and perturbation experiments shows that the proposed generative-predictive approach offers significant improvements in predicting protein expression from images in comparison to baseline methods as well as accurate generation of virtual GBM sample images. This proof of concept study can contribute to advancing our understanding of histological responses to protein expression perturbations and lays the foundations for further developments in this area. Availability and implementation Implementation and associated data for the proposed approach are available at the URL: https://github.com/Srijay/Ouroboros.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67ded4a0599c7810f90359134e15ada22ea589a" target='_blank'>
              Ouroboros: cross-linking protein expression perturbations and cancer histology imaging with generative-predictive modeling
              </a>
            </td>
          <td>
            Srijay Deshpande, Sokratia Georgaka, Michael Haley, Robert Sellers, James Minshull, J. Nallala, Martin Fergie, Nicholas Stone, N. Rajpoot, Syed-Murtuza Baker, Mudassar Iqbal, Kevin Couper, Federico Roncaroli, F. Minhas
          </td>
          <td>2024-09-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMNs) are critical precursors to pancreatic ductal adenocarcinoma (PDAC), a highly lethal cancer due to late detection and rapid progression. Using multi-region whole-genome and transcriptome sequencing, we traced the evolution of PDAC from IPMN, constructing detailed phylogenetic trees to provide insights into subclonal architectures and progression pathways. Our analysis identified two distinct evolutionary trajectories: one driven by a single ancestral clone, and another involving multiple independent ancestral clones, potentially influencing the timing and nature of PDAC onset. We further explored the roles of mutational signatures and structural variants (SVs) in promoting clonal evolution. Complementing these genomic findings, our transcriptomic analysis revealed unique gene expression profiles and variations in the immune landscape, correlating with the different progression stages from IPMN to PDAC. These insights reveal the complex molecular dynamics of IPMN progression to PDAC, highlighting the need to refine early detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15f2134ea7c52643fe484a9a257be6fc80940a8" target='_blank'>
              Clonal evolutionary analysis reveals patterns of malignant transformation in pancreatic cancer from Intraductal Papillary Mucinous IPMN Neoplasms (IPMN)
              </a>
            </td>
          <td>
            A. Pea, Xiaotong He, Rosie Upstill-Goddard, C. Luchini, Leonor Patricia Schubert Santana, Stephan B. Dreyer, F. Duthie, N. Jamieson, Colin J. McKay, E. Dickson, Alessandra Pulvirenti, Selma Rebus, Paola Piccoli, Nicola Sperandio, R. Lawlor, M. Milella, F. Froeling, Roberto Salvia, Aldo Scarpa, Andrew V. Bainkin, David C Wedge, David K. Chang
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Tumor heterogeneity often leads to substantial differences in responses to same drug treatment. The presence of pre-existing or acquired drug-resistant cell sub-populations within a tumor survive and proliferate, ultimately resulting in tumor relapse and metastasis. The drug resistance is the leading cause of failure in clinical tumor therapy. Therefore, accurate identification of drug-resistant tumor cell subpopulations could greatly facilitate the precision medicine and novel drug development. However, the scarcity of single-cell drug response data significantly hinders the exploration of tumor cell resistance mechanisms and the development of computational predictive methods. In this paper, we propose scDrugAtlas, a comprehensive database devoted to integrating the drug response data at single-cell level. We manually compiled more than 100 datasets containing single-cell drug responses from various public resources. The current version comprises large-scale single-cell transcriptional profiles and drug response labels from more than 1,000 samples (cell line, mouse, PDX models, patients and bacterium), across 66 unique drugs and 13 major cancer types. Particularly, we assigned a confidence level to each response label based on the tissue source (primary or relapse/metastasis), drug exposure time and drug-induced cell phenotype. We believe scDrugAtlas could greatly facilitate the Bioinformatics community for developing computational models and biologists for identifying drug-resistant tumor cells and underlying molecular mechanism. The scDrugAtlas database is available at: http://drug.hliulab.tech/scDrugAtlas/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c67f110b19aa753b80dffa15ad6e0a2ac3b41aaf" target='_blank'>
              scDrugAtlas: an integrative single-cell drug response atlas for unraveling tumor heterogeneity in therapeutic efficacy
              </a>
            </td>
          <td>
            Wei Huang, Xinda Ren, Yinpu Bai, Hui Liu
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Genetically engineered CD8+ T cells are being explored for the treatment of various cancers. Analytical characterization represents a major challenge in the development of genetically engineered cell therapies, especially assessing the potential off-target editing and product heterogeneity. As conventional sequencing techniques only provide information at the bulk level, they are unable to detect off-target CRISPR translocation or editing events occurring in minor cell subpopulations. In this study, we report the analytical development of a single-cell multi-omics DNA and protein assay to characterize genetically engineered cell products for safety and genotoxicity assessment. We were able to quantify on-target edits, off-target events, and potential translocations at the targeting loci with per-cell granularity, providing important characterization data of the final cell product. Conclusion: A single-cell multi-omics approach provides the resolution required to understand the composition of cellular products and identify critical quality attributes (CQAs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a403ecc7f675cd4ff2179f421e45809aa747f" target='_blank'>
              Assessing a single-cell multi-omic analytic platform to characterize ex vivo-engineered T-cell therapy products
              </a>
            </td>
          <td>
            Maryam Moshref, Jerry Hung-Hao Lo, Andrew McKay, Julien Camperi, Joseph Schroer, Norikiyo Ueno, Shu Wang, Saurabh Gulati, Somayeh Tarighat, Steffen Durinck, Ho Young Lee, Dayue Chen
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction This study aims at elucidating mechanisms of immune infiltration in osteosarcoma (OSA) to identify novel therapeutic strategies. Methods Here, joint analysis of spatial transcriptomics (ST, Visium) data from 26 formalin-fixed paraffin embedded OSA samples at diagnosis (n=9) and relapse (n=17, MAPPYACTS NCT02613962 & OS2006 NCT00470223) from 19 donors (including 5 diagnosis/relapse pairs) was performed using the CellsFromSpace unbiased reference-free signal deconvolution and analysis workflow. This methodology enabled the definition of the topology and the molecular signatures of the cancerous, stromal and immune components across all samples, and allowed for the phenotyping of individual tumors based on cancer subset composition. Spatial colocalization analysis was performed using Lee’s bivariate spatial association measure of different cell populations, enabling the characterization of tumor-infiltrating immune cells found across samples. Similar ST analysis of Patient-derived xenograft (PDX) samples was performed on fresh-frozen samples. Subsequent separation of human and mouse reads using the Xenome algorithm allowed for a clear dissection of the cancer vs. microenvironment compartments. Preliminary signature validation on larger cohorts was performed on bulk RNAseq samples from the OS2006 and MAPPYACTS cohorts. Results Our joint spatial transcriptomics analysis of 26 primary and relapse tumors revealed 64 transcriptomic signatures associated with cancer cells, some of which were shared across multiple samples. These signatures could be consolidated into 18 broad phenotypic categories frequently detectable in various samples, showcasing significant intratumoral heterogeneity within the cancerous compartment. By averaging the composition per sample, we found that samples clustered based on the relative intratumoral abundance of cancer phenotypes, indicating the presence of at least two main differentiation archetypes in OSA: “High intratumoral heterogeneity” and “Undifferentiated.” Additionally, spatial niche analysis of immune cells within tumors identified immune populations consistently associated with tumors across samples. This analysis revealed a specific myeloid lineage capable of infiltrating tumors in all samples, likely mediated by chemotactic signals linked to cancer differentiation, as suggested by ligand-receptor analysis. To validate these findings, PDX models of OSA were analyzed, confirming the presence of a similar phenotype of tumor-infiltrating myeloid cells and underscoring the relevance of PDX models in OSA research. Furthermore, preliminary analyses querying the transcriptomic signature of infiltrating myeloid cells in larger bulk RNAseq cohorts revealed promising associations with patient outcomes, detectable even at diagnosis. Conclusions Our research elucidates patterns of intratumoral phenotypic heterogeneity in OSA and identifies the primary immune cell populations specifically infiltrating OSA tumors. This paves the way for developing novel therapies by leveraging these cellular tropism mechanisms.
 Citation Format: Gaël Moquin-Beaudry, Maria Eugenia Marques Da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, Jean-Yves Scoazec, Birgit Geoerger, Nathalie Gaspar, Antonin Marchais. Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f2ff33e6fea180aea3b91082872e0d960db848" target='_blank'>
              Abstract B033: Characterizing the spatial transcriptomic landscape of Osteosarcoma from diagnosis to relapse
              </a>
            </td>
          <td>
            G. Moquin-Beaudry, Maria Eugénia Marques da Costa, Hanane Zair, Corentin Thuilliez, Pierre Khneisser, Nicolas Signolle, J. Scoazec, Birgit Geoerger, Nathalie Gaspar, A. Marchais
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccca4d1f03908b798585e26a62fe66aed3b57bd1" target='_blank'>
              Cell competition in primary and metastatic colorectal cancer
              </a>
            </td>
          <td>
            M. V. van Luyk, Ana Krotenberg Garcia, Maria Lamprou, Saskia JE Suijkerbuijk
          </td>
          <td>2024-07-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Follicular helper CD4hi T cells (TFH) are a major cellular pool for the maintenance of the HIV reservoir. Therefore, the delineation of the follicular (F)/germinal center (GC) immune landscape will significantly advance our understanding of HIV pathogenesis. We have applied multiplex confocal imaging, in combination with the relevant computational tools, to investigate F/GC in situ immune dynamics in viremic (vir-HIV), antiretroviral-treated (cART HIV) People Living With HIV (PLWH) and compare them to reactive, non-infected controls. Lymph nodes (LNs) from viremic and cART PLWH could be further grouped based on their TFH cell densities in high-TFH and low-TFH subgroups. These subgroups were also characterized by different in situ distributions of PD1hi TFH cells. Furthermore, a significant accumulation of follicular FOXP3hiCD4hi T cells, which were characterized by a low scattering in situ distribution profile and strongly correlated with the cell density of CD8hi T cells, was found in the cART-HIV low-TFH group. An inverse correlation between plasma viral load and LN GrzBhiCD8hi T and CD16hiCD15lo cells was found. Our data reveal the complex GC immune landscaping in HIV infection and suggest that follicular FOXP3hiCD4hi T cells could be negative regulators of TFH cell prevalence in cART-HIV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1706696771c242f4d5c443be3edfe84d2b43c530" target='_blank'>
              Follicular Immune Landscaping Reveals a Distinct Profile of FOXP3hiCD4hi T Cells in Treated Compared to Untreated HIV
              </a>
            </td>
          <td>
            Spiros Georgakis, Michail Orfanakis, Cloé Brenna, Simon Burgermeister, P. M. Del Rio Estrada, Mauricio González-Navarro, Fernanda Torres-Ruiz, G. Reyes-Terán, S. Ávila-Ríos, Y. Luna-Villalobos, Oliver Y Chen, G. Pantaleo, R. Koup, Constantinos Petrovas
          </td>
          <td>2024-08-01</td>
          <td>Vaccines</td>
          <td>0</td>
          <td>127</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Testicular tumors represent the most common malignancy among young men. Nevertheless, the pathogenesis and molecular underpinning of testicular tumors remain largely elusive. We aimed to delineate the intricate intra-tumoral heterogeneity and the network of intercellular communication within the tumor microenvironment. A total of 40,760 single-cell transcriptomes were analyzed, encompassing samples from six individuals with seminomas, two patients with mixed germ cell tumors, one patient with a Leydig cell tumor, and three healthy donors. Five distinct malignant subclusters were identified in the constructed landscape. Among them, malignant 1 and 3 subclusters were associated with a more immunosuppressive state and displayed worse disease-free survival. Further analysis identified that APP-CD74 interactions were significantly strengthened between malignant 1 and 3 subclusters and 14 types of immune subpopulations. In addition, we established an aberrant spermatogenesis trajectory and delineated the global gene alterations of somatic cells in seminoma testes. Sertoli cells were identified as the somatic cell type that differed the most from healthy donors to seminoma testes. Cellular communication between spermatogonial stem cells and Sertoli cells is disturbed in seminoma testes. Our study delineates the intra-tumoral heterogeneity and the tumor immune microenvironment in testicular tumors, offering novel insights for targeted therapy. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd34b54c068c07d9abc9ebf5e69a601ef764ff7" target='_blank'>
              Single-cell transcriptomic analysis reveals that the APP-CD74 axis promotes immunosuppression and progression of testicular tumors.
              </a>
            </td>
          <td>
            Guo Chen, Wei Wang, Xin Wei, Yulin Chen, Liao Peng, Rui Qu, Yi Luo, Shengyin He, Yugao Liu, Jie Du, Ran Lu, Siying Li, Chuangwen Fan, Sujun Chen, Yi Dai, Luo Yang
          </td>
          <td>2024-08-19</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell sequencing technologies have revolutionized biomedical research by enabling deconvolution of cell type-specific properties in highly heterogeneous tissue. While robust tools have been developed to handle bioinformatic challenges posed by single-cell RNA and ATAC data, options for emergent modalities such as methylation are much more limited, impeding the utility of results. Here we present Amethyst, a comprehensive R package for atlas-scale single-cell methylation sequencing data analysis. Amethyst begins with base-level methylation calls and expedites batch integration, doublet detection, dimensionality reduction, clustering, cell type annotation, differentially methylated region calling, and interpretation of results, facilitating rapid data interaction in a local environment. We introduce the workflow using published single-cell methylation human peripheral blood mononuclear cell (PBMC) and human cortex data. We further leverage Amethyst on an atlas-scale brain dataset to describe a noncanonical methylation pattern in human astrocytes and oligodendrocytes, challenging the notion that this form of methylation is principally relevant to neurons in the brain. Tools such as Amethyst will increase accessibility to single-cell methylation data analysis, catalyzing research progress across diverse contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/345bc3f744d769c3b98a5c6e3d9a3dfc3016fecd" target='_blank'>
              Single-cell DNA methylation analysis tool Amethyst reveals distinct noncanonical methylation patterns in human glial cells
              </a>
            </td>
          <td>
            Lauren E. Rylaarsdam, Ruth V. Nichols, Brendan L O'Connell, Stephen Coleman, G. Yardımcı, Andrew C. Adey
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdec74a3009c70bf0c5f23f4d67afa20718ffa77" target='_blank'>
              Single-cell RNA sequencing and cell-cell communication analysis reveal tumor microenvironment associated with chemotherapy responsiveness in ovarian cancer.
              </a>
            </td>
          <td>
            Xiaoyan Jiang, Ningxuan Chen, Qinglv Wei, Xin Luo, Xiaoyi Liu, Lingcui Xie, Ping Yi, Jing Xu
          </td>
          <td>2024-08-09</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Introduction: Pediatric sarcomas are the second most common solid tumor in children and adolescence. Despite advances in multimodal therapies, one-third of sarcoma patients do not survive the disease. The development of new treatments for sarcomas is hindered by a limited understanding of the tumor microenvironment (TME), especially the role of immune cells such as tumor-associated macrophages (TAMs). The abundance of TAMs significantly impacts the clinical outcomes of pediatric sarcoma patients, yet our knowledge of their molecular heterogeneity and interactions with other cell types in the TME remains limited. Methods: To characterize the transcriptomic and epigenomic heterogeneity of sarcoma TME, we profiled 32 treatment-naïve tumor samples from 5 pediatric sarcoma subtypes using the single-nuclei RNA+ATAC-Seq multiome assay. We also interrogated the spatial organization of the sarcoma TME using the multiplexed immunohistochemistry assay, Co-detection by indexing (CODEX), with a focus on the diversity and inter-cellular interactions of TAMs. We utilized in vitro co-culture models of human monocyte-derived macrophages, CD8 T cells and malignant sarcoma cells to validate immunosuppressive signaling interactions between TAMs and CD8 T cells predicted based on our multiome and CODEX data. Results: We sequenced 175,964 and 178,000 high-quality cells from RNA and ATAC modality respectively. Analysis of cell type frequencies across sarcoma subtypes showed that TAMs were the most abundant immune cells in the TME. We identified seven TAM subpopulations with distinct gene expression and epigenomic signatures. Among these, SPP1+ and C1QC+ TAM subsets exhibited higher immunosuppressive potential. By integrating gene expression and chromatin accessibility data, we identified key transcription factors that drive the transcriptional phenotypes of SPP1+/C1QC+ TAMs, including RUNX1 and ELF2. By ligand-receptor analysis using our snRNA-Seq data, we identified signaling ligand-receptor pairs that potentially mediate immunosuppression by SPP1+/C1QC+ TAMs, including SPP1-CD44 and HLA-E-KLRC1. Additionally, interactions like SEMA3D-NRP1 and LTBP3-ITGB5 between malignant cells and SPP1+/C1QC+ TAMs were observed in multiple sarcoma subtypes and significantly associated with inferior prognosis. Furthermore, our CODEX data revealed spatial co-localization between SPP1+TAMs and T cells, further corroborating the immune regulatory role of SPP1+TAMs in the sarcoma TME. Finally, our in vitro experiments revealed that macrophages co-cultured with sarcoma cell lines (RD and RH30) showed increased expression of C1QC/SPP1+ TAM markers, including CD163 and CD109, and demonstrated enhanced immunosuppressive effect on CD8+ T cells. Conclusion: This study presents a comprehensive single-cell multi-omics and spatial omics analysis of the pediatric sarcoma microenvironment. By identifying specific pro-tumorigenic TAM subsets and signaling interactions associated with inferior patient outcome, this study reveals novel insights into the role of TAMs in pediatric sarcomas.
 Citation Format: Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik-Sit, Changya Chen, Anusha Thadi, Chia-hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin Bernt, Vinodh Pillai, Kai Tan. Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B061.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea4e7ffe970f4c90d3fbc5950148fc0b9f42efbd" target='_blank'>
              Abstract B061: Dissecting pediatric sarcoma microenvironment using single-cell and spatial multi-omics
              </a>
            </td>
          <td>
            Zhan Zhang, Kyung Jin Ahn, Rumeysa Biyik‐Sit, Changya Chen, Anusha Thadi, Chia-Hui Chen, William Molina, Brian Lockhart, Theodore Laetsch, Lea Surrey, Malay Haldar, Kathrin M Bernt, Vinodh Pillai, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Multimodal analysis of single-cell samples from healthy and diseased tissues at various stages provides a comprehensive view that identifies disease-specific cells, their molecular features and aids in patient stratification. Here, we present MultiMIL, a novel weakly-supervised multimodal model designed to construct multimodal single-cell references and prioritize phenotype-specific cells via patient classification. MultiMIL effectively integrates single-cell modalities, even when they only partially overlap, providing robust representations for downstream analyses such as phenotypic prediction and cell prioritization. Using a multiple-instance learning approach, MultiMIL aggregates cell-level measurements into sample-level representations and identifies disease-specific cell states through attention-based scoring. We demonstrate that MultiMIL accurately identifies disease-specific cell states in blood and lung samples, identifying novel disease-associated genes and achieving superior patient classification accuracy compared to existing methods. We anticipate MultiMIL will become an essential tool for querying single-cell multiomic atlases, enhancing our understanding of disease mechanisms and informing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8efc9b9ff38309e496bbcf1bd6af1cf5075840" target='_blank'>
              Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases
              </a>
            </td>
          <td>
            Anastasia Litinetskaya, Maiia Shulman, Soroor Hediyeh-zadeh, A. Moinfar, Fabiola Curion, Artur Szałata, Alireza Omidi, M. Lotfollahi, F. Theis
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is the third leading cause of cancer-related death in the US. This poor prognosis is related to the complex tumor microenvironment and tumor heterogeneity that drives resistance to chemotherapy. Here the heterogeneity of the stromal, immune, and malignant epithelial cell populations enables signaling and metabolic crosstalk mechanisms that drive immune suppression and resistance to therapy. Recently, our group unveiled that multiple distinct metabolic subclasses of cancer cells coexist within individual PDAC tumors. Among these, two PDAC clonal populations can engage in metabolic symbiosis that supports mitochondrial metabolism. One subtype has a constitutively high integrated stress response (ISR), enabling resistance to mitochondrial inhibitors. Further, these cells release asparagine that supports the mitochondrial metabolism of the other subtype that are more sensitive to mitochondrial inhibition. Given these observations, we are developing approaches to disrupt this metabolic symbiosis. Clonal PDAC cells with constitutively active ISR exhibit increased sensitivity to inhibition of the redox activity of Redox factor 1/Apurinic-apyrimidinic endonuclease 1 (Ref-1). Importantly, this can be leveraged to disrupt the support of the mitochondrial metabolism of clonal symbiotic pairs. Moreover, polyclonal PDAC cell lines representative of the heterogeneity seen in PDAC patients show a highly synergistic response to the combination of Ref-1 redox inhibitors with mitochondrial inhibitors. These data suggest this treatment combination will be synergistic in polyclonal tumors that exist in most PDAC patients. Overall, these data have the potential to lead to new avenues to directly target heterogenous populations of cancer cells to treat pancreatic cancer- an urgent clinical need.
 Citation Format: Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, Michelle Digman, Aimee Edinger, Claus Jorgensen, Melissa Fishel, Christopher J Halbrook. Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C045.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0412df26641d71ff1a5183fc1b222c74aa39cae6" target='_blank'>
              Abstract C045: Targeting metabolic heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Taryn Morningstar, Cecily Anaraki, Lorenzo Scipioni, Giulia Tedeschi, M. Digman, Aimee Edinger, Claus Jorgensen, M. Fishel, Christopher J. Halbrook
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c56d7dfef26c62111d5f106e161ec720c881a32b" target='_blank'>
              A new exploration: characterization of the differentiation trajectory of prostate cancer cells
              </a>
            </td>
          <td>
            Jiyu Yang, Changyou Wang, Xiao Ma, Jie Li, Haoran Yuan, Renzhen Tan, Ling Ling, Xianchun Zhou
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Adenoid cystic carcinoma (ACC) is a malignant tumor primarily originating from the salivary glands, capable of affecting multiple organs. Although ACC typically exhibits slow growth, it is notorious for its propensity for neural invasion, local recurrence, and distant metastasis, making it a particularly challenging cancer to treat. The complexity of ACC's histological and molecular features poses significant challenges to current treatment modalities, which often show limited effectiveness. Recent advancements in single‐cell RNA‐sequencing (scRNA‐seq) have begun to unravel unprecedented insights into the heterogeneity and subpopulation diversity within ACC, revealing distinct cellular phenotypes and origins. This review delves into the intricate pathological and molecular characteristics of ACC, focusing on recent therapeutic advancements. We particularly emphasize the insights gained from scRNA‐seq studies that shed light on the cellular landscape of ACC, underscoring its heterogeneity and pathobiology. Moreover, by integrating analyses from public databases, this review proposes novel perspectives for advancing treatment strategies in ACC. This review contributes to the academic understanding of ACC by proposing novel therapeutic approaches informed by cutting‐edge molecular insights, paving the way for more effective, personalized therapeutic approaches for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e81e2fac0a605e61e6708a46d6b2502839fd4" target='_blank'>
              Adenoid cystic carcinoma: insights from molecular characterization and therapeutic advances
              </a>
            </td>
          <td>
            Yunxuan Jia, Yupeng Liu, Haitang Yang, Feng Yao
          </td>
          <td>2024-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) that is refractory to treatment frequently leads to extensive systemic and multi-organ metastases. PDAC is characterized as an immune “cold” cancer; however, the persistence of this “cold” tumor immune microenvironment (TIME) across distant metastases, and the mechanisms by which metastases influence their respective TIME, remain poorly understood. In this study, we conducted spatial protein expression profiling of the TIME using the Nanostring GeoMx Digital Spatial Profiling (DSP) platform. We employed a 43- plex panel on 54 tissue samples, including 14 primary tumors and 40 matched metastases from organs commonly affected by PDAC metastases (liver, lung, and peritoneum), obtained through a rapid (“warm”) autopsy program. We quantified protein expression in three cellular compartments: PanCK+ epithelial cells, α-SMA+ fibroblasts, and CD45+ immune cells, across 181 regions of interest (ROIs). Our analysis revealed a significant enrichment of CD45+ immune cells in lung metastases compared to other organ sites, with a notably lower presence of these cells in liver metastases. Focusing on the immune cell compartment, we analyzed the CD45 segments for further investigation. After normalization using housekeeping proteins, including Histone H3, GAPDH, and ribosomal protein S6, we found that the immune profiles were most distinct between liver and lung metastases. Specifically, liver metastases exhibited the greatest deviation in immune profile compared to primary PDAC, lung, and peritoneal metastases. We scored protein expression signatures for various immune cell subsets based on specific marker expressions. When comparing liver to lung metastases, scores for T cells, myeloid cells, checkpoint markers, macrophages, and interferon signaling were significantly lower in the liver, underscoring the immune “coldness” of the liver metastasis environment. Our cohort included six untreated patients and nine patients treated with chemotherapies. We compared protein expressions in each organ site between untreated and treated patients. Notably, treatment induced divergent expression patterns of multiple proteins between liver and lung metastases. While treatment significantly reduced the expression of several immune markers, including CD45 (total immune), CD3 (T cell), CD4 (T helper cell), CD45RO (memory T cell), CD163 (M2 macrophage), and B7-H3 (checkpoint marker), in lung metastases, these markers were significantly upregulated in liver metastases of treated patients. This finding indicates a pronounced divergence in TIME modulation in response to treatment between liver and lung metastases, potentially fostering a treatment-resistant environment in PDAC patients with multi- organ metastasis. Consequently, this urges the need to develop tailored therapeutic strategies for PDAC metastasis, taking into account the distinct immune landscapes of different metastasis sites.
 Citation Format: Jimin Min, Vittorio Branchi, Kimal I Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L Grem, Thomas C Caffrey, Paul M Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra. Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B073.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd4226ecd000fb900607928bb9c305d2b12ea81" target='_blank'>
              Abstract B073: Organ-specific diversity of tumor immune microenvironment in metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Jimin Min, Vittorio Branchi, K. Rajapakshe, Nisha Narayanan, Guangsheng Pei, Linghua Wang, Jean L. Grem, T. Caffrey, P. Grandgenett, Michael A Hollingsworth, Paola A Guerrero, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is characterized by uncontrolled cell growth and spread into surrounding tissue. According to the North American Association of Central Cancer Registries (NAACCR) and the National Center for Health Statistics, there were 1,918,030 cancer cases and a total of 609,360 deaths estimated in the US in 2022. Although the direct cause of cancer is not fully understood, it is recognized that the immune system plays a major role in the development and progression of tumors. Investigating tumor-associated macrophages (TAMs) is crucial for understanding the immunosuppressive role that they may play in the tumor microenvironment, which makes them a target for immunotherapies to combat cancer. However, angiogenesis, tumor initiation and invasion promoting signals prevent gaining deeper insight into the intricate molecular mechanisms involved in TAMs and the tumor microenvironment. While researchers in the past have used population samples to analyze cells, recent efforts have incorporated single-cell technologies to study the complex mixture of cells in a tumor, monitor the effects of immunotherapies and more. With the revolutionary advent of the first single-cell RNA sequencing (scRNA-seq), researchers have since been able to gain insight into single-cell data with unprecedented accuracy and develop new immunotherapies through different single-cell technologies, such as polymethylsiloxane (PDMS) and Multiplexed Ion Beam Imaging (MIBI) multiplexing. In this review, we underline the benefits of adopting single-cell technologies to identify distinct cell types in complex environments (i.e., tumors) and highlight the potential application of these technologies along with various multiplexing techniques to immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f7bdccd7146f521a4c249388cdbf8225adf0ff" target='_blank'>
              A Review on Single-Cell analysis of Macrophages in Cancer
              </a>
            </td>
          <td>
            Isaiah Yim, Amirali Assylkhan
          </td>
          <td>2024-09-17</td>
          <td>STEM Fellowship Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Molecular and cellular characterization of tumors is essential due to the complex and heterogeneous nature of cancer. In recent decades, many bioinformatic tools and experimental techniques have been developed to achieve personalized characterization of tumors. However, sample handling continues to be a major challenge as limitations such as prior treatments before sample acquisition, the amount of tissue obtained, transportation, or the inability to process fresh samples pose a hurdle for experimental strategies that require viable cell suspensions. Here, we present an optimized protocol that allows the recovery of highly viable cell suspensions from breast cancer primary tumor biopsies. Using these cell suspensions we have successfully characterized genome architecture through Hi-C. Also, we have evaluated single-cell gene expression and the tumor cellular microenvironment through single-cell RNAseq. Both technologies are key in the detailed and personalized molecular characterization of tumor samples. The protocol described here is a cost-effective alternative to obtain viable cell suspensions from biopsies simply and efficiently.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6963d9d1f4834d1881e0dfb691c6907687ec39cc" target='_blank'>
              Obtention of viable cell suspensions from breast cancer tumor biopsies for 3D chromatin conformation and single-cell transcriptome analysis
              </a>
            </td>
          <td>
            Aura Stephenson-Gussinye, Luis A. Rendón-Bautista, B. E. Ruiz-Medina, Eduardo Blanco-Olais, Rosario Pérez-Molina, Cleofas Marcial-Medina, Y. Chávarri-Guerra, Enrique Soto-Pérez-de-Celis, Andrea Morales-Alfaro, A. Esquivel-López, Fernando Candanedo-González, Armando Gamboa-Domínguez, Rubén Cortes-González, Alejandro Alfaro-Goldaracena, Sara E. Vázquez-Manjarrez, Guido Grajales-Figueroa, Beatriz Astudillo-Romero, Jesús Ruiz-Manriquez, A. C. Poot-Hernández, Paula Licona-Limón, Mayra Furlan-Magaril
          </td>
          <td>2024-08-22</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The presence of pre-existing or acquired drug-resistant cells within the tumor often leads to tumor relapse and metastasis. Single-cell RNA sequencing (scRNA-seq) enables to elucidate the subtle differences in drug responsiveness among distinct cell subpopulations within tumors. A few methods have employed scRNA-seq data to predict the drug response of individual cells to date, but their performance is far from satisfactory. In this study, we propose SSDA4Drug, a semi-supervised few-shot transfer learning method for inferring drug-resistant cancer cells. SSDA4Drug extracts pharmacogenomic features from both bulk and single-cell transcriptomic data by utilizing semi-supervised adversarial domain adaptation. This allows us to transfer knowledge of drug sensitivity from bulk-level cell lines to single cells. We conduct extensive performance evaluation experiments across multiple independent scRNA-seq datasets, and demonstrate the state-of-the-art performance of SSDA4Drug. Remarkably, with only one or two labeled target-domain samples, SSDA4Drug significantly boosts the predictive performance of single-cell drug responses. Moreover, SSDA4Drug accurately recapitulates the temporally dynamic changes of drug responses during continuous drug exposure of tumor cells, and successfully identifies reversible drug-responsive states in lung cancer cells, which initially acquired resistance through drug exposure but later restore sensitivity induced by drug holiday. Also, our predicted drug responses consistently align with the developmental patterns of drug sensitivity observed along the evolutionary trajectory of oral squamous cell carcinoma cells. In addition, our derived SHAP values and integrated gradients effectively pinpoint the key genes involved in drug resistance in prostate cancer cells. These findings highlight the exceptional performance of our method in determining single-cell drug responses. This powerful tool holds the potential for identifying drug-resistant tumor cell subpopulations, paving the way for strides in precision medicine and novel drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25abb010de95fa69c67d91f158879e6866ff5c92" target='_blank'>
              Identifying drug-resistant individual cells within tumors by semi-supervised domain adaptation
              </a>
            </td>
          <td>
            Kaishun Huang, Hui Liu
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Tumor‐associatedmacrophages (TAMs) exhibit remarkable heterogeneity in glioblastoma. Spatially resolved single‐cell transcriptomic studies identified a monocyte‐derived TAM subset localized in the peri‐necrotic niche, driven by hypoxic cues to acquire ahypoxia response signature. These hypoxia‐TAMs destabilize endothelial adherens junctions through adrenomedullin paracrine signaling, promoting the formation of hyperpermeable neovasculature that impedes drug delivery. Blocking adrenomedullin produced by hypoxia‐TAMs restores vascular integrity, increases drug deliveryinto tumors, and provides combinatorial therapeutic benefits. Here we discuss the heterogeneity of TAMs regarding functional states and locations in glioblastomas, and propose future directions for studying the temporospatial dynamics of multifaceted TAM. Highlights Single‐cell omics reveal a functionally and spatially distinct hypoxia‐TAM subset in glioblastoma. Adrenomedullin secreted by hypoxia‐TAM destabilizes tumor vasculature and its blockade enhances vessel integrity and drug delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fdc1e11579a80529898e9b0f1ec0aa652779e" target='_blank'>
              Identification of hypoxic macrophages in glioblastoma: Unveiling therapeutic insights from tumour microenvironment analysis
              </a>
            </td>
          <td>
            Zhen Qin, Xiu-Wu Bian, Yu Shi
          </td>
          <td>2024-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5b4e38880804cee273ee49510e4f038a8ad036" target='_blank'>
              scPriorGraph: constructing biosemantic cell–cell graphs with prior gene set selection for cell type identification from scRNA-seq data
              </a>
            </td>
          <td>
            Xiyue Cao, Yu-An Huang, Zhu-Hong You, Xuequn Shang, Lun Hu, Peng-Wei Hu, Zhi-an Huang
          </td>
          <td>2024-08-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) patients with tumors enriched for the basal-like molecular subtype exhibit severe resistance to standard of care treatments and have significantly worse overall survival, compared to patients with classical subtype enriched tumors. It is therefore important to develop resources that allow identification of novel targets in a statistically rigorous way to overcome therapeutic challenges. Here we compiled a single cell RNA sequencing (scRNAseq) atlas of the human pancreas with 229 individual patient samples, aggregated from raw data obtained from published studies. We mapped cell-type specific scRNAseq gene signatures in newly generated (n=12) and existing (n=13) spatial transcriptomics (ST) data from PDAC samples. Analysis of tumor cells from our scRNAseq atlas revealed nine transcriptionally distinct populations, two of which strongly align with a basal gene signature, correlating with worse overall survival in PDAC bulk RNAseq datasets. Deconvolution showed one of the basal populations identified to be the predominant tumor subtype in non- dissociated ST tissues and in vitro tumor cell and organoid PDAC lines. In bulk RNAseq datasets, we uncovered that myofibroblasts (myFb) expressing CXCL10 consistently correlated with both basal subtypes and were found near basal tumor cells using immunostaining. We also identified that besides immune cells, ductal cells also express CXCR3, the receptor for CXCL10, further suggesting a relationship between these cell types in PDAC tumor microenvironment (TME). Overall, we show that our newly built scRNAseq atlas, containing over 700,000 cells, integrated with ST data is a powerful resource, given that it corroborates numerous PDAC TME observations in the literature, with increased statistical power. Moreover, we uncovered a novel association between CXCL10+ myFb and basal tumor cells, underscoring the presence of unique spatial niches in PDAC that should be explored for the development of targeted therapies for this dismal disease.
 Citation Format: Ian M Loveless, Samantha B Kempt, Yuesong Wu, Daniel J Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel D Zwernik, Kendyll Gartrelle, Rohit G Reddy, Daniel Long, Allison Wombell, Julie M Clark, Albert M Levin, David Kwon, Ling Huang, Ralph Francescone, Débora B Vendramini-Costa, Ben Stanger, Adam Alessio, Andrew M Waters, Yuehua Cui, Brian Theisen, Howard C Crawford, Nina G Steele. Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb2cdcff90781c8782f8a20266c95fd99d51d7f6" target='_blank'>
              Abstract B068: Human pancreatic cancer single cell atlas reveals association of CXCL10 positive fibroblasts and basal subtype tumor cells
              </a>
            </td>
          <td>
            Ian Loveless, Samantha B Kempt, Yuesong Wu, Daniel J. Salas-Escabillas, Kailee Hartway, Madison George, Yetunde Makinwa, Thais Stockdale, Samuel Zwernik, K. Gartrelle, Rohit G Reddy, Daniel W Long, Allison Wombell, Julie Clark, Albert Levin, David Kwon, Ling Huang, R. Francescone, D. Vendramini-Costa, Ben Z. Stanger, Adam Alessio, Andrew M. Waters, Yuehua Cui, Brian Theisen, Howard C. Crawford, Nina G. Steele
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="
 Introduction: A complex immunosuppressive tumor microenvironment (TME) is a distinctive feature of pancreatic ductal adenocarcinoma (PDAC). The TME consists of an admixture of cell types including epithelial, endothelial, immune cells of both myeloid and lymphoid origin, and cancer-associated fibroblasts (CAFs). The traditionally defined subtypes of PDAC tumor cells include classical, associated with the retention of epithelial markers and a better prognosis, and basal, which is characterized by a quasi-mesenchymal phenotype with poorer responses to chemotherapy and worse overall survival. Additionally, CAF subtypes are associated with specific roles within the TME. These include inflammatory (iCAF), with the capacity to secrete IL-6, myofibroblastic (myCAF), defined by high levels of alpha smooth muscle actin (αSMA) expression, and antigen presenting (apCAF), defined by expression of HLA-DR and antigen processing and presentation capacity. We hypothesized that interactions between CAFs and tumor cells altered tumor cell phenotype and could be delineated. Methods: Using multi-omics platforms, we evaluated the impact of CAF populations on tumor cell phenotype. Our approach integrated multiplex immunohistochemistry, imaging mass cytometry (IMC) and RNA-scope of patient tumor tissues with functional in vitro studies from matched patient-derived organoid (PDO) and CAF cocultures. The dynamic impact of CAFs on PDOs was examined using a combination of bulk RNA sequencing, multiparametric flow cytometry and secretome analysis, in addition to RT-qPCR and western blot comparing cocultures to matched PDO monocultures. Results: IMC of 15 patient tumors revealed distinct neighborhoods of CAFs and tumor cells. iCAFs were in close proximity to classical tumor cells, while myCAFs were in close proximity to basal tumor cells. Further, in our PDO-CAF co-cultures the PDOs underwent a shift from classical to basal subtype in 42% of co-cultures, a phenomenon also inferred in patient tissues by associating collagen content with classical or basal designation. We also captured a tumor cell population expressing both basal and classical markers that may represent a transition state. The shift from classical to basal tumor cell subtype was accompanied by enhanced epithelial-mesenchymal transition and a loss of IL-8 secretion. Interestingly, when we examined immune cell distribution within tumor tissues, T cells were both the most proximal and the most abundant immune cell type in relation to tumor cells. Moreover, the highest frequency of total and activated T cells was seen in close association with basal tumor cells. Conclusions: Our study underscores the importance of understanding the complex interplay between CAFs and tumor cells in PDAC. Further, we identify a relationship between T cells and tumor cell subtypes. By leveraging high-dimensional data and patient-derived models, we provide new insights into the spatial and functional dynamics of the PDAC TME, paving the way for novel therapeutic strategies to reprogram the TME and enhance patient survival.
 Citation Format: Jacquelyn W Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J Zabransky, Jae W Lee, Emma Kartalia, Jignasha Patel, Sarah Shin, Alexei Hernandez, Courtney Cannon, Nicole Gross, Soren Charmsaz, Elana J Fertig, Richard A Burkhart, Won J Ho, Elizabeth M Jaffee. Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b1b3de60c255399613042a94100d74cd1acaab3" target='_blank'>
              Abstract A032: Cancer associated fibroblasts drive classical to basal change associated with increased T cell presence in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jacquelyn W. Zimmerman, Samantha Guinn, Mili Ramani, Brayan Perez, Joseph A Tandurella, Daniel J. Zabransky, Jae W. Lee, Emma Kartalia, Jignasha Patel, Sarah M. Shin, Alexei Hernandez, Courtney Cannon, Nicole E. Gross, Soren Charmsaz, E. Fertig, Richard A Burkhart, W. Ho, Elizabeth M. Jaffee
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron A. Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature.Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature.A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo.This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, , Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7649961c3767077277c2ff33140a98201a7bd6" target='_blank'>
              Characterization and spatial distribution of infiltrating lymphocytes in medullary, and lymphocyte-predominant triple negative breast cancers
              </a>
            </td>
          <td>
            A. Alfaro, C. Catelain, H. El-Masri, P. Rameau, M. Lacroix-Triki, JY Scoazec, V. Marty, F. Mosele, B. Pistilli
          </td>
          <td>2024-09-14</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Heloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="The success of chimeric antigen receptor (CAR) T-cell therapy in treating hematologic malignancies has generated widespread interest in translating this technology to solid cancers. However, issues like tumor infiltration, the immunosuppressive tumor microenvironment, and tumor heterogeneity limit its efficacy in the solid tumor setting. Recent experimental and clinical studies propose local administration directly into the tumor or at the tumor site to increase CAR T-cell infiltration and improve treatment outcomes. Characteristics of the types of solid tumors that may be the most receptive to this treatment approach remain unclear. In this work, we develop a spatiotemporal model for CAR T-cell treatment of solid tumors, and use numerical simulations to compare the effect of introducing CAR T cells via intratumoral injection versus intracavitary administration in diverse cancer types. We demonstrate that the model can recapitulate tumor and CAR T-cell data from imaging studies of local administration of CAR T cells in mouse models. Our results suggest that locally administered CAR T cells will be most successful against slowly proliferating, highly diffusive tumors, which have the lowest average tumor cell density. These findings affirm the clinical observation that CAR T cells will not perform equally across different types of solid tumors, and suggest that measuring tumor density may be helpful when considering the feasibility of CAR T-cell therapy and planning dosages for a particular patient. We additionally find that local delivery of CAR T cells can result in deep tumor responses, provided that the initial CAR T-cell dose does not contain a significant fraction of exhausted cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7a6bf493c0b33ad1aad3083fdba685e7c59e1d" target='_blank'>
              Spatiotemporal dynamics of tumor - CAR T-cell interaction following local administration in solid cancers
              </a>
            </td>
          <td>
            Katherine Owens, Aminur Rahman, Ivana Bozic
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9797ff0fc8da1478461b88997f4f8cb4c2bab8" target='_blank'>
              Single-cell and spatial sequencing identifies senescent and germinal tumor cells in adamantinomatous craniopharyngiomas
              </a>
            </td>
          <td>
            Xianlong Wang, Jincheng Lin, Hongxing Liu, Chuan Zhao, Zhiwei Tu, Dapeng Xu, En Zhang, Zhongqing Zhou, Xueling Qi, Xingfu Wang, Zhixiong Lin
          </td>
          <td>2024-09-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, Jian-Qiang Lu, John S Lazo, T. Kislinger, Yu Lu, Kevin R. Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) is a deadly cancer with poor response to targeted therapy, largely driven by an immunosuppressive tumor microenvironment (TME). Here we examine the immune-modulatory role of the receptor tyrosine kinase EphA4 in HNSCC progression. Within the TME, EphA4 is primarily expressed on regulatory T cells (Tregs) and macrophages. In contrast ephrinB2, an activating ligand of EphA4, is expressed in tumor blood vessels. Using genetically engineered mouse models, we show that EphA4 expressed in Tregs promotes tumor growth, whereas EphA4 expressed in monocytes inhibits tumor growth. In contrast, ephrinB2 knockout in blood vessels reduces both intratumoral Tregs and macrophages. A novel specific EphA4 inhibitor, APY-d3-PEG4, reverses the accelerated tumor growth we had previously reported with EphB4 cancer cell knockout. EphA4 knockout in macrophages not only enhanced their differentiation into M2 macrophage but also increased Treg suppressive activity. APY-d3-PEG4 reversed the accelerated growth seen in the EphA4 knockout of monocytes but conferred no additional benefit when EphA4 was knocked out on Tregs. Underscoring an EphA4-mediated interplay between Tregs and macrophages, we found that knockout of EphA4 in Tregs not only decreases their activation but also reduces tumor infiltration of pro-tumoral M2 macrophages. These data identify Tregs as a primary target of APY-d3-PEG4 and suggest a role for Tregs in regulating macrophage conversion. These data also support the possible anti-cancer therapeutic value of bispecific peptides or antibodies capable of promoting EphA4 blockade in Tregs but not macrophages. Significance EphA4 in regulatory T cells has a pro-tumoral effect while EphA4 in macrophages plays an anti-tumoral role underscoring the necessity of developing biologically rational therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99406a05b43999ff5eb6dc79db7091b1d866ebc8" target='_blank'>
              The pro-tumoral and anti-tumoral roles of EphA4 on T regulatory cells and tumor associated macrophages during HNSCC tumor progression
              </a>
            </td>
          <td>
            S. Corbo, D. Nguyen, Shilpa Bhatia, L. Darragh, K. Abdelazeem, B. V. Court, Nicholas A. Olimpo, J. Gadwa, Justin Yu, Chloe Hodgson, Von Samedi, Enrique Sanz Garcia, Lillian Siu, Anthony J. Saviola, Lynn E. Heasley, M. Knitz, Elena B. Pasquale, S. D. Karam
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Human cytomegalovirus (HCMV) infection is common in tumor tissues across different types of cancer. While HCMV has not been recognized as a cancer-causing virus, numerous studies hint at its potential role in cancer development where its presence in various cancers corresponds with the hallmarks of cancer. Herein, we discuss and demonstrate that high-risk HCMV-DB and BL strains have the potential to trigger transformation in epithelial cells, including human mammary epithelial cells (HMECs), ovarian epithelial cells (OECs), and prostate epithelial cells (PECs), through the generation of polyploid giant cancer cells (PGCCs). A discussion is provided on how HCMV infection creates a cellular environment that promotes oncogenesis, supporting the continuous growth of CMV-transformed cells. The aforementioned transformed cells, named CTH, CTO, and CTP cells, underwent giant cell cycling with PGCC generation parallel to dedifferentiation, displaying stem-like characteristics and an epithelial–mesenchymal transition (EMT) phenotype. Furthermore, we propose that giant cell cycling through PGCCs, increased EZH2 expression, EMT, and the acquisition of malignant traits represent a deleterious response to the cellular stress induced by high-risk oncogenic HCMV strains, the latter being the origin of the transformation process in epithelial cells upon HCMV infection and leading to adenocarcinoma of poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139ca38ce75a83fdf75794021da421194cdfffd" target='_blank'>
              Polyploid Giant Cancer Cells: A Distinctive Feature in the Transformation of Epithelial Cells by High-Risk Oncogenic HCMV Strains
              </a>
            </td>
          <td>
            Georges Herbein, Ranim El Baba
          </td>
          <td>2024-07-31</td>
          <td>Viruses</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alpha-smooth muscle actin (α-SMA) expression in the stroma is linked to the presence of cancer-associated fibroblasts and is known to correlate with worse outcomes in various tumors. In this study, using a GeoMx digital spatial profiling approach, we characterized the gene expression of the tumor and α-SMA-expressing stromal cell compartments in pancreatic neuroendocrine tumors (PanNETs). The profiling was performed on tissues from eight retrospective cases (three grade 1, four grade 2, and one grade 3). Selected regions of interest were segmented geometrically based on tissue morphology and fluorescent signals from synaptophysin and α-SMA markers. The α-SMA-expressing stromal-cell-associated genes were involved in pathways of extracellular matrix modification, whereas, in tumor cells, the gene expression profiles were associated with pathways involved in cell proliferation. The comparison of gene expression profiles across all three PanNET grades revealed that the differences between grades are not only present at the level of the tumor but also in the α-SMA-expressing stromal cells. Furthermore, the tumor cells from regions with a rich presence of adjacent α-SMA-expressing stromal cells revealed an upregulation of matrix metalloproteinase-9 (MMP9) expression in grade 3 tumors. This study provides an in-depth characterization of gene expression profiles in α-SMA-expressing stromal and tumor cells, and outlines potential crosstalk mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fe06c12f5865d2f0fe11fa3ad5ad66746e58fe" target='_blank'>
              Tumor and α-SMA-expressing stromal cells in pancreatic neuroendocrine tumors have a distinct RNA profile depending on tumor grade.
              </a>
            </td>
          <td>
            H. Niedra, R. Peculis, Rihards Saksis, Ilona Mandrika, Sofija Vilisova, Jurijs Nazarovs, Austra Breikša, Aija Gerina, Julie Earl, Ignacio Ruz-Caracuel, M. Rosas, A. Puķītis, Natalja Senterjakova, V. Rovite
          </td>
          <td>2024-09-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Pancreatic cancer, recognized as one of the most lethal cancer types, exhibits a dismal five-year survival rate below 12%. The genetic and epigenetic heterogeneity in human cancer poses a tremendous challenge in understanding disease mechanisms and developing targeted therapeutics. Current mouse models in the field can recapitulate some features of human pathophysiology but have limitations. While tumor formation in patients is focal, most genetically engineered mouse models (GEMMs) express oncogenes throughout the pancreas. GEMMs also have limited flexibility to manipulate gene expression and model the heterogeneity of human cancer. Additionally, GEMMs require significant time for crossing and tumor formation. Tumor transplantation models offer greater flexibility, however, they often require immunodeficient mice, which limits their applications in modeling the tumor microenvironment. To overcome the limits of current mouse models, we combined the piggyBac transposon system, CRISPR/Cas9, and in vivo electroporation to engineer somatic pancreatic tissue to induce focal tumor formation in immune-competent mice. With this approach, we engineered wildtype pancreata to express oncogenic KrasG12D while knocking out multiple tumor suppressors. We optimized this protocol by generating multiplexed gRNA constructs and cell type-specific promoters to improve reproducibility. Low-grade tumors formed within 10 days after electroporation, progressed to higher grade within a month, and metastasized to peritoneal organs during disease progression. Mouse tumors also recapitulated histopathological features in human patients. Lastly, we used our novel mouse model to study how mutant Kras disrupts post-transcriptional regulation by microRNA to drive pancreatic tumorigenesis. This model offers a powerful platform for manipulating gene expression and functionality to model cancer heterogeneity, ultimately advancing precision oncology's prospects for treating human patients.
 Citation Format: Ziyue (Zoey) Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem. Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02653c15c1c4443d9e4685c7c408a134b796978a" target='_blank'>
              Abstract A006: Generation of a flexible pancreatic cancer mouse model via somatic tissue engineering
              </a>
            </td>
          <td>
            Z. Yang, Mara Cardenas, Akihiko Miyauchi, Caroline Snider, Ronald Parchem
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The bone marrow microenvironment is critical for B-cell acute lymphoblastic leukemia (B-ALL) but its cellular heterogeneity remains poorly defined. Here, we employed single-cell RNA sequencing to comprehensively characterize the stromal and hematopoietic niches in pediatric B-ALL. Our analysis revealed two distinct mesenchymal stromal cell (MSC) populations as primary leukemia-supportive niches: early mesenchymal progenitors and adipogenic progenitors. Single-cell transcriptomic analysis infers that ALL blasts use distinct cell-cell interactions to communicate with the different stromal populations. Purified adipogenic progenitors from the bone of children with ALL support survival of the leukemic blasts ex vivo and their signature is enriched in relapse samples. Our data establish adipogenic progenitors as a distinct and novel component of the ALL niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f24fbb432c05f1e5ee3ed61b3b778c9a293df20" target='_blank'>
              Distinct Stromal Cell Populations Define the B-cell Acute Lymphoblastic Leukemia Microenvironment
              </a>
            </td>
          <td>
            Mauricio N. Ferrao Blanco, Bexultan Kazybay, Mirjam E Belderbos, Olaf Heidenreich, H. J. Vormoor
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Tumor initiation and progression rely on intricate cellular pathways that promote proliferation while suppressing differentiation, yet the importance of pathways inhibiting differentiation in cancer remains incompletely understood. Here, we reveal a novel mechanism centered on the repression of the neuronal-specific transcription factor ARNT2 by the MYC oncogene that governs the balance between proliferation and differentiation. We found that MYC coordinates the transcriptional repression of ARNT2 through the activity of polycomb repressive complex 2 (PRC2). Notably, ARNT2, highly and specifically expressed in the central nervous system, is diminished in glioblastoma, inversely correlating with patient survival. Utilizing in vitro and in vivo models, we demonstrate that ARNT2 knockout (KO) exerts no discernible effect on the in vitro proliferation of glioblastoma cells, but significantly enhances the growth of glioblastoma cells in vivo. Conversely, ARNT2 overexpression severely dampens the growth of fully transformed glioblastoma cells subcutaneously or orthotopically xenografted in mice. Mechanistically, ARNT2 depletion diminishes differentiation and enhances stemness of glioblastoma cells. Our findings provide new insights into the complex mechanisms used by oncogenes to limit differentiation in cancer cells and define ARNT2 as a tumor suppressor in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fc2b3a844fbe1cf3418432eb71140217e74feb" target='_blank'>
              MYC-Mediated Inhibition of ARNT2 Uncovers a Key Tumor Suppressor in Glioblastoma
              </a>
            </td>
          <td>
            Maralice Conacci-Sorrell, Yi-Heng Hao, N. Borenstein-Auerbach, Anthony Grichuk, Li Li, M. Lafita-Navarro, Shun Fang, Pedro A S Nogueira, Jiwoong Kim, Lin Xu, J. Shay
          </td>
          <td>2024-08-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01a31875ebe3aec993187c2c6c6d773a5efac6f4" target='_blank'>
              Human intermediate prostate cancer stem cells contribute to the initiation and development of prostate adenocarcinoma
              </a>
            </td>
          <td>
            Jie Mu, Ruizhi Li, Yu Zheng, Yi Lu, Lei Ma, Lin Yin, Miao Zhang, Wenyu Ma, Mengjia Chang, Aihua Liu, Jing Li, Haihui Zhu, Dong Wang
          </td>
          <td>2024-09-11</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484efc8a4ee08b37da85a0f29cd67a40a824997b" target='_blank'>
              Spatial analysis reveals targetable macrophage-mediated mechanisms of immune evasion in hepatocellular carcinoma minimal residual disease.
              </a>
            </td>
          <td>
            Lea Lemaitre, Nia Adeniji, Akanksha Suresh, Reshma Reguram, Josephine Zhang, Jangho Park, Amit Reddy, Alexandro E. Trevino, Aaron T. Mayer, A. Deutzmann, Aida S Hansen, Ling Tong, Vinodhini Arjunan, N. Kambham, Brendan C Visser, M. Dua, C. A. Bonham, N. Kothary, H. B. D'Angio, Ryan Preska, Yanay Rosen, James Zou, Vivek Charu, Dean W Felsher, R. Dhanasekaran
          </td>
          <td>2024-09-20</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary The classification of breast cancer has been a complex subject, with diverse origins of normal breast epithelial cells and evolutionary pathways contributing to its extensive heterogeneity, which further complicates the treatment process. We developed a dynamic network model to simulate the emergence of different cellular states and explore the underlying biological mechanisms. This model revealed cellular states that align with four recognized molecular subtypes of breast cancer. It also identified feedback loops that either reinforce or facilitate phenotypic transitions, and allowed us to trace these transitions through the model’s topological structure. This approach provides new insights into the mechanisms driving breast cancer heterogeneity and may benefit the development of more targeted therapeutic strategies. Abstract Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hematopoietic stem and progenitor cells (HSPCs) rely on intercellular signaling to maintain and adjust their production of blood and immune cells. This process occurs in the semi-fluidic bone marrow, hosting dozens of cell types that constantly migrate and interact. To elucidate the dynamic network of cell-cell interaction and signaling transduction underlying hematopoiesis, we developed an algorithm to measure Cell-cell Spatial Interaction Probability (CellSIP) by integrating data on ligand and receptor expression, cell type abundance, and cellular spatial positioning. Using new and published mouse datasets, we validated CellSIP and uncovered signaling transductions indicating feedback mechanisms underlying hematopoiesis. Moreover, we identified significant correlations between signaling pathways across individual HSPCs at the same hematopoiesis stage. These pathway correlations illuminate the organization of cellular and signaling networks underlying hematopoiesis, revealing new regulators through their associations with established ones. The signaling quantification and correlation data are available through the Hematopoiesis Intercellular Signaling Explorer (HISE).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf26ff56c34b3ddada82e3d4951fade44feb8315" target='_blank'>
              Cellular and molecular networks of intercellular signaling in bone marrow hematopoiesis
              </a>
            </td>
          <td>
            Zachary Thomas, Bowen Wang, Wade R. Boohar, Mary Vergel, Jiya Eerdeng, Dayeon J. Shon, M. Elowitz, Rong Lu
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Biological systems can be understood by perturbing individual components and studying the system’s response. Cell biology experiments are defined by the applied treatment, cellular state, and the assayed phenotype. Given the vast number of possible combinations, testing every scenario is impractical. We present Prophet, a transformer-based computational model for cellular phenotype prediction. Prophet learns a representation of the cell biology experiment space, enabling it to predict the outcomes of untested small molecule or genetic perturbations in new cellular contexts across diverse phenotypes including gene expression, cell viability, and cell morphology. Its scalable architecture facilitates training across independent assays, using transfer learning to enhance performance across phenotypes. In vitro validation shows Prophet’s potential to guide experimental design, making it a valuable tool for accelerating biological discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8123a1ada6d3520bdccd8d89073356381fcdd64" target='_blank'>
              Scalable and universal prediction of cellular phenotypes
              </a>
            </td>
          <td>
            Yuge Ji, Alejandro Tejada-Lapuerta, Niklas A. Schmacke, Zihe Zheng, Xinyue Zhang, Simrah Khan, I. Rothenaigner, Juliane Tschuck, K. Hadian, F. Theis
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 The complex biology of pancreatic ductal adenocarcinoma (PDAC) remains a significant challenge and limits overcoming its dismal prognosis. The tumor microenvironment (TME), shaped by dynamic interactions between cancer-associated fibroblasts (CAFs), immune and tumor cells, plays a crucial role in PDAC pathogenesis. Uncharted field lies in the impact of specific gene mutations within the tumor epithelium on orchestrating distinct oncogenic TME patterns. Our investigation into homologous recombination deficiency (HRD), caused by mutations in genes like ATM, reveals its role in promoting PDAC aggressiveness and triggering a pronounced desmoplastic reaction, suggesting a unique TME programing. Through single-cell-resolved and multiomics analyses of murine ATM and/or P53-depleted PDACs, and validation in human cases, we unveil transcriptional, translational, and secretory regulatory events, shed light on the intricate interplay between tumor genetics and PDAC microenvironment and pave the way for the elaboration of novel personalized therapies. Murine tumor analysis using single-nucleus multiomics RNA and ATAC sequencing, alongside histological validation in human tissues, revealed specific enrichment of myCAFs in ATM-deficient tumors and reduced populations of T cells and macrophages. Molecular signatures and cell-cell communication inference uncovered ATM-specific interactions within the TME, highlighting TGF-β signaling as pivotal in mediating dialog between CAFs and an aggressive EMT-like tumor cell subset prevalent in ATM-depleted tumors. Additionally, distinct communication patterns were observed between CAFs and immune cells, associated with specific epigenetic and molecular profiles. Mechanistically, transcriptomics, secretomics and proteomics investigations confirmed increased TGFβ release by ATM-deficient PDAC cells and identified a reactive oxygen species (ROS)-mediated mechanism affecting tumor cell behavior and interactions with CAFs. Consistently, a combinatorial therapy targeting TGF-β-mediated tumor-stroma dialog reversed cancer-promoting TME remodeling and exacerbated FOLFIRINOX cytotoxic effects, in vivo, exclusively in ATM-deficient HRD PDACs. Dissecting the biology of ATM-deficient HRD malignant cells, our study unveils their pivotal role in orchestrating oncogenic communication networks with CAFs and immune cells, along with specific tumor promoting stromal reprogramming. Our findings paint a comprehensive picture of how distinct tumor-stroma interactions reshape the TME, steering it towards a cancer-promoting outcome and shed light on the potential to exploit genotype-specific vulnerabilities via tailored tumor-stroma interference strategies.
 Citation Format: Elodie Roger, Hannah M. Mummey, Eleni Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, Lukas Perkhofer, Johann Gout, Alexander Kleger. Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A055.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fd93ca5c41a43eb8ee9eb151fa6f7aeb554c533" target='_blank'>
              Abstract A055: Multiomics profiling reveals druggable tumor-stroma interactions in ATM-deficient pancreatic cancer
              </a>
            </td>
          <td>
            E. Roger, Hannah M. Mummey, E. Zimmer, Julia Ragg, Yuna Lee, Kyle J. Gaulton, Alica K. Beutel, Christopher J. Halbrook, L. Perkhofer, J. Gout, A. Kleger
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G. Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab20abff3d5ab2e3023b99dd1dd6ab980ad6081" target='_blank'>
              Mutual exclusivity and co-occurrence patterns of immune checkpoints indicate NKG2A relates to anti-PD-1 resistance in gastric cancer
              </a>
            </td>
          <td>
            Guanjun Li, Xiaohan Liu, Chunhui Gu, Guichuang Ma, Shaowei Li, Zhenfeng Ma, Yanqiu Xiong, Yu Jiang, Qiong Huang, Jianhua Wu, Zhenzhen Wu, Wangjun Liao, Qijing Wu, Min Shi
          </td>
          <td>2024-08-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cbe85e7267d943e47fa9cdeee4b8e805b7a651" target='_blank'>
              CD24 flags anastasis in melanoma cells.
              </a>
            </td>
          <td>
            Martina H Vasileva, Anette Bennemann, Karolin Zachmann, Michael P Schön, Jorge Frank, Vijay Kumar Ulaganathan
          </td>
          <td>2024-08-13</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the analysis of changes in phenotype, such as in vivo and ex vivo morphology and localization of cellular proteins and organelles can provide important insights into patient treatment options. Novel therapies are needed to extend survival in metastatic castration-resistant prostate cancer (mCRPC). Prostate-specific membrane antigen (PSMA), a cell surface glycoprotein that is commonly overexpressed by prostate cancer (PC) cells relative to normal prostate cells, provides a validated target. We developed a software for image analysis designed to identify PSMA expression on the surface of epithelial cells in order to extract prognostic metrics. In addition, our software can deliver predictive information and inform clinicians regarding the efficacy of PC therapy. We can envisage additional applications of our software system, beyond PC, as PSMA is expressed in a variety of tissues. Our method is based on image denoising, topologic partitioning, and edge detection. These three steps allow to segment the area of each PSMA spot in an image of a coverslip with epithelial cells. Our objective has been to present the community with an integrated, easy to use by all, tool for resolving the complex cytoskeletal organization and it is our goal to have such software system approved for use in the clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926fb40317d30bcdb538d1f8f9766fb5c180eb87" target='_blank'>
              Quantitative Microscopy in Medicine
              </a>
            </td>
          <td>
            Alexandre Matov, Shayan Modiri
          </td>
          <td>2024-08-03</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) -- the expansion of somatically-mutated hematopoietic cells in blood -- is common in solid cancers. CH is associated with systemic inflammation that may lead to cancer, but its impact on tumor biology is underexplored. Here, we report the effects of CH on the tumor microenvironment (TME) using 1,550 treatment-naive patient samples from the CPTAC cohort. CH was present in 18.3% of patients, with one-third of CH mutations also detectable in tumor-derived DNA from the same individual (CH-Tum), reflecting CH-mutant leukocyte infiltration. The presence of CH-Tum was associated with worse survival across cancers, particularly for glioblastoma. Transcriptomics and proteomics revealed that CH drives inflammation in the TME in a cancer- and CH driver-specific manner, and may improve immunotherapy responses. In glioblastoma, CH associated with pronounced macrophage infiltration, inflammation, and an aggressive, mesenchymal phenotype. Our findings demonstrate that CH shapes the TME, with potential applications as a biomarker in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df335cc5550a0c8767d76bf480da6e95ae3c38e1" target='_blank'>
              Inflammatory reprogramming of the tumor microenvironment by infiltrating clonal hematopoiesis is associated with adverse outcomes in solid cancer
              </a>
            </td>
          <td>
            M. Buttigieg, C. Vlasschaert, A. G. Bick, R. Vanner, M. Rauh
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Effective drug discovery relies on combining target knowledge with functional assays and multi-omics data to understand chemicals’ molecular actions. However, the relationship between cell morphology and gene expression over time and across cell lines remains unclear. To explore this, we analyzed Cell Painting and L1000 data for 106 compounds across three cell lines from osteoblast, lung, and breast tumors (U2OS, A549, and MCF7) at three time points (6h, 24h, 48h) using a 10µM concentration. We found significant time and cell line effects in Cell Painting data, with less pronounced effects in transcriptomics. Using Weighted Gene Co-expression Network Analysis (WGCNA) and enrichment analysis, we identified connections between cell morphology and gene deregulation for chemicals with similar biological effects (e.g., HDAC and CDK inhibitors). These findings suggest that while Cell Painting shows distinct patterns, both technologies offer complementary insights into compound-induced cellular changes, enhancing drug discovery and chemical risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e8eb6d8184a42daa0a1c6534359ca9aca19c28" target='_blank'>
              Cell morphology and gene expression: tracking changes and complementarity across time and cell lines
              </a>
            </td>
          <td>
            Vanille Lejal, David Rouquié, Olivier Taboureau
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study presents an interdisciplinary approach to analyse the distribution of cancer stem cell markers (CSCMs) across various cancer-affected organs using hypergraphs. Cancer stem cells (CSCs) play a crucial role in cancer initiation, progression, and metastasis. By employing hypergraphs, we model the relationships between CSCM locations and cancerous organs, providing a comprehensive representation of these interactions. Initially, we utilised an unweighted incidence matrix and its Markov transition matrices to gain a dynamic perspective on CSCM distributions. This method allows us to observe how these markers spread and influence cancer progression in a dynamical context. By calculating mutual information for each node and hyperedge, our analysis uncovers complex interaction patterns between CSCMs and organs, highlighting the critical roles of certain markers in cancer progression and metastasis. Our approach offers a detailed representation of cancer stem cell networks, enhancing our understanding of the mechanisms driving cancer heterogeneity and metastasis. By integrating hypergraph theory with cancer biology, this study provides valuable insights for developing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48368f26aa7447550426a623ac42529547196d96" target='_blank'>
              Mapping Cancer Stem Cell Markers Distribution:A Hypergraph Analysis Across Organs
              </a>
            </td>
          <td>
            D. H. Margarit, Gustavo Paccosi, M. V. Reale, Lilia M. Romanelli
          </td>
          <td>2024-07-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8018e573a8779af2c11f1aae1832a3b88d066791" target='_blank'>
              Targeting SWI/SNF ATPases reduces neuroblastoma cell plasticity.
              </a>
            </td>
          <td>
            Man Xu, Jason J Hong, Xiyuan Zhang, Ming Sun, Xingyu Liu, Jeeyoun Kang, Hannah Stack, Wendy Fang, Haiyan Lei, Xavier Lacoste, Reona Okada, Raina Jung, Rosa Nguyen, J. Shern, Carol J Thiele, Zhihui Liu
          </td>
          <td>2024-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The WDR5/MLL1-H3K4me3 epigenetic axis is often activated in both tumor cells and tumor-infiltrating immune cells to drive various cellular responses in the tumor microenvironment and has been extensively studied in hematopoietic cancer, but its respective functions in tumor cells and immune cells in the context of tumor growth regulation of solid tumor is still incompletely understood. We report here that WDR5 exhibits a higher expression level in human pancreatic tumor tissues compared with adjacent normal pancreas. Moreover, WDR5 expression is negatively correlated with patients’ response to chemotherapy or immunotherapy in human colon cancer and melanoma. However, WDR5 expression is positively correlated with the HLA level in human cancer cells, and H3K4me3 enrichment is observed at the promoter region of the HLA-A, HLA-B, and HLA-C genes in pancreatic cancer cells. Using mouse tumor cell lines and in vivo tumor models, we determined that WDR5 deficiency or inhibition significantly represses MHC I expression in vitro and in vivo in pancreatic tumor cells. Mechanistically, we determine that WDR5 deficiency inhibits H3K4me3 deposition at the MHC I (H2K) promoter region to repress MHC I (H2K) transcription. On the other hand, WDR5 depletion leads to the effective downregulation of immune checkpoints and immunosuppressive cytokines, including TGFβ and IL6, in the pancreatic tumor microenvironments. Our data determine that WDR5 not only regulates tumor cell immunogenicity to suppress tumor growth but also activates immune suppressive pathways to promote tumor immune evasion. Selective activation of the WDR5-MHC I pathway and/or selective inhibition of the WDR5–immune checkpoint and WDR5–cytokine pathways should be considered in WDR5-based epigenetic cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855836379cadca724defacc71dcd89c3fbbdd74b" target='_blank'>
              The Wdr5-H3K4me3 Epigenetic Axis Regulates Pancreatic Tumor Immunogenicity and Immune Suppression
              </a>
            </td>
          <td>
            Kaidi Deng, Liyan Liang, Yingcui Yang, Yanmin Wu, Yan Li, Rongrong Zhang, Yulin Tian, Chunwan Lu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) it though to develop through induction of different precursor lesions, however, the normal cell type that gives rise to these precancerous lesions and PDAC is unclear. Recent studies using mouse models and ex vivo cultured human cells suggest that the cellular origin of PDAC may be either acinar or ductal cells. Potential transcriptional markers associated with PDAC of different cellular origins have been identified. Here, we interrogate whether these markers are predictive of cellular of origin across models using mouse models where acinar and ductal cells (Ptf1aCreER or Sox9CreER) expressing oncogenic Kras in the absence of Trp53 or Pten give rise to PDAC tumors. We found that acinar- and ductal-cell-derived PDAC exhibit transcriptional differences both in vivo and in vitro when comparing within one genotype or sample source type (e.g. bulk tumor or primary cell lines). However, previously predicted markers of cellular origin had variable expression across models and sample types suggestive of differences specific to those sample sets. Using our larger sample sets and multiple genotypes and source types, we proposed a more comprehensive set of cell of origin markers for consideration. Application of these markers to our own data sets, however, failed to properly segregate acinar- and ductal-cell-derived PDAC into separate groups. This suggests that the transcriptional heterogeneity observed in our sample set was not driven by the tumors arising from different cellular origins. We also examined the relationship between cell of origin and patient PDAC transcriptional subtypes using our mouse model datasets. Our findings support previous research indicating that mouse models of acinar-cell-derived PDAC favor those of a classical subtype. However, our mouse PDAC samples all express similar or very low levels of the genes denoting the basal subtype, suggesting that the basal subtype was not strongly present in any of the datasets we examined. In summary, our data suggest that acinar and ductal cells converge into a very similar PDAC transcriptional state and that, if differences can be identified, they will likely be identified by non-transcript based readouts.
 Citation Format: Ken Chu, Alex Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire Dubois, Stephane Flibotte, Janel Kopp. Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7619576b4dd9f9188da32a5b5d7af49469c5cb88" target='_blank'>
              Abstract A025: Cell of origin does not underlie transcription heterogeneity in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Ken Chu, Alex Y. L. Lee, Karnjit Sarai, Yan Dou, Wesley Hunt, Emma Croft, Atefeh Samani, Farnaz Taghizedeh, Claire L. Dubois, Stephane Flibotte, Janel L. Kopp
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Background: Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemoradiation and immune therapies. There is limited biomarker to identify subsets of PDAC that responds to immunotherapy or targeted therapy. We have identified a subset of PDACs (approximately 25%) carrying COMPASS-like complex (CLC) gene mutations. Analysis of the clinicopathologic and molecular features of this PDAC subset revealed that these tumors are an aggressive group of PDACs with poor or squamous histologic differentiation, increased rates of TP53 mutations, and poor patient survival compared to control PDACs. Our preliminary data showed that PDACs carrying mutations in two members of the CLC genes (KDM6A and KMT2D) have immunosuppressive tumor microenvironment (TME), suggesting that CLC genes are candidate biomarkers for immunotherapy. However, a comprehensive analysis of the immune composition, immune checkpoint expression, and the relationship between tumor cells and immune cells in CLC gene mutated PDACs compared to controls has not been performed. In this study, we characterized the immune composition in CLC gene mutated PDACs compared to paired control PDACs using tyramide signal amplification multiplex fluorescent immunohistochemistry (mfIHC). Methods: Human PDAC tissue slides were stained with 2 mfIHC panels and cell types were defined by different markers: panel 1 (PanCK, CD163, PD-L1, CD3, CD8, FoxP3) and panel 2 (PanCK, CD163, CD3, CD8, TIGIT, TIM3). Random image sections were selected in each sample (5∼8 sections per sample) and divided into wild-type (WT) group (n=57) and mutant group (n=41) and analyzed using InForm Cell Analysis software. Tissue segmentation training was performed to separate stromal and epithelial elements. Cell percentage, cell density, and nearest neighbor distance of each cell type were calculated using R program. Results: The CLC gene mutation is associated with elevated epithelial cells (ECs) and PDL1+ ECs and increased infiltration of exhausted cytotoxic T cells (TIGIT+/TIM3+ CD8 T cells). CLC gene mutated PDACs are characterized by spatial proximity between exhausted T cells (TIGIT+) and epithelial cells/antigen presenting cells (APCs). Higher number of epithelial cells is associated with reduced infiltration of helper T cells in the mutant PDAC tissue. We also observed that high numbers of APC in proximity to CD4 T cells were positively correlated with the proximity of EC and CD8 cytotoxic T cells as well as the proximity of CD4 T cells and cytotoxic T cells in the WT group but not in the CLC gene mutant group. Conclusion: Our results provide insight into the effects of CLC gene mutations on shaping the immune microenvironment of PDAC and suggest that CLC gene mutated PDACs have an immunosuppressive immune microenvironment than other PDACs and may respond differently to immunotherapy.
 Citation Format: Shungang Zhang, Elaina Daniels, Jake McGue, Hong Sun Kim, Dafydd Thomas, Timothy Frankel, Jiaqi Shi. Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2790a085665b4a36a3c8b84d6bede9ac8c16c70" target='_blank'>
              Abstract A040: Multiplex fluorescent immunohistochemistry reveals immunosuppressive tumor microenvironment in COMPASS-like complex gene mutated pancreatic cancer
              </a>
            </td>
          <td>
            Shungang Zhang, Elaina Daniels, Jake McGue, H. Kim, Dafydd Thomas, Timothy L. Frankel, Jiaqi Shi
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
The efficacy of immune checkpoint blockade therapy in patients with hepatocellular carcinoma (HCC) remains poor. Although serine- and arginine-rich splicing factor (SRSF) family members play crucial roles in tumors, their impact on tumor immunology remains unclear. This study aimed to elucidate the role of SRSF10 in HCC immunotherapy.


METHODS
To identify the key genes associated with immunotherapy resistance, we conducted single-nuclear RNA sequencing, multiplex immunofluorescence, and The Cancer Genome Atlas and Gene Expression Omnibus database analyses. We investigated the biological functions of SRSF10 in immune evasion using in vitro co-culture systems, flow cytometry, various tumor-bearing mouse models, and patient-derived organotypic tumor spheroids.


RESULTS
SRSF10 was upregulated in various tumors and associated with poor prognosis. Moreover, SRSF10 positively regulated lactate production, and SRSF10/glycolysis/ histone H3 lysine 18 lactylation (H3K18la) formed a positive feedback loop in tumor cells. Increased lactate levels promoted M2 macrophage polarization, thereby inhibiting CD8+ T cell activity. Mechanistically, SRSF10 interacted with the 3'-untranslated region of MYB, enhancing MYB RNA stability, and subsequently upregulating key glycolysis-related enzymes including glucose transporter 1 (GLUT1), hexokinase 1 (HK1), lactate dehydrogenase A (LDHA), resulting in elevated intracellular and extracellular lactate levels. Lactate accumulation induced histone lactylation, which further upregulated SRSF10 expression. Additionally, lactate produced by tumors induced lactylation of the histone H3K18la site upon transport into macrophages, thereby activating transcription and enhancing pro-tumor macrophage activity. M2 macrophages, in turn, inhibited the enrichment of CD8+ T cells and the proportion of interferon-γ+CD8+ T cells in the tumor microenvironment (TME), thus creating an immunosuppressive TME. Clinically, SRSF10 could serve as a biomarker for assessing immunotherapy resistance in various solid tumors. Pharmacological targeting of SRSF10 with a selective inhibitor 1C8 enhanced the efficacy of programmed cell death 1 (PD-1) monoclonal antibodies (mAbs) in both murine and human preclinical models.


CONCLUSIONS
The SRSF10/MYB/glycolysis/lactate axis is critical for triggering immune evasion and anti-PD-1 resistance. Inhibiting SRSF10 by 1C8 may overcome anti-PD-1 tolerance in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e482999679814655172e8da8a22ac0dbfafa590" target='_blank'>
              Targeting SRSF10 might inhibit M2 macrophage polarization and potentiate anti-PD-1 therapy in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Jialiang Cai, Lina Song, Feng Zhang, Suiyi Wu, Guiqi Zhu, Peiling Zhang, Shiping Chen, Jun-Ze Du, Biao Wang, Yufan Cai, Yi Yang, Jinglei Wan, Jian Zhou, Jia Fan, Zhi Dai
          </td>
          <td>2024-09-02</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf3da76b81a975c5e4f8d3102cc9399708ed2b0" target='_blank'>
              Mathematical modeling and quantitative analysis of phenotypic plasticity during tumor evolution based on single-cell data.
              </a>
            </td>
          <td>
            Yuyang Xiao, Xiufen Zou
          </td>
          <td>2024-08-20</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62fe0d4817b1cce8f7a8474cb3953d097a1d805f" target='_blank'>
              The spatial distribution of intermediate fibroblasts and myeloid-derived cells dictate lymph node metastasis dynamics in oral cancer
              </a>
            </td>
          <td>
            Soni Shaikh, Harsh Dhar, Manju Moorthy, Vijayalakshmi Bhat, Sangramjit Basu, Devmalya Banerjee, Deepak Kumar Mishra, Sourav Datta, Geetashree Mukherjee
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Phosphoinositides (PIs) are minor but essential phospholipids that play crucial roles in cellular signaling pathways, membrane dynamics, and the regulation of various cellular processes. We developed and utilized a novel PI-focused CRISPR gRNA library to perform negative-selection and positive-selection screens in PANC-1 and OCI-AML2 cells, models of pancreatic ductal adenocarcinoma (PDAC) and acute myeloid leukemia (AML), respectively. Through these screens, we identified 28 essential genes in PANC-1, 84 essential genes in OCI-AML2, and 28 regulators of colony formation in OCI-AML2. Our results using this small and focused library uncovered false negatives and subtle effects that may be missed in genome-wide approaches, while enabling adaptation to different screening conditions. Overall, our results uncovered previously uncharacterized essential genes in PDAC and AML that can be leveraged as therapeutic targets and biomarkers. We also demonstrate that focused libraries offer a more efficient and targeted approach to uncovering critical genetic determinants of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468ec30b5ec6c72c51b0f5fb5f674977328a427d" target='_blank'>
              Phosphoinositide-focused CRISPR screen identifies novel genetic vulnerabilities in PDAC and AML cells
              </a>
            </td>
          <td>
            Daniel K.C. Lee, Ryan Loke, Jonathan TS Chow, M. Gabra, L. Salmena
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Langerhans cell Histiocytosis (LCH) and Erdheim-Chester disease (ECD) are clonal myeloid disorders, associated with MAP-Kinase activating mutations and an increased risk of neurodegeneration. Surprisingly, we found pervasive PU.1+ microglia mutant clones across the brain of LCH and ECD patients with and without neurological symptoms, associated with microgliosis, reactive astrocytosis, and neuronal loss. The disease predominated in the grey nuclei of the rhombencephalon, a topography attributable to a local proliferative advantage of mutant microglia. Presence of clinical symptoms was associated with a longer evolution of the disease and a larger size of PU.1+ clones (p= 0.0003). Genetic lineage tracing of PU.1+ clones suggest a resident macrophage lineage or a bone marrow precursor origin depending on patients. Finally, a CSF1R-inhibitor depleted mutant microglia and limited neuronal loss in mice suggesting an alternative to MAPK inhibitors. These studies characterize a progressive neurodegenerative disease, caused by clonal proliferation of inflammatory microglia (CPIM), with a decade(s)-long preclinical stage of incipient disease that represent a therapeutic window for prevention of neuronal death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2ee538175a393697537cbcb21555fea3da58384" target='_blank'>
              Mechanism of neurodegeneration mediated by clonal inflammatory microglia
              </a>
            </td>
          <td>
            Rocio Vicario, Stamatina Fragkogianni, Maria Pokrovskii, Carina Mayer, Estibaliz Lopez-Rodrigo, Yang Hu, Masato Ogishi, Araitz Alberdi, Ann Baako, Oyku Ay, I. Plu, Véronique Sazdovitch, S. Héritier, F. Cohen-Aubart, Natalia Shor, M. Miyara, Florence Nguyen-Khac, Agnes Viale, A. Idbaih, Zahir Amoura, Marc K. Rosenblum, Haochen Zhang, Elias-Ramzey Karnoub, P. Sashittal, Akhil Jakatdar, Christine A. Iacobuzio-Donahue, O. Abdel-Wahab, Viviane Tabar, N. Socci, O. Elemento, Eli L Diamond, B. Boisson, J. Casanova, Danielle Seilhean, J. Haroche, Jean Donadieu, F. Geissmann
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is crucial in cancer development and therapeutic response. Immunotherapy is increasingly recognized as a critical component of cancer treatment. While immunotherapies have shown efficacy in various cancers, including breast cancer, patient responses vary widely. Some patients receive significant benefits, while others experience minimal or no improvement. This disparity underscores the complexity and diversity of the immune system. In this study, we investigated the immune landscape and cell–cell communication within the TME of breast cancer through integrated analysis of bulk and single-cell RNA sequencing data. We established profiles of tumor immune infiltration that span across a broad spectrum of adaptive and innate immune cells. Our clustering analysis of immune infiltration identified three distinct patient groups: high T cell abundance, moderate infiltration, and low infiltration. Patients with low immune infiltration exhibited the poorest survival rates, while those in the moderate infiltration group showed better outcomes than those with high T cell abundance. Moreover, the high cell abundance group was associated with a greater tumor burden and higher rates of TP53 mutations, whereas the moderate infiltration group was characterized by a lower tumor burden and elevated PIK3CA mutations. Analysis of an independent single-cell RNA-seq breast cancer dataset confirmed the presence of similar infiltration patterns. Further investigation into ligand–receptor interactions within the TME unveiled significant variations in cell–cell communication patterns among these groups. Notably, we found that the signaling pathways SPP1 and EGF were exclusively active in the low immune infiltration group, suggesting their involvement in immune suppression. This work comprehensively characterizes the composition and dynamic interplay in the breast cancer TME. Our findings reveal associations between the extent of immune infiltration and clinical outcomes, providing valuable prognostic information for patient stratification. The unique mutations and signaling pathways associated with different patient groups offer insights into the mechanisms underlying diverse tumor immune infiltration and the formation of an immunosuppressive tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ff73b13094e5afc913b4266529d0100e73db7ee" target='_blank'>
              Characterizing the Tumor Microenvironment and Its Prognostic Impact in Breast Cancer
              </a>
            </td>
          <td>
            Wenjuan Zhang, Alex Lee, Amit K. Tiwari, Mary Qu Yang
          </td>
          <td>2024-09-10</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Cell migration is a fundamental process in both normal and cancerous tissues, playing a crucial role in development, immune responses, and, in the case of cancer cells, metastasis-a leading cause of cancer-related mortality. Understanding the differences between healthy and cancerous cell migration is essential for uncovering potential therapeutic targets. This study aims to elucidate these differences by comparing the migratory behaviors of healthy cells (nHDF cells) and cancer cells (MDA-MB-231 cells). Our findings reveal that cancer cells significantly reduce their stiffness during migration through narrow channels, a phenomenon not observed in healthy cells. Additionally, DNA and membrane repair mechanisms are more active in healthy cells during migration compared to tumor cells. Notably, the use of MDA-MB-231 FUCCI cells demonstrates that the cell cycle profoundly influences cell migration under confined conditions. These insights provide a deeper understanding of the cellular mechanisms driving migration in both healthy and cancerous cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80200f4051176bbf9268203283d720b6727862b7" target='_blank'>
              The role of disease state in confined migration
              </a>
            </td>
          <td>
            Annika Meid
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95736f8e150d70ccc63e3057497b476be28ab1d9" target='_blank'>
              CCKBR+ cancer cells contribute to the intratumor heterogeneity of gastric cancer and confer sensitivity to FOXO inhibition.
              </a>
            </td>
          <td>
            Zhenya Tan, Ke Pan, Minqiong Sun, Xianzhu Pan, Zhi Yang, Zhiling Chang, Xue Yang, Jicheng Zhu, Li Zhan, Yakun Liu, Xiaofei Li, Keqiong Lin, Lin Chen, Hui Mo, Wei Luo, Chen Kan, Lunxi Duan, Hong Zheng
          </td>
          <td>2024-08-20</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, , Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3fe31e6aa7a99673a4d6d194c90f42b74a1aa1" target='_blank'>
              Immunogenomic features of radiologically distinctive nodules in multiple primary lung cancer
              </a>
            </td>
          <td>
            Mei-Cheng Chen, Haolong Yang, Zhi Dong, Lu-Jie Li, Xiang-Min Li, Hong-He Luo, Qiong Li, Ying Zhu
          </td>
          <td>2024-09-05</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Modest response rates to immunotherapy observed in advanced lung cancer patients underscore the need to identify reliable biomarkers and targets, enhancing both treatment decision-making and efficacy. Factors such as PD-L1 expression, tumor mutation burden, and a ‘hot’ tumor microenvironment with heightened effector T cell infiltration have consistently been associated with positive responses. In contrast, the predictive role of the abundantly present tumor-infiltrating myeloid cell (TIMs) fraction remains somewhat uncertain, partly explained by their towering variety in terms of ontogeny, phenotype, location, and function. Nevertheless, numerous preclinical and clinical studies established a clear link between lung cancer progression and alterations in intra- and extramedullary hematopoiesis, leading to emergency myelopoiesis at the expense of megakaryocyte/erythroid and lymphoid differentiation. These observations affirm that a continuous crosstalk between solid cancers such as lung cancer and the bone marrow niche (BMN) must take place. However, the BMN, encompassing hematopoietic stem and progenitor cells, differentiated immune and stromal cells, remains inadequately explored in solid cancer patients. Subsequently, no clear consensus has been reached on the exact breadth of tumor installed hematopoiesis perturbing cues nor their predictive power for immunotherapy. As the current era of single-cell omics is reshaping our understanding of the hematopoietic process and the subcluster landscape of lung TIMs, we aim to present an updated overview of the hierarchical differentiation process of TIMs within the BMN of solid cancer bearing subjects. Our comprehensive overview underscores that lung cancer should be regarded as a systemic disease in which the cues governing the lung tumor-BMN crosstalk might bolster the definition of new biomarkers and druggable targets, potentially mitigating the high attrition rate of leading immunotherapies for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09f4d97d42cb2976ecd4a13f91ebddf956195779" target='_blank'>
              The crosstalk between lung cancer and the bone marrow niche fuels emergency myelopoiesis
              </a>
            </td>
          <td>
            Evelyn Calderon-Espinosa, K. De Ridder, Thomas Benoot, Y. Jansen, Domien Vanhonacker, Robbe Heestermans, A. De Becker, Ivan Van Riet, Lore Decoster, C. Goyvaerts
          </td>
          <td>2024-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37a0227f94104fac217dd5983f562df5f9fbaad8" target='_blank'>
              Characterization of the stem cell landscape and identification of a stemness-associated prognostic signature in bladder cancer
              </a>
            </td>
          <td>
            Gaoteng Lin, Jiamei Lin, Hao Wang, Liucheng Wang, Fangfang Zhan, Liqian Wu, Liang Xue, Yang Dong, Wanqing Wei, Lin Liu
          </td>
          <td>2024-08-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3d4f61a2577f3255d5a96a1f62094418b1629b" target='_blank'>
              A cell state-specific metabolic vulnerability to GPX4-dependent ferroptosis in glioblastoma.
              </a>
            </td>
          <td>
            Matei A. Banu, Athanassios Dovas, Michael G. Argenziano, Wenting Zhao, Colin P. Sperring, Henar Cuervo Grajal, Zhouzerui Liu, Dominique Mo Higgins, Misha Amini, B. Pereira, Ling F Ye, A. Mahajan, N. Humala, J. Furnari, P. Upadhyayula, Fereshteh Zandkarimi, Trang Tt Nguyen, Damian E Teasley, Peter B Wu, Li Hai, Charles Karan, Tyrone Dowdy, Aida Razavilar, M. Siegelin, Jan Kitajewski, Mioara Larion, Jeffrey N Bruce, Brent R. Stockwell, Peter A. Sims, Peter D Canoll
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) is a type of blood and bone marrow cancer characterized by the proliferation of abnormal clonal haematopoietic cells in the bone marrow leading to bone marrow failure. Over the course of the disease, angiogenic factors released by leukaemic cells drastically alter the bone marrow vascular niches resulting in observable structural abnormalities. We use a technique from topological data analysis - persistent homology - to quantify the images and infer on the disease through the imaged morphological features. We find that persistent homology uncovers succinct dissimilarities between the control, early, and late stages of AML development. We then integrate persistent homology into stage-dependent Gaussian mixture models for the first time, proposing a new class of models which are applicable to persistent homology summaries and able to both infer patterns in morphological changes between different stages of progression as well as provide a basis for prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/908fdd9f4a85fe4603f9b7353c67f06a0aab3abc" target='_blank'>
              A Topological Gaussian Mixture Model for Bone Marrow Morphology in Leukaemia
              </a>
            </td>
          <td>
            Qiquan Wang, Anna Song, A. Batsivari, Dominique Bonnet, Anthea Monod
          </td>
          <td>2024-08-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Intraductal papillary mucinous neoplasms (IPMNs), a common precursor to pancreatic ductal adenocarcinoma, present as a spectrum of benign indolent behavior and progression to invasive carcinoma, which highlights the importance of identifying biomarkers of IPMN progression. This study utilized whole transcriptome spatial transcriptomics to define the molecular characteristics of Epithelial, Immune, and Fibroblast cell populations of the three distinct histological sub-types of IPMN to understand the dynamics between the cell types and identify biomarkers of neoplastic progression. Fifty-two surgically resected IPMN cores from 21 patients with resected IPMNs comprising 24 Gastric, 14 Intestinal, and 14 Pancreaticobiliary with a distribution of low-grade dysplasia (LGD), high-grade dysplasia (HGD), and Invasive carcinoma (IC) were analyzed. Epithelial, Immune, and Fibroblast components were analyzed using the GeoMx spatial profiler. Unsupervised hierarchical clustering of epithelial areas of interest (AOI) identified 3 distinct clusters demonstrating a spectrum of molecular phenotypes in IPMN that were correlated with the three histological subtypes. Comparing the gene expression profile based on the shared genes expressed on all three subtypes, the Intestinal type tends to have an overexpression of genes whereas the Pancreaticobiliary represents a repressed phenotype relative to the Gastric phenotype. Supervised clustering using Moffitt’s classifier to identify the Epithelial-Mesenchymal transition demonstrates the Pancreaticobiliary subtype to be more Basal-like versus the Gastric and Intestinal inclined towards the classical phenotype. Supervised clustering based on the degree of dysplasia within the epithelial compartments of each of these subtypes, demonstrated a spectrum of molecular heterogeneity and identified biomarkers specific to High-Grade dysplasia in the Gastric and Intestinal subtype. Immune AOI deconvolution analysis demonstrated that Tregs were associated with Pancreaticobiliary, Macrophage enrichment associated with the Intestinal phenotype, while there is enrichment of CD8 T-cell and CD4 T-cell gene signatures associated with the Gastric subtype. Fibroblast AOI analyses demonstrated heterogenous myCAF- iCAF signatures between the subtypes. In summary, Spatial whole transcriptome analysis of IPMN identifies molecular signatures in epithelial and immune compartments that differentiate between the three histological subtypes of IPMN and helps understand their progression from Low-grade dysplasia to High-grade dysplasia.
 Citation Format: Amaya Pankaj, Michael J Raabe, Yuhui Song, Bidish K Patel, Katherine Xu, Joshua R Kocher, Nicholas J Caldwell, Maria L Ganci, Linda T Nieman, Mari Mino-Kenudson, Vikram Deshpande, Nabeel Bardeesy, Carlos Fernandez-Del Castillo, Martin J Aryee, David T Ting, Yasmin G Hernandez-Barco. Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a43f4d0c8d98a15ce28e74661a5da83f45aa30c" target='_blank'>
              Abstract B091: Spatial Transcriptomics Unveils Intraductal Papillary Mucinous Neoplasm Heterogeneity: From Novel Clusters to Immune Dynamics
              </a>
            </td>
          <td>
            Amaya Pankaj, Michael J. Raabe, Yuhui Song, Bidish Kumar Patel, Katherine Xu, Joshua R. Kocher, Nicholas J Caldwell, M. L. Ganci, Linda T. Nieman, M. Mino-Kenudson, Vikram Deshpande, N. Bardeesy, C. F. Fernandez-Del Castillo, Martin J. Aryee, David Ting, Yasmin G Hernandez-Barco
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Patient-derived tumor organoids (PDTOs) are novel cellular models that maintain the genetic, phenotypic and structural features of patient tumor tissue and are useful for studying tumorigenesis and drug response. When integrated with advanced 3D imaging and analysis techniques, PDTOs can be used to establish physiologically relevant high-throughput and high-content drug screening platforms that support the development of patient-specific treatment strategies. However, in order to effectively leverage high-throughput PDTO observations for clinical predictions, it is critical to establish a quantitative understanding of the basic properties and variability of organoid growth dynamics. In this work, we introduced an innovative workflow for analyzing and understanding PDTO growth dynamics, by integrating a high-throughput imaging deep learning platform with mathematical modeling, incorporating flexible growth laws and variable dormancy times. We applied the workflow to colon cancer organoids and demonstrated that organoid growth is well-described by the Gompertz model of growth. Our analysis showed significant intrapatient heterogeneity in PDTO growth dynamics, with the initial exponential growth rate of an organoid following a lognormal distribution within each dataset. The level of intrapatient heterogeneity varied between patients, as did organoid growth rates and dormancy times of single seeded cells. Our work contributes to an emerging understanding of the basic growth characteristics of PDTOs, and it highlights the heterogeneity in organoid growth both within and between patients. These results pave the way for further modeling efforts aimed at predicting treatment response dynamics and drug resistance timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d868d7a68c9f23a73010e77ac074ffad6193b786" target='_blank'>
              Understanding patient-derived tumor organoid growth through an integrated imaging and mathematical modeling framework
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Seung-Jin Kim, Brandon Choi, J. K. Schmid, Karn Kaura, H. Lenz, S. Mumenthaler, Jasmine Foo
          </td>
          <td>2024-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0f02ab9166ce43f53694fdc71a386bd1c5188a5" target='_blank'>
              The presence of cancer-associated fibroblast in breast cavity side margins is in correlation with the expression of oncoproteins by adjacent epithelial cells: a new era in cancerous potential
              </a>
            </td>
          <td>
            Zohreh Miripour, Mina Aminifar, Parisa Hoseinpour, F. Abbasvandi, Koosha Karimi, Alireza Ghahremani, Mohammad Parniani, Mohammadreza Ghaderinia, Faride Makiyan, P. Aghaee, Mohammad Esmaeil Akbari, M. Abdolahad
          </td>
          <td>2024-09-17</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880d6fbe977b9fa0abf41b1330834beac1e2a6aa" target='_blank'>
              A patient-derived cell model for malignant transformation in IDH-mutant glioma
              </a>
            </td>
          <td>
            Olga Kim, Zach Sergi, Guangyang Yu, Kazu Yamamoto, Martha Quezado, Zied Abdullaev, Danel R Crooks, Shun Kishimoto, Qi Li, Peng Lu, Burchelle N. Blackman, T. Andresson, Xiaolin Wu, Bao Tran, Jun S. Wei, Wei Zhang, Meili Zhang, Hua Song, Javed Khan, Murali C. Krishna, J. Brender, Jing Wu
          </td>
          <td>2024-09-10</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Vulvar diseases are a critical yet often neglected area of women’s health, profoundly affecting patients’ quality of life and frequently resulting in long-term physical and psychological challenges. Lichen sclerosus (LS) is a chronic inflammatory skin disorder that predominantly affects the vulva, leading to severe itching, pain, scarring, and an increased risk of malignancy. Despite its profound impact on affected individuals, the molecular pathogenesis of vulvar LS (VLS) is not well understood, hindering the development of FDA-approved therapies. Here, we utilize single-cell and spatial transcriptomics to analyze lesional and non-lesional skin from VLS patients, as well as healthy control vulvar skin. Our findings demonstrate histologic, cellular, and molecular heterogeneities within VLS, yet highlight unifying molecular changes across keratinocytes, fibroblasts, immune cells, and melanocytes in lesional skin. They reveal cellular stress and damage in fibroblasts and keratinocytes, enhanced T cell activation and cytotoxicity, aberrant cell-cell signaling, and increased activation of the IFN, JAK/STAT, and p53 pathways in specific cell types. Using both monolayer and organotypic culture models, we also demonstrate that knockdown of select genes, which are downregulated in VLS lesional keratinocytes, partially recapitulates VLS-like stress-associated changes. Collectively, these data provide novel insights into the pathogenesis of VLS, identifying potential biomarkers and therapeutic targets for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a1ede19b990ffc821eea644860eccc4ba0949" target='_blank'>
              Single-cell and spatial transcriptomics of vulvar lichen sclerosus reveal multi-compartmental alterations in gene expression and signaling cross-talk
              </a>
            </td>
          <td>
            Peng Sun, Christina N. Kraus, Wei Zhao, Jiahui Xu, Susie Suh, Quy Nguyen, Yunlong Jia, Arjun Nair, Melanie Oakes, Roberto Tinoco, Jessica Shiu, Bryan Sun, Ashley Elsensohn, Scott X. Atwood, Qing Nie, Xing Dai
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Immunotherapy represents a groundbreaking and monumental achievement in the field of cancer therapy, marking a significant advancement in fighting against this devastating disease. Lung cancer has showed consistent clinical improvements in response to immunotherapy treatments, yet, it is undeniable that challenges such as limited response rates acquire resistance, and the unclear fundamental mechanisms were inevitable problems.


METHODS
The cellular composition was defined and distinguished through single-cell RNA sequencing (scRNA-seq) analysis of MPR (major pathologic response) and NMPR (non-major pathologic response) samples in GSE207422, including four primary MPR samples and eight primary NMPR samples.


RESULTS
We found obvious difference in CD8+ T cell population between MPR and NMPR samples, with high expression of TYMS, RRM2, and BIRC5 in NPMR samples. Meanwhile, the proportion of macrophages and tumor epithelial cells infiltration increased in the NMPR samples. We discovered biomarkers (ACTN4, ATF3, BRD2, CDKN1A, and CHMP4B) in epithelial cells which were potentially represented worse outcomes.


CONCLUSIONS
By exploring the difference of tumor microenvironment (TME) in samples with different corresponding degrees of neoadjuvant immunotherapy, this research introduces a number of novel biomarkers for predicting the response of treatment and a theoretical basis for overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eb58f4e74cf2a065722c81b24475ac3132ac67a" target='_blank'>
              Single-cell RNA sequencing reveals microenvironmental infiltration in non-small cell lung cancer with different responses to immunotherapy.
              </a>
            </td>
          <td>
            Xinnan Hu, Yonghui Wu, Lixin Wang, Fujun Yang, Lingyun Ye, Xiaoxia Chen, Xiao Song, Ping Wei
          </td>
          <td>2024-09-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="High-grade gliomas are a major health challenge with poor prognosis and high morbidity. Immune-checkpoint inhibitors (ICI) have emerged as promising therapeutic options for several malignancies yet show little efficacy against central nervous system (CNS) tumors. CD200 is a newly recognized immune checkpoint that modulates immune homeostasis. CD200 protein is expressed by a variety of cells, including immune cells and stromal cells, and is overexpressed by many tumors. The shedding of CD200 from tumor cells can create an immunosuppressive environment that dampens anti-tumor immunity by modulating cytolytic activity and cytokine expression both within and outside the tumor microenvironment (TME). While it is well-accepted that CD200 induces a pro-tumorigenic environment through its ability to suppress the immune response, we sought to determine the role of glioma-specific expression of CD200. We show that CD200 is expressed across glioma types, is shed from tumor cells, and increases over time in the serum of patients undergoing immunotherapy. Using CD200 knockout (KO) glioma models, we demonstrated that glioma cell-derived CD200 promotes tumor growth in vivo and in vitro. Notably, CD200 KO gliomas are spontaneously rejected by their host, a process that required a fully functional immune system, including NK and T-cells. Moreover, we report that glioma-derived or brain-injected soluble CD200 contributes to the suppression of antigen-specific CD8 T-cells in the draining lymph nodes (dLNs). Our work provides new mechanistic insights regarding CD200-mediated immunosuppression by gliomas. Statement of significance We demonstrate mechanisms of the druggable glioma-derived CD200 checkpoint on tumor growth and immune suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b634d44ec918f6bd798f77ff4dc7cd21cc67c2e" target='_blank'>
              CD200 depletion in glioma enhances antitumor immunity and induces tumor rejection
              </a>
            </td>
          <td>
            Itay Raphael, Anzar A. Mujeeb, Elisabet Ampudia-Mesias, ReidAnn E. Sever, Brandon L. McClellan, Stephen C. Frederico, Chaim T. Sneiderman, Apoorva Mirji, Ali Daba, Francisco Puerta-Martínez, Michal Nisnboym, W. Edwards, Michael W. Graner, Christopher L. Moertel, Maria G. Castro, Gary Kohanbash, Michael Olin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Recent advances in single cell sequencing have led to an increased focus on the role of cell-type composition in phenotypic presentation and disease progression. Cell-type composition research in the heart is challenging due to large, frequently multinucleated cardiomyocytes that preclude most single cell approaches from obtaining accurate measurements of cell composition. Our in silico studies reveal that ignoring cell type composition when calculating differentially expressed genes (DEGs) can have significant consequences. For example, a relatively small change in cell abundance of only 10% can result in over 25% of DEGs being false positives. Methods We have implemented an algorithmic approach that uses snRNAseq datasets as a reference to accurately calculate cell type compositions from bulk RNAseq datasets through robust data cleaning, gene selection, and multi-sample cross-subject and cross-cell-type deconvolution. We applied our approach to cardiomyocyte-specific α1A adrenergic receptor (CM-α1A-AR) knockout mice. 8-12 week-old mice (either WT or CM-α1A-KO) were subjected to permanent left coronary artery (LCA) ligation or sham surgery (n=4 per group). Transcriptomes from the infarct border zones were collected 3 days later and analyzed using our algorithm to determine cell-type abundances, corrected differential expression calculations using DESeq2, and validated these findings using RNAscope. Results Uncorrected DEGs for the CM-α1A-KO X LCA interaction term featured many cell-type specific genes such as Timp4 (fibroblasts) and Aplnr (cardiomyocytes) and overall GO enrichment for terms pertaining to cardiomyocyte differentiation (P=3.1E-4). Using our algorithm, we observe a striking loss of cardiomyocytes and gain in fibroblasts in the α1A-KO + LCA mice that was not recapitulated in WT + LCA animals, although we did observe a similar increase in macrophage abundance in both conditions. This recapitulates prior results that showed a much more severe heart failure phenotype in CM-α1A-KO + LCA mice. Following correction for cell-type, our DEGs now highlight a novel set of genes enriched for GO terms such as cardiac contraction (P=3.7E-5) and actin filament organization (P=6.3E-5). Conclusions Our algorithm identifies and corrects for cell-type abundance in bulk RNAseq datasets opening new avenues for research on novel genes and pathways as well as an improved understanding of the role of cardiac cell types in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217ee4d32d0eb92d12b53e00c08af7cc8fb059d2" target='_blank'>
              Novel Insights into Post-Myocardial Infarction Cardiac Remodeling through Algorithmic Detection of Cell-Type Composition Shifts
              </a>
            </td>
          <td>
            Brian Gural, Logan Kirkland, Abbey Hockett, Peyton Sandroni, Jiandong Zhang, M. Rosa-Garrido, Samantha K. Swift, Douglas J. Chapski, Michael A. Flinn, Caitlin C. O’Meara, T. Vondriska, Michaela Patterson, Brian C. Jensen, Christoph D Rau
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Introduction: High-risk neuroblastoma (NBL) is a leading cause of pediatric cancer death and is characterized by substantial intratumoral heterogeneity leading to therapeutic resistance. Despite the significant insights from recent single-cell sequencing, the evolution of the tumor microenvironment under therapy remains obscure. We longitudinally profiled 22 high-risk NBL patients before and after induction chemotherapy through single-nucleus RNA and ATAC sequencing. We identified profound shifts in tumor and immune cell subpopulations. In particular, macrophages significantly expanded towards pro-angiogenic, immunosuppressive, and metabolic phenotypes. Through a prediction of paracrine signaling using our single-cell sequencing data, we predicted a role of the heparin-binding epidermal growth factor (HB-EGF)/Erb receptor tyrosine kinase 4 (ERBB4) signaling pathway between VCAN+ macrophages and neuroblasts. In this study, we sought to characterize the role of this signaling pathway in therapy response in high-risk neuroblastoma. Methods: We examined the interaction between malignant neuroblasts and macrophages using transwell co-culture assay. Macrophages were differentiated from the THP1 monocytes. Neuroblasts were cultured using NB1643, CHLA15, CHLA20, COG-N-297, and COG-N-590 neuroblastoma cell lines. To determine the source and activity of the HB-EGF/ERBB4/ERK signaling pathway, protein levels were determined using enzyme-linked immunosorbent assay and Western blot using co-culture supernatant and neuroblast and macrophage cell lysates. Pathway activity was modulated using CRM197 as the HB-EGF inhibitor and afatinib as the ERBB tyrosine kinase inhibitor. We examined the role of macrophage-derived HB-EGF on cell proliferation and migration by colony formation and transwell migration assays, respectively. Furthermore, we assessed the adrenergic (PHOX2B, GATA3, ALK, PHOX2A) and mesenchymal (YAP1 and PRRX1) marker expressions by real-time qPCR and Western blot. Results: We found that expression of HB-EGF in macrophages was significantly induced when co-cultured with multiple neuroblast cell lines. Binding of HB-EGF to ERBB4 in neuroblasts led to the activation of the kinase pathway and increased proliferation and migration of neuroblast cells, which can be inhibited by either HB-EGF or tyrosine kinase inhibitors. We examined adrenergic and mesenchymal marker expression in neuroblasts after prolonged co-culture with macrophages. While ALK and PHOX2B expressions were reduced, YAP1 and PRRX1 expressions increased in post-therapy CHLA20 cells but not in the paired diagnostic CHLA15 or NB1643 cells. Inhibition of the HB-EGF/ERBB pathway partially restored PHOX2B expression and reduced YAP1 and PRRX1 expression in macrophage co-cultured CHLA20 cells. Conclusion: We established the HB-EGF/ERBB4 axis between macrophage and neoplastic neuroblast, inducing ERK signaling, tumor cell proliferation, migration, and adrenergic-to-mesenchymal transition. These findings collectively reveal a novel extrinsic mechanism of therapy resistance in high-risk NBL.
 Citation Format: Rumeysa Biyik-Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea F. Surrey, Michael D. Hogarty, Kathrin Bernt, Nancy Zhang, John M. Maris, Kai Tan. VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf38a2142d9aa00bb31bb908d3c2ec97aaec27dc" target='_blank'>
              Abstract B062: VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis
              </a>
            </td>
          <td>
            Rumeysa Biyik‐Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea Surrey, M. Hogarty, Kathrin M Bernt, Nancy Zhang, John M. Maris, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/587d631ba6a320ce9960d57b667aeb776ca548db" target='_blank'>
              Development of 3D-iNET ORION: a novel, pre-clinical, three-dimensional in vitro cell model for modeling human metastatic neuroendocrine tumor of the pancreas
              </a>
            </td>
          <td>
            J. Strnadel, M. Valasek, Grace Y Lin, Huahui Lin, Ann M Ponsford Tipps, Sang Myung Woo, Ken Fujimura, Huawei Wang, Sunkyu Choi, Jack Bui, Christopher Hermosillo, Kristen Jepsen, Michael R Navarro, J. Kelber, R. Klemke, Michael Bouvet
          </td>
          <td>2024-08-05</td>
          <td>Human Cell</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Yvette Hayat, Sarah Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, Emmanuelle Huillard, Nina Pottier, Aurore Desmons, Inès Fayache, Garrett A. Kaas, Philip J. Kingsley, Lawrence J. Marnett, Eric Duplus, Elias El-Habr, Lucas Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, Michele Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, Mirjam E Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Myelodysplastic syndrome and acute myeloid leukemia (AML) belong to a continuous disease spectrum of myeloid malignancies with poor prognosis in the relapsed/refractory setting necessitating novel therapies. Natural killer (NK) cells from patients with myeloid malignancies display global dysfunction with impaired killing capacity, altered metabolism, and an exhausted phenotype at the single-cell transcriptomic and proteomic levels. In this study, we identified that this dysfunction was mediated through a cross-talk between NK cells and myeloid blasts necessitating cell-cell contact. NK cell dysfunction could be prevented by targeting the αvβ-integrin/TGF-β/SMAD pathway but, once established, was persistent because of profound epigenetic reprogramming. We identified BATF as a core transcription factor and the main mediator of this NK cell dysfunction in AML. Mechanistically, we found that BATF was directly regulated and induced by SMAD2/3 and, in turn, bound to key genes related to NK cell exhaustion, such as HAVCR2, LAG3, TIGIT, and CTLA4. BATF deletion enhanced NK cell function against AML in vitro and in vivo. Collectively, our findings reveal a previously unidentified mechanism of NK immune evasion in AML manifested by epigenetic rewiring and inactivation of NK cells by myeloid blasts. This work highlights the importance of using healthy allogeneic NK cells as an adoptive cell therapy to treat patients with myeloid malignancies combined with strategies aimed at preventing the dysfunction by targeting the TGF-β pathway or BATF. BATF deletion prevents epigenetic rewiring and enhances the antitumor activity of NK cells in AML. Editor’s summary Patients with myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML), often relapse after standard of care. To improve outcomes, Kumar et al. have investigated the role of natural killer (NK) cells in antitumor immunity against myeloid blasts and have identified global dysfunction of NK cells in these patients. They identified BATF as responsible for this dysfunction, and by knocking out BATF downstream of the TGF-β canonical pathway in NK cells, they were able to enhance NK cell function against AML in vitro and in vivo. This has led to the initiation of a promising clinical trial using these TGFBR2 knockout NK cells against myeloid malignancies. —Dorothy Hallberg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1bf3fa2449127d2f41a09515bf651d31d92c80" target='_blank'>
              BATF is a major driver of NK cell epigenetic reprogramming and dysfunction in AML
              </a>
            </td>
          <td>
            Bijender Kumar, A. Singh, R. Başar, N. Uprety, Ye Li, Huihui Fan, Ana Karen Nunez Cortes, Mecit Kaplan, S. Acharya, Hila Shaim, Anna C Xu, Manrong Wu, Emily Ensley, Dexing Fang, P. Banerjee, L. M. Garcia, S. Tiberti, Paul Lin, H. Rafei, Maliha Nuzhat Munir, Madison Moore, M. Shanley, M. Mendt, L. Kerbauy, Bin Liu, Alexander Biederstädt, Elif Gokdemir, Susmita Ghosh, Kiran Kundu, Francia Reyes-Silva, X. Jiang, Xinhai Wan, April L. Gilbert, Merve Dede, V. Mohanty, J. Dou, Patrick Zhang, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Abhinav K. Jain, J. J. Rodríguez-Sevilla, S. Colla, Guillermo Garcia-Manero, Elizabeth J. Shpall, Ken Chen, Hussein A. Abbas, Kunal Rai, Katayoun Rezvani, M. Daher
          </td>
          <td>2024-09-11</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D'Amato, Yi Xue, Kristina Fontanez, Aaron A May-Zhang, Trinity Smithers, Yigal Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aging increases breast cancer risk while an early first pregnancy reduces a woman’s life-long risk. Several studies have explored the effect of either aging or pregnancy on mammary epithelial cells (MECs), but the combined effect of both remains unclear. Here, we interrogate the functional and transcriptomic changes at single cell resolution in the mammary gland of aged nulliparous and parous mice to discover that pregnancy normalizes age-related imbalances in lineage composition, while also inducing a differentiated cell state. Importantly, we uncover a minority population of Il33-expressing hybrid MECs with high cellular potency that accumulate in aged nulliparous mice but is significantly reduced in aged parous mice. Functionally, IL33 treatment of basal, but not luminal, epithelial cells from young mice phenocopies aged nulliparous MECs and promotes formation of organoids with Trp53 knockdown. Collectively, our study demonstrates that pregnancy blocks the age-associated loss of lineage integrity in the basal layer through a decrease in Il33+ hybrid MECs, potentially contributing to pregnancy-induced breast cancer protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3189ca70d80763a95a696e8eb95db69af76072ee" target='_blank'>
              Pregnancy Reduces Il33+ Hybrid Progenitor Accumulation in the Aged Mammary Gland
              </a>
            </td>
          <td>
            Andrew Olander, Cynthia M Ramirez, Veronica Haro Acosta, Paloma Medina, Sara Kaushik, Vanessa D. Jonsson, Shaheen S. Sikandar
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80a90a7fa373a0b5dc0cb49c622a37b283337d8" target='_blank'>
              Barriers and opportunities in pancreatic cancer immunotherapy
              </a>
            </td>
          <td>
            Yixin Ju, Dongzhi Xu, Miao-Miao Liao, Yutong Sun, Wen-Dai Bao, Fan Yao, Li Ma
          </td>
          <td>2024-09-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) originates from precursor lesions, the most common of which are known as pancreatic intraepithelial neoplasia (PanIN). PanIN formation and progression is accompanied by accumulation of a precursor microenvironment (PME) composed of desmoplastic stroma that plays an important role in promoting tumor growth and immune evasion. The crosstalk between tumor cells and stromal cells is an area of active investigation. While PanIN and the PME have been extensively studied in mouse models, their nature in humans have remained elusive as there is no clinical indication to sample healthy pancreas. Through a unique collaboration with the Gift of Life Organ and Tissue donation program, we performed transcriptomic analysis on normal pancreata declined for transplant. We noted that epithelial cells in the PanIN and cancer microenvironment were transcriptomically similar; however, their surrounding stromal fibroblasts were markedly distinct. Given that most precursor lesions remain quiescent while some progress to tumor, we sought out to elucidate how the transcriptional programs in epithelial cells influence fibroblast cell fate. To do so, we developed a 3D ex-vivo organoid-fibroblast coculture system using primary organoid and fibroblast cultures successfully derived from Gift of Life pancreata or PDAC patients undergoing surgical resection. Our coculture system retains histological features of heterogeneity observed in vivo. Using single cell RNA-sequencing analysis of matched fibroblast-organoid cocultures we identified factors that were highly expressed in fibroblasts in tumor cocultures, but not in normal cocultures. We are currently interrogating candidate ligand/receptor interactions in our cocultures to determine the mediators of this epithelial-mediated fibroblast polarization. This model will allow us to gain an understanding about how components of the microenvironment restrain or promote malignant transformation and serve as a great tool to study therapies targeting tumor and stromal cells in a patient specific manner.
 Citation Format: Padma Kadiyala, Ahmed Elhossiny, Jianhua Liu, Alexander Bray, Katelyn Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy Frankel, Eileen Carpenter, Marina Pasca Di Magliano. Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9841941d0df90af0a793863b27fe1e8b8a9497" target='_blank'>
              Abstract A057: Investigating Fibroblast-Epithelial crosstalk in pre-malignant and cancerous microenvironment
              </a>
            </td>
          <td>
            Padma Kadiyala, Ahmed M. Elhossiny, Jianhua Liu, Alexander W Bray, Katelyn L. Donahue, Arvind Rao, Jiaqi Shi, Yaqing Zhang, Filip Bednar, Timothy L. Frankel, Eileen S Carpenter, M. P. di Magliano
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Most patients with prostate adenocarcinoma develop resistance to therapies targeting the androgen receptor (AR). Consequently, a portion of these patients develop AR-independent neuroendocrine prostate cancer (NEPC), a rapidly progressing cancer with limited therapies and poor survival outcomes. Current research to understand the progression to NEPC suggests a model of lineage plasticity whereby AR-dependent luminal-like tumors progress towards an AR-independent NEPC state. Genetic analysis of human NEPC identified frequent loss of RB1 and TP53, and the loss of both genes in experimental models mediates the transition to a neuroendocrine lineage. Transcriptomics studies have shown that lineage transcription factors ASCL1 and NEUROD1 are present in NEPC. In this study, we modeled the progression of prostate adenocarcinoma to NEPC by establishing prostate organoids and subsequently generating subcutaneous allograft tumors from genetically-engineered mouse models harboring Cre-induced loss of Rb1 and Trp53 with Myc overexpression (RPM). These tumors were heterogeneous and displayed adenocarcinoma, squamous, and neuroendocrine features. ASCL1 and NEUROD1 were expressed within neuroendocrine-defined regions, with ASCL1 being predominant. Genetic loss of Ascl1 in this model did not decrease tumor incidence, growth, or metastasis; however, there was a notable decrease in neuroendocrine identity and an increase in basal-like identity. This study provides an in vivo model to study progression to NEPC and establishes the requirement for ASCL1 in driving neuroendocrine differentiation in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b06924a4ca0fcd63effebd03ee725e6bf5e8759" target='_blank'>
              Neuroendocrine Differentiation in Prostate Cancer Requires ASCL1.
              </a>
            </td>
          <td>
            Kathia E Rodarte, Shaked Nir Heyman, Lei Guo, Lydia Flores, T. Savage, Juan Villarreal, Su Deng, Lin Xu, Rajal B Shah, T. Oliver, Jane E Johnson
          </td>
          <td>2024-09-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa682eab050e4a15f2145e68ead15052709fd5de" target='_blank'>
              SMARCA4 controls state plasticity in small cell lung cancer through regulation of neuroendocrine transcription factors and REST splicing
              </a>
            </td>
          <td>
            E. Redin, Harsha Sridhar, Y. Zhan, Barbara Pereira Mello, Hong Zhong, Vidushi Durani, Amin Sabet, P. Manoj, I. Linkov, Juan Qiu, R. Koche, E. de Stanchina, Maider Astorkia, D. Betel, Á. Quintanal-Villalonga, Charles M. Rudin
          </td>
          <td>2024-07-30</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, Philip Maini, Tyler Cassidy
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, Philip Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b7af48216a0c84346ea77c18dc911a1e93afbf" target='_blank'>
              Endothelial cell heterogeneity in colorectal cancer: tip cells drive angiogenesis
              </a>
            </td>
          <td>
            Zhenyu Xie, Liaoran Niu, Kunli Du, Ling Chen, Gaozan Zheng, Songchen Dai, Han-Long Dan, Lili Duan, Xinyu Dou, Fan Feng, Jian Zhang, Jianyong Zheng
          </td>
          <td>2024-08-22</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Background: Gastrointestinal cancer cells display both morphology and physiology diversity, thus posing a significant challenge for precise representation by a single data model. We conducted an in-depth study of gastrointestinal cancer heterogeneity by integrating and analyzing data from multiple modalities. Methods: We used a modified Canny algorithm to identify edges from tumor images, capturing intricate nonlinear interactions between pixels. These edge features were then combined with differentially expressed mRNA, miRNA, and immune cell data. Before data integration, we used the K-medoids algorithm to pre-cluster individual data types. The results of pre-clustering were used to construct the kernel matrix. Finally, we applied spectral clustering to the fusion matrix to identify different tumor subtypes. Furthermore, we identified hub genes linked to these subtypes and their biological roles through the application of Weighted Gene Co-expression Network Analysis (WGCNA) and Gene Ontology (GO) enrichment analysis. Results: Our investigation categorized patients into three distinct tumor subtypes and pinpointed hub genes associated with each. Genes MAGI2-AS3, MALAT1, and SPARC were identified as having a differential impact on the metastatic and invasive capabilities of cancer cells. Conclusion: By harnessing multimodal features, our study enhances the understanding of gastrointestinal tumor heterogeneity and identifies biomarkers for personalized medicine and targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c68f2f427231b32eddf4811a28d710e5bb1cf" target='_blank'>
              Tumor Heterogeneity in Gastrointestinal Cancer Based on Multimodal Data Analysis
              </a>
            </td>
          <td>
            Dongmei Ai, Yang Du, Hongyu Duan, Juan Qi, Yuduo Wang
          </td>
          <td>2024-09-13</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Despite extensive molecular stratification efforts, improvements in molecularly informed RT-based therapeutic strategies in the neoadjuvant setting have been limited. This study aims to elucidate the adaptive mechanisms and interactions within PDAC to identify new combinatorial therapeutic targets with RT. MethodsWe constructed a high-resolution molecular landscape of the cellular subtypes and spatial communities that constitute PDAC. Single-cell RNA sequencing (scRNA- seq) was performed on 16 PDAC biopsies of matched pre- and post-treatment samples from a Phase I/II dose-escalation clinical trial of pancreatic stereotactic body radiation therapy (SBRT). Additionally, Visium spatial transcriptomics (ST) was conducted on biopsies from primary PDAC patients collected at the time of surgery. These patients received either no neoadjuvant therapy (n = 8) or neoadjuvant chemoradiation (chemoRT) (n = 20). Cell type profiles were determined from scRNA-seq to compare changes in cell type proportions and signatures before and after RT. From these profiles, we employed robust cell type decomposition to analyze cell type signatures in our ST dataset. ResultsChemoRT induced several significant effects compared to controls within the tumor microenvironment, including an increase in the proportion of myofibroblasts, specifically senescent cancer-associated fibroblasts (CAFs), with signatures consistent with immune cell dysfunction. We observed an increase in CD8 and CD4 effector memory cells with RT; however, this occurred alongside a rise in immunosuppressive alternatively activated tumor-associated macrophages. Additionally, RT upregulated pathways related to HIF, cytokine production, B cell proliferation, and negative regulation of lymphocytes. Patients with increased infiltration of NK, memory B, and CD8 effector memory cells demonstrated improved overall survival outcomes. Inference of copy number variations enabled us to identify cancer subclones as either responsive or resistant in matched pre and post RT biopsies. Resistance was characterized by augmented hypoxia, KRAS activation, epithelial- mesenchymal transition, and interferon-gamma response signaling. Conversely, responders exhibited increased oxidative phosphorylation signaling and pathway enrichment associated with cell proliferation and cell cycle progression. Interestingly, RT induced a decrease in cancer cells characterized as harboring a chemoresistant basal-like molecular subtype. ConclusionThrough both longitudinal and therapy stratified tumor biopsies, we were able to observe significant selection pressures on stromal and cancer cells following RT. This includes upregulation of innate and adaptive immune pathways with a compensatory immunosuppressive response driven by myeloid cells and CAFs. This may provide opportunities for development of clinical trials to therapeutically target specific cellular phenotypes and their interactions.
 Citation Format: Vincent Bernard, Galia Jacobson, Kimal Rajapakshe, Jimin Min, Guangsheng Pei, Daniel Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, Manoop S Bhutani, Linghua Wang, Paola A Guerrero, Ethan B Ludmir, Albert C Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J Koay. A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee951a6470ee1fe16a77439ce9fbdb87aac3a94c" target='_blank'>
              Abstract B078: A comprehensive spatial atlas of neoadjuvant chemoradiation therapy in resected pancreatic cancer identifies cellular & microenvironmental determinants of persister populations
              </a>
            </td>
          <td>
            V. Bernard, Galia Jacobson, K. Rajapakshe, Jimin Min, Guangsheng Pei, D. Lin, Ayush Suresh, Dadi Jiang, Ching-Wei Tzeng, M. Bhutani, Linghua Wang, Paola A Guerrero, E. Ludmir, Albert C. Koong, Anirban Maitra, Cullen M Taniguchi, Eugene J Koay
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinomas (RCC) are characterized by their largely diverse clinical outcomes. Leveraging genomic diversity among individual tumors, our research during the past decade has concentrated on developing personalized medicine approaches for the most common and the most aggressive type of RCC, clear cell renal carcinoma (ccRCC). We generated the largest dataset of ccRCC including somatic genomic and clinical annotations for over 940 ccRCC patients, and developed a genomic classifier, based on mutational status of 12 RCC-relevant genes, which is able to stratify patients according to their risk of relapse after nephrectomy and/or death due to RCC. Furthermore, we noticed the presence of mesenchymal-like cellular phenotypes in tumors of high-risk patients, representsing a de-differentiation process, known as sarcomatoic or rhabdoid (S/R) de-differentation. Methods The current knowledge about molecular mechanisms that may drive S/R de-differentiation is primarily generated from molecular profiling of bulk tumors, collecting data from a mixture of cells that co-exist in the tumor milieu. Therefore, high-resolution studies that precisely define molecular characteristics of different cellular phenotypes associated with S/R features are missing. We have used spatial transcriptomic profiling to investigate molecular mechanisms that underline S/R de-differentationin RCC. While preserving tissue context, we applied spatial whole human transcriptome profiling to areas exhibiting S/R or clear cell phenotypes within the same tumor specimens to generate phenotype-specific transcriptome profiles. In addition, we applied whole-exome sequencing (WES) to independent areas that exhibit different differentiation phenotype withing same tumors. Results Pathway and network analysis of genesets with upregulation in each area have revealed meaningful differences in cellular pathways which are active in each phenotype. Specifically, we observed that dysregulation of extracellular matrix (ECM) is a hallmark of S/R areas within RCC tumors. Furthermore, WES of phenotypically-distinct areas within each tumor has shed new light son the genome evoltuion of S/R phenotypes. Conclusions S/R dedifferentiation in RCC tumors is characterized by specific genomic evolutionary patterns and substantial dysregulation of ECM components. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad0d8a5a2c80baecbe7aa769327da96aacd5a05" target='_blank'>
              9 Delving into molecular underpinnings of sarcomatoid/rhabdoid dedifferentioation in renal cancers using spatial profiling
              </a>
            </td>
          <td>
            Y. Riazalhosseini
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Background Current diagnostic tools are unable to distinguish low‐grade indolent prostate cancer (PrCa) from that with a propensity to become metastatic and/or lethal. Recent evidence suggests that reprogramming of the transcriptome may drive the metastatic phenotype, and that this reprogramming is controlled, at least in part, by epigenetic changes to the DNA of cancer cells, including methylation. These changes, referred to as ‘epigenetic drivers,’ have previously been associated with cancer cell survival. Methods Here, using Illumina Methylation EPIC array data of paired primary PrCa and metastatic bone samples, we identified WNT5A as a putative epi‐driver of PrCa metastasis to the bone, which was further validated in vitro. Results Significantly higher WNT5A methylation was observed in primary PrCa samples and 22Rv1 cells compared to metastatic bone samples and PC‐3 cells. This higher methylation was associated with significantly lower WNT5A gene expression. Conclusion Given the limited effective therapies available for metastatic cancer sufferers, particularly those whose disease has metastasised to the bone, WNT5A presents as a potential putative target for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b055e884a7d4d2969ce7c73d7f93b2c051be4c83" target='_blank'>
              WNT5A is a putative epi‐driver of prostate cancer metastasis to the bone
              </a>
            </td>
          <td>
            Emma J Wilkinson, Kelsie Raspin, R. Malley, S. Donovan, Louise M Nott, Adele F Holloway, Joanne L Dickinson
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6640869137efbf815e49406335626c0344fac69" target='_blank'>
              Age-associated clonal B cells drive B cell lymphoma in mice.
              </a>
            </td>
          <td>
            José P Castro, A. Shindyapina, Alessandro Barbieri, Kejun Ying, Olga S. Strelkova, Joao A. Paulo, A. Tyshkovskiy, Rico Meinl, C. Kerepesi, Anna P. Petrashen, Marco Mariotti, Margarita V Meer, Yan Hu, Alexander Karamyshev, Grigoriy Losyev, Mafalda Galhardo, Elsa Logarinho, Artur A. Indzhykulian, Steven P. Gygi, J. M. Sedivy, John P Manis, V. N. Gladyshev
          </td>
          <td>2024-08-08</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Motivation The combination of single-cell sequencing with ligand-receptor analysis paves the way for the characterization of cell communication events in complex tissues. In particular, directed weighted graphs stand out as a natural representation of cell-cell communication events. However, current computational methods cannot analyze sample-specific cell-cell communication events, as measured in single-cell data produced in large patient cohorts. Cohort-based cell-cell communication analysis presents many challenges, such as the non-linear nature of cell-cell communication and the high variability presented by the patient-specific single-cell RNAseq datasets. Results Here, we present scACCorDiON (single-cell Analysis of Cell-Cell Communication in Disease clusters using Optimal transport in Directed Networks), an optimal transport algorithm exploring node distances on the Markov Chain as the ground metric between directed weighted graphs. Additionally, we derive a k-barycenter algorithm using the Wasserstein-based distance, which is able to cluster directed weighted graphs. We compare our approach with competing methods in several large cohorts of scRNA-seq data. Our results show that scACCorDiON can predict clusters better, matching the disease status of samples. Moreover, we show that barycenters provide a robust and explainable representation of cell cell communication events related to the detected clusters. We also provide a case study of pancreas adenocarcinoma, where scACCorDion detects a sub-cluster of disease samples associated with changes in the tumor microenvironment. Availability The code of scACCorDiON is available at https://scaccordion.readthedocs.io/en/latest Contact ivan.costa@rwth-aachen.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12408229b87ae8ceac3d6b2906ea4bd12adc6c6e" target='_blank'>
              scACCorDiON: A clustering approach for explainable patient level cell cell communication graph analysis
              </a>
            </td>
          <td>
            J. Nagai, Michael T. Schaub, Ivan G.Costa
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b983ff02239e55a5a088e11ea44463f2a191dc" target='_blank'>
              Spatial computational modelling illuminates the role of the tumour microenvironment for treating glioblastoma with immunotherapies
              </a>
            </td>
          <td>
            Blanche Mongeon, Julien Hébert-Doutreloux, Anudeep Surendran, Elham Karimi, B. Fiset, D. Quail, Logan A Walsh, A. Jenner, Morgan Craig
          </td>
          <td>2024-08-18</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e0df7d22a45444a437c740c4db84a4c490aa49" target='_blank'>
              Membrane RRM2-positive cells represent a malignant population with cancer stem cell features in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Yongzhi Zhao, Shuting Xue, Danduo Wei, Jianjuan Zhang, Nachuan Zhang, Liping Mao, Niya Liu, Lei Zhao, Jianing Yan, Yifan Wang, Xiujun Cai, Saiyong Zhu, Stephanie Roessler, Junfang Ji
          </td>
          <td>2024-09-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b528f1c27b39c2d67f13d20607910c504ce5735" target='_blank'>
              Spatial proteomic profiling elucidates immune determinants of neoadjuvant chemo-immunotherapy in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Chao Wu, Guoqing Zhang, Lin Wang, Jinlong Hu, Zhongjian Ju, H. Tao, Qing Li, Jian Li, Wei Zhang, Jianpeng Sheng, Xiaobin Hou, Yi Hu
          </td>
          <td>2024-08-09</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, M. Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16df671c98a37a9e428d4261d351be144c2e34b" target='_blank'>
              Monitoring imatinib decreasing pericyte coverage and HIF-1α level in a colorectal cancer model by an ultrahigh-field multiparametric MRI approach
              </a>
            </td>
          <td>
            Xinpeng Hu, Kunlin Ye, Shaowei Bo, Zeyu Xiao, Mengjie Ma, Jinghua Pan, Xing Zhong, Dong Zhang, Xukai Mo, Xiaojun Yu, Minfeng Chen, L. Luo, Changzheng Shi
          </td>
          <td>2024-07-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d1e5e2f8fff698fa2e7aaaa422dd95deef960f" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis identifies a prognostic T-cell signature in colorectal cancer
              </a>
            </td>
          <td>
            Peng Cui, Haibo Wang, Zhigang Bai
          </td>
          <td>2024-08-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dysregulation of the mitogen-activated protein kinase interacting kinases 1/2 (MNK1/2)–eukaryotic initiation factor 4E (eIF4E) signaling axis promotes breast cancer progression. MNK1 is known to influence cancer stem cells (CSCs); self-renewing populations that support metastasis, recurrence, and chemotherapeutic resistance, making them a clinically relevant target. The precise function of MNK1 in regulating CSCs, however, remains unexplored. Here, we generated MNK1 knockout cancer cell lines, resulting in diminished CSC properties in vitro and slowed tumor growth in vivo. Using a multiomics approach, we functionally demonstrated that loss of MNK1 restricts tumor cell metabolic adaptation by reducing glycolysis and increasing dependence on oxidative phosphorylation. Furthermore, MNK1-null breast and pancreatic tumor cells demonstrated suppressed metastasis to the liver, but not the lung. Analysis of The Cancer Genome Atlas (TCGA) data from breast cancer patients validated the positive correlation between MNK1 and glycolytic enzyme protein expression. This study defines metabolic perturbations as a previously unknown consequence of targeting MNK1/2, which may be therapeutically exploited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd46eff0abc3612dd21dd74ba5f7acbfb0e77403" target='_blank'>
              Blocking tumor-intrinsic MNK1 kinase restricts metabolic adaptation and diminishes liver metastasis
              </a>
            </td>
          <td>
            Samuel E. J. Preston, Michael S. Dahabieh, Raúl Ernesto Flores González, Christophe Gonçalves, Vincent R. Richard, M. Leibovitch, Eleanor Dakin, Theodore Papadopoulos, Carolina Lopez Naranjo, Paige McCallum, F. Huang, N. Gagnon, Stephanie Perrino, R. Zahedi, Christoph H Borchers, Russell G. Jones, P. Brodt, Wilson H Miller, Sonia V. del Rincón
          </td>
          <td>2024-09-13</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The transcription factor achaete-scute complex homolog 1 (ASCL1) is a lineage oncogene that is central in growth and survival of the majority of small cell lung cancers (SCLC) and neuroendocrine non-small cell lung cancers (NSCLC-NE) that express it. Targeting ASCL1, or its downstream pathways, remains a challenge. SCLCs and NSCLC-NE that express ASCL1 exhibit relatively low ERK1/2 activity, in dramatic contrast to NSCLCs in which the ERK pathway has a major role in pathogenesis. ERK1/2 inhibition in ASCL1-expressing lung tumor cells revealed down-regulation of ERK1/2 pathway suppressors SPRY4, SPRED1, DUSP6, and the transcription factor ETV5, which regulates DUSP6. CHIP-seq demonstrated that these genes are bound by ASCL1. Availability of a pharmacological inhibitor directed mechanistic studies towards DUSP6, an ERK1/2-selective phosphatase, in a subset of ASCL1-high NE lung tumors. Inhibition of DUSP6 increased active ERK1/2, which accumulated in the nucleus. Pharmacologic and genetic inhibition of DUSP6 reduced proliferation and survival of these cancers. Resistance developed in DUSP6 KO cells, indicating a bypass mechanism. Although targeting ASCL1 remains a challenge, our findings suggest that expression of ASCL1, DUSP6 and low phospho-ERK1/2 identify neuroendocrine lung cancers for which DUSP6 may be a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f8d9d7ace7366903d7b9e7f385544cb77bfd1b" target='_blank'>
              ASCL1 restrains ERK1/2 to promote survival of a subset of neuroendocrine lung cancers.
              </a>
            </td>
          <td>
            Ana Martín-Vega, Svetlana A Earnest, Alexander Augustyn, Chonlarat Wichaidit, Luc Girard, M. Peyton, John D Minna, Jane E Johnson, Melanie H. Cobb
          </td>
          <td>2024-09-19</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Osteosarcoma is an aggressive form of bone cancer that spreads quickly and is challenging to treat. Genetic screening in this study discovered that the loss of the Arid1a gene is a crucial factor in the development and progression of osteosarcoma. In vitro and in vivo studies validated that the loss of Arid1a increased proliferation and chemoresistance in human cell lines and led to higher disease penetrance, metastases, and shorter survival in mice. Unbiased pathway analyses suggest that Arid1a may contribute to the aggressiveness of osteosarcoma by dysregulating genomic instability. Abstract Osteosarcoma is an aggressive bone malignancy, molecularly characterized by acquired genome complexity and frequent loss of TP53 and RB1. Obtaining a molecular understanding of the initiating mutations of osteosarcomagenesis has been challenged by the difficulty of parsing between passenger and driver mutations in genes. Here, a forward genetic screen in a genetic mouse model of osteosarcomagenesis initiated by Trp53 and Rb1 conditional loss in pre-osteoblasts identified that Arid1a loss contributes to OS progression. Arid1a is a member of the canonical BAF (SWI/SNF) complex and a known tumor suppressor gene in other cancers. We hypothesized that the loss of Arid1a increases the rate of tumor progression and metastasis. Phenotypic evaluation upon in vitro and in vivo deletion of Arid1a validated this hypothesis. Gene expression and pathway analysis revealed a correlation between Arid1a loss and genomic instability, and the subsequent dysregulation of genes involved in DNA DSB or SSB repair pathways. The most significant of these transcriptional changes was a concomitant decrease in DCLRE1C. Our findings suggest that Arid1a plays a role in genomic instability in aggressive osteosarcoma and a better understanding of this correlation can help with clinical prognoses and personalized patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84d420fc77f5594ad3093563dc03dd9bc7eaa72d" target='_blank'>
              Arid1a Loss Enhances Disease Progression in a Murine Model of Osteosarcoma
              </a>
            </td>
          <td>
            Kaniz Fatema, Yanliang Wang, Adriene Pavek, Zachary Larson, Christopher Nartker, Shawn Plyler, Amanda Jeppesen, Breanna Mehling, Mario R. Capecchi, Kevin B Jones, Jared J. Barrott
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Developing accurate in vitro models that replicate the in vivo tumor environment is essential for advancing cancer research and therapeutic development. Traditional 2D cell cultures often fail to capture the complex structural and functional heterogeneity of tumors, limiting the translational relevance of findings. In contrast, 3D culture systems, such as spheroids, provide a more physiologically relevant context by replicating key aspects of the tumor microenvironment. This study aimed to compare the metabolism of three intrahepatic cholangiocarcinoma cell lines in 2D and 3D cultures to identify metabolic shifts associated with spheroid formation. Cells were cultured in 2D on adhesion plates and in 3D using ultra-low attachment plates. Metabolic exchange rates were measured using NMR, and intracellular metabolites were analyzed using LC-MS. Significant metabolic differences were observed between 2D and 3D cultures, with notable changes in central carbon and glutathione metabolism in 3D spheroids. The results suggest that 3D cultures, which more closely mimic the in vivo environment, may offer a more accurate platform for cancer research and drug testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d509b84c812a08dde0b9446f0ac93f32c85c3a96" target='_blank'>
              3D Modeling: Insights into the Metabolic Reprogramming of Cholangiocarcinoma Cells
              </a>
            </td>
          <td>
            Giorgia Ciufolini, Serena Zampieri, Simona Cesaroni, Valentina Pasquale, M. Bonanomi, D. Gaglio, E. Sacco, Marco Vanoni, M. Pastore, F. Marra, D. O. Cicero, C. Raggi, G. Petrella
          </td>
          <td>2024-09-13</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63ccd845ca415506eac680283d46074762ee2fa" target='_blank'>
              Orai1 and Orai3 act through distinct signalling axes to promote stemness and tumorigenicity of breast cancer stem cells
              </a>
            </td>
          <td>
            Duan Zhuo, Zhenchuan Lei, Lin Dong, Andrew Man Lok Chan, Jiacheng Lin, Liwen Jiang, Beibei Qiu, Xiaohua Jiang, Youhua Tan, Xiaoqiang Yao
          </td>
          <td>2024-08-13</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Intermediate cell carcinoma (Int-CA) is a rare and enigmatic primary liver cancer characterized by uniform tumor cells exhibiting mixed features of both HCC and intrahepatic cholangiocarcinoma. Despite the unique pathological features of int-CA, its molecular characteristics remain unclear yet. Methods: RNA sequencing and whole genome sequencing profiling were performed on int-CA tumors and compared with those of HCC and intrahepatic cholangiocarcinoma. Results: Int-CAs unveiled a distinct and intermediate transcriptomic feature that is strikingly different from both HCC and intrahepatic cholangiocarcinoma. The marked abundance of splicing events leading to intron retention emerged as a signature feature of int-CA, along with a prominent expression of Notch signaling. Further exploration revealed that METTL16 was suppressed within int-CA, showing a DNA copy number–dependent transcriptional deregulation. Notably, experimental investigations confirmed that METTL16 suppression facilitated invasive tumor characteristics through the activation of the Notch signaling cascade. Conclusions: Our results provide a molecular landscape of int-CA featured by METTL16 suppression and frequent intron retention events, which may play pivotal roles in the acquisition of the aggressive phenotype of Int-CA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab7b9d18c0e5e1be8d12c7361eff07994e2e2a" target='_blank'>
              Transcriptomic profiling of intermediate cell carcinoma of the liver
              </a>
            </td>
          <td>
            Byungchan Jang, S. Kwon, Jang Hyun Kim, Jung Mo Kim, Taek Chung, Jeong Eun Yoo, Haeryoung Kim, Julien Calderaro, Hyun Goo Woo, Young Nyun Park
          </td>
          <td>2024-08-01</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immunotherapy response is associated with the presence of conventional dendritic cells (cDCs). cDC type 1 (cDC1) is critically important for CD8+ T cell activation, cDC type 2 (cDC2) regulates CD4+ T cell responses, and mature regulatory cDCs may dampen T cell responses in the tumor microenvironment (TME). However, we lack a clear understanding of cDC distribution in the human TME, cDC prevalence in metastatic sites, and cDC differences in early- versus late-stage disease. Rapid autopsy specimens of 10 patients with lung adenocarcinoma were evaluated to detect cDCs and immune cells via multiplex immunofluorescence using 18 markers and 42 tumors. First, we found that T cells, cDC1, and cDC2 were confined to stroma, whereas mature regulatory DCs were enriched in tumor, suggesting unique localization-specific functions. Second, lung and lymph node tumors were more enriched in T cells and cDCs than liver tumors, underscoring differences in the TME of metastatic sites. Third, although the proportion of T cells and cDC1 did not differ in different stages, an increase in the proportion of cDC2 and macrophages in late stage suggests potential differences in regulation of T cell responses in different stages. Collectively, these findings provide new, to our knowledge, insights into cDC biology in human cancer that may have important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8512ba6cd3679e5a0f0f3d1f8e432ba86900fbeb" target='_blank'>
              Rapid Autopsy to Define Dendritic Cell Spatial Distribution and T Cell Association in Lung Adenocarcinoma.
              </a>
            </td>
          <td>
            Hilal Ozakinci, Xiaofei Song, Gina S Nazario, Thomas Lila, Benjamin J Chen, Tyler Simpson, Jonathan V Nguyen, Carlos M Moran Segura, Zachary Thompson, Ram Thapa, Trevor A Rose, E. Haura, B. Pellini, Xiaoqing Yu, B. Ruffell, Dung-tsa Chen, Theresa A. Boyle, Amer A Beg
          </td>
          <td>2024-08-09</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a8cbaf63e3462f27923725db4d9a627507061cb" target='_blank'>
              Radiogenomic profiling of global DNA methylation associated with molecular phenotypes and immune features in glioma
              </a>
            </td>
          <td>
            Z. Zhuang, Jinxin Lin, Zixiao Wan, Jingrong Weng, Ze Yuan, Yumo Xie, Zongchao Liu, Peiyi Xie, Siyue Mao, Zongming Wang, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2024-09-02</td>
          <td>BMC Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b21391e9d7b4e36087d981b4ffc2b93179db8ea" target='_blank'>
              Combined microfluidic enrichment and staining workflow for single-cell analysis of circulating tumor cells in metastatic prostate cancer patients
              </a>
            </td>
          <td>
            Caroline Løppke, Amalie M Jørgensen, Nicolai T. Sand, Rasmus B Klitgaard, G. Daugaard, Mette Ø. Agerbæk
          </td>
          <td>2024-07-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Malignant rhabdoid tumors (MRT) are rare aggressive tumors of infancy characterized by the biallelic inactivation of SMARCB1. Their lineage of origin remains uncertain, but increasing evidence suggest a neural-crest origin. By analyzing the transcriptome of MRT occurring in peripheral and cranial nerves, we observe that their expression profiling was not distinct from any other tumor localization. We therefore investigated whether the development of Smarcb1-deficient tumors emerging from embryonic precursors of peripheral nerves in mice could recapitulate human MRT. we generated conditional Smarcb1 knockout in the Prss56-Cre mouse strain thereby targeting boundary cap cells which are neural crest derived Schwann cell precursor. All Smarcb1Flox/Flox;Prss56Cre+/+ showed ataxia within a median of two months, due to the development of massive intracranial tumor in the ponto-cerebellum angle. Smarcb1Flox/Flox;Prss56Cre+/- mainly developed paraplegia related to nerve root tumors, within a median delay of 7 months. Interestingly, transcriptome profiling confirmed the good correlation between mouse peripheral nerve tumors and human MRT; intracranial tumors in this new model correlated with MYC-ATRTs but with some specific features not observed in their human counterparts. All tumors harbored a significant immune infiltration, with some clonal T-cell expansion. Altogether, the Smarcb1Flox/Flox;Prss56-Cre mouse tumors offer a new model for human MRT and promising perspectives for immune-based innovative treatments.
 Citation Format: Zhi-Yan HAN, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, Arnault Tauziède-Espariat, Kevin Beccaria, Julien Masliah-Planchon, Christine Bourneix, Delphine Guillemot, Stéphanie Reynaud, Gaelle Pierron, Maëva Veyssiere, Owen Hoare, Céline Chauvin, Grégory Thomson, Didier Surdez, Pascale Varlet, Christelle Dufour, Volodia Dangouloff-Ros, Joshua J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, Franck Bourdeaut. Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624aa14765b3c481a1fc38ac19c8ad7677137eda" target='_blank'>
              Abstract A006 Targeting boundary cap cells leads to mouse Smarcb1-deficient peripheral nerve tumors recapitulating human peripheral rhabdoid tumors
              </a>
            </td>
          <td>
            Zhi-Yan Han, Mamy Andrianteranagna, Stéphanie Fitte-Duval, Valeria Manriquez, Magali Frah, Rachida Bouarich, Sandrina Turcynski, A. Tauziède-Espariat, K. Beccaria, J. Masliah-Planchon, C. Bourneix, D. Guillemot, S. Reynaud, G. Pierron, Maëva Veyssiere, Owen Hoare, C. Chauvin, Grégory Thomson, D. Surdez, Pascale Varlet, Christelle Dufour, V. Dangouloff-Ros, J. Waterfall, Eliane Piaggio, Olivier Delattre, Piotr Topilko, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim U. Rashid, Gary L. Johnson, J. J. Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 21],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>