<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-12 06:29:39 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E. Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, , F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, , M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) technologies, like GeoMx Digital Spatial Profiler, are increasingly utilized to reveal the role of diverse tumor microenvironment components, particularly in relation to cancer progression, treatment response, and therapeutic resistance. However, in many ST studies, the spatial information obtained from immunofluorescence imaging is primarily used for identifying regions of interest, rather than as an integral part of downstream transcriptomic data interpretation. We developed ROICellTrack, a deep learning-based framework, to better integrate cellular imaging with spatial transcriptomic profiling. By examining 56 ROIs from urothelial carcinoma of the bladder (UCB) and upper tract urothelial carcinoma (UTUC), ROICellTrack accurately identified cancer-immune mixtures and associated cellular morphological features. This approach also revealed different sets of spatial clustering patterns and receptor-ligand interactions. Our findings underscore the importance of combining imaging and transcriptomics for comprehensive spatial omics analysis, offering potential new insights into within-sample heterogeneity and implications for targeted therapies and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92bbcac98e9a080d35c5feacfadc1d55b66b9ab7" target='_blank'>
              Utility of cellular imaging modality in subcellular spatial transcriptomic profiling of tumor tissues
              </a>
            </td>
          <td>
            Xiaofei Song, Xiaoqing Yu, Carlos M. Moran-Segura, G. D. Grass, Roger Li, Xuefeng Wang
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The tumor microenvironment (TME) of medulloblastoma (MB) influences progression and therapy response, presenting a promising target for therapeutic advances. Prior single-cell analyses have characterized the cellular components of the TME but lack spatial context. To address this, we performed spatial transcriptomic sequencing on sixteen pediatric MB samples obtained at diagnosis, including two matched diagnosis-relapse pairs. Our analyses revealed inter- and intra-tumoral heterogeneity within the TME, comprised of tumor-associated astrocytes (TAAs), macrophages (TAMs), stromal components, and distinct subpopulations of MB cells at different stages of neuronal differentiation and cell cycle progression. We identified dense regions of quiescent progenitor-like MB cells enriched in patients with high-risk (HR) features and an increase in TAAs, TAMs, and dysregulated vascular endothelium following relapse. Our study presents novel insights into the spatial architecture and cellular landscape of the medulloblastoma TME, highlighting spatial patterns linked to HR features and relapse, which may serve as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b39423470039d57ba772c250043854baef4821" target='_blank'>
              Medulloblastoma Spatial Transcriptomics Reveals Tumor Microenvironment Heterogeneity with High-Density Progenitor Cell Regions Correlating with High-Risk Disease
              </a>
            </td>
          <td>
            Franklin Chien, Marina E. Michaud, Mojtaba Bakhtiari, Chanel Schroff, M. Snuderl, José E. Velázquez Vega, Tobey J. MacDonald, Manoj K. Bhasin
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) presents significant challenges due to its heterogeneity and late-stage diagnoses. Using single-cell and spatial transcriptomics to elucidate the complex landscape of HGSOC to understand its underlying mechanism. Our analysis reveals significant inter- and intra-tumoral diversity, manifested through distinct cellular subpopulations and varied microenvironmental niches. Notably, our findings highlight a widespread immunosuppressive environment, marked by complex networks of cell-cell interactions, particularly evident in areas of elevated tumor cell density within metastatic samples. We identify the exclusive presence of COL14A1+ neoplastic cells in metastatic specimens, alongside a strong correlation between CD8A+ NKT cells and poor prognosis, and elevated CHODL expression in HGSOC metastasis tissues. Furthermore, knockdown experiments targeting CHODL demonstrate its role in reducing migration and invasion abilities in HGSOC cells. A pivotal discovery of our study is the delineation of specific cellular signatures correlated with adverse outcomes, notably a subset of CHODL+ neoplastic cells characterized by a distinct metabolic phenotype with a predilection for lipid metabolism. The therapeutic targeting of this metabolic pathway with existing inhibitors appears promising in curbing tumor proliferation. These findings enhance our understanding of HGSOC heterogeneity and reveal potential therapeutic targets, promising more effective management strategies for this aggressive cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ff004a52301690996ef94c99200186510d19e37" target='_blank'>
              Integrating single-cell and spatial transcriptomic analysis to unveil heterogeneity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Haixia Luo, Kunyu Wang, Bin Li
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d769c8e2a874add71e222eabf3bcfb7177fb2c04" target='_blank'>
              Spatial iTME analysis of KRAS mutant NSCLC and immunotherapy outcome
              </a>
            </td>
          <td>
            Dan Zhao, Haiqing Li, Isa Mambetsariev, T. Mirzapoiazova, Chen Chen, Jeremy Fricke, Deric Wheeler, Leonidas Arvanitis, Raju Pillai, M. Afkhami, Bihong T. Chen, Martin Sattler, L. Erhunmwunsee, E. Massarelli, Prakash Kulkarni, A. Amini, Brian Armstrong, Ravi Salgia
          </td>
          <td>2024-06-19</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb8ba6a424922fdea28451ef81ee4284340b35a6" target='_blank'>
              Advances in spatial transcriptomics and its applications in cancer research
              </a>
            </td>
          <td>
            Yang Jin, Yuanli Zuo, Gang Li, Wenrong Liu, Yitong Pan, Ting Fan, Xin Fu, Xiaojun Yao, Yong Peng
          </td>
          <td>2024-06-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The patterns by which primary tumors spread to metastatic sites remain poorly understood. Here, we define patterns of metastatic seeding in prostate cancer (PCa) using a novel injection-based mouse model - EvoCaP (Evolution in Cancer of the Prostate), featuring aggressive metastatic cancer to bone, liver, lungs, and lymph nodes. To define migration histories between primary and metastatic sites, we used our EvoTraceR pipeline to track distinct tumor clones containing recordable barcodes. We detected widespread intratumoral heterogeneity from the primary tumor in metastatic seeding, with few clonal populations (CPs) instigating most migration. Metastasis-to-metastasis seeding was uncommon, as most cells remained confined within the tissue. Migration patterns in our model were congruent with human PCa seeding topologies. Our findings support the view of metastatic PCa as a systemic disease driven by waves of aggressive clones expanding their niche, infrequently overcoming constraints that otherwise keep them confined in the primary or metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7ba92880acaf1b934f559d90646f82a1f001d3" target='_blank'>
              Clonal Lineage Tracing with Somatic Delivery of Recordable Barcodes Reveals Migration Histories of Metastatic Prostate Cancer.
              </a>
            </td>
          <td>
            Ryan N Serio, Armin Scheben, Billy Lu, Domenic V Gargiulo, Lucrezia Patruno, Caroline L Buckholtz, Ryan J Chaffee, Megan C Jibilian, Steven G Persaud, Stephen J. Staklinski, Rebecca Hassett, Lise M Brault, Daniele Ramazzotti, Christopher E Barbieri, Adam Siepel, Dawid G Nowak
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99dcce816f6a19dca7baf05569c687c99b3105e9" target='_blank'>
              Unbiasedly decoding the tumor microenvironment with single-cell multiomics analysis in pancreatic cancer
              </a>
            </td>
          <td>
            Yifan Fu, Jinxin Tao, Tao Liu, Yueze Liu, J. Qiu, Dan Su, Ruobing Wang, W. Luo, Zhe Cao, Guihu Weng, Taiping Zhang, Yupei Zhao
          </td>
          <td>2024-07-09</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The current challenge in analyzing sequencing-based spatial transcriptomics (ST) is to precisely resolve the spatial distribution of cell subpopulations with low abundance and detect context-dependent variation in cell states within complex tissue microenvironments. Here, we introduce an ultra-resolution ST deconvolution algorithm (UCASpatial) that improves the resolution of mapping cell subpopulations to spatial locations by leveraging the contribution of genes indicative of cell identity through entropy-based weighting. Using both in silico and real ST datasets, we demonstrate that UCASpatial improves the robustness and accuracy in identifying low-abundant cell subpopulations and distinguishing transcriptionally heterogeneous cell subpopulations. Applying UCASpatial to murine wound healing, we recapitulate known spatiotemporal dynamics of multiple cell subpopulations involved in wound healing and reveal the spatial segregation of distinct macrophage subpopulations embedded within different cellular communities at the wound center. In human colorectal cancer (CRC), we link genomic alterations in individual cancer clones to multi-cellular characteristics of the tumor immune microenvironment (TIME) and reveal the co-evolution of tumor cells and TIME within the same tumor. We show that the copy number gain on chromosome 20q (chr20q-gain) in tumor cells orchestrates a T cell-excluded TIME, indicative of resistance to immunotherapy in CRC, and is associated with tumor-intrinsic endogenous retrovirus silencing and impaired type I interferon response. Our findings present UCASpatial as a versatile tool for deciphering ultra-resolution cellular landscapes in ST and exploring intercellular mechanisms in complex and dynamic microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36aa9aa2d8c5c7dbae3c0976fedf2735c37f839" target='_blank'>
              Ultra-resolution Deconvolution of Spatial Transcriptomics Unveils Spatiotemporal Cellular Dynamics in Complex Microenvironments
              </a>
            </td>
          <td>
            Yin Xu, Zurui Huang, Yawei Zhang, Zhenghang Wang, Peijin Guo, Feifan Zhang, Wenxuan Gong, Guanghao Liang, Boyuan Mei, Lihui Dong, Renbao Chang, Minghui Gong, Yu Xia, Haochen Ni, Jing Yang, Yuan Gao, Zhaoqi Liu, Lin Shen, Jian Li, Meng Michelle Xu, Dali Han
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Understanding the specific type of brain malignancy, source of brain metastasis, and underlying transformation mechanisms can help provide better treatment and less harm to patients. The tumor microenvironment plays a fundamental role in cancer progression and affects both primary and metastatic cancers. The use of single-cell RNA sequencing to gain insights into the heterogeneity profiles in the microenvironment of brain malignancies is useful for guiding treatment decisions. To comprehensively investigate the heterogeneity in gliomas and brain metastasis originating from different sources (lung and breast), we integrated data from three groups of single-cell RNA-sequencing datasets obtained from GEO. We gathered and processed single-cell RNA sequencing data from 90,168 cells obtained from 17 patients. We then employed the R package Seurat for dataset integration. Next, we clustered the data within the UMAP space and acquired differentially expressed genes for cell categorization. Our results underscore the significance of macrophages as abundant and pivotal constituents of gliomas. In contrast, lung-to-brain metastases exhibit elevated numbers of AT2, cytotoxic CD4+ T, and exhausted CD8+ T cells. Conversely, breast-to-brain metastases are characterized by an abundance of epithelial and myCAF cells. Our study not only illuminates the variation in the TME between brain metastasis with different origins but also opens the door to utilizing established markers for these cell types to differentiate primary brain metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a696f1d28423bca688b38bdd5d7a10f5760ecb" target='_blank'>
              Comprehensive integrated single-cell RNA sequencing analysis of brain metastasis and glioma microenvironment: Contrasting heterogeneity landscapes
              </a>
            </td>
          <td>
            S. F. Sajjadi, N. Salehi, Mehdi Sadeghi
          </td>
          <td>2024-07-26</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the spatial organization of the bone marrow (BM) microenvironment at single-cell resolution constitutes a challenge in multiple myeloma (MM). Using formalin-fixed paraffin-embedded (FFPE) samples from the MIcγ1 mouse model of MM and aged-matched healthy YFPcγ1 mice, we performed spatial transcriptional profiling with Visium Spatial Gene Expression. A custom data-analysis framework that combines spatial with single-cell transcriptomic profiling defined the BM cellular composition and established specific cell relations, visualizing the spatial distribution of transcriptionally heterogeneous MM plasma cells (MM-PC). MM pathogenesis transcriptional programs were spatially delineated within the BM microenvironment. A high-to-low MM-PC density gradient spatially correlated with effector-to-exhausted T cell phenotype abundance. In this context, MM cells in high-density MM-PC areas coexisted with dendritic cells while displaced neutrophils to the tumor border. Increased neutrophil extracellular trap formation, IL-17-driven inflammatory signaling, and osteoclast differentiation were spatially delineated within the BM microenvironment. The spatial identification of different areas of BM and the interaction between malignant cells and their microenvironment were validated in FFPE BM biopsies from MM patients with varying degrees of MM-PC infiltration. In summary, spatial transcriptomics depicts the BM cellular architecture of MM and reveals deregulated mechanisms underlying MM intercellular communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6b45562f865baeadce9704ad2edc7a5b178a26" target='_blank'>
              Spatial Transcriptomics Reveals a Myeloma Cell Architecture with Dysfunctional T-Cell Distribution, Neutrophil Traps, and Inflammatory Signaling
              </a>
            </td>
          <td>
            Laura Sudupe, Emma Muiños-López, A. Lopez-Perez, A. Vilas-Zornoza, S. Sarvide, Purificación Ripalda-Cemboráin, Paula Aguirre-Ruiz, Patxi San Martín-Úriz, Marta Larrayoz, Laura Alvarez-Gigli, Marta Abengozar-Muela, I. Cenzano, Miguel Cócera, Javier Ruiz, Ignacio Sancho González, Azari Bantan, Aleksandra Kurowska, Jin Ye, Phillip T. Newton, B. Paiva, J. R. Rodríguez-Madoz, Vincenzo Lagani, Jesper N. Tegnér, Borja Saez, J. Martinez-Climent, Isabel A. Calvo, D. Gómez-Cabrero, F. Prósper
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Biliary tract cancers (BTCs) are a group of deadly malignancies encompassing intrahepatic and extrahepatic cholangiocarcinoma, gallbladder carcinoma, and ampullary carcinoma. Here, we present the integrative analysis of 63 BTC cell lines via multi-omics clustering and genome- scale CRISPR screens, providing a platform to illuminate BTC biology and inform therapeutic development. We identify dependencies broadly enriched in BTC compared to other cancers as well as dependencies selective to the anatomic subtypes. Notably, cholangiocarcinoma cell lines are stratified into distinct lineage subtypes based on biliary or dual biliary/hepatocyte marker signatures, associated with dependency on specific lineage survival factors. Transcriptional analysis of patient specimens demonstrates the prognostic significance of these lineage subtypes. Additionally, we delineate strategies to enhance targeted therapies or to overcome resistance in cell lines with key driver gene mutations. Furthermore, clustering based on dependencies and proteomics data elucidates unexpected functional relationships, including a BTC subgroup with partial squamous differentiation. Thus, this cell line atlas reveals potential therapeutic targets in molecularly defined BTCs, unveils biologically distinct disease subtypes, and offers a vital resource for BTC research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef975abc103c925058ab977fa358ad6a654baecf" target='_blank'>
              Generation of a biliary tract cancer cell line atlas reveals molecular subtypes and therapeutic targets
              </a>
            </td>
          <td>
            Vindhya Vijay, Negin Karisani, Lei Shi, Yu-Han Hung, Phuong Vu, Prabhat Kattel, Lauren Kenney, J. Merritt, Ramzi Adil, Qibiao Wu, Yuanli Zhen, Robert Morris, J. Kreuzer, Meena Kathiresan, Xcanda Ixchel Herrera Lopez, Haley Ellis, Ilaria Gritti, Lilian Lecorgne, Ines Farag, Alexandra Popa, William Shen, Hiroyuki Kato, Qin Xu, E. Balasooriya, Meng-Ju Wu, Saireudee Chaturantabut, R. Kate Kelley, James M. Cleary, Michael S. Lawrence, David Root, C. Benes, Vikram Deshpande, D. Juric, William R. Sellers, Cristina R. Ferrone, Wilhelm Haas, Francisca Vazquez, G. Getz, N. Bardeesy
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>194</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Adult-type diffuse gliomas are a family of aggressive brain tumours with few effective treatments. Their complex cellular makeup adds to the challenge of finding successful therapies. This intratumoural heterogeneity is fuelled by a subpopulation of glioma stem-like cells (GSCs) that drive tumour growth and resistance to standard treatments. Previous research focused on the three glioma types (astrocytoma, oligodendroglioma, glioblastoma) individually revealed malignant cells mimic the transcriptional profiles of normal brain cell types. Whether these diverse cellular states stem from a shared biological origin is unknown. Here, we show through single-cell RNA sequencing of 40 glioma tumours that all gliomas are described by seven recurring cell states. We also identify a shared astrocyte-like GSC population. Our unique method of identifying GSCs, based on reconstructed tumour phylogenies, repositions astrocyte-like cells at the apex of a differentiation hierarchy in glioma. Our findings indicate the transcriptional heterogeneity observed in gliomas stems from a GSC population recapitulating lineages of healthy adult neural stem cells. These results suggest a shared lineage drives the intratumoural heterogeneity observed in adult-type diffuse gliomas. We anticipate that a deeper understanding of the molecular mechanisms maintaining the GSC state will provide a new framework for future therapeutic development and research into glioma cell biology. Highlights Recurring cell states are shared across adult-type diffuse gliomas Reconstructed tumour phylogenies identify an astrocyte-like glioma stem cell population Tumour subclones are segregated non-randomly across cell states">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a7ceb66735b2467794675778803760e2c35252" target='_blank'>
              Adult-type diffuse gliomas share recurring cell states driven by a common astrocyte-like glioma stem cell population
              </a>
            </td>
          <td>
            Walter Muskovic, Ashraf Zaman, Madhavi Pandya, Emily Holton, Chia-Ling Chan, Rachael A McCloy, Himanshi Arora, Venessa T Chin, Joseph E Powell
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The molecular underpinnings of High Grade Endometrial Carcinoma (HGEC) metastatic growth and survival are poorly understood. Here we show that ascites-derived and primary tumor HGEC cell lines in 3D spheroid culture faithfully recapitulate key features of malignant peritoneal effusion and exhibit fundamentally distinct transcriptomic, proteomic and metabolomic landscapes when compared with conventional 2D monolayers. Using genetic screening platform we identify MAPK14 (which encodes the protein kinase p38α) as a specific requirement for HGEC in spheroid culture. MAPK14/p38α has broad roles in programing the phosphoproteome, transcriptome and metabolome of HGEC spheroids, yet has negligible impact on monolayer cultures. MAPK14 promotes tumorigenicity in vivo and is specifically required to sustain a sub-population of spheroid cells that is enriched in cancer stemness markers. Therefore, spheroid growth of HGEC activates unique biological programs, including p38α signaling, that cannot be captured using 2D culture models and are highly relevant to malignant disease pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e7c989b30535b3174da390f7011080ff704c1d1" target='_blank'>
              MAPK14 /p38α Shapes the Molecular Landscape of Endometrial Cancer and promotes Tumorigenic Characteristics
              </a>
            </td>
          <td>
            Sayali Joseph, Xingyuan Zhang, G. Droby, Di Wu, V. Bae-Jump, Scott Lyons, Angie Mordant, Allie Mills, Laura Herring, Blake R Rushing, Jessica L. Bowser, Cyrus Vaziri
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rationale: The heterogeneity of tumor cells within the glioblastoma (GBM) microenvironment presents a complex challenge in curbing GBM progression. Understanding the specific mechanisms of interaction between different GBM cell subclusters and non-tumor cells is crucial. Methods: In this study, we utilized a comprehensive approach integrating glioma single-cell and spatial transcriptomics. This allowed us to examine the molecular interactions and spatial localization within GBM, focusing on a specific tumor cell subcluster, GBM subcluster 6, and M2-type tumor-associated macrophages (M2 TAMs). Results: Our analysis revealed a significant correlation between a specific tumor cell subcluster, GBM cluster 6, and M2-type TAMs. Further in vitro and in vivo experiments demonstrated the specific regulatory role of the CEBPB transcriptional network in GBM subcluster 6, which governs its tumorigenicity, recruitment of M2 TAMs, and polarization. This regulation involves molecules such as MCP1 for macrophage recruitment and the SPP1-Integrin αvβ1-Akt signaling pathway for M2 polarization. Conclusion: Our findings not only deepen our understanding of the formation of M2 TAMs, particularly highlighting the differential roles played by heterogeneous cells within GBM in this process, but also provided new insights for effectively controlling the malignant progression of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f32ae634b60547b99c952f2958aef6e7b9e1f2ae" target='_blank'>
              The CEBPB+ glioblastoma subcluster specifically drives the formation of M2 tumor-associated macrophages to promote malignancy growth
              </a>
            </td>
          <td>
            Yongchang Yang, Xingyu Jin, Yang Xie, Chunlan Ning, Yiding Ai, Haotian Wei, Xing Xu, Xianglian Ge, Tailong Yi, Qiang Huang, Xuejun Yang, Tao Jiang, Xiaoguang Wang, Yingzhe Piao, Xun Jin
          </td>
          <td>2024-07-02</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Motivation Cells exhibit a wide array of morphological features, enabling computer vision methods to identify and track relevant parameters. Morphological analysis has long been implemented to identify specific cell types and cell responses. Here we asked whether morphological features might also be used to classify transcriptomic subpopulations within in vitro cancer cell lines. Identifying cell subpopulations furthers our understanding of morphology as a reflection of underlying cell phenotype and could enable a better understanding of how subsets of cells compete and cooperate in disease progression and treatment. Results We demonstrate that cell morphology can reflect underlying transcriptomic differences in vitro using convolutional neural networks. First, we find that changes induced by chemotherapy treatment are highly identifiable in a breast cancer cell line. We then show that the intra cell line subpopulations that comprise breast cancer cell lines under standard growth conditions are also identifiable using cell morphology. We find that cell morphology is influenced by neighborhood effects beyond the cell boundary, and that including image information surrounding the cell can improve model discrimination ability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116b8716438d464b34fa34caed09100f7664c755" target='_blank'>
              Deep learning identifies heterogeneous subpopulations in breast cancer cell lines
              </a>
            </td>
          <td>
            T. Jost, Andrea L. Gardner, Daylin Morgan, Amy Brock
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Targeted therapies can induce strong tumor regression, but they typically fail to eradicate metastatic cancers. The elucidation of the causes that enable cancers to survive within a residual disease is essential for finding eradication strategies. The ability of cancers to survive eradication reflects not only cell-intrinsic sensitivities, but also microenvironmental effects. Paracrine signals produced by fibroblast, non-neoplastic cells that make tumor stroma, can provide strong but spatially limited therapy protection. Even though this phenomenon is well-established, its contribution to the ability of tumors to escape eradication remains poorly understood. To address this gap of knowledge, we developed an in silico model that recapitulates the effect of stroma on therapy responses in tumor tissues. This model enabled us to evaluate the contribution of spatial aspects of stroma-mediated resistance. Our analyses reveal that stroma dispersal might be the most important yet overlooked aspect of stromal resistance that determines the overall tumor responses to therapy. Abstract The response of tumors to anti-cancer therapies is defined not only by cell-intrinsic therapy sensitivities but also by local interactions with the tumor microenvironment. Fibroblasts that make tumor stroma have been shown to produce paracrine factors that can strongly reduce the sensitivity of tumor cells to many types of targeted therapies. Moreover, a high stroma/tumor ratio is generally associated with poor survival and reduced therapy responses. However, in contrast to advanced knowledge of the molecular mechanisms responsible for stroma-mediated resistance, its effect on the ability of tumors to escape therapeutic eradication remains poorly understood. To a large extent, this gap of knowledge reflects the challenge of accounting for the spatial aspects of microenvironmental resistance, especially over longer time frames. To address this problem, we integrated spatial inferences of proliferation-death dynamics from an experimental animal model of targeted therapy responses with spatial mathematical modeling. With this approach, we dissected the impact of tumor/stroma distribution, magnitude and distance of stromal effects. While all of the tested parameters affected the ability of tumor cells to resist elimination, spatial patterns of stroma distribution within tumor tissue had a particularly strong impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d34125347f89218581236d182c69f2cff0ff55a" target='_blank'>
              Dissecting the Spatially Restricted Effects of Microenvironment-Mediated Resistance on Targeted Therapy Responses
              </a>
            </td>
          <td>
            Tatiana Miti, Bina Desai, D. Miroshnychenko, D. Basanta, A. Marusyk
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have heralded a remarkable shift in cancer care, significantly extending survival for advanced cancer patients. However, despite their remarkable clinical successes, a substantial majority of patients fail to achieve a lasting response to ICIs. To address this challenge and gain insights into the complex landscape of the tumor microenvironment (TME), we conducted an extensive analysis using single-cell RNA sequencing (scRNA; ∼216K cells across 39 samples) and single-nucleus transposase-accessible chromatin sequencing (snATAC; ∼43K cells from 15 samples) in a metastatic melanoma cohort. This systematic approach delineates 14 distinct cell types and 55 cell subtypes, including the identification of 15 transcriptional hallmarks of malignant cells. Through correlation analysis of cell subtype proportions, we unveiled six distinct clusters associated with varying tumor responses. Particularly intriguing was the identification of the mature DC enriched in immunoregulatory molecules (mregDC) subtype exhibiting correlations with naive T and B cells, forming an anti-tumor program that underscores the importance of multiple immune cell types in mediating anti-tumor immunity. Notably, we found that mregDC abundance represents a good prognostic predictor of progression-free survival (PFS) in the context of ICI treatment, and when combined with the TCF7+/- CD8 T cell ratio, it reliably predicts patient PFS across treatments beyond ICI. We validated our findings using an independent cohort of 274 ICI-treated melanoma samples analyzed using tissue-level expression. We next compared mregDCs and conventional dendritic cell types 1 and 2 (cDC1 and cDC2) using transcriptome signature, differentiation trajectory, interactome, cytokine milieu, and epigenome landscape analyses. This comparative analysis shed light on the unique attributes of mregDCs within the TME. Finally, we investigated cell type/subtype-specific genes, pathways, immune response enrichment, and ligand-receptor interactions closely associated with the proportions of mregDCs within the TME. These molecular and cellular insights, with their critical roles in enhancing the immune response against cancer, offer valuable prospects for predicting the efficacy of ICI regimens, and potentially guiding the selection of rational combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd285dffb8067a1b036abc12a6336c80dd568b9" target='_blank'>
              Mature and migratory dendritic cells promote immune infiltration and response to anti-PD-1 checkpoint blockade in metastatic melanoma
              </a>
            </td>
          <td>
            Jie-Ying Yang, Cassia Wang, Doris Fu, Li-Lun Ho, K. Galani, Lee Chen, Dennie T. Frederick, Liang He, Amy Y. Huang, Emily J. Robitschek, Sandeep K. Yadav, Wentao Deng, Rahul Tacke, Kelly P. Burke, T. Sharova, Kunal Rai, David Liu, Genevieve M. Boland, M. Kellis
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Intracerebral hemorrhage (ICH) poses a formidable challenge in stroke management, with limited therapeutic options, particularly in the realm of immune‐targeted interventions. Clinical trials targeting immune responses post‐ICH have encountered setbacks, potentially attributable to the substantial cellular heterogeneity and intricate intercellular networks within the brain. Here, we present a pioneering investigation utilizing single‐cell RNA sequencing and spatial transcriptome profiling at hyperacute (1 h), acute (24 h), and subacute (7 days) intervals post‐ICH, aimed at unraveling the dynamic immunological landscape and spatial distributions within the cerebral tissue. Our comprehensive analysis revealed distinct cell differentiation patterns among myeloid and lymphocyte populations, along with delineated spatial distributions across various brain regions. Notably, we identified a subset of lymphocytes characterized by the expression of Spp1 and Lyz2, termed macrophage‐associated lymphocytes, which exhibited close interactions with myeloid cells. Specifically, we observed prominent interactions between Lgmn+Macro‐T cells and microglia through the spp1–cd44 pathway during the acute phase post‐ICH in the choroid plexus. These findings represent a significant advancement in our understanding of immune cell dynamics at single‐cell resolution across distinct post‐ICH time points, thereby laying the groundwork for exploring critical temporal windows and informing the development of targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af27d3cf37340430057062faa5cf04622e91eea" target='_blank'>
              Unraveling dynamic immunological landscapes in intracerebral hemorrhage: insights from single‐cell and spatial transcriptomic profiling
              </a>
            </td>
          <td>
            Lingui Gu, Hualin Chen, Mingjiang Sun, Yihao Chen, Qinglei Shi, Jianbo Chang, Junji Wei, Wenbin Ma, X. Bao, Renzhi Wang
          </td>
          <td>2024-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, still most patients experience limited or no clinical benefit. The immune component dynamics during treatment and how it correlated with treatment efficacy remained largely known. We hypothesize that understanding the modulation dynamics of the TME upon ICI treatment would be key to elucidate the mechanisms of response and resistance. To systematically characterize these dynamics, we aggregated a large-scale single-cell RNA-sequencing (scRNA-seq) dataset of 141 patients across multiple cancer types and ICI treatment modalities, with longitudinally paired samples. We developed a robust, integration-free, and cluster-free deep phenotyping framework, leveraging multiple pan-cancer references to annotate 876,410 high-quality cells into 80 granular cell states. Our analysis revealed consistent compositional changes in 17 cell subtypes following ICI treatment, including an enhanced adaptive immune response such as CD8+ IL7R+ central memory T cells, GZMK+ effector memory T cells, germinal center B cells, precursor exhausted T cells, and CD4+ TfhTh1 cells, accompanied by a broad reduction in interferon responsive immune cells quiescent CD4+CREM-memory T cells. We uncovered co-regulated cell communities within the TME, highlighting the coordinated interplay between adaptive and innate immune cells, as well as immune and non-immune components. Furthermore, we identified two distinct patient groups exhibiting tightly correlated cellular dynamics within the TME post-treatment. The first group, enriched for responders, displayed a marked expansion of naive lymphocytes, while the second group, predominantly composed of non-responders, showed an increased abundance of immune experienced/suppressive cell states. This dichotomy in TME dynamics offers a potential predictive biomarker for patient stratification and personalized therapeutic strategies. Our study presents a comprehensive landscape of the cellular dynamics within the TME during ICI treatment, enabled by a powerful deep phenotyping approach showcasing the importance of a systems-level understanding of the TME dynamics in improving patient stratification and advancing personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e90093265f7348aad7b591e51e89915aa6d67b" target='_blank'>
              Cellular Dynamics Upon Immune Checkpoint Inhibition
              </a>
            </td>
          <td>
            Zhongyang Lin, Eketerina Petrenko, Mitalee Chandra, Nalini Srinivas, Selma Ugurel, Jürgen C. Becker, Dvir Aran
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intratumor heterogeneity has been shown to play a role in the malignant progression of cancer. Although clonal evolution in primary cancer has been well studied, that in metastatic tumorigenesis is not fully understood. In this study, we established human colon cancer-derived organoids and investigated clonal dynamics during liver metastasis development by tracking barcode-labelled subclones. Long-term subclone co-cultures showed clonal drift, with a single subclone becoming dominant in the cell population. Interestingly, the selected subclones were not always the same, suggesting that clonal selection was not based on cell intrinsic properties. Furthermore, liver tumors developed by co-transplantation of organoid subclones into the immunodeficient mouse spleen showed a progressive drastic reduction in clonal diversity, and only one or two subclones predominated in the majority of large metastatic tumors. Importantly, selections were not limited to particular subclones but appeared to be random. A trend towards a reduction in clonal diversity was also found in liver metastases of multiple color-labeled organoids of mouse intestinal tumors. Based on these results, we propose a novel mechanism of metastasis development, i.e. a subclone population of the disseminated tumor cells in the liver is selected by neutral selection during colonization and constitutes large metastatic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2918909f5cedae4e48ddf4cd75fefc07c76ff3b" target='_blank'>
              Neutral selection and clonal expansion during the development of colon cancer metastasis.
              </a>
            </td>
          <td>
            Xuelian Lei, Daisuke Yamamoto, Hirotaka Kitamura, Kenji Kita, Noriyuki Inaki, Kazuhiro Murakami, M. Nakayama, H. Oshima, Masanobu Oshima
          </td>
          <td>2024-06-18</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Cancers evolve in a dynamic ecosystem. Thus, characterizing cancer’s ecological dynamics is crucial to understanding cancer evolution and can lead to discovering novel biomarkers to predict disease progression. Ductal carcinoma in situ (DCIS) is an early-stage breast cancer characterized by abnormal epithelial cell growth confined within the milk ducts. Although there has been extensive research on genetic and epigenetic causes of breast carcinogenesis, none of these studies have successfully identified a biomarker for the progression and/or upstaging of DCIS. In this study, we show that ecological habitat analysis of hypoxia and acidosis biomarkers can significantly improve prediction of DCIS upstaging. First, we developed a novel eco-evolutionary designed approach to define habitats in the tumor intra-ductal microenvironment based on oxygen diffusion distance in our DCIS cohort of 84 patients. Then, we identify cancer cells with metabolic phenotypes attributed to their habitat conditions, such as the expression of CA9 indicating hypoxia responding phenotype, and LAMP2b indicating a hypoxia-induced acid adaptation. Traditionally these markers have shown limited predictive capabilities for DCIS upstaging, if any. However, when analyzed from an ecological perspective, their power to differentiate between indolent and upstaged DCIS increased significantly. Second, using eco-evolutionary guided computational and digital pathology techniques, we discovered distinct spatial patterns of these biomarkers and used the distribution of such patterns to predict patient upstaging. The patterns were characterized by both cellular features and spatial features. With a 5-fold validation on the biopsy cohort, we trained a random forest classifier to achieve the area under curve(AUC) of 0.74. Our results affirm the importance of using eco-evolutionary-designed approaches in biomarkers discovery studies in the era of digital pathology by demonstrating the role of eco-evolution dynamics in predicting cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786cf3e3e85927b01c56064b452b334cb91ffed1" target='_blank'>
              Eco-evolutionary Guided Pathomic Analysis to Predict DCIS Upstaging
              </a>
            </td>
          <td>
            Yujie Xiao, Manal Elmasry, Ji Dong K. Bai, Andrew Chen, Yuzhu Chen, Brooke Jackson, Joseph O. Johnson, Robert J. Gillies, Prateek Prasanna, Chao Chen, Mehdi Damaghi
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid growth and early metastasis and is susceptible to treatment resistance and recurrence. Understanding the intra-tumoral spatial heterogeneity in SCLC is crucial for improving patient outcomes and clinically relevant subtyping. In this study, a spatial whole transcriptome-wide analysis of 25 SCLC patients at sub-histological resolution using GeoMx Digital Spatial Profiling technology is performed. This analysis deciphered intra-tumoral multi-regional heterogeneity, characterized by distinct molecular profiles, biological functions, immune features, and molecular subtypes within spatially localized histological regions. Connections between different transcript-defined intra-tumoral phenotypes and their impact on patient survival and therapeutic response are also established. Finally, a gene signature, termed ITHtyper, based on the prevalence of intra-tumoral heterogeneity levels, which enables patient risk stratification from bulk RNA-seq profiles is identified. The prognostic value of ITHtyper is rigorously validated in independent multicenter patient cohorts. This study introduces a preliminary tumor-centric, regionally targeted spatial transcriptome resource that sheds light on previously unexplored intra-tumoral spatial heterogeneity in SCLC. These findings hold promise to improve tumor reclassification and facilitate the development of personalized treatments for SCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28041b5c4214338317866c30c837c28570a97950" target='_blank'>
              Spatial Transcriptome-Wide Profiling of Small Cell Lung Cancer Reveals Intra-Tumoral Molecular and Subtype Heterogeneity.
              </a>
            </td>
          <td>
            Zicheng Zhang, Xu-min Sun, Yutao Liu, Yibo Zhang, Zijian Yang, J. Dong, Nan Wang, Jianming Ying, Meng Zhou, Lin Yang
          </td>
          <td>2024-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Annotation of the cis-regulatory elements that drive transcriptional dysregulation in cancer cells is critical to improving our understanding of tumor biology. Herein, we present a compendium of matched chromatin accessibility (scATAC-seq) and transcriptome (scRNA-seq) profiles at single-cell resolution from human breast tumors and healthy mammary tissues processed immediately following surgical resection. We identify the most likely cell-of-origin for luminal breast tumors and basal breast tumors and then introduce a novel methodology that implements linear mixed-effects models to systematically quantify associations between regions of chromatin accessibility (i.e. regulatory elements) and gene expression in malignant cells versus normal mammary epithelial cells. These data unveil regulatory elements with that switch from silencers of gene expression in normal cells to enhancers of gene expression in cancer cells, leading to the upregulation of clinically relevant oncogenes. To translate the utility of this dataset into tractable models, we generated matched scATAC-seq and scRNA-seq profiles for breast cancer cell lines, revealing, for each subtype, a conserved oncogenic gene expression program between in vitro and in vivo cells. Together, this work highlights the importance of non-coding regulatory mechanisms that underlie oncogenic processes and the ability of single-cell multi-omics to define the regulatory logic of BC cells at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bf4929898d9a3caa32f9bca65e96287e05306d" target='_blank'>
              Defining the Regulatory Logic of Breast Cancer Using Single-Cell Epigenetic and Transcriptome Profiling
              </a>
            </td>
          <td>
            Matthew J. Regner, Susana Garcia-Recio, A. Thennavan, K. Wiśniewska, Raul Mendez-Giraldez, Brooke Felsheim, Philip M. Spanheimer, Joel S. Parker, C. Perou, Hector L. Franco
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>160</td>
        </tr>

        <tr id="Understanding transcriptional heterogeneity in cancer cells and its implication for treatment response is critical to identify how resistance occurs and may be targeted. Such heterogeneity can be captured by in vitro studies through clonal barcoding methods. We present TraCSED (Transformer-based modeling of Clonal Selection and Expression Dynamics), a dynamic deep learning approach for modeling clonal selection. Using single-cell gene expression and the fitness of barcoded clones, TraCSED identifies interpretable gene programs and the timepoints at which they are associated with clonal selection. When applied to cells treated with either giredestrant, an estrogen receptor (ER) antagonist and degrader, or palbociclib, a CDK4/6 inhibitor, time-dependent resistance pathways are revealed. For example, ER activity is associated with positive selection around day four under palbociclib treatment and this adaptive response can be suppressed by combining the drugs. Yet, in the combination treatment, one clone still emerged. Clustering based on partial least squares regression found that high baseline expression of both SNHG25 and SNCG genes was the primary marker of positive selection to co-treatment and thus potentially associated with innate resistance – an aspect that traditional differential analysis methods missed. In conclusion, TraCSED enables associating pathways with phenotypes in a time-dependent manner from scRNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb809968ef629bdd6c8b90e0032f40dbb52452b" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics (TraCSED) reveals resistance signatures in breast cancer
              </a>
            </td>
          <td>
            Nathan Maulding, Jun Zou, Wei Zhou, Ciara Metcalfe, Josh Stuart, Xin Ye, Marc Hafner
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The acidic tumor microenvironment favors cancer aggressiveness via incompletely understood pathways. Here, we asked whether acidic environments select for cancer stem cell (CSC) properties. Bulk RNA-seq of Panc-1 human pancreatic cancer cells adapted to extracellular pH 6.5 revealed upregulation of CSC markers including CD44, EpCam, Nestin and aldehyde dehydrogenases, and CSC pathway enrichment. We therefore assessed CSC characteristics of acid-adapted (AA) and non-adapted (Ctrl) PaTu8988s and MiaPaca-2 pancreatic cancer cells. Compared to Ctrl, AA cells exhibited increased ALDH- and β-catenin activity and pancreatosphere-forming efficiency, classical CSC characteristics. Panc-1, PaTu8988s and MiaPaCa-2 AA cells differed in CSC marker expression, and AA cells did not exhibit typical flow cytometric CSC populations. However, single-nucleus sequencing identified the acid adaptation-induced emergence of a population with clear CSC characteristics. Finally, in an orthotopic mouse model, AA Panc-1 cells drove strongly increased aggressiveness and liver metastasis compared to Ctrl cells. We conclude that acid-adaptation of pancreatic cancer cells leads to enrichment of a CSC phenotype with unusual traits, providing new insight into how acidic tumor microenvironments favor cancer aggressiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87f31abeb1d47ff4ba195ad016a61c5c1131c375" target='_blank'>
              Tumor microenvironment acidosis favors pancreatic cancer stem cell properties and in vivo metastasis
              </a>
            </td>
          <td>
            Michala G Rolver, J. C. Roda, Yifan Dai, Mette Flinck, Renata Ialchina, Julie Hindkær, Rigmor T. Dyhr, August N. Bodilsen, N. S. Prasad, Jonathan Baldan, Jiayi Yao, Albin Sandelin, Luis Arnes, S. F. Pedersen
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The spatial arrangement of cells is vital in developmental processes and organogenesis in multicellular life forms. Deep learning models trained with spatial omics data uncover complex patterns and relationships among cells, genes, and proteins in a high-dimensional space, providing new insights into biological processes and diseases. State-of-the-art in silico spatial multi-cell gene expression methods using histological images of tissue stained with hematoxylin and eosin (H&E) to characterize cellular heterogeneity. These computational techniques offer the advantage of analyzing vast amounts of spatial data in a scalable and automated manner, thereby accelerating scientific discovery and enabling more precise medical diagnostics and treatments. In this work, we developed a vision transformer (ViT) framework to map histological signatures to spatial single-cell transcriptomic signatures, named SPiRiT (Spatial Omics Prediction and Reproducibility integrated Transformer). Our framework was enhanced by integrating cross validation with model interpretation during hyper-parameter tuning. SPiRiT predicts single-cell spatial gene expression using the matched histopathological image tiles of human breast cancer and whole mouse pup, evaluated by Xenium (10x Genomics) datasets. Furthermore, ViT model interpretation reveals the high-resolution, high attention area (HAR) that the ViT model uses to predict the gene expression, including marker genes for invasive cancer cells (FASN), stromal cells (POSTN), and lymphocytes (IL7R). In an apple-to-apple comparison with the ST-Net Convolutional Neural Network algorithm, SPiRiT improved predictive accuracy by 40% using human breast cancer Visium (10x Genomics) dataset. Cancer biomarker gene prediction and expression level are highly consistent with the tumor region annotation. In summary, our work highlights the feasibility to infer spatial single-cell gene expression using tissue morphology in multiple-species, i.e., human and mouse, and multi-organs, i.e., mouse whole body morphology. Importantly, incorporating model interpretation and vision transformer is expected to serve as a general-purpose framework for spatial transcriptomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2dd7dd123b282d7d557858b4a296ca45cae841" target='_blank'>
              Inferring single-cell spatial gene expression with tissue morphology via explainable deep learning
              </a>
            </td>
          <td>
            Yue Zhao, Elaheh Alizadeh, Yang Liu, Ming Xu, J. M. Mahoney, Sheng Li
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Melanoma, a malignant skin cancer arising from melanocytes, exhibits rapid metastasis and a high mortality rate, especially in advanced stages. Current treatment modalities, including surgery, radiation, and immunotherapy, offer limited success, with immunotherapy using immune checkpoint inhibitors (ICIs) being the most promising. However, the high mortality rate underscores the urgent need for robust, non-invasive biomarkers to predict patient response to adjuvant therapies. The immune microenvironment of melanoma comprises various immune cells, which influence tumor growth and immune response. Melanoma cells employ multiple mechanisms for immune escape, including defects in immune recognition and epithelial-mesenchymal transition (EMT), which collectively impact treatment efficacy. Single-cell analysis technologies, such as single-cell RNA sequencing (scRNA-seq), have revolutionized the understanding of tumor heterogeneity and immune microenvironment dynamics. These technologies facilitate the identification of rare cell populations, co-expression patterns, and regulatory networks, offering deep insights into tumor progression, immune response, and therapy resistance. In the realm of biomarker discovery for melanoma, single-cell analysis has demonstrated significant potential. It aids in uncovering cellular composition, gene profiles, and novel markers, thus advancing diagnosis, treatment, and prognosis. Additionally, tumor-associated antibodies and specific genetic and cellular markers identified through single-cell analysis hold promise as predictive biomarkers. Despite these advancements, challenges such as RNA-protein expression discrepancies and tumor heterogeneity persist, necessitating further research. Nonetheless, single-cell analysis remains a powerful tool in elucidating the mechanisms underlying therapy response and resistance, ultimately contributing to the development of personalized melanoma therapies and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824bd1e506d1d79fb980c0e85de9df9031024731" target='_blank'>
              Advancing immunotherapy for melanoma: the critical role of single-cell analysis in identifying predictive biomarkers
              </a>
            </td>
          <td>
            Ru He, Jiaan Lu, Jianglong Feng, Ziqing Lu, Kaixin Shen, Ke Xu, Huiyan Luo, Guanhu Yang, Hao Chi, Shangke Huang
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Simple Summary This study investigates the role of visual intratumor heterogeneity (ITH) in breast cancer progression. By analyzing histologic images from the Carolina Breast Cancer Study (CBCS) and the Cancer Genome Atlas Breast Invasive Carcinoma (TCGA-BRCA) data using advanced image processing and machine learning techniques, we developed a measure of tumor heterogeneity based on visual features. Our findings indicate that tumors with low visual heterogeneity exhibited a higher risk of recurrence and were more likely to come from patients whose tumors comprised of only one subclone or had a TP53 mutation. Conversely, high visual heterogeneity was correlated with a more favorable prognosis. These results suggest that visual heterogeneity provides complementary information to molecular markers. A comprehensive understanding of both the visual and molecular aspects of heterogeneity has the potential to offer novel insights for treatment strategies. Abstract High intratumoral heterogeneity is thought to be a poor prognostic indicator. However, the source of heterogeneity may also be important, as genomic heterogeneity is not always reflected in histologic or ‘visual’ heterogeneity. We aimed to develop a predictor of histologic heterogeneity and evaluate its association with outcomes and molecular heterogeneity. We used VGG16 to train an image classifier to identify unique, patient-specific visual features in 1655 breast tumors (5907 core images) from the Carolina Breast Cancer Study (CBCS). Extracted features for images, as well as the epithelial and stromal image components, were hierarchically clustered, and visual heterogeneity was defined as a greater distance between images from the same patient. We assessed the association between visual heterogeneity, clinical features, and DNA-based molecular heterogeneity using generalized linear models, and we used Cox models to estimate the association between visual heterogeneity and tumor recurrence. Basal-like and ER-negative tumors were more likely to have low visual heterogeneity, as were the tumors from younger and Black women. Less heterogeneous tumors had a higher risk of recurrence (hazard ratio = 1.62, 95% confidence interval = 1.22–2.16), and were more likely to come from patients whose tumors were comprised of only one subclone or had a TP53 mutation. Associations were similar regardless of whether the image was based on stroma, epithelium, or both. Histologic heterogeneity adds complementary information to commonly used molecular indicators, with low heterogeneity predicting worse outcomes. Future work integrating multiple sources of heterogeneity may provide a more comprehensive understanding of tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18cc90a062bf3cee9a0bd73529b68cc2b5c51dfd" target='_blank'>
              Visual Intratumor Heterogeneity and Breast Tumor Progression
              </a>
            </td>
          <td>
            Yao Li, Sarah C. Van Alsten, Dong Neuck Lee, Taebin Kim, Benjamin C. Calhoun, C. Perou, Sara E. Wobker, J. S. Marron, Katie Hoadley, M. A. Troester
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>160</td>
        </tr>

        <tr id="
 Therapeutics targeting tumor endothelial cells (TECs) have been explored for decades, with only suboptimal efficacy achieved, partly due to an insufficient understanding of the TEC heterogeneity across cancer patients. We integrated single-cell RNA-seq data of 575 cancer patients from 19 solid tumor types, comprehensively charting the TEC phenotypic diversities. Our analyses uncovered underappreciated compositional and functional heterogeneity in TECs from a pan-cancer perspective. Two subsets, CXCR4+ tip cells and SELE+ veins, represented the prominent angiogenic and proinflammatory phenotypes of TECs, respectively. They exhibited distinct spatial organization patterns, and compared to adjacent non-tumor tissues, tumor tissue showed an increased prevalence of CXCR4+ tip cells, yet with SELE+ veins depleted. Such functional and spatial characteristics underlie their differential associations with the response of anti-angiogenic therapies and immunotherapies. Our integrative resources and findings open new avenues to understand and clinically intervene in the tumor vasculature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc571fa1d1ff50ea9e24190fe0da0d50d2cd7b6f" target='_blank'>
              Pan-cancer integrative analyses dissect the remodeling of endothelial cells in human cancers
              </a>
            </td>
          <td>
            Jinhu Li, Dongfang Wang, Fei Tang, Xinnan Ling, Wenjie Zhang, Zemin Zhang
          </td>
          <td>2024-07-11</td>
          <td>National Science Review</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary In our study, we investigated immune regulation in a subtype of lymphoma called mantle cell lymphoma (MCL). Our goal was to understand how different types of immune cells, specifically T cells, behave in MCL. To do this, we combined image analysis and a technology called digital spatial profiling. We looked at tumor tissue from 102 MCL patients and analyzed different T-cell subsets. Interestingly, we found that late-stage differentiated T cells (CD57+) were more common in tumor-rich areas. These T cells also showed an increased expression of immune suppressive markers. We identified potential targets for treatment, such as CD47, IDO1, and CTLA-4. Additionally, we found that patients with sparse T-cell infiltration had an increased amount of GITR, which tentatively can be therapeutically targeted. Our findings shed light on previously unknown features of T-cell behavior in MCL. Abstract With the aim to advance the understanding of immune regulation in MCL and to identify targetable T-cell subsets, we set out to combine image analysis and spatial omic technology focused on both early and late differentiation stages of T cells. MCL patient tissue (n = 102) was explored using image analysis and GeoMx spatial omics profiling of 69 proteins and 1812 mRNAs. Tumor cells, T helper (TH) cells and cytotoxic (TC) cells of early (CD57−) and late (CD57+) differentiation stage were analyzed. An image analysis workflow was developed based on fine-tuned Cellpose models for cell segmentation and classification. TC and CD57+ subsets of T cells were enriched in tumor-rich compared to tumor-sparse regions. Tumor-sparse regions had a higher expression of several key immune suppressive proteins, tentatively controlling T-cell expansion in regions close to the tumor. We revealed that T cells in late differentiation stages (CD57+) are enriched among MCL infiltrating T cells and are predictive of an increased expression of immune suppressive markers. CD47, IDO1 and CTLA-4 were identified as potential targets for patients with T-cell-rich MCL TIME, while GITR might be a feasible target for MCL patients with sparse T-cell infiltration. In subgroups of patients with a high degree of CD57+ TC-cell infiltration, several immune checkpoint inhibitors, including TIGIT, PD-L1 and LAG3 were increased, emphasizing the immune-suppressive features of this highly differentiated T-cell subset not previously described in MCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd671c1ae7f90cd22784b36ac1b15cb30c9901a3" target='_blank'>
              Quantification and Profiling of Early and Late Differentiation Stage T Cells in Mantle Cell Lymphoma Reveals Immunotherapeutic Targets in Subsets of Patients
              </a>
            </td>
          <td>
            L. Lokhande, Daniel Nilsson, Joana de Matos Rodrigues, May Hassan, L. Olsson, P. Pyl, Louella Vasquez, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-06-21</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell spatial omics analysis requires consideration of biological functions and mechanisms in a microenvironment. However, microenvironment analysis using bioinformatic methods is limited by the need to detect histological morphology. In this study, we developed SpatialKNife (SKNY), an image-processing-based toolkit that detects spatial domains that potentially reflect histology and extends these domains to the microenvironment. The SKNY algorithm identified tumour spatial domains from spatial transcriptomic data of breast cancer, followed by clustering of these domains, trajectory estimation, and spatial extension to the tumour microenvironment (TME). The results of the trajectory estimation were consistent with the known mechanisms of cancer progression. We observed endothelial cell and macrophage infiltration into the TME at mid-stage progression. Our results suggest that analysis using the spatial domain as a unit reflects pathological mechanisms in the TME. This approach may be applicable to the biological estimation of diverse microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37e699643db70ef39e651b4f4b87acc5b96626c" target='_blank'>
              Spatial domain analysis to estimate spatiotemporal pathological mechanisms in microenvironment with single-cell spatial omics data
              </a>
            </td>
          <td>
            Shunsuke A. Sakai, Ryosuke Nomura, Satoi Nagasawa, SungGi Chi, Ayako Suzuki, Yutaka Suzuki, Shumpei Ishikawa, K. Tsuchihara, S. Kageyama, Riu Yamashita
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background. Interactions among tumor, immune, and vascular niches play major roles in driving glioblastoma (GBM) malignancy and treatment responses. The composition, heterogeneity, and localization of extracellular core matrix proteins (CMPs) that mediate such interactions, however, are not well understood. Methods. Here, through computational genomics and proteomics approaches, we analyzed the functional and clinical relevance of CMP expression in GBM at bulk, single cell, and spatial anatomical resolution. Results. We identified genes encoding CMPs whose expression levels categorize GBM tumors into CMP expression-high (M-H) and CMP expression-low (M-L) groups. CMP enrichment is associated with worse patient survival, specific driver oncogenic alterations, mesenchymal state, infiltration of pro-tumor immune cells, and immune checkpoint gene expression. Anatomical and single-cell transcriptome analyses indicate that matrisome gene expression is enriched in vascular and leading edge/infiltrative niches that are known to harbor glioma stem cells driving GBM progression. Finally, we identified a 17-gene CMP expression signature, termed Matrisome 17 (M17) signature that further refines the prognostic value of CMP genes. The M17 signature is a significantly stronger prognostic factor compared to MGMT promoter methylation status as well as canonical subtypes, and importantly, potentially predicts responses to PD1 blockade. Conclusion. The matrisome gene expression signature provides a robust stratification of GBM patients by survival and potential biomarkers of functionally relevant GBM niches that can mediate mesenchymal-immune cross talk. Patient stratification based on matrisome profiles can contribute to selection and optimization of treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0881b040066587c281ed9ef82afae916e63793e" target='_blank'>
              A prognostic matrix gene expression signature defines functional glioblastoma phenotypes and niches
              </a>
            </td>
          <td>
            Monika Vishnoi, Z. Dereli, Zheng Yin, Elisabeth K. Kong, Meric Kinali, Kisan Thapa, O. Babur, Kyuson Yun, Nourhan Abdelfattah, Xubin Li, B. Bozorgui, M. Farach-Carson, Robert Rostomily, Anil Korkut
          </td>
          <td>2024-06-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 Peripheral T-cell lymphomas (PTCL) are heterogeneous and highly aggressive hematologic malignancies with dismal outcomes, highlighting the need for novel targeted therapies. PTCLs are characterized by complex tumor-microenvironment (TME) ecosystems with low tumor cell content, admixed with rich immune infiltrates and stromal elements correlated with clinical outcomes. Direct targeting of the TME is an attractive strategy for PTCLs; however, the specific mechanisms that regulate the interdependence between lymphoma and TME remain poorly understood, hampering the development of TME-directed therapies. Our previous work based on driver genetic alterations in PTCL patients and genetic mouse models identified an important role of these driver mutations in remodeling the TME (Cortes et al., 2022). To address the specific role and mechanisms of tumor cell-TME interactions in the pathogenesis of PTCL, we developed a novel experimental platform and cutting-edge computational methodologies using single-nucleus RNAseq analysis to comprehensively profile lymphoma cells and their microenvironment from 27 PTCL patient samples of two major nodal pathologic subtypes, including 18 Nodal T follicular helper cell lymphomas (TFHcL) and 9 PTCL, not otherwise specified (PTCL, NOS). Our approach allowed the identification and in-depth characterization of tumor cells, lymphoid and myeloid TME, and vascular endothelium and non-endothelial stromal cells, which were challenging to capture with conventional single-cell RNAseq. The malignant T-cells in PTCL, NOS were grouped into patient-specific clusters demonstrating significant inter-tumor heterogeneity associated with diverse and frequently complex chromosomal abnormalities identified by copy number variation (CNV) analysis. In contrast, malignant TFHcL cells show a low degree to no apparent chromosomal abnormalities and divide into two major transcriptional subclusters, with tumor cells from the same patient falling into one of the two subclusters. Additionally, genomic profiles of TFHcL tumors revealed that the inter-tumor transcriptional heterogeneity of TFHcL tumor cells was strongly associated with the driver RHOA G17V mutation status. To profile oncogenic pathways critical for PTCL growth, we compared PTCL tumor cells to normal CD4 T-cells from reactive lymphoid tissue. PTCL, NOS and TFHcL tumor cells were equally enriched in the TCR signaling pathway, mitosis, and cell cycle related genes. However, exploitation of KRAS and cytokine-driven inflammatory signals was prevalent in TFHcL but not PTCL, NOS tumor cells. Analysis of the immune TME identified an increase in T regulatory cells, CD8 T-cells, immunoblasts, and macrophages in TFHcL. Furthermore, while dysregulated B-cells with heightened cytokine-driven inflammatory signals were unique to TFHcL, inflammatory macrophages and classic dendritic cells (cDC) were prominent in the TME of both subtypes. Our results uncover shared and subtype-specific pathogenic features in PTCL and pave the way for designing novel precision therapies for PTCL patients.
 Citation Format: Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig Soderquist, Ryan D Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero. Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17bf46cac83bb1af31141fb58102ffe6ac218af" target='_blank'>
              Abstract PR05: Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas
              </a>
            </td>
          <td>
            Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B. Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig R. Soderquist, Ryan Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Objectives Non‐small‐cell lung carcinoma (NSCLC) is the most prevalent and lethal form of lung cancer. The need for biomarker‐informed stratification of targeted therapies has underpinned the need to uncover the underlying properties of the tumor microenvironment (TME) through high‐plex quantitative assays. Methods In this study, we profiled resected NSCLC tissues from 102 patients by targeted spatial proteomics of 78 proteins across tumor, immune activation, immune cell typing, immune‐oncology, drug targets, cell death and PI3K/AKT modules to identify the tumor and stromal signatures associated with overall survival (OS). Results Survival analysis revealed that stromal CD56 (HR = 0.384, P = 0.06) and tumoral TIM3 (HR = 0.703, P = 0.05) were associated with better survival in univariate Cox models. In contrast, after adjusting for stage, BCLXL (HR = 2.093, P = 0.02) and cleaved caspase 9 (HR = 1.575, P = 0.1) negatively influenced survival. Delta testing indicated the protective effect of TIM‐3 (HR = 0.614, P = 0.04) on OS. In multivariate analysis, CD56 (HR = 0.172, P = 0.001) was associated with better survival in the stroma, while B7.H3 (HR = 1.72, P = 0.008) was linked to poorer survival in the tumor. Conclusions Deciphering the TME using high‐plex spatially resolved methods is giving us new insights into compartmentalised tumor and stromal protein signatures associated with clinical endpoints in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c86af101b13bc4f2f501f32d2e227ba48bb3c7a" target='_blank'>
              Spatial proteomic profiling of tumor and stromal compartments in non‐small‐cell lung cancer identifies signatures associated with overall survival
              </a>
            </td>
          <td>
            Vahid Yaghoubi Naei, J. Monkman, Habib Sadeghirad, Ahmed M Mehdi, Tony Blick, W. Mullally, K. O’Byrne, M. Warkiani, A. Kulasinghe
          </td>
          <td>2024-01-01</td>
          <td>Clinical & Translational Immunology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) mortality rates continue to increase faster than those of other cancer types due to high heterogeneity, which limits diagnosis and treatment. Pathological and molecular subtyping have identified that HCC tumors with poor outcomes are characterized by intratumoral collagenous accumulation. However, the translational and post-translational regulation of tumor collagen, which is critical to the outcome, remains largely unknown. Here, we investigate the spatial extracellular proteome to understand the differences associated with HCC tumors defined by Hoshida transcriptomic subtypes of poor outcome (Subtype 1; S1; n = 12) and better outcome (Subtype 3; S3; n = 24) that show differential stroma-regulated pathways. Collagen-targeted mass spectrometry imaging (MSI) with the same-tissue reference libraries, built from untargeted and targeted LC-MS/MS was used to spatially define the extracellular microenvironment from clinically-characterized, formalin-fixed, paraffin-embedded tissue sections. Collagen α-1(I) chain domains for discoidin-domain receptor and integrin binding showed distinctive spatial distribution within the tumor microenvironment. Hydroxylated proline (HYP)-containing peptides from the triple helical regions of fibrillar collagens distinguished S1 from S3 tumors. Exploratory machine learning on multiple peptides extracted from the tumor regions could distinguish S1 and S3 tumors (with an area under the receiver operating curve of ≥0.98; 95% confidence intervals between 0.976 and 1.00; and accuracies above 94%). An overall finding was that the extracellular microenvironment has a high potential to predict clinically relevant outcomes in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53efde0c062c9d0f5b393a6e975a03778bb11cae" target='_blank'>
              The Spatial Extracellular Proteomic Tumor Microenvironment Distinguishes Molecular Subtypes of Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Jade K Macdonald, Harrison B Taylor, Mengjun Wang, A. DelaCourt, Christin Edge, David N. Lewin, Naoto Kubota, Naoto Fujiwara, Fahmida Rasha, Cesia A. Marquez, A. Ono, Shiro Oka, K. Chayama, Sara Lewis, B. Taouli, Myron Schwartz, M. Fiel, Richard R Drake, Y. Hoshida, A. Mehta, Peggi M Angel
          </td>
          <td>2024-07-09</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Significance Treatment of prostate cancer is rapidly evolving with several promising drugs targeting different cell surface antigens. Selection of patients most likely to benefit from these therapies requires an understanding of how expression of these cell surface antigens varies across patients and how they change during disease progression, particularly in tumors that undergo lineage plasticity. Using immunohistochemistry and single-cell mRNA sequencing, we reveal heterogeneity of cell states across a cohort of advanced disease prostate cancer patients; this heterogeneity is not captured by conventional histology-based designations of adenocarcinoma and neuroendocrine prostate cancer. We show that these cell states can be identified by gene regulatory networks that could provide additional diagnostic precision based on their correlation with clinically relevant cell surface antigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c609499e96060bae9781113aade46c45aa614e3d" target='_blank'>
              Single-cell analysis of treatment-resistant prostate cancer: Implications of cell state changes for cell surface antigen–targeted therapies
              </a>
            </td>
          <td>
            S. Zaidi, Jooyoung Park, J. Chan, Martine P. Roudier, Jimmy L. Zhao, Anuradha Gopalan, K. Wadosky, Radhika A Patel, Erolcan Sayar, W. Karthaus, D. H. Kates, O. Chaudhary, Tianhao Xu, I. Masilionis, L. Mazutis, R. Chaligné, Aleksandar Z Obradovic, I. Linkov, A. Barlas, Achim A. Jungbluth, Natasha Rekhtman, J. Silber, Katia Manova-Todorova, Philip A Watson, Lawrence D. True, C. Morrissey, H. Scher, D. Rathkopf, Michael J Morris, David W. Goodrich, Jungmin Choi, Peter S Nelson, Michael C. Haffner, C. Sawyers
          </td>
          <td>2024-07-05</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumors have a complex metabolism that differs from most metabolic processes in healthy tissues. It is highly dynamic and driven by the tumor cells themselves, as well as by the non‐transformed stromal infiltrates and immune components. Each of these cell populations has a distinct metabolism that depends on both their cellular state and the availability of nutrients. Consequently, to fully understand the individual metabolic states of all tumor‐forming cells, correlative mass spectrometric imaging (MSI) up to cellular resolution with minimal metabolite shift needs to be achieved. By using a secondary ion mass spectrometer (SIMS) equipped with an Orbitrap mass analyzer, we present a workflow to image primary murine tumor tissues up to cellular resolution and correlate these ion images with post acquisition immunofluorescence or histological staining. In a murine breast cancer model, we could identify metabolic profiles that clearly distinguish tumor tissue from stromal cells and immune infiltrates. We demonstrate the robustness of the classification by applying the same profiles to an independent murine model of lung cancer, which is accurately segmented by histological traits. Our pipeline allows metabolic segmentation with simultaneous cell identification, which in the future will enable the design of subpopulation‐targeted metabolic interventions for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e6d2390afe35044a275be6c42d6f73bf5b238a" target='_blank'>
              Orbi‐SIMS Mediated Metabolomics Analysis of Pathogenic Tissue up to Cellular Resolution
              </a>
            </td>
          <td>
            Christine Kern, Astrid Scherer, Laura Gambs, Mariia Yuneva, H. Walczak, G. Liccardi, J. Saggau, Peter Kreuzaler, M. Rohnke
          </td>
          <td>2024-07-12</td>
          <td>Chemistry–Methods</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The discovery of genetic alterations in patient samples over the last 20 years has promoted a cell-autonomous view of proliferative expansion during T-cell acute lymphoblastic leukemia (T-ALL) development in the thymus. However, the potential contribution of non-cell-autonomous factors, particularly the impact of thymic epithelial cells (TECs) within the thymic niche during the initiation phase, remains unexplored. In this study, we employ a unique combination of a cell-based computational model of the thymus and in vivo experiments. We systematically analyze the impact of 12 cell-autonomous and non-autonomous factors, either alone or in combinations, on the proliferation of normal and malignant thymocytes with interleukin-7 receptor (IL7R) gain-of-function mutations or elevated IL7R levels, as observed in T-ALL patients. By simulating over 1500 scenarios, we show that while a dense TEC network favored the proliferation of normal thymocytes, it inhibited the proliferation of malignant lineages, which achieved their maximal proliferative capacity when TECs were sparsely distributed. Our in silico model predicts that certain mutations could accelerate proliferative expansion within a few days. This was experimentally validated, revealing rapid onset of thymus lymphoma and infiltration of malignant T-cells into other organs within 8 days of medaka (Oryzias latipes) embryonic development, thus revealing that modifications in the thymic niche and oncogenes in thymocytes together accelerate the disease development. Our results also suggest that negative feedback from the proliferative state inhibits differentiation of thymocytes, thereby prolonging the proliferative state and further fueling malignant expansion. Overall, this work reveals the critical impact of TEC-thymocyte interactions in both the initiation and progression of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc3507e8bc25fac66027e188edcf5b0ac8b541ea" target='_blank'>
              Altered thymic niche synergistically drives the massive proliferation of malignant thymocytes
              </a>
            </td>
          <td>
            E. Tsingos, Advaita M. Dick, B. Bajoghli
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. This study enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained genomic features and histopathological traits of primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, correlating ex vivo drug sensitivity with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program is associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributes to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine. One Sentence Summary This study establishes patient-derived tumor organoids (PDOs) from 31 Head and Neck Squamous Cell Carcinoma (HNSCC) patients, faithfully recapitulating characteristics of primary tumors and accurately predicting clinical responses to cisplatin treatment. We reveal intertumoral heterogeneity within PDOs and a hybrid epithelial-mesenchymal transition (hEMT) program conferring cisplatin resistance, highlighting amphiregulin (AREG) as a regulator of cellular plasticity and potential therapeutic target for HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82266d25cc2972a0e3e79d403bbffccb3c4b2f76" target='_blank'>
              Genomic and single-cell characterization of patient-derived tumor organoid models of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Young Min Park, De-Chen Lin
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Understanding the spatial dynamics within tissue microenvironments is crucial for deciphering cellular interactions and molecular signaling in living systems. These spatial characteristics govern cell distribution, extracellular matrix components, and signaling molecules, influencing local biochemical and biophysical conditions. Decoding these features offers insights into physiological processes, disease progression, and clinical outcomes. By elucidating spatial relationships between cell types, researchers uncover tissue architecture, cell communication networks, and microenvironment dynamics, aiding in the identification of biomarkers and therapeutic targets. Digital pathology imaging, including Hematoxylin and Eosin (H&E) staining, provides high-resolution histological information that offer intricate insights into cell-cell spatial relationships with greater details. However, current methods for capturing cell-cell spatial interactions are constrained by either methodological scopes or implementations restricted to script-level access. This limitation undermines generalizability and standardization, crucial for ensuring reproducibility. To address these limitations, we introduce SpatialQPFs, an extendable R package designed for extraction of interpretable spatial features from digital pathology images. By leveraging segmented cell information, our package provides researchers with a comprehensive toolkit for applying a range of spatial statistical methods within a stochastic process framework which includes analysis of point pattern data, areal data, and geostatistical data. This allows for a thorough analysis of cell spatial relationships, enhancing the depth and accuracy of spatial insights derived from the tissue, thereby empowering researchers to conduct comprehensive spatial analyses efficiently and reproducibly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da34c04318b2cac3bb2622ebe7b27ef2fe040d" target='_blank'>
              SpatialQPFs: An R package for deciphering cell-cell spatial relationship
              </a>
            </td>
          <td>
            Xiao Li
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b652a22909b12491feaf9d1745a2154d89fbc8e0" target='_blank'>
              Immune profiling of mouse lung adenocarcinoma paraffin tissues using multiplex immunofluorescence panel: a pilot study
              </a>
            </td>
          <td>
            Jie Zhai, A. Tamegnon, Mei Jiang, R. Pandurengan, Edwin Roger Parra
          </td>
          <td>2024-06-14</td>
          <td>Laboratory Animal Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Local inflammation in the pancreas is transient but imprints a durable epigenetic memory on epithelial cells, making them more amenable to oncogenic transformation. However, it is unclear whether epithelial cell heterogeneity is impacted by acute pancreatitis (AP) or whether population dynamics during regeneration contributes to the establishment of inflammation memory. To tackle those questions, we deployed experimental pancreatitis in mice and performed paired sequencing of transcriptomic and chromatin accessibility profiles at single nucleus resolution. We documented cell type abundance but also applied integrative analyses to infer phenotypically-distinct clusters of mesenchymal and exocrine cells. We found that AP perturbs a subset of “idling” acinar cells, which separate from more canonical “secretory” acini based on a more diversified proteome, which include elevated expression of signal transduction receptors. We linked acinar cell heterogeneity to epigenetic differences that also endow idling cells with superior plasticity. These constitute about 40% of acinar cells but can proliferate and skew their phenotype in response to AP. This leads to a remarkable recovery of pancreas histology and function, but also to the dissemination of idling-like features across the exocrine parenchyma. Mechanistically, idling acinar cells are characterized by enhanced transcriptional activity and protein synthesis. After recovery from pancreatitis, acini show elevation of both and establishment of chronic Unfolded Protein Response (UPR). We finally demonstrated that AP-primed pancreata show signs of elevated UPR and that ER stress promotes acinar cell metaplasia. Our data interrogate phenotypical dynamics during tissue regeneration to identify cell states amenable to epigenetic imprinting. They also suggest that UPR-alleviating strategies might curtail the risk of developing pancreatic cancer for individuals who experiences AP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0889f4e0ef737079e14a1df3e2a78bed4691d30d" target='_blank'>
              Population dynamics after pancreatitis dictates long-lasting epigenetic reprogramming and mediates tumor predisposition
              </a>
            </td>
          <td>
            Marco Fantuz, J. Liebig, Giovanni Fontana, Valerio Iebba, Carmine Carbone, Sören Lukassen, Christian Conrad, Alessandro Carrer
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metastasis, the leading cause of cancer-related deaths, involves a complex cascade of events, including extravasation. Despite extensive research into metastasis, the mechanisms underlying extravasation remain unclear. Molecular targeted therapies have advanced cancer treatment, yet their efficacy is limited, prompting exploration into novel therapeutic targets. Here, we showed the association of polyploidy in MDA-MB-231 breast cancer cells and their extravasation, using microfluidic systems to reproduce the in vivo microvascular environment. We observed enhanced extravasation in polyploid cells alongside upregulated expression of genes involved in cell-substrate adhesion and cell mechanical dynamics. These findings offer insights into the relationship between polyploidy and extravasation, highlighting potential targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53afd6a9eed41e19e707b6c0bf0023754788d413" target='_blank'>
              Polyploidy of MDA-MB-231 cells drives increased extravasation with enhanced cell-matrix adhesion
              </a>
            </td>
          <td>
            Satomi Hirose, Tatsuya Osaki, R. Kamm
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is the most common form of pancreatic cancer, a disease with dismal overall survival. Advances in treatment are hindered by a lack of preclinical models. Here, we show how a personalized organotypic “avatar” created from resected tissue allows spatial and temporal reporting on a complete in situ tumor microenvironment and mirrors clinical responses. Our perfusion culture method extends tumor slice viability, maintaining stable tumor content, metabolism, stromal composition, and immune cell populations for 12 days. Using multiplexed immunofluorescence and spatial transcriptomics, we identify immune neighborhoods and potential for immunotherapy. We used avatars to assess the impact of a preclinically validated metabolic therapy and show recovery of stromal and immune phenotypes and tumor redifferentiation. To determine clinical relevance, we monitored avatar response to gemcitabine treatment and identify a patient avatar-predictable response from clinical follow-up. Thus, avatars provide valuable information for syngeneic testing of therapeutics and a truly personalized therapeutic assessment platform for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361225cc3eb1a10b3d464aab534da9288640614c" target='_blank'>
              Development of human pancreatic cancer avatars as a model for dynamic immune landscape profiling and personalized therapy
              </a>
            </td>
          <td>
            Daniel Hughes, Alice Evans, Simei Go, Michael Eyres, Liuliu Pan, Somnath Mukherjee, Z. Soonawalla, Frances Willenbrock, Eric O’Neill
          </td>
          <td>2024-07-05</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The genomic landscape of clear cell renal cell carcinoma (ccRCC) has a considerable intra-tumor heterogeneity, which is a significant obstacle in the field of precision oncology and plays a pivotal role in metastasis, recurrence, and therapeutic resistance of cancer. The mechanisms of intra-tumor heterogeneity in ccRCC have yet to be fully established. We integrated single-cell RNA sequencing (scRNA-seq) and transposase-accessible chromatin sequencing (scATAC-seq) data from a single-cell multi-omics perspective. Based on consensus non-negative matrix factorization (cNMF) algorithm, functionally heterogeneous cancer cells were classified into metabolism, inflammatory, and EMT meta programs, with spatial transcriptomics sequencing (stRNA-seq) providing spatial information of such disparate meta programs of cancer cells. The bulk RNA sequencing (RNA-seq) data revealed high clinical prognostic values of functionally heterogeneous cancer cells of three meta programs, with transcription factor regulatory network and motif activities revealing the key transcription factors that regulate functionally heterogeneous ccRCC cells. The interactions between varying meta programs and other cell subpopulations in the microenvironment were investigated. Finally, we assessed the sensitivity of cancer cells of disparate meta programs to different anti-cancer agents. Our findings inform on the intra-tumor heterogeneity of ccRCC and its regulatory networks and offers new perspectives to facilitate the designs of rational therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56d844ad93caafe1a1fe2435935aca07cee4238" target='_blank'>
              Unfolding the mysteries of heterogeneity from a high-resolution perspective: integration analysis of single-cell multi-omics and spatial omics revealed functionally heterogeneous cancer cells in ccRCC
              </a>
            </td>
          <td>
            Jie Zheng, Wenhao Lu, Chengbang Wang, Shaohua Chen, Qingyun Zhang, Cheng Su
          </td>
          <td>2024-06-26</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immunotherapies have shown promising results in treating patients with hematological malignancies like multiple myeloma, which is an incurable but treatable bone marrow-resident plasma cell cancer. Choosing the most efficacious treatment for a patient remains a challenge in such cancers. However, pre-clinical assays involving patient-derived tumor cells co-cultured in an ex vivo reconstruction of immune-tumor micro-environment have gained considerable notoriety over the past decade. Such assays can characterize a patient’s response to several therapeutic agents including immunotherapies in a high-throughput manner, where bright-field images of tumor (target) cells interacting with effector cells (T cells, Natural Killer (NK) cells, and macrophages) are captured once every 30 minutes for upto six days. Cell detection, tracking, and classification of thousands of cells of two or more types in each frame is bound to test the limits of some of the most advanced computer vision tools developed to date and requires a specialized approach. We propose TLCellClassifier (time-lapse cell classifier) for live cell detection, cell tracking, and cell type classification, with enhanced accuracy and efficiency obtained by integrating convolutional neural networks (CNN), metric learning, and long short-term memory (LSTM) networks, respectively. State-of-the-art computer vision software like KTH-SE and YOLOv8 are compared with TLCellClassifier, which shows improved accuracy in detection (CNN) and tracking (metric learning). A two-stage LSTM-based cell type classification method is implemented to distinguish between multiple myeloma (tumor/target) cells and macrophages/monocytes (immune/effector cells). Validation of cell type classification was done both using synthetic datasets and ex vivo experiments involving patient-derived tumor/immune cells. Availability and implementation https://github.com/QibingJiang/cell classification ml">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0af147d9156c5d559b6eeac8f7b6ba9f3178525c" target='_blank'>
              TLCellClassifier: Machine Learning Based Cell Classification for Bright-Field Time-Lapse Images
              </a>
            </td>
          <td>
            Qibing Jiang, P. Sudalagunta, M. Meads, Xiaohong Zhao, Alexandra Achille, David Noyes, Maria Silva, Rafael R. Canevarolo, Ken Shain, Ariosto Silva, Wei Zhang
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND AIMS
Surgical resection serves as the principal curative strategy for hepatocellular carcinoma (HCC), yet the incidence of postoperative recurrence remains alarmingly high. However, the spatial molecular structural alterations contributing to postoperative recurrence in HCC are still poorly understood.


APPROACH RESULTS
We employed imaging mass cytometry to profile the in-situ expression of 33 proteins within 358,729 single cells of 92 clinically annotated surgical specimens from 46 patients who were treated with surgical resections for primary and relapsed tumors. We revealed the recurrence progression of HCC was governed by the dynamic spatial distribution and functional interplay of diverse cell types across adjacent normal, tumor margin, and intratumor regions. Our exhaustive analyses revealed an aggressive, immunosuppression-related spatial ecosystem in relapsed HCC. Additionally, we illustrated the prominent implications of the TME of tumor margins in association with relapse HCC. Moreover, we identified a novel subpopulation of dendritic cells (PDL1+CD103+ DCs) enriched in the peritumoral area that correlated with early postoperative recurrence, which was further validated in an external cohort. Through the analysis of scRNA-seq data, we found the interaction of PDL1+CD103+ DCs with regulatory T cells and exhausted T cells enhanced immunosuppression and immune escape via multiple ligand-receptor pathways.


CONCLUSIONS
We comprehensively depicted the spatial landscape of single-cell dynamics and multicellular architecture within primary and relapsed HCC. Our findings highlight spatial organization is a prominent determinant of HCC recurrence and provide a valuable insight into the immune evasion mechanisms driving recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a63dde0bf07aaed3ddea67316c202ee6463051a" target='_blank'>
              Spatial proteomic landscape of primary and relapsed hepatocellular carcinoma reveals immune escape characteristics in early relapse.
              </a>
            </td>
          <td>
            Meilin Yang, Xiaoyi Song, Fan Zhang, Mingan Li, Wuguang Chang, Zheyan Wang, Man Li, Hong Shan, Dan Li
          </td>
          <td>2024-06-20</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Severe Coronavirus disease 2019 (COVID-19) induces heterogeneous and progressive diffuse alveolar damage (DAD) highly disrupting lung tissue architecture and homeostasis, hampering disease management leading to fatal outcomes. Characterizing DAD pathophysiology across disease progression is of ultimate importance to better understand the molecular and cellular features driving different DAD patterns and to optimize treatment strategies. To contextualize the interplay between cell types and assess their distribution, spatial transcriptomics (ST) techniques have emerged, allowing unprecedented resolution to investigate spatial architecture of tissues. To this end, post-mortem lung tissue provides valuable insights into cellular composition and their spatial relationships at the time of death. Here, we have leveraged VisumST technology in post-mortem COVID-19 induced acute and proliferative DAD lungs including control samples with normal morphological appearance, to unravel the immunopathological mechanisms underlying DAD, providing novel insights into cellular and molecular communication events driving DAD progression in fatal COVID-19. We report a progressive loss of endothelial cell types, pneumocytes type I and natural killer cells coupled with a continuous increase of myeloid and stromal cells, mostly peribronchial fibroblasts, over disease progression. Spatial organization analysis identified variable cellular compartments, ranging from major compartments defined by cell type lineages in control lungs to increased and more specific compartmentalization including immune-specific clusters across DAD spectrum. Importantly, spatially informed ligand-receptor interaction (LRI) analysis revealed an intercellular communication signature defining COVID-19 induced DAD lungs. Transcription factor (TF) activity enrichment analysis identified TGF-B pathway as DAD driver, highlighting SMAD3 and SMAD7 TFs activity role during lung fibrosis. Integration of deregulated LRIs and TFs activity allowed us to propose a downstream intracellular signaling pathway in peribronchial fibroblasts, suggesting potential novel therapeutic targets. Finally, spatio-temporal trajectories analysis provided insights into the alveolar epithelium regeneration program, characterizing markers of pneumocytes type II differentiation towards pneumocytes type I. In conclusion, we provide a spatial characterization of lung tissue architecture upon COVID-19 induced DAD progression, identifying molecular and cellular hallmarks that may help optimize treatment and patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a4db747d52faae9abbf25e5031f73c4597d1cc" target='_blank'>
              Spatial transcriptomics unveils the in situ cellular and molecular hallmarks of the lung in fatal COVID-19
              </a>
            </td>
          <td>
            C. García-Prieto, Eva Musulen, V. Davalos, Gerardo Ferrer, D. Grases, E. Porta, Belén Pérez-Miés, T. Caniego-Casas, José Palacios, Xavier Saenz-Sardà, Elisabet Englund, Manel Esteller
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Neuroblastoma is a heterogeneous disease with adrenergic (ADRN)- and therapy resistant mesenchymal (MES)-like cells driven by distinct transcription factor networks. Here, we investigate the expression of immunotherapeutic targets in each neuroblastoma subtype and propose pan-neuroblastoma and cell state specific targetable cell-surface proteins. Methods We characterized cell lines, patient-derived xenografts, and patient samples as ADRN-dominant or MES- dominant to define subtype-specific and pan-neuroblastoma gene sets. Targets were validated with ChIP- sequencing, immunoblotting, and flow cytometry in neuroblastoma cell lines and isogenic ADRN-to-MES transition cell line models. Finally, we evaluated the activity of MES-specific agents in vivo and in vitro. Results Most immunotherapeutic targets being developed for neuroblastoma showed significantly higher expression in the ADRN subtype with limited expression in MES-like tumor cells. In contrast, CD276 (B7-H3) and L1CAM maintained expression across both ADRN and MES states. We identified several receptor tyrosine kinases (RTKs) enriched in MES-dominant samples and showed that AXL targeting with ADCT-601 was potently cytotoxic in MES-dominant cell lines and showed specific anti-tumor activity in a MES cell line-derived xenograft. Conclusions Immunotherapeutic strategies for neuroblastoma must address the potential of epigenetic downregulation of antigen density as a mechanism for immune evasion. We identified several RTKs as candidate MES-specific immunotherapeutic target proteins for the elimination of therapy-resistant cells. We hypothesize that the phenomena of immune escape will be less likely when targeting pan-neuroblastoma cell surface proteins such as B7-H3 and L1CAM, and/or dual targeting strategies that consider both the ADRN- and MES-cell states. Key Points Cellular plasticity influences the abundance of immunotherapeutic targets. Subtype-specific targets may be susceptible to epigenetically-mediated downregulation. Immunotherapeutic targets in development, B7-H3 and L1CAM, show “pan-subtype” expression. Importance of Study Neuroblastoma is a lethal childhood malignancy that shows cellular plasticity in response to anti-cancer therapies. Several plasma membrane proteins are being developed as immunotherapeutic targets in this disease. Here we define which cell surface proteins are susceptible to epigenetically regulated downregulation during an adrenergic to mesenchymal cell state switch and propose immunotherapeutic strategies to anticipate and circumvent acquired immunotherapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c43da3990dff1c35a4148b920ff54013c4ee8ec" target='_blank'>
              Lineage-dependence of the neuroblastoma surfaceome defines tumor cell state-dependent and independent immunotherapeutic targets
              </a>
            </td>
          <td>
            N. Kendsersky, Michal Odrobina, Nathaniel W. Mabe, Alvin Farrel, Liron D. Grossmann, M. Tsang, David Groff, A. Wolpaw, F. Zammarchi, Patrick H van Berkel, Chi V. Dang, Y. Mossé, K. Stegmaier, John M. Maris
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Simple Summary Immune checkpoint blockade has shown remarkable efficacy across various cancers but has failed to improve outcomes in patients with relapsed medulloblastoma (MB). While it is thought that the cold, immunosuppressive tumor microenvironment of MB accounts for this poor efficacy, the precise mechanisms contributing to immune suppression in this context remain unclear. In this study, we explore the immune landscape of MB using a previously unexplored syngeneic mouse model. Our study demonstrates that this model faithfully recapitulates the cold, immunosuppressive tumor microenvironment observed in human disease. Importantly, our research uncovers mechanisms employed by myeloid cells and tumor cells to evade immune detection while highlighting the therapeutic potential of targeting V-domain Ig Suppressor of T-cell Activation (VISTA), a novel inhibitory immune checkpoint, to enhance anti-tumor immunity. Abstract Background: Relapsed medulloblastoma (MB) poses a significant therapeutic challenge due to its highly immunosuppressive tumor microenvironment. Immune checkpoint inhibitors (ICIs) have struggled to mitigate this challenge, largely due to low T-cell infiltration and minimal PD-L1 expression. Identifying the mechanisms driving low T-cell infiltration is crucial for developing more effective immunotherapies. Methods: We utilize a syngeneic mouse model to investigate the tumor immune microenvironment of MB and compare our findings to transcriptomic and proteomic data from human MB. Results: Flow cytometry reveals a notable presence of CD45hi/CD11bhi macrophage-like and CD45int/CD11bint microglia-like tumor-associated macrophages (TAMs), alongside regulatory T-cells (Tregs), expressing high levels of the inhibitory checkpoint molecule VISTA. Compared to sham control mice, the CD45hi/CD11bhi compartment significantly expands in tumor-bearing mice and exhibits a myeloid-specific signature composed of VISTA, CD80, PD-L1, CTLA-4, MHCII, CD40, and CD68. These findings are corroborated by proteomic and transcriptomic analyses of human MB samples. Immunohistochemistry highlights an abundance of VISTA-expressing myeloid cells clustering at the tumor–cerebellar border, while T-cells are scarce and express FOXP3. Additionally, tumor cells exhibit immunosuppressive properties, inhibiting CD4 T-cell proliferation in vitro. Identification of VISTA’s binding partner, VSIG8, on tumor cells, and its correlation with increased VISTA expression in human transcriptomic analyses suggests a potential therapeutic target. Conclusions: This study underscores the multifaceted mechanisms of immune evasion in MB and highlights the therapeutic potential of targeting the VISTA–VSIG axis to enhance anti-tumor responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8972b71fd9c803882e0f815586aa8205257f7159" target='_blank'>
              VISTA Emerges as a Promising Target against Immune Evasion Mechanisms in Medulloblastoma
              </a>
            </td>
          <td>
            S. Harrabi, A. Hormigo, Natalia Muñoz Perez, Juliana Pensabene, Jr Phillip M. Galbo, Negar Sadeghipour, Joanne Xiu, Kirsten Moziak, Rita M. Yazejian, Rachel L. Welch, W. R. Bell, Soma Sengupta, Sonikpreet Aulakh, Charles G Eberhart, David M Loeb, Emad Eskandar, Deyou Zheng, Xingxing Zang, Allison M. Martin
          </td>
          <td>2024-07-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Accumulating evidence suggests that the tumor immune microenvironment (TIME) significantly influences the response to immunotherapy, yet this complex relationship remains elusive. To address this issue, we developed TimiGP-Response (TIME Illustration based on Gene Pairing designed for immunotherapy Response), a computational framework leveraging single-cell and bulk transcriptomic data, along with response information, to construct cell-cell interaction networks associated with responders and estimate the role of immune cells in treatment response. This framework was showcased in triple-negative breast cancer treated with immune checkpoint inhibitors targeting the PD-1:PD-L1 interaction, and orthogonally validated with imaging mass cytometry. As a result, we identified CD8+ GZMB+ T cells associated with responders and its interaction with regulatory T cells emerged as a potential feature for selecting patients who may benefit from these therapies. Subsequently, we analyzed 3,410 patients with seven cancer types (melanoma, non-small cell lung cancer, renal cell carcinoma, metastatic urothelial carcinoma, hepatocellular carcinoma, breast cancer, and esophageal cancer) treated with various immunotherapies and combination therapies, as well as several chemo- and targeted therapies as controls. Using TimiGP-Response, we depicted the pan-cancer immune landscape associated with immunotherapy response at different resolutions. At the TIME level, CD8 T cells and CD4 memory T cells were associated with responders, while anti-inflammatory (M2) macrophages and mast cells were linked to non-responders across most cancer types and datasets. Given that T cells are the primary targets of these immunotherapies and our TIME analysis highlights their importance in response to treatment, we portrayed the pan-caner landscape on 40 T cell subtypes. Notably, CD8+ and CD4+ GZMK+ effector memory T cells emerged as crucial across all cancer types and treatments, while IL-17-producing CD8+ T cells were top candidates associated with immunotherapy non-responders. In summary, this study provides a computational method to study the association between TIME and response across the pan-cancer immune landscape, offering resources and insights into immune cell interactions and their impact on treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883be60621995b3d688f88f7d3f09de5740baf9e" target='_blank'>
              TimiGP-Response: the pan-cancer immune landscape associated with response to immunotherapy
              </a>
            </td>
          <td>
            Chenyang Li, Wei Hong, A. Reuben, Linghua Wang, Anirban Maitra, Jianjun Zhang, Chao Cheng
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Interactions between mutant cells and their environment play a key role in determining cancer susceptibility. However, our understanding of how the pre-cancer microenvironment contributes to early tumorigenesis remains limited. Here, we show that newly emerging tumours at their most incipient stages shape their microenvironment in a critical process that determines their survival. Analysis of nascent squamous tumours in the upper gastrointestinal tract of the mouse reveals that the stress response of early tumour cells instructs the underlying mesenchyme to form a supportive “pre-cancer niche”, which dictates the long-term outcome of epithelial lesions. Stimulated fibroblasts beneath emerging tumours activate a wound healing response that triggers a dramatic remodelling of the underlying extracellular matrix, resulting in the formation of a fibronectin-rich stromal scaffold that promotes tumour growth. Functional heterotypic 3D culture assays and in vivo grafting experiments, combining carcinogen-free healthy epithelium and tumour-derived stroma, demonstrate that the pre-cancerous niche alone is sufficient to confer tumour properties to healthy epithelial cells. We propose a model where both mutations and the stromal response to genetic stress defines the likelihood of early tumours to survive and progress towards more advanced disease stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be23254ceedda48875f58bb5d700c5fd073374a7" target='_blank'>
              Pre-cancerous Niche Remodelling Dictates Nascent Tumour Survival
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. E. Rojo Arias, Y. Dang, S. Han, M. T. Bejar, B. Colom, J. C. Fowler, P. Jones, S. Rulands, B. D. Simons, M. P. Alcolea
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer/testis antigens (CTAs) are widely expressed in melanoma and lung cancer, emerging as promising targets for vaccination strategies and T-cell-based therapies in these malignancies. Despite recognizing the essential impact of intratumoral heterogeneity on clinical responses to immunotherapy, our understanding of intratumoral heterogeneity in CTA expression has remained limited. We employed single-cell mRNA sequencing to delineate the CTA expression profiles of cancer cells in clinically derived melanoma and lung cancer samples. Our findings reveal a high degree of intratumoral transcriptional heterogeneity in CTA expression. In melanoma, every cell expressed at least one CTA. However, most individual CTAs, including the widely used therapeutic targets NY-ESO-1 and MAGE, were confined to subpopulations of cells and were uncoordinated in their expression, resulting in mosaics of cancer cells with diverse CTA profiles. Coordinated expression was observed, however, mainly among highly structurally and evolutionarily related CTA genes. Importantly, a minor subset of CTAs, including PRAME and several members of the GAGE and MAGE-A families, were homogenously expressed in melanomas, highlighting their potential as therapeutic targets. Extensive heterogeneity in CTA expression was also observed in lung cancer. However, the frequency of CTA-positive cancer cells was notably lower and homogenously expressed CTAs were only identified in one of five tumors in this cancer type. Our findings underscore the need for careful CTA target selection in immunotherapy development and clinical testing and offer a rational framework for identifying the most promising candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d59cb995a024b6b3c733ef9a23d2fbf82f85441" target='_blank'>
              Single-cell sequencing unveils extensive intratumoral heterogeneity of cancer/testis antigen expression in melanoma and lung cancer
              </a>
            </td>
          <td>
            S. Traynor, Mie K. Jakobsen, Tina M Green, Hana Komic, Y. Palarasah, C. B. Pedersen, Henrik J Ditzel, Fredrik B Thoren, Per Guldberg, M. Gjerstorff
          </td>
          <td>2024-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Immunotherapies have shown great promise in pleural mesothelioma (PM), yet most patients still do not achieve significant clinical response, highlighting the importance of improving understanding of the tumor microenvironment (TME). Here, we utilized high-throughput, single-cell RNA-sequencing to de novo identify 54 expression programs and construct a comprehensive cellular catalogue of the PM TME. We found four cancer-intrinsic programs associated with poor disease outcome and a novel fetal-like, endothelial cell population that likely responds to VEGF signaling and promotes angiogenesis. Throughout cellular compartments, we observe substantial difference in the TME associated with a cancer-intrinsic sarcomatoid signature, including enrichment in fetal-like endothelial cells, CXCL9+ macrophages, cytotoxic, exhausted, and regulatory T cells, which we validated using imaging and bulk deconvolution analyses on independent cohorts. Finally, we show, both computationally and experimentally, that NKG2A-HLA-E interaction between NK and tumor cells represents an important new therapeutic axis in PM, especially for epithelioid cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c84bfe3e9c66b621b1832492f79b2264c3e648" target='_blank'>
              Single cell view of tumor microenvironment gradients in pleural mesothelioma.
              </a>
            </td>
          <td>
            Bruno Giotti, Komal Dolasia, William Zhao, Peiwen Cai, Robert Sweeney, Elliot Merritt, Evgeny Kiner, Grace S Kim, Atharva Bhagwat, Thinh Nguyen, Samarth Hegde, Bailey G Fitzgerald, S. Shroff, T. Dawson, M. Garcia-Barros, Jamshid Abdul-Ghafar, Rachel Chen, Sacha Gnjatic, Alan Soto, Rachel Brody, S. Kim-Schulze, Zhihong Chen, Kristin G Beaumont, Miram Merad, R.M. Flores, R. Sebra, Amir Horowitz, Thomas U. Marron, Anna Tocheva, Andrea Wolf, A. Tsankov
          </td>
          <td>2024-07-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is a complex ecosystem of diverse cell types whose interactions govern tumor growth and clinical outcome. While the TME’s impact on immunotherapy has been extensively studied, its role in chemotherapy response remains less explored. To address this, we developed DECODEM (DEcoupling Cell-type-specific Outcomes using DEconvolution and Machine learning), a generic computational framework leveraging cellular deconvolution of bulk transcriptomics to associate the gene expression of individual cell types in the TME with clinical response. Employing DECODEM to analyze the gene expression of breast cancer (BC) patients treated with neoadjuvant chemotherapy, we find that the gene expression of specific immune cells (myeloid, plasmablasts, B-cells) and stromal cells (endothelial, normal epithelial, CAFs) are highly predictive of chemotherapy response, going beyond that of the malignant cells. These findings are further tested and validated in a single-cell cohort of triple negative breast cancer. To investigate the possible role of immune cell-cell interactions (CCIs) in mediating chemotherapy response, we extended DECODEM to DECODEMi to identify such CCIs, validated in single-cell data. Our findings highlight the importance of active pre-treatment immune infiltration for chemotherapy success. The tools developed here are made publicly available and are applicable for studying the role of the TME in mediating response from readily available bulk tumor expression in a wide range of cancer treatments and indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec38ea4f715fa72ae5e3c095b32d63b6ede6629" target='_blank'>
              The expression patterns of different cell types and their interactions in the tumor microenvironment are predictive of breast cancer patient response to neoadjuvant chemotherapy
              </a>
            </td>
          <td>
            S. R. Dhruba, S. Sahni, Binbin Wang, Di Wu, P. Rajagopal, Yael Schmidt, E. Shulman, Sanju Sinha, Stephen-John Sammut, Carlos Caldas, Kun Wang, E. Ruppin
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H M Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary We utilized publicly available perturbation phosphoproteomic data to construct models elucidating cell state transitions across multiple breast cancer and normal breast tissue-derived cell lines. Employing a hybrid methodology, which integrates machine learning and mechanistic modeling, we separated luminal, basal, and normal cell states and revealed core networks that control cell state transitions. We determined causal connections within the core networks and developed interpretable mechanistic models that elucidated the drivers of cell phenotypes. Significantly, these models can predict synergistic drug combinations capable of potentially reversing oncogenic transformation in breast cancer cell lines. Our methodology will enable designer approaches to identify targeted perturbations that convert cell states and mechanistically underpin therapeutic interventions. Abstract Understanding signaling patterns of transformation and controlling cell phenotypes is a challenge of current biology. Here we applied a cell State Transition Assessment and Regulation (cSTAR) approach to a perturbation dataset of single cell phosphoproteomic patterns of multiple breast cancer (BC) and normal breast tissue-derived cell lines. Following a separation of luminal, basal, and normal cell states, we identified signaling nodes within core control networks, delineated causal connections, and determined the primary drivers underlying oncogenic transformation and transitions across distinct BC subtypes. Whereas cell lines within the same BC subtype have different mutational and expression profiles, the architecture of the core network was similar for all luminal BC cells, and mTOR was a main oncogenic driver. In contrast, core networks of basal BC were heterogeneous and segregated into roughly four major subclasses with distinct oncogenic and BC subtype drivers. Likewise, normal breast tissue cells were separated into two different subclasses. Based on the data and quantified network topologies, we derived mechanistic cSTAR models that serve as digital cell twins and allow the deliberate control of cell movements within a Waddington landscape across different cell states. These cSTAR models suggested strategies of normalizing phosphorylation networks of BC cell lines using small molecule inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/092c4f2c92e0be64eb32694e8a69506f512145ae" target='_blank'>
              Cell State Transition Models Stratify Breast Cancer Cell Phenotypes and Reveal New Therapeutic Targets
              </a>
            </td>
          <td>
            Oleksii S. Rukhlenko, Hiroaki Imoto, Ayush Tambde, Amy McGillycuddy, Philipp Junk, Anna Tuliakova, Walter Kolch, Boris N. Kholodenko
          </td>
          <td>2024-06-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Transcription factors (TFs) drive significant cellular changes in response to environmental cues and intercellular signaling. Neighboring cells influence TF activity and, consequently, cellular fate and function. Spatial transcriptomics (ST) captures mRNA expression patterns across tissue samples, enabling characterization of the local microenvironment. However, these datasets have not been fully leveraged to systematically estimate TF activity governing cell identity. Here, we present STAN (Spatially informed Transcription factor Activity Network), a linear mixed-effects computational method that predicts spot-specific, spatially informed TF activities by integrating curated TF-target gene priors, mRNA expression, spatial coordinates, and morphological features from corresponding imaging data. We tested STAN using lymph node, breast cancer, and glioblastoma ST datasets to demonstrate its applicability by identifying TFs associated with specific cell types, spatial domains, pathological regions, and ligand-receptor pairs. STAN augments the utility of ST to reveal the intricate interplay between TFs and spatial organization across a spectrum of cellular contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c72cf6525dcea64d3c42d0ffde8f466249deeae" target='_blank'>
              STAN, a computational framework for inferring spatially informed transcription factor activity across cellular contexts
              </a>
            </td>
          <td>
            Linan Zhang, April Sagan, Bin Qin, Baoli Hu, H. Osmanbeyoglu
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, O. Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The haematological malignancy multiple myeloma is associated with skewed T-cell activation and function. T-cell alterations are detectable in asymptomatic myeloma precursor conditions and have the potential to identify precursor patients at imminent risk of progression. However, what myeloma-associated T-cells alterations represent mechanistically, how they relate to tumour burden and gene expression, and what influences high inter-patient variability in immune composition remains unknown. Here, we assembled the largest ever dataset of published and newly-generated single-cell RNA and TCR sequencing of the marrow and blood from patients with myeloma, precursor conditions, and age-matched non-cancer controls. We show myeloma is not associated with T-cell exhaustion and instead defined by a pattern of T-cell differentiation resembling antigen-driven terminal memory differentiation. Myeloma-associated T-cell differentiation was dependent on tumour-intrinsic features including tumour burden and tumour expression of antigen-presentation genes. Expanded TCR clones accumulating in myeloma were not enriched for viral specificity and were detected in effector states in highly infiltrated marrows. Together, these results suggest anti-tumour immunity drives a novel form of cancer-associated T-cell memory differentiation in myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d674fe6f864f5fa613feeb41c61f5327c58bd541" target='_blank'>
              Tumour-intrinsic features shape T-cell differentiation through myeloma disease evolution
              </a>
            </td>
          <td>
            K. Foster, Elise Rees, L. Ainley, Eileen M. Boyle, Lydia Lee, Gwennan Ward, D. Galas-Filipowicz, Anna Mikolajczak, Emma J Lyon, Dylan Jankovic, Jasmine Rahman, Mahima Turakhia, Imran Uddin, Gordon Beattie, Yvette Hoade, Catherine Zhu, J. Reading, Ieuan G Walker, Michael Chapman, Karthik Ramasamy, Javier Herrero, B. Chain, S. Quezada, Kwee Yong
          </td>
          <td>2024-06-23</td>
          <td>None</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Durable serological memory following vaccination is critically dependent on the production and survival of long-lived plasma cells (LLPCs). Yet, the factors that control LLPC specification and survival remain poorly resolved. Using intravital two-photon imaging, we find that in contrast to most plasma cells (PCs) in the bone marrow (BM), LLPCs are uniquely sessile and organized into clusters that are dependent on APRIL, an important survival factor. Using deep, bulk RNA sequencing, and surface protein flow-based phenotyping, we find that LLPCs express a unique transcriptome and phenotype compared to bulk PCs, fine-tuning expression of key cell surface molecules, CD93, CD81, CXCR4, CD326, CD44, and CD48, important for adhesion and homing. Conditional deletion of Cxcr4 in PCs following immunization leads to rapid mobilization from the BM, reduced survival of antigen-specific PCs, and ultimately accelerated decay of antibody titer. In naïve mice, the endogenous LLPCs BCR repertoire exhibits reduced diversity, reduced somatic mutations, and increased public clones and IgM isotypes, particularly in young mice, suggesting LLPC specification is non-random. As mice age, the BM PC compartment becomes enriched in LLPCs, which may outcompete and limit entry of new PCs into the LLPC niche and pool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae2a0a86a1858977f32349e08a4ecc5beb84322" target='_blank'>
              Fine-tuning spatial-temporal dynamics and surface receptor expression support plasma cell-intrinsic longevity
              </a>
            </td>
          <td>
            Zhixin Jing, Phillip M. Galbo, Luis Ovando, Megan Demouth, Skylar Welte, Rosa Park, K. Chandran, Yinghao Wu, Thomas MacCarthy, Deyou Zheng, David R Fooksman
          </td>
          <td>2024-06-18</td>
          <td>eLife</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a complex tumor microenvironment (TME) that promotes cancer progression. The tumor stroma is characterized by cancer-associated fibroblasts (CAFs) that secrete factors stimulating tumor growth, invasion, and immunosuppression. Macrophages tend to polarize toward a pro-tumor phenotype. Various other components, including endothelial cells, pericytes, and neural cells, likely contribute to the complexity of the PDAC TME. Epidemiological investigations have highlighted obesity as a risk factor for pancreatic cancer, suggesting that adipocytes may also play a significant role. This manuscript offers an overview of the main actors in TME, as well as dysregulated signaling pathways and gene mutations. It underlines how strategies modulating the TME hold promise for improving outcomes. Lastly, it describes how integrating multi-omic approaches may represent a future trajectory to explore novel therapeutic possibilities. Abstract Pancreatic ductal adenocarcinoma (PDAC) presents significant oncological challenges due to its aggressive nature and poor prognosis. The tumor microenvironment (TME) plays a critical role in progression and treatment resistance. Non-neoplastic cells, such as cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), contribute to tumor growth, angiogenesis, and immune evasion. Although immune cells infiltrate TME, tumor cells evade immune responses by secreting chemokines and expressing immune checkpoint inhibitors (ICIs). Vascular components, like endothelial cells and pericytes, stimulate angiogenesis to support tumor growth, while adipocytes secrete factors that promote cell growth, invasion, and treatment resistance. Additionally, perineural invasion, a characteristic feature of PDAC, contributes to local recurrence and poor prognosis. Moreover, key signaling pathways including Kirsten rat sarcoma viral oncogene (KRAS), transforming growth factor beta (TGF-β), Notch, hypoxia-inducible factor (HIF), and Wnt/β-catenin drive tumor progression and resistance. Targeting the TME is crucial for developing effective therapies, including strategies like inhibiting CAFs, modulating immune response, disrupting angiogenesis, and blocking neural cell interactions. A recent multi-omic approach has identified signature genes associated with anoikis resistance, which could serve as prognostic biomarkers and targets for personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23cd29cbd6982ee3e4b132b1c93695ffca70902" target='_blank'>
              Decoding the Intricate Landscape of Pancreatic Cancer: Insights into Tumor Biology, Microenvironment, and Therapeutic Interventions
              </a>
            </td>
          <td>
            A. Argentiero, Alessandro Andriano, Ingrid Catalina Caradonna, Giulia de Martino, V. Desantis
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Achieving long-term disease control using therapeutic immunomodulation is a long-standing concept with a strong tradition in blood malignancies. Besides allogeneic hematopoietic stem cell transplantation that continues to provide potentially curative treatment for otherwise challenging diagnoses, recent years have seen impressive progress in immunotherapies for leukemias and lymphomas with immune checkpoint blockade, bispecific monoclonal antibodies, and CAR T cell therapies. Despite their success, non-response, relapse, and immune toxicities remain frequent, thus prioritizing the elucidation of the underlying mechanisms and identifying predictive biomarkers. The increasing availability of single-cell genomic tools now provides a system’s immunology view to resolve the molecular and cellular mechanisms of immunotherapies at unprecedented resolution. Here, we review recent studies that leverage these technological advancements for tracking immune responses, the emergence of immune resistance, and toxicities. As single-cell immune monitoring tools evolve and become more accessible, we expect their wide adoption for routine clinical applications to catalyze more precise therapeutic steering of personal immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a83dbce941baa359f9c950fa30e353ef39f9d564" target='_blank'>
              Single-cell genomics-based immune and disease monitoring in blood malignancies
              </a>
            </td>
          <td>
            Anja C Rathgeber, Leif S. Ludwig, Livius Penter
          </td>
          <td>2024-06-14</td>
          <td>Clinical Hematology International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Effective cancer immunotherapy relies on the clonal proliferation and expansion of CD8+ T cells in the tumor1,2. However, our insights into clonal expansions are limited, owing to an inability to track the same clones in tumors over time. Here, we developed a multi-tumor mouse model system to track hundreds of expanding and contracting CD8+ T cell clones over multiple timepoints in tumors of the same individual. Through coupling of clonal expansion dynamics and single-cell RNA/TCR-seq data, we identified a transcriptomic signature in PD-1+Ly108+ precursor exhausted cells3,4 that strongly predicts rates of intratumoral clone expansion in mice and humans. We found that expression of the signature successfully stratifies melanoma patient outcomes to PD-1/PD-L1 blockade5,6. Downregulation of the signature precedes clone contraction – a phase in which clones contract but maintain revivable precursor exhausted cells in the tumor. LAG-3 blockade – an FDA-approved therapy whose effects on CD8+ T cell responses are currently unclear7, re-activates the expansion signature, re-expanding pre-existing clones, including previously contracted clones. These findings reveal how the study of clonal expansion dynamics provide a powerful ‘pan-immunotherapy’ signature for monitoring immunotherapies, including PD-1/PD-L1 and LAG-3 blockade, with implications for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6681eced55a92fc5ceb65f99517fea6fa8caf1df" target='_blank'>
              LAG-3 blockade reactivates the CD8+ T cell expansion program to re-expand contracted clones in the tumor
              </a>
            </td>
          <td>
            Munetomo Takahashi, Mikiya Tsunoda, Hiroyasu Aoki, Masaki Kurosu, H. Ogiwara, S. Shichino, D. Bending, Shumpei Ishikawa, James E. D. Thaventhiran, Kouji Matsushima, S. Ueha
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Melanoma, a highly malignant tumour, presents significant challenges due to its cellular heterogeneity, yet research on this aspect in cutaneous melanoma remains limited. In this study, we utilized single‐cell data from 92,521 cells to explore the tumour cell landscape. Through clustering analysis, we identified six distinct cell clusters and investigated their differentiation and metabolic heterogeneity using multi‐omics approaches. Notably, cytotrace analysis and pseudotime trajectories revealed distinct stages of tumour cell differentiation, which have implications for patient survival. By leveraging markers from these clusters, we developed a tumour cell‐specific machine learning model (TCM). This model not only predicts patient outcomes and responses to immunotherapy, but also distinguishes between genomically stable and unstable tumours and identifies inflamed (‘hot’) versus non‐inflamed (‘cold’) tumours. Intriguingly, the TCM score showed a strong association with TOMM40, which we experimentally validated as an oncogene promoting tumour proliferation, invasion and migration. Overall, our findings introduce a novel biomarker score that aids in selecting melanoma patients for improved prognoses and targeted immunotherapy, thereby guiding clinical treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceff46cb2d3d44eadc13e557a6116c6ae860759" target='_blank'>
              Unravelling tumour cell diversity and prognostic signatures in cutaneous melanoma through machine learning analysis
              </a>
            </td>
          <td>
            Wenhao Cheng, Ping Ni, Hao Wu, Xiaye Miao, Xiaodong Zhao, Dali Yan
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors with dismal prognosis. One of the main bottlenecks for developing more effective therapies for glioblastoma stems from their histologic and molecular heterogeneity, leading to distinct tumor microenvironments and disease phenotypes. Effectively characterizing these features would improve the clinical management of glioblastoma. Glucose flux rates through glycolysis and mitochondrial oxidation have been recently shown to quantitatively depict glioblastoma proliferation in mouse models (GL261 and CT2A tumors, 38±3 mm3) using dynamic glucose-enhanced (DGE) deuterium spectroscopy. However, the spatial features of tumor microenvironment phenotypes remain hitherto unresolved. Here, we develop a DGE Deuterium Metabolic Imaging (DMI) approach for profiling tumor microenvironments through glucose conversion kinetics. Using a multimodal combination of tumor mouse models, novel strategies for spectroscopic imaging and noise attenuation, and histopathological correlations, we show that tumor lactate turnover mirrors phenotype differences between GL261 and CT2A mouse glioblastoma (59±7 mm3), whereas peritumoral glutamate-glutamine recycling is a potential marker of invasion capacity in pooled cohorts, linked to secondary brain lesions. Our findings were validated by histopathological characterization of each tumor, including cell density and proliferation, peritumoral infiltration, and distant migration. Our study bodes well for precision neuro-oncology, highlighting the importance of mapping glucose flux rates to better understand the metabolic heterogeneity of glioblastoma and its links to disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2927f90371bde6768c35753a6b358801ccb1fb15" target='_blank'>
              Deuterium Metabolic Imaging Phenotypes Mouse Glioblastoma Heterogeneity Through Glucose Turnover Kinetics
              </a>
            </td>
          <td>
            Rui V. Simões, Rafael N. Henriques, J. Olesen, Beatriz M Cardoso, Francisca F. Fernandes, Mariana A.V. Monteiro, S. Jespersen, Tânia Carvalho, N. Shemesh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Spatially resolved transcriptomics simultaneously measure the spatial location, histology images, and transcriptional profiles of the same cells or regions in undissociated tissues. Integrative analysis of multi-modal spatially resolved data holds immense potential for understanding the mechanisms of biology. Here we present a flexible multi-modal contrastive learning for the integration of spatially resolved transcriptomics (MuCST), which jointly perform denoising, elimination of heterogeneity, and compatible feature learning. We demonstrate that MuCST robustly and accurately identifies tissue subpopulations from simulated data with various types of perturbations. In cancer-related tissues, MuCST precisely identifies tumor-associated domains, reveals gene biomarkers for tumor regions, and exposes intra-tumoral heterogeneity. We also validate that MuCST is applicable to diverse datasets generated from various platforms, such as STARmap, Visium, and omsFISH for spatial transcriptomics, and hematoxylin and eosin or fluorescence microscopy for images. Overall, MuCST not only facilitates the integration of multi-modal spatially resolved data, but also serves as pre-processing for data restoration (Python software is available at https://github.com/xkmaxidian/MuCST).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e93e7b00fc6643c076f03733108be8401ccb4c7" target='_blank'>
              MuCST: restoring and integrating heterogeneous morphology images and spatial transcriptomics data with contrastive learning
              </a>
            </td>
          <td>
            Yu Wang, Xiaoke Ma
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) enables interrogating the molecular composition of tissue with ever-increasing resolution, depth, and sensitivity. However, costs, rapidly evolving technology, and lack of standards have constrained computational methods in ST to narrow tasks and small cohorts. In addition, the underlying tissue morphology as reflected by H&E-stained whole slide images (WSIs) encodes rich information often overlooked in ST studies. Here, we introduce HEST-1k, a collection of 1,108 spatial transcriptomic profiles, each linked to a WSI and metadata. HEST-1k was assembled using HEST-Library from 131 public and internal cohorts encompassing 25 organs, two species (Homo Sapiens and Mus Musculus), and 320 cancer samples from 25 cancer types. HEST-1k processing enabled the identification of 1.5 million expression--morphology pairs and 60 million nuclei. HEST-1k is tested on three use cases: (1) benchmarking foundation models for histopathology (HEST-Benchmark), (2) biomarker identification, and (3) multimodal representation learning. HEST-1k, HEST-Library, and HEST-Benchmark can be freely accessed via https://github.com/mahmoodlab/hest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/351b175485ca22362ace3258ab42fe3b708be864" target='_blank'>
              HEST-1k: A Dataset for Spatial Transcriptomics and Histology Image Analysis
              </a>
            </td>
          <td>
            Guillaume Jaume, Paul Doucet, Andrew H. Song, Ming Y. Lu, Cristina Almagro-P'erez, Sophia J. Wagner, Anurag J. Vaidya, Richard J. Chen, Drew F. K. Williamson, Ahrong Kim, Faisal Mahmood
          </td>
          <td>2024-06-23</td>
          <td>ArXiv</td>
          <td>3</td>
          <td>22</td>
        </tr>

        <tr id="Ductal carcinoma in situ (DCIS) is a heterogeneous breast disease that remains challenging to treat due to its unpredictable progression to invasive breast cancer (IBC). Contemporary literature has become increasingly focused on extracellular matrix (ECM) alterations with breast cancer progression. However, the spatial regulation of the ECM proteome in DCIS has yet to be investigated in relation to IBC. We hypothesized that DCIS and IBC present distinct ECM proteomes that could discriminate between these pathologies. Tissue sections of pure DCIS, mixed DCIS-IBC, or pure IBC (n = 22) with detailed pathological annotations were investigated by multiplexed spatial proteomics. Across tissues, 1,005 ECM peptides were detected in pathologically annotated regions and their surrounding extracellular microenvironments. A comparison of DCIS to IBC pathologies demonstrated 43 significantly altered ECM peptides. Notably, eight fibrillar collagen peptides could distinguish with high specificity and sensitivity between DCIS and IBC. Lesion-targeted proteomic imaging revealed heterogeneity of the ECM proteome surrounding individual DCIS lesions. Multiplexed spatial proteomics reported an invasive cancer field effect, in which DCIS lesions in closer proximity to IBC shared a more similar ECM profile to IBC than distal counterparts. Defining the ECM proteomic microenvironment provides novel molecular insights relating to DCIS and IBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae606988443d439f3b711991f79a070c7351e672" target='_blank'>
              Extracellular Microenvironment Alterations in Ductal Carcinoma In Situ and Invasive Breast Cancer Pathologies by Multiplexed Spatial Proteomics
              </a>
            </td>
          <td>
            Taylor S. Hulahan, Laura Spruill, Elizabeth N. Wallace, Yeonhee Park, Robert B. West, Jeffrey R. Marks, E. S. Hwang, Richard R Drake, Peggi M Angel
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most malignant primary brain tumor in adults, has poor prognosis irrespective of therapeutic advances due to its radio-resistance and infiltrative growth into brain tissue. The present study assessed functions and putative druggability of protein breast cancer type 1 susceptibility protein (BRCA1)-associated Ataxia telangiectasia mutated (ATM)-activator 1 (BRAT1) as a crucial factor driving key aspects of GBM, including enhanced DNA damage response and tumor migration. By a stable depletion of BRAT1 in GBM and glioma stem-like (GSC) cell lines, we observed a delay in DNA double-strand break repair and increased sensitivity to radiation treatment, corroborated by in vitro and in vivo studies demonstrating impaired tumor growth and invasion. Proteomic analyses further emphasize the role of BRAT1’s cell migration and invasion capacity, with a notable proportion of downregulated proteins associated with these processes. In line with the genetic manipulation, we found that treatment with the BRAT1 inhibitor Curcusone D (CurD) significantly reduced GSC migration and invasion in an ex vivo slice culture model, particularly when combined with irradiation, resulting a synergistic inhibition of tumor growth and infiltration. Our results reveal that BRAT1 contributes to GBM growth and invasion and suggest that therapeutic inhibition of BRAT1 with CurD or similar compounds might constitute a novel approach for anti-GBM directed treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e398faf7890164e1dab60592356505fb1252c" target='_blank'>
              BRAT1 - a new therapeutic target for glioblastoma
              </a>
            </td>
          <td>
            Alicia Haydo, Jennifer Schmidt, Alisha Crider, Tim Kögler, Johanna Ertl, S. Hehlgans, Marina E. Hoffmann, Rajeshwari Rathore, Ömer Güllülü, Yecheng Wang, Xiangke Zhang, Christel Herold-Mende, Francesco Pampaloni, Irmgard Tegeder, Ivan Dikic, Mingji Dai, Franz Rödel, D. Kögel, B. Linder
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 Background: Mantle cell lymphoma (MCL) is an aggressive subtype of B-cell non-Hodgkin lymphoma that exhibits clinical, pathological, and genetic heterogeneity. While generally thought to be incurable, recently demonstrated clinical activity of both chimeric antigen receptor T (CAR-T) cells and bispecific antibodies suggest that immune-mediated therapies offer substantial benefit in this disease and underscores the value of understanding mechanisms of immune evasion in this disease.
 Aim: The aim of this project is to uncover distinct cellular networks driving immune evasion in MCL across disease sites.
 Methods: We analyzed 10 archival samples collected from 8 clinically annotated patients with untreated MCL, including mononuclear cells from peripheral blood (PB), lymph nodes (LN) and bone marrow (BM). After generating single-cell suspensions from tumors, we performed Cellular Indexing of Transcriptomes and Epitopes sequencing (CITE-seq) and identified malignant B cells based on the co-expression of cyclin D1 (CCND1) and B cell markers (MS4A1, CD79A, CD79B). B and T cells were then subclustered using a reference mapping geneset. Molecular signatures database was utilized to perform geneset enrichment analysis (GSEA).
 Results: Our study first confirms the heterogeneity of mantle cell lymphoma across compartments (i.e PB, LN, BM), showing 5 different clusters of malignant cells, with different abundance in cell proportion among patients. We found that malignant B cells expressed different levels of CCND1 expression across tissue compartments. Furthermore, we identified a specific cluster of malignant cells, enriched in the PB, marked by decreased expression of the costimulatory receptor CD40 and higher levels of B-cell receptor signaling. This may explain the rapid response of circulating MCL cells to Bruton Tyrosine Kinase (BTK) inhibition. Interestingly, the higher expression of CD40 on LN-resident MCL cells suggests that MCL cells may co-opt LN-resident T-cells to support their growth, as has been previously described. Consistent with this, T-cells from LN were phenotypically exhausted, suggesting that MCL cells within the LN leverage local interactions with T-cells to promote survival while avoiding T-cell-driven cytotoxicity. Finally, we also found that high level of Ki67 correlates with myeloid-cell abundance in the TME suggesting that monocytes may support tumor cell proliferation.
 Conclusion: These results provide a deeper understanding of tissue-specific heterogeneity in MCL, including distinct tumor-immune interactions at different tissue sites. In ongoing analyses, we will correlate these findings with patient outcomes and response to BTK inhibition and are currently analyzing samples from patients with relapsed disease to identify microenvironmental features associated with disease progression.
 Citation Format: Amira Marouf, Jahan Rahman, Ashlee Joseph, Irem Isgor, Ya-Hui Lin, Ronan Chaligne, Shenon Sethi, Ahmet Dogan, Gilles Salles, Anita Kumar, Santosha A Vardhana. Identifying tissue-specific drivers of immune evasion in Mantle Cell Lymphoma at single cell level [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93708d0eca850dbd52911bae7638726472d8fb06" target='_blank'>
              Abstract PO-025: Identifying tissue-specific drivers of immune evasion in Mantle Cell Lymphoma at single cell level
              </a>
            </td>
          <td>
            Amira Marouf, Jahan Rahman, A. Joseph, Irem Isgor, Ya-Hui Lin, R. Chaligné, Shenon Sethi, Ahmet Dogan, G. Salles, Anita Kumar, S. Vardhana
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In vitro models coupled with multimodal approaches are needed to dissect the dynamic response of local tumor immune microenvironment (TIME) to immunotherapy. Here the patient-derived primary lung cancer organoids (pLCOs) are generated by isolating tumor cell clusters, including the infiltrated immune cells. A function-associated single-cell RNA sequencing (FascRNA-seq) platform allowing both phenotypic evaluation and scRNA-seq at single-organoid level is developed to dissect the TIME of individual pLCOs. The analysis of 171 individual pLCOs derived from seven patients reveals that pLCOs retain the TIME heterogeneity in the parenchyma of parental tumor tissues, providing models with identical genetic background but various TIME. Linking the scRNA-seq data of individual pLCOs with their responses to anti-PD-1 (αPD-1) immune checkpoint blockade (ICB) allows to confirm the central role of CD8+ T cells in anti-tumor immunity, to identify potential tumor-reactive T cells with a set of 10 genes, and to unravel the factors regulating T cell activity, including CD99 gene. In summary, the study constructs a joint phenotypic and transcriptomic FascRNA-seq platform to dissect the dynamic response of local TIME under ICB treatment, providing a promising approach to evaluate novel immunotherapies and to understand the underlying molecular mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902a93c4a9cfc533a5004cf44836921bd67b326d" target='_blank'>
              Patient-Derived Tumor Organoids Combined with Function-Associated ScRNA-Seq for Dissecting the Local Immune Response of Lung Cancer.
              </a>
            </td>
          <td>
            Chang Liu, Kaiyi Li, X. Sui, Tian Zhao, Ting Zhang, Zhongyao Chen, Hainan Wu, Chao Li, Hao Li, Fan Yang, Zhidong Liu, You-Yong Lu, Jun Wang, Xiaofang Chen, Peng Liu
          </td>
          <td>2024-06-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Immune checkpoint (IC) blockade and adoptive transfer of tumor-specific T-cells (ACT) are two major strategies to treat metastatic melanoma. Their combination can potentiate T-cell activation in the suppressive tumor microenvironment, but the autoimmune adverse effects associated with systemic injection of IC blockers persist with this strategy. ACT of tumor-reactive T-cells defective for IC expression would overcome this issue. For this purpose, PD-1 and TIGIT appear to be relevant candidates, because their co-expression on highly tumor-reactive lymphocytes limits their therapeutic efficacy within the tumor microenvironme,nt. Our study compares the consequences of PDCD1 or TIGIT genetic deletion on anti-tumor properties and T-cell fitness of melanoma-specific T lymphocytes. Transcriptomic analyses revealed down-regulation of cell cycle-related genes in PD-1KO T-cells, consistent with biological observations, whereas proliferative pathways were preserved in TIGITKO T-cells. Functional analyses showed that PD-1KO and TIGITKO T-cells displayed superior antitumor reactivity than their wild-type counterpart in vitro and in a preclinical melanoma model using immunodeficient mice. Interestingly, it appears that TIGITKO T-cells were more effective at inhibiting tumor cell proliferation in vivo, and persist longer within tumors than PD-1KO T-cells, consistent with the absence of impact of TIGIT deletion on T-cell fitness. Taken together, these results suggest that TIGIT deletion, over PD-1 deletion, in melanoma-specific T-cells is a compelling option for future immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d39a7191a7050c3023b69888c21ce3d7915b91a" target='_blank'>
              Differential impact of genetic deletion of TIGIT or PD-1 on melanoma-specific T-lymphocytes
              </a>
            </td>
          <td>
            G. Cadiou, T. Beauvais, Lucine Marotte, Sylvia Lambot, C. Deleine, Caroline Vignes, Malika Gantier, Melanie Hussong, Samuel Rulli, Anne Jarry, Sylvain Simon, Bernard Malissen, N. Labarrière
          </td>
          <td>2024-07-08</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7f25a39411b25f30ce76ee209200149b2b9966" target='_blank'>
              Non-stem cell lineages as an alternative origin of intestinal tumorigenesis in the context of inflammation
              </a>
            </td>
          <td>
            Mathijs P. Verhagen, Rosalie Joosten, Mark Schmitt, Niko Välimäki, A. Sacchetti, K. Rajamäki, Jiahn Choi, Paola Procopio, Sara Silva, Berdine van der Steen, T. P. van den Bosch, Danielle Seinstra, Annemarie C de Vries, M. Doukas, Leonard H. Augenlicht, L. Aaltonen, Riccardo Fodde
          </td>
          <td>2024-06-20</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19b937a783d0083d4f357f86050f667df91ba2a" target='_blank'>
              ScRNA-seq revealed the tumor microenvironment heterogeneity related to the occurrence and metastasis in upper urinary tract urothelial carcinoma.
              </a>
            </td>
          <td>
            Shiyong Xin, Yanwei Zhang, Zhenhua Zhang, Ziyao Li, Xianchao Sun, Xiang Liu, L. Jin, Weiyi Li, Chaozhi Tang, Wangli Mei, Qiong Cao, Haojie Wang, Zhihao Wei, Zhen Zhou, Rongbing Li, Xiaofei Wen, Guosheng Yang, Weihua Chen, Junhua Zheng, L. Ye
          </td>
          <td>2024-06-14</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancers are commonly characterised by a complex pathology encompassing genetic, microscopic and macroscopic features, which can be probed individually using imaging and omics technologies. Integrating this data to obtain a full understanding of pathology remains challenging. We introduce a new method called Deep Latent Variable Path Modelling (DLVPM), which combines the representational power of deep learning with the capacity of path modelling to identify relationships between interacting elements in a complex system. To evaluate the capabilities of DLVPM, we initially trained a foundational model to map dependencies between SNV, Methylation, miRNA-Seq, RNA-Seq and Histological data using Breast Cancer data from The Cancer Genome Atlas (TCGA). This method exhibited superior performance in mapping associations between data types compared to classical path modelling. We additionally performed successful applications of the model to: stratify single-cell data, identify synthetic lethal interactions using CRISPR-Cas9 screens derived from cell-lines, and detect histologic-transcriptional associations using spatial transcriptomic data. Results from each of these data types can then be understood with reference to the same holistic model of illness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36a969a03ce741afe8ca32855f97d3ec725e814d" target='_blank'>
              Integrating Multi-Modal Cancer Data Using Deep Latent Variable Path Modelling
              </a>
            </td>
          <td>
            Alex Ing, Alvaro Andrades, M. R. Cosenza, J. Korbel
          </td>
          <td>2024-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Pericytes, recognized as mural cells, have long been described as components involved in blood vessel formation, playing a mere supporting role for endothelial cells (ECs). Emerging evidence strongly suggests their multifaceted roles in tissues and organs. Indeed, pericytes exhibit a remarkable ability to anticipate endothelial cell behavior and adapt their functions based on the specific cells they interact with. Pericytes can be activated by pro-inflammatory stimuli and crosstalk with immune cells, actively participating in their transmigration into blood vessels. Moreover, they can influence the immune response, often sustaining an immunosuppressive phenotype in most of the cancer types studied. In this review, we concentrate on the intricate crosstalk between pericytes and immune cells in cancer, highlighting the primary evidence regarding pericyte involvement in primary tumor mass dynamics, their contributions to tumor reprogramming for invasion and migration of malignant cells, and their role in the formation of pre-metastatic niches. Finally, we explored recent and emerging pharmacological approaches aimed at vascular normalization, including novel strategies to enhance the efficacy of immunotherapy through combined use with anti-angiogenic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76bcdf2dcb6f3841dc22f4ab784010532400194" target='_blank'>
              Pericytes: jack-of-all-trades in cancer-related inflammation
              </a>
            </td>
          <td>
            Marianna Moro, Federica Carolina Balestrero, A. A. Grolla
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tertiary lymphoid structures (TLS) are ectopic lymphoid aggregates associated with improved prognosis in numerous tumors. To date, their prognostic value in central nervous system cancers and the events underlying their formation remain unclear. Here, we find that TLS correlate with improved survival in glioblastoma patients. Furthermore, combining spatial transcriptomics of human tissues with longitudinal studies in murine glioma, we establish that the assembly of T cell-rich clusters is a prerequisite for the development of canonical TLS, and show that CD4 T cells play a central role in this process. Indeed, we provide evidence that IL7R+CCR7+ Th1 lymphocytes with lymphoid tissue-inducing potential are recruited to TLS nucleation sites, where they can initiate TLS assembly by expressing lymphotoxin β. Our work defines TLS as a potential prognostic factor in glioblastoma and identifies cellular and molecular mechanisms of TLS formation. These findings have broader implications for the development of TLS-inducing therapies for cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368cd006ed1ae7b496e449db1b6253ab2d80a3fe" target='_blank'>
              Cellular and molecular events organizing the assembly of tertiary lymphoid structures in glioblastoma
              </a>
            </td>
          <td>
            Alessandra Vaccaro, Fan Yang, Tiarne van de Walle, Saskia Franke, R. Lugano, A. Dénes, Anastasia Magoulopoulou, Inés Cid-Fariña, Anja Smits, Lene Uhrbom, Sylwia Libard, Francesco Latini, Magnus Essand, Mats Nilsson, Liqun He, T. Olsson Bontell, A. Jakola, Mohanraj Ramachandran, A. Dimberg
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Glioblastomas are known for their poor clinical prognosis, with recurrent tumors often exhibiting greater invasiveness and faster growth rates compared to primary tumors. To understand the intratumoral changes driving this phenomenon, we employed single-cell sequencing to analyze the differences between two pairs of primary and recurrent glioblastomas. Our findings revealed an upregulation of ferroptosis in endothelial cells within recurrent tumors, identified by the significant overexpression of the NOX4 gene. Further analysis indicated that knocking down NOX4 in endothelial cells reduced the activity of the ferroptosis pathway. Utilizing conditioned media from endothelial cells with lower ferroptosis activity, we observed a decrease in the growth rate of glioblastoma cells. These results highlighted the complex role of ferroptosis within tumors and suggested that targeting ferroptosis in the treatment of glioblastomas requires careful consideration of its effects on endothelial cells, as it may otherwise produce counterproductive outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb06c19dc2e1dad39392c5e5a73da1e9d0d09630" target='_blank'>
              Endothelial cell ferroptosis influences IDH wild-type glioblastoma growth in recurrent glioblastoma multiforme patients
              </a>
            </td>
          <td>
            Bo Liang, Xinghuan Ding, Siyuan Yang, Enshan Feng
          </td>
          <td>2024-07-08</td>
          <td>Brazilian Journal of Medical and Biological Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fec58e3a7ae44488bf9b6bdb1c9aefc99aa39bfd" target='_blank'>
              Prediction of Tumor-Associated Macrophages and Immunotherapy Benefits Using Weakly Supervised Contrastive Learning in Breast Cancer Pathology Images.
              </a>
            </td>
          <td>
            Guobang Yu, Yi Zuo, Bin Wang, Hui Liu
          </td>
          <td>2024-06-17</td>
          <td>Journal of imaging informatics in medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Since the introduction of the first immune checkpoint inhibitor (ICI), immunotherapy has changed the landscape of molecular therapeutics for cancers. However, ICIs do not work equally well on all cancers and for all patients. There has been a growing interest in using mathematical and computational models to optimize clinical responses. Ordinary differential equations (ODEs) have been widely used for mechanistic modeling in immuno-oncology and immunotherapy because they allow rapid simulations of temporal changes in the cellular and molecular populations involved. Nonetheless, ODEs cannot describe the spatial structure in the tumor microenvironment or quantify the influence of spatially-dependent characteristics of tumor-immune dynamics. For these reasons, agent-based models (ABMs) have gained popularity because they can model more detailed phenotypic and spatial heterogeneity that better reflect the complexity seen in vivo. In the context of anti-PD-1 ICIs, we compare treatment outcomes simulated from an ODE model and an ABM to show the importance of including spatial components in computational models of cancer immunotherapy. We consider tumor cells of high and low antigenicity and two distinct cytotoxic T lymphocyte (CTL) killing mechanisms. The preferred mechanism differs based on the antigenicity of tumor cells. Our ABM reveals varied phenotypic shifts within the tumor and spatial organization of tumor and CTLs, despite similarities in key immune parameters, initial conditions of simulation, and early temporal trajectories of the cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38a51133539694b984ad15aa283204f18aa1920" target='_blank'>
              Agent-Based Modeling of Virtual Tumors Reveals the Critical Influence of Microenvironmental Complexity on Immunotherapy Efficacy
              </a>
            </td>
          <td>
            Yixuan Wang, Daniel R. Bergman, Erica Trujillo, Anthony A. Fernald, Lie Li, Alexander T. Pearson, Randy F. Sweis, Trachette Jackson
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The spatial composition and cellular heterogeneity of the tumor microenvironment plays a critical role in cancer development and progression. High-definition pathology imaging of tumor biopsies provide a high-resolution view of the spatial organization of different types of cells. This allows for systematic assessment of intra- and inter-patient spatial cellular interactions and heterogeneity by integrating accompanying patient-level genomics data. However, joint modeling across tumor biopsies presents unique challenges due to non-conformability (lack of a common spatial domain across biopsies) as well as high-dimensionality. To address this problem, we propose the Dual random effect and main effect selection model for Spatially structured regression model (DreameSpase). DreameSpase employs a Bayesian variable selection framework that facilitates the assessment of spatial heterogeneity with respect to covariates both within (through fixed effects) and between spaces (through spatial random effects) for non-conformable spatial domains. We demonstrate the efficacy of DreameSpase via simulations and integrative analyses of pathology imaging and gene expression data obtained from $335$ melanoma biopsies. Our findings confirm several existing relationships, e.g. neutrophil genes being associated with both inter- and intra-patient spatial heterogeneity, as well as discovering novel associations. We also provide freely available and computationally efficient software for implementing DreameSpase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f4788cc422b8751f7a8dd15c650e8347a66bb49" target='_blank'>
              Spatially Structured Regression for Non-conformable Spaces: Integrating Pathology Imaging and Genomics Data in Cancer
              </a>
            </td>
          <td>
            Nathaniel Osher, Jian Kang, Arvind Rao, Veerabhadran Baladandayuthapani
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Biomarkers, including proteins, nucleic acids, antibodies, and peptides, are essential for identifying diseases such as cancer and differentiating between healthy and abnormal cells in patients. To date, studies have shown that cancer stem cells have DNA repair mechanisms that deter the effects of medicinal treatment. Experiments with cell cultures and chemotherapy treatments of these cultures have revealed the presence of small cells, with a small amount of cytoplasm that can be intensively stained with azure eosin, called microcells. Microcells develop during sporosis from a damaged tumor macrocell. After anticancer therapy in tumor cells, a defective macrocell may produce one or more microcells. This study aims to characterize microcell morphology in melanoma cell lines. In this investigation, we characterized the population of cancer cell microcells after applying paclitaxel treatment to a Sk-Mel-28 melanoma cell line using immunocytochemical cell marker detection and fluorescent microscopy. Paclitaxel-treated cancer cells show stronger expression of stem-associated ALDH2, SOX2, and Nanog markers than untreated cells. The proliferation of nuclear antigens in cells and the synthesis of RNA in microcells indicate cell self-defense, promoting resistance to applied therapy. These findings improve our understanding of microcell behavior in melanoma, potentially informing future strategies to counteract drug resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8ed6b1eaab2b7374876491c10185ffcc9a5b4c3" target='_blank'>
              Morphological and Immunocytochemical Characterization of Paclitaxel-Induced Microcells in Sk-Mel-28 Melanoma Cells
              </a>
            </td>
          <td>
            Z. Simsone, Tālivaldis Feivalds, L. Harju, I. Miķelsone, Ilze Blāķe, Juris Bērziņš, Indulis Buiķis
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Carotid atherosclerosis is orchestrated by cell-cell communication that drives progression along a clinical continuum (asymptomatic to symptomatic). Extracellular vesicles (EVs) are cell-derived nanoparticles representing a new paradigm in cellular communication. Little is known about their biological cargo, cellular origin/destination, and functional roles in human atherosclerotic plaque. Methods: EVs were enriched via size exclusion chromatography from human carotid endarterectomy samples dissected into paired plaque and marginal zones (symptomatic n=16, asymptomatic n=13). EV cargos were assessed via whole transcriptome miRNA sequencing and mass spectrometry-based proteomics. EV multi-omics were integrated with bulk and single cell RNA-sequencing (scRNA-seq) datasets to predict EV cellular origin and ligand-receptor interactions, and multi-modal biological network integration of EV-cargo was completed. EV functional impact was assessed with endothelial angiogenesis assays. Results: Carotid plaques contained more EVs than adjacent marginal zones, with differential enrichment for EV-miRNAs and EV-proteins in key atherogenic pathways. EV cellular origin analysis suggested that tissue EV signatures originated from endothelial cells (EC), smooth muscle cells (SMC), and immune cells. Integrated tissue vesiculomics and scRNA-seq indicated complex EV-vascular cell communication that changed with disease progression and plaque vulnerability (i.e., symptomatic disease). Plaques from symptomatic patients, but not asymptomatic patients, were characterized by increased involvement of endothelial pathways and more complex ligand-receptor interactions, relative to their marginal zones. Plaque-EVs were predicted to mediate communication with ECs. Pathway enrichment analysis delineated an endothelial signature with roles in angiogenesis and neovascularization – well-known indices of plaque instability. This was validated functionally, wherein human carotid symptomatic plaque EVs induced sprouting angiogenesis in comparison to their matched marginal zones. Conclusion: Our findings indicate that EVs may drive dynamic changes in plaques through EV- vascular cell communication and effector functions that typify vulnerability to rupture, precipitating symptomatic disease. The discovery of endothelial-directed angiogenic processes mediated by EVs creates new therapeutic avenues for atherosclerosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85baba07ce353659c168561fd588df1f617dc3d1" target='_blank'>
              Multiomics unveils extracellular vesicle-driven mechanisms of endothelial communication in human carotid atherosclerosis
              </a>
            </td>
          <td>
            S. Raju, Mandy E Turner, C. Cao, Majed Abdul-Samad, Neil Punwasi, M. Blaser, Rachel M E Cahalane, S. Botts, Kamal K. Prajapati, Sarvatit Patel, Ruilin Wu, D. Gustafson, Natalie J. Galant, Lindsey Fiddes, M. Chemaly, Ulf Hedin, L. Matic, Michael Seidman, Vallijah Subasri, Sasha A. Singh, Elena Aikawa, Jason E. Fish, Kathryn L. Howe
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Immune cell interactions and metabolic changes are crucial in determining the tumor microenvironment and affecting various clinical outcomes. However, the clinical significance of metabolism evolution of immune cell evolution in colorectal cancer (CRC) remains unexplored. Methods Single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing data were acquired from TCGA and GEO datasets. For the analysis of macrophage differentiation trajectories, we employed the R packages Seurat and Monocle. Consensus clustering was further applied to identify the molecular classification. Immunohistochemical results from AOM and AOM/DSS models were used to validate macrophage expression. Subsequently, GSEA, ESTIMATE scores, prognosis, clinical characteristics, mutational burden, immune cell infiltration, and the variance in gene expression among different clusters were compared. We constructed a prognostic model and nomograms based on metabolic gene signatures identified through the MEGENA framework. Results We found two heterogeneous groups of M2 macrophages with various clinical outcomes through the evolutionary process. The prognosis of Cluster 2 was poorer. Further investigation showed that Cluster 2 constituted a metabolically active group while Cluster 1 was comparatively metabolically inert. Metabolic variations in M2 macrophages during tumor development are related to tumor prognosis. Additionally, Cluster 2 showed the most pronounced genomic instability and had highly elevated metabolic pathways, notably those associated with the ECM. We identified eight metabolic genes (PRELP, NOTCH3, CNOT6, ASRGL1, SRSF1, PSMD4, RPL31, and CNOT7) to build a predictive model validated in CRC datasets. Then, a nomogram based on the M2 risk score improved predictive performance. Furthermore, our study demonstrated that immune checkpoint inhibitor therapy may benefit patients with low-risk. Discussion Our research reveals underlying relationships between metabolic phenotypes and immunological profiles and suggests a unique M2 classification technique for CRC. The identified gene signatures may be key factors linking immunity and tumor metabolism, warranting further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b77333ce5794bf43c82ae5c8985b3dc9293d6dd" target='_blank'>
              M2 Macrophage Classification of Colorectal Cancer Reveals Intrinsic Connections with Metabolism Reprogramming and Clinical Characteristics
              </a>
            </td>
          <td>
            Fengxing Huang, Youwei Wang, Yu Shao, Runan Zhang, Mengting Li, Lan Liu, Qiu-xia Zhao
          </td>
          <td>2024-07-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background The heterogeneity and dynamic changes of endometrial cells have a significant impact on health as they determine the normal function of the endometrium during the menstrual cycle. Dysfunction of the endometrium can lead to the occurrence of various gynecological diseases. Therefore, deconvolution of immune microenvironment that drives transcriptional programs throughout the menstrual cycle is key to understand regulatory biology of endometrium. Methods Herein, we comprehensively analyzed single-cell transcriptome of 59,397 cells across ten human endometrium samples and revealed the dynamic cellular heterogeneity throughout the menstrual cycle. Results We identified two perivascular cell subtypes, four epithelial subtypes and four fibroblast cell types in endometrium. Moreover, we inferred the cell type-specific transcription factor (TF) activities and linked critical TFs to transcriptional output of diverse immune cell types, highlighting the importance of transcriptional regulation in endometrium. Dynamic interactions between various types of cells in endometrium contribute to a range of biological pathways regulating differentiation of secretory. Integration of the molecular biomarkers identified in endometrium and bulk transcriptome of 535 endometrial cancers (EC), we revealed five RNA-based molecular subtypes of EC with highly intratumoral heterogeneity and different clinical manifestations. Mechanism analysis uncovered clinically relevant pathways for pathogenesis of EC. Conclusion In summary, our results revealed the dynamic immune microenvironment of endometrium and provided novel insights into future development of RNA-based treatments for endometriosis and endometrial carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ccb0924331867e645195c52b43129abb7308e5" target='_blank'>
              Mapping Single-Cell Transcriptomes of Endometrium Reveals Potential Biomarkers in Endometrial Cancer
              </a>
            </td>
          <td>
            Gang Xu, Tao Pan, Si Li, Jing Guo, Ya Zhang, Qi Xu, Renwei Chen, Yanlin Ma, Yongsheng Li
          </td>
          <td>2024-07-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="NF1 is recurrently mutated in glioblastoma yet the molecular landscape and efficacy of targeted therapies remain unclear. Here, we combine bulk and single cell genomics of human somatic NF1 mutant, IDH-wildtype glioblastomas with functional genomic analysis of cell lines and mouse intracranial tumor models to identify molecular subgroups within NF1 mutant glioblastomas and mechanisms underlying MEK inhibitor response. Targeted DNA sequencing showed homozygous deletion of the cell cycle regulator CDKN2A/B is a poor prognostic marker in somatic NF1 mutant, but not NF1 wildtype, tumors. DNA methylation array profiling revealed three epigenetic groups highlighted by distinct clinical features, co-mutation patterns, and reference methylation classifier identities. Genome-wide CRISPRi screens in glioblastoma cells revealed cell cycle regulators are conserved mediators of cell growth while response to the MEK inhibitor selumetinib converges on Ras/RAF/MEK pathway activation. Repression of the RAF regulator SHOC2 sensitizes glioblastomas to selumetinib in vitro and in vivo in mouse intracranial glioblastoma models. Single cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib reveals distinct responses between mesenchymal-like (MES-like) and non MES-like subpopulations suggesting non-MES like cells are intrinsically resistant to MEK inhibition. Finally, single nuclear RNA-sequencing (snRNA-seq) of human NF1 mutant, CDKN2A/B deleted glioblastomas reveals MES-like tumor cells are associated with selumetinib sensitivity signatures while non MES-like cells exhibit increased cell cycle progression and lack selumetinib sensitivity, further supporting the notion that MEK inhibition specifically targets MES-like tumor cell subpopulations. Taken together, our data underscores the heterogeneity between and within somatic NF1 mutant glioblastomas and delineates mechanisms of MEK inhibitor response across distinct tumor subpopulations, guiding the development of future therapeutic strategies that may synergize with MEK inhibition for NF1 mutant tumors. The tumor suppressor NF1 is mutated in 15% of glioblastomas,1–3 the most common malignant brain tumor with poor outcomes and few effective treatments.4 NF1 is a GTPase activating protein (GAP) that negatively regulates Ras, and thus, NF1 loss leads to induction of Ras/RAF/MEK/ERK signaling, driving tumorigenesis and comprising a targetable molecular cascade.5,6 Genomic analysis of glioblastoma demonstrates NF1 mutation is associated with a mesenchymal-like (MES-like) transcriptomic tumor cell subpopulation and altered tumor microenvironment.7,8 More broadly, DNA methylation analysis reveals multiple epigenetic subgroups with overlapping relationships to transcriptomic subtype and DNA alterations, underscoring the complex relationship between genetic drivers and molecular signatures.9 While the updated 2021 Central Nervous System WHO tumor classification incorporates an ever increasing amount of molecular criteria for diffuse astrocytic tumors,10 the existence and clinical significance of molecular subgroups within somatic NF1 mutant, IDH-wildtype glioblastomas based on genetic co-mutations, epigenetic profile, or transcriptomic signatures remain unclear. Preclinical data support the utility of MEK inhibition in NF1 mutant gliomas,11,12 and the MEK inhibitor selumetinib is FDA approved for tumors arising in patients with syndromic neurofibromatosis type 1 (NF-1) harboring a germline NF1 mutation.13,14 In NF-1 associated gliomas, MEK inhibition demonstrates efficacy in a limited case series,15 and combined BRAF/MEK inhibition shows efficacy in BRAF p.V600E mutant gliomas,16 further supporting the translational potential of Ras/Raf/MEK/ERK blockade within genetically defined glioma subtypes. Nevertheless, treatment resistance to molecular monotherapy remains a challenge,17–20 and the mechanisms underlying MEK inhibitor resistance in NF1 mutant glioma are unknown. Here, we integrate targeted DNA sequencing, DNA methylation profiling, and single nuclear RNA-sequencing (snRNA-seq) of human patient somatic NF1 mutant, IDH-wildtype glioblastomas with single cell RNA-sequencing (scRNA-seq), genome-wide clustered regularly interspaced short palindromic repeats interference (CRISPRi) screens, and pharmacologic studies in cell lines and mouse intracranial glioblastoma models to define molecular subgroups and functional mediators of MEK inhibitor response. Targeted DNA sequencing of NF1 mutant, IDH-wildtype glioblastomas (n=186 tumors) revealed CDKN2A/B deletion was associated with poor outcomes in NF1 mutant, but not NF1 wildtype, glioblastomas. DNA methylation profiling (n=129 tumors) demonstrated three epigenetic subgroups with distinct clinical features, co- mutation patterns across cell cycle genes, and reference methylation classifier identities. Genome-wide CRISPRi screens in mouse SB28 and human GBM43 glioblastoma cells identified a conserved cell cycle gene network mediating cell growth, consistent with the clinical importance of additional hits affecting the cell cycle in human somatic NF1 mutant glioblastomas. Moreover, genome-wide mediators of selumetinib response converged upon two Ras pathway effectors mediating selumetinib sensitivity: BRAF and SHOC2. SHOC2 repression in glioblastoma cells significantly improved selumetinib response both in vitro and in intracranial allografts in vivo. Single cell RNA-sequencing (scRNA-seq) of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib revealed MES-like tumor cells correlated with CDKN2A retention and the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells displaying Ras pathway induction. In contrast, non-MES like tumor cells were CDKN2A deficient and lacked expression of the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells inducing a glial de-differentiation program. Finally, snRNA-seq of NF1 mutant, CDKN2A/B deleted, IDH-wildtype glioblastomas (n=9) showed non MES-like tumor cells exhibit increased cell cycle progression and were not associated with the CRISPRi screen selumetinib sensitivity signature. MES-like tumor cells within newly diagnosed, but not recurrent, tumors retained expression of the CRISPRi screen selumetinib sensitivity signature, suggesting resistance can arise both between and within specific transcriptomic glioblastoma cell tumor cell subpopulations. Taken together, our data identifies clinically important subgroups of NF1 mutant, IDH-wildtype glioblastomas and supports a model in which heterogeneity between tumors and within tumor cell subpopulations underlies MEK inhibitor response, supporting the need for additional synergistic therapeutic approaches beyond maximal Ras pathway blockade for NF1 mutant glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e53f09e755435876ce84b544a3051562b27f2f" target='_blank'>
              Multiplatform molecular profiling and functional genomic screens identify prognostic signatures and mechanisms underlying MEK inhibitor response in somatic NF1 mutant glioblastoma
              </a>
            </td>
          <td>
            Sixuan Pan, Kanish Mirchia, E. Payne, S. J. Liu, Nadeem N Al-Adli, Zain Peeran, Poojan Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, K. Seo, T. Ozawa, Brian Na, Alyssa T. Reddy, S. Braunstein, Joanna J. Phillips, Susan Chang, D. A. Solomon, Arie Perry, D. Raleigh, Mitchell S. Berger, A. Abate, H. Vasudevan
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Glioma, a prevalent and deadly brain tumor, is marked by significant cellular heterogeneity and metabolic alterations. However, the comprehensive cell-of-origin and metabolic landscape in high-grade (Glioblastoma Multiforme, WHO grade IV) and low-grade (Oligoastrocytoma, WHO grade II) gliomas remains elusive. Methods In this study, we undertook single-cell transcriptome sequencing of these glioma grades to elucidate their cellular and metabolic distinctions. Following the identification of cell types, we compared metabolic pathway activities and gene expressions between high-grade and low-grade gliomas. Results Notably, astrocytes and oligodendrocyte progenitor cells (OPCs) exhibited the most substantial differences in both metabolic pathways and gene expression, indicative of their distinct origins. The comprehensive analysis identified the most altered metabolic pathways (MCPs) and genes across all cell types, which were further validated against TCGA and CGGA datasets for clinical relevance. Discussion Crucially, the metabolic enzyme phosphodiesterase 8B (PDE8B) was found to be exclusively expressed and progressively downregulated in astrocytes and OPCs in higher-grade gliomas. This decreased expression identifies PDE8B as a metabolism-related oncogene in IDH-mutant glioma, marking its dual role as both a protective marker for glioma grading and prognosis and as a facilitator in glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32a8dd10db0de50c204790d7bd0cb6e815520890" target='_blank'>
              Single-cell transcriptomic analysis identifies downregulated phosphodiesterase 8B as a novel oncogene in IDH-mutant glioma
              </a>
            </td>
          <td>
            Z. He, Yu Peng, Duo Wang, Chen Yang, Chengzhi Zhou, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) represents the most heterogeneous and aggressive subtype of breast carcinomas, characterized by the absence of clinical biomarkers (ERα, PR, and HER2) and the lack of targeted therapies. In this regard, several clinical trials have consistently failed to effectively stratify patients and identify specific treatments that elicit substantial responses. This study aims to pinpoint biomarkers expressed exclusively in the basal mammary epithelial compartment, facilitating a refined subclassification of this breast cancer subtype. Using computational analyses of single-cell RNA sequencing data, we have identified a list of genes associated with basal identity (BC-markers). Histological validation in 137 human samples has enabled us to categorize TNBC patients into BC-positive and BC-negative TNBC subtypes. Significantly, the presence of these markers correlates with a poorer prognosis in TNBC patients. Functional analyses have revealed a pivotal role for TAGLN in cell migration, likely influencing tumor aggressiveness. Further, we discovered that BC-marker expression is associated with the mesenchymal phenotype and increased sensitivity to the tyrosine kinase inhibitor dasatinib, particularly in BC-positive TNBC, suggesting novel therapeutic avenues. In our study, TAGLN emerged as a potential predictive biomarker for dasatinib responsiveness, offering new directions for personalized therapy for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37a72d0f9fe14dc4dcdef733872b863c9bfc2b5a" target='_blank'>
              Discovery and evaluation of novel biomarkers reveal dasatinib as a potential treatment for a specific subtype of Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            D. Ortega-Álvarez, D. Tébar-García, M. Casado-Peláez, E. Castillo-Agea, J. Balibrea-Rull, D. Olivares-Osuna, D. Pérez-Parra, C. Guardia, E. Musulén, E. Vinuesa-Pitarch, E. Sánchez-López, F. Postigo-Corrales, E. Ballana, A. Martínez-Cardús, M. Margeli, M. Esteller, PL Fernández-Ruiz, G. Luengo-Gil, E. Mereu, EM Galán-Moya, V. Rodilla
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f58160d8cdc5bd6e2466eb7c04abf11234ebf3" target='_blank'>
              Patient-derived mini-colons enable long-term modeling of tumor-microenvironment complexity.
              </a>
            </td>
          <td>
            L. F. Lorenzo-Martín, N. Broguiere, Jakob Langer, Lucie Tillard, Mike Nikolaev, George Coukos, K. Homicsko, Matthias P. Lutolf
          </td>
          <td>2024-07-02</td>
          <td>Nature biotechnology</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Breast cancer stem cells (BCSCs) are a small population of cells in breast cancer tumors, playing a putative role in the cancer’s progression. BCSC biomarkers are associated with cells’ enhanced growth, adhesion, migration, and invasion potential and are responsible for poor outcomes. Due to the vast heterogeneity of breast cancer signaling pathways and associated therapeutic targets, treatment strategies vary and depend on the molecular subtype of BC. Nevertheless, drugs affecting the Wnt, Notch, Hedgehog, PI3K/Akt/mTOR, and HER2 signaling pathways have been effectively applied in BCSC elimination strategies. The complexity of the characteristics of each molecular subtype of breast cancer, with an emphasis on the involvement of breast cancer stem cells in the development, metastasis, and recurrence of the tumor, requires greater understanding. Continuous updating of knowledge and the development of new therapies will allow the effective elimination of cancer cells and the rapid recovery of patients. Abstract Breast cancer is one of the most frequently detected malignancies worldwide. It is responsible for more than 15% of all death cases caused by cancer in women. Breast cancer is a heterogeneous disease representing various histological types, molecular characteristics, and clinical profiles. However, all breast cancers are organized in a hierarchy of heterogeneous cell populations, with a small proportion of cancer stem cells (breast cancer stem cells (BCSCs)) playing a putative role in cancer progression, and they are responsible for therapeutic failure. In different molecular subtypes of breast cancer, they present different characteristics, with specific marker profiles, prognoses, and treatments. Recent efforts have focused on tackling the Wnt, Notch, Hedgehog, PI3K/Akt/mTOR, and HER2 signaling pathways. Developing diagnostics and therapeutic strategies enables more efficient elimination of the tumor mass together with the stem cell population. Thus, the knowledge about appropriate therapeutic methods targeting both “normal” breast cancer cells and breast cancer stem cell subpopulations is crucial for success in cancer elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ec80e0acde078cb7edbaa565c5edbda8feeb01" target='_blank'>
              Breast Cancer Stem Cells and Tumor Heterogeneity: Characteristics and Therapeutic Strategies
              </a>
            </td>
          <td>
            Aleksandra Romaniuk-Drapała, Ewa Totoń, Magdalena Taube, M. Idzik, B. Rubiś, N. Lisiak
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Kidney allograft rejections are orchestrated by a variety of immune cells. Because of the complex histopathologic features, accurate pathological diagnosis poses challenges even for expert pathologists. The objective of this study was to unveil novel spatial indices associated with transplant rejection by using a spatial bioinformatic approach using 36-plex immunofluorescence image data.


METHODS
The image obtained from 11 T cell-mediated rejection (TCMR) and 12 antibody-mediated rejection (AMR) samples were segmented into 753 737 single cells using DeepCell's Mesmer algorithm. These cells were categorized into 13 distinct cell types through unsupervised clustering based on their biomarker expression profiles. Cell neighborhood analysis allowed us to stratify kidney tissue into 8 distinct neighborhood components consisting of unique cell type enrichment profiles.


RESULTS
In contrast to TCMR samples, AMR samples exhibited a higher frequency of neighborhood components that were characterized by an enrichment of CD31+ endothelial cells. Although the overall frequency of CD68+ macrophages in AMR samples was not significantly high, CD68+ macrophages within endothelial cell-rich lesions exhibited a significantly higher frequency in AMR samples than TCMR samples. Furthermore, the frequency of interactions between CD31+ cells and CD68+ cells was significantly increased in AMR samples, implying the pivotal role of macrophages in AMR pathogenesis. Importantly, patients demonstrating a high frequency of CD31:CD68 interactions experienced significantly poorer outcomes in terms of chronic AMR progression.


CONCLUSIONS
Collectively, these data indicate the potential of spatial bioinformatic as a valuable tool for aiding in pathological diagnosis and for uncovering new insights into the mechanisms underlying transplant rejection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e65530b2545156cbbf6161b36bc3af83e357d01" target='_blank'>
              Unveiling Spatial Immune Cell Profile in Kidney Allograft Rejections Using 36-plex Immunofluorescence Imaging.
              </a>
            </td>
          <td>
            Toshihito Hirai, Ayano Kondo, Tomokazu Shimizu, H. Fukuda, Daisuke Tokita, Toshio Takagi, Aaron T. Mayer, Hideki Ishida
          </td>
          <td>2024-06-24</td>
          <td>Transplantation</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf457125c2dfc7b5223e6aad5f2e990ae804842e" target='_blank'>
              Breast cancer patient-derived organoids for the investigation of patient-specific tumour evolution
              </a>
            </td>
          <td>
            S. Mazzucchelli, L. Signati, L. Messa, Alma Franceschini, A. Bonizzi, L. Castagnoli, P. Gasparini, C. Consolandi, E. Mangano, P. Pelucchi, I. Cifola, Tania Camboni, M. Severgnini, Laura Villani, Barbara Tagliaferri, S. Carelli, S. Pupa, Cristina Cereda, Fabio Corsi
          </td>
          <td>2024-06-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab01bc978e62346ed66c2ee4492c4c5372e12a2c" target='_blank'>
              Characterization of cuproptosis signature in clear cell renal cell carcinoma by single cell and spatial transcriptome analysis
              </a>
            </td>
          <td>
            Xiaohong Zou, Xiaoqing Liu, Huiting Wang, Zhenhua Li, Chen Zhou
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pro-inflammatory CD4+ T cells are major drivers of autoimmune diseases, yet therapies modulating T cell phenotypes to promote an anti-inflammatory state are lacking. Here, we identify T helper 17 (TH17) cell plasticity in the kidneys of patients with antineutrophil cytoplasmic antibody–associated glomerulonephritis on the basis of single-cell (sc) T cell receptor analysis and scRNA velocity. To uncover molecules driving T cell polarization and plasticity, we established an in vivo pooled scCRISPR droplet sequencing (iCROP-seq) screen and applied it to mouse models of glomerulonephritis and colitis. CRISPR-based gene targeting in TH17 cells could be ranked according to the resulting transcriptional perturbations, and polarization biases into T helper 1 (TH1) and regulatory T cells could be quantified. Furthermore, we show that iCROP-seq can facilitate the identification of therapeutic targets by efficient functional stratification of genes and pathways in a disease- and tissue-specific manner. These findings uncover TH17 to TH1 cell plasticity in the human kidney in the context of renal autoimmunity. A single-cell in vivo CRISPR screen highlights genes driving T cell tissue adaptation and plasticity in autoimmunity. Editor’s summary Modulating plasticity in pathogenic T cells to promote a tolerogenic phenotype is a promising therapeutic approach in autoimmunity and cancer. To explore how pathogenic TH17 cell plasticity is regulated in mouse models of kidney and gut inflammation, Enk et al. optimized an in vivo pooled single-cell CRISPR droplet sequencing (iCROP-seq) screen. iCROP-seq enabled the assessment and quantification of CRISPR-modified TH17 cell polarization states in vivo. Furthermore, T cells from patients with antineutrophil cytoplasmic antibody (ANCA)–associated glomerulonephritis exhibited plasticity. These findings indicate that iCROP-seq can be applied to preclinical mouse models of inflammation and suggest that targeting TH17 plasticity in patients with ANCA-glomerulonephritis is a promising therapeutic approach. —Hannah Isles">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dec04d242989fa1742f8f03f089c28ee5f90cc6" target='_blank'>
              Targeting T cell plasticity in kidney and gut inflammation by pooled single-cell CRISPR screening
              </a>
            </td>
          <td>
            Leon U. B. Enk, Malte Hellmig, K. Riecken, Christoph Kilian, Paul Datlinger, Saskia-L. Jauch-Speer, Tobias Neben, Zeba Sultana, Varshi Sivayoganathan, Alina Borchers, H. Paust, Yu Zhao, N. Asada, Shuya Liu, T. Agalioti, P. Pelczar, Thorsten Wiech, Christoph Bock, Tobias B. Huber, Samuel Huber, Stefan Bonn, Nicola Gagliani, B. Fehse, U. Panzer, Christian F. Krebs
          </td>
          <td>2024-06-14</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Introduction Ovarian and other peritoneal cancers have a strong tendency to metastasize into the surrounding adipose tissue. This study describes an effect of the adipose microenvironment on upregulation of sialic acid-containing glycans in ovarian cancer (OC). Heterogeneous populations of glycosylated OC tumors converged to a highly sialylated cell state that regulates tumorigenesis in an immune-dependent manner. Methods We modeled the adipose microenvironment by conditioning growth media with human patient-derived adipose tissue. OC cell lines grown in the presence vs. absence of adipose conditioned media (ACM) were characterized by transcriptomics, western blotting, and chemical biology glycan labeling methods. Fluorescence-activated cell sorting was used to separate adipose-driven upregulation of hypersialylated (“SNA-high”) vs. hyposialylated (“SNA-low”) OC subpopulations. The two subpopulations were characterized by further transcriptomic and quantitative polymerase chain reaction analyses, then injected into a syngeneic mouse model. Immune system involvement was implicated using wild type and athymic nude mice with a primary endpoint of overall survival. Results Adipose conditioning resulted in upregulation of sialyltransferases ST3GAL1, ST6GAL1, ST6GALNAC3, and ST8Sia1. In culture, OC cells displayed two distinct sialylated subpopulations that were stable for up to 9 passages, suggesting inherent heterogeneity in sialylation that is maintained throughout cell division and media changes. OC tumors that implanted in the omental adipose tissue exclusively reprogrammed to the highly sialylated subpopulation. In wild type C57BL/6 mice, only the hypersialylated SNA-high subpopulation implanted in the adipose, whereas the hyposialylated SNA-low subpopulation failed to be tumorigenic (p=0.023, n=5). In the single case where SNA-low established a tumor, post-mortem analysis revealed reprogramming of the tumor to the SNA-high state in vivo. In athymic nude mice, both subpopulations rapidly formed tumors, implicating a role of the adaptive immune system. Conclusions These findings suggest a model of glycan-dependent tumor evolution wherein the adipose microenvironment reprograms OC to a tumorigenic state that resists the adaptive immune system. Mechanistically, adipose factors upregulate sialyltransferases. To our knowledge, this is the first demonstration of the effect of adipose microenvironment on OC tumor sialylation. Our results set the stage for translational applications targeting sialic acid pathways in OC and other peritoneal cancer tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ade0f16e59c010463533d8ecb04aec54d965a7b" target='_blank'>
              Adipose microenvironment promotes hypersialylation of ovarian cancer cells
              </a>
            </td>
          <td>
            Alexandra Fox, Garry D. Leonard, Nicholas K Adzibolosu, Terrence Wong, Roslyn Tedja, Sapna Sharma, Radhika Gogoi, Robert Morris, Gil Mor, C. Fehl, A. Alvero
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cellular biomechanics plays critical roles in cancer metastasis and tumor progression. Existing studies on cancer cell biomechanics are mostly conducted in flat 2D conditions, where cells’ behavior can differ considerably from those in 3D physiological environments. Despite great advances in developing 3D in vitro models, probing cellular elasticity in 3D conditions remains a major challenge for existing technologies. In this work, we utilize optical Brillouin microscopy to longitudinally acquire mechanical images of growing cancerous spheroids over the period of eight days. The dense mechanical mapping from Brillouin microscopy enables us to extract spatially resolved and temporally evolving mechanical features that were previously inaccessible. Using an established machine learning algorithm, we demonstrate that incorporating these extracted mechanical features significantly improves the classification accuracy of cancer cells, from 74% to 95%. Building on this finding, we have developed a deep learning pipeline capable of accurately differentiating cancerous spheroids from normal ones solely using Brillouin images, suggesting the mechanical features of cancer cells could potentially serve as a new biomarker in cancer classification and detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872d3dafd293741f9c83632444cfe12e1fdf3df8" target='_blank'>
              Mechanical evolution of metastatic cancer cells in three-dimensional microenvironment
              </a>
            </td>
          <td>
            Karlin Hilai, Daniil Grubich, Marcus Akrawi, Hui Zhu, Razanne Zaghloul, Chenjun Shi, Man Do, Dongxiao Zhu, Jitao Zhang
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="ABSTRACT Conventional type 1 dendritic cells (cDC1) are critical regulators of anti-tumoral T-cell responses. The structure and abundance of intercellular contacts between cDC1 and CD8 T cells in cancer tissues is important to determine the outcome of the T-cell response. However, the molecular determinants controlling the stability of cDC1–CD8 interactions during cancer progression remain poorly investigated. Here, we generated a genetic model of non-small cell lung cancer crossed to a fluorescent cDC1 reporter (KP-XCR1venus) to allow the detection of cDC1-CD8T cell clusters in tumor tissues across tumor stages. We found that cDC1-CD8 clusters are abundant and productive at the early stages of tumor development but progressively diminish in advanced tumors. Transcriptional profiling and flow cytometry identified the adhesion molecule ALCAM/CD166 (Activated Leukocyte Cell Adhesion Molecule, ligand of CD6) as highly expressed by lung cDC1 and significantly downregulated in advanced tumors. Analysis of human datasets indicated that ALCAM is downregulated in non-small cell lung cancer and its expression correlates to better prognosis. Mechanistically, triggering ALCAM on lung cDC1 induces cytoskeletal remodeling and contact formation whereas its blockade prevents T-cell activation. Together, our results indicate that ALCAM is important to stabilize cDC1–CD8 interactions at early tumor stages, while its loss in advanced tumors contributes to immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd1ca4d0f78203d20e733a6f3ff393bf9fa0588" target='_blank'>
              ALCAM-mediated cDC1 CD8 T cells interactions are suppressed in advanced lung tumors
              </a>
            </td>
          <td>
            L. Morosi, G. M. Piperno, Lucía López, Roberto Amadio, Sonal Joshi, A. Rustighi, Giannino Del Sal, F. Benvenuti
          </td>
          <td>2024-06-14</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Intra-tumoral B cells mediate a plethora of immune effector mechanisms with key roles in anti-tumor immunity and serve as positive prognostic indicators in a variety of solid tumor types, including epithelial ovarian cancer (EOC). Several aspects of intra-tumoral B cells remain unclear, such as their state of activation, antigenic repertoires, and capacity to mature into plasma cells. Methods B lymphocytes were isolated from primary EOC tissue and malignant ascites and were maintained in cell culture medium. The stably maintained cell lines were profiled with flow cytometry and B cell receptor sequencing. Secreted antibodies were tested with a human proteome array comprising more than 21,000 proteins, followed by ELISA for validation. Originating tumor samples were used for spatial profiling with chip cytometry. Results Antibody-secreting B lymphocytes were isolated from the ovarian tumor microenvironment (TME) of four different EOC patients. The highly clonal cell populations underwent spontaneous immortalization in vitro, were stably maintained in an antibody-secreting state, and showed presence of Epstein-Barr viral (EBV) proteins. All originating tumors had high frequency of tumor-infiltrating B cells, present as lymphoid aggregates, or tertiary lymphoid structures. The antigens recognized by three of the four cell lines are coil-coil domain containing protein 155 (CCDC155), growth factor receptor-bound protein 2 (GRB2), and pyruvate dehydrogenase phosphatase2 (PDP2), respectively. Anti-CCDC155 circulating IgG antibodies were detected in 9 of 20 (45%) of EOC patients’ sera. Tissue analyses with multiparameter chip cytometry shows that the antibodies secreted by these novel human B cell lines engage their cognate antigens on tumor cells. Discussion These studies demonstrate that within the tumor-infiltrating lymphocyte population in EOC resides a low frequency population of antibody-secreting B cells that have been naturally exposed to EBV. Once stably maintained, these novel cell lines offer unique opportunities for future studies on intratumor B cell biology and new target antigen recognition, and for studies on EBV latency and/or viral reactivation in the TME of non-EBV related solid tumors such as the EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91568ddf7942710ae6ed5b0bc9abf380b3db7b97" target='_blank'>
              Characterization of latently infected EBV+ antibody-secreting B cells isolated from ovarian tumors and malignant ascites
              </a>
            </td>
          <td>
            Lixin Zhang, Mary Strange, E. Elishaev, Syed Zaidi, F. Modugno, M. Radolec, Robert P Edwards, Olivera J Finn, A. Vlad
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Single cell and multiplex imaging modalities have opened new insights into the organization and architecture of the immune microenvironment of cancer. Our group was one of the first to demonstrate these methods in the analysis of Hodgkin [1,2] and non-Hodgkin lymphoma [3]. Most recently, we demonstrated that spatial biomarkers outperform gene-expression biomarkers in predicting relapse after autologous transplant [4]. Here we present previously unpublished data demonstrating single cell multiplexed spatial analysis, using the imaging mass cytometry platform, in a cohort of 426 patients with large cell lymphoma. Gene expression and mutational analysis are available for a majority of cases allowing us to correlate spatial signature with genetic classes such as LymphGen [5], and deconvolution methods such as EcoTyper [6]. Using this cohort, we identify spatial biomarkers that improve prediction of clinical outcomes beyond cell of origin or genetic subgroup and identify novel targets for clinical development. Finally, we will present our approaches to move our discovery level spatial profiling into clinical diagnostic space through application of automated multiplex immunofluorescence imaging and a machine learning based image analysis pipeline. [1] Aoki T, et al. Single-Cell Transcriptome Analysis Reveals Disease-Defining T-cell Subsets in the Tumor Microenvironment of Classic Hodgkin Lymphoma. Cancer Discov. 2020 doi: 10.1158/2159-8290.CD-19-0680. [2] Xu, A. et al. Single cell spatial analysis and biomarker discovery in Hodgkin lymphoma. Biorxiv, 2023. https://doi.org/10.1101/2023.05.24.542195. [3] Colombo AR, et al. Single-cell spatial analysis of tumor immune architecture in diffuse large B-cell lymphoma. Blood Adv. 2022 doi: 10.1182/bloodadvances.2022007493. [4] Aoki T, et al. Spatially Resolved Tumor Microenvironment Predicts Treatment Outcomes in Relapsed/Refractory Hodgkin Lymphoma. J Clin Oncol. 2024 doi: 10.1200/JCO.23.01115. [5] Wright GW, et al. A Probabilistic Classification Tool for Genetic Subtypes of Diffuse Large B Cell Lymphoma with Therapeutic Implications. Cancer Cell. 2020 doi: 10.1016/j.ccell.2020.03.015. [6] Steen CB, et al. The landscape of tumor cell states and ecosystems in diffuse large B cell lymphoma. Cancer Cell. 2021 doi: 10.1016/j.ccell.2021.08.011.
 Citation Format: Alexander Xu, Joseph Lownik, Anton Villamejor, Simeon Mahov, Ying Li, Yidan Xu-Monette, Ken Young, Akil Merchant. Single cell spatial analysis of large B cell lymphoma identifies biomarkers correlated with genetic subtypes and clincal outcomes [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-030.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127a30be3f605294a50171a3c1596c81524ecc96" target='_blank'>
              Abstract PO-030: Single cell spatial analysis of large B cell lymphoma identifies biomarkers correlated with genetic subtypes and clincal outcomes
              </a>
            </td>
          <td>
            Alexander M Xu, Joseph Lownik, A. Villamejor, Simeon Mahov, Ying Li, Yidan Xu-Monette, Ken Young, Akil Merchant
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive and fatal tumor in the eye, and due the special biology of UM, immunotherapy showed little effect in UM patients. To improve the efficacy of immunotherapy for UM patients is of great clinical importance. Single-cell RNA sequencing(scRNA-seq) provides a critical perspective for deciphering the complexity of intratumor heterogeneity and tumor microenvironment(TME). Combing the bioinformatics analysis, scRNA-seq could help to find prognosis-related molecular indicators, develop new therapeutic targets especially for immunotherapy, and finally to guide the clinical treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c4d47ceeecc5ed316ad19f6c8292032f2ac8a3" target='_blank'>
              Single cell RNA-sequencing in uveal melanoma: advances in heterogeneity, tumor microenvironment and immunotherapy
              </a>
            </td>
          <td>
            Shiyi Tang, Yun Zhang, Shengmei Huang, Tengfei Zhu, Xiaojing Huang
          </td>
          <td>2024-06-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors are the most aggressive primary brain tumors in adults that, despite maximum treatment, carry a dismal prognosis. GBM tumors exhibit tissue hypoxia, which promotes tumor aggressiveness and maintenance of glioma stem cells and creates an overall immunosuppressive landscape. This article reviews how hypoxic conditions overlap with inflammatory responses, favoring the proliferation of immunosuppressive cells and inhibiting cytotoxic T cell development. Immunotherapies, including vaccines, immune checkpoint inhibitors, and CAR-T cell therapy, represent promising avenues for GBM treatment. However, challenges such as tumor heterogeneity, immunosuppressive TME, and BBB restrictiveness hinder their effectiveness. Strategies to address these challenges, including combination therapies and targeting hypoxia, are actively being explored to improve outcomes for GBM patients. Targeting hypoxia in combination with immunotherapy represents a potential strategy to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01181d182e89bbea741de8c2380cb4e17ca4fac4" target='_blank'>
              Hypoxia within the glioblastoma tumor microenvironment: a master saboteur of novel treatments
              </a>
            </td>
          <td>
            Lisa Feldman
          </td>
          <td>2024-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c3b8fcb2d930ae2a4d496aebe20c735fc43c688" target='_blank'>
              Pan-cancer profiling of tumor-infiltrating natural killer cells through transcriptional reference mapping
              </a>
            </td>
          <td>
            Herman Netskar, Aline Pfefferle, J. Goodridge, Ebba Sohlberg, Olli Dufva, Sarah A. Teichmann, D. Brownlie, J. Michaëlsson, N. Marquardt, Trevor Clancy, Amir Horowitz, Karl-Johan Malmberg
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>2</td>
          <td>53</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Motivation Spatial proteomics can reveal the spatial organization of immune cells in the tumor immune microenvironment. Relating measures of spatial clustering, such as Ripley’s K or Besag’s L, to patient outcomes may offer important clinical insights. However, these measures require pre-specifying a radius in which to quantify clustering, yet no consensus exists on the optimal radius which may be context-specific. Results We propose a SPatial Omnibus Test (SPOT) which conducts this analysis across a range of candidate radii. At each radius, SPOT evaluates the association between the spatial summary and outcome, adjusting for confounders. SPOT then aggregates results across radii using the Cauchy combination test, yielding an omnibus P-value characterizing the overall degree of association. Using simulations, we verify that the type I error rate is controlled and show SPOT can be more powerful than alternatives. We also apply SPOT to ovarian and lung cancer studies. Availability and implementation An R package and tutorial are provided at https://github.com/sarahsamorodnitsky/SPOT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec30fce6fd25e1c87e31190798a50855db79eb0" target='_blank'>
              A Spatial Omnibus Test (SPOT) for Spatial Proteomic Data
              </a>
            </td>
          <td>
            Sarah Samorodnitsky, Katie Campbell, Antoni Ribas, Michael C. Wu
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Simple Summary Colorectal cancer (CRC) is a major global health issue, causing many deaths each year. The current treatments often face challenges due to the cancer’s resistance to drugs, making it hard to treat effectively. This research focuses on using patient-derived tumor organoids (PDTOs), which are small, three-dimensional models grown from a patient’s tumor, to better understand and predict how CRC responds to treatments. PDTOs mimic the actual tumor environment closely, making them more reliable than traditional models. By studying PDTOs, researchers hope to find new ways to use existing drugs, understand why resistance happens, and develop better treatment plans tailored to individual patients. This approach could lead to significant advancements in how CRC is managed, potentially improving survival rates and patient outcomes. Abstract Colorectal cancer (CRC) remains a significant health burden globally, being the second leading cause of cancer-related mortality. Despite significant therapeutic advancements, resistance to systemic antineoplastic agents remains an important obstacle, highlighting the need for innovative screening tools to tailor patient-specific treatment. This review explores the application of patient-derived tumor organoids (PDTOs), three-dimensional, self-organizing models derived from patient tumor samples, as screening tools for drug resistance in CRC. PDTOs offer unique advantages over traditional models by recapitulating the tumor architecture, cellular heterogeneity, and genomic landscape and are a valuable ex vivo predictive drug screening tool. This review provides an overview of the current literature surrounding the use of PDTOs as an instrument for predicting therapy responses in CRC. We also explore more complex models, such as co-cultures with important stromal cells, such as cancer-associated fibroblasts, and organ-on-a-chip models. Furthermore, we discuss the use of PDTOs for drug repurposing, offering a new approach to identify the existing drugs effective against drug-resistant CRC. Additionally, we explore how PDTOs serve as models to gain insights into drug resistance mechanisms, using newer techniques, such as single-cell RNA sequencing and CRISPR-Cas9 genome editing. Through this review, we aim to highlight the potential of PDTOs in advancing our understanding of predicting therapy responses, drug resistance, and biomarker identification in CRC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ec72abe4a373540217d7952ed80cc0e065ef2" target='_blank'>
              Advancements in Research and Treatment Applications of Patient-Derived Tumor Organoids in Colorectal Cancer
              </a>
            </td>
          <td>
            Denise van der Graaff, Sofie Seghers, P. Vanclooster, C. Deben, T. Vandamme, Hans Prenen
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Introduction Glioma is the most frequent and lethal form of primary brain tumor. The molecular mechanism of oncogenesis and progression of glioma still remains unclear, rendering the therapeutic effect of conventional radiotherapy, chemotherapy, and surgical resection insufficient. In this study, we sought to explore the function of HEC1 (highly expressed in cancer 1) in glioma; a component of the NDC80 complex in glioma is crucial in the regulation of kinetochore. Methods Bulk RNA and scRNA‐seq analyses were used to infer HEC1 function, and in vitro experiments validated its function. Results HEC1 overexpression was observed in glioma and was indicative of poor prognosis and malignant clinical features, which was confirmed in human glioma tissues. High HEC1 expression was correlated with more active cell cycle, DNA‐associated activities, and the formation of immunosuppressive tumor microenvironment, including interaction with immune cells, and correlated strongly with infiltrating immune cells and enhanced expression of immune checkpoints. In vitro experiments and RNA‐seq further confirmed the role of HEC1 in promoting cell proliferation, and the expression of DNA replication and repair pathways in glioma. Coculture assay confirmed that HEC1 promotes microglial migration and the transformation of M1 phenotype macrophage to M2 phenotype. Conclusion Altogether, these findings demonstrate that HEC1 may be a potential prognostic marker and an immunotherapeutic target in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d939cbc805d07514492b6fb30a900e128b56d6" target='_blank'>
              NDC80/HEC1 promotes macrophage polarization and predicts glioma prognosis via single‐cell RNA‐seq and in vitro experiment
              </a>
            </td>
          <td>
            Weijie Ye, Xisong Liang, Ge Chen, Qiao Chen, Hao Zhang, Nan Zhang, Yuanfei Huang, Q. Cheng, Xiaoping Chen
          </td>
          <td>2024-07-01</td>
          <td>CNS Neuroscience & Therapeutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bone pain is a presenting feature of bone cancers such as osteosarcoma (OS), relayed by skeletal-innervating peripheral afferent neurons. Potential functions of tumor-associated sensory neurons in bone cancers beyond pain sensation are unknown. To uncover neural regulatory functions, a chemical-genetic approach in mice with a knock-in allele for TrkA was used to functionally perturb sensory nerve innervation during OS growth and disease progression. TrkA inhibition in transgenic mice led to significant reductions in sarcoma-associated sensory innervation and vascularization, tumor growth and metastasis, and prolonged overall survival. Single-cell transcriptomics revealed that sarcoma denervation was associated with phenotypic alterations in both OS tumor cells and cells within the tumor microenvironment, and with reduced calcitonin gene-related peptide (CGRP) and vascular endothelial growth factor (VEGF) signaling. Multimodal and multi-omics analyses of human OS bone samples and human dorsal root ganglia neurons further implicated peripheral innervation and neurotrophin signaling in OS tumor biology. In order to curb tumor-associated axonal ingrowth, we next leveraged FDA-approved bupivacaine liposomes leading to significant reductions in sarcoma growth, vascularity, as well as alleviation of pain. In sum, TrkA-expressing peripheral neurons positively regulate key aspects of OS progression and sensory neural inhibition appears to disrupt calcitonin receptor signaling (CALCR) and VEGF signaling within the sarcoma microenvironment leading to significantly reduced tumor growth and improved survival. These data suggest that interventions to prevent pathological innervation of osteosarcoma represent a novel adjunctive therapy to improve clinical outcomes and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38e8f176dcc8dd09eaf33b5874e7e585465ebfc" target='_blank'>
              TrkA+ sensory neurons regulate osteosarcoma proliferation and vascularization to promote disease progression
              </a>
            </td>
          <td>
            Q. Qin, Sowmya Ramesh, Zhao Li, Lingke Zhong, M. Cherief, Mary Archer, X. Xing, N. Thottappillil, M. Gomez-Salazar, Mingxin Xu, Manyu Zhu, Leslie Chang, Ankit Uniyal, Khadija Mazhar, Monisha Mittal, Edward F. McCarthy, Carol D. Morris, Benjamin Levi, Yun Guan, Thomas L. Clemens, Theodore J. Price, Aaron W. James
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Over decades, cancer understanding has advanced significantly at molecular and cellular levels, leading to various therapies based on intra-/inter-cellular networks. Despite this, cancer still remains a leading cause of death globally. The primary driver of cancer mortality is metastasis, responsible for about 90% of cancer deaths, due to unclear pathophysiological mechanisms that complicate treatment development. The Notch signaling pathway, a crucial intercellular network in many cancers, has been extensively studied and therapies targeting the Notch pathway also have been well-studied based on inhibiting various stages of Notch activation. On the other hand, Notch signaling’s role varies between cancers; for instance, in non-small cell lung cancer, Notch1 and Notch2 have opposing effects compared to their roles in embryonal brain tumors. In this study, we assumed a scenario of multiple cancers with contrasting Notch signaling pathways and explored optimal targeted therapies for reducing cancer cells by developing two mathematical models with contrasting Notch signaling pathways. The proposed therapies were compared with existing ones, and strategies were investigated to reduce cancer cell numbers for different stage of cancer. We found that that multiple cancers with contrasting Notch networks can be controlled by a common targeted signal network. Combination therapy enhancing Notch production may be most effective in early-stage cancer, while cleavage therapies may be more effective in late-stage cancer. Our study also suggests that optimal treatment should consider the cancer stage, with careful selection and ordering of medication therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd0b0aca24e705cffc10843b8484f56eae84233b" target='_blank'>
              Optimal targeted therapy for multiple cancers based on contrastive Notch signaling networks
              </a>
            </td>
          <td>
            Tamaki Wakamoto, S. Seirin-Lee
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma poses a formidable challenge among primary brain tumors: its tumorigenic stem cells, capable of self-renewal, proliferation, and differentiation, contribute substantially to tumor initiation and therapy resistance. These glioblastoma stem cells (GSCs), resembling conventional stem and progenitor cells, adopt pathways critical for tissue development and repair, promoting uninterrupted tumor expansion. Long non-coding RNAs (lncRNAs), a substantial component of the human transcriptome, have garnered considerable interest for their pivotal roles in normal physiological processes and cancer pathogenesis. They display cell- or tissue-specific expression patterns, and extensive investigations have highlighted their impact on regulating GSC properties and cellular differentiation, thus offering promising avenues for therapeutic interventions. Consequently, lncRNAs, with their ability to exert regulatory control over tumor initiation and progression, have emerged as promising targets for innovative glioblastoma therapies. This review explores notable examples of GSC-associated lncRNAs and elucidates their functional roles in driving glioblastoma progression. Additionally, we delved deeper into utilizing a 3D in vitro model for investigating GSC biology and elucidated four primary methodologies for targeting lncRNAs as potential therapeutics in managing glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ed3ed8d6a1c37f2db5f6f03648c49bfe8a7955" target='_blank'>
              Glioblastoma stem cell long non-coding RNAs: therapeutic perspectives and opportunities
              </a>
            </td>
          <td>
            R. Hazra, Rinku Debnath, Arati Tuppad
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Papillary renal cell carcinoma (pRCC) accounts for up to 15% of all kidney cancer cases, yet our understanding of its tumor immune microenvironment (TIME) remains limited. We utilized multiplex immunofluorescence (mIF) and spatial transcriptomics (ST) to evaluate immune cell spatial architecture in pRCC and compared to that of clear cell RCC (ccRCC). Methods Surgical tumor specimens were obtained from localized RCC tumors, followed by mIF using markers for T cells, B cells, and tumor-associated macrophages (TAMs). Spatial data were derived in regions of interest (ROIs) manually selected from spatially distinct tissue compartments of the TIME. Single-cell ST was performed on a subset of patient samples, utilizing probes against 960 transcripts. Cell abundance, cell spatial clustering, and spatially varying gene expression were analyzed to identify unique features of the TIME in pRCC. Results Sixteen pRCC and 70 ccRCC patient samples underwent mIF. Compared to ccRCC, global pRCC immune cell abundance was statistically lower amongst functional CD8 T cells, while global cell spatial clustering was higher amongst M2-like macrophages as measured by mIF, including PDL1+ subsets. Using ST, seven genes were significantly associated with spatial clustering of M2-like macrophages in pRCC. Three of seven genes (CCL18, GPNMB, CD9) are known markers of lipid-associated TAMs (LAMs) (adjusted p-value < 0.1). Conclusions Compared to ccRCC, pRCC has fewer T cells but greater M2-like macrophage spatial clustering. Using ST, we found that multiple LAM-associated genes are spatially enriched in pRCC. Additional resources should be dedicated to investigating myeloid biomarkers and the impact of myeloid modulating therapeutics in pRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c40ee543a137bacb770b063bafef44d824bc2e7" target='_blank'>
              27 Spatial Clustering of Immunosuppressive Macrophages in Papillary Renal Cell Carcinoma
              </a>
            </td>
          <td>
            Alex C. Soupir, Mitchell T Hayes, José Laborde, Mr Kirill Sabitov, Taylor Peak, Mr Jonathan Nguyen, Carlos M. Moran-Segura, Daryoush Saeed-Vafa, Neale Lopez-Blanco, Ms Paola Ramos Echevarria, Christopher Guske, Jodi Blasi, J. Dhillon, Youngchul Kim, James Mulé, Brooke L. Fridley, Brandon J. Manley
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive neuroendocrine skin cancer with a ~50% response rate to immune checkpoint blockade (ICB) therapy. To identify predictive biomarkers, we integrated bulk and single-cell RNA-seq with spatial transcriptomics from a cohort of 186 samples from 116 patients, including bulk RNA-seq from 14 matched pairs pre- and post-ICB. In non-responders, tumors show evidence of increased tumor proliferation, neuronal stem cell markers, and IL-1. Responders have increased type I/II interferons and pre-existing tissue resident (Trm) CD8 or Vd1 gd T cells that functionally converge with overlapping antigen-specific transcriptional programs and clonal expansion of public TCRs. Spatial transcriptomics demonstrated co-localization of T cells with B and dendritic cells, which supply chemokines and co-stimulation. Lastly, ICB significantly increased clonal expansion or recruitment of Trm and Vd1 cells in tumors specifically in responders, underscoring their therapeutic importance. These data identify potential clinically actionable biomarkers and therapeutic targets for MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc8fff7c0c805192f9c9a67df2e29b038f02c52" target='_blank'>
              Pre-existing skin-resident CD8 and gd T cell circuits mediate immune response in Merkel cell carcinoma and predict immunotherapy efficacy.
              </a>
            </td>
          <td>
            Z. Reinstein, Yue Zhang, Oscar E. Ospina, Matt D Nichols, Victoria A Chu, Alvaro de Mingo Pulido, Karol Prieto, Jonathan V Nguyen, Rui Yin, Carlos M Moran Segura, Ahmed Usman, B. Sell, Spencer Ng, Janis V de la Iglesia, Sunandana Chandra, Jeffrey A Sosman, Raymond J. Cho, Jeffrey B. Cheng, E. Ivanova, S. Koralov, R. Slebos, C. Chung, N. Khushalani, Jane L. Messina, A. Sarnaik, J. Zager, V. Sondak, Charles J. Vaske, Sungjune Kim, A. Brohl, Xinlei Mi, Brian G. Pierce, Xuefeng Wang, Brooke L. Fridley, Kenneth Y. Tsai, Jaehyuk Choi
          </td>
          <td>2024-07-26</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Hypopharyngeal squamous cell carcinoma (SCC) is a highly aggressive cancer with a poor prognosis, particularly in advanced stages where concurrent chemoradiotherapy (CCRT) is used for treatment. However, resistance to CCRT poses a significant challenge, often leading to treatment failure and disease progression. This study explores the tumor microenvironment (TME) of hypopharyngeal SCC to understand the molecular mechanisms underlying CCRT resistance. Using spatial transcriptomics (ST), we analyzed tissue samples from patients with locally advanced hypopharyngeal SCC, distinguishing between those who were CCRT-resistant and those who were CCRT-naive. The analysis revealed six distinct cellular clusters within the TME, including a prominent epithelio-immune cellular area in CCRT-resistant tissues. SPP1 was identified as a key gene with significantly higher expression in CCRT-resistant samples, specifically within macrophages. Further investigation showed that SPP1+ macrophages interacted with malignant epithelial cells through SPP1-CD44 and SPP1-ITGB1 ligand-receptor pairs. These interactions were primarily localized in the peri-tumoral and intra-tumoral regions, highlighting their potential role in driving CCRT resistance. Our findings suggest that SPP1+ macrophages contribute to the resistant phenotype in hypopharyngeal SCC by modulating the TME and interacting with cancer cells. Understanding these interactions offers valuable insights into the mechanisms of CCRT resistance and may inform the development of targeted therapies to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d07108a9430fa63556b14b31c6b832a1633f5b4" target='_blank'>
              Spatial transcriptomics unveils landscape of resistance to concurrent chemo-radiotherapy in hypopharyngeal squamous cell carcinoma: the role of SPP1+ macrophages
              </a>
            </td>
          <td>
            J. Ohn, Sungwoo Bae, Hongyoon Choi, In Gul Kim, Kwon Joong Na, Eun-Jae Chung
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite extensive research efforts, cancer continues to stand as one of the leading causes of death on a global scale. To gain profound insights into the intricate mechanisms underlying cancer onset and progression, it is imperative to possess methodologies that allow the study of cancer cells at the single-cell level, focusing on critical parameters such as cell morphology, metabolism, and molecular characteristics. These insights are essential for effectively discerning between healthy and cancerous cells and comprehending tumoral progression. Recent advancements in microscopy techniques have significantly advanced the study of cancer cells, with Raman microspectroscopy (RM) emerging as a particularly powerful tool. Indeed, RM can provide both biochemical and spatial details at the single-cell level without the need for labels or causing disruptions to cell integrity. Moreover, RM can be correlated with other microscopy techniques, creating a synergy that offers a spectrum of complementary insights into cancer cell morphology and biology. This review aims to explore the correlation between RM and other microscopy techniques such as confocal fluoresce microscopy (CFM), atomic force microscopy (AFM), digital holography microscopy (DHM), and mass spectrometry imaging (MSI). Each of these techniques has their own strengths, providing different perspectives and parameters about cancer cell features. The correlation between information from these various analysis methods is a valuable tool for physicians and researchers, aiding in the comprehension of cancer cell morphology and biology, unraveling mechanisms underlying cancer progression, and facilitating the development of early diagnosis and/or monitoring cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/493ff8b7363d13b88cedadb4ac3e4389b6265366" target='_blank'>
              Correlative Raman Imaging: Development and Cancer Applications
              </a>
            </td>
          <td>
            Hossein Khadem, Maria Mangini, Somayeh Farazpour, Anna Chiara De Luca
          </td>
          <td>2024-06-28</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a86fe03c5eeba9c5514e3529ad3773b3e2ecef7" target='_blank'>
              Three-dimensional environment sensitizes pancreatic cancer cells to the anti-proliferative effect of budesonide by reprogramming energy metabolism
              </a>
            </td>
          <td>
            Eduardo Ibello, Federica Saracino, Donatella Delle Cave, S. Buonaiuto, Filomena Amoroso, Gennaro Andolfi, Marco Corona, O. Guardiola, V. Colonna, Bruno Sainz, Lucia Altucci, Dario De Cesare, G. Cobellis, E. Lonardo, E. Patriarca, Cristina d’Aniello, G. Minchiotti
          </td>
          <td>2024-06-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/251acda7df20c5eff2f9bfbbad717de02554a201" target='_blank'>
              Unlocking the potential of HHLA2: identifying functional immune infiltrating cells in the tumor microenvironment and predicting clinical outcomes in laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Wenjing Li, Jianqing You, Haixiang Xue, Yi Liu, Junjun Chen, Xiao Zheng, Lujun Chen, Changping Wu
          </td>
          <td>2024-08-06</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Improving the prognosis for patients diagnosed with aggressive tumors remains a critical challenge in clinical oncology. One crucial aspect of addressing this challenge involves the development of preclinical models that can accurately predict the effectiveness of potential therapies for clinical application. A pivotal element in enhancing these models lies in their ability to accurately recapitulate the tumor microenvironment. In this review, the authors focus on investigating the tumor microenvironment in the context of ex vivo glioma modelling, with a specific emphasis on the extracellular matrix. The review discusses the changes that occur within the extracellular matrix of pediatric and adult gliomas. Furthermore, it explores the potential of advanced cellular models, particularly those employing bioprinting technologies, to replicate the extracellular matrix ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57df2f73960d1564d3bc87ee081e279feba2a7e0" target='_blank'>
              Ex Vivo Modeling of the Tumor Microenvironment to Develop Therapeutic Strategies for Gliomas
              </a>
            </td>
          <td>
            Philipp Graber, M. E. M. Dolman, MoonSun Jung, Maria Kavallaris
          </td>
          <td>2024-06-16</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2526497a55bfddb0a519406f7853c48bb5f37540" target='_blank'>
              Digital spatial profiling of the microenvironment of muscle invasive bladder cancer
              </a>
            </td>
          <td>
            Michael Eyers, J. Irlam, Gayle Marshall, V. Smith, Alexander Baker, Lucy Frost, P. Hoskin, Ananya Choudhury, C. West
          </td>
          <td>2024-06-18</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is a multifaceted condition characterized by significant diversity in its molecular and pathological subtypes and clinical manifestation. Despite the progress made in the treatment of DLBCL through the development of novel drugs, an estimated one-third of patients encounter relapse or acquire refractory disease. The tumor microenvironment (TME) of DLBCL, a complex network consisting of cellular and noncellular components that engage in interactions with the tumor, is a parameter that is gaining increasing attention. The TME comprises both the immune and nonimmune microenvironments. The immune microenvironment comprises natural killer (NK) cells, dendritic cells (DCs), tumor-associated macrophages (TAMs), neutrophils, myeloid-derived suppressor cells (MDSCs), and T and B lymphocytes. The nonimmune microenvironment consists of the extracellular matrix (ECM), cancer-associated fibroblasts (CAFs), mesenchymal stromal cells, and other molecules that are secreted. Despite ongoing research, the exact impact of these components and their interaction on the progression of the disease remains elusive. A comprehensive review of significant discoveries concerning the cellular and noncellular constituents, molecular characteristics, and treatment response and prognosis of the TME in DLBCL, as well as the potential targeting of the TME with novel therapeutic approaches, is provided in this article.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842f1b179cae9c35039714b759e7f43b1c1c5ae1" target='_blank'>
              Unraveling the Immune Microenvironment in Diffuse Large B-Cell Lymphoma: Prognostic and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            E. Koumpis, A. Papoudou-Bai, K. Papathanasiou, E. Kolettas, Panagiotis Kanavaros, E. Hatzimichael
          </td>
          <td>2024-07-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e877f4d47a5850d3ceab780a2d2a2472c654fb" target='_blank'>
              High-dimensional single-cell analysis of human natural killer cell heterogeneity
              </a>
            </td>
          <td>
            Lucas Rebuffet, J. Melsen, Bertrand Escalière, Daniela Basurto-Lozada, A. Bhandoola, N. Björkström, Y. Bryceson, R. Castriconi, Frank Cichocki, Marco Colonna, Daniel M Davis, Andreas Diefenbach, Yi Ding, M. Haniffa, Amir Horowitz, Lewis L. Lanier, Karl-Johan Malmberg, Jeffrey S Miller, Lorenzo Moretta, Émilie Narni-Mancinelli, L. A. O'Neill, C. Romagnani, Dylan G Ryan, S. Sivori, Dan Sun, Constance Vagne, Eric Vivier
          </td>
          <td>2024-07-02</td>
          <td>Nature Immunology</td>
          <td>4</td>
          <td>66</td>
        </tr>

        <tr id="Substantial heterogeneity in molecular features, patient prognoses, and therapeutic responses in head and neck squamous cell carcinomas (HNSCC) highlights the urgent need to develop molecular classifications that reliably and accurately reflect tumor behavior and inform personalized therapy. Here, we leveraged the similarity network fusion bioinformatics approach to jointly analyze multi-omics datasets spanning copy number variations, somatic mutations, DNA methylation, and transcriptomic profiling and derived a prognostic classification system for HNSCC. The integrative model consistently identified three subgroups (IMC1-3) with specific genomic features, biological characteristics, and clinical outcomes across multiple independent cohorts. The IMC1 subgroup included proliferative, immune-activated tumors and exhibited a more favorable prognosis. The IMC2 subtype harbored activated EGFR signaling and an inflamed tumor microenvironment with cancer-associated fibroblast/vascular infiltrations. Alternatively, the IMC3 group featured highly aberrant metabolic activities and impaired immune infiltration and recruiting. Pharmacogenomics analyses from in silico predictions and from patient-derived xenograft model data unveiled subtype-specific therapeutic vulnerabilities including sensitivity to cisplatin and immunotherapy in IMC1 and EGFR inhibitors (EGFRi) in IMC2, which was experimentally validated in patient-derived organoid models. Two signatures for prognosis and EGFRi sensitivity were developed via machine learning. Together, this integrative multi-omics clustering for HNSCC improves current understanding of tumor heterogeneity and facilitates patient stratification and therapeutic development tailored to molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c3e275f858f75f81c9f658d4d03e003b3adc1b" target='_blank'>
              Integrative multi-omics analyses identify molecular subtypes of head and neck squamous cell carcinoma with distinct therapeutic vulnerabilities.
              </a>
            </td>
          <td>
            Pengfei Diao, Yibin Dai, An Wang, Xiaoxuan Bu, Ziyu Wang, Jin Li, Yaping Wu, Hongbing Jiang, Yanling Wang, Jie Cheng
          </td>
          <td>2024-07-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/980719e09512c7538d00c8fd96d0dd2473599e42" target='_blank'>
              Loss of tumor suppressors promotes inflammatory tumor microenvironment and enhances LAG3+T cell mediated immune suppression
              </a>
            </td>
          <td>
            Sara Zahraeifard, Zhiguang Xiao, Jae Young So, Abdul Ahad, Selina Montoya, Woo-Yong Park, Trinadharao Sornapudi, Tiffany Andohkow, Abigail Read, Noemi Kedei, Vishal N. Koparde, H. Yang, M. Lee, Nathan Wong, Maggie Cam, Kun Wang, E. Ruppin, Ji Luo, C. Hollander, Li Yang
          </td>
          <td>2024-07-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM), a malignant melanocyte-derived skin cancer, potentially leads to fatal outcomes without effective treatment. The variability in immunotherapy responses among melanoma patients is significantly influenced by the intricate immune microenvironment, particularly due to the status of tumor T cells, encompassing their activity, exhaustion levels, and antigen recognition capabilities. This study utilized single-cell RNA sequencing (scRNA-seq) to analyze 34 melanoma samples from two public datasets (GSE215120 and GSE115978). Herein, we extracted 706 marker genes associated with immune checkpoint (ICP) therapy from these T cells, 509 markers of T cells from 11 melanoma tissues, and eventually identified 33 candidate genes. These genes underwent LASSO and COX regression analyses to identify the signature genes. Of the initial 33 candidate genes, we successfully isolated six distinct T cell-associated immunotherapy-related genes (IRTGs). Additionally, the computation of each patient risk score proved beneficial in evaluating the immune cell infiltration level and functions as an independent prognostic factor for melanoma patient survival. The risk score results revealed promising predictive outcomes in determining the response of melanoma patients to immunotherapy. Notably, our study is the first to reveal the potential correlation between signature gene PEB4B and the immune microenvironment in melaoma, which was explored with multiple immunofluorescence (IF) and Immune Infiltration Assessment. In a conclusion, our findings demonstrate the potential utility of a risk score dependent on signature genes as a predictive tool for assessing the prognosis and response to immunotherapeutic interventions in melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4480d7b0dac6fe343faaffd214e0f795e255eda" target='_blank'>
              The combination of single-cell and RNA sequencing analysis decodes the melanoma tumor microenvironment and identifies novel T cell-associated signature genes
              </a>
            </td>
          <td>
            Sihan Luo, Daiyue Wang, Jiajie Chen, Shaocheng Hong, Yuanyuan Fang, Lu Cao, Liang Yong, Shengxiu Liu
          </td>
          <td>2024-08-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e1810e2d03e004eeb58c6ca7cae3dc32ba5c9" target='_blank'>
              Single-cell transcriptional analysis of irradiated skin reveals changes in fibroblast subpopulations and variability in caveolin expression
              </a>
            </td>
          <td>
            Lionel E. Kameni, Michelle F. Griffin, Charlotte E. Berry, Siavash Shariatzadeh, Mauricio A. Downer, Caleb Valencia, Alexander Z. Fazilat, Rahim S Nazerali, Arash Momeni, Michael Januszyk, M. Longaker, Derrick C. Wan
          </td>
          <td>2024-06-26</td>
          <td>Radiation Oncology (London, England)</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaac037eca190874316bd2b34e72ed4d604d818c" target='_blank'>
              Sulfatide imaging identifies tumor cells in colorectal cancer peritoneal metastases
              </a>
            </td>
          <td>
            G. M. Sarcinelli, L. Varinelli, S. Ghislanzoni, F. Padelli, D. Lorenzini, A. Vingiani, M. Milione, M. Guaglio, S. Kusamura, M. Deraco, G. Pruneri, M. Gariboldi, D. Baratti, I. Bongarzone
          </td>
          <td>2024-06-28</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinoma (RCC) with histological sarcomatoid de-differentiation (sRCC) has a historically poor prognosis across all RCC subtypes. Recently, immunotherapy with anti-PD1 + anti-CTLA4 combo has significantly improved disease-specific survival, and complete response are seen in upwards of 20% of patients. However, half of sRCC patients still have no response to this immunotherapy, and the mechanism of response remains unclear. Methods Our preliminary bulk RNA sequencing data reveal a distinct tumor microenvironment (TME) in human sRCC tumors, characterized by heightened immune cell infiltration, particularly enriched for regulatory T cells (Tregs). Subsequently, we delineated the spatial gene expression topography within sRCC and clear cell renal cell carcinoma (ccRCC) regions, culminating in the development of the sRCC signature. Concurrently, we established an innovative immunocompetent murine model of sRCC. Leveraging CRISPR technology, we replicated the human sRCC genotype by knocking out Vhl, Bap1, and Cdkn2a/2b genes in a mouse telomerase reverse transcriptase (mTERT) overexpressing renal proximal tubule epithelial cell line. This effort yielded a successfully engineered mouse implantable sRCC cell line bearing VhlMutBap1DelCdkn2aDelCdkn2bDel (HJRCC68N). Results The syngeneic mouse sRCC model we developed faithfully replicates human sRCC histology and tumor microenvironment and recapitulates the sRCC signature identified in human sRCC regions. Importantly, like in humans, mouse sRCC tumors exhibit a heterogeneous response to anti-PD1 + anti-CTLA4 combination, and anti-CTLA4 monotherapy, reflecting the clinical variability observed in human patients. Furthermore, transcriptomic analysis from our mouse model reveals the involvement of type 1 immunity in responder, providing insights into overcoming immunotherapy resistance. Conclusions In conclusion, our study unveiled distinct changes in the TME at the transcriptomic and spatial gene expression levels in human sRCC. Leveraging our novel murine model, which faithfully replicates human sRCC histology, TME characteristics, and mixed response to immunotherapy, we elucidated the involvement of type 1 immunity in treatment response. These insights offer valuable pathways for overcoming immunotherapy resistance and pave the way for the development of tailored treatment strategies aimed at improving outcomes for sRCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5fbfe11a82cb5fed55f6641955ed49526e183d8" target='_blank'>
              18 Unraveling the Impact of Tumor Microenvironment on Immunotherapy Response/Resistance in Human Sarcomatoid Clear Cell Renal Cell Carcinoma: Insights from a Novel Mouse Model
              </a>
            </td>
          <td>
            Hui Jiang
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the quest for improved therapeutics targeting immune checkpoints (ICs), we turn to spontaneously developing dog (canine) cancers, which are unique models that genetically and clinically mirror human equivalents. Despite its potential, canine cancer immunology remains largely unexplored. Here, we examine the RNA-seq-based expression of 44 ICs across 14 canine cancer types and an extensive human dataset. We unveil diverse canine IC expression patterns and unique human IC signatures that reflect the histological type and primary site of cancer. We uncover a striking similarity between canine brain cancers, osteosarcoma, and their human counterparts, identifying them as prospective immunotherapy models. Four ICs—CD160, A2AR, NKG2A, and OX40—are key to the differences observed between species. Moreover, individual patient IC signatures exhibit varying alignment with their respective cancer types, a finding with profound implications for personalized human therapy. This exploration illuminates new aspects of canine and human cancer immunology, setting the stage for discoveries at their crossroads.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f77ebd885f80c0d134fa31e9198696b90786b6" target='_blank'>
              Barking Up the Right Tree: Immune Checkpoint Signatures of Human and Dog Cancers
              </a>
            </td>
          <td>
            M. Kocikowski, M. Yébenes Mayordomo, Javier Alfaro, Maciej Parys
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Oral squamous cell carcinomas (OSCC), the predominant head and neck cancer, pose significant challenges due to late-stage diagnoses and low five-year survival rates. Spatial transcriptomics offers a promising avenue to decipher the genetic intricacies of OSCC tumor microenvironments. In spatial transcriptomics, Cell-type deconvolution is a crucial inferential goal; however, current methods fail to consider the high zero-inflation present in OSCC data. To address this, we develop a novel zero-inflated version of the hierarchical generalized transformation model (ZI-HGT) and apply it to the Conditional AutoRegressive Deconvolution (CARD) for cell-type deconvolution. The ZI-HGT serves as an auxiliary Bayesian technique for CARD, reconciling the highly zero-inflated OSCC spatial transcriptomics data with CARD’s normality assumption. The combined ZI-HGT + CARD framework achieves enhanced cell-type deconvolution accuracy and quantifies uncertainty in the estimated cell-type proportions. We demonstrate the superior performance through simulations and analysis of the OSCC data. Furthermore, our approach enables the determination of the locations of the diverse fibroblast population in the tumor microenvironment, critical for understanding tumor growth and immunosuppression in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/090b060bd9dd44f20620bc9cc982e8c6875f2751" target='_blank'>
              A Zero-Inflated Hierarchical Generalized Transformation Model to Address Non-Normality in Spatially-Informed Cell-Type Deconvolution
              </a>
            </td>
          <td>
            Hunter J. Melton, Jonathan R. Bradley, Chong Wu
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30215d511cd6c83bd7fde1586962eb127268dc49" target='_blank'>
              Comprehensive analysis of single cell and bulk RNA sequencing reveals the heterogeneity of melanoma tumor microenvironment and predicts the response of immunotherapy.
              </a>
            </td>
          <td>
            Yuan Zhang, Cong Zhang, Jing He, Guichuan Lai, Wenlong Li, Haijiao Zeng, Xiaoni Zhong, Biao Xie
          </td>
          <td>2024-06-19</td>
          <td>Inflammation research : official journal of the European Histamine Research Society ... [et al.]</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2a1e8a0b040b1a9b17302a42daea756ed498f33" target='_blank'>
              Cancer cell plasticity defines response to immunotherapy in cutaneous squamous cell carcinoma
              </a>
            </td>
          <td>
            Laura Lorenzo-Sanz, Marta Lopez-Cerda, Victoria da Silva-Diz, Marta H Artés, Sandra Llop, Rosa M. Penín, Josep Oriol Bermejo, Eva Gonzalez-Suarez, Manel Esteller, Francesc Viñals, Enrique Espinosa, Marc Oliva, J. Piulats, J. Martín-Liberal, Purificación Muñoz
          </td>
          <td>2024-06-24</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Novel therapies targeting KRAS offer treatment options for previously untreatable patients. However, in colorectal cancer (CRC), resistance to KRAS-targeted therapy develops rapidly, making it imperative to understand its underlying mechanisms. Cancer-associated fibroblasts (CAFs) contribute to therapy resistance by generating and maintaining the cancer stem cell niche. This study investigates whether CAF-secreted factors induce resistance to KRAS inhibition by enhancing cancer stemness. Our findings demonstrate that while KRAS silencing reduced the expression of stem cell markers and stemness, CAF-secreted factors counteracted those effects by activating pro-tumorigenic pathways, such as epithelial-to-mesenchymal transition, and increasing cell proliferation. Overall, we provide novel mechanistic insights into how CAF-secreted factors oppose KRAS silencing-induced growth inhibition, which may be crucial for improving CRC therapy. Abstract Colorectal cancer (CRC) responses to KRAS-targeted inhibition have been limited due to low response rates, the mechanisms of which remain unknown. Herein, we explored the cancer-associated fibroblasts (CAFs) secretome as a mediator of resistance to KRAS silencing. CRC cell lines HCT15, HCT116, and SW480 were cultured either in recommended media or in conditioned media from a normal colon fibroblast cell line (CCD-18Co) activated with rhTGF-β1 to induce a CAF-like phenotype. The expression of membrane stem cell markers was analyzed by flow cytometry. Stem cell potential was evaluated by a sphere formation assay. RNAseq was performed in KRAS-silenced HCT116 colonospheres treated with either control media or conditioned media from CAFs. Our results demonstrated that KRAS-silencing up-regulated CD24 and down-regulated CD49f and CD104 in the three cell lines, leading to a reduction in sphere-forming efficiency. However, CAF-secreted factors restored stem cell marker expression and increased stemness. RNA sequencing showed that CAF-secreted factors up-regulated genes associated with pro-tumorigenic pathways in KRAS-silenced cells, including KRAS, TGFβ, NOTCH, WNT, MYC, cell cycle progression and exit from quiescence, epithelial-mesenchymal transition, and immune regulation. Overall, our results suggest that resistance to KRAS-targeted inhibition might derive not only from cell-intrinsic causes but also from external elements, such as fibroblast-secreted factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9813f2738a9fd05b2f447429fc53a99374d146" target='_blank'>
              Fibroblasts Promote Resistance to KRAS Silencing in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Susana Mendonça Oliveira, P. D. Carvalho, André Serra-Roma, Patrícia Oliveira, A. Ribeiro, J. Carvalho, F. Martins, A. Machado, Maria José Oliveira, S. Velho
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9120ab8ec6281dd0ae5d345b0a1c2eb5d5dfca5c" target='_blank'>
              Mutation analysis in individual circulating tumor cells depicts intratumor heterogeneity in melanoma
              </a>
            </td>
          <td>
            Mark Sementsov, L. Ott, Julian Kött, A. Sartori, Amélie Lusque, Sarah Degenhardt, Bertille Segier, I. Heidrich, B. Volkmer, Rüdiger Greinert, Peter Mohr, R. Simon, J. Stadler, Darryl Irwin, Claudia Koch, Antje Andreas, Benjamin Deitert, V. Thewes, A. Trumpp, A. Schneeweiss, Yassine Belloum, S. Peine, H. Wikman, S. Riethdorf, Stefan W. Schneider, C. Gebhardt, Klaus Pantel, Laura Keller
          </td>
          <td>2024-06-19</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Tumor structure is heterogeneous and complex, and it is difficult to obtain complete characteristics by two‐dimensional analysis. The aim of this study was to visualize and characterize volumetric vascular information of clear cell renal cell carcinoma (ccRCC) tumors using whole tissue phenotyping and three‐dimensional light‐sheet microscopy. Here, we used the diagnosing immunolabeled paraffin‐embedded cleared organs pipeline for tissue clearing, immunolabeling, and three‐dimensional imaging. The spatial distributions of CD34, which targets blood vessels, and LYVE‐1, which targets lymphatic vessels, were examined by calculating three‐dimensional density, vessel length, vessel radius, and density curves, such as skewness, kurtosis, and variance of the expression. We then examined those associations with ccRCC outcomes and genetic alteration state. Formalin‐fixed paraffin‐embedded tumor samples from 46 ccRCC patients were included in the study. Receiver operating characteristic curve analyses revealed the associations between blood vessel and lymphatic vessel distributions and pathological factors such as a high nuclear grade, large tumor size, and the presence of venous invasion. Furthermore, three‐dimensional imaging parameters stratified ccRCC patients regarding survival outcomes. An analysis of genomic alterations based on volumetric vascular information parameters revealed that PI3K‐mTOR pathway mutations related to the blood vessel radius were significantly different. Collectively, we have shown that the spatial elucidation of volumetric vasculature information could be prognostic and may serve as a new biomarker for genomic alterations. High‐end tissue clearing techniques and volumetric immunohistochemistry enable three‐dimensional analysis of tumors, leading to a better understanding of the microvascular structure in the tumor space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63465b7766c5458fa29857848c92dc3c85ce6311" target='_blank'>
              Volumetric imaging of the tumor microvasculature reflects outcomes and genomic states of clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Yuta Kaneko, Tsukasa Masuda, Kimiharu Takamatsu, S. Mikami, Kohei Nakamura, Hiroshi Nishihara, R. Mizuno, Nobuyuki Tanaka, M. Oya
          </td>
          <td>2024-06-26</td>
          <td>The Journal of Pathology: Clinical Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background: Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain. Methods: Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package “Seurat” was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes. Results: Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA. Conclusion: A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CD8+ T cells play pivotal roles in combating intracellular pathogens and eliminating malignant cells in cancer. However, the prognostic role of CD8+ T cells in ovarian carcinoma is insufficiently exploited. Herein, through univariate Cox regression along with least absolute shrinkage and selection operator (LASSO) regression analyses, we developed a novel prognostic model based on CD8+ T cell markers identified by single-cell sequencing (scRNA-seq) analyses. Patient grouping by the median risk score reveals an excellent prognostic efficacy of this model in both training and validation cohorts. Of note, patients classified as low-risk group exhibit a dramatically improved prognosis. In addition, higher enrichment level of immune-related pathways and increased infiltration level of multiple immune cells are found in patients with lower risk score. Importantly, low-risk patients also exhibited higher response rate to immunotherapies. Summarily, this developed CD8+ T cell-associated prognostic model serves as an excellent predictor for clinical outcomes and aids in guiding therapeutic strategy choices for ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/409d1a4ad42cb1c888c3ed09d0a52f53075d8d47" target='_blank'>
              Comprehensive single-cell and bulk RNA-seq analyses reveal a novel CD8+ T cell-associated prognostic signature in ovarian cancer
              </a>
            </td>
          <td>
            Yi-Ling Han, Zhou Fang, Zhijie Gao, Wen-Ge Li, Jing Yang
          </td>
          <td>2024-06-25</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
In multiple myeloma (MM), therapy-induced clonal evolution is associated with treatment resistance and is one of the most important hindrances toward a cure for MM. To further understand the molecular mechanisms controlling the clonal evolution of MM, we applied single-cell RNA-sequencing (scRNA-seq) to paired diagnostic and post-treatment bone marrow (BM) samples.


EXPERIMENTAL DESIGN
scRNA-seq was performed on 38 BM samples from patients with monoclonal gammopathy of undetermined significance (MGUS) (n = 1), MM patients at diagnosis (n = 19), MM post-treatment (n = 17), and one healthy donor. The single-cell transcriptome data of malignant plasma cells and the surrounding immune microenvironment were analyzed.


RESULTS
Profiling by scRNA-seq data revealed three primary trajectories of transcriptional evolution after treatment: clonal elimination in patients with undetectable minimal residual disease (MRD-), as well as clonal stabilization and clonal selection in detectable MRD (MRD+) patients. We noted a metabolic shift towards fatty acid oxidation in cycling-resistant plasma cells (PCs), while selective PCs favored the NF-κB pathway. Intriguingly, when comparing the genetic and transcriptional dynamics, we found a significant correlation between genetic and non-genetic factors in driving the clonal evolution. Furthermore, we identified variations in cellular interactions between malignant plasma cells and the tumor microenvironment (TME). Selective PCs showed the most robust cellular interactions with the TME.


CONCLUSIONS
These data suggest that MM cells could rapidly adapt to induction treatment through transcriptional adaptation, metabolic adaptation, and specialized immune evasion. Targeting therapy-induced resistance mechanisms may help to avert refractory disease in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4967fd77a9456d257d33b503e573eb358b554d" target='_blank'>
              Identification of therapy-induced clonal evolution and resistance pathways in minimal residual clones in multiple myeloma through single-cell sequencing.
              </a>
            </td>
          <td>
            Jian Cui, Xiaoyun Li, S. Deng, C. Du, H. Fan, W. Yan, Jingyu Xu, Xiaoqing Li, Tengteng Yu, Shuaishuai Zhang, R. Lv, W. Sui, Mu Hao, Xin Du, Yan Xu, S. Yi, D. Zou, Tao Cheng, Lugui Qiu, Xin Gao, Gang An
          </td>
          <td>2024-06-20</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02ae1a6e8ff74963e3bc365b3bbb12a975c7a9f" target='_blank'>
              Spatial profiling of non-small cell lung cancer provides insights into tumorigenesis and immunotherapy response
              </a>
            </td>
          <td>
            Joon Kim, S. Yong, Gyuho Jang, Yumin Kim, Raekil Park, Hyun-Hee Koh, Sehui Kim, Chang-Myung Oh, Sang Hoon Lee
          </td>
          <td>2024-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this study, we explore the possibility of inferring characteristics of the tumor-immune microenvironment (TIME) from the blood. Specifically, we investigate two datasets of head and neck squamous cell carcinoma (HNSCC) patients with matched scRNA-Seq from peripheral blood mononuclear cells (PBMCs) and tumor tissues. Our analysis shows that the immune cell fractions and gene expression profiles of various immune cells within the tumor microenvironment can be inferred from the matched PBMC scRNA-Seq data. We find that the established exhausted T-cell signature can be predicted from the blood and serve as a valuable prognostic blood biomarker of immunotherapy response. Additionally, our study reveals that the inferred ratio between tumor memory B and regulatory T cell fractions is predictive of immunotherapy response and is superior to the well-established cytolytic and exhausted T-cell signatures. These results highlight the promising potential of PBMC scRNA-Seq in cancer immunotherapy and warrant, and will hopefully facilitate, further investigations on a larger scale. The code for predicting tumor immune microenvironment from PBMC scRNA-Seq, TIMEP, is provided, offering other researchers the opportunity to investigate its prospective applications in various other indications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ade53f587544d91914fb929dd7efe4a6e7cd50" target='_blank'>
              Inferring characteristics of the tumor-immune microenvironment of HNSCC patients from single-cell transcriptomics of peripheral blood.
              </a>
            </td>
          <td>
            Yingying Cao, Tiangen Chang, Fiorella Schischlik, Kun Wang, Sanju Sinha, Sridhar Hannenhalli, Peng Jiang, E. Ruppin
          </td>
          <td>2024-08-08</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf3fb50dabab3c628e11b6f429b04114a9702d4" target='_blank'>
              Brain Chimeroids reveal individual susceptibility to neurotoxic triggers.
              </a>
            </td>
          <td>
            Noelia Antón-Bolaños, Irene Faravelli, Tyler Faits, Sophia Andreadis, Rahel Kastli, Sebastiano Trattaro, Xian Adiconis, Anqi Wei, Abhishek Sampath Kumar, Daniela J. Di Bella, M. Tegtmeyer, R. Nehme, J. Levin, Aviv Regev, Paola Arlotta
          </td>
          <td>2024-06-26</td>
          <td>Nature</td>
          <td>5</td>
          <td>58</td>
        </tr>

        <tr id="Cancer cells show remarkable plasticity and can switch lineages in response to the tumor microenvironment. Cellular plasticity drives invasiveness and metastasis and helps cancer cells to evade therapy by developing resistance to radiation and cytotoxic chemotherapy. Increased understanding of cell fate determination through epigenetic reprogramming is critical to discover how cancer cells achieve transcriptomic and phenotypic plasticity. Glioblastoma is a perfect example of cancer evolution where cells retain an inherent level of plasticity through activation or maintenance of progenitor developmental programs. However, the principles governing epigenetic drivers of cellular plasticity in glioblastoma remain poorly understood. Here, using machine learning (ML) we employ cross-patient prediction of transcript expression using a combination of epigenetic features (ATAC-seq, CTCF ChIP-seq, RNAPII ChIP-seq, H3K27Ac ChIP-seq, and RNA-seq) of glioblastoma stem cells (GSCs). We investigate different ML and deep learning (DL) models for this task and build our final pipeline using XGBoost. The model trained on one patient generalizes to another one suggesting that the epigenetic signals governing gene transcription are consistent across patients even if GSCs can be very different. We demonstrate that H3K27Ac is the epigenetic feature providing the most significant contribution to cross-patient prediction of gene expression. In addition, using H3K27Ac signals from patients-derived GSCs, we can predict gene expression of human neural crest stem cells suggesting a shared developmental epigenetic trajectory between subpopulations of these malignant and benign stem cells. Our cross-patient ML/DL models determine weighted patterns of influence of epigenetic marks on gene expression across patients with glioblastoma and between GSCs and neural crest stem cells. We propose that broader application of this analysis could reshape our view of glioblastoma tumor evolution and inform the design of new epigenetic targeting therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f06de9705bc503e591aae9a45d9fab127b04d79" target='_blank'>
              Machine learning on multiple epigenetic features reveals H3K27Ac as a driver of gene expression prediction across patients with glioblastoma
              </a>
            </td>
          <td>
            Yusuke Suita, Hardy Bright, Yuan Pu, M. D. Toruner, Jordan Idehen, Nikos Tapinos, Ritambhara Singh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65ae438e85d1f5a8bcba96886546fdec069966a7" target='_blank'>
              IRF8 deficiency-induced myeloid-derived suppressor cell promote immune evasion in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zhen-Yang Gao, Shang Liu, Han Xiao, Mengdi Li, Wan-gang Ren, Lin Xu, Zhong-Min Peng
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a dynamic population of tumor cells characterized by long-term self-renewal, high tumorigenicity, resistance to conventional therapies such as radio- and chemotherapy, and capacity to recapitulate the tumor heterogeneity. Similar to other tumor cells, CSCs need to carry critical mutations and epigenetic changes to acquire their aberrant phenotype. Confirmed in various hematologic and solid malignancies, the critical need to deepen our understanding of CSC biology, including identification of CSC biomarkers, and develop novel CSC-targeted therapies has been clearly recognized. Here, we review the L1 cell adhesion molecule (L1CAM) as a CSC-associated biomarker in ovarian cancer. Furthermore, we inform on the promising potential of anti-L1CAM radioimmunotherapy with 161Tb as a novel CSC-targeted therapeutic approach to overcome CSC radioresistance in comparison to 177Lu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd80ab2d2b7a4a88dde09b0705beb033af954a4" target='_blank'>
              Targeting Cancer Stem Cells with Radioimmunotherapy: The Case of the Ovarian Cancer Stemness-Associated Biomarker L1CAM
              </a>
            </td>
          <td>
            Tihomir Zh. Todorov, R. Schibli, Martin Béhé, Jürgen Grünberg
          </td>
          <td>2024-07-18</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The global incidence of cancer is increasing, with estimates suggesting that there will be 26 million new cases and 17 million deaths per year by 2030. Cancer stem cells (CSCs) and extracellular vesicles (EVs) are key to the resistance and advancement of cancer. They play a crucial role in tumor dynamics and resistance to therapy. CSCs, initially discovered in acute myeloid leukemia, are well-known for their involvement in tumor initiation, progression, and relapse, mostly because of their distinct characteristics, such as resistance to drugs and the ability to self-renew. EVs, which include exosomes, microvesicles, and apoptotic bodies, play a vital role in facilitating communication between cells within the tumor microenvironment (TME). They have a significant impact on cellular behaviors and contribute to genetic and epigenetic changes. This paper analyzes the mutually beneficial association between CSCs and EVs, emphasizing their role in promoting tumor spread and developing resistance mechanisms. This review aims to investigate the interaction between these entities in order to discover new approaches for attacking the complex machinery of cancer cells. It highlights the significance of CSCs and EVs as crucial targets in the advancement of novel cancer treatments, which helps stimulate additional research, promote progress in ideas for cancer treatment, and provide renewed optimism in the effort to reduce the burden of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b17e9728aa76af6a02ab4a44623aab4c0608b6" target='_blank'>
              Extracellular vesicles and cancer stem cells: a deadly duo in tumor progression
              </a>
            </td>
          <td>
            Akram Tayanloo-Beik, Azin Eslami, Masoumeh Sarvari, H. Jalaeikhoo, Mohsen Rajaeinejad, Mohsen Nikandish, Ali Faridfar, Mostafa Rezaei-Tavirani, A. R. Mafi, Bagher Larijani, B. Arjmand
          </td>
          <td>2024-07-18</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b564fa77c9ecb886651a32d892f7a470d9fe2e51" target='_blank'>
              Reconstructed colorectal cancer model to dissect the anti-tumor effect of mesenchymal stromal cells derived extracellular vesicles
              </a>
            </td>
          <td>
            E. D’Angelo, Sarah Tassinari, Andrea Biccari, S. Crotti, Francesca Sensi, Asia Marangio, Ombretta Repetto, Giuseppe Corona, Linda Bellucci, Federica Antico, Federico Caicci, G. Spolverato, Giovanni Montini, Benedetta Bussolati, Marco Agostini, Federica Collino
          </td>
          <td>2024-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 What are the molecular and cellular features that contribute to the transformation of myometrial stem cells into tumor-initiating cells within uterine leiomyoma?



 Transformative changes in differentiation and genetic processes, in a distinct subset of M/LM cells drives their progression into a tumorigenic state.



 Uterine leiomyomas (LM), which are benign smooth muscle-derived tumours of myometrium, impact approximately 75% of women, causing significant physical and psychological challenges and imposing substantial healthcare costs. The widespread consensus postulates a monoclonal origin for LM, implying their derivation from a dysregulated single multipotent stem cell that could give rise to tumor-initiating cells (TICs). While previous studies have shown cellular heterogeneity in both myometrium (M) and LM, the precise identity of the originating cells is unknown.



 Prospective, observational, and biomedical study conducted at Hospital La Fe (Valencia, Spain) for one year. Eight sample pairs of LM and M underwent single-nuclei RNAseq (snRNA-seq; n = 16) and single-cell proteomic (scP; n = 16) analyses, to generate a detailed transcriptomic and proteomic map decoupled from cell type, state, and spatial location. We further employed single-cell RNA velocity inference for differentiation trajectory [BR1] analysis of mesenchymal cells, followed by the identification of potential driver genes in LM tumorigenesis.



 Upon obtaining informed consent, we procured LM and M samples from eight patients aged 35-50 undergoing hysterectomies. Part of these samples were preserved in paraffin for spatial transcriptomics utilizing VISIUM (10x Genomics). The remaining tissues were dissociated into single-cell suspensions and underwent scRNA-seq and scP using Chromium Controller and Orbitrap Eclipse Tribid mass spectrometry, respectively. The analysis of the collected data was carried out using publicly accessible R/Python tools.



 Analysis of snRNAseq (∼52,000 cells) revealed similar cellular compositions in M and LM, including endothelial, perivascular, smooth muscle cells (SMC), fibroblasts, and immune cells. Further trajectory analysis unveiled a subset of myometrial cells with potential as TICs, exhibiting reduced expression of hormone receptors like PGR. Analysis of these cells [BR1] suggested differentiation into two distinct cell types -fibroblasts and SMCs- with elevated hormone expression over pseudotime. Differential analysis between M and LM also showed a dysregulated transcriptomic profile in the TICs within the tumor, as evidenced by upregulation of RAD51B and HMGA2 and downregulation PGR among other genes. These genes are indicative markers, underscoring their significant roles in the tumorigenesis of LM by influencing hormone response and disrupting DNA repair mechanisms. Spatial transcriptomic analysis confirmed these findings and suggested that the distribution of TICs could be ubiquitous, since the expression of potential myometrial-stem markers, was markedly increased only on certain spots within the tissue across both LM and M. Lastly, scProt profiling (∼5,000 cells) also allowed the identification of putative TICs, which showed upregulation in LM of specific proteins involved in replication, transcription, and translation.



 Our study sets the basis for TICs isolation in LM, yet functional validation studies need to be performed to address the clinical potential of our findings. Further studies including more patients, and addressing racial disparities will help to generalize these findings to a broader population.



 Our findings indicate the presence of a particular subset of M/LM cells whose changes in differentiation path and genetic processes seem to drive the transformation of these cells into a tumorigenic state to develop LM. Targeting these markers might show promise for effectively treating these tumors.



 NCT04214457
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e5f1c7a81c7b6baceb7a49d4c94f2c7bde8e8c" target='_blank'>
              O-264 Profiling Tumor-Initiating Cells in Uterine Leiomyoma through Single-Cell Multi-omics
              </a>
            </td>
          <td>
            A. Machado-Lopez, P. Punzón-Jiménez, J. Llera-Oyola, R. Perez-Moraga, M. Gálvez-Viedma, D. Grases, E. Satorres, E. Porta-Pardo, E. Schooff, B. Roson, C. Simon, A. Mas
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
Classic Hodgkin lymphoma (cHL) is a B cell lymphoma that occurs primarily in young adults and, less frequently, in elderly individuals. A hallmark of cHL is the exceptional scarcity (1-5%) of the malignant Hodgkin Reed-Sternberg (HRS) cells within a network of non-malignant immune cells. Molecular determinants governing the relationship between HRS cells and their proximal microenvironment remain largely unknown.


EXPERIMENTAL DESIGN
We performed spatially resolved multiplexed protein imaging and transcriptomic sequencing to characterize HRS cell states, cellular neighborhoods, and gene expression signatures of 23.6 million cells from 36 newly diagnosed Epstein-Barr virus (EBV) positive and EBV-negative cHL tumors.


RESULTS
We show that MHC-I expression on HRS cells is associated with immune inflamed neighborhoods containing CD8+ T cells, MHC-II+ macrophages, and immune checkpoint expression (i.e., PD-1 and VISTA). We identified spatial clustering of HRS cells, consistent with the syncytial variant of cHL, and its association with T cell excluded neighborhoods in a subset of EBV-negative tumors. Finally, a subset of both EBV-positive and EBV-negative tumors contained regulatory T cells high neighborhoods harboring HRS cells with augmented proliferative capacity.


CONCLUSIONS
Our study links HRS cell properties with distinct immunophenotypes and potential immune escape mechanisms in cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/645b8738be3b7a5dce2bff2635b7f7c7780b894d" target='_blank'>
              Multiplexed spatial profiling of Hodgkin Reed-Sternberg cell neighborhoods in classic Hodgkin lymphoma.
              </a>
            </td>
          <td>
            Maryam Pourmaleki, Caitlin J Jones, Sabrina D Mellinghoff, Bryan D. Greenstein, Priyadarshini Kumar, Miguel Foronda, Daniel A Navarrete, Carl Campos, M. Roshal, N. Schultz, Sohrab P. Shah, A. Schietinger, N. Socci, Travis J Hollmann, A. Dogan, I. Mellinghoff
          </td>
          <td>2024-07-01</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db39575c0e33ee4fe58dec1930d2f78980fd1497" target='_blank'>
              Angiopoietin-Like Protein 2 Expression in Tumor Cells Supports Tumor-Associated Macrophage-Induced Tumor Progression in Esophageal Cancer.
              </a>
            </td>
          <td>
            Taichi Horino, Haruki Horiguchi, Shinsei Yumoto, T. Kadomatsu, Yoshihiro Hara, T. Yagi, Y. Baba, Yuji Miyamoto, Hideo Baba, Yuichi Oike
          </td>
          <td>2024-07-09</td>
          <td>Annals of surgical oncology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Basal cell carcinoma (BCC) is the most frequently diagnosed skin cancer and the most common malignancy in humans. Different morphological subtypes of BCC are associated with low- or high-risk of recurrence and aggressiveness, but the underlying biology of how the individual subtypes arise remains largely unknown. Because the majority of BCCs appear to arise from mutations in the same pathway, we hypothesized that BCC development, growth and invasive potential is also influenced by the tumor microenvironment and in particular by cancer-associated fibroblasts (CAFs) and their secreted factors.


OBJECTIVE
We aimed to characterize the stroma of the different BCC subtypes with a focus on CAF populations.


METHODS
To investigate the stromal features of the different BCC subtypes, we applied laser-capture microdissection (LCM) followed by RNA sequencing. A cohort of 15 BCC samples from 5 different "pure" subtypes (superficial, nodular, micronodular, sclerosing and basosquamous; n=3 each) were selected and included in the analysis. Healthy skin was used as a control (n=6). We confirmed the results by immunohistochemistry. We validated our findings in two independent, public single-cell RNA sequencing (scRNAseq) datasets and by RNAscope.


RESULTS
The stroma of the different BCC subtypes have distinct gene expression signatures. Nodular and micronodular seem to have the most similar signatures, while superficial and sclerosing the most different. By comparing low- and high-risk BCC subtypes, we observed that Collagen 10A1 (COL10A1) is overexpressed in the stroma of sclerosing/infiltrative and basosquamous but not micronodular high-risk subtypes. Those findings were confirmed by immunohistochemistry in a cohort of 89 different BCC and 13 healthy skin samples. Moreover, scRNAseq analysis of BCCs of two independent datasets showed that the COL10A1-expressing population of cells is associated with the stroma adjacent to invasive BCC and shows extracellular matrix remodeling features.


CONCLUSION
We identified COL10A1 as a marker of high-risk BCC, in particular of the sclerosing/infiltrative and basosquamous subtypes. We demonstrated at the single cell level that COL10A1 is expressed by a specific CAF population associated with the stroma of invasive BCC. This opens up new tailored treatment options as well as a new prognostic biomarker for BCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5131e4d68fffbdfca6cced052102c01223f2d2d9" target='_blank'>
              COL10A1 expression distinguishes a subset of cancer-associated fibroblasts present in the stroma of high-risk basal cell carcinoma.
              </a>
            </td>
          <td>
            Mauro Esposito, L. Yerly, Prachi Shukla, Victoria Hermes, Federica Sella, Zsolt Balázs, Evelyn Lattmann, A. Tastanova, P. Turko, Ramon Lang, I. Kolm, Ramon Staeger, François Kuonen, Michael Krauthammer, J. Hafner, Mitchell P Levesque, G. Restivo
          </td>
          <td>2024-06-25</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="A distinct feature of pancreatic ductal adenocarcinoma (PDAC) is a prominent tumor microenvironment (TME) with remarkable cellular and spatial heterogeneity that meaningfully impacts disease biology and treatment resistance. The dynamic crosstalk between cancer cells and the dense stromal compartment leads to spatially and temporally heterogeneous metabolic alterations, such as acidic pH that contributes to drug resistance in PDAC. Thus, monitoring the extracellular pH metabolic fluctuations within the TME is crucial to predict and to quantify anticancer drug efficacy. Here, a simple and reliable alginate-based 3D PDAC model embedding ratiometric optical pH sensors and cocultures of tumor (AsPC-1) and stromal cells for simultaneously monitoring metabolic pH variations and quantify drug response is presented. By means of time-lapse confocal laser scanning microscopy (CLSM) coupled with a fully automated computational analysis, the extracellular pH metabolic variations are monitored and quantified over time during drug testing with gemcitabine, folfirinox, and paclitaxel, commonly used in PDAC therapy. In particular, the extracellular acidification is more pronounced after drugs treatment, resulting in increased antitumor effect correlated with apoptotic cell death. These findings highlight the importance of studying the influence of cellular metabolic mechanisms on tumor response to therapy in 3D tumor models, this being crucial for the development of personalized medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dceab42d7b7c7a1f2eb735efea89f11000f608" target='_blank'>
              A 3D Pancreatic Cancer Model with Integrated Optical Sensors for Noninvasive Metabolism Monitoring and Drug Screening.
              </a>
            </td>
          <td>
            Anna Chiara Siciliano, S. Forciniti, V. Onesto, Helena Iuele, Donatella Delle Cave, Federica Carnevali, Giuseppe Gigli, E. Lonardo, L. L. del Mercato
          </td>
          <td>2024-07-08</td>
          <td>Advanced healthcare materials</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac24e0e7a39e90599e20a198363cbf479bb3749d" target='_blank'>
              Using a pan-cancer atlas to investigate tumour associated macrophages as regulators of immunotherapy response
              </a>
            </td>
          <td>
            Alexander Coulton, Jun Murai, Danwen Qian, Krupa Thakkar, Claire E Lewis, Kevin Litchfield
          </td>
          <td>2024-07-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35eea075db8a8c1977ce6381cea9a663e6162ef" target='_blank'>
              PTGR1-mediated immune evasion mechanisms in late-stage triple-negative breast cancer: mechanisms of M2 macrophage infiltration and CD8+ T cell suppression.
              </a>
            </td>
          <td>
            Fang Huang, Fuhe Wang, Qilu Hu, Ying Li, Da Jiang
          </td>
          <td>2024-07-28</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are small RNAs that are often dysregulated in many diseases, including cancers. They are highly tissue-specific and stable, thus, making them particularly useful as biomarkers. As the spatial transcriptomics field advances, protocols that enable highly sensitive and spatially resolved detection become necessary to maximize the information gained from samples. This is especially true of miRNAs where the location their expression within tissue can provide prognostic value with regard to patient outcome. Equally as important as detection are ways to assess and visualize the miRNA's spatial information in order to leverage the power of spatial transcriptomics over that of traditional nonspatial bulk assays. We present a highly sensitive methodology that simultaneously quantitates and spatially detects seven miRNAs in situ on formalin-fixed paraffin-embedded tissue sections. This method utilizes rolling circle amplification (RCA) in conjunction with a dual scanning approach in nanoliter well arrays with embedded hydrogel posts. The hydrogel posts are functionalized with DNA probes that enable the detection of miRNAs across a large dynamic range (4 orders of magnitude) and a limit of detection of 0.17 zeptomoles (1.7 × 10-4 attomoles). We applied our methodology coupled with a data analysis pipeline to K14-Cre Brca1f/fTp53f/f murine breast tumors to showcase the information gained from this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbaad333d7c14fb2504c61497f5a898eaa1ab830" target='_blank'>
              Sensitive Multiplexed MicroRNA Spatial Profiling and Data Classification Framework Applied to Murine Breast Tumors.
              </a>
            </td>
          <td>
            Omar N Mohd, Y. Heng, Lin Wang, Abhishek Thavamani, Erica S. Massicott, Gerburg M Wulf, Frank J Slack, Patrick S. Doyle
          </td>
          <td>2024-07-23</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neuroblastoma is a childhood developmental cancer; however, its embryonic origins remain poorly understood. Moreover, in-depth studies of early tumor-driving events are limited because of the lack of appropriate models. Herein, we analyzed RNA sequencing data obtained from human neuroblastoma samples and found that loss of expression of trunk neural crest–enriched gene MOXD1 associates with advanced disease and worse outcome. Further, by using single-cell RNA sequencing data of human neuroblastoma cells and fetal adrenal glands and creating in vivo models of zebrafish, chick, and mouse, we show that MOXD1 is a determinate of tumor development. In addition, we found that MOXD1 expression is highly conserved and restricted to mesenchymal neuroblastoma cells and Schwann cell precursors during healthy development. Our findings identify MOXD1 as a lineage-restricted tumor-suppressor gene in neuroblastoma, potentiating further stratification of these tumors and development of novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4a1e60d743c1788421ba02de583dc1b69f419e" target='_blank'>
              MOXD1 is a lineage-specific gene and a tumor suppressor in neuroblastoma
              </a>
            </td>
          <td>
            Elina Fredlund, Stina Andersson, Elien Hilgert, Ezequiel Monferrer, Guadalupe Álvarez-Hernán, Sinan Karakaya, Siebe Loontiens, J. Bek, Tomas Gregor, Estelle Lecomte, Emma Magnusson, Enrika Miltenyte, Marie Cabirol, Michail Kyknas, Niklas Engström, Marie Arsenian Henriksson, Emma Hammarlund, J. Rosenblum, Rosa Noguera, Frank Speleman, J. van Nes, S. Mohlin
          </td>
          <td>2024-06-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Targeted imaging of cancer lymphatic metastasis remains challenging due to its highly heterogeneous molecular and phenotypic diversity. Herein, we introduced triple-targeted protein nanoprobes capable of specifically binding to three targets for imaging cancer lymphatic metastasis, through a data-driven design approach combined with a synthetic biology-based assembly strategy. Specifically, to address the diversity of metastatic lymph nodes (LNs), a combination of three targets, including C-X-C motif chemokine receptor 4 (CXCR4), transferrin receptor protein 1 (TfR1) and vascular endothelial growth factor receptor 3 (VEGFR3) was identified, leveraging machine leaning-based bioinformatics analysis and examination of lymphatic metastatic tissues from patients with gastric cancer. Using this identified target combination, ferritin nanocage-based nanoprobes capable of specifically binding to all three targets were designed through the self-assembly of genetically engineered ferritin subunits using a synthetic biology approach. Multiplexed imaging of heterogeneous metastatic LNs using these nanoprobes resulted in approximately 80% binding with heterogeneous tumor cells, which was 60% higher than that of single-targeted nanoprobes in a polyclonal lymphatic metastasis animal model. In 19 freshly resected human gastric specimens, the signal from the triple-targeted nanoprobes significantly differentiated metastatic LNs from benign LNs, in high accordance with pathological examination. This study not only provides an effective nanoprobe for imaging highly heterogeneous lymphatic metastasis but also proposes a potential strategy for guiding the design of targeted nanomedicines for cancer lymphatic metastasis. This article is protected by copyright. All rights reserved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/595024240b444e89cf21627f1500404247fe4401" target='_blank'>
              Data-Driven Design of Triple-Targeted Protein Nanoprobes for Multiplexed Imaging of Cancer Lymphatic Metastasis.
              </a>
            </td>
          <td>
            Guodong Shen, Xiao-hua Jia, Tianyi Qi, Zhenhua Hu, Anqi Xiao, Qiqi Liu, Keyu He, Weihong Guo, Dan Zhang, Wanjun Li, Genmao Cao, Guoxin Li, Jie Tian, Xinglu Huang, Yanfeng Hu
          </td>
          <td>2024-06-18</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9c5b5959b41a250a5ef58adb37088e34a07297" target='_blank'>
              Immuno-oncologic profiling of pediatric brain tumors reveals major clinical significance of the tumor immune microenvironment
              </a>
            </td>
          <td>
            Adrian B Levine, L. Nobre, Anirban Das, Scott Milos, V. Bianchi, Monique Johnson, Nick Fernandez, L. Stengs, S. Ryall, Michelle Ku, Mansuba Rana, Benjamin Laxer, J. Sheth, Stefanie-Grace Sbergio, Ivana Fedorakova, Vijay Ramaswamy, J. Bennett, R. Siddaway, U. Tabori, Cynthia Hawkins
          </td>
          <td>2024-07-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Motivation Cell–cell interactions (CCIs) consist of cells exchanging signals with themselves and neighboring cells by expressing ligand and receptor molecules and play a key role in cellular development, tissue homeostasis, and other critical biological functions. Since direct measurement of CCIs is challenging, multiple methods have been developed to infer CCIs by quantifying correlations between the gene expression of the ligands and receptors that mediate CCIs, originally from bulk RNA-sequencing data and more recently from single-cell or spatially resolved transcriptomics (SRT) data. SRT has a particular advantage over single-cell approaches, since ligand–receptor correlations can be computed between cells or spots that are physically close in the tissue. However, the transcript counts of individual ligands and receptors in SRT data are generally low, complicating the inference of CCIs from expression correlations. Results We introduce Copulacci, a count-based model for inferring CCIs from SRT data. Copulacci uses a Gaussian copula to model dependencies between the expression of ligands and receptors from nearby spatial locations even when the transcript counts are low. On simulated data, Copulacci outperforms existing CCI inference methods based on the standard Spearman and Pearson correlation coefficients. Using several real SRT datasets, we show that Copulacci discovers biologically meaningful ligand–receptor interactions that are lowly expressed and undiscoverable by existing CCI inference methods. Availability and implementation Copulacci is implemented in Python and available at https://github.com/raphael-group/copulacci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7018d88732147a5c725f1a06701187686f85ae3c" target='_blank'>
              A count-based model for delineating cell–cell interactions in spatial transcriptomics data
              </a>
            </td>
          <td>
            Hirak Sarkar, Uthsav Chitra, Julian Gold, Benjamin J. Raphael
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, V. Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Hereditary Hemorrhagic Telangiectasia (HHT) is a rare congenital disease in which fragile vascular malformations (VM) – including small telangiectasias and large arteriovenous malformations (AVMs) – focally develop in multiple organs. There are few treatment options and no cure for HHT. Most HHT patients are heterozygous for loss-of-function mutations affecting Endoglin (ENG) or Alk1 (ACVRL1); however, why loss of these genes manifests as VMs remains poorly understood. To complement ongoing work in animal models, we have developed a fully human, cell-based microphysiological model based on our Vascularized Micro-organ (VMO) platform (the HHT-VMO) that recapitulates HHT patient VMs. Using inducible ACVRL1 -knockdown, we control timing and extent of endogenous Alk1 expression in primary human endothelial cells (EC). Resulting HHT-VMO VMs develop over several days. Interestingly, in chimera experiments AVM-like lesions can be comprised of both Alk1-intact and Alk1-deficient EC, suggesting possible cell non-autonomous effects. Single cell RNA sequencing data are consistent with microvessel pruning/regression as contributing to AVM formation, while loss of PDGFB implicates mural cell recruitment. Finally, lesion formation is blocked by the VEGFR inhibitor pazopanib, mirroring positive effects of this drug in patients. In summary, we have developed a novel HHT-on-a-chip model that faithfully reproduces HHT patient lesions and that can be used to better understand HHT disease biology and identify potential new HHT drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08fd425d062824896a0b14bbeff3a330e5c1bcbd" target='_blank'>
              A Microphysiological HHT-on-a-Chip Platform Recapitulates Patient Vascular Lesions
              </a>
            </td>
          <td>
            Christopher C W Hughes, Jennifer S. Fang, Christopher J. Hatch, Jillian W. Andrejecsk, William Van Trigt, Damie J Juat, Yu-Hsi Chen, Satomi Matsumoto, Abraham P. Lee
          </td>
          <td>2024-06-18</td>
          <td>Research Square</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Most melanomas progress from radial to vertical growth phase before spreading locoregionally and distally. Much is still unknown about the metabolic changes in the tumor cells and their microenvironment during this metastatic progression. We aimed to gain new insight into the molecular characteristics of melanoma in regard to spatial lipidomics to deliver new knowledge regarding tumor metastatic progression. We included 10 fresh tumor samples from 10 patients including two in situ melanomas, two invasive primary melanomas, and six metastatic melanomas (four in-transit metastases and two distant metastases). In addition, we analyzed four healthy skin controls from the same patients. Time-of-flight imaging secondary ion mass spectrometry (ToF-SIMS) enabled detailed spatial-lipidomics that could be directly correlated with conventional histopathological analysis of consecutive H&E-stained tissue sections. Significant differences in the lipid profiles were found in primary compared to metastatic melanomas, notably an increase in phosphatidylethanolamine lipids relative to phosphatidylinositol lipids and an increase in GM3 gangliosides in the metastatic samples. Furthermore, analysis of the data from in transit versus distant metastases samples highlighted that specific phospholipids, and a difference in the long versus shorter chain GM3 gangliosides, discriminated the metastatic routes. Further studies are warranted to verify these preliminary findings. Lipidomic changes could serve as a novel biomarker for tumor progression and even serve as a target for novel treatments. Furthermore, analyzing the lipid profiles could help to differentiate between primary and metastatic melanomas in challenging cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dd5da01fb93ce9422a464070818f23116949e9f" target='_blank'>
              ToF-SIMS imaging reveals changes in tumor cell lipids during metastatic progression of melanoma.
              </a>
            </td>
          <td>
            N. Neittaanmäki, O. Zaar, Kevin Sjögren Cehajic, Kelly Dimovska Nilsson, D. Katsarelias, R. O. Bagge, John Paoli, John S Fletcher
          </td>
          <td>2024-06-29</td>
          <td>Pigment cell & melanoma research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Bispecific antibodies (bsAbs) that bind to two distinct surface antigens on cancer cells are emerging as an appealing therapeutic strategy in cancer immunotherapy. However, considering the vast number of surface proteins, experimental identification of potential antigen pairs that are selectively expressed in cancer cells and not in normal cells is both costly and time-consuming. Recent studies have utilized large bulk RNA-seq databases to propose bispecific targets for various cancers. However, co-expressed pairs derived from bulk RNA-seq do not necessarily indicate true co-expression of both markers in the malignant cell. Single-cell RNA-seq (scRNA-seq) can circumvent this issue but the issues in low-coverage of transcripts impede the large-scale characterization of co-expressed pairs.



 We present a computational pipeline for bsAbs target identification which combines the advantages of bulk and scRNA-seq while minimizing the issues associated with using these approaches separately. We select Hepatocellular Carcinoma (HCC) as a case study to demonstrate the utility of the approach. First, using the bulk RNA-seq samples in the OCTAD database, we identified target pairs that most distinctly differentiate tumor cases from healthy controls. Next, we confirmed our findings on the scRNA-seq database comprising 39,361 healthy cells from vital organs and 18,000 cells from HCC. The top pair was GPC3∼MUC13, where both genes are co-expressed on the surface of over 30% of malignant HCCs and have very low expression in other cells. Finally, we leveraged the emerging spatial transcriptomic to validate the co-expressed pair in situ.



 A standalone R package (https://github.com/Bin-Chen-Lab/bsAbsFinder)



 Supplementary data are available at Bioinformatics Advances online
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2059ebaa34bf1fd7c83bd94cb6ade0525ae3296b" target='_blank'>
              Computational discovery of co-expressed antigens as dual targeting candidates for cancer therapy through bulk, single-cell, and spatial transcriptomics
              </a>
            </td>
          <td>
            E. Chekalin, Shreya Paithankar, Rama Shankar, Jing Xing, Wenfeng Xu, Bin Chen
          </td>
          <td>2024-06-20</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf72209777e452dc93a39bae0d7ff7fff3f49ae6" target='_blank'>
              Effects of tumor treating fields (TTFields) on human mesenchymal stromal cells.
              </a>
            </td>
          <td>
            M. Strack, Jan Kückelhaus, M. Diebold, Patrick Wuchter, Peter E Huber, Oliver Schnell, R. Sankowski, Marco Prinz, Anca-Ligia Grosu, D. Heiland, N. H. Nicolay, A. Rühle
          </td>
          <td>2024-06-20</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bf603a0e9cde601ac9006c72d111f317f71de81" target='_blank'>
              Single-cell profiling reveals altered immune landscape and impaired NK cell function in gastric cancer liver metastasis.
              </a>
            </td>
          <td>
            Xiaolong Tang, Lei Gao, Xingzhi Jiang, Zhenyu Hou, Yiwen Wang, Shiyang Hou, Hui Qu
          </td>
          <td>2024-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PURPOSE
Co-occurring mutations in KEAP1 and STK11KRAS have emerged as determinants of survival outcomes in non-small cell lung cancer (NSCLC) patients treated with immunotherapy. However, these mutational contexts identify a fraction of non-responders to immune checkpoint inhibitors. We hypothesized that KEAP1 wild-type tumors recapitulate the transcriptional footprint of KEAP1 mutations, and that this KEAPness phenotype can determine immune responsiveness with higher precision compared to mutation-based models.


EXPERIMENTAL DESIGN
The TCGA was used to infer the KEAPness phenotype and explore its immunological correlates at the pan-cancer level. The association between KEAPness and survival outcomes was tested in two independent cohorts of advanced NSCLC patients treated with immunotherapy and profiled by RNA-Seq (SU2C n=153; OAK/POPLAR n=439). The NSCLC TRACERx421 multi-region sequencing study (tumor regions n=947) was used to investigate evolutionary trajectories.


RESULTS
KEAPness-dominant tumors represented 50% of all NSCLCs and were associated with shorter progression-free survival (PFS) and overall survival (OS) compared to KEAPness-free cases in independent cohorts of NSCLC patients treated with immunotherapy (SU2C PFS P=0.042, OS P=0.008; OAK/POPLAR PFS P=0.0014, OS P<0.001). Patients with KEAPness tumors had survival outcomes comparable to those with KEAP1-mutant tumors. In the TRACERx421, KEAPness exhibited limited transcriptional intratumoral heterogeneity and immune exclusion, resembling the KEAP1-mutant disease. This phenotypic state occurred across genetically divergent tumors, exhibiting shared and private cancer genes under positive selection when compared to KEAP1-mutant tumors.


CONCLUSIONS
We identified a KEAPness phenotype across evolutionary divergent tumors. KEAPness outperforms mutation-based classifiers as a biomarker of inferior survival outcomes in NSCLC patients treated with immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea8af08e6c600cf8a24f58a7981fb66121f005e" target='_blank'>
              Transcriptional phenocopies of deleterious KEAP1 mutations correlate with survival outcomes in lung cancer treated with immunotherapy.
              </a>
            </td>
          <td>
            S. Scalera, B. Ricciuti, D. Marinelli, M. Mazzotta, Laura Cipriani, Giulia Bon, Giulia Schiavoni, I. Terrenato, Alessandro Di Federico, J. Alessi, Maurizio Fanciulli, L. Ciuffreda, F. de Nicola, F. Goeman, G. Caravagna, Daniele Santini, R. De Maria, F. Cappuzzo, G. Ciliberto, M. Jamal-Hanjani, Mark M Awad, N. Mcgranahan, M. Maugeri-Saccà
          </td>
          <td>2024-07-09</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, Chin L Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, Michael Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly invasive, aggressive brain cancer that carries a median survival of 15 months and is resistant to standard therapeutics. Recent studies have demonstrated that intratumoral heterogeneity plays a critical role in promoting resistance by mediating tumor adaptation through microenvironmental cues. GBM can be separated into two distinct regions—a core and a rim, which are thought to drive specific aspects of tumor evolution. These differences in tumor progression are regulated by the diverse biomolecular and biophysical signals in these regions, but the acellular biophysical characteristics remain poorly described. This study investigates the mechanical and ultrastructural characteristics of the tumor extracellular matrix (ECM) in patient-matched GBM core and rim tissues. Seven patient-matched tumor core and rim samples and one non-neoplastic control were analyzed using atomic force microscopy, scanning electron microscopy, and immunofluorescence imaging to quantify mechanical, ultrastructural, and ECM composition changes. The results reveal significant differences in biophysical parameters between GBM core, rim, and non-neoplastic tissues. The GBM core is stiffer, denser, and is rich in ECM proteins hyaluronic acid and tenascin-C when compared to tumor rim and non-neoplastic tissues. These alterations are intimately related and have prognostic effect with stiff, dense tissue correlating with longer progression-free survival. These findings reveal new insights into the spatial heterogeneity of biophysical parameters in the GBM tumor microenvironment and identify a set of characteristics that may correlate with patient prognosis. In the long term, these characteristics may aid in the development of strategies to combat therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b5313dd86c28916e3c6cd3d13c84d5b7be07567" target='_blank'>
              The prognostic effect of mechanical, ultrastructural, and ECM signatures in glioblastoma core and rim
              </a>
            </td>
          <td>
            Bradley J. Mahaffey, Zachary P. Fowler, Zoe Lung, Vivien Dang, Hyunchul Lee, Allison McKenzie Johnson, Marco A. Munoz, Dylan A. Goodin, Hermann Frieboes, Brian J. Williams, Joseph Chen
          </td>
          <td>2024-06-24</td>
          <td>APL Bioengineering</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common malignancy of the central nervous system in adults. The current standard of care includes surgery, radiation therapy, temozolomide; and tumor-treating fields leads to dismal overall survival. There are far limited treatments upon recurrence. Therapies to date are ineffective as a result of several factors, including the presence of the blood-brain barrier, blood tumor barrier, glioma stem-like cells and genetic heterogeneity in GBM. In the present review, the potential mechanisms that lead to treatment resistance in GBM and the measures which have been taken so far to attempt to overcome the resistance were discussed. The complex biology of GBM and lack of comprehensive understanding of the development of therapeutic resistance in GBM demands discovery of novel antigens that are targetable and provide effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3693326221de1b300389f5e1e34c85fa1b817b7d" target='_blank'>
              Targeting drug resistance in glioblastoma (Review)
              </a>
            </td>
          <td>
            Jonathan H Sherman, Adam Bobak, T. Arsiwala, P. Lockman, Sonikpreet Aulakh
          </td>
          <td>2024-07-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative apocrine breast carcinoma (TNAC) is a rare type of triple-negative breast cancer (TNBC) characterized by apocrine morphology. Due to its low clinical incidence, TNAC has not been well studied at the molecular level in terms of cellular mechanisms of tumorigenesis and treatment response. In this study, we collected formalin-fixed paraffin-embedded samples from 31 patients diagnosed with invasive TNAC. Mass spectrometry analysis and whole-exome sequencing were performed to systematically construct the proteome and mutation landscape of the TNAC. Comparing ductal carcinoma in situ (DCIS), invasive, and adjacent normal tissues, we observed increased cadherin binding and ligase activity during tumorigenesis, heightened extracellular matrix signaling at tumor initiation, and elevated GTPase activity during tumor progression. Besides, we discovered genes such as C3 and COL18A1 enriched in somatic mutations are also significantly dysregulated in invasive TNAC. Additionally, by assessing biopsies before neo-adjacent chemotherapy, we provided molecular evidence supporting clinical observations of chemotherapy insensitivity in TNAC patients. We further identified a protein panel (CAPN1, CORO1B, HK1, RAP1B, and VPS4B) differentiating TNAC from non-apocrine TNBC, which could potentially serve as diagnostic markers. Taken together, this study represents the first large-scale proteomics analysis of TNAC and may provide guidance on decisions regarding the chemotherapy treatment of TNAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0126eb87e77314e6e97348e0d5a5a4cb7c491972" target='_blank'>
              Proteomics Landscape of Triple-Negative Apocrine Breast Carcinoma Reveals Molecular Mechanisms of Tumorigenesis and Characteristics of Chemo-insensitivity
              </a>
            </td>
          <td>
            Yiying Zhu, M. Long, Wenhao Shi, Tianlong He, Fangzhou Xie, Annan Qian, Yuqiao Liu, Taobo Hu, Shaojun Tang
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Despite immense interest in biomarker applications of extracellular vesicles (EVs) from blood, our understanding of their physiological population in healthy humans remains limited. Using imaging and multiplex bead-based flow cytometry, we comprehensively quantified circulating EVs with respect to their cellular origin in a large cohort of healthy blood donors. We assessed coefficients of variations to characterise their biological variability and explored demographic, clinical, and lifestyle factors contributing to this variability. Cell-specific circulating EV subsets show a wide range of concentrations, which do not directly reflect concentrations of blood cells, indicating diverse patterns of EV subset release and/or uptake, even for EVs originating from the same cell type. Interestingly, tetraspanin+ circulating EVs largely originate from platelets and to a lesser extent from lymphocytes. PCA and association analyses demonstrate high biological inter-individual variability in circulating EVs across healthy humans, which can be only partly explained by the influence of sex, menopausal status, age and smoking on specific circulating EV and/or tetraspanin+ circulating EV subsets. No global influence of the explored subject’s factors on circulating EVs was detected. Our findings provide the first comprehensive, quantitative data towards the cell-origin atlas of blood EVs, with important implications in the clinical use of EVs as biomarkers of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034915449166dc66e67d52a49a16c6d0d29b2917" target='_blank'>
              Comprehensive Phenotyping of Extracellular Vesicles in Blood of Healthy Humans – Insights into Cellular Origin and Biological Variability
              </a>
            </td>
          <td>
            M. Holcar, Ivica Marić, T. Tertel, K. Goričar, Urška Čegovnik Primožič, Darko Černe, Bernd Giebel, Metka Lenassi
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) is a rare, incurable, and aggressive B-cell non-Hodgkin lymphoma (NHL). Early MCL diagnosis and treatment is critical and puzzling due to inter/intra-tumoral heterogeneity and limited understanding of the underlying molecular mechanisms. We developed and applied a multifaceted analysis of selected publicly available transcriptomic data of well-defined MCL stages, integrating network-based methods for pathway enrichment analysis, co-expression module alignment, drug repurposing, and prediction of effective drug combinations. We demonstrate the “butterfly effect” emerging from a small set of initially differentially expressed genes, rapidly expanding into numerous deregulated cellular processes, signaling pathways, and core machineries as MCL becomes aggressive. We explore pathogenicity-related signaling circuits by detecting common co-expression modules in MCL stages, pointing out, among others, the role of VEGFA and SPARC proteins in MCL progression and recommend further study of precise drug combinations. Our findings highlight the benefit that can be leveraged by such an approach for better understanding pathobiology and identifying high-priority novel diagnostic and prognostic biomarkers, drug targets, and efficacious combination therapies against MCL that should be further validated for their clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac8aca4ef99bd3761be0b240141c93cbef6b50b1" target='_blank'>
              Insight into Mantle Cell Lymphoma Pathobiology, Diagnosis, and Treatment Using Network-Based and Drug-Repurposing Approaches
              </a>
            </td>
          <td>
            Georgia Orfanoudaki, K. Psatha, M. Aivaliotis
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="
 Multiple therapies with immune mechanisms of action such as CAR T-cells and bispecific antibodies are now approved for relapsed/refractory large B-cell lymphoma (rrLBCL). The efficacy of these therapies is likely influenced by lymphoma microenvironment (LME) characteristics, but these have not been directly characterized in a comprehensive fashion. We therefore performed single-nucleus multiome (RNA + ATAC), bulk RNA, and WES of 119 biopsies from 114 patients to capture both hematopoietic and non-hematopoietic cell types. After stringent quality control, 970,239 cells remained. Non-B-cell lineages were classified into 76 transcriptionally-distinct cell types using unsupervised clustering. LME archetypes were defined by non-negative matrix factorization (NMF) of non-B cell types. Associations with CAR T-cell outcomes were validated using published Nanostring data from the ZUMA7 clinical trial. NMF clustering identified 5 cellular modules: lymph-node 1 [LN1] and lymph-node 2 [LN2] that are characterized by lymph-node structural cell types, antigen presenting cells and naïve and memory T-cells; T-effector/exhausted [TEX] characterized by high frequencies of effector and exhausted CD8 T-cells; and fibroblast/macrophage 1 [FMAC1] and 2 [FMAC2] that were characterized by high frequencies of macrophage subsets and fibroblast subsets including cancer associated fibroblasts (CAFs). The pairs of LN1 and LN2 modules, and FMAC1 and FMAC2 modules, were correlated and therefore considered collectively in tumor archetype construction, resulting in three major archetypes: LN (33% of tumors), TEX (25% of tumors) and FMAC (42% of tumors). Among these biopsies, 17 were obtained immediately prior to, and 13 were subsequent to CD19 CAR T-cell therapy. Although numbers were limited, these cases suggested that the LN archetype was associated with greater benefit from CAR T cell therapy. We therefore leveraged our bulk RNA-sequencing data plus published Nanostring data from the ZUMA7 study of axicabtagene ciloleucel (axi-cel) in second line rrLBCL to develop a Naïve Bayes classifier for these archetypes. Evaluation of response data from ZUMA7 showed that the greatest benefit for axi-cel compared to chemotherapy was observed within the LN subtype (HR=0.18; P<0.0001), compared to the FMAC (HR=0.4; P<0.0001) and TEX (HR=0.71; P=0.23), with median PFS in the axi-cel arm of 30.16, 4.94, and 4.53 months respectively. Furthermore, LN subtype patients had significantly better outcomes compared to FMAC and TEX subtype patients in the axi-cel arm (HR=0.5, P=0.006) but not in the chemotherapy arm (HR=1.3; P=0.23). Therefore, accurate construction of the rrLBCL LME using direct cell measurements of both hematopoietic and non-hematopoietic cells with single-cell genomics permits the identification of cell types, cell modules, and LME archetypes with important implications for LBCL biology, and may present an opportunity for LME-guided selection of patients most likely to benefit from cellular therapy.
 Citation Format: Xubin Li, Kartik Singhal, Qing Deng, Dai Chihara, David Russler-Germain, Usama Hussein, Jennifer A. Foltz, Jared Henderson, Ashley Wilson, Evelyn Schmidt, Imran Nizamuddin, Tommy Dinh, Ryan Sun, Francisco Vega, Christopher Flowers, Obi Griffith, Todd Fehniger, Malachi Griffith, Michael R. Green. Comprehensive characterization of the non-tumor microenvironment of relapsed/refractory large B-cell lymphoma identifies patients with greatest benefit from CD19 CAR T-cell therapy [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731635a6f268d61f8670c7f3dbebaf34d1a0dda8" target='_blank'>
              Abstract PR06: Comprehensive characterization of the non-tumor microenvironment of relapsed/refractory large B-cell lymphoma identifies patients with greatest benefit from CD19 CAR T-cell therapy
              </a>
            </td>
          <td>
            Xubin Li, Kartik Singhal, Q. Deng, Dai Chihara, David A. Russler-Germain, Usama Hussein, J. Foltz, J. Henderson, Ashley Wilson, Evelyn Schmidt, Imran Nizamuddin, Tommy Dinh, Ryan Sun, F. Vega, C. Flowers, O. Griffith, T. Fehniger, M. Griffith, M. Green
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131fbeb07da63c8da0152ac32458fba6ab9aeb1b" target='_blank'>
              Chip collection of hepatocellular carcinoma based on O2 heterogeneity from patient tissue
              </a>
            </td>
          <td>
            Sewoom Baek, H. Ha, Jeong Su Park, Min Jeong Cho, Hye-Seon Kim, Seung Eun Yu, Seyong Chung, Chansik Kim, Jueun Kim, Ji Youn Lee, Yerin Lee, Hyunjae Kim, Yujin Nam, Sungwoo Cho, Kyubae Lee, Ja Kyung Yoon, Jin Sub Choi, Dai Hoon Han, Hak-Joon Sung
          </td>
          <td>2024-06-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a rare disease associated with a poor prognosis, primarily due to early recurrence and metastasis. An important feature of this condition is microvascular invasion (MVI). However, current predictive models based on imaging have limited efficacy in this regard. This study employed a random forest model to construct a predictive model for MVI identification and uncover its biological basis. Single-cell transcriptome sequencing, whole exome sequencing, and proteome sequencing were performed. The area under the curve of the prediction model in the validation set was 0.93. Further analysis indicated that MVI-associated tumor cells exhibited functional changes related to epithelial-mesenchymal transition and lipid metabolism due to alterations in the NF-kappa B and MAPK signaling pathways. Tumor cells were also differentially enriched for the IL-17 signaling pathway. There was less infiltration of SLC30A1+ CD8+ T cells expressing cytotoxic genes in MVI-associated ICC, whereas there was more infiltration of myeloid cells with attenuated expression of the MHC II pathway. Additionally, MVI-associated intercellular communication was closely related to the SPP1-CD44 and ANXA1-FPR1 pathways. These findings resulted in a brilliant predictive model and fresh insights into MVI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4891c8bc5aef729b930179251eb7e060720eef85" target='_blank'>
              Machine learning based on biological context facilitates the identification of microvascular invasion in intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Shuaishuai Xu, Mingyu Wan, C. Ye, Ruyin Chen, Qiong Li, Xiaochen Zhang, Jian Ruan
          </td>
          <td>2024-08-01</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ovarian cancer (OC) remains the deadliest gynecological cancer, primarily due to late stage diagnosis and high rates of chemotherapy resistance and recurrence. Lack of representative preclinical models exacerbates the challenges of discovering effective therapies to treat platinum-resistant OC. Patient-derived xenograft (PDX) models maintain the genetic characteristics of the original tumor, but require considerable development time and have limited screening capabilities. Organoid models mimic the tumor’s 3D structure and preserve intra-tumoral heterogeneity, so are ideal for drug screening. The purpose of this study was to determine if PDX-derived organoids (PDXOs) can reflect patient responses to chemotherapy similar to organoids derived from primary patient tumors (PDOs). In drug response assays, PDXOs and PDOs demonstrated similar sensitivity to standard chemotherapy. Furthermore, both PDXOs and PDOs reliably reflected patient response based on the clinical designation of platinum sensitivity. Seven out of eight models derived from six platinum-sensitive cases showed a significant reduction in cell viability when treated with carboplatin, paclitaxel, or the combination therapy. Six of seven organoid models derived from four platinum-resistant or refractory patients demonstrated little to no reduction in cell viability with carboplatin or combination treatment. In these platinum-resistant models, response to single agent paclitaxel was mixed, suggesting that organoid models could predict response to second-line paclitaxel. This study demonstrates that PDXOs can effectively mirror patient responses to chemotherapy, underscoring their potential as valuable, renewable models for screening novel therapies and developing personalized treatment strategies in OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b32a4ff45082909d5bdf821603cc33a53a0c3a" target='_blank'>
              Organoid Models Derived from Primary Tumors and Patient-Derive Xenograft Tumors Reflect Platinum Sensitivity of Ovarian Cancer Patients
              </a>
            </td>
          <td>
            Parisa Nikeghbal, Danielle Burke, Mara P. Steinkamp
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Medulloblastoma (MB) stands as one of the prevalent malignant brain tumors among pediatric patients. Despite its prevalence, the intricate interplay between the regulatory program driving malignancy in MB cells and their interactions with the microenvironment remains insufficiently understood. Leveraging the capabilities of single-cell Assay for Transposase-Accessible Chromatin sequencing (scATAC-seq), the chromatin accessibility landscape is unveiled across 59,015 distinct MB cells. This expansive dataset encompasses cells belonging to discrete molecular subgroups, namely SHH, WNT, Group3, and Group4. Within these chromatin accessibility profiles, specific regulatory elements tied to individual subgroups are uncovered, shedding light on the distinct activities of transcription factors (TFs) that likely orchestrate the tumorigenesis process. Moreover, it is found that certain neurotransmitter receptors (NTRs) are subgroup-specific and can predict MB subgroup classification when combined with their associated transcription factors. Notably, targeting essential NTRs within tumors influences both the in vitro sphere-forming capability and the in vivo tumorigenic capacity of MB cells. These findings collectively provide fresh insights into comprehending the regulatory networks and cellular dynamics within MBs. Furthermore, the significance of the TF-NTR regulatory circuits is underscored as prospective biomarkers and viable therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b19ab545540365c4fc326bff03e9d828165b6" target='_blank'>
              Single-Cell Chromatin Accessibility Analysis Reveals Subgroup-Specific TF-NTR Regulatory Circuits in Medulloblastoma.
              </a>
            </td>
          <td>
            Xiaoyue Gao, Qiyuan Zhuang, Yun Li, Guochao Li, Zheng Huang, Shenzhi Chen, Shaoxing Sun, Hui Yang, Lan Jiang, Ying Mao
          </td>
          <td>2024-06-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Intermediate cell carcinoma (Int-CA) is a rare and enigmatic primary liver cancer characterized by uniform tumor cells exhibiting mixed features of both HCC and intrahepatic cholangiocarcinoma. Despite the unique pathological features of int-CA, its molecular characteristics remain unclear yet. Methods: RNA sequencing and whole genome sequencing profiling were performed on int-CA tumors and compared with those of HCC and intrahepatic cholangiocarcinoma. Results: Int-CAs unveiled a distinct and intermediate transcriptomic feature that is strikingly different from both HCC and intrahepatic cholangiocarcinoma. The marked abundance of splicing events leading to intron retention emerged as a signature feature of int-CA, along with a prominent expression of Notch signaling. Further exploration revealed that METTL16 was suppressed within int-CA, showing a DNA copy number–dependent transcriptional deregulation. Notably, experimental investigations confirmed that METTL16 suppression facilitated invasive tumor characteristics through the activation of the Notch signaling cascade. Conclusions: Our results provide a molecular landscape of int-CA featured by METTL16 suppression and frequent intron retention events, which may play pivotal roles in the acquisition of the aggressive phenotype of Int-CA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dab7b9d18c0e5e1be8d12c7361eff07994e2e2a" target='_blank'>
              Transcriptomic profiling of intermediate cell carcinoma of the liver
              </a>
            </td>
          <td>
            Byungchan Jang, S. Kwon, Jang Hyun Kim, Jung Mo Kim, Taek Chung, Jeong Eun Yoo, Haeryoung Kim, Julien Calderaro, Hyun Goo Woo, Young Nyun Park
          </td>
          <td>2024-08-01</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde19b1f96c1cfcaa9c403d17be06cabeee637d7" target='_blank'>
              Pan-cell death-related signature reveals tumor immune microenvironment and optimizes personalized therapy alternations in lung adenocarcinoma
              </a>
            </td>
          <td>
            Linzhi Han, Jingyi He, Hongxin Xie, Yan Gong, Conghua Xie
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease.Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion.Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group.This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Objective Hepatocellular carcinoma (HCC) poses a significant clinical challenge because the long-term benefits of immune checkpoint blockade therapy are limited. A comprehensive understanding of the mechanisms underlying immunotherapy resistance in HCC is imperative for improving patient prognosis. Design In this study, to systematically investigate the characteristics of cancer-associated fibroblast (CAF) subsets and the dynamic communication among the tumor microenvironment (TME) components regulated by CAF subsets, we generated an HCC atlas by compiling single-cell RNA sequencing (scRNA-seq) datasets on 220 samples from six datasets. We combined spatial transcriptomics with scRNA-seq and multiplexed immunofluorescence to identify the specific CAF subsets in the TME that determine the efficacy of immunotherapy in HCC patients. Results Our findings highlight the pivotal role of POSTN+ CAFs as potent immune response barriers at specific tumor locations, as they hinder effective T-cell infiltration and decrease the efficacy of immunotherapy. Additionally, we elucidated the interplay between POSTN+ CAFs and SPP1+ macrophages, whereby the former recruits the latter and triggers increased SPP1 expression via the IL-6/STAT3 signaling pathway. Moreover, we demonstrated a spatial correlation between POSTN+ CAFs and SPP1+ macrophages, revealing an immunosuppressive microenvironment that limits the immunotherapy response. Notably, we found that patients with elevated expression levels of both POSTN+ CAFs and SPP1+ macrophages achieved less therapeutic benefit in an immunotherapy cohort. Conclusion Our research elucidates light on the role of a particular subset of CAFs in immunotherapy resistance, emphasizing the potential benefits of targeting specific CAF subpopulations to improve clinical responses to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321d7431417f89b80a8af6ab7f043a0cb955e621" target='_blank'>
              POSTN+ cancer-associated fibroblasts determine the efficacy of immunotherapy in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hao Wang, Yuan Liang, Zheng Liu, Rui Zhang, Jiashuo Chao, Mingming Wang, Mu Liu, Lei Qiao, Zhengfeng Xuan, Haitao Zhao, Ling Lu
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e1b4639cfb922b131986ddda7afd728b72c924a" target='_blank'>
              Deciphering the molecular landscape: evolutionary progression from gynecomastia to aggressive male breast cancer.
              </a>
            </td>
          <td>
            Chuang Yang, Zhonglin Wang, Lijun Qian, Jingyue Fu, H. Sun
          </td>
          <td>2024-06-18</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Combined gene and cell therapy are promising strategies for cancer treatment. Given the complexity of cancer, several approaches are actively studied to fight this disease. Using mesenchymal stem cells (MSCs) has demonstrated dual antitumor and protumor effects as they exert massive immune/regulatory effects on the tissue microenvironment. MSCs have been widely investigated to exploit their antitumor target delivery system. They can be genetically modified to overexpress genes and selectively or more efficiently eliminate tumor cells. Current approaches tend to produce more effective and safer therapies using MSCs or derivatives; however, the effect achieved by engineered MSCs in solid tumors is still limited and depends on several factors such as the cell source, transgene, and tumor target. This review describes the progress of gene and cell therapy focused on MSCs as a cornerstone against solid tumors, addressing the different MSC-engineering methods that have been approached over decades of research. Furthermore, we summarize the main objectives of engineered MSCs against the most common cancers and discuss the challenges, limitations, risks, and advantages of targeted treatments combined with conventional ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cab318e5fa2d83e67296cbf3bec076eaa3e16df" target='_blank'>
              Applications of Modified Mesenchymal Stem Cells as Targeted Systems against Tumor Cells
              </a>
            </td>
          <td>
            Elsa N. Garza Treviño, Adriana G Quiroz Reyes, Paulina Delgado Gonzalez, Juan Antonio Rojas Murillo, J. Islas, Santiago Saavedra Alonso, Carlos A. Gonzalez Villarreal
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genome-level clonal decomposition of a single specimen has been widely studied; however, it is mostly limited to cancer research. In this study, we developed a new algorithm CLEMENT, which conducts accurate decomposition and reconstruction of multiple subclones in genome sequencing of non-tumor (normal) samples. CLEMENT employs the Expectation-Maximization (EM) algorithm with optimization strategies specific to non-tumor subclones, including false variant call identification, non-disparate clone fuzzy clustering, and clonal allele fraction confinement. In the simulation and in vitro cell line mixture data, CLEMENT outperformed current cancer decomposition algorithms in estimating the number of clones (root-mean-square-error = 0.58-0.78 versus 1.43-3.34) and in the variant-clone membership agreement (∼85.5% versus 70.1-76.7%). Additional testing on human multi-clonal normal tissue sequencing confirmed the accurate identification of subclones that originated from different cell types. Clone-level analysis, including mutational burden and signatures, provided a new understanding of normal-tissue composition. We expect that CLEMENT will serve as a crucial tool in the currently emerging field of non-tumor genome analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b10b8eaeda7c6b37ca2e0e9728849f940c0c059" target='_blank'>
              CLEMENT: genomic decomposition and reconstruction of non-tumor subclones.
              </a>
            </td>
          <td>
            Young-Soo Chung, Seungseok Kang, Jisu Kim, Sangbo Lee, Sangwoo Kim
          </td>
          <td>2024-06-26</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Mesenchymal stem cells (MSCs) are recruited by malignant tumor cells to the tumor microenvironment (TME) and play a crucial role in the initiation and progression of malignant tumors. This role encompasses immune evasion, promotion of angiogenesis, stimulation of cancer cell proliferation, correlation with cancer stem cells, multilineage differentiation within the TME, and development of treatment resistance. Simultaneously, extensive research is exploring the homing effect of MSCs and MSC‐derived extracellular vesicles (MSCs‐EVs) in tumors, aiming to design them as carriers for antitumor substances. These substances are targeted to deliver antitumor drugs to enhance drug efficacy while reducing drug toxicity. This paper provides a review of the supportive role of MSCs in tumor progression and the associated molecular mechanisms. Additionally, we summarize the latest therapeutic strategies involving engineered MSCs and MSCs‐EVs in cancer treatment, including their utilization as carriers for gene therapeutic agents, chemotherapeutics, and oncolytic viruses. We also discuss the distribution and clearance of MSCs and MSCs‐EVs upon entry into the body to elucidate the potential of targeted therapies based on MSCs and MSCs‐EVs in cancer treatment, along with the challenges they face.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/585f69f9945ccba64634f80eb9b13a3dd4c96a35" target='_blank'>
              The role of mesenchymal stem cells in cancer and prospects for their use in cancer therapeutics
              </a>
            </td>
          <td>
            Jian Tang, Yu Chen, Chunhua Wang, Ying Xia, Tingyu Yu, Mengjun Tang, Kun Meng, Lijuan Yin, Yang Yang, Liang Shen, Hui Xing, Xiaogang Mao
          </td>
          <td>2024-07-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1c26de2e8d9a710cffe1117e49d0f50cd225d69" target='_blank'>
              Cancer organoids 2.0: modelling the complexity of the tumour immune microenvironment.
              </a>
            </td>
          <td>
            Roel Polak, Elisa T. Zhang, Calvin J Kuo
          </td>
          <td>2024-07-08</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bf390baec4f133e1159b69971ccead0d2deed57" target='_blank'>
              Establishment and characterization of the PDAC-X3 cell line: a novel Chinese-origin pancreatic ductal adenocarcinoma cell line.
              </a>
            </td>
          <td>
            Changpeng Chai, Huan-zhang Tang, Xin Miao, Yuanhui Su, Lu Li, Cheng Yu, Jian-Fen Yi, Zhenzhen Ye, L. Miao, Bo Zhang, Zhengfeng Wang, Wei Luo, Jinjing Hu, Hui Zhang, Wence Zhou, Hao Xu
          </td>
          <td>2024-07-16</td>
          <td>Human cell</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Therapy resistance is a major clinical hurdle in bone cancer treatment and seems to be largely driven by poorly understood microenvironmental factors. Recent evidence suggests a critical role for a unique subpopulation of mesenchymal stem cells with inflammatory features (iMSCs), though their origin and function remained unexplored. We demonstrate that cancer-secreted extracellular vesicles (EVs) trigger the development of iMSCs, which hinder therapy response in vivo, and set out to identify strategies to counteract their function.


EXPERIMENTAL DESIGN
The role of iMSCs in therapy resistance was evaluated in an orthotopic xenograft mouse model of osteosarcoma. EV-induced alterations of the MSC transcriptome were analyzed and compared with scRNA-seq data of osteosarcoma and multiple myeloma patient biopsies. Functional assays identified EV components driving iMSC development. We assessed the efficacy of clinical drugs in blocking iMSC-induced resistance in vivo.


RESULTS
We found that iMSCs are induced by interaction with cancer EVs and completely abrogate the antimetastatic effect of TGFb signaling inhibition. Importantly, EV-induced iMSCs faithfully recapitulate the inflammatory single-cell RNA signature of stromal cells enriched in multiple myeloma and osteosarcoma patient biopsies. Mechanistically, cancer EVs act through two distinct mechanisms. EV-associated TGFb induces IL6 production, while the EV-RNA cargo enhances TLR3-mediated chemokine production. We reveal that simultaneous blockade of downstream EV-activated pathways with ladarixin and tocilizumab disrupts metastasis formation and overcomes iMSC-induced resistance.


CONCLUSIONS
Our observations establish iMSCs as major contributors to drug resistance, reveal EVs as physiological triggers of iMSC development and highlight a promising combination strategy to improve therapy response in bone cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fb87f3a9aca9bc675c26582efc89f032b59273" target='_blank'>
              Tumor-secreted extracellular vesicles counteract therapy response by triggering inflammatory mesenchymal stem cell development.
              </a>
            </td>
          <td>
            Crescenzo Massaro, Hilal N Sensoy, Manon Mulders, Celine De Schrijver, C. Gómez-Martín, Juan Simon-Nieto, T. Lagerweij, Alisha Atmopawiro, Jennifer Pérez-Boza, Maarten P. Bebelman, Leontien Bosch, Simone Foderaro, Mafalda Neves Ferreira, Monique A J van Eijndhoven, Jan R. T. van Weering, C. Dell'Aversana, L. Altucci, C. D. Savci-Heijink, Niels W C J van de Donk, Cristina Giorgio, L. Brandolini, M. Allegretti, D. M. Pegtel, S. R. Baglio
          </td>
          <td>2024-08-08</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="PURPOSE
The molecular drivers underlying mucinous tumor pathogenicity are poorly understood. GNAS mutations predict metastatic burden and treatment resistance in mucinous appendiceal adenocarcinoma. We investigated the pan-cancer clinicopathologic relevance of GNAS variants.


METHODS
We assessed 58,043 patients with Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (IMPACT)-sequenced solid tumors to identify oncogenic variants, including GNAS, associated with mucinous tumor phenotype. We then performed comprehensive molecular analyses to compare GNAS-mutant (mut) and wild-type tumors across cancers. Gene expression patterns associated with GNAS-mut tumors were assessed in a The Cancer Genome Atlas cohort. Associations between GNAS variant status and peritoneal metastasis, first-line systemic therapy response, progression-free survival (PFS), and overall survival (OS) were determined using a propensity-matched subcohort of patients with metastatic disease.


RESULTS
Mucinous tumors were enriched for oncogenic GNAS variants. GNAS was mutated in >1% of small bowel, cervical, colorectal, pancreatic, esophagogastric, hepatobiliary, and GI neuroendocrine cancers. Across these cancers, GNAS-mut tumors exhibited a generally conserved C-to-T mutation-high, aneuploidy-low molecular profile with co-occurring prevalent KRAS variants (65% of GNAS-mut tumors) and fewer TP53 alterations. GNAS-mut tumors exhibited recurrently comutated alternative tumor suppressors (RBM10, INPPL1) and upregulation of MAPK and cell surface modulators. GNAS-mut tumors demonstrate an increased prevalence of peritoneal metastases (odds ratio [OR], 1.7 [95% CI, 1.1 to 2.5]; P = .006), worse response to first-line systemic therapy (OR, 2.2 [95% CI, 1.3 to 3.8]; P = .003), and shorter PFS (median, 5.6 v 7.0 months; P = .047). In a multivariable analysis, GNAS mutated status was independently prognostic of worse OS (hazard ratio, 1.25 [95% CI, 1.01 to 1.56]; adjusted P = .04).


CONCLUSION
Across the assessed cancers, GNAS-mut tumors exhibit a conserved molecular and clinical phenotype defined by mucinous tumor status, increased peritoneal metastasis, poor response to first-line systemic therapy, and worse survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a7fd5d8b4825c722728924fd719f0a0d9f3a8e" target='_blank'>
              Molecular and Clinicopathologic Impact of GNAS Variants Across Solid Tumors.
              </a>
            </td>
          <td>
            P. Johannet, Somer Abdelfattah, Callahan Wilde, Shrey Patel, H. Walch, B. Rousseau, Guillem Argiles, O. Artz, Miteshkumar Patel, Andrea Arfe, A. Cercek, R. Yaeger, K. Ganesh, N. Schultz, Luis A. Diaz, M. Foote
          </td>
          <td>2024-08-09</td>
          <td>Journal of clinical oncology : official journal of the American Society of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Preclinical models that replicate patient tumours as closely as possible are crucial for translational cancer research. While in vitro cancer models have many advantages in assessing tumour response therapy, in vivo systems are essential to enable evaluation of the role of the tumour cell extrinsic factors, such as the tumour microenvironment and host immune system. The requirement for a functional immune system is particularly important given the current focus on immunotherapies. Therefore, we set out to generate an immunocompetent, transplantable model of colorectal cancer suitable for in vivo assessment of immune-based therapeutic approaches. Intestinal tumours from a genetically engineered mouse model, driven by expression of a Pik3ca mutation and loss of Apc, were transplanted into wild type C57BL/6 host mice and subsequently passaged to form a novel syngeneic transplant model of colorectal cancer. Our work confirms the potential to develop a panel of mouse syngeneic grafts, akin to human PDX panels, from different genetically engineered, or carcinogen-induced, mouse models. Such panels would allow the in vivo testing of new pharmaceutical and immunotherapeutic treatment approaches across a range of tumours with a variety of genetic driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e96ac527246a723f0c584b8d2c966fbc703899" target='_blank'>
              Development of an in vivo syngeneic mouse transplant model of invasive intestinal adenocarcinoma driven by endogenous expression of Pik3caH1047R and Apc loss
              </a>
            </td>
          <td>
            Francesc de las Heras, Camilla B. Mitchell, William K. Murray, N. Clemons, Wayne A. Phillips
          </td>
          <td>2024-08-02</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03aef8e2ade23cd8110e6a9d02003c1b5ecd551f" target='_blank'>
              Nanoscale insights into hematology: super-resolved imaging on blood cell structure, function, and pathology
              </a>
            </td>
          <td>
            Jinghan Liu, Yuping Yolanda Tan, Wen Zheng, Yao Wang, L. Ju, Qian Peter Su
          </td>
          <td>2024-06-24</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Brain tumors, particularly glioblastoma (GBM), are devastating and challenging to treat, with a low 5-year survival rate of only 6.6%. Mouse models are established to understand tumorigenesis and develop new therapeutic strategies. Large-scale genomic studies have facilitated the identification of genetic alterations driving human brain tumor development and progression. Genetically engineered mouse models (GEMMs) with clinically relevant genetic alterations are widely used to investigate tumor origin. Additionally, syngeneic implantation models, utilizing cell lines derived from GEMMs or other sources, are popular for their consistent and relatively short latency period, addressing various brain cancer research questions. In recent years, the success of immunotherapy in specific cancer types has led to a surge in cancer immunology-related research which specifically necessitates the utilization of immunocompetent mouse models. In this review, we provide a comprehensive summary of GEMMs and syngeneic mouse models for adult brain tumors, emphasizing key features such as model origin, genetic alteration background, oncogenic mechanisms, and immune-related characteristics. Our review serves as a valuable resource for the brain tumor research community, aiding in the selection of appropriate models to study cancer immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff748c1f4b0e36e21dc90ffb3f8ae15d403a0e7" target='_blank'>
              In Vivo Mouse Models for Adult Brain Tumors: Exploring Tumorigenesis and Advancing Immunotherapy Development.
              </a>
            </td>
          <td>
            John Figg, Dongjiang Chen, Laura Falceto Font, Catherine Flores, Dan Jin
          </td>
          <td>2024-07-11</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background PIK3CA mutations are implicated in various cancers, but the implications of multiple concurrent mutations and their orientations within the gene have not been fully explored. Methods In this study, we analyzed multi‐PIK3CA mutations across a diverse pan‐cancer cohort comprising 3564 tumors. Results Multi‐PIK3CA mutations were present in 10.3% of all PIK3CA‐mutant tumors, predominantly occurring in breast and gynecological cancers. Notably, mutations within the helical domain (E542:E545) exclusively occurred in the trans‐orientation, contrasting with mutations in the kinase ABD and C2 domains, which mainly appeared in the cis orientation. Conclusions The distinct pattern of mutation orientations in PIK3CA suggests variable oncogenic potential, with helical domain mutations in the trans‐orientation potentially being less oncogenic. These findings highlight the importance of mutation orientation in the PIK3CA gene as potential biomarkers for targeted therapy. This understanding is crucial for designing clinical trials that leverage PI3K inhibitors, aiming for more effective and precise cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c039fa2d1ed75da420038d18b0f0cfdcf748c0d" target='_blank'>
              Characterizing multi‐PIK3CA mutations across cancer types: Toward precision oncology
              </a>
            </td>
          <td>
            Kohei Nakamura, Marin Ishikawa, R. Kawano, E. Aimono, Takaaki Mizuno, S. Nohara, Shigeki Tanishima, Hideyuki Hayashi, H. Nishihara
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Gliomas’ aggressive nature and resistance to therapy make them a major problem in oncology. Gliomas continue to have dismal prognoses despite significant advancements in medical science, and traditional treatments like surgery, radiation (RT), and chemotherapy (CT) frequently prove to be ineffective. After glioma stem cells (GSCs) were discovered, the traditional view of gliomas as homogeneous masses changed. GSCs are essential for tumor growth, treatment resistance, and recurrence. These cells’ distinct capacities for differentiation and self-renewal are changing our knowledge of the biology of gliomas. This systematic literature review aims to uncover the molecular mechanisms driving glioma progression associated with GSCs. The systematic review adhered to PRISMA guidelines, with a thorough literature search conducted on PubMed, Ovid MED-LINE, and Ovid EMBASE. The first literature search was performed on 1 March 2024, and the search was updated on 15 May 2024. Employing MeSH terms and Boolean operators, the search focused on molecular mechanisms associated with GCSs-mediated glioma progression. Inclusion criteria encompassed English language studies, preclinical studies, and clinical trials. A number of 957 papers were initially identified, of which 65 studies spanning from 2005 to 2024 were finally included in the review. The main GSC model distribution is arranged in decreasing order of frequency: U87: 20 studies (32.0%); U251: 13 studies (20.0%); A172: 4 studies (6.2%); and T98G: 2 studies (3.17%). From most to least frequent, the distribution of the primary GSC pathway is as follows: Notch: 8 studies (12.3%); STAT3: 6 studies (9.2%); Wnt/β-catenin: 6 studies (9.2%); HIF: 5 studies (7.7%); and PI3K/AKT: 4 studies (6.2%). The distribution of molecular effects, from most to least common, is as follows: inhibition of differentiation: 22 studies (33.8%); increased proliferation: 18 studies (27.7%); enhanced invasive ability: 15 studies (23.1%); increased self-renewal: 5 studies (7.7%); and inhibition of apoptosis: 3 studies (4.6%). This work highlights GSC heterogeneity and the dynamic interplay within the glioblastoma microenvironment, underscoring the need for a tailored approach. A few key pathways influencing GSC behavior are JAK/STAT3, PI3K/AKT, Wnt/β-catenin, and Notch. Therapy may target these pathways. This research urges more study to fill in knowledge gaps in the biology of GSCs and translate findings into useful treatment approaches that could improve GBM patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a85e605f55b9c1f91ff004830ccec505bfa08a" target='_blank'>
              Glioma Stem Cells as Promoter of Glioma Progression: A Systematic Review of Molecular Pathways and Targeted Therapies
              </a>
            </td>
          <td>
            Edoardo Agosti, Sara Antonietti, T. Ius, M. Fontanella, Marco Zeppieri, P. Panciani
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Inflammation is a key component in the development of abdominal aortic aneurysm (AAA), yet insights into the roles of immune cells and their interactions in this process are limited.


METHODS
Using single-cell RNA transcriptomic analysis, we deconstructed the CD45+ cell population in elastase-induced murine AAA at the single-cell level. We isolated each group of immune cells from murine AAA tissue at different time points and divided them into several subtypes, listed the remarkable differentially expressed genes, explored the developmental trajectories of immune cells, and demonstrated the interactions among them.


RESULTS
Our findings reveal significant differences in several immune cell subsets, including macrophages, dendritic cells, and T cells, within the AAA microenvironment compared with the normal aorta. Especially, conventional dendritic cell type 1 exclusively existed in the AAA tissue rather than the normal aortas. Via CellChat analysis, we identified several intercellular communication pathways like visfatin, which targets monocyte differentiation and neutrophil extracellular trap-mediated interaction between neutrophils and dendritic cells, which might contribute to AAA development. Some of these pathways were validated in human AAA.


CONCLUSIONS
Despite the absence of external pathogenic stimuli, AAA tissues develop a complex inflammatory microenvironment involving numerous immune cells. In-depth studies of the inflammatory network shall provide new strategies for patients with AAA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ebb97e30e8ccddcf0b4ca430f1bb3ad1428083" target='_blank'>
              Single-Cell RNA Sequencing Deconstructs the Distribution of Immune Cells Within Abdominal Aortic Aneurysms in Mice.
              </a>
            </td>
          <td>
            Zhen Yuan, Li Shu, Jiantao Fu, Peipei Yang, Yidong Wang, Jie Sun, Mengsha Zheng, Zhenjie Liu, Jin Yang, Jiangping Song, Shen Song, Zhejun Cai
          </td>
          <td>2024-07-25</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="A first‐in‐human trial demonstrated that a vaccine targeting the histone mutation H3K27M can induce an immune response, in a mutation‐specific manner, in patients with diffuse midline glioma. In a recent study by Boschert et al., the same group now dissects the functional immune response triggered after effective vaccination of one of the patients, who has been in remission for over 3 years. The H3K27M peptide vaccine, named H3‐vac, induces a CD4+ T‐cell‐specific immune response in this patient and expands the repertoire of polyclonal H3K27M‐specific T‐cell receptors. A clonal H3K27M‐reactive B‐cell population was also detected in the patient's cerebrospinal fluid. Importantly, the immune response is induced across various human leukocyte antigen alleleotypes, indicating the potential efficacy of the vaccine in diverse populations. By exploring in detail the immune response linked to this patient's long‐term survival, the authors prove peptide vaccinations as a viable therapeutic approach. This paves the way for personalised therapies harnessing immunogenic T‐ and B‐cell responses against different tumour types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d159638592d6fc47e8468ec726c4ac5d5865738" target='_blank'>
              The immune response behind peptide vaccination in diffuse midline glioma
              </a>
            </td>
          <td>
            Craig A. Vincent, Silvia Remeseiro
          </td>
          <td>2024-06-16</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Invasive mucinous adenocarcinoma (IMA) comprises ∼5% of lung adenocarcinoma. There is no effective therapy for IMA when surgical resection is not possible. IMA is sometimes confused with adenocarcinoma with signet ring cell features (SRCC) pathologically since both adenocarcinomas feature tumor cells with abundant intracellular mucin. The molecular mechanisms by which such mucin-producing lung adenocarcinomas develop remain unknown. Methods Using a Visium spatial transcriptomics approach, we analyzed IMA and compared it with SRCC patho-transcriptomically. Combining spatial transcriptomics data with in vitro studies using RNA-seq and ChIP-seq, we assessed downstream targets of transcription factors HNF4A and SPDEF that are highly expressed in IMA and/or SRCC. Results Spatial transcriptomics analysis indicated that there are 6 distinct cell clusters in IMA and SRCC. Notably, two clusters (C1 and C3) of mucinous tumor cells exist in both adenocarcinomas albeit at a different ratio. Importantly, a portion of genes (e.g., NKX2-1, GKN1, HNF4A and FOXA3) are distinctly expressed while some mucous-related genes (e.g., SPDEF and FOXA2) are expressed in both adenocarcinomas. We determined that HNF4A induces MUC3A/B and TM4SF4 and that BI 6015, an HNF4A antagonist, suppressed the growth of IMA cells. Using mutant SPDEF that is associated with COVID-19, we also determined that an intact DNA-binding domain of SPDEF is required for SPDEF-mediated induction of mucin genes (MUC5AC, MUC5B and AGR2). Additionally, we found that XMU-MP-1, a SPDEF inhibitor, suppressed the growth of IMA cells. Conclusion These results revealed that IMA and SRCC contain heterogenous tumor cell types, some of which are targetable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405c78189e11982c1b68c2b274f03ca88f30d620" target='_blank'>
              Patho-transcriptomic analysis of invasive mucinous adenocarcinoma of the lung (IMA): comparison with lung adenocarcinoma with signet ring cell features (SRCC)
              </a>
            </td>
          <td>
            William D. Stuart, M. Ito, Iris-Fink Baldauf, T. Fukazawa, T. Yamatsuji, Tomoshi Tsuchiya, Hideo Watanabe, Morihito Okada, Eric L. Snyder, M. Mino-Kenudson, M. Guo, Yutaka Maeda
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Brain metastases are common in triple-negative breast cancer (TNBC), suggesting a complex process of cancer spread. The mechanisms enabling TNBC cell adaptation and proliferation in the brain remain unclear. Small extracellular vesicles (sEVs) play a crucial role in communication between breast carcinoma cells and the brain. However, the lack of relevant models hinders understanding of sEV-mediated communication. The present study assesses the impact of brain organoid-derived sEVs (BO-sEVs) on various behaviours of the MDA-MB-231 cell line, chosen as a representative of TNBC in a 3D microfluidic model. Our results demonstrate that 150-200 nm sEVs expressing CD63, CD9, and CD81 from brain organoid media decrease MDA-MB-231 cell proliferation, enhance their wound-healing capacity, alter their morphology into more mesenchymal mode, and increase their stemness. BO-sEVs led to heightened PD-L1, CD49f, and vimentin levels of expression in MDA-MB-231 cells, suggesting an amplified immunosuppressive, stem-like, and mesenchymal phenotype. Furthermore, these sEVs also induced the expression of neural markers such as GFAP in carcinoma cells. The cytokine antibody profiling array also showed that BO-sEVs enhanced the secretion of MCP-1, IL-6, and IL-8 by MDA-MB-231 cells. Moreover, sEVs significantly enhance the migration and invasion of carcinoma cells toward brain organoids in a 3D organoid-on-a-chip system. Our findings emphasize the potential significance of metastatic site-derived sEVs as pivotal mediators in carcinoma progression and adaptation to the brain microenvironment, thereby unveiling novel therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/595396c0f8ca78aed55c1d7058ba15655e060da1" target='_blank'>
              Impact of brain organoid-derived sEVs on metastatic adaptation and invasion of breast carcinoma cells through a microphysiological system.
              </a>
            </td>
          <td>
            Hojjatollah Nazari, Ann-Na Cho, Dale Goss, Jean Paul Thiery, Majid Ebrahimi Warkiani
          </td>
          <td>2024-06-18</td>
          <td>Lab on a chip</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Modest response rates to immunotherapy observed in advanced lung cancer patients underscore the need to identify reliable biomarkers and targets, enhancing both treatment decision-making and efficacy. Factors such as PD-L1 expression, tumor mutation burden, and a ‘hot’ tumor microenvironment with heightened effector T cell infiltration have consistently been associated with positive responses. In contrast, the predictive role of the abundantly present tumor-infiltrating myeloid cell (TIMs) fraction remains somewhat uncertain, partly explained by their towering variety in terms of ontogeny, phenotype, location, and function. Nevertheless, numerous preclinical and clinical studies established a clear link between lung cancer progression and alterations in intra- and extramedullary hematopoiesis, leading to emergency myelopoiesis at the expense of megakaryocyte/erythroid and lymphoid differentiation. These observations affirm that a continuous crosstalk between solid cancers such as lung cancer and the bone marrow niche (BMN) must take place. However, the BMN, encompassing hematopoietic stem and progenitor cells, differentiated immune and stromal cells, remains inadequately explored in solid cancer patients. Subsequently, no clear consensus has been reached on the exact breadth of tumor installed hematopoiesis perturbing cues nor their predictive power for immunotherapy. As the current era of single-cell omics is reshaping our understanding of the hematopoietic process and the subcluster landscape of lung TIMs, we aim to present an updated overview of the hierarchical differentiation process of TIMs within the BMN of solid cancer bearing subjects. Our comprehensive overview underscores that lung cancer should be regarded as a systemic disease in which the cues governing the lung tumor-BMN crosstalk might bolster the definition of new biomarkers and druggable targets, potentially mitigating the high attrition rate of leading immunotherapies for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09f4d97d42cb2976ecd4a13f91ebddf956195779" target='_blank'>
              The crosstalk between lung cancer and the bone marrow niche fuels emergency myelopoiesis
              </a>
            </td>
          <td>
            Evelyn Calderon-Espinosa, K. De Ridder, Thomas Benoot, Yanina Jansen, Domien Vanhonacker, Robbe Heestermans, A. De Becker, Ivan Van Riet, Lore Decoster, C. Goyvaerts
          </td>
          <td>2024-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: The immune response against tumors relies on distinguishing between self and non-self, the basis of cancer immunotherapy. Neoantigens from somatic mutations are central to many immunotherapeutic strategies and understanding their landscape in breast cancer is crucial for targeted interventions. We aimed to profile neoantigens in Kenyan breast cancer patients using genomic DNA and total RNA from paired tumor and adjacent non-cancerous tissue samples of 23 patients. Methods: We sequenced the genome-wide exome (WES) and RNA, from which somatic mutations were identified and their expression quantified, respectively. Neoantigen prediction focused on human leukocyte antigens (HLA) crucial to cancer, HLA type I. HLA alleles were predicted from WES data covering the adjacent non-cancerous tissue samples, identifying four alleles that were present in at least 50% of the patients. Neoantigens were deemed potentially immunogenic if their predicted median IC50 binding scores were [≤]500nM and were expressed [transcripts per million (TPM) >1] in tumor samples. Results: An average of 1465 neoantigens covering 10260 genes had [≤]500nM median IC50 binding score and >1 TPM in the 23 patients and their presence significantly correlated with the somatic mutations (R2 =0.570, P=0.001). Assessing 58 genes reported in the catalog of somatic mutations in cancer (COSMIC, v99) to be commonly mutated in breast cancer, 44 (76%) produced >2 neoantigens among the 23 patients, with a mean of 10.5 ranging from 2 to 93. For the 44 genes, a total of 477 putative neoantigens were identified, predominantly derived from missense mutations (88%), indels (6%), and frameshift mutations (6%). Notably, 78% of the putative breast cancer neoantigens were patient-specific. HLA-C*06:01 allele was associated with the majority of neoantigens (194), followed by HLA-A*30:01 (131), HLA-A*02:01 (103), and HLA-B*58:01 (49). Among the genes of interest that produced putative neoantigens were MUC17, TTN, MUC16, AKAP9, NEB, RP1L1, CDH23, PCDHB10, BRCA2, TP53, TG, and RB1. Conclusions: The unique neoantigen profiles in our patient group highlight the potential of immunotherapy in personalized breast cancer treatment as well as potential biomarkers for prognosis. The unique mutations producing these neoantigens, compared to other populations, provide an opportunity for validation in a much larger sample cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4be4a56941f362fcf3f465561b9453240cfdfd" target='_blank'>
              Whole exome-seq and RNA-seq data reveal unique neoantigen profiles in Kenyan breast cancer patients
              </a>
            </td>
          <td>
            G. Wagutu, J. Gitau, K. Mwangi, M. Murithi, E. Melly, A. R. Harris, S. Sayed, S. Ambs, F. Makokha
          </td>
          <td>2024-06-20</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Previous characterization of the genome and transcriptome of glioblastoma (GBM) has revealed molecular alterations that potentially drive GBM pathogenesis and heterogeneity 1-6. These open-resources are evolving, such as The Cancer Genome Atlas (TCGA) and The Cancer Imaging Atlas (TCIA) at the National Institute of Health comprising a large cohort of molecular and MRI data. Yet, no report deciphers the link between molecular signatures and MRI-classified GBM. The necessity to re-form molecular and imaging data motivated our computational approach to integrate TCIA and TCGA datasets derived from GBM. We uncovered common and distinct molecular signatures across GBM patients and specific to tumor locations, respectively. Despite heterogeneity in GBM, the top 12 genes from our analysis highlights that the dysregulation of a subset of neurotransmitter receptor or transporter and synaptic activity is common across GBM patients. The coherent layer of imaging and molecular information would help us stratify precision neuro-oncology and treatment options in ways that are not possible through MRI or genomic data alone. Our findings provide molecular targets in the disrupted neurocircuit of GBM, suggesting imbalanced excitation and inhibition. Given the fact that GBM patients exhibit similar symptoms resembling patients with neurodegenerative diseases and seizures, our results supported the hypothesis-GBM in the context of neurological disorders beyond a solely cancerous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bcee33160d7d19048e98fc72e314300b743ed5" target='_blank'>
              Differential Gene Expression in MRI-classified Glioblastoma
              </a>
            </td>
          <td>
            Christopher T. Rhodes, Yufeng Wang, Chin-Hsing Annie Lin
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary The expression of the mechanosensory Piezo2 channel has already been described in different malignant tumors. There is discordance in the literature regarding breast carcinoma, with its expression described either as decreased or increased in neoplasms with respect to benign tissue. A retrospective cohort of 125 patients whose breasts were resected for carcinoma was chosen to determine the relationship between Piezo2 and different clinical and histological variables. A significant association was found with the Ki67 proliferation index, with a tendency for most proliferative tumors to be positive for Piezo2. Abstract In the last decade, a group of Ca2+ channels called Piezo were discovered, demonstrating a decisive role in the cellular response to mechanical stimuli and being essential in the biological behavior of cells regarding the extracellular compartment. Several investigations have suggested a potential role in carcinogenesis, with a tumor suppressor role in some cases but increased expression in several high-grade neoplasms. Regarding Piezo2 expression in mammary gland neoplasms, a protective role for Piezo2 was initially suggested, but a subsequent study demonstrated a relationship between Piezo2 expression and the highly aggressive triple-negative phenotype of breast carcinoma. A cohort of 125 patients with clinical follow-up was chosen to study Piezo2 expression and clarify its clinical implications using the same immunohistochemical evaluation performed for other breast carcinoma parameters. Fisher’s exact test was chosen to identify potential relationships between the different variables. A significant association was found with the Ki67 proliferation index, but not with mitoses. The tendency of most proliferative tumors was to have an increased score for Piezo2. A similar association was found between Piezo2 expression and perineural invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6531181900e59a16e97bcf0664ec51d36a032225" target='_blank'>
              Prognostic Evaluation of Piezo2 Channels in Mammary Gland Carcinoma
              </a>
            </td>
          <td>
            R. Martín-Sanz, Aline Rodrigues-Françoso, Y. García-Mesa, F. J. García-Alonso, María Asunción Gómez-Muñoz, Sandra Malmierca-González, Rocío Salazar-Blázquez, Olivia García-Suárez, J. Feito
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c66c83c1a10b2686b4f5d3cde67d2c730b0ba4" target='_blank'>
              Computational approaches identify a transcriptomic fingerprint of drug-induced structural cardiotoxicity
              </a>
            </td>
          <td>
            Victoria P W Au Yeung, Olga Obrezanova, Jiarui Zhou, Hongbin Yang, Tara J Bowen, Delyan Ivanov, Izzy Saffadi, Alfie S Carter, Vigneshwari Subramanian, Inken Dillmann, Andrew R Hall, Adam Corrigan, M. Viant, Amy Pointon
          </td>
          <td>2024-06-28</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ff0f7d3a22a2ccb36635a3877c0f50c26a651" target='_blank'>
              MHC-I-presented non-canonical antigens expand the cancer immunotherapy targets in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yangyang Cai, Donghao Li, Dezhong Lv, Jiaxin Yu, Yingying Ma, Tiantongfei Jiang, Na Ding, Zhigang Liu, Yongsheng Li, Juan Xu
          </td>
          <td>2024-08-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary This study investigates the impact of the BRAFV600E mutation, present in half of melanoma cases, on immune microenvironment and therapeutic response. Analyzing the Cancer Genome Atlas data, we uncover that the mutation correlates with reduced tumor mutation burden, indicating a lower generation of immune-stimulating neopeptides. Examination of immune subtypes reveal heightened immunosuppression in BRAFV600E-mutated tumors. Using melanoma cell lines harboring different genomic profiles, we observed an enhanced response to immunogenic cell death (ICD)—a crucial process eliciting anti-tumor immune responses—mediated by photodynamic therapy (PDT). Transcriptomic analysis highlights upregulation of IFNAR1, IFNAR2, and CXCL10 genes in BRAFV600E-mutated cells, suggesting their role in ICD. Our results describe the intricate relationship between the BRAFV600E mutation and immune responses, hinting at a potential link between this mutation and responsiveness to ICD-inducing therapies, particularly PDT, putatively linked to increased IFN-1 pathway activation. Abstract The BRAFV600E mutation, found in approximately 50% of melanoma cases, plays a crucial role in the activation of the MAPK/ERK signaling pathway, which promotes tumor cell proliferation. This study aimed to evaluate its impact on the melanoma immune microenvironment and therapeutic responses, particularly focusing on immunogenic cell death (ICD), a pivotal cytotoxic process triggering anti-tumor immune responses. Through comprehensive in silico analysis of the Cancer Genome Atlas data, we explored the association between the BRAFV600E mutation, immune subtype dynamics, and tumor mutation burden (TMB). Our findings revealed that the mutation correlated with a lower TMB, indicating a reduced generation of immunogenic neoantigens. Investigation into immune subtypes reveals an exacerbation of immunosuppression mechanisms in BRAFV600E-mutated tumors. To assess the response to ICD inducers, including doxorubicin and Me-ALA-based photodynamic therapy (PDT), compared to the non-ICD inducer cisplatin, we used distinct melanoma cell lines with wild-type BRAF (SK-MEL-2) and BRAFV600E mutation (SK-MEL-28, A375). We demonstrated a differential response to PDT between the WT and BRAFV600E cell lines. Further transcriptomic analysis revealed upregulation of IFNAR1, IFNAR2, and CXCL10 genes associated with the BRAFV600E mutation, suggesting their involvement in ICD. Using a gene reporter assay, we showed that PDT robustly activated the IFN-1 pathway through cGAS-STING signaling. Collectively, our results underscore the complex interplay between the BRAFV600E mutation and immune responses, suggesting a putative correlation between tumors carrying the mutation and their responsiveness to therapies inducing the IFN-1 pathway, such as the ICD inducer PDT, possibly mediated by the elevated expression of IFNAR1/2 receptors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ed26388b396064bd1f63bcbdf14dd7a6adc97c" target='_blank'>
              Impact of Genomic Mutation on Melanoma Immune Microenvironment and IFN-1 Pathway-Driven Therapeutic Responses
              </a>
            </td>
          <td>
            F. Mentucci, Elisa Ayelén Romero Nuñez, Agustina Ercole, Valentina Silvetti, J. Dal Col, M. Lamberti
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) is a heterogenous disease that disproportionately affects Black women. TNBC outcomes among Black women are dismal secondary to multiple factors, such as poor healthcare accessibility resulting in delays in diagnosis, and aggressive disease biology in addition to a pro-tumor immune microenvironment (TME). Black women with breast cancer exhibit elevated levels of serum pro-inflammatory cytokines, and a pro-tumorigenic TME with higher immunosuppressive regulatory T cells (Tregs), M2 macrophages and exhausted CD8+ T cells. We have shown that the combined use of toll-like receptor 3 (TLR3) ligands with interferon-α (chemokine modulation: CKM) is able to enrich the tumor with CD8+ T cells, while not increasing immunosuppressive cells. Recent clinical trials have revealed the efficacy of immune checkpoint inhibitors (ICI) in rejuvenizing exhausted CD8+ T cells. We hypothesize that strategies to modulate the TME by enriching chemokines that attract CD8+T cells followed by reversal of CD8+ T cell exhaustion (ICI), when added to standard treatment, could potentially improve clinical outcomes, and mitigate the racial disparities in TNBC outcomes between Black and White Women.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a19fbb259d7a213132cc235d155d56ac9120b99" target='_blank'>
              Targeting the tumor microenvironment to improve clinical outcomes in triple negative breast cancer patients and bridge the current disparity gap
              </a>
            </td>
          <td>
            Malak Alharbi, A. M. Roy, Jayasree Krishnan, Pawel Kalinski, Song Yao, Shipra Gandhi
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) is dependent on a supportive tumor immune microenvironment (TIME), where infiltration of CD163+ macrophages has a negative prognostic impact. This study explores how abundance and spatial localization of CD163+ cells are associated with the biology of the MCL TIME. This is achieved through spatial multi-omic investigations of tumor and infiltrating CD163+ and CD3+ cells, respectively. We analyzed diagnostic MCL tissue from 100 patients. Sixty-three proteins were measured by GeoMx® digital spatial profiling in tissue microarrays. Regions of interests (ROIs) were selected in tumor-rich and tumor-sparse tissue regions. Molecular profiling of CD163+ macrophage segments, CD20+ MCL tumor cell segments and CD3+ T-cell segments was performed. To validate protein profiles, 1811 mRNAs were measured in CD20+ cells and two subsets of T-cells. Image analysis was used to extract the phenotype and position of each targeted cell allowing exploration of cell frequencies and cellular neighborhoods. Proteomic investigations revealed that CD163+ cells modulate their immune profile depending on the localization and that the immune inhibitory molecules VISTA and B7-H3 have higher expression in tumor-sparse versus tumor-rich tissue regions and targeting should be explored. We show that MCL tissues with more abundant infiltration of CD163+ cells have a higher expression of key components of the mitogen-activated protein kinase (MAPK) pathway, which was validated by complementary mRNA analyses. Thus, the MAPK pathway may be a feasible therapeutic target in MCL patients with CD163+ cell infiltration. We further show the independent and combined prognostic value of CD11c and CD163 beyond established risk factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d71801d630d06d650258878ed1d5d14b266a8e5" target='_blank'>
              CD163+ macrophages in mantle cell lymphoma induce activation of pro-survival pathways and immune suppression.
              </a>
            </td>
          <td>
            J. Rodrigues, L. Lokhande, L. Olsson, May Hassan, Angelica Johansson, Anna Janská, Darshan Kumar, Lina Schmidt, A. Nikkarinen, P. Hollander, I. Glimelius, A. Porwit, A. Gerdtsson, Mats Jerkeman, Sara Ek
          </td>
          <td>2024-07-03</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Single-cell technology has allowed researchers to probe tissue complexity and dynamics at unprecedented depth in health and disease. However, the generation of high-dimensionality single-cell atlases and virtual three-dimensional tissues requires integrated reference maps that harmonize disparate experimental designs, analytical pipelines, and taxonomies. Here, we present a comprehensive single-cell transcriptome integration map of cardiac fibrosis, which underpins pathophysiology in most cardiovascular diseases. Our findings reveal similarity between cardiac fibroblast (CF) identities and dynamics in ischemic versus pressure overload models of cardiomyopathy. We also describe timelines for commitment of activated CFs to proliferation and myofibrogenesis, profibrotic and antifibrotic polarization of myofibroblasts and matrifibrocytes, and CF conservation across mouse and human healthy and diseased hearts. These insights have the potential to inform knowledge-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39d1bacb1f7ac6282e099a55f3c47a0b78634e" target='_blank'>
              Integration mapping of cardiac fibroblast single-cell transcriptomes elucidates cellular principles of fibrosis in diverse pathologies
              </a>
            </td>
          <td>
            R. Patrick, Vaibhao Janbandhu, Vikram Tallapragada, Shannon S M Tan, Emily E. McKinna, Osvaldo Contreras, S. Ghazanfar, David T. Humphreys, Nicholas J. Murray, Yen T. H. Tran, Robert D Hume, James J. H. Chong, Richard P. Harvey
          </td>
          <td>2024-06-21</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Single-cell multi-omics data is a valuable resource to promote the understanding of cancer. However, multimodal data analysis is challenging for most cancer researchers, and there is still a lack of online databases that can bridge the gap. Here we introduced scCancerExplorer, which is a user-friendly database designed for exploring single-cell multi-omics data of human pan-cancer. We processed more than 110 datasets covering single-cell multiomics data from 51 human cancer types, and provides 5 major modules and 12 useful functions for conveniently exploring those data. (1) The “Integrated multi-omics analysis” module enables users explore a gene from multimodal perspectives, such as the gene expression levels, survival analysis, transcription factor activity, and the DNA methylation levels of promoter regions. (2) With single-cell transcriptome module, users can explore the integrated pan-cancer datasets, compare the expression profiles between primary tumor and metastases, and generate customized figures. (3) With single-cell epigenome module, users can explore promoter DNA methylation levels in both normal and cancer cells, DNA methylation state at single CpG resolution, and chromatin accessibility patterns of different cell types. (4) For single-cell genome data, users can visualize the copy number variations of cancer cells for each patient. (5) Moreover, TCGA survival analysis can be performed conveniently. Users can not only group the patients according to gene expression levels, but also select patients by gene mutation states, pathological subtypes, and cancer stages. In summary, scCancerExplorer is a powerful database for convenient data mining by simple clicking, and gaining novel insights into human pan-cancer. scCancerExplorer is freely accessible at https://bianlab.cn/scCancerExplorer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0bfe81970da09d476152acc53e069683a992ff" target='_blank'>
              scCancerExplorer: a comprehensive database for interactively exploring single-cell multi-omics data of human pan-cancer
              </a>
            </td>
          <td>
            Changzhi Huang, Zekai Liu, Yunlei Guo, Wanchu Wang, Zhen Yuan, Yusheng Guan, Deng Pan, Zhibin Hu, Linhua Sun, Zan Fu, Shuhui Bian
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Organoid tumor models have emerged as a powerful tool in the fields of biology and medicine as such 3D structures grown from tumor cells recapitulate better tumor characteristics, making these tumoroids unique for personalized cancer research. Assessment of their functional behavior, particularly protein secretion, is of significant importance to provide comprehensive insights. Here, a label-free spectroscopic imaging platform is presented with advanced integrated optofluidic nanoplasmonic biosensor that enables real-time secretion analysis from single tumoroids. A novel two-layer microwell design isolates tumoroids, preventing signal interference, and the microarray configuration allows concurrent analysis of multiple tumoroids. The dual imaging capability combining time-lapse plasmonic spectroscopy and bright-field microscopy facilitates simultaneous observation of secretion dynamics, motility, and morphology. The integrated biosensor is demonstrated with colorectal tumoroids derived from both cell lines and patient samples to investigate their vascular endothelial growth factor A (VEGF-A) secretion, growth, and movement under various conditions, including normoxia, hypoxia, and drug treatment. This platform, by offering a label-free approach with nanophotonics to monitor tumoroids, can pave the way for new applications in fundamental biological studies, drug screening, and the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738f082d29ba965763765f12b51207081cea62bd" target='_blank'>
              Nanoplasmonic Single-Tumoroid Microarray for Real-Time Secretion Analysis.
              </a>
            </td>
          <td>
            Yen‐Cheng Liu, Saeid Ansaryan, Jiayi Tan, N. Broguiere, L. F. Lorenzo-Martín, K. Homicsko, G. Coukos, Matthias P. Lütolf, Hatice Altug
          </td>
          <td>2024-06-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78f32ec794232189fcdd67219dcf2441b706129" target='_blank'>
              Innate immune dynamics in the context of multisite EGFR mutations in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yuan Peng, Chuan Zeng, Rongxin Liao, Lu Shen, Yan Zhou, Zhenzhou Yang
          </td>
          <td>2024-08-06</td>
          <td>Genes and immunity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8c4d0ef06e7916deea0b92c894ed2b387bbc58" target='_blank'>
              Unraveling the Drug Response Heterogeneity with Single-Cell Vibrational Phenomics.
              </a>
            </td>
          <td>
            Yue Wang, Yadi Wang, Junhong Lü, Xueling Li
          </td>
          <td>2024-06-24</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8ae75854f197a5d035df3021257a7a07da8e59" target='_blank'>
              Leading Edge Techniques in the Quest for Characterizing Rare Hematopoietic Stem Cells
              </a>
            </td>
          <td>
            S. Patel, Alana M Franceski, Brittany L. Crown, Robert S. Welner
          </td>
          <td>2024-06-28</td>
          <td>Current Stem Cell Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Brain metastases (BrMs) constitute the most common type of brain cancer with abysmal prognosis. Given the high incidence and poor prognosis of BrM, understanding the tumor microenvironment (TME) architecture across BrM subtypes is increasingly crucial, especially with the emergence of novel immunotherapeutic agents like immune checkpoint inhibitors, which have demonstrated efficacy in melanoma and lung cancer BrM. Tertiary lymphoid structures (TLS) are spatially organized lymphoid aggregates in the TME that mediate antitumor immunity. Despite being established as prognostic and predictive markers in various cancers, their significance in BrM lacks extensive investigation. We performed gene expression profiling using paired-end RNA sequencing and multiplex immunofluorescence combined with spatial imaging on 149 human BrM samples. Cell type deconvolution analysis revealed that BrMs from non-small cell lung carcinoma (NSCLC) and malignant melanoma exhibited significantly higher immune cell infiltration compared to BrMs from breast carcinoma. Furthermore, B cell infiltration was detected in some patients, suggesting potential TLS formation. We created a metagene signature comprising TLS hallmarks in BrMs using transcriptomic profiling, allowing us to categorize BrM patients into TLS-positive and TLS-negative groups. This finding was corroborated by multiplex immunofluorescence staining of corresponding BrM tissue sections, which confirmed TLS-like aggregate formation in 40% of the BrM patients. Differential gene expression analysis highlighted the presence of activated lymphocyte-mediated immunity as well as increased immunoglobulin production signature in TLS-positive tumors. Additionally, a higher TLS signature score was associated with improved overall survival of lung carcinoma patients after BrM diagnosis. Our results underscore the heterogeneity of the tumor microenvironment in BrM, and the identification of intratumoral TLS may guide patient stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f904325c9c40fd220a1c52e4c6b7474d7dea43" target='_blank'>
              BMRK-09 PROGNOSTIC ROLE OF TUMOR-INFILTRATING LYMPHOCYTES AND TERTIARY LYMPHOID STRUCTURES IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            S. S. Mughal, Y. Reiss, J. Felsberg, L. Meyer, J. Macas, S. Schlue, Tatjana Starzetz, K. Köhrer, Tanja N Fehm, Volkmar Müller, K. Lamszus, Iris Helfrich, H. Wikman, A. Berghoff, B. Brors, Karlheinz Plate, G. Reifenberger
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>81</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [26],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>