<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Drug-Resistance%20forecasting/">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Spatial-Heterogeneity forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Spatial-Heterogeneity forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-09 06:33:14 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Spatial-Heterogeneity forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Spatial-Heterogeneity forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Spatial-Heterogeneity forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Spatial-Heterogeneity forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="Intratumoral variability is a seminal feature of human tumors contributing to tumor progression and response to treatment. Current technologies are unsuitable to accurately track phenotypes and subclonal evolution within tumors, especially in response to genetic manipulations. Here, we developed epitope combinatorial tags (EpicTags), which we coupled to multiplexed ion beam imaging (EpicMIBI) for in situ tracking of barcodes within tissue microenvironments. Using this platform, we dissected the spatial component of cell lineages and phenotypes in a xenograft model of small-cell lung cancer. We observed emergent properties from mixed clones leading to the preferential expansion of subclonal patches for both neuroendocrine and non-neuroendocrine cancer cell states in this model. In tumors harboring a fraction of PTEN-deficient cancer cells, we uncovered a non-autonomous increase of subclonal patch size in PTEN wildtype cancer cells. EpicMIBI can facilitate in situ interrogation of cell-intrinsic and cell-extrinsic processes involved in intratumoral heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6224677472f42cbcbbf1cd09cc54069086fdf" target='_blank'>
                Spatial epitope barcoding reveals subclonal tumor patch behaviors
                </a>
              </td>
          <td>
            Xavier Rovira-Clavé, Alexandros P. Drainas, Sizun Jiang, Yunhao Bai, M. Baron, Bokai Zhu, Maxim Markovic, Garry L. Coles, M. Bassik, J. Sage, G. Nolan
          </td>
          <td>2021-06-11</td>
          <td>bioRxiv</td>
          <td>12</td>
          <td>101</td>

            <td><a href='../recommendations/e7b6224677472f42cbcbbf1cd09cc54069086fdf' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Spatial-Heterogeneity forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Spatial-Heterogeneity forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM) is the deadliest form of primary brain tumor with limited treatment options. Recent studies have profiled GBM tumor heterogeneity, revealing numerous axes of variation that explain the molecular and spatial features of the tumor. Here, we seek to bridge descriptive characterization of GBM cell type heterogeneity with the functional role of individual populations within the tumor. Our lens leverages a gene program-centric meta-atlas of published transcriptomic studies to identify commonalities between diverse tumors and cell types in order to decipher the mechanisms that drive them. This approach led to the discovery of a tumor-derived stem cell population with mixed vascular and neural stem cell features, termed a neurovascular progenitor (NVP). Following in situ validation and molecular characterization of NVP cells in GBM patient samples, we characterized their function in vivo. Genetic depletion of NVP cells resulted in altered tumor cell composition, fewer cycling cells, and extended survival, underscoring their critical functional role. Clonal analysis of primary patient tumors in a human organoid tumor transplantation system demonstrated that the NVP has dual potency, generating both neuronal and vascular tumor cells. Although NVP cells comprise a small fraction of the tumor, these clonal analyses demonstrated that they strongly contribute to the total number of cycling cells in the tumor and generate a defined subset of the whole tumor. This study represents a paradigm by which cell type-specific interrogation of tumor populations can be used to study functional heterogeneity and therapeutically targetable vulnerabilities of GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8360f6cc028c180aeed6d27dc114f7d56cdab1" target='_blank'>
              Glioblastoma Neurovascular Progenitor Orchestrates Tumor Cell Type Diversity
              </a>
            </td>
          <td>
            Elisa Fazzari, Daria J. Azizad, Kwanha Yu, Weihong Ge, Matthew Li, Patricia R. Nano, Ryan L. Kan, Hong A. Tum, C. Tse, Nicholas A. Bayley, Vjola Haka, Dimitri Cadet, Travis Perryman, Jose A. Soto, Brittney Wick, D. Raleigh, Elizabeth E Crouch, Kunal S. Patel, Linda M Liau, Benjamin Deneen, David A. Nathanson, A. Bhaduri
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Intravital microscopy (IVM) enables live imaging of animals at single-cell level, offering essential insights into cancer progression. This technique allows for the observation of single-cell behaviors within their natural 3D tissue environments, shedding light on how genetic and microenvironmental changes influence the complex dynamics of tumors. The complexity of data generated by IVM often surpasses the capabilities of conventional analyses accessible to biomedical scientists, thereby neglecting single-cell heterogeneity and limiting the exploration of microenvironmental influences on cellular behavior without bias. To address this challenge, here we introduce BEHAV3D Tumor Profiler (BEHAV3D-TP), a user-friendly computational framework designed for the comprehensive analysis of single tumor cell behaviors and their interactions with the tumor microenvironment (TME). BEHAV3D-TP facilitates unbiased profiling of cancer cell dynamics without requiring advanced computational expertise. Here, we apply BEHAV3D-TP to study diffuse midline glioma (DMG), a highly aggressive pediatric brain tumor characterized by invasive growth. Our analysis reveals that distinct migratory behaviors of DMG cells correlate with specific TME components such as tumor-associated macrophages and vasculature. This approach, initially aimed at uncovering tumor invasive patterns and their interactions with the TME, holds promise for understanding additional cancer cell behaviors like intravasation and metastasis. BEHAV3D-TP represents a significant advancement in democratizing the analysis of heterogeneous cancer cell behaviors and their TME interactions, providing accessible computational insights into tumor dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/148dca420d0f76e0533267a0df34c8b8969cf9bd" target='_blank'>
              BEHAV3D Tumor Profiler to map heterogeneous cancer cell behavior in the tumor microenvironment
              </a>
            </td>
          <td>
            Emilio Rios-Jimenez, A. Zomer, R. Collot, Mario Barrera Román, H. Ariese, Ravian L. van Ineveld, Michiel Kleinnijenhuis, Nils Bessler, H. Johnson, A. Rios, Maria Alieva
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Quantitative assessment of multiplex immunofluorescence (mIF) data represents a powerful tool for immunotherapy biomarker discovery in melanoma and other solid tumors. In addition to providing detailed phenotypic information of immune cells of the tumor microenvironment, these datasets contain spatial information that can reveal biologically relevant interactions among cell types. To assess quantitative mIF analysis as a platform for biomarker discovery, we used a 12-plex mIF panel to characterize tumor samples collected from 50 patients with melanoma prior to treatment with immune checkpoint inhibitors (ICI). Consistent with prior studies, we identified a strong association between stromal B cell percentage and response to ICI therapy. We then compared pathologist assessment of lymphoid aggregates with a density based clustering algorithm, DBSCAN, to both automatically detect B cell aggregates and quantify their size, morphology, and distance to tumor. Spatial neighborhood analysis identified TCF1+ and LAG3- T cell subpopulations enriched near stromal B cells. These analyses provide a roadmap for the further development and validation of spatial immunotherapy biomarkers in melanoma and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cd6fec56a56340edd9c1c9c15dc8c5fb6df05ec" target='_blank'>
              Spatial assessment of stromal B cell aggregates predicts response to checkpoint inhibitors in unresectable melanoma
              </a>
            </td>
          <td>
            J. Smithy, X. Peng, F. D. Ehrich, A. P. Moy, M. Yosofvand, C. Maher, N. Aleynick, R. Vanguri, M. Zhuang, J. Lee, M. Bleile, Y. Li, M. Postow, K. Panageas, T. Hollmann, M. K. Callahan, R. Shen
          </td>
          <td>2024-08-10</td>
          <td>None</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Glioblastoma (GBM) is characterized by fast progression, an infiltrative growth pattern, and a high rate of relapse. A defining feature of GBM is the existence of spatially and functionally distinct cellular niches, i.e. a hypoxic niche, a leading-edge niche, and a perivascular niche, in which malignant cells engage in paracrine crosstalk with cell types comprising the tumor microenvironment. Here, by analysis of single-cell transcriptomic data of human GBM and transgenic mouse models of GBM, we unexpectedly identified pericytes, mural cells intimately associated with the endothelium, as the most active paracrine signaling hub within the tumor parenchyma. Exclusive signaling axes emanating from pericytes were received by endothelial cells, malignant cells, astrocytes, and immune cells. Depletion of pericytes through genetic engineering in several different transgenic and orthotopic mouse models of GBM demonstrated accelerated tumor progression, a disrupted blood-brain-barrier, and premature death of pericyte-poor mice. Mechanistic studies revealed that pericyte deficiency altered the cellular composition of GBM, remodeled the endothelium, and impacted on the immune cell landscape, exacerbating tumor cell invasion and immune suppression. Specifically, endothelial cells deprived of pericyte association altered their signaling programs, which in turn attracted perivascular, tumor-associated macrophages polarized towards an immune-suppressive phenotype. The recruited macrophages expressed Hepatocyte Growth Factor (HGF), which reinforced activation of its receptor tyrosine kinase MET on GBM cells harboring an extreme mesenchymal subtype driven by the key phenotypic regulator Fosl1 within hypoxic regions. Indeed, orthotopic implantation of isolated, MET-expressing GBM cells corroborated their superior tumor-initiating capability and invasive phenotype. In patients, low expression of a pericyte core gene signature was reduced in recurrent GBM, compared to primary tumors. Consistently, gene signatures for transcriptional programs of Fosl1+Met+ GBM cells were indicative of poor survival in human tumors, and spatial transcriptomics corroborated their superior invasive capacity. Taken together, we infer that the pericyte represents a critical modulator of GBM development by orchestrating a tumor-suppressive microenvironment; our findings thus highlight the importance of pericyte preservation in the face of current and future GBM therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1287c43b4f299f76e0c243131298cd37830d70a7" target='_blank'>
              Pericytes orchestrate a tumor-restraining microenvironment in glioblastoma
              </a>
            </td>
          <td>
            Sebastian Braun, Paulina Bolivar, Clara R. L. Oudenaarden, Jonas Sjölund, M. Bocci, Katja Harbst, Mehrnaz Safaee Talkhoncheh, Bengt Phung, E. Cordero, R. Rosberg, Elinn Johansson, Göran Jönsson, A. Pietras, Kristian Pietras
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The characteristics of the tumor microenvironment (TME) are closely linked to tumor progression and treatment response. The TME comprises various cell types, their spatial distribution, cell-cell interactions, and their organization into cellular niches or neighborhoods. To capture this complexity, several spatial profiling technologies have been developed. However, challenges such as low throughput, high costs, and complicated data analysis have limited their widespread use in immune research. In this study, we introduce the Cyclic-multiplex TSA (CmTSA) staining platform, a high-throughput superplex staining technology based on tyramide signal amplification (TSA) immunostaining combined with an efficient fluorophore recycling method. The CmTSA platform allows for the labeling of 30-60 antigens across multiple parallel formalin-fixed paraffin-embedded (FFPE) slides. Furthermore, the automated CmTSA workflow requires only standard histological equipment and conventional immunohistochemistry (IHC) primary antibodies (Abs), significantly reducing costs. While the superplex images produced contain extensive multidimensional information, extracting the spatial features of the TME from raw pixel data can be challenging. To address this, we present a computer vision-based analysis pipeline, which begins with deep learning-based algorithms to segment individual cells and identify cell types based on defined annotation rules. It then evaluates the spatial distribution tendencies of each cell type, the interaction intensity between paired cells, and the multicellular functional niches. This comprehensive approach enables researchers to visualize and quantify the types, states, and levels of immune activities within the TME effectively, advancing tumor immunology research and precision immune medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd339bff0e8e2231f57027618c2bef4a563bab3" target='_blank'>
              Pipeline for Assessing Tumor Immune Status Using Superplex Immunostaining and Spatial Immune Interaction Analysis
              </a>
            </td>
          <td>
            Chaoxin Xiao, Ruihan Zhou, Qin Chen, Wanting Hou, Xiaoying Li, Yulin Wang, Lu Liu, Huanhuan Wang, Xiaohong Yao, Tongtong Xu, Fujun Cao, Banglei Yin, Ouying Yan, Lili Jiang, Wei Wang, Dan Cao, Chengjian Zhao
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Senescence has been demonstrated to either inhibit or promote tumorigenesis. Resolving this paradox requires spatial mapping and functional characterization of senescent cells in the native tumor niche. Here, we identified senescent p16Ink4a+ cancer-associated fibroblasts with a secretory phenotype that promotes fatty acid uptake and utilization by aggressive lung adenocarcinoma driven by Kras and p53 mutations. Furthermore, rewiring of lung cancer metabolism by p16Ink4a+ cancer- associated fibroblasts also altered tumor cell identity to a highly plastic/dedifferentiated state associated with progression in murine and human LUAD. Our ex vivo senolytic screening platform identified XL888, a HSP90 inhibitor, that cleared p16Ink4a+ cancer- associated fibroblasts in vivo. XL888 administration after establishment of advanced lung adenocarcinoma significantly reduced tumor burden concurrent with the loss of plastic tumor cells. Our study identified a druggable component of the tumor stroma that fulfills the metabolic requirement of tumor cells to acquire a more aggressive phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6f71ba4dd04b12a2d236b3e450863e954b3784" target='_blank'>
              Senescent fibroblasts in the tumor stroma rewire lung cancer metabolism and plasticity
              </a>
            </td>
          <td>
            Jin Young Lee, Nabora S. Reyes, Sang-Ho Woo, Sakshi Goel, Fia Stratton, Chaoyuan Kuang, Aaron S. Mansfield, Lindsay M. LaFave, Tien Peng
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Secondary lymphoid organs (SLOs), including tonsils (TS), lymph nodes (LN), and Peyer's Patches, exhibit complementary immune functions. However, little is known about the spatial organization of immune cells and extracellular matrix (ECM) in the SLOs. Traditional imaging is limited to a few markers, confining our understanding of the differences between the SLOs. Herein, imaging mass cytometry addressed this gap by simultaneously profiling 25-plex proteins in SLO tissues at subcellular resolution. The antibody panel targeted immune, stromal, chemokine, epigenetic, and functional markers. For robust cell identification, a computational workflow SpatialVizPheno was developed to spatially phenotype 999,970 cells using two approaches, including manual gating and semi-supervised gating, iterative clustering, and annotation. LN exhibited the highest density of B cells while the intestinal tissues contained the highest proportion of regulatory and follicular helper T cells. SpatialVizPheno identified the most prevalent interaction between follicular dendritic cells and stromal cells (SCs), plasmablasts/plasma cells, and the SCs across the lymphoid tissues. Collagen-enriched regions were associated with the spatial orientation of B cell follicles in both TS and LN tissues, but not in intestinal lymphoid tissues. Such spatial differences of immunophenotypes and ECM in different SLO tissues can be used to quantify the relationship between cellular organization and ultimate immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7d9342b523b0b50e2fb3715a6d362425d2a168c" target='_blank'>
              Spatial immunophenotyping using multiplexed imaging of immune follicles in secondary lymphoid tissues
              </a>
            </td>
          <td>
            Mayar Allam, Thomas Hu, Zhou Fang, Michelle Pi, Ankur Singh, Ahmet F. Coskun
          </td>
          <td>2024-07-22</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Supratentorial ependymomas are aggressive childhood brain cancers that retain features of neurodevelopmental cell types and segregate into molecularly and clinically distinct subgroups, suggesting different developmental roots. The developmental signatures as well as microenvironmental factors underlying aberrant cellular transformation and behavior across each supratentorial ependymoma subgroup are unknown. Here we integrated single cell- and spatial transcriptomics, as well as in vitro and in vivo live-cell imaging to define supratentorial ependymoma cell states, spatial organization, and dynamic behavior within the neural microenvironment. We find that individual tumor subgroups harbor two distinct progenitor-like cell states reminiscent of early human brain development and diverge in the extent of neuronal or ependymal differentiation. We further uncover several modes of spatial organization of these tumors, including a high order architecture influenced by mesenchymal and hypoxia signatures. Finally, we identify an unappreciated role for brain-resident cells in shifting supratentorial ependymoma cellular heterogeneity towards neuronal-like cells that co-opt immature neuronal morphology and invasion mechanisms. Collectively, these findings provide a multidimensional framework to integrate transcriptional and phenotypic characterization of tumor heterogeneity in supratentorial ependymoma and its potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682f497ee480d416656ebedce292e26818b138e5" target='_blank'>
              Single-cell multidimensional profiling of tumor cell heterogeneity in supratentorial ependymomas
              </a>
            </td>
          <td>
            Daeun Jeong, Sara G Danielli, K. Maass, David R. Ghasemi, Svenja K. Tetzlaff, Ekin Reyhan, Carlos Alberto Oliveira de Biagi-Junior, S. Neyazi, Andrezza Nascimento, Rebecca D Haase, Costanza Lo Cascio, Bernhard Englinger, Li Jiang, Cuong Nguyen, A. Baumgartner, Sophia Castellani, Jacob S Rozowsky, Olivia A Hack, McKenzie L. Shaw, Daniela Lotsch-Gojo, K. Bruckner, Stefan M. Pfister, M. Kool, Tomasz J. Nowakowski, J. Gojo, Lissa Baird, S. Alexandrescu, K. Pajtler, V. Venkataramani, Mariella G. Filbin
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) resides in an immune rich microenvironment, yet, immune-based therapies have faltered in eliciting durable responses. Bridging this paradox requires a comprehensive understanding of leukemic interactions within the bone marrow microenvironment. We optimized a high-throughput tissue-microarray based pipeline for high-plex spatial immunofluorescence and mass cytometry imaging on a single slide, capturing immune, tumor, and structural components. Using unbiased clustering on the spatial K function, we unveiled the presence of tertiary lymphoid-like aggregates in bone marrow which we validated using spatial transcriptomics and an independent proteomics approach. We then found validated TLS signatures predictive of outcomes in AML using an integrated public 480 patient transcriptomic dataset. By harnessing high-plex spatial proteomics, we open the possibility of discovering of novel structures and interactions that underpin leukemic immune response. Further, our study’s methodologies and resources can be adapted for other bone marrow diseases where decalcification and autofluorescence present challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1b1bad3b36f52c6886c710f3a8b1e66619ef44" target='_blank'>
              Multimodal Spatial Proteomic Profiling in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Christopher Ly, Ivo Veletic, Christopher D. Pacheco, Enes Dasdemir, F. Jelloul, Sammy Ferri-Borgongo, Akshay Basi, Javier A. Gomez, Jessica L Root, P. Reville, Padmanee Sharma, Sreyashi Basu, Andres E. Quesada, Carlos Bueso-Ramos, T. Manshouri, Miriam Garcia, Jared K. Burks, Hussein A. Abbas
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Neuroendocrine tumors (NETs) occur primarily in the small intestine, lung and pancreas. Due to their rarity compared to other malignancies in these organs, their complex biology remains poorly understood, including their oncogenesis, tumor composition and the intriguing phenomena of mixed neuroendocrine non-neuroendocrine neoplasms (MiNEN). Here we profiled ten low-grade small intestine NET (SiNET) tumor samples as well as one mixed lung tumor by single-cell or single-nuclei RNA-seq. We find that SiNETs are largely separated into two distinct subtypes, in which the neuroendocrine cells upregulate epithelial or neuronal markers, respectively. Surprisingly, in both subtypes the neuroendocrine cells are largely non-proliferative while higher proliferation is observed in multiple non-malignant cell types. Specifically, B and plasma cells are highly proliferative in the epithelial-like SiNET subtype, potentially reflecting the outcome of high Migration Inhibitory Factor (MIF) expression in those tumors, which may constitute a relevant target. Finally, our analysis of a mixed lung neuroendocrine tumor identifies a population of putative progenitor cells that may give rise to both neuroendocrine and non-neuroendocrine (squamous) cells, potentially explaining the origin of the mixed histology. Taken together, our results provide important insights and hypotheses regarding the biology of neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b4bac5cd6807fff863bab344300f5dcfb31ca7" target='_blank'>
              Subtypes and proliferation patterns of small intestine neuroendocrine tumors revealed by single cell RNA sequencing
              </a>
            </td>
          <td>
            Einav Someach, Debdatta Halder, Avishay Spitzer, Chaya Barbolin, Michael Tyler, Reut Halperin, Moshe Biton, Amit Tirosh, I. Tirosh
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59fae692b36b9cb8dfff150568e46638865d4946" target='_blank'>
              Single-cell and spatial proteo-transcriptomic profiling reveals immune infiltration heterogeneity associated with neuroendocrine features in small cell lung cancer
              </a>
            </td>
          <td>
            Ying Jin, Yuefeng Wu, Alexandre Reuben, Liang Zhu, C. Gay, Qingzhe Wu, Xintong Zhou, Haomin Mo, Qi Zheng, Junyu Ren, Zhaoyuan Fang, Teng Peng, Nan Wang, Liang Ma, Yun Fan, Hai Song, Jianjun Zhang, Ming Chen
          </td>
          <td>2024-09-04</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b2b2736cd24fffcca3410f55115541def7e515" target='_blank'>
              METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics
              </a>
            </td>
          <td>
            Jiahui Jiang, Yunhe Liu, Jiang-Jiang Qin, Jianfeng Chen, JingJing Wu, Melissa P Pizzi, R. Lazcano, Kohei Yamashita, Zhiyuan Xu, Guangsheng Pei, K. S. Cho, Yanshuo Chu, Ansam Sinjab, F. Peng, Xinmiao Yan, Guangchun Han, Ruiping Wang, Enyu Dai, Yibo Dai, Bogdan Czerniak, Andrew Futreal, Anirban Maitra, Alexander Lazar, H. Kadara, A. Jazaeri, Xiangdong Cheng, Jaffer J. Ajani, Jianjun Gao, Jian Hu, Linghua Wang
          </td>
          <td>2023-10-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Notch signaling can have either an oncogenic or tumor-suppressive function in cancer depending on the cancer type and cellular context. While Notch can be oncogenic in early prostate cancer, we identified significant downregulation of the Notch pathway during prostate cancer progression from adenocarcinoma to neuroendocrine (NE) prostate cancer, where it functions as a tumor suppressor. Activation of Notch in NE and Rb1/Trp53-deficient prostate cancer models led to phenotypic conversion toward a more indolent, non-NE state with glandular features and expression of luminal lineage markers. This was accompanied by upregulation of MHC and type I IFN and immune cell infiltration. Overall, these data support Notch signaling as a suppressor of NE differentiation in advanced prostate cancer and provide insights into how Notch signaling influences lineage plasticity and the tumor microenvironment (TME).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de21afbb6c368d901e1bd4e6360df58ff3e7cece" target='_blank'>
              Notch signaling suppresses neuroendocrine differentiation and alters the immune microenvironment in advanced prostate cancer
              </a>
            </td>
          <td>
            S. Ku, Yanqing Wang, Maria Mica Garcia, Yasutaka Yamada, Kei Mizuno, M. Long, Spencer Rosario, M. Chinnam, Majd Al Assaad, L. Puca, Min Jin Kim, Martin K. Bakht, V. B. Venkadakrishnan, Brian D. Robinson, Andrés M Acosta, K. Wadosky, Juan Miguel Mosquera, David W. Goodrich, H. Beltran
          </td>
          <td>2024-07-18</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81ef2d83f1b6031cda6c0a9dde272764ae505dda" target='_blank'>
              Single-cell tumor heterogeneity landscape of hepatocellular carcinoma: unraveling the pro-metastatic subtype and its interaction loop with fibroblasts
              </a>
            </td>
          <td>
            De-Zhen Guo, Xin Zhang, Sen-Quan Zhang, Shi-Yu Zhang, Xiangyu Zhang, Jialun Yan, San-Yuan Dong, Kai Zhu, Xinchun Yang, Jia Fan, Jian Zhou, Ao Huang
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Motivation Analyzing the local microenvironment of tumor cells can provide significant insights into their complex interactions with their cellular surroundings, including immune cells. By quantifying the prevalence and distances of certain immune cells in the vicinity of tumor cells through a neighborhood analysis, patterns may emerge that indicate specific associations between cell populations. Such analyses can reveal important aspects of tumor-immune dynamics, which may inform therapeutic strategies. This method enables an in-depth exploration of spatial interactions among different cell types, which is crucial for research in oncology, immunology, and developmental biology. Results We introduce an R Markdown script called SNAQTM (Single-cell Spatial Neighborhood Analysis and Quantification), which conducts a neighborhood analysis on immunofluorescent images without the need for extensive coding knowledge. As a demonstration, SNAQTM was used to analyze images of pancreatic ductal adenocarcinoma. Samples stained for DAPI, PanCK, CD68, and PD-L1 were segmented and classified using QuPath. The resulting CSV files were exported into RStudio for further analysis and visualization using SNAQTM. Visualizations include plots revealing the cellular composition of neighborhoods around multiple cell types within a customizable radius. Additionally, the analysis includes measuring the distances between cells of certain types relative to others across multiple regions of interest. Availability and implementation The R Markdown files that comprise the SNAQTM algorithm and the input data from this paper are freely available on the web at https://github.com/AryehSilver1/SNAQ. Visual Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/536ae41dfee8df569ec9d770e09c6f0416cb7a5b" target='_blank'>
              Get to know your neighbors with a SNAQ™: A framework for single cell spatial neighborhood analysis in immunohistochemical images
              </a>
            </td>
          <td>
            Aryeh Silver, Avirup Chakraborty, Avinash Pittu, Diana Feier, Miruna Anica, Illeana West, Matthew R. Sarkisian, L. Deleyrolle
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b45d887e551ac160bbe696e1a2e41877583a53a" target='_blank'>
              Single-cell resolution profiling of the immune microenvironment in primary and metastatic nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Qiuping Liu, Jingping Xu, Bingyi Dai, Danni Guo, Changling Sun, Xiaodong Du
          </td>
          <td>2024-08-19</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="Despite promising results with immune checkpoint blockade (ICB) therapy, outcomes of patients with brain metastasis (BrM) remain poor. Identifying determinant of resistance is crucial but has been hindered by limited access to patient samples and the scarcity of relevant preclinical models. Here, we developed two mouse melanoma BrM models that mirror the disparate response to ICB seen in patients. By characterizing their mutational landscape and performing single-cell phenotypic and transcriptomic analysis, we demonstrated that these models recapitulate both the cellular and molecular features of human disease. Through our comparative analysis, we identified key factors contributing to ICB response. In the responsive model, we found that tumor cells expressed inflammatory programs that polarized microglia toward reactive states, expressing high levels of MHC-I and -II and immunostimulatory molecules, eliciting robust T cell recruitment and activation, which favored anti-tumor immunity. Conversely, the resistant melanoma cells were characterized by neurological molecular signatures and expression of ligands that maintain microglia homeostatic states, resulting in poor T cell infiltration and therapy resistance. We validated our findings in BrM patients, showing that BR1 and BR3 signatures correlated with T cell infiltration and were associated with better and worse patient outcomes, respectively. Our study fills a critical gap by providing clinically relevant models and revealing mechanistic insights into BrM ICB responses, identifying potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b06309c3c54287af3d2a6158bdb011f7654e629" target='_blank'>
              Brain-Metastatic Melanoma Models Uncover Immune and Molecular Correlates of Response to Immunotherapy
              </a>
            </td>
          <td>
            Amélie Daugherty-Lopès, Eva Pérez-Guijarro, Vishaka Gopalan, Jessica Rappaport, Quanyi Chen, April Huang, Khiem C. Lam, Sung Chin, Jessica Ebersole, Emily Wu, Gabe Needle, Isabella Church, George Kyriakopoulos, Yongmei Zhao, Charli Gruen, Antonella Sassano, Romina E. Araya, Andres Thorkelsson, Cari Smith, M. Lee, Sridhar Hannenhalli, Chi-Ping Day, Glenn Merlino, R. Goldszmid
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, Andreas Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Identifying the key molecular pathways that enable metastasis by analyzing the eventual metastatic tumor is challenging because the state of the founder subclone likely changes following metastatic colonization. To address this challenge, we labeled primary mouse pancreatic ductal adenocarcinoma (PDAC) subclones with DNA barcodes to characterize their pre-metastatic state using ATAC-seq and RNA-seq and determine their relative in vivo metastatic potential prospectively. We identified a gene signature separating metastasis-high and metastasis-low subclones orthogonal to the normal-to-PDAC and classical-to-basal axes. The metastasis-high subclones feature activation of IL-1 pathway genes and high NF-κB and Zeb/Snail family activity and the metastasis-low subclones feature activation of neuroendocrine, motility, and Wnt pathway genes and high CDX2 and HOXA13 activity. In a functional screen, we validated novel mediators of PDAC metastasis in the IL-1 pathway, including the NF-κB targets Fos and Il23a, and beyond the IL-1 pathway including Myo1b and Tmem40. We scored human PDAC tumors for our signature of metastatic potential from mouse and found that metastases have higher scores than primary tumors. Moreover, primary tumors with higher scores are associated with worse prognosis. We also found that our metastatic potential signature is enriched in other human carcinomas, suggesting that it is conserved across epithelial malignancies. This work establishes a strategy for linking cancer cell state to future behavior, reveals novel functional regulators of PDAC metastasis, and establishes a method for scoring human carcinomas based on metastatic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130863ca923658f30d397c218a109254f4ad9093" target='_blank'>
              Identifying a gene signature of metastatic potential by linking pre-metastatic state to ultimate metastatic fate
              </a>
            </td>
          <td>
            Jesse S Handler, Zijie Li, Rachel K Dveirin, Weixiang Fang, Hani Goodarzi, E. Fertig, Reza Kalhor
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04093c08888f78b5e8c1e948744775d304ab8d34" target='_blank'>
              Decoding the spatiotemporal heterogeneity of tumor-associated macrophages
              </a>
            </td>
          <td>
            Xiangyuan Chu, Yu Tian, Chao Lv
          </td>
          <td>2024-07-27</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aa9b0e609eead675b8ed0662ea364a8b1ad7f" target='_blank'>
              Tumor-associated macrophage clusters linked to immunotherapy in a pan-cancer census
              </a>
            </td>
          <td>
            Chen Wei, Yijie Ma, Mengyu Wang, Siyi Wang, Wenyue Yu, Shuailei Dong, Wenying Deng, L. Bie, Chi Zhang, Wei Shen, Qingxin Xia, Suxia Luo, Ning Li
          </td>
          <td>2024-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a lethal brain cancer with no effective treatment; understanding how GBM cells respond to tumor microenvironment remains challenging as conventional cell cultures lack proper cytoarchitecture while in vivo animal models present complexity all at once. Developing a culture system to bridge the gap is thus crucial. Here, we employed a multicellular approach using human glia and vascular cells to optimize a 3-dimensional (3D) brain vascular niche model that enabled not only long-term culture of patient derived GBM cells but also recapitulation of key features of GBM heterogeneity, in particular invasion behavior and vascular association. Comparative transcriptomics of identical patient derived GBM cells in 3D and in vivo xenotransplants models revealed that glia-vascular contact induced genes concerning neural/glia development, synaptic regulation, as well as immune suppression. This gene signature displayed region specific enrichment in the leading edge and microvascular proliferation zones in human GBM and predicted poor prognosis. Gene variance analysis also uncovered histone demethylation and xylosyltransferase activity as main themes for gene adaption of GBM cells in vivo. Furthermore, our 3D model also demonstrated the capacity to provide a quiescence and a protective niche against chemotherapy. In summary, an advanced 3D brain vascular model can bridge the gap between 2D cultures and in vivo models in capturing key features of GBM heterogeneity and unveil previously unrecognized influence of glia-vascular contact for transcriptional adaption in GBM cells featuring neural/synaptic interaction and immunosuppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d6f0191f7de29cccb27dddaa61e12cbec1517e" target='_blank'>
              3D Brain Vascular Niche Model Captures Invasive Behavior and Gene Signatures of Glioblastoma
              </a>
            </td>
          <td>
            Vivian K. Lee, R. Tejero, Nathaniel Silvia, Anirudh Sattiraju, Aarthi Ramakrishnan, Li Shen, Alexandre Wojcinski, Santosh Kesari, R. Friedel, Hongyan Zou, Guohao Dai
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor heterogeneity refers to the diversity observed among tumor cells: both between different tumors (inter-tumor heterogeneity) and within a single tumor (intra-tumor heterogeneity). These cells can display distinct morphological and phenotypic characteristics, including variations in cellular morphology, metastatic potential and variability treatment responses among patients. Therefore, a comprehensive understanding of such heterogeneity is necessary for deciphering tumor-specific mechanisms that may be diagnostically and therapeutically valuable. Innovative and multidisciplinary approaches are needed to understand this complex feature. In this context, proteogenomics has been emerging as a significant resource for integrating omics fields such as genomics and proteomics. By combining data obtained from both Next-Generation Sequencing (NGS) technologies and mass spectrometry (MS) analyses, proteogenomics aims to provide a comprehensive view of tumor heterogeneity. This approach reveals molecular alterations and phenotypic features related to tumor subtypes, potentially identifying therapeutic biomarkers. Many achievements have been made; however, despite continuous advances in proteogenomics-based methodologies, several challenges remain: in particular the limitations in sensitivity and specificity and the lack of optimal study models. This review highlights the impact of proteogenomics on characterizing tumor phenotypes, focusing on the critical challenges and current limitations of its use in different clinical and preclinical models for tumor phenotypic characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/958a7a3c1fd552ec9c7dc2019625a3d4361653d4" target='_blank'>
              Phenotyping Tumor Heterogeneity through Proteogenomics: Study Models and Challenges
              </a>
            </td>
          <td>
            Diletta Piana, Federica Iavarone, E. De Paolis, Gennaro Daniele, Federico Parisella, Angelo Minucci, V. Greco, Andrea Urbani
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Intraductal papillary mucinous neoplasms (IPMN) is one of the known precancerous lesions. Patients’ prognosis is aggravated as IPMN transforms into invasive Pancreatic Ductal Adenocarcinoma (PDAC). The molecular mechanisms underlying this progression lack effective experimental models and urgently need to be elaborated. We performed spatial transcriptomics (ST) on fresh tissue samples from the same patient including normal pancreas, high-grade IPMN, and invasive PDAC, and described the step-by-step development of transcriptional landscape including clone evolution and adjacent TME feature variation. Our findings identified the master transcript factors and critical signaling pathways promoting IPMN progression to invasive PDAC. Additionally, both IPMN and PDAC harbored the ELF3, MYC, and KLF4 amplification. The Spatial CNV profile demonstrated significant heterogeneity among PDAC in their spatial distribution compared to IPMN, with seven distinct subclones showing diverse functions, such as hypoxia, oxidative phosphorylation, and epithelial-mesenchymal Transition. We observed a marked shift in the immune landscape, with the depletion of CD4+ T-cells and dendritic cells and an increase in immune-suppressive M2 macrophages in invasive PDAC, indicating a transition to an immune-evasive microenvironment. Additionally, cancer-associated fibroblasts (CAFs), particularly myofibroblastic CAFs, were enriched adjacent to invasive PDAC, suggesting their active role in tumor progression. By leveraging spatial transcriptomic analysis, our study provides comprehensive insights into the intricate molecular landscape that underlies the progression of IPMNs to invasive PDAC. These findings not only enhance our understanding of this complex process but also offer valuable knowledge for early diagnosis and intervention. Highlights Spatial CNV analysis reveals clonal evolution and distinct subclones in PDAC. Key drivers like ELF3, MYC, and KLF4 are amplified in both IPMN and invasive PDAC. Immune landscape shifts from pro-inflammatory in IPMN to immune-evasive in PDAC. Enrichment of myofibroblastic CAFs suggests their role in tumor progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4caecc6532101fd5cd79dde472c73a99ffa9c7b0" target='_blank'>
              Evolution of the Spatial transcriptomic landscape during the progression of high-grade pancreatic intraductal papillary mucinous neoplasms to invasive cancer
              </a>
            </td>
          <td>
            Sirui Peng, Qiangxing Chen, Zixin Chen, Mengling Yao, Yu Cai, Du He, Yu Cai, K. Cheng, Jun Li, H. Cai, P. Gao, Xiafei Gu, Xin Wang, Yongbin Li, Man Zhang, L. Meng, Qi Xia, Panpan Xu, Jin Zhou, Zhong Wu, Bing Peng
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) displays an infiltrative growth characteristic that recruits neighboring normal cells to facilitate tumor growth, maintenance, and invasion into the brain. While the blood-brain barrier serves as a critical natural defense mechanism for the central nervous system, GBM disrupts this barrier, resulting in the infiltration of macrophages from the peripheral bone marrow and the activation of resident microglia. Recent advancements in single-cell transcriptomics and spatial transcriptomics have refined the categorization of cells within the tumor microenvironment for precise identification. The intricate interactions and influences on cell growth within the tumor microenvironment under multi-omics conditions are succinctly outlined. The factors and mechanisms involving microglia, macrophages, endothelial cells, and T cells that impact the growth of GBM are individually examined. The collaborative mechanisms of tumor cell-immune cell interactions within the tumor microenvironment synergistically promote the growth, infiltration, and metastasis of gliomas, while also influencing the immune status and therapeutic response of the tumor microenvironment. As immunotherapy continues to progress, targeting the cells within the inter-tumor microenvironment emerges as a promising novel therapeutic approach for GBM. By comprehensively understanding and intervening in the intricate cellular interactions within the tumor microenvironment, novel therapeutic modalities may be developed to enhance treatment outcomes for patients with GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7eb60b2084b150ce4896eaa955b9d8cf4d435d1" target='_blank'>
              Advancing precision medicine in gliomas through single-cell sequencing: unveiling the complex tumor microenvironment
              </a>
            </td>
          <td>
            Jinwei Li, Yang Zhang, Cong Liang, Xianlei Yan, Xuhui Hui, Quan Liu
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, Tae Kyung Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="We present GBM-cRegMap, an online resource providing a comprehensive coregulatory influence network perspective on glioblastoma (GBM) heterogeneity and plasticity. Using representation learning algorithms, we derived two components of this resource: GBM-CoRegNet, a highly specific coregulatory network of tumor cells, and GBM-CoRegMap, a unified network influence map based on 1612 tumors from 16 studies. As a widely applicable closed-loop system connecting cellular models and tumors, GBM-cRegMap will provide the GBM research community with an easy-to-use web tool (https://gbm.cregmap.com) that maps any existing or newly generated transcriptomic “query” data to a reference coregulatory network and a large-scale manifold of disease heterogeneity. Using GBM-cRegMap, we demonstrated the synergy between the two components by refining the molecular classification of GBM, identifying potential key regulators, and aligning the transcriptional profiles of tumors and in vitro models. Through the amalgamation of a vast dataset, we validated the proneural (PN)-mesenchymal (MES) axis and identified three subclasses of classical (CL) tumors: astrocyte-like (CL-A), epithelial basal-like (CL-B), and cilium-rich (CL-C). We revealed the CL-C subclass, an intermediate state demonstrating the plasticity of GBM cells along the PN-MES axis under chemotherapy. We identified key regulators, such as PAX8, and NKX2.5, involved in TMZ resistance. Notably, NKX2.5, more expressed in higher-grade gliomas, negatively impacts patient survival and regulates genes involved in glucose metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd1c463b43fec2813fd1baca33de19d265544140" target='_blank'>
              A coregulatory influence map of glioblastoma heterogeneity and plasticity – From cells in vitro to tumors and back again
              </a>
            </td>
          <td>
            Chloé Bernhard, Konstantinos Geles, Geoffrey Pawlak, Wajdi Dhifli, Aurélien Dispot, Jules Dusol, Maria Kondratova, Sophie Martin, M. Messé, Damien Reita, David Tulasne, I. Seuningen, N. Entz-Werlé, S. Ciafrè, Monique Dontenwill, Mohamed Elati
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The study presents a novel approach for quantifying cellular interactions in digital pathology using deep learning-based image cytometry. Traditional methods struggle with the diversity and heterogeneity of cells within tissues. To address this, we introduce the Spatial Interaction Potential (SIP) and the Co-Localization Index (CLI), leveraging deep learning classification probabilities. SIP assesses the potential for cell-to-cell interactions, similar to an electric field, while CLI incorporates distances between cells, accounting for dynamic cell movements. Our approach enhances traditional methods, providing a more sophisticated analysis of cellular interactions. We validate SIP and CLI through simulations and apply them to colorectal cancer specimens, demonstrating strong correlations with actual biological data. This innovative method offers significant improvements in understanding cellular interactions and has potential applications in various fields of digital pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949ba712b098fead7bc4bf83be6ad4882df65fb2" target='_blank'>
              Novel Methods for Analyzing Cellular Interactions in Deep Learning-Based Image Cytometry: Spatial Interaction Potential and Co-Localization Index
              </a>
            </td>
          <td>
            Toru Nagasaka, Kimihiro Yamashita, Mitsugu Fujita
          </td>
          <td>2024-08-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer for which few effective therapies exist. Immunotherapies specifically are ineffective in pancreatic cancer, in part due to its unique stromal and immune microenvironment. Pancreatic intraepithelial neoplasia, or PanIN, is the main precursor lesion to PDAC. Recently it was discovered that PanINs are remarkably abundant in the grossly normal pancreas, suggesting that the vast majority will never progress to cancer. Here, through construction of 48 samples of cm3-sized human pancreas tissue, we profiled the immune microenvironment of 1,476 PanINs in 3D and at single-cell resolution to better understand the early evolution of the pancreatic tumor microenvironment and to determine how inflammation may play a role in cancer progression. We found that bulk pancreatic inflammation strongly correlates to PanIN cell fraction. We found that the immune response around PanINs is highly heterogeneous, with distinct immune hotspots and cold spots that appear and disappear in a span of tens of microns. Immune hotspots generally mark locations of higher grade of dysplasia or locations near acinar atrophy. The immune composition at these hotspots is dominated by naïve, cytotoxic, and regulatory T cells, cancer associated fibroblasts, and tumor associated macrophages, with little similarity to the immune composition around less-inflamed PanINs. By mapping FOXP3+ cells in 3D, we found that regulatory T cells are present at higher density in larger PanIN lesions compared to smaller PanINs, suggesting that the early initiation of PanINs may not exhibit an immunosuppressive response. This analysis demonstrates that while PanINs are common in the pancreases of most individuals, inflammation may play a pivotal role, both at the bulk and the microscopic scale, in demarcating regions of significance in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/194ae57756ba7688c48e283a00a59550489311e0" target='_blank'>
              3D histology reveals that immune response to pancreatic precancers is heterogeneous and depends on global pancreas structure
              </a>
            </td>
          <td>
            A. Kiemen, Cristina Almagro-Pérez, Valentina Matos, André Forjaz, Alicia M. Braxton, Lucie Dequiedt, Jeeun Parksong, Courtney Cannon, Xuan Yuan, Sarah M. Shin, Jaanvi Mahesh Babu, Elizabeth D. Thompson, T. Cornish, W. Ho, Laura D. Wood, Pei-Hsun Wu, Arrate Muñoz Barrutia, Ralph H. Hruban, Denis Wirtz
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer with a poor survival rate. Despite hundreds of clinical trials, there is no effective targeted therapy. Glioblastoma stem cells (GSCs) are an important GBM model system. In culture, these cells form spatial structures that share morphological aspects with their source tumors. We collected 17,000 phase contrast images of 15 patient-derived GSC lines growing to confluence. We find that GSCs grow in characteristic multicellular patterns depending on their transcriptional state. Interpretable computer vision algorithms identified specific image features that predict transcriptional state across multiple cell confluency levels. This relationship will be useful in developing GSC screens where image features can be used to identify how GSC biology changes in response to perturbations simply by imaging cultured cells on plates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1043d1e1cafd115cae393754618f9b76789fb01" target='_blank'>
              Glioblastoma stem cells show transcriptionally correlated spatial organization
              </a>
            </td>
          <td>
            Shamini Ayyadhury, Patty Sachamitr, M. Kushida, Nicole I Park, F. Coutinho, Owen Whitley, Panagiotis Prinos, Cheryl H. Arrowsmith, Peter B. Dirks, Trevor Pugh, Gary D. Bader
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Background Gastric cancer brain metastasis (GCBM) represents a rare but highly aggressive malignancy. Metastatic cancer cells are highly heterogeneous and differentially remodels brain vasculature and immune microenvironments, which affects the treatment effectiveness and patient outcome. This study aimed to investigate the spatial interactions among different cell components, especially the vasculature system and the brain microenvironment of GCBM patients. Methods We used digital spatial profiling to examine 140 regions composing tumor, immune, and brain tissues from three GCBM patients. Transcriptomic data with spatial information were analyzed for tissue areas related to different blood recruitment strategies. For validation, independent analysis of patient bulk transcriptomic data and in vivo single-cell transcriptomic data were performed. Results Angiogenesis and blood vessel co-option co-existed within the same GCBM lesion. Tumors with high epithelial-mesenchymal transition and an enhanced transcriptomic gene signature composed of CTNNB1, SPARC, VIM, SMAD3, SMAD4, TGFB1, TGFB2, and TGFB3 were more prone to adopt blood vessel co-option than angiogenesis. Enriched macrophage infiltration, angiogenic chemokines, and NAMPT were found in angiogenic areas, while increased T cells, T cell activating cytokines, and reduced NAMPT were found in vessel co-option regions. Spatially, angiogenesis was enriched at the tumor edge, which showed higher DMBT1 expression than the tumor center. Conclusions This study mapped the orchestrated spatial characteristics of tumor and immunological compositions that support the conventional and atypical vascularization strategies in GCBM. Our data provided molecular insights for more effective combinations of anti-vascular and immune therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd5b21775a9a6dd1dd682d9fd31f88883a74f9" target='_blank'>
              Spatial transcriptomics of gastric cancer brain metastasis reveals atypical vasculature strategies with supportive immune profiles
              </a>
            </td>
          <td>
            Kaijing Liu, Ying Wang, Chunhua Wang, Chengcheng Guo, Dun Zhang, Yu Zhong, Lin Yin, Yunxin Lu, Furong Liu, Yang Zhang, Dongsheng Zhang
          </td>
          <td>2024-07-18</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function.
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667052bc6e34331d4c683bafe12d9a9b03403b8b" target='_blank'>
              Single-cell landscape of intratumoral heterogeneity and tumor microenvironment remolding in pre-nodal metastases of breast cancer
              </a>
            </td>
          <td>
            Kaidong Liu, Huiming Han, Kai Xiong, Songmei Zhai, Xiuqi Yang, Xinmiao Yu, Bo Chen, Mingyue Liu, Qi Dong, Hongxue Meng, Yunyan Gu
          </td>
          <td>2024-08-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Discovering the role of fibroblasts residing in the tumor microenvironment (TME) requires controlled, localized perturbations because fibroblasts play critical roles in regulating immunity and tumor biology at multiple sites. Systemic perturbations can lead to unintended, confounding secondary effects, and methods to locally genetically engineer fibroblasts are lacking. To specifically investigate murine stromal cell perturbations restricted to the TME, we developed an adeno-associated virus (AAV)-based method to target any gene-of-interest in fibroblasts at high efficiency (>80%). As proof of concept, we generated single (sKO) and double gene KOs (dKO) of Osmr, Tgfbr2, and Il1r1 in cancer-associated fibroblasts (CAFs) and investigated how their cell states and those of other cells of the TME subsequently change in mouse models of melanoma and pancreatic ductal adenocarcinoma (PDAC). Furthermore, we developed an in vivo knockin-knockout (KIKO) strategy to achieve long-term tracking of CAFs with target gene KO via knocked-in reporter gene expression. This validated in vivo gene editing toolbox is fast, affordable, and modular, and thus holds great potential for further exploration of gene function in stromal cells residing in tumors and beyond. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7267ed0c36ee868cfaeb352a86d2167c2c260e" target='_blank'>
              Localized in vivo gene editing of murine cancer-associated fibroblasts
              </a>
            </td>
          <td>
            N. Kuhn, Itzia Zaleta-Linares, William A. Nyberg, Justin Eyquem, M. Krummel
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="98% of T cells reside in tissues, yet nearly all human T cell analyses are performed from peripheral blood. We single-cell sequenced 5.7 million T cells from ten donors’ autologous blood and tonsils and sought to answer key questions about T cell receptor biology previously unanswerable by smaller-scale experiments. We identified distinct clonal expansions and distributions in blood compared to tonsils, with surprisingly low (1-7%) clonal sharing. These few shared clones exhibited divergent phenotypes across bodily sites. Analysis of antigen-specific CD8 T cells revealed location as a main determinant of frequency, phenotype, and immunodominance. Finally, diversity estimates from the tissue recalibrates current repertoire diversity estimates, and we provide a refined estimate of whole-body repertoire. Given the tissue-restricted nature of T cell phenotypes, functions, differentiation, and clonality revealed by this dataset, we conclude that tissue analyses are crucial for accurate repertoire analysis and monitoring changes after perturbing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8645a932ebb1aa3bc8da664348a030e7cf1f0fd4" target='_blank'>
              Tissue determinants of the human T cell receptor repertoire
              </a>
            </td>
          <td>
            Suhas Sureshchandra, James Henderson, Elizabeth Levendosky, Sankalan Bhattacharyya, Jenna M Kastenschmidt, Andrew Sorn, Mahina Tabassum Mitul, Aviv I. Benchorin, Kyle Batucal, Allyssa Daugherty, Samuel JH Murphy, Chandrani Thakur, Douglas Trask, Gurpreet Ahuja, Qiu Zhong, Annie Moisan, Andreas Tiffeau-Mayer, N. Saligrama, Lisa E. Wagar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Heterogeneity of tumor metabolism is an important, but still poorly understood aspect of tumor biology. Present work is focused on the visualization and quantification of cellular metabolic heterogeneity of colorectal cancer using fluorescence lifetime imaging (FLIM) of redox cofactor NAD(P)H. FLIM-microscopy of NAD(P)H was performed in vitro in four cancer cell lines (HT29, HCT116, CaCo2 and CT26), in vivo in the four types of colorectal tumors in mice and ex vivo in patients’ tumor samples. The dispersion and bimodality of the decay parameters were evaluated to quantify the intercellular metabolic heterogeneity. Our results demonstrate that patients’ colorectal tumors have significantly higher heterogeneity of energy metabolism compared with cultured cells and tumor xenografts, which was displayed as a wider and frequently bimodal distribution of a contribution of a free (glycolytic) fraction of NAD(P)H within a sample. Among patients’ tumors, the dispersion was larger in the high-grade and early stage ones, without, however, any association with bimodality. These results indicate that cell-level metabolic heterogeneity assessed from NAD(P)H FLIM has a potential to become a clinical prognostic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d39beee686da3cb9d1fa5f224388f8f6c20fccf" target='_blank'>
              Insights into metabolic heterogeneity of colorectal cancer gained from fluorescence lifetime imaging
              </a>
            </td>
          <td>
            A. D. Komarova, S. D. Sinyushkina, Ilia D Shchechkin, I. Druzhkova, Sofia A Smirnova, Vitaliy M Terekhov, A. Mozherov, Nadezhda I. Ignatova, Elena E. Nikonova, E. Shirshin, L. Shimolina, Sergey V Gamayunov, V. Shcheslavskiy, M. Shirmanova
          </td>
          <td>2024-08-28</td>
          <td>eLife</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) tumor heterogeneity impedes the development of biomarker assays suitable for early disease detection that would improve patient outcomes. The CA19-9 glycan is currently used as a standalone biomarker for PDAC. Furthermore, previous studies have shown that cancer cells may display aberrant membrane-associated glycans. We therefore hypothesized that PDAC cancer cell subpopulations could be distinguished by aberrant glycan signatures. We used multiplexed glycan immunofluorescence combined with pathologist annotation and automated image processing to distinguish between PDAC cancer cell subpopulations within tumor tissue. Using a training-set/test-set approach, we found that PDAC cancer cells may be identified by signatures comprising 4 aberrant glycans (VVL, CA19-9, sTRA, and GM2) and that there are three glycan-defined PDAC tumor types: sTRA type, CA19-9 type, and intermixed. To determine whether the aberrant glycan signatures could be detected in blood samples, we developed hybrid glycan sandwich assays for membrane-associated glycans. In both patient-matched tumor and blood samples, the proportion of aberrant glycans detected was consistent. Furthermore, our multiplexed glycan immunofluorescent approach proved to be more sensitive and more specific than CA19-9 alone. Our results provide proof of concept for a novel methodology to improve early PDAC detection and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6b226460c28415c919ed9fc15c5b9945dc42aa" target='_blank'>
              Multiplexed Glycan Immunofluorescence Identification of Pancreatic Cancer Cell Subpopulations in Both Tumor and Blood Samples
              </a>
            </td>
          <td>
            Braelyn Binkowski, Zachary L. Klamer, ChongFeng Gao, Ben Staal, Anna Repesh, Hoang-Le Tran, David M. Brass, Pamela Bartlett, Steven Gallinger, Maria Blomqvist, J. B. Morrow, Peter Allen, Chanjuan Shi, A. Singhi, Randall Brand, Ying Huang, G. Hostetter, Brian Haab
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b99f7435300e4c22850539415838d54ebcfa2c5" target='_blank'>
              Spatial dynamics of tertiary lymphoid aggregates in head and neck cancer: insights into immunotherapy response
              </a>
            </td>
          <td>
            Habib Sadeghirad, J. Monkman, Chin Wee Tan, Ning Liu, J. Yunis, Meg L. Donovan, Afshin Moradi, N. Jhaveri, Christopher Perry, Mark N Adams, K. O’Byrne, M. Warkiani, Rahul Ladwa, Brett Hughes, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Although localized prostate cancer is relatively indolent, advanced prostate cancer manifests with aggressive and often lethal variants, including neuroendocrine prostate cancer (NEPC). To identify drivers of aggressive prostate cancer, we leveraged Sleeping Beauty (SB) transposon mutagenesis in a mouse model based on prostate-specific loss-of-function of Pten and Tp53. Compared with control mice, SB mice developed more aggressive prostate tumors, with increased incidence of metastasis. Notably, a significant percentage of the SB prostate tumors display NEPC phenotypes, and the transcriptomic features of these SB mouse tumors recapitulated those of human NEPC. We identified common SB transposon insertion sites (CIS) and prioritized associated CIS-genes differentially expressed in NEPC versus non-NEPC SB tumors. Integrated analysis of CIS-genes encoding for proteins representing upstream, post-translational modulators of master regulators controlling the transcriptional state of SB-mouse and human NEPC tumors identified sirtuin 1 (Sirt1) as a candidate mechanistic determinant of NEPC. Gain-of-function studies in human prostate cancer cell lines confirmed that SIRT1 promotes NEPC, while its loss-of-function or pharmacological inhibition abrogates NEPC. This integrative analysis is generalizable and can be used to identify novel cancer drivers for other malignancies. Summary Using an unbiased forward mutagenesis screen in an autochthonous mouse model, we have investigated mechanistic determinants of aggressive prostate cancer. SIRT1 emerged as a key regulator of neuroendocrine prostate cancer differentiation and a potential target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef638b5be4a6bc97dbe24dca3c6171d01e593aa" target='_blank'>
              A forward genetic screen identifies Sirtuin1 as a driver of neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Francisca Nunes de Almeida, A. Vasciaveo, Ainsley Mike Antao, Min Zou, Matteo Di Bernardo, Simone de Brot, A. Rodriguez-Calero, Alexander Chui, Alexander L.E. Wang, Nicolas Floc’h, Jaime Y Kim, Stephanie N Afari, Timur Mukhammadov, Juan M Arriaga, Jinqiu Lu, Michael M. Shen, Mark A. Rubin, Andrea Califano, Cory Abate-Shen
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest-derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma.
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell techniques have convincingly demonstrated that tumor tissue usually contains multiple genetically defined cell subclones with different gene mutation sets as well as various transcriptional profiles, but the spatial heterogeneity of the microenvironment and the macrobiological characteristics of the tumor ecosystem have not been described. For the past few years, spatial multi-omics technologies have revealed the cellular interactions, microenvironment, and even systemic tumor-host interactions in the tumor ecosystem at the spatial level, which can not only improve classical therapies such as surgery, radiotherapy, and chemotherapy but also promote the development of emerging targeted therapies in immunotherapy. Here, we review some emerging spatial omics techniques in cancer research and therapeutic applications and propose prospects for their future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef38d3cdba0bda6123b389b5084be973c8790711" target='_blank'>
              Advances in spatial multi-omics in tumors.
              </a>
            </td>
          <td>
            Junyan Wang, Ahmad Alhaskawi, Yanzhao Dong, Tu Tian, Sahar Abdalbary, Hui Lu
          </td>
          <td>2024-08-26</td>
          <td>Tumori</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumors have a complex metabolism that differs from most metabolic processes in healthy tissues. It is highly dynamic and driven by the tumor cells themselves, as well as by the non‐transformed stromal infiltrates and immune components. Each of these cell populations has a distinct metabolism that depends on both their cellular state and the availability of nutrients. Consequently, to fully understand the individual metabolic states of all tumor‐forming cells, correlative mass spectrometric imaging (MSI) up to cellular resolution with minimal metabolite shift needs to be achieved. By using a secondary ion mass spectrometer (SIMS) equipped with an Orbitrap mass analyzer, we present a workflow to image primary murine tumor tissues up to cellular resolution and correlate these ion images with post acquisition immunofluorescence or histological staining. In a murine breast cancer model, we could identify metabolic profiles that clearly distinguish tumor tissue from stromal cells and immune infiltrates. We demonstrate the robustness of the classification by applying the same profiles to an independent murine model of lung cancer, which is accurately segmented by histological traits. Our pipeline allows metabolic segmentation with simultaneous cell identification, which in the future will enable the design of subpopulation‐targeted metabolic interventions for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04e6d2390afe35044a275be6c42d6f73bf5b238a" target='_blank'>
              Orbi‐SIMS Mediated Metabolomics Analysis of Pathogenic Tissue up to Cellular Resolution
              </a>
            </td>
          <td>
            Christine Kern, Astrid Scherer, Laura Gambs, Mariia Yuneva, H. Walczak, G. Liccardi, J. Saggau, Peter Kreuzaler, M. Rohnke
          </td>
          <td>2024-07-12</td>
          <td>Chemistry–Methods</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood.We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles.We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation.Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c649f4e5bac69e4b666a1bcdc0297d323411c9" target='_blank'>
              Re-analysis of single cell and spatial transcriptomics data reveals B cell landscape in gastric cancer microenvironment and its potential crosstalk with tumor cells for clinical prognosis
              </a>
            </td>
          <td>
            Xing Cai, Jinru Yang, Yusheng Guo, Yanchao Yu, Chuansheng Zheng, Xiaofang Dai
          </td>
          <td>2024-08-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell analysis has become an essential tool in modern biological research, providing unprecedented insights into cellular behavior and heterogeneity. By examining individual cells, this approach surpasses conventional population-based methods, revealing critical variations in cellular states, responses to environmental cues, and molecular signatures. In the context of cancer, with its diverse cell populations, single-cell analysis is critical for investigating tumor evolution, metastasis, and therapy resistance. Understanding the phenotype-genotype relationship at the single-cell level is crucial for deciphering the molecular mechanisms driving tumor development and progression. This review highlights innovative strategies for selective cell isolation based on desired phenotypes, including robotic aspiration, laser detachment, microraft arrays, optical traps, and droplet-based microfluidic systems. These advanced tools facilitate high-throughput single-cell phenotypic analysis and sorting, enabling the identification and characterization of specific cell subsets, thereby advancing therapeutic innovations in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1bb0534cf6f72176c966878e2924974c47306ed" target='_blank'>
              Advances in Single-Cell Techniques for Linking Phenotypes to Genotypes.
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yushu Ma, Jinxiong Cheng, Yu-Chih Chen
          </td>
          <td>2024-07-25</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Many aspects of breast cancer metastasis remain poorly understood, despite its clinical importance. In this issue of the JCI, Winkler et al. have applied an elegant patient-derived xenograft (PDX) model to map the transcriptomes of single cells in matched primary tumors and lung metastases across 13 breast cancer PDX models. They identified distinct transcriptional changes associated with metastatic evolution in lowly and highly metastatic primary tumors. Furthermore, by classifying the “epithelial-mesenchymal plasticity” (EMP) state of single cells, they revealed that considerable EMP heterogeneity exists among primary and metastatic human breast cancer cells. However, the EMP profile of a tumor does not change substantially upon metastasis. These findings give an unprecedentedly detailed view into the transcriptional heterogeneity and evolution of metastatic human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54971b7e64340d8e740cdf1de341d57f2e78ee5a" target='_blank'>
              Mapping the transcriptional evolution of human metastatic breast cancer
              </a>
            </td>
          <td>
            Melissa Q. Reeves
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Uveal melanoma is the most common non-cutaneous melanoma and is an intraocular malignancy affecting nearly 7,000 individuals per year worldwide. Of these, approximately 50% will progress to metastatic disease for which there are currently no effective curative therapies. Despite advances in molecular profiling and metastatic stratification of uveal melanoma tumors, little is known regarding their underlying biology of metastasis. Our group has identified a disseminated neoplastic cell population characterized by co-expression of immune and melanoma proteins, circulating hybrid cells (hybrids), in patients with uveal melanoma. Compared to circulating tumor cells, which lack expression of immune proteins, hybrids are detected at an increased prevalence in peripheral blood and can be used as a non-invasive biomarker to predict metastatic progression. Methods To ascertain mechanisms underlying enhanced hybrid cell dissemination we identified hybrid cells within primary uveal melanoma tumors using single cell RNA sequencing ( n  = 8) and evaluated their gene expression and predicted ligand-receptor interactions in relation to other melanoma and immune cells within the primary tumor. We then verified expression of upregulated hybrid pathways within patient-matched tumor and peripheral blood hybrids ( n  = 4) using cyclic immunofluorescence and quantified their protein expression relative to other non-hybrid tumor and disseminated tumor cells. Results Among the top upregulated genes and pathways in hybrid cells were those involved in enhanced cell motility and cytoskeletal rearrangement, immune evasion, and altered cellular metabolism. In patient-matched tumor and peripheral blood, we verified gene expression by examining concordant protein expression for each pathway category: TMSB10 (cell motility), CD74 (immune evasion) and GPX1 (metabolism). Both TMSB10 and GPX1 were expressed on significantly higher numbers of disseminated hybrid cells compared to circulating tumor cells, and CD74 and GPX1 were expressed on more disseminated hybrids than tumor-resident hybrids. Lastly, we identified that hybrid cells express ligand-receptor signaling pathways implicated in promoting metastasis including GAS6-AXL, CXCL12-CXCR4, LGALS9-P4HB and IGF1-IGFR1. Conclusion These findings highlight the importance of TMSB10, GPX1 and CD74 for successful hybrid cell dissemination and survival in circulation. Our results contribute to the understanding of uveal melanoma tumor progression and interactions between tumor cells and immune cells in the tumor microenvironment that may promote metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20782b15e64557dc3e5252547de53160b7d85219" target='_blank'>
              Detection of neoplastic-immune hybrid cells with metastatic properties in uveal melanoma
              </a>
            </td>
          <td>
            Ashley N. Anderson, P. Conley, Christopher Klocke, Sidharth K. Sengupta, Amara Pang, Hannah C. Farley, Abigail R. Gillingham, Aubrey D. Dawson, Yichen Fan, Jocelyn A. Jones, Summer L. Gibbs, Alison H. Skalet, Guanming Wu, Melissa H. Wong
          </td>
          <td>2023-12-06</td>
          <td>Research Square</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6152a163a331bb6cdaad732712072fe74da5cd31" target='_blank'>
              Spatial cell interplay networks of regulatory T cells predict recurrence in patients with operable non-small cell lung cancer
              </a>
            </td>
          <td>
            Siqi Cai, Guanqun Yang, Mengyu Hu, Chaozhuo Li, Liying Yang, Wei Zhang, Jujie Sun, Fenghao Sun, Ligang Xing, Xiaorong Sun
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer cells have remarkable plasticity allowing them to acquire many biological states. Melanoma cells have the ability to switch from a proliferative melanocytic state to an invasive mesenchymal state and back again resulting in intratumoral heterogeneity. While microphthalmia-associated transcription factor (MITF) promotes the melanocytic phenotype, it is unclear what transcription factors drive the mesenchymal phenotype, and what mechanisms regulate the switch from the proliferative state to the mesenchymal state. We show that nuclear localization of the MITF paralog TFE3 correlates positively with metastatic potential in melanoma cell lines and tumors, and that deletion of TFE3 in MITF-low melanoma cell lines eliminates migration and metastatic ability. Further, we find that MITF suppresses the mesenchymal phenotype by activating expression of FNIP2, which encodes a component of an mTORC1-stimulated pathway promoting cytoplasmic retention and lysosomal degradation of TFE3. These findings point to the mTOR pathway and TFE3 as key regulators of melanoma plasticity. Highlights In melanoma, high MITF activity correlate with lower levels of the full-length form of TFE3 and with TFE3 being in the cytoplasm. In an MITF-high melanoma cell line, deletion of MITF results in increased levels of the full-length form of TFE3 and with its nuclear localization. In an MITF-low melanoma cell line, deletion of TFE3 reduces expression levels of genes promoting epithelial-mesenchymal transition and prevents distant cell colonization and metastasis in a xenograft model. MITF promotes expression of FNIP2 which encodes a component of the folliculin complex. This complex activates RagC/D guanosine triphosphatase which recruits mTOR and TFE3 to the lysosomal surface, resulting in cytoplasmic localization and degradation of full-length TFE3. Inhibition of mTORC1 signaling in MITF-high cells reverses the effect of MITF on TFE3 stability and nucular localization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a5e351ace57f99fd25c92d1c6e1950143252fc" target='_blank'>
              An MITF- and mTOR-dependent FLCN pathway suppresses TFE3-driven metastasis in melanoma
              </a>
            </td>
          <td>
            Jeremy Chang, Katelyn R. Campbell-Hanson, Marion Vanneste, Julius Yevdash, Nicholas Bartschat, Jiarui Jiang, Ardith Bhinu, Annika Helverson, Michael D. Henry, E. Steingrímsson, Ronald J. Weigel, Robert A. Cornell, Colin Kenny
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Recent technological advancements have enabled spatially resolved transcriptomic profiling but at a multicellular resolution that is more cost-effective. The task of cell type deconvolution has been introduced to disentangle discrete cell types from such multicellular spots. However, existing benchmark datasets for cell type deconvolution are either generated from simulation or limited in scale, predominantly encompassing data on mice and are not designed for human immuno-oncology. To overcome these limitations and promote comprehensive investigation of cell type deconvolution for human immuno-oncology, we introduce a large-scale spatial transcriptomic deconvolution benchmark dataset named SpatialCTD, encompassing 1.8 million cells and 12,900 pseudo spots from the human tumor microenvironment across the lung, kidney, and liver. In addition, SpatialCTD provides more realistic reference than those generated from single-cell RNA sequencing (scRNA-seq) data for most reference-based deconvolution methods. To utilize the location-aware SpatialCTD reference, we propose a graph neural network-based deconvolution method (i.e., GNNDeconvolver). Extensive experiments show that GNNDeconvolver often outperforms existing state-of-the-art methods by a substantial margin, without requiring scRNA-seq data. To enable comprehensive evaluations of spatial transcriptomics data from flexible protocols, we provide an online tool capable of converting spatial transcriptomic data from various platforms (e.g., 10× Visium, MERFISH, and sci-Space) into pseudo spots, featuring adjustable spot size. The SpatialCTD dataset and GNNDeconvolver implementation are available at https://github.com/OmicsML/SpatialCTD, and the online converter tool can be accessed at https://omicsml.github.io/SpatialCTD/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/868514c71cf6aea4de94427fd6a0a67ca0f210d0" target='_blank'>
              SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.
              </a>
            </td>
          <td>
            Jiayuan Ding, Lingxiao Li, Qiaolin Lu, Julian Venegas, Yixin Wang, Lidan Wu, Wei Jin, Hongzhi Wen, Renming Liu, Wenzhuo Tang, Xinnan Dai, Zhaoheng Li, Wangyang Zuo, Yi Chang, Yu Leo Lei, Lulu Shang, Patrick Danaher, Yuying Xie, Jiliang Tang
          </td>
          <td>2024-08-08</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51210c4137b11dcdb52e668475ae791c89b00c10" target='_blank'>
              Integrative single-cell and bulk transcriptome analyses identify a distinct pro-tumor macrophage signature that has a major prognostic impact on glioblastomas
              </a>
            </td>
          <td>
            Peilin Li, Guolei Su, Yinglin Cui
          </td>
          <td>2024-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Methods that predict fate potential or degree of differentiation from transcriptomic data have identified rare progenitor populations and uncovered developmental regulatory mechanisms. However, some state-of-the-art methods are too computationally burdensome for emerging large-scale data and all methods make inaccurate predictions in certain biological systems. We developed a method in R (stemFinder) that predicts single cell differentiation time based on heterogeneity in cell cycle gene expression. Our method is computationally tractable and is as good as or superior to competitors. As part of our benchmarking, we implemented four different performance metrics to assist potential users in selecting the tool that is most apt for their application. Finally, we explore the relationship between differentiation time and cell fate potential by analyzing a lineage tracing dataset with clonally labelled hematopoietic cells, revealing that metrics of differentiation time are correlated with the number of downstream lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fe7aece967bf7efe6ff7ba7d9a3f9b4b16c676a" target='_blank'>
              Cell cycle expression heterogeneity predicts degree of differentiation
              </a>
            </td>
          <td>
            K. Noller, Patrick Cahan
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Current rheumatoid arthritis (RA) treatments do not restore immune tolerance. Investigating dendritic cell (DC) populations in human synovial tissue (ST) may reveal pathways to re-instate tolerance in RA. With single-cell and spatial-transcriptomics of synovial tissue biopsies, validated by micro co-culture systems, we identified condition and niche-specific myeloid DC clusters with distinct differentiation trajectories and functions. Healthy synovium contains a unique tolerogenic AXLpos DC2 cluster in the superficial sublining layer. In active RA, a macrophage-rich lining-layer niche becomes populated with inflammatory DC3 clusters that specifically activate memory CCL5pos TEM and CCL5posCXCL13pos TPH, promoting synovitis. In the sublining lymphoid niche, CCR7pos DC2 mReg specifically interact with naïve-T-cells, potentially driving the local expansion of new effector T-cells. Sustained remission sees the resolution of these niches but lacks the recovery of tolerogenic AXLpos DC2, indicating latent potential for disease flare. A human RA disease-flare model showed that the activation of blood predecessor of ST-DC3 clusters precedes the onset of inflammation in joints. Therapeutic strategies targeting pathogenic ST-DC3 clusters, or reinstating tolerogenic AXLpos DC2, may restore immune homeostasis in RA. In brief Deconstruction of human RA synovium, using single-cell spatial transcriptomics and micro-culture systems, reveals distinct neighbourhoods within the synovial architecture across health, and RA patients with active disease or sustained remission. Discrete niches are identified that contain distinct myeloid DC clusters that differ in frequency, differentiation trajectories, and effector functions. Highlights Human RA synovium exhibits condition and niche specific myeloid DC clusters that vary in their tissue differentiation trajectories and functions. ST-CD14pos DC3 (iDC3) support inflammatory CCL5pos TEM and CCL5pos TPH cell activation in the hyperplastic lining layer. ST-CCR7pos DC2 (mReg), driven by MIR155, interact with naïve-T-cells in sublining lymphoid niches. A specific inflammatory signature of blood predecessors of ST-DC3s predict flare in RA. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bc93e71948d0e300f8f2b2043725d09545d814" target='_blank'>
              Distinct tissue-niche localization and function of synovial tissue myeloid DC subsets in health, and in active and remission Rheumatoid Arthritis
              </a>
            </td>
          <td>
            L. MacDonald, A. Elmesmari, D. Somma, Jack Frew, C. Di Mario, Roopa Madhu, Audrey Paoletti, Teontor Simakou, Olympia Hardy, B. Tolusso, Denise Campobasso, S. Perniola, Marco Gessi, M. Gigante, L. Petricca, Dario Bruno, L. Coletto, Roberta Benvenuto, J. Isaacs, A. Filby, David McDonald, Jasmine P. X. Sim, N. Jamieson, Kevin Wei, Maria-Antonietta D'Agostino, Neal L. Millar, S. Milling, C. McSharry, E. Gremese, Karen Affleck, Kenneth F. Baker, Iain B. McInnes, Thomas D. Otto, I. Korsunsky, S. Alivernini, Mariola Kurowska-Stolarska
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cells are generally exposed to numerous extrinsic stimulations in the tumor microenvironment. In this environment, cancer cells change their expression profiles to fight against circumstantial stresses, allowing their progression in the challenging tissue space. Technological advancements of spatial omics have had substantial influence on cancer genomics. This technical progress, especially that occurring in the spatial transcriptome, has been drastic and rapid. Here, we describe the latest spatial analytical technologies that have allowed omics feature characterization to retain their spatial and histopathological information in cancer tissues. Several spatial omics platforms have been launched, and the latest platforms finally attained single-cell level or even higher subcellular level resolution. We discuss several key papers elucidating the initial utility of the spatial analysis. In fact, spatial transcriptome analyses reveal comprehensive omics characteristics not only in cancer cells but also their surrounding cells, such as tumor infiltrating immune cells and cancer-associated fibroblasts. We also introduce several spatial omics platforms. We describe our own attempts to investigate molecular events associated with cancer progression. Furthermore, we discuss the next challenges in analyzing the multiomics status of cells, including their morphology and location. These novel technologies, in conjunction with spatial transcriptome analysis and, more importantly, with histopathology, will elucidate even novel key aspects of the intratumor heterogeneity of cancers. Such enhanced knowledge is expected to open a new path for overcoming therapeutic resistance and eventually to precisely stratify patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca089c4742cacf38e064d93c1785f53f8df9974" target='_blank'>
              Spatial omics technologies for understanding molecular status associated with cancer progression.
              </a>
            </td>
          <td>
            Satoi Nagasawa, Junko Zenkoh, Yutaka Suzuki, Ayako Suzuki
          </td>
          <td>2024-07-23</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Granulomas (defining tuberculosis histopathological feature) are central to the host’s defense against Mycobacterium tuberculosis, critically influencing patient outcomes. However, knowledge of human granulomas’ structure and function are incomplete. This study employs single-cell and spatial transcriptomics to dissect human granuloma’s cellular composition, structure, communication and function from 19 pulmonary, lymphatic and skeletal samples. Results Our study identified nine key immune-activated/signaling-active cell clusters. Notably, we delineated a three-layered granuloma structure: a core with macrophages (Macro-c09, Macro-c10) and occasional fibroblasts (Fib-c03); a fibroblast-rich (Fib-c01) periphery; and an immune-infiltrated intermediate layer comprising diverse immune-cells recruited by strong signaling-molecules (SPP1/MIF) from core/periphery cells. This study also shows granuloma heterogeneity across individuals and tissues. Conclusions By merging scRNA-seq with ST-seq, we offer an intricate single-cell perspective of granulomas’ spatial-structure and formation mechanisms, identify signaling-molecules and significantly changed genes as potential targets for host-directed tuberculosis immunotherapy, highlight fibroblasts’ crucial role in granuloma formation, and provide an important reference/improved understanding of TB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c14b14dcc946c712193e06986c65772b8af84b9" target='_blank'>
              Single-Cell and Spatial Transcriptomic Analyses Deciphering the Three-Layer Architecture of Human Tuberculosis Granulomas
              </a>
            </td>
          <td>
            Xia Yu, Jie Wang, Peihan Wang, Xiaoqiang Liu, Cuidan Li, Yingjiao Ju, Sitong Liu, Yujie Dong, Jing Wang, Bahetibieke Tuohetaerbaike, Hao Wen, Wenbao Zhang, Haitao Niu, Sihong Xu, Chunlai Jiang, Xiaoyi Jiang, Jing Wu, Hairong Huang, Fei Chen
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Epithelial–mesenchymal plasticity plays a significant role in various biological processes including tumour progression and chemoresistance. However, the expression programmes underlying the epithelial–mesenchymal transition (EMT) in cancer are diverse, and accurately defining the EMT status of tumour cells remains a challenging task. In this study, we employed a pre-trained single-cell large language model (LLM) to develop an EMT-language model (EMT-LM) that allows us to capture discrete states within the EMT continuum in single cell cancer data. In capturing EMT states, we achieved an average Area Under the Receiver Operating Characteristic curve (AUROC) of 90% across multiple cancer types. We propose a new metric, ADESI, to aid the biological interpretability of our model, and derive EMT signatures liked with energy metabolism and motility reprogramming underlying these state switches. We further employ our model to explore the emergence of EMT states in spatial transcriptomics data, uncovering hybrid EMT niches with contrasting potential for antitumour immunity or immune evasion. Our study provides a proof of concept that LLMs can be applied to characterise cell states in single cell data, and proposes a generalisable framework to predict EMT in single cell RNA-seq that can be adapted and expanded to characterise other cellular states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54879184bde6542b97b756a010bae7401c49921" target='_blank'>
              Classifying epithelial-mesenchymal transition states in single cell cancer data using large language models
              </a>
            </td>
          <td>
            Shi Pan, Eloise Withnell, M. Secrier
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Squamous cell carcinomas (SCCs) across different anatomical locations possess common molecular features. Recent studies showed that stromal cells may contribute to tumor progression and metastasis of SCCs. Limited by current sequencing technology and analysis methods, it has been difficult to combine stroma expression profiles with a large number of clinical information. Methods With the help of transfer learning on the cell line, single-cell, and bulk tumor sequencing data, we identified and validated 2 malignant gene patterns (V1 and V5) expressed by stromal cells of SCCs from head and neck (HNSCC), lung (LUSC), cervix (CESC), esophagus, and breast. Results Pattern V5 reflected a novel malignant feature that explained the mixed signals of HNSCC molecular subtypes. Higher expression of pattern V5 was related to shorter PFI with gender and cancer-type specificity. The other stromal gene pattern V1 was associated with poor PFI in patients after surgery in all the three squamous cancer types (HNSCC p = 0.0055, LUSC p = 0.0292, CESC p = 0.0451). Cancer-associated fibroblasts could induce HNSCC cancer cells to express pattern V1. Adjuvant radiotherapy may weaken the effect of high V1 on recurrence and metastasis, depending on the tumor radiosensitivity. Conclusion Considering the prognostic value of stromal gene patterns and its universality, we suggest that the genetic subtype classification of SCCs may be improved to a new system that integrates both malignant and non-malignant components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec6008ecf1c1695bb535c0cacf60d9734a6a36e5" target='_blank'>
              Stromal cell-expressed malignant gene patterns contribute to the progression of squamous cell carcinomas across different sites
              </a>
            </td>
          <td>
            Kaiyan Qi, Guangqi Li, Yuanjun Jiang, Xuexin Tan, Qiao Qiao
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8eb93ddf32d281284931746da27ab53243de093" target='_blank'>
              Tumoroids, a valid preclinical screening platform for monitoring cancer angiogenesis
              </a>
            </td>
          <td>
            Zahra Abbasi-Malati, Parisa Khanicheragh, Maryam Taghavi Narmi, Narges Mardi, Nafiseh Didar Khosrowshahi, Amirataollah Hiradfar, Aysa Rezabakhsh, Fatemeh Sadeghsoltani, Somayyeh Rashidi, S. A. Chegeni, Golbarg Roozbahani, R. Rahbarghazi
          </td>
          <td>2024-08-26</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86603b6c8f27f9a5a7d6a05553f0d41b941be5e5" target='_blank'>
              Spatial landscapes of cancers: insights and opportunities.
              </a>
            </td>
          <td>
            Julia Chen, L. Larsson, Alexander Swarbrick, J. Lundeberg
          </td>
          <td>2024-07-23</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are heterogenous in size, biogenesis, cargo and function. Beside small EVs, aggressive tumor cells release a population of particularly large EVs, namely large oncosomes (LO). This study provides the first resource of label-free quantitative proteomics of LO and small EVs obtained from distinct cancer cell types (prostate, breast, and glioma). This dataset was integrated with a SWATH Proteomic assay on LO, rigorously isolated from the plasma of patients with metastatic prostate cancer (PC). Proteins enriched in LO, which were identified also at the RNA level, and found in plasma LO significantly correlated with PC progression. Single EV RNA-Seq of the PC cell-derived LO confirmed some of the main findings from the bulk RNA-Seq, providing first evidence that single cell technologies can be successfully applied to EVs. This multiomics resource of cancer EVs can be leveraged for developing a multi-analyte approach for liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da254bd82830fbf0461d37b66c142c956fe3dc3" target='_blank'>
              Extracellular Vesicles heterogeneity through the lens of multiomics
              </a>
            </td>
          <td>
            Taylon F. Silva, E. Hutchins, Wenyan Zhao, Yari Ciani, Minhyung Kim, Emily Ko, Javier Mariscal, Zhuyu Qiu, Agnes Kittel, Bo Zhou, Yang Wang, Megan Hall, Francesca Galasso, Rebecca A. Reiman, Michael R Freeman, Sarah Parker, Jennifer Van Eyk, Wei Yang, Edwin Posadas, Jlenia Guarnerio, John Nolan, C. Théry, Andries Zijlstra, Shannon Stott, Sungyong You, F. Demichelis, P. Boutros, K. Van Keuren-Jensen, Dolores Di Vizio
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequently occurring cancers, but prognostic biomarkers identifying patients at risk of recurrence are still lacking. In this study, we aimed to investigate in more detail the spatial relationship between intratumoural T cells, cancer cells, and cancer cell hallmarks as prognostic biomarkers in stage III colorectal cancer patients. We conducted multiplexed imaging of 56 protein markers at single-cell resolution on resected fixed tissue from stage III CRC patients who received adjuvant 5-fluorouracil (5FU)-based chemotherapy. Images underwent segmentation for tumour, stroma, and immune cells, and cancer cell 'state' protein marker expression was quantified at a cellular level. We developed a Python package for estimation of spatial proximity, nearest neighbour analysis focusing on cancer cell-T-cell interactions at single-cell level. In our discovery cohort (Memorial Sloan Kettering samples), we processed 462 core samples (total number of cells: 1,669,228) from 221 adjuvant 5FU-treated stage III patients. The validation cohort (Huntsville Clearview Cancer Center samples) consisted of 272 samples (total number of cells: 853,398) from 98 stage III CRC patients. While there were trends for an association between the percentage of cytotoxic T cells (across the whole cancer core), it did not reach significance (discovery cohort: p = 0.07; validation cohort: p = 0.19). We next utilised our region-based nearest neighbour approach to determine the spatial relationships between cytotoxic T cells, helper T cells, and cancer cell clusters. In both cohorts, we found that shorter distance between cytotoxic T cells, T helper cells, and cancer cells was significantly associated with increased disease-free survival. An unsupervised trained model that clustered patients based on the median distance between immune cells and cancer cells, as well as protein expression profiles, successfully classified patients into low-risk and high-risk groups (discovery cohort: p = 0.01; validation cohort: p = 0.003). © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e5d285772eda83349528bbd7ffce569b58bcfc" target='_blank'>
              Spatial effects of infiltrating T cells on neighbouring cancer cells and prognosis in stage III CRC patients.
              </a>
            </td>
          <td>
            Mohammadreza Azimi, Sanghee Cho, E. Bozkurt, E. McDonough, Batuhan Kısakol, Anna Matveeva, M. Salvucci, H. Dussmann, Simon McDade, C. Firat, Nil Urganci, J. Shia, Daniel B Longley, Fiona Ginty, Jochen H. M. Prehn
          </td>
          <td>2024-08-02</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="During the development from epithelial neoplasms to invasive carcinoma, uncontrolled growth and proliferation in a confined space cause tumor cells survive in a prolonged overcrowding condition. However, the role of this aberrant physical stimulus induced by overcrowding in the invasiveness acquisition of epithelial cells remains unclear. Here, we first identified overcrowded cancer cells were more abundant at invasive foci in human cancer samples and mouse xenograft models. Combining with mechanical modelling, genetic manipulations and biophysical measurements, we revealed that prolonged overcrowding induces retentive invasiveness in cancer cells by triggering nanocorrugated topographic transition of plasma membrane. Further study demonstrated that the nanocorrugated topographic transition is induced by overcrowding-increased Laplace pressure and disrupts lipid raft-like domains organization to promote tumor invasion. Finally, suppressing nanocorrugated topographical transition of plasma membrane by strengthening membrane-cortex-attachment resists to tumor cell invasion in a mouse model of oncogenesis. This work reveals the profound effect of tissue-scale mechanical revolution-induced biophysical transition of cancer cells in driving tumor invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96309d0c656c868fef684ed435f46b0a78a1517e" target='_blank'>
              Overcrowding Drives Tumor Invasion by Inducing Nanotopographical Transition of Plasma Membrane
              </a>
            </td>
          <td>
            Xinbin Zhao, Min Tan, Long Li, Xubo Lin, Zekun Li, Xiaohuan Wang, Bingqi Song, Zheng Guo, Tailin Chen, Sen Hou, Jiehou Fan, Shijiang Wang, Yun Zhang, Yubo Fan, Jing Du
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Hepatoblastoma (HB) and hepatocellular carcinoma (HCC) are the most common malignant hepatocellular tumors seen in children. Patients with vascular invasion and metastasis generally have poor outcomes. The aim of this study was to delineate the unique cellular changes that drive dissemination by performing comprehensive molecular and phenotypic profiling of samples representing disseminating cells, including primary circulating tumor cells (CTCs) from patients and orthotopic patient-derived xenograt (PDX) mouse models and cell lines grown in vitro in the presence of other cells that represent the tumor microenvironment (TME). Methods: With primary patient whole blood samples, we used a RosetteSep CD45 Depletion Cocktail (Stem Cell Technologies) to enrich CTCs. We validated that these cells were CTCs by staining for a panel of validated markers, including indocyanine green (ICG), Glypican-3 (GPC3), and DAPI. We then performed single cell RNA sequencing (scRNA-seq, 10x Genomics) to comprehensively analyze gene expression of these cells. To understand tumor dissemination mechanisms influenced by the TME, we grew HepT1 HB cells in the presence of supernatant from human umbilical vein endothelial cells (HUVECs). Specifically, HUVEC and HepT1 cells were cultured in separate plates in a combination of endothelial cell medium and minimal essential medium in a 1:1 ratio. The supernatant from each plate was then added to HepT1 cells, and we examined resulting transcriptomic changes with bulk RNA-seq and phenotypic alterations with proliferation (CCK8), scratch (Incucyte), and invasion (Boyden chamber) assays. We also worked to develop a pipeline to establish stable cell lines with primary patient- and murine-derived CTCs. Results: Using our ICG/GPC3/DAPI panel, we showed that the cells isolated from patient whole blood samples were CTCs. scRNA-seq analyses of these samples revealed that there was upregulation of key pathways in CTCs, including NRF2 activity, compared to low-risk primary HB tumors. Factors secreted by HUVECs influenced HepT1 cell phenotypes and gene expression. Specifically, we showed significantly higher migration and faster wound closure when HepT1 cells were grown in the presence of secreted factors from HUVECs, compared to control HepT1 cells. Conclusions: This study provides a comprehensive transcriptomic landscape of pediatric liver cancer CTCs. This work also underscores the importance of the interactions between endothelial cells and tumor cells in the initiation of metastasis. Taken together, this work builds a strong foundation for future studies elucidating the specific mechanisms of how pediatric liver tumor cells successfully disseminate with the overall goal of developing novel therapeutic regimens that target these mechanisms.
 Citation Format: Priyanka Rao, Andres Espinoza, Roma Patel, Mohammad J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield. Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165a476b6bcad6280ce2606b1f9945d425fc4ed9" target='_blank'>
              Abstract B072: Cell alterations that drive vascular invasion and dissemination in pediatric liver cancer
              </a>
            </td>
          <td>
            Priyanka Rao, Andres F. Espinoza, Roma Patel, M. J. Najaf Panah, Pavel Sumazin, Sanjeev A. Vasudevan, Sarah E. Woodfield
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Accurate phenotyping of cells in the tumor microenvironment is crucial for understanding cancer biology and developing effective therapies. However, current methods require precise cell segmentations and struggle to generalize across different imaging modalities, limiting their utility in digital pathology. Here, we show that Contrastive Learning Enabled Accurate Registration of Immune and Tumor Cells (CLEAR-IT) overcomes these limitations, providing a robust and versatile tool for cell phenotyping. CLEAR-IT accurately phenotypes cells comparable to state-of-the-art methods, generalizes across multiplex imaging modalities, maintains high performance even with limited number of labels, and enables the extraction of prognostic markers. Additionally, CLEAR-IT can be combined with existing methods to boost their performance, whereas its lack of need for precise cell segmentations significantly reduces training efforts. This method enhances the robustness and efficiency of digital pathology workflows, making it a valuable tool for cancer research and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02cd2bfec0cff758ded2e68209475ad537d70fc" target='_blank'>
              CLEAR-IT: Contrastive Learning to Capture the Immune Composition of Tumor Microenvironments
              </a>
            </td>
          <td>
            Daniel Spengler, Serafim Korovin, Kirti Prakash, Peter Bankhead, R. Debets, Carlas S. Smith, H. E. Balcioglu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Glioblastomas are aggressive brain tumors for which effective therapy is still lacking, resulting in dismal survival rates. These tumors display significant phenotypic plasticity, harboring diverse cell populations ranging from tumor core cells to dispersed, highly invasive cells. Neuron navigator 3 (NAV3), a microtubule-associated protein affecting microtubule growth and dynamics, is downregulated in various cancers, including glioblastoma, and has thus been considered a tumor suppressor. In this study, we challenge this designation and unveil distinct expression patterns of NAV3 across different invasion phenotypes. Using glioblastoma cell lines and patient-derived glioma stem-like cell cultures, we disclose an upregulation of NAV3 in invading glioblastoma cells, contrasting with its lower expression in cells residing in tumor spheroid cores. Furthermore, we establish an association between low and high NAV3 expression and the amoeboid and mesenchymal invasive phenotype, respectively, and demonstrate that overexpression of NAV3 directly stimulates glioblastoma invasive behavior in both 2D and 3D environments. Consistently, we observed increased NAV3 expression in cells migrating along blood vessels in mouse xenografts. Overall, our results shed light on the role of NAV3 in glioblastoma invasion, providing insights into this lethal aspect of glioblastoma behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e666e6ac20954126f3ae7a942589c280786af039" target='_blank'>
              Microtubule-associated NAV3 regulates invasive phenotypes in glioblastoma cells.
              </a>
            </td>
          <td>
            A. Škarková, Markéta Pelantová, Ondřej Tolde, Anna Legátová, R. Mateu, P. Bušek, E. Garcia-Borja, A. Šedo, S. Étienne-Manneville, D. Rösel, J. Brábek
          </td>
          <td>2024-08-03</td>
          <td>Brain pathology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7565f576b9dc62d0204231ebf9aa97eaaf0e3374" target='_blank'>
              Single-cell transcriptome profiles the heterogeneity of tumor cells and microenvironments for different pathological endometrial cancer and identifies specific sensitive drugs
              </a>
            </td>
          <td>
            Fang Ren, Lingfang Wang, Yuyouye Wang, Jiaxuan Wang, Yuanpei Wang, Xiaole Song, Gong Zhang, Fangfang Nie, Shitong Lin
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b298e495488218b37b27f2977ca66330db94ae6" target='_blank'>
              Spatial tumor immune microenvironment phenotypes in ovarian cancer
              </a>
            </td>
          <td>
            Claudia Mateiou, L. Lokhande, Lan Hoa Diep, Mattis Knulst, Elias Carlsson, Sara Ek, K. Sundfeldt, A. Gerdtsson
          </td>
          <td>2024-07-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract While paediatric blood cancers are deadly, modern medical advances have enabled clinicians to measure levels of residual cancer cells to manage therapeutic strategies for patients. However, blood cancers, including leukaemias and lymphomas, are highly heterogeneous and is comprised of complex clonal populations that can hinder efforts in detecting the cancer cells as well as managing treatments. Furthermore, the tumour microenvironment is comprised of heterogenous immune dynamics that may be different between patients. High‐throughput sequencing has constributed to new discoveries in genetic and transcriptomic alterations underpinning cancer, including blood cancers, and has changed how patients are monitored and managed. Here we discuss the recent efforts using single‐cell approach, particularly on efforts to track clonal heterogenity of paediatric blood cancer and the underlying immune response, highlighting avenues for novel biomarker discovery that may have significant impact on clinical oncology practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9b7aa16b92f8240bd156cad7d9dc00c572051c" target='_blank'>
              Measuring single‐cell immune clonality to track haematological cancers
              </a>
            </td>
          <td>
            Amos Choo, Z. Tuong
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Interactions between tumor and stromal cells are well known to play a prominent roles in progression of pancreatic ductal adenocarcinoma (PDAC). As knowledge of stromal crosstalk in PDAC has evolved, it has become clear that cancer associated fibroblasts can play both tumor promoting and tumor suppressive roles through a combination of paracrine crosstalk and juxtacrine interactions involving direct physical contact. Another major contributor to dismal survival statistics for PDAC is development of resistance to chemotherapy drugs. Though less is known about how the acquisition of chemoresistance impacts upon tumor-stromal crosstalk. Here, we use 3D co-culture geometries to recapitulate juxtacrine interactions between epithelial and stromal cells. In particular, extracellular matrix (ECM) overlay cultures in which stromal cells (pancreatic stellate cells, or normal human fibroblasts) are placed adjacent to PDAC cells (PANC1), result in direct heterotypic cell adhesions accompanied by dramatic fibroblast contractility which leads to highly condensed macroscopic multicellular aggregates as detected using particle image velocimetry (PIV) analysis to quantify cell velocities over the course of time lapse movie sequences. To investigate how drug resistance impacts these juxtacrine interactions we contrast cultures in which PANC1 are substituted with a drug resistant subline (PANC1-OR) previously established in our lab. We find that heterotypic cell-cell interactions are highly suppressed in drug-resistant cells relative to the parental PANC1 cells. To investigate further we conduct RNA-seq and bioinformatics analysis to identify differential gene expression in PANC1 and PANC1-OR, which shows that negative regulation of cell adhesion molecules, consistent with increased epithelial mesenchymal transition (EMT), is also consistent with loss of hetrotypic cell-cell contact necessary for the contractile behavior observed in drug naïve cultures. Overall these findings elucidate the role of drug-resistance in inhibiting an avenue of stromal crosstalk which is associated with tumor suppression and also help to establish cell culture conditions useful for further mechanistic investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a4fe450dafc0e09ef1ac6571627884db5342e41" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, J. Celli
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Background High-grade gliomas including glioblastoma (GBM) and diffuse midline gliomas (DMG) represent the most lethal and aggressive brain cancers where current treatment modalities offer limited efficacy. Chimeric antigen receptor (CAR) T cell therapies have emerged as a promising strategy, boasting tumor-specific targeting and the unique ability to penetrate the blood-brain barrier. However, the effective clinical application hinges on the optimal choice of antigen, with a limited number, currently under investigation. Methods We employed cell surface proteomic analysis of primary human high-grade glioma samples from both adult and pediatric patients. This led to the identification of Ephrin type-A receptor 3 (EphA3) as a prevalently expressed target. We engineered a second-generation EphA3-targeted CAR T cell and assessed function using in vitro and in vivo models of GBM and DMG. Results EphA3-targeted CAR T cells demonstrated robust antigen-specific killing of human GBM and DMG cell lines in vitro. In an orthotopic xenograft NSG mouse model, EphA3-targeted CAR T cells not only effectively eradicated tumors but also established a functional T cell population protective on rechallenge. Remarkably, mice rechallenged with a second contralateral orthotopic tumor implantation achieved complete tumor clearance and maintained a sustained complete response 6 months following initial treatment. Conclusion Building on the proven safety profile of EphA3 antibodies in clinical settings, our study provides compelling preclinical evidence supporting the efficacy of EphA3-targeted CAR T cells against high-grade gliomas. These findings underscore the potential for transitioning this innovative therapy into clinical trials, aiming to revolutionize the treatment landscape for patients afflicted with these formidable brain cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ccdf5eef190aeacc1b356281cbe1f1aeb78930" target='_blank'>
              EphA3-targeted chimeric antigen receptor T cells are effective in glioma and generate curative memory T cell responses
              </a>
            </td>
          <td>
            Leesa Lertsumitkul, Melinda Iliopoulos, Stacie S Wang, Sarah J McArthur, Lisa M Ebert, Alexander J Davenport, R. Endersby, Jordan R Hansford, Kate Drummond, Ryan S Cross, Misty R. Jenkins
          </td>
          <td>2024-08-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9c71ae891c14b29b8011948e682b3c7afb701b" target='_blank'>
              Unsupervised representation learning of chromatin images identifies changes in cell state and tissue organization in DCIS
              </a>
            </td>
          <td>
            Xinyi Zhang, Saradha Venkatachalapathy, Daniel Paysan, Paulina Schaerer, C. Tripodo, Caroline Uhler, G. Shivashankar
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Cancer initiation, progression, and immune evasion depend on the tumor microenvironment (TME). Thus, understanding the TME immune architecture is essential for understanding tumor metastasis and therapy response. This study aimed to create an immune cell states (CSs) atlas using bulk RNA-seq data enriched by eco-type analyses to resolve the complex immune architectures in the TME. Methods: We employed EcoTyper, a machine-learning (ML) framework, to study the real-world prognostic significance of immune CSs and multicellular ecosystems, utilizing molecular data from 1,610 patients with multiple malignancies who underwent immune checkpoint inhibitor (ICI) therapy within the ORIEN Avatar cohort, a well-annotated real-world dataset. Results: Our analysis revealed consistent ICI-specific prognostic TME carcinoma ecotypes (CEs) (including CE1, CE9, CE10) across our pan-cancer dataset, where CE1 being more lymphocyte-deficient and CE10 being more proinflammatory. Also, the analysis of specific immune CSs across different cancers showed consistent CD8+ and CD4+ T cell CS distribution patterns. Furthermore, survival analysis of the ORIEN ICI cohort demonstrated that ecotype CE9 is associated with the most favorable survival outcomes, while CE2 is linked to the least favorable outcomes. Notably, the melanoma-specific prognostic EcoTyper model confirmed that lower predicted risk scores are associated with improved survival and better response to immunotherapy. Finally, de novo discovery of ecotypes in the ORIEN ICI dataset identified Ecotype E3 as significantly associated with poorer survival outcomes. Conclusion: Our findings offer important insights into refining the patient selection process for immunotherapy in real-world practice and guiding the creation of novel therapeutic strategies to target specific ecotypes within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21e6c1192cee70c66fced389170cb753c806eb9" target='_blank'>
              Multicellular immune ecotypes within solid tumors predict real-world therapeutic benefits with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Xuefeng Wang, Tingyi Li, Islam Eljilany, V. Sukrithan, A. Ratan, Martin D McCarter, J. Carpten, Howard Colman, A. Ikeguchi, Igor Puzanov, S. Arnold, Michelle L Churchman, Patrick Hwu, Paulo C Rodriguez, William S. Dalton, George J Weiner, A. Tarhini
          </td>
          <td>2024-07-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Despite being the leading cause of childhood mortality, pediatric gliomas have been relatively understudied, and the repurposing of immunotherapies has not been successful. Whole transcriptome sequencing, single-cell sequencing, and sequential multiplex immunofluorescence were used to identify an immunotherapy strategy evaluated in multiple preclinical glioma models. MAPK-driven pediatric gliomas have a higher interferon signature relative to other molecular subgroups. Single-cell sequencing identified an activated and cytotoxic microglia population designated MG-Act in BRAF-fused MAPK-activated pilocytic astrocytoma (PA), but not in high-grade gliomas or normal brain. TIM3 is expressed on MG-Act and on the myeloid cells lining the tumor vasculature but not normal brain. TIM3 expression becomes upregulated on immune cells in the PA microenvironment and anti-TIM3 reprograms ex vivo immune cells from human PAs to a pro-inflammatory cytotoxic phenotype. In a genetically engineered murine model of MAPK-driven low-grade gliomas, anti-TIM3 treatment increased median survival over IgG and anti-PD1 treated mice. ScRNA sequencing data during the therapeutic window of anti-TIM3 demonstrates enrichment of the MG-Act population. The therapeutic activity of anti-TIM3 is abrogated in the CX3CR1 microglia knockout background. These data support the use of anti-TIM3 in clinical trials of pediatric low-grade MAPK-driven gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f93c9fb9117641aac2f760a5c311e07768d3cba6" target='_blank'>
              Pediatric glioma immune profiling identifies TIM3 as a therapeutic target in BRAF-Fusion pilocytic astrocytoma.
              </a>
            </td>
          <td>
            Shashwat Tripathi, Hinda Najem, Corey Dussold, Sebastian Pacheco, Ruochen Du, Moloud Sooreshjani, Lisa A Hurley, James P Chandler, R. Stupp, A. Sonabend, C. Horbinski, R. Lukas, Joanne Xiu, Giselle Y López, Theodore Nicolaides, Valerie Brown, N. R. Wadhwani, Sandi K Lam, C. D. James, Ganesh Rao, Maria G Castro, Amy B. Heimberger, Michael DeCuypere
          </td>
          <td>2024-08-13</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Keratinocytes, the dominant cell type in the melanoma microenvironment during tumor initiation, exhibit diverse effects on melanoma progression. Using a zebrafish model of melanoma and human cell co-cultures, we observed that keratinocytes undergo an Epithelial–Mesenchymal Transition (EMT)-like transformation in the presence of melanoma, reminiscent of their behavior during wound healing. Surprisingly, overexpression of the EMT transcription factor Twist in keratinocytes led to improved overall survival in zebrafish melanoma models, despite no change in tumor initiation rates. This survival benefit was attributed to reduced melanoma invasion, as confirmed by human cell co-culture assays. Single-cell RNA-sequencing revealed a unique melanoma cell cluster in the Twist-overexpressing condition, exhibiting a more differentiated, less invasive phenotype. Further analysis nominated homotypic jam3b-jam3b and pgrn-sort1a interactions between Twist-overexpressing keratinocytes and melanoma cells as potential mediators of the invasive restraint. Our findings suggest that EMT in the tumor microenvironment (TME) may limit melanoma invasion through altered cell-cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8055a65d64a36749562fed897a59067bde3fe48" target='_blank'>
              Restraint of melanoma progression by cells in the local skin environment
              </a>
            </td>
          <td>
            Yilun Ma, Mohita M. Tagore, Miranda V. Hunter, Ting-Hsiang Huang, Emily D. Montal, Joshua M. Weiss, Richard M. White
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background CD24 plays a crucial role not only in promoting tumor progression and metastasis but also in modulating macrophage-mediated anti-tumor immunity. However, the impact of tumor CD24 on the immune landscape of the tumor microenvironment (TME) remains poorly explored. Here, we investigated the role of CD24a, murine CD24 gene, in the progression and immune dynamics of the tumor microenvironment (TME) in the 4T1 murine model of triple-negative breast cancer (TNBC). Methods: We employed Clustered Regularly Interspaced Short Palindromic Repeat (CRISPR)/Cas9 technology to perform a gene knockout of Cd24a in 4T1 cells. Flow cytometry was utilized to analyze the distribution and number of immune cells, including myeloid-derived suppressor cells (MDSCs), natural killer (NK) cells, T cells, and macrophages, within tumors, spleens, and bone marrow. Immunofluorescence (IF) staining was used to detect these immune cells in tumor sections. Additionally, ANDOR Dragonfly High-Speed Confocal was used to perform three-dimensional (3D) mapping of mouse tumors. Results Our study showed that knocking out CD24a significantly impeded tumor progression and prolonged mouse survival. Flow cytometry and IF analysis revealed that the loss of CD24a transformed tumor microenvironment from cold to hot by promoting the infiltration of M1 macrophages, cytotoxic CD8+ T cells, and CD49b+ natural killer (NK) cells while reducing the recruitment and expansion of granulocytic myeloid-derived suppressor cells (gMDSCs) in the TME. Additionally, the 3D mapping of TME further validated the “hot state” of CD24a knockout tumors. Conclusions Our study provides the first evidence that targeting CD24a could effectively reprogram the TME, enhancing its immunogenicity, and transforming immune cold tumors into hot tumors. This strategy may offer a promising therapeutic strategy for enhancing the immune response against poorly immunogenic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f888be94df7f979fefb9bdbfa437c0f898437cb" target='_blank'>
              CD24a knockout transforms the tumor microenvironment from cold to hot by promoting tumor-killing immune cell infiltration in a murine triple-negative breast cancer model
              </a>
            </td>
          <td>
            Shih-Hsuan Chan, Hsuan-Jung Tseng, Lu-Hai Wang
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSC) remains a disease of poor prognosis that is unresponsive to current immune checkpoint inhibitors. Although PI3K pathway alterations, such as PTEN loss, are common in HGSC, attempts to target this pathway have been unsuccessful. We hypothesized that aberrant PI3K pathway activation may alter the HGSC immune microenvironment and present a targeting opportunity. Single-cell RNA sequencing identified populations of resident macrophages specific to Pten-null omental tumors in murine models, which were confirmed by flow cytometry. These macrophages derived from peritoneal fluid macrophages and had a unique gene expression program, marked by high expression of the enzyme heme oxygenase-1 (HMOX1). Targeting resident peritoneal macrophages prevented the appearance of HMOX1hi macrophages and reduced tumor growth. Furthermore, direct inhibition of HMOX1 extended survival in vivo. RNA sequencing identified IL33 in Pten-null tumor cells as a likely candidate driver leading to the appearance of HMOX1hi macrophages. Human HGSC tumors also contained HMOX1hi macrophages with a corresponding gene expression program. Moreover, the presence of these macrophages correlated with activated tumoral PI3K/mTOR signaling and poor overall survival in HGSC patients. In contrast, tumors with low numbers of HMOX1hi macrophages were marked by increased adaptive immune response gene expression. These data suggest targeting HMOX1hi macrophages as a potential therapeutic strategy for treating poor prognosis HGSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d6297fbdda342e2a463b24c533bad3a26f20d1" target='_blank'>
              PTEN loss shapes macrophage dynamics in high grade serous ovarian carcinoma.
              </a>
            </td>
          <td>
            S. Spear, O. Le Saux, H. B. Mirza, Nayana Iyer, Katie Tyson, Fabio Grundland Freile, J. Walton, Chloe Woodman, Sheba Jarvis, D. Ennis, Carmen Aguirre Hernandez, Yuewei Xu, P. Spiliopoulou, James D. Brenton, A. P. Costa-Pereira, D. Cook, B. C. Vanderhyden, Hector C Keun, E. Triantafyllou, James N Arnold, I. McNeish
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer bone metastasis is the leading cause of mortality in patients with advanced breast cancer. Although decreased mineral density is a known risk factor for bone metastasis, the underlying mechanisms remain poorly understood because studying the isolated effect of bone mineral density on tumor heterogeneity is challenging with conventional approaches. Here, we investigate how bone mineral content affects tumor growth and microenvironmental complexity in vivo by combining single-cell RNA-sequencing with mineral-containing or mineral-free decellularized bone matrices. We discover that the absence of bone mineral significantly influences fibroblast and immune cell heterogeneity, promoting phenotypes that increase tumor growth and alter the response to injury or disease. Importantly, we observe that the stromal response to matrix mineral content depends on host immunocompetence and the murine tumor model used. Collectively, our findings suggest that bone mineral density affects tumor growth by altering microenvironmental complexity in an organism-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29d79f5245668a68eb4e20b21ea961a72baab370" target='_blank'>
              Bone mineral density affects tumor growth by shaping microenvironmental heterogeneity
              </a>
            </td>
          <td>
            Matthew A. Whitman, Madhav Mantri, Emmanuel Spanos, L. Estroff, I. De Vlaminck, Claudia Fischbach
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="BACKGROUND & AIMS
Metastases from gastric adenocarcinoma (GAC) lead to high morbidity and mortality. Developing innovative and effective therapies requires a comprehensive understanding of the tumor and immune biology of advanced GAC. Yet, collecting matched specimens from advanced, treatment-naïve GAC patients poses a significant challenge, limiting the scope of current research, which has predominantly focused on localized tumors. This gap hinders a deeper insight into the metastatic dynamics of GAC.


METHODS
We performed in-depth single-cell transcriptome and immune profiling on 68 paired, treatment-naïve, primary-metastatic tumors to delineate alterations in cancer cells and their tumor microenvironment (TME) during metastatic progression. To validate our observations, we conducted comprehensive functional studies both in vitro and in vivo, employing cell lines, multiple PDX and novel mouse models of GAC.


RESULTS
Liver and peritoneal metastases exhibited distinct properties in cancer cells and dynamics of TME phenotypes, supporting the notion that cancer cells and their local TMEs co-evolve at metastatic sites. Our study also revealed differential activation of cancer meta-programs across metastases. We observed evasion of cancer cell ferroptosis via GPX4 upregulation during GAC progression. Conditional depletion of Gpx4 or pharmacological inhibition of ferroptosis resistance significantly attenuated tumor growth and metastatic progression. Additionally, ferroptosis-resensitizing treatments augmented the efficacy of CAR T-cell therapy.


CONCLUSIONS
This study represents the largest single-cell dataset of metastatic GACs to date. High-resolution mapping of the molecular and cellular dynamics of GAC metastasis has revealed a rationale for targeting ferroptosis defense in combination with CAR T-cell therapy as a novel therapeutic strategy with potential immense clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c4774948f3c82306cdc21b3d5730989f11741b" target='_blank'>
              Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response.
              </a>
            </td>
          <td>
            Xiangdong Cheng, Enyu Dai, Jibo Wu, Natasha M. Flores, Yanshuo Chu, Ruiping Wang, M. Dang, Zhiyuan Xu, Guangchun Han, Yunhe Liu, D. Chatterjee, Can Hu, Jieer Ying, Yi-an Du, Litao Yang, Xiaoqing Guan, Shaowei Mo, Xuanye Cao, Guangsheng Pei, Jiahui Jiang, Xiaoyin Lu, Ana Morales Benitez, R. Waters, Melissa P Pizzi, N. Shanbhag, Yibo Fan, F. Peng, Samir Hanash, George A. Calin, Andrew Futreal, Shumei Song, Cassian Yee, Pawel K Mazur, Jiang-Jiang Qin, J. Ajani, Linghua Wang
          </td>
          <td>2024-08-01</td>
          <td>Gastroenterology</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="Tumor growth dynamics serve as a critical aspect of understanding cancer progression and treatment response to mitigate one of the most pressing challenges in healthcare. The in silico approach to understanding tumor behavior computationally provides an efficient, cost-effective alternative to wet-lab examinations and are adaptable to different environmental conditions, time scales, and unique patient parameters. As a result, this paper explored modeling of free tumor growth in cancer, surveying contemporary literature on continuum, discrete, and hybrid approaches. Factors like predictive power and high-resolution simulation competed against drawbacks like simulation load and parameter feasibility in these models. Understanding tumor behavior in different scenarios and contexts became the first step in advancing cancer research and revolutionizing clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5814b4312284d51679616c4dcd34ca6a0385e075" target='_blank'>
              Modeling free tumor growth: Discrete, continuum, and hybrid approaches to interpreting cancer development.
              </a>
            </td>
          <td>
            Dashmi Singh, Dana Paquin
          </td>
          <td>2024-07-19</td>
          <td>Mathematical biosciences and engineering : MBE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The functional heterogeneity and ecological niche of prostate cancer stem cells (PCSCs), which are major drivers of prostate cancer development and treatment resistance, have attracted considerable research attention. Cancer-associated fibroblasts (CAFs), which are crucial components of the tumor microenvironment (TME), substantially affect PCSC stemness. Additionally, CAFs promote PCSC growth and survival by releasing signaling molecules and modifying the surrounding environment. Conversely, PCSCs may affect the characteristics and behavior of CAFs by producing various molecules. This crosstalk mechanism is potentially crucial for prostate cancer progression and the development of treatment resistance. Using organoids to model the TME enables an in-depth study of CAF-PCSC interactions, providing a valuable preclinical tool to accurately evaluate potential target genes and design novel treatment strategies for prostate cancer. The objective of this review is to discuss the current research on the multilevel and multitarget regulatory mechanisms underlying CAF-PCSC interactions and crosstalk, aiming to inform therapeutic approaches that address challenges in prostate cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebc78492c002bb6ca79859426afb09d4c0e6ab75" target='_blank'>
              Cancer-associated fibroblasts and prostate cancer stem cells: crosstalk mechanisms and implications for disease progression
              </a>
            </td>
          <td>
            Haoran Chen, Suping Fang, Xudong Zhu, Hao Liu
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The efficiency of immunotherapy stays limited for solid tumors. It is mainly caused by the tumoral structural heterogeneity and its complex microenvironment, which impede the infiltration of immune cells into malignant tissues. Mimicking this environment in frames of microphysiological models remains a challenge, significantly increasing costs of the clinical translation for the new therapies. Here, we study a 3D multi-spheroid model incorporating prostate stem cell antigen (PSCA) modified PC3 human prostate cancer cells and fibroblast activation protein (FAP) expressing fibrosarcoma HT1080 cells embedded within the soft hydrogel microbeads. We use this model to trial the immunotherapy based on the universal chimeric antigen receptor (UniCAR) T cells, and to better understand the impact of FAP on the immunotherapeutic treatment of solid tumors. First, we demonstrate the successful chemoattraction and infiltration of UniCAR T cells into the area of solid tumors, as well as the ability of UniCAR T cells to navigate through artificial extracellular matrix barriers. We further observe the synergistic efficacy of a dual-targeting UniCAR T cell approach against FAP and PSCA antigens, which represent the tumor microenvironment and the tumor, respectively. The results of our studies offer valuable methodologies and insights for engineering different 3D tumor models and studying immunotargeting of small-sized solid tumors (e.g., metastases and residual tumors). The developed microphysiological system has great potential to advance cancer research efforts aiming to elucidate the pivotal role of microenvironment in solid tumor development, enabling therapy trials and more precise prognosis for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2135fca4a8d632d002305df6b26cbcaa6ed0dc5d" target='_blank'>
              3D Microphysiological Tumor Model for Dual-Targeting CAR T Cell Immunotherapy
              </a>
            </td>
          <td>
            Xuan Peng, Ž. Janićijević, L. R. Loureiro, Lydia Hoffmann, Poh Soo Lee, Isli Cela, B. Kruppke, A. Kegler, A. Feldmann, Michael P Bachmann, L. Baraban
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="While early prostate cancer (PCa) depends on the androgen receptor (AR) signaling pathway, which is predominant in luminal cells, there is much to be understood about the contribution of epithelial basal cells in cancer progression. Herein, we observe cell-type specific differences in the importance of the metabolic enzyme phosphatidylinositol 5-phosphate 4-kinase alpha (PI5P4Kα; gene name PIP4K2A) in the prostate epithelium. We report the development of a basal-cell-specific genetically engineered mouse model (GEMM) targeting Pip4k2a alone or in combination with the tumor suppressor phosphatase and tensin homolog (Pten). PI5P4Kα is enriched in basal cells, and no major histopathological changes were detectable following gene deletion. Notably, the combined loss of Pip4k2a slowed the development of Pten mutant mouse prostatic intraepithelial neoplasia (mPIN). Through the inclusion of a lineage tracing reporter, we utilize single-cell RNA sequencing to evaluate changes resulting from in vivo downregulation of Pip4k2a and characterize cell populations influenced in the established Probasin-Cre and Cytokeratin 5 (CK5)- Cre driven GEMMs. Transcriptomic pathway analysis points towards the disruption of lipid metabolism as a mechanism for reduced tumor progression. This was functionally supported by shifts of carnitine lipids in LNCaP PCa cells treated with siPIP4K2A. Overall, these data nominate PI5P4Kα as a target for PTEN mutant PCa. One Sentence Summary Loss of PI5P4Kα slows cancer progression in prostate basal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3808ffd57acf44cbae9d5ed91835f3ddfcf8c6b3" target='_blank'>
              Loss of PI5P4Kα slows the progression of a Pten mutant basal cell model of prostate cancer
              </a>
            </td>
          <td>
            Joanna Triscott, Marika Lehner, Andrej Benjak, Matthias Reist, Brooke M. Emerling, C. K. Ng, Simone de Brot, Mark A. Rubin
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Immunologic treatment options are uncommon in low-grade gliomas, although such therapies might be beneficial for inoperable and aggressive cases. Knowledge of the immune and stromal cells in low-grade gliomas is highly relevant for such approaches but still needs to be improved. Published gene-expression data from 400 low-grade gliomas and 193 high-grade gliomas were gathered to quantify 10 microenvironment cell populations with a deconvolution method designed explicitly for brain tumors. First, we investigated general differences in the microenvironment of low- and high-grade gliomas. Lower-grade and high-grade tumors cluster together, respectively, and show a general similarity within and distinct differences between these groups, the main difference being a higher infiltration of fibroblasts and T cells in high-grade gliomas. Among the analyzed entities, gangliogliomas and pleomorphic xanthoastrocytomas presented the highest overall immune cell infiltration. Further analyses of the low-grade gliomas presented three distinct microenvironmental signatures of immune cell infiltration, which can be divided into T-cell/dendritic/natural killer cell-, neutrophilic/B lineage/natural killer cell-, and monocytic/vascular/stromal-cell-dominated immune clusters. These clusters correlated with tumor location, age, and histological diagnosis but not with sex or progression-free survival. A survival analysis showed that the prognosis can be predicted from gene expression, clinical data, and a combination of both with a support vector machine and revealed the negative prognostic relevance of vascular markers. Overall, our work shows that low- and high-grade gliomas can be characterized and differentiated by their immune cell infiltration. Low-grade gliomas cluster into three distinct immunologic tumor microenvironments, which may be of further interest for upcoming immunotherapeutic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44cc97fe1933721f54254897b73fb405b889309" target='_blank'>
              Transcriptomics-based characterization of the immuno-stromal microenvironment in pediatric low-grade glioma
              </a>
            </td>
          <td>
            M. Körner, Michael Spohn, U. Schüller, M. Bockmayr
          </td>
          <td>2024-08-09</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Lung cancer, including non-small-cell lung cancer (NSCLC), is one of the leading causes of cancer-related deaths worldwide. NSCLC treatment has undergone a major paradigm shift with the advent of targeted therapy and immunotherapy, which require the specific tumor and immune signatures of NSCLC to be tested to ensure careful patient selection and prediction of clinical benefits from the selected therapy. Tumor samples with large fibrotic areas and the focal nature of biomarker expression may be a source of bias in histological assays. The ex vivo analysis developed in this work allows us to rapidly explore various characteristics of cancer and tumor-infiltrating immune cells after their separation from fibrotic tissue in solid NSCLC samples and to have a better understanding of the immunological landscape and expression of different cell markers in the tumor as a whole, thereby avoiding some false negatives in the histological assay of the same tumor samples. Abstract NSCLC is a very aggressive solid tumor, with a poor prognosis due to post-surgical recurrence. Analysis of the specific tumor and immune signatures of NSCLC samples is a critical step in prognostic evaluation and management decisions for patients after surgery. Routine histological assays have some limitations. Therefore, new diagnostic tools with the capability to quickly recognize NSCLC subtypes and correctly identify various markers are needed. We developed a technique for ex vivo isolation of cancer and immune cells from surgical tumor and lung tissue samples of patients with NSCLC (adenocarcinomas and squamous cell carcinomas) and their examination on ex vivo cell preparations and, parallelly, on histological sections after Romanovsky–Giemsa and immunofluorescent/immunochemical staining for cancer-specific and immune-related markers. As a result, PD-L1 expression was detected for some patients only by ex vivo analysis. Immune cell profiling in the tumor microenvironment revealed significant differences in the immunological landscapes between the patients’ tumors, with smokers’ macrophages with simultaneous expression of pro- and anti-inflammatory cytokines, neutrophils, and eosinophils being the dominant populations. The proposed ex vivo analysis may be used as an additional diagnostic tool for quick examination of cancer and immune cells in whole tumor samples and to avoid false negatives in histological assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d29642b6785049d2e4dfd77736c81a19d7e54a" target='_blank'>
              Development of Ex Vivo Analysis for Examining Cell Composition, Immunological Landscape, Tumor and Immune Related Markers in Non-Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Elena G. Ufimtseva, M.S. Gileva, Ruslan V. Kostenko, V. V. Kozlov, L. F. Gulyaeva
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58cf60748c00c682a8179debf2fde37709954fb0" target='_blank'>
              Highlighting immune features of the tumor ecosystem and prognostic value of Tfh and Th17 cell infiltration in head and neck squamous cell carcinoma by single-cell RNA-seq
              </a>
            </td>
          <td>
            Yan Tian, Chao Liu, Wenhui Yang, Xiaohui Li, Min Zhang, Yan Xiong, Xueying Ren, Zhiguo Ma, Xuan Jin, Yanping Wu, Xin Dong, Nanlin Hu, Zhijun Xie, Yong Qin, Shikai Wu
          </td>
          <td>2024-08-02</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeting the tumor microenvironment (TME) is an attractive strategy for cancer therapy, as tumor cells in vivo are surrounded by many different influential cell types, with complex interactions strongly affecting tumor progression and therapeutic outcome. Cancer-associated fibroblasts (CAFs) represent an abundant stromal cell type in the TME that modulate tumor development by exerting an immunosuppressive effect to influence effector immune cell activation. One promising target for TME-directed therapy is the CAF marker fibroblast activation protein-α (FAP). In this study, we employ a multicellular three-dimensional (3D) spheroid model, including tumor cells, fibroblast cells, and naïve T cells and could observe a protective effect of fibroblasts on tumor cells. Subsequently, we demonstrate that fibroblasts express FAP at differing expression levels in two-dimensional (2D) versus 3D cells. Lastly, we show that in a triple-culture of tumor cells, T cells and fibroblasts, the simultaneous assembly of fibroblasts using the high-affinity ligand oncoFAP with an engineered α-CD3-scFv-Fc-dextran-oncoFAP construct resulted in effective T cell activation to augment immunogenicity. Overall, this model can be routinely used for preclinical screening to study the effects of fibroblasts on the TME in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ded138adba947a8cd32556d8f5f10db81c5760a" target='_blank'>
              Mimicking the immunosuppressive impact of fibroblasts in a 3D multicellular spheroid model
              </a>
            </td>
          <td>
            Melanie Grotz, Lieke van Gijzel, Peter Bitsch, Stefania C. Carrara, Harald Kolmar, Sakshi Garg
          </td>
          <td>2024-07-26</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Biomarkers, including proteins, nucleic acids, antibodies, and peptides, are essential for identifying diseases such as cancer and differentiating between healthy and abnormal cells in patients. To date, studies have shown that cancer stem cells have DNA repair mechanisms that deter the effects of medicinal treatment. Experiments with cell cultures and chemotherapy treatments of these cultures have revealed the presence of small cells, with a small amount of cytoplasm that can be intensively stained with azure eosin, called microcells. Microcells develop during sporosis from a damaged tumor macrocell. After anticancer therapy in tumor cells, a defective macrocell may produce one or more microcells. This study aims to characterize microcell morphology in melanoma cell lines. In this investigation, we characterized the population of cancer cell microcells after applying paclitaxel treatment to a Sk-Mel-28 melanoma cell line using immunocytochemical cell marker detection and fluorescent microscopy. Paclitaxel-treated cancer cells show stronger expression of stem-associated ALDH2, SOX2, and Nanog markers than untreated cells. The proliferation of nuclear antigens in cells and the synthesis of RNA in microcells indicate cell self-defense, promoting resistance to applied therapy. These findings improve our understanding of microcell behavior in melanoma, potentially informing future strategies to counteract drug resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8ed6b1eaab2b7374876491c10185ffcc9a5b4c3" target='_blank'>
              Morphological and Immunocytochemical Characterization of Paclitaxel-Induced Microcells in Sk-Mel-28 Melanoma Cells
              </a>
            </td>
          <td>
            Z. Simsone, Tālivaldis Feivalds, L. Harju, I. Miķelsone, Ilze Blāķe, Juris Bērziņš, Indulis Buiķis
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multimodal analysis of single-cell samples from healthy and diseased tissues at various stages provides a comprehensive view that identifies disease-specific cells, their molecular features and aids in patient stratification. Here, we present MultiMIL, a novel weakly-supervised multimodal model designed to construct multimodal single-cell references and prioritize phenotype-specific cells via patient classification. MultiMIL effectively integrates single-cell modalities, even when they only partially overlap, providing robust representations for downstream analyses such as phenotypic prediction and cell prioritization. Using a multiple-instance learning approach, MultiMIL aggregates cell-level measurements into sample-level representations and identifies disease-specific cell states through attention-based scoring. We demonstrate that MultiMIL accurately identifies disease-specific cell states in blood and lung samples, identifying novel disease-associated genes and achieving superior patient classification accuracy compared to existing methods. We anticipate MultiMIL will become an essential tool for querying single-cell multiomic atlases, enhancing our understanding of disease mechanisms and informing targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d8efc9b9ff38309e496bbcf1bd6af1cf5075840" target='_blank'>
              Multimodal weakly supervised learning to identify disease-specific changes in single-cell atlases
              </a>
            </td>
          <td>
            Anastasia Litinetskaya, Maiia Shulman, Soroor Hediyeh-zadeh, A. Moinfar, Fabiola Curion, Artur Szałata, Alireza Omidi, M. Lotfollahi, F. Theis
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7254d2e767a2a42779b5a71fd655bef014a5b881" target='_blank'>
              A deep learning method that identifies cellular heterogeneity using nanoscale nuclear features
              </a>
            </td>
          <td>
            Davide Carnevali, Limei Zhong, Esther González-Almela, Carlotta Viana, Mikhail Rotkevich, Aiping Wang, Daniel Franco-Barranco, Aitor González-Marfil, M. V. Neguembor, Álvaro Castells-García, I. Arganda-Carreras, M. Cosma
          </td>
          <td>2024-08-27</td>
          <td>Nature Machine Intelligence</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562977a8b24cea0e019cd93dcf3b43822ec9b974" target='_blank'>
              Leveraging neighborhood representations of single-cell data to achieve sensitive DE testing with miloDE
              </a>
            </td>
          <td>
            A. Missarova, Emma Dann, Leah Rosen, Rahul Satija, J. Marioni
          </td>
          <td>2024-07-18</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54642882e6df67d3e39c892d3f67f253c726f46d" target='_blank'>
              A single-cell pan-cancer analysis to show the variability of tumor-infiltrating myeloid cells in immune checkpoint blockade
              </a>
            </td>
          <td>
            Weiyuan Li, Lu Pan, Weifeng Hong, F. Ginhoux, Xuan Zhang, Chunjie Xiao, Xuexin Li
          </td>
          <td>2024-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55bf88cd2f361cbe0e78275013e0864970ee199" target='_blank'>
              In vivo CRISPR screens identify a dual function of MEN1 in regulating tumor-microenvironment interactions.
              </a>
            </td>
          <td>
            Peiran Su, Yin Liu, Tianyi Chen, Yibo Xue, Yong Zeng, Guanghui Zhu, Sujun Chen, Mona Teng, Xinpei Ci, Mengdi Guo, Michael Y. He, Jun Hao, Vivian Chu, Wenxi Xu, Shiyan Wang, P. Mehdipour, Xin Xu, Sajid A. Marhon, F. Soares, Nhu-An Pham, Bell Xi Wu, Peter Hyunwuk Her, Shengrui Feng, Najd Alshamlan, Maryam Khalil, Rehna Krishnan, Fangyou Yu, Chang Chen, Francis Burrows, Razqallah Hakem, Mathieu Lupien, Shane Harding, Benjamin H. Lok, Catherine O'Brien, Alejandro Berlin, D. D. De Carvalho, David G Brooks, Daniel Schramek, Ming-Sound Tsao, Housheng Hansen He
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Predicting the transcriptional response of chemical perturbations is crucial to understanding gene function and developing drug candidates, promising a streamlined drug development process. Single-cell sequencing has provided an ideal data basis for training machine learning models for this task. Recent advances in deep learning have led to significant improvements in predictions of chemical as well as genetic perturbations at the single cell level. Experiments have shown that different cell types exhibit distinct transcriptional patterns and responses to perturbation. This poses a fundamental problem for predicting transcriptional responses of drugs or cell types outside the training data. Accordingly, existing methods lack cell-type-specific modeling or do not explicitly provide an interpretable mechanism for the gene features. In this study, we introduce a novel approach that employs a network representation of various cell types as an inductive bias, improving prediction performance in scenarios with limited data while acknowledging cellular differences. We applied our framework to four small-scale single-cell perturbation datasets and one large-scale screening experiment, demonstrating that this representation can inherently generalize to previously unseen cell types. Furthermore, our method outperforms the state-of-the-art methods in predicting the post-perturbation response in unobserved cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/800792362ef249f4ba2c5bd2c2590459a14303c2" target='_blank'>
              PrePR-CT: Predicting Perturbation Responses in Unseen Cell Types Using Cell-Type-Specific Graphs
              </a>
            </td>
          <td>
            Reem Alsulami, Robert Lehmann, S. Khan, Vincenzo Lagani, D. Gómez-Cabrero, N. Kiani, Jesper N. Tegnér
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/257cd6f239d65625f17a430679485034087c6c92" target='_blank'>
              Advances in targeting cancer-associated fibroblasts through single-cell spatial transcriptomic sequencing
              </a>
            </td>
          <td>
            Pin Lyu, X. Gu, Fuqi Wang, Haifeng Sun, Quanbo Zhou, Shuaixi Yang, Weitang Yuan
          </td>
          <td>2024-07-29</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0fe9cf1feacb859f4a39031fb5dd0bee75bed8" target='_blank'>
              Pre-existing cell subpopulations in primary prostate cancer tumors display surface fingerprints of docetaxel-resistant cells.
              </a>
            </td>
          <td>
            S. Drápela, Barbora Kvokačková, Eva Slabáková, A. Kotrbová, Kristína Gömöryová, R. Fedr, Daniela Kurfürstová, Martin Eliáš, Vladimír Študent, Frederika Lenčéšová, Ganji Sri Ranjani, Vendula Pospíchalová, Vítězslav Bryja, W. V. van Weerden, M. Puhr, Z. Culig, J. Bouchal, Karel Souček
          </td>
          <td>2024-08-20</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Skeletal muscle regeneration is mediated by resident muscle stem cells that produce progeny to repair or recreate muscle. Critical to this function is the ability to transition between states of proliferation and quiescence. This balancing act is disrupted with age, leading to eroded regenerative capacity. Notwistanding, mechanisms by which the regenerating niche directs MuSCs return to the dormant state are largely unknown. Since single-cell RNAseq methods exclude the analysis of multinucleated cells, we generated single-nuclei RNAseq datasets of regenerating muscle to capture the full breadth of myogenic progression. With this, we uncovered new transition states between differentiating myocytes and syncytial multinucleated cells. Using cell communication inference tools, we highlighted receptor-ligand interactions between MuSCs and fusing nuclei. We leveraged a bespoke biomimetic 3D niche that induces MuSC quiescence, to filter the predicted interactions using a Cas9-based functional genomics approach. We found the proteoglycan Perlecan (Hspg2) promotes MuSC re-quiescence. Hspg2 silencing in vivo, during muscle regeneration, induced an aging-like phenotype and perturbed MuSC re-quiescence. Notably, the temporal profile and overall levels of Perlecan were altered with age. Exogenous supplementation with Endorepellin (truncated Perlecan) rescued MuSC decline following aged muscle regeneration, offering a new therapeutic target. Thus, coupling in silico predictions with a 3D in vitro assay followed by in vivo investigations revealed a previously inaccessible window of biology; that spatiotemporal coordination of MuSC re-quiescence is directed by fusing myonuclei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8472bde10017480be8b7a03b85bf49bab5ad8ea6" target='_blank'>
              Integrating in silico predictions with an engineered tissue assay identifies Perlecan as an age-perturbed re-quiescence cue for muscle stem cells
              </a>
            </td>
          <td>
            Erik Jacques, Pauline Garcia, Orane Mercier, Yechen Hu, Cyril Dégletagne, Jade Ravent, Sidy Fall, Maira P. Almeida, Aaron R. Wheeler, S. Angers, P. Gilbert, F. Le Grand
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e281f15c1b3ddbddd2f378f32dbc2fd4db79f0e" target='_blank'>
              Transcriptomic analysis unveils bona fide molecular signatures of microglia under conditions of homeostasis and viral encephalitis
              </a>
            </td>
          <td>
            Felix Mulenge, Olivia Luise Gern, Lena Mareike Busker, Angela Aringo, L. Ghita, I. Waltl, A. Pavlou, Ulrich Kalinke
          </td>
          <td>2024-08-17</td>
          <td>Journal of Neuroinflammation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Starting from one totipotent cell, complex multicellular organisms form through a series of differentiation and morphogenetic events, culminating in a multitude of cell types arranged in a functional and intricate spatial pattern. To do so, cells coordinate with each other, resulting in dynamics which follow a precise developmental trajectory, constraining the space of possible embryo-to-embryo variation. Using recent single-cell sequencing data of early ascidian embryos, we leverage natural variation together with modeling and inference techniques from statistical physics to investigate development at the level of a complete interconnected embryo – an embryonic transcriptome. After developing a robust and biophysically motivated approach to identifying distinct transcriptomic states or cell types, a statistical analysis reveals correlations within embryos and across cell types demonstrating the presence of collective variation. From these intra-embryo correlations, we infer minimal networks of cell-cell interactions, which reveal the collective modes of gene expression. Our work demonstrates how the existence and nature of spatial interactions along with the collective modes of expression that they give rise to can be inferred from single-cell gene expression measurements, opening up a wider range of biological questions that can be addressed using sequencing-based modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917441574a1989eb6aea19dc5d9b06ebec4e3fa8" target='_blank'>
              Physical modeling of embryonic transcriptomes identifies collective modes of gene expression
              </a>
            </td>
          <td>
            Dominic J. Skinner, Patrick Lemaire, Madhav Mani
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma clinical outcomes emerge from incompletely understood genetic mechanisms operating within the tumor and its microenvironment. Here, we used single-cell RNA-based spatial molecular imaging (RNA-SMI) in patient-derived archival tumors to reveal clinically relevant markers of malignancy progression and prognosis. We examined spatial gene expression of 203,472 cells inside benign and malignant melanocytic neoplasms, including melanocytic nevi and primary invasive and metastatic melanomas. Algorithmic cell clustering paired with intratumoral comparative two-dimensional analyses visualized synergistic, spatial gene signatures linking cellular proliferation, metabolism, and malignancy, validated by protein expression. Metastatic niches included up-regulation of CDK2 and FABP5, which independently predicted poor clinical outcome in 473 patients with melanoma via Cox regression analysis. More generally, our work demonstrates a framework for applying single-cell RNA-SMI technology toward identifying gene regulatory landscapes pertinent to cancer progression and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c7032e41f64aa5fcf950a9326c9ff13d3e22a6" target='_blank'>
              Melanoma progression and prognostic models drawn from single-cell, spatial maps of benign and malignant tumors
              </a>
            </td>
          <td>
            Nick R. Love, Claire Williams, E. Killingbeck, A. Merleev, Mohammad Saffari Doost, Lan Yu, John D. McPherson, H. Mori, Alexander D. Borowsky, E. Maverakis, Maija Kiuru
          </td>
          <td>2024-07-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Metastasis is a hallmark of cancer that is responsible for the greatest number of cancer-related deaths (~90%). For the metastasis to take place, the primary tumor mass engineers migratory cells that can invade adjacent tissues and circulate to distant organs where cancer cells have sufficient nutrients to grow and form new colonies. The migrating cell group typically consists of “LEADER” cells at the front, which navigate the path lead collective invasive packs, and “FOLLOWER” cells that trail behind and support the movement. Both leader and follower cells communicate and cooperate to achieve collective cancer migration and invasion. Recent studies showed that the elimination of a leader cell disrupts collective cell migration. However, The mechanisms controlling the synchronized migration of leader and follower cells are still unresolved. Therefore, the genetic and metabolic traits of leader and follower cells, enabling them to navigate their way into secondary nutrient-rich sites and escape the immune system surveillance, need further elucidation. Interestingly, the differential Multi-omic analyses of cancer leader and follower cells are an emerging area of research within the study of tumor heterogeneity and metastasis. This review highlights the recent findings about leader/follower cancer cells focusing on their unique proteomic, metabolic, and transcriptomic signatures rendering them their unique involvement in cancer metastasis and invasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04fc44097d774157998b41ef30091e3fd7a1d4f3" target='_blank'>
              Differential Analyses of Leader and Follower Cancer Cells
              </a>
            </td>
          <td>
            Ashraf M. Omar, Amira A. Abdellatef
          </td>
          <td>2024-07-25</td>
          <td>Future Perspectives of Medical, Pharmaceutical and Environmental Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Development of subtype heterogeneity in breast cancer requires stochastic transitions between cell types. Here, we sought to characterize how NF-κB contributes to breast cancer progression. We built a gene regulatory network model and calibrated it to the expression levels of NF-κB, TWIST1, SIP1, and SLUG in HER2+ and TNBC cell line, using patient and in vitro published data for validation. The model’s epigenetic landscape exhibits two attractor basins separated by a fluctuation-susceptible slow route. NF-κB stochastic fluctuations induce irreversible transitions from HER2+ to TNBC basins at different times, increasing heterogeneity. Mutations altering the availability of NF-κB change the size of the subtype basins, changing the transition probabilities. Our findings enhance the established attractor landscape formulation and deepen understanding of breast cancer heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8337b6fd94c2dc5d7610f3b772efc6d10bcc115" target='_blank'>
              NF-κB Epigenetic Attractor Landscape Drives Breast Cancer Heterogeneity
              </a>
            </td>
          <td>
            Francisco Lopes, B. Pires, Alexandre A. B. Lima, R. Binato, Eliana Abdelhay
          </td>
          <td>2024-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="
 Despite the growing insights into the cellular heterogeneity of pediatric brain tumors, the transcriptional changes and spatial reorganization during disease progression remain largely unexplored. Here, we present an in-depth single-cell and 10X Xenium spatial transcriptomic characterization of 24 matched pediatric high-grade gliomas (pHGGs) in children, adolescents and young adults (AYA) profiled at both diagnosis and recurrence - constituting a total of 55 tissue samples. By stratifying the transcriptional programs activated at recurrence, we identify two distinct global response sets. Hemispheric AYA tumors predominantly exhibit upregulation of extracellular matrix pathways at recurrence, whereas midline tumors preferentially activate generic stress response programs. On a cellular level, we find that heterogeneity within the malignant tumor compartment remains largely unchanged at recurrence. However, a subset of AYA patients with hemispheric pHGG shows enrichment of the neural-progenitor-like (NPC-like) tumor cell state, and compositional changes in the immune and normal cell landscape, characterized by a global decrease in myeloid cells and an increase in oligodendrocyte fractions. Those changes lead, in turn, to enhanced NPC-like tumor cell-to-oligodendrocyte interactions. Spatial analysis of our extensive cohort at unprecedented subcellular resolution further reveals distinct cellular and spatial features associated with tumor progression, particularly in the remodeling of cell-cell interactions and tumor cell neighborhoods. Overall, this study provides a comprehensive longitudinal single-cell and spatial atlas of pHGG, uncovering the extensive cellular heterogeneity associated with tumor progression. Our findings highlight the role of intrinsic and extrinsic tumor adaptations in shaping disease progression in pHGG, offering potential novel therapeutic targets.
 Citation Format: Sara G. Danielli, Sina Neyazi, Olivia A. Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, Johannes Gojo, Keith L. Ligon, Lissa Baird, Sanda Alexandrescu, Jennifer Cotter, Michael Prados, Adam Resnick, Lin Wang, Michelle Monje, Aaron Diaz, Mariella G. Filbin. Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A052.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca1e500481d6ce9fb3ed15507a312915878f86c" target='_blank'>
              Abstract A052 Project HOPE: A spatiotemporal single-cell landscape of high-grade gliomas in children, adolescents and young adults
              </a>
            </td>
          <td>
            Sara G Danielli, S. Neyazi, Olivia A Hack, Li Jiang, Costanza Lo Cascio, Andrezza Nascimento, Cuong Nguyen, Jacob S Rozowsky, Ilon Liu, Owen Hoare, Karin Shamardani, Kennedy Cunliffe-Koehler, J. Gojo, Keith L. Ligon, Lissa Baird, S. Alexandrescu, Jennifer Cotter, Michael Prados, Adam C. Resnick, Lin Wang, M. Monje, Aaron Diaz, Mariella G. Filbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gallbladder carcinoma (GBC) is a malignant hepatobiliary cancer characterized by an intricate tumor microenvironments (TME) and heterogeneity. The traditional GBC 2D culture models cannot faithfully recapitulate the characteristics of the TME. Three-dimensional (3D) bioprinting enables the establishment of high-throughput and high-fidelity multicellular GBC models. In this study, we designed a concentric cylindrical tetra-culture model to reconstitute the spatial distribution of cells in tumor tissue, with the inner portion containing GBC cells, and the outer ring containing a mixture of endothelial cells, fibroblasts, and macrophages. We confirmed the survival, proliferation, biomarker expression and gene expression profiles of GBC 3D tetra-culture models. Hematoxylin-eosin (HE) and immunofluorescence staining verified the morphology and robust expression of GBC/endothelial/fibroblast/macrophage biomarkers in GBC 3D tetra-culture models. Single-cell RNA sequencing revealed two distinct subtypes of GBC cells within the model, glandular epithelial and squamous epithelial cells, suggesting the mimicry of intratumoral heterogeneity. Comparative transcriptome profile analysis among various in vitro models revealed that cellular interactions and the TME in 3D tetra-culture models reshaped the biological processes of tumor cells to a more aggressive phenotype. GBC 3D tetra-culture models restored the characteristics of the TME as well as intratumoral heterogeneity. Therefore, this model is expected to have future applications in tumor biology research and antitumor drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc6d2cc903e389bd57ed19225ad293a9f8472b5b" target='_blank'>
              Multicellular 3D bioprinted human gallbladder carcinoma forin vitromimicry of tumor microenvironment and intratumoral heterogeneity.
              </a>
            </td>
          <td>
            Yukai Jin, Jiangang Zhang, Jiali Xing, Yiran Li, Huiyu Yang, Liujian Ouyang, Zhiyuan Fang, Lejia Sun, Bao Jin, Pengyu Huang, Huayu Yang, Shunda Du, X. Sang, Yilei Mao
          </td>
          <td>2024-08-09</td>
          <td>Biofabrication</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The success of chimeric antigen receptor (CAR) T-cell therapy in treating hematologic malignancies has generated widespread interest in translating this technology to solid cancers. However, issues like tumor infiltration, the immunosuppressive tumor microenvironment, and tumor heterogeneity limit its efficacy in the solid tumor setting. Recent experimental and clinical studies propose local administration directly into the tumor or at the tumor site to increase CAR T-cell infiltration and improve treatment outcomes. Characteristics of the types of solid tumors that may be the most receptive to this treatment approach remain unclear. In this work, we develop a spatiotemporal model for CAR T-cell treatment of solid tumors, and use numerical simulations to compare the effect of introducing CAR T cells via intratumoral injection versus intracavitary administration in diverse cancer types. We demonstrate that the model can recapitulate tumor and CAR T-cell data from imaging studies of local administration of CAR T cells in mouse models. Our results suggest that locally administered CAR T cells will be most successful against slowly proliferating, highly diffusive tumors, which have the lowest average tumor cell density. These findings affirm the clinical observation that CAR T cells will not perform equally across different types of solid tumors, and suggest that measuring tumor density may be helpful when considering the feasibility of CAR T-cell therapy and planning dosages for a particular patient. We additionally find that local delivery of CAR T cells can result in deep tumor responses, provided that the initial CAR T-cell dose does not contain a significant fraction of exhausted cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7a6bf493c0b33ad1aad3083fdba685e7c59e1d" target='_blank'>
              Spatiotemporal dynamics of tumor - CAR T-cell interaction following local administration in solid cancers
              </a>
            </td>
          <td>
            Katherine Owens, Aminur Rahman, Ivana Bozic
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab01bc978e62346ed66c2ee4492c4c5372e12a2c" target='_blank'>
              Characterization of cuproptosis signature in clear cell renal cell carcinoma by single cell and spatial transcriptome analysis
              </a>
            </td>
          <td>
            Xiaohong Zou, Xiaoqing Liu, Huiting Wang, Zhenhua Li, Chen Zhou
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common type of malignancy in children. ALL prognosis after initial diagnosis is generally good; however, patients suffering from relapse have a poor outcome. The tumor microenvironment is recognized as an important contributor to relapse, yet the cell-cell interactions involved are complex and difficult to study in traditional experimental models. In the present study, we established an innovative larval zebrafish xenotransplantation model, that allows the analysis of leukemic cells (LCs) within an orthotopic niche using time-lapse microscopic and flow cytometric approaches. LCs homed, engrafted and proliferated within the hematopoietic niche at the time of transplant, the caudal hematopoietic tissue (CHT). A specific dissemination pattern of LCs within the CHT was recorded, as they extravasated over time and formed clusters close to the dorsal aorta. Interactions of LCs with macrophages and endothelial cells could be quantitatively characterized. This zebrafish model will allow the quantitative analysis of LCs in a functional and complex microenvironment, to study mechanisms of niche mediated leukemogenesis, leukemia maintenance and relapse development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93d8b671ae9c63ac39a6096f61f5f4b266ff65" target='_blank'>
              In vivo monitoring of leukemia-niche interactions in a zebrafish xenograft model
              </a>
            </td>
          <td>
            A. Arner, Andreas Ettinger, Bradley Wayne Blaser, Bettina Schmid, I. Jeremias, Nadia Rostam, Vera Binder-Blaser
          </td>
          <td>2024-08-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="3D bioprinting has become a valuable tool for studying the biology of solid tumors, including glioblastoma multiforme (GBM). Our analysis of publicly available bulk RNA and single-cell sequencing data has allowed us to define the chemotactic profile of GBM tumors and identify the cell types that secrete particular chemokines in the GBM tumor microenvironment (TME). Our findings indicate that primary GBM tissues express multiple chemokines, whereas spherical monocultures of GBM cells significantly lose this diversity. Subsequently, the comparative analysis of GBM spherical monocultures vs. 3D-bioprinted multicultures of cells showed a restoration of chemokine profile diversity in 3D-bioprinted cultures. Furthermore, single-cell RNA-Seq analysis showed that cells of the perivascular niche (pericytes and endocytes) express multiple chemokines in the GBM TME. Next, we 3D-bioprinted cells from two glioblastoma cell lines, U-251 and DK-MG, alone and as co-cultures with mesenchymal stromal cells (representing cells of the perivascular niche) and assessed the chemokine secretome. The results clearly demonstrated that the interaction of tumors and mesenchymal cells leads to in a significant increase in the repertoire and levels of secreted chemokines under culture in 21% O2 and 1% O2. Our study indicates that cells of the perivascular niche may perform a substantial role in shaping the chemokine microenvironment in GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae245299abc68f53e5e3fd8e4dc327ea2b0cb63" target='_blank'>
              3D-Bioprinted Co-Cultures of Glioblastoma Multiforme and Mesenchymal Stromal Cells Indicate a Role for Perivascular Niche Cells in Shaping Glioma Chemokine Microenvironment
              </a>
            </td>
          <td>
            K. Zielniok, Kinga Rusinek, A. Słysz, Mieszko Lachota, Ewa Bączyńska, Natalia Wiewiórska-Krata, Anna Szpakowska, Martyna Ciepielak, B. Foroncewicz, K. Mucha, Radosław Zagożdżon, Zygmunt Pojda
          </td>
          <td>2024-08-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d5f0ed69b574960ec0cf8e7cfd67555c4fa973" target='_blank'>
              Drug resistant pancreatic cancer cells exhibit altered biophysical interactions with stromal fibroblasts in imaging studies of 3D co-culture models.
              </a>
            </td>
          <td>
            Eric Struth, M. Labaf, V. Karimnia, Yiran Liu, Gwendolyn Cramer, Joanna B. Dahl, Frank J Slack, Kourosh Zarringhalam, Jonathan P Celli
          </td>
          <td>2024-09-05</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80f973d71a8ce36e5c86d44d1a5719a54c336" target='_blank'>
              Targeting axonal guidance dependencies in glioblastoma with ROBO1 CAR T cells.
              </a>
            </td>
          <td>
            C. Chokshi, Muhammad Vaseem Shaikh, Benjamin A Brakel, Martin Rossotti, D. Tieu, W. Maich, Alisha Anand, Shawn C Chafe, Kui Zhai, Yujin Suk, Agata M Kieliszek, P. Miletic, N. Mikolajewicz, David Chen, Jamie D McNicol, Katherine Chan, Amy H Y Tong, L. Kuhlmann, Lina Liu, Zahra Alizada, Daniel Mobilio, N. Tatari, N. Savage, Nikoo Aghaei, Shan Grewal, Anish Puri, M. Subapanditha, D. Mckenna, V. Ignatchenko, Joseph M. Salamoun, Jacek M Kwiecien, Peter Wipf, E. Sharlow, J. Provias, Jian-Qiang Lu, John S Lazo, T. Kislinger, Yu Lu, Kevin R. Brown, C. Venugopal, Kevin Henry, Jason Moffat, Sheila K Singh
          </td>
          <td>2024-08-02</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic disease is the leading cause of cancer-related death, despite recent advances in therapeutic interventions. Prior modeling approaches have accounted for the adaptive immune system’s role in combating tumors, which has led to the development of stochastic models that explain cancer immunoediting and tumor-immune co-evolution. However, cancer immunemediated dormancy, wherein the adaptive immune system maintains a micrometastatic population by keeping its growth in check, remains poorly understood. Immune-mediated dormancy can significantly delay the emergence (and therefore detection) of metastasis. An improved quantitative understanding of this process will thereby improve our ability to identify and treat cancer during the micrometastatic period. Here, we introduce a generalized stochastic model that incorporates the dynamic effects of immunomodulation within the tumor microenvironment on T cell-mediated cancer killing. This broad class of nonlinear birth-death model can account for a variety of cytotoxic T cell immunosuppressive effects, including regulatory T cells, cancer-associated fibroblasts, and myeloid-derived suppressor cells. We develop analytic expressions for the likelihood and mean time of immune escape. We also develop a method for identifying a corresponding diffusion approximation applicable to estimating population dynamics across a wide range of nonlinear birth-death processes. Lastly, we apply our model to estimate the nature and extent of immunomodulation that best explains the timing of disease recurrence in bladder and breast cancer patients. Our findings quantify the effects that stochastic tumor-immune interaction dynamics can play in the timing and likelihood of disease progression. Our analytical approximations provide a method of studying population escape in other ecological contexts involving nonlinear transition rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01deb4ea3ea2c479555870ecd9542f8fc8d05376" target='_blank'>
              How Modulation of the Tumor Microenvironment Drives Cancer Immune Escape Dynamics
              </a>
            </td>
          <td>
            Pujan Shrestha, Zahra S. Ghoreyshi, Jason T. George
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier B. Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Understanding interplay of breast cancer and microenvironment is critical. Here, we identified two transcriptomic subtypes and five immune infiltration patterns from RNA-seq and multiplex immunohistochemistry from 21 ER+/HER2- invasive lobular breast cancers. The proliferative subtype associated with increased immune infiltration especially by immunosuppressive regulatory T-cells and macrophages. We also defined a TAM-Low signature, which associated with lower infiltration of proliferative, pro-inflammatory TAM, and improved outcome in patients with ER+ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13ba7e3cf03c88a995520cfd413c5d66f042dd69" target='_blank'>
              Immune Infiltration Correlates with Transcriptomic Subtypes in Primary ER+ Invasive Lobular Breast Cancer
              </a>
            </td>
          <td>
            Fangyuan Chen, Sayali S Onkar, Jian Zou, Yujia Li, Haley Arbore, Sai Maley, George C. Tseng, Peter Lucas, Tullia Bruno, Dario Vignali, Julia Foldi, Marija Balic, Adrian Lee, S. Oesterreich
          </td>
          <td>2024-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Human cytomegalovirus (HCMV) infection is common in tumor tissues across different types of cancer. While HCMV has not been recognized as a cancer-causing virus, numerous studies hint at its potential role in cancer development where its presence in various cancers corresponds with the hallmarks of cancer. Herein, we discuss and demonstrate that high-risk HCMV-DB and BL strains have the potential to trigger transformation in epithelial cells, including human mammary epithelial cells (HMECs), ovarian epithelial cells (OECs), and prostate epithelial cells (PECs), through the generation of polyploid giant cancer cells (PGCCs). A discussion is provided on how HCMV infection creates a cellular environment that promotes oncogenesis, supporting the continuous growth of CMV-transformed cells. The aforementioned transformed cells, named CTH, CTO, and CTP cells, underwent giant cell cycling with PGCC generation parallel to dedifferentiation, displaying stem-like characteristics and an epithelial–mesenchymal transition (EMT) phenotype. Furthermore, we propose that giant cell cycling through PGCCs, increased EZH2 expression, EMT, and the acquisition of malignant traits represent a deleterious response to the cellular stress induced by high-risk oncogenic HCMV strains, the latter being the origin of the transformation process in epithelial cells upon HCMV infection and leading to adenocarcinoma of poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139ca38ce75a83fdf75794021da421194cdfffd" target='_blank'>
              Polyploid Giant Cancer Cells: A Distinctive Feature in the Transformation of Epithelial Cells by High-Risk Oncogenic HCMV Strains
              </a>
            </td>
          <td>
            Georges Herbein, Ranim El Baba
          </td>
          <td>2024-07-31</td>
          <td>Viruses</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/191b232f0cd9a546f0ef1129ae0b68bee5dbd12e" target='_blank'>
              In vivo interaction screening reveals liver-derived constraints to metastasis
              </a>
            </td>
          <td>
            Costanza Borrelli, Morgan Roberts, Davide Eletto, Marie-Didiée Hussherr, H. Fazilaty, Tomas Valenta, Atefeh Lafzi, Jonas A. Kretz, Elena Guido Vinzoni, Andromachi Karakatsani, Srivathsan Adivarahan, Ardian Mannhart, Shoichiro Kimura, Ab Meijs, Farah Baccouche Mhamedi, I. E. Acar, Kristina Handler, Xenia Ficht, Randall J. Platt, Salvatore Piscuoglio, A. Moor
          </td>
          <td>2024-07-24</td>
          <td>Nature</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity and tissue transcriptomic complexity. However, the high frequency of dropout events in scRNA-seq data complicates downstream analyses such as cell type identification and trajectory inference. Existing imputation methods address the dropout problem but face limitations such as high computational cost and risk of over-imputation. We present SmartImpute, a novel computational framework designed for targeted imputation of scRNA-seq data. SmartImpute focuses on a predefined set of marker genes, enhancing the biological relevance and computational efficiency of the imputation process while minimizing the risk of model misspecification. Utilizing a modified Generative Adversarial Imputation Network architecture, SmartImpute accurately imputes the missing gene expression and distinguishes between true biological zeros and missing values, preventing overfitting and preserving biologically relevant zeros. To ensure reproducibility, we also provide a function based on the GPT4 model to create target gene panels depending on the tissue types and research context. Our results, based on scRNA-seq data from head and neck squamous cell carcinoma and human bone marrow, demonstrate that SmartImpute significantly enhances cell type annotation and clustering accuracy while reducing computational burden. Benchmarking against other imputation methods highlights SmartImpute’s superior performance in terms of both accuracy and efficiency. Overall, SmartImpute provides a lightweight, efficient, and biologically relevant solution for addressing dropout events in scRNA-seq data, facilitating deeper insights into cellular heterogeneity and disease progression. Furthermore, SmartImpute’s targeted approach can be extended to spatial omics data, which also contain many missing values.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6236ffea74eeb68e587d5adb9d7ee59d89926243" target='_blank'>
              SmartImpute: A Targeted Imputation Framework for Single-cell Transcriptome Data
              </a>
            </td>
          <td>
            Sijie Yao, Xiaoqing Yu, Xuefeng Wang
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune composition within the tumor microenvironment (TME) plays a central role in the propensity of cancer to metastasize and to respond to therapy. Previous studies suggested that the metastatic TME is immune suppressed. However, limited accessibility to multiple metastatic sites within patients has made assessment of the immune TME in the context of multi-organ metastases difficult. We utilized a rapid postmortem tissue collection protocol to assess immune composition in numerous sites of breast cancer metastasis and paired tumor-free tissues. Metastases were found to have comparable immune cell densities and composition to paired tumor-free tissues of the same organ type. In contrast, immune cell densities in both metastatic and tumor-free tissues were significantly different between organ types, with lung immune infiltration consistently greater than liver. These immune profiling results were consistent between both flow cytometry and multiplex immunofluorescence-based spatial analysis. Furthermore, we found granulocytes were a predominant tumor-infiltrating immune cell in both lung and liver metastases and these granulocytes made up the majority of PD-L1-expressing cells in many tissue sites. We also identified distinct potential mechanisms of immunosuppression in lung and liver metastases, with lung having increased expression of PD-L1+ antigen-presenting cells and liver having higher numbers of activated regulatory T cells and HLA-DRlow monocytes. Together these results demonstrate that immune contexture of metastases is dictated by organ type, and that immunotherapy strategies may benefit from unique tailoring to tissue-specific features of the immune TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e192c02a5aeb7dbe62c265c54853095175504802" target='_blank'>
              Organ-specific immune setpoints underlie divergent immune profiles across metastatic sites in breast cancer.
              </a>
            </td>
          <td>
            Colt A. Egelston, Weihua Guo, D. Simons, Jian Ye, Christian Avalos, Shawn Solomon, M. Nwangwu, Michael S. Nelson, Jiayi Tan, Eliza R. Bacon, Kena Ihle, Dan Schmolze, L. Tumyan, James R Waisman, Peter P Lee
          </td>
          <td>2024-07-25</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a3cd7f4d8e503d53aded204c426d8531003ab1" target='_blank'>
              State of the Art Modelling of the Breast Cancer Metastatic Microenvironment: Where Are We?
              </a>
            </td>
          <td>
            Mia Nuckhir, David Withey, Sara Cabral, H. Harrison, Robert B. Clarke
          </td>
          <td>2024-07-16</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor-educated platelets (TEPs) are a potential method of liquid biopsy for the diagnosis and monitoring of cancer. However, the mechanism underlying tumor education of platelets is not known, and transcripts associated with TEPs are often not tumor-associated transcripts. We demonstrated that direct tumor transfer of transcripts to circulating platelets is an unlikely source of the TEP signal. We used CDSeq, a latent Dirichlet allocation algorithm, to deconvolute the TEP signal in blood samples from patients with glioblastoma. We demonstrated that a substantial proportion of transcripts in the platelet transcriptome are derived from non-platelet cells, and the use of this algorithm allows the removal of contaminant transcripts. Furthermore, we used the results of this algorithm to demonstrate that TEPs represent a subset of more activated platelets, which also contain transcripts normally associated with non-platelet inflammatory cells, suggesting that these inflammatory cells, possibly in the tumor microenvironment, transfer transcripts to platelets that are then found in circulation. Our analysis suggests a useful and efficient method of processing TEP transcriptomic data to enable the isolation of a unique TEP signal associated with specific tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1407d9261db4f9fb6a0492efe30995b31483fd" target='_blank'>
              Deconvolution of the tumor-educated platelet transcriptome reveals activated platelet and inflammatory cell transcript signatures.
              </a>
            </td>
          <td>
            Jerome M Karp, A. Modrek, R. Ezhilarasan, Ze-Yan Zhang, Yingwen Ding, Melanie Graciani, Ali Sahimi, Michele Silvestro, Ting Chen, Shuai Li, Kwok-Kin Wong, Bhama Ramkhelawon, Krishna Pl Bhat, Erik P Sulman
          </td>
          <td>2024-08-27</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c1721498988e1f8ecba462d4b3ad1583a1728e2" target='_blank'>
              Exploring the prognostic value of BRMS1 + microglia based on single-cell anoikis regulator patterns in the immunologic microenvironment of GBM.
              </a>
            </td>
          <td>
            Songyun Zhao, Kaixiang Ni, Jiaheng Xie, Chao Cheng, Ning Zhao, Jinhui Liu, Wei Ji, Qi Wang, Pengpeng Zhang, Yuan Liu
          </td>
          <td>2024-08-15</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Head and neck cancer (HNC) is the seventh most common cancer globally, resulting in 440 000 deaths per year. While there have been advancements in chemoradiotherapy and surgery, relapse occurs in more than half of HNCs, and these patients have a median survival of 10 months and a 2-year survival of < 20%. Only a subset of patients displays durable benefits from immunotherapies in metastatic and recurrent HNC, making it critical to understand the tumor microenvironment (TME) underpinning therapy responses in HNC. To recognize biological differences within the TME that may be predictive of immunotherapy response, we applied cutting-edge geospatial whole-transcriptome profiling (NanoString GeoMx Digital Spatial Profiler) and spatial proteomics profiling (Akoya PhenoCycler-Fusion) on a tumor microarray consisting of 25 cores from 12 patients that included 4 immunotherapy-unresponsive (8 cores) and 2 immunotherapy-responsive patients (5 cores), as well as 6 immunotherapy naïve patients (12 cores). Through high-plex, regional-based transcriptomic mapping of the tumor and TME, pathways involved with the complement system and hypoxia were identified to be differentially expressed in patients who went on to experience a poor immunotherapy response. Single-cell, targeted proteomic analysis found that immune cell infiltration of the cancer cell mass and interactions of CD8 T cells with tumor and other immune cells were associated with positive immunotherapy response. The relative abundance of specific tumor phenotypes and their interactions with various immune cells was identified to be different between response groups. This study demonstrates how spatial transcriptomics and proteomics can resolve novel alterations in the TME of HNC that may contribute to therapy sensitivity and resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4fc8bbe5e0cd0c92b9460e8e2157b1414a4678" target='_blank'>
              Spatial resolution of the head and neck cancer tumor microenvironment to identify tumor and stromal features associated with therapy response.
              </a>
            </td>
          <td>
            Naomi Berrell, J. Monkman, Meg L. Donovan, Tony Blick, K. O’Byrne, Rahul Ladwa, Chin Wee Tan, A. Kulasinghe
          </td>
          <td>2024-07-24</td>
          <td>Immunology and cell biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Colorectal cancer is a predominant malignancy with a second mortality worldwide. Despite its prevalence, therapeutic options remain constrained and surgical operation is still the most useful therapy. In this regard, a comprehensive spatially resolved quantitative proteome atlas was constructed to explore the functional proteomic landscape of colorectal cancer. This strategy integrates histopathological analysis, laser capture microdissection, and proteomics. Spatial proteome profiling of 200 tissue section samples facilitated by the fully integrated sample preparation technology SISPROT enabled the identification of more than 4000 proteins on the Orbitrap Exploris 240 from 2 mm2 × 10 μm tissue sections. Compared with normal adjacent tissues, we identified a spectrum of cancer-associated proteins and dysregulated pathways across various regions of colorectal cancer including ascending colon, transverse colon, descending colon, sigmoid colon, and rectum. Additionally, we conducted proteomic analysis on tumoral epithelial cells and paracancerous epithelium from early to advanced stages in hallmark rectum cancer and sigmoid colon cancer. Bioinformatics analysis revealed functional proteins and cell-type signatures associated with different regions of colorectal tumors, suggesting potential clinical implications. Overall, this study provides a comprehensive spatially resolved functional proteome landscape of colorectal cancer, serving as a valuable resource for exploring potential biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3187a8d97cb9da8d288b849296a5e0819056a10" target='_blank'>
              Spatial Proteomic Profiling of Colorectal Cancer Revealed Its Tumor Microenvironment Heterogeneity.
              </a>
            </td>
          <td>
            Lifen Xie, Qian Kong, Meiling Ai, An He, Bin Yao, Luobin Zhang, Keren Zhang, Chaowei Zhu, Yangqiu Li, Ligang Xia, Ruijun Tian, Ruilian Xu
          </td>
          <td>2024-07-18</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Pancreatic ductal adenocarcinoma (PDAC) is considered a low immunogenic tumor with "cold" tumor microenvironment (TME) and is mostly unresponsive to immune checkpoint blockade therapies. Here we decipher the impact of intratumoral heterogeneity of immune determinants on antitumor response.


EXPERIMENTAL DESIGN
We performed spatial proteomic and transcriptomic analyses and multiplexed immunofluorescence on multiple tumor regions, including tumor center (TC) and invasive front (IF), from 220 PDAC-patients, classified according to their transcriptomic immune signaling into high-immunogenic (HI-PDACs, n=54) and low-immunogenic tumors (LI-PDACs, n=166). Spatial compartments (tumor: Pancytokeratin+/CD45- and leukocytes: Pancytokeratin-/CD45+) were defined by fluorescent imaging.


RESULTS
HI-PDACs exhibited higher densities of cytotoxic T lymphocytes with upregulation of T-cell priming-associated immune determinants, including CD40, ITGAM, GITR, CXCL10, GZMB, IFNG and HLA-DR, which was significantly more prominent at the IF than the TC. In contrast, LI-PDACs exhibited immune evasive TMEs with downregulation of immune determinants and a negative gradient from TC to IF. Patients with HI-PDACs had significantly better outcomes; however, they showed more frequently exhausted immune phenotypes.


CONCLUSIONS
Our results indicate strategic differences in the regulation of immune determinants, which lead to different levels of effectiveness of antitumor responses between high- and low-immunogenic tumors and dynamic spatial changes, which affect the evolution of immune evasion and patient outcomes. This supports coevolution of tumor and immune cells and may help define therapeutic vulnerabilities to improve antitumor immunity and harness the responsiveness to immune checkpoint inhibitors in PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73b451979296ce7922c3490629c4d76fd0b70420" target='_blank'>
              Spatial heterogeneity of immune regulators drives dynamic changes of local immune responses, affecting disease outcomes in pancreatic cancer.
              </a>
            </td>
          <td>
            E. Karamitopoulou, A. Wenning, A. Acharjee, P. Aeschbacher, I. Marinoni, Inti Zlobec, B. Gloor, Aurel Perren
          </td>
          <td>2024-07-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary The classification of breast cancer has been a complex subject, with diverse origins of normal breast epithelial cells and evolutionary pathways contributing to its extensive heterogeneity, which further complicates the treatment process. We developed a dynamic network model to simulate the emergence of different cellular states and explore the underlying biological mechanisms. This model revealed cellular states that align with four recognized molecular subtypes of breast cancer. It also identified feedback loops that either reinforce or facilitate phenotypic transitions, and allowed us to trace these transitions through the model’s topological structure. This approach provides new insights into the mechanisms driving breast cancer heterogeneity and may benefit the development of more targeted therapeutic strategies. Abstract Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The evaluation of DNA damage response, particularly DNA damage foci formation, is crucial for understanding tumor biology and assessing the impacts of various drugs. We have developed a sophisticated semi-automated image analysis pipeline which generates quantitative map of the spatiotemporal distribution of DNA damage foci within live tumor spheroids. Our framework seamlessly integrates live imaging of tumor spheroids via Light Sheet Fluorescence Microscopy with a DNA damage foci formation assay using a genetically encoded fluorescently labeled DNA damage sensor. By combining advanced imaging techniques with computational tools, our framework offers a powerful tool for studying DNA damage response mechanisms in complex 3D cellular environments. MOTIVATION The motivation of this work is to propose a comprehensive framework that facilitates the study of DNA repair mechanisms within 3D contexts, specifically using tumor spheroid models. By integrating advanced imaging technologies and genetically encoded fluorescent sensors, our goal is to offer researchers a robust methodology for observing and analyzing DNA damage dynamics in realistic tissue-like environments. This framework is designed to enhance accessibility and streamline data processing, thereby empowering the scientific community to investigate DNA repair processes in 3D with greater precision and efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a639e1c3cfceca777754bcf45cdd52cb289b07" target='_blank'>
              A Comprehensive Framework for Spatio-Temporal Analysis of DNA Damage Foci in Tumor Spheroids
              </a>
            </td>
          <td>
            Martine Cazalès, Théo Liu, David Bernard, Denis Jullien, Corinne Lorenzo
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Head and neck cancer disparities in outcomes for Black Americans have been well recognized. However, the specific drivers of the inferior outcomes remain poorly understood. We investigated the biologic features of patient oral cancers and performed a follow-up study of the patient post-surgery recurrences and metastases aiming to explore potential mechanisms that might underpin the poorer outcomes among Black American patients. It was found that high levels of tumor stemness and tumor-promoting stromal characteristics were linked to patient recurrence and metastasis. There were more cases of Black American than White American exhibiting high stemness traits and strong tumor-promoting stromal features associated with tumor recurrences and metastases, although the investigated cases displayed comparable clinical diagnoses. Our findings revealed that the differences in tumor stemness and stromal property among cancers with similar diagnoses contribute to patient outcome disparities. Abstract Black Americans (BAs) with head and neck cancer (HNC) have worse survival outcomes compared to the White patients. While HNC disparities in patient outcomes for BAs have been well recognized, the specific drivers of the inferior outcomes remain poorly understood. Here, we investigated the biologic features of patient tumor specimens obtained during the surgical treatment of oral cancers and performed a follow-up study of the patients’ post-surgery recurrences and metastases with the aim to explore whether tumor biologic features could be associated with the poorer outcomes among BA patients compared with White American (WA) patients. We examined the tumor stemness traits and stromal properties as well as the post-surgery recurrence and metastasis of oral cancers among BA and WA patients. It was found that high levels of tumor self-renewal, invasion, tumorigenesis, metastasis, and tumor-promoting stromal characteristics were linked to post-surgery recurrence and metastasis. There were more BA than WA patients demonstrating high stemness traits and strong tumor-promoting stromal features in association with post-surgery tumor recurrences and metastases, although the investigated cases displayed clinically comparable TNM stages and histological grades. These findings demonstrated that the differences in tumor stemness and stromal property among cancers with comparable clinical diagnoses contribute to the outcome disparity in HNCs. More research is needed to understand the genetic and molecular basis of the biologic characteristics underlying the inferior outcomes among BA patients, so that targeting strategies can be developed to reduce HNC disparity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcad9291ce4667fd7a96a4d38826ce6e7b1c58" target='_blank'>
              Tumor Cell Stemness and Stromal Cell Features Contribute to Oral Cancer Outcome Disparity in Black Americans
              </a>
            </td>
          <td>
            S. Mirshahidi, Isabella J. Yuan, Zhong Chen, Alfred Simental, Steve C. Lee, Pedro A Andrade Filho, Thomas Murry, Feng Zeng, Penelope Duerksen-Hughes, Charles Wang, Xiangpeng Yuan
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mismatch repair proficient (MMRp) tumors of colorectal origin are one of the prevalent yet unpredictable clinical challenges. Despite earnest efforts, optimal treatment modalities have yet to emerge for this class. The poor prognosis and limited actionability of MMRp are ascribed to a low neoantigen burden and a desert-like microenvironment. This review focuses on the critical roadblocks orchestrated by an immune evasive mechanistic milieu in the context of MMRp. The low density of effector immune cells, their weak spatiotemporal underpinnings, and the high-handedness of the IL-17-TGF-β signaling are intertwined and present formidable challenges for the existing therapies. Microbiome niche decorated by Fusobacterium nucleatum alters the metabolic program to maintain an immunosuppressive state. We also highlight the evolving strategies to repolarize and reinvigorate this microenvironment. Reconstruction of anti-tumor chemokine signaling, rational drug combinations eliciting T cell activation, and reprograming the maladapted microbiome are exciting developments in this direction. Alternative vulnerability of other DNA damage repair pathways is gaining momentum. Integration of liquid biopsy and ex vivo functional platforms provide precision oncology insights. We illustrated the perspectives and changing landscape of MMRp-CRC. The emerging opportunities discussed in this review can turn the tide in favor of fighting the treatment dilemma for this elusive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823c184571d6413de5f857c9ff7de13bd6953768" target='_blank'>
              Mismatch repair-proficient tumor footprints in the sands of immune desert: mechanistic constraints and precision platforms
              </a>
            </td>
          <td>
            Biswanath Majumder, N. Nataraj, Leela Maitreyi, Santanu Datta
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract INTRODUCTION Brain metastases pose a significant and growing challenge in clinical practice, yet the molecular variants of this tumor type has not previously been examined comprehensively across multiple cancer types. In particular, there has been limited focus on contrasting the genetic profiles of common brain metastases originating from primary sites like lung, breast, melanoma, colorectal, and renal cancers, with those arising infrequently such as prostate and others. METHODS We have prospectively collected over 125 fresh frozen brain metastases and matching germline reference samples from 14 different primary tumor types and to date have performed WGS, RNA-Seq and Epic 850k methylation profiling on 60 of these samples spanning both common and rare brain metastases. We have employed machine learning algorithms, network analysis techniques, and integrative bioinformatics pipelines to extract meaningful insights into the biological underpinnings of brain metastasis formation. RESULTS We conducted an in-depth investigation into the genomic, transcriptomic, and epigenomic profiles of brain metastases, spanning both common and rare types. By merging these diverse datasets, we identified distinct molecular modifications linked to brain metastases with low incidence rates compared to those more frequently observed. Notably, we observed heightened alterations in the regulation of Golgi dynamics, sensing of lipid species, chromatin remodeling factors, and cytoskeletal remodeling in rare brain metastases. These changes likely influence cell migration and invasion dynamics, elucidating the potential for unique characteristics of less common brain metastasis types. CONCLUSIONS This research indicates that the molecular mechanisms driving the formation of brain metastases may vary depending on whether the primary malignancy frequently or infrequently spreads to the brain. Studies such as these, will aid in eventually driving innovations in precision oncology and ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1030204350b61714078feb89357298754ef4bc7e" target='_blank'>
              BSBM-04 THE MULTI-OMIC LANDSCAPE OF SOMATIC VARIATION ACROSS A PAN-CANCER COHORT OF BRAIN METASTASES
              </a>
            </td>
          <td>
            Grace Hall, Bethany Campbell, Stanley Stylli, Chol-hee Jung, Bernard Pope, Daniel Park, R. Molania, Justin Bedo, Jennifer Ureta, Silvia Rodrigues, Caroline Fidalgo-Ribeiro, Massimo Loda, Patrick McCoy, Clint Gray, Swee Tan, A. Wickremesekera, Christobel Saunders, Kate Drummond, Niall Corcoran, James Dimou, T. Papenfuss, Christopher Hovens
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological malignancy, and therapeutic options and mortality rates over the last three decades have largely not changed. Recent studies indicate that the composition of the tumor immune microenvironment (TIME) influences patient outcomes. To improve spatial understanding of the TIME, we performed multiplexed ion beam imaging on 83 human high-grade serous carcinoma tumor samples, identifying about 160,000 cells across 23 cell types. For 77 of these samples meeting inclusion criteria, we generated composition features based on cell type proportions, spatial features based on the distances between cell types, and spatial network features representing cell interactions and cell clustering patterns, which we linked to traditional clinical and immunohistochemical variables and patient overall survival (OS) and progression-free survival (PFS) outcomes. Among these features, we found several significant univariate correlations, including B-cell contact with M1 macrophages (OS hazard ratio HR=0.696, p=0.011, PFS HR=0.734, p=0.039). We then used high-dimensional random forest models to evaluate out-of-sample predictive performance for OS and PFS outcomes and to derive relative feature importance scores for each feature. The top model for predicting low or high PFS used TIME composition and spatial features and achieved an average AUC (area under the receiver-operating characteristic curve) score of 0.71. The results demonstrate the importance of spatial structure in understanding how the TIME contributes to treatment outcomes. Furthermore, the present study provides a generalizable roadmap for spatial analyses of the TIME in ovarian cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bdeb2935e1da214543c14d951cb57782236e97" target='_blank'>
              The spatial structure of the tumor immune microenvironment can explain and predict patient response in high-grade serous carcinoma.
              </a>
            </td>
          <td>
            Lucy B Van Kleunen, Mansooreh Ahmadian, Miriam D Post, Rebecca J Wolsky, Chrisitan Rickert, Kimberly R. Jordan, Junxiao Hu, Jennifer K Richer, Lindsay W Brubaker, Nicole A. Marjon, K. Behbakht, Matthew J. Sikora, Benjamin G Bitler, Aaron Clauset
          </td>
          <td>2024-08-08</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the analysis of changes in phenotype, such as in vivo and ex vivo morphology and localization of cellular proteins and organelles can provide important insights into patient treatment options. Novel therapies are needed to extend survival in metastatic castration-resistant prostate cancer (mCRPC). Prostate-specific membrane antigen (PSMA), a cell surface glycoprotein that is commonly overexpressed by prostate cancer (PC) cells relative to normal prostate cells, provides a validated target. We developed a software for image analysis designed to identify PSMA expression on the surface of epithelial cells in order to extract prognostic metrics. In addition, our software can deliver predictive information and inform clinicians regarding the efficacy of PC therapy. We can envisage additional applications of our software system, beyond PC, as PSMA is expressed in a variety of tissues. Our method is based on image denoising, topologic partitioning, and edge detection. These three steps allow to segment the area of each PSMA spot in an image of a coverslip with epithelial cells. Our objective has been to present the community with an integrated, easy to use by all, tool for resolving the complex cytoskeletal organization and it is our goal to have such software system approved for use in the clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926fb40317d30bcdb538d1f8f9766fb5c180eb87" target='_blank'>
              Quantitative Microscopy in Medicine
              </a>
            </td>
          <td>
            Alexandre Matov, Shayan Modiri
          </td>
          <td>2024-08-03</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Endometriosis, affecting 1 in 9 women, presents treatment and diagnostic challenges. To address these issues, we generated the biggest single-cell atlas of endometrial tissue to date, comprising 466,371 cells from 35 endometriosis and 25 non-endometriosis patients without exogenous hormonal treatment. Detailed analysis reveals significant gene expression changes and altered receptor-ligand interactions present in the endometrium of endometriosis patients, including increased inflammation, adhesion, proliferation, cell survival, and angiogenesis in various cell types. These alterations may enhance endometriosis lesion formation and offer novel therapeutic targets. Using ScaiVision, we developed neural network models predicting endometriosis of varying disease severity (median AUC = 0.83), including an 11-gene signature-based model (median AUC = 0.83) for hypothesis-generation without external validation. In conclusion, our findings illuminate numerous pathway and ligand-receptor changes in the endometrium of endometriosis patients, offering insights into pathophysiology, targets for novel treatments, and diagnostic models for enhanced outcomes in endometriosis management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de9557348ef1327c4800ef29c1c6e1843f22fe83" target='_blank'>
              Tracing Endometriosis: Coupling deeply phenotyped, single-cell based Endometrial Differences and AI for disease pathology and prediction
              </a>
            </td>
          <td>
            Lea Duempelmann, Shaoline Sheppard, Angelo Duo, Jitka Skrabalova, Brett McKinnon, Thomas Andrieu, Dennis Goehlsdorf, S. Muzumdar, Cinzia Donato, Ryan Lusby, Wiebke Solass, Hans Bösmüller, Peter Nestorov, Michael D. Mueller
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Stable isotope-resolved metabolomics delineates reprogrammed intersecting metabolic networks in human cancers. Knowledge gained from in vivo patient studies provides the "benchmark" for cancer models to recapitulate. It is particularly difficult to model patients' tumor microenvironment (TME) with its complex cell-cell/cell-matrix interactions, which shapes metabolic reprogramming crucial to cancer development/drug resistance. Patient-derived organotypic tissue cultures (PD-OTCs) represent a unique model that retains an individual patient's TME. PD-OTCs of non-small-cell lung cancer better recapitulated the in vivo metabolic reprogramming of patient tumors than the patient-derived tumor xenograft (PDTX), while enabling interrogation of immunometabolic response to modulators and TME-dependent resistance development. Patient-derived organoids (PDOs) are also good models for reconstituting TME-dependent metabolic reprogramming and for evaluating therapeutic responses. Single-cell based 'omics on combinations of PD-OTC and PDO models will afford an unprecedented understanding on TME dependence of human cancer metabolic reprogramming, which should translate into the identification of novel metabolic targets for regulating TME interactions and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6597c82613b1bff5869b53a6356a5e51a2359cc" target='_blank'>
              Metabolic Reprogramming in Human Cancer Patients and Patient-Derived Models.
              </a>
            </td>
          <td>
            T. W. Fan, R. Higashi, A. Lane
          </td>
          <td>2024-07-15</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/609bb25145d5246600ce4feb000ce7f09c4474b7" target='_blank'>
              Building a translational cancer dependency map for The Cancer Genome Atlas
              </a>
            </td>
          <td>
            Xu Shi, Christos Gekas, Daniel Verduzco, Sakina M. Petiwala, Cynthia Jeffries, Charles Lu, Erin Murphy, Tifani Anton, Andy H Vo, Zhiguang Xiao, Padmini Narayanan, Bee-Chun Sun, Aloma L D'Souza, J. M. Barnes, Somdutta Roy, Cyril Ramathal, Michael J. Flister, Zoltan Dezso
          </td>
          <td>2024-07-15</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="This study presents an interdisciplinary approach to analyse the distribution of cancer stem cell markers (CSCMs) across various cancer-affected organs using hypergraphs. Cancer stem cells (CSCs) play a crucial role in cancer initiation, progression, and metastasis. By employing hypergraphs, we model the relationships between CSCM locations and cancerous organs, providing a comprehensive representation of these interactions. Initially, we utilised an unweighted incidence matrix and its Markov transition matrices to gain a dynamic perspective on CSCM distributions. This method allows us to observe how these markers spread and influence cancer progression in a dynamical context. By calculating mutual information for each node and hyperedge, our analysis uncovers complex interaction patterns between CSCMs and organs, highlighting the critical roles of certain markers in cancer progression and metastasis. Our approach offers a detailed representation of cancer stem cell networks, enhancing our understanding of the mechanisms driving cancer heterogeneity and metastasis. By integrating hypergraph theory with cancer biology, this study provides valuable insights for developing targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48368f26aa7447550426a623ac42529547196d96" target='_blank'>
              Mapping Cancer Stem Cell Markers Distribution:A Hypergraph Analysis Across Organs
              </a>
            </td>
          <td>
            D. H. Margarit, Gustavo Paccosi, M. V. Reale, Lilia M. Romanelli
          </td>
          <td>2024-07-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite improvements in patient outcomes, pediatric cancer remains a leading cause of non-accidental death in children. Recent genetic analysis of patients with pediatric cancers indicates an important role for both germline genetic predisposition and cancer-specific somatic driver mutations. Increasingly, evidence demonstrates that the developmental timepoint at which the cancer cell-of-origin transforms is critical to tumor identity and therapeutic response. Therefore, future therapeutic development would be bolstered by the use of disease models that faithfully recapitulate the genetic context, cell-of-origin, and developmental window of vulnerability in pediatric cancers. Human stem cells have the potential to incorporate all of these characteristics into a pediatric cancer model, while serving as a platform for rapid genetic and pharmacological testing. In this review, we describe how human stem cells have been used to model pediatric cancers and how these models compare to other pediatric cancer model modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5451f14ab5137f3c1ef0749fac76fd503e0d7a63" target='_blank'>
              Pediatric Tumors as Disorders of Development: The Case for In Vitro Modeling Based on Human Stem Cells
              </a>
            </td>
          <td>
            Cullen Clairmont, Joanna J Gell, Ching C Lau
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) exert clinical efficacy against various types of cancers by reinvigorating exhausted CD8+ T cells that can expand and directly attack cancer cells (cancer-specific T cells) among tumor-infiltrating lymphocytes (TILs). Although some reports have identified somatic mutations in TILs, their effect on antitumor immunity remains unclear. In this study, we successfully established 18 cancer-specific T cell clones, which have an exhaustion phenotype, from the TILs of four patients with melanoma. We conducted whole-genome sequencing for these T cell clones and identified various somatic mutations in them with high clonality. Among the somatic mutations, an SH2D2A loss-of-function frameshift mutation and TNFAIP3 deletion could activate T cell effector functions in vitro. Furthermore, we generated CD8+ T cell-specific Tnfaip3 knockout mice and showed that Tnfaip3 function loss in CD8+ T cell increased antitumor immunity, leading to remarkable response to PD-1 blockade in vivo. In addition, we analyzed bulk CD3+ T cells from TILs in additional 12 patients and identified an SH2D2A mutation in one patient through amplicon sequencing. These findings suggest that somatic mutations in TILs can affect antitumor immunity and suggest unique biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d5ca5b4ab22ea980ea6c4a3574f4132db1aef93" target='_blank'>
              Somatic mutations in tumor-infiltrating lymphocytes impact on antitumor immunity.
              </a>
            </td>
          <td>
            Fumiaki Mukohara, Kazuma Iwata, Takamasa Ishino, Takashi Inozume, J. Nagasaki, Youki Ueda, K. Suzawa, Toshihide Ueno, Hideki Ikeda, Katsushige Kawase, Yuka Saeki, Shusuke Kawashima, Kazuo Yamashita, Yu Kawahara, Yasuhiro Nakamura, Akiko Honobe-Tabuchi, Hiroko Watanabe, H. Dansako, T. Kawamura, Yutaka Suzuki, Hiroaki Honda, Hiroyuki Mano, Shinichi Toyooka, Masahito Kawazu, Yosuke Togashi
          </td>
          <td>2024-08-21</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by the selective expansion of hematopoietic stem and progenitor cells (HSPCs) carrying somatic mutations. While CHIP is typically asymptomatic, it has garnered substantial attention due to its association with the pathogenesis of multiple disease conditions, including cardiovascular disease (CVD) and hematological malignancies. In this Review, we will discuss seminal and recent studies that have advanced our understanding of mechanisms that drive selection for mutant HSPCs in the BM niche. Next, we will address recent studies evaluating potential relationships between the clonal dynamics of CHIP and hematopoietic development across the lifespan. Next, we will examine the roles of systemic factors that can influence hematopoietic stem cell (HSC) fitness, including inflammation, and exposures to cytotoxic agents in driving selection for CHIP clones. Furthermore, we will consider how — through their impact on the BM niche — lifestyle factors, including diet, exercise, and psychosocial stressors, might contribute to the process of somatic evolution in the BM that culminates in CHIP. Finally, we will review the role of old age as a major driver of selection in CHIP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4403bef9498a4a5c2e26a32490bcbf4e8de6435a" target='_blank'>
              CHIP: a clonal odyssey of the bone marrow niche
              </a>
            </td>
          <td>
            Wolfgang E Schleicher, Bridget Hoag, Marco De Dominici, James DeGregori, Eric Pietras
          </td>
          <td>2024-08-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mixed invasive ductal and lobular carcinoma (MDLC) is a rare histologic subtype of breast cancer displaying both E-cadherin positive ductal and E-cadherin negative lobular morphologies within the same tumor, posing challenges with regard to anticipated clinical management. It remains unclear whether these distinct morphologies also have distinct biology and risk of recurrence. Our spatially resolved transcriptomic, genomic, and single-cell profiling revealed clinically significant differences between ductal and lobular tumor regions including distinct intrinsic subtype heterogeneity - e.g., MDLC with triple-negative breast cancer (TNBC) or basal ductal and estrogen receptor positive (ER+) luminal lobular regions, distinct enrichment of cell cycle arrest/senescence and oncogenic (ER and MYC) signatures, genetic and epigenetic CDH1 inactivation in lobular but not ductal regions, and single-cell ductal and lobular subpopulations with unique oncogenic signatures further highlighting intraregional heterogeneity. Altogether, we demonstrated that the intratumoral morphological/histological heterogeneity within MDLC is underpinned by intrinsic subtype and oncogenic heterogeneity which may result in prognostic uncertainty and therapeutic dilemma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b946691678dc96aa33837697b37dc03e3a68321d" target='_blank'>
              Spatial molecular profiling of mixed invasive ductal and lobular breast cancers reveals heterogeneity in intrinsic molecular subtypes, oncogenic signatures, and mutations.
              </a>
            </td>
          <td>
            O. Shah, A. Nasrazadani, J. Foldi, J. Atkinson, Celina G. Kleer, Priscilla F McAuliffe, Tyler J Johnston, Wayne Stallaert, E. D. da Silva, P. Selenica, Higinio Dopeso, F. Pareja, D. Mandelker, B. Weigelt, Jorge S Reis-Filho, Rohit Bhargava, Peter C. Lucas, Adrian V. Lee, S. Oesterreich
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7a41ef98c35752ab8b215566e394e307d31a58" target='_blank'>
              PHF12 regulates HDAC1 to promote tumorigenesis via EGFR/AKT signaling pathway in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yiru Kong, Rongrong Jiang, Hui Zhou, M. Ge, Hao Lin, Yu Wang, Rongrong Yao, Qing Wang, Xiaohua Liang, Jing Li, Xinli Zhou
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Bone metastasis (BM) is a mortality-related event of late-stage cancer, with non-small cell lung cancer (NSCLC) being a common origin for BM. However, the detailed molecular profiling of the metastatic bone ecosystem is not fully understood, hindering the development of effective therapies for advanced patients. In this study, we examined the cellular heterogeneity between primary tumours and BM from tissues and peripheral blood by single-cell transcriptomic analysis, which was verified using multiplex immunofluorescence staining and public datasets. Our results demonstrate a senescent microenvironment in BM tissues of NSCLC. BM has a significantly higher infiltration of malignant cells with senescent characteristics relative to primary tumours, accompanied by aggravated metastatic properties. The endothelial-mesenchymal transition involved with SOX18 activation is related to the cellular senescence of vascular endothelial cells from BM. CD4Tstr cells, with pronounced stress and senescence states, are preferentially infiltrated in BM, indicating stress-related dysfunction contributing to the immunocompromised environment during tumour metastasis to bone. Moreover, we identify the SPP1 pathway-induced cellular crosstalk among T cells, vascular ECs and malignant cells in BM, which activates SOX18 and deteriorates patient survival. Our findings highlight the roles of cellular senescence in modulating the microenvironment of BM and implicate anti-senescence therapy for advanced NSCLC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae5ee75b17730b9ef56c88719d46b498e6167453" target='_blank'>
              Single-cell transcriptomic analysis of the senescent microenvironment in bone metastasis.
              </a>
            </td>
          <td>
            Shenglin Wang, Lu Ao, Huangfeng Lin, Hongxiang Wei, Zhaoyang Wu, Shuting Lu, Fude Liang, R. Shen, Huarong Zhang, Tongjie Miao, Xiaopei Shen, Jianhua Lin, Guangxian Zhong
          </td>
          <td>2024-09-04</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The WDR5/MLL1-H3K4me3 epigenetic axis is often activated in both tumor cells and tumor-infiltrating immune cells to drive various cellular responses in the tumor microenvironment and has been extensively studied in hematopoietic cancer, but its respective functions in tumor cells and immune cells in the context of tumor growth regulation of solid tumor is still incompletely understood. We report here that WDR5 exhibits a higher expression level in human pancreatic tumor tissues compared with adjacent normal pancreas. Moreover, WDR5 expression is negatively correlated with patients’ response to chemotherapy or immunotherapy in human colon cancer and melanoma. However, WDR5 expression is positively correlated with the HLA level in human cancer cells, and H3K4me3 enrichment is observed at the promoter region of the HLA-A, HLA-B, and HLA-C genes in pancreatic cancer cells. Using mouse tumor cell lines and in vivo tumor models, we determined that WDR5 deficiency or inhibition significantly represses MHC I expression in vitro and in vivo in pancreatic tumor cells. Mechanistically, we determine that WDR5 deficiency inhibits H3K4me3 deposition at the MHC I (H2K) promoter region to repress MHC I (H2K) transcription. On the other hand, WDR5 depletion leads to the effective downregulation of immune checkpoints and immunosuppressive cytokines, including TGFβ and IL6, in the pancreatic tumor microenvironments. Our data determine that WDR5 not only regulates tumor cell immunogenicity to suppress tumor growth but also activates immune suppressive pathways to promote tumor immune evasion. Selective activation of the WDR5-MHC I pathway and/or selective inhibition of the WDR5–immune checkpoint and WDR5–cytokine pathways should be considered in WDR5-based epigenetic cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855836379cadca724defacc71dcd89c3fbbdd74b" target='_blank'>
              The Wdr5-H3K4me3 Epigenetic Axis Regulates Pancreatic Tumor Immunogenicity and Immune Suppression
              </a>
            </td>
          <td>
            Kaidi Deng, Liyan Liang, Yingcui Yang, Yanmin Wu, Yan Li, Rongrong Zhang, Yulin Tian, Chunwan Lu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) is the most common primary malignant brain tumor. The aim of this study was to elucidate the role of microenvironment and intrinsic T-type calcium channels (Cav3) in regulating tumor growth and progression. Methods We grafted syngeneic GBM cells into Cav3.2 knockout mice to assess the role of microenvironment T-Type calcium channels on GBM tumor growth. We performed single-cell RNA-seq (scRNA-seq) of tumors from WT and Cav3.2 KO mice to elucidate the regulation of tumors by the microenvironment. We used neurons from WT and Cav3.2 KO mice in co-culture with GBM stem cells (GSC) to assess the effects of Cav3.2 on neuron/GSC synaptic connections and tumor cell growth. Results Cav3.2 KO in the microenvironment led to significant reduction of GBM growth and prolongation of animal survival. scRNA-seq showed that microenvironment Cav3.2 regulates neuronal and glial biological processes. Microenvironment Cav3.2 downregulated numerous genes associated with regulating the OPC cell state in GBM tumors such as SOX10 and Olig2. Neuronal Cav3.2 promoted neuron/GSC synaptic connections and GSC growth. Treatment of GSCs with the Cav3 blocker mibefradil downregulated genes associated with neuronal processes. The Cav3 blocker drug mibefradil synergized with temozolomide (TMZ) and radiation to reduce in vivo tumor growth and prolong animal survival. Conclusions Together these data reveal a role for microenvironment Cav3 in promoting GBM tumor progression through regulating neuronal and glial processes particularly associated with the OPC-cell state. Targeting both intrinsic and microenvironment Cav3 with the inhibitor mibefradil significantly enhanced the anti-GBM effects of TMZ and radiation. Key Points Microenvironment Cav3.2 promotes GBM progression Microenvironment Cav3.2 promotes neuronal and glial processes Pharmacological targeting of intrinsic and microenvironment Cav3 synergizes with TMZ/radiation Importance of the Study In this study, we demonstrate for the first time that microenvironment Cav3.2 contributes to GBM progression and growth by regulating neuronal and glial processes. Our findings highlight the importance of T-type calcium channels in the microenvironment as well as the tumor and provides preclinically relevant data for the use of mibefradil to inhibit GBM growth in combination with standard of care therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303b17198abd5c192529012a365daa1747ef06e0" target='_blank'>
              Microenvironment T-Type calcium channels regulate neuronal and glial processes to promote glioblastoma growth
              </a>
            </td>
          <td>
            Collin J. Dube, Ying Zhang, Shekhar Saha, Michelle Lai, Myron K. Gibert, Miguel Escalante, Kadie Hudson, Doris Wong, Pawel Marcinkiewicz, Ulas Yener, Yunan Sun, Esther Xu, Aditya Sorot, Elizabeth Mulcahy, B. Kefas, Farina Hanif, Fadilla Guessous, Ashley Vernon, Manoj K. Patel, David Schiff, Hui Zong, Benjamin Purow, Eric Holland, Swapnil Sonkusare, Harald Sontheimer, R. Abounader
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common malignant primary brain tumor. GBM has an extremely poor prognosis and new treatments are badly needed. Efficient drug delivery to GBM is a major obstacle as the blood-brain barrier (BBB) prevents passage of the majority of cancer drugs into the brain. It is also recognized that the blood-brain tumor barrier (BTB) in the growing tumor represents a challenge. The BTB is heterogeneous and poorly characterized, but similar to the BBB it can prevent therapeutics from reaching effective intra-tumoral doses, dramatically hindering their potential. Here, we identified a 12-gene signature associated with the BTB, with functions related to vasculature development, morphogenesis and cell migration. We identified CDH5 as a core molecule in this set and confirmed its over-expression in GBM vasculature using spatial transcriptomics of GBM patient specimens. We found that the indirubin-derivative, 6-bromoindirubin acetoxime (BIA), could downregulate CDH5 and other BTB signature genes, causing endothelial barrier disruption in endothelial monolayers and BBB 3D spheroids in vitro. Treatment of tumor-bearing mice with BIA enabled increased intra-tumoral accumulation of the BBB non-penetrant chemotherapeutic drug cisplatin and potentiated cisplatin-mediated DNA damage by targeting DNA repair pathways. Finally, using an injectable BIA nanoparticle formulation, PPRX-1701, we significantly improved the efficacy of cisplatin in patient-derived GBM xenografts and prolonged their survival. Overall, our work reveals potential targets at the BTB for improved chemotherapy delivery and the bifunctional properties of BIA as a BTB modulator and potentiator of chemotherapy, supporting its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca2d6e3b8b9f2b3209e769ca054b665691b61645" target='_blank'>
              Modulation of blood-tumor barrier transcriptional programs improves intra-tumoral drug delivery and potentiates chemotherapy in GBM
              </a>
            </td>
          <td>
            J. Jimenez-Macias, Philippa Vaughn-Beaucaire, Ayush Bharati, Zheyun Xu, Megan Forrest, Jason Hong, Michael Sun, Andrea Schmidt, Jasmine S. Clark, William Hawkins, Noe Mercado, Jacqueline Real, Kelsey Huntington, M. Zdioruk, Michal O. Nowicki, C. Cho, Bin Wu, Weiyi Li, Theresa Logan, K. Manz, K. Pennell, B. Fedeles, Alexander S. Brodsky, Sean Lawler
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf3da76b81a975c5e4f8d3102cc9399708ed2b0" target='_blank'>
              Mathematical modeling and quantitative analysis of phenotypic plasticity during tumor evolution based on single-cell data.
              </a>
            </td>
          <td>
            Yuyang Xiao, Xiufen Zou
          </td>
          <td>2024-08-20</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62fe0d4817b1cce8f7a8474cb3953d097a1d805f" target='_blank'>
              The spatial distribution of intermediate fibroblasts and myeloid-derived cells dictate lymph node metastasis dynamics in oral cancer
              </a>
            </td>
          <td>
            Soni Shaikh, Harsh Dhar, Manju Moorthy, Vijayalakshmi Bhat, Sangramjit Basu, Devmalya Banerjee, Deepak Kumar Mishra, Sourav Datta, Geetashree Mukherjee
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accurately simulating the tumor immune microenvironment (TIME), which consists of a tumor, extracellular matrix (ECM), vascular network, and a variety of stromal and immune cells, is crucial for advancing and testing immunotherapies such as checkpoint inhibitors, chimeric antigen receptor-T (CAR-T) cells, and cancer vaccines. Traditional models, such as animal models, are limited by their differences from human immune environments. Bioprinting addresses these limitations by incorporating tumors, immune cells, and vascular cells within an extracellular matrix, thereby reflecting the complex interactions, including trafficking, between cancer and immune cells. These models provide better predictive accuracy for human immune responses, reducing translational failures and improving preclinical testing. While bioprinting methods for simulating the tumor microenvironment (TME), where cancer cells form spheroids surrounded by blood vessels, are well reviewed, bioprinting methods for recapitulating the TIME are not as thoroughly explored. This review aims to fill this gap by exploring the development, application, and potential of bioprinted TIME (bTIME) models in enhancing the study and efficacy of immunotherapies, ultimately offering a more realistic and personalized approach to cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9065193d8c14d70dae381de83d8789b50860d87e" target='_blank'>
              Bioprinting of tumor immune microenvironment for immunotherapy
              </a>
            </td>
          <td>
            Sein Kim, Seokgyu Han, Jaehyun Lee, Chanyang Lee, Sungsu Park
          </td>
          <td>2024-08-15</td>
          <td>International Journal of Bioprinting</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ab10bbc0562e400afe55f02a7e1bbeddfbcff21" target='_blank'>
              Reinvigoration of cytotoxic T lymphocytes in microsatellite instability-high colon adenocarcinoma through lysosomal degradation of PD-L1
              </a>
            </td>
          <td>
            Dan Liu, Jin Yan, Fang Ma, Jingmei Wang, Siqi Yan, Wangxiao He
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9965a9579beaaf870afee3260a8af2cf6b7ea8" target='_blank'>
              IL-1R signaling drives enteric glia-macrophage interactions in colorectal cancer
              </a>
            </td>
          <td>
            L. van Baarle, V. De Simone, Linda Schneider, Sneha Santhosh, S. Abdurahiman, F. Biscu, Reiner Schneider, Lisa Zanoletti, Renata Siqueira de Mello, Sara Verbandt, Zedong Hu, M. Stakenborg, B. Ke, N. Stakenborg, Raquel Salvador Laureano, Balbina García-Reyes, Jonas Henn, Marieta Toma, M. Vanmechelen, G. Boeckxstaens, F. De Smet, A. Garg, S. Ibiza, Sabine Tejpar, Sven Wehner, G. Matteoli
          </td>
          <td>2024-07-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Blood vessels are essential for maintaining tumor growth, progression, and metastasis, yet the tumor vasculature is under a constant state of remodeling. Since the tumor vasculature is an attractive therapeutic target, there is a need to predict the dynamic changes in intratumoral fluid pressure and velocity that occur across the tumor microenvironment (TME). The goal of this study was to obtain insight into perfusion anisotropy within lung tumors. To achieve this goal, we used the perfusion marker Hoechst 33342 and vascular endothelial marker CD31 to stain tumor sections from C57BL/6 mice harboring Lewis lung carcinoma tumors on their flank. Vasculature, capillary diameter, and permeability distribution were extracted at different time points along the tumor growth curve. A computational model was generated by applying a unique modeling approach based on the smeared physical fields (Kojic Transport Model, KTM). KTM predicts spatial and temporal changes in intratumoral pressure and fluid velocity within the growing tumor. Anisotropic perfusion occurs within two domains: capillary and extracellular space. Anisotropy in tumor structure causes the nonuniform distribution of pressure and fluid velocity. These results provide insights regarding local vascular distribution for optimal drug dosing and delivery to better predict distribution and duration of retention within the TME.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3ac7b07a7229391ca9b44facf997075832d488d" target='_blank'>
              An Insight into Perfusion Anisotropy within Solid Murine Lung Cancer Tumors
              </a>
            </td>
          <td>
            Antonio Martino, Rossana Terracciano, B. Milićević, M. Milošević, V. Simić, Blake C. Fallon, Yareli Carcamo-Bahena, A. L. R. Royal, Aileen A. Carcamo-Bahena, Edward Brian Butler, Richard C. Willson, M. Kojic, Carly S. Filgueira
          </td>
          <td>2024-07-30</td>
          <td>Pharmaceutics</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Langerhans cell Histiocytosis (LCH) and Erdheim-Chester disease (ECD) are clonal myeloid disorders, associated with MAP-Kinase activating mutations and an increased risk of neurodegeneration. Surprisingly, we found pervasive PU.1+ microglia mutant clones across the brain of LCH and ECD patients with and without neurological symptoms, associated with microgliosis, reactive astrocytosis, and neuronal loss. The disease predominated in the grey nuclei of the rhombencephalon, a topography attributable to a local proliferative advantage of mutant microglia. Presence of clinical symptoms was associated with a longer evolution of the disease and a larger size of PU.1+ clones (p= 0.0003). Genetic lineage tracing of PU.1+ clones suggest a resident macrophage lineage or a bone marrow precursor origin depending on patients. Finally, a CSF1R-inhibitor depleted mutant microglia and limited neuronal loss in mice suggesting an alternative to MAPK inhibitors. These studies characterize a progressive neurodegenerative disease, caused by clonal proliferation of inflammatory microglia (CPIM), with a decade(s)-long preclinical stage of incipient disease that represent a therapeutic window for prevention of neuronal death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2ee538175a393697537cbcb21555fea3da58384" target='_blank'>
              Mechanism of neurodegeneration mediated by clonal inflammatory microglia
              </a>
            </td>
          <td>
            Rocio Vicario, Stamatina Fragkogianni, Maria Pokrovskii, Carina Mayer, Estibaliz Lopez-Rodrigo, Yang Hu, Masato Ogishi, Araitz Alberdi, Ann Baako, Oyku Ay, I. Plu, Véronique Sazdovitch, S. Héritier, F. Cohen-Aubart, Natalia Shor, M. Miyara, Florence Nguyen-Khac, Agnes Viale, A. Idbaih, Zahir Amoura, Marc K. Rosenblum, Haochen Zhang, Elias-Ramzey Karnoub, P. Sashittal, Akhil Jakatdar, Christine A. Iacobuzio-Donahue, O. Abdel-Wahab, Viviane Tabar, N. Socci, O. Elemento, Eli L Diamond, B. Boisson, J. Casanova, Danielle Seilhean, J. Haroche, Jean Donadieu, F. Geissmann
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Despite advancements in antifibrotic therapy, idiopathic pulmonary fibrosis (IPF) remains a medical condition with unmet needs. Single-cell RNA sequencing (scRNA-seq) has enhanced our understanding of IPF but lacks the cellular tissue context and gene expression localization that spatial transcriptomics provides. To bridge this gap, we profiled IPF and control patient lung tissue using spatial transcriptomics, integrating the data with an IPF scRNA-seq atlas. We identified three disease-associated niches with unique cellular compositions and localizations. These include a fibrotic niche, consisting of myofibroblasts and aberrant basaloid cells, located around airways and adjacent to an airway macrophage niche in the lumen, containing SPP1+ macrophages. In addition, we identified an immune niche, characterized by distinct lymphoid cell foci in fibrotic tissue, surrounded by remodeled endothelial vessels. This spatial characterization of IPF niches will facilitate the identification of drug targets that disrupt disease-driving niches and aid in the development of disease relevant in vitro models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5351f23c92a86d2b314cc84f770e78f3ef9e013f" target='_blank'>
              Spatial transcriptomic characterization of pathologic niches in IPF
              </a>
            </td>
          <td>
            Christoph H. Mayr, Diana Santacruz, Sebastian Jarosch, Marina Bleck, John Dalton, Angela McNabola, Charlotte Lempp, L. Neubert, Berenice Rath, Jan C. Kamp, D. Jonigk, Mark Kühnel, Holger Schlüter, Alexander Klimowicz, Jonas Doerr, Alec Dick, F. Ramírez, Matthew J. Thomas
          </td>
          <td>2024-08-09</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="In the first pan-cancer analysis of the tumor vasculature, Pan and colleagues profile nearly 200,000 endothelial and mural cells (ECs and MCs), identifying novel subclusters and cell states using consensus trajectory inference. They identify differentiation trajectories in vascular and lymphatic endothelial cells, and subtype the pericyte population. During sprouting angiogenesis, venous cells dedifferentiate and transition to capillary and, finally, arterial ECs. Capillary ECs transition via "three angiogenic stages" (SI-SIII) during which APLN+ TipS1 cells were identified as potential modulators of tumor-induced neovascularization and anti-angiogenic therapy response. In lymphatic ECs, differentiation was inversely correlated between the lymphangiogenic (T1) and antigen-presenting (T2) trajectories, with T2 associated with a better prognosis. While several pericyte clusters were identified, BASP1+ matrix-associated pericytes were associated with APLN+ TipS1 cells and had a worse prognosis. These findings present transcriptional validation of previous experimental findings and serve as a resource to examine the tumor vascular microenvironment in detail.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5f776484aa47a465eea06a0fd10d87081b85a4" target='_blank'>
              The Pan-tumor Vasculature Under the Transcriptomic Magnifying Glass.
              </a>
            </td>
          <td>
            Krish Skandha Gopalan, G. Bergers
          </td>
          <td>2024-09-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd631ed2dced3a5624f3bbb39c2cea6f5bacc47" target='_blank'>
              Tumor microenvironment dynamics in oral cancer: unveiling the role of inflammatory cytokines in a syngeneic mouse model.
              </a>
            </td>
          <td>
            Ayano Tobe-Nishimoto, Yoshihiro Morita, Junya Nishimura, Yukiko Kitahira, Shun Takayama, Satoko Kishimoto, Yuka Matsumiya-Matsumoto, Kazuhide Matsunaga, Tomoaki Imai, N. Uzawa
          </td>
          <td>2024-08-10</td>
          <td>Clinical & experimental metastasis</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Renal cell carcinoma (RCC) is the most prevalent type of malignant kidney tumor in adults, with clear cell renal cell carcinoma (ccRCC) comprising about 75% of all cases. The SETD2 gene, which is involved in the modification of histone proteins, is often found to have alterations in ccRCC. Yet, our understanding of how these SETD2 mutations affect ccRCC characteristics and behavior within the tumor microenvironment is still not fully understood. Methods We conducted a detailed analysis of single-cell RNA sequencing (scRNA-seq) data from ccRCC. First, the data was preprocessed using the Python package, “scanpy.” High variability genes were pinpointed through Pearson’s correlation coefficient. Dimensionality reduction and clustering identification were performed using Principal Component Analysis (PCA) and the Leiden algorithm. Malignant cell identification was conducted with the “InferCNV” R package, while cell trajectories and intercellular communication were depicted using the Python packages “VIA” and “cellphoneDB.” We then employed the R package “Deseq2” to determine differentially expressed genes (DEGs) between groups. Using high-dimensional weighted gene correlation network analysis (hdWGCNA), co-expression modules were identified. We intersected these modules with DEGs to establish prognostic models through univariate Cox and the least absolute shrinkage and selection operator (LASSO) method. Results We identified 69 and 53 distinctive cell clusters, respectively. These were classified further into 12 unique cell types. This analysis highlighted the presence of an abnormal tumor sub-cluster (MT + group), identified by high mitochondrial-encoded protein gene expression and an indication of unfavorable prognosis. Investigation of cellular interactions spotlighted significant interactions between the MT + group and endothelial cells, macrophaes. In addition, we developed a prognostic model based on six characteristic genes. Notably, risk scores derived from these genes correlated significantly with various clinical features. Finally, a nomogram model was established to facilitate more accurate outcome prediction, incorporating four independent risk factors. Conclusion Our findings provide insight into the crucial transcriptomic characteristics of ccRCC associated with SETD2 mutation. We discovered that this mutation-induced subcluster could stimulate M2 polarization in macrophages, suggesting a heightened propensity for metastasis. Moreover, our prognostic model demonstrated effectiveness in forecasting overall survival for ccRCC patients, thus presenting a valuable clinical tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838967df027ef703d03c6fe47cf35883f45c841" target='_blank'>
              Revealing the characteristics of SETD2-mutated clear cell renal cell carcinoma through tumor heterogeneity analysis
              </a>
            </td>
          <td>
            Shansen Peng, Zhouzhou Xie, Huiming Jiang, Guihao Zhang, Nanhui Chen
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ef194ae944a9d14469f29854aa5464aa9dae02" target='_blank'>
              Immune cell landscapes are associated with high-grade serous ovarian cancer survival
              </a>
            </td>
          <td>
            Guoan Zhang, Yan Zhang, Jingjing Zhang, Xiaohui Yang, Wenjie Sun, Ying Liu, Yingfu Liu
          </td>
          <td>2024-07-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Glioblastoma is a highly aggressive brain cancer and, unlike many other cancers types, the median survival for patients after treatment (14.6 months) has barely improved in the last 20 years. Infiltrative growth into the surrounding brain parenchyma facilitates tumor recurrence and ultimately the death of the patient - novel therapies targeting this process are desperately needed. Lysyl oxidase inhibition has been shown to decrease invasive growth in a variety of solid tumours and is a potential therapy for glioblastoma patients. Methods Genes highly expressed in the mesenchymal subtype of glioblastoma were analyzed in a data set from the Cancer Genome Atlas and tissue microarrays. Two patient-derived human glioblastoma stem cell lines were used to assess the involvement of lysyl oxidase (LOX). The effect of LOX on infiltration was examined in an organotypic brain slice assay and in an orthotopic mouse model. Chemotactic assays, protease and cleavage arrays were used to assess the underlying mechanism behind LOX-mediated infiltration. The orthotopic model was used to evaluate potential clinical utility of targeting LOX in glioblastoma. Results LOX is overexpressed in the mesenchymal glioblastoma subtype and strongly associated with poor patient survival. LOX expression upregulates MMP7 expression, which subsequently cleaves the vascular matrix resulting in increased chemotaxis of glioblastoma cells. Conclusions We have uncovered a novel mechanism of glioblastoma infiltration and suggest that targeting LOX represent an effective therapeutic approach blocking glioblastoma infiltration. Importance of the study The ability of glioblastoma cells to infiltrate the surrounding normal brain tissue facilitates their evasion of current therapies, leading to tumor recurrence and ultimately the death of the patient. To improve targeted therapies for glioblastoma patients we need to understand the molecular mechanisms of glioblastoma cell infiltration and how cells interact with the unique microenvironment of the brain. We have identified a novel mechanism whereby tumor-derived LOX mediates chemotaxis of glioblastoma cells to the laminin rich perivascular niche, enabling infiltrative growth. Inhibiting this infiltrative pathway is a potential anti-invasive therapy that is desperately needed for glioblastoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb3e6c468837c0b0bfb72d57ecf46fe10ba327b" target='_blank'>
              Cleavage of the vascular matrix attracts glioblastoma cells to infiltrate the brain parenchyma
              </a>
            </td>
          <td>
            Lara Perryman, Anette M. Høye, T. Cox, Ann-Marie Baker, Jan E. Strøbech, Lidia Leonte, Lukram B Singh, Sergey Popov, Lasse G Lorentzen, Raphael Reuten, Hans Skovgaard Poulsen, Michael J Davies, Chris Jones, J. Erler
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348b736e6277f146d74b3ce8aea790786645eb9" target='_blank'>
              The role of progranulin in macrophages of a glioblastoma model.
              </a>
            </td>
          <td>
            Shohei Tsuji, Urara Kudo, Kei Takahashi, Shinsuke Nakamura, M. Shimazawa
          </td>
          <td>2024-08-14</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="High-dimensional single-cell data poses significant challenges in identifying underlying biological patterns due to the complexity and heterogeneity of cellular states. We propose a comprehensive gene-cell dependency visualization via unsupervised clustering, Growing Hierarchical Self-Organizing Map (GHSOM), specifically designed for analyzing high-dimensional single-cell data like single-cell sequencing and CRISPR screens. GHSOM is applied to cluster samples in a hierarchical structure such that the self-growth structure of clusters satisfies the required variations between and within. We propose a novel Significant Attributes Identification Algorithm to identify features that distinguish clusters. This algorithm pinpoints attributes with minimal variation within a cluster but substantial variation between clusters. These key attributes can then be used for targeted data retrieval and downstream analysis. Furthermore, we present two innovative visualization tools: Cluster Feature Map and Cluster Distribution Map. The Cluster Feature Map highlights the distribution of specific features across the hierarchical structure of GHSOM clusters. This allows for rapid visual assessment of cluster uniqueness based on chosen features. The Cluster Distribution Map depicts leaf clusters as circles on the GHSOM grid, with circle size reflecting cluster data size and color customizable to visualize features like cell type or other attributes. We apply our analysis to three single-cell datasets and one CRISPR dataset (cell-gene database) and evaluate clustering methods with internal and external CH and ARI scores. GHSOM performs well, being the best performer in internal evaluation (CH=4.2). In external evaluation, GHSOM has the third-best performance of all methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/140c0a81733fa80520acd13c32e97603d9f10e5c" target='_blank'>
              scGHSOM: Hierarchical clustering and visualization of single-cell and CRISPR data using growing hierarchical SOM
              </a>
            </td>
          <td>
            Shang-Jung Wen, Jia-Ming Chang, Fang Yu
          </td>
          <td>2024-07-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytotoxic T lymphocytes effectively eliminate cancer cells. Their abundance in the tumour microenvironment is one of the strongest pan-cancer predictors of clinical response. Here, we show that Hedgehog (Hh) signaling regulates T cell migration into tumours. Using conditional knockout mouse models of central Hh signaling components Ihh, Smo and Gli1 in CD8 T cells, we show that Smo deletion greatly impairs the anti-tumour response in vivo due to diminished CD8 T cell migration into the tumour microenvironment. The migration defect is mediated exclusively by Smo, both in in vivo cancer models and in vitro migration assays. This effect is independent of the canonical Hh pathway and relies on the GPCR function of Smo to regulate the migration of murine and human CD8 T cells via RhoA. Hh signaling is critical during embryonic development and adult stem cell homeostasis, but is also amplified in multiple cancer types. Hh inhibitors targeting SMO have been clinically-approved and shown efficacy in the treatment of Hh-driven basal cell carcinoma and medulloblastoma but have failed in clinical trials in other solid cancers with upregulated Hh signaling. We demonstrate that SMO inhibitors specifically decrease CD8 T cell migration into the tumour microenvironment, both in murine cancer models and resected BCCs from patients treated with the SMO inhibitor vismodegib, providing the first mechanistic explanation as to why Hh inhibitors have failed in solid cancers. Our data establishes a novel link between Hh inhibition in vivo and the anti-tumour immune response and reveals a fundamental mechanism controlling T cell migration. The work provides the basis for improved Hh targeting approaches in the clinic and new entry points into enhancing migration in T cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ea8b12b2d000a1cd2622a843a03890464fd3c2" target='_blank'>
              Hedgehog signaling controls cytotoxic T cell migration in the tumour microenvironment
              </a>
            </td>
          <td>
            C. Kapeni, L. O’Brien, Dilyara Sabirova, Oliver Cast, V. Carbonaro, Stephen Clark-Leonard, F. Beke, Sarah McDonald, Kate Fife, M. de la Roche
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Coronary atherosclerotic plaques susceptible to acute coronary syndrome have traditionally been characterized by their surrounding cellular architecture. However, with the advent of intravascular imaging, novel mechanisms of coronary thrombosis have emerged, challenging our contemporary understanding of acute coronary syndrome. These intriguing findings underscore the necessity for a precise molecular definition of plaque stability. Considering this, our study aimed to investigate the vascular microenvironment in patients with stable and unstable plaques using spatial transcriptomics.


METHODS
Autopsy-derived coronary arteries were preserved and categorized by plaque stability (n=5 patients per group). We utilized the GeoMx spatial profiling platform and Whole Transcriptome Atlas to link crucial histological morphology markers in coronary lesions with differential gene expression in specific regions of interest, thereby mapping the vascular transcriptome. This innovative approach allowed us to conduct cell morphological and spatially resolved transcriptional profiling of atherosclerotic plaques while preserving crucial intercellular signaling.


RESULTS
We observed intriguing spatial and cell-specific transcriptional patterns in stable and unstable atherosclerotic plaques, showcasing regional variations within the intima and media. These regions exhibited differential expression of proinflammatory molecules (eg, IFN-γ [interferon-γ], MHC class II, proinflammatory cytokines) and prothrombotic signaling pathways. By using lineage tracing through spatial deconvolution of intimal CD68+ (cluster of differentiation 68) cells, we characterized unique, intraplaque subpopulations originating from endothelial, smooth muscle, and myeloid lineages with distinct regional locations associated with plaque instability. In addition, unique transcriptional signatures were observed in vascular smooth muscle and CD68+ cells among plaques exhibiting coronary calcification.


CONCLUSIONS
Our study illuminates distinct cell-specific and regional transcriptional alterations present in unstable plaques. Furthermore, we characterize the first spatially resolved, in situ evidence supporting cellular transdifferentiation and intraplaque plasticity as significant contributors to plaque instability in human coronary atherosclerosis. Our results provide a powerful resource for the identification of novel mediators of acute coronary syndrome, opening new avenues for preventative and therapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a73fe256c8ad5204b69d364f2110a060834134" target='_blank'>
              Spatial Transcriptomic Approach to Understanding Coronary Atherosclerotic Plaque Stability.
              </a>
            </td>
          <td>
            M. Gastanadui, Camilla Margaroli, Silvio Litovsky, Robert P Richter, Dezhi Wang, D.D. Xing, J. Wells, Amit Gaggar, V. Nanda, Rakesh P Patel, Gregory A Payne
          </td>
          <td>2024-09-05</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patient-derived tumor organoids (PDTOs) are novel cellular models that maintain the genetic, phenotypic and structural features of patient tumor tissue and are useful for studying tumorigenesis and drug response. When integrated with advanced 3D imaging and analysis techniques, PDTOs can be used to establish physiologically relevant high-throughput and high-content drug screening platforms that support the development of patient-specific treatment strategies. However, in order to effectively leverage high-throughput PDTO observations for clinical predictions, it is critical to establish a quantitative understanding of the basic properties and variability of organoid growth dynamics. In this work, we introduced an innovative workflow for analyzing and understanding PDTO growth dynamics, by integrating a high-throughput imaging deep learning platform with mathematical modeling, incorporating flexible growth laws and variable dormancy times. We applied the workflow to colon cancer organoids and demonstrated that organoid growth is well-described by the Gompertz model of growth. Our analysis showed significant intrapatient heterogeneity in PDTO growth dynamics, with the initial exponential growth rate of an organoid following a lognormal distribution within each dataset. The level of intrapatient heterogeneity varied between patients, as did organoid growth rates and dormancy times of single seeded cells. Our work contributes to an emerging understanding of the basic growth characteristics of PDTOs, and it highlights the heterogeneity in organoid growth both within and between patients. These results pave the way for further modeling efforts aimed at predicting treatment response dynamics and drug resistance timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d868d7a68c9f23a73010e77ac074ffad6193b786" target='_blank'>
              Understanding patient-derived tumor organoid growth through an integrated imaging and mathematical modeling framework
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Seung-Jin Kim, Brandon Choi, J. K. Schmid, Karn Kaura, H. Lenz, S. Mumenthaler, Jasmine Foo
          </td>
          <td>2024-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Assays that measure morphology, proliferation, motility, deformability, and migration are used to study the invasiveness of cancer cells. However, native invasive potential of cells may be hidden from these contextual metrics because they depend on culture conditions. We created a micropatterned chip that mimics the native environmental conditions, quantifies the invasive potential of tumor cells, and improves our understanding of the malignancy signatures. Unlike conventional assays, which rely on indirect measurements of metastatic potential, our method uses three-dimensional microchannels to measure the basal native invasiveness without chemoattractants or microfluidics. No change in cell death or proliferation is observed on our chips. Using six cancer cell lines, we show that our system is more sensitive than other motility-based assays, measures of nuclear deformability, or cell morphometrics. In addition to quantifying metastatic potential, our platform can distinguish between motility and invasiveness, help study molecular mechanisms of invasion, and screen for targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/402f836172ab24cb948457a85d0d2a480f5d424a" target='_blank'>
              A high-throughput microfabricated platform for rapid quantification of metastatic potential
              </a>
            </td>
          <td>
            Smiti Bhattacharya, Abora Ettela, Jonathan C. Haydak, Chad M. Hobson, Alan Stern, Miran Yoo, T. Chew, G. L. Gusella, Emily J Gallagher, James C Hone, E. Azeloglu
          </td>
          <td>2024-08-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9e350582b4bfc2de6392f5ccba4d7d4587f93b" target='_blank'>
              T cell receptor clonotype in tumor microenvironment contributes to intratumoral signaling network in patients with colorectal cancer.
              </a>
            </td>
          <td>
            In Hye Song, Seung-Been Lee, Byung-Kwan Jeong, Jungwook Park, Honggeun Kim, GunHee Lee, Su Min Cha, H. Lee, Gyungyub Gong, Nak-Jung Kwon, H. Lee
          </td>
          <td>2024-08-08</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748399fff88dfb0406bee15abcee4ba08d334ec0" target='_blank'>
              An 18-gene signature of recurrence-associated endothelial cells predicts tumor progression and castration resistance in prostate cancer
              </a>
            </td>
          <td>
            Bing-Biao Lin, Qingqing Huang, Binyuan Yan, Mingcheng Liu, Zhiqian Zhang, Hanqi Lei, Ronghua Huang, Jintang Dong, Jun Pang
          </td>
          <td>2024-07-12</td>
          <td>British Journal of Cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Ability to genetically edit primary B cells via CRISPR/Cas9 technology represents a powerful tool to study molecular mechanisms of B ‐ cell pathogenesis. In this context, employing ribonucleoprotein complexes (RNPs), formed by recombinant Cas9 and genome ‐ targetting single guide RNA molecules, brings in advantage of accelerated set ‐ up and protocol robustness. Gene editing via RNP electroporation has been recently applied to primary tumor cells isolated from patients chronic lymphocytic leukemia (CLL), suggesting an efficient and valuable tool for studying leukemic cell biology and biomarker validation. 1,2 The work by Nardi et al. on this topic proposed to electroporate unmanipulated primary CLL cells that are subsequently put in culture with human CD40L ‐ expressing fibroblasts and soluble stimuli to promote CLL cell proliferation. In this context, cellular proliferation is required to achieve homozygous gene editing, whereas in unstimulated CLL cells it is possible to achieve only the heterozygous editing. 1 The method published by Mateos ‐ Jaimez et al. relies on the preactivation of CLL cells with CD40L/BAFF/IL ‐ 21 ‐ expressing stromal cells, followed by RNP electroporation and continuation of the stimulatory coculture. 2 Both methods approach 80% – 90% of editing efficiency and allow to perform downstream in vitro experiments on edited leukemic cells. Application of a similar RNP ‐ based editing approach to the widely used murine">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce4b4f9dae608c0b0ba780ba2b6869894794e80" target='_blank'>
              Robust and cost‐effective CRISPR/Cas9 gene editing of primary tumor B cells in Eµ‐TCL1 model of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Rosita Del Prete, Roberta Drago, Federica Nardi, Gaia Bartolini, Erika Bellini, Antonella De Rosa, Silvia Valensin, Anna Kabanova
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomics analyses in clinical specimens, with extensive in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment using in vitro and in vivo models. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9549906de5a8fb5e451fc40dd7e32f3d0a7b1f0" target='_blank'>
              BSLM-02 SPATIAL TRANSCRIPTOMICS ANALYSIS IDENTIFIES A UNIQUE TUMOR-PROMOTING FUNCTION OF THE MENINGEAL STROMA IN MELANOMA LEPTOMENINGEAL DISEASE
              </a>
            </td>
          <td>
            Hasan Alhaddad, Oscar E. Ospina, M. Khaled, Yuan Ren, Ethan Vallebuona, Mohammad Baraa Boozo, Peter Forsyth, Yolanda Pina, Robert Macaulay, V. Law, Kenneth Y. Tsai, Douglas Cress, Brooke L. Fridley, Inna Smalley
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3471e25fd3d93db955069ff8713da95ffc83f905" target='_blank'>
              Single-cell multi-omic and spatial profiling of human kidneys implicates the fibrotic microenvironment in kidney disease progression.
              </a>
            </td>
          <td>
            Amin Abedini, Jonathan Levinsohn, Konstantin A. Klötzer, Bernhard Dumoulin, Ziyuan Ma, Julia Frederick, Poonam Dhillon, Michael S. Balzer, Rojesh Shrestha, Hongbo Liu, Steven Vitale, Andi M. Bergeson, Kishor B. Devalaraja-Narashimha, Paola Grandi, Tanmoy Bhattacharyya, Erding Hu, Steven S. Pullen, Carine M. Boustany-Kari, Paolo Guarnieri, Anil Karihaloo, D. Traum, Hanying Yan, Kyle Coleman, Matthew Palmer, Lea Sarov-Blat, L. Morton, Christopher A. Hunter, Klaus H. Kaestner, Mingyao Li, K. Suszták
          </td>
          <td>2024-07-24</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Systematic investigation of tumor-infiltrating immune (TII) cells is important to the development of immunotherapies, and the clinical response prediction in cancers. There exists complex transcriptional regulation within TII cells, and different immune cell types display specific regulation patterns. To dissect transcriptional regulation in TII cells, we first integrated the gene expression profiles from single-cell datasets, and proposed a computational pipeline to identify TII cell type-specific transcription factor (TF) mediated activity immune modules (TF-AIMs). Our analysis revealed key TFs, such as BACH2 and NFKB1 play important roles in B and NK cells, respectively. We also found some of these TF-AIMs may contribute to tumor pathogenesis. Based on TII cell type-specific TF-AIMs, we identified eight CD8+ T cell subtypes. In particular, we found the PD1 + CD8+ T cell subset and its specific TF-AIMs associated with immunotherapy response. Furthermore, the TII cell type-specific TF-AIMs displayed the potential to be used as predictive markers for immunotherapy response of cancer patients. At the pan-cancer level, we also identified and characterized six molecular subtypes across 9680 samples based on the activation status of TII cell type-specific TF-AIMs. Finally, we constructed a user-friendly web interface CellTF-AIMs (http://bio-bigdata.hrbmu.edu.cn/CellTF-AIMs/) for exploring transcriptional regulatory pattern in various TII cell types. Our study provides valuable implications and a rich resource for understanding the mechanisms involved in cancer microenvironment and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b9ff76c2393752409d847895c8cb3295294da" target='_blank'>
              Identifying cell type-specific transcription factor-mediated activity immune modules reveal implications for immunotherapy and molecular classification of pan-cancer
              </a>
            </td>
          <td>
            Feng Li, Jingwen Wang, Mengyue Li, Xiaomeng Zhang, Yongjuan Tang, Xinyu Song, Yifang Zhang, Liying Pei, Jiaqi Liu, Chunlong Zhang, Xia Li, Yanjun Xu, Yunpeng Zhang
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Breast cancer stands as the most prevalent cancer diagnosed worldwide, often leading to brain metastasis, a challenging complication characterized by high mortality rates and a grim prognosis. Understanding the intricate mechanisms governing breast cancer brain metastasis (BCBM) remains an ongoing challenge. The unique microenvironment in the brain fosters an ideal setting for the colonization of breast cancer cells. The tumor microenvironment (TME) in brain metastases plays a pivotal role in the initiation and progression of BCBM, shaping the landscape for targeted therapeutic interventions. Current research primarily concentrates on unraveling the complexities of the TME in BCBM, with a particular emphasis on neuroglia and immune cells, such as microglia, monocyte-derived macrophages (MDMs), astrocytes and T cells. This comprehensive review delves deeply into these elements within the TME of BCBM, shedding light on their interplay, mechanisms, and potential as therapeutic targets to combat BCBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89f58be8a99bf67c919a1c80a20e8d34360b0f86" target='_blank'>
              Analysis of neuroglia and immune cells in the tumor microenvironment of breast cancer brain metastasis.
              </a>
            </td>
          <td>
            Haixin Mo, Xin Zhang, Liangliang Ren
          </td>
          <td>2024-09-05</td>
          <td>Cancer biology & therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Recent advances in single cell sequencing have led to an increased focus on the role of cell-type composition in phenotypic presentation and disease progression. Cell-type composition research in the heart is challenging due to large, frequently multinucleated cardiomyocytes that preclude most single cell approaches from obtaining accurate measurements of cell composition. Our in silico studies reveal that ignoring cell type composition when calculating differentially expressed genes (DEGs) can have significant consequences. For example, a relatively small change in cell abundance of only 10% can result in over 25% of DEGs being false positives. Methods We have implemented an algorithmic approach that uses snRNAseq datasets as a reference to accurately calculate cell type compositions from bulk RNAseq datasets through robust data cleaning, gene selection, and multi-sample cross-subject and cross-cell-type deconvolution. We applied our approach to cardiomyocyte-specific α1A adrenergic receptor (CM-α1A-AR) knockout mice. 8-12 week-old mice (either WT or CM-α1A-KO) were subjected to permanent left coronary artery (LCA) ligation or sham surgery (n=4 per group). Transcriptomes from the infarct border zones were collected 3 days later and analyzed using our algorithm to determine cell-type abundances, corrected differential expression calculations using DESeq2, and validated these findings using RNAscope. Results Uncorrected DEGs for the CM-α1A-KO X LCA interaction term featured many cell-type specific genes such as Timp4 (fibroblasts) and Aplnr (cardiomyocytes) and overall GO enrichment for terms pertaining to cardiomyocyte differentiation (P=3.1E-4). Using our algorithm, we observe a striking loss of cardiomyocytes and gain in fibroblasts in the α1A-KO + LCA mice that was not recapitulated in WT + LCA animals, although we did observe a similar increase in macrophage abundance in both conditions. This recapitulates prior results that showed a much more severe heart failure phenotype in CM-α1A-KO + LCA mice. Following correction for cell-type, our DEGs now highlight a novel set of genes enriched for GO terms such as cardiac contraction (P=3.7E-5) and actin filament organization (P=6.3E-5). Conclusions Our algorithm identifies and corrects for cell-type abundance in bulk RNAseq datasets opening new avenues for research on novel genes and pathways as well as an improved understanding of the role of cardiac cell types in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217ee4d32d0eb92d12b53e00c08af7cc8fb059d2" target='_blank'>
              Novel Insights into Post-Myocardial Infarction Cardiac Remodeling through Algorithmic Detection of Cell-Type Composition Shifts
              </a>
            </td>
          <td>
            Brian Gural, Logan Kirkland, Abbey Hockett, Peyton Sandroni, Jiandong Zhang, M. Rosa-Garrido, Samantha K. Swift, Douglas J. Chapski, Michael A. Flinn, Caitlin C. O’Meara, T. Vondriska, Michaela Patterson, Brian C. Jensen, Christoph D Rau
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-associated macrophages and microglia (TAMs) are highly abundant myeloid cells in gliomas, with their phenotype and immune response determined by ontogeny and microenvironment. TAMs display distinctive transcriptional programs according to the IDH mutation status but the underlying regulatory mechanisms remain largely unknown. Herein, we uncover that CD11B+ myeloid cells in human IDHmut gliomas exhibit DNA hypermethylation predominantly at distal enhancers. This hypermethylation was linked to decreased expression of genes involved in inflammatory responses and glycolytic metabolism, and the inactivation of transcription factors that regulate microglial responses to environmental stimuli. Prolonged exposure of human primary microglia to D-2-hydroxyglutarate (D-2HG) inhibited TET-mediated 5mC oxidation, resulting in a reduced accumulation of global 5hmC levels. We confirmed high 5mC/5hmC ratios at lineage-specific enhancers, by analyzing CpGs at single-base resolution. D-2HG-treated microglia show reduced proinflammatory capacity and enhanced oxidative phosphorylation, consistent with the remodeled enhancer landscape. Conversely, depletion of D-2HG following treatment of a glioma patient with an IDH mutant inhibitor was associated with enhanced microglial responses, as assessed by snRNA-seq. Our findings provide a mechanistic rationale for the hyporesponsive state of microglia in IDHmut gliomas and support the concept that oncometabolites may disrupt the function of immune cells residing in the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc5ed6e8a9af5e3aa64ecf135e384212bae5953" target='_blank'>
              The oncometabolite D-2-hydroxyglutarate promotes DNA hypermethylation at lineage-specific enhancers controlling microglial activation in IDHmut gliomas
              </a>
            </td>
          <td>
            A. Laurenge, Pietro Pugliese, Q. Richard, B. Mathon, S. Jouannet, Yvette Hayat, Sarah Scuderi, Laurent Capelle, P. Marijon, Karim Labreche, A. Alentorn, M. Verreault, A. Idbaih, C. Birzu, Emmanuelle Huillard, Nina Pottier, Aurore Desmons, Inès Fayache, Garrett A. Kaas, Philip J. Kingsley, Lawrence J. Marnett, Eric Duplus, Elias El-Habr, Lucas Salas, K. Mokhtari, S. Tran, M. Touat, Franck Bielle, M. Suvà, A. Iavarone, Michele Ceccarelli, M. Mallat, Marc Sanson, L. Castro-Vega
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6392838cbd8fd43bfdbde70561f2b8df6cd16609" target='_blank'>
              A multidimensional analysis reveals distinct immune phenotypes and the composition of immune aggregates in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            J. Koedijk, I. van der Werf, Livius Penter, M. Vermeulen, F. Barneh, A. Perzolli, J. Meesters-Ensing, Dennis S Metselaar, T. Margaritis, M. Fiocco, H. D. de Groot-Kruseman, Rubina Moeniralam, Kristina Bang Christensen, Billie Porter, Kathleen L. Pfaff, Jacqueline S Garcia, Scott J. Rodig, Catherine J. Wu, Henrik Hasle, S. Nierkens, Mirjam E Belderbos, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-08-26</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic cancer has a poor prognosis. Targeting Kirsten Rat Sarcoma (KRAS) mutation and its related pathways may enhance immunotherapy efficacy. While in vivo monitoring of therapeutic response and immune cell migration remains challenging, Fluorine-19 MRI (19F MRI) may allow noninvasive longitudinal imaging of immune cells.


PURPOSE
Evaluating the potential of 19F MRI for monitoring changes in the tumor immune microenvironment, in response to combined SHP2/MEK inhibition.


STUDY TYPE
Pre-clinical animal study.


ANIMAL MODEL
Murine genetically engineered pancreatic cancer model (N = 20, both sexes).


FIELD STRENGTH/SEQUENCE
9.4-T, two-dimensional multi-slice Rapid Acquisition with Relaxation Enhancement sequence. Intravenous injection of 19F-perfluorocarbon (PFC) nanoparticles.


ASSESSMENT
Upon tumor detection by conventional 1H MRI screening, 19F MRI was performed in mice 24 hours after PFC nanoparticle administration. Animals were randomly assigned to four treatment groups: allosteric Src-homology-2-containing protein tyrosine phosphatase 2 (SHP2) inhibitor SHP099, the mitogen-activated extracellular signal-regulated kinase 1/2 (MEK1/2) inhibitor Trametinib, the combination of both, or a vehicle control (4 to 6 mice each group), administered every other day per oral gavage. 1H and 19F MRI was repeated 7 days and 14 days later. Pancreatic immune cell infiltrates were analyzed by flow cytometry and multiplex immunohistofluorescence (mIHF) upon sacrifice.


STATISTICAL TESTS
Independent t-tests and one-way analysis of variance.


RESULTS
19F MRI revealed continuous decrease of PFC-signals in tumors from vehicle controls (100%, 80%, and 74% on days 0, 7, and 14, respectively), contrasting with stable or increasing signals under KRAS-pathway-directed treatment. MEK inhibition showed 100%, 152%, and 84% and dual SHP2/MEK1/2 inhibition demonstrated signals of 100%, 134%, and 100% on days 0, 7, 14, respectively. mIHF analyses indicated CD11b+ macrophages/monocytes as primary contributors to the observed 19F MRI signal differences.


DATA CONCLUSION
19F MRI might provide non-invasive longitudinal estimates for abundance and spatial distribution of CD11b+ macrophages/monocytes in pancreatic cancer.


EVIDENCE LEVEL
1 TECHNICAL EFFICACY: Stage 2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ec946a8fce42040b3fa092ddfe665a193003215" target='_blank'>
              19Fluorine-MRI Based Longitudinal Immuno-Microenvironment-Monitoring for Pancreatic Cancer.
              </a>
            </td>
          <td>
            Wilfried Reichardt, Tabea Gewalt, Philipp Hafner, S. J. Keller, Xun Chen, Asma Y. Alrawashdeh, Yayu Li, Solène Besson, Stefan Fichtner-Feigl, Dominik von Elverfeldt, Huda Jumaa, Dietrich A Ruess
          </td>
          <td>2024-08-27</td>
          <td>Journal of magnetic resonance imaging : JMRI</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8701da24858299d2bd116213df22e0fe6a06dea4" target='_blank'>
              Titration of RAS alters senescent state and influences tumour initiation.
              </a>
            </td>
          <td>
            Adelyne Chan, Haoran Zhu, M. Narita, Liam D. Cassidy, Andrew R J Young, Camino Bermejo-Rodriguez, Aleksandra T Janowska, Hung-Chang Chen, Sarah Gough, Naoki Oshimori, L. Zender, Sarah J. Aitken, Matthew Hoare, Masashi Narita
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70504aeb9b8ca67b917dcc2474590fbeb0bd285d" target='_blank'>
              Single-cell transcriptomic analysis reveals a decrease in the frequency of macrophage-RGS1high subsets in patients with osteoarticular tuberculosis
              </a>
            </td>
          <td>
            Ying Jiang, Xinqiang Zhang, Bo Wang, Liping Tang, Xin Liu, Xiudong Ding, Yueming Dong, Hong Lei, Di Wang, Huicheng Feng
          </td>
          <td>2024-08-10</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Adaptive resistance contributes significantly to treatment failure in many cancers. Despite the increased prevalence of experimental studies that interrogate this phenomenon, there remains a lack of applicable quantitative tools to characterise data, and importantly to distinguish between resistance as a discrete phenotype and a (potentially heterogeneous) continuous distribution of phenotypes. To address this, we develop a stochastic individual-based model of adaptive resistance in low-cell-count proliferation assays. That our model corresponds probabilistically to common partial differential equation models of resistance allows us to formulate a likelihood that captures the intrinsic noise ubiquitous to such experiments. We apply our framework to assess the identifiability of key model parameters in several population-level data collection regimes; in particular, parameters relating to the adaptation velocity and within-population heterogeneity. Significantly, we find that heterogeneity is practically non-identifiable from both cell count and proliferation marker data, implying that population-level behaviours may be well characterised by homogeneous ordinary differential equation models. Additionally, we demonstrate that population-level data are insufficient to distinguish resistance as a discrete phenotype from a continuous distribution of phenotypes. Our results inform the design of both future experiments and future quantitative analyses that probe adaptive resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ebb08eee1f5c61607df8342bbe91a4c2f7a927" target='_blank'>
              Identifiability of heterogeneous phenotype adaptation from low-cell-count experiments and a stochastic model
              </a>
            </td>
          <td>
            A. Browning, Rebecca M. Crossley, Chiara Villa, Philip Maini, A. Jenner, Tyler Cassidy, Sara Hamis
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb0bdd74bef55463193922578fe18a07c1d496b" target='_blank'>
              Single-cell multiomics reveals ENL mutation perturbs kidney developmental trajectory by rewiring gene regulatory landscape
              </a>
            </td>
          <td>
            Lele Song, Qinglan Li, Lingbo Xia, A. Sahay, Q. Qiu, Yuanyuan Li, Haitao Li, Kotaro Sasaki, Katalin Susztak, Hao Wu, Liling Wan
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b7af48216a0c84346ea77c18dc911a1e93afbf" target='_blank'>
              Endothelial cell heterogeneity in colorectal cancer: tip cells drive angiogenesis
              </a>
            </td>
          <td>
            Zhenyu Xie, Liaoran Niu, Kunli Du, Ling Chen, Gaozan Zheng, Songchen Dai, Han-Long Dan, Lili Duan, Xinyu Dou, Fan Feng, Jian Zhang, Jianyong Zheng
          </td>
          <td>2024-08-22</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Elderly individuals have higher disease susceptibility and lower vaccine responsiveness, highlighting the need to better comprehend the aging immune system and its clinical associations. Here we conducted a deep immune profiling study of 550 elderly individuals (61–94 years) and 100 young adults (22–38 years). Utilizing high-dimensional spectral flow cytometry to identify 97 immune cell populations and 48-plex cytokine profiling, we detailed intricate age-and sex-related changes in the elderly immune system at an unprecedented depth. Synthesizing information from clinical, laboratory, and immunological data through an integrative multi-block analysis, we reveal overarching systems-level signatures of aging, such as increased concentrations of specific cytokines and frequencies of defined innate and adaptive immune cell subpopulations. Extending this approach, we identified unique immune signatures of smoking, obesity, and several diseases including osteoporosis, heart failure and gout. Our systems biology approach enables to uncover new relationships between clinical characteristics and immunological traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75977a9130047fe0c681122378f90c39329e640" target='_blank'>
              Integrative deep immune profiling of the elderly reveals systems-level signatures of aging, sex, smoking, and clinical traits
              </a>
            </td>
          <td>
            Lennart Riemann, Rodrigo Gutierrez, Iva Odak, Joana Barros-Martins, L. Roesner, Ximena Leon Lara, Christine Falk, T. F. Schulz, Gesine Hansen, T. Werfel, Reinhold Förster
          </td>
          <td>2024-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Background Renal cell carcinomas (RCC) are characterized by their largely diverse clinical outcomes. Leveraging genomic diversity among individual tumors, our research during the past decade has concentrated on developing personalized medicine approaches for the most common and the most aggressive type of RCC, clear cell renal carcinoma (ccRCC). We generated the largest dataset of ccRCC including somatic genomic and clinical annotations for over 940 ccRCC patients, and developed a genomic classifier, based on mutational status of 12 RCC-relevant genes, which is able to stratify patients according to their risk of relapse after nephrectomy and/or death due to RCC. Furthermore, we noticed the presence of mesenchymal-like cellular phenotypes in tumors of high-risk patients, representsing a de-differentiation process, known as sarcomatoic or rhabdoid (S/R) de-differentation. Methods The current knowledge about molecular mechanisms that may drive S/R de-differentiation is primarily generated from molecular profiling of bulk tumors, collecting data from a mixture of cells that co-exist in the tumor milieu. Therefore, high-resolution studies that precisely define molecular characteristics of different cellular phenotypes associated with S/R features are missing. We have used spatial transcriptomic profiling to investigate molecular mechanisms that underline S/R de-differentationin RCC. While preserving tissue context, we applied spatial whole human transcriptome profiling to areas exhibiting S/R or clear cell phenotypes within the same tumor specimens to generate phenotype-specific transcriptome profiles. In addition, we applied whole-exome sequencing (WES) to independent areas that exhibit different differentiation phenotype withing same tumors. Results Pathway and network analysis of genesets with upregulation in each area have revealed meaningful differences in cellular pathways which are active in each phenotype. Specifically, we observed that dysregulation of extracellular matrix (ECM) is a hallmark of S/R areas within RCC tumors. Furthermore, WES of phenotypically-distinct areas within each tumor has shed new light son the genome evoltuion of S/R phenotypes. Conclusions S/R dedifferentiation in RCC tumors is characterized by specific genomic evolutionary patterns and substantial dysregulation of ECM components. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad0d8a5a2c80baecbe7aa769327da96aacd5a05" target='_blank'>
              9 Delving into molecular underpinnings of sarcomatoid/rhabdoid dedifferentioation in renal cancers using spatial profiling
              </a>
            </td>
          <td>
            Y. Riazalhosseini
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d4ba3cf7cf5603a608ca85287f8b1899486efd" target='_blank'>
              Targeting CERS6-AS1/FGFR1 axis as synthetic vulnerability to constrain stromal cells supported proliferation in Mantle cell lymphoma.
              </a>
            </td>
          <td>
            Udita Jindal, Mukesh Mamgain, U. Nath, Isha Sharma, Bhaskar Pant, Ankita Sharma, Archita Gupta, Khaliqur Rahman, Sunil Yadav, Manish Pratap Singh, Shaktiprasad Mishra, Chandra Praksah Chaturvedi, José Courty, Navin Singh, Seema Gupta, Sanjeev Kumar, S. Verma, S. Mallick, A. Gogia, Sunil K Raghav, Jayanta Sarkar, Kinshuk Raj Srivastava, Dipak Datta, Neeraj Jain
          </td>
          <td>2024-07-13</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The microenvironment of a cancer stem cell (CSC) niche is often found in coexistence with cancer-associated fibroblasts (CAFs). Here, we show the first in-depth analysis of the interaction between primary triple-negative breast cancer stem cells (BCSCs) with fibroblasts. Using 2D co-culture models with specific seeding ratios, we identified stromal fibroblast aggregation at the BCSC cluster periphery, and, on closer observation, the aggregated fibroblasts was found to encircle BCSC clusters in nematic organization. In addition, collagen type I and fibronectin accumulation were also found at the BCSC–stromal periphery. MACE-Seq analysis of BCSC-encapsulating fibroblasts displayed the transformation of stromal fibroblasts to CAFs and the upregulation of fibrosis regulating genes of which the Interferon Regulatory Factor 6 (IRF6) gene was identified. Loss of function experiments with the IRF6 gene decreased fibroblast encapsulation around BCSC clusters in 2D co-cultures. In BCSC xenografts, fibroblast IRF6 expression led to an increase in the stromal area and fibroblast density in tumors, in addition to a reduction in necrotic growth. Based on our findings, we propose that fibroblast IRF6 function is an important factor in the development of the stromal microenvironment and in sustaining the BCSC tumor niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffc09f9dbaba3a0d11e41b85e9f9e54309333f44" target='_blank'>
              Breast Cancer Stem Cells Upregulate IRF6 in Stromal Fibroblasts to Induce Stromagenesis
              </a>
            </td>
          <td>
            Harshini Muralidharan, Thomas Hansen, Anja Steinle, David Schumacher, E. Stickeler, Jochen Maurer
          </td>
          <td>2024-08-31</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6640869137efbf815e49406335626c0344fac69" target='_blank'>
              Age-associated clonal B cells drive B cell lymphoma in mice.
              </a>
            </td>
          <td>
            José P Castro, A. Shindyapina, Alessandro Barbieri, Kejun Ying, Olga S. Strelkova, Joao A. Paulo, A. Tyshkovskiy, Rico Meinl, C. Kerepesi, Anna P. Petrashen, Marco Mariotti, Margarita V Meer, Yan Hu, Alexander Karamyshev, Grigoriy Losyev, Mafalda Galhardo, Elsa Logarinho, Artur A. Indzhykulian, Steven P. Gygi, J. M. Sedivy, John P Manis, V. N. Gladyshev
          </td>
          <td>2024-08-08</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Motivation The combination of single-cell sequencing with ligand-receptor analysis paves the way for the characterization of cell communication events in complex tissues. In particular, directed weighted graphs stand out as a natural representation of cell-cell communication events. However, current computational methods cannot analyze sample-specific cell-cell communication events, as measured in single-cell data produced in large patient cohorts. Cohort-based cell-cell communication analysis presents many challenges, such as the non-linear nature of cell-cell communication and the high variability presented by the patient-specific single-cell RNAseq datasets. Results Here, we present scACCorDiON (single-cell Analysis of Cell-Cell Communication in Disease clusters using Optimal transport in Directed Networks), an optimal transport algorithm exploring node distances on the Markov Chain as the ground metric between directed weighted graphs. Additionally, we derive a k-barycenter algorithm using the Wasserstein-based distance, which is able to cluster directed weighted graphs. We compare our approach with competing methods in several large cohorts of scRNA-seq data. Our results show that scACCorDiON can predict clusters better, matching the disease status of samples. Moreover, we show that barycenters provide a robust and explainable representation of cell cell communication events related to the detected clusters. We also provide a case study of pancreas adenocarcinoma, where scACCorDion detects a sub-cluster of disease samples associated with changes in the tumor microenvironment. Availability The code of scACCorDiON is available at https://scaccordion.readthedocs.io/en/latest Contact ivan.costa@rwth-aachen.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12408229b87ae8ceac3d6b2906ea4bd12adc6c6e" target='_blank'>
              scACCorDiON: A clustering approach for explainable patient level cell cell communication graph analysis
              </a>
            </td>
          <td>
            J. Nagai, Michael T. Schaub, Ivan G.Costa
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The expression of PD-L1 on tumor cells (TCs) is used as an immunotherapy biomarker in lung cancer, but heterogeneous intratumoral expression is often observed. Using a Digital Spatial Profiling, we performed proteomic and whole-transcriptomic analyses of TCs and immune cells (ICs) in spatially matched areas based on tumor PD-L1 expression and the status of the immune microenvironment. Our findings were validated using immunohistochemistry, The Cancer Genome Atlas, and immunotherapy cohorts. ICs in areas with high PD-L1 expression on TCs showed more features indicative of immunosuppression and exhaustion than ICs in areas with low PD-L1 expression on TCs. TCs highly expressing PD-L1 within immune-inflamed (IF) areas show up-regulation of pro-inflammatory processes, whereas TCs highly expressing PD-L1 within immune-deficient (ID) areas show up-regulation of various metabolic processes. Using differentially expressed genes of TCs between the IF and ID areas, we identified a novel prognostic gene signature for lung cancer. In addition, a high ratio of CD8+ cells to M2 macrophages was found to predict favorable outcomes in patients with PD-L1-expressing lung cancer after immune checkpoint inhibitor therapy. This study demonstrates that TCs and ICs have distinct spatial features within the tumor microenvironment that are related to tumor PD-L1 expression and IC infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc0a9207938d739ad9fb6635845dbbb9be7ec54" target='_blank'>
              Spatially resolved whole-transcriptomic and proteomic profiling of lung cancer and its immune-microenvironment according to PD-L1 expression.
              </a>
            </td>
          <td>
            Yoon Kyung Jeon, Jaemoon Koh, Dongjoo Lee, Sehui Kim, Seung Geun Song, Bogyeong Han, Hyein Jeong, Young A Kim, B. Keam, Se-Hoon Lee, Kwangsoo Kim, Doo Hyun Chung
          </td>
          <td>2024-09-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Prostate cancer remains the most commonly diagnosed and the second leading cause of cancer-related deaths in men in the United States. The neoplastic transformation of prostate epithelia, concomitant with modulations in the stromal compartment, known as reactive stromal response, is critical for the growth, development, and progression of prostate cancer. Reactive stroma typifies an emergent response to disrupted tissue homeostasis commonly observed in wound repair and pathological conditions such as cancer. Despite the significance of reactive stroma in prostate cancer pathobiology, our understanding of the ontogeny, phenotypic and functional heterogeneity, and reactive stromal regulation of the immune microenvironment in prostate cancer remains limited. Traditionally characterized to have an immunologically "cold" tumor microenvironment, prostate cancer presents significant challenges for advancing immunotherapy compared to other solid tumors. This review explores the detrimental role of reactive stroma in prostate cancer, particularly its immunomodulatory function. Understanding the molecular characteristics and dynamic transcriptional program of the reactive stromal populations in tandem with tumor progression could offer insights into enhancing immunotherapy efficacy against prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd12cec3359f8ec93eb34750a8c77daa465b3613" target='_blank'>
              The reactive stroma response regulates the immune landscape in prostate cancer
              </a>
            </td>
          <td>
            Rintu Thomas, John Michael Jerome, Kimiko L. Krieger, Naghmana Ashraf, David R. Rowley
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b32dd10288cda745785973a2a8b2bb050d2f62" target='_blank'>
              Cranioencephalic functional lymphoid units in glioblastoma.
              </a>
            </td>
          <td>
            Celia Dobersalske, L. Rauschenbach, Yichao Hua, Christoph Berliner, Anita Steinbach, Anika Grüneboom, Konstantinos D Kokkaliaris, D. H. Heiland, Pia Berger, Sarah Langer, C. L. Tan, Martin Stenzel, Somaya Landolsi, Flora Weber, M. Darkwah Oppong, Rudolf A. Werner, H. Gull, Thomas Schröder, T. Linsenmann, Andreas K. Buck, Matthias Gunzer, M. Stuschke, K. Keyvani, M. Forsting, Martin Glas, Jonathan Kipnis, D. A. Steindler, Hans Christian Reinhardt, E. Green, M. Platten, A. Tasdogan, Ken Herrmann, Florian Rambow, Igor Cima, Ulrich Sure, Björn Scheffler
          </td>
          <td>2024-07-31</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Spatial transcriptomics (ST) examines gene expression within its spatial context on tissue, linking morphology and function. Advances in ST resolution and throughput have led to an increase in scientific interest, notably in cancer research. This scoping study reviews the challenges and practical applications of ST, summarizing current methods, trends, and data analysis techniques for ST in neoplasm research. We analyzed 41 articles published by the end of 2023 alongside public data repositories. The findings indicate cancer biology is an important focus of ST research, with a rising number of studies each year. Visium (10x Genomics, Pleasanton, CA, USA) is the leading ST platform, and SCTransform from Seurat R library is the preferred method for data normalization and integration. Many studies incorporate additional data types like single-cell sequencing and immunohistochemistry. Common ST applications include discovering the composition and function of tumor tissues in the context of their heterogeneity, characterizing the tumor microenvironment, or identifying interactions between cells, including spatial patterns of expression and co-occurrence. However, nearly half of the studies lacked comprehensive data processing protocols, hindering their reproducibility. By recommending greater transparency in sharing analysis methods and adapting single-cell analysis techniques with caution, this review aims to improve the reproducibility and reliability of future studies in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546384150762685b69a172f984b5438d683edf6a" target='_blank'>
              Scoping Review: Methods and Applications of Spatial Transcriptomics in Tumor Research
              </a>
            </td>
          <td>
            Kacper Maciejewski, Patrycja Czerwińska
          </td>
          <td>2024-09-06</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16df671c98a37a9e428d4261d351be144c2e34b" target='_blank'>
              Monitoring imatinib decreasing pericyte coverage and HIF-1α level in a colorectal cancer model by an ultrahigh-field multiparametric MRI approach
              </a>
            </td>
          <td>
            Xinpeng Hu, Kunlin Ye, Shaowei Bo, Zeyu Xiao, Mengjie Ma, Jinghua Pan, Xing Zhong, Dong Zhang, Xukai Mo, Xiaojun Yu, Minfeng Chen, L. Luo, Changzheng Shi
          </td>
          <td>2024-07-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d1e5e2f8fff698fa2e7aaaa422dd95deef960f" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis identifies a prognostic T-cell signature in colorectal cancer
              </a>
            </td>
          <td>
            Peng Cui, Haibo Wang, Zhigang Bai
          </td>
          <td>2024-08-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Bladder cancers are heterogeneous in nature, showing diverse molecular profiles and histopathological characteristics, which pose challenges for diagnosis and treatment. However, understanding the molecular basis of such heterogeneity has remained elusive. This study aimed to elucidate the molecular landscape of neuroendocrine-like bladder tumors, focusing on the involvement of β-catenin localization. Analyzing the transcriptome data and benefiting from the molecular classification tool, we undertook an in-depth analysis of muscle-invasive bladder cancers to uncover the molecular characteristics of the neuroendocrine-like differentiation. The study explored the contribution of transcription factors and chromatin remodeling complexes to neuroendocrine differentiation in bladder cancer. The study revealed a significant correlation between β-catenin localization and neuroendocrine differentiation in muscle-invasive bladder tumors, highlighting the molecular complexity of neuroendocrine-like tumors. Enrichment of YY1 transcription factor, E2F family members, and Polycomb repressive complex components in β-catenin-positive tumors suggest their potential contribution to neuroendocrine phenotypes. Our findings contribute valuable insights into the molecular complexity of neuroendocrine-like bladder tumors. By identifying potential therapeutic targets and refining diagnostic strategies, this study advances our understanding of endocrinology in the context of bladder cancer. Further investigations into the functional implications of these molecular relationships are warranted to enhance our knowledge and guide future therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c6b5215bbb6583476be728532e64b764e1d96" target='_blank'>
              Nucleocytoplasmic β-catenin expression contributes to neuroendocrine differentiation in muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            Burcu Akman, Ahmet Bursali, Mustafa Gürses, A. Suner, Gökhan Karakülah, U. Mungan, K. Yörükoğlu, Serap Erkek-Ozhan
          </td>
          <td>2024-07-14</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Solid tumors represent the most common type of cancer in humans and are classified into sarcomas, lymphomas, and carcinomas based on the originating cells. Among these, carcinomas, which arise from epithelial and glandular cells lining the body's tissues, are the most prevalent. Around the world, a significant increase in the incidence of solid tumors is observed during recent years. In this context, efforts to discover more effective cancer treatments have led to a deeper understanding of the tumor microenvironment and its components. Currently, the interactions between cancer cells and elements of the tumor microenvironment are being intensely investigated. Remarkable progress in research is noted, largely due to the development of advanced in vitro models, such as Tumor-on-a-Chip models that assist in understanding and ultimately discovering new effective treatments for a specific type of cancer. The purpose of this paper is to provide a review of the tumor microenvironment and cancer cell components, along with the advances on tumor-on-a-chip models designed to mimic tumors, offering a perspective on the current state-of-the-art. Recent studies using this kind of microdevices that reproduce the tumor microenvironment have allowed a better understanding of the cancer and its treatments. Nevertheless, current applications of this technology present some limitations that must be overcome in order to achieve a broad application by researchers looking for a deeper knowledge of cancer and new strategies to improve current therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d9d7fa6f0b9b61f8c31ffe7d59e62a370ae7ef" target='_blank'>
              The Microenvironment of Solid Tumors: Components and current challenges of Tumor-on-a-Chip models.
              </a>
            </td>
          <td>
            Ilva de Fátima Souza, João Paulo Jesus Vieira, E. Bonifácio, Bethânia Alves Avelar Freitas, Libardo Andrés González Torres
          </td>
          <td>2024-07-15</td>
          <td>Tissue engineering. Part B, Reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Cells must adopt flexible regulatory strategies to make decisions regarding their fate, including differentiation, apoptosis, or survival in the face of various external stimuli. One key cellular strategy that enables these functions is stochastic gene expression programs. However, understanding how transcriptional bursting, and consequently, cell fate, responds to DNA damage on a genome-wide scale poses a challenge. In this study, we propose an interpretable and scalable inference framework, DeepTX, that leverages deep learning methods to connect mechanistic models and scRNA-seq data, thereby revealing genome-wide transcriptional burst kinetics. This framework enables rapid and accurate solutions to transcription models and the inference of transcriptional burst kinetics from scRNA-seq data. Applying this framework to several scRNA-seq datasets of DNA-damaging drug treatments, we observed that fluctuations in transcriptional bursting induced by different drugs could lead to distinct fate decisions: IdU treatment induces differentiation in mouse embryonic stem cells by increasing the burst size of gene expression, while 5FU treatment with low and high dose increases the burst frequency of gene expression to induce cell apoptosis and survival in human colon cancer cells. Together, these results show that DeepTX can be used to analyze single-cell transcriptomics data and can provide mechanistic insights into cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08e79a79f41500c7b1baabfdd652bb03e9ecc3e" target='_blank'>
              Deep learning linking mechanistic models to single-cell transcriptomics data reveals transcriptional bursting in response to DNA damage
              </a>
            </td>
          <td>
            Zhiwei Huang, Songhao Luo, Zihao Wang, Zhenquan Zhang, Benyuan Jiang, Qing Nie, Jiajun Zhang
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome editing has demonstrated its utility in generating isogenic cell-based disease models, enabling the precise introduction of genetic alterations into wild-type cells to mimic disease phenotypes and explore underlying mechanisms. However, its application in liver-related diseases has been limited by challenges in genetic modification of mature hepatocytes in a dish. Here, we conducted a systematic comparison of various methods for primary hepatocyte culture and gene delivery to achieve robust genome editing of hepatocytes ex vivo. Our efforts yielded editing efficiencies of up to 80% in primary murine hepatocytes cultured in monolayer and 20% in organoids. To model human hepatic tumorigenesis, we utilized hepatocytes differentiated from human pluripotent stem cells (hPSCs) as an alternative human hepatocyte source. We developed a series of cellular models by introducing various single or combined oncogenic alterations into hPSC-derived hepatocytes. Our findings demonstrated that distinct mutational patterns led to phenotypic variances, affecting both overgrowth and transcriptional profiles. Notably, we discovered that the PI3KCA E542K mutant, whether alone or in combination with exogenous c-MYC, significantly impaired hepatocyte functions and facilitated cancer metabolic reprogramming, highlighting the critical roles of these frequently mutated genes in driving liver neoplasia. In conclusion, our study demonstrates genome-engineered hepatocytes as valuable cellular models of hepatocarcinoma, providing insights into early tumorigenesis mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0098e14922960e96572df20d519a8a83fa8956" target='_blank'>
              Genome Engineering of Primary and Pluripotent Stem Cell-Derived Hepatocytes for Modeling Liver Tumor Formation
              </a>
            </td>
          <td>
            Lulu Zhang, Xunting Wang, Xuelian Yang, Yijia Chi, Yihang Chu, Yi Zhang, Yufan Gong, Fei Wang, Qian Zhao, Dongxin Zhao
          </td>
          <td>2024-09-02</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cardiac allograft vasculopathy (CAV), a diffuse thickening of the intima of the coronary arteries and microvasculature, is the leading cause of late graft failure and mortality after heart transplantation (HT). Diagnosis involves invasive coronary angiography, which carries substantial risk, and minimally-invasive approaches to CAV diagnosis are urgently needed. Using single-cell RNA-sequencing in peripheral blood mononuclear cells (PBMCs), we sought to identify cell-specific gene expression profiles in CAV. Methods Whole blood was collected from 22 HT recipients with angiographically-confirmed CAV and 18 HT recipients without CAV. PBMCs were isolated and subjected to single-cell RNA-sequencing using a 10X Genomics microfluidic platform. Downstream analyses focused on differential expression of genes, cell compositional changes, and T cell receptor repertoire analyses. Results Across 40 PBMC samples, we isolated 134,984 cells spanning 8 major clusters and 31 subclusters of cell types. Compositional analyses showed subtle, but significant increases in CD4+ T central memory cells, and CD14+ and CD16+ monocytes in high-grade CAV (CAV-2 and CAV-3) as compared to low-grade or absent CAV. After adjusting for age, gender, and prednisone use, 745 genes were differentially expressed in a cell-specific manner in high-grade CAV. Weighted gene co-expression network analyses showed enrichment for putative pathways involved in inflammation and angiogenesis. There were no significant differences in T cell clonality or diversity with increasing CAV severity. Conclusions Unbiased whole transcriptomic analyses at single-cell resolution identify unique, cell-specific gene expression patterns in CAV, suggesting the potential utility of peripheral gene expression biomarkers in diagnosing CAV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567376b4c2c8bc3b9b425a0a70eb9e9401f4f4a7" target='_blank'>
              Single-cell RNA-sequencing identifies unique cell-specific gene expression profiles in high-grade cardiac allograft vasculopathy
              </a>
            </td>
          <td>
            K. Amancherla, K. H. Schlendorf, Nelson Chow, Quanhu Sheng, Jane E. Freedman, J.C. Rathmell
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Immunotherapies targeting T cells in solid cancers are revolutionizing clinical treatment. Novel immunotherapies have had extremely limited benefit for acute myeloid leukemia (AML). Here, we characterized the immune microenvironment of t(8;21) AML patients to determine how immune cell infiltration status influenced prognosis. Methods Through multi-omics studies of primary and longitudinal t(8;21) AML samples, we characterized the heterogeneous immune cell infiltration in the tumor microenvironment and their immune checkpoint gene expression. Further external cohorts were also included in this research. Results CD8+ T cells were enriched and HAVCR2 and TIGIT were upregulated in the CD34+CD117dim%-High group; these features are known to be associated with immune exhaustion. Data integration analysis of single-cell dynamics revealed that a subset of T cells (cluster_2) (highly expressing GZMB, NKG7, PRF1 and GNLY) evolved and expanded markedly in the drug-resistant stage after relapse. External cohort analysis confirmed that the cluster_2 T-cell signature could be utilized to stratify patients by overall survival outcome. Discussion In conclusion, we discovered a distinct T-cell signature by scRNA-seq that was correlated with disease progression and drug resistance. Our research provides a novel system for classifying patients based on their immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc810c4ae6890ff0d2634f30ff5c8e59f928a8c" target='_blank'>
              Integrative single-cell analysis of longitudinal t(8;21) AML reveals heterogeneous immune cell infiltration and prognostic signatures
              </a>
            </td>
          <td>
            Xue-Ping Li, Jiang-Tao Song, Yu-ting Dai, Wei-Na Zhang, Bai-Tian Zhao, J. Mao, Yan Gao, Lu Jiang, Yang Liang
          </td>
          <td>2024-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are the predominant immune population within the tumor microenvironment (TME), playing a key role in promoting tumor growth and establishing an immunosuppressive environment that facilitate immune evasion. Here we report that the immune receptor CD300e is highly expressed by TAM in colorectal (CRC) and drives their immunosuppressive and pro-tumorigenic, correlating with reduced expression of MHC-II molecules, essential for antigen presentation. In vitro, CD300e-deficient macrophages exhibit enhanced pro-inflammatory activity and phagocytic capacity, coupled with reduced efferocytosis, suggesting a critical role for CD300e in promoting tumor progression. The depletion of CD300e, in vivo, results in a reduced tumor burden and enhanced survival in CRC mouse models, accompanied by a more robust anti-tumor immune response characterized by increased infiltration of activated CD4+ and CD8+ T cells producing IFN-γ. Our study provides comprehensive insights into the roles of CD300e in myeloid cells in CRC, highlighting its potential as a therapeutic target for reprogramming TAMs to support anti-tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce54a08c3fd49574486dc4b007db233ab49008e" target='_blank'>
              CD300e as a Driver of Immunosuppressive Tumor Microenvironment in Colorectal Cancer
              </a>
            </td>
          <td>
            Annica Barizza, Stefania Vassallo, L. Masatti, S. Lonardi, M. Bugatti, Sara Coletta, Sofia Giacometti, Simone Pizzini, W. Vermi, Fabio Munari, Nicolò Gnoato, Matteo Fassan, Giulia Nigro, G. Spolverato, E. Calura, Gaia Codolo
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b21391e9d7b4e36087d981b4ffc2b93179db8ea" target='_blank'>
              Combined microfluidic enrichment and staining workflow for single-cell analysis of circulating tumor cells in metastatic prostate cancer patients
              </a>
            </td>
          <td>
            Caroline Løppke, Amalie M Jørgensen, Nicolai T. Sand, Rasmus B Klitgaard, G. Daugaard, Mette Ø Agerbæk
          </td>
          <td>2024-07-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Numerous aspects of cellular signaling are regulated by the kinome—the network of over 500 protein kinases that guides and modulates information transfer throughout the cell. The key role played by both individual kinases and assemblies of kinases organized into functional subnetworks leads to kinome dysregulation driving many diseases, particularly cancer. In the case of pancreatic ductal adenocarcinoma (PDAC), a variety of kinases and associated signaling pathways have been identified for their key role in the establishment of disease as well as its progression. However, the identification of additional relevant therapeutic targets has been slow and is further confounded by interactions between the tumor and the surrounding tumor microenvironment. In this work, we attempt to link the state of the human kinome, or kinotype, with cell viability in treated, patient-derived PDAC tumor and cancer-associated fibroblast cell lines. We applied classification models to independent kinome perturbation and kinase inhibitor cell screen data, and found that the inferred kinotype of a cell has a significant and predictive relationship with cell viability. We further find that models are able to identify a set of kinases whose behavior in response to perturbation drive the majority of viability responses in these cell lines, including the understudied kinases CSNK2A1/3, CAMKK2, and PIP4K2C. We next utilized these models to predict the response of new, clinical kinase inhibitors that were not present in the initial dataset for model devlopment and conducted a validation screen that confirmed the accuracy of the models. These results suggest that characterizing the perturbed state of the human protein kinome provides significant opportunity for better understanding of signaling behavior and downstream cell phenotypes, as well as providing insight into the broader design of potential therapeutic strategies for PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd178f02946551d2f3d6c0478dc67e5c83de2358" target='_blank'>
              Kinome state is predictive of cell viability in pancreatic cancer tumor and cancer-associated fibroblast cell lines
              </a>
            </td>
          <td>
            Matthew E. Berginski, Madison R. Jenner, Chinmaya U. Joisa, Gabriela Herrera Loeza, B. Golitz, Matthew B Lipner, J. R. Leary, Naim Rashid, Gary L. Johnson, J. J. Yeh, Shawn M Gomez
          </td>
          <td>2024-08-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing is a valuable technique for identifying diverse cell subtypes. A key challenge in this process is that the detection of rare cells is often missed by conventional methods due to low abundance and subtle features of these cells. To overcome this, we developed SCLCNF (Local Connectivity Network Feature Sharing in Single-Cell RNA sequencing), a novel approach that identifies rare cells by analyzing features uniquely expressed in these cells. SCLCNF creates a cellular connectivity network, considering how each cell relates to its neighbors. This network helps to pinpoint coexpression patterns unique to rare cells, utilizing a rarity score to confirm their presence. Our method performs better in detecting rare cells than existing techniques, offering enhanced robustness. It has proven to be effective in human gastrula data sets for accurately pinpointing rare cells, and in sepsis data sets where it uncovers previously unidentified rare cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41629b08b97b96e2a5087885fb3403f0009f9b9" target='_blank'>
              Connectivity Network Feature Sharing in Single-Cell RNA Sequencing Data Identifies Rare Cells.
              </a>
            </td>
          <td>
            Shudong Wang, Hengxiao Li, Yahui Liu, Shanchen Pang, Sibo Qiao, Jionglong Su, Shaoqiang Wang, Yulin Zhang
          </td>
          <td>2024-08-03</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The tumor microenvironment (TME) consists of complex interactions between cellular and extracellular components, among which the immune system is known to play an integral role in disease progression and response to therapy. Cytokines and chemokines are cell signaling proteins used by immune cells to communicate with each other as well as with other cell types in the body. These proteins control systemic and local immune responses and levels of cytokines/chemokines in the TME have been associated with tumor outcomes. However, cytokines and chemokines have varied expression across cell types, tumors, and host conditions. Therefore, approaches to effectively study the production of these proteins at the single-cell level in the TME are needed to fully elucidate the mechanisms governing the anti-cancer immune response. Here, we detail a protocol to assess the production of cytokines/chemokines across leukocyte populations in mouse tumors using RNA flow cytometry. Importantly, this method can be adapted with minimal changes to study various mouse and human tumors, other RNA analytes, and non-tumor tissues. With this approach, we characterize single-cell production of Ifnb1, Xcl1 and Ccl5 in mouse tumors and identify monocytes and monocyte-derived macrophages as the main producers of type I interferon transcript Ifnb1 consistent across 4 different syngeneic tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e3dc9db7634219e9551930578d0d75b0478447" target='_blank'>
              Use of optimized single-cell RNA flow cytometry protocol identifies monocytes as main producers of type I interferon in mouse syngeneic tumors
              </a>
            </td>
          <td>
            Khiem C. Lam, Quanyi Chen, R. Goldszmid
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cervical cancer has become a major worldwide health concern that demands attention to women's health and often needs more effective and specialized treatment options. Cervical cancer claims the lives of over 300,000 women globally, ranking as the fourth most prevalent cancer among women. The tumor microenvironment (TME) shapes a distinctive landscape for tumor survival, characterized by factors like immunosuppression, hypoxia, acidity, and nutrient scarcity. Comprising tumor cells, immune cells, mesenchymal cells, cancer-associated fibroblasts, and extracellular matrix, the TME reprograms key aspects of tumor development, uncontrolled proliferation, invasion, metastasis, and response to treatments. Recognizing the TME's pivotal role in tumor progression and treatment responsiveness, targeting the TME has emerged as a potential strategy in cancer therapy. This publication delves into recent TME research, offering a comprehensive overview of the specific functions of each TME component in cancer development and progression. Based on the reviewed literature, it appears that women with cervical cancer may benefit from more effective therapy, fewer side effects, and a higher quality of life in the future. By addressing pressing problems and unmet needs in the field, this review has the potential to significantly alter the course of cervical cancer treatment in the future. Furthermore, it outlines the primary therapeutic targets identified by researchers, which may prove valuable in treating tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627e4e1b395f65a8bd5b6e52e925ef93c39c676d" target='_blank'>
              Tumor Microenvironment: From Cervical Carcinogenesis to Therapeutic Advancements.
              </a>
            </td>
          <td>
            Srishti Sharma, Meenakshi Verma, Indra Rautela, Fahad Khan, P. Pandey
          </td>
          <td>2024-09-04</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
The distinct functions of immune cells in atherosclerosis have been mostly defined by preclinical mouse studies. Contrastingly, the immune cell composition of human atherosclerotic plaques and their contribution to disease progression is only poorly understood. It remains uncertain whether genetic animal models allow for valuable translational approaches.


METHODS AND RESULTS
Single cell RNA-sequencing (scRNA-seq) was performed to define the immune cell landscape in human carotid atherosclerotic plaques. The human immune cell repertoire demonstrated an unexpectedly high heterogeneity and was dominated by cells of the T-cell lineage, a finding confirmed by immunohistochemistry. Bioinformatical integration with 7 mouse scRNA-seq data sets from adventitial and atherosclerotic vascular tissue revealed a total of 51 identities of cell types and differentiation states, of which some were only poorly conserved between species and exclusively found in humans. Locations, frequencies, and transcriptional programs of immune cells in mouse models did not resemble the immune cell landscape in human carotid atherosclerosis. In contrast to standard mouse models of atherosclerosis, human plaque leukocytes were dominated by several T-cell phenotypes with transcriptional hallmarks of T-cell activation and memory formation, T-cell receptor-, and pro-inflammatory signaling. Only mice at the age of 22 months partially resembled the activated T-cell phenotype. In a validation cohort of 43 patients undergoing carotid endarterectomy, the abundance of activated immune cell subsets in the plaque defined by multi-color flow cytometry associated with the extend of clinical atherosclerosis.


CONCLUSIONS
Integrative scRNA-seq reveals a substantial difference in the immune cell composition of murine and human carotid atherosclerosis - a finding that questions the translational value of standard mouse models for adaptive immune cell studies. Clinical associations suggest a specific role for T-cell driven (auto-) immunity in human plaque formation and -instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80b139f986c47757e5b62dbac2100eb09e11707" target='_blank'>
              Cross-species single-cell RNA sequencing reveals divergent phenotypes and activation states of adaptive immunity in human carotid and experimental murine atherosclerosis.
              </a>
            </td>
          <td>
            H. Horstmann, N. Michel, Xia Sheng, Sophie Hansen, A. Lindau, Katharina Pfeil, Marbely C Fernández, T. Marchini, Holger Winkels, L. Mitre, Tijani Abogunloko, Xiaowei Li, T. Mwinyella, M. Gissler, Heiko Bugger, Timo Heidt, Konrad Buscher, I. Hilgendorf, Peter Stachon, S. Piepenburg, Nicolas Verheyen, Thomas Rathner, Teresa Gerhardt, P. Siegel, W. Oswald, T. Cohnert, A. Zernecke, J. Madl, P. Kohl, A. Foks, C. von zur Muehlen, Dirk Westermann, Andreas Zirlik, Dennis Wolf
          </td>
          <td>2024-07-23</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="25 Background: The brain tumor immune microenvironment (TiME) is characterised by suppression of tumor-infiltrating leukocytes. The lymphocyte niche activity is particularly downregulated within this TiME milieu. Methods: Here, using data-driven approaches in flow cytometry and RNA sequencing analysis, we aimed at dissecting leukocyte cell types infiltrating human brain tumor TiME. Results: Unsupervised clustering provided T lymphocyte subsets including double-negative (CD3+CD4- CD8-) T cells (DNT). Our DNT phenotype was validated with an independent dataset, suggesting DNT to be compatible with γδ T cell phenotype. In our cohort, myeloid immune cell frequencies were associated with malignancy type as previously reported, while a gliosarcoma notably presented unexpectedly high lymphocyte frequencies. Consistently, double-positive (CD4+ CD8+) T cells (DPT) had higher frequency in the gliosarcoma than other tumors. The flow cytometry results were supported by transcriptome patterns and the activity of immune-related genes in these tumors. Conclusions: Our findings feature the benefit of data-driven approaches for brain TiME lymphocyte analysis, and show that primary brain tumors can entail atypical lymphocyte subsets like DNTs and DPT cells. Unbiased lymphocyte characterisation can potentially outline novel targets for immune checkpoint blockade therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb1ae6998ef7cf2b065eaabbce0c558410319b8" target='_blank'>
              Unsupervised clustering of brain tumor leukocytes to reveal atypical lymphoid cell subsets.
              </a>
            </td>
          <td>
            Ismaïl Hermelo, Tuomo Virtanen, I. Salonen, R. Nätkin, Sofia Keitaanniemi, A. Tiihonen, Suvi Lehtipuro, Laura Kummola, K. Nordfors, H. Haapasalo, M. Rauhala, J. Kesseli, M. Nykter, J. Haapasalo, K. Rautajoki
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma is a devastating disease accounting for ~15% of all childhood cancer deaths. Collagen content and fiber association within the tumor stroma influence tumor progression and metastasis. High expression levels of collagen receptor kinase, Discoidin domain receptor II (DDR2), are associated with poor survival of neuroblastoma patients. Additionally, cancer cells generate and sustain mechanical forces within their enviroment as a part of their normal physiology. Despite this, whether collagen activated DDR2 signaling dysregulate these migration forces is still elusive. To address these questions, a shRNA DDR2 knockdown neuroblastoma cell line (SH-SY5Y) was engineered to evaluate the consequence of DDR2 on cellular mechanics. Atomic force microscopy and traction force microscopy were utlizing to unveil the biophysical altercations. DDR2 down-regulation was found to significantly reduce proliferation, cell stiffness, and cellular elongation. Aditionally, DDR2 down-regulated cells had decreased traction forces when plated on collagen coated elastic substrates. Together, these results highlight the important role that DDR2 has in reducing migration mechanics in neuroblastoma and might be a promising target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f707587725fd2ca416d376c5d57d87998251580c" target='_blank'>
              Investigation of cell mechanics and migration on DDR2-expressing neuroblastoma cell line
              </a>
            </td>
          <td>
            Theadora Vessella, Esteban J. Rozen, Jason Shohet, Qi Wen, Hong Susan Zhou
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Inflammation is a key component in the development of abdominal aortic aneurysm (AAA), yet insights into the roles of immune cells and their interactions in this process are limited.


METHODS
Using single-cell RNA transcriptomic analysis, we deconstructed the CD45+ cell population in elastase-induced murine AAA at the single-cell level. We isolated each group of immune cells from murine AAA tissue at different time points and divided them into several subtypes, listed the remarkable differentially expressed genes, explored the developmental trajectories of immune cells, and demonstrated the interactions among them.


RESULTS
Our findings reveal significant differences in several immune cell subsets, including macrophages, dendritic cells, and T cells, within the AAA microenvironment compared with the normal aorta. Especially, conventional dendritic cell type 1 exclusively existed in the AAA tissue rather than the normal aortas. Via CellChat analysis, we identified several intercellular communication pathways like visfatin, which targets monocyte differentiation and neutrophil extracellular trap-mediated interaction between neutrophils and dendritic cells, which might contribute to AAA development. Some of these pathways were validated in human AAA.


CONCLUSIONS
Despite the absence of external pathogenic stimuli, AAA tissues develop a complex inflammatory microenvironment involving numerous immune cells. In-depth studies of the inflammatory network shall provide new strategies for patients with AAA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0ebb97e30e8ccddcf0b4ca430f1bb3ad1428083" target='_blank'>
              Single-Cell RNA Sequencing Deconstructs the Distribution of Immune Cells Within Abdominal Aortic Aneurysms in Mice.
              </a>
            </td>
          <td>
            Zhen Yuan, Li Shu, Jiantao Fu, Peipei Yang, Yidong Wang, Jie Sun, Mengsha Zheng, Zhenjie Liu, Jin Yang, Jiangping Song, Shen Song, Zhejun Cai
          </td>
          <td>2024-07-25</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The cell cycle is crucial for maintaining normal cellular functions and preventing replication errors. FOXM1, a key transcription factor, plays a pivotal role in regulating cell cycle progression and is implicated in various physiological and pathological processes, including cancers like liver, prostate, breast, lung, and colon cancer. Despite previous research, our understanding of FOXM1 dynamics under different cell cycle perturbations and its connection to heterogeneous cell fate decisions remains limited. In this study, we investigated FOXM1 behavior in individual cells exposed to various perturbagens. We found that different drugs induce diverse responses due to heterogeneous FOXM1 dynamics at the single-cell level. Single-cell analysis identified six distinct cellular phenotypes: on-time cytokinesis, cytokinesis delay, cell cycle delay, G1 arrest, G2 arrest, and cell death, observed across different drug types and doses. Specifically, treatments with PLK1, CDK1, CDK1/2, and Aurora kinase inhibitors revealed varied FOXM1 dynamics leading to heterogeneous cellular outcomes. Our findings affirm that FOXM1 dynamics are pivotal in determining cellular outcomes, independent of the specific inhibitor employed. Our results gave insights into how FOXM1 dynamics contribute to cell cycle fate decisions, especially under different cell-cycle perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c75453a839e87a7d8f39be8553c63d1773d34aa" target='_blank'>
              Exploring the Single-Cell Dynamics of FOXM1 Under Cell Cycle Perturbations
              </a>
            </td>
          <td>
            Tooba Jawwad, M. Kamkaew, Kriengkrai Phongkitkarun, Porncheera Chusorn, S. Sampattavanich
          </td>
          <td>2024-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab20abff3d5ab2e3023b99dd1dd6ab980ad6081" target='_blank'>
              Mutual exclusivity and co-occurrence patterns of immune checkpoints indicate NKG2A relates to anti-PD-1 resistance in gastric cancer
              </a>
            </td>
          <td>
            Guanjun Li, Xiaohan Liu, Chunhui Gu, Guichuang Ma, Shaowei Li, Zhenfeng Ma, Yanqiu Xiong, Yu Jiang, Qiong Huang, Jianhua Wu, Zhenzhen Wu, Wangjun Liao, Qijing Wu, Min Shi
          </td>
          <td>2024-08-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Deciphering the intricate cell‐state transitions orchestrating immune adaptation over time stands as a cornerstone for advancing biological understanding. However, the lack of empirical in vivo genomic technologies capable of capturing cellular dynamics has posed a significant challenge. In response to this gap, a groundbreaking study introduces Zman‐seq, a single‐cell technology that records transcriptomic dynamics across time by incorporating time stamps into circulating immune cells, enabling their tracking in tissues for extended periods. The application of Zman‐seq in glioblastoma research has successfully unraveled the cell state and molecular trajectories underlying the dysfunctional immune microenvironment. Understanding the temporal aspects of cell‐state transitions during immune responses is pivotal for advancing our knowledge in biology. The emergence of Zman‐seq addresses the current limitations in empirical in vivo genomic technologies, offering a revolutionary approach to studying the dynamics of immune cells over time. This highlight comprehensively explores the implications of Zman‐seq in resolving cell‐state transitions and molecular trajectories within the dysfunctional immune microenvironment in different types of immunotherapy. This technique has particular potential for chimeric antigen receptor T‐cell therapy, overriding drug resistance, clinical medication optimization, and facilitating drug development. In particular, this article discusses potential strategies for improving the efficacy of clinical treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6fd82670000d1f72f4d7503c3853bc7af1763a3" target='_blank'>
              Time – The fourth dimension of immune cells
              </a>
            </td>
          <td>
            Guiming Li, Wenjun Zhang, Jing Yang
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioma is a highly heterogeneous malignancy of the central nervous system. This heterogeneity is driven by various molecular processes, including neoplastic transformation, cell cycle dysregulation, and angiogenesis. Among these biomolecular events, inflammation and stress pathways in the development and driving factors of glioma heterogeneity have been reported. However, the mechanisms of glioma heterogeneity under stress response remain unclear, especially from a spatial aspect.This study employed single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) to explore the impact of oxidative stress response genes in oligodendrocyte precursor cells (OPCs). Our analysis identified distinct pathways activated by oxidative stress in two different types of gliomas: high- and low- grade (HG and LG) gliomas.In HG gliomas, oxidative stress induced a metabolic shift from oxidative phosphorylation to glycolysis, promoting cell survival by preventing apoptosis. This metabolic reprogramming was accompanied by epithelial-to-mesenchymal transition (EMT) and an upregulation of stress response genes. Furthermore, SCENIC (Single-Cell rEgulatory Network Inference and Clustering) analysis revealed that oxidative stress activated the AP1 transcription factor in HG gliomas, thereby enhancing tumor cell survival and proliferation.Our findings provide a novel perspective on the mechanisms of oxidative stress responses across various grades of gliomas. This insight enhances our comprehension of the evolutionary processes and heterogeneity within gliomas, potentially guiding future research and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56dee2729032b7749900d8cd04b93b8180eff50e" target='_blank'>
              Single-cell and spatial transcriptome assays reveal heterogeneity in gliomas through stress responses and pathway alterations
              </a>
            </td>
          <td>
            Z. He, Zheng Liu, Qi Wang, Xingjian Sima, Wei Zhao, Chunmei He, Wenjie Yang, Han Chen, Bo Gong, Siyuan Song, Yi Wang
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) data, susceptible to noise arising from biological variability and technical errors, can distort gene expression analysis and impact cell similarity assessments, particularly in heterogeneous populations. Current methods, including deep learning approaches, often struggle to accurately characterize cell relationships due to this inherent noise. To address these challenges, we introduce scAMF (Single-cell Analysis via Manifold Fitting), a framework designed to enhance clustering accuracy and data visualization in scRNA-seq studies. At the heart of scAMF lies the manifold fitting module, which effectively denoises scRNA-seq data by unfolding their distribution in the ambient space. This unfolding aligns the gene expression vector of each cell more closely with its underlying structure, bringing it spatially closer to other cells of the same cell type. To comprehensively assess the impact of scAMF, we compile a collection of 25 publicly available scRNA-seq datasets spanning various sequencing platforms, species, and organ types, forming an extensive RNA data bank. In our comparative studies, benchmarking scAMF against existing scRNA-seq analysis algorithms in this data bank, we consistently observe that scAMF outperforms in terms of clustering efficiency and data visualization clarity. Further experimental analysis reveals that this enhanced performance stems from scAMF's ability to improve the spatial distribution of the data and capture class-consistent neighborhoods. These findings underscore the promising application potential of manifold fitting as a tool in scRNA-seq analysis, signaling a significant enhancement in the precision and reliability of data interpretation in this critical field of study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acbaf46cdaedc6e0bd116a59835656d80ed78ff3" target='_blank'>
              Single-cell analysis via manifold fitting: A framework for RNA clustering and beyond.
              </a>
            </td>
          <td>
            Zhigang Yao, Bingjie Li, Yukun Lu, S. Yau
          </td>
          <td>2024-09-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Ionizing radiation exposure can cause damage to diverse tissues and organs, with the hematopoietic system being the most sensitive. However, limited information is available regarding the radiosensitivity of various hematopoietic cell populations in the bone marrow due to the high heterogeneity of the hematopoietic system. In this study, we observed that granulocyte–macrophage progenitors, hematopoietic stem/progenitor cells, and B cells within the bone marrow showed the highest sensitivity, exhibiting a rapid decrease in cell numbers following irradiation. Nonetheless, neutrophils, natural killer (NK) cells, T cells, and dendritic cells demonstrated a certain degree of radioresistance, with neutrophils exhibiting the most pronounced resistance. By employing single-cell transcriptome sequencing, we investigated the early responsive genes in various cell types following irradiation, revealing that distinct gene expression profiles emerged between radiosensitive and radioresistant cells. In B cells, radiation exposure led to a specific upregulation of genes associated with mitochondrial respiratory chain complexes, suggesting a connection between these complexes and cell radiosensitivity. In neutrophils, radiation exposure resulted in fewer gene alterations, indicating their potential for distinct mechanisms in radiation resistance. Collectively, this study provides insights into the molecular mechanism for the heterogeneity of radiosensitivity among the various bone marrow hematopoietic cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb664766831a6e9ac2aeeb0e263cf03abab8ee2" target='_blank'>
              Single-Cell Transcriptomics Reveals Early Effects of Ionizing Radiation on Bone Marrow Mononuclear Cells in Mice
              </a>
            </td>
          <td>
            Yun-Qiang Wu, Ke-Xin Ding, Zhi-Chun Lv, Zheng-Yue Cao, Ke Zhao, Huiying Gao, Hui-Ying Sun, Jing-Jing Li, Si-Yu Li, Xiong-Wei Zhao, Yang Xue, Shen-Si Xiang, Xiao-Fei Zheng, Xiao-Ming Yang, Chang-Yan Li
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="OBJECTIVE
Pancreatic ductal adenocarcinoma (PDAC) stands as one of the most lethal cancers, marked by its lethality and limited treatment options, including the utilisation of checkpoint blockade (ICB) immunotherapy. Epigenetic dysregulation is a defining feature of tumourigenesis that is implicated in immune surveillance, but remains elusive in PDAC.


DESIGN
To identify the factors that modulate immune surveillance, we employed in vivo epigenetic-focused CRISPR-Cas9 screen in mouse PDAC tumour models engrafted in either immunocompetent or immunodeficient mice.


RESULTS
Here, we identified MED12 as a top hit, emerging as a potent negative modulator of immune tumour microenviroment (TME) in PDAC. Loss of Med12 significantly promoted infiltration and cytotoxicity of immune cells including CD8+ T cells, natural killer (NK) and NK1.1+ T cells in tumours, thereby heightening the sensitivity of ICB treatment in a mouse model of PDAC. Mechanistically, MED12 stabilised heterochromatin protein HP1A to repress H3K9me3-marked endogenous retroelements. The derepression of retrotransposons induced by MED12 loss triggered cytosolic nucleic acid sensing and subsequent activation of type I interferon pathways, ultimately leading to robust inflamed TME . Moreover, we uncovered a negative correlation between MED12 expression and immune resposne pathways, retrotransposon levels as well as the prognosis of patients with PDAC undergoing ICB therapy.


CONCLUSION
In summary, our findings underscore the pivotal role of MED12 in remodelling immnue TME through the epigenetic silencing of retrotransposons, offering a potential therapeutic target for enhancing tumour immunogenicity and overcoming immunotherapy resistance in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18010d0edb9998ef713f34115b0a8998ae88296" target='_blank'>
              MED12 loss activates endogenous retroelements to sensitise immunotherapy in pancreatic cancer.
              </a>
            </td>
          <td>
            Yingying Tang, Shijie Tang, Wenjuan Yang, Zhengyan Zhang, Teng Wang, Yuyun Wu, Junyi Xu, Christian Pilarsky, Massimiliano Mazzone, Lei-Wei Wang, Yongwei Sun, Ruijun Tian, Yujie Tang, Yu Wang, Chao Wang, Jing Xue
          </td>
          <td>2024-08-31</td>
          <td>Gut</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Prostate cancer is a complex and heterogeneous disease with multiple tumor foci, each potentially harboring distinct driver molecular aberrations. This complexity poses challenges to effective management. We took an innovative approach to gain a comprehensive understanding of the genetic underpinnings of each tumor focus and avoid overlooking more minor yet clinically significant foci. Instead of relying solely on a systematic sampling of dominant foci, we conducted molecular analysis on whole-mount radical prostatectomy specimens. Our study aimed to find distinct molecular subsets of prostate cancer and assess their correlation with clinical outcomes, focusing on Caucasians (CA) and African Americans (AA). Method: We randomly selected 2201 whole-mount radical prostatectomy cases, with 1207 (54.8%) from CA and 994 (45.1%) from AA patients evaluated for a 5-year biochemical recurrence-free survival rate (BCR). Of these 2201 cases, 834 (463 -56% were from CA and 371 -44% from AA patients) were subjected to molecular analysis using dual immunohistochemistry (IHC) for ERG and SPINK1, along with dual RNA in-situ hybridization (RNA-ISH) for ETV1 and ETV4 to evaluate tumor molecular heterogeneity on whole-mount specimens. The Chi-squared test examined racial disparities in aberrant oncogene expression. To assess BCR-free survival, we employed the Kaplan-Meier method and Cox-PH models for patients with distinct molecular subsets of prostate cancer. Additionally, Gleason Grade groups of prostate biopsies were summarized using a spaghetti plot and compared using linear mixed models. Results: Analysis of the 2201 cases revealed that AA with localized prostate cancer behaved differently with better 5-year BCR-free survival than CA after radical prostatectomy (AA: 0.82, 95% CI 0.80-0.85; CA: 0.71, 95% CI 0.68-0.75; p<.001). Molecular profiling of whole-mount specimens from 834 cases revealed that 16.4%, 58.4%, 21.7%, and 3.5% of patients with localized prostate cancer expressed none, one, two, and three of the four oncogenes, respectively. This finding identified new molecular subsets of prostate cancer with more than one driver mutation in a mutually exclusive manner within the multifocal disease. ERG and SPINK1 expression showed a negative correlation (p<.001). Notably, AA patients exhibited a lower incidence of ERG (38.8% vs. 60.3%) but a higher incidence of SPINK1 (63.3% vs. 35.6%) than CA patients. The incidences of ETV1 (9.4% vs. 9.3%) and ETV4 (4.6% vs. 3.9%) were not statistically significant between the two racial groups. However, significantly, ETV1 expression was associated with worse BCR-free survival in CA patients (hazard ratio [HR]=2.36, 95% CI 1.22-4.57, p=.02), while ETV4 expression was linked to worse BCR-free survival in AA patients (HR=2.65, 95% CI 1.15-6.09, p=.02). Moreover, ETV4 expression was associated with regional lymph node metastasis in AA patients (odds ratio [OR]=5.14, 95% CI 1.3-17.4, p=.01) but not in CA patients (OR=0.60, 95% CI 0.03-3.17, p=.63) at the time of radical prostatectomy. Additionally, in patients who underwent multiple biopsies before radical prostatectomy, the Gleason Grade group increased over time in AA patients (0.25 per year, p<.001) but remained unchanged in CA patients. ERG expression was associated with a lower Gleason Grade group (-0.20, p=.03), while ETV4 expression was linked to a higher Gleason Grade (0.54, p=.01). Conclusions: Our study reveals that AA with localized prostate cancer behaves differently and has better BCR-free survival than CA after radical prostatectomy, even after adjusting for known prognostic factors. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion within a multifocal prostate shows significant molecular heterogeneity between localized prostate cancer in CA and AA patients. Importantly, given the association of ETV1 and ETV4 expression with worse BCR-free survival in CA and AA, respectively, ETV1 and ETV4 emerge as potential prognostic markers, offering insights for clinical practice to predict prostate cancer recurrence after radical prostatectomy. Identification of new molecular subsets of prostate cancer with more than one ETS gene fusion and SPINK1 in a mutually exclusive pattern indicates the clonal origin of independent tumor foci, which is a rare and unique phenomenon in prostate cancer hitherto unidentified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863b31d824c9eeb2a43dee5a1d15a9728d0743ea" target='_blank'>
              Unveiling Prostate Cancer's Molecular Tapestry: Ethnic Disparities and Prognostic Insights from Whole-Mount Prostatectomy Tissue Analysis
              </a>
            </td>
          <td>
            Wei Zhao, Pin Li, Shannon Carskadon, Sunita Ghosh, Craig G. Rogers, James Peabody, D. Chitale, Mani Menon, Sean Williamson, Nilesh Gupta, N. Palanisamy
          </td>
          <td>2024-07-16</td>
          <td>None</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Transcription factor 3 (TCF3), a pivotal member of the TCF/LEF family, plays a critical role in tumorigenesis. Nonetheless, its impact on the tumor microenvironment (TME) and cancer phenotypes remains elusive. We perform an exhaustive analysis of TCF3 expression, DNA variation profiles, prognostic implications, and associations with the TME and immunological aspects. This study is based on a large-scale pan-cancer cohort, encompassing over 17,000 cancer patients from multiple independent datasets, validated by in vitro assays. Our results show that TCF3/4/7 exhibits differential expression patterns between normal and tumor tissues across pan-cancer analyses. Mutational analysis of TCF3 across diverse cancer types reveals the highest alteration rates in biliary tract cancer. Additionally, mutations and single nucleotide variants in TCF3/4/7 are found to exert varied effects on patient prognosis. Importantly, TCF3 emerges as a robust predictor of survival across all cancer cohorts and among patients receiving immune checkpoint inhibitors. Elevated TCF3 expression is correlated with more aggressive cancer subtypes, as validated by immunohistochemistry and diverse cohort data. Furthermore, TCF3 expression is positively correlated with intratumoral heterogeneity and angiogenesis. In vitro investigations demonstrate that TCF3 is involved in epithelial-mesenchymal transition, migration, invasion, and angiogenesis. These effects are likely mediated through the interaction of TCF3 with the NF-κB/MMP2 pathway, which is modulated by IL-17A in human uveal melanoma MUM2B cells. This study elucidates, for the first time, the significant associations of TCF3 with DNA variation profiles, prognostic outcomes, and the TME in multiple cancer contexts. TCF3 holds promise as a molecular marker for diagnosis and as a potential target for novel therapeutic strategies, particularly in uveal melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bec429a74c0a32bc4300e0227ccff7d9af711e" target='_blank'>
              TCF3 as a multidimensional biomarker: oncogenicity, genomic alterations, and immune landscape in pan-cancer analysis.
              </a>
            </td>
          <td>
            Huiling Nie, Yang Yu, Siqi Zhou, Yue Xu, Xi Chen, Xun Qin, Zhangyu Liu, Jiayu Huang, Hailiang Zhang, Jin Yao, Qin Jiang, Bingbing Wei, Xiaojian Qin
          </td>
          <td>2024-08-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0de2bb42c9d38bb212c4141ac747130ed1d8b87" target='_blank'>
              MYH6 suppresses tumor progression by downregulating KIT expression in human prostate cancer
              </a>
            </td>
          <td>
            Fei Wang, Hua Shen, Kai Li, Yanhong Ding, Jianqing Wang, Jian Sun
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Keloids are defined as a benign dermal fibroproliferative disorder, with excessive fibroblast proliferation, and excessive overproduction of collagen. Although the heterogeneity during keloid development has been extensively studied, the heterogeneity across different skin states is still unclear. So, a global comparison across skin states is needed. In this study, we collected samples from 5 states of skin, including melanoma, cutaneous squamous cell carcinoma, keloid skin, scar skin, and healthy control samples. The heterogeneity of cell types and subtypes was analyzed and compared across 5 states, and we observed significant differences among them. Our results showed a cancer-like fibroblast, which is not in normal samples, may play an important role in antigen processing and presentation. We also noticed that the mesenchymal fibroblast increased in keloid samples, which highly expressed POSTN. And POSTN may participate in epithelial-mesenchymal transition and collagen overexpression to promote keloid growth. These findings help to understand the alteration among different skin states and provide potential genetic basis for keloid therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab6fcb2479ef9bf96031a9f84c5fa5956988e58f" target='_blank'>
              The fibroblast heterogeneity across keloid, normal and tumor samples from single-cell resolution.
              </a>
            </td>
          <td>
            Xia Wang, Weihu Huang, Yongtie Li, Chen Zhu
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Heterogeneity can impact biomarker identification. Thus, we investigated the somatic copy number alterations (SCNAs) of individual tumor cells in the vitreous humor of a retinoblastoma patient using single-cell whole-genome profiling and explored the genomic concordance among vitreous and aqueous humor, vitreous seeds, and tumor.


METHODS
Aqueous humor (AH), vitreous humor (VH), and tumor biopsy were obtained from an enucleated globe with retinoblastoma and vitreous seeding. Micromanipulation was used to manually isolate 39 live single tumor cells from vitreous seeds harvested from the VH. The SCNA profiles of these individual cells were generated via whole-genome sequencing and analyzed alongside profiles from the tumor mass and cell-free DNA (cfDNA) from AH and VH.


RESULTS
Heatmap of VH single-cell SCNA profiles demonstrates heterogeneity among individual vitreous seeds with one clearly dominant subclone (23 of 37 cells). The SCNA profiles from the cells in this subclone demonstrate an average concordance of 98% with cfDNA profiles from acellular AH and VH and with the tumor profile.


CONCLUSIONS
Our findings reveal some heterogeneity among single-cell SCNA profiles in individual VH seeds. Despite this heterogeneity, the dominant vitreous subclone exhibits extremely (>98%) high concordance with the SCNA profile from tumor and AH, suggesting AH cfDNA is representative of the dominant genomic subclone. This may facilitate tumoral biomarker identification via the AH. This preliminary work supports the potential of applying single-cell technology to VH seeds in retinoblastoma as a platform to study tumor subclones, which may provide insight into the genomic complexity of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b2d1d4be3422b57718cb912ac9759429e36257" target='_blank'>
              Single-cell somatic copy number alteration profiling of vitreous humor seeds in retinoblastoma.
              </a>
            </td>
          <td>
            Shreya Sirivolu, Michael J. Schmidt, Rishvanth K. Prabakar, Peter Kuhn, James B. Hicks, Jesse L Berry, Liya Xu
          </td>
          <td>2024-07-17</td>
          <td>Ophthalmic genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Microglia, the innate immune cells in the central nervous system, exhibit distinct transcriptional profiles across brain regions that are important for facilitating their specialized function. There has been recent interest in identifying the epigenetic modifications associated with these distinct transcriptional profiles, as these may improve our understanding of the underlying mechanisms governing the functional specialization of microglia. One obstacle to achieving this goal is the large number of microglia required to obtain a genome-wide profile for a single histone modification. Given the cellular and regional heterogeneity of the brain, this would require pooling many samples which would impede biological applications that are limited by numbers of available animals. To overcome this obstacle, we have adapted a method of chromatin profiling known as Cleavage Under Targets and Tagmentation (CUT&Tag-Direct) to profile histone modifications associated with regional differences in gene expression throughout the brain reward system. Consistent with previous studies, we find that transcriptional profiles of microglia vary by brain region. However, here we report that these regional differences also exhibit transcriptional network signatures specific to each region. Additionally, we find that these region-dependent network signatures are associated with differential deposition of H3K27ac and H3K7me3, and while the H3K27me3 landscape is remarkably stable across brain regions, the H3K27ac landscape is most consistent with the anatomical location of microglia which explain their distinct transcriptional profiles. Altogether, these findings underscore the established role of H3K27me3 in cell fate determination and support the active role of H3K27ac in the dynamic regulation of microglial gene expression. In this study, we report a molecular and computational framework that can be applied to improve our understanding of the role of epigenetic regulation in microglia in both health and disease, using as few as 2,500 cells per histone mark. Figure 1. Pipeline of tissue processing and data analysis for the characterization of the microglial transcriptome and epigenome on a regional scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/419c0ef2ba145f51cc94c8365f393a5229e804d5" target='_blank'>
              Epigenetic heterogeneity shapes the transcriptional landscape of regional microglia
              </a>
            </td>
          <td>
            Alexander V. Margetts, Samara J. Vilca, Florence Bourgain-Guglielmetti, Luis M. Tuesta
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e722e4e423764d35e29b66ac1408c4a29aa4fd" target='_blank'>
              Targeting SOX4/PCK2 signaling suppresses neuroendocrine trans-differentiation of castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Nan Jing, Zhenkeke Tao, Xinxing Du, Zhenzhen Wen, Wei-Qiang Gao, Baijun Dong, Yu-Xiang Fang
          </td>
          <td>2024-07-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Brain metastases (BrMs) constitute the most common type of brain cancer with abysmal prognosis. Given the high incidence and poor prognosis of BrM, understanding the tumor microenvironment (TME) architecture across BrM subtypes is increasingly crucial, especially with the emergence of novel immunotherapeutic agents like immune checkpoint inhibitors, which have demonstrated efficacy in melanoma and lung cancer BrM. Tertiary lymphoid structures (TLS) are spatially organized lymphoid aggregates in the TME that mediate antitumor immunity. Despite being established as prognostic and predictive markers in various cancers, their significance in BrM lacks extensive investigation. We performed gene expression profiling using paired-end RNA sequencing and multiplex immunofluorescence combined with spatial imaging on 149 human BrM samples. Cell type deconvolution analysis revealed that BrMs from non-small cell lung carcinoma (NSCLC) and malignant melanoma exhibited significantly higher immune cell infiltration compared to BrMs from breast carcinoma. Furthermore, B cell infiltration was detected in some patients, suggesting potential TLS formation. We created a metagene signature comprising TLS hallmarks in BrMs using transcriptomic profiling, allowing us to categorize BrM patients into TLS-positive and TLS-negative groups. This finding was corroborated by multiplex immunofluorescence staining of corresponding BrM tissue sections, which confirmed TLS-like aggregate formation in 40% of the BrM patients. Differential gene expression analysis highlighted the presence of activated lymphocyte-mediated immunity as well as increased immunoglobulin production signature in TLS-positive tumors. Additionally, a higher TLS signature score was associated with improved overall survival of lung carcinoma patients after BrM diagnosis. Our results underscore the heterogeneity of the tumor microenvironment in BrM, and the identification of intratumoral TLS may guide patient stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f904325c9c40fd220a1c52e4c6b7474d7dea43" target='_blank'>
              BMRK-09 PROGNOSTIC ROLE OF TUMOR-INFILTRATING LYMPHOCYTES AND TERTIARY LYMPHOID STRUCTURES IN BRAIN METASTASIS
              </a>
            </td>
          <td>
            S. S. Mughal, Y. Reiss, J. Felsberg, L. Meyer, J. Macas, S. Schlue, Tatjana Starzetz, K. Köhrer, Tanja N Fehm, Volkmar Müller, K. Lamszus, Iris Helfrich, H. Wikman, A. Berghoff, B. Brors, Karlheinz Plate, G. Reifenberger
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Background Radiotherapy (RT) is the primary treatment for diffuse midline glioma (DMG), a lethal pediatric malignancy defined by histone H3 lysine 27-to-methionine (H3K27M) mutation. Based on the loss of H3K27 trimethylation producing broad epigenomic alterations, we hypothesized that H3K27M causes a functional double-strand break (DSB) repair defect that could be leveraged therapeutically with PARP inhibitor and RT for selective radiosensitization and antitumor immune responses. Methods H3K27M isogenic DMG cells and orthotopic brainstem DMG tumors in immune deficient and syngeneic, immune competent mice were used to evaluate the efficacy and mechanisms of PARP1/2 inhibition by olaparib or PARP1 inhibition by AZD9574 with concurrent RT. Results H3K27M mutation caused an HRR defect characterized by impaired RT-induced K63-linked polyubiquitination of histone H1 and inhibition of HRR protein recruitment. H3K27M DMG cells were selectively radiosensitized by olaparib in comparison to isogenic controls, and this effect translated to efficacy in H3K27M orthotopic brainstem tumors. Olaparib and RT induced an innate immune response and induction of NK cell (NKG2D) activating ligands leading to increased NK cell-mediated lysis of DMG tumor cells. In immunocompetent syngeneic orthotopic DMG tumors, either olaparib or AZD9574 in combination with RT enhanced intratumoral NK cell infiltration and activity in association with NK cell-mediated therapeutic responses and favorable activity of AZD9574. Conclusions The HRR deficiency in H3K27M DMG can be therapeutically leveraged with PARP inhibitors to radiosensitize and induce an NK cell-mediated antitumor immune response selectively in H3K27M DMG, supporting the clinical investigation of best-in-class PARP inhibitors with RT in DMG patients. Key points H3K27M DMG are HRR defective and selectively radiosensitized by PARP inhibitor. PARP inhibitor with RT enhances NKG2D ligand expression and NK cell-mediated lysis. NK cells are required for the therapeutic efficacy of PARP inhibitor and RT. Importance of the Study Radiotherapy is the cornerstone of H3K27M-mutant diffuse midline glioma treatment, but almost all patients succumb to tumor recurrence with poor overall survival, underscoring the need for RT-based precision combination therapy. Here, we reveal HRR deficiency as an H3K27M-mediated vulnerability and identify a novel mechanism linking impaired RT-induced histone H1 polyubiquitination and the subsequent RNF168/BRCA1/RAD51 recruitment in H3K27M DMG. This model is supported by selective radiosensitization of H3K27M DMG by PARP inhibitor. Notably, the combination treatment results in NKG2D ligand expression that confers susceptibility to NK cell killing in H3K27M DMG. We also show that the novel brain penetrant, PARP1-selective inhibitor AZD9574 compares favorably to olaparib when combined with RT, prolonging survival in a syngeneic orthotopic model of H3K27M DMG. This study highlights the ability of PARP1 inhibition to radiosensitize and induce an NK cell-mediated antitumor immunity in H3K27M DMG and supports future clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa409894637aca4cc204fe5ba28199f9da5c66db" target='_blank'>
              H3K27M diffuse midline glioma is homologous recombination defective and sensitized to radiotherapy and NK cell-mediated antitumor immunity by PARP inhibition
              </a>
            </td>
          <td>
            Yupei Guo, Zian Li, L. A. Parsels, Zhuwen Wang, J. Parsels, Anushka Dalvi, Stephanie The, Nan Hu, Victoria M. Valvo, Robert Doherty, Erik Peterson, Xinjun Wang, S. Venkataraman, S. Agnihotri, S. Venneti, Daniel R. Wahl, Michael D. Green, Theodore S. Lawrence, C. Koschmann, Meredith A. Morgan, Qiang Zhang
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cbf2aead1d0ad8473a6756261dd1579d6d4208b" target='_blank'>
              The potential of cancer stem cells for personalized risk assessment and therapeutic intervention in individuals with intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Jian Zhang, Tao Cui, Jiaobang Xu, Peng Wang, Chongqing Lv, Guozheng Pan
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bladder cancer prognosis remains a pressing clinical challenge, necessitating the identification of novel biomarkers for precise survival prediction and improved quality of life outcomes. This study proposes a comprehensive strategy to uncover key prognostic biomarkers in bladder cancer using DNA methylation analysis and extreme survival pattern observations in matched pairs of cancer and adjacent normal cells. Unlike traditional approaches that overlook cancer heterogeneity by analyzing entire samples, our methodology leverages patient-matched samples to account for this variability. Specifically, DNA methylation profiles from adjacent normal bladder tissue and bladder cancer tissue collected from the same individuals were analyzed to pinpoint critical methylation changes specific to cancer cells while mitigating confounding effects from individual genetic differences. Utilizing differential threshold settings for methylation levels within cancer-associated pathways enabled the identification of biomarkers that significantly impact patient survival. Our analysis identified distinct survival patterns associated with specific CpG sites, underscoring these sites’ pivotal roles in bladder cancer outcomes. By hypothesizing and testing the influence of methylation levels on survival, we pinpointed CpG biomarkers that profoundly affect the prognosis. Notably, CpG markers, such as cg16269144 (PRKCZ), cg16624272 (PTK2), cg11304234, and cg26534425 (IL18), exhibited critical methylation thresholds that correlate with patient mortality. This study emphasizes the importance of tailored approaches to enhancing prognostic accuracy and refining therapeutic strategies for bladder cancer patients. The identified biomarkers pave the way for personalized prognostication and targeted interventions, promising advancements in bladder cancer management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96750ad297c98f5a6f2185dd55161203f8047d69" target='_blank'>
              Unveiling Bladder Cancer Prognostic Insights by Integrating Patient-Matched Sample and CpG Methylation Analysis
              </a>
            </td>
          <td>
            Chanbyeol Kim, Sangwon Oh, Hamin Im, Jungsoo Gim
          </td>
          <td>2024-07-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Immune checkpoint-inhibitors have become a standard treatment for clear cell renal cell carcinoma (ccRCC). However, responses to checkpoint-inhibition are heterogeneous and many patients are resistant or eventually develop resistance. A better understanding of functional mechanisms governing immune responses in ccRCC is needed to identify biomarkers for therapeutic responses and to unravel novel therapeutic targets to overcome primary or acquired resistance to therapies. Methods We developed a fully human explant model derived from freshly resected ccRCC tissue (Fig. 1A). Our tissue explant model preserves all components of the tumor microenvironment including not only tumor cells, but also the immune cell compartment as well as tumor-stroma and -vasculature in culture (Fig. 1B). Following treatment with combined checkpoint-inhibition (Nivolumab plus Ipilimumab) or an anti-CCR5 inhibitor (Maraviroc), the tissue explants were analyzed by immunohistochemistry staining and multiplex cytokine profiling of 50 cytokines. Spatial and functional insights were incorporated in an agent-based in silico model (PhysiCell, Ghaffarizadeh et al., PLoS Comput. Biol., 2018) for unlimited exploration of functional cellular dynamics in the tumor microenvironment (Fig. 1D). Results Immunohistochemical analyses and cytokine profiling of ccRCC-tissue explants showed heterogeneous immune responses among different patients and indicated an impaired cytotoxic T cell response following checkpoint-inhibition (Fig. 1C). Spatial analyses of immune cell populations revealed clusters of CD8+ T cells and CD163+ macrophages localized closely to CD31+ endothelial cells. Further characterization of the tumor microenvironment revealed high CCR5-expression in the tumor, particularly on the tumor blood vessels (Fig. 1B). Treatment of the tissue explants with the anti-CCR5 inhibitor Maraviroc led to an increase of CD8+ T cells and cytotoxic cytokines (Granzyme B, IFNgamma, TNFalpha, IFNalpha2) in the tumor in comparison to checkpoint-inhibition (Fig. 1C). Spatial and functional information from cultured ccRCC-tissue explants were integrated into an agent-based in silico model of the ccRCC tumor microenvironment, which includes tumor cells, T cells, macrophages and endothelial cells (Fig. 1D). In silico simulation of different immune cell modulating treatment conditions showed an increase of T cells and cytotoxic cytokines upon blockade of the interaction between T cells with endothelial cells or macrophages (Fig. 1E). Conclusions Our combined ex vivo and in silico analyses provide evidence for immunosuppression in the ccRCC tumor microenvironment mediated by macrophages and endothelial cells. CCR5 was shown to be a potential target to overcome immune resistance to checkpoint-inhibition in ccRCC. Further characterization of immune cell compositions and functional mechanisms in the perivascular area is imperative to enhance therapeutic responses to immunotherapies for ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78af278232c241a08a9645b6c5f42c4fea1d9f55" target='_blank'>
              30 Synergistic systematic analysis with fully human tissue models andin silico modeling of “copycats” reveals mechanisms of T cell suppression in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Xin-Wen Zhang, Alexandra Pöchmann, Xu Holtkotte, M. Suarez-Carmona, Prof Stefan Duensing, S. Zschäbitz, Prof Dirk Jäger, Prof Matthias Gaida, Prof Niels Halama
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="While often represented as static entities, gene networks are highly context-dependent. Here, we developed a multi-task learning strategy to yield context-specific representations of gene network dynamics. We assembled a corpus comprising ∼103 million human single-cell transcriptomes from a broad range of tissues and diseases and performed a two stage pretraining, first with non-malignant cells to generate a foundational model and then with continual learning on cancer cells to tune the model to the cancer domain. We performed multi-task learning with the foundational model to learn context-specific representations of a broad range of cell types, tissues, developmental stages, and diseases. We then leveraged the cancer-tuned model to jointly learn cell states and predict tumor-restricting factors within the colorectal tumor microenvironment. Model quantization allowed resource-efficient fine-tuning and inference while preserving biological knowledge. Overall, multi-task learning enables context-specific disease modeling that can yield contextual predictions of candidate therapeutic targets for human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9913f0b488b5e3e74870d991a16be130e495ac98" target='_blank'>
              Quantized multi-task learning for context-specific representations of gene network dynamics
              </a>
            </td>
          <td>
            Han Chen, Madhavan S. Venkatesh, Javier Gómez Ortega, Siddharth V. Mahesh, T. Nandi, R. Madduri, Karin Pelka, Christina V. Theodoris
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 22],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>